<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="76C1AB06747931C306AB060015EF9BCF.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="biomol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Biomolecules">
<meta name="citation_title" content="Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma">
<meta name="citation_author" content="Kankamol Kerdkumthong">
<meta name="citation_author_institution" content="Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)">
<meta name="citation_author" content="Sittiruk Roytrakul">
<meta name="citation_author_institution" content="Functional Proteomics Technology Laboratory, National Center for Genetic Engineering and Biotechnology, National Science and Technology Development Agency, Pathum Tani 12120, Thailand; sittiruk@biotec.or.th">
<meta name="citation_author" content="Kawinnath Songsurin">
<meta name="citation_author_institution" content="Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)">
<meta name="citation_author" content="Kandawasri Pratummanee">
<meta name="citation_author_institution" content="Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)">
<meta name="citation_author" content="Phanthipha Runsaeng">
<meta name="citation_author_institution" content="Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)">
<meta name="citation_author_institution" content="Center of Excellence for Biochemistry, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand">
<meta name="citation_author" content="Sumalee Obchoei">
<meta name="citation_author_institution" content="Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)">
<meta name="citation_author_institution" content="Center of Excellence for Biochemistry, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand">
<meta name="citation_publication_date" content="2024 Aug 8">
<meta name="citation_volume" content="14">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="969">
<meta name="citation_doi" content="10.3390/biom14080969">
<meta name="citation_pmid" content="39199357">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/pdf/biomolecules-14-00969.pdf">
<meta name="description" content="Drug resistance is a major challenge in the treatment of advanced cholangiocarcinoma (CCA). Understanding the mechanisms of drug resistance can aid in identifying novel prognostic biomarkers and therapeutic targets to improve treatment efficacy. ...">
<meta name="og:title" content="Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Drug resistance is a major challenge in the treatment of advanced cholangiocarcinoma (CCA). Understanding the mechanisms of drug resistance can aid in identifying novel prognostic biomarkers and therapeutic targets to improve treatment efficacy. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11352417">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.3390/biom14080969" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/biomolecules-14-00969.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11352417%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/11352417/" data-citation-style="nlm" data-download-format-link="/resources/citations/11352417/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-biomol.png" alt="Biomolecules logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Biomolecules" title="Link to Biomolecules" shape="default" href="http://www.mdpi.com/journal/biomolecules" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Biomolecules</button></div>. 2024 Aug 8;14(8):969. doi: <a href="https://doi.org/10.3390/biom14080969" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.3390/biom14080969</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Biomolecules%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Biomolecules%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Biomolecules%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Biomolecules%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kerdkumthong%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Kankamol Kerdkumthong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kankamol Kerdkumthong</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div>Conceptualization, Methodology, Formal analysis, Investigation, Data curation, Writing – original draft, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kerdkumthong%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kankamol Kerdkumthong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roytrakul%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Sittiruk Roytrakul</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sittiruk Roytrakul</span></h3>
<div class="p">
<sup>2</sup>Functional Proteomics Technology Laboratory, National Center for Genetic Engineering and Biotechnology, National Science and Technology Development Agency, Pathum Tani 12120, Thailand; sittiruk@biotec.or.th</div>
<div>Validation, Formal analysis, Investigation, Resources, Data curation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roytrakul%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sittiruk Roytrakul</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Songsurin%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Kawinnath Songsurin</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kawinnath Songsurin</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div>Investigation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Songsurin%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kawinnath Songsurin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pratummanee%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Kandawasri Pratummanee</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kandawasri Pratummanee</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div>Investigation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pratummanee%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kandawasri Pratummanee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Runsaeng%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Phanthipha Runsaeng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Phanthipha Runsaeng</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div class="p">
<sup>3</sup>Center of Excellence for Biochemistry, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand</div>
<div>Validation, Resources, Data curation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Runsaeng%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Phanthipha Runsaeng</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Obchoei%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Sumalee Obchoei</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sumalee Obchoei</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div class="p">
<sup>3</sup>Center of Excellence for Biochemistry, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand</div>
<div>Conceptualization, Methodology, Validation, Resources, Data curation, Writing – original draft, Writing – review &amp; editing, Funding acquisition</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Obchoei%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sumalee Obchoei</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Frank Krause</span>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="af1-biomolecules-14-00969">
<sup>1</sup>Department of Biochemistry, Division of Health and Applied Sciences, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand; kkerdkumthong@gmail.com (K.K.); kawinnath.ss@gmail.com (K.S.); kandawasri1122@gmail.com (K.P.); phanthipha.r@psu.ac.th (P.R.)</div>
<div id="af2-biomolecules-14-00969">
<sup>2</sup>Functional Proteomics Technology Laboratory, National Center for Genetic Engineering and Biotechnology, National Science and Technology Development Agency, Pathum Tani 12120, Thailand; sittiruk@biotec.or.th</div>
<div id="af3-biomolecules-14-00969">
<sup>3</sup>Center of Excellence for Biochemistry, Faculty of Science, Prince of Songkla University, Hat Yai District, Songkhla 90110, Thailand</div>
<div class="author-notes p"><div class="fn" id="c1-biomolecules-14-00969">
<sup>*</sup><p class="display-inline">Correspondence: <span>sumalee.o@psu.ac.th</span>; Tel.: +66-74-288-270</p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Kankamol Kerdkumthong</span></strong>: <span class="role">Conceptualization, Methodology, Formal analysis, Investigation, Data curation, Writing – original draft, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sittiruk Roytrakul</span></strong>: <span class="role">Validation, Formal analysis, Investigation, Resources, Data curation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kawinnath Songsurin</span></strong>: <span class="role">Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kandawasri Pratummanee</span></strong>: <span class="role">Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Phanthipha Runsaeng</span></strong>: <span class="role">Validation, Resources, Data curation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sumalee Obchoei</span></strong>: <span class="role">Conceptualization, Methodology, Validation, Resources, Data curation, Writing – original draft, Writing – review &amp; editing, Funding acquisition</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Frank Krause</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 2; Revised 2024 Jul 21; Accepted 2024 Aug 5; Collection date 2024 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2024 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11352417&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/39199357/" class="usa-link">39199357</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2 data-anchor-id="abstract1"><a id="abstract1-anchor" data-anchor-id="abstract1" class="usa-anchor"></a>Abstract</h2>
<p>Drug resistance is a major challenge in the treatment of advanced cholangiocarcinoma (CCA). Understanding the mechanisms of drug resistance can aid in identifying novel prognostic biomarkers and therapeutic targets to improve treatment efficacy. This study established 5-fluorouracil- (5-FU) and gemcitabine-resistant CCA cell lines, KKU-213FR and KKU-213GR, and utilized comparative proteomics to identify differentially expressed proteins in drug-resistant cells compared to parental cells. Additionally, bioinformatics analyses were conducted to explore the biological and clinical significance of key proteins. The drug-resistant phenotypes of KKU-213FR and KKU-213GR cell lines were confirmed. In addition, these cells demonstrated increased migration and invasion abilities. Proteomics analysis identified 81 differentially expressed proteins in drug-resistant cells, primarily related to binding functions, biological regulation, and metabolic processes. Protein–protein interaction analysis revealed a highly interconnected network involving MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1. siRNA-mediated knockdown of these genes in drug-resistant cell lines attenuated cell migration and cell invasion abilities and increased sensitivity to 5-FU and gemcitabine. The mRNA expression of these genes is upregulated in CCA patient samples and is associated with poor prognosis in gastrointestinal cancers. Furthermore, the functions of these proteins are closely related to the epithelial–mesenchymal transition (EMT) pathway. These findings elucidate the potential molecular mechanisms underlying drug resistance and tumor progression in CCA, providing insights into potential therapeutic targets.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> cholangiocarcinoma, quantitative proteomics, drug resistance, 5-fluorouracil, gemcitabine, EMT</p></section></section><section id="sec1-biomolecules-14-00969"><h2 class="pmc_sec_title" data-anchor-id="sec1-biomolecules-14-00969"><a id="sec1-biomolecules-14-00969-anchor" data-anchor-id="sec1-biomolecules-14-00969" class="usa-anchor"></a>1. Introduction</h2>
<p>Cholangiocarcinoma (CCA), a malignancy affecting the bile ducts, is relatively rare on a global scale. However, its prevalence and mortality rates are notably elevated in specific Asian regions, with the highest occurrences observed in Northeast Thailand [<a href="#B1-biomolecules-14-00969" class="usa-link" aria-describedby="B1-biomolecules-14-00969" aria-expanded="false">1</a>]. Due to late-stage diagnosis, most CCA cases are ineligible for curative surgical interventions. Consequently, chemotherapy plays a crucial role in the treatment of these late-diagnosed CCA patients [<a href="#B2-biomolecules-14-00969" class="usa-link" aria-describedby="B2-biomolecules-14-00969" aria-expanded="false">2</a>]. However, the effectiveness of chemotherapy is limited by the acquisition of drug resistance by CCA cells, leading to inevitable tumor recurrence [<a href="#B3-biomolecules-14-00969" class="usa-link" aria-describedby="B3-biomolecules-14-00969" aria-expanded="false">3</a>]. A deeper understanding of the mechanisms underlying drug resistance holds the potential to enhance the effectiveness of CCA treatment. In the field of cancer research, to identify novel drug-resistant mechanisms or investigate the potency of novel treatments for combating drug resistance, drug-resistant cell lines are often employed [<a href="#B4-biomolecules-14-00969" class="usa-link" aria-describedby="B4-biomolecules-14-00969" aria-expanded="false">4</a>,<a href="#B5-biomolecules-14-00969" class="usa-link" aria-describedby="B5-biomolecules-14-00969" aria-expanded="false">5</a>,<a href="#B6-biomolecules-14-00969" class="usa-link" aria-describedby="B6-biomolecules-14-00969" aria-expanded="false">6</a>,<a href="#B7-biomolecules-14-00969" class="usa-link" aria-describedby="B7-biomolecules-14-00969" aria-expanded="false">7</a>].</p>
<p>Proteomics techniques have played a crucial role in advancing our understanding of cancer biology by investigating modifications in signaling pathways within tumor cells. This progress has led to the creation of comprehensive cancer proteome databases, merging molecular and clinical data, and identifying potential therapeutic targets and prognostic markers [<a href="#B8-biomolecules-14-00969" class="usa-link" aria-describedby="B8-biomolecules-14-00969" aria-expanded="false">8</a>,<a href="#B9-biomolecules-14-00969" class="usa-link" aria-describedby="B9-biomolecules-14-00969" aria-expanded="false">9</a>,<a href="#B10-biomolecules-14-00969" class="usa-link" aria-describedby="B10-biomolecules-14-00969" aria-expanded="false">10</a>,<a href="#B11-biomolecules-14-00969" class="usa-link" aria-describedby="B11-biomolecules-14-00969" aria-expanded="false">11</a>]. Recent examples of such revelations have been observed in various cancers including hepatocellular carcinoma [<a href="#B12-biomolecules-14-00969" class="usa-link" aria-describedby="B12-biomolecules-14-00969" aria-expanded="false">12</a>], pancreatic cancer [<a href="#B13-biomolecules-14-00969" class="usa-link" aria-describedby="B13-biomolecules-14-00969" aria-expanded="false">13</a>], glioblastoma [<a href="#B14-biomolecules-14-00969" class="usa-link" aria-describedby="B14-biomolecules-14-00969" aria-expanded="false">14</a>], and lung adenocarcinoma [<a href="#B15-biomolecules-14-00969" class="usa-link" aria-describedby="B15-biomolecules-14-00969" aria-expanded="false">15</a>]. However, to our knowledge, proteomic profiling for the elucidation of chemotherapeutic drug-resistant CCA cells has not yet been undertaken.</p>
<p>In the present study, we established 5-fluorouracil (5-FU) and gemcitabine-resistant cell lines named KKU-213FR and KKU-213GR, respectively, from the KKU-213A parental cell line. Various tumor characteristics such as the drug sensitivity, cell migration, and cell invasion capability of these cells were investigated. Furthermore, quantitative proteomic analysis was conducted to identify differentially expressed proteins in drug-resistant cell lines compared with the parental cell line and to explore the underlying drug-resistant mechanisms. Additionally, various bioinformatic tools were utilized to explore functions and protein–protein interaction networks. The Cancer Genome Atlas Program (TCGA) data were analyzed to investigate the clinical relevance of the key identified proteins.</p></section><section id="sec2-biomolecules-14-00969"><h2 class="pmc_sec_title" data-anchor-id="sec2-biomolecules-14-00969"><a id="sec2-biomolecules-14-00969-anchor" data-anchor-id="sec2-biomolecules-14-00969" class="usa-anchor"></a>2. Materials and Methods</h2>
<section id="sec2dot1-biomolecules-14-00969"><h3 class="pmc_sec_title">2.1. Cell Culture</h3>
<p>The KKU-213A, formerly known as M213, was established from liver fluke-associated intrahepatic mass-forming CCA tissue with an adenosquamous carcinoma nature from a Thai patient [<a href="#B16-biomolecules-14-00969" class="usa-link" aria-describedby="B16-biomolecules-14-00969" aria-expanded="false">16</a>,<a href="#B17-biomolecules-14-00969" class="usa-link" aria-describedby="B17-biomolecules-14-00969" aria-expanded="false">17</a>]. It was obtained from the Japanese Collection of Research Bioresources (JCRB) Cell Bank in Osaka, Japan. The cell line was cultured under standard conditions in a humidified atmosphere with 5% CO<sub>2</sub> at 37 °C using Dulbecco’s Modified Eagle’s Medium (DMEM, Invitrogen, Carlsbad, CA, USA), supplemented with 25 mM glucose, 10% fetal bovine serum (FBS, Gibco, Paisley, Scotland), and 1% antibiotic-antimycotic mixture (Gibco, Grand Island, NY, USA).</p></section><section id="sec2dot2-biomolecules-14-00969"><h3 class="pmc_sec_title">2.2. Establishment of Drug-Resistant CCA Cell Lines</h3>
<p>The chemotherapeutic drugs 5-FU and gemcitabine were purchased from Sigma (Sigma Aldrich, Saint Louis, MO, USA). Drug-resistant cell lines were established using a stepwise concentration incremental method, as previously described [<a href="#B18-biomolecules-14-00969" class="usa-link" aria-describedby="B18-biomolecules-14-00969" aria-expanded="false">18</a>], with necessary modifications. The parental cell line of KKU-213A cells was initially cultured in DMEM containing the IC<sub>25</sub> concentrations of 5-FU and gemcitabine. After four passages, the drug concentration was increased two-fold. Finally, cell lines growing exponentially in the presence of 7 µM 5-FU and 3 µM gemcitabine were designated as KKU-213FR and KKU-213GR, respectively.</p></section><section id="sec2dot3-biomolecules-14-00969"><h3 class="pmc_sec_title">2.3. Chemotherapeutic Drug Sensitivity Assay</h3>
<p>The drug-resistant characteristic of the established KKU-213FR and KKU-213GR cell lines was verified through the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl-2H-tetrazolium bromide (MTT) assay. In this process, both drug-resistant cells and their parental counterparts were seeded at a density of 3 × 10<sup>3</sup> cells/well in a 96-well plate and incubated for 24 h. Subsequently, they were exposed to varying concentrations of 5-FU or gemcitabine for 72 h. Afterward, an MTT solution at a concentration of 0.5 mg/mL was introduced into each well and incubated at 37 °C for 2 h. The resulting formazan crystals were dissolved by replacing the medium with 100 µL of dimethyl sulfoxide (DMSO). The absorbance was measured at 540 nm using a microplate reader, enabling the calculation of cell viability percentages for comparison between the drug-resistant and parental cell lines.</p></section><section id="sec2dot4-biomolecules-14-00969"><h3 class="pmc_sec_title">2.4. Cell Migration and Invasion Assay</h3>
<p>The migration and invasion abilities of cells were evaluated using a modified Boyden chamber technique. In short, 2 × 10<sup>4</sup> cells were placed onto an 8-µm porous polycarbonate membrane in the upper chamber. For the invasion assay, the membrane was pre-coated with 0.5 mg/mL Matrigel (Corning, Bedford, MA, USA). The lower chamber contained a complete medium to encourage cell migration in response to chemoattractants. After 12 h, any non-migrated cells in the upper chamber were removed and the membrane was fixed and stained using a 0.4% sulforhodamine B (SRB) solution. The numbers of migrated and invaded cells were then counted and compared across five randomly chosen low-power fields.</p></section><section id="sec2dot5-biomolecules-14-00969"><h3 class="pmc_sec_title">2.5. Proteomics Analysis Using LC-MS/MS</h3>
<p>Cell pellets from KKU-213A, KKU-213FR, and KKU-213GR were lysed with 0.5% SDS and centrifuged at 10,000× <em>g</em> for 15 min. The resulting supernatant was transferred to a new tube, mixed with two volumes of cold acetone, and incubated overnight at −20 °C. The mixture was thawed and centrifuged again at 10,000× <em>g</em> for 15 min. Following the removal of the supernatant, the pellet was dried and stored at −80 °C until needed. Protein concentration of all samples was determined using the Lowry protein assay, with bovine serum albumin (BSA) as the standard protein. To reduce disulfide bonds, a solution of 10 mM dithiothreitol in 10 mM ammonium bicarbonate was added to the protein solution. To prevent reformation of disulfide bonds, the solution was treated with 30 mM iodoacetamide in 10 mM ammonium bicarbonate. Protein samples were digested with sequencing-grade porcine trypsin (at a ratio of 1:20) for 16 h at 37 °C. Subsequently, 100 ng of digested peptides were injected into an Ultimate3000 Nano/Capillary LC System (Thermo Scientific, Oxford, UK) coupled to an HCTUltra LC-MS system (Bruker Daltonics Ltd., Hamburg, Germany) equipped with a nano-captive spray ion source.</p>
<p>Briefly, the peptide digests were enriched on a μ-Precolumn (300 μm I.D. × 5 mm) C18 PepMap 100 with 5 μm, 100 Å (Thermo Scientific, UK). The peptides were then separated on a 75 μm I.D. × 15 cm column packed with Acclaim PepMap RSLC C18, 2 μm, 100 Å, nanoViper (Thermo Scientific, UK). This column was maintained in a thermostatted column oven at 60 °C. The column was supplied with solvents A and B, which contained 0.1% formic acid in water and 0.1% formic acid in 80% acetonitrile, respectively. Tryptic peptides were eluted from the column using a gradient of 5–55% solvent B at a constant flow rate of 0.30 μL/min for 30 min. Electrospray ionization was performed at 1.6 kV using the CaptiveSpray method with nitrogen as the drying gas at a flow rate of 50 L/h. Mass spectra (MS) and MS/MS spectra were acquired in the positive-ion mode at 2 Hz, covering a range of 150–2200 <em>m</em>/<em>z</em>. The collision energy was set at 10 eV based on the <em>m</em>/<em>z</em> value. MS/MS data-dependent acquisition was enabled. The ion trap was configured for positive-ion mode utilizing the manufacturer-specified standard enhanced mode with a scan rate of 8100 <em>m</em>/z/s. The scan range spanned from <em>m/z</em> 400 to 1500 for MS, averaging five spectra, and accumulated either 100,000 charges [by ion charge control (ICC)] or 200 ms, whichever came first. Collision-induced dissociation (CID) fragmentation was performed on the five most intense ions within the <em>m/z</em> range of 200–2800. The threshold for precursor ion selection was set at an absolute intensity of 5000. To prevent redundancy, strict active exclusion was applied: a precursor ion was excluded after two spectra and then reintroduced after a brief 0.3-min interval. The samples were run in three biological replicates of KKU-213A, KKU-213FR, and KKU-213GR, with each replicate undergoing LC-MS analysis in triplicate. A spike of digested BSA was used as an internal standard.</p>
<p>The DecyderMS 2.0 Differential Analysis software was employed to quantify peptides in individual samples from MS/MS data [<a href="#B19-biomolecules-14-00969" class="usa-link" aria-describedby="B19-biomolecules-14-00969" aria-expanded="false">19</a>,<a href="#B20-biomolecules-14-00969" class="usa-link" aria-describedby="B20-biomolecules-14-00969" aria-expanded="false">20</a>]. It provides novel 2D and 3D visualizations of LC-MS data to allow for raw data quality assessment and interactive confirmation of results obtained through automated methods for peptide detection, charge state assignments, and peptide matching across multiple LC-MS experiments. Univariate statistical tools such as Student’s <em>t</em>-test and ANOVA are available to identify significantly varying peptides among different groups of samples. Briefly, the raw LC-MS/MS data files were imported into the program, which transformed the information into a virtual peptide image. These images were then processed within the Pepdetect module of DecyderMS. The program initially identifies individual peptides by placing a box around them. Subsequently, ion counts for each peptide are integrated, and the Log<sub>2</sub> of this number is reported. The process is repeated for each replicate sample. This resulting data are then uploaded to the Pepmatch module, where peptides within different virtual images are matched based on <em>m</em>/<em>z</em> and retention time. Notably, because the data was acquired in an ion trap mass spectrometer, tandem MS ensures accurate peptide matching (i.e., MS/MS spectra consistency). Within the Pepmatch module, each virtual image is assigned to a specific group (KKU-213A, KKU-213FR, and KKU-213GR). Group-to-group comparisons are performed. Integrated peptide counts serve as the basis for statistical analyses, assessing the probability that observed changes are not due to chance. False discovery rates are determined from the data.</p>
<p>MASCOT software, version 2.2 (Matrix Science, London, UK), was used to correlate MS/MS spectra obtained from DecyderMS software to the Homo sapiens protein database (downloaded on 18 August 2021). The search criteria included taxonomy (Homo sapiens), enzyme (trypsin), variable modifications (oxidation of methionine residues), mass values (monoisotopic), protein mass (unrestricted), peptide mass tolerance (1.0 Da), fragment mass tolerance (±0.4 Da), peptide charge states (1+, 2+, and 3+), and a maximum number of missed cleavages [<a href="#B21-biomolecules-14-00969" class="usa-link" aria-describedby="B21-biomolecules-14-00969" aria-expanded="false">21</a>]. Proteins were identified using one or more peptides with an individual MASCOT score corresponding to <em>p</em> &lt; 0.05 and subsequently annotated by UniProtKB/Swiss-Prot entries (<a href="http://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.uniprot.org/</a>; accessed on 18 August 2021).</p></section><section id="sec2dot6-biomolecules-14-00969"><h3 class="pmc_sec_title">2.6. Identification of Differentially Expressed Proteins</h3>
<p>MetaboAnalyst 6.0 was used for the statistical evaluation of the identified proteins [<a href="#B22-biomolecules-14-00969" class="usa-link" aria-describedby="B22-biomolecules-14-00969" aria-expanded="false">22</a>]. To illustrate the distinction between various groups of identified proteins, partial least squares-discriminant analysis (PLS-DA) was utilized. Differentially expressed proteins (DEPs) were selected if they showed at least a 3-fold significant difference between drug-resistant cell lines and the parental line, with a false discovery rate (FDR) &lt; 0.05.</p>
<p>Venn diagrams were generated to show the total number of upregulated and downregulated proteins. To visualize and compare the Log2 expression values of all DEPs, a heatmap was generated using MetaboAnalyst 6.0 [<a href="#B22-biomolecules-14-00969" class="usa-link" aria-describedby="B22-biomolecules-14-00969" aria-expanded="false">22</a>].</p></section><section id="sec2dot7-biomolecules-14-00969"><h3 class="pmc_sec_title">2.7. Bioinformatics Analysis</h3>
<p>Protein organization and biological action were conducted on differentially expressed proteins using the protein analysis through evolutionary relationships (Panther software version 18.0; <a href="http://pantherdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pantherdb.org/</a>; accessed on 5 December 2022) protein classification [<a href="#B23-biomolecules-14-00969" class="usa-link" aria-describedby="B23-biomolecules-14-00969" aria-expanded="false">23</a>]. The categorization included molecular function, biological process, and protein class.</p>
<p>The gene symbols of all upregulated proteins sharing in both KKU-213FR and KKU-213GR were queried against TCGA data through The University of Alabama at Birmingham Cancer data analysis portal (UALCAN) (<a href="http://ualcan.path.uab.edu/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ualcan.path.uab.edu/index.html</a>; accessed on 12 July 2023) [<a href="#B24-biomolecules-14-00969" class="usa-link" aria-describedby="B24-biomolecules-14-00969" aria-expanded="false">24</a>] to analyze their mRNA expression levels in CCA patients’ tissues. The genes that were upregulated in both drug-resistant cell lines, as indicated by proteomics results, and whose mRNA expression levels were significantly elevated in CCA patients’ tissues according to TCGA data, were filtered and presented in a heatmap. These genes were then subjected to protein–protein interaction prediction and generation using STRING version 11.5 (<a href="https://version-11-5.string-db.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://version-11-5.string-db.org</a>; accessed on 30 July 2023) [<a href="#B25-biomolecules-14-00969" class="usa-link" aria-describedby="B25-biomolecules-14-00969" aria-expanded="false">25</a>]. The following parameters were used: Organism: Homo sapiens; Network type: Full STRING network; Required score: medium confidence (0.4); and FDR stringency: medium (5 percent). The largest protein network, comprising six proteins, was assigned as the ‘focused network’ and further analyzed. The mRNA expression of the six selected genes and the correlations among them were assessed through expression and correlation analysis using the Gene Expression Profiling Interactive Analysis 2 (GEPIA2) platform (<a href="http://gepia2.cancer-pku.cn/#correlation" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia2.cancer-pku.cn/#correlation</a>; accessed on 7 August 2023). The dataset of cholangiocarcinoma (CHOL) and gastrointestinal tract (GI) cancer patients was utilized. The correlation between genes was computed with Pearson correlation coefficients and presented in scatter plots.</p></section><section id="sec2dot8-biomolecules-14-00969"><h3 class="pmc_sec_title">2.8. Survival Analysis</h3>
<p>The prognostic values of the six genes within the focus network were assessed using TCGA data through the GEPIA2 web portal. Overall survival (OS) and disease-free survival (DFS) were analyzed through Kaplan–Meier survival plots and heatmaps (<a href="http://gepia2.cancer-pku.cn/#survival" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia2.cancer-pku.cn/#survival</a>; accessed on 7 August 2023). Heatmap analysis was employed to compare the contributions of genes to survival in gastrointestinal (GI) tract cancers, with estimates generated using the Mantel–Cox test at a significance level of less than 0.05. Survival analysis was performed based on the expression status of genes in the focused network, and a Kaplan–Meier curve was plotted. The Cox proportional hazards model, including a 95% confidence interval (CI), was used to evaluate the significance of genes in predicting OS and DFS.</p></section><section id="sec2dot9-biomolecules-14-00969"><h3 class="pmc_sec_title">2.9. RT-qPCR</h3>
<p>The mRNA levels of the six chosen genes were assessed using RT-qPCR, with the beta-actin gene (ACTB) serving as an internal control. Initially, total RNA was isolated from KKU-213A, KKU-213FR, and KKU-213GR cells, as well as from the drug-resistant cells transfected with siRNAs, using the GF-1 total RNA extraction kit (Vivantis, Malaysia), and then converted into cDNA using the RevertAid First Strand cDNA Synthesis Kit (Thermo Scientific, Vilnius, Lithuania). The resulting cDNA was amplified using gene-specific primers (<a href="#biomolecules-14-00969-t001" class="usa-link">Table 1</a>) and Maxima SYBR Green/ROX qPCR master Mix (Thermo Scientific, Vilnius, Lithuania), following the manufacturer’s instructions. The qPCR analysis was conducted on an Agilent Technologies Stratagene Mx3005P instrument. The relative mRNA expression levels were normalized using the internal control gene and calculated using the 2<sup>−ΔΔCt</sup> method.</p>
<section class="tw xbox font-sm" id="biomolecules-14-00969-t001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Lists of gene-specific primers for RT-qPCR.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Gene</th>
<th colspan="2" align="left" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Sequence (5′ to 3′)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>ACTB</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GGATTCCTATGTGGGCGACG</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">TTGTAGAAGGTGTGGTGCCAG</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>MET</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CGCACAAAGCAAGCCAGATT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">AGTGCTCATGATTGGGTCCG</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>LAMB1</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GGCAATCTGAAAATGGTGTGGA</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ACGAGGCCTCACAGTCATAG</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>ITGA3</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GGGACAGTGATGGGTGAGTC</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GTAGGGCCACTCCAGACCTA</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>NOTCH2</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">AGGTGTCAGAATGGAGGGGT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GCCGTTGACACATACACAGC</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>CDH2</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">TGCAAGACTGGATTTCCTGAAGA</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">AGCTTCTCACGGCATACACC</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" style="border-bottom:solid thin" colspan="1">
<em>NDRG1</em>
</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">For:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ATTGGCATGGGAACAGGAGC</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Rev:</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CATCCTGAGATCTTGGAGGCG</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/biomolecules-14-00969-t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2dot10-biomolecules-14-00969"><h3 class="pmc_sec_title">2.10. Gene Knockdown Using siRNA</h3>
<p>Specific siRNA sequences targeting MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1 mRNA, as well as a non-targeting negative control siRNA (siNC), were obtained from Gene Universal Inc. (Newark, DE, USA) and are shown in <a href="#biomolecules-14-00969-t002" class="usa-link">Table 2</a>. Briefly, for gene knockdown, 2 × 10<sup>5</sup> cells per well were plated in a six-well plate overnight. The next day, the cells were washed and serum-starved in 800 µL per well of Opti-MEM<sup>®</sup> (Gibco, Grand Island, NY, USA) for 1 h. Then, 200 µL of transfection complex containing 100 pmoles siRNA and 2 µL Lipofectamine™ 2000 (Thermo Scientific, Carlsbad, CA, USA) were added to each well. After 6 h of incubation, the medium was changed to complete medium. Twenty-four hours after transfection, the cells were subjected to cell migration, invasion, and drug sensitivity assays, and were harvested for RNA extraction and subsequent RT-qPCR.</p>
<section class="tw xbox font-sm" id="biomolecules-14-00969-t002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>List of siRNAs for gene knockdown.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">siRNA</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Target Gene</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">siRNA Sequence (5′ to 3′)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siMET</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MET</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GGACCGGUUCAUCAACUUCTT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siLAMB1</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">LAMB1</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">AAUGUAACUGCAAUGAACATT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siITGA3</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ITGA3</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GGAAAGGAAACAGCUACAUGATT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siNOTCH2</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NOTCH2</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GAAUUGUCAGACAGUAUUGTT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siCDH2</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CDH2</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">UGACAACAGACCUGAGUUCTT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siNDRG1</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NDRG1</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">GACCACUCUCCUCAAGAUGTT</td>
</tr>
<tr>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">siNC</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">-</td>
<td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">UUCUCCGAACGUGUCACGUTT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/biomolecules-14-00969-t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2dot11-biomolecules-14-00969"><h3 class="pmc_sec_title">2.11. Statistical Analysis</h3>
<p>The results are displayed as mean ± standard deviation (SD). The significance of comparisons between groups was evaluated using the two-tailed Student’s <em>t</em>-test, with a significance level set at <em>p</em> &lt; 0.05. Each experiment was performed with at least three replicates.</p></section></section><section id="sec3-biomolecules-14-00969"><h2 class="pmc_sec_title" data-anchor-id="sec3-biomolecules-14-00969"><a id="sec3-biomolecules-14-00969-anchor" data-anchor-id="sec3-biomolecules-14-00969" class="usa-anchor"></a>3. Results</h2>
<section id="sec3dot1-biomolecules-14-00969"><h3 class="pmc_sec_title">3.1. Drug-Resistant CCA Cell Lines Exhibit Aggressive Behaviors</h3>
<p>We established two chemotherapeutic drug-resistant CCA cell lines, namely KKU-213FR and KKU-213GR, representing the KKU-213A cells resistant to 5-FU and gemcitabine, respectively. These drug-resistant CCA cell lines underwent drug sensitivity assays under various concentrations of 5-FU and gemcitabine. The results confirmed that KKU-213FR cells were resistant to 5-FU, and KKU-213GR cells were resistant to gemcitabine, respectively (<a href="#biomolecules-14-00969-f001" class="usa-link">Figure 1</a>a,b).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/8bbf4df69915/biomolecules-14-00969-g001.jpg" loading="lazy" height="655" width="787" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic study of drug-resistant CCA cell lines. Drug sensitivity assay of KKU-213FR (<strong>a</strong>) and KKU-213GR (<strong>b</strong>). (<strong>c</strong>) Cell migration assay. (<strong>d</strong>) Cell invasion assay: magnification 100×; scale bar, 100 μm. The data are presented as mean ± SD from three replicates. Significant differences between treated versus untreated controls or parental versus drug-resistant cells are indicated by * <em>p</em> &lt; 0.05, ** <em>p</em> &lt; 0.01, and *** <em>p</em> &lt; 0.001.</p></figcaption></figure><p>The Boyden chamber migration assays demonstrated that both drug-resistant cell lines exhibited significantly higher migration capabilities than the parental cell line (<a href="#biomolecules-14-00969-f001" class="usa-link">Figure 1</a>c). Similarly, the invasion assay revealed that both KKU-213FR and KKU-213GR had higher invasion rates compared to their parental counterparts (<a href="#biomolecules-14-00969-f001" class="usa-link">Figure 1</a>d).</p></section><section id="sec3dot2-biomolecules-14-00969"><h3 class="pmc_sec_title">3.2. Comparative Proteomic Analysis of Drug-Resistant CCA Cell Lines</h3>
<p>We employed liquid chromatography with tandem mass spectrometry (LC-MS/MS) to conduct a comparative proteomic analysis of the parental CCA cell line (KKU-213A) and the drug-resistant cell lines (KKU-213FR and KKU-213GR). The proteomics analysis workflow is depicted in <a href="#biomolecules-14-00969-f002" class="usa-link">Figure 2</a>a. In total, 5444, 5894, and 5848 proteins were identified from KKU-213A, KKU-213FR, and KKU-213GR, respectively. The partial least squares-discriminant analysis (PLS-DA) illustrated a good separation of identified proteins among groups (<a href="#biomolecules-14-00969-f002" class="usa-link">Figure 2</a>b). Proteins with expression differences exceeding 3-fold between drug-resistant cell lines and the parental line, with a false discovery rate (FDR) &lt; 0.05, were classified as differentially expressed proteins (DEPs). In KKU-213A vs. KKU-213FR, we identified a total of 279 DEPs, including 137 upregulated and 142 downregulated proteins. In KKU-213A vs. KKU-213GR, we identified 127 DEPs, comprised of 112 upregulated and 15 downregulated proteins. To identify important genes and pathways in these drug-resistant cells, we searched for overlapping DEPs and found 81 DEPs shared by both KKU-213FR and KKU-213GR, with 80 proteins upregulated (<a href="#biomolecules-14-00969-f002" class="usa-link">Figure 2</a>c) and only 1 protein downregulated (<a href="#biomolecules-14-00969-f002" class="usa-link">Figure 2</a>d). The log2 expression values of these proteins in KKU-213A, KKU-213FR, and KKU-213GR were visualized using a heatmap (<a href="#biomolecules-14-00969-f002" class="usa-link">Figure 2</a>e).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/456c85e59ef3/biomolecules-14-00969-g002.jpg" loading="lazy" height="983" width="735" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proteomics analysis of KKU-213A, KKU-213FR, and KKU-213GR cell lines. (<strong>a</strong>) Proteomics analysis workflow. (<strong>b</strong>) Partial least squares-discriminant analysis (PLS-DA) of all identified proteins. Venn diagram of upregulated proteins (<strong>c</strong>) and down-regulated proteins (<strong>d</strong>) in KKU-213FR and KKU-213GR compared to KKU-213A. (<strong>e</strong>) Heatmap with group averages of upregulated (red frame) and down-regulated (green frame) proteins shared among KKU-213A, KKU-213FR, and KKU-213GR.</p></figcaption></figure></section><section id="sec3dot3-biomolecules-14-00969"><h3 class="pmc_sec_title">3.3. Gene Ontology Analysis</h3>
<p>To gain an overall perspective on proteome changes following the development of drug resistance, we subjected the list of DEPs from proteomics data to Gene Ontology (GO) analysis based on the GO database and protein analysis Through Evolutionary Relationships (PANTHER) classification. This allowed us to categorize these proteins based on their molecular function, biological processes, and protein classes (<a href="#biomolecules-14-00969-f003" class="usa-link">Figure 3</a>). As depicted in <a href="#biomolecules-14-00969-f003" class="usa-link">Figure 3</a>a, the largest fraction of the upregulated proteins is associated with the molecular function of binding proteins. Other prominent groups of proteins that exhibited changes include those with catalytic activity, transcriptional regulation, molecular transduction, and transporter activity. Furthermore, the analysis of biological processes revealed that most proteins were involved in cellular processes, followed by metabolic processes, localization, and development processes. Smaller percentages of proteins fell into categories related to responses to stimuli, multicellular organismal processes, signaling, and locomotion (<a href="#biomolecules-14-00969-f003" class="usa-link">Figure 3</a>b). Additionally, these upregulated proteins were mainly classified into categories such as metabolite interconversion enzymes, transcriptional regulators, protein-modifying enzymes, and cytoskeleton proteins (<a href="#biomolecules-14-00969-f003" class="usa-link">Figure 3</a>c).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/f47878f2dd9a/biomolecules-14-00969-g003.jpg" loading="lazy" height="915" width="719" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene Ontology analysis of upregulated proteins shared in KKU-213FR and KKU-213GR. The analysis represents the proteins hits and their respective percentages. Analysis based on molecular functions (<strong>a</strong>), biological processes (<strong>b</strong>), and protein classes (<strong>c</strong>).</p></figcaption></figure></section><section id="sec3dot4-biomolecules-14-00969"><h3 class="pmc_sec_title">3.4. The mRNA Expression of Upregulated Proteins in CCA Patients’ Tissues</h3>
<p>To investigate whether the 80 upregulated proteins shared in both KKU-213FR and KKU-213GR were also upregulated in CCA patients’ tissues compared to their corresponding normal tissue counterparts, we utilized the UALCAN web portal as an analysis tool based on TCGA data. The results reveal that 25 out of 80 genes (<a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>a) were significantly upregulated in CCA patients’ tissues compared with adjacent normal tissues, as shown in the heatmap in <a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>b. These include TRAP1, CUL3, ARIH1, RPN2, NOTCH2, CDH2, TTLL4, PSMD13, DHX9, WDR74, NDRG1, MET, COMMD3, TERF2, TCP1, DDX19B, DMTF1, EGLN1, LAMB1, SNRPG, TMEM115, ITGA3, GRINA, RCOR3, and TJAP1 (<a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>b).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/43ae67ce71e0/biomolecules-14-00969-g004.jpg" loading="lazy" height="990" width="744" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of significant genes in CCA. (<strong>a</strong>) The Venn diagram represents 80 upregulated proteins from the proteomic analysis and 25 upregulated genes in CCA tissue compared to normal tissue. (<strong>b</strong>) Heatmap analysis displaying the expression of 25 overlapping genes that were significantly upregulated in CCA patients’ tissues based on TCGA data (left) and proteomics data (right). (<strong>c</strong>) PPI analysis of the 25 upregulated genes via STRING. (<strong>d</strong>) Box plot showing the mRNA expression of six selected genes (a focused network) in CCA patients’ tissues (red box) vs. normal tissues (grey box) based on TCGA data. Significant differences between groups are indicated by * <em>p</em> &lt; 0.05. (<strong>e</strong>) The mRNA expression of six selected genes verified by RT-qPCR in KKU-213A, KKU-213FR, and KKU-213GR. Significant differences between the KKU-213A and KKU-213FR groups, as well as between the KKU-213A and KKU-213GR groups, are indicated by *** <em>p</em> &lt; 0.001.</p></figcaption></figure></section><section id="sec3dot5-biomolecules-14-00969"><h3 class="pmc_sec_title">3.5. Identification of Protein–Protein Interaction Networks</h3>
<p>The 25 upregulated proteins were further investigated for protein–protein interactions (PPI) using STRING analysis. The results are represented in <a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>c. The PPI network contains 23 nodes with 14 edges (versus 6 expected edges), a clustering coefficient of 0.514, an enrichment <em>p</em>-value of 0.00611, and an average node degree of 1.22, revealing the largest protein network that includes MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1. Smaller networks were also identified, including CUL3, COMMD3, ARIH1, and EGLN1. Additionally, connecting proteins such as DDX19B, DHX9, and SNRPG, along with two more connecting proteins, TCP1 and TRAP1, were identified. Subsequent analyses focused on the largest protein network, designated as the ‘focused network’. The mRNA expression of six genes in the focused network was further analyzed in CCA patients’ samples through the GEPIA2 web portal and verified in drug-resistant CCA cell lines using RT-qPCR. The results, presented as box plots in <a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>d, demonstrate that LAMB1, ITGA3, NOTCH2, and NDRG1 exhibit significantly higher mRNA expression in CCA tissues compared to normal tissues, while the difference was not significant for MET and CDH2. Furthermore, the mRNA levels of all six genes were confirmed to be upregulated in both KKU-213GR and KKU-213FR (<a href="#biomolecules-14-00969-f004" class="usa-link">Figure 4</a>e).</p></section><section id="sec3dot6-biomolecules-14-00969"><h3 class="pmc_sec_title">3.6. Assessment of the Prognostic Value of Six Selected Genes</h3>
<p>We evaluated the correlation between the overall survival and disease-free survival of CCA patients and the mRNA expression levels of MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1 using TCGA data through the GEPIA2 web portal. No significant correlations were observed, possibly due to the small sample size of CCA patients’ tissues (<a href="#app1-biomolecules-14-00969" class="usa-link">Figure S1</a>). We further explored the prognostic value of these six proteins in a pooled dataset of GI tract cancer patients, including cholangiocarcinoma (CHOL), colon adenocarcinoma (COAD), esophageal carcinoma (ESCA), liver hepatocellular carcinoma (LIHC), pancreatic adenocarcinoma (PAAD), rectum adenocarcinoma (READ), and stomach adenocarcinoma (STAD). The results are presented in survival maps (<a href="#biomolecules-14-00969-f005" class="usa-link">Figure 5</a>a,b) and Kaplan–Meier plots (<a href="#biomolecules-14-00969-f005" class="usa-link">Figure 5</a>c,d). High expression levels of LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1 are associated with poor overall survival of GI tract cancer patients (<a href="#biomolecules-14-00969-f005" class="usa-link">Figure 5</a>c). Additionally, elevated expressions of ITGA3, NOTCH2, and CDH2 are indicative of a shorter disease-free survival time (<a href="#biomolecules-14-00969-f005" class="usa-link">Figure 5</a>d).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/777d8064544f/biomolecules-14-00969-g005.jpg" loading="lazy" height="899" width="791" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Survival analyses of six selected genes in GI tract cancer patients. The overall survival map (<strong>a</strong>) and disease-free survival map (<strong>b</strong>) are displayed. Red shading indicates poor prognosis, and blue shading indicates good prognostic potential for each gene. The box with a prominent border represents statistical significance. Kaplan–Meier plots are used to represent overall survival (<strong>c</strong>) and disease-free survival (<strong>d</strong>).</p></figcaption></figure></section><section id="sec3dot7-biomolecules-14-00969"><h3 class="pmc_sec_title">3.7. Correlation among Six Genes in the Focused Network</h3>
<p>To further explore whether the connected proteins found in STRING analysis exhibit correlation in patients’ samples, we investigated the correlation of gene expression between connected proteins in the focused network using the Pearson correlation coefficient through the GEPIA2 platform. Except for CDH2 vs. NDRG1, we identified significant positive correlations between 10 gene pairs in GI tract cancer patients’ data, including MET vs. LAMB1, MET vs. ITGA3, MET vs. NOTCH2, MET vs. NDRG1, LAMB1 vs. ITGA3, LAMB1 vs. NOTCH2, LAMB1 vs. NDRG1, ITGA3 vs. NOTCH2, ITGA3 vs. NDRG1, and NOTCH2 vs. NDRG1. On the other hand, four gene pairs, including MET vs. CDH2, LAMB1 vs. CDH2, ITGA3 vs. CDH2, and CDH2 vs. NDRG1, showed negative correlations (<a href="#biomolecules-14-00969-f006" class="usa-link">Figure 6</a>). Additionally, when analyzing the CHOL dataset alone, we found positive correlations between five gene pairs in CCA patients’ data, including MET vs. LAMB1, MET vs. ITGA3, MET vs. NOTCH2, MET vs. CDH2, and NOTCH2 vs. CDH2 (<a href="#app1-biomolecules-14-00969" class="usa-link">Figure S2</a>).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/6d25bc1fe4d2/biomolecules-14-00969-g006.jpg" loading="lazy" height="719" width="766" alt="Figure 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The correlation analysis of six focused genes in GI tract cancers includes. (<strong>a</strong>) Scatter plots depicting the expression correlation between gene pairs in GI tract cancers. (<strong>b</strong>) Summary of the significant correlations among gene pairs in GI tract cancers. Yellow color indicates a positive correlation, blue color indicates a negative correlation, and grey color indicates no significant correlation between genes. Significant correlations between gene pairs are indicated by *** <em>p</em> &lt; 0.001, while <em>NS</em> denotes non-significance.</p></figcaption></figure></section><section id="sec3dot8-biomolecules-14-00969"><h3 class="pmc_sec_title">3.8. siRNA-Mediated Knockdown of Six Selected Genes Attenuates Cell Migration, Cell Invasion, and Reverses Drug-Resistant Phenotypes</h3>
<p>We further confirmed the significance of the six selected genes using a loss-of-function study with siRNA. Gene knockdown efficiency was confirmed with RT-qPCR, and the results showed that siMET, siLAMB1, siITGA3, siNOTCH2, siCDH2, and siNDRG1 could significantly and effectively suppress the mRNA expression of the respective genes (<a href="#biomolecules-14-00969-f007" class="usa-link">Figure 7</a>a). The drug-resistant CCA cells (KKU-213FR and KKU-213GR) transfected with all six gene-specific siRNAs showed reduced cell migration (<a href="#biomolecules-14-00969-f007" class="usa-link">Figure 7</a>b) and cell invasion (<a href="#biomolecules-14-00969-f007" class="usa-link">Figure 7</a>c) capability when compared to those cells transfected with siNC. Moreover, knockdown of these six genes suppressed the drug-resistant phenotype of these cells (<a href="#biomolecules-14-00969-f007" class="usa-link">Figure 7</a>d).</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f007"><h4 class="obj_head">Figure 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/511ed2dfe06f/biomolecules-14-00969-g007.jpg" loading="lazy" height="1001" width="751" alt="Figure 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic study after knockdown of six selected genes in KKU-213FR and KKU-213GR cells. (<strong>a</strong>) mRNA expression after siRNA transfection. (<strong>b</strong>) Cell migration assay. (<strong>c</strong>) Cell invasion assay. (<strong>d</strong>) Drug sensitivity assay. Data are presented as mean ± SD from three replicates. Significant differences between siRNA-transfected cells and siNC-transfected controls are indicated by * <em>p</em> &lt; 0.05, ** <em>p</em> &lt; 0.01, and *** <em>p</em> &lt; 0.001, magnification 100×; scale bar, 100 μm.</p></figcaption></figure></section></section><section id="sec4-biomolecules-14-00969"><h2 class="pmc_sec_title" data-anchor-id="sec4-biomolecules-14-00969"><a id="sec4-biomolecules-14-00969-anchor" data-anchor-id="sec4-biomolecules-14-00969" class="usa-anchor"></a>4. Discussion</h2>
<p>Despite recent advancements in surgical techniques, chemotherapy, and the development of targeted therapies at the molecular level, the primary cause of cancer-related deaths is still the progressive growth of metastases that show resistance to existing treatments [<a href="#B26-biomolecules-14-00969" class="usa-link" aria-describedby="B26-biomolecules-14-00969" aria-expanded="false">26</a>]. Although chemotherapy is a commonly employed approach for managing advanced CCA patients, even with combinations of drugs, the median survival rate remains below one year [<a href="#B27-biomolecules-14-00969" class="usa-link" aria-describedby="B27-biomolecules-14-00969" aria-expanded="false">27</a>]. This challenge arises from the intricate mechanisms of chemoresistance, which typically aid cancer cells in evading the lethal effects of anticancer drugs. Thus, gaining a deeper comprehension of the molecular foundations of drug resistance is crucial for unraveling the complex mechanisms and indicators associated with drug-resistant characteristics.</p>
<p>Previous research has successfully developed cell lines that are resistant to commonly used chemotherapeutic drugs 5-FU and gemcitabine in various cancer types, and these cell lines have become invaluable tools for elucidating drug resistance mechanisms and identifying potential therapeutic targets in cancer research. For instance, 5-FU resistance has been achieved in hepatocellular carcinoma cells (HLF-cell line) [<a href="#B28-biomolecules-14-00969" class="usa-link" aria-describedby="B28-biomolecules-14-00969" aria-expanded="false">28</a>], colorectal cancer cells (HCT-8 and H630) [<a href="#B29-biomolecules-14-00969" class="usa-link" aria-describedby="B29-biomolecules-14-00969" aria-expanded="false">29</a>,<a href="#B30-biomolecules-14-00969" class="usa-link" aria-describedby="B30-biomolecules-14-00969" aria-expanded="false">30</a>], breast cancer cells (T47D) [<a href="#B30-biomolecules-14-00969" class="usa-link" aria-describedby="B30-biomolecules-14-00969" aria-expanded="false">30</a>], triple-negative breast cancer cells (MDA-MB-231) [<a href="#B31-biomolecules-14-00969" class="usa-link" aria-describedby="B31-biomolecules-14-00969" aria-expanded="false">31</a>], and gastric cancer cells (SNU638) [<a href="#B32-biomolecules-14-00969" class="usa-link" aria-describedby="B32-biomolecules-14-00969" aria-expanded="false">32</a>]. In pancreatic cancer, gemcitabine-resistant cell lines have been established, such as SW1990 [<a href="#B33-biomolecules-14-00969" class="usa-link" aria-describedby="B33-biomolecules-14-00969" aria-expanded="false">33</a>], PaCa-2 [<a href="#B34-biomolecules-14-00969" class="usa-link" aria-describedby="B34-biomolecules-14-00969" aria-expanded="false">34</a>], PANC [<a href="#B35-biomolecules-14-00969" class="usa-link" aria-describedby="B35-biomolecules-14-00969" aria-expanded="false">35</a>], and in pancreatic ductal carcinoma cell lines [<a href="#B36-biomolecules-14-00969" class="usa-link" aria-describedby="B36-biomolecules-14-00969" aria-expanded="false">36</a>]. Additionally, gemcitabine-resistant cell lines have also been developed in CCA, specifically KKU-M139 and KKU-M214 cell lines [<a href="#B18-biomolecules-14-00969" class="usa-link" aria-describedby="B18-biomolecules-14-00969" aria-expanded="false">18</a>]. In this research, we successfully established 5-FU- and gemcitabine-resistant CCA cell lines KKU-213FR and KKU-213GR (<a href="#biomolecules-14-00969-f001" class="usa-link">Figure 1</a>a,b).</p>
<p>The intricate process of drug resistance in cancer cells linked to metastasis-related differentiation signaling involves dynamic changes in both cancer and stromal cells [<a href="#B37-biomolecules-14-00969" class="usa-link" aria-describedby="B37-biomolecules-14-00969" aria-expanded="false">37</a>]. Metastasis encompasses a decrease in tumor cell adhesion receptor expression, promoting heightened cell motility, while invasion entails the disassembly of the extracellular matrix (ECM) [<a href="#B38-biomolecules-14-00969" class="usa-link" aria-describedby="B38-biomolecules-14-00969" aria-expanded="false">38</a>]. Patients with chemotherapeutic resistance in CCA often undergo advanced cancer recurrence with metastasis [<a href="#B39-biomolecules-14-00969" class="usa-link" aria-describedby="B39-biomolecules-14-00969" aria-expanded="false">39</a>]. Subsequently, we explored the metastatic potential of 5-FU- and gemcitabine-resistant CCA cell lines. Further assessment through the Boyden chamber assay demonstrated a higher number of migrated and invaded cells, affirming the heightened metastatic potential of the established drug-resistant cell line (<a href="#biomolecules-14-00969-f001" class="usa-link">Figure 1</a>c,d).</p>
<p>To investigate alterations in cellular mechanisms related to drug resistance, we identified changes in the proteome profiles. A total of 80 proteins were upregulated in both KKU-213FR and KKU-213GR when compared to the parental cell line. These proteins underwent classification using GO analysis, categorizing them based on their molecular function, biological process, and protein class. The upregulated proteins were primarily classified under the molecular function category of ‘binding’ (GO: 0005488). Previous studies have highlighted the significant roles played by various binding proteins in cancer progression and the development of drug resistance. These proteins function in diverse ways such as transmitting signals through signaling molecules [<a href="#B40-biomolecules-14-00969" class="usa-link" aria-describedby="B40-biomolecules-14-00969" aria-expanded="false">40</a>,<a href="#B41-biomolecules-14-00969" class="usa-link" aria-describedby="B41-biomolecules-14-00969" aria-expanded="false">41</a>], altering the number or mutation of receptors [<a href="#B42-biomolecules-14-00969" class="usa-link" aria-describedby="B42-biomolecules-14-00969" aria-expanded="false">42</a>,<a href="#B43-biomolecules-14-00969" class="usa-link" aria-describedby="B43-biomolecules-14-00969" aria-expanded="false">43</a>,<a href="#B44-biomolecules-14-00969" class="usa-link" aria-describedby="B44-biomolecules-14-00969" aria-expanded="false">44</a>], and releasing cytokines [<a href="#B45-biomolecules-14-00969" class="usa-link" aria-describedby="B45-biomolecules-14-00969" aria-expanded="false">45</a>,<a href="#B46-biomolecules-14-00969" class="usa-link" aria-describedby="B46-biomolecules-14-00969" aria-expanded="false">46</a>].</p>
<p>Drug resistance mechanisms involve alterations in various cellular processes such as increased cell replication [<a href="#B47-biomolecules-14-00969" class="usa-link" aria-describedby="B47-biomolecules-14-00969" aria-expanded="false">47</a>,<a href="#B48-biomolecules-14-00969" class="usa-link" aria-describedby="B48-biomolecules-14-00969" aria-expanded="false">48</a>], transcriptional activation of drug-resistant genes or oncogenes [<a href="#B49-biomolecules-14-00969" class="usa-link" aria-describedby="B49-biomolecules-14-00969" aria-expanded="false">49</a>,<a href="#B50-biomolecules-14-00969" class="usa-link" aria-describedby="B50-biomolecules-14-00969" aria-expanded="false">50</a>], and DNA repair in response to exposure to chemotherapeutic drugs to prevent cell death [<a href="#B51-biomolecules-14-00969" class="usa-link" aria-describedby="B51-biomolecules-14-00969" aria-expanded="false">51</a>,<a href="#B52-biomolecules-14-00969" class="usa-link" aria-describedby="B52-biomolecules-14-00969" aria-expanded="false">52</a>,<a href="#B53-biomolecules-14-00969" class="usa-link" aria-describedby="B53-biomolecules-14-00969" aria-expanded="false">53</a>].</p>
<p>The second largest group of proteins was categorized under biological regulation and metabolic processes. In response to chemotherapeutic agents, cancer cells that possess the ability to evade cell death caused by the drugs can alter their biological processes to sustain growth or proliferation [<a href="#B54-biomolecules-14-00969" class="usa-link" aria-describedby="B54-biomolecules-14-00969" aria-expanded="false">54</a>,<a href="#B55-biomolecules-14-00969" class="usa-link" aria-describedby="B55-biomolecules-14-00969" aria-expanded="false">55</a>], migrate to other parts of the body by reducing cell adhesion [<a href="#B56-biomolecules-14-00969" class="usa-link" aria-describedby="B56-biomolecules-14-00969" aria-expanded="false">56</a>,<a href="#B57-biomolecules-14-00969" class="usa-link" aria-describedby="B57-biomolecules-14-00969" aria-expanded="false">57</a>], and secrete enzymes or signals to modulate the extracellular matrix [<a href="#B58-biomolecules-14-00969" class="usa-link" aria-describedby="B58-biomolecules-14-00969" aria-expanded="false">58</a>,<a href="#B59-biomolecules-14-00969" class="usa-link" aria-describedby="B59-biomolecules-14-00969" aria-expanded="false">59</a>].</p>
<p>When categorized via protein class, the majority of upregulated proteins were classified as metabolite interconversion enzymes. Metabolic reprogramming of cancer cells can hinder immune responses and create obstacles to cancer treatment. Cancer cells must balance increasing metabolic demands associated with uncontrolled cell division and metastasis, which often render them resistant to current therapies. Several studies have discovered abnormalities in metabolism in chemotherapy-resistant cancers [<a href="#B60-biomolecules-14-00969" class="usa-link" aria-describedby="B60-biomolecules-14-00969" aria-expanded="false">60</a>,<a href="#B61-biomolecules-14-00969" class="usa-link" aria-describedby="B61-biomolecules-14-00969" aria-expanded="false">61</a>,<a href="#B62-biomolecules-14-00969" class="usa-link" aria-describedby="B62-biomolecules-14-00969" aria-expanded="false">62</a>].</p>
<p>We further compared upregulated proteins from the proteomic data to their mRNA expression in patients’ tissues from the TCGA database through the UALCAN platform [<a href="#B24-biomolecules-14-00969" class="usa-link" aria-describedby="B24-biomolecules-14-00969" aria-expanded="false">24</a>]. Among the 80 upregulated proteins identified in drug-resistant cell lines based on proteomics analysis, only 25 were found to be upregulated in cholangiocarcinoma (CCA) tissues compared to normal tissues when analyzed using TCGA data through the UALCAN web portal. This discrepancy may be attributed to the absence of comparative gene expression data between chemosensitive and chemoresistant CCA patient samples within the CHOL dataset. The 25 overlapping genes may play a crucial role in both drug resistance and carcinogenesis in CCA, whereas the remaining 56 non-overlapping genes may solely contribute to the development of drug resistance. Future research involving the analysis of these 80 upregulated genes in drug-sensitive and drug-resistant clinical samples will further elucidate their roles in drug-resistant CCA. When subjected to STRING analysis, we identified a highly interconnected network comprising six proteins: MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1. It is notable that a medium confidence level (0.4) was used as a parameter in the STRING analysis. If high confidence (0.7) was used, the protein network would remain largely similar, except for the exclusion of NOTCH2 and NDRG1. To avoid excluding significant genes, the medium confidence value was utilized. Nearly all of the proteins in this network, except for MET, were associated with the poor prognosis in GI tract cancer patients (<a href="#biomolecules-14-00969-f005" class="usa-link">Figure 5</a>). Although statistical significance was not found when analyzing CCA data alone, it was likely due to the small sample size. Furthermore, the mRNA expression in patients’ samples of genes in the focused network was positively correlated with each other (<a href="#biomolecules-14-00969-f006" class="usa-link">Figure 6</a>). Interestingly, the silencing of these six genes reduces cell motility rates and attenuates the chemotherapeutic-resistant phenotype of both KKU-213FR and KKU-213GR (<a href="#biomolecules-14-00969-f007" class="usa-link">Figure 7</a>).</p>
<p>Several past investigations have demonstrated the oncogenic functions of these proteins in different cancer types. The mesenchymal epithelial transition factor (MET), alternatively termed hepatocyte growth factor receptor (HGFR), is a receptor tyrosine kinase. When it interacts with its ligand, hepatocyte growth factor triggers various cellular signaling pathways related to proliferation, motility, migration, and invasion [<a href="#B63-biomolecules-14-00969" class="usa-link" aria-describedby="B63-biomolecules-14-00969" aria-expanded="false">63</a>,<a href="#B64-biomolecules-14-00969" class="usa-link" aria-describedby="B64-biomolecules-14-00969" aria-expanded="false">64</a>].</p>
<p>Laminin beta-1 (LAMB1), a type of extracellular matrix (ECM) glycoprotein, is present in most tissues and is responsible for initiating cell assembly. This assembly is crucial for the invasion and spread of cancer cells [<a href="#B65-biomolecules-14-00969" class="usa-link" aria-describedby="B65-biomolecules-14-00969" aria-expanded="false">65</a>]. Studies have shown that LAMB1 is significantly expressed in multiple invasive cancers [<a href="#B66-biomolecules-14-00969" class="usa-link" aria-describedby="B66-biomolecules-14-00969" aria-expanded="false">66</a>] and plays a significant role in cancer progression, as observed in gastric cancer [<a href="#B67-biomolecules-14-00969" class="usa-link" aria-describedby="B67-biomolecules-14-00969" aria-expanded="false">67</a>,<a href="#B68-biomolecules-14-00969" class="usa-link" aria-describedby="B68-biomolecules-14-00969" aria-expanded="false">68</a>], and lung adenocarcinoma [<a href="#B66-biomolecules-14-00969" class="usa-link" aria-describedby="B66-biomolecules-14-00969" aria-expanded="false">66</a>].</p>
<p>Integrin alpha-3 (ITGA3) is a cell membrane-bound integrin protein receptor that plays a role within the ECM. Its function involves serving as a cell surface adhesion molecule and engaging with various ECM-receptor proteins such as integrin beta-1 (ITGB1), members of the laminin family, and fibronectin 1 [<a href="#B69-biomolecules-14-00969" class="usa-link" aria-describedby="B69-biomolecules-14-00969" aria-expanded="false">69</a>].</p>
<p>Neurogenic locus notch homolog protein 2 (NOTCH2) is a product of the <em>NOTCH2</em> gene. It plays dual roles in cell interactions and serves as a highly conserved signal transduction system once its ligand binds. Overexpression of NOTCH2 has been identified in multiple cancer types including breast cancer [<a href="#B70-biomolecules-14-00969" class="usa-link" aria-describedby="B70-biomolecules-14-00969" aria-expanded="false">70</a>], lung squamous cell carcinoma [<a href="#B71-biomolecules-14-00969" class="usa-link" aria-describedby="B71-biomolecules-14-00969" aria-expanded="false">71</a>], leukemia [<a href="#B72-biomolecules-14-00969" class="usa-link" aria-describedby="B72-biomolecules-14-00969" aria-expanded="false">72</a>], ovarian cancer [<a href="#B73-biomolecules-14-00969" class="usa-link" aria-describedby="B73-biomolecules-14-00969" aria-expanded="false">73</a>], and colorectal cancer [<a href="#B74-biomolecules-14-00969" class="usa-link" aria-describedby="B74-biomolecules-14-00969" aria-expanded="false">74</a>]. The activation of NOTCH2 is linked to various mechanisms in tumor development such as regulating the properties of tumor-initiating cells, controlling signaling pathways like MYC [<a href="#B75-biomolecules-14-00969" class="usa-link" aria-describedby="B75-biomolecules-14-00969" aria-expanded="false">75</a>] or P53 [<a href="#B76-biomolecules-14-00969" class="usa-link" aria-describedby="B76-biomolecules-14-00969" aria-expanded="false">76</a>,<a href="#B77-biomolecules-14-00969" class="usa-link" aria-describedby="B77-biomolecules-14-00969" aria-expanded="false">77</a>], promoting angiogenesis [<a href="#B78-biomolecules-14-00969" class="usa-link" aria-describedby="B78-biomolecules-14-00969" aria-expanded="false">78</a>], regulating tumor invasion [<a href="#B79-biomolecules-14-00969" class="usa-link" aria-describedby="B79-biomolecules-14-00969" aria-expanded="false">79</a>,<a href="#B80-biomolecules-14-00969" class="usa-link" aria-describedby="B80-biomolecules-14-00969" aria-expanded="false">80</a>], and managing the cell cycle [<a href="#B81-biomolecules-14-00969" class="usa-link" aria-describedby="B81-biomolecules-14-00969" aria-expanded="false">81</a>,<a href="#B82-biomolecules-14-00969" class="usa-link" aria-describedby="B82-biomolecules-14-00969" aria-expanded="false">82</a>].</p>
<p>CDH2, also known as neuronal cadherin (N-cadherin), is a marker of epithelial-to-mesenchymal transition (EMT). CDH2 is referred to as a ‘mesenchymal cadherin’, which replaces epithelial cadherin (E-cadherin) during the epithelial-to-mesenchymal transition known as the cadherin ‘switch’ [<a href="#B83-biomolecules-14-00969" class="usa-link" aria-describedby="B83-biomolecules-14-00969" aria-expanded="false">83</a>,<a href="#B84-biomolecules-14-00969" class="usa-link" aria-describedby="B84-biomolecules-14-00969" aria-expanded="false">84</a>]. CDH2’s functions are associated with fibroblast growth factor receptors (FGFRs). This interaction leads to continuous FGFR signaling and the progression of carcinoma [<a href="#B85-biomolecules-14-00969" class="usa-link" aria-describedby="B85-biomolecules-14-00969" aria-expanded="false">85</a>]. Additionally, CDH2 enhances the activity of nuclear β-catenin-mediated drug resistance in myeloma [<a href="#B86-biomolecules-14-00969" class="usa-link" aria-describedby="B86-biomolecules-14-00969" aria-expanded="false">86</a>]. The roles of CDH2 have been demonstrated in several cancers such as increasing invasiveness in melanoma cancer [<a href="#B87-biomolecules-14-00969" class="usa-link" aria-describedby="B87-biomolecules-14-00969" aria-expanded="false">87</a>] and esophageal squamous cell carcinomas [<a href="#B88-biomolecules-14-00969" class="usa-link" aria-describedby="B88-biomolecules-14-00969" aria-expanded="false">88</a>].</p>
<p>N-myc downregulated gene-1 (NDRG1) is known to have diverse roles, functioning both as a suppressor of metastasis and as an indicator of poor prognosis, while also contributing to disease progression across various cancer types [<a href="#B89-biomolecules-14-00969" class="usa-link" aria-describedby="B89-biomolecules-14-00969" aria-expanded="false">89</a>]. Although NDRG1 is primarily recognized for its anti-oncogenic and anti-metastatic functions [<a href="#B90-biomolecules-14-00969" class="usa-link" aria-describedby="B90-biomolecules-14-00969" aria-expanded="false">90</a>,<a href="#B91-biomolecules-14-00969" class="usa-link" aria-describedby="B91-biomolecules-14-00969" aria-expanded="false">91</a>,<a href="#B92-biomolecules-14-00969" class="usa-link" aria-describedby="B92-biomolecules-14-00969" aria-expanded="false">92</a>], studies have revealed its pro-oncogenic role in certain cancers like gastric cancer and hepatocellular carcinoma [<a href="#B93-biomolecules-14-00969" class="usa-link" aria-describedby="B93-biomolecules-14-00969" aria-expanded="false">93</a>]. In clinical samples of esophageal cancer, elevated levels of NDRG1 were associated with the malignant advancement of the disease [<a href="#B94-biomolecules-14-00969" class="usa-link" aria-describedby="B94-biomolecules-14-00969" aria-expanded="false">94</a>]. This study proposed that NDRG1 influences the Wnt signaling pathway and the accumulation of β-catenin through the mediation of Wnt-associated genes, thereby promoting metastasis [<a href="#B83-biomolecules-14-00969" class="usa-link" aria-describedby="B83-biomolecules-14-00969" aria-expanded="false">83</a>]. Indeed, reducing NDRG1 levels in gastric cancer cells led to increased E-cadherin expression and decreased vimentin expression, indicating a link between high NDRG1 levels and the metastatic potential of gastric cancer cells [<a href="#B95-biomolecules-14-00969" class="usa-link" aria-describedby="B95-biomolecules-14-00969" aria-expanded="false">95</a>]. Overall, the role of NDRG1 in either promoting or suppressing tumor progression is highly dependent on the specific type of tumor cell and its degree of differentiation.</p>
<p>Taken together, MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1 were upregulated in drug-resistant CCA cell lines and in CCA patients’ tissues. These genes not only contribute to acquired drug resistance but also play a crucial role in the progression of CCA cells. They share a common role in activating tumor metastasis through EMT pathways. <a href="#biomolecules-14-00969-f008" class="usa-link">Figure 8</a> provides a schematic picture illustrating the predicted functional role of these proteins in the development of drug resistance and progression of CCA.</p>
<figure class="fig xbox font-sm" id="biomolecules-14-00969-f008"><h3 class="obj_head">Figure 8.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11352417_biomolecules-14-00969-g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/568c/11352417/3309b0340476/biomolecules-14-00969-g008.jpg" loading="lazy" height="515" width="720" alt="Figure 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/biomolecules-14-00969-f008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hypothetical role of six selected genes in CCA progression. A schematic illustration delineating the putative role of the selected genes, elucidating their hypothesized involvement in driving drug resistance and promoting the progression of CCA cells. This figure was constructed using BioRender (2023).</p></figcaption></figure></section><section id="sec5-biomolecules-14-00969"><h2 class="pmc_sec_title" data-anchor-id="sec5-biomolecules-14-00969"><a id="sec5-biomolecules-14-00969-anchor" data-anchor-id="sec5-biomolecules-14-00969" class="usa-anchor"></a>5. Conclusions</h2>
<p>Despite advancements in cancer therapies, metastasis-driven drug resistance poses a formidable challenge leading to high mortality rates. Our study successfully established 5-FU- and gemcitabine-resistant CCA cell lines KKU-213FR and KKU-213GR, which exhibited increased cell motility and invasion. Through the integration of proteomics analysis and bioinformatics, we identified 25 genes that were upregulated in drug-resistant cell lines as well as in patients’ tissues. Further investigation revealed a network of six highly interconnected genes (MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1) whose expressions are correlated with each other. Furthermore, the mRNA expression of these genes is associated with a poor prognosis in GI tract cancer patients, emphasizing their potential as prognostic markers and possibly therapeutic targets.</p></section><section id="app1-biomolecules-14-00969" class="app"><h2 class="pmc_sec_title" data-anchor-id="app1-biomolecules-14-00969"><a id="app1-biomolecules-14-00969-anchor" data-anchor-id="app1-biomolecules-14-00969" class="usa-anchor"></a>Supplementary Materials</h2>
<p>The following supporting information can be downloaded at: <a href="https://www.mdpi.com/article/10.3390/biom14080969/s1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mdpi.com/article/10.3390/biom14080969/s1</a>, Figure S1. Survival analyses of six selected genes in CCA patients; Figure S2. The correlation analysis of six focused genes in CCA cancers.</p>
<section class="sm xbox font-sm" id="biomolecules-14-00969-s001"><div class="media p"><div class="caption">
<a href="/articles/instance/11352417/bin/biomolecules-14-00969-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">biomolecules-14-00969-s001.zip</a><sup> (1.6MB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title" data-anchor-id="notes1"><a id="notes1-anchor" data-anchor-id="notes1" class="usa-anchor"></a>Author Contributions</h2>
<p>Conceptualization, K.K. and S.O.; data curation, K.K., S.R., P.R. and S.O.; formal analysis, K.K. and S.R.; funding acquisition, S.O., investigation, K.K., S.R., K.S. and K.P.; methodology, K.K. and S.O.; resources, S.R., P.R. and S.O.; validation, S.R., P.R. and S.O.; visualization, K.K., K.S. and K.P.; writing–original draft and editing, K.K. and S.O.; writing–review &amp; editing, S.O. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title" data-anchor-id="notes2"><a id="notes2-anchor" data-anchor-id="notes2" class="usa-anchor"></a>Institutional Review Board Statement</h2>
<p>Not applicable.</p></section><section id="notes3"><h2 class="pmc_sec_title" data-anchor-id="notes3"><a id="notes3-anchor" data-anchor-id="notes3" class="usa-anchor"></a>Informed Consent Statement</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title" data-anchor-id="notes4"><a id="notes4-anchor" data-anchor-id="notes4" class="usa-anchor"></a>Data Availability Statement</h2>
<p>All data generated or analyzed during this study are included in this published article and its <a href="#app1-biomolecules-14-00969" class="usa-link">Supplementary Information Files</a>. The MS/MS raw data and analysis have been deposited in the ProteomeXchange Consortium via the jPOST partner repository with the data set identifier JPST002506 and PXD049309 (<a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309</a>, accessed on 29 July 2024).</p></section><section id="notes5"><h2 class="pmc_sec_title" data-anchor-id="notes5"><a id="notes5-anchor" data-anchor-id="notes5" class="usa-anchor"></a>Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title" data-anchor-id="funding-statement1"><a id="funding-statement1-anchor" data-anchor-id="funding-statement1" class="usa-anchor"></a>Funding Statement</h2>
<p>This research was supported by the National Science, Research and Innovation Fund (NSRF) and Prince of Songkla University (Ref. No. SCI6701131S) to Sumalee Obchoei. Kankamol Kerdkumthong was supported by a Prince of Songkla University—Ph.D. Scholarship (PSU_PHD2561-001). Kandawasri Pratummanee was supported by Graduate Fellowship (Research Assistant), Faculty of Science, Prince of Songkhla University, Contract no. 1-2565-02-023.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Disclaimer/Publisher’s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></div></div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-biomolecules-14-00969">
<span class="label">1.</span><cite>Banales J.M., Marin J.J., Lamarca A., Rodrigues P.M., Khan S.A., Roberts L.R., Cardinale V., Carpino G., Andersen J.B., Braconi C. Cholangiocarcinoma 2020: The next horizon in mechanisms and management. Nat. Rev. Gastroenterol. Hepatol. 2020;17:557–588. doi: 10.1038/s41575-020-0310-z.</cite> [<a href="https://doi.org/10.1038/s41575-020-0310-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7447603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32606456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Gastroenterol.%20Hepatol.&amp;title=Cholangiocarcinoma%202020:%20The%20next%20horizon%20in%20mechanisms%20and%20management&amp;author=J.M.%20Banales&amp;author=J.J.%20Marin&amp;author=A.%20Lamarca&amp;author=P.M.%20Rodrigues&amp;author=S.A.%20Khan&amp;volume=17&amp;publication_year=2020&amp;pages=557-588&amp;pmid=32606456&amp;doi=10.1038/s41575-020-0310-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2-biomolecules-14-00969">
<span class="label">2.</span><cite>Tawarungruang C., Khuntikeo N., Chamadol N., Laopaiboon V., Thuanman J., Thinkhamrop K., Kelly M., Thinkhamrop B. Survival after surgery among patients with cholangiocarcinoma in Northeast Thailand according to anatomical and morphological classification. BMC Cancer. 2021;21:497.  doi: 10.1186/s12885-021-08247-z.</cite> [<a href="https://doi.org/10.1186/s12885-021-08247-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8094526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33941120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&amp;title=Survival%20after%20surgery%20among%20patients%20with%20cholangiocarcinoma%20in%20Northeast%20Thailand%20according%20to%20anatomical%20and%20morphological%20classification&amp;author=C.%20Tawarungruang&amp;author=N.%20Khuntikeo&amp;author=N.%20Chamadol&amp;author=V.%20Laopaiboon&amp;author=J.%20Thuanman&amp;volume=21&amp;publication_year=2021&amp;pages=497&amp;pmid=33941120&amp;doi=10.1186/s12885-021-08247-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3-biomolecules-14-00969">
<span class="label">3.</span><cite>Marin J.J., Lozano E., Herraez E., Asensio M., Di Giacomo S., Romero M.R., Briz O., Serrano M.A., Efferth T., Macias R.I. Chemoresistance and chemosensitization in cholangiocarcinoma. Biochim. Biophys. Acta Mol. Basis Dis. 2018;1864:1444–1453. doi: 10.1016/j.bbadis.2017.06.005.</cite> [<a href="https://doi.org/10.1016/j.bbadis.2017.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28600147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Mol.%20Basis%20Dis.&amp;title=Chemoresistance%20and%20chemosensitization%20in%20cholangiocarcinoma&amp;author=J.J.%20Marin&amp;author=E.%20Lozano&amp;author=E.%20Herraez&amp;author=M.%20Asensio&amp;author=S.%20Di%20Giacomo&amp;volume=1864&amp;publication_year=2018&amp;pages=1444-1453&amp;pmid=28600147&amp;doi=10.1016/j.bbadis.2017.06.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4-biomolecules-14-00969">
<span class="label">4.</span><cite>Kamangar F., Dores G.M., Anderson W.F. Patterns of cancer incidence, mortality, and prevalence across five continents: Defining priorities to reduce cancer disparities in different geographic regions of the world. J. Clin. Oncol. 2006;24:2137–2150. doi: 10.1200/JCO.2005.05.2308.</cite> [<a href="https://doi.org/10.1200/JCO.2005.05.2308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16682732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&amp;title=Patterns%20of%20cancer%20incidence,%20mortality,%20and%20prevalence%20across%20five%20continents:%20Defining%20priorities%20to%20reduce%20cancer%20disparities%20in%20different%20geographic%20regions%20of%20the%20world&amp;author=F.%20Kamangar&amp;author=G.M.%20Dores&amp;author=W.F.%20Anderson&amp;volume=24&amp;publication_year=2006&amp;pages=2137-2150&amp;pmid=16682732&amp;doi=10.1200/JCO.2005.05.2308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5-biomolecules-14-00969">
<span class="label">5.</span><cite>Govindan S.V., Kulsum S., Pandian R.S., Das D., Seshadri M., Hicks W., Kuriakose M.A., Suresh A. Establishment and characterization of triple drug resistant head and neck squamous cell carcinoma cell lines. Mol. Med. Rep. 2015;12:3025–3032. doi: 10.3892/mmr.2015.3768.</cite> [<a href="https://doi.org/10.3892/mmr.2015.3768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25962396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.%20Rep.&amp;title=Establishment%20and%20characterization%20of%20triple%20drug%20resistant%20head%20and%20neck%20squamous%20cell%20carcinoma%20cell%20lines&amp;author=S.V.%20Govindan&amp;author=S.%20Kulsum&amp;author=R.S.%20Pandian&amp;author=D.%20Das&amp;author=M.%20Seshadri&amp;volume=12&amp;publication_year=2015&amp;pages=3025-3032&amp;pmid=25962396&amp;doi=10.3892/mmr.2015.3768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6-biomolecules-14-00969">
<span class="label">6.</span><cite>Tan Y., Li J., Zhao G., Huang K.-C., Cardenas H., Wang Y., Matei D., Cheng J.-X. Metabolic reprogramming from glycolysis to fatty acid uptake and beta-oxidation in platinum-resistant cancer cells. Nat. Commun. 2022;13:4554. doi: 10.1038/s41467-022-32101-w.</cite> [<a href="https://doi.org/10.1038/s41467-022-32101-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9356138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35931676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Metabolic%20reprogramming%20from%20glycolysis%20to%20fatty%20acid%20uptake%20and%20beta-oxidation%20in%20platinum-resistant%20cancer%20cells&amp;author=Y.%20Tan&amp;author=J.%20Li&amp;author=G.%20Zhao&amp;author=K.-C.%20Huang&amp;author=H.%20Cardenas&amp;volume=13&amp;publication_year=2022&amp;pages=4554&amp;pmid=35931676&amp;doi=10.1038/s41467-022-32101-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7-biomolecules-14-00969">
<span class="label">7.</span><cite>Zuo Q., Liu J., Zhang J., Wu M., Guo L., Liao W. Development of trastuzumab-resistant human gastric carcinoma cell lines and mechanisms of drug resistance. Sci. Rep. 2015;5:11634.  doi: 10.1038/srep11634.</cite> [<a href="https://doi.org/10.1038/srep11634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4479993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26108989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Development%20of%20trastuzumab-resistant%20human%20gastric%20carcinoma%20cell%20lines%20and%20mechanisms%20of%20drug%20resistance&amp;author=Q.%20Zuo&amp;author=J.%20Liu&amp;author=J.%20Zhang&amp;author=M.%20Wu&amp;author=L.%20Guo&amp;volume=5&amp;publication_year=2015&amp;pages=11634&amp;pmid=26108989&amp;doi=10.1038/srep11634&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8-biomolecules-14-00969">
<span class="label">8.</span><cite>Posadas E., Simpkins F., Liotta L., MacDonald C., Kohn E. Proteomic analysis for the early detection and rational treatment of cancer—Realistic hope? Ann. Oncol. 2005;16:16–22. doi: 10.1093/annonc/mdi004.</cite> [<a href="https://doi.org/10.1093/annonc/mdi004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15598930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&amp;title=Proteomic%20analysis%20for%20the%20early%20detection%20and%20rational%20treatment%20of%20cancer%E2%80%94Realistic%20hope?&amp;author=E.%20Posadas&amp;author=F.%20Simpkins&amp;author=L.%20Liotta&amp;author=C.%20MacDonald&amp;author=E.%20Kohn&amp;volume=16&amp;publication_year=2005&amp;pages=16-22&amp;pmid=15598930&amp;doi=10.1093/annonc/mdi004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9-biomolecules-14-00969">
<span class="label">9.</span><cite>Nanjundan M., Byers L.A., Carey M.S., Siwak D.R., Raso M.G., Diao L., Wang J., Coombes K.R., Roth J.A., Mills G.B. Proteomic profiling identifies pathways dysregulated in non-small cell lung cancer and an inverse association of AMPK and adhesion pathways with recurrence. J. Thorac. Oncol. 2010;5:1894–1904. doi: 10.1097/JTO.0b013e3181f2a266.</cite> [<a href="https://doi.org/10.1097/JTO.0b013e3181f2a266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3374718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21124077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&amp;title=Proteomic%20profiling%20identifies%20pathways%20dysregulated%20in%20non-small%20cell%20lung%20cancer%20and%20an%20inverse%20association%20of%20AMPK%20and%20adhesion%20pathways%20with%20recurrence&amp;author=M.%20Nanjundan&amp;author=L.A.%20Byers&amp;author=M.S.%20Carey&amp;author=D.R.%20Siwak&amp;author=M.G.%20Raso&amp;volume=5&amp;publication_year=2010&amp;pages=1894-1904&amp;pmid=21124077&amp;doi=10.1097/JTO.0b013e3181f2a266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10-biomolecules-14-00969">
<span class="label">10.</span><cite>Sahukar Shruthi B., Vinodhkumar P., Amani S. Proteomics: A new perspective for cancer. Adv. Biomed. Res. 2016;2016:67.  doi: 10.4103/2277-9175.180636.</cite> [<a href="https://doi.org/10.4103/2277-9175.180636" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4854039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27169098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Biomed.%20Res.&amp;title=Proteomics:%20A%20new%20perspective%20for%20cancer&amp;author=B.%20Sahukar%20Shruthi&amp;author=P.%20Vinodhkumar&amp;author=S.%20Amani&amp;volume=2016&amp;publication_year=2016&amp;pages=67&amp;pmid=27169098&amp;doi=10.4103/2277-9175.180636&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11-biomolecules-14-00969">
<span class="label">11.</span><cite>Chang L., Ni J., Beretov J., Wasinger V.C., Hao J., Bucci J., Malouf D., Gillatt D., Graham P.H., Li Y. Identification of protein biomarkers and signaling pathways associated with prostate cancer radioresistance using label-free LC-MS/MS proteomic approach. Sci. Rep. 2017;7:41834.  doi: 10.1038/srep41834.</cite> [<a href="https://doi.org/10.1038/srep41834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5320484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28225015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Identification%20of%20protein%20biomarkers%20and%20signaling%20pathways%20associated%20with%20prostate%20cancer%20radioresistance%20using%20label-free%20LC-MS/MS%20proteomic%20approach&amp;author=L.%20Chang&amp;author=J.%20Ni&amp;author=J.%20Beretov&amp;author=V.C.%20Wasinger&amp;author=J.%20Hao&amp;volume=7&amp;publication_year=2017&amp;pages=41834&amp;pmid=28225015&amp;doi=10.1038/srep41834&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12-biomolecules-14-00969">
<span class="label">12.</span><cite>Gao Q., Zhu H., Dong L., Shi W., Chen R., Song Z., Huang C., Li J., Dong X., Zhou Y. Integrated proteogenomic characterization of HBV-related hepatocellular carcinoma. Cell. 2019;179:561–577.e22. doi: 10.1016/j.cell.2019.08.052.</cite> [<a href="https://doi.org/10.1016/j.cell.2019.08.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31585088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated%20proteogenomic%20characterization%20of%20HBV-related%20hepatocellular%20carcinoma&amp;author=Q.%20Gao&amp;author=H.%20Zhu&amp;author=L.%20Dong&amp;author=W.%20Shi&amp;author=R.%20Chen&amp;volume=179&amp;publication_year=2019&amp;pages=561-577.e22&amp;pmid=31585088&amp;doi=10.1016/j.cell.2019.08.052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13-biomolecules-14-00969">
<span class="label">13.</span><cite>Brandi J., Dando I., Dalla Pozza E., Biondani G., Jenkins R., Elliott V., Park K., Fanelli G., Zolla L., Costello E. Proteomic analysis of pancreatic cancer stem cells: Functional role of fatty acid synthesis and mevalonate pathways. J. Proteomics. 2017;150:310–322. doi: 10.1016/j.jprot.2016.10.002.</cite> [<a href="https://doi.org/10.1016/j.jprot.2016.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27746256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Proteomic%20analysis%20of%20pancreatic%20cancer%20stem%20cells:%20Functional%20role%20of%20fatty%20acid%20synthesis%20and%20mevalonate%20pathways&amp;author=J.%20Brandi&amp;author=I.%20Dando&amp;author=E.%20Dalla%20Pozza&amp;author=G.%20Biondani&amp;author=R.%20Jenkins&amp;volume=150&amp;publication_year=2017&amp;pages=310-322&amp;pmid=27746256&amp;doi=10.1016/j.jprot.2016.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14-biomolecules-14-00969">
<span class="label">14.</span><cite>Gupta M.K., Polisetty R.V., Sharma R., Ganesh R.A., Gowda H., Purohit A.K., Ankathi P., Prasad K., Mariswamappa K., Lakshmikantha A. Altered transcriptional regulatory proteins in glioblastoma and YBX1 as a potential regulator of tumor invasion. Sci. Rep. 2019;9:10986.  doi: 10.1038/s41598-019-47360-9.</cite> [<a href="https://doi.org/10.1038/s41598-019-47360-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6662741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31358880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Altered%20transcriptional%20regulatory%20proteins%20in%20glioblastoma%20and%20YBX1%20as%20a%20potential%20regulator%20of%20tumor%20invasion&amp;author=M.K.%20Gupta&amp;author=R.V.%20Polisetty&amp;author=R.%20Sharma&amp;author=R.A.%20Ganesh&amp;author=H.%20Gowda&amp;volume=9&amp;publication_year=2019&amp;pages=10986&amp;pmid=31358880&amp;doi=10.1038/s41598-019-47360-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15-biomolecules-14-00969">
<span class="label">15.</span><cite>Lignitto L., LeBoeuf S.E., Homer H., Jiang S., Askenazi M., Karakousi T.R., Pass H.I., Bhutkar A.J., Tsirigos A., Ueberheide B. Nrf2 activation promotes lung cancer metastasis by inhibiting the degradation of Bach1. Cell. 2019;178:316–329.e18. doi: 10.1016/j.cell.2019.06.003.</cite> [<a href="https://doi.org/10.1016/j.cell.2019.06.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6625921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31257023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Nrf2%20activation%20promotes%20lung%20cancer%20metastasis%20by%20inhibiting%20the%20degradation%20of%20Bach1&amp;author=L.%20Lignitto&amp;author=S.E.%20LeBoeuf&amp;author=H.%20Homer&amp;author=S.%20Jiang&amp;author=M.%20Askenazi&amp;volume=178&amp;publication_year=2019&amp;pages=316-329.e18&amp;pmid=31257023&amp;doi=10.1016/j.cell.2019.06.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16-biomolecules-14-00969">
<span class="label">16.</span><cite>Sripa B., Leungwattanawanit S., Nitta T., Wongkham C., Bhudhisawasdi V., Puapairoj A., Sripa C., Miwa M. Establishment and characterization of an opisthorchiasis-associated cholangiocarcinoma cell line (KKU-100) World J. Gastroenterol. 2005;11:3392. doi: 10.3748/wjg.v11.i22.3392.</cite> [<a href="https://doi.org/10.3748/wjg.v11.i22.3392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4315993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15948244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J.%20Gastroenterol.&amp;title=Establishment%20and%20characterization%20of%20an%20opisthorchiasis-associated%20cholangiocarcinoma%20cell%20line%20(KKU-100)&amp;author=B.%20Sripa&amp;author=S.%20Leungwattanawanit&amp;author=T.%20Nitta&amp;author=C.%20Wongkham&amp;author=V.%20Bhudhisawasdi&amp;volume=11&amp;publication_year=2005&amp;pages=3392&amp;pmid=15948244&amp;doi=10.3748/wjg.v11.i22.3392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17-biomolecules-14-00969">
<span class="label">17.</span><cite>Sripa B., Seubwai W., Vaeteewoottacharn K., Sawanyawisuth K., Silsirivanit A., Kaewkong W., Muisuk K., Dana P., Phoomak C., Lert-Itthiporn W. Functional and genetic characterization of three cell lines derived from a single tumor of an Opisthorchis viverrini- associated cholangiocarcinoma patient. Human Cell. 2020;33:695–708. doi: 10.1007/s13577-020-00334-w.</cite> [<a href="https://doi.org/10.1007/s13577-020-00334-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32207095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Cell&amp;title=Functional%20and%20genetic%20characterization%20of%20three%20cell%20lines%20derived%20from%20a%20single%20tumor%20of%20an%20Opisthorchis%20viverrini-%20associated%20cholangiocarcinoma%20patient&amp;author=B.%20Sripa&amp;author=W.%20Seubwai&amp;author=K.%20Vaeteewoottacharn&amp;author=K.%20Sawanyawisuth&amp;author=A.%20Silsirivanit&amp;volume=33&amp;publication_year=2020&amp;pages=695-708&amp;pmid=32207095&amp;doi=10.1007/s13577-020-00334-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18-biomolecules-14-00969">
<span class="label">18.</span><cite>Wattanawongdon W., Hahnvajanawong C., Namwat N., Kanchanawat S., Boonmars T., Jearanaikoon P., Leelayuwat C., Techasen A., Seubwai W. Establishment and characterization of gemcitabine-resistant human cholangiocarcinoma cell lines with multidrug resistance and enhanced invasiveness. Int. J. Oncol. 2015;47:398–410. doi: 10.3892/ijo.2015.3019.</cite> [<a href="https://doi.org/10.3892/ijo.2015.3019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25998688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Oncol.&amp;title=Establishment%20and%20characterization%20of%20gemcitabine-resistant%20human%20cholangiocarcinoma%20cell%20lines%20with%20multidrug%20resistance%20and%20enhanced%20invasiveness&amp;author=W.%20Wattanawongdon&amp;author=C.%20Hahnvajanawong&amp;author=N.%20Namwat&amp;author=S.%20Kanchanawat&amp;author=T.%20Boonmars&amp;volume=47&amp;publication_year=2015&amp;pages=398-410&amp;pmid=25998688&amp;doi=10.3892/ijo.2015.3019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19-biomolecules-14-00969">
<span class="label">19.</span><cite>Johansson C., Samskog J., Sundström L., Wadensten H., Björkesten L., Flensburg J. Differential expression analysis of Escherichia coli proteins using a novel software for relative quantitation of LC-MS/MS data. Proteomics. 2006;6:4475–4485. doi: 10.1002/pmic.200500921.</cite> [<a href="https://doi.org/10.1002/pmic.200500921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16858737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Differential%20expression%20analysis%20of%20Escherichia%20coli%20proteins%20using%20a%20novel%20software%20for%20relative%20quantitation%20of%20LC-MS/MS%20data&amp;author=C.%20Johansson&amp;author=J.%20Samskog&amp;author=L.%20Sundstr%C3%B6m&amp;author=H.%20Wadensten&amp;author=L.%20Bj%C3%B6rkesten&amp;volume=6&amp;publication_year=2006&amp;pages=4475-4485&amp;pmid=16858737&amp;doi=10.1002/pmic.200500921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20-biomolecules-14-00969">
<span class="label">20.</span><cite>Thorsell A., Portelius E., Blennow K., Westman-Brinkmalm A. Evaluation of sample fractionation using micro-scale liquid-phase isoelectric focusing on mass spectrometric identification and quantitation of proteins in a SILAC experiment. Rapid Commun. Mass Spectrom. 2007;21:771–778. doi: 10.1002/rcm.2898.</cite> [<a href="https://doi.org/10.1002/rcm.2898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17279600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Commun.%20Mass%20Spectrom.&amp;title=Evaluation%20of%20sample%20fractionation%20using%20micro-scale%20liquid-phase%20isoelectric%20focusing%20on%20mass%20spectrometric%20identification%20and%20quantitation%20of%20proteins%20in%20a%20SILAC%20experiment&amp;author=A.%20Thorsell&amp;author=E.%20Portelius&amp;author=K.%20Blennow&amp;author=A.%20Westman-Brinkmalm&amp;volume=21&amp;publication_year=2007&amp;pages=771-778&amp;pmid=17279600&amp;doi=10.1002/rcm.2898&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21-biomolecules-14-00969">
<span class="label">21.</span><cite>Perkins D.N., Pappin D.J., Creasy D.M., Cottrell J.S. Probability-based protein identification by searching sequence databases using mass spectrometry data. ELECTROPHORESIS Int. J. 1999;20:3551–3567. doi: 10.1002/(SICI)1522-2683(19991201)20:18&amp;#x0003c;3551::AID-ELPS3551&amp;#x0003e;3.0.CO;2-2.</cite> [<a href="https://doi.org/10.1002/(SICI)1522-2683(19991201)20:18&amp;#x0003c;3551::AID-ELPS3551&amp;#x0003e;3.0.CO;2-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10612281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ELECTROPHORESIS%20Int.%20J.&amp;title=Probability-based%20protein%20identification%20by%20searching%20sequence%20databases%20using%20mass%20spectrometry%20data&amp;author=D.N.%20Perkins&amp;author=D.J.%20Pappin&amp;author=D.M.%20Creasy&amp;author=J.S.%20Cottrell&amp;volume=20&amp;publication_year=1999&amp;pages=3551-3567&amp;pmid=10612281&amp;doi=10.1002/(SICI)1522-2683(19991201)20:18&amp;#x0003c;3551::AID-ELPS3551&amp;#x0003e;3.0.CO;2-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22-biomolecules-14-00969">
<span class="label">22.</span><cite>Pang Z., Zhou G., Ewald J., Chang L., Hacariz O., Basu N., Xia J. Using MetaboAnalyst 5.0 for LC–HRMS spectra processing, multi-omics integration and covariate adjustment of global metabolomics data. Nat. Protoc. 2022;17:1735–1761. doi: 10.1038/s41596-022-00710-w.</cite> [<a href="https://doi.org/10.1038/s41596-022-00710-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35715522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Using%20MetaboAnalyst%205.0%20for%20LC%E2%80%93HRMS%20spectra%20processing,%20multi-omics%20integration%20and%20covariate%20adjustment%20of%20global%20metabolomics%20data&amp;author=Z.%20Pang&amp;author=G.%20Zhou&amp;author=J.%20Ewald&amp;author=L.%20Chang&amp;author=O.%20Hacariz&amp;volume=17&amp;publication_year=2022&amp;pages=1735-1761&amp;pmid=35715522&amp;doi=10.1038/s41596-022-00710-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23-biomolecules-14-00969">
<span class="label">23.</span><cite>Mi H., Muruganujan A., Ebert D., Huang X., Thomas P.D. PANTHER version 14: More genomes, a new PANTHER GO-slim and improvements in enrichment analysis tools. Nucleic Acids Res. 2019;47:D419–D426. doi: 10.1093/nar/gky1038.</cite> [<a href="https://doi.org/10.1093/nar/gky1038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30407594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PANTHER%20version%2014:%20More%20genomes,%20a%20new%20PANTHER%20GO-slim%20and%20improvements%20in%20enrichment%20analysis%20tools&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=D.%20Ebert&amp;author=X.%20Huang&amp;author=P.D.%20Thomas&amp;volume=47&amp;publication_year=2019&amp;pages=D419-D426&amp;pmid=30407594&amp;doi=10.1093/nar/gky1038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24-biomolecules-14-00969">
<span class="label">24.</span><cite>Chandrashekar D.S., Karthikeyan S.K., Korla P.K., Patel H., Shovon A.R., Athar M., Netto G.J., Qin Z.S., Kumar S., Manne U. UALCAN: An update to the integrated cancer data analysis platform. Neoplasia. 2022;25:18–27. doi: 10.1016/j.neo.2022.01.001.</cite> [<a href="https://doi.org/10.1016/j.neo.2022.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8788199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35078134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;title=UALCAN:%20An%20update%20to%20the%20integrated%20cancer%20data%20analysis%20platform&amp;author=D.S.%20Chandrashekar&amp;author=S.K.%20Karthikeyan&amp;author=P.K.%20Korla&amp;author=H.%20Patel&amp;author=A.R.%20Shovon&amp;volume=25&amp;publication_year=2022&amp;pages=18-27&amp;pmid=35078134&amp;doi=10.1016/j.neo.2022.01.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25-biomolecules-14-00969">
<span class="label">25.</span><cite>Szklarczyk D., Gable A.L., Nastou K.C., Lyon D., Kirsch R., Pyysalo S., Doncheva N.T., Legeay M., Fang T., Bork P. The STRING database in 2021: Customizable protein–protein networks, and functional characterization of user-uploaded gene/measurement sets. Nucleic Acids Res. 2021;49:D605–D612. doi: 10.1093/nar/gkaa1074.</cite> [<a href="https://doi.org/10.1093/nar/gkaa1074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33237311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20STRING%20database%20in%202021:%20Customizable%20protein%E2%80%93protein%20networks,%20and%20functional%20characterization%20of%20user-uploaded%20gene/measurement%20sets&amp;author=D.%20Szklarczyk&amp;author=A.L.%20Gable&amp;author=K.C.%20Nastou&amp;author=D.%20Lyon&amp;author=R.%20Kirsch&amp;volume=49&amp;publication_year=2021&amp;pages=D605-D612&amp;pmid=33237311&amp;doi=10.1093/nar/gkaa1074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26-biomolecules-14-00969">
<span class="label">26.</span><cite>Housman G., Byler S., Heerboth S., Lapinska K., Longacre M., Snyder N., Sarkar S. Drug resistance in cancer: An overview. Cancers. 2014;6:1769–1792. doi: 10.3390/cancers6031769.</cite> [<a href="https://doi.org/10.3390/cancers6031769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4190567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25198391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Drug%20resistance%20in%20cancer:%20An%20overview&amp;author=G.%20Housman&amp;author=S.%20Byler&amp;author=S.%20Heerboth&amp;author=K.%20Lapinska&amp;author=M.%20Longacre&amp;volume=6&amp;publication_year=2014&amp;pages=1769-1792&amp;pmid=25198391&amp;doi=10.3390/cancers6031769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27-biomolecules-14-00969">
<span class="label">27.</span><cite>Thongprasert S. The role of chemotherapy in cholangiocarcinoma. Ann. Oncol. 2005;16:ii93–ii96. doi: 10.1093/annonc/mdi712.</cite> [<a href="https://doi.org/10.1093/annonc/mdi712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15958484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&amp;title=The%20role%20of%20chemotherapy%20in%20cholangiocarcinoma&amp;author=S.%20Thongprasert&amp;volume=16&amp;publication_year=2005&amp;pages=ii93-ii96&amp;pmid=15958484&amp;doi=10.1093/annonc/mdi712&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28-biomolecules-14-00969">
<span class="label">28.</span><cite>Uchibori K., Kasamatsu A., Sunaga M., Yokota S., Sakurada T., Kobayashi E., Yoshikawa M., Uzawa K., Ueda S., Tanzawa H. Establishment and characterization of two 5-fluorouracil-resistant hepatocellular carcinoma cell lines. Int. J. Oncol. 2012;40:1005–1010. doi: 10.3892/ijo.2011.1300.</cite> [<a href="https://doi.org/10.3892/ijo.2011.1300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3584526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22179686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Oncol.&amp;title=Establishment%20and%20characterization%20of%20two%205-fluorouracil-resistant%20hepatocellular%20carcinoma%20cell%20lines&amp;author=K.%20Uchibori&amp;author=A.%20Kasamatsu&amp;author=M.%20Sunaga&amp;author=S.%20Yokota&amp;author=T.%20Sakurada&amp;volume=40&amp;publication_year=2012&amp;pages=1005-1010&amp;pmid=22179686&amp;doi=10.3892/ijo.2011.1300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29-biomolecules-14-00969">
<span class="label">29.</span><cite>He L., Zhu H., Zhou S., Wu T., Wu H., Yang H., Mao H., SekharKathera C., Janardhan A., Edick A.M. Wnt pathway is involved in 5-FU drug resistance of colorectal cancer cells. Exp. Mol. Med. 2018;50:1–12. doi: 10.1038/s12276-018-0128-8.</cite> [<a href="https://doi.org/10.1038/s12276-018-0128-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6093888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30111797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Med.&amp;title=Wnt%20pathway%20is%20involved%20in%205-FU%20drug%20resistance%20of%20colorectal%20cancer%20cells&amp;author=L.%20He&amp;author=H.%20Zhu&amp;author=S.%20Zhou&amp;author=T.%20Wu&amp;author=H.%20Wu&amp;volume=50&amp;publication_year=2018&amp;pages=1-12&amp;pmid=30111797&amp;doi=10.1038/s12276-018-0128-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30-biomolecules-14-00969">
<span class="label">30.</span><cite>Wang W., Cassidy J., O’Brien V., Ryan K.M., Collie-Duguid E. Mechanistic and predictive profiling of 5-Fluorouracil resistance in human cancer cells. Cancer Res. 2004;64:8167–8176. doi: 10.1158/0008-5472.CAN-04-0970.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-04-0970" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15548681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Mechanistic%20and%20predictive%20profiling%20of%205-Fluorouracil%20resistance%20in%20human%20cancer%20cells&amp;author=W.%20Wang&amp;author=J.%20Cassidy&amp;author=V.%20O%E2%80%99Brien&amp;author=K.M.%20Ryan&amp;author=E.%20Collie-Duguid&amp;volume=64&amp;publication_year=2004&amp;pages=8167-8176&amp;pmid=15548681&amp;doi=10.1158/0008-5472.CAN-04-0970&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31-biomolecules-14-00969">
<span class="label">31.</span><cite>Takahashi K., Tanaka M., Inagaki A., Wanibuchi H., Izumi Y., Miura K., Nagayama K., Shiota M., Iwao H. Establishment of a 5-fluorouracil-resistant triple-negative breast cancer cell line. Int. J. Oncol. 2013;43:1985–1991. doi: 10.3892/ijo.2013.2135.</cite> [<a href="https://doi.org/10.3892/ijo.2013.2135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24126575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Oncol.&amp;title=Establishment%20of%20a%205-fluorouracil-resistant%20triple-negative%20breast%20cancer%20cell%20line&amp;author=K.%20Takahashi&amp;author=M.%20Tanaka&amp;author=A.%20Inagaki&amp;author=H.%20Wanibuchi&amp;author=Y.%20Izumi&amp;volume=43&amp;publication_year=2013&amp;pages=1985-1991&amp;pmid=24126575&amp;doi=10.3892/ijo.2013.2135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32-biomolecules-14-00969">
<span class="label">32.</span><cite>Chung Y.M., Park S.-H., Park J.K., Kim Y.-T., Kang Y.-K., Do Yoo Y. Establishment and characterization of 5-fluorouracil-resistant gastric cancer cells. Cancer Lett. 2000;159:95–101. doi: 10.1016/S0304-3835(00)00535-8.</cite> [<a href="https://doi.org/10.1016/S0304-3835(00)00535-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10974411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett.&amp;title=Establishment%20and%20characterization%20of%205-fluorouracil-resistant%20gastric%20cancer%20cells&amp;author=Y.M.%20Chung&amp;author=S.-H.%20Park&amp;author=J.K.%20Park&amp;author=Y.-T.%20Kim&amp;author=Y.-K.%20Kang&amp;volume=159&amp;publication_year=2000&amp;pages=95-101&amp;pmid=10974411&amp;doi=10.1016/S0304-3835(00)00535-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33-biomolecules-14-00969">
<span class="label">33.</span><cite>Yu Y., Ding F., Gao M., Jia Y.F., Ren L. Establishment and characterization of the gemcitabine-resistant human pancreatic cancer cell line SW1990/gemcitabine. Oncol. Lett. 2019;18:3065–3071. doi: 10.3892/ol.2019.10627.</cite> [<a href="https://doi.org/10.3892/ol.2019.10627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6676397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31452783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett.&amp;title=Establishment%20and%20characterization%20of%20the%20gemcitabine-resistant%20human%20pancreatic%20cancer%20cell%20line%20SW1990/gemcitabine&amp;author=Y.%20Yu&amp;author=F.%20Ding&amp;author=M.%20Gao&amp;author=Y.F.%20Jia&amp;author=L.%20Ren&amp;volume=18&amp;publication_year=2019&amp;pages=3065-3071&amp;pmid=31452783&amp;doi=10.3892/ol.2019.10627&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34-biomolecules-14-00969">
<span class="label">34.</span><cite>Togawa A., Ito H., Kimura F., Shimizu H., Ohtsuka M., Shimamura F., Yoshidome H., Katoh A., Miyazaki M. Establishment of gemcitabine-resistant human pancreatic cancer cells and effect of brefeldin-a on the resistant cell line. Pancreas. 2003;27:220–224. doi: 10.1097/00006676-200310000-00004.</cite> [<a href="https://doi.org/10.1097/00006676-200310000-00004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14508125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pancreas&amp;title=Establishment%20of%20gemcitabine-resistant%20human%20pancreatic%20cancer%20cells%20and%20effect%20of%20brefeldin-a%20on%20the%20resistant%20cell%20line&amp;author=A.%20Togawa&amp;author=H.%20Ito&amp;author=F.%20Kimura&amp;author=H.%20Shimizu&amp;author=M.%20Ohtsuka&amp;volume=27&amp;publication_year=2003&amp;pages=220-224&amp;pmid=14508125&amp;doi=10.1097/00006676-200310000-00004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35-biomolecules-14-00969">
<span class="label">35.</span><cite>Wang C., Zhang W., Fu M., Yang A., Huang H., Xie J. Establishment of human pancreatic cancer gemcitabine-resistant cell line with ribonucleotide reductase overexpression. Oncol. Rep. 2015;33:383–390. doi: 10.3892/or.2014.3599.</cite> [<a href="https://doi.org/10.3892/or.2014.3599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25394408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep.&amp;title=Establishment%20of%20human%20pancreatic%20cancer%20gemcitabine-resistant%20cell%20line%20with%20ribonucleotide%20reductase%20overexpression&amp;author=C.%20Wang&amp;author=W.%20Zhang&amp;author=M.%20Fu&amp;author=A.%20Yang&amp;author=H.%20Huang&amp;volume=33&amp;publication_year=2015&amp;pages=383-390&amp;pmid=25394408&amp;doi=10.3892/or.2014.3599&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36-biomolecules-14-00969">
<span class="label">36.</span><cite>Fujiwara-Tani R., Sasaki T., Takagi T., Mori S., Kishi S., Nishiguchi Y., Ohmori H., Fujii K., Kuniyasu H. Gemcitabine resistance in pancreatic ductal carcinoma cell lines stems from reprogramming of energy metabolism. Int. J. Mol. Sci. 2022;23:7824.  doi: 10.3390/ijms23147824.</cite> [<a href="https://doi.org/10.3390/ijms23147824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9323155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35887170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Gemcitabine%20resistance%20in%20pancreatic%20ductal%20carcinoma%20cell%20lines%20stems%20from%20reprogramming%20of%20energy%20metabolism&amp;author=R.%20Fujiwara-Tani&amp;author=T.%20Sasaki&amp;author=T.%20Takagi&amp;author=S.%20Mori&amp;author=S.%20Kishi&amp;volume=23&amp;publication_year=2022&amp;pages=7824&amp;pmid=35887170&amp;doi=10.3390/ijms23147824&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37-biomolecules-14-00969">
<span class="label">37.</span><cite>Mansoori B., Mohammadi A., Davudian S., Shirjang S., Baradaran B. The different mechanisms of cancer drug resistance: A brief review. Adv. Pharm. Bull. 2017;7:339. doi: 10.15171/apb.2017.041.</cite> [<a href="https://doi.org/10.15171/apb.2017.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5651054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29071215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Pharm.%20Bull.&amp;title=The%20different%20mechanisms%20of%20cancer%20drug%20resistance:%20A%20brief%20review&amp;author=B.%20Mansoori&amp;author=A.%20Mohammadi&amp;author=S.%20Davudian&amp;author=S.%20Shirjang&amp;author=B.%20Baradaran&amp;volume=7&amp;publication_year=2017&amp;pages=339&amp;pmid=29071215&amp;doi=10.15171/apb.2017.041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38-biomolecules-14-00969">
<span class="label">38.</span><cite>Khan S.U., Fatima K., Aisha S., Malik F. Unveiling the mechanisms and challenges of cancer drug resistance. Cell Commun. Signal. 2024;22:109. doi: 10.1186/s12964-023-01302-1.</cite> [<a href="https://doi.org/10.1186/s12964-023-01302-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10860306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38347575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Commun.%20Signal&amp;title=Unveiling%20the%20mechanisms%20and%20challenges%20of%20cancer%20drug%20resistance&amp;author=S.U.%20Khan&amp;author=K.%20Fatima&amp;author=S.%20Aisha&amp;author=F.%20Malik&amp;volume=22&amp;publication_year=2024&amp;pages=109&amp;pmid=38347575&amp;doi=10.1186/s12964-023-01302-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39-biomolecules-14-00969">
<span class="label">39.</span><cite>Rao L., Ma N., Liu Y., Du L., Qu B. Correlation between adjuvant chemotherapy regimen, recurrence pattern and prognosis of cholangiocarcinoma after radical surgery. Front. Oncol. 2022;12:324.  doi: 10.3389/fonc.2022.695228.</cite> [<a href="https://doi.org/10.3389/fonc.2022.695228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8873576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35223461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&amp;title=Correlation%20between%20adjuvant%20chemotherapy%20regimen,%20recurrence%20pattern%20and%20prognosis%20of%20cholangiocarcinoma%20after%20radical%20surgery&amp;author=L.%20Rao&amp;author=N.%20Ma&amp;author=Y.%20Liu&amp;author=L.%20Du&amp;author=B.%20Qu&amp;volume=12&amp;publication_year=2022&amp;pages=324&amp;pmid=35223461&amp;doi=10.3389/fonc.2022.695228&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40-biomolecules-14-00969">
<span class="label">40.</span><cite>Rahl P.B., Lin C.Y., Seila A.C., Flynn R.A., McCuine S., Burge C.B., Sharp P.A., Young R.A. c-Myc regulates transcriptional pause release. Cell. 2010;141:432–445. doi: 10.1016/j.cell.2010.03.030.</cite> [<a href="https://doi.org/10.1016/j.cell.2010.03.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2864022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20434984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=c-Myc%20regulates%20transcriptional%20pause%20release&amp;author=P.B.%20Rahl&amp;author=C.Y.%20Lin&amp;author=A.C.%20Seila&amp;author=R.A.%20Flynn&amp;author=S.%20McCuine&amp;volume=141&amp;publication_year=2010&amp;pages=432-445&amp;pmid=20434984&amp;doi=10.1016/j.cell.2010.03.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41-biomolecules-14-00969">
<span class="label">41.</span><cite>Cichowski K., Jacks T. NF1 tumor suppressor gene function: Narrowing the GAP. Cell. 2001;104:593–604. doi: 10.1016/S0092-8674(01)00245-8.</cite> [<a href="https://doi.org/10.1016/S0092-8674(01)00245-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11239415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=NF1%20tumor%20suppressor%20gene%20function:%20Narrowing%20the%20GAP&amp;author=K.%20Cichowski&amp;author=T.%20Jacks&amp;volume=104&amp;publication_year=2001&amp;pages=593-604&amp;pmid=11239415&amp;doi=10.1016/S0092-8674(01)00245-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42-biomolecules-14-00969">
<span class="label">42.</span><cite>Krook M.A., Reeser J.W., Ernst G., Barker H., Wilberding M., Li G., Chen H.-Z., Roychowdhury S. Fibroblast growth factor receptors in cancer: Genetic alterations, diagnostics, therapeutic targets and mechanisms of resistance. Br. J. Cancer. 2021;124:880–892. doi: 10.1038/s41416-020-01157-0.</cite> [<a href="https://doi.org/10.1038/s41416-020-01157-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7921129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33268819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&amp;title=Fibroblast%20growth%20factor%20receptors%20in%20cancer:%20Genetic%20alterations,%20diagnostics,%20therapeutic%20targets%20and%20mechanisms%20of%20resistance&amp;author=M.A.%20Krook&amp;author=J.W.%20Reeser&amp;author=G.%20Ernst&amp;author=H.%20Barker&amp;author=M.%20Wilberding&amp;volume=124&amp;publication_year=2021&amp;pages=880-892&amp;pmid=33268819&amp;doi=10.1038/s41416-020-01157-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43-biomolecules-14-00969">
<span class="label">43.</span><cite>Sharma S.V., Bell D.W., Settleman J., Haber D.A. Epidermal growth factor receptor mutations in lung cancer. Nat. Rev. Cancer. 2007;7:169–181. doi: 10.1038/nrc2088.</cite> [<a href="https://doi.org/10.1038/nrc2088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17318210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Epidermal%20growth%20factor%20receptor%20mutations%20in%20lung%20cancer&amp;author=S.V.%20Sharma&amp;author=D.W.%20Bell&amp;author=J.%20Settleman&amp;author=D.A.%20Haber&amp;volume=7&amp;publication_year=2007&amp;pages=169-181&amp;pmid=17318210&amp;doi=10.1038/nrc2088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44-biomolecules-14-00969">
<span class="label">44.</span><cite>Morova T., McNeill D.R., Lallous N., Gönen M., Dalal K., Wilson D.M., III, Gürsoy A., Keskin Ö., Lack N.A. Androgen receptor-binding sites are highly mutated in prostate cancer. Nat. Commun. 2020;11:832. doi: 10.1038/s41467-020-14644-y.</cite> [<a href="https://doi.org/10.1038/s41467-020-14644-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7012874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32047165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Androgen%20receptor-binding%20sites%20are%20highly%20mutated%20in%20prostate%20cancer&amp;author=T.%20Morova&amp;author=D.R.%20McNeill&amp;author=N.%20Lallous&amp;author=M.%20G%C3%B6nen&amp;author=K.%20Dalal&amp;volume=11&amp;publication_year=2020&amp;pages=832&amp;pmid=32047165&amp;doi=10.1038/s41467-020-14644-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45-biomolecules-14-00969">
<span class="label">45.</span><cite>Fantini M.C., Pallone F. Cytokines: From gut inflammation to colorectal cancer. Curr. Drug Targets. 2008;9:375–380. doi: 10.2174/138945008784221206.</cite> [<a href="https://doi.org/10.2174/138945008784221206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18473765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Targets&amp;title=Cytokines:%20From%20gut%20inflammation%20to%20colorectal%20cancer&amp;author=M.C.%20Fantini&amp;author=F.%20Pallone&amp;volume=9&amp;publication_year=2008&amp;pages=375-380&amp;pmid=18473765&amp;doi=10.2174/138945008784221206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46-biomolecules-14-00969">
<span class="label">46.</span><cite>Matanić D., Beg-Zec Z., Stojanović D., Matakorić N., Flego V., Milevoj-Ribić F. Cytokines in patients with lung cancer. Scand. J. Immunol. 2003;57:173–178. doi: 10.1046/j.1365-3083.2003.01205.x.</cite> [<a href="https://doi.org/10.1046/j.1365-3083.2003.01205.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12588664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Immunol.&amp;title=Cytokines%20in%20patients%20with%20lung%20cancer&amp;author=D.%20Matani%C4%87&amp;author=Z.%20Beg-Zec&amp;author=D.%20Stojanovi%C4%87&amp;author=N.%20Matakori%C4%87&amp;author=V.%20Flego&amp;volume=57&amp;publication_year=2003&amp;pages=173-178&amp;pmid=12588664&amp;doi=10.1046/j.1365-3083.2003.01205.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47-biomolecules-14-00969">
<span class="label">47.</span><cite>Nikolaou M., Pavlopoulou A., Georgakilas A.G., Kyrodimos E. The challenge of drug resistance in cancer treatment: A current overview. Clin. Exp. Metastasis. 2018;35:309–318. doi: 10.1007/s10585-018-9903-0.</cite> [<a href="https://doi.org/10.1007/s10585-018-9903-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29799080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Metastasis&amp;title=The%20challenge%20of%20drug%20resistance%20in%20cancer%20treatment:%20A%20current%20overview&amp;author=M.%20Nikolaou&amp;author=A.%20Pavlopoulou&amp;author=A.G.%20Georgakilas&amp;author=E.%20Kyrodimos&amp;volume=35&amp;publication_year=2018&amp;pages=309-318&amp;pmid=29799080&amp;doi=10.1007/s10585-018-9903-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48-biomolecules-14-00969">
<span class="label">48.</span><cite>Holm P.S., Lage H., Bergmann S., Jürchott K., Glockzin G., Bernshausen A., Mantwill K., Ladhoff A., Wichert A., Mymryk J.S. Multidrug-resistant cancer cells facilitate E1-independent adenoviral replication: Impact for cancer gene therapy. Cancer Res. 2004;64:322–328. doi: 10.1158/0008-5472.CAN-0482-2.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-0482-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14729641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Multidrug-resistant%20cancer%20cells%20facilitate%20E1-independent%20adenoviral%20replication:%20Impact%20for%20cancer%20gene%20therapy&amp;author=P.S.%20Holm&amp;author=H.%20Lage&amp;author=S.%20Bergmann&amp;author=K.%20J%C3%BCrchott&amp;author=G.%20Glockzin&amp;volume=64&amp;publication_year=2004&amp;pages=322-328&amp;pmid=14729641&amp;doi=10.1158/0008-5472.CAN-0482-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49-biomolecules-14-00969">
<span class="label">49.</span><cite>Cole S., Bhardwaj G., Gerlach J., Mackie J., Grant C., Almquist K., Stewart A., Kurz E., Duncan A., Deeley R.G. Overexpression of a transporter gene in a multidrug-resistant human lung cancer cell line. Science. 1992;258:1650–1654. doi: 10.1126/science.1360704.</cite> [<a href="https://doi.org/10.1126/science.1360704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1360704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Overexpression%20of%20a%20transporter%20gene%20in%20a%20multidrug-resistant%20human%20lung%20cancer%20cell%20line&amp;author=S.%20Cole&amp;author=G.%20Bhardwaj&amp;author=J.%20Gerlach&amp;author=J.%20Mackie&amp;author=C.%20Grant&amp;volume=258&amp;publication_year=1992&amp;pages=1650-1654&amp;pmid=1360704&amp;doi=10.1126/science.1360704&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50-biomolecules-14-00969">
<span class="label">50.</span><cite>Morrow C., Cowan K. Glutathione S-transferases and drug resistance. Cancer Cells. 1990;2:15–22. doi: 10.1016/0300-9629(91)90616-K.</cite> [<a href="https://doi.org/10.1016/0300-9629(91)90616-K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2201336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cells&amp;title=Glutathione%20S-transferases%20and%20drug%20resistance&amp;author=C.%20Morrow&amp;author=K.%20Cowan&amp;volume=2&amp;publication_year=1990&amp;pages=15-22&amp;pmid=2201336&amp;doi=10.1016/0300-9629(91)90616-K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51-biomolecules-14-00969">
<span class="label">51.</span><cite>Sharma S., Shah N.A., Joiner A.M., Roberts K.H., Canman C.E. DNA polymerase ζ is a major determinant of resistance to platinum-based chemotherapeutic agents. Mol. Pharmacol. 2012;81:778–787. doi: 10.1124/mol.111.076828.</cite> [<a href="https://doi.org/10.1124/mol.111.076828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3362893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22387291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&amp;title=DNA%20polymerase%20%CE%B6%20is%20a%20major%20determinant%20of%20resistance%20to%20platinum-based%20chemotherapeutic%20agents&amp;author=S.%20Sharma&amp;author=N.A.%20Shah&amp;author=A.M.%20Joiner&amp;author=K.H.%20Roberts&amp;author=C.E.%20Canman&amp;volume=81&amp;publication_year=2012&amp;pages=778-787&amp;pmid=22387291&amp;doi=10.1124/mol.111.076828&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52-biomolecules-14-00969">
<span class="label">52.</span><cite>Zamble D.B., Lippard S.J. Cisplatin and DNA repair in cancer chemotherapy. Trends Biochem. Sci. 1995;20:435–439. doi: 10.1016/S0968-0004(00)89095-7.</cite> [<a href="https://doi.org/10.1016/S0968-0004(00)89095-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8533159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Cisplatin%20and%20DNA%20repair%20in%20cancer%20chemotherapy&amp;author=D.B.%20Zamble&amp;author=S.J.%20Lippard&amp;volume=20&amp;publication_year=1995&amp;pages=435-439&amp;pmid=8533159&amp;doi=10.1016/S0968-0004(00)89095-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53-biomolecules-14-00969">
<span class="label">53.</span><cite>Pang B., Qiao X., Janssen L., Velds A., Groothuis T., Kerkhoven R., Nieuwland M., Ovaa H., Rottenberg S., Van Tellingen O. Drug-induced histone eviction from open chromatin contributes to the chemotherapeutic effects of doxorubicin. Nat. Commun. 2013;4:1908. doi: 10.1038/ncomms2921.</cite> [<a href="https://doi.org/10.1038/ncomms2921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3674280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23715267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Drug-induced%20histone%20eviction%20from%20open%20chromatin%20contributes%20to%20the%20chemotherapeutic%20effects%20of%20doxorubicin&amp;author=B.%20Pang&amp;author=X.%20Qiao&amp;author=L.%20Janssen&amp;author=A.%20Velds&amp;author=T.%20Groothuis&amp;volume=4&amp;publication_year=2013&amp;pages=1908&amp;pmid=23715267&amp;doi=10.1038/ncomms2921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54-biomolecules-14-00969">
<span class="label">54.</span><cite>Kamphorst A.O., Pillai R.N., Yang S., Nasti T.H., Akondy R.S., Wieland A., Sica G.L., Yu K., Koenig L., Patel N.T. Proliferation of PD-1+ CD8 T cells in peripheral blood after PD-1–targeted therapy in lung cancer patients. Proc. Natl. Acad. Sci. USA. 2017;114:4993–4998. doi: 10.1073/pnas.1705327114.</cite> [<a href="https://doi.org/10.1073/pnas.1705327114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5441721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28446615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Proliferation%20of%20PD-1+%20CD8%20T%20cells%20in%20peripheral%20blood%20after%20PD-1%E2%80%93targeted%20therapy%20in%20lung%20cancer%20patients&amp;author=A.O.%20Kamphorst&amp;author=R.N.%20Pillai&amp;author=S.%20Yang&amp;author=T.H.%20Nasti&amp;author=R.S.%20Akondy&amp;volume=114&amp;publication_year=2017&amp;pages=4993-4998&amp;pmid=28446615&amp;doi=10.1073/pnas.1705327114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55-biomolecules-14-00969">
<span class="label">55.</span><cite>Roninson I.B., Broude E.V., Chang B.-D. If not apoptosis, then what? Treatment-induced senescence and mitotic catastrophe in tumor cells. Drug Resist. Updat. 2001;4:303–313. doi: 10.1054/drup.2001.0213.</cite> [<a href="https://doi.org/10.1054/drup.2001.0213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11991684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Resist.%20Updat.&amp;title=If%20not%20apoptosis,%20then%20what?%20Treatment-induced%20senescence%20and%20mitotic%20catastrophe%20in%20tumor%20cells&amp;author=I.B.%20Roninson&amp;author=E.V.%20Broude&amp;author=B.-D.%20Chang&amp;volume=4&amp;publication_year=2001&amp;pages=303-313&amp;pmid=11991684&amp;doi=10.1054/drup.2001.0213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56-biomolecules-14-00969">
<span class="label">56.</span><cite>O’Brien S., Golubovskaya V.M., Conroy J., Liu S., Wang D., Liu B., Cance W.G. FAK inhibition with small molecule inhibitor Y15 decreases viability, clonogenicity, and cell attachment in thyroid cancer cell lines and synergizes with targeted therapeutics. Oncotarget. 2014;5:7945. doi: 10.18632/oncotarget.2381.</cite> [<a href="https://doi.org/10.18632/oncotarget.2381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4202172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25277206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=FAK%20inhibition%20with%20small%20molecule%20inhibitor%20Y15%20decreases%20viability,%20clonogenicity,%20and%20cell%20attachment%20in%20thyroid%20cancer%20cell%20lines%20and%20synergizes%20with%20targeted%20therapeutics&amp;author=S.%20O%E2%80%99Brien&amp;author=V.M.%20Golubovskaya&amp;author=J.%20Conroy&amp;author=S.%20Liu&amp;author=D.%20Wang&amp;volume=5&amp;publication_year=2014&amp;pages=7945&amp;pmid=25277206&amp;doi=10.18632/oncotarget.2381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57-biomolecules-14-00969">
<span class="label">57.</span><cite>Wang Z., Li Y., Kong D., Banerjee S., Ahmad A., Azmi A.S., Ali S., Abbruzzese J.L., Gallick G.E., Sarkar F.H. Acquisition of epithelial-mesenchymal transition phenotype of gemcitabine-resistant pancreatic cancer cells is linked with activation of the notch signaling pathway. Cancer Res. 2009;69:2400–2407. doi: 10.1158/0008-5472.CAN-08-4312.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-08-4312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2657919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19276344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Acquisition%20of%20epithelial-mesenchymal%20transition%20phenotype%20of%20gemcitabine-resistant%20pancreatic%20cancer%20cells%20is%20linked%20with%20activation%20of%20the%20notch%20signaling%20pathway&amp;author=Z.%20Wang&amp;author=Y.%20Li&amp;author=D.%20Kong&amp;author=S.%20Banerjee&amp;author=A.%20Ahmad&amp;volume=69&amp;publication_year=2009&amp;pages=2400-2407&amp;pmid=19276344&amp;doi=10.1158/0008-5472.CAN-08-4312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58-biomolecules-14-00969">
<span class="label">58.</span><cite>Erdogan B., Webb D.J. Cancer-associated fibroblasts modulate growth factor signaling and extracellular matrix remodeling to regulate tumor metastasis. Biochem. Soc. Trans. 2017;45:229–236. doi: 10.1042/BST20160387.</cite> [<a href="https://doi.org/10.1042/BST20160387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5371349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28202677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Soc.%20Trans.&amp;title=Cancer-associated%20fibroblasts%20modulate%20growth%20factor%20signaling%20and%20extracellular%20matrix%20remodeling%20to%20regulate%20tumor%20metastasis&amp;author=B.%20Erdogan&amp;author=D.J.%20Webb&amp;volume=45&amp;publication_year=2017&amp;pages=229-236&amp;pmid=28202677&amp;doi=10.1042/BST20160387&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59-biomolecules-14-00969">
<span class="label">59.</span><cite>Sprenger C.C., Plymate S.R., Reed M.J. Aging-related alterations in the extracellular matrix modulate the microenvironment and influence tumor progression. Int. J. Cancer. 2010;127:2739–2748. doi: 10.1002/ijc.25615.</cite> [<a href="https://doi.org/10.1002/ijc.25615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3024458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21351253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&amp;title=Aging-related%20alterations%20in%20the%20extracellular%20matrix%20modulate%20the%20microenvironment%20and%20influence%20tumor%20progression&amp;author=C.C.%20Sprenger&amp;author=S.R.%20Plymate&amp;author=M.J.%20Reed&amp;volume=127&amp;publication_year=2010&amp;pages=2739-2748&amp;pmid=21351253&amp;doi=10.1002/ijc.25615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60-biomolecules-14-00969">
<span class="label">60.</span><cite>He W., Liang B., Wang C., Li S., Zhao Y., Huang Q., Liu Z., Yao Z., Wu Q., Liao W. MSC-regulated lncRNA MACC1-AS1 promotes stemness and chemoresistance through fatty acid oxidation in gastric cancer. Oncogene. 2019;38:4637–4654. doi: 10.1038/s41388-019-0747-0.</cite> [<a href="https://doi.org/10.1038/s41388-019-0747-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6756048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=MSC-regulated%20lncRNA%20MACC1-AS1%20promotes%20stemness%20and%20chemoresistance%20through%20fatty%20acid%20oxidation%20in%20gastric%20cancer&amp;author=W.%20He&amp;author=B.%20Liang&amp;author=C.%20Wang&amp;author=S.%20Li&amp;author=Y.%20Zhao&amp;volume=38&amp;publication_year=2019&amp;pages=4637-4654&amp;pmid=30742067&amp;doi=10.1038/s41388-019-0747-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61-biomolecules-14-00969">
<span class="label">61.</span><cite>Iwamoto H., Abe M., Yang Y., Cui D., Seki T., Nakamura M., Hosaka K., Lim S., Wu J., He X. Cancer lipid metabolism confers antiangiogenic drug resistance. Cell Metab. 2018;28:104–117.e5. doi: 10.1016/j.cmet.2018.05.005.</cite> [<a href="https://doi.org/10.1016/j.cmet.2018.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29861385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=Cancer%20lipid%20metabolism%20confers%20antiangiogenic%20drug%20resistance&amp;author=H.%20Iwamoto&amp;author=M.%20Abe&amp;author=Y.%20Yang&amp;author=D.%20Cui&amp;author=T.%20Seki&amp;volume=28&amp;publication_year=2018&amp;pages=104-117.e5&amp;pmid=29861385&amp;doi=10.1016/j.cmet.2018.05.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62-biomolecules-14-00969">
<span class="label">62.</span><cite>Wang T., Fahrmann J.F., Lee H., Li Y.-J., Tripathi S.C., Yue C., Zhang C., Lifshitz V., Song J., Yuan Y. JAK/STAT3-regulated fatty acid β-oxidation is critical for breast cancer stem cell self-renewal and chemoresistance. Cell Metab. 2018;27:136–150. doi: 10.1016/j.cmet.2017.11.001.</cite> [<a href="https://doi.org/10.1016/j.cmet.2017.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5777338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29249690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=JAK/STAT3-regulated%20fatty%20acid%20%CE%B2-oxidation%20is%20critical%20for%20breast%20cancer%20stem%20cell%20self-renewal%20and%20chemoresistance&amp;author=T.%20Wang&amp;author=J.F.%20Fahrmann&amp;author=H.%20Lee&amp;author=Y.-J.%20Li&amp;author=S.C.%20Tripathi&amp;volume=27&amp;publication_year=2018&amp;pages=136-150&amp;pmid=29249690&amp;doi=10.1016/j.cmet.2017.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B63-biomolecules-14-00969">
<span class="label">63.</span><cite>Peruzzi B., Bottaro D.P. Targeting the c-Met signaling pathway in cancer. Clin. Cancer Res. 2006;12:3657–3660. doi: 10.1158/1078-0432.CCR-06-0818.</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-06-0818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16778093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&amp;title=Targeting%20the%20c-Met%20signaling%20pathway%20in%20cancer&amp;author=B.%20Peruzzi&amp;author=D.P.%20Bottaro&amp;volume=12&amp;publication_year=2006&amp;pages=3657-3660&amp;pmid=16778093&amp;doi=10.1158/1078-0432.CCR-06-0818&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B64-biomolecules-14-00969">
<span class="label">64.</span><cite>Comoglio P.M., Trusolino L. Series Introduction: Invasive growth: From development to metastasis. J. Clin. Investig. 2002;109:857–862. doi: 10.1172/JCI0215392.</cite> [<a href="https://doi.org/10.1172/JCI0215392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC150936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11927611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&amp;title=Series%20Introduction:%20Invasive%20growth:%20From%20development%20to%20metastasis&amp;author=P.M.%20Comoglio&amp;author=L.%20Trusolino&amp;volume=109&amp;publication_year=2002&amp;pages=857-862&amp;pmid=11927611&amp;doi=10.1172/JCI0215392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B65-biomolecules-14-00969">
<span class="label">65.</span><cite>Aumailley M. The laminin family. Cell Adh. Migr. 2013;7:48–55. doi: 10.4161/cam.22826.</cite> [<a href="https://doi.org/10.4161/cam.22826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3544786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23263632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Adh.%20Migr.&amp;title=The%20laminin%20family&amp;author=M.%20Aumailley&amp;volume=7&amp;publication_year=2013&amp;pages=48-55&amp;pmid=23263632&amp;doi=10.4161/cam.22826&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B66-biomolecules-14-00969">
<span class="label">66.</span><cite>Chen Z., Yang X., Bi G., Liang J., Hu Z., Zhao M., Li M., Lu T., Zheng Y., Sui Q. Ligand-receptor interaction atlas within and between tumor cells and T cells in lung adenocarcinoma. Int. J. Biol. Sci. 2020;16:2205.  doi: 10.7150/ijbs.42080.</cite> [<a href="https://doi.org/10.7150/ijbs.42080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7294944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32549766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Biol.%20Sci.&amp;title=Ligand-receptor%20interaction%20atlas%20within%20and%20between%20tumor%20cells%20and%20T%20cells%20in%20lung%20adenocarcinoma&amp;author=Z.%20Chen&amp;author=X.%20Yang&amp;author=G.%20Bi&amp;author=J.%20Liang&amp;author=Z.%20Hu&amp;volume=16&amp;publication_year=2020&amp;pages=2205&amp;pmid=32549766&amp;doi=10.7150/ijbs.42080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67-biomolecules-14-00969">
<span class="label">67.</span><cite>Ran T., Chen Z., Zhao L., Ran W., Fan J., Hong S., Yang Z. LAMB1 is related to the T stage and indicates poor prognosis in gastric cancer. Technol. Cancer Res. Treat. 2021;20:1–11. doi: 10.1177/15330338211004944.</cite> [<a href="https://doi.org/10.1177/15330338211004944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8020091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33784890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Technol.%20Cancer%20Res.%20Treat.&amp;title=LAMB1%20is%20related%20to%20the%20T%20stage%20and%20indicates%20poor%20prognosis%20in%20gastric%20cancer&amp;author=T.%20Ran&amp;author=Z.%20Chen&amp;author=L.%20Zhao&amp;author=W.%20Ran&amp;author=J.%20Fan&amp;volume=20&amp;publication_year=2021&amp;pages=1-11&amp;pmid=33784890&amp;doi=10.1177/15330338211004944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B68-biomolecules-14-00969">
<span class="label">68.</span><cite>Lee H., Kim W.-J., Kang H.-G., Jang J.-H., Choi I.J., Chun K.-H., Kim S.-J. Upregulation of LAMB1 via ERK/c-Jun axis promotes gastric cancer growth and motility. Int. J. Mol. Sci. 2021;22:626.  doi: 10.3390/ijms22020626.</cite> [<a href="https://doi.org/10.3390/ijms22020626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7826975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33435161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Upregulation%20of%20LAMB1%20via%20ERK/c-Jun%20axis%20promotes%20gastric%20cancer%20growth%20and%20motility&amp;author=H.%20Lee&amp;author=W.-J.%20Kim&amp;author=H.-G.%20Kang&amp;author=J.-H.%20Jang&amp;author=I.J.%20Choi&amp;volume=22&amp;publication_year=2021&amp;pages=626&amp;pmid=33435161&amp;doi=10.3390/ijms22020626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B69-biomolecules-14-00969">
<span class="label">69.</span><cite>Sakaguchi T., Yoshino H., Yonemori M., Miyamoto K., Sugita S., Matsushita R., Itesako T., Tatarano S., Nakagawa M., Enokida H. Regulation of ITGA3 by the dual-stranded microRNA-199 family as a potential prognostic marker in bladder cancer. Br. J. Cancer. 2017;116:1077–1087. doi: 10.1038/bjc.2017.43.</cite> [<a href="https://doi.org/10.1038/bjc.2017.43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5396103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28324890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&amp;title=Regulation%20of%20ITGA3%20by%20the%20dual-stranded%20microRNA-199%20family%20as%20a%20potential%20prognostic%20marker%20in%20bladder%20cancer&amp;author=T.%20Sakaguchi&amp;author=H.%20Yoshino&amp;author=M.%20Yonemori&amp;author=K.%20Miyamoto&amp;author=S.%20Sugita&amp;volume=116&amp;publication_year=2017&amp;pages=1077-1087&amp;pmid=28324890&amp;doi=10.1038/bjc.2017.43&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B70-biomolecules-14-00969">
<span class="label">70.</span><cite>Stoeck A., Lejnine S., Truong A., Pan L., Wang H., Zang C., Yuan J., Ware C., MacLean J., Garrett-Engele P.W. Discovery of biomarkers predictive of GSI response in triple-negative breast cancer and adenoid cystic carcinoma. Cancer Discov. 2014;4:1154–1167. doi: 10.1158/2159-8290.CD-13-0830.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-0830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4184927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25104330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov.&amp;title=Discovery%20of%20biomarkers%20predictive%20of%20GSI%20response%20in%20triple-negative%20breast%20cancer%20and%20adenoid%20cystic%20carcinoma&amp;author=A.%20Stoeck&amp;author=S.%20Lejnine&amp;author=A.%20Truong&amp;author=L.%20Pan&amp;author=H.%20Wang&amp;volume=4&amp;publication_year=2014&amp;pages=1154-1167&amp;pmid=25104330&amp;doi=10.1158/2159-8290.CD-13-0830&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B71-biomolecules-14-00969">
<span class="label">71.</span><cite>Wang N.J., Sanborn Z., Arnett K.L., Bayston L.J., Liao W., Proby C.M., Leigh I.M., Collisson E.A., Gordon P.B., Jakkula L. Loss-of-function mutations in Notch receptors in cutaneous and lung squamous cell carcinoma. Proc. Natl. Acad. Sci. USA. 2011;108:17761–17766. doi: 10.1073/pnas.1114669108.</cite> [<a href="https://doi.org/10.1073/pnas.1114669108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3203814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22006338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Loss-of-function%20mutations%20in%20Notch%20receptors%20in%20cutaneous%20and%20lung%20squamous%20cell%20carcinoma&amp;author=N.J.%20Wang&amp;author=Z.%20Sanborn&amp;author=K.L.%20Arnett&amp;author=L.J.%20Bayston&amp;author=W.%20Liao&amp;volume=108&amp;publication_year=2011&amp;pages=17761-17766&amp;pmid=22006338&amp;doi=10.1073/pnas.1114669108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B72-biomolecules-14-00969">
<span class="label">72.</span><cite>Pancewicz J., Taylor J.M., Datta A., Baydoun H.H., Waldmann T.A., Hermine O., Nicot C. Notch signaling contributes to proliferation and tumor formation of human T-cell leukemia virus type 1–associated adult T-cell leukemia. Proc. Natl. Acad. Sci. USA. 2010;107:16619–16624. doi: 10.1073/pnas.1010722107.</cite> [<a href="https://doi.org/10.1073/pnas.1010722107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2944748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20823234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Notch%20signaling%20contributes%20to%20proliferation%20and%20tumor%20formation%20of%20human%20T-cell%20leukemia%20virus%20type%201%E2%80%93associated%20adult%20T-cell%20leukemia&amp;author=J.%20Pancewicz&amp;author=J.M.%20Taylor&amp;author=A.%20Datta&amp;author=H.H.%20Baydoun&amp;author=T.A.%20Waldmann&amp;volume=107&amp;publication_year=2010&amp;pages=16619-16624&amp;pmid=20823234&amp;doi=10.1073/pnas.1010722107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B73-biomolecules-14-00969">
<span class="label">73.</span><cite>Kuhnert F., Chen G., Coetzee S., Thambi N., Hickey C., Shan J., Kovalenko P., Noguera-Troise I., Smith E., Fairhurst J. Dll4 blockade in stromal cells mediates antitumor effects in preclinical models of ovarian cancer. Cancer Res. 2015;75:4086–4096. doi: 10.1158/0008-5472.CAN-14-3773.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-14-3773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26377940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Dll4%20blockade%20in%20stromal%20cells%20mediates%20antitumor%20effects%20in%20preclinical%20models%20of%20ovarian%20cancer&amp;author=F.%20Kuhnert&amp;author=G.%20Chen&amp;author=S.%20Coetzee&amp;author=N.%20Thambi&amp;author=C.%20Hickey&amp;volume=75&amp;publication_year=2015&amp;pages=4086-4096&amp;pmid=26377940&amp;doi=10.1158/0008-5472.CAN-14-3773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B74-biomolecules-14-00969">
<span class="label">74.</span><cite>Lu J., Ye X., Fan F., Xia L., Bhattacharya R., Bellister S., Tozzi F., Sceusi E., Zhou Y., Tachibana I. Endothelial cells promote the colorectal cancer stem cell phenotype through a soluble form of Jagged-1. Cancer Cell. 2013;23:171–185. doi: 10.1016/j.ccr.2012.12.021.</cite> [<a href="https://doi.org/10.1016/j.ccr.2012.12.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3574187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23375636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Endothelial%20cells%20promote%20the%20colorectal%20cancer%20stem%20cell%20phenotype%20through%20a%20soluble%20form%20of%20Jagged-1&amp;author=J.%20Lu&amp;author=X.%20Ye&amp;author=F.%20Fan&amp;author=L.%20Xia&amp;author=R.%20Bhattacharya&amp;volume=23&amp;publication_year=2013&amp;pages=171-185&amp;pmid=23375636&amp;doi=10.1016/j.ccr.2012.12.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B75-biomolecules-14-00969">
<span class="label">75.</span><cite>Zou Y., Fang F., Ding Y.J., Dai M.Y., Yi X., Chen C., Tao Z.Z., Chen S.M. Notch 2 signaling contributes to cell growth, anti-apoptosis and metastasis in laryngeal squamous cell carcinoma. Mol. Med. Rep. 2016;14:3517–3524. doi: 10.3892/mmr.2016.5688.</cite> [<a href="https://doi.org/10.3892/mmr.2016.5688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5042778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27572051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.%20Rep.&amp;title=Notch%202%20signaling%20contributes%20to%20cell%20growth,%20anti-apoptosis%20and%20metastasis%20in%20laryngeal%20squamous%20cell%20carcinoma&amp;author=Y.%20Zou&amp;author=F.%20Fang&amp;author=Y.J.%20Ding&amp;author=M.Y.%20Dai&amp;author=X.%20Yi&amp;volume=14&amp;publication_year=2016&amp;pages=3517-3524&amp;pmid=27572051&amp;doi=10.3892/mmr.2016.5688&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B76-biomolecules-14-00969">
<span class="label">76.</span><cite>Beverly L.J., Felsher D.W., Capobianco A.J. Suppression of p53 by Notch in lymphomagenesis: Implications for initiation and regression. Cancer Res. 2005;65:7159–7168. doi: 10.1158/0008-5472.CAN-05-1664.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-05-1664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16103066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Suppression%20of%20p53%20by%20Notch%20in%20lymphomagenesis:%20Implications%20for%20initiation%20and%20regression&amp;author=L.J.%20Beverly&amp;author=D.W.%20Felsher&amp;author=A.J.%20Capobianco&amp;volume=65&amp;publication_year=2005&amp;pages=7159-7168&amp;pmid=16103066&amp;doi=10.1158/0008-5472.CAN-05-1664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B77-biomolecules-14-00969">
<span class="label">77.</span><cite>Dotto G.P. Crosstalk of Notch with p53 and p63 in cancer growth control. Nat. Rev. Cancer. 2009;9:587–595. doi: 10.1038/nrc2675.</cite> [<a href="https://doi.org/10.1038/nrc2675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6059364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19609265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Crosstalk%20of%20Notch%20with%20p53%20and%20p63%20in%20cancer%20growth%20control&amp;author=G.P.%20Dotto&amp;volume=9&amp;publication_year=2009&amp;pages=587-595&amp;pmid=19609265&amp;doi=10.1038/nrc2675&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B78-biomolecules-14-00969">
<span class="label">78.</span><cite>Capaccione K.M., Pine S.R. The Notch signaling pathway as a mediator of tumor survival. Carcinogenesis. 2013;34:1420–1430. doi: 10.1093/carcin/bgt127.</cite> [<a href="https://doi.org/10.1093/carcin/bgt127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3697894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23585460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=The%20Notch%20signaling%20pathway%20as%20a%20mediator%20of%20tumor%20survival&amp;author=K.M.%20Capaccione&amp;author=S.R.%20Pine&amp;volume=34&amp;publication_year=2013&amp;pages=1420-1430&amp;pmid=23585460&amp;doi=10.1093/carcin/bgt127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B79-biomolecules-14-00969">
<span class="label">79.</span><cite>Qu J., Song M., Xie J., Huang X.-Y., Hu X.-M., Gan R.-H., Zhao Y., Lin L.-S., Chen J., Lin X. Notch2 signaling contributes to cell growth, invasion, and migration in salivary adenoid cystic carcinoma. Mol. Cell. Biochem. 2016;411:135–141. doi: 10.1007/s11010-015-2575-z.</cite> [<a href="https://doi.org/10.1007/s11010-015-2575-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26427670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biochem.&amp;title=Notch2%20signaling%20contributes%20to%20cell%20growth,%20invasion,%20and%20migration%20in%20salivary%20adenoid%20cystic%20carcinoma&amp;author=J.%20Qu&amp;author=M.%20Song&amp;author=J.%20Xie&amp;author=X.-Y.%20Huang&amp;author=X.-M.%20Hu&amp;volume=411&amp;publication_year=2016&amp;pages=135-141&amp;pmid=26427670&amp;doi=10.1007/s11010-015-2575-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B80-biomolecules-14-00969">
<span class="label">80.</span><cite>Zhu H.-M., Jiang X.-S., Li H.-Z., Qian L.-X., Du M.-Y., Lu Z.-W., Wu J., Tian X.-K., Fei Q., He X. miR-184 inhibits tumor invasion, migration and metastasis in nasopharyngeal carcinoma by targeting Notch2. Cell. Physiol. Biochem. 2018;49:1564–1576. doi: 10.1159/000493459.</cite> [<a href="https://doi.org/10.1159/000493459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30223264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Physiol.%20Biochem.&amp;title=miR-184%20inhibits%20tumor%20invasion,%20migration%20and%20metastasis%20in%20nasopharyngeal%20carcinoma%20by%20targeting%20Notch2&amp;author=H.-M.%20Zhu&amp;author=X.-S.%20Jiang&amp;author=H.-Z.%20Li&amp;author=L.-X.%20Qian&amp;author=M.-Y.%20Du&amp;volume=49&amp;publication_year=2018&amp;pages=1564-1576&amp;pmid=30223264&amp;doi=10.1159/000493459&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B81-biomolecules-14-00969">
<span class="label">81.</span><cite>Tanaka M., Setoguchi T., Hirotsu M., Gao H., Sasaki H., Matsunoshita Y., Komiya S. Inhibition of Notch pathway prevents osteosarcoma growth by cell cycle regulation. Br. J. Cancer. 2009;100:1957–1965. doi: 10.1038/sj.bjc.6605060.</cite> [<a href="https://doi.org/10.1038/sj.bjc.6605060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2714252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19455146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&amp;title=Inhibition%20of%20Notch%20pathway%20prevents%20osteosarcoma%20growth%20by%20cell%20cycle%20regulation&amp;author=M.%20Tanaka&amp;author=T.%20Setoguchi&amp;author=M.%20Hirotsu&amp;author=H.%20Gao&amp;author=H.%20Sasaki&amp;volume=100&amp;publication_year=2009&amp;pages=1957-1965&amp;pmid=19455146&amp;doi=10.1038/sj.bjc.6605060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B82-biomolecules-14-00969">
<span class="label">82.</span><cite>Zhao W.-X., Wu Z.-M., Liu W., Lin J.-H. Notch2 and Notch3 suppress the proliferation and mediate invasion of trophoblast cell lines. Biol. Open. 2017;6:1123–1129. doi: 10.1242/bio.025767.</cite> [<a href="https://doi.org/10.1242/bio.025767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5576079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28606936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Open&amp;title=Notch2%20and%20Notch3%20suppress%20the%20proliferation%20and%20mediate%20invasion%20of%20trophoblast%20cell%20lines&amp;author=W.-X.%20Zhao&amp;author=Z.-M.%20Wu&amp;author=W.%20Liu&amp;author=J.-H.%20Lin&amp;volume=6&amp;publication_year=2017&amp;pages=1123-1129&amp;pmid=28606936&amp;doi=10.1242/bio.025767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B83-biomolecules-14-00969">
<span class="label">83.</span><cite>Craene B.D., Berx G. Regulatory networks defining EMT during cancer initiation and progression. Nat. Rev. Cancer. 2013;13:97–110. doi: 10.1038/nrc3447.</cite> [<a href="https://doi.org/10.1038/nrc3447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23344542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Regulatory%20networks%20defining%20EMT%20during%20cancer%20initiation%20and%20progression&amp;author=B.D.%20Craene&amp;author=G.%20Berx&amp;volume=13&amp;publication_year=2013&amp;pages=97-110&amp;pmid=23344542&amp;doi=10.1038/nrc3447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B84-biomolecules-14-00969">
<span class="label">84.</span><cite>Thiery J.P., Acloque H., Huang R.Y., Nieto M.A. Epithelial-mesenchymal transitions in development and disease. Cell. 2009;139:871–890. doi: 10.1016/j.cell.2009.11.007.</cite> [<a href="https://doi.org/10.1016/j.cell.2009.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19945376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Epithelial-mesenchymal%20transitions%20in%20development%20and%20disease&amp;author=J.P.%20Thiery&amp;author=H.%20Acloque&amp;author=R.Y.%20Huang&amp;author=M.A.%20Nieto&amp;volume=139&amp;publication_year=2009&amp;pages=871-890&amp;pmid=19945376&amp;doi=10.1016/j.cell.2009.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B85-biomolecules-14-00969">
<span class="label">85.</span><cite>Otte J., Dizdar L., Behrens B., Goering W., Knoefel W.T., Wruck W., Stoecklein N.H., Adjaye J. FGF signalling in the self-renewal of colon cancer organoids. Sci. Rep. 2019;9:17365.  doi: 10.1038/s41598-019-53907-7.</cite> [<a href="https://doi.org/10.1038/s41598-019-53907-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6874569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31758153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=FGF%20signalling%20in%20the%20self-renewal%20of%20colon%20cancer%20organoids&amp;author=J.%20Otte&amp;author=L.%20Dizdar&amp;author=B.%20Behrens&amp;author=W.%20Goering&amp;author=W.T.%20Knoefel&amp;volume=9&amp;publication_year=2019&amp;pages=17365&amp;pmid=31758153&amp;doi=10.1038/s41598-019-53907-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B86-biomolecules-14-00969">
<span class="label">86.</span><cite>Ye S., Chen Y., Hu B., Huang H., Sun Y., Stewart J.P., Johnson S.K., Barlogie B., Zangari M., Tricot G. N-Cadherin Stabilizes β-Catenin and Promotes β-Catenin/TCF Transcriptional Activation and Cell Adhesion-Mediated Drug Resistance in Multiple Myeloma. Blood. 2021;138:1572. doi: 10.1182/blood-2021-148624.</cite> [<a href="https://doi.org/10.1182/blood-2021-148624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=N-Cadherin%20Stabilizes%20%CE%B2-Catenin%20and%20Promotes%20%CE%B2-Catenin/TCF%20Transcriptional%20Activation%20and%20Cell%20Adhesion-Mediated%20Drug%20Resistance%20in%20Multiple%20Myeloma&amp;author=S.%20Ye&amp;author=Y.%20Chen&amp;author=B.%20Hu&amp;author=H.%20Huang&amp;author=Y.%20Sun&amp;volume=138&amp;publication_year=2021&amp;pages=1572&amp;doi=10.1182/blood-2021-148624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B87-biomolecules-14-00969">
<span class="label">87.</span><cite>Ciołczyk-Wierzbicka D., Laidler P. The inhibition of invasion of human melanoma cells through N-cadherin knock-down. Med. Oncol. 2018;35:42. doi: 10.1007/s12032-018-1104-9.</cite> [<a href="https://doi.org/10.1007/s12032-018-1104-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5830464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29492694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Oncol.&amp;title=The%20inhibition%20of%20invasion%20of%20human%20melanoma%20cells%20through%20N-cadherin%20knock-down&amp;author=D.%20Cio%C5%82czyk-Wierzbicka&amp;author=P.%20Laidler&amp;volume=35&amp;publication_year=2018&amp;pages=42&amp;pmid=29492694&amp;doi=10.1007/s12032-018-1104-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B88-biomolecules-14-00969">
<span class="label">88.</span><cite>Li K., He W., Lin N., Wang X., Fan Q.-X. N-cadherin knock-down decreases invasiveness of esophageal squamous cell carcinoma in vitro. World J. Gastroenterol. WJG. 2009;15:697. doi: 10.3748/wjg.15.697.</cite> [<a href="https://doi.org/10.3748/wjg.15.697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2653438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19222093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J.%20Gastroenterol.%20WJG&amp;title=N-cadherin%20knock-down%20decreases%20invasiveness%20of%20esophageal%20squamous%20cell%20carcinoma%20in%20vitro&amp;author=K.%20Li&amp;author=W.%20He&amp;author=N.%20Lin&amp;author=X.%20Wang&amp;author=Q.-X.%20Fan&amp;volume=15&amp;publication_year=2009&amp;pages=697&amp;pmid=19222093&amp;doi=10.3748/wjg.15.697&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B89-biomolecules-14-00969">
<span class="label">89.</span><cite>Joshi V., Lakhani S.R., McCart Reed A.E. NDRG1 in Cancer: A Suppressor, Promoter, or Both? Cancers. 2022;14:5739.  doi: 10.3390/cancers14235739.</cite> [<a href="https://doi.org/10.3390/cancers14235739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9737586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36497221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=NDRG1%20in%20Cancer:%20A%20Suppressor,%20Promoter,%20or%20Both?&amp;author=V.%20Joshi&amp;author=S.R.%20Lakhani&amp;author=A.E.%20McCart%20Reed&amp;volume=14&amp;publication_year=2022&amp;pages=5739&amp;pmid=36497221&amp;doi=10.3390/cancers14235739&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B90-biomolecules-14-00969">
<span class="label">90.</span><cite>Guan R.J., Ford H.L., Fu Y., Li Y., Shaw L.M., Pardee A.B. Drg-1 as a differentiation-related, putative metastatic suppressor gene in human colon cancer. Cancer Res. 2000;60:749–755.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10676663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Drg-1%20as%20a%20differentiation-related,%20putative%20metastatic%20suppressor%20gene%20in%20human%20colon%20cancer&amp;author=R.J.%20Guan&amp;author=H.L.%20Ford&amp;author=Y.%20Fu&amp;author=Y.%20Li&amp;author=L.M.%20Shaw&amp;volume=60&amp;publication_year=2000&amp;pages=749-755&amp;pmid=10676663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B91-biomolecules-14-00969">
<span class="label">91.</span><cite>Maruyama Y., Ono M., Kawahara A., Yokoyama T., Basaki Y., Kage M., Aoyagi S., Kinoshita H., Kuwano M. Tumor growth suppression in pancreatic cancer by a putative metastasis suppressor gene Cap43/NDRG1/Drg-1 through modulation of angiogenesis. Cancer Res. 2006;66:6233–6242. doi: 10.1158/0008-5472.CAN-06-0183.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-06-0183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16778198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Tumor%20growth%20suppression%20in%20pancreatic%20cancer%20by%20a%20putative%20metastasis%20suppressor%20gene%20Cap43/NDRG1/Drg-1%20through%20modulation%20of%20angiogenesis&amp;author=Y.%20Maruyama&amp;author=M.%20Ono&amp;author=A.%20Kawahara&amp;author=T.%20Yokoyama&amp;author=Y.%20Basaki&amp;volume=66&amp;publication_year=2006&amp;pages=6233-6242&amp;pmid=16778198&amp;doi=10.1158/0008-5472.CAN-06-0183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B92-biomolecules-14-00969">
<span class="label">92.</span><cite>Bandyopadhyay S., Pai S.K., Hirota S., Hosobe S., Takano Y., Saito K., Piquemal D., Commes T., Watabe M., Gross S.C. Role of the putative tumor metastasis suppressor gene Drg-1 in breast cancer progression. Oncogene. 2004;23:5675–5681. doi: 10.1038/sj.onc.1207734.</cite> [<a href="https://doi.org/10.1038/sj.onc.1207734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15184886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Role%20of%20the%20putative%20tumor%20metastasis%20suppressor%20gene%20Drg-1%20in%20breast%20cancer%20progression&amp;author=S.%20Bandyopadhyay&amp;author=S.K.%20Pai&amp;author=S.%20Hirota&amp;author=S.%20Hosobe&amp;author=Y.%20Takano&amp;volume=23&amp;publication_year=2004&amp;pages=5675-5681&amp;pmid=15184886&amp;doi=10.1038/sj.onc.1207734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B93-biomolecules-14-00969">
<span class="label">93.</span><cite>Akiba J., Ogasawara S., Kawahara A., Nishida N., Sanada S., Moriya F., Kuwano M., Nakashima O., Yano H. N-myc downstream regulated gene 1 (NDRG1)/Cap43 enhances portal vein invasion and intrahepatic metastasis in human hepatocellular carcinoma. Oncol. Rep. 2008;20:1329–1335. doi: 10.3892/or_00000148.</cite> [<a href="https://doi.org/10.3892/or_00000148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19020710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep.&amp;title=N-myc%20downstream%20regulated%20gene%201%20(NDRG1)/Cap43%20enhances%20portal%20vein%20invasion%20and%20intrahepatic%20metastasis%20in%20human%20hepatocellular%20carcinoma&amp;author=J.%20Akiba&amp;author=S.%20Ogasawara&amp;author=A.%20Kawahara&amp;author=N.%20Nishida&amp;author=S.%20Sanada&amp;volume=20&amp;publication_year=2008&amp;pages=1329-1335&amp;pmid=19020710&amp;doi=10.3892/or_00000148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B94-biomolecules-14-00969">
<span class="label">94.</span><cite>Ai R., Sun Y., Guo Z., Wei W., Zhou L., Liu F., Hendricks D.T., Xu Y., Zhao X. NDRG1 overexpression promotes the progression of esophageal squamous cell carcinoma through modulating Wnt signaling pathway. Cancer Biol. Ther. 2016;17:943–954. doi: 10.1080/15384047.2016.1210734.</cite> [<a href="https://doi.org/10.1080/15384047.2016.1210734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5036407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27414086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol.%20Ther.&amp;title=NDRG1%20overexpression%20promotes%20the%20progression%20of%20esophageal%20squamous%20cell%20carcinoma%20through%20modulating%20Wnt%20signaling%20pathway&amp;author=R.%20Ai&amp;author=Y.%20Sun&amp;author=Z.%20Guo&amp;author=W.%20Wei&amp;author=L.%20Zhou&amp;volume=17&amp;publication_year=2016&amp;pages=943-954&amp;pmid=27414086&amp;doi=10.1080/15384047.2016.1210734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B95-biomolecules-14-00969">
<span class="label">95.</span><cite>Ureshino H., Murakami Y., Watari K., Izumi H., Kawahara A., Kage M., Arao T., Nishio K., Yanagihara K., Kinoshita H. N-myc downstream regulated gene 1 (NDRG1) promotes metastasis of human scirrhous gastric cancer cells through epithelial mesenchymal transition. PLoS ONE. 2012;7:e41312.  doi: 10.1371/journal.pone.0041312.</cite> [<a href="https://doi.org/10.1371/journal.pone.0041312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3402489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22844455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=N-myc%20downstream%20regulated%20gene%201%20(NDRG1)%20promotes%20metastasis%20of%20human%20scirrhous%20gastric%20cancer%20cells%20through%20epithelial%20mesenchymal%20transition&amp;author=H.%20Ureshino&amp;author=Y.%20Murakami&amp;author=K.%20Watari&amp;author=H.%20Izumi&amp;author=A.%20Kawahara&amp;volume=7&amp;publication_year=2012&amp;pages=e41312&amp;pmid=22844455&amp;doi=10.1371/journal.pone.0041312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11352417/bin/biomolecules-14-00969-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">biomolecules-14-00969-s001.zip</a><sup> (1.6MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article and its <a href="#app1-biomolecules-14-00969" class="usa-link">Supplementary Information Files</a>. The MS/MS raw data and analysis have been deposited in the ProteomeXchange Consortium via the jPOST partner repository with the data set identifier JPST002506 and PXD049309 (<a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309</a>, accessed on 29 July 2024).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Biomolecules are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.3390/biom14080969" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/biomolecules-14-00969.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.8&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/11352417/" data-citation-style="nlm" data-download-format-link="/resources/citations/11352417/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11352417%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11352417/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11352417/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC11352417/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/39199357/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC11352417/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/39199357/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC11352417/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/11352417/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="abstract1" href="#abstract1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec1-biomolecules-14-00969" href="#sec1-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="1. Introduction">1. Introduction</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec2-biomolecules-14-00969" href="#sec2-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="2. Materials and Methods">2. Materials and Methods</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec3-biomolecules-14-00969" href="#sec3-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="3. Results">3. Results</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec4-biomolecules-14-00969" href="#sec4-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="4. Discussion">4. Discussion</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec5-biomolecules-14-00969" href="#sec5-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="5. Conclusions">5. Conclusions</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="app1-biomolecules-14-00969" href="#app1-biomolecules-14-00969" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary Materials">Supplementary Materials</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes1" href="#notes1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Author Contributions">Author Contributions</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes2" href="#notes2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Institutional Review Board Statement">Institutional Review Board Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes3" href="#notes3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Informed Consent Statement">Informed Consent Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes4" href="#notes4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Data Availability Statement">Data Availability Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes5" href="#notes5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Conflicts of Interest">Conflicts of Interest</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="funding-statement1" href="#funding-statement1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Funding Statement">Funding Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AW9psOAINASdOoyMKCCi6Z0mRt3wEAidJynO5GQ3QI6iN4FyeZ5u6UzZ0rHx9MFU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>