<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EAuthor Manuscript%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="E5FD68FE747948D30668FE000539D125.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="nihpa">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="manuscript">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of proteome research">
<meta name="citation_title" content="Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems">
<meta name="citation_author" content="Zongkai Peng">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States">
<meta name="citation_author" content="Nagib Ahsan">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; Mass Spectrometry, Proteomics and Metabolomics Core Facility, Stephenson Life Sciences Research Center, University of Oklahoma, Norman, Oklahoma 73019, United States">
<meta name="citation_author" content="Zhibo Yang">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States">
<meta name="citation_publication_date" content="2024 Jun 22">
<meta name="citation_volume" content="23">
<meta name="citation_issue" content="7">
<meta name="citation_firstpage" content="2608">
<meta name="citation_doi" content="10.1021/acs.jproteome.4c00338">
<meta name="citation_pmid" content="38907724">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/pdf/nihms-2020672.pdf">
<meta name="description" content="Cell–cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with ...">
<meta name="og:title" content="Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cell–cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11425778">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.1021/acs.jproteome.4c00338" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/nihms-2020672.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11425778%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/11425778/" data-citation-style="nlm" data-download-format-link="/resources/citations/11425778/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nihpa.png" alt="NIHPA Author Manuscripts logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NIHPA Author Manuscripts" title="Link to NIHPA Author Manuscripts" shape="default" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">J Proteome Res</button></div>. Author manuscript; available in PMC: 2024 Sep 26.</div>
<div>
<em>Published in final edited form as: </em>J Proteome Res. 2024 Jun 22;23(7):2608–2618. doi: <a href="https://doi.org/10.1021/acs.jproteome.4c00338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acs.jproteome.4c00338</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Proteome%20Res%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Proteome%20Res%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Proteome%20Res%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Proteome%20Res%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Zongkai Peng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zongkai Peng</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zongkai Peng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahsan%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Nagib Ahsan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nagib Ahsan</span></h3>
<div class="p">
<sup>2</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; Mass Spectrometry, Proteomics and Metabolomics Core Facility, Stephenson Life Sciences Research Center, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahsan%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nagib Ahsan</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Zhibo Yang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Zhibo Yang</span></h3>
<div class="p">
<sup>3</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhibo Yang</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div id="A2">
<sup>2</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; Mass Spectrometry, Proteomics and Metabolomics Core Facility, Stephenson Life Sciences Research Center, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div id="A3">
<sup>3</sup>Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States</div>
<div class="author-notes p"><div class="fn" id="CR1">
<sup>*</sup><p class="display-inline"><strong>Corresponding Authors: Nagib Ahsan</strong> – Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; Mass Spectrometry, Proteomics and Metabolomics Core Facility, Stephenson Life Sciences Research Center, University of Oklahoma, Norman, Oklahoma 73019, United States; <span>nahsan@ou.edu</span>, <strong>Zhibo Yang</strong> – Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; <span>zhibo.yang@ou.edu</span></p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Issue date 2024 Jul 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC11425778&nbsp;&nbsp;NIHMSID: NIHMS2020672&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38907724/" class="usa-link">38907724</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>The publisher's version of this article is available at <a href="https://doi.org/10.1021/acs.jproteome.4c00338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">J Proteome Res</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2 data-anchor-id="ABS1"><a id="ABS1-anchor" data-anchor-id="ABS1" class="usa-anchor"></a>Abstract</h2>
<p id="P1">Cell–cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with drug-sensitive cells to elevate their drug resistance level, which is partially responsible for cancer recurrence. Studying protein targets and pathways involved in cell–cell communication provides essential information for fundamental cell biology studies and therapeutics of human diseases. In the current studies, we performed direct coculture and indirect coculture of drug-resistant and drug-sensitive cell lines, aiming to investigate intracellular proteins responsible for cell communication. Comparative studies were carried out using monoculture cells. Shotgun bottom-up proteomics results indicate that the P53 signaling pathway has a strong association with drug resistance mechanisms, and multiple TP53-related proteins were upregulated in both direct and indirect coculture systems. In addition, cell–cell communication pathways, including the phagosome and the HIF-signaling pathway, contribute to both direct and indirect coculture systems. Consequently, AK3 and H3–3A proteins were identified as potential targets for cell–cell interactions that are relevant to drug resistance mechanisms. We propose that the P53 signaling pathway, in which mitochondrial proteins play an important role, is responsible for inducing drug resistance through communication between drug-resistant and drug-sensitive cancer cells.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> drug resistance, cell–cell communication, quantitative proteomics, direct cell coculture, indirect cell coculture</p></section></section><section class="abstract" id="ABS2"><h2 data-anchor-id="ABS2"><a id="ABS2-anchor" data-anchor-id="ABS2" class="usa-anchor"></a>Graphical Abstract</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c6/11425778/602496163767/nihms-2020672-f0001.jpg" loading="lazy" height="400" width="640" alt="graphic file with name nihms-2020672-f0001.jpg"></p></section><section id="S1"><h2 class="pmc_sec_title" data-anchor-id="S1"><a id="S1-anchor" data-anchor-id="S1" class="usa-anchor"></a>INTRODUCTION</h2>
<p id="P3">Cell–cell interactions play an important role in the development, homeostasis, and function of cells, and they influence multicellular organisms and diseases. These interactions enable communication among cells, allowing them to respond to changes in their microenvironment. Cell–cell interactions are of great interest for cancer research because they are critical in cancer cell progression, invasion, and metastasis. In cancer studies and treatments, the efficacy of chemotherapy is limited by the drug resistance of tumors, resulting in a treatment failure. Drug-resistant cells not only possess resistance by themselves but can facilitate drug-sensitive cells to increase their resistance to drugs through interactions in the microenvironment.<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a>–<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup> In-depth studies of chemoresistance induced by cell–cell interactions can help us to understand tumor recurrence mechanisms and improve clinical treatment strategies. However, the exact mechanisms of resistance in drug-resistant cells and cell–cell communication are not fully understood.<sup><a href="#R4" class="usa-link" aria-describedby="R4" aria-expanded="false">4</a></sup></p>
<p id="P4">To understand cell–cell communication, different mechanisms have been proposed. In respect of species involved in molecular transmission, secreted soluble molecules (e.g., growth factors<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup> and estrogen<sup><a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup>) and extracellular vesicles (e.g., exosomes<sup><a href="#R6" class="usa-link" aria-describedby="R6" aria-expanded="false">6</a></sup>) have been reported. Regarding the interaction distance, both short-distance and long-distance cell–cell communication approaches have been proposed.<sup><a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a>–<a href="#R11" class="usa-link" aria-describedby="R11" aria-expanded="false">11</a></sup> Accordingly, two different types of <em>in vitro</em> experimental models have been developed to study cell–cell communication affected by distances: indirect and direct coculture systems. In indirect coculture systems, cells are separated by different barriers (e.g., membranes<sup><a href="#R12" class="usa-link" aria-describedby="R12" aria-expanded="false">12</a></sup>) or cultured in different containers,<sup><a href="#R13" class="usa-link" aria-describedby="R13" aria-expanded="false">13</a></sup> whereas cell signaling molecules are transmitted through a shared culture medium. These coculture models can be used to mimic long-distance communication. These indirect coculture devices (e.g., Transwell) are commonly used for studies of cell–cell interactions primarily because of their commercial availability, ease-of-use, and excellent reproducibility. These indirect coculture systems are suitable to study certain long-distance communication but cannot effectively mimic short-distance communication. For example, intercellular transport proteins and RNA are distance-sensitive and even contact-dependent factors in actual vivid microenvironments.<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a>,<a href="#R15" class="usa-link" aria-describedby="R15" aria-expanded="false">15</a></sup> Thus, direct coculture systems, in which different types of cells are cultured in the same device with direct cell–cell contact, have been developed to better mimic <em>in vivo</em> systems.</p>
<p id="P5">Although direct coculture systems possess unique advantages, the analysis of different types of cells in these systems requires cells to be distinguished or separated prior to analysis. To achieve this goal, both fluorescence-labeled (e.g., by fluorescent proteins or dyes) cells and regular (i.e., nonlabeled) cells are commonly used for direct coculture, whereas standard fluorescence-based techniques, e.g., fluorescence microscopy and fluorescence-assisted cell sorting (FACS), can be used to distinguish and separate different types of cells in the direct coculture systems.</p>
<p id="P6">We have recently established both direct and indirect coculture systems using drug-sensitive and drug-resistant HCT116 (human colorectal) cancer cell lines in which one of them was labeled with GFP. We then conducted single-cell mass spectrometry (SCMS) experiments to study the influence of drug-resistant cells on the metabolism of drug-sensitive cells. Our results indicated that drug-resistant cells facilitated the drug-sensitive cells to increase their drug resistance.<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a>,<a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> In addition, metabolites’ profiles of drug-sensitive cells were changed (e.g., upregulated sphingomyelin lipids and lactic acid but downregulated TCA cycle intermediate<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup>) after coculture. However, there is a lack of comprehensive understanding of proteins involved in cell–cell communication, particularly distance mediated interactions influencing drug resistance of cancer cells.<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a></sup> In addition, investigating protein targets responsible for the induction of drug resistance potentially provides a great opportunity to discover personalized anticancer drugs,<sup><a href="#R17" class="usa-link" aria-describedby="R17" aria-expanded="false">17</a></sup> such as antibody–drug conjugates (ADCs), to combat drug resistance and cancer recurrence.</p>
<p id="P7">Here, we utilized the GFP-labeled and regular (unlabeled) HCT116 cell lines for both direct and indirect coculture systems for label-free quantitative mass spectrometry (MS) bottom-up proteomics studies (<a href="#F1" class="usa-link">Figure 1</a>). Our primary goal is to study intracellular target proteins involved in long-distance and short-distance cell–cell interactions that are responsible for elevated drug resistance. We are particularly interested in the influence of drug-resistant cancer cells on drug-sensitive cells, which gained increased drug-resistance through coculture with drug-resistant cells. We produced irinotecan (Iri)-resistant HCT116 cells (no GFP labeling) using our established protocols.<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> The Iri-resistant and regular (drug-sensitive) cells were subjected to both direct and indirect cocultures, followed by cell separation and bottom-up quantitative MS proteomics analyses. We discovered a series of proteins responsible for the facilitation of drug resistance in drug-sensitive cells through cell–cell communication. Our systematic proteomic analysis made it possible to find potential protein targets related to drug resistance mechanisms through cell–cell interactions, which might provide new inspiration for cancer treatment and clinical biomarker targets.</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c6/11425778/a57e284dfbda/nihms-2020672-f0002.jpg" loading="lazy" height="1950" width="1816" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P46">Flowchart of the experiments and data analysis. Cell lines include HCT116 drug-sensitive cell lines (H and GH), irinotecan-resistant cell lines (H-Res and GH-Res), and drug-sensitive cell lines from indirect coculture (H-IC) and direct coculture (GH-DC).</p></figcaption></figure></section><section id="S2"><h2 class="pmc_sec_title" data-anchor-id="S2"><a id="S2-anchor" data-anchor-id="S2" class="usa-anchor"></a>EXPERIMENTAL PROCEDURES</h2>
<section id="S3"><h3 class="pmc_sec_title">Monoculture System</h3>
<p id="P8">Monoculture systems were prepared using four cell lines, including two drug-sensitive and two drug-resistant cell lines. Human colorectal cancer HCT116 (H) cells, which were used as the control cells, were purchased from the American Type Culture Collection (ATCC; Rockville, MD, USA). HCT116 cells were cultured at 37 °C with 5% CO<sub>2</sub> in Petri dishes using McCoy’s 5A media supplemented with 10% FBS and 1% Pen Strep. HCT116-GFP (GH) cells, generated from HCT116 cells by lentiviral vector transduction, were purchased from Cellomics Technology (Halethorpe, MD, USA). Puromycin at 1 <em>μ</em>g/mL was added to the complete growth medium to maintain the expression of green fluorescence protein in HCT116-GFP cells. Both H and GH were used as the drug-sensitive cell lines. Irinotecan (Iri) was used as the anticancer drug to generate drug-resistant cell lines (H-Res and GH-Res) by culturing their corresponding parental cells (H and GH) in the complete growth medium containing 1.0 <em>μ</em>M Iri for 20 days. These drug-resistant cells were then cultured in drug-free complete medium for 4 days to deplete intracellular anticancer drug molecules. The medium was changed every 24 h during the depletion process.</p></section><section id="S4"><h3 class="pmc_sec_title">Coculture System</h3>
<p id="P9">Both indirect and direct coculture systems were prepared using one drug-resistant (H-Res) and one drug-sensitive cell line (i.e., H (for indirect coculture) or GH (for direct coculture)).</p></section><section id="S5"><h3 class="pmc_sec_title">Indirect Coculture</h3>
<p id="P10">A Corning Transwell insert (Costar, part no. 3450) and six-well plates (Costar, part no. 3516) were used for the indirect coculture. Cells in this indirect coculture model were separated by a 0.4 <em>μ</em>m membrane, which allowed proteins and small molecules to be transferred through the membrane for communication without direct cell–cell contact between two cell lines. Briefly, 1.5 mL of H-Res cells (5 × 10<sup>5</sup> cells) were added into one Transwell insert, and 2.6 mL of H cells (1 × 10<sup>5</sup> cells) were added into a well of the 6-well plates. Then the inserts containing drug-resistant cells were combined with the 6-well plates containing drug-sensitive cells. After 72 h of incubation (at 37 °C with 5% CO<sub>2</sub>), drug-sensitive cells were collected from the 6-well plates for preparation for bottom-up MS proteomics analysis. These drug-sensitive cells collected from the indirect coculture were referred to as H-IC.</p></section><section id="S6"><h3 class="pmc_sec_title">Direct Coculture</h3>
<p id="P11">GFP-labeled drug-sensitive cells (GH) and drug-resistant cells (H-Res) were directly cocultured in the same 6-well plates, allowing for direct cell–cell contact between two cell lines. Briefly, a mixture of 4 mL of cell culture medium containing GH (1 × 10<sup>5</sup>) and H-Res (5 × 10<sup>5</sup>) cells was added into a well in the 6-well plates and incubated for 72 h (at 37 °C with 5% CO<sub>2</sub>). Then, after removing the culture medium and washing with Gibco Dulbecco’s Phosphate Buffered Saline (DPBS, 14190–250, MilliporeSigma, Rockville, MD), Trypsin EDTA was added to detach the cells. Trypsinization was stopped by adding the complete medium (containing 10% FBS). Cells were transferred into a 1.5 mL LoBind tube (Sarstedt ref. 72.706.600, Nümbrecht, Germany), centrifuged at 500 g at 4 °C for 5 min, washed twice with DPBS, and stored on ice. FACS was used to isolate GH cells from direct coculture, which were referred to as GH-DC. Cells were washed twice with DPBS prior to protein extraction steps.</p></section><section id="S7"><h3 class="pmc_sec_title">Cell Protein Extraction and Sample Preparation for Label-Free MS Proteomics Analysis</h3>
<p id="P12">Cell protein extraction was performed following previously published protocols.<sup><a href="#R18" class="usa-link" aria-describedby="R18" aria-expanded="false">18</a></sup> Briefly, 8.0 M urea solution was added to the cell pellet to extract and denature proteins. Protein concentration was measured at 280 nm using a NanoDrop One instrument (Thermo Fisher Scientific, IL, USA). For each group of cell lines, measurements of three biological replicates were performed. In total, 100 <em>μ</em>g of protein was taken from each sample, and 1.0 <em>μ</em>g of BSA (catalog no. 23225, Thermo Scientific, Waltham, MA) was spiked in as the internal standard. In-solution digestion was performed using Trypsin/Lys-C (catalog no. V5071, Promega, WI, USA). Peptides were desalted using a Waters C18 Sep-Pak cartridge (cat. no. WAT023590, Waters, Milford, MA, USA) and then dried using speed-vac (SVC 100H, Savant) for 8 h. Each sample was resuspended with 200 <em>μ</em>L of mobile phase A (water with 0.1% FA). An equal amount (3.0 <em>μ</em>L) of sample was injected into the LC-MS/MS analysis.</p></section><section id="S8"><h3 class="pmc_sec_title">LC-MS/MS and Data Analysis</h3>
<p id="P13">The LC-MS/MS analysis was performed according to our earlier published protocols<sup><a href="#R18" class="usa-link" aria-describedby="R18" aria-expanded="false">18</a></sup> using a Dionex UltiMate 3000 UHPLC system (Thermo Fisher Scientific, CA, USA) connected to a Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific, Waltham, MA). The resuspended tryptic peptides were loaded onto an in-house packed trap column (150 <em>μ</em>m × 3 cm, packed with Bio-C18 3 <em>μ</em>m resin, Sepax Technologies, DE, USA) using mobile phase A (0.1% formic acid in LC-MS grade water) at a flow rate of 3 <em>μ</em>L/min for 10 min and separated on an analytical column (75 <em>μ</em>m × 30 cm, packed in house with Bio-C18 3 <em>μ</em>m resin, Sepax Technologies) at 350 nL/min. The total LC-MS/MS run time was 90 min including column washing and re-equilibration.</p>
<p id="P14">MS RAW files were searched against the UniProt reviewed human (TaxID: UP000005640) protein database using the Sequest algorithm within Proteome Discoverer v 2.4 (Thermo Fisher Scientific, San Jose, CA). The Sequest database search was performed with the following parameters: trypsin enzyme cleavage specificity, two possible missed cleavages, 10 ppm mass tolerance for precursor ions, and 0.02 Da mass tolerance for fragment ions. Search parameters permitted dynamic modification of methionine oxidation (+15.9949 Da) and static modification of carbamidomethylation (+57.0215 Da) on cysteine. Peptide assignments from the database search were filtered down to a 1% FDR (false discovery rate). Label-free quantitation across the samples employed the Minora algorithm and the adjoining bioinformatics tools available in Proteome Discoverer. A 1.5-fold increase or decrease in abundance with a <em>p</em>-value &lt; 0.05 was considered statistically significant.</p></section><section id="S9"><h3 class="pmc_sec_title">ELISA (Enzyme-Linked Immunosorbent Assay)</h3>
<p id="P15">To verify functional proteins detected in LC-MS/MS experiments, ELISA verification of two selected proteins (adenylate kinase 3 (AK3) and H3 histone 3A (H3–3A)) was performed for both drug-resistant cells (H-Res) and drug-sensitive cells (H and GH) from monoculture systems. Antibodies of AK3 (RDR-AK3-Hu, Reddot Biotech, Houston, TX) and H3–3A (abx387828, Abbexa, Sugar Land, TX) were used to quantify their corresponding cellular proteins, following the protocols provided by their vendors. Briefly, 1 × 10<sup>5</sup> drug-sensitive (H and GH) or 5 × 10<sup>5</sup> drug-resistant (H-Res) cells were seeded in 6-well plates. After 72 h of culture, cells were detached, washed twice using DPBS, and then lysed with 100 <em>μ</em>L of 8.0 M urea. Following protocols from the manufacturers, 300 <em>μ</em>L of dilution buffer from the kit (additional 300 <em>μ</em>L DPBS was added for AK3 measurements) were added in cell lysates prior to the quantification steps. Protein concentration was measured using a NanoDrop One (Thermo Fisher Scientific, Waltham, MA). Each sample was analyzed in triplicate, and standard curves were generated with GraphPad Prism. AK3 and H3–3A concentrations were normalized to the total protein concentration for a fair comparison.</p></section><section id="S10"><h3 class="pmc_sec_title">Bioinformatics</h3>
<p id="P16">All functional and pathway analyses were performed using the ShinyGO 0.77 bioinformatics platform.<sup><a href="#R19" class="usa-link" aria-describedby="R19" aria-expanded="false">19</a></sup> Heatmaps and volcano plots were plotted by SRplot (<a href="https://www.bioinformatics.com.cn/en" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.com.cn/en</a>), a free online platform for data analysis and visualization. Protein–protein interaction analysis of changed proteins was searched through STRING 12.0 (<a href="https://string-db.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://string-db.org/</a>).<sup><a href="#R20" class="usa-link" aria-describedby="R20" aria-expanded="false">20</a></sup> The potential protein pathway was searched through KEGG<sup><a href="#R21" class="usa-link" aria-describedby="R21" aria-expanded="false">21</a>–<a href="#R23" class="usa-link" aria-describedby="R23" aria-expanded="false">23</a></sup> (<a href="https://www.genome.jp/kegg/pathway.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/kegg/pathway.html</a>). A Venn diagram was generated using Molbiotools (<a href="https://molbiotools.com/listcompare.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://molbiotools.com/listcompare.php</a>). An abstract graph and P53 pathway graph were generated using Biorender (<a href="https://www.biorender.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.biorender.com/</a>). All the bar plots was generated with GraphPad Prism version 7.0 for Windows, GraphPad Software, Boston, Massachusetts USA (<a href="https://www.graphpad.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.graphpad.com</a>). The remaining parts were generated by Microsoft PowerPoint and Excel 365.</p></section></section><section id="S11"><h2 class="pmc_sec_title" data-anchor-id="S11"><a id="S11-anchor" data-anchor-id="S11" class="usa-anchor"></a>RESULTS</h2>
<p id="P17">In this study, we prepared both mono- and coculture models using drug-sensitive cells and drug-resistant cells. We then performed global MS proteomics studies of cells to understand cell–cell communication (<a href="#F1" class="usa-link">Figure 1</a>). GFP-labeled cells were used in coculture systems to discriminate different cell types. It is known that GFP labeling has no significant influence on metabolism and functions of cells.<sup><a href="#R24" class="usa-link" aria-describedby="R24" aria-expanded="false">24</a></sup> In addition, our previous studies show that GFP labeling has no significant influence on the drug resistance of cancer cells.<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> To generate drug-resistant cells, irinotecan (Iri, CPT-11) was used to treat drug-sensitive cells (H and GH) for 20 days. In most studies, drug-resistant cancer cells are produced using gradually increasing doses of anticancer drugs for relatively long treatment times (e.g., for 6–12 months or longer), whereas limited studies have been performed to investigate cells with early stage drug resistance.<sup><a href="#R25" class="usa-link" aria-describedby="R25" aria-expanded="false">25</a></sup> Irinotecan is a potent anticancer drug used in the first-line therapy of advanced colorectal cancer, and its activated CPT-11 form, SN-38, binds to topoisomerase 1 and stops DNA replication. Our previous studies indicated that these 20-day resistant cells (H-Res) acquired 5.72-fold increased drug resistance levels compared with their parental cells.<sup><a href="#R25" class="usa-link" aria-describedby="R25" aria-expanded="false">25</a>–<a href="#R27" class="usa-link" aria-describedby="R27" aria-expanded="false">27</a></sup></p>
<p id="P18">A total of six cell lines were obtained from the mono- and coculture systems, including drug-sensitive (H and GH) and drug-resistant (H-Res and GH-Res) cells in monocultures as well as drug-sensitive cells in indirect (H-IC) and direct coculture (RH-DC) with the drug-resistant cell line. A total of 2,542 and 1,983 unique protein groups were successfully identified and quantified in monoculture and coculture systems, respectively. Comprehensive data analyses, including principal component analysis (PCA), heat map clustering, volcano plot illustration, and protein pathway analysis (using KEGG), were performed for all identified proteins.</p>
<section id="S12"><h3 class="pmc_sec_title">Proteome Profiling of Monoculture Cells</h3>
<p id="P19">In total, 12 samples (i.e., three analytical replicates from each of monocultured cell groups (H, GH, H-Res, and GH-Res) were analyzed, and 2,542 unique protein groups were successfully identified and quantified. Proteins’ quantities were normalized. The sequence coverage of spiked BSA (1 <em>μ</em>g of BSA/100 <em>μ</em>g of proteins) is higher than 82% in all monoculture samples, indicating efficient peptide trypsin digestion and sensitive MS detection (<a href="#SD1" class="usa-link">Figure S1</a>). The relative abundances of BSA were compared among different groups of cell samples, and a small standard deviation suggests that the same protein in different groups was accurately quantified for a fair comparison (<a href="#SD1" class="usa-link">Figure S1</a>).</p>
<p id="P20">PCA was used to illustrate the overall profiles of all identified proteins (with normalized abundances by peak areas) from different cell groups. A reasonably good reproducibility among the three analytical replicates was observed (<a href="#F2" class="usa-link">Figure 2A</a>). The influence on GFP labeling on protein expression can be also evaluated using PCA plots. By comparing protein profiles of paired GFP-labeled and nonlabeled cells, it is shown that, compared with the differences between drug-sensitive and drug-resistant cells, GFP labeling has no significant influence on the overall protein expression in drug-sensitive (H vs GH) and drug-resistant (H-Res vs GH-Res) cells (<a href="#F2" class="usa-link">Figure 2A</a>). Thus, GFP-labeled cells are suitable models for both drug-sensitive and drug-resistant cells in coculture systems.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c6/11425778/d1a39678c6e4/nihms-2020672-f0003.jpg" loading="lazy" height="1017" width="1950" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P47">Bioinformatic analyses of proteins relevant to irinotecan resistance in monoculture cell models. (A) Principal component analysis (PCA) of total proteins in each sample. (B) Heat map clustering of 2,542 unique proteins showing differences between the drug-sensitive (H and GH) and drug-resistant (H-Res and GH-Res) cell groups. (C) Volcano plot analyses of the significantly increased and decreased proteins between two groups. (Fold change &gt; 1.5 or &lt; 0.66 and <em>p</em>-value &lt; 0.05). (D) KEEG pathway analysis of all significantly upregulated (fold change &gt; 1.5 and <em>p</em>-value &lt; 0.05) proteins in drug-resistant groups. Red asterisks highlight the pathways subjected to protein network analysis using STRING. (E) Upregulated protein networks of the oxidative phosphorylation pathway, cellular senescence, and P53 signaling pathway (from STRING). (F) P53 signaling pathway response after chemotherapy (upregulated proteins are highlighted in pink boxes).</p></figcaption></figure><p id="P21">Using PCA, significant differences were observed between the drug-sensitive and drug-resistant cells (i.e., H vs H-Res and GH vs GH-Res) in the monoculture systems (<a href="#F2" class="usa-link">Figure 2A</a>). Similarly, the heat map further demonstrated that clustered proteins with differential abundance were observed between the drug-sensitive and drug-resistant cells (<a href="#F2" class="usa-link">Figure 2B</a>). Volcano plots were utilized to illustrate proteins with significant upregulation or downregulation (with fold change &gt; 1.5 or &lt; 0.66, adjusted <em>p-</em>value &lt; 0.05). A total of 159 proteins were significantly upregulated, whereas 55 proteins were downregulated in the drug-resistant group (H-Res) compared with the drug-sensitive (H) group (<a href="#F2" class="usa-link">Figure 2B</a>). We also compared the proteins between GFP-labeled and nonlabeled cells. For drug-sensitive cell lines, 101 and 27 proteins were significantly upregulated and downregulated, respectively, in GH cells compared with H cells. For drug-resistant cell lines, 72 and 46 proteins were upregulated and downregulated, respectively, in the H-Res group compared with the GH-Res group (<a href="#SD1" class="usa-link">Figure S2D</a>). Comparative studies were also carried out between drug-resistant and drug-sensitive cells: the abundances of 255 proteins increased and 93 proteins decreased in the GH-Res group compared with the H group (<a href="#SD1" class="usa-link">Figure S2B</a>). These dysregulated proteins are likely related to both GFP expression and enhancement of drug resistance.</p>
<p id="P22">KEGG pathway analysis was performed for significantly upregulated proteins (&gt;1.5-fold change and <em>p</em>-value &lt; 0.05) in the Iri-resistant group (H-Res) compared to control group (H) (<a href="#F2" class="usa-link">Figure 2D</a>). To further understand protein–protein interactions among significantly upregulated proteins, STRING was used to illustrate the protein network for those involved in oxidative phosphorylation, cellular senescence, and the P53 signaling pathway (<a href="#F2" class="usa-link">Figure 2E</a>). The P53 signaling pathway is a key network that prevents gene mutations. P53 stabilizes genes in cells by interacting with multiple signal transduction pathways that affect cellular processes such as DNA repair and replication. It is known that TP53-related proteins are related to induced drug resistance.<sup><a href="#R28" class="usa-link" aria-describedby="R28" aria-expanded="false">28</a></sup> The abundances of multiple P53 signaling pathway proteins (e.g., RRM2B, CYCS, SERPINB5, CDK1A, and CDK4) were significantly increased (<a href="#SD1" class="usa-link">Figure S3</a>) in drug-resistant cell lines (H-Res and GH-Res) compared with their control groups (H and GH), indicating a strong association of P53 signaling pathways (<a href="#F2" class="usa-link">Figure 2F</a>) in the Iri resistance mechanisms.</p></section><section id="S13"><h3 class="pmc_sec_title">Proteome Profiling of Direct and Indirect Coculture Cells</h3>
<p id="P23">Our studies are focused on the influence of drug-resistant cells on drug-sensitive cells through cell–cell interactions. Thus, only drug-sensitive cells in the direct coculture (GH-DC) and indirect coculture (H-IC) were collected. In total, 1,983 proteins (&lt;1% FDR) were successfully identified and quantified in GH-DC and H-IC cells. We then determined proteins with significantly changed abundances (fold change &gt; 1.5 or &lt; 0.66, <em>p</em>-value &lt; 0.05). A total of 296 (253 upregulated and 43 downregulated) (<a href="#F3" class="usa-link">Figure 3A</a>) and 158 (58 upregulated and 100 downregulated) (<a href="#F3" class="usa-link">Figure 3E</a>) proteins were significantly changed in the direct (GH-DC) and indirect coculture (H-IC) models, respectively. Interestingly, some proteins (e.g., SRP14 and ECHDC1) showed coculture-specific alternation, i.e., they were not detected in monoculture systems, suggesting that they are relevant to the acquisition of drug resistance through cell–cell communication.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c6/11425778/9b1c144b8512/nihms-2020672-f0004.jpg" loading="lazy" height="1950" width="1195" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P48">Bioinformatic analyses of proteomics data obtained from direct (A–D) and indirect coculture (E–H) models. (A and E) Volcano plot of significantly upregulated proteins (<em>p</em>-value &lt; 0.05 and FC &gt; 1.5) in drug-sensitive cells from direct coculture (A) and indirect coculture (E). (B and F) Significantly upregulated pathways in direct coculture (B) and indirect coculture (F). Red asterisks highlight the pathways that are important to these coculture systems: oxidative phosphorylation was found upregulated in drug-resistant cells as well; phagosome shows the activity of those membrane ATP related proteins; and the HIF-1 signaling pathway and TCA cycle proteins have direct interactions with TP53 proteins. (C and G) Upregulated proteins relevant to TP53 pathway transcriptional regulation in direct coculture (C) and indirect coculture (G). (D and H) Heat map of regulated proteins in TP53 transcriptional regulation in direct coculture (D) and TP53 transcriptional regulation and TCA cycle in indirect coculture (H).</p></figcaption></figure></section><section id="S14"><h3 class="pmc_sec_title">Direct Coculture Cell System</h3>
<p id="P24">GFP-labeled drug-sensitive cells (GH-DC) were isolated from the direct coculture system containing drug-resistant cells (H-Res) (<a href="#SD1" class="usa-link">Figure S4A</a>) using FACS. Drug-resistant cells (H-Res) can interact with the drug-sensitive cells (GH) through cell–cell contact and cell–matrix interactions in the direct coculture system, allowing the drug-sensitive cells to change their protein profiles and construct new pathways with elevated drug resistance. The volcano plot illustrated 253 upregulated and 43 downregulated proteins in GH-DC cells compared to the control (GH) (<a href="#SD1" class="usa-link">Figure S4B</a>). Similar to that observed in the Iri-resistant cells (H-Res), Reactome pathway analysis of the upregulated proteins revealed that the oxidative phosphorylation pathway was also upregulated in direct coculture cells (GH-DC). This pathway has a strong association with P53 signaling pathways (<a href="#F3" class="usa-link">Figure 3B</a>). GH-DC cells also activated transcriptional regulation by TP53 pathways, and the relevant proteins (MBD3, HDAC1, CARM1, NPM1, PPP2R1B, RFC3, RFC2, RFC5, and TNRSF10A) were upregulated (<a href="#F3" class="usa-link">Figure 3C</a>). These results indicate that transcriptional regulation by TP53 is important for drug-sensitive cells to gain drug resistance from drug-resistant cells through direct coculture. In particular, upregulated TNFRSF10A (DR4),<sup><a href="#R29" class="usa-link" aria-describedby="R29" aria-expanded="false">29</a></sup> a surface receptor in tumor cells, indicates that this protein is likely a target for cell–cell communication (<a href="#F3" class="usa-link">Figure 3D</a>). In addition, the phagosome pathway was upregulated, and multiple relevant proteins (e.g., LAMP2, ATP6 V1E1, TUBAL3, TUBB3, TUBB6, and TUBB8) were overexpressed (<a href="#F3" class="usa-link">Figure 3B</a>).</p></section><section id="S15"><h3 class="pmc_sec_title">Indirect Coculture Cell System</h3>
<p id="P25">Drug-sensitive cells (H-IC) were harvested from the Transwell device used in an indirect coculture with drug-resistant cells (H-Res) (<a href="#F3" class="usa-link">Figure 3F</a>). A membrane (0.4 <em>μ</em>m pore size) separated these two different types of cells, whereas molecular species can be transported for only cell–matrix (including metabolites, proteins, and hormone<sup><a href="#R30" class="usa-link" aria-describedby="R30" aria-expanded="false">30</a></sup>) interactions without cell–cell contact, allowing cells to gain enhanced drug resistance. A total of 58 proteins were upregulated, whereas 100 proteins were down-regulated in the H-IC cells (<a href="#F3" class="usa-link">Figure 3E</a>). Multiple pathways (e.g., citrate cycle (TCA cycle) and HIF-1 signaling pathway) were upregulated (<a href="#F3" class="usa-link">Figure 3F</a>). Among them, ALDOA and ALDOC, which play significant roles in glycolysis and fructolysis steps for synthesis of ATP through TCA cycle,<sup><a href="#R31" class="usa-link" aria-describedby="R31" aria-expanded="false">31</a></sup> were upregulated in the HIF- 1 signaling pathway, resulting in a reduced oxygen consumption process (<a href="#F3" class="usa-link">Figures 3F</a> and <a href="#SD1" class="usa-link">S5</a>).</p>
<p id="P26">Similar to the trend observed from the direct coculture cells (GH-DC), the TP53-related protein–protein network was found to correlate with the interactions with drug-resistant cells in the indirect coculture cells. In H-IC cells, the upregulated proteins, including membrane glucose transporter (SLC2A1), isocitrate dehydrogenase (NADP(+)) 2 (IDH2), and histone proteins (H1–0, H4C1, and H3–3A), were directly related to TP53 proteins (<a href="#F3" class="usa-link">Figures 3G</a> and <a href="#F3" class="usa-link">3H</a>).</p></section><section id="S16"><h3 class="pmc_sec_title">Potential Protein Targets in Drug Resistance and Cell–Cell Communication</h3>
<p id="P27">In this study, the P53 signaling pathway was found to be upregulated and strongly correlated with drug resistance mechanisms in Iri-resistant cells (<a href="#F2" class="usa-link">Figures 2D</a> and <a href="#F2" class="usa-link">2F</a>). As the common proteins relevant to Iri resistance, AK3 and H3–3A were significantly upregulated in monoculture (H-Res), indirect coculture (H-IDC), and direct coculture (GH-DC) (<a href="#F4" class="usa-link">Figure 4A</a>). Overexpressed AK3 and H3–3A in coculture cells indicates that Iri resistance can be induced through molecular transfer during cell–cell communication.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c6/11425778/4245cd8e1eb2/nihms-2020672-f0005.jpg" loading="lazy" height="1556" width="1950" alt="Figure 4."></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P49">Significantly upregulated proteins relevant to Iri resistance. (A) Venn diagram of significantly upregulated (FC &gt; 1.5 and <em>p</em>-value &lt; 0.05) proteins in direct coculture (GH-DC), indirect coculture (H-IC), and drug-resistant (H-Res) cells. (B and C) ELISA quantification of AK3 (B) and H3–3A (C) proteins in Iri-resistant (H-Res) and control (H and GH) cells. (****<em>p</em>-value &lt; 0.0001; *<em>p</em>-value &lt; 0.05; ns, <em>p</em>-value &gt; 0.05).</p></figcaption></figure><p id="P28">To verify results obtained from label-free MS quantitative proteomics experiments, ELISA experiments were carried out for two selected proteins (H3–3A and AK3). Similar to MS results, ELISA measurements provided the same trends: both H3–3A and AK3 proteins were highly expressed in drug-resistant cells (H-Res) compared with control groups (H and GH) (<a href="#F4" class="usa-link">Figures 4B</a> and <a href="#F4" class="usa-link">4C</a>). The AK3 protein, a GTP:ATP phosphotransferase present in the mitochondrial matrix, is responsible for purine homeostasis. AK3 is directly related to both P53 (via RRM2B proteins) and oxidative phosphorylation pathways (<a href="#F2" class="usa-link">Figure 2E</a>), and the latter was also upregulated in direct coculture systems (<a href="#F2" class="usa-link">Figure 2D</a>), indicating AK3 is a potential biomarker for cell–cell communication of drug resistance. Multiple histones, including H3–3A, H1–0, and H4C1, are directly associated with TP53 proteins. These histones were upregulated in indirect coculture cells (GH-DC) (<a href="#F3" class="usa-link">Figure 3G</a>), contributing to the induced drug resistance through indirect interactions with drug resistance associated with the P53 signaling pathways.</p></section></section><section id="S17"><h2 class="pmc_sec_title" data-anchor-id="S17"><a id="S17-anchor" data-anchor-id="S17" class="usa-anchor"></a>DISCUSSION</h2>
<section id="S18"><h3 class="pmc_sec_title">Drug Resistance Related Mechanisms Observed in Monoculture Systems</h3>
<p id="P29">Our previous studies indicated that the drug resistance level of HCT116 cells (H; IC<sub>50</sub> = 2.90 ± 0.1 <em>μ</em>M) can be significantly enhanced (H-IC; IC<sub>50</sub> = 10.7 ± 1.9 <em>μ</em>M) through indirect coculture with 20-day Iri-resistant HCT116 cells (H-Res; IC<sub>50</sub> = 16.6 ± 1.4 <em>μ</em>M).<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> In addition, numerous lipids, including ceramides and sphingolipids, were significantly upregulated.<sup><a href="#R32" class="usa-link" aria-describedby="R32" aria-expanded="false">32</a></sup> In the current studies, we observed that several proteins associated with the sphingolipid signaling pathway (e.g., <em>N-</em>acylsphingosine amidohydrolase 1 (ASAH1) and ceramide synthase 2 (CERS2)) were abundant in Iri-resistant cell lines. ASAH1 gene-encoded protein catalyzes the degradation of ceramide and fatty acids, whereas CERS2 catalyzes the synthesis of very long acyl chain ceramides. It has been reported that radiation leads to ASAH1 overexpression in glioblastoma cells;<sup><a href="#R33" class="usa-link" aria-describedby="R33" aria-expanded="false">33</a>,<a href="#R34" class="usa-link" aria-describedby="R34" aria-expanded="false">34</a></sup> thus, inhibiting the activity of ASAH1 results in glioblastoma cell death.<sup><a href="#R35" class="usa-link" aria-describedby="R35" aria-expanded="false">35</a></sup> These studies suggest that the ASAH1 protein is relevant to the development of radioresistance and recurrency of glioblastoma. Similarly, CERS2 expression is correlated to the degree of recurrence of numerous cancers.<sup><a href="#R36" class="usa-link" aria-describedby="R36" aria-expanded="false">36</a></sup> Because ASAH1 and CERS2 affect the synthesis of sphingolipids and ceramides, the production of these lipids is likely critical for cells to establish resistance to the anticancer drug Iri. In fact, our previous studies indicate that a large number of ceramides and sphingolipids were upregulated in drug-resistant cells and cells cocultured with drug-resistant cells.<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> Thus, we have experimental evidence from both proteomics and metabolomics studies to demonstrate that the regulation of sphingolipids and ceramides is related to drug resistance.</p>
<p id="P30">As we previously demonstrated, cell-cycle arrest, senescence, and apoptosis are the three main outcomes of P53 activation.<sup><a href="#R37" class="usa-link" aria-describedby="R37" aria-expanded="false">37</a></sup> Cell-cycle arrest allows cell repair and recovery from the stress (i.e., drug treatment) so that cell survival occurs, and resistance is gained. As observed in the monoculture systems, the P53 signaling pathway and relevant proteins, including cyclin-dependent kinase 4 (CDK4), cyclin-dependent kinase inhibitor 1A (CDKN1A), and cytochrome c (CYCS), were significantly upregulated in drug-resistant cells (<a href="#F2" class="usa-link">Figures 2C</a> and <a href="#SD1" class="usa-link">3S</a>). CDK4 affects the cell cycle pathway by progressing it from the G1 to the S phase, in which DNA synthesis occurs.<sup><a href="#R38" class="usa-link" aria-describedby="R38" aria-expanded="false">38</a></sup> CDK4 is upregulated in multiple tumors, and its hyperactivity leads to drug resistance. In contrast, suppressing the expression or activity of CDK4 by its inhibitors (e.g., palbociclib, ribociclib, and abemaciclib<sup><a href="#R39" class="usa-link" aria-describedby="R39" aria-expanded="false">39</a></sup>) is regarded as an effective strategy for cancer treatment.<sup><a href="#R40" class="usa-link" aria-describedby="R40" aria-expanded="false">40</a>,<a href="#R41" class="usa-link" aria-describedby="R41" aria-expanded="false">41</a></sup> CYCS is among proteins (&gt;300) involved in the oxidative phosphorylation processes. These proteins are encoded by nuclear and mitochondrial genes, demonstrating that mitochondria play a key role during the drug resistance mechanism.<sup><a href="#R42" class="usa-link" aria-describedby="R42" aria-expanded="false">42</a></sup> Increased CYCS is correlated with drug resistance and poor survival of cancer (e.g., lymphoma<sup><a href="#R43" class="usa-link" aria-describedby="R43" aria-expanded="false">43</a></sup>). Thus, it is not surprising that several proteins associated with the P53 pathway, including TP53I3, CDKN1A, CDK4, CYCS, and SERPINB5 proteins, were increased in resistant cells (H-Res and GH-Res). This provides the drug-resistant cells with a better chance to survive after treatment.</p>
<p id="P31">We also observed other potential pathways related to Iri resistance. For example, propanoate metabolism governs propionic acid metabolism, which is enriched in metastatic tumors to increase metastatic aggressiveness.<sup><a href="#R44" class="usa-link" aria-describedby="R44" aria-expanded="false">44</a></sup> Multiple proteins involved in the propanoate metabolism pathway were identified in Iri-resistant cells, indicating that they are correlated with the drug-resistance mechanism. These proteins include the hydroxyacyl-CoA dehydrogenase alpha subunit (HADHA), the cytosolic enzyme ethylmalonyl-CoA decarboxylase (ECHDC1), and 3-hydroxyisobutyryl-CoA hydrolase (HIBCH). It has been reported that HADAH is overexpressed in cisplatin-resistant cancer cells.<sup><a href="#R45" class="usa-link" aria-describedby="R45" aria-expanded="false">45</a></sup> ECHDC1 is one of the side products of acetyl-CoA carboxylase.<sup><a href="#R46" class="usa-link" aria-describedby="R46" aria-expanded="false">46</a></sup> The expression of ECHDC1 is significantly increased in the gemcitabine-resistant cells.<sup><a href="#R47" class="usa-link" aria-describedby="R47" aria-expanded="false">47</a></sup> High expression of HIBCH is correlated with poor survival in patients with colorectal cancer.<sup><a href="#R48" class="usa-link" aria-describedby="R48" aria-expanded="false">48</a></sup></p></section><section id="S19"><h3 class="pmc_sec_title">Short-Distance Communication</h3>
<p id="P32">Cell–cell communication can be classified into four types: autocrine signaling (secreted signaling molecules bind to the same cell), paracrine signaling (secreted signaling molecules act on nearby cells), endocrine signaling (secreted signaling molecules are carried through the circulation to act on distant cells), and juxtacrine (signaling processes require close contact).<sup><a href="#R30" class="usa-link" aria-describedby="R30" aria-expanded="false">30</a></sup> According to the distance of molecular transfer between cells, cell–cell communication can be classified as short- and long-distance communication.<sup><a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a></sup> In the short-distance communication, cells communicate with each other by sending and accepting signal molecules (e.g., ligands, nitric oxide, and <em>P-</em>glycoproteins<sup><a href="#R49" class="usa-link" aria-describedby="R49" aria-expanded="false">49</a></sup>) over short distances or through physical contact. The <em>in vitro</em> direct coculture systems are regarded as vivid models mimicking short-distance communication (i.e., paracrine and juxtacrine signaling) in <em>in vivo</em> microenvironments. Our previous studies showed that drug-sensitive cells gained drug resistance after being cocultured with drug-resistant cells through cell–cell and cell–matrix communication. Cell metabolites and related pathways were significantly changed (i.e., upregulated lactic acid and downregulated TCA cycle intermediate) after coculture.<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> Our current work indicates that the expression of some proteins relevant to Iri resistance (e.g., SRP14 and ECHDC1) were altered only in cocultured cells (H-IC and GH-DC), suggesting that the induction of drug resistance was through cell–cell communication.</p>
<p id="P33">We have observed multiple pathways that likely contribute to short-distance communication in direct coculture models. For example, phagosome and oxidative phosphorylation pathways were significantly upregulated in the direct coculture system (<a href="#F3" class="usa-link">Figure 3B</a>). In the phagosome process, upregulated vacuolar adenosine triphosphatases (V-ATPases), including ATP6 V1V1 and ATP6 V1G1, enhance drug resistance of cancer cells by reducing anticancer drug uptake or trapping drug molecules in acidic vesicles.<sup><a href="#R29" class="usa-link" aria-describedby="R29" aria-expanded="false">29</a>,<a href="#R50" class="usa-link" aria-describedby="R50" aria-expanded="false">50</a></sup></p>
<p id="P34">Using Reactome pathway analysis of results obtained from direct coculture cells (GH-DC), we also discovered that tumor protein 53 (TP53) and tumor necrosis factor receptor superfamily member 10A (TNFRSF10A) were upregulated (<a href="#F3" class="usa-link">Figure 3D</a>). In fact, multiple upregulated proteins, including AK3, H3–3A, and TNFRSF10A, have direct relations with the TP53 protein, indicating the importance of P53 signaling pathways in direct coculture systems through short-distance communication.</p></section><section id="S20"><h3 class="pmc_sec_title">Long-Distance Communication</h3>
<p id="P35">In long-distance communication, cells usually send signaling molecules using circulatory systems such as the bloodstream. For <em>in vitro</em> models, a physical barrier (i.e., membrane) is needed to mimic this endocrine signaling. The indirect coculture model used in the current work involves culturing two types of cells (i.e., drug-sensitive and drug-resistant cells) in the same culture medium, whereas these types of cells were separated by using a Transwell (0.4 <em>μ</em>m membrane) device. After 72 h of indirect coculture, our results indicated that the TCA cycle, which plays an important role in drug resistance development,<sup><a href="#R51" class="usa-link" aria-describedby="R51" aria-expanded="false">51</a></sup> was significantly upregulated in the drug-sensitive cells. The IC<sub>50</sub> data shows that the IC<sub>50</sub> value of the sensitive cells increases three times after indirect coculture (H-IC, 10.7 <em>μ</em>M vs H, 2.9 <em>μ</em>M), indicating an effective transfer of resistance to the sensitive cells.<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup> In addition, Aconitase 2 (ACO2), which is a protein that catalyzes the interconversion of citrate to isocitrate in the TCA cycle, was upregulated (<a href="#F2" class="usa-link">Figure 2 H</a>). According to previous studies, ACO2 is a potential immunotherapeutic biomarker for multiple cancer types, including colon carcinoma.<sup><a href="#R52" class="usa-link" aria-describedby="R52" aria-expanded="false">52</a></sup> Associated with the TCA cycle, hypoxia-inducible factor 1 (HIF-1) was also upregulated (<a href="#F2" class="usa-link">Figure 2F</a>). HIF-1 is responsible for sensing and adapting the cellular response to oxygen availability. This indicates the Warburg effect and shows that cells have an increased glycolysis activity. Eventually, the proteins encoded by the HIF-1 gene increase O<sub>2</sub> delivery, enabling cells to adapt to O<sub>2</sub> deprivation. As observed in multiple human cancers, activation of HIF-1 signaling increases multidrug resistance by enhancing the proliferation, invasion, and migration of tumor cells.<sup><a href="#R53" class="usa-link" aria-describedby="R53" aria-expanded="false">53</a></sup> In the HIF-1 signaling pathway, glucose transporter member 1 GLUT1 (SLC2A1), a glucose transporter on the cell membrane controlling glucose concentration, was found to be highly upregulated after indirect coculture (<a href="#F2" class="usa-link">Figure 2H</a>). Previous studies indicate that SLC2A1 is highly expressed in many cancers.<sup><a href="#R54" class="usa-link" aria-describedby="R54" aria-expanded="false">54</a></sup> In addition, we have observed other upregulated proteins, such as isocitrate dehydrogenase [NADP] IDH2 and histone proteins (e.g., H3–3A), directly related to the TP53 protein in this indirect protein–protein interaction network.</p></section></section><section id="S21"><h2 class="pmc_sec_title" data-anchor-id="S21"><a id="S21-anchor" data-anchor-id="S21" class="usa-anchor"></a>CONCLUSION</h2>
<p id="P36">Overall, we studied intracellular proteins involved in cell–cell communication in two coculture models (indirect and direct cocultures) using drug-sensitive cells (i.e., HCT116 and HCT116GFP) and drug-resistant cells (i.e., IRI-HCT116 and IRI-HCT116-GFP). Label-free bottom-up shotgun proteomics results show that the P53 signaling pathway has a strong association with Iri drug resistance mechanisms. It is fascinating that P53 signaling pathway proteins were upregulated in both direct and indirect coculture systems, indicating that P53 contributes to the facilitation of drug resistance in drug-sensitive cells through their communication with drug-resistant cells. Upregulated pathways (e.g., oxidative phosphorylation) were observed in both direct coculture and drug-resistant cells, showing that they have a strong association with short-distance communication activities and could be the potential pathway of the Iri resistance mechanism. As for the long-distance communication model, the energy-related TCA cycle and HIF signaling pathways were upregulated in indirect coculture systems. We propose protein targets (e.g., AK3 and H3–3A) contribute to cell–cell communication for drug-resistance information; by inhibiting these proteins, it may slow down recurrence for colon cancer Iri treatments. Our current studies are focused on the analysis of intracellular proteins in drug-sensitive cancer cells affected by drug-resistant cancer cells through cell–cell communications. Although extracellular species (e.g., growth factors and extracellular vesicles) are critical for cell–cell communication, these studies are beyond the scope of the current work.</p></section><section id="SM1"><h2 class="pmc_sec_title" data-anchor-id="SM1"><a id="SM1-anchor" data-anchor-id="SM1" class="usa-anchor"></a>Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>SI</span></div>
<div class="media p" id="d67e776"><div class="caption">
<a href="/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2020672-supplement-SI.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section><section id="S23" class="ack"><h2 class="pmc_sec_title" data-anchor-id="S23"><a id="S23-anchor" data-anchor-id="S23" class="usa-anchor"></a>ACKNOWLEDGMENTS</h2>
<p id="P38">This work was supported by funds from National Science Foundation (2305182), the National Institutes of Health (1R01AI177469), and the University of Oklahoma’s Office of the Vice President for Research and Partnerships.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN1">
<p id="P39">Supporting Information</p>
<p id="P40">The Supporting Information is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338</a>.</p>
<p id="P41"><a href="#SD1" class="usa-link">Figure S1</a>, quality control of label-free quantitative proteomics; <a href="#SD1" class="usa-link">Figure S2</a>, comparison between the monoculture drug-resistant and drug-sensitive cells; <a href="#SD1" class="usa-link">Figure S3</a>, P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a href="#SD1" class="usa-link">Figure S4</a>, workflow of the direct coculture and indirect coculture; and <a href="#SD1" class="usa-link">Figure S5</a>, upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p>
<p id="P42">All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p>
<p id="P420">Volcano plot of proteins in monoculture and coculture cells (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p>
<p id="P43">Enriched upregulated KEGG pathways in monoculture and coculture systems (<a href="https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p>
</div>
<div class="fn p" id="FN2"><p id="P44">The authors declare no competing financial interest.</p></div>
<div class="fn p" id="FN3"><p id="P45">Complete contact information is available at: <a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/10.1021/acs.jproteome.4c00338</a></p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title" data-anchor-id="_ci93_"><a id="_ci93_-anchor" data-anchor-id="_ci93_" class="usa-anchor"></a>Contributor Information</h2>
<p>Zongkai Peng, Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States.</p>
<p>Nagib Ahsan, Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States; Mass Spectrometry, Proteomics and Metabolomics Core Facility, Stephenson Life Sciences Research Center, University of Oklahoma, Norman, Oklahoma 73019, United States.</p>
<p>Zhibo Yang, Department of Chemistry and Biochemistry, University of Oklahoma, Norman, Oklahoma 73019, United States.</p></section><section id="S22"><h2 class="pmc_sec_title" data-anchor-id="S22"><a id="S22-anchor" data-anchor-id="S22" class="usa-anchor"></a>Data Availability Statement</h2>
<p id="P37">Proteomics data can be found at the MassIVE database via MSV000092944.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">(1).</span><cite>Brücher BL; Jamall IS
Cell-cell communication in the tumor microenvironment, carcinogenesis, and anticancer treatment. Cellular Physiology and Biochemistry
2014, 34 (2), 213–243.
</cite> [<a href="https://doi.org/10.1159/000362978" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25034869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cellular%20Physiology%20and%20Biochemistry&amp;title=Cell-cell%20communication%20in%20the%20tumor%20microenvironment,%20carcinogenesis,%20and%20anticancer%20treatment&amp;author=BL%20Br%C3%BCcher&amp;author=IS%20Jamall&amp;volume=34&amp;issue=2&amp;publication_year=2014&amp;pages=213-243&amp;pmid=25034869&amp;doi=10.1159/000362978&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">(2).</span><cite>Frankfurt OS; Seckinger D; Sugarbaker EV
Intercellular transfer of drug resistance. Cancer Research
1991, 51 (4), 1190–1195.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1997161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Intercellular%20transfer%20of%20drug%20resistance&amp;author=OS%20Frankfurt&amp;author=D%20Seckinger&amp;author=EV%20Sugarbaker&amp;volume=51&amp;issue=4&amp;publication_year=1991&amp;pages=1190-1195&amp;pmid=1997161&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">(3).</span><cite>Emond R; Griffiths JI; Grolmusz VK; Nath A; Chen J; Medina EF; Sousa RS; Synold T; Adler FR; Bild AH
Cell facilitation promotes growth and survival under drug pressure in breast cancer. Nat. Commun
2023, 14 (1), 3851.
</cite> [<a href="https://doi.org/10.1038/s41467-023-39242-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10310817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37386030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Cell%20facilitation%20promotes%20growth%20and%20survival%20under%20drug%20pressure%20in%20breast%20cancer&amp;author=R%20Emond&amp;author=JI%20Griffiths&amp;author=VK%20Grolmusz&amp;author=A%20Nath&amp;author=J%20Chen&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=3851&amp;pmid=37386030&amp;doi=10.1038/s41467-023-39242-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">(4).</span><cite>Housman G; Byler S; Heerboth S; Lapinska K; Longacre M; Snyder N; Sarkar S
Drug resistance in cancer: an overview. Cancers
2014, 6 (3), 1769–1792.
</cite> [<a href="https://doi.org/10.3390/cancers6031769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4190567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25198391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Drug%20resistance%20in%20cancer:%20an%20overview&amp;author=G%20Housman&amp;author=S%20Byler&amp;author=S%20Heerboth&amp;author=K%20Lapinska&amp;author=M%20Longacre&amp;volume=6&amp;issue=3&amp;publication_year=2014&amp;pages=1769-1792&amp;pmid=25198391&amp;doi=10.3390/cancers6031769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">(5).</span><cite>Korkaya H; Liu S; Wicha MS
Breast cancer stem cells, cytokine networks, and the tumor microenvironment. J. Clin. Invest
2011, 121 (10), 3804–3809.
</cite> [<a href="https://doi.org/10.1172/JCI57099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3223613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21965337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&amp;title=Breast%20cancer%20stem%20cells,%20cytokine%20networks,%20and%20the%20tumor%20microenvironment&amp;author=H%20Korkaya&amp;author=S%20Liu&amp;author=MS%20Wicha&amp;volume=121&amp;issue=10&amp;publication_year=2011&amp;pages=3804-3809&amp;pmid=21965337&amp;doi=10.1172/JCI57099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">(6).</span><cite>Lindoso RS; Collino F; Vieyra A
Extracellular vesicles as regulators of tumor fate: crosstalk among cancer stem cells, tumor cells and mesenchymal stem cells. Stem Cell Investigation
2017, 4 (9), 75–75.
</cite> [<a href="https://doi.org/10.21037/sci.2017.08.08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5639036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29057247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Investigation&amp;title=Extracellular%20vesicles%20as%20regulators%20of%20tumor%20fate:%20crosstalk%20among%20cancer%20stem%20cells,%20tumor%20cells%20and%20mesenchymal%20stem%20cells&amp;author=RS%20Lindoso&amp;author=F%20Collino&amp;author=A%20Vieyra&amp;volume=4&amp;issue=9&amp;publication_year=2017&amp;pages=75-75&amp;pmid=29057247&amp;doi=10.21037/sci.2017.08.08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">(7).</span><cite>Vis MA; Ito K; Hofmann S
Impact of culture medium on cellular interactions in in vitro co-culture systems. Frontiers in bioengineering and biotechnology
2020, 8, 911.
</cite> [<a href="https://doi.org/10.3389/fbioe.2020.00911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7417654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32850750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20bioengineering%20and%20biotechnology&amp;title=Impact%20of%20culture%20medium%20on%20cellular%20interactions%20in%20in%20vitro%20co-culture%20systems&amp;author=MA%20Vis&amp;author=K%20Ito&amp;author=S%20Hofmann&amp;volume=8&amp;publication_year=2020&amp;pages=911&amp;pmid=32850750&amp;doi=10.3389/fbioe.2020.00911&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">(8).</span><cite>Pandurangan M; Hwang I
Application of cell co-culture system to study fat and muscle cells. Applied microbiology and biotechnology
2014, 98, 7359–7364.
</cite> [<a href="https://doi.org/10.1007/s00253-014-5935-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25038928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20microbiology%20and%20biotechnology&amp;title=Application%20of%20cell%20co-culture%20system%20to%20study%20fat%20and%20muscle%20cells&amp;author=M%20Pandurangan&amp;author=I%20Hwang&amp;volume=98&amp;publication_year=2014&amp;pages=7359-7364&amp;pmid=25038928&amp;doi=10.1007/s00253-014-5935-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">(9).</span><cite>Kuppusamy P; Kim D; Soundharrajan I; Hwang I; Choi K
Adipose and Muscle Cell Co-Culture System: A Novel In Vitro Tool to Mimic the In Vivo Cellular Environment. Biology
2021, 10, 6.</cite> [<a href="https://doi.org/10.3390/biology10010006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7823969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33374127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology&amp;title=Adipose%20and%20Muscle%20Cell%20Co-Culture%20System:%20A%20Novel%20In%20Vitro%20Tool%20to%20Mimic%20the%20In%20Vivo%20Cellular%20Environment&amp;author=P%20Kuppusamy&amp;author=D%20Kim&amp;author=I%20Soundharrajan&amp;author=I%20Hwang&amp;author=K%20Choi&amp;volume=10&amp;publication_year=2021&amp;pages=6&amp;pmid=33374127&amp;doi=10.3390/biology10010006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">(10).</span><cite>Goers L; Freemont P; Polizzi KM
Co-culture systems and technologies: taking synthetic biology to the next level. Journal of The Royal Society Interface
2014, 11 (96), No. 20140065.
</cite> [<a href="https://doi.org/10.1098/rsif.2014.0065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4032528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24829281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20The%20Royal%20Society%20Interface&amp;title=Co-culture%20systems%20and%20technologies:%20taking%20synthetic%20biology%20to%20the%20next%20level&amp;author=L%20Goers&amp;author=P%20Freemont&amp;author=KM%20Polizzi&amp;volume=11&amp;issue=96&amp;publication_year=2014&amp;pages=20140065&amp;pmid=24829281&amp;doi=10.1098/rsif.2014.0065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">(11).</span><cite>Borciani G; Montalbano G; Baldini N; Cerqueni G; Vitale-Brovarone C; Ciapetti G
Co–culture systems of osteoblasts and osteoclasts: Simulating in vitro bone remodeling in regenerative approaches. Acta biomaterialia
2020, 108, 22–45.
</cite> [<a href="https://doi.org/10.1016/j.actbio.2020.03.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32251782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20biomaterialia&amp;title=Co%E2%80%93culture%20systems%20of%20osteoblasts%20and%20osteoclasts:%20Simulating%20in%20vitro%20bone%20remodeling%20in%20regenerative%20approaches&amp;author=G%20Borciani&amp;author=G%20Montalbano&amp;author=N%20Baldini&amp;author=G%20Cerqueni&amp;author=C%20Vitale-Brovarone&amp;volume=108&amp;publication_year=2020&amp;pages=22-45&amp;pmid=32251782&amp;doi=10.1016/j.actbio.2020.03.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">(12).</span><cite>Li W; Khan M; Mao S; Feng S; Lin J-M
Advances in tumor-endothelial cells co-culture and interaction on microfluidics. Journal of Pharmaceutical Analysis
2018, 8 (4), 210–218.
</cite> [<a href="https://doi.org/10.1016/j.jpha.2018.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30140484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20Analysis&amp;title=Advances%20in%20tumor-endothelial%20cells%20co-culture%20and%20interaction%20on%20microfluidics&amp;author=W%20Li&amp;author=M%20Khan&amp;author=S%20Mao&amp;author=S%20Feng&amp;author=J-M%20Lin&amp;volume=8&amp;issue=4&amp;publication_year=2018&amp;pages=210-218&amp;pmid=30140484&amp;doi=10.1016/j.jpha.2018.07.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">(13).</span><cite>Renaud J; Martinoli M-G
Development of an insert co-culture system of two cellular types in the absence of cell-cell contact. JoVE (Journal of Visualized Experiments)
2016, 113, No. e54356.</cite> [<a href="https://doi.org/10.3791/54356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27500972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JoVE%20(Journal%20of%20Visualized%20Experiments)&amp;title=Development%20of%20an%20insert%20co-culture%20system%20of%20two%20cellular%20types%20in%20the%20absence%20of%20cell-cell%20contact&amp;author=J%20Renaud&amp;author=M-G%20Martinoli&amp;volume=113&amp;publication_year=2016&amp;pages=e54356&amp;pmid=27500972&amp;doi=10.3791/54356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">(14).</span><cite>Bogdanowicz DR; Lu HH
Studying cell-cell communication in co-culture. Biotechnology journal
2013, 8 (4), 395.
</cite> [<a href="https://doi.org/10.1002/biot.201300054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4230534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23554248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20journal&amp;title=Studying%20cell-cell%20communication%20in%20co-culture&amp;author=DR%20Bogdanowicz&amp;author=HH%20Lu&amp;volume=8&amp;issue=4&amp;publication_year=2013&amp;pages=395&amp;pmid=23554248&amp;doi=10.1002/biot.201300054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">(15).</span><cite>Heng BC; Cao T; Lee EH
Directing stem cell differentiation into the chondrogenic lineage in vitro. Stem cells
2004, 22 (7), 1152–1167.
</cite> [<a href="https://doi.org/10.1634/stemcells.2004-0062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15579636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20cells&amp;title=Directing%20stem%20cell%20differentiation%20into%20the%20chondrogenic%20lineage%20in%20vitro&amp;author=BC%20Heng&amp;author=T%20Cao&amp;author=EH%20Lee&amp;volume=22&amp;issue=7&amp;publication_year=2004&amp;pages=1152-1167&amp;pmid=15579636&amp;doi=10.1634/stemcells.2004-0062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">(16).</span><cite>Chen X; Peng Z; Yang Z
Metabolomics studies of cell–cell interactions using single cell mass spectrometry combined with fluorescence microscopy. Chemical Science
2022, 13 (22), 6687–6695.
</cite> [<a href="https://doi.org/10.1039/d2sc02298b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9172575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35756524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Science&amp;title=Metabolomics%20studies%20of%20cell%E2%80%93cell%20interactions%20using%20single%20cell%20mass%20spectrometry%20combined%20with%20fluorescence%20microscopy&amp;author=X%20Chen&amp;author=Z%20Peng&amp;author=Z%20Yang&amp;volume=13&amp;issue=22&amp;publication_year=2022&amp;pages=6687-6695&amp;pmid=35756524&amp;doi=10.1039/d2sc02298b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">(17).</span><cite>An Y; Zhou L; Huang Z; Nice EC; Zhang H; Huang C
Molecular insights into cancer drug resistance from a proteomics perspective. Expert review of proteomics
2019, 16 (5), 413–429.
</cite> [<a href="https://doi.org/10.1080/14789450.2019.1601561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30925852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20review%20of%20proteomics&amp;title=Molecular%20insights%20into%20cancer%20drug%20resistance%20from%20a%20proteomics%20perspective&amp;author=Y%20An&amp;author=L%20Zhou&amp;author=Z%20Huang&amp;author=EC%20Nice&amp;author=H%20Zhang&amp;volume=16&amp;issue=5&amp;publication_year=2019&amp;pages=413-429&amp;pmid=30925852&amp;doi=10.1080/14789450.2019.1601561&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">(18).</span><cite>Ahsan N; Fornelli L; Najar FZ; Gamagedara S; Hossan MR; Rao RSP; Punyamurtula U; Bauer A; Yang Z; Foster SB; et al. 
Proteomics evaluation of five economical commercial abundant protein depletion kits for enrichment of diseases-specific biomarkers from blood serum. Proteomics
2023, 23, No. 2300150.</cite> [<a href="https://doi.org/10.1002/pmic.202300150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11166006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37199141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Proteomics%20evaluation%20of%20five%20economical%20commercial%20abundant%20protein%20depletion%20kits%20for%20enrichment%20of%20diseases-specific%20biomarkers%20from%20blood%20serum&amp;author=N%20Ahsan&amp;author=L%20Fornelli&amp;author=FZ%20Najar&amp;author=S%20Gamagedara&amp;author=MR%20Hossan&amp;volume=23&amp;publication_year=2023&amp;pages=2300150&amp;pmid=37199141&amp;doi=10.1002/pmic.202300150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">(19).</span><cite>Ge SX; Jung D; Yao R
ShinyGO: a graphical gene-set enrichment tool for animals and plants. Bioinformatics
2020, 36 (8), 2628–2629.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ShinyGO:%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants&amp;author=SX%20Ge&amp;author=D%20Jung&amp;author=R%20Yao&amp;volume=36&amp;issue=8&amp;publication_year=2020&amp;pages=2628-2629&amp;pmid=31882993&amp;doi=10.1093/bioinformatics/btz931&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">(20).</span><cite>Szklarczyk D; Kirsch R; Koutrouli M; Nastou K; Mehryary F; Hachilif R; Gable AL; Fang T; Doncheva NT; Pyysalo S; Bork P; Jensen LJ; von Mering C
The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Res. 2023, 51 (D1), D638–d646.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest&amp;author=D%20Szklarczyk&amp;author=R%20Kirsch&amp;author=M%20Koutrouli&amp;author=K%20Nastou&amp;author=F%20Mehryary&amp;volume=51&amp;issue=D1&amp;publication_year=2023&amp;pages=D638-d646&amp;pmid=36370105&amp;doi=10.1093/nar/gkac1000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">(21).</span><cite>Kanehisa M
Toward understanding the origin and evolution of cellular organisms. Protein Sci. 2019, 28 (11), 1947–1951.
</cite> [<a href="https://doi.org/10.1002/pro.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Toward%20understanding%20the%20origin%20and%20evolution%20of%20cellular%20organisms&amp;author=M%20Kanehisa&amp;volume=28&amp;issue=11&amp;publication_year=2019&amp;pages=1947-1951&amp;pmid=31441146&amp;doi=10.1002/pro.3715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">(22).</span><cite>Kanehisa M; Furumichi M; Sato Y; Kawashima M; Ishiguro-Watanabe M
KEGG for taxonomy-based analysis of pathways and genomes. Nucleic Acids Res. 2023, 51 (D1), D587–d592.
</cite> [<a href="https://doi.org/10.1093/nar/gkac963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36300620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG%20for%20taxonomy-based%20analysis%20of%20pathways%20and%20genomes&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=Y%20Sato&amp;author=M%20Kawashima&amp;author=M%20Ishiguro-Watanabe&amp;volume=51&amp;issue=D1&amp;publication_year=2023&amp;pages=D587-d592&amp;pmid=36300620&amp;doi=10.1093/nar/gkac963&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">(23).</span><cite>Kanehisa M; Goto S
KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000, 28 (1), 27–30.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M%20Kanehisa&amp;author=S%20Goto&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;doi=10.1093/nar/28.1.27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">(24).</span><cite>Yang J; Wang N; Chen D; Yu J; Pan Q; Wang D; Liu J; Shi X; Dong X; Cao H
The impact of GFP reporter gene transduction and expression on metabolomics of placental mesenchymal stem cells determined by UHPLC-Q/TOF-MS. Stem cells international
2017, 2017, 1.</cite> [<a href="https://doi.org/10.1155/2017/3167985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5694582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29230249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20cells%20international&amp;title=The%20impact%20of%20GFP%20reporter%20gene%20transduction%20and%20expression%20on%20metabolomics%20of%20placental%20mesenchymal%20stem%20cells%20determined%20by%20UHPLC-Q/TOF-MS&amp;author=J%20Yang&amp;author=N%20Wang&amp;author=D%20Chen&amp;author=J%20Yu&amp;author=Q%20Pan&amp;volume=2017&amp;publication_year=2017&amp;pages=1&amp;pmid=29230249&amp;doi=10.1155/2017/3167985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">(25).</span><cite>McDermott M; Eustace AJ; Busschots S; Breen L; Crown J; Clynes M; O’Donovan N; Stordal B
In vitro development of chemotherapy and targeted therapy drug-resistant cancer cell lines: a practical guide with case studies. Frontiers in oncology
2014, 4, 40.
</cite> [<a href="https://doi.org/10.3389/fonc.2014.00040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3944788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24639951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20oncology&amp;title=In%20vitro%20development%20of%20chemotherapy%20and%20targeted%20therapy%20drug-resistant%20cancer%20cell%20lines:%20a%20practical%20guide%20with%20case%20studies&amp;author=M%20McDermott&amp;author=AJ%20Eustace&amp;author=S%20Busschots&amp;author=L%20Breen&amp;author=J%20Crown&amp;volume=4&amp;publication_year=2014&amp;pages=40&amp;pmid=24639951&amp;doi=10.3389/fonc.2014.00040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">(26).</span><cite>Biedler JL; Spengler BA
Reverse transformation of multidrug-resistant cells. Cancer and Metastasis Reviews
1994, 13, 191–207.
</cite> [<a href="https://doi.org/10.1007/BF00689636" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7923550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20and%20Metastasis%20Reviews&amp;title=Reverse%20transformation%20of%20multidrug-resistant%20cells&amp;author=JL%20Biedler&amp;author=BA%20Spengler&amp;volume=13&amp;publication_year=1994&amp;pages=191-207&amp;pmid=7923550&amp;doi=10.1007/BF00689636&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">(27).</span><cite>Liu R; Sun M; Zhang G; Lan Y; Yang Z
Towards early monitoring of chemotherapy-induced drug resistance based on single cell metabolomics: Combining single-probe mass spectrometry with machine learning. Anal. Chim. Acta
2019, 1092, 42–48.
</cite> [<a href="https://doi.org/10.1016/j.aca.2019.09.065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6878984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31708031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chim.%20Acta&amp;title=Towards%20early%20monitoring%20of%20chemotherapy-induced%20drug%20resistance%20based%20on%20single%20cell%20metabolomics:%20Combining%20single-probe%20mass%20spectrometry%20with%20machine%20learning&amp;author=R%20Liu&amp;author=M%20Sun&amp;author=G%20Zhang&amp;author=Y%20Lan&amp;author=Z%20Yang&amp;volume=1092&amp;publication_year=2019&amp;pages=42-48&amp;pmid=31708031&amp;doi=10.1016/j.aca.2019.09.065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">(28).</span><cite>Hientz K; Mohr A; Bhakta-Guha D; Efferth T
The role of p53 in cancer drug resistance and targeted chemotherapy. Oncotarget
2017, 8 (5), 8921–8946.
</cite> [<a href="https://doi.org/10.18632/oncotarget.13475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5352454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27888811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=The%20role%20of%20p53%20in%20cancer%20drug%20resistance%20and%20targeted%20chemotherapy&amp;author=K%20Hientz&amp;author=A%20Mohr&amp;author=D%20Bhakta-Guha&amp;author=T%20Efferth&amp;volume=8&amp;issue=5&amp;publication_year=2017&amp;pages=8921-8946&amp;pmid=27888811&amp;doi=10.18632/oncotarget.13475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">(29).</span><cite>Stransky L; Cotter K; Forgac M
The Function of V-ATPases in Cancer. Physiol. Rev
2016, 96 (3), 1071–1091.
</cite> [<a href="https://doi.org/10.1152/physrev.00035.2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4982037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27335445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rev&amp;title=The%20Function%20of%20V-ATPases%20in%20Cancer&amp;author=L%20Stransky&amp;author=K%20Cotter&amp;author=M%20Forgac&amp;volume=96&amp;issue=3&amp;publication_year=2016&amp;pages=1071-1091&amp;pmid=27335445&amp;doi=10.1152/physrev.00035.2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">(30).</span><cite>Armingol E; Officer A; Harismendy O; Lewis NE
Deciphering cell–cell interactions and communication from gene expression. Nat. Rev. Genet
2021, 22 (2), 71–88.
</cite> [<a href="https://doi.org/10.1038/s41576-020-00292-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7649713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33168968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&amp;title=Deciphering%20cell%E2%80%93cell%20interactions%20and%20communication%20from%20gene%20expression&amp;author=E%20Armingol&amp;author=A%20Officer&amp;author=O%20Harismendy&amp;author=NE%20Lewis&amp;volume=22&amp;issue=2&amp;publication_year=2021&amp;pages=71-88&amp;pmid=33168968&amp;doi=10.1038/s41576-020-00292-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">(31).</span><cite>Chang Y-C; Yang Y-C; Tien C-P; Yang C-J; Hsiao M
Roles of Aldolase Family Genes in Human Cancers and Diseases. Trends in Endocrinology &amp; Metabolism
2018, 29 (8), 549–559.
</cite> [<a href="https://doi.org/10.1016/j.tem.2018.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29907340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Endocrinology%20&amp;%20Metabolism&amp;title=Roles%20of%20Aldolase%20Family%20Genes%20in%20Human%20Cancers%20and%20Diseases&amp;author=Y-C%20Chang&amp;author=Y-C%20Yang&amp;author=C-P%20Tien&amp;author=C-J%20Yang&amp;author=M%20Hsiao&amp;volume=29&amp;issue=8&amp;publication_year=2018&amp;pages=549-559&amp;pmid=29907340&amp;doi=10.1016/j.tem.2018.05.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">(32).</span><cite>Sun M; Chen X; Yang Z
Single cell mass spectrometry studies reveal metabolomic features and potential mechanisms of drug-resistant cancer cell lines. Anal. Chim. Acta
2022, 1206, No. 339761.
</cite> [<a href="https://doi.org/10.1016/j.aca.2022.339761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9046687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35473873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chim.%20Acta&amp;title=Single%20cell%20mass%20spectrometry%20studies%20reveal%20metabolomic%20features%20and%20potential%20mechanisms%20of%20drug-resistant%20cancer%20cell%20lines&amp;author=M%20Sun&amp;author=X%20Chen&amp;author=Z%20Yang&amp;volume=1206&amp;publication_year=2022&amp;pages=339761&amp;pmid=35473873&amp;doi=10.1016/j.aca.2022.339761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">(33).</span><cite>Doan NB; Nguyen HS; Al-Gizawiy MM; Mueller WM; Sabbadini RA; Rand SD; Connelly JM; Chitambar CR; Schmainda KM; Mirza SP
Acid ceramidase confers radioresistance to glioblastoma cells. Oncol. Rep
2017, 38 (4), 1932–1940.
</cite> [<a href="https://doi.org/10.3892/or.2017.5855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5652937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28765947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep&amp;title=Acid%20ceramidase%20confers%20radioresistance%20to%20glioblastoma%20cells&amp;author=NB%20Doan&amp;author=HS%20Nguyen&amp;author=MM%20Al-Gizawiy&amp;author=WM%20Mueller&amp;author=RA%20Sabbadini&amp;volume=38&amp;issue=4&amp;publication_year=2017&amp;pages=1932-1940&amp;pmid=28765947&amp;doi=10.3892/or.2017.5855&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">(34).</span><cite>Tan S-F; Dunton W; Liu X; Fox TE; Morad SA; Desai D; Doi K; Conaway MR; Amin S; Claxton DF; et al. 
Acid ceramidase promotes drug resistance in acute myeloid leukemia through NF-<em>κ</em>B-dependent P-glycoprotein upregulation. Journal of lipid research
2019, 60 (6), 1078–1086.
</cite> [<a href="https://doi.org/10.1194/jlr.M091876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30962310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20lipid%20research&amp;title=Acid%20ceramidase%20promotes%20drug%20resistance%20in%20acute%20myeloid%20leukemia%20through%20NF-%CE%BAB-dependent%20P-glycoprotein%20upregulation&amp;author=S-F%20Tan&amp;author=W%20Dunton&amp;author=X%20Liu&amp;author=TE%20Fox&amp;author=SA%20Morad&amp;volume=60&amp;issue=6&amp;publication_year=2019&amp;pages=1078-1086&amp;pmid=30962310&amp;doi=10.1194/jlr.M091876&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">(35).</span><cite>Doan NB; Alhajala H; Al-Gizawiy MM; Mueller WM; Rand SD; Connelly JM; Cochran EJ; Chitambar CR; Clark P; Kuo J; et al. 
Acid ceramidase and its inhibitors: A de novo drug target and a new class of drugs for killing glioblastoma cancer stem cells with high efficiency. Oncotarget
2017, 8 (68), No. 112662.
</cite> [<a href="https://doi.org/10.18632/oncotarget.22637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Acid%20ceramidase%20and%20its%20inhibitors:%20A%20de%20novo%20drug%20target%20and%20a%20new%20class%20of%20drugs%20for%20killing%20glioblastoma%20cancer%20stem%20cells%20with%20high%20efficiency&amp;author=NB%20Doan&amp;author=H%20Alhajala&amp;author=MM%20Al-Gizawiy&amp;author=WM%20Mueller&amp;author=SD%20Rand&amp;volume=8&amp;issue=68&amp;publication_year=2017&amp;pages=112662&amp;pmid=29348854&amp;doi=10.18632/oncotarget.22637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">(36).</span><cite>Aldoghachi AF; Baharudin A; Ahmad U; Chan SC; Ong TA; Yunus R; Razack AH; Yusoff K; Veerakumarasivam A
Evaluation of CERS2 Gene as a Potential Biomarker for Bladder Cancer. Dis Markers
2019, 2019, No. 1.</cite> [<a href="https://doi.org/10.1155/2019/3875147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6766133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31636736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&amp;title=Evaluation%20of%20CERS2%20Gene%20as%20a%20Potential%20Biomarker%20for%20Bladder%20Cancer&amp;author=AF%20Aldoghachi&amp;author=A%20Baharudin&amp;author=U%20Ahmad&amp;author=SC%20Chan&amp;author=TA%20Ong&amp;volume=2019&amp;publication_year=2019&amp;pages=1&amp;pmid=31636736&amp;doi=10.1155/2019/3875147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">(37).</span><cite>Tian X; Ahsan N; Lulla A; Lev A; Abbosh P; Dicker DT; Zhang S; El-Deiry WS
P53-independent partial restoration of the p53 pathway in tumors with mutated p53 through ATF4 transcriptional modulation by ERK1/2 and CDK9. Neoplasia
2021, 23 (3), 304–325.
</cite> [<a href="https://doi.org/10.1016/j.neo.2021.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7890376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33582407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;title=P53-independent%20partial%20restoration%20of%20the%20p53%20pathway%20in%20tumors%20with%20mutated%20p53%20through%20ATF4%20transcriptional%20modulation%20by%20ERK1/2%20and%20CDK9&amp;author=X%20Tian&amp;author=N%20Ahsan&amp;author=A%20Lulla&amp;author=A%20Lev&amp;author=P%20Abbosh&amp;volume=23&amp;issue=3&amp;publication_year=2021&amp;pages=304-325&amp;pmid=33582407&amp;doi=10.1016/j.neo.2021.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">(38).</span><cite>Chesnokova V; Melmed S
Pituitary senescence: the evolving role of Pttg. Molecular and cellular endocrinology
2010, 326 (1–2), 55–59.
</cite> [<a href="https://doi.org/10.1016/j.mce.2010.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2906651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20153804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20cellular%20endocrinology&amp;title=Pituitary%20senescence:%20the%20evolving%20role%20of%20Pttg&amp;author=V%20Chesnokova&amp;author=S%20Melmed&amp;volume=326&amp;issue=1%E2%80%932&amp;publication_year=2010&amp;pages=55-59&amp;pmid=20153804&amp;doi=10.1016/j.mce.2010.02.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">(39).</span><cite>Shah M; Nunes MR; Stearns V
CDK4/6 Inhibitors: Game Changers in the Management of Hormone Receptor–Positive Advanced Breast Cancer?
Oncology (Williston Park)
2018, 32 (5), 216–22.
</cite> [<a href="/articles/PMC6424488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29847850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncology%20(Williston%20Park)&amp;title=CDK4/6%20Inhibitors:%20Game%20Changers%20in%20the%20Management%20of%20Hormone%20Receptor%E2%80%93Positive%20Advanced%20Breast%20Cancer?&amp;author=M%20Shah&amp;author=MR%20Nunes&amp;author=V%20Stearns&amp;volume=32&amp;issue=5&amp;publication_year=2018&amp;pages=216-22&amp;pmid=29847850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">(40).</span><cite>Cao Y; Li X; Kong S; Shang S; Qi Y
CDK4/6 inhibition suppresses tumour growth and enhances the effect of Temozolomide in glioma cells. J. Cell Mol. Med
2020, 24 (9), 5135–5145.
</cite> [<a href="https://doi.org/10.1111/jcmm.15156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7205809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32277580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Mol.%20Med&amp;title=CDK4/6%20inhibition%20suppresses%20tumour%20growth%20and%20enhances%20the%20effect%20of%20Temozolomide%20in%20glioma%20cells&amp;author=Y%20Cao&amp;author=X%20Li&amp;author=S%20Kong&amp;author=S%20Shang&amp;author=Y%20Qi&amp;volume=24&amp;issue=9&amp;publication_year=2020&amp;pages=5135-5145&amp;pmid=32277580&amp;doi=10.1111/jcmm.15156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="/articles/PMC10941511/" class="text-red">Retracted</a>]</li>
<li id="R41">
<span class="label">(41).</span><cite>Goel S; Bergholz JS; Zhao JJ
Targeting CDK4 and CDK6 in cancer. Nature Reviews Cancer
2022, 22 (6), 356–372.
</cite> [<a href="https://doi.org/10.1038/s41568-022-00456-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9149100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35304604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&amp;title=Targeting%20CDK4%20and%20CDK6%20in%20cancer&amp;author=S%20Goel&amp;author=JS%20Bergholz&amp;author=JJ%20Zhao&amp;volume=22&amp;issue=6&amp;publication_year=2022&amp;pages=356-372&amp;pmid=35304604&amp;doi=10.1038/s41568-022-00456-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">(42).</span><cite>Calhoun MW; Thomas JW; Gennis RB
The cytochrome oxidase superfamily of redox-driven proton pumps. Trends Biochem. Sci
1994, 19 (8), 325–330.
</cite> [<a href="https://doi.org/10.1016/0968-0004(94)90071-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7940677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci&amp;title=The%20cytochrome%20oxidase%20superfamily%20of%20redox-driven%20proton%20pumps&amp;author=MW%20Calhoun&amp;author=JW%20Thomas&amp;author=RB%20Gennis&amp;volume=19&amp;issue=8&amp;publication_year=1994&amp;pages=325-330&amp;pmid=7940677&amp;doi=10.1016/0968-0004(94)90071-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">(43).</span><cite>Wilkinson ST; Johnson DBF; Tardif HL; Tome ME; Briehl MM
Increased cytochrome c correlates with poor survival in aggressive lymphoma. Oncol Lett. 2010, 1 (2), 227–230.
</cite> [<a href="https://doi.org/10.3892/ol_00000040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2927837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20798784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett&amp;title=Increased%20cytochrome%20c%20correlates%20with%20poor%20survival%20in%20aggressive%20lymphoma&amp;author=ST%20Wilkinson&amp;author=DBF%20Johnson&amp;author=HL%20Tardif&amp;author=ME%20Tome&amp;author=MM%20Briehl&amp;volume=1&amp;issue=2&amp;publication_year=2010&amp;pages=227-230&amp;pmid=20798784&amp;doi=10.3892/ol_00000040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">(44).</span><cite>Gomes AP; Ilter D; Low V; Drapela S; Schild T; Mullarky E; Han J; Elia I; Broekaert D; Rosenzweig A; Nagiec M; Nunes JB; Schaffer BE; Mutvei AP; Asara JM; Cantley LC; Fendt S-M; Blenis J
Altered propionate metabolism contributes to tumour progression and aggressiveness. Nature Metabolism
2022, 4 (4), 435–443.</cite> [<a href="https://doi.org/10.1038/s42255-022-00553-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9050834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35361954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Metabolism&amp;title=Altered%20propionate%20metabolism%20contributes%20to%20tumour%20progression%20and%20aggressiveness&amp;author=AP%20Gomes&amp;author=D%20Ilter&amp;author=V%20Low&amp;author=S%20Drapela&amp;author=T%20Schild&amp;volume=4&amp;issue=4&amp;publication_year=2022&amp;pages=435-443&amp;pmid=35361954&amp;doi=10.1038/s42255-022-00553-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">(45).</span><cite>Nasgashio R; Sato Y; Matsumoto T; Kageyama T; Iyoda A; Satoh Y; Ryuge S; Masuda N; Jiang S-X; Saegusa M
HADHA is a potential predictor of response to platinum-based chemotherapy for lung cancer. Asian Pac J. Cancer Prev
2011, 12 (12), 3457–63.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/22471497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J.%20Cancer%20Prev&amp;title=HADHA%20is%20a%20potential%20predictor%20of%20response%20to%20platinum-based%20chemotherapy%20for%20lung%20cancer&amp;author=R%20Nasgashio&amp;author=Y%20Sato&amp;author=T%20Matsumoto&amp;author=T%20Kageyama&amp;author=A%20Iyoda&amp;volume=12&amp;issue=12&amp;publication_year=2011&amp;pages=3457-63&amp;pmid=22471497&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">(46).</span><cite>Dewulf JP; Paquay S; Marbaix E; Achouri Y; Van Schaftingen E; Bommer GT
ECHDC1 knockout mice accumulate ethyl-branched lipids and excrete abnormal intermediates of branched-chain fatty acid metabolism. J. Biol. Chem
2021, 297 (4), No. 101083.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2021.101083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8473548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34419447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=ECHDC1%20knockout%20mice%20accumulate%20ethyl-branched%20lipids%20and%20excrete%20abnormal%20intermediates%20of%20branched-chain%20fatty%20acid%20metabolism&amp;author=JP%20Dewulf&amp;author=S%20Paquay&amp;author=E%20Marbaix&amp;author=Y%20Achouri&amp;author=E%20Van%20Schaftingen&amp;volume=297&amp;issue=4&amp;publication_year=2021&amp;pages=101083&amp;pmid=34419447&amp;doi=10.1016/j.jbc.2021.101083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">(47).</span><cite>Asai S; Miura N; Sawada Y; Noda T; Kikugawa T; Tanji N; Saika T
Silencing of ECHDC1 inhibits growth of gemcitabine-resistant bladder cancer cells. Oncology Letters
2017, 15 (1), 522–527.
</cite> [<a href="https://doi.org/10.3892/ol.2017.7269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5769416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29391886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncology%20Letters&amp;title=Silencing%20of%20ECHDC1%20inhibits%20growth%20of%20gemcitabine-resistant%20bladder%20cancer%20cells&amp;author=S%20Asai&amp;author=N%20Miura&amp;author=Y%20Sawada&amp;author=T%20Noda&amp;author=T%20Kikugawa&amp;volume=15&amp;issue=1&amp;publication_year=2017&amp;pages=522-527&amp;pmid=29391886&amp;doi=10.3892/ol.2017.7269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">(48).</span><cite>Shan Y; Gao Y; Jin W; Fan M; Wang Y; Gu Y; Shan C; Sun L; Li X; Yu B; Luo Q; Xu Q
Targeting HIBCH to reprogram valine metabolism for the treatment of colorectal cancer. Cell Death &amp; Disease
2019, 10 (8), 618.
</cite> [<a href="https://doi.org/10.1038/s41419-019-1832-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6692300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31409769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20&amp;%20Disease&amp;title=Targeting%20HIBCH%20to%20reprogram%20valine%20metabolism%20for%20the%20treatment%20of%20colorectal%20cancer&amp;author=Y%20Shan&amp;author=Y%20Gao&amp;author=W%20Jin&amp;author=M%20Fan&amp;author=Y%20Wang&amp;volume=10&amp;issue=8&amp;publication_year=2019&amp;pages=618&amp;pmid=31409769&amp;doi=10.1038/s41419-019-1832-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">(49).</span><cite>Levchenko A; Mehta BM; Niu X; Kang G; Villafania L; Way D; Polycarpe D; Sadelain M; Larson SM
Intercellular transfer of P-glycoprotein mediates acquired multidrug resistance in tumor cells. Proc. Natl. Acad. Sci. U. S. A
2005, 102 (6), 1933–1938.
</cite> [<a href="https://doi.org/10.1073/pnas.0401851102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC545583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15671173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Intercellular%20transfer%20of%20P-glycoprotein%20mediates%20acquired%20multidrug%20resistance%20in%20tumor%20cells&amp;author=A%20Levchenko&amp;author=BM%20Mehta&amp;author=X%20Niu&amp;author=G%20Kang&amp;author=L%20Villafania&amp;volume=102&amp;issue=6&amp;publication_year=2005&amp;pages=1933-1938&amp;pmid=15671173&amp;doi=10.1073/pnas.0401851102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">(50).</span><cite>Wang Y; Zhang L; Wei Y; Huang W; Li L; Wu A.-a.; Dastur A; Greninger P; Bray WM; Zhang C-S; Li M; Lian W; Hu Z; Wang X; Liu G; Yao L; Guh J-H; Chen L; Wang H-R; Zhou D; Lin S-C; Xu Q; Shen Y; Zhang J; Jurica MS; Benes CH; Deng X
Pharmacological Targeting of Vacuolar H+-ATPase via Subunit V1G Combats Multidrug-Resistant Cancer. Cell Chemical Biology
2020, 27 (11), 1359–1370.
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2020.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32649904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chemical%20Biology&amp;title=Pharmacological%20Targeting%20of%20Vacuolar%20H+-ATPase%20via%20Subunit%20V1G%20Combats%20Multidrug-Resistant%20Cancer&amp;author=Y%20Wang&amp;author=L%20Zhang&amp;author=Y%20Wei&amp;author=W%20Huang&amp;author=L%20Li&amp;volume=27&amp;issue=11&amp;publication_year=2020&amp;pages=1359-1370&amp;pmid=32649904&amp;doi=10.1016/j.chembiol.2020.06.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">(51).</span><cite>Eniafe J; Jiang S
The functional roles of TCA cycle metabolites in cancer. Oncogene
2021, 40 (19), 3351–3363.
</cite> [<a href="https://doi.org/10.1038/s41388-020-01639-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33864000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=The%20functional%20roles%20of%20TCA%20cycle%20metabolites%20in%20cancer&amp;author=J%20Eniafe&amp;author=S%20Jiang&amp;volume=40&amp;issue=19&amp;publication_year=2021&amp;pages=3351-3363&amp;pmid=33864000&amp;doi=10.1038/s41388-020-01639-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">(52).</span><cite>Wang Z; Zheng W; Chen Z; Wu S; Chang H; Cai M; Cai H
Pan-Cancer analysis shows that ACO2 is a potential prognostic and immunotherapeutic biomarker for multiple cancer types including hepatocellular carcinoma. Front Oncol
2022, 12, No. 1055376.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1055376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9748622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36531056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Pan-Cancer%20analysis%20shows%20that%20ACO2%20is%20a%20potential%20prognostic%20and%20immunotherapeutic%20biomarker%20for%20multiple%20cancer%20types%20including%20hepatocellular%20carcinoma&amp;author=Z%20Wang&amp;author=W%20Zheng&amp;author=Z%20Chen&amp;author=S%20Wu&amp;author=H%20Chang&amp;volume=12&amp;publication_year=2022&amp;pages=1055376&amp;pmid=36531056&amp;doi=10.3389/fonc.2022.1055376&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">(53).</span><cite>Yong L; Tang S; Yu H; Zhang H; Zhang Y; Wan Y; Cai F
The role of hypoxia-inducible factor-1 alpha in multidrug-resistant breast cancer. Frontiers in Oncology
2022, 12, No. 964934.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.964934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9393754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36003773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Oncology&amp;title=The%20role%20of%20hypoxia-inducible%20factor-1%20alpha%20in%20multidrug-resistant%20breast%20cancer&amp;author=L%20Yong&amp;author=S%20Tang&amp;author=H%20Yu&amp;author=H%20Zhang&amp;author=Y%20Zhang&amp;volume=12&amp;publication_year=2022&amp;pages=964934&amp;pmid=36003773&amp;doi=10.3389/fonc.2022.964934&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">(54).</span><cite>Zheng H; Long G; Zheng Y; Yang X; Cai W; He S; Qin X; Liao H
Glycolysis-Related SLC2A1 Is a Potential Pan-Cancer Biomarker for Prognosis and Immunotherapy. Cancers
2022, 14 (21), 5344.
</cite> [<a href="https://doi.org/10.3390/cancers14215344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9657346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36358765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Glycolysis-Related%20SLC2A1%20Is%20a%20Potential%20Pan-Cancer%20Biomarker%20for%20Prognosis%20and%20Immunotherapy&amp;author=H%20Zheng&amp;author=G%20Long&amp;author=Y%20Zheng&amp;author=X%20Yang&amp;author=W%20Cai&amp;volume=14&amp;issue=21&amp;publication_year=2022&amp;pages=5344&amp;pmid=36358765&amp;doi=10.3390/cancers14215344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>SI</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2020672-supplement-SI.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Proteomics data can be found at the MassIVE database via MSV000092944.</p></section></section></section></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1021/acs.jproteome.4c00338" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/nihms-2020672.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.4&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/11425778/" data-citation-style="nlm" data-download-format-link="/resources/citations/11425778/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11425778%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11425778/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11425778/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC11425778/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/38907724/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC11425778/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/38907724/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC11425778/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/11425778/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="ABS1" href="#ABS1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ABS2" href="#ABS2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Graphical Abstract">Graphical Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S1" href="#S1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="INTRODUCTION">INTRODUCTION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S2" href="#S2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="EXPERIMENTAL PROCEDURES">EXPERIMENTAL PROCEDURES</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S11" href="#S11" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="RESULTS">RESULTS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S17" href="#S17" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="DISCUSSION">DISCUSSION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S21" href="#S21" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="CONCLUSION">CONCLUSION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="SM1" href="#SM1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary Material">Supplementary Material</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S23" href="#S23" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="ACKNOWLEDGMENTS">ACKNOWLEDGMENTS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ci93_" href="#_ci93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Contributor Information">Contributor Information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S22" href="#S22" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Data Availability Statement">Data Availability Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="REFERENCES">REFERENCES</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wwJBZLSLZLeUV3WN99FcI77JkdafYIWFF8X0CD862TsZUJ3zDw8oI2GmtbOgtUjm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Amanuscript" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>