{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:12.271920Z",
     "start_time": "2025-06-11T21:33:09.605111Z"
    }
   },
   "source": "from data_gatherer.orchestrator import Orchestrator",
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:13.986721Z",
     "start_time": "2025-06-11T21:33:13.980005Z"
    }
   },
   "cell_type": "code",
   "source": "data_gatherer = Orchestrator() ",
   "id": "ae808f5efe1a0566",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 72 - INFO - Data_Gatherer Orchestrator initialized. Extraction Model: gpt-4o-mini\n"
     ]
    }
   ],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:14.384511Z",
     "start_time": "2025-06-11T21:33:14.380733Z"
    }
   },
   "cell_type": "code",
   "source": [
    "input_urls = [\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11252349/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11320025/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11423856/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11062922/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11208500/\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10972747\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10982990\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4836692\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915822\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11061608\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5460863\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9506422\"]\n",
    "\n",
    "input_url = input_urls[2]"
   ],
   "id": "a46e34d49293e72a",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:14.910303Z",
     "start_time": "2025-06-11T21:33:14.714379Z"
    }
   },
   "cell_type": "code",
   "source": [
    "raw_data = data_gatherer.fetch_data(input_url, write_htmls_xmls=True, html_xml_dir='tmp/html_xmls/')\n",
    "raw_data"
   ],
   "id": "1d596b324a0f4e5f",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 179 - INFO - Setting up data fetcher...\n",
      "orchestrator.py - line 207 - INFO - Data fetcher setup completed.\n",
      "orchestrator.py - line 115 - INFO - Fetching data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11425778&retmode=xml\n",
      "orchestrator.py - line 123 - INFO - Raw Data is XML.\n",
      "data_fetcher.py - line 624 - INFO - File already exists: tmp/html_xmls/PMC/Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems.xml. Skipping download.\n",
      "orchestrator.py - line 129 - INFO - Raw XML saved in tmp/html_xmls/PMC/ directory\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/': <Element pmc-articleset at 0x129f60e40>}"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:15.889715Z",
     "start_time": "2025-06-11T21:33:15.883113Z"
    }
   },
   "cell_type": "code",
   "source": [
    "for key, value in raw_data.items():\n",
    "    informative = data_gatherer.raw_data_contains_required_sections(value, key, [\"data_availability\", \"supplementary_data\"])\n",
    "    print(f\"PMC API Response for article at {key} is complete: {informative}\")"
   ],
   "id": "c3fab81720d02d93",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PMC API Response for article at https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/ is complete: False\n"
     ]
    }
   ],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:22.777557Z",
     "start_time": "2025-06-11T21:33:16.568480Z"
    }
   },
   "cell_type": "code",
   "source": [
    "raw_data = data_gatherer.fetch_data(input_url, HTML_fallback=True)\n",
    "print(f\"raw_data fetched for {len(raw_data.values())} input url\")"
   ],
   "id": "60cdfbc1195ebc8d",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 179 - INFO - Setting up data fetcher...\n",
      "orchestrator.py - line 207 - INFO - Data fetcher setup completed.\n",
      "orchestrator.py - line 115 - INFO - Fetching data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: True\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: False\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: True\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x12ade8dc0>.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "raw_data fetched for 1 input url\n"
     ]
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:25.481159Z",
     "start_time": "2025-06-11T21:33:22.779088Z"
    }
   },
   "cell_type": "code",
   "source": "data_cited = data_gatherer.parse_data(raw_data, input_url, full_document_read=False, raw_data_format=\"full_HTML\")",
   "id": "792fc418f267bd24",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 159 - INFO - Parsing data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/ with publisher: PMC\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/)\n",
      "parser.py - line 967 - INFO - Extracted 14 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 6 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 3468\n",
      "parser.py - line 1285 - INFO - Content length: 10807\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 12180 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 3805 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-eaad1d96501f94fabc9e89c80def198354689d81df2860883669ae3747b5afb7\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 12180 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 3805 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"MSV000092944\",\n",
      "        \"data_repository\": \"MassIVE\",\n",
      "        \"dataset_webpage\": \"https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=3c1c1b1e1c4e4c3b8c3b8c3b8c3b8c3b&dataset=MSV000092944\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'MSV000092944', 'data_repository': 'MassIVE', 'dataset_webpage': 'https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=3c1c1b1e1c4e4c3b8c3b8c3b8c3b8c3b&dataset=MSV000092944'}]\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: MSV000092944\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:33:31.268806Z",
     "start_time": "2025-06-11T21:33:31.258029Z"
    }
   },
   "cell_type": "code",
   "source": "data_cited",
   "id": "4bf356ecb2125174",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "  dataset_identifier data_repository  \\\n",
       "0       MSV000092944         MassIVE   \n",
       "0                NaN             NaN   \n",
       "8                NaN             NaN   \n",
       "\n",
       "                                     dataset_webpage  \\\n",
       "0  https://massive.ucsd.edu/ProteoSAFe/dataset.js...   \n",
       "0                                                NaN   \n",
       "8                                                NaN   \n",
       "\n",
       "                                                link  \\\n",
       "0                                                NaN   \n",
       "0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       "8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       "\n",
       "                            title     file_info description source_section  \\\n",
       "0                             NaN           NaN         NaN            NaN   \n",
       "0   10.1021/acs.jproteome.4c00338           n/a         n/a            n/a   \n",
       "8  NIHMS2020672-supplement-SI.pdf  (1.4MB, pdf)         n/a        d67e776   \n",
       "\n",
       "                                 section_class  \\\n",
       "0                                          NaN   \n",
       "0  pmc-layout__citation font-secondary font-xs   \n",
       "8                                      media p   \n",
       "\n",
       "                                       download_link file_extension  \\\n",
       "0                                                NaN           None   \n",
       "0                                               None           None   \n",
       "8  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...            pdf   \n",
       "\n",
       "                                         a_attr_href  \\\n",
       "0                                                NaN   \n",
       "0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       "8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       "\n",
       "                  a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0                          NaN                   NaN           NaN   \n",
       "0  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       "8                     usa-link      click_feat_suppl           NaN   \n",
       "\n",
       "            a_attr_rel                                         source_url  \n",
       "0                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "0  noopener noreferrer  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "8                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>section_class</th>\n",
       "      <th>download_link</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>source_url</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MSV000092944</td>\n",
       "      <td>MassIVE</td>\n",
       "      <td>https://massive.ucsd.edu/ProteoSAFe/dataset.js...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>pmc-layout__citation font-secondary font-xs</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>https://doi.org/10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/11425778/bin/NIHMS2020672-s...</td>\n",
       "      <td>NIHMS2020672-supplement-SI.pdf</td>\n",
       "      <td>(1.4MB, pdf)</td>\n",
       "      <td>n/a</td>\n",
       "      <td>d67e776</td>\n",
       "      <td>media p</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/11425778/bin/NIHMS2020672-s...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:34:36.976943Z",
     "start_time": "2025-06-11T21:33:32.954358Z"
    }
   },
   "cell_type": "code",
   "source": "results = data_gatherer.process_articles(input_urls[:10])",
   "id": "676abbf7d3c01af2",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 416 - INFO - 0th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6141466&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" notes-type=\"data-availability\"><title>Data availability</title><p>Data has been deposited in dbGAP with accession #PHS001049 (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049\">https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049</ext-link>), and SRA with project accession PRJNA306801 (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)\">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)</ext-link>.</p></notes>\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for supplementary_data_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"supplementary-material\"><title>Electronic supplementary material</title><sec id=\"Sec36\"><p>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM1\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM1_ESM.pdf\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Information</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM2\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM2_ESM.pdf\" position=\"float\" orientation=\"portrait\"><caption><p>Peer Review File</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM3\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM3_ESM.pdf\" position=\"float\" orientation=\"portrait\"><caption><p>Description of Additional Supplementary Files</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM4\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM4_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 1</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM5\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM5_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 2</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM6\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM6_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 3</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM7\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM7_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 4</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM8\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM8_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 5</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM9\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM9_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 6</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM10\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM10_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 7</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM11\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM11_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 8</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM12\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM12_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 9</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM13\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM13_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 10</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM14\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM14_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 11</p></caption></media></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM15\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41467_2018_6162_MOESM15_ESM.xlsx\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Data 12</p></caption></media></supplementary-material>\n",
      "</p></sec></sec>\n",
      "data_fetcher.py - line 701 - INFO - XML data contains all required sections.\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 311 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466/, additional_data, XML)\n",
      "parser.py - line 531 - INFO - Extracted title:'Recurrent WNT pathway alterations are frequent in relapsed small cell lung cancer'\n",
      "parser.py - line 983 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//sec[@sec-type='supplementary-material']. cont: [<Element sec at 0x12c0ab580>]\n",
      "parser.py - line 994 - INFO - Found 15 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x12c0ab640>, <Element supplementary-material at 0x12c0aaf80>, <Element supplementary-material at 0x12c0abd40>, <Element supplementary-material at 0x12c0ab7c0>, <Element supplementary-material at 0x12c0abe00>, <Element supplementary-material at 0x12c0aba80>, <Element supplementary-material at 0x12c0abdc0>, <Element supplementary-material at 0x12c0a8d80>, <Element supplementary-material at 0x12c0ab740>, <Element supplementary-material at 0x12c0ab140>, <Element supplementary-material at 0x12ac05a00>, <Element supplementary-material at 0x12c0a8b40>, <Element supplementary-material at 0x12c0abfc0>, <Element supplementary-material at 0x12c0a9040>, <Element supplementary-material at 0x12c0a8a40>]\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': \".//sec[@sec-type='supplementary-material']\", 'cont': [<Element sec at 0x12c0ab580>]}\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x12c0ab640>, <Element supplementary-material at 0x12c0aaf80>, <Element supplementary-material at 0x12c0abd40>, <Element supplementary-material at 0x12c0ab7c0>, <Element supplementary-material at 0x12c0abe00>, <Element supplementary-material at 0x12c0aba80>, <Element supplementary-material at 0x12c0abdc0>, <Element supplementary-material at 0x12c0a8d80>, <Element supplementary-material at 0x12c0ab740>, <Element supplementary-material at 0x12c0ab140>, <Element supplementary-material at 0x12ac05a00>, <Element supplementary-material at 0x12c0a8b40>, <Element supplementary-material at 0x12c0abfc0>, <Element supplementary-material at 0x12c0a9040>, <Element supplementary-material at 0x12c0a8a40>]}\n",
      "parser.py - line 1573 - INFO - Found 16 supplementary data sections\n",
      "parser.py - line 1597 - INFO - Found data availability content: [' Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) ']\n",
      "parser.py - line 1225 - INFO - Processing DAS_content: [' Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) ']\n",
      "parser.py - line 1237 - INFO - Repos elements: ['bigd.big.ac.cn (BIGD)', 'ega-archive.org (European Genome-Phenome Archive)', 'massive.ucsd.edu (MassIVE)', 'www.ebi.ac.uk (PRIDE)', 'www.iprox.cn (iProX)', 'proteomecentral.proteomexchange.org (ProteomeCentral)', 'www.proteomexchange.org (ProteomeXchange)', 'pdc.cancer.gov (Proteomic Data Commons)', 'gdc.cancer.gov (Genomic Data Commons)', 'cancerimagingarchive.net (The Cancer Imaging Archive)', 'imaging.datacommons.cancer.gov (Imaging Data Commons)', 'repository.jpostdb.org (JPOST)', 'doi.org', 'geo (GEO)', 'dbgap (dbGaP)', 'sra (sra)', 'ArrayExpress (ArrayExpress)']\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 116\n",
      "parser.py - line 1285 - INFO - Content length: 353\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1726 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 452 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-ce6f552859cf2eead778e2fea3d07d4b5eb6e24b70262116329e848e53fbc2a2\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1726 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 452 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PHS001049\",\n",
      "        \"data_repository\": \"dbGaP\",\n",
      "        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PRJNA306801\",\n",
      "        \"data_repository\": \"SRA\",\n",
      "        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PHS001049', 'data_repository': 'dbGaP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049'}, {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801'}]\n",
      "parser.py - line 1248 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PHS001049', 'data_repository': 'dbGaP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049'}, {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801'}]\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PHS001049', 'data_repository': 'dbGaP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801'}\n",
      "parser.py - line 1255 - INFO - Final ret additional data: 2 items\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 2 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PHS001049\n",
      "parser.py - line 1953 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PRJNA306801\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 2\n",
      "parser.py - line 591 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 32, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'surrounding_text',\n",
      "       'description'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 328 - INFO - Parsed data extraction completed. Elements collected: 17\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 360 - INFO - XML element classification not needed. Using parsed_data.\n",
      "orchestrator.py - line 424 - INFO - \n",
      "Progress: 1/10 (10.00%) | Elapsed: 00:00:03 | ETA: 00:00:32\n",
      "\n",
      "orchestrator.py - line 416 - INFO - 1th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11252349/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11252349/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11252349&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" notes-type=\"data-availability\"><title>Data availability</title><p>All data generated or analyzed and its supplementary information files during this study are included in this published article. The mass spectrometry proteomics data have been deposited to the PRIDE Archive (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://www.ebi.ac.uk/pride/archive/\">http://www.ebi.ac.uk/pride/archive/</ext-link>) via the PRIDE partner repository with the data set identifier PXD053567.</p></notes>\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for supplementary_data_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"supplementary-material\"><sec id=\"Sec15\"><title>Supplementary Information</title><p>\n",
      "<supplementary-material content-type=\"local-data\" id=\"MOESM1\" position=\"float\" orientation=\"portrait\"><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"41598_2024_67079_MOESM1_ESM.zip\" position=\"float\" orientation=\"portrait\"><caption><p>Supplementary Information.</p></caption></media></supplementary-material></p></sec></sec>\n",
      "data_fetcher.py - line 701 - INFO - XML data contains all required sections.\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 311 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11252349/, additional_data, XML)\n",
      "parser.py - line 531 - INFO - Extracted title:'Proteomics profile in encapsulated follicular patterned thyroid neoplasms'\n",
      "parser.py - line 983 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//sec[@sec-type='supplementary-material']. cont: [<Element sec at 0x12c0aa700>]\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x12c1b8d40>]\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': \".//sec[@sec-type='supplementary-material']\", 'cont': [<Element sec at 0x12c0aa700>]}\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x12c1b8d40>]}\n",
      "parser.py - line 1573 - INFO - Found 2 supplementary data sections\n",
      "parser.py - line 1597 - INFO - Found data availability content: [' Data availability  All data generated or analyzed and its supplementary information files during this study are included in this published article.\\xa0The mass spectrometry proteomics data have been deposited to the PRIDE Archive (  http://www.ebi.ac.uk/pride/archive/  ) via the PRIDE partner repository with the data set identifier PXD053567.  http://www.ebi.ac.uk/pride/archive/ ']\n",
      "parser.py - line 1225 - INFO - Processing DAS_content: [' Data availability  All data generated or analyzed and its supplementary information files during this study are included in this published article.\\xa0The mass spectrometry proteomics data have been deposited to the PRIDE Archive (  http://www.ebi.ac.uk/pride/archive/  ) via the PRIDE partner repository with the data set identifier PXD053567.  http://www.ebi.ac.uk/pride/archive/ ']\n",
      "parser.py - line 1237 - INFO - Repos elements: ['bigd.big.ac.cn (BIGD)', 'ega-archive.org (European Genome-Phenome Archive)', 'massive.ucsd.edu (MassIVE)', 'www.ebi.ac.uk (PRIDE)', 'www.iprox.cn (iProX)', 'proteomecentral.proteomexchange.org (ProteomeCentral)', 'www.proteomexchange.org (ProteomeXchange)', 'pdc.cancer.gov (Proteomic Data Commons)', 'gdc.cancer.gov (Genomic Data Commons)', 'cancerimagingarchive.net (The Cancer Imaging Archive)', 'imaging.datacommons.cancer.gov (Imaging Data Commons)', 'repository.jpostdb.org (JPOST)', 'doi.org', 'geo (GEO)', 'dbgap (dbGaP)', 'sra (sra)', 'ArrayExpress (ArrayExpress)']\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 83\n",
      "parser.py - line 1285 - INFO - Content length: 380\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1753 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 419 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-adf48c0a2de8080d5914f95df67350399e21632fdb0eb3989cd6d72a70ace449\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1753 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 419 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    [\n",
      "        \"dataset_identifier\": \"PXD053567\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD053567\"\n",
      "    ]\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PXD053567', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD053567'}]\n",
      "parser.py - line 1248 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PXD053567', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD053567'}]\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD053567', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD053567'}\n",
      "parser.py - line 1255 - INFO - Final ret additional data: 1 items\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD053567\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 591 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'surrounding_text',\n",
      "       'description'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 328 - INFO - Parsed data extraction completed. Elements collected: 2\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 360 - INFO - XML element classification not needed. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 2th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11425778&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n",
      "orchestrator.py - line 271 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/.\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: False\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: True\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: False\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x11c3d9a50>.\n",
      "data_fetcher.py - line 221 - INFO - No cookie pattern 1 found in HTML\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 333 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778/)\n",
      "parser.py - line 967 - INFO - Extracted 14 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 6 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 3468\n",
      "parser.py - line 1285 - INFO - Content length: 10807\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 12180 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 3805 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-eaad1d96501f94fabc9e89c80def198354689d81df2860883669ae3747b5afb7\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 12180 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 3805 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"MSV000092944\",\n",
      "        \"data_repository\": \"MassIVE\",\n",
      "        \"dataset_webpage\": \"https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=5c1c1e1e1e4e4e1e8e1e1e1e1e1e1e1e&dataset=MSV000092944\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'MSV000092944', 'data_repository': 'MassIVE', 'dataset_webpage': 'https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=5c1c1e1e1e4e4e1e8e1e1e1e1e1e1e1e&dataset=MSV000092944'}]\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: MSV000092944\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 338 - INFO - Parsed data extraction completed. Elements collected: 3\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 367 - INFO - Full HTML element classification not supported. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 3th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11320025/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11320025/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11320025&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\" id=\"s2-7\"><title>Data availability</title><p>Individual participant data associated with the clinical trials sponsored by AstraZeneca can be requested by qualified researchers via the request portal, with request evaluated according to the AstraZeneca disclosure commitment (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"https://astrazenecagroup-dt.pharmacm.com/DT/Home\" ext-link-type=\"uri\">https://astrazenecagroup-dt.pharmacm.com/DT/Home</ext-link>) and Responsible Data Sharing Principles. Global proteomics spectra and associated were deposited to the ProteomeXchange Consortium (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"https://www.proteomexchange.org/\" ext-link-type=\"uri\">https://www.proteomexchange.org/</ext-link>) via the PRIDE partner repository with the dataset identifier PXD052949. Samples of the polyclonal antibodies for AKT1 and AKT2 can be requested through the Segal Cancer Proteomics Centre (Montreal, QC, Canada). LC-MRM-MS assay information and validation data are publicly available in the CPTAC Assay Portal (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"https://assays.cancer.gov/available_assays\" ext-link-type=\"uri\">https://assays.cancer.gov/available_assays</ext-link>, IDs: 6077-6129; see Supplementary Methods S1 for links to each assay). MRM-MS assay kits including the required peptide standards and instrument files are now commercially available from MRM Proteomics Inc. (Montreal, QC, Canada).</p></sec>\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for supplementary_data_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"supplementary-material\"><title>Supplementary Material</title><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Methods S1</label><caption><p>Supplemental Materials and Methods concerning patient samples, cell lines, tissue culture, iMALDI-MS, Nano-LC-Orbitrap-MS, LC-MRM-MS</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_methods_s1_suppsm1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Validation Data S1</label><caption><p>Supplementary Validation Data demonstrating the performance of the iMALDI-MS assay</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_validation_data_s1_suppsd1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Validation Data S2</label><caption><p>Quality Assessment of the Global Proteomics Data</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_validation_data_s2_suppsd2.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Figure S1</label><caption><p>Individual protein concentrations measured by iMALDI-MS for each sample, comparison of measured PTEN concentration to IHC H-score</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_figure_s1_suppsf1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Figure S2</label><caption><p>Individual protein concentrations measured by iMALDI-MS for PTEN vs. IHC H-score</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_figure_s2_suppsf2.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Figure S3</label><caption><p>Comparison of iMALDI and Global Proteomics results for primary vs metastatic samples</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_figure_s3_suppsf3.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Figure S4</label><caption><p>Combination treatment of cell lines with capivasertib and an eIF2α inhibitor</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_figure_s4_suppsf4.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Table S1</label><caption><p>Detailed results of the label-free quantitation</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_table_s1_suppst1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Table S2</label><caption><p>Proteins expression differences between capivasertib-resistant vs. capivasertib-sensitive cells quantified by validated MRM-MS assays as compared to the profile observed in patient samples</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_table_s2_suppst2.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material><supplementary-material position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>Supplementary Table S3</label><caption><p>Detailed results of pathway mapping by Qiagen IPA</p></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"crc-24-0113_supplementary_table_s3_suppst3.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material></sec>\n",
      "data_fetcher.py - line 701 - INFO - XML data contains all required sections.\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 311 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11320025/, additional_data, XML)\n",
      "parser.py - line 531 - INFO - Extracted title:'mTORC1-Driven Protein Translation Correlates with Clinical Benefit of Capivasertib within a Genetically Preselected Cohort of '\n",
      "parser.py - line 983 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//sec[@sec-type='supplementary-material']. cont: [<Element sec at 0x129ef2dc0>]\n",
      "parser.py - line 994 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x12c2bf340>, <Element supplementary-material at 0x12b8f5e40>, <Element supplementary-material at 0x12b8f59c0>, <Element supplementary-material at 0x12c3c05c0>, <Element supplementary-material at 0x12c3c1100>, <Element supplementary-material at 0x12c3c0800>, <Element supplementary-material at 0x12c0f59c0>, <Element supplementary-material at 0x12c0f7d80>, <Element supplementary-material at 0x12c0f4980>, <Element supplementary-material at 0x12c0f4680>]\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': \".//sec[@sec-type='supplementary-material']\", 'cont': [<Element sec at 0x129ef2dc0>]}\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x12c2bf340>, <Element supplementary-material at 0x12b8f5e40>, <Element supplementary-material at 0x12b8f59c0>, <Element supplementary-material at 0x12c3c05c0>, <Element supplementary-material at 0x12c3c1100>, <Element supplementary-material at 0x12c3c0800>, <Element supplementary-material at 0x12c0f59c0>, <Element supplementary-material at 0x12c0f7d80>, <Element supplementary-material at 0x12c0f4980>, <Element supplementary-material at 0x12c0f4680>]}\n",
      "parser.py - line 1573 - INFO - Found 11 supplementary data sections\n",
      "parser.py - line 1597 - INFO - Found data availability content: [' Data availability  Individual participant data associated with the clinical trials sponsored by AstraZeneca can be requested by qualified researchers via the request portal, with request evaluated according to the AstraZeneca disclosure commitment (  https://astrazenecagroup-dt.pharmacm.com/DT/Home  ) and Responsible Data Sharing Principles. Global proteomics spectra and associated were deposited to the ProteomeXchange Consortium (  https://astrazenecagroup-dt.pharmacm.com/DT/Home  https://www.proteomexchange.org/  ) via the PRIDE partner repository with the dataset identifier PXD052949. Samples of the polyclonal antibodies for AKT1 and AKT2 can be requested through the Segal Cancer Proteomics Centre (Montreal, QC, Canada). LC-MRM-MS assay information and validation data are publicly available in the CPTAC Assay Portal (  https://www.proteomexchange.org/  https://assays.cancer.gov/available_assays  , IDs: 6077-6129; see Supplementary Methods S1 for links to each assay). MRM-MS assay kits including the required peptide standards and instrument files are now commercially available from MRM Proteomics Inc. (Montreal, QC, Canada).  https://assays.cancer.gov/available_assays ']\n",
      "parser.py - line 1225 - INFO - Processing DAS_content: [' Data availability  Individual participant data associated with the clinical trials sponsored by AstraZeneca can be requested by qualified researchers via the request portal, with request evaluated according to the AstraZeneca disclosure commitment (  https://astrazenecagroup-dt.pharmacm.com/DT/Home  ) and Responsible Data Sharing Principles. Global proteomics spectra and associated were deposited to the ProteomeXchange Consortium (  https://astrazenecagroup-dt.pharmacm.com/DT/Home  https://www.proteomexchange.org/  ) via the PRIDE partner repository with the dataset identifier PXD052949. Samples of the polyclonal antibodies for AKT1 and AKT2 can be requested through the Segal Cancer Proteomics Centre (Montreal, QC, Canada). LC-MRM-MS assay information and validation data are publicly available in the CPTAC Assay Portal (  https://www.proteomexchange.org/  https://assays.cancer.gov/available_assays  , IDs: 6077-6129; see Supplementary Methods S1 for links to each assay). MRM-MS assay kits including the required peptide standards and instrument files are now commercially available from MRM Proteomics Inc. (Montreal, QC, Canada).  https://assays.cancer.gov/available_assays ']\n",
      "parser.py - line 1237 - INFO - Repos elements: ['bigd.big.ac.cn (BIGD)', 'ega-archive.org (European Genome-Phenome Archive)', 'massive.ucsd.edu (MassIVE)', 'www.ebi.ac.uk (PRIDE)', 'www.iprox.cn (iProX)', 'proteomecentral.proteomexchange.org (ProteomeCentral)', 'www.proteomexchange.org (ProteomeXchange)', 'pdc.cancer.gov (Proteomic Data Commons)', 'gdc.cancer.gov (Genomic Data Commons)', 'cancerimagingarchive.net (The Cancer Imaging Archive)', 'imaging.datacommons.cancer.gov (Imaging Data Commons)', 'repository.jpostdb.org (JPOST)', 'doi.org', 'geo (GEO)', 'dbgap (dbGaP)', 'sra (sra)', 'ArrayExpress (ArrayExpress)']\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 266\n",
      "parser.py - line 1285 - INFO - Content length: 1190\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 2563 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 602 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-83c8f23810a8450f327772ef755c53225f2430578503e69fa2971eb4190b26eb\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 2563 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 602 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD052949\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD052949\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PXD052949', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD052949'}]\n",
      "parser.py - line 1248 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PXD052949', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD052949'}]\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD052949', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD052949'}\n",
      "parser.py - line 1255 - INFO - Final ret additional data: 1 items\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD052949\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 591 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'surrounding_text',\n",
      "       'description'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 328 - INFO - Parsed data extraction completed. Elements collected: 11\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 360 - INFO - XML element classification not needed. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 4th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11066909&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\" id=\"s19\"><title>Data Availability</title><p>The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE<sup><xref rid=\"CIT0028\" ref-type=\"bibr\">28</xref></sup> partner repository with the dataset identifier PXD045268.</p></sec>\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n",
      "orchestrator.py - line 271 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/.\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: False\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: True\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: False\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x12c35c1f0>.\n",
      "data_fetcher.py - line 221 - INFO - No cookie pattern 1 found in HTML\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 333 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11066909/)\n",
      "parser.py - line 967 - INFO - Extracted 25 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 5 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 7484\n",
      "parser.py - line 1285 - INFO - Content length: 22597\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 23970 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 7821 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-ccfc3035be05fd08b1e41941b30c88004d2c92ae1a14b668f39d35f2064ad811\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 23970 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 7821 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD045268\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD045268\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PXD045268', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD045268'}]\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD045268\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 338 - INFO - Parsed data extraction completed. Elements collected: 17\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 367 - INFO - Full HTML element classification not supported. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 5th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11352417&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" notes-type=\"data-availability\"><title>Data Availability Statement</title><p>All data generated or analyzed during this study are included in this published article and its <xref rid=\"app1-biomolecules-14-00969\" ref-type=\"app\">Supplementary Information Files</xref>. The MS/MS raw data and analysis have been deposited in the ProteomeXchange Consortium via the jPOST partner repository with the data set identifier JPST002506 and PXD049309 (<uri xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309\">https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309</uri>, accessed on 29 July 2024).</p></notes>\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n",
      "orchestrator.py - line 271 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/.\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: False\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: True\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: False\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1043a8d00>.\n",
      "data_fetcher.py - line 221 - INFO - No cookie pattern 1 found in HTML\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 333 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11352417/)\n",
      "parser.py - line 967 - INFO - Extracted 11 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 6 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 2006\n",
      "parser.py - line 1285 - INFO - Content length: 7603\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 8976 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 2343 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-dfaed4266eee6c6348d4c2140b6b5495b45f7827fee1e816abdaaf7aebca2bae\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 8976 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 2343 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"JPST002506\",\n",
      "        \"data_repository\": \"ProteomeXchange Consortium via jPOST\",\n",
      "        \"dataset_webpage\": \"https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=JPST002506\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD049309\",\n",
      "        \"data_repository\": \"ProteomeXchange Consortium\",\n",
      "        \"dataset_webpage\": \"https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'JPST002506', 'data_repository': 'ProteomeXchange Consortium via jPOST', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=JPST002506'}, {'dataset_identifier': 'PXD049309', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD049309'}]\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 2 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: JPST002506\n",
      "parser.py - line 1953 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD049309\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 2\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 338 - INFO - Parsed data extraction completed. Elements collected: 4\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 367 - INFO - Full HTML element classification not supported. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 6th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10855131&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" notes-type=\"data-availability\"><title>Data Availability Statement</title><p>The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD047284 and 10.6019/PXD047284 (<uri xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"https://www.proteomexchange.org\">https://www.proteomexchange.org</uri>, accessed on 1 January 2024).</p></notes>\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n",
      "orchestrator.py - line 271 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/.\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: False\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: True\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: False\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1409e38b0>.\n",
      "data_fetcher.py - line 221 - INFO - No cookie pattern 1 found in HTML\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 333 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10855131/)\n",
      "parser.py - line 967 - INFO - Extracted 18 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 6 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 1081\n",
      "parser.py - line 1285 - INFO - Content length: 4402\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 5775 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 1418 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-f44ead57e7ecd55e35bf4e1b5358fe33903a36e2b8d92568a940bac4b54003e5\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 5775 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 1418 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD047284\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD047284\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"10.6019/PXD047284\",\n",
      "        \"data_repository\": \"ProteomeXchange Consortium\",\n",
      "        \"dataset_webpage\": \"https://www.proteomexchange.org\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PXD047284', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD047284'}, {'dataset_identifier': '10.6019/PXD047284', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'https://www.proteomexchange.org'}]\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 2 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD047284\n",
      "parser.py - line 1953 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: 10.6019/PXD047284\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 2\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 338 - INFO - Parsed data extraction completed. Elements collected: 3\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 367 - INFO - Full HTML element classification not supported. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 7th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11129317&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\" id=\"sec4.2.3\"><title>Data and code availability</title><p id=\"p0195\">\n",
      "<list list-type=\"simple\" id=\"ulist0015\"><list-item id=\"u0030\"><label>•</label><p id=\"p0200\">In this study, a comprehensive dataset was assembled, comprising in total of 911 tumor tissues and 558 adjacent normal tissues across multiple cancer types. The Dataset was assembled from PDC commons (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://pdc.cancer.gov/pdc/\" id=\"intref0120\">https://pdc.cancer.gov/pdc/</ext-link>), and the PDC IDs are reported in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref> (above). The Mass spectrometry proteomics data have been deposited at Pride, and original western blot images have been deposited in Mendeley, all will be publicly available as of the date of publication. The DOI is listed in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref>.</p></list-item><list-item id=\"u0035\"><label>•</label><p id=\"p0205\">All original codes have been deposited at Mendeley Data and are publicly available as of the date of publication. DOIs are listed in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref>.</p></list-item><list-item id=\"u0040\"><label>•</label><p id=\"p0210\">Any additional information required to reanalyze the data reported in the paper is available from the <xref rid=\"sec4.2.1\" ref-type=\"sec\">lead contact</xref> upon request.</p></list-item></list>\n",
      "</p></sec>\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for supplementary_data_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" id=\"app2\" sec-type=\"supplementary-material\"><title>Supplemental information</title><p id=\"p0425\">\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc1\" position=\"float\" orientation=\"portrait\"><caption><title>Document S1. Figures S1–S6</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc2\" position=\"float\" orientation=\"portrait\"><caption><title>File S1. Detailed Philosopher parameters for substitutant library scanning</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc2.zip\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc3\" position=\"float\" orientation=\"portrait\"><caption><title>Table S1. Identified arginine to all other amino acid substitution peptides in different cancers, related to Figure 1</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc3.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc4\" position=\"float\" orientation=\"portrait\"><caption><title>Table S2. Arginine-to-cysteine substitutants with codon-specific analysis in LSCC, related to Figure 3</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc4.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc5\" position=\"float\" orientation=\"portrait\"><caption><title>Table S3. Arginine substitutants codon-specific analysis in different cancers, related to Figure 1</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc5.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc6\" position=\"float\" orientation=\"portrait\"><caption><title>Table S4. Proteomic analysis of MDA-MB-231 cells treated with arginine depletion, related to Figure 4</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc6.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc7\" position=\"float\" orientation=\"portrait\"><caption><title>Table S5. Ontology-enrichment analysis of arginine-to-cysteine substitutants, related to Figure 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc7.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc8\" position=\"float\" orientation=\"portrait\"><caption><title>Table S6. Proteomics analysis of A549 cells treated with arginine depletion, related to Figure 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc8.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc9\" position=\"float\" orientation=\"portrait\"><caption><title>Table S7. DNA fragments of the plasmids used in this study, related to Figures 3, 4, and 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc9.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc10\" position=\"float\" orientation=\"portrait\"><caption><title>Document S2. Article plus supplemental information</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc10.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "</p></sec>\n",
      "data_fetcher.py - line 701 - INFO - XML data contains all required sections.\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 311 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317/, additional_data, XML)\n",
      "parser.py - line 531 - INFO - Extracted title:'Arginine deprivation enriches lung cancer proteomes with cysteine by inducing arginine-to-cysteine substitutants'\n",
      "parser.py - line 983 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//sec[@sec-type='supplementary-material']. cont: [<Element sec at 0x140b7a580>]\n",
      "parser.py - line 994 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x12cb6be80>, <Element supplementary-material at 0x140a53e80>, <Element supplementary-material at 0x140a52640>, <Element supplementary-material at 0x140a53780>, <Element supplementary-material at 0x140a50780>, <Element supplementary-material at 0x140a52780>, <Element supplementary-material at 0x12cba7b40>, <Element supplementary-material at 0x12f8e4cc0>, <Element supplementary-material at 0x12fce8700>, <Element supplementary-material at 0x12cb50a00>]\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': \".//sec[@sec-type='supplementary-material']\", 'cont': [<Element sec at 0x140b7a580>]}\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x12cb6be80>, <Element supplementary-material at 0x140a53e80>, <Element supplementary-material at 0x140a52640>, <Element supplementary-material at 0x140a53780>, <Element supplementary-material at 0x140a50780>, <Element supplementary-material at 0x140a52780>, <Element supplementary-material at 0x12cba7b40>, <Element supplementary-material at 0x12f8e4cc0>, <Element supplementary-material at 0x12fce8700>, <Element supplementary-material at 0x12cb50a00>]}\n",
      "parser.py - line 1573 - INFO - Found 11 supplementary data sections\n",
      "parser.py - line 1592 - INFO - Found key resources table: <Element table-wrap at 0x12cb50e00>\n",
      "parser.py - line 1594 - INFO - Table text: REAGENT or RESOURCE\tSOURCE\tIDENTIFIER\n",
      "Antibodies\n",
      "\n",
      "Mouse anti V5 tag monoclonal\tInvitrogen\tCatalog # R960-25; RRID:  AB_2556564\n",
      "rabbit anti-ASS1\tProteintech\tCatalog# 16210-1-AP; RRID: AB_2060466\n",
      "Mouse anti-HSP90\tBD bioscience\tCatalog # 610418; RRID:  AB_397798\n",
      "rat anti-tubulin\tSanta Cruz\tCatalog # sc-53029; RRID:  AB_793541\n",
      "Rabbit anti-RARS\tProteintech\tCatalog # 27344-1-AP; RRID: AB_2880849\n",
      "Rb anti-phospho-eIF2a\tCell signaling\t3398\n",
      "IRDye 680RD donkey anti-mouse\tLI-COR\tCatalog # 926-68072; RRID:  AB_10953628\n",
      "IRDye 800CW goat anti-rabbit\tLICOR\tCatalog # 926-32211; RRID:  AB_621843\n",
      "IRDye 800RD goat anti-rat\tLI-COR\tCatalog # 926-32219; RRID:  AB_1850025\n",
      "\n",
      "Bacterial and virus strains\n",
      "\n",
      "DH5-alpha\tThermo Fisher Scientific\tCatalog# 18265017\n",
      "\n",
      "Chemicals, peptides, and recombinant proteins\n",
      "\n",
      "T4 RNA Ligase 2 (dsRNA Ligase)\tNew England biolabs\tCatalog #M0239L\n",
      "Zeba Spin Desalting Columns, 7K MWCO, 0.5 mL\tThermo Fisher scientific\tCatalog # 89883\n",
      "BCA Protein Assay Kit\tThermo Fisher Scientific\tCatalog # 23225\n",
      "Blasticidin S Hydrochloride\tThermo Fisher Scientific\tCatalog # 10264913\n",
      "Crystal violet solution\tSigma Aldrich\tCatalog # V5265\n",
      "DAPI (4',6-Diamidino-2-Phenylindole, Dihydrochloride)\tThermo Fisher Scientific\tCatalog # D1306\n",
      "Dulbecco’s MEM (DMEM) F-12 w/o Tryptophan\tUSBiological life Sciences\tCatalog # D9807-04-10\n",
      "Fetal Bovine Serum, dialyzed US origin\tThermo Fisher Scientific\tCatalog # 26400044\n",
      "Formaldehyde solution\tSigma Aldrich\tCatalog # 252549\n",
      "Gibco HEPES (1M)\tThermo Fisher Scientific\tCatalog # 15630056 or # 15630-080\n",
      "GlutaMAX Supplement\tThermo Fisher Scientific\tCatalog # 35050038\n",
      "Sodium bicarbonate\tSigma Aldrich\tCatalog # S6014\n",
      "Trizol\tThermo Fisher Scientific\tCatalog # 15596018\n",
      "Tween 20\tSigma Aldrich\tCatalog # P1379\n",
      "Phusion HF DNA Polymerase\tThermo Fisher Scientific\tCatalog # F530\n",
      "Polyethylenimine Hydrochloride\tPolysciences\tCatalog # 25439-2\n",
      "Puromycin\tBio-connect\tCatalog # AG-CN2-0078-M500\n",
      "ChromoTek V5-Trap Magnetic Agarose\tproteintech\tCatalog # v5tma-20\n",
      "Sodium acetate, 1M aq. soln., pH 4.5\tSanta Cruz\tsc-296390\n",
      "Glucose solution\tSigma\t49163-100ML\n",
      "Amersham MicroSpin G-50 Columns\tCytivalifesciences\tCatalog # 27533001\n",
      "Acid-Phenol:Chloroform, pH 4.5 (with IAA, 125:24:1)\tThermo Fisher Scientific\tCatalog # AM9720\n",
      "(+)-Biotin-(PEO)3-iodoacetamide\tSanta Cruz\tCatalog # sc-470524A\n",
      "Pemetrexed (LY-231514) disodium\tSelleckchem\tCatalog # S1135\n",
      "DY-547P1-Maleimide\tdyomics\tCatalog # 547P1-03\n",
      "V5 peptide\tchromotek\tCatalog # v5p-1\n",
      "\n",
      "Critical commercial assays\n",
      "\n",
      "EZ-PCR Mycoplasma Detection Kit\tBiological Industries\tCatalog # 20-700-20\n",
      "PureLink Quick Midiprep Kit\tThermo Fisher Scientific\tCatalog # K210004\n",
      "PureLink Quick Miniprep Kit\tThermo Fisher Scientific\tCatalog # K210002\n",
      "SensiFAST SYBR No-ROX Kit\tBioline\tCatalog # BIO-98050\n",
      "SuperScript III Reverse Transcriptase\tThermo Fisher Scientific\tCatalog # 18080093\n",
      "Wizard SV Gel and PCR Clean-Up System\tPromega\tCatalog # A9282\n",
      "\n",
      "Deposited data\n",
      "\n",
      "Mass spectrometry proteomics\tThis study\tPride:  PXD043612\n",
      "Philosopher parameters\tThis study\thttps://doi.org/10.17632/3wfxrz66w2.1\n",
      "Western blot\tThis study\thttps://doi.org/10.17632/bvdn865y9c.1\n",
      "LSCC proteome\tCPTAC\tPDC000234\n",
      "CCRCC proteome\tCPTAC\tPDC000127\n",
      "GBM proteome\tCPTAC\tPDC000204\n",
      "HNSCC proteome\tCPTAC\tPDC000221\n",
      "HCC proteome\tCPTAC\tPDC000198\n",
      "OVSCC proteome\tCPTAC\tPDC000110\n",
      "PDA proteome\tCPTAC\tPDC000270\n",
      "UCEC proteome\tCPTAC\tPDC000125\n",
      "LUAD proteome\tCPTAC\tPDC000153\n",
      "\n",
      "Experimental models: Cell lines\n",
      "\n",
      "293T\tInternal stock\tRRID:CVCL_0063\n",
      "A549\tInternal stock\tRRID:CVCL_0023\n",
      "MDA-MB-231\tInternal stock\tRRID:CVCL_0062\n",
      "\n",
      "Oligonucleotides\n",
      "\n",
      "ON-TARGETplus Human RARS siRNA\thorizon discovery\tLQ-009820-02-0002\n",
      "\n",
      "Recombinant DNA\n",
      "\n",
      "pCDH- KRT8S\tThis study\tN/A\n",
      "pTeton-KRT8s\tThis study\tN/A\n",
      "pLX304-KRT8s\tThis study\tN/A\n",
      "pCDH- KRT8S R>K\tThis study\tN/A\n",
      "pCDH- KRT8S 10R>10C\tThis study\tN/A\n",
      "pCDH- KRT8S 10R>10K\tThis study\tN/A\n",
      "pCDH- KRT8S agg\tThis study\tN/A\n",
      "pCDH- KRT8S cgc\tThis study\tN/A\n",
      "pCDH- KRT8S cgg\tThis study\tN/A\n",
      "pCDH- KRT8S agt\tThis study\tN/A\n",
      "pCDH-KEAP1\tThis study\tN/A\n",
      "\n",
      "Software and algorithms\n",
      "\n",
      "Philosopher 4.0.0\tNesviLab\thttps://github.com/Nesvilab/philosopher/releases\n",
      "Philosopher 5.0\tNesviLab\thttps://github.com/Nesvilab/philosopher/releases\n",
      "PEPQUERY2\tPepquery.org\thttp://www.pepquery.org/\n",
      "Adobe Illustrator CC 2017\tAdobe acrobat\thttp://www.adobe.com\n",
      "ImageJ\tNIH\thttps://imagej.nih.gov/ij/\n",
      "FlowJo V10 software (FlowJo)\tFlowJo\thttps://www.flowjo.com/\n",
      "TraceFinder software\tThermo Fisher Scientific\thttps://www.thermofisher.com/\n",
      "Prism9\tGraphPad software\thttps://www.graphpad.com/scientific-software/prism/\n",
      "parser.py - line 1597 - INFO - Found data availability content: [' Data and code availability  \\n  \\n  •  In this study, a comprehensive dataset was assembled, comprising in total of 911 tumor tissues and 558 adjacent normal tissues across multiple cancer types. The Dataset was assembled from PDC commons (  https://pdc.cancer.gov/pdc/  ), and the PDC IDs are reported in the   https://pdc.cancer.gov/pdc/  key resources table   (above). The Mass spectrometry proteomics data have been deposited at Pride, and original western blot images have been deposited in Mendeley, all will be publicly available as of the date of publication. The DOI is listed in the   key resources table  key resources table  .  key resources table  •  All original codes have been deposited at Mendeley Data and are publicly available as of the date of publication. DOIs are listed in the   key resources table  .  key resources table  •  Any additional information required to reanalyze the data reported in the paper is available from the   lead contact   upon request.  lead contact ', \"REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\nAntibodies\\n\\nMouse anti V5 tag monoclonal\\tInvitrogen\\tCatalog # R960-25; RRID:  AB_2556564\\nrabbit anti-ASS1\\tProteintech\\tCatalog# 16210-1-AP; RRID: AB_2060466\\nMouse anti-HSP90\\tBD bioscience\\tCatalog # 610418; RRID:  AB_397798\\nrat anti-tubulin\\tSanta Cruz\\tCatalog # sc-53029; RRID:  AB_793541\\nRabbit anti-RARS\\tProteintech\\tCatalog # 27344-1-AP; RRID: AB_2880849\\nRb anti-phospho-eIF2a\\tCell signaling\\t3398\\nIRDye 680RD donkey anti-mouse\\tLI-COR\\tCatalog # 926-68072; RRID:  AB_10953628\\nIRDye 800CW goat anti-rabbit\\tLICOR\\tCatalog # 926-32211; RRID:  AB_621843\\nIRDye 800RD goat anti-rat\\tLI-COR\\tCatalog # 926-32219; RRID:  AB_1850025\\n\\nBacterial and virus strains\\n\\nDH5-alpha\\tThermo Fisher Scientific\\tCatalog# 18265017\\n\\nChemicals, peptides, and recombinant proteins\\n\\nT4 RNA Ligase 2 (dsRNA Ligase)\\tNew England biolabs\\tCatalog #M0239L\\nZeba Spin Desalting Columns, 7K MWCO, 0.5\\xa0mL\\tThermo Fisher scientific\\tCatalog # 89883\\nBCA Protein Assay Kit\\tThermo Fisher Scientific\\tCatalog # 23225\\nBlasticidin S Hydrochloride\\tThermo Fisher Scientific\\tCatalog # 10264913\\nCrystal violet solution\\tSigma Aldrich\\tCatalog # V5265\\nDAPI (4',6-Diamidino-2-Phenylindole, Dihydrochloride)\\tThermo Fisher Scientific\\tCatalog # D1306\\nDulbecco’s MEM (DMEM) F-12 w/o Tryptophan\\tUSBiological life Sciences\\tCatalog # D9807-04-10\\nFetal Bovine Serum, dialyzed US origin\\tThermo Fisher Scientific\\tCatalog # 26400044\\nFormaldehyde solution\\tSigma Aldrich\\tCatalog # 252549\\nGibco HEPES (1M)\\tThermo Fisher Scientific\\tCatalog # 15630056 or # 15630-080\\nGlutaMAX Supplement\\tThermo Fisher Scientific\\tCatalog # 35050038\\nSodium bicarbonate\\tSigma Aldrich\\tCatalog # S6014\\nTrizol\\tThermo Fisher Scientific\\tCatalog # 15596018\\nTween 20\\tSigma Aldrich\\tCatalog # P1379\\nPhusion HF DNA Polymerase\\tThermo Fisher Scientific\\tCatalog # F530\\nPolyethylenimine Hydrochloride\\tPolysciences\\tCatalog # 25439-2\\nPuromycin\\tBio-connect\\tCatalog # AG-CN2-0078-M500\\nChromoTek V5-Trap Magnetic Agarose\\tproteintech\\tCatalog # v5tma-20\\nSodium acetate, 1M aq. soln., pH 4.5\\tSanta Cruz\\tsc-296390\\nGlucose solution\\tSigma\\t49163-100ML\\nAmersham MicroSpin G-50 Columns\\tCytivalifesciences\\tCatalog # 27533001\\nAcid-Phenol:Chloroform, pH 4.5 (with IAA, 125:24:1)\\tThermo Fisher Scientific\\tCatalog # AM9720\\n(+)-Biotin-(PEO)3-iodoacetamide\\tSanta Cruz\\tCatalog # sc-470524A\\nPemetrexed (LY-231514) disodium\\tSelleckchem\\tCatalog # S1135\\nDY-547P1-Maleimide\\tdyomics\\tCatalog # 547P1-03\\nV5 peptide\\tchromotek\\tCatalog # v5p-1\\n\\nCritical commercial assays\\n\\nEZ-PCR Mycoplasma Detection Kit\\tBiological Industries\\tCatalog # 20-700-20\\nPureLink Quick Midiprep Kit\\tThermo Fisher Scientific\\tCatalog # K210004\\nPureLink Quick Miniprep Kit\\tThermo Fisher Scientific\\tCatalog # K210002\\nSensiFAST SYBR No-ROX Kit\\tBioline\\tCatalog # BIO-98050\\nSuperScript III Reverse Transcriptase\\tThermo Fisher Scientific\\tCatalog # 18080093\\nWizard SV Gel and PCR Clean-Up System\\tPromega\\tCatalog # A9282\\n\\nDeposited data\\n\\nMass spectrometry proteomics\\tThis study\\tPride:  PXD043612\\nPhilosopher parameters\\tThis study\\thttps://doi.org/10.17632/3wfxrz66w2.1\\nWestern blot\\tThis study\\thttps://doi.org/10.17632/bvdn865y9c.1\\nLSCC proteome\\tCPTAC\\tPDC000234\\nCCRCC proteome\\tCPTAC\\tPDC000127\\nGBM proteome\\tCPTAC\\tPDC000204\\nHNSCC proteome\\tCPTAC\\tPDC000221\\nHCC proteome\\tCPTAC\\tPDC000198\\nOVSCC proteome\\tCPTAC\\tPDC000110\\nPDA proteome\\tCPTAC\\tPDC000270\\nUCEC proteome\\tCPTAC\\tPDC000125\\nLUAD proteome\\tCPTAC\\tPDC000153\\n\\nExperimental models: Cell lines\\n\\n293T\\tInternal stock\\tRRID:CVCL_0063\\nA549\\tInternal stock\\tRRID:CVCL_0023\\nMDA-MB-231\\tInternal stock\\tRRID:CVCL_0062\\n\\nOligonucleotides\\n\\nON-TARGETplus Human RARS siRNA\\thorizon discovery\\tLQ-009820-02-0002\\n\\nRecombinant DNA\\n\\npCDH- KRT8S\\tThis study\\tN/A\\npTeton-KRT8s\\tThis study\\tN/A\\npLX304-KRT8s\\tThis study\\tN/A\\npCDH- KRT8S R>K\\tThis study\\tN/A\\npCDH- KRT8S 10R>10C\\tThis study\\tN/A\\npCDH- KRT8S 10R>10K\\tThis study\\tN/A\\npCDH- KRT8S agg\\tThis study\\tN/A\\npCDH- KRT8S cgc\\tThis study\\tN/A\\npCDH- KRT8S cgg\\tThis study\\tN/A\\npCDH- KRT8S agt\\tThis study\\tN/A\\npCDH-KEAP1\\tThis study\\tN/A\\n\\nSoftware and algorithms\\n\\nPhilosopher 4.0.0\\tNesviLab\\thttps://github.com/Nesvilab/philosopher/releases\\nPhilosopher 5.0\\tNesviLab\\thttps://github.com/Nesvilab/philosopher/releases\\nPEPQUERY2\\tPepquery.org\\thttp://www.pepquery.org/\\nAdobe Illustrator CC 2017\\tAdobe acrobat\\thttp://www.adobe.com\\nImageJ\\tNIH\\thttps://imagej.nih.gov/ij/\\nFlowJo V10 software (FlowJo)\\tFlowJo\\thttps://www.flowjo.com/\\nTraceFinder software\\tThermo Fisher Scientific\\thttps://www.thermofisher.com/\\nPrism9\\tGraphPad software\\thttps://www.graphpad.com/scientific-software/prism/\"]\n",
      "parser.py - line 1225 - INFO - Processing DAS_content: [' Data and code availability  \\n  \\n  •  In this study, a comprehensive dataset was assembled, comprising in total of 911 tumor tissues and 558 adjacent normal tissues across multiple cancer types. The Dataset was assembled from PDC commons (  https://pdc.cancer.gov/pdc/  ), and the PDC IDs are reported in the   https://pdc.cancer.gov/pdc/  key resources table   (above). The Mass spectrometry proteomics data have been deposited at Pride, and original western blot images have been deposited in Mendeley, all will be publicly available as of the date of publication. The DOI is listed in the   key resources table  key resources table  .  key resources table  •  All original codes have been deposited at Mendeley Data and are publicly available as of the date of publication. DOIs are listed in the   key resources table  .  key resources table  •  Any additional information required to reanalyze the data reported in the paper is available from the   lead contact   upon request.  lead contact ', \"REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\nAntibodies\\n\\nMouse anti V5 tag monoclonal\\tInvitrogen\\tCatalog # R960-25; RRID:  AB_2556564\\nrabbit anti-ASS1\\tProteintech\\tCatalog# 16210-1-AP; RRID: AB_2060466\\nMouse anti-HSP90\\tBD bioscience\\tCatalog # 610418; RRID:  AB_397798\\nrat anti-tubulin\\tSanta Cruz\\tCatalog # sc-53029; RRID:  AB_793541\\nRabbit anti-RARS\\tProteintech\\tCatalog # 27344-1-AP; RRID: AB_2880849\\nRb anti-phospho-eIF2a\\tCell signaling\\t3398\\nIRDye 680RD donkey anti-mouse\\tLI-COR\\tCatalog # 926-68072; RRID:  AB_10953628\\nIRDye 800CW goat anti-rabbit\\tLICOR\\tCatalog # 926-32211; RRID:  AB_621843\\nIRDye 800RD goat anti-rat\\tLI-COR\\tCatalog # 926-32219; RRID:  AB_1850025\\n\\nBacterial and virus strains\\n\\nDH5-alpha\\tThermo Fisher Scientific\\tCatalog# 18265017\\n\\nChemicals, peptides, and recombinant proteins\\n\\nT4 RNA Ligase 2 (dsRNA Ligase)\\tNew England biolabs\\tCatalog #M0239L\\nZeba Spin Desalting Columns, 7K MWCO, 0.5\\xa0mL\\tThermo Fisher scientific\\tCatalog # 89883\\nBCA Protein Assay Kit\\tThermo Fisher Scientific\\tCatalog # 23225\\nBlasticidin S Hydrochloride\\tThermo Fisher Scientific\\tCatalog # 10264913\\nCrystal violet solution\\tSigma Aldrich\\tCatalog # V5265\\nDAPI (4',6-Diamidino-2-Phenylindole, Dihydrochloride)\\tThermo Fisher Scientific\\tCatalog # D1306\\nDulbecco’s MEM (DMEM) F-12 w/o Tryptophan\\tUSBiological life Sciences\\tCatalog # D9807-04-10\\nFetal Bovine Serum, dialyzed US origin\\tThermo Fisher Scientific\\tCatalog # 26400044\\nFormaldehyde solution\\tSigma Aldrich\\tCatalog # 252549\\nGibco HEPES (1M)\\tThermo Fisher Scientific\\tCatalog # 15630056 or # 15630-080\\nGlutaMAX Supplement\\tThermo Fisher Scientific\\tCatalog # 35050038\\nSodium bicarbonate\\tSigma Aldrich\\tCatalog # S6014\\nTrizol\\tThermo Fisher Scientific\\tCatalog # 15596018\\nTween 20\\tSigma Aldrich\\tCatalog # P1379\\nPhusion HF DNA Polymerase\\tThermo Fisher Scientific\\tCatalog # F530\\nPolyethylenimine Hydrochloride\\tPolysciences\\tCatalog # 25439-2\\nPuromycin\\tBio-connect\\tCatalog # AG-CN2-0078-M500\\nChromoTek V5-Trap Magnetic Agarose\\tproteintech\\tCatalog # v5tma-20\\nSodium acetate, 1M aq. soln., pH 4.5\\tSanta Cruz\\tsc-296390\\nGlucose solution\\tSigma\\t49163-100ML\\nAmersham MicroSpin G-50 Columns\\tCytivalifesciences\\tCatalog # 27533001\\nAcid-Phenol:Chloroform, pH 4.5 (with IAA, 125:24:1)\\tThermo Fisher Scientific\\tCatalog # AM9720\\n(+)-Biotin-(PEO)3-iodoacetamide\\tSanta Cruz\\tCatalog # sc-470524A\\nPemetrexed (LY-231514) disodium\\tSelleckchem\\tCatalog # S1135\\nDY-547P1-Maleimide\\tdyomics\\tCatalog # 547P1-03\\nV5 peptide\\tchromotek\\tCatalog # v5p-1\\n\\nCritical commercial assays\\n\\nEZ-PCR Mycoplasma Detection Kit\\tBiological Industries\\tCatalog # 20-700-20\\nPureLink Quick Midiprep Kit\\tThermo Fisher Scientific\\tCatalog # K210004\\nPureLink Quick Miniprep Kit\\tThermo Fisher Scientific\\tCatalog # K210002\\nSensiFAST SYBR No-ROX Kit\\tBioline\\tCatalog # BIO-98050\\nSuperScript III Reverse Transcriptase\\tThermo Fisher Scientific\\tCatalog # 18080093\\nWizard SV Gel and PCR Clean-Up System\\tPromega\\tCatalog # A9282\\n\\nDeposited data\\n\\nMass spectrometry proteomics\\tThis study\\tPride:  PXD043612\\nPhilosopher parameters\\tThis study\\thttps://doi.org/10.17632/3wfxrz66w2.1\\nWestern blot\\tThis study\\thttps://doi.org/10.17632/bvdn865y9c.1\\nLSCC proteome\\tCPTAC\\tPDC000234\\nCCRCC proteome\\tCPTAC\\tPDC000127\\nGBM proteome\\tCPTAC\\tPDC000204\\nHNSCC proteome\\tCPTAC\\tPDC000221\\nHCC proteome\\tCPTAC\\tPDC000198\\nOVSCC proteome\\tCPTAC\\tPDC000110\\nPDA proteome\\tCPTAC\\tPDC000270\\nUCEC proteome\\tCPTAC\\tPDC000125\\nLUAD proteome\\tCPTAC\\tPDC000153\\n\\nExperimental models: Cell lines\\n\\n293T\\tInternal stock\\tRRID:CVCL_0063\\nA549\\tInternal stock\\tRRID:CVCL_0023\\nMDA-MB-231\\tInternal stock\\tRRID:CVCL_0062\\n\\nOligonucleotides\\n\\nON-TARGETplus Human RARS siRNA\\thorizon discovery\\tLQ-009820-02-0002\\n\\nRecombinant DNA\\n\\npCDH- KRT8S\\tThis study\\tN/A\\npTeton-KRT8s\\tThis study\\tN/A\\npLX304-KRT8s\\tThis study\\tN/A\\npCDH- KRT8S R>K\\tThis study\\tN/A\\npCDH- KRT8S 10R>10C\\tThis study\\tN/A\\npCDH- KRT8S 10R>10K\\tThis study\\tN/A\\npCDH- KRT8S agg\\tThis study\\tN/A\\npCDH- KRT8S cgc\\tThis study\\tN/A\\npCDH- KRT8S cgg\\tThis study\\tN/A\\npCDH- KRT8S agt\\tThis study\\tN/A\\npCDH-KEAP1\\tThis study\\tN/A\\n\\nSoftware and algorithms\\n\\nPhilosopher 4.0.0\\tNesviLab\\thttps://github.com/Nesvilab/philosopher/releases\\nPhilosopher 5.0\\tNesviLab\\thttps://github.com/Nesvilab/philosopher/releases\\nPEPQUERY2\\tPepquery.org\\thttp://www.pepquery.org/\\nAdobe Illustrator CC 2017\\tAdobe acrobat\\thttp://www.adobe.com\\nImageJ\\tNIH\\thttps://imagej.nih.gov/ij/\\nFlowJo V10 software (FlowJo)\\tFlowJo\\thttps://www.flowjo.com/\\nTraceFinder software\\tThermo Fisher Scientific\\thttps://www.thermofisher.com/\\nPrism9\\tGraphPad software\\thttps://www.graphpad.com/scientific-software/prism/\"]\n",
      "parser.py - line 1237 - INFO - Repos elements: ['bigd.big.ac.cn (BIGD)', 'ega-archive.org (European Genome-Phenome Archive)', 'massive.ucsd.edu (MassIVE)', 'www.ebi.ac.uk (PRIDE)', 'www.iprox.cn (iProX)', 'proteomecentral.proteomexchange.org (ProteomeCentral)', 'www.proteomexchange.org (ProteomeXchange)', 'pdc.cancer.gov (Proteomic Data Commons)', 'gdc.cancer.gov (Genomic Data Commons)', 'cancerimagingarchive.net (The Cancer Imaging Archive)', 'imaging.datacommons.cancer.gov (Imaging Data Commons)', 'repository.jpostdb.org (JPOST)', 'doi.org', 'geo (GEO)', 'dbgap (dbGaP)', 'sra (sra)', 'ArrayExpress (ArrayExpress)']\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 218\n",
      "parser.py - line 1285 - INFO - Content length: 997\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 2370 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 554 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-0cb7d4109d68dfc3135db7bab7d7f20166d6d8fd3ab1e577750f4773d09f3d99\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 2370 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 554 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC commons\",\n",
      "        \"data_repository\": \"PDC\",\n",
      "        \"dataset_webpage\": \"https://pdc.cancer.gov/pdc/\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD029921\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"Mendeley Data\",\n",
      "        \"data_repository\": \"Mendeley\",\n",
      "        \"dataset_webpage\": \"https://data.mendeley.com/\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PDC commons', 'data_repository': 'PDC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/'}, {'dataset_identifier': 'PXD029921', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD029921'}, {'dataset_identifier': 'Mendeley Data', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}]\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 1590\n",
      "parser.py - line 1285 - INFO - Content length: 4478\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 5851 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 1925 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-94aab37a96b7649f10a13893a0d80c0bc3a0ed9bad4986e281d2b1d1fbf1c18f\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 5851 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 1925 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    {\n",
      "        \"dataset_identifier\": \"PXD043612\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD043612\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"10.17632/3wfxrz66w2.1\",\n",
      "        \"data_repository\": \"This study\",\n",
      "        \"dataset_webpage\": \"https://doi.org/10.17632/3wfxrz66w2.1\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"10.17632/bvdn865y9c.1\",\n",
      "        \"data_repository\": \"This study\",\n",
      "        \"dataset_webpage\": \"https://doi.org/10.17632/bvdn865y9c.1\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000234\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000234\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000127\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000127\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000204\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000204\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000221\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000221\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000198\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000198\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000110\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000110\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000270\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000270\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000125\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000125\"\n",
      "    },\n",
      "    {\n",
      "        \"dataset_identifier\": \"PDC000153\",\n",
      "        \"data_repository\": \"CPTAC\",\n",
      "        \"dataset_webpage\": \"https://cptac-data-portal.georgetown.edu/study/summary/PDC000153\"\n",
      "    }\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PXD043612', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}, {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/3wfxrz66w2.1'}, {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/bvdn865y9c.1'}, {'dataset_identifier': 'PDC000234', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000234'}, {'dataset_identifier': 'PDC000127', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000127'}, {'dataset_identifier': 'PDC000204', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000204'}, {'dataset_identifier': 'PDC000221', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000221'}, {'dataset_identifier': 'PDC000198', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000198'}, {'dataset_identifier': 'PDC000110', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000110'}, {'dataset_identifier': 'PDC000270', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000270'}, {'dataset_identifier': 'PDC000125', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000125'}, {'dataset_identifier': 'PDC000153', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000153'}]\n",
      "parser.py - line 1248 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PDC commons', 'data_repository': 'PDC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/'}, {'dataset_identifier': 'PXD029921', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD029921'}, {'dataset_identifier': 'Mendeley Data', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}, {'dataset_identifier': 'PXD043612', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}, {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/3wfxrz66w2.1'}, {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/bvdn865y9c.1'}, {'dataset_identifier': 'PDC000234', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000234'}, {'dataset_identifier': 'PDC000127', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000127'}, {'dataset_identifier': 'PDC000204', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000204'}, {'dataset_identifier': 'PDC000221', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000221'}, {'dataset_identifier': 'PDC000198', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000198'}, {'dataset_identifier': 'PDC000110', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000110'}, {'dataset_identifier': 'PDC000270', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000270'}, {'dataset_identifier': 'PDC000125', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000125'}, {'dataset_identifier': 'PDC000153', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000153'}]\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC commons', 'data_repository': 'PDC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD029921', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD029921'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'Mendeley Data', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD043612', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/3wfxrz66w2.1'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/bvdn865y9c.1'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000234', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000234'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000127', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000127'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000204', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000204'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000221', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000221'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000198', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000198'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000110', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000110'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000270', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000270'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000125', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000125'}\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000153', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://cptac-data-portal.georgetown.edu/study/summary/PDC000153'}\n",
      "parser.py - line 1255 - INFO - Final ret additional data: 15 items\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 15 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC commons\n",
      "parser.py - line 1953 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD029921\n",
      "parser.py - line 1953 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: Mendeley Data\n",
      "parser.py - line 1953 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PXD043612\n",
      "parser.py - line 1953 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: 10.17632/3wfxrz66w2.1\n",
      "parser.py - line 1953 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: 10.17632/bvdn865y9c.1\n",
      "parser.py - line 1953 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000234\n",
      "parser.py - line 1953 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000127\n",
      "parser.py - line 1953 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000204\n",
      "parser.py - line 1953 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000221\n",
      "parser.py - line 1953 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000198\n",
      "parser.py - line 1953 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000110\n",
      "parser.py - line 1953 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000270\n",
      "parser.py - line 1953 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000125\n",
      "parser.py - line 1953 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: PDC000153\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 15\n",
      "parser.py - line 591 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 35, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'surrounding_text',\n",
      "       'description'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 328 - INFO - Parsed data extraction completed. Elements collected: 25\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 360 - INFO - XML element classification not needed. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 8th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11423856/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11423856/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11423856&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\"><title>Data availability</title><p>All data and search results have been deposited to the iProX database (iProX: IPX0004230000), which are publicly accessible at <ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"http://www.iprox.org\" ext-link-type=\"uri\">http://www.iprox.org</ext-link>.</p></sec>\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for supplementary_data_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"supplementary-material\"><title>Supplementary Material</title><supplementary-material id=\"sup1\" position=\"float\" content-type=\"local-data\" orientation=\"portrait\"><label>qzae033_Supplementary_Data</label><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"qzae033_supplementary_data.zip\" position=\"float\" orientation=\"portrait\"/></supplementary-material></sec>\n",
      "data_fetcher.py - line 701 - INFO - XML data contains all required sections.\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 311 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11423856/, additional_data, XML)\n",
      "parser.py - line 531 - INFO - Extracted title:'Proteomic Stratification of Prognosis and Treatment Options for Small Cell Lung Cancer'\n",
      "parser.py - line 983 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//sec[@sec-type='supplementary-material']. cont: [<Element sec at 0x140b57300>]\n",
      "parser.py - line 994 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x12c282a80>]\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': \".//sec[@sec-type='supplementary-material']\", 'cont': [<Element sec at 0x140b57300>]}\n",
      "parser.py - line 1002 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x12c282a80>]}\n",
      "parser.py - line 1573 - INFO - Found 2 supplementary data sections\n",
      "parser.py - line 1597 - INFO - Found data availability content: [' Data availability  All data and search results have been deposited to the iProX database (iProX: IPX0004230000), which are publicly accessible at   http://www.iprox.org  .  http://www.iprox.org ']\n",
      "parser.py - line 1225 - INFO - Processing DAS_content: [' Data availability  All data and search results have been deposited to the iProX database (iProX: IPX0004230000), which are publicly accessible at   http://www.iprox.org  .  http://www.iprox.org ']\n",
      "parser.py - line 1237 - INFO - Repos elements: ['bigd.big.ac.cn (BIGD)', 'ega-archive.org (European Genome-Phenome Archive)', 'massive.ucsd.edu (MassIVE)', 'www.ebi.ac.uk (PRIDE)', 'www.iprox.cn (iProX)', 'proteomecentral.proteomexchange.org (ProteomeCentral)', 'www.proteomexchange.org (ProteomeXchange)', 'pdc.cancer.gov (Proteomic Data Commons)', 'gdc.cancer.gov (Genomic Data Commons)', 'cancerimagingarchive.net (The Cancer Imaging Archive)', 'imaging.datacommons.cancer.gov (Imaging Data Commons)', 'repository.jpostdb.org (JPOST)', 'doi.org', 'geo (GEO)', 'dbgap (dbGaP)', 'sra (sra)', 'ArrayExpress (ArrayExpress)']\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 51\n",
      "parser.py - line 1285 - INFO - Content length: 195\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1568 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 387 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0-26fb5ff922f66a6111c6358d9594ef2a333af2684c4ee2ce9a6d5e5cba025a42\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1568 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages: 387 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: [\n",
      "    [\n",
      "        \"dataset_identifier\": \"IPX0004230000\",\n",
      "        \"data_repository\": \"iProX\",\n",
      "        \"dataset_webpage\": \"http://www.iprox.org\"\n",
      "    ]\n",
      "]\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'IPX0004230000', 'data_repository': 'iProX', 'dataset_webpage': 'http://www.iprox.org'}]\n",
      "parser.py - line 1248 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'IPX0004230000', 'data_repository': 'iProX', 'dataset_webpage': 'http://www.iprox.org'}]\n",
      "parser.py - line 1250 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'IPX0004230000', 'data_repository': 'iProX', 'dataset_webpage': 'http://www.iprox.org'}\n",
      "parser.py - line 1255 - INFO - Final ret additional data: 1 items\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 1 datasets\n",
      "parser.py - line 1953 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "parser.py - line 1964 - INFO - Raw accession ID: IPX0004230000\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 1\n",
      "parser.py - line 591 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'surrounding_text',\n",
      "       'description'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 328 - INFO - Parsed data extraction completed. Elements collected: 2\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 360 - INFO - XML element classification not needed. Using parsed_data.\n",
      "orchestrator.py - line 416 - INFO - 9th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/)\n",
      "orchestrator.py - line 228 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/\n",
      "data_fetcher.py - line 95 - INFO - Initializing APIClient(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 567 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11015306&retmode=xml\n",
      "orchestrator.py - line 263 - INFO - Raw data fetched from source: PMC_API\n",
      "data_fetcher.py - line 721 - INFO - ----Checking for data_availability_sections section in raw data.\n",
      "data_fetcher.py - line 732 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\"><title>Data availability statement</title><p>Data are available in a public, open access repository.</p></sec>\n",
      "data_fetcher.py - line 698 - INFO - Missing section in XML: data_availability_sections\n",
      "orchestrator.py - line 271 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/.\n",
      "data_fetcher.py - line 107 - INFO - WebScraper instance: False\n",
      "data_fetcher.py - line 108 - INFO - APIClient instance: True\n",
      "data_fetcher.py - line 109 - INFO - scraper_tool attribute: False\n",
      "data_fetcher.py - line 111 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1043a8d90>.\n",
      "data_fetcher.py - line 221 - INFO - No cookie pattern 1 found in HTML\n",
      "orchestrator.py - line 286 - INFO - Successfully fetched Raw content.\n",
      "orchestrator.py - line 333 - INFO - Using LLMParser to parse data.\n",
      "parser.py - line 137 - INFO - Parser initialized.\n",
      "parser.py - line 517 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/, additional_data, full_HTML)\n",
      "parser.py - line 631 - INFO - Chunking the HTML content for the parsing step.\n",
      "parser.py - line 907 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015306/)\n",
      "parser.py - line 967 - INFO - Extracted 2 unique supplementary material links from HTML.\n",
      "parser.py - line 1630 - INFO - Extracting data availability elements from HTML\n",
      "parser.py - line 1641 - INFO - Using selector: section.data-availability-statement\n",
      "parser.py - line 1641 - INFO - Using selector: section.associated-data\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "parser.py - line 1641 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "parser.py - line 1679 - INFO - Found 5 data availability elements from HTML.\n",
      "parser.py - line 1278 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gpt-4o-mini\n",
      "prompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 1382 char\n",
      "parser.py - line 2035 - INFO - Number of tokens: 2599\n",
      "parser.py - line 1285 - INFO - Content length: 9814\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 11187 char\n",
      "parser.py - line 1296 - INFO - Prompt messages total length: 2936 tokens\n",
      "parser.py - line 1302 - INFO - Prompt ID: gpt-4o-mini-0.0-2182240fa966a57e3122b1d7705e1c9b1b0510306943e763dcedc497787c7cc4\n",
      "parser.py - line 2052 - INFO - Expected string but got list. Converting list to string.\n",
      "parser.py - line 2059 - INFO - Counting tokens for model: gpt-4o-mini, prompt length: 11187 char\n",
      "parser.py - line 1321 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0.0, messages: 2936 tokens\n",
      "parser.py - line 1348 - INFO - GPT response: []\n",
      "parser.py - line 1359 - INFO - Response is <class 'list'>: []\n",
      "parser.py - line 1945 - INFO - Fetching metadata for 0 datasets\n",
      "parser.py - line 2015 - INFO - Updated datasets len: 0\n",
      "parser.py - line 649 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['link', 'title', 'file_info', 'description', 'source_section',\n",
      "       'section_class', 'download_link', 'file_extension', 'a_attr_href',\n",
      "       'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\n",
      "orchestrator.py - line 338 - INFO - Parsed data extraction completed. Elements collected: 1\n",
      "orchestrator.py - line 346 - INFO - Raw Data parsing completed.\n",
      "orchestrator.py - line 367 - INFO - Full HTML element classification not supported. Using parsed_data.\n"
     ]
    }
   ],
   "execution_count": 9
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-06-11T21:34:36.986635Z",
     "start_time": "2025-06-11T21:34:36.980162Z"
    }
   },
   "cell_type": "code",
   "source": "len(results)",
   "id": "f02d795897f4ba4b",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {},
   "cell_type": "code",
   "outputs": [],
   "execution_count": null,
   "source": "",
   "id": "2d33834fd1c43dd8"
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
