<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EAuthor Manuscript%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Comprehensive genomic profiles of small cell lung cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="6E4B408F747933B306408F0010CC88C7.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="nihpa">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="manuscript">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature">
<meta name="citation_title" content="Comprehensive genomic profiles of small cell lung cancer">
<meta name="citation_author" content="Julie George">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Jing Shan Lim">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Se Jin Jang">
<meta name="citation_author_institution" content="Department of Pathology and Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Yupeng Cun">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Luka Ozretić">
<meta name="citation_author_institution" content="Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Gu Kong">
<meta name="citation_author_institution" content="Department of Pathology, College of Medicine, Hanyang University. 222 Wangsimniro, Seongdong-gu, Seoul 133-791, Korea">
<meta name="citation_author" content="Frauke Leenders">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Xin Lu">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Lynnette Fernández-Cuesta">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Graziella Bosco">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Christian Müller">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Ilona Dahmen">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Nadine S Jahchan">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Kwon-Sik Park">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Dian Yang">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Anthony N Karnezis">
<meta name="citation_author_institution" content="Vancouver General Hospital, Terry Fox laboratory, Vancouver, British Columbia V5Z 1L3, Canada">
<meta name="citation_author" content="Dedeepya Vaka">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Angela Torres">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Maia Segura Wang">
<meta name="citation_author_institution" content="European Molecular Biology Laboratory, Genome Biology Unit, 69117 Heidelberg, Germany">
<meta name="citation_author" content="Jan O Korbel">
<meta name="citation_author_institution" content="European Molecular Biology Laboratory, Genome Biology Unit, 69117 Heidelberg, Germany">
<meta name="citation_author" content="Roopika Menon">
<meta name="citation_author_institution" content="Institute of Pathology, Center of Integrated Oncology Cologne–Bonn, University Hospital of Bonn, 53127 Bonn, Germany">
<meta name="citation_author" content="Sung-Min Chun">
<meta name="citation_author_institution" content="Department of Pathology and Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Deokhoon Kim">
<meta name="citation_author_institution" content="Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Matt Wilkerson">
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, The University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA">
<meta name="citation_author" content="Neil Hayes">
<meta name="citation_author_institution" content="UNC Lineberger Comprehensive Cancer Center School of Medicine, University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA">
<meta name="citation_author" content="David Engelmann">
<meta name="citation_author_institution" content="Institute of Experimental Gene Therapy and Cancer Research, Rostock University Medical Center, 18057 Rostock, Germany">
<meta name="citation_author" content="Brigitte Pützer">
<meta name="citation_author_institution" content="Institute of Experimental Gene Therapy and Cancer Research, Rostock University Medical Center, 18057 Rostock, Germany">
<meta name="citation_author" content="Marc Bos">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Sebastian Michels">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Ignacija Vlasic">
<meta name="citation_author_institution" content="Department of Internal Medicine, University Hospital of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Danila Seidel">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Berit Pinther">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Philipp Schaub">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Christian Becker">
<meta name="citation_author_institution" content="Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Janine Altmüller">
<meta name="citation_author_institution" content="Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Institute of Human Genetics, University Hospital Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Jun Yokota">
<meta name="citation_author_institution" content="Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan">
<meta name="citation_author_institution" content="Genomics and Epigenomics of Cancer Prediction Program, Institute of Predictive and Personalized Medicine of Cancer (IMPPC), Barcelona 08916, Spain">
<meta name="citation_author" content="Takashi Kohno">
<meta name="citation_author_institution" content="Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan">
<meta name="citation_author" content="Reika Iwakawa">
<meta name="citation_author_institution" content="Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan">
<meta name="citation_author" content="Koji Tsuta">
<meta name="citation_author_institution" content="Department of Pathology and Clinical Laboratories, National Cancer Center Hospital Chuo-ku, Tokyo 1040045, Japan">
<meta name="citation_author" content="Masayuki Noguchi">
<meta name="citation_author_institution" content="Department of Pathology, Faculty of Medicine, University of Tsukuba, Ibaraki 305-8575, Japan">
<meta name="citation_author" content="Thomas Muley">
<meta name="citation_author_institution" content="Thoraxklinik at University Hospital Heidelberg, Amalienstrasse 5, 69126 Heidelberg, Germany">
<meta name="citation_author_institution" content="Translational Lung Research Center Heidelberg (TLRC-H), Member of German Center for Lung Research (DZL), Amalienstrasse 5, 69126 Heidelberg, Germany">
<meta name="citation_author" content="Hans Hoffmann">
<meta name="citation_author_institution" content="Thoraxklinik at University Hospital Heidelberg, Amalienstrasse 5, 69126 Heidelberg, Germany">
<meta name="citation_author" content="Philipp A Schnabel">
<meta name="citation_author_institution" content="Translational Lung Research Center Heidelberg (TLRC-H), Member of German Center for Lung Research (DZL), Amalienstrasse 5, 69126 Heidelberg, Germany">
<meta name="citation_author_institution" content="Institute of Pathology, University of Heidelberg, Im Neuenheimer Feld 220, 69120 Heidelberg, Germany">
<meta name="citation_author" content="Iver Petersen">
<meta name="citation_author_institution" content="Institute of Pathology, Jena University Hospital, Friedrich-Schiller-University, 07743 Jena, Germany">
<meta name="citation_author" content="Yuan Chen">
<meta name="citation_author_institution" content="Institute of Pathology, Jena University Hospital, Friedrich-Schiller-University, 07743 Jena, Germany">
<meta name="citation_author" content="Alex Soltermann">
<meta name="citation_author_institution" content="Institute of Surgical Pathology, University Hospital Zürich, 8091 Zürich, Switzerland">
<meta name="citation_author" content="Verena Tischler">
<meta name="citation_author_institution" content="Institute of Surgical Pathology, University Hospital Zürich, 8091 Zürich, Switzerland">
<meta name="citation_author" content="Chang-min Choi">
<meta name="citation_author_institution" content="Department of Oncology, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Yong-Hee Kim">
<meta name="citation_author_institution" content="Department of Thoracic and Cardiovascular Surgery, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Pierre P Massion">
<meta name="citation_author_institution" content="Thoracic Program, Vanderbilt-Ingram Cancer Center PRB 640, 2220 Pierce Avenue, Nashville, Tennessee 37232, USA">
<meta name="citation_author" content="Yong Zou">
<meta name="citation_author_institution" content="Thoracic Program, Vanderbilt-Ingram Cancer Center PRB 640, 2220 Pierce Avenue, Nashville, Tennessee 37232, USA">
<meta name="citation_author" content="Dragana Jovanovic">
<meta name="citation_author_institution" content="University Hospital of Pulmonology, Clinical Center of Serbia, Medical School, University of Belgrade, 11000 Belgrade, Serbia">
<meta name="citation_author" content="Milica Kontic">
<meta name="citation_author_institution" content="University Hospital of Pulmonology, Clinical Center of Serbia, Medical School, University of Belgrade, 11000 Belgrade, Serbia">
<meta name="citation_author" content="Gavin M Wright">
<meta name="citation_author_institution" content="Department of Surgery, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia">
<meta name="citation_author" content="Prudence A Russell">
<meta name="citation_author_institution" content="Department of Pathology, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia">
<meta name="citation_author" content="Benjamin Solomon">
<meta name="citation_author_institution" content="Department of Haematology and Medical Oncology, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia">
<meta name="citation_author" content="Ina Koch">
<meta name="citation_author_institution" content="Asklepios Biobank für Lungenerkrankungen, Comprehensive Pneumology Center Munich, Member of the German Center for Lung Research (DZL), Asklepios Fachkliniken München-Gauting 82131, Germany">
<meta name="citation_author" content="Michael Lindner">
<meta name="citation_author_institution" content="Asklepios Biobank für Lungenerkrankungen, Comprehensive Pneumology Center Munich, Member of the German Center for Lung Research (DZL), Asklepios Fachkliniken München-Gauting 82131, Germany">
<meta name="citation_author" content="Lucia A Muscarella">
<meta name="citation_author_institution" content="Laboratory of Oncology, IRCCS Casa Sollievo della Sofferenza, Viale Cappuccini, 71013 San Giovanni, Rotondo, Italy">
<meta name="citation_author" content="Annamaria la Torre">
<meta name="citation_author_institution" content="Laboratory of Oncology, IRCCS Casa Sollievo della Sofferenza, Viale Cappuccini, 71013 San Giovanni, Rotondo, Italy">
<meta name="citation_author" content="John K Field">
<meta name="citation_author_institution" content="Roy Castle Lung Cancer Research Programme, Department of Molecular and Clinical Cancer Medicine, Institute of Translational Medicine, The University of Liverpool Cancer Research Centre, 200 London Road, L69 3GA Liverpool, UK">
<meta name="citation_author" content="Marko Jakopovic">
<meta name="citation_author_institution" content="University of Zagreb, School of Medicine, Department for Respiratory Diseases Jordanovac, University Hospital Center Zagreb, 10000 Zagreb, Croatia">
<meta name="citation_author" content="Jelena Knezevic">
<meta name="citation_author_institution" content="Laboratory for Translational Medicine, Rudjer Boskovic Institute, 10000 Zagreb, Croatia">
<meta name="citation_author" content="Esmeralda Castaños-Vélez">
<meta name="citation_author_institution" content="Charité Comprehensive Cancer Center, Charité Campus Mitte, 10115 Berlin, Germany">
<meta name="citation_author" content="Luca Roz">
<meta name="citation_author_institution" content="Tumor Genomics Unit, Department of Experimental Oncology and Molecular Medicine, Fondazione IRCCS — Istituto Nazionale Tumori, Via Venezian 1, 20133 Milan, Italy">
<meta name="citation_author" content="Ugo Pastorino">
<meta name="citation_author_institution" content="Thoracic Surgery Unit, Department of Surgery, Fondazione IRCCS Istituto Nazionale Tumori, 20133 Milan, Italy">
<meta name="citation_author" content="Odd-Terje Brustugun">
<meta name="citation_author_institution" content="Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, N-0424 Oslo, Norway">
<meta name="citation_author_institution" content="Department of Oncology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway">
<meta name="citation_author" content="Marius Lund-Iversen">
<meta name="citation_author_institution" content="Department of Pathology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway">
<meta name="citation_author" content="Erik Thunnissen">
<meta name="citation_author_institution" content="Department of Pathology, VU University Medical Center, 1007 MB Amsterdam, The Netherlands">
<meta name="citation_author" content="Jens Köhler">
<meta name="citation_author_institution" content="West German Cancer Center, Department of Medical Oncology, University Hospital Essen, 45147 Essen, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), 69120 Heidelberg, Germany">
<meta name="citation_author" content="Martin Schuler">
<meta name="citation_author_institution" content="West German Cancer Center, Department of Medical Oncology, University Hospital Essen, 45147 Essen, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), 69120 Heidelberg, Germany">
<meta name="citation_author" content="Johan Botling">
<meta name="citation_author_institution" content="Departments of Immunology, Genetics and Pathology, and Medical Sciences, Respiratory, Allergy and Sleep Research, Uppsala University, 75185 Uppsala, Sweden">
<meta name="citation_author" content="Martin Sandelin">
<meta name="citation_author_institution" content="Departments of Immunology, Genetics and Pathology, and Medical Sciences, Respiratory, Allergy and Sleep Research, Uppsala University, 75185 Uppsala, Sweden">
<meta name="citation_author" content="Montserrat Sanchez-Cespedes">
<meta name="citation_author_institution" content="Genes and Cancer Group, Cancer Epigenetics and Biology Program (PEBC), Bellvitge Biomedical Research Institute (IDIBELL), 08908 Hospitalet de Llobregat, Barcelona, Spain">
<meta name="citation_author" content="Helga B Salvesen">
<meta name="citation_author_institution" content="Department of Clinical Science, Center for Cancer Biomarkers, University of Bergen, N-5058 Bergen, Norway">
<meta name="citation_author_institution" content="Department of Gynecology and Obstetrics, Haukeland University Hospital, N-5058 Bergen, Norway">
<meta name="citation_author" content="Viktor Achter">
<meta name="citation_author_institution" content="Computing Center, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Ulrich Lang">
<meta name="citation_author_institution" content="Computing Center, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Department of Informatics, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Magdalena Bogus">
<meta name="citation_author_institution" content="Institute of Legal Medicine, University of Cologne, 50823 Cologne, Germany">
<meta name="citation_author" content="Peter M Schneider">
<meta name="citation_author_institution" content="Institute of Legal Medicine, University of Cologne, 50823 Cologne, Germany">
<meta name="citation_author" content="Thomas Zander">
<meta name="citation_author_institution" content="Gastrointestinal Cancer Group Cologne, Center of Integrated Oncology Cologne–Bonn, Department I for Internal Medicine, University Hospital of Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Sascha Ansén">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Michael Hallek">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author_institution" content="Cologne Excellence Cluster on Cellular Stress Responses in Aging-Associated Diseases (CECAD), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Jürgen Wolf">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Martin Vingron">
<meta name="citation_author_institution" content="Computational Molecular Biology Group, Max Planck Institute for Molecular Genetics, 14195 Berlin, Germany">
<meta name="citation_author" content="Yasushi Yatabe">
<meta name="citation_author_institution" content="Department of Pathology and Molecular Diagnostics, Aichi Cancer Center, 464-8681 Nagoya, Japan">
<meta name="citation_author" content="William D Travis">
<meta name="citation_author_institution" content="Department of Pathology, Memorial Sloan-Kettering Cancer Center, New York 10065, USA">
<meta name="citation_author" content="Peter Nürnberg">
<meta name="citation_author_institution" content="Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Cologne Excellence Cluster on Cellular Stress Responses in Aging-Associated Diseases (CECAD), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Christian Reinhardt">
<meta name="citation_author_institution" content="Department of Internal Medicine, University Hospital of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Sven Perner">
<meta name="citation_author_institution" content="Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea">
<meta name="citation_author" content="Lukas Heukamp">
<meta name="citation_author_institution" content="Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Reinhard Büttner">
<meta name="citation_author_institution" content="Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_author" content="Stefan A Haas">
<meta name="citation_author_institution" content="Computational Molecular Biology Group, Max Planck Institute for Molecular Genetics, 14195 Berlin, Germany">
<meta name="citation_author" content="Elisabeth Brambilla">
<meta name="citation_author_institution" content="Department of Pathology, CHU Grenoble INSERM U823, University Joseph Fourier, Institute Albert Bonniot 38043, CS10217 Grenoble, France">
<meta name="citation_author" content="Martin Peifer">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author" content="Julien Sage">
<meta name="citation_author_institution" content="Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA">
<meta name="citation_author" content="Roman K Thomas">
<meta name="citation_author_institution" content="Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany">
<meta name="citation_author_institution" content="Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany">
<meta name="citation_publication_date" content="2015 Jul 13">
<meta name="citation_volume" content="524">
<meta name="citation_issue" content="7563">
<meta name="citation_firstpage" content="47">
<meta name="citation_doi" content="10.1038/nature14664">
<meta name="citation_pmid" content="26168399">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/pdf/nihms782739.pdf">
<meta name="description" content="We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours ...">
<meta name="og:title" content="Comprehensive genomic profiles of small cell lung cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script src="/static/assets/mathjax/ui/lazy.js" charset="UTF-8"></script><style>.MJX_LiveRegion {
  position: absolute;
  top: 0;
  display: block;
  width: auto;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  left: 0;
  right: 0;
  margin: 0 auto;
  background-color: white;
  box-shadow: 0px 5px 20px #888;
  border: 2px solid #CCCCCC;
}</style><style>.MJX_LiveRegion {
  position: absolute;
  top: 0;
  display: block;
  width: auto;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  left: 0;
  right: 0;
  margin: 0 auto;
  background-color: white;
  box-shadow: 0px 5px 20px #888;
  border: 2px solid #CCCCCC;
}</style><style>.MJX_HoverRegion {
  display: block;
  position: absolute;
  width: max-content;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  margin: 0 auto;
  background-color: white;
  line-height: 0;
  box-shadow: 0px 10px 20px #888;
  border: 2px solid #CCCCCC;
}

.MJX_HoverRegion > div {
  overflow: hidden;
}</style><style>.MJX_ToolTip {
  width: auto;
  height: auto;
  opacity: 1;
  text-align: center;
  border-radius: 6px;
  padding: 0;
  border-bottom: 1px dotted black;
  position: absolute;
  display: inline-block;
  background-color: white;
  z-index: 202;
}</style><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container[jax="CHTML"] :focus {
  outline: solid 3px;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-em.woff") format("woff");
}
</style><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="4861069">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.1038/nature14664" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/nihms782739.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC4861069%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/4861069/" data-citation-style="nlm" data-download-format-link="/resources/citations/4861069/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nihpa.png" alt="NIHPA Author Manuscripts logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NIHPA Author Manuscripts" title="Link to NIHPA Author Manuscripts" shape="default" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Nature</button></div>. Author manuscript; available in PMC: 2016 May 9.</div>
<div>
<em>Published in final edited form as: </em>Nature. 2015 Jul 13;524(7563):47–53. doi: <a href="https://doi.org/10.1038/nature14664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/nature14664</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nature%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nature%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nature%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nature%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comprehensive genomic profiles of small cell lung cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22George%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Julie George</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Julie George</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22George%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julie George</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20JS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Jing Shan Lim</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jing Shan Lim</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20JS%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Shan Lim</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20SJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Se Jin Jang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Se Jin Jang</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology and Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20SJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Se Jin Jang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cun%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Yupeng Cun</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yupeng Cun</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cun%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yupeng Cun</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozreti%C4%87%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Luka Ozretić</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Luka Ozretić</span></h3>
<div class="p">
<sup>4</sup>Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozreti%C4%87%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luka Ozretić</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Gu Kong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Gu Kong</span></h3>
<div class="p">
<sup>5</sup>Department of Pathology, College of Medicine, Hanyang University. 222 Wangsimniro, Seongdong-gu, Seoul 133-791, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gu Kong</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leenders%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7" aria-expanded="false"><span class="name western">Frauke Leenders</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Frauke Leenders</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leenders%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Frauke Leenders</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8" aria-expanded="false"><span class="name western">Xin Lu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xin Lu</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Lu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fern%C3%A1ndez-Cuesta%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9" aria-expanded="false"><span class="name western">Lynnette Fernández-Cuesta</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lynnette Fernández-Cuesta</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fern%C3%A1ndez-Cuesta%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lynnette Fernández-Cuesta</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bosco%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id10" aria-expanded="false"><span class="name western">Graziella Bosco</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Graziella Bosco</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bosco%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Graziella Bosco</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11" aria-expanded="false"><span class="name western">Christian Müller</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Christian Müller</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Müller</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahmen%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id12" aria-expanded="false"><span class="name western">Ilona Dahmen</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Ilona Dahmen</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahmen%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilona Dahmen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jahchan%20NS%22%5BAuthor%5D" class="usa-link" aria-describedby="id13" aria-expanded="false"><span class="name western">Nadine S Jahchan</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Nadine S Jahchan</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jahchan%20NS%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadine S Jahchan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20KS%22%5BAuthor%5D" class="usa-link" aria-describedby="id14" aria-expanded="false"><span class="name western">Kwon-Sik Park</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Kwon-Sik Park</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20KS%22%5BAuthor%5D" class="usa-link"><span class="name western">Kwon-Sik Park</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id15" aria-expanded="false"><span class="name western">Dian Yang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Dian Yang</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dian Yang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karnezis%20AN%22%5BAuthor%5D" class="usa-link" aria-describedby="id16" aria-expanded="false"><span class="name western">Anthony N Karnezis</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Anthony N Karnezis</span></h3>
<div class="p">
<sup>6</sup>Vancouver General Hospital, Terry Fox laboratory, Vancouver, British Columbia V5Z 1L3, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karnezis%20AN%22%5BAuthor%5D" class="usa-link"><span class="name western">Anthony N Karnezis</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaka%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id17" aria-expanded="false"><span class="name western">Dedeepya Vaka</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Dedeepya Vaka</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaka%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dedeepya Vaka</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id18" aria-expanded="false"><span class="name western">Angela Torres</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Angela Torres</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Angela Torres</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id19" aria-expanded="false"><span class="name western">Maia Segura Wang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Maia Segura Wang</span></h3>
<div class="p">
<sup>7</sup>European Molecular Biology Laboratory, Genome Biology Unit, 69117 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Maia Segura Wang</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Korbel%20JO%22%5BAuthor%5D" class="usa-link" aria-describedby="id20" aria-expanded="false"><span class="name western">Jan O Korbel</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Jan O Korbel</span></h3>
<div class="p">
<sup>7</sup>European Molecular Biology Laboratory, Genome Biology Unit, 69117 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Korbel%20JO%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan O Korbel</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Menon%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id21" aria-expanded="false"><span class="name western">Roopika Menon</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Roopika Menon</span></h3>
<div class="p">
<sup>8</sup>Institute of Pathology, Center of Integrated Oncology Cologne–Bonn, University Hospital of Bonn, 53127 Bonn, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Menon%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roopika Menon</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chun%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id22" aria-expanded="false"><span class="name western">Sung-Min Chun</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Sung-Min Chun</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology and Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chun%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Sung-Min Chun</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id23" aria-expanded="false"><span class="name western">Deokhoon Kim</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Deokhoon Kim</span></h3>
<div class="p">
<sup>9</sup>Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Deokhoon Kim</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilkerson%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id24" aria-expanded="false"><span class="name western">Matt Wilkerson</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Matt Wilkerson</span></h3>
<div class="p">
<sup>10</sup>Department of Genetics, Lineberger Comprehensive Cancer Center, The University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilkerson%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matt Wilkerson</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayes%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id25" aria-expanded="false"><span class="name western">Neil Hayes</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Neil Hayes</span></h3>
<div class="p">
<sup>11</sup>UNC Lineberger Comprehensive Cancer Center School of Medicine, University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayes%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neil Hayes</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Engelmann%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id26" aria-expanded="false"><span class="name western">David Engelmann</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">David Engelmann</span></h3>
<div class="p">
<sup>12</sup>Institute of Experimental Gene Therapy and Cancer Research, Rostock University Medical Center, 18057 Rostock, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Engelmann%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Engelmann</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%BCtzer%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id27" aria-expanded="false"><span class="name western">Brigitte Pützer</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Brigitte Pützer</span></h3>
<div class="p">
<sup>12</sup>Institute of Experimental Gene Therapy and Cancer Research, Rostock University Medical Center, 18057 Rostock, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%BCtzer%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Brigitte Pützer</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bos%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id28" aria-expanded="false"><span class="name western">Marc Bos</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Marc Bos</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bos%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marc Bos</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Michels%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id29" aria-expanded="false"><span class="name western">Sebastian Michels</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Sebastian Michels</span></h3>
<div class="p">
<sup>13</sup>Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Michels%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sebastian Michels</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vlasic%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id30" aria-expanded="false"><span class="name western">Ignacija Vlasic</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Ignacija Vlasic</span></h3>
<div class="p">
<sup>14</sup>Department of Internal Medicine, University Hospital of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vlasic%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ignacija Vlasic</span></a>
</div>
</div>
<sup>14</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seidel%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id31" aria-expanded="false"><span class="name western">Danila Seidel</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Danila Seidel</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seidel%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Danila Seidel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pinther%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id32" aria-expanded="false"><span class="name western">Berit Pinther</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Berit Pinther</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pinther%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Berit Pinther</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schaub%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id33" aria-expanded="false"><span class="name western">Philipp Schaub</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Philipp Schaub</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schaub%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Philipp Schaub</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Becker%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id34" aria-expanded="false"><span class="name western">Christian Becker</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Christian Becker</span></h3>
<div class="p">
<sup>15</sup>Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Becker%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Becker</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Altm%C3%BCller%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id35" aria-expanded="false"><span class="name western">Janine Altmüller</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Janine Altmüller</span></h3>
<div class="p">
<sup>15</sup>Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>16</sup>Institute of Human Genetics, University Hospital Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Altm%C3%BCller%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Janine Altmüller</span></a>
</div>
</div>
<sup>15,</sup><sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yokota%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id36" aria-expanded="false"><span class="name western">Jun Yokota</span></a><div hidden="hidden" id="id36">
<h3><span class="name western">Jun Yokota</span></h3>
<div class="p">
<sup>17</sup>Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan</div>
<div class="p">
<sup>18</sup>Genomics and Epigenomics of Cancer Prediction Program, Institute of Predictive and Personalized Medicine of Cancer (IMPPC), Barcelona 08916, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yokota%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jun Yokota</span></a>
</div>
</div>
<sup>17,</sup><sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kohno%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id37" aria-expanded="false"><span class="name western">Takashi Kohno</span></a><div hidden="hidden" id="id37">
<h3><span class="name western">Takashi Kohno</span></h3>
<div class="p">
<sup>17</sup>Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kohno%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Takashi Kohno</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iwakawa%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id38" aria-expanded="false"><span class="name western">Reika Iwakawa</span></a><div hidden="hidden" id="id38">
<h3><span class="name western">Reika Iwakawa</span></h3>
<div class="p">
<sup>17</sup>Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iwakawa%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reika Iwakawa</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsuta%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id39" aria-expanded="false"><span class="name western">Koji Tsuta</span></a><div hidden="hidden" id="id39">
<h3><span class="name western">Koji Tsuta</span></h3>
<div class="p">
<sup>19</sup>Department of Pathology and Clinical Laboratories, National Cancer Center Hospital Chuo-ku, Tokyo 1040045, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsuta%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Koji Tsuta</span></a>
</div>
</div>
<sup>19</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Noguchi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id40" aria-expanded="false"><span class="name western">Masayuki Noguchi</span></a><div hidden="hidden" id="id40">
<h3><span class="name western">Masayuki Noguchi</span></h3>
<div class="p">
<sup>20</sup>Department of Pathology, Faculty of Medicine, University of Tsukuba, Ibaraki 305-8575, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Noguchi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Masayuki Noguchi</span></a>
</div>
</div>
<sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muley%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id41" aria-expanded="false"><span class="name western">Thomas Muley</span></a><div hidden="hidden" id="id41">
<h3><span class="name western">Thomas Muley</span></h3>
<div class="p">
<sup>21</sup>Thoraxklinik at University Hospital Heidelberg, Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div class="p">
<sup>22</sup>Translational Lung Research Center Heidelberg (TLRC-H), Member of German Center for Lung Research (DZL), Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muley%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Muley</span></a>
</div>
</div>
<sup>21,</sup><sup>22</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoffmann%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id42" aria-expanded="false"><span class="name western">Hans Hoffmann</span></a><div hidden="hidden" id="id42">
<h3><span class="name western">Hans Hoffmann</span></h3>
<div class="p">
<sup>21</sup>Thoraxklinik at University Hospital Heidelberg, Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoffmann%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hans Hoffmann</span></a>
</div>
</div>
<sup>21</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schnabel%20PA%22%5BAuthor%5D" class="usa-link" aria-describedby="id43" aria-expanded="false"><span class="name western">Philipp A Schnabel</span></a><div hidden="hidden" id="id43">
<h3><span class="name western">Philipp A Schnabel</span></h3>
<div class="p">
<sup>22</sup>Translational Lung Research Center Heidelberg (TLRC-H), Member of German Center for Lung Research (DZL), Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div class="p">
<sup>23</sup>Institute of Pathology, University of Heidelberg, Im Neuenheimer Feld 220, 69120 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schnabel%20PA%22%5BAuthor%5D" class="usa-link"><span class="name western">Philipp A Schnabel</span></a>
</div>
</div>
<sup>22,</sup><sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petersen%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id44" aria-expanded="false"><span class="name western">Iver Petersen</span></a><div hidden="hidden" id="id44">
<h3><span class="name western">Iver Petersen</span></h3>
<div class="p">
<sup>24</sup>Institute of Pathology, Jena University Hospital, Friedrich-Schiller-University, 07743 Jena, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petersen%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Iver Petersen</span></a>
</div>
</div>
<sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id45" aria-expanded="false"><span class="name western">Yuan Chen</span></a><div hidden="hidden" id="id45">
<h3><span class="name western">Yuan Chen</span></h3>
<div class="p">
<sup>24</sup>Institute of Pathology, Jena University Hospital, Friedrich-Schiller-University, 07743 Jena, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Chen</span></a>
</div>
</div>
<sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Soltermann%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id46" aria-expanded="false"><span class="name western">Alex Soltermann</span></a><div hidden="hidden" id="id46">
<h3><span class="name western">Alex Soltermann</span></h3>
<div class="p">
<sup>25</sup>Institute of Surgical Pathology, University Hospital Zürich, 8091 Zürich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Soltermann%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alex Soltermann</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tischler%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id47" aria-expanded="false"><span class="name western">Verena Tischler</span></a><div hidden="hidden" id="id47">
<h3><span class="name western">Verena Tischler</span></h3>
<div class="p">
<sup>25</sup>Institute of Surgical Pathology, University Hospital Zürich, 8091 Zürich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tischler%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Verena Tischler</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id48" aria-expanded="false"><span class="name western">Chang-min Choi</span></a><div hidden="hidden" id="id48">
<h3><span class="name western">Chang-min Choi</span></h3>
<div class="p">
<sup>26</sup>Department of Oncology, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Chang-min Choi</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id49" aria-expanded="false"><span class="name western">Yong-Hee Kim</span></a><div hidden="hidden" id="id49">
<h3><span class="name western">Yong-Hee Kim</span></h3>
<div class="p">
<sup>27</sup>Department of Thoracic and Cardiovascular Surgery, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong-Hee Kim</span></a>
</div>
</div>
<sup>27</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massion%20PP%22%5BAuthor%5D" class="usa-link" aria-describedby="id50" aria-expanded="false"><span class="name western">Pierre P Massion</span></a><div hidden="hidden" id="id50">
<h3><span class="name western">Pierre P Massion</span></h3>
<div class="p">
<sup>28</sup>Thoracic Program, Vanderbilt-Ingram Cancer Center PRB 640, 2220 Pierce Avenue, Nashville, Tennessee 37232, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massion%20PP%22%5BAuthor%5D" class="usa-link"><span class="name western">Pierre P Massion</span></a>
</div>
</div>
<sup>28</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id51" aria-expanded="false"><span class="name western">Yong Zou</span></a><div hidden="hidden" id="id51">
<h3><span class="name western">Yong Zou</span></h3>
<div class="p">
<sup>28</sup>Thoracic Program, Vanderbilt-Ingram Cancer Center PRB 640, 2220 Pierce Avenue, Nashville, Tennessee 37232, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Zou</span></a>
</div>
</div>
<sup>28</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jovanovic%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id52" aria-expanded="false"><span class="name western">Dragana Jovanovic</span></a><div hidden="hidden" id="id52">
<h3><span class="name western">Dragana Jovanovic</span></h3>
<div class="p">
<sup>29</sup>University Hospital of Pulmonology, Clinical Center of Serbia, Medical School, University of Belgrade, 11000 Belgrade, Serbia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jovanovic%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dragana Jovanovic</span></a>
</div>
</div>
<sup>29</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kontic%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id53" aria-expanded="false"><span class="name western">Milica Kontic</span></a><div hidden="hidden" id="id53">
<h3><span class="name western">Milica Kontic</span></h3>
<div class="p">
<sup>29</sup>University Hospital of Pulmonology, Clinical Center of Serbia, Medical School, University of Belgrade, 11000 Belgrade, Serbia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kontic%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Milica Kontic</span></a>
</div>
</div>
<sup>29</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wright%20GM%22%5BAuthor%5D" class="usa-link" aria-describedby="id54" aria-expanded="false"><span class="name western">Gavin M Wright</span></a><div hidden="hidden" id="id54">
<h3><span class="name western">Gavin M Wright</span></h3>
<div class="p">
<sup>30</sup>Department of Surgery, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wright%20GM%22%5BAuthor%5D" class="usa-link"><span class="name western">Gavin M Wright</span></a>
</div>
</div>
<sup>30</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Russell%20PA%22%5BAuthor%5D" class="usa-link" aria-describedby="id55" aria-expanded="false"><span class="name western">Prudence A Russell</span></a><div hidden="hidden" id="id55">
<h3><span class="name western">Prudence A Russell</span></h3>
<div class="p">
<sup>31</sup>Department of Pathology, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Russell%20PA%22%5BAuthor%5D" class="usa-link"><span class="name western">Prudence A Russell</span></a>
</div>
</div>
<sup>31</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id56" aria-expanded="false"><span class="name western">Benjamin Solomon</span></a><div hidden="hidden" id="id56">
<h3><span class="name western">Benjamin Solomon</span></h3>
<div class="p">
<sup>32</sup>Department of Haematology and Medical Oncology, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Benjamin Solomon</span></a>
</div>
</div>
<sup>32</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id57" aria-expanded="false"><span class="name western">Ina Koch</span></a><div hidden="hidden" id="id57">
<h3><span class="name western">Ina Koch</span></h3>
<div class="p">
<sup>33</sup>Asklepios Biobank für Lungenerkrankungen, Comprehensive Pneumology Center Munich, Member of the German Center for Lung Research (DZL), Asklepios Fachkliniken München-Gauting 82131, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ina Koch</span></a>
</div>
</div>
<sup>33</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id58" aria-expanded="false"><span class="name western">Michael Lindner</span></a><div hidden="hidden" id="id58">
<h3><span class="name western">Michael Lindner</span></h3>
<div class="p">
<sup>33</sup>Asklepios Biobank für Lungenerkrankungen, Comprehensive Pneumology Center Munich, Member of the German Center for Lung Research (DZL), Asklepios Fachkliniken München-Gauting 82131, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Lindner</span></a>
</div>
</div>
<sup>33</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muscarella%20LA%22%5BAuthor%5D" class="usa-link" aria-describedby="id59" aria-expanded="false"><span class="name western">Lucia A Muscarella</span></a><div hidden="hidden" id="id59">
<h3><span class="name western">Lucia A Muscarella</span></h3>
<div class="p">
<sup>34</sup>Laboratory of Oncology, IRCCS Casa Sollievo della Sofferenza, Viale Cappuccini, 71013 San Giovanni, Rotondo, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muscarella%20LA%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucia A Muscarella</span></a>
</div>
</div>
<sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22la%20Torre%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id60" aria-expanded="false"><span class="name western">Annamaria la Torre</span></a><div hidden="hidden" id="id60">
<h3><span class="name western">Annamaria la Torre</span></h3>
<div class="p">
<sup>34</sup>Laboratory of Oncology, IRCCS Casa Sollievo della Sofferenza, Viale Cappuccini, 71013 San Giovanni, Rotondo, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22la%20Torre%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annamaria la Torre</span></a>
</div>
</div>
<sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id61" aria-expanded="false"><span class="name western">John K Field</span></a><div hidden="hidden" id="id61">
<h3><span class="name western">John K Field</span></h3>
<div class="p">
<sup>35</sup>Roy Castle Lung Cancer Research Programme, Department of Molecular and Clinical Cancer Medicine, Institute of Translational Medicine, The University of Liverpool Cancer Research Centre, 200 London Road, L69 3GA Liverpool, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">John K Field</span></a>
</div>
</div>
<sup>35</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jakopovic%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id62" aria-expanded="false"><span class="name western">Marko Jakopovic</span></a><div hidden="hidden" id="id62">
<h3><span class="name western">Marko Jakopovic</span></h3>
<div class="p">
<sup>36</sup>University of Zagreb, School of Medicine, Department for Respiratory Diseases Jordanovac, University Hospital Center Zagreb, 10000 Zagreb, Croatia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jakopovic%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marko Jakopovic</span></a>
</div>
</div>
<sup>36</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knezevic%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id63" aria-expanded="false"><span class="name western">Jelena Knezevic</span></a><div hidden="hidden" id="id63">
<h3><span class="name western">Jelena Knezevic</span></h3>
<div class="p">
<sup>37</sup>Laboratory for Translational Medicine, Rudjer Boskovic Institute, 10000 Zagreb, Croatia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knezevic%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jelena Knezevic</span></a>
</div>
</div>
<sup>37</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casta%C3%B1os-V%C3%A9lez%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id64" aria-expanded="false"><span class="name western">Esmeralda Castaños-Vélez</span></a><div hidden="hidden" id="id64">
<h3><span class="name western">Esmeralda Castaños-Vélez</span></h3>
<div class="p">
<sup>38</sup>Charité Comprehensive Cancer Center, Charité Campus Mitte, 10115 Berlin, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casta%C3%B1os-V%C3%A9lez%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Esmeralda Castaños-Vélez</span></a>
</div>
</div>
<sup>38</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roz%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id65" aria-expanded="false"><span class="name western">Luca Roz</span></a><div hidden="hidden" id="id65">
<h3><span class="name western">Luca Roz</span></h3>
<div class="p">
<sup>39</sup>Tumor Genomics Unit, Department of Experimental Oncology and Molecular Medicine, Fondazione IRCCS — Istituto Nazionale Tumori, Via Venezian 1, 20133 Milan, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roz%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luca Roz</span></a>
</div>
</div>
<sup>39</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pastorino%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id66" aria-expanded="false"><span class="name western">Ugo Pastorino</span></a><div hidden="hidden" id="id66">
<h3><span class="name western">Ugo Pastorino</span></h3>
<div class="p">
<sup>40</sup>Thoracic Surgery Unit, Department of Surgery, Fondazione IRCCS Istituto Nazionale Tumori, 20133 Milan, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pastorino%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Ugo Pastorino</span></a>
</div>
</div>
<sup>40</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brustugun%20OT%22%5BAuthor%5D" class="usa-link" aria-describedby="id67" aria-expanded="false"><span class="name western">Odd-Terje Brustugun</span></a><div hidden="hidden" id="id67">
<h3><span class="name western">Odd-Terje Brustugun</span></h3>
<div class="p">
<sup>41</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, N-0424 Oslo, Norway</div>
<div class="p">
<sup>42</sup>Department of Oncology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brustugun%20OT%22%5BAuthor%5D" class="usa-link"><span class="name western">Odd-Terje Brustugun</span></a>
</div>
</div>
<sup>41,</sup><sup>42</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lund-Iversen%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id68" aria-expanded="false"><span class="name western">Marius Lund-Iversen</span></a><div hidden="hidden" id="id68">
<h3><span class="name western">Marius Lund-Iversen</span></h3>
<div class="p">
<sup>43</sup>Department of Pathology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lund-Iversen%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marius Lund-Iversen</span></a>
</div>
</div>
<sup>43</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thunnissen%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id69" aria-expanded="false"><span class="name western">Erik Thunnissen</span></a><div hidden="hidden" id="id69">
<h3><span class="name western">Erik Thunnissen</span></h3>
<div class="p">
<sup>44</sup>Department of Pathology, VU University Medical Center, 1007 MB Amsterdam, The Netherlands</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thunnissen%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erik Thunnissen</span></a>
</div>
</div>
<sup>44</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6hler%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id70" aria-expanded="false"><span class="name western">Jens Köhler</span></a><div hidden="hidden" id="id70">
<h3><span class="name western">Jens Köhler</span></h3>
<div class="p">
<sup>45</sup>West German Cancer Center, Department of Medical Oncology, University Hospital Essen, 45147 Essen, Germany</div>
<div class="p">
<sup>46</sup>German Cancer Consortium (DKTK), 69120 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6hler%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Köhler</span></a>
</div>
</div>
<sup>45,</sup><sup>46</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schuler%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id71" aria-expanded="false"><span class="name western">Martin Schuler</span></a><div hidden="hidden" id="id71">
<h3><span class="name western">Martin Schuler</span></h3>
<div class="p">
<sup>45</sup>West German Cancer Center, Department of Medical Oncology, University Hospital Essen, 45147 Essen, Germany</div>
<div class="p">
<sup>46</sup>German Cancer Consortium (DKTK), 69120 Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schuler%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Schuler</span></a>
</div>
</div>
<sup>45,</sup><sup>46</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Botling%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id72" aria-expanded="false"><span class="name western">Johan Botling</span></a><div hidden="hidden" id="id72">
<h3><span class="name western">Johan Botling</span></h3>
<div class="p">
<sup>47</sup>Departments of Immunology, Genetics and Pathology, and Medical Sciences, Respiratory, Allergy and Sleep Research, Uppsala University, 75185 Uppsala, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Botling%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Johan Botling</span></a>
</div>
</div>
<sup>47</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandelin%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id73" aria-expanded="false"><span class="name western">Martin Sandelin</span></a><div hidden="hidden" id="id73">
<h3><span class="name western">Martin Sandelin</span></h3>
<div class="p">
<sup>47</sup>Departments of Immunology, Genetics and Pathology, and Medical Sciences, Respiratory, Allergy and Sleep Research, Uppsala University, 75185 Uppsala, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandelin%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Sandelin</span></a>
</div>
</div>
<sup>47</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez-Cespedes%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id74" aria-expanded="false"><span class="name western">Montserrat Sanchez-Cespedes</span></a><div hidden="hidden" id="id74">
<h3><span class="name western">Montserrat Sanchez-Cespedes</span></h3>
<div class="p">
<sup>48</sup>Genes and Cancer Group, Cancer Epigenetics and Biology Program (PEBC), Bellvitge Biomedical Research Institute (IDIBELL), 08908 Hospitalet de Llobregat, Barcelona, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez-Cespedes%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Montserrat Sanchez-Cespedes</span></a>
</div>
</div>
<sup>48</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salvesen%20HB%22%5BAuthor%5D" class="usa-link" aria-describedby="id75" aria-expanded="false"><span class="name western">Helga B Salvesen</span></a><div hidden="hidden" id="id75">
<h3><span class="name western">Helga B Salvesen</span></h3>
<div class="p">
<sup>49</sup>Department of Clinical Science, Center for Cancer Biomarkers, University of Bergen, N-5058 Bergen, Norway</div>
<div class="p">
<sup>50</sup>Department of Gynecology and Obstetrics, Haukeland University Hospital, N-5058 Bergen, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salvesen%20HB%22%5BAuthor%5D" class="usa-link"><span class="name western">Helga B Salvesen</span></a>
</div>
</div>
<sup>49,</sup><sup>50</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achter%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id76" aria-expanded="false"><span class="name western">Viktor Achter</span></a><div hidden="hidden" id="id76">
<h3><span class="name western">Viktor Achter</span></h3>
<div class="p">
<sup>51</sup>Computing Center, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achter%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Viktor Achter</span></a>
</div>
</div>
<sup>51</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lang%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id77" aria-expanded="false"><span class="name western">Ulrich Lang</span></a><div hidden="hidden" id="id77">
<h3><span class="name western">Ulrich Lang</span></h3>
<div class="p">
<sup>51</sup>Computing Center, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>52</sup>Department of Informatics, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lang%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Ulrich Lang</span></a>
</div>
</div>
<sup>51,</sup><sup>52</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogus%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id78" aria-expanded="false"><span class="name western">Magdalena Bogus</span></a><div hidden="hidden" id="id78">
<h3><span class="name western">Magdalena Bogus</span></h3>
<div class="p">
<sup>53</sup>Institute of Legal Medicine, University of Cologne, 50823 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogus%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Magdalena Bogus</span></a>
</div>
</div>
<sup>53</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schneider%20PM%22%5BAuthor%5D" class="usa-link" aria-describedby="id79" aria-expanded="false"><span class="name western">Peter M Schneider</span></a><div hidden="hidden" id="id79">
<h3><span class="name western">Peter M Schneider</span></h3>
<div class="p">
<sup>53</sup>Institute of Legal Medicine, University of Cologne, 50823 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schneider%20PM%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter M Schneider</span></a>
</div>
</div>
<sup>53</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zander%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id80" aria-expanded="false"><span class="name western">Thomas Zander</span></a><div hidden="hidden" id="id80">
<h3><span class="name western">Thomas Zander</span></h3>
<div class="p">
<sup>54</sup>Gastrointestinal Cancer Group Cologne, Center of Integrated Oncology Cologne–Bonn, Department I for Internal Medicine, University Hospital of Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zander%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Zander</span></a>
</div>
</div>
<sup>54</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ans%C3%A9n%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id81" aria-expanded="false"><span class="name western">Sascha Ansén</span></a><div hidden="hidden" id="id81">
<h3><span class="name western">Sascha Ansén</span></h3>
<div class="p">
<sup>13</sup>Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ans%C3%A9n%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sascha Ansén</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hallek%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id82" aria-expanded="false"><span class="name western">Michael Hallek</span></a><div hidden="hidden" id="id82">
<h3><span class="name western">Michael Hallek</span></h3>
<div class="p">
<sup>13</sup>Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">
<sup>55</sup>Cologne Excellence Cluster on Cellular Stress Responses in Aging-Associated Diseases (CECAD), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hallek%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Hallek</span></a>
</div>
</div>
<sup>13,</sup><sup>55</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id83" aria-expanded="false"><span class="name western">Jürgen Wolf</span></a><div hidden="hidden" id="id83">
<h3><span class="name western">Jürgen Wolf</span></h3>
<div class="p">
<sup>13</sup>Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jürgen Wolf</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vingron%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id84" aria-expanded="false"><span class="name western">Martin Vingron</span></a><div hidden="hidden" id="id84">
<h3><span class="name western">Martin Vingron</span></h3>
<div class="p">
<sup>56</sup>Computational Molecular Biology Group, Max Planck Institute for Molecular Genetics, 14195 Berlin, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vingron%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Vingron</span></a>
</div>
</div>
<sup>56</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yatabe%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id85" aria-expanded="false"><span class="name western">Yasushi Yatabe</span></a><div hidden="hidden" id="id85">
<h3><span class="name western">Yasushi Yatabe</span></h3>
<div class="p">
<sup>57</sup>Department of Pathology and Molecular Diagnostics, Aichi Cancer Center, 464-8681 Nagoya, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yatabe%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yasushi Yatabe</span></a>
</div>
</div>
<sup>57</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Travis%20WD%22%5BAuthor%5D" class="usa-link" aria-describedby="id86" aria-expanded="false"><span class="name western">William D Travis</span></a><div hidden="hidden" id="id86">
<h3><span class="name western">William D Travis</span></h3>
<div class="p">
<sup>58</sup>Department of Pathology, Memorial Sloan-Kettering Cancer Center, New York 10065, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Travis%20WD%22%5BAuthor%5D" class="usa-link"><span class="name western">William D Travis</span></a>
</div>
</div>
<sup>58</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22N%C3%BCrnberg%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id87" aria-expanded="false"><span class="name western">Peter Nürnberg</span></a><div hidden="hidden" id="id87">
<h3><span class="name western">Peter Nürnberg</span></h3>
<div class="p">
<sup>15</sup>Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>55</sup>Cologne Excellence Cluster on Cellular Stress Responses in Aging-Associated Diseases (CECAD), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>59</sup>Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22N%C3%BCrnberg%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Nürnberg</span></a>
</div>
</div>
<sup>15,</sup><sup>55,</sup><sup>59</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reinhardt%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id88" aria-expanded="false"><span class="name western">Christian Reinhardt</span></a><div hidden="hidden" id="id88">
<h3><span class="name western">Christian Reinhardt</span></h3>
<div class="p">
<sup>14</sup>Department of Internal Medicine, University Hospital of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reinhardt%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Reinhardt</span></a>
</div>
</div>
<sup>14</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perner%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id89" aria-expanded="false"><span class="name western">Sven Perner</span></a><div hidden="hidden" id="id89">
<h3><span class="name western">Sven Perner</span></h3>
<div class="p">
<sup>9</sup>Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perner%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sven Perner</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heukamp%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id90" aria-expanded="false"><span class="name western">Lukas Heukamp</span></a><div hidden="hidden" id="id90">
<h3><span class="name western">Lukas Heukamp</span></h3>
<div class="p">
<sup>4</sup>Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heukamp%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lukas Heukamp</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%BCttner%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id91" aria-expanded="false"><span class="name western">Reinhard Büttner</span></a><div hidden="hidden" id="id91">
<h3><span class="name western">Reinhard Büttner</span></h3>
<div class="p">
<sup>4</sup>Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%BCttner%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reinhard Büttner</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haas%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id92" aria-expanded="false"><span class="name western">Stefan A Haas</span></a><div hidden="hidden" id="id92">
<h3><span class="name western">Stefan A Haas</span></h3>
<div class="p">
<sup>56</sup>Computational Molecular Biology Group, Max Planck Institute for Molecular Genetics, 14195 Berlin, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haas%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefan A Haas</span></a>
</div>
</div>
<sup>56</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brambilla%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id93" aria-expanded="false"><span class="name western">Elisabeth Brambilla</span></a><div hidden="hidden" id="id93">
<h3><span class="name western">Elisabeth Brambilla</span></h3>
<div class="p">
<sup>60</sup>Department of Pathology, CHU Grenoble INSERM U823, University Joseph Fourier, Institute Albert Bonniot 38043, CS10217 Grenoble, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brambilla%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elisabeth Brambilla</span></a>
</div>
</div>
<sup>60</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peifer%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id94" aria-expanded="false"><span class="name western">Martin Peifer</span></a><div hidden="hidden" id="id94">
<h3><span class="name western">Martin Peifer</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>59</sup>Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peifer%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Peifer</span></a>
</div>
</div>
<sup>1,</sup><sup>59</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sage%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id95" aria-expanded="false"><span class="name western">Julien Sage</span></a><div hidden="hidden" id="id95">
<h3><span class="name western">Julien Sage</span></h3>
<div class="p">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sage%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julien Sage</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id96" aria-expanded="false"><span class="name western">Roman K Thomas</span></a><div hidden="hidden" id="id96">
<h3><span class="name western">Roman K Thomas</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div class="p">
<sup>4</sup>Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman K Thomas</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Department of Translational Genomics, Center of Integrated Oncology Cologne–Bonn, Medical Faculty, University of Cologne, 50931 Cologne, Germany</div>
<div id="A2">
<sup>2</sup>Departments of Pediatrics and Genetics, Stanford University, Stanford, California 94305, USA</div>
<div id="A3">
<sup>3</sup>Department of Pathology and Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div id="A4">
<sup>4</sup>Department of Pathology, University Hospital Cologne, 50937 Cologne, Germany</div>
<div id="A5">
<sup>5</sup>Department of Pathology, College of Medicine, Hanyang University. 222 Wangsimniro, Seongdong-gu, Seoul 133-791, Korea</div>
<div id="A6">
<sup>6</sup>Vancouver General Hospital, Terry Fox laboratory, Vancouver, British Columbia V5Z 1L3, Canada</div>
<div id="A7">
<sup>7</sup>European Molecular Biology Laboratory, Genome Biology Unit, 69117 Heidelberg, Germany</div>
<div id="A8">
<sup>8</sup>Institute of Pathology, Center of Integrated Oncology Cologne–Bonn, University Hospital of Bonn, 53127 Bonn, Germany</div>
<div id="A9">
<sup>9</sup>Center for Cancer Genome Discovery, University of Ulsan College of Medicine, Asan Medical Center 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div id="A10">
<sup>10</sup>Department of Genetics, Lineberger Comprehensive Cancer Center, The University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA</div>
<div id="A11">
<sup>11</sup>UNC Lineberger Comprehensive Cancer Center School of Medicine, University of North Carolina at Chapel Hill, North Carolina 27599-7295, USA</div>
<div id="A12">
<sup>12</sup>Institute of Experimental Gene Therapy and Cancer Research, Rostock University Medical Center, 18057 Rostock, Germany</div>
<div id="A13">
<sup>13</sup>Department I of Internal Medicine, Center of Integrated Oncology Cologne-Bonn, University Hospital Cologne, 50937 Cologne, Germany</div>
<div id="A14">
<sup>14</sup>Department of Internal Medicine, University Hospital of Cologne, 50931 Cologne, Germany</div>
<div id="A15">
<sup>15</sup>Cologne Center for Genomics (CCG), University of Cologne, 50931 Cologne, Germany</div>
<div id="A16">
<sup>16</sup>Institute of Human Genetics, University Hospital Cologne, 50931 Cologne, Germany</div>
<div id="A17">
<sup>17</sup>Division of Genome Biology, National Cancer Center Research Institute, Chuo-ku, Tokyo 1040045, Japan</div>
<div id="A18">
<sup>18</sup>Genomics and Epigenomics of Cancer Prediction Program, Institute of Predictive and Personalized Medicine of Cancer (IMPPC), Barcelona 08916, Spain</div>
<div id="A19">
<sup>19</sup>Department of Pathology and Clinical Laboratories, National Cancer Center Hospital Chuo-ku, Tokyo 1040045, Japan</div>
<div id="A20">
<sup>20</sup>Department of Pathology, Faculty of Medicine, University of Tsukuba, Ibaraki 305-8575, Japan</div>
<div id="A21">
<sup>21</sup>Thoraxklinik at University Hospital Heidelberg, Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div id="A22">
<sup>22</sup>Translational Lung Research Center Heidelberg (TLRC-H), Member of German Center for Lung Research (DZL), Amalienstrasse 5, 69126 Heidelberg, Germany</div>
<div id="A23">
<sup>23</sup>Institute of Pathology, University of Heidelberg, Im Neuenheimer Feld 220, 69120 Heidelberg, Germany</div>
<div id="A24">
<sup>24</sup>Institute of Pathology, Jena University Hospital, Friedrich-Schiller-University, 07743 Jena, Germany</div>
<div id="A25">
<sup>25</sup>Institute of Surgical Pathology, University Hospital Zürich, 8091 Zürich, Switzerland</div>
<div id="A26">
<sup>26</sup>Department of Oncology, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div id="A27">
<sup>27</sup>Department of Thoracic and Cardiovascular Surgery, University of Ulsan College of Medicine, Asan Medical Center, 88, Olympic-ro 43-gil, Songpa-gu, Seoul 138-736, Korea</div>
<div id="A28">
<sup>28</sup>Thoracic Program, Vanderbilt-Ingram Cancer Center PRB 640, 2220 Pierce Avenue, Nashville, Tennessee 37232, USA</div>
<div id="A29">
<sup>29</sup>University Hospital of Pulmonology, Clinical Center of Serbia, Medical School, University of Belgrade, 11000 Belgrade, Serbia</div>
<div id="A30">
<sup>30</sup>Department of Surgery, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div id="A31">
<sup>31</sup>Department of Pathology, St. Vincent’s Hospital, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div id="A32">
<sup>32</sup>Department of Haematology and Medical Oncology, Peter MacCallum Cancer Centre, 3065 Melbourne, Victoria, Australia</div>
<div id="A33">
<sup>33</sup>Asklepios Biobank für Lungenerkrankungen, Comprehensive Pneumology Center Munich, Member of the German Center for Lung Research (DZL), Asklepios Fachkliniken München-Gauting 82131, Germany</div>
<div id="A34">
<sup>34</sup>Laboratory of Oncology, IRCCS Casa Sollievo della Sofferenza, Viale Cappuccini, 71013 San Giovanni, Rotondo, Italy</div>
<div id="A35">
<sup>35</sup>Roy Castle Lung Cancer Research Programme, Department of Molecular and Clinical Cancer Medicine, Institute of Translational Medicine, The University of Liverpool Cancer Research Centre, 200 London Road, L69 3GA Liverpool, UK</div>
<div id="A36">
<sup>36</sup>University of Zagreb, School of Medicine, Department for Respiratory Diseases Jordanovac, University Hospital Center Zagreb, 10000 Zagreb, Croatia</div>
<div id="A37">
<sup>37</sup>Laboratory for Translational Medicine, Rudjer Boskovic Institute, 10000 Zagreb, Croatia</div>
<div id="A38">
<sup>38</sup>Charité Comprehensive Cancer Center, Charité Campus Mitte, 10115 Berlin, Germany</div>
<div id="A39">
<sup>39</sup>Tumor Genomics Unit, Department of Experimental Oncology and Molecular Medicine, Fondazione IRCCS — Istituto Nazionale Tumori, Via Venezian 1, 20133 Milan, Italy</div>
<div id="A40">
<sup>40</sup>Thoracic Surgery Unit, Department of Surgery, Fondazione IRCCS Istituto Nazionale Tumori, 20133 Milan, Italy</div>
<div id="A41">
<sup>41</sup>Institute of Clinical Medicine, Faculty of Medicine, University of Oslo, N-0424 Oslo, Norway</div>
<div id="A42">
<sup>42</sup>Department of Oncology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway</div>
<div id="A43">
<sup>43</sup>Department of Pathology, Norwegian Radium Hospital, Oslo University Hospital, N-0310 Oslo, Norway</div>
<div id="A44">
<sup>44</sup>Department of Pathology, VU University Medical Center, 1007 MB Amsterdam, The Netherlands</div>
<div id="A45">
<sup>45</sup>West German Cancer Center, Department of Medical Oncology, University Hospital Essen, 45147 Essen, Germany</div>
<div id="A46">
<sup>46</sup>German Cancer Consortium (DKTK), 69120 Heidelberg, Germany</div>
<div id="A47">
<sup>47</sup>Departments of Immunology, Genetics and Pathology, and Medical Sciences, Respiratory, Allergy and Sleep Research, Uppsala University, 75185 Uppsala, Sweden</div>
<div id="A48">
<sup>48</sup>Genes and Cancer Group, Cancer Epigenetics and Biology Program (PEBC), Bellvitge Biomedical Research Institute (IDIBELL), 08908 Hospitalet de Llobregat, Barcelona, Spain</div>
<div id="A49">
<sup>49</sup>Department of Clinical Science, Center for Cancer Biomarkers, University of Bergen, N-5058 Bergen, Norway</div>
<div id="A50">
<sup>50</sup>Department of Gynecology and Obstetrics, Haukeland University Hospital, N-5058 Bergen, Norway</div>
<div id="A51">
<sup>51</sup>Computing Center, University of Cologne, 50931 Cologne, Germany</div>
<div id="A52">
<sup>52</sup>Department of Informatics, University of Cologne, 50931 Cologne, Germany</div>
<div id="A53">
<sup>53</sup>Institute of Legal Medicine, University of Cologne, 50823 Cologne, Germany</div>
<div id="A54">
<sup>54</sup>Gastrointestinal Cancer Group Cologne, Center of Integrated Oncology Cologne–Bonn, Department I for Internal Medicine, University Hospital of Cologne, 50937 Cologne, Germany</div>
<div id="A55">
<sup>55</sup>Cologne Excellence Cluster on Cellular Stress Responses in Aging-Associated Diseases (CECAD), University of Cologne, 50931 Cologne, Germany</div>
<div id="A56">
<sup>56</sup>Computational Molecular Biology Group, Max Planck Institute for Molecular Genetics, 14195 Berlin, Germany</div>
<div id="A57">
<sup>57</sup>Department of Pathology and Molecular Diagnostics, Aichi Cancer Center, 464-8681 Nagoya, Japan</div>
<div id="A58">
<sup>58</sup>Department of Pathology, Memorial Sloan-Kettering Cancer Center, New York 10065, USA</div>
<div id="A59">
<sup>59</sup>Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany</div>
<div id="A60">
<sup>60</sup>Department of Pathology, CHU Grenoble INSERM U823, University Joseph Fourier, Institute Albert Bonniot 38043, CS10217 Grenoble, France</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Correspondence and requests for materials should be addressed to R.K.T. (<span>roman.thomas@uni-koeln.de</span>), J.S. (<span>julsage@stanford.edu</span>) or M.P. (<span>mpeifer@uni-koeln.de</span>)</p>
</div>
<div class="fn" id="FN1">
<sup>†</sup><p class="display-inline" id="P1">Present address: Genetic Cancer Susceptibility Group, Section of Genetics, International Agency for Research on Cancer (IARC-WHO), 69008 Lyon, France.</p>
</div>
<div class="fn" id="FN2">
<sup>*</sup><p class="display-inline" id="P2">These authors contributed equally to this work.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Issue date 2015 Aug 6.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<p>Reprints and permissions information is available at <a href="http://www.nature.com/reprints" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.nature.com/reprints</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4861069&nbsp;&nbsp;NIHMSID: NIHMS782739&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/26168399/" class="usa-link">26168399</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>The publisher's version of this article is available at <a href="https://doi.org/10.1038/nature14664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Nature</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2 data-anchor-id="abstract1"><a id="abstract1-anchor" data-anchor-id="abstract1" class="usa-anchor"></a>Abstract</h2>
<p id="P3">We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of <em>TP53</em> and <em>RB1</em>, sometimes by complex genomic rearrangements. Two tumours with wild-type <em>RB1</em> had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the <em>CCND1</em> gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors <em>TP53</em> and <em>RB1</em> is obligatory in SCLC. We discovered somatic genomic rearrangements of <em>TP73</em> that create an oncogenic version of this gene, <em>TP73Δex2/3</em>. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in <em>NOTCH</em> family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.</p></section><hr class="headless">
<p id="P4">Small cell lung cancer (SCLC) accounts for approximately 15% of all lung cancers, arises in heavy smokers, and the tumour cells express neuroendocrine markers. Although chemotherapy is initially effective in the treatment of SCLC, recurrence arises rapidly in the vast majority of cases, usually killing the patient within only a few months<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a></sup>. SCLC is rarely treated by surgery and few specimens are available for genomic characterization. Previous studies applying mostly exome sequencing in a limited number of tumour specimens have revealed only a few recurrently mutated genes<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup>.</p>
<p id="P5">We hypothesized that complex genomic rearrangements, which are undetectable by exome sequencing, might further contribute to the pathogenesis of SCLC and thus performed whole-genome sequencing of 110 human SCLC specimens (<a href="#SD1" class="usa-link">Supplementary Tables 1–4</a>). One of the hallmarks of SCLC is the high frequency of mutations in <em>TP53</em> and <em>RB1</em> (refs <a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>–<a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a>). As mice lacking <em>Trp53</em> and <em>Rb1</em> in the lung develop SCLC<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a>,<a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>, we also sequenced 8 of these murine SCLC tumours in order to identify mutations that may promote SCLC development following loss of <em>Trp53</em> and <em>Rb1</em> and that may overlap with such accessory genes in human SCLC<sup><a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a></sup> (<a href="#SD1" class="usa-link">Supplementary Table 5</a>).</p>
<section id="S1"><h2 class="pmc_sec_title" data-anchor-id="S1"><a id="S1-anchor" data-anchor-id="S1" class="usa-anchor"></a>Samples and clinical data</h2>
<p id="P6">We collected 152 fresh-frozen clinical tumour specimens obtained from patients diagnosed with stage I–IV SCLC under institutional review board approval (<a href="#SD1" class="usa-link">Supplementary Table 1</a> and <a href="#F6" class="usa-link">Extended Data Fig. 1</a>). The tumour samples were enriched for earlier stages and consisted of primary lung (<em>n</em> = 148) and metastatic tumours (<em>n</em> = 4) obtained by surgical resection (<em>n</em> = 132), biopsy (<em>n</em> = 4), pleural effusion (<em>n</em> = 1) or through autopsy (<em>n</em> = 15). We performed whole-genome sequencing on 110 of these tumours and their matched normal DNA. A total of 42 cases were excluded from the analysis because of insufficient quality or amount of DNA. Most of these 110 tumours were treatment-naive, with only five cases obtained at the time of relapse. We analysed transcriptome sequencing data in 71 of the 110 specimens that had undergone genome sequencing and in 10 additional specimens. Finally, 103 of the 110 genome-sequenced specimens and 39 additional specimens were analysed by Affymetrix 6.0 SNP arrays (<a href="#SD1" class="usa-link">Supplementary Table 1</a> and <a href="#F6" class="usa-link">Extended Data Fig. 1</a>). Eight tumour samples from preclinical SCLC mouse models were analysed by whole-exome sequencing (<em>n</em> = 6) or whole-genome sequencing (<em>n</em> = 2) (<a href="#SD1" class="usa-link">Supplementary Table 5</a>).</p></section><section id="S2"><h2 class="pmc_sec_title" data-anchor-id="S2"><a id="S2-anchor" data-anchor-id="S2" class="usa-anchor"></a>Recurrent somatic alterations in SCLC</h2>
<p id="P7">SCLC genomes exhibited extremely high mutation rates<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup> of 8.62 nonsynonomous mutations per million base pairs (Mb). C:G&gt;A:T transversions were found in 28% of all mutations on average, a pattern indicative of heavy smoking (<a href="#F1" class="usa-link">Fig. 1a</a> and <a href="#SD1" class="usa-link">Supplementary Tables 2 and 3</a>). The smoking history or clinical stage of the tumours did not correlate with the type and number of mutations (<a href="#F7" class="usa-link">Extended Data Fig. 2</a>). The median tumour content was 84% (<a href="#F8" class="usa-link">Extended Data Fig. 3a</a> and <a href="#SD1" class="usa-link">Supplementary Table 2</a>). By contrast, murine SCLC tumours showed a low number of somatic alterations (on average 28.5 protein-altering mutations per sample on average)<sup><a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a></sup> (<a href="#SD1" class="usa-link">Supplementary Table 5</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Figure 1. Genomic alterations in small cell lung cancer.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/54aa2128880d/nihms782739f1.jpg" loading="lazy" height="568" width="1050" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P93"><strong>a</strong>, Tumour samples are arranged from left to right. Alterations of SCLC candidate genes are annotated for each sample according to the colour panel below the image. The somatic mutation frequencies for each candidate gene are plotted on the right panel. Mutation rates and type of base-pair substitution are displayed in the top and bottom panel, respectively. Significant candidate genes are highlighted in bold (*corrected <em>q</em>-values &lt; 0.05, †<em>P</em> &lt; 0.05, ‡<em>P</em> &lt; 0.01). The respective level of significance is displayed as a heatmap on the right panel. Genes that are also mutated in murine SCLC tumours are denoted with a § symbol. Mutated cancer census genes of therapeutic relevance are denoted with a + symbol. <strong>b</strong>, Somatic copy number alterations determined for 142 human SCLC tumours by single nucleotide polymorphism (SNP) arrays. Significant amplifications (red) and deletions (blue) were determined for the chromosomal regions and are plotted as <em>q</em>-values (significance &lt; 0.05).</p></figcaption></figure><p id="P8">In order to assess the amount of genetic heterogeneity of SCLC, we developed a subclonality score, which can be interpreted as the probability that an arbitrary point mutation in a randomly selected cancer cell is subclonal throughout the entire tumour (Methods). A reliable reconstruction of the subclonal architecture was possible in 55 of the cases (<a href="#F8" class="usa-link">Extended Data Fig. 3b</a>). A comparison to lung adenocarcinoma<sup><a href="#R11" class="usa-link" aria-describedby="R11" aria-expanded="false">11</a></sup> indicated a threefold lower subclonal diversity in SCLC (<em>P</em> = 0.00023, <a href="#F8" class="usa-link">Extended Data Fig. 3b</a>), pointing to pronounced differences in the evolution of SCLC and lung adenocarcinoma<sup><a href="#R12" class="usa-link" aria-describedby="R12" aria-expanded="false">12</a>,<a href="#R13" class="usa-link" aria-describedby="R13" aria-expanded="false">13</a></sup>. In contrast to adenocarcinomas, the level of heterogeneity in SCLC did not correlate with clinical stage (<a href="#F7" class="usa-link">Extended Data Fig. 2b</a>).</p>
<p id="P9">We applied several analytical filters in order to identify mutations with a probable relevance in SCLC biology in the context of the high load of background mutations<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a></sup> (<a href="#F6" class="usa-link">Extended Data Fig. 1</a>, <a href="#SD1" class="usa-link">Supplementary Table 6</a> and Methods). They include (I) analyses of significance determined by a comparison of observed and expected mutation rates followed by a correction for expressed genes, (II) a survey of regional clustering of mutations that may indicate mutational targeting of functionally enriched areas in tumour suppressors or proto-oncogenes, (III) determination of genes that are enriched for likely damaging mutations, (IV) a comparison with genes whose biological relevance has been established in SCLC mouse models, and (V) a listing of genes with a likely therapeutic relevance or that are otherwise frequently affected by genetic alterations in human cancers (that is, genes in the Cancer Gene Census<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a></sup> and COSMIC<sup><a href="#R15" class="usa-link" aria-describedby="R15" aria-expanded="false">15</a></sup> database).</p>
<p id="P10">Among the significantly mutated genes (I), (<em>q</em>-values &lt; 0.05, Methods) were <em>TP53</em> and <em>RB1</em> (refs <a href="#R4" class="usa-link" aria-describedby="R4" aria-expanded="false">4</a>–<a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a>), <em>KIAA1211</em> and <em>COL22A1</em>, as well as <em>RGS7</em> and <em>FPR1</em>, both of which are involved in G-protein-coupled receptor signalling (<a href="#F1" class="usa-link">Fig. 1a</a>).</p>
<p id="P11">Locally clustered mutations (II) are indicative of functional selection (<em>P</em> &lt; 0.05, <a href="#SD1" class="usa-link">Supplementary Table 6</a>, Methods)<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup>. Of all genes, <a href="#F1" class="usa-link">Fig. 1a</a> lists those alterations that occurred in more than 8% of the samples, were otherwise affected by recurrent genomic rearrangements (<a href="#SD1" class="usa-link">Supplementary Table 4</a>), or were mutated in <em>Trp53</em><sup>−/−</sup>, <em>Rb1</em><sup>−/−</sup> or <em>Trp53</em><sup>−/−</sup>, <em>Rb1</em><sup>−/−</sup>, <em>Rbl2</em><sup>−/−</sup> SCLC tumours arising in mice<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a>,<a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup> (<a href="#SD1" class="usa-link">Supplementary Table 5</a>). Confirming previous results and our analytical strategy, the histone acetyltransferase genes <em>CREBBP</em> and <em>EP300</em> exhibited significantly clustered mutations and recurrent inactivating translocations (<a href="#F1" class="usa-link">Fig. 1a</a> and <a href="#F8" class="usa-link">Extended Data Fig. 3c</a>)<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup>. Furthermore, significant mutation clustering occurred in genes with functional roles in the centrosome (<em>ASPM, ALMS1</em> and <em>PDE4DIP</em>), in the RNA-regulating gene <em>XRN1</em> and the tetraspanin gene <em>PTGFRN</em>; the latter was also mutated in murine SCLC (<a href="#F8" class="usa-link">Extended Data Fig. 3c</a>). The <em>TP53</em> homologue <em>TP73</em>, which was also affected by recurrent somatic rearrangements (<a href="#F1" class="usa-link">Fig. 1a</a>), also showed clustered mutations.</p>
<p id="P12">In the group of significantly damaged genes (III) we also found <em>TP53, RB1, CREBBP</em> and <em>COL22A1</em>, further highlighting their likely biological relevance in SCLC. Additional inactivating mutations occurred in <em>FMN2</em> and <em>NOTCH1</em> (<em>P</em> &lt; 0.01). <em>NOTCH</em> family genes were recurrently mutated with a pattern of frequent inactivation. <em>Notch3</em> was also mutated in a <em>Trp53</em><sup>−/−</sup>,<em>Rb1</em><sup>−/−</sup>,<em>Rbl2</em><sup>−/−</sup> mouse tumour (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#SD1" class="usa-link">Supplementary Table 5</a> and Methods).</p>
<p id="P13">Of the genes with an established role in murine SCLC (IV), we confirmed <em>PTEN</em><sup><a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a>,<a href="#R17" class="usa-link" aria-describedby="R17" aria-expanded="false">17</a></sup>. <em>RBL1</em> and <em>RBL2</em>, which are closely related to <em>RB1</em> (ref. <a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a>), similarly exhibited inactivating translocations and mutations (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#F8" class="usa-link">Extended Data Fig. 3d</a> and <a href="#SD1" class="usa-link">Supplementary Table 4</a>). Mice with inactivation of <em>Trp53, Rb1</em> and <em>Rbl2</em> develop SCLC with shorter latency than mice lacking <em>Trp53</em> and <em>Rb1</em> alone<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>, thus validating <em>RBL2</em> as another accessory tumour suppressor in SCLC.</p>
<p id="P14">Given the lack of therapeutic options in SCLC, we sought mutations that are known oncogenic drivers in other cancers and sometimes associated with response to targeted drugs (V)<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a>,<a href="#R15" class="usa-link" aria-describedby="R15" aria-expanded="false">15</a></sup>. Of these, we found mutations in four tumours with a potential therapeutic implication, including mutations in <em>BRAF</em><sup><a href="#R18" class="usa-link" aria-describedby="R18" aria-expanded="false">18</a></sup>, <em>KIT</em><sup><a href="#R19" class="usa-link" aria-describedby="R19" aria-expanded="false">19</a>,<a href="#R20" class="usa-link" aria-describedby="R20" aria-expanded="false">20</a></sup> and <em>PIK3CA</em><sup><a href="#R21" class="usa-link" aria-describedby="R21" aria-expanded="false">21</a></sup> (<a href="#F8" class="usa-link">Extended Data Fig. 3e</a>). Thus, genotyping of SCLC patients may reveal individual patients who might have a possible benefit from targeted therapeutic intervention.</p>
<p id="P15">Across these five categories, mutations in <em>CREBBP, EP300, TP73, RBL1, RBL2</em> and <em>NOTCH</em> family genes were largely mutually exclusive (<a href="#F1" class="usa-link">Fig. 1a</a>), suggesting that they may exert similar pro-tumorigenic functions in the development of SCLC. We did not observe significant correlations of global mutational signatures (for example, predominance of C:G&gt;A:T transversions) with the mutational status of these genes (<a href="#F7" class="usa-link">Extended Data Fig. 2b</a>). Furthermore, mutations in these genes were not significantly associated with the total number of mutations, overall survival or other clinical parameters (<a href="#F9" class="usa-link">Extended Data Fig. 4</a>). The mutation status of 22 of the most frequently mutated genes was confirmed in an independent data set (Methods and <a href="#SD1" class="usa-link">Supplementary Table 7</a>).</p>
<p id="P16">By analysing somatic copy number alterations, we confirmed previously known genomic losses within 3p pointing to focal events on 3p14.3–3p14.2 (harbouring <em>FHIT</em><sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup>) and 3p12.3–3p12.2 (harbouring <em>ROBO1</em> (ref. <a href="#R22" class="usa-link" aria-describedby="R22" aria-expanded="false">22</a>)) (<a href="#F1" class="usa-link">Fig. 1b</a>, <a href="#F10" class="usa-link">Extended Data Fig. 5a</a> and <a href="#SD1" class="usa-link">Supplementary Table 8</a>)<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a>,<a href="#R22" class="usa-link" aria-describedby="R22" aria-expanded="false">22</a>,<a href="#R23" class="usa-link" aria-describedby="R23" aria-expanded="false">23</a></sup>. <em>FHIT</em> expression was also reduced in cases with focal deletions (<a href="#F10" class="usa-link">Extended Data Fig. 5b</a>). In addition to homozygous losses in the <em>CDKN2A</em> locus (<a href="#F10" class="usa-link">Extended Data Fig. 5c</a>), amplification of the <em>MYC</em> family genes<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup>, <em>MYCL1, MYCN</em> and <em>MYC</em>, as well as of the tyrosine kinase gene, <em>FGFR1</em> (refs <a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>, <a href="#R24" class="usa-link" aria-describedby="R24" aria-expanded="false">24</a>), and <em>IRS2</em> were recurrent genomic events (<a href="#F1" class="usa-link">Fig. 1b</a>). Focal <em>IRS2</em> amplifications occurred in 2% of the cases (<a href="#F10" class="usa-link">Extended Data Fig. 5d, e</a>).</p></section><section id="S3"><h2 class="pmc_sec_title" data-anchor-id="S3"><a id="S3-anchor" data-anchor-id="S3" class="usa-anchor"></a>Universal inactivation of <em>TP53</em> and <em>RB1</em>
</h2>
<p id="P17">Inactivating mutations in <em>TP53</em> and <em>RB1</em> have been shown to affect up to 90% and up to 65% of SCLC, respectively<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>–<a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a></sup>. By contrast, our whole-genome sequencing analyses revealed that both genes were altered in all but two cases that exhibited signs of chromothripsis<sup><a href="#R25" class="usa-link" aria-describedby="R25" aria-expanded="false">25</a></sup> (<a href="#F1" class="usa-link">Figs 1</a> and <a href="#F2" class="usa-link">2</a>). <em>TP53</em> and <em>RB1</em> alterations were mostly inactivating (<a href="#SD1" class="usa-link">Supplementary Table 9</a> and <a href="#F11" class="usa-link">Extended Data Fig. 6a</a>). Missense mutations in <em>TP53</em> affected the functionally critical DNA binding domain, while <em>RB1</em> was frequently altered by complex genomic translocations. Many mutations in <em>RB1</em> occurred at exon–intron junctions, which caused protein-damaging splice events as confirmed by transcriptome sequencing (<a href="#F11" class="usa-link">Extended Data Fig. 6b–e</a> and <a href="#SD1" class="usa-link">Supplementary Tables 10 and 11</a>). In the 108 tumours without chromothripsis, <em>TP53</em> and <em>RB1</em> had bi-allelic losses in 100% and 93% of the cases, respectively. Inactivating events included mutations, translocations, homozygous deletions, hemizygous losses, copy-neutral losses of heterozygosity (LOH) and LOH at higher ploidy (<a href="#F2" class="usa-link">Fig. 2a</a>, <a href="#F11" class="usa-link">Extended Data Fig. 6</a> and <a href="#SD1" class="usa-link">Supplementary Table 9</a>). Loss of <em>CDKN2A</em> occurred in cases with both bi-allelic inactivation of <em>TP53</em> and <em>RB1</em> and hemizygous loss of <em>RB1</em> (<a href="#F2" class="usa-link">Fig. 2a</a>). Although emerging data supports a continuum model of inactivation of tumour suppressors across multiple cancers, <em>TP53</em> and <em>RB1</em> follow the classical discrete ‘two-hit paradigm’ pattern of Knudson-type tumour suppressors in SCLC<sup><a href="#R26" class="usa-link" aria-describedby="R26" aria-expanded="false">26</a>,<a href="#R27" class="usa-link" aria-describedby="R27" aria-expanded="false">27</a></sup>.</p>
<figure class="fig xbox font-sm" id="F2"><h3 class="obj_head">Figure 2. Universal bi-allelic inactivation of <em>TP53</em> and <em>RB1</em> in human SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/d52f2d5d7ae6/nihms782739f2.jpg" loading="lazy" height="1050" width="791" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P94"><strong>a</strong>, Alterations of <em>TP53</em> and <em>RB1</em> were determined based on whole-genome sequencing data of 108 SCLC cases. Samples are plotted from left to right. Alleles A and B are represented for each case and colour-coded according to the somatic alteration. The integral copy number (iCN) state of each allele is plotted; hemizygous losses are annotated as loss of heterozygosity (LOH), copy-neutral LOH or LOH at higher ploidy. Samples retaining allele A and B show alterations on both alleles (bi-allelic alterations). <strong>b</strong>, Circos plot of case <a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a> showing intra- and interchromosomal translocations between chromosome 3 and 11. The copy number state of the respective chromosomal regions (iCN) is plotted as a heatmap. The genomic context of <em>CCND1</em> (on chromosome 11) is highlighted. <strong>c</strong>, Significantly differentially expressed genes encoded on chromosome 11 are analysed in both chromothripsis cases in comparison to all other tumours. Positive and negative <em>z</em>-scores show upregulation and downregulation of genes, respectively (<em>P</em> &lt; 0.05; *<em>q</em>-value &lt; 0.05). <strong>d</strong>, Distribution of <em>CCND1</em> expression over 81 SCLC samples. Chromothripsis cases are highlighted in red. <strong>e</strong>, Haematoxylin and eosin (H&amp;E) and immunohistochemistry staining for cyclin D1 and Rb1 for sample <a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a>. Original magnification, ×400.</p></figcaption></figure><p id="P18">The two tumours affected by chromothripsis displayed a similar pattern of massive genomic rearrangements between chromosomes 3 and 11 (<a href="#F2" class="usa-link">Fig. 2b</a> and <a href="#F12" class="usa-link">Extended Data Fig. 7a</a>), but lacked shared fusion transcripts in the transcriptome sequencing data, suggesting that a particular fusion is not a common target (<a href="#F12" class="usa-link">Extended Data Fig. 7b</a> and <a href="#SD1" class="usa-link">Supplementary Table 12</a>). Of the genes on chromosomes 3 and 11, <em>CCND1</em> (encoding cyclin D1) was retained (<a href="#F2" class="usa-link">Fig. 2b</a> and <a href="#F12" class="usa-link">Extended Data Fig. 7a</a>) resulting in significant <em>CCND1</em> overexpression in both tumours, but not in the other SCLC specimens (<a href="#F2" class="usa-link">Fig. 2c, d</a> and <a href="#SD1" class="usa-link">Supplementary Table 13</a>). Immunohistochemistry confirmed high expression of cyclin D1 and a lack of nuclear Rb1 (<a href="#F2" class="usa-link">Fig. 2e</a>, <a href="#F12" class="usa-link">Extended Data Fig. 7c</a>). There were fewer proliferating Ki67-positive cells in these two cases. As cyclin D1 negatively regulates Rb family proteins<sup><a href="#R28" class="usa-link" aria-describedby="R28" aria-expanded="false">28</a></sup>, these findings suggest that chromothripsis in cases with wild-type <em>RB1</em> may compensate for genomic loss of <em>RB1</em>.</p>
<p id="P19">Together, our findings provide evidence for the notion that complete genomic loss of both <em>TP53</em> and <em>RB1</em> function is obligatory in the pathogenesis of SCLC.</p></section><section id="S4"><h2 class="pmc_sec_title" data-anchor-id="S4"><a id="S4-anchor" data-anchor-id="S4" class="usa-anchor"></a>Oncogenic genomic events affecting <em>TP73</em>
</h2>
<p id="P20">We analysed the genome sequencing data for the presence of clustered chromosomal breakpoints that may indicate a common biological target (<a href="#SD1" class="usa-link">Supplementary Table 14</a>)<sup><a href="#R29" class="usa-link" aria-describedby="R29" aria-expanded="false">29</a></sup> and found 5 major clusters affecting <em>RB1</em>, as well as regions on chromosomes 1, 3 (3q26), 6 (affecting <em>CDKAL1</em>) and 22 (<a href="#F3" class="usa-link">Fig. 3a</a>). Breakpoints in chromosome 22 caused inactivating translocations of <em>TTC28</em> (<a href="#F13" class="usa-link">Extended Data Fig. 8a</a>)<sup><a href="#R30" class="usa-link" aria-describedby="R30" aria-expanded="false">30</a></sup>. Breakpoints also clustered downstream of the L1HS retrotransposon in SCLC, further supporting a role for this element in cancer<sup><a href="#R31" class="usa-link" aria-describedby="R31" aria-expanded="false">31</a>,<a href="#R32" class="usa-link" aria-describedby="R32" aria-expanded="false">32</a></sup>, (<a href="#F13" class="usa-link">Extended Data Fig. 8b</a>). Breakpoints on chromosomes 3, 6 and 22 did not result in changes of expression of the affected genes (<a href="#SD1" class="usa-link">Supplementary Table 10</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h3 class="obj_head">Figure 3. Recurrent rearrangements generating oncogenic variants of <em>TP73</em>.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/91a7822560c5/nihms782739f3.jpg" loading="lazy" height="1050" width="686" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P95"><strong>a</strong>, Genomic breakpoints identified by whole-genome sequencing were mapped to their chromosomal locations. Recurrent breakpoints (<em>n</em> &gt; 6 samples) are highlighted in colours. <strong>b</strong>, Schematic representation of the <em>TP73</em> locus (hg19) illustrating intragenic translocations. Coding and non-coding regions of the annotated exons are shown as black and white boxes, respectively. <strong>c</strong>, Schematic representation of exons encoding p73, p73Δex2, p73Δex2/3 and p73Δex10. <strong>d</strong>, Exon skipping events were assessed in the transcriptome data of samples with genomic translocations resulting in p73Δex2, p73Δex2/3 and p73Δex10 transcript variants. S02139 served as a reference sample without <em>TP73</em> alterations. The expression of uncommon exon combinations is highlighted in red.</p></figcaption></figure><p id="P21">By contrast, genomic breakpoints affecting chromosome 1 clustered precisely in the <em>TP73</em> locus in 7% of the cases (<em>n</em> = 8). To our surprise, several breakpoints were recurrently located in introns 1, 2 and 3 of <em>TP73</em>. In two cases, breakpoints led to complex intrachromosomal rearrangements (<a href="#F13" class="usa-link">Extended Data Fig. 8c</a> and <a href="#SD1" class="usa-link">Supplementary Table 4</a>), while the majority of breaks caused intragenic fusions and, thus, exclusion of either exon 2, or exons 2 and 3, which were all somatic (<a href="#F3" class="usa-link">Fig. 3b</a> and <a href="#F13" class="usa-link">Extended Data Fig. 8d</a>). Some rearrangements were copy-neutral events, while others occurred on the background of copy number gains (<a href="#F13" class="usa-link">Extended Data Fig. 8e</a>). One tumour sample revealed genomic exclusion of exon 10 (<a href="#F3" class="usa-link">Fig. 3b</a>). Analyses of transcriptome sequencing data confirmed that these rearrangements created the N-terminally truncated transcript variants p73Δex2 and p73Δex2/3, as well as p73Δex10 (ref. <a href="#R33" class="usa-link" aria-describedby="R33" aria-expanded="false">33</a>) (<a href="#F3" class="usa-link">Fig. 3c, d</a> and <a href="#SD1" class="usa-link">Supplementary Table 11</a>). Genomic validation and comparative profiling of transcript variants confirmed that p73Δex2/3 were not naturally occurring splice variants in SCLC and were only found in cases with genomic rearrangements (<a href="#F3" class="usa-link">Fig. 3d</a> and <a href="#SD1" class="usa-link">Supplementary Table 11</a>). Some tumours expressed p73Δex2, in which we failed to identify genomic rearrangements (<a href="#SD1" class="usa-link">Supplementary Table 11</a>).</p>
<p id="P22">p73Δex2 and p73Δex2/3 lack a fully competent transactivation domain and are known tumour-derived variants of <em>TP73</em> (ref. <a href="#R33" class="usa-link" aria-describedby="R33" aria-expanded="false">33</a>). p73 with N-terminal truncations has dominant negative functions on wild-type p73 and p53, and is a confirmed oncogene <em>in vivo</em><sup><a href="#R34" class="usa-link" aria-describedby="R34" aria-expanded="false">34</a>,<a href="#R35" class="usa-link" aria-describedby="R35" aria-expanded="false">35</a></sup>. p73Δex10 results in an early stop codon; C-terminal truncations can similarly exert dominant-negative effects on wild-type p73 (ref. <a href="#R33" class="usa-link" aria-describedby="R33" aria-expanded="false">33</a>).</p>
<p id="P23">Altogether, <em>TP73</em> was somatically altered by mutations and genomic rearrangements in 13% of the cases (<a href="#F1" class="usa-link">Fig. 1</a> and <a href="#F13" class="usa-link">Extended Data Fig. 8c–e</a>). To our knowledge, this is the first study describing p73Δex2/3 variants to emerge as a consequence of precise genomic rearrangements.</p></section><section id="S5"><h2 class="pmc_sec_title" data-anchor-id="S5"><a id="S5-anchor" data-anchor-id="S5" class="usa-anchor"></a>Tumour suppressive roles of Notch in SCLC</h2>
<p id="P24">In an unsupervised hierarchical clustering analysis of transcriptome sequencing data (Methods), we observed two major clusters of SCLC tumours (<a href="#F4" class="usa-link">Fig. 4a</a> and <a href="#F14" class="usa-link">Extended Data Fig. 9a</a>). The majority (77%, <em>n</em> = 53/69) of tumours exhibited high expression of the neuroendocrine markers <em>CHGA</em> (chromogranin A)<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a></sup> and <em>GRP</em> (gastrin releasing peptide)<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup>, had high levels of <em>DLK1</em> (ref. <a href="#R36" class="usa-link" aria-describedby="R36" aria-expanded="false">36</a>), a non-canonical inhibitor of Notch signalling<sup><a href="#R36" class="usa-link" aria-describedby="R36" aria-expanded="false">36</a></sup>, and <em>ASCL1</em>, a lineage oncogene of neuroendocrine cells whose expression is inhibited by active Notch signalling (<a href="#F14" class="usa-link">Extended Data Fig. 9b</a>)<sup><a href="#R37" class="usa-link" aria-describedby="R37" aria-expanded="false">37</a>,<a href="#R38" class="usa-link" aria-describedby="R38" aria-expanded="false">38</a></sup>. The remaining cases (23%, <em>n</em> = 16/69) also expressed <em>SYP</em> (synaptophysin) or <em>NCAM1</em> (CD56), thus confirming that all tumours were of the typical SCLC subtype<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a></sup> (<a href="#F14" class="usa-link">Extended Data Fig. 9c</a>). Furthermore, no significant difference in the distribution of the major known SCLC mutations (for example, <em>TP53, RB1</em> or <em>CREBBP</em>) existed between the two transcriptional subtypes (<a href="#F14" class="usa-link">Extended Data Fig. 9a</a>). Thus, although all SCLC tumours shared the most frequent mutations as well as key neuroendocrine markers, the majority had a gene expression pattern suggestive of low Notch pathway activity (high levels of <em>ASCL1</em> and <em>DLK1</em>).</p>
<figure class="fig xbox font-sm" id="F4"><h3 class="obj_head">Figure 4. Notch is a tumour suppressor and a key regulator of neuroendocrine differentiation in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/f45d8a0af2e7/nihms782739f4.jpg" loading="lazy" height="508" width="1050" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P96"><strong>a</strong>, Unsupervised expression analysis of human SCLC tumours. Tumour samples are arranged in columns and grouped by the expression of differentially expressed genes (rows). Expression values are represented as a heatmap; yellow and blue indicate high and low expression, respectively. <strong>b</strong>, Schematic representation of <em>NOTCH1</em> and <em>NOTCH2</em>. Somatic mutations are mapped to the respective protein domains. Damaging and missense mutations are highlighted in red and black, respectively. <strong>c</strong>, Representative H&amp;E images of lungs from <em>Trp53;Rb1;Rbl2</em> triple-knockout (TKO) or TKO;N2ICD (Notch2) mice collected 3 months after Ad-Cre instillation. Scale bar, 1 mm. Tumours were quantified for each genotype (<em>n</em> = 8). Statistical significance was determined by two-tailed unpaired Student’s <em>t</em>-test. <strong>d</strong>, Survival analysis of TKO (<em>n</em> = 7, median survival = 210 days) and TKO;N2ICD (<em>n</em> = 8, median survival = 274 days) mice. Statistical significance was determined by log-rank test. <strong>e</strong>, Cell viability assay of the murine SCLC cell line KP1 transfected with a N1ICD (Notch1) expression plasmid or empty vector control (Ctrl) (3 independent biological replicas with 3 technical replicas each). Fold growth is normalized to day 0; representative images were taken on day 8. Scale bar, 50 µm. Statistical significance was determined by two-tailed paired Student’s <em>t</em>-test. <strong>f</strong>, Mouse SCLC cells were transfected with control or N1ICD and analysed 48 h after transfection by gene expression microarrays. The heatmap describes differentially expressed genes in control or N1ICD-transfected cells (<em>n</em> = 3, each); red and green indicate high and low expression, respectively. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001. Data are represented as mean ± s.d.</p></figcaption></figure><p id="P25">Mutations affected <em>NOTCH</em> family genes in both human and murine SCLC (<a href="#F1" class="usa-link">Fig. 1a</a> and <a href="#SD1" class="usa-link">Supplementary Table 5</a>). The mutations did not cluster significantly in any individual domains, but frequent damaging mutations occurred in the extracellular domain (<a href="#F4" class="usa-link">Fig. 4b</a> and <a href="#F15" class="usa-link">Extended Data Fig. 10a</a>), suggesting that <em>NOTCH</em> may be a tumour suppressor in SCLC. Overall, the <em>NOTCH</em> family was affected by genomic alterations in 25% of human SCLC (<a href="#F1" class="usa-link">Fig. 1</a>).</p>
<p id="P26">Based on these observations and emerging evidence that activation of Notch signalling may inhibit the expansion of neuroendocrine tumour cells<sup><a href="#R39" class="usa-link" aria-describedby="R39" aria-expanded="false">39</a>,<a href="#R40" class="usa-link" aria-describedby="R40" aria-expanded="false">40</a></sup>, we examined the consequences of Notch pathway activation in <em>Trp53;Rb1;Rbl2</em> conditional triple-knockout (TKO) mice<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>.We crossed <em>Rosa26<sup>Lox-stop-Lox-Notch2ICD</sup></em> (<em>LSL-N2ICD</em>) mice that conditionally express an activated form of Notch2 (Notch2 intracellular domain, N2ICD) to TKO mice and found a significant reduction in the number of tumours that arose in the presence of N2ICD (<em>P</em> &lt; 0.001; <a href="#F4" class="usa-link">Fig. 4c</a>). Similar results were obtained upon activation of Notch1, reflecting a general inhibition of SCLC initiation by active Notch signalling (<a href="#F15" class="usa-link">Extended Data Fig. 10b</a>). The recombination efficiency of an innocuous inducible reporter allele (<em>Rosa26<sup>mT/mG</sup></em>) by Cre was much greater than that of the <em>N2ICD</em> allele, providing further support for a strong negative selection against active Notch signalling during SCLC development (<a href="#F15" class="usa-link">Extended Data Fig. 10c–e</a>). Importantly, the inhibitory effects of Notch observed in the early stages of tumorigenesis correlated with a prolongation of survival of the mutant mice expressing N2ICD (<a href="#F4" class="usa-link">Fig. 4d</a>). Similarly, ectopic expression of N1ICD in both mouse and human SCLC cell lines significantly inhibited their growth (<a href="#F4" class="usa-link">Fig. 4e</a> and <a href="#F15" class="usa-link">Extended Data Fig. 10f</a>).</p>
<p id="P27">SCLC tumours in TKO mice showed typical patterns of neuroendocrine differentiation with high expression of synaptophysin and Ascl1. Consistent with the notion that Notch regulates neuroendocrine differentiation in SCLC, overexpression of N2ICD resulted in the upregulation of Hes1 and abrogated expression of neuroendocrine markers (<a href="#F15" class="usa-link">Extended Data Fig. 10g</a>). Similarly, N1ICD induced upregulation of Notch targets (for example, <em>Hes1, Hey1, Hey2</em>) in murine SCLC cells (<a href="#F4" class="usa-link">Fig. 4f</a>, <a href="#F15" class="usa-link">Extended Data Fig. 10h</a> and <a href="#SD1" class="usa-link">Supplementary Table 15</a>), as well as gene expression signatures consistent with cell cycle inhibition (<a href="#F15" class="usa-link">Extended Data Fig. 10i</a>). Ectopic expression of N1ICD inhibited cell cycle progression in murine and human SCLC cell lines (<a href="#F15" class="usa-link">Extended Data Fig. 10j, k</a>). This cell cycle inhibition is reminiscent of what has been seen in other contexts where Notch activation acts as a tumour suppressor<sup><a href="#R41" class="usa-link" aria-describedby="R41" aria-expanded="false">41</a>,<a href="#R42" class="usa-link" aria-describedby="R42" aria-expanded="false">42</a></sup>.</p>
<p id="P28">Altogether, our analyses involving genome and transcriptome sequencing of human and murine SCLC tumours, as well as studies in genetically manipulated mice, identify and validate Notch as a tumour suppressor and master regulator of neuroendocrine differentiation in SCLC.</p></section><section id="S6"><h2 class="pmc_sec_title" data-anchor-id="S6"><a id="S6-anchor" data-anchor-id="S6" class="usa-anchor"></a>Discussion</h2>
<p id="P29">Here we provide a comprehensive analysis of somatic genome alterations in SCLC, identifying many novel candidate genes, some of which may have therapeutic implications. Such alterations with immediate therapeutic consequences are rare but present in SCLC (for example, in <em>BRAF</em> or <em>KIT</em>), suggesting that individual patients may benefit from genotyping and subsequent targeted kinase inhibitor therapy. We further discovered recurrent expression of p73Δex2/3 in SCLC and established a genetic mechanistic basis for this oncogenic variant. <em>TP73</em>Δex2/3 has recently been demonstrated to function as an oncogene<sup><a href="#R34" class="usa-link" aria-describedby="R34" aria-expanded="false">34</a>,<a href="#R35" class="usa-link" aria-describedby="R35" aria-expanded="false">35</a></sup> and therapeutic options were identified to restrict p73-dependent tumour growth <em>in vivo</em>, including in <em>Trp53</em>-deficient tumours<sup><a href="#R35" class="usa-link" aria-describedby="R35" aria-expanded="false">35</a></sup>. Given the frequent occurrence of genomic <em>TP73</em> alterations in SCLC, such approaches may potentially be promising in SCLC tumours. Our results furthermore provide proof for universal bi-allelic inactivation of <em>TP53</em> and <em>RB1</em>, thereby establishing these two genes as obligatory tumour suppressors in SCLC.</p>
<p id="P30">Our genomic analyses also identified <em>NOTCH</em> family genes as tumour suppressors and master regulators of neuroendocrine differentiation in SCLC, and we validated this finding <em>in vivo</em> in a pre-clinical mouse model of this disease. Our observations may thus provide an initial link between Notch and the neuroendocrine phenotype in SCLC. In contrast to the involvement of <em>TP73</em> and <em>NOTCH</em> family genes (<a href="#F5" class="usa-link">Fig. 5</a>), the functional role of most of the other newly discovered genes (for example, <em>KIAA1211, COL22A1, ASPM, PDE4DIP</em> or <em>PTGFRN</em>) is much less clear. Although our analytical filters support their involvement in the tumour pathogenesis, functional experiments will be required to clarify their biological role.</p>
<figure class="fig xbox font-sm" id="F5"><h3 class="obj_head">Figure 5. Signalling pathways recurrently affected in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/263ef7b85353/nihms782739f5.jpg" loading="lazy" height="824" width="1050" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P97">Red and blue boxes denote genes with activating and inactivating alterations, respectively. Deep blue boxes highlight the bi-allelic inactivation of <em>TP53</em> and <em>RB1</em>. Genes found expressed at high levels are shown in red font.</p></figcaption></figure><p id="P31">In summary, we have provided the first, to our knowledge, comprehensive genomic analysis of SCLC, implicating several previously unknown genes and biological processes (<a href="#F5" class="usa-link">Fig. 5</a>) in the pathogenesis of this disease as possible targets for more efficacious targeted therapeutic intervention against this deadly cancer.</p>
<p id="P32"><strong>Online Content</strong> Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.</p></section><section id="S7"><h2 class="pmc_sec_title" data-anchor-id="S7"><a id="S7-anchor" data-anchor-id="S7" class="usa-anchor"></a>METHODS</h2>
<section id="S8"><h3 class="pmc_sec_title">Human lung tumour specimens</h3>
<p id="P33">The institutional review board of the University of Cologne approved this study. We collected and analysed fresh-frozen tumour samples of 152 SCLC patients, which were provided by multiple collaborating institutions as fresh-frozen tissue specimen, frozen sections or as genomic DNA extracted from fresh-frozen material (<a href="#F6" class="usa-link">Extended Data Fig. 1</a>). Human tumour samples were obtained from patients under IRB-approved protocols following written informed consent.</p>
<p id="P34">The fresh-frozen SCLC samples were primary tumours diagnosed as stage I–IV tumours, and snap-frozen after tissue sampling. All tumour samples were pathologically assessed to have a purity of at least 60% and no extensive signs of necrosis. Additionally, these tumour samples were reviewed by at least two independent expert pathologists and the diagnosis of SCLC was histomorphologically confirmed by H&amp;E staining and immunohistochemistry for chromogranin A, synaptophysin, CD56 and Ki67. Matching normal material was provided in the form of EDTA-anticoagulated blood or adjacent non-tumorigenic lung tissue (<a href="#SD1" class="usa-link">Supplementary Table 1</a>). The matched normal tissue was confirmed to be free of tumour contaminants by pathological assessment. Furthermore, tumour and matching normal material were confirmed to be acquired from the same patient by short tandem repeat (STR) analysis conducted at the Institute of Legal Medicine at the University of Cologne (Germany), or confirmed by subsequent SNP 6.0 array and sequencing analyses. Patient material was stored at −80 °C.</p>
<p id="P35">Whole-genome sequencing was performed on 110 SCLC fresh-frozen tumour samples and matched normal material. Additionally, we analysed RNA-seq data of 81 SCLC primary tumours (<a href="#F6" class="usa-link">Extended Data Fig. 1</a> and <a href="#SD1" class="usa-link">Supplementary Table 1</a>), among which 20 cases were previously published<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R43" class="usa-link" aria-describedby="R43" aria-expanded="false">43</a></sup>. Furthermore, we studied the copy-number alterations of a total of 142 fresh-frozen tumour specimen by Affymetrix SNP 6.0, among which 74 cases were described before<sup><a href="#R44" class="usa-link" aria-describedby="R44" aria-expanded="false">44</a></sup>.</p>
<p id="P36">Clinical correlation studies were performed with the study cohort of 110 SCLC patients considering age of diagnosis, gender, tumour stage, surgery, treatment with chemotherapeutics, smoking status, smoking history and overall survival (<a href="#F7" class="usa-link">Extended Data Figs 2</a> and <a href="#F9" class="usa-link">4</a> and <a href="#SD1" class="usa-link">Supplementary Table 1</a>). The median follow-up time for this cohort of 110 SCLC patients was 69 months, and 31% of the patients were alive at the time of last follow-up (<a href="#F7" class="usa-link">Extended Data Fig. 2a</a> and <a href="#SD1" class="usa-link">Supplementary Table 1</a>). Smoking status was available for 88% (<em>n</em> = 97) of the patients; 63% (<em>n</em> = 69) reported a smoking history amounting to a median of 45 pack-years. Patients with a known smoking history were further subcategorized to heavy smokers (&gt;30 pack-years), average smokers (10–30 pack-years) and light/never smokers (&lt;10 pack-years).</p>
<p id="P37">Primary findings on somatic mutations were further studied in a second independent cohort consisting of 112 SCLC cases. This validation cohort refers to the exome sequencing data of 28 fresh-frozen SCLC primary tumours and 9 SCLC cell lines<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup> which were re-analysed in this present study (<a href="#SD1" class="usa-link">Supplementary Table 7</a>). Additionally, we performed targeted sequencing on 8 fresh-frozen and 67 formalin fixed paraffin embedded (FFPE) samples from SCLC patients (<a href="#SD1" class="usa-link">Supplementary Table 1</a>).</p></section><section id="S9"><h3 class="pmc_sec_title">Mouse SCLC models and tumour samples</h3>
<p id="P38">Mice were maintained according to practices prescribed by the NIH (Bethesda, MD) at Stanford’s Research Animal Facility, accredited by the Association for the Assessment and Accreditation of Laboratory Animal Care (AAALAC). The <em>Trp53;Rb1</em> double-knockout (DKO) and the <em>Trp53;Rb1;Rbl2</em> triple-knockout (TKO) mouse models for SCLC have been previously described<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a>,<a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>. Mice were bred onto a mixed genetic background composed of C57BL/6, 129/SvJ and 129/SvOla. SCLC tumours were induced in 8-week-old mice by intratracheal instillation with 4 × 10<sup>7</sup> plaque-forming units (p.f.u.) of adenovirus expressing the Cre recombinase (Ad-Cre, Baylor College of Medicine, Houston, TX).</p>
<p id="P39">Whole-genome and whole-exome sequencing was performed on 8 murine SCLC tumours isolated from DKO and TKO mice. Primary tumours and metastases were dissected, snap-frozen, and stored at −80 °C. The material was pathologically confirmed to have a tumour content of at least 90%. The respective tail tissue was similarly processed and served as a normal reference for 6 tumour samples (<a href="#SD1" class="usa-link">Supplementary Table 5</a>). Average mutation rates were calculated for cases with tumour-normal pairs (<em>n</em> = 6).</p>
<p id="P40">SCLC tumours expressing the activated intracellular domain (ICD) of Notch1 (Notch1 ICD, N1ICD) and Notch2 (Notch2 ICD, N2ICD) were analysed in mouse models. <em>Rosa26<sup>Lox-stop-Lox-Notch1ICD</sup></em> (<em>LSL-N1ICD</em>) or <em>Rosa26<sup>Lox-stop-Lox-Notch2ICD</sup></em> (<em>LSL-N2ICD</em>) mice were obtained from Spyros Artavanis-Tsakonas and Exelixis. These mice are similar to recently published <em>Rosa26<sup>+/LSL-Notch3ICD</sup></em> mice<sup><a href="#R45" class="usa-link" aria-describedby="R45" aria-expanded="false">45</a></sup>. <em>Rosa26<sup>+/LSL-N1ICD</sup></em> or <em>Rosa26<sup>+/LSL-N2ICD</sup></em> mice were crossed with TKO mice. TKO or TKO;<em>Rosa26<sup>+/LSL-NICD</sup></em> mice were infected with Ad-Cre at week 8 and their survival was monitored. The sample size was chosen based on our experience with these mouse models of cancer (a minimum of 3–5 mice usually ensures statistical significance if the phenotypes are robust). We used both males and females in these experiments, littermates served as controls. Tumour initiation was studied three months after Ad-Cre instillation. The lungs were fixed and tumour burden was quantified using ImageJ software. To control for the efficiency of deletion, we also crossed TKO mice to <em>Rosa26<sup>mT/mG</sup></em> reporter mice<sup><a href="#R46" class="usa-link" aria-describedby="R46" aria-expanded="false">46</a></sup>. For all tumour quantifications, the investigator was blinded to the genotypes when the H&amp;E pictures were taken, and during the quantification of tumour number and area. No samples or animals were excluded from the analyses, and no randomization was performed.</p></section><section id="S10"><h3 class="pmc_sec_title">DNA and RNA extractions</h3>
<p id="P41">Nucleic acids were extracted from fresh-frozen tissue specimen which were processed to 15–30 sections each of 20 µm thickness at a cryostat maintaining a temperature of −20 °C (Leica). In the case of FFPE samples, 6–10 sections of 10 µm thickness were prepared.</p>
<p id="P42">DNA was extracted from fresh-frozen tissues, EDTA blood, or FFPE samples using the Gentra Puregene DNA extraction kit (Qiagen) following the protocol of the manufacturer. DNA isolates were hydrated in TE-buffer and confirmed to be of high molecular weight (&gt;10 kb) by agarose gel electrophoresis. Genomic DNA from fresh-frozen samples with evident signs of degradation were excluded from further sequencing studies.</p>
<p id="P43">For RNA extractions, tissue sections were first lysed and homogenized with the Tissue Lyzer (Qiagen). Subsequent RNA extractions were performed with the Qiagen RNAeasy Mini Kit according to the instructions of the manufacturer. The RNA quality was assessed at the Bioanalyzer 2100 DNA Chip 7500 (Agilent Technologies) and samples with a RNA integrity number (RIN) of over 7 were further analysed by RNA-seq.</p></section><section id="S11"><h3 class="pmc_sec_title">Next-generation sequencing</h3>
<p id="P44">All sequencing reactions were performed on an Illumina HiSeq 2000 instrument (Illumina, San Diego, CA, USA).</p>
<section id="S12"><h4 class="pmc_sec_title">Whole-genome sequencing</h4>
<p id="P45">Whole-genome sequencing was performed with DNA extracted from fresh-frozen tumour and normal material. Short insert DNA libraries were prepared with the TruSeq DNA PCRfree sample preparation kit (Illumina) for paired-end sequencing at a minimum read length of 2 × 100 bp. Human DNA libraries were sequenced with the aim to obtain a coverage of minimum 30× for both tumour and matched normal. Murine DNA libraries of tumour and matched normal were both sequenced to a coverage of 25×.</p></section><section id="S13"><h4 class="pmc_sec_title">Whole-exome sequencing</h4>
<p id="P46">Whole-exome sequencing was performed on fresh-frozen tissue specimen from mice. The enrichment for the exome was performed with the SureSelectXT Mouse All Exon kit (Agilent) following the protocol of the manufacturer. The exon-enriched libraries were subjected to paired-end sequencing with a read-length of 2 × 100 bp. Both tumour and normal material was sequenced to a minimum coverage of 60×</p></section><section id="S14"><h4 class="pmc_sec_title">RNA-sequencing</h4>
<p id="P47">RNA-sequencing (RNA-seq) was performed with RNA extracted from fresh-frozen human tumour tissue samples. cDNA libraries were prepared from poly(A) selected RNA applying the Illumina TruSeq protocol for mRNA. The libraries were then sequenced with a 2 × 100 bp paired-end protocol to a minimum mean coverage of 30× of the annotated transcriptome.</p></section><section id="S15"><h4 class="pmc_sec_title">Targeted enrichment sequencing</h4>
<p id="P48">Targeted enrichment sequencing was performed on human FFPE and fresh-frozen tumour and normal specimen for the purpose of validating genome alterations in an independent cohort. The custom probe design was constructed with SureDesign (Agilent Technologies) enriching for the exons of 22 genes of interest. DNA libraries were prepared with the SureSelect XT reagent kit according to the manufacturer’s instructions (Agilent Technologies) and sequenced with the aim to obtain a coverage of at least 200×.</p></section></section><section id="S16"><h3 class="pmc_sec_title">Dideoxy sequencing for validation of somatic alterations</h3>
<p id="P49">If available, RNA-seq or exome sequencing was used to validate somatic mutations determined by genome sequencing. Alternatively, dideoxynucleotide chain termination sequencing (Sanger sequencing) was performed to validate mutations, genomic rearrangements, and chimaeric fusion transcripts. Primer pairs were designed to amplify the target region encompassing the somatic alteration. The PCR reactions were performed either with genomic DNA, whole-genome amplified DNA or cDNA. The amplified products were subjected to Sanger sequencing and the respective electropherogram was analysed with Geneious (<a href="http://www.geneious.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneious.com</a>).</p></section><section id="S17"><h3 class="pmc_sec_title">Copy number analysis by Affymetrix SNP 6.0 arrays</h3>
<p id="P50">Human DNA extracted from fresh-frozen tumour specimen was hybridized to Affymetrix Genome-Wide Human SNP array 6.0 following the manufacturer’s instructions. The signal intensities were processed to analyse for chromosomal gene copy number data. Raw copy number signals and segmented copy number data were computed following the procedure described previously<sup><a href="#R24" class="usa-link" aria-describedby="R24" aria-expanded="false">24</a></sup>.</p>
<p id="P51">The raw, unsegmented copy number signals were used to analyse for significant copy number alterations applying the method CGARS<sup><a href="#R47" class="usa-link" aria-describedby="R47" aria-expanded="false">47</a></sup>. Significant amplifications were determined with the upper quantiles 0.25, 0.15, 0.1, and 0.05; deletions were computed in reference to the 0.25 lower quantile. The significance threshold was set at a <em>q</em>-value of 0.05 (<a href="#SD1" class="usa-link">Supplementary Table 8</a>).</p></section><section id="S18"><h3 class="pmc_sec_title">Data processing</h3>
<p id="P52">The raw sequencing reads of human and mouse samples acquired from whole-genome, whole-exome or targeted enrichment sequencing were aligned to the respective human (NCBI37/hg19) or mouse reference genome (NCBI37/mm9). The alignment was performed with the BWA aligner<sup><a href="#R48" class="usa-link" aria-describedby="R48" aria-expanded="false">48</a></sup> (version 0.6.1-r104). Concordant read-pairs were identified as potential PCR duplicates and were subsequently masked in the alignment file. The quality of the sequencing data was determined and is summarized in <a href="#SD1" class="usa-link">Supplementary Table 2</a>.</p>
<p id="P53">The whole-genome sequencing data of human samples was analysed for purity and ploidy with methods previously described<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a></sup> (<a href="#F8" class="usa-link">Extended Data Fig. 3a</a> and <a href="#SD1" class="usa-link">Supplementary Table 2</a>).</p>
<p id="P54">Somatic mutations and copy number alterations were determined with our in-house analysis pipeline<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R49" class="usa-link" aria-describedby="R49" aria-expanded="false">49</a></sup>. The calling of somatic mutations in human samples was further improved by filtering the identified variant against the sequencing data of more than 500 normal samples (including exome or genome sequencing data). Additionally, an estimation of human DNA library contamination was implemented to enhance sensitivity and specificity of mutation calling.</p></section><section id="S19"><h3 class="pmc_sec_title">Analysis of significantly mutated and biologically relevant genes</h3>
<p id="P55">The significance of recurrently mutated genes was analysed for the whole-genome sequencing data set of 110 human SCLC samples (<a href="#F6" class="usa-link">Extended Data Fig. 1a</a>).</p>
<p id="P56">As previously described<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a></sup>, the analysis first estimated the background mutation rate for each gene and corrected for its expression by referring to the RNA-seq data of 81 human primary SCLC tumour specimen analysed in the present study. The analysis included those genes which had FPKM values (fragments per kilobase of exon per million fragments mapped) of over 1 in at least 50 samples. Following corrections for the occurrence of synonymous mutations, significantly mutated genes were determined with <em>q</em>-values of &lt; 0.05 (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#F6" class="usa-link">Extended Data Fig. 1a</a> (filter I) and <a href="#SD1" class="usa-link">Supplementary Table 6</a>).</p>
<p id="P57">Mutations that cluster within a gene are defined as a mutational hotspot similar to our previously described method<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a></sup>. Here we used an analytical derivation of the test statistics, rather than resampling. To this end, the mutated positions are rescaled to lie within zero and one (using the protein length). Under the null hypothesis of having no particular mutational hotspot, the rescaled mutated positions are uniformly distributed between zero and one, thus its expected value is 0.5. We therefore chose the final statistics as sum over the modulus of the rescaled position minus 0.5. This allows that the distribution under the null hypothesis can analytically be calculated; hence, also the <em>P</em> values. The analysis was calculated for genes that were significantly mutated in at least 5% of the samples with <em>P</em> &lt; 0.05 (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#F6" class="usa-link">Extended Data Fig. 1a</a> (filter II) and <a href="#SD1" class="usa-link">Supplementary Table 6</a>). In order to further filter for the genes of relevance, subsequent analysis considered those genes recurrently mutated in more than 8% (<em>n</em> &gt; 8) samples. The called genes were scored for their relevance by either analysing recurrent translocations affecting these genes (<a href="#SD1" class="usa-link">Supplementary Table 4</a>) or by comparison with the mouse SCLC mutation data to identify alterations in common genes (<a href="#SD1" class="usa-link">Supplementary Table 5</a>).</p>
<p id="P58">Additionally, recurrent mutations were scored for the accumulation of clearly damaging mutations in which splice site, frameshift and nonsense mutations were considered as damaging mutations. Here, we restricted the aforementioned significance analysis only to this class of mutations (by restricting the background mutation rate only to damaging mutations) and determined significance at <em>P</em> &lt; 0.01 (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#F6" class="usa-link">Extended Data Fig. 1a</a> (filter III) and <a href="#SD1" class="usa-link">Supplementary Table 6</a>).</p>
<p id="P59">Genetic alterations were further scored for their relevance by comparison with genes that were functionally characterized in genetically engineered mouse models (GEMM) for SCLC<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a>,<a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a></sup>, or by comparing somatic mutations in SCLC with mutations in other cancer types reported in the Cancer Gene Census<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a></sup> and in COSMIC (catalogue of somatic mutations in cancer)<sup><a href="#R15" class="usa-link" aria-describedby="R15" aria-expanded="false">15</a></sup> (<a href="#F1" class="usa-link">Fig. 1a</a>, <a href="#F6" class="usa-link">Extended Data Fig. 1a</a> (filter IV and V) and <a href="#SD1" class="usa-link">Supplementary Table 6</a>). Additionally, the sequencing data of mouse SCLC specimen was used to identify alterations in common genes.</p></section><section id="S20"><h3 class="pmc_sec_title">Analysis of subclonal architecture</h3>
<p id="P60">To determine the subclonal architecture from genome sequencing data, we first computed the cancer cell fraction (CCF; that is, the fraction of cancer cells carrying a particular mutation) of each called somatic point mutation. To this end, we first estimated the tumour purity, absolute copy numbers, and subclonal copy number changes using our previously described method<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a></sup> and computed for each mutation the expected allelic fraction under clonality assumption. The quotient between the observed allelic fraction of a mutation with its corresponding expected allelic fraction then yields the CCF. To assess the clonal and subclonal populations we next identified distinct clusters in the CCF profile and assigned each mutation to the cluster of highest probability. In order to provide a measure for the subclonal architecture, we proposed the following score:
</p>
<table class="disp-formula p" id="FD1"><tbody><tr><td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="0"><mjx-lazy data-mjx-lazy="0"></mjx-lazy></mjx-container></td></tr></tbody></table>
<p>
where <em>i</em> = 0 represents the clonal population, <em>i</em> = 1,…,<em>nc</em> the subclonal populations; φ<sub><em>i</em></sub> is the CCF of each population (thus, φ<sub>0</sub> ≈ 1), and <em>m<sub>i</sub></em> is the number of mutations assigned to cluster <em>i</em>. This subclonality score can be interpreted as the probability that a randomly selected mutation present in a single cancer cell is subclonal throughout the entire tumour.</p>
<p id="P61">As a low sequencing depth limits the robust identification of subclonal populations, we computed the genome-wide average contribution of a single mutated read to the CCF. For a given tumour purity <em>p</em>, average ploidy π, and mean coverage <em>c</em>, this measure is given by:
</p>
<table class="disp-formula p" id="FD2"><tbody><tr><td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="1"><mjx-lazy data-mjx-lazy="1"></mjx-lazy></mjx-container></td></tr></tbody></table>
<p>
The smaller the average increase of CCF per read, the more accurately the subclonality score can be determined since more subclonal mutations can be called from the sequencing data. In this study, the most limiting factor for assessing the subclonal diversity is the relatively low sequencing depth (35× on average). We therefore used this measure to select the samples that are suitable for a reliable calculation of the subclonality score. To this end, we systematically scanned from the average increase of CCF per read form large to small values and detected the point of the most prominent change in the distribution of the subclonality score (<a href="#SD1" class="usa-link">Supplementary Table 2</a>).</p></section><section id="S21"><h3 class="pmc_sec_title">Analysis of genomic breakpoints</h3>
<p id="P62">Genomic rearrangements were reconstructed from the whole-genome sequencing data of 110 human SCLC samples following the procedure as previously described<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R49" class="usa-link" aria-describedby="R49" aria-expanded="false">49</a></sup>. The genomic rearrangements called from each tumour sample were further filtered against a library of 110 normal genomes to thus minimize the detection of false-positive rearrangements. Genomic breakpoints of SCLC candidate driver genes are listed in <a href="#SD1" class="usa-link">Supplementary Table 4</a>.</p>
<p id="P63">The genomic breakpoints of all samples were mapped to their chromosomal locations and recurrent breakpoints clustering within the range of 100 kb were identified with a similar approach described previously<sup><a href="#R29" class="usa-link" aria-describedby="R29" aria-expanded="false">29</a></sup> (<a href="#SD1" class="usa-link">Supplementary Table 14</a>).</p></section><section id="S22"><h3 class="pmc_sec_title">Processing and analysis of RNA-seq data</h3>
<p id="P64">RNA-seq data was processed as previously described<sup><a href="#R2" class="usa-link" aria-describedby="R2" aria-expanded="false">2</a>,<a href="#R49" class="usa-link" aria-describedby="R49" aria-expanded="false">49</a></sup> to detect chimaeric transcripts and to determine the transcriptional abundance of annotated transcript variants. In brief, paired-end RNA-seq reads were mapped to the human reference genome (NCBI37/hg19) using GSNAP. Potential chimaeric fusion transcripts were identified by discordant read pairs and by individual reads mapping to distinct chromosomal locations. The sequence context of rearranged transcripts was reconstructed around the identified breakpoint and the assembled fusion transcript was then aligned to the human reference genome to determine the genes involved in the fusion.</p>
<p id="P65">Cufflinks was used to determine the expression levels of annotated transcripts referring to unique paired-end reads which align within the expected mapping distance. The expression is represented as FPKM values (<a href="#SD1" class="usa-link">Supplementary Table 10</a>).</p></section><section id="S23"><h3 class="pmc_sec_title">Transcript splicing analysis</h3>
<p id="P66">RNA-seq data was used to analyse for alternative splicing events of <em>TP53, RB1</em> and <em>TP73</em> caused by exon skipping or intron retention (<a href="#SD1" class="usa-link">Supplementary Table 11</a>). The paired-end reads were mapped to the reference genome (hg19) using STAR mapper. In reference to the annotation of exon junctions provided from UCSC genes and RefSeq the following parameters were applied: ref <a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a>, options:–alignIntronMin 20,–alignIntronMax 500000,–outFilterMismatchNmax 10, and–chimSegmentMin 10. The coordinates of reads potentially crossing exon boundaries were derived from the respective “SJ.out.tab” file and compared to the reference annotation. Subsequently, junction read counts were assigned to all transcripts containing the respective exon combination. If the exon combination is novel, read counts were assigned to those transcripts sharing one of the exons contributing to the novel junction. For subsequent analyses the transcript with the highest number of junction read counts was used as a reference. Additionally, for exon combinations unique to alternative transcripts a representative transcript was selected based on total read counts. The read counts of each exon junction were normalized to the reads per kilobase per million mapped reads (RPKM) per sample. These expression values were further normalized per gene by dividing by the average expression of the exons of the reference transcript. Potentially novel exon combinations were rejected if the average expression of the reference transcript was &lt;2 or if their expression were &lt;10% of the reference transcript expression.</p></section><section id="S24"><h3 class="pmc_sec_title">Differential expression for outlier studies</h3>
<p id="P67">Differential gene expression analysis was performed to compare the transcriptional profile of the two chromothripsis cases (<a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a> and S02353) with other non-chromothripsis SCLC cases and to thus identify outliers in the expression profile. The expression was analysed by computing <em>z</em>-scores for all samples referring to the RPKM values and using the R function ‘scale’; RPKM values smaller than 3 were set to 0. In order to prioritize for genes differentially expressed in the two samples <a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a> and S02353, genes were ranked by their respective <em>z</em>-scores. Statistical testing was performed for genes on Chr 3 and Chr 11, respectively. The <em>P</em> values were then combined from the two samples using Fisher’s method and corrected for multiple hypothesis testing by using the Benjamini–Hochberg approach. Differentially expressed genes with a <em>P</em> &lt; 0.05 and <em>q</em>-values &lt;0.01 are provided in <a href="#SD1" class="usa-link">Supplementary Table 13</a>.</p></section><section id="S25"><h3 class="pmc_sec_title">Unsupervised expression clustering</h3>
<p id="P68">Unsupervised clustering was performed with RNA-seq data of 69 SCLC cases for which matching genome sequencing data was available (<a href="#F4" class="usa-link">Fig. 4a</a> and <a href="#F14" class="usa-link">Extended Data Fig. 9a</a>). As expression values are approximately following log-normal distribution, we transformed raw FPKM of each transcript by log<sub>2</sub>(1 + FPKM). The resulting expression scores were then searched for a high and low expression characteristic over the samples. To this end, expression scores of each transcript were divided into two states using <em>k</em>-means clustering. To prevent an accumulation of artificial signals, only transcripts with at least 6 samples in each state and having a state-averaged fold change larger than 3 are considered for further analysis. A <em>t</em>-test is then computed between the two states of the remaining transcripts and corrected for multiple hypothesis testing using the false discovery rate framework. Next, transcripts having a <em>q</em>-value smaller than 0.01 were selected. For genes with multiple transcript variants, the transcript with the smallest <em>q</em>-value was chosen as representative transcript. Then, invariant genes were removed (having a standard deviation across all samples &lt;2). To improve clustering, only genes that share a similar pattern of the two states in at least 6 other genes are finally selected (using a Fisher’s exact test with a significance threshold of 10<sup>−6</sup>). Using the determined list of transcripts/genes, hierarchical clustering (Euclidean distance, complete linkage) was performed on the raw expression scores.</p></section><section id="S26"><h3 class="pmc_sec_title">
<em>IRS2</em> amplification FISH assay</h3>
<p id="P69">A fluorescence <em>in situ</em> hybridization (FISH) assay was used to detect and confirm <em>IRS2</em> amplifications at the chromosomal level. We performed a signal detection approach, with two probes on chromosome 13: the reference probe is located on the centromeric region of chromosome 13 (Empire Genomics, Art.Nr.: CHR13-10-GR) and was labelled with green 5-fluorescein dUTP to produce a green signal; the target probe is located on the <em>IRS2</em> locus spanning 13q33.3–34 and was labelled with biotin to produce a red signal using the CTD-2083015 BAC clone (Life Technologies, CA, USA). As previously described<sup><a href="#R24" class="usa-link" aria-describedby="R24" aria-expanded="false">24</a></sup>, slides of FFPE and fresh-frozen samples of tumour tissues were prepared, stained and analysed at a fluorescence microscope (Zeiss, Jena, Germany) with a 63× oil immersion objective. A non-amplified nucleus showed one red target signal for every corresponding green reference signal, with a red/green ratio of 1:1 (<a href="#F10" class="usa-link">Extended Data Fig. 5e</a>). High-level amplifications were determined for at least 10 red signals. In some cases the red signals were observed as clusters in the cells. At least 100 nuclei per case were evaluated.</p></section><section id="S27"><h3 class="pmc_sec_title">Immunohistochemistry</h3>
<p id="P70">Immunohistochemistry was performed on human tumour FFPE samples to analyse for the protein expression of Rb, cyclin D1, p53, p14 (ARF), and p16. The staining was performed with the BenchMark XT automated immunohistochemistry slide staining system (Roche). The following antibodies and conditions were applied: Rb (C-15) rabbit polyclonal (Santa Cruz; FFPE retrieving conditions: 60 min at pH 6.0; dilution: 1:500; incubation: 60 min, 37 °C); Cyclin D1 clone SP4 rabbit monoclonal (Microm France; FFPE retrieving conditions: 90 min at pH 8.4; dilution: 1:200; incubation: 60 min, 37 °C); p53 clone DO7 mouse monoclonal (Dako; FFPE retrieving conditions: 60 min at pH 8.4; dilution: 1:25; incubation: 60 min, 37 °C); p14ARF clone 4C6/4 mouse monoclonal (Cell Signaling; FFPE retrieving conditions: 60 min, water-bath 98 °C at pH 6.0; manual immunohistochemistry staining with Novolink Max polymer detection system (Leica); dilution: 1:4,000; incubation: overnight, 4 °C); p16 INK4 Ab-7 clone PO7 mouse monoclonal (Neomarkers; FFPE retrieving conditions: 60 min at pH 8.4; dilution: 1:800; incubation: 60 min, room temperature).</p>
<p id="P71">For immunohistochemistry on mouse tumour FFPE samples, sections were permeabilized for antigen retrieval by microwaving in a citrate-based antigen unmasking solution (Vector Laboratories). The following antibodies were used: GFP (Invitrogen; A11122; dilution: 1:400), RFP/Tomato (Rockland Immunochemicals; 600-401-379; dilution: 1:500), Notch2 (Cell Signaling; 5732; dilution: 1:200), Hes1 (Cell Signaling; 11988; dilution: 1:200), Ascl1 (BD Biosciences; 556604; dilution: 1:200) and Synaptophysin (Neuromics; MO20000; dilution: 1:200). Sections were developed with DAB (Vector Labs) and counterstained with haematoxylin.</p></section><section id="S28"><h3 class="pmc_sec_title">Cell lines, tissue culture and transfections</h3>
<p id="P72">Mouse (KP1) and human (NJH29, NCI-H82 and NCI-H187) SCLC cell lines were grown in RPMI-1640 media supplemented with 10% bovine growth serum (BGS) (Fisher Scientific) and penicillin-streptomycin-glutamine (Gibco), as described before<sup><a href="#R50" class="usa-link" aria-describedby="R50" aria-expanded="false">50</a></sup>. KP1, NJH29 were generated at Stanford. NCI-H82 and NCI-H187 were purchased from ATCC. These cells grow as suspension spheres or aggregates in culture. All cell lines were maintained at 37 °C in a humidified chamber with 5% CO<sub>2</sub>. All cell lines tested negative for mycoplasma infection. For transient expression of Notch ICD, cells were trypsinized and transfected with either MigR1-IRES-GFP (Ctrl) or MigR1-Notch1-ICD-IRES-GFP (NICD) using Lipofectamine 2000 (Life Technologies). The plasmids were gifts from W.S. Pear (University of Pennsylvania, Philadelphia). Then 48 h after transfection, cells were trypsinized and resuspended in phosphate-buffered saline (PBS) containing 10% BGS and 1 µg ml<sup>−1</sup> 7-aminoactinomycin D (Life Technologies) that labels dead cells. Live GFP<sup>+</sup> cells were then sorted for subsequent experiments using a BD FACSAria fluorescence-activated cell sorting (FACS) machine.</p></section><section id="S29"><h3 class="pmc_sec_title">Gene expression and microarray analysis</h3>
<p id="P73">Gene expression and microarray analyses were performed with the mouse cell line KP1 transiently transfected with MigR1-IRES-GFP (Ctrl) or MigR1-Notch1-ICD-IRES-GFP (N1ICD).</p>
<p id="P74">Then 1 × 10<sup>5</sup> GFP<sup>+</sup> cells were sorted and the RNA isolated using the AllPrep DNA/RNA micro kit (Qiagen). RNA quality assessment using the 2100 Bioanalyzer (Agilent) as well as the subsequent cDNA preparation steps for microarray analysis were performed at the Stanford Protein and Nucleic Acid (PAN) facility using the GeneChip Mouse Gene 2.0 ST Array (Affymetrix). For gene expression analysis, the Robust Multichip Average (RMA) Express 1.0.4 program was used for background adjustment and quantile RMA normalization of the 41,345 probe sets encoding mouse genome transcripts. Linear models for microarray data (LIMMA) was used to compare Ctrl or N1ICD samples on RMA normalized signal intensities. Only genes with an adjusted <em>P</em> value of 0.05 or less were considered as significantly differentially expressed. A total of 769 probes accounting for 760 genes were significant, and the expression levels of these genes were represented as a heatmap using the heatmap.2 function in R. The analysis was performed in triplicates. A list of significant genes is provided in <a href="#SD1" class="usa-link">Supplementary Table 15</a>.</p></section><section id="S30"><h3 class="pmc_sec_title">MTT cell viability assay</h3>
<p id="P75">Sorted GFP<sup>+</sup> cells were seeded at 1 × 10<sup>4</sup> per well in 96-well plates. The MTT reagents (Roche) were added on days 0, 2, 4, 6 and 8 for mouse SCLC cell lines or on days 0, 2, 4 and 6 for the human SCLC cell line NJH29. The absorbance wavelength was 570 nm with a reference wavelength of 650 nm.</p></section><section id="S31"><h3 class="pmc_sec_title">EdU incorporation assay</h3>
<p id="P76">Transfected cells were treated with 10 µM EdU (5-ethynyl-2′-deoxyuridine) (Life Technologies) for 3 h before trypsinization for FACS. 1 × 10<sup>5</sup> live, GFP<sup>+</sup> cells were sorted and labelled with EdU using the Click-iT EdU Pacific Blue flow cytometry assay kit (Life Technologies). Cells were then run through the BD FACSAria to analyse for per cent EdU incorporation.</p></section><section id="S32"><h3 class="pmc_sec_title">Data reporting</h3>
<p id="P77">No statistical methods were used to predetermine sample size.</p></section></section><section id="S33"><h2 class="pmc_sec_title" data-anchor-id="S33"><a id="S33-anchor" data-anchor-id="S33" class="usa-anchor"></a>Extended Data</h2>
<figure class="fig xbox font-sm" id="F6"><h3 class="obj_head">Extended Data Figure 1. Genomic analyses in SCLC tumours.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/81a1ac5529c5/nihms782739f6.jpg" loading="lazy" height="1050" width="870" alt="Extended Data Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P78"><strong>a</strong>, Schematic detailing the genomic study and number of samples as well as various steps of analyses for the identification of candidate genes in SCLC. <strong>b</strong>, Illustration of the number of samples analysed in this study.</p></figcaption></figure><figure class="fig xbox font-sm" id="F7"><h3 class="obj_head">Extended Data Figure 2. Clinical molecular-correlation analyses.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/06331923fe44/nihms782739f7.jpg" loading="lazy" height="765" width="1050" alt="Extended Data Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P79"><strong>a</strong>, Survival analysis of SCLC patients based on clinical stage and treatment options (surgery and/or chemotherapy). Statistical significance was determined by log-rank test. <strong>b</strong>, Analyses of clinical stage and smoking status and the respective effect on number and type of mutations, as well as mutational subclonality in tumours. Statistical significance was determined by Kruskal–Wallis analysis.</p></figcaption></figure><figure class="fig xbox font-sm" id="F8"><h3 class="obj_head">Extended Data Figure 3. Genomic characterization of SCLC tumours.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/4dac6d56f0a2/nihms782739f8.jpg" loading="lazy" height="1050" width="854" alt="Extended Data Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P80"><strong>a</strong>, Purity and ploidy determined in SCLC tumours by whole-genome sequencing presented as dot density plots showing median and the interquartile range (IQR) <strong>b</strong>, Subclonal architecture of SCLC in comparison to lung adenocarcinoma (AD). Whole-genome sequencing data of SCLC and of adenocarcinoma (<em>n</em> = 15)<sup><a href="#R11" class="usa-link" aria-describedby="R11" aria-expanded="false">11</a></sup> was analysed for the presence of subclonal populations using clustering of the derived cancer cell fraction (CCF) of all single nucleotide mutations. To compare the emerging subclonal structure, we derived a subclonality score that takes into account the CCF of each subpopulation as well as its mutational burden (see Methods). In order to prevent the low sequencing coverage (35 × for SCLC and 63 × for AD) from causing a systematic underrepresentation of the subclonal diversity in the mutation calls, we computed the contribution of a single read to the CCF on genome-wide average. After systematically determining a threshold within the average increase of CCF per read values (see Methods for details), we determined the group of samples for which a reliable estimation of the subclonality score is not possible (grey area). The subclonality scores of the remaining SCLC cases were then compared to those of the adenocarcinoma cases (<em>P</em> = 0.000232; Mann–Whitney test). <strong>c</strong>, Schematic representation of candidate genes with significant clustering of mutations in respective protein domains. Somatic mutations and genomic translocations are mapped to the respective protein regions. Hotspot mutations are highlighted in red. <strong>d, e</strong>, Genomic alterations in the <em>RB1</em> family proteins p107 (<em>RBL1</em>) and p130 (<em>RBL2</em>) (<strong>d</strong>), and in <em>KIT</em> and <em>PIK3CA</em> (<strong>e</strong>). Somatic mutations in therapeutic target genes are listed and mapped to the protein domains of <em>KIT</em> and <em>PIK3CA.</em> Mutations with potential therapeutic implications are highlighted in red.</p></figcaption></figure><figure class="fig xbox font-sm" id="F9"><h3 class="obj_head">Extended Data Figure 4. Clinical molecular-correlations of significantly mutated genes.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/6b6e2c0db93f/nihms782739f9.jpg" loading="lazy" height="597" width="1050" alt="Extended Data Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P81"><strong>a</strong>, Survival analysis of SCLC patients based on the status of <em>CREBBP/EP300, TP73</em> or <em>NOTCH</em> alterations. Statistical significance was determined by log-rank test. <strong>b</strong>, Analysis of <em>CREBBP/EP300, TP73</em> and <em>NOTCH</em> alterations and their effect on clinical and genetic parameters. Statistical significance was analysed by multinomial logistic regression.</p></figcaption></figure><figure class="fig xbox font-sm" id="F10"><h3 class="obj_head">Extended Data Figure 5. Significant somatic copy number alterations in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f10.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/7a419f6dea27/nihms782739f10.jpg" loading="lazy" height="1050" width="1015" alt="Extended Data Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/F10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P82"><strong>a</strong>, Deletions of the chromosomal arm 3p point to the 3p14 (<em>FHIT</em>) and 3p12 (<em>ROBO1</em>) locus. <strong>b</strong>, Expression analyses of genes encoded on the 3p14.3–3p14.2 and 3p12.2–3p12.2 locus. Histogram displaying the expression of samples with focal deletions (blue) and samples without any copy number alterations (white). Mean and standard error of the mean is plotted for each gene in each group. Significant differences were determined by Mann–Whitney test; *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01. <strong>c, d</strong>, Focal deletions of the <em>CDKN2A</em> (<strong>c</strong>) and focal amplifications of <em>IRS2</em> (<strong>d</strong>) were found on chromosome 9 and 13, respectively. The copy number (CN) states were computed from SNP array (SNP 6.0) and from whole-genome sequencing (WGS) data. The samples are sorted according to their amplitude of deletions or amplifications. <strong>e</strong>, Amplifications of <em>IRS2</em> were determined by FISH analysis. <em>IRS</em> amplifications were quantified based on the ratio of red signals (<em>IRS2</em>-specific probe) to green signals (centromere probe for chromosome 13). Lymphocyte spreads and SCLC tumours without detectable <em>IRS2</em> amplifications served as negative controls. Scale bar, 100 µm.</p></figcaption></figure><figure class="fig xbox font-sm" id="F11"><h3 class="obj_head">Extended Data Figure 6. <em>TP53</em> and <em>RB1</em> alterations in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f11.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/000b54937505/nihms782739f11.jpg" loading="lazy" height="1050" width="888" alt="Extended Data Figure 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/F11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P83"><strong>a</strong>, Distribution of somatic mutations in <em>TP53</em> and <em>RB1</em> according to the colour panel provided. <strong>b, c</strong>, Complex genomic rearrangements in <em>RB1</em> showing homozygous deletions of exon 1 (<strong>b</strong>) or inversions within the <em>RB1</em> gene (<strong>c</strong>). <strong>d, e</strong>, Annotated silent or missense mutations in <em>RB1</em> occur at intron-exon junctions resulting in alternative splicing, intron retention (<strong>d</strong>) or exon skipping events (<strong>e</strong>). The coverage at the respective exon junctions is quantified as RPKM values. Sample <a href="https://www.ncbi.nlm.nih.gov/protein/S02194" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02194</a> is not holding any mutations at intron–exon junctions and is displayed as an example for unaltered splicing of <em>RB1</em>.</p></figcaption></figure><figure class="fig xbox font-sm" id="F12"><h3 class="obj_head">Extended Data Figure 7. Chromothripsis in human SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f12.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/1b1e0557321c/nihms782739f12.jpg" loading="lazy" height="1050" width="964" alt="Extended Data Figure 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/F12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P84"><strong>a</strong>, Circos plot of the chromothripsis sample S02353 showing intra- and interchromosomal rearrangements between chromosome 3 and 11. The integral copy number state (iCN) is plotted as a heatmap and assigned to the respective chromosomal regions. The chromosomal context of <em>CCND1</em> (on chromosome 11) is highlighted. <strong>b</strong>, Circos plots displaying fusion transcripts identified in the SCLC chromothripsis cases (<a href="#SD1" class="usa-link">Supplementary Table 12</a>) are represented as blue (<a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a>) or red (S02353) lines for genes located on chromosome 3 and 11. <strong>c</strong>, Immunohistochemistry staining for p53, p14 (ARF) and p16 on FFPE material of the chromothripsis sample <a href="https://www.ncbi.nlm.nih.gov/protein/S02297" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02297</a>. Original magnification, ×400.</p></figcaption></figure><figure class="fig xbox font-sm" id="F13"><h3 class="obj_head">Extended Data Figure 8. Recurrent genomic translocations in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f13.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/b7d2b385a117/nihms782739f13.jpg" loading="lazy" height="949" width="1050" alt="Extended Data Figure 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/F13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P85"><strong>a</strong>, Recurrent genomic translocations (<em>n</em> = 14) affecting chromosome 22 are illustrated as a Circos plot highlighting the respective rearrangements as red connecting lines. <strong>b</strong>, Breakpoints in chromosome 22 map to intron 1 of <em>TTC28</em> and cluster downstream of the LINE1 (L1Hs) retrotransposon. Each arrow indicates the sample and the respective chromosomal position the segment translocates to. <strong>c</strong>, Schematic representation of the <em>TP73</em> locus (hg19) describing complex intrachromosomal rearrangements of <em>TP73</em> identified for S02397 and <a href="https://www.ncbi.nlm.nih.gov/protein/S02243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02243</a>. Recurrent somatic mutations identified in <a href="#F1" class="usa-link">Fig. 1a</a> are mapped to the respective exons. <strong>d</strong>, Validation of somatic <em>TP73</em> translocations. Genomic regions involved in the <em>TP73</em> rearrangements were amplified in matched normal (N) and tumour (T) samples. The expected band size is indicated in brackets. The respective PCR products were subjected to Sanger sequencing to confirm the genomic breakpoint. <strong>e</strong>, Copy-number state of the <em>TP73</em> gene in samples involved in genomic translocations.</p></figcaption></figure><figure class="fig xbox font-sm" id="F14"><h3 class="obj_head">Extended Data Figure 9. Transcriptome profile of human SCLC tumours.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f14.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/ba6e936b23c7/nihms782739f14.jpg" loading="lazy" height="1050" width="789" alt="Extended Data Figure 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/F14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P86"><strong>a</strong>, Unsupervised hierarchical clustering of transcriptome sequencing data of 69 SCLC specimens as described in <a href="#F4" class="usa-link">Fig. 4a</a>. Each sample is annotated for the genomic alterations described in <a href="#F1" class="usa-link">Fig. 1</a>. Black filled boxes describe the presence of a genomic event. <strong>b</strong>, Expression values of <em>CHGA, GRP, ASCL1</em> and <em>DLK1</em> (FPKM) are represented as dot density plots for the subgroups identified in (<strong>a</strong>). Red lines highlight the median value for each group. <strong>c</strong>, Expression values of the neuroendocrine markers <em>SYP</em> (synaptophysin) and <em>NCAM1</em> (CD56) plotted as scatter plots for all SCLC samples. Green lines indicate thresholds for no expression (FPKM&lt;1).</p></figcaption></figure><figure class="fig xbox font-sm" id="F15"><h3 class="obj_head">Extended Data Figure 10. Notch is a tumour suppressor in SCLC regulating neuroendocrine differentiation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4861069_nihms782739f15.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6c55/4861069/98327eb74e94/nihms782739f15.jpg" loading="lazy" height="853" width="1050" alt="Extended Data Figure 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/F15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P87"><strong>a</strong>, Somatic mutations identified in <em>NOTCH3</em> and <em>NOTCH4</em> are mapped to the protein domains. Damaging mutations are highlighted in red. Mutations found in murine SCLC tumours are highlighted in blue. <strong>b</strong>, Quantification of tumour lesions and per cent tumour area to lung in TKO (<em>n</em> = 5) and TKO;N1ICD (<em>n</em> = 4) mice 3 months after Ad-Cre instillation. Statistical significance was determined by two-tailed unpaired Student’s <em>t</em>-test. <strong>c</strong>, Representative immunohistochemistry for GFP or tdTomato in lungs from <em>TKO;Rosa26<sup>mT/mG</sup></em> mice approximately 6 months after tumour induction. Left scale bar, 500 µm; right and middle: scale bar, 50 µm. <strong>d</strong>, Representative immunostaining for Notch2 in lungs from <em>TKO;Rosa26<sup>N2ICD</sup></em> mice approximately 6 months after tumour induction. Left scale bar, 500 µm; right scale bar, 50 µm. <strong>e</strong>, Quantification of the per cent recombination at the <em>Rosa26</em> locus in <em>TKO;Rosa26<sup>mT/mG</sup></em> (<em>n</em> = 6) and <em>TKO;Rosa26<sup>N2ICD</sup></em> mice (<em>n</em> = 10; two-tailed unpaired Student’s <em>t</em>-test). <strong>f</strong>, Cell viability assay of the human SCLC cell line NJH29 transfected with a N1ICD (Notch1) expression plasmid or empty vector control (Ctrl) (3 independent biological replicas with 3 technical replicas each). Fold growth was normalized to day 0; representative images were taken on day 6. Scale bar, 50 µm. <strong>g</strong>, Immunohistochemistry staining in FFPE embedded tissues of TKO and TKO;N2ICD mice. Scale bar, 50 µm. <strong>h</strong>, Quantitative RT–PCR validation of <em>Notch1</em> induction and the expression of common Notch target genes after N1ICD transfection in murine SCLC cells (three biological replicas; two-tailed paired Student’s <em>t</em>-test). <strong>i</strong>, Mouse SCLC cells transfected with control or N1ICD (Notch1) were analysed 48 h later by gene expression microarrays. Gene Set Enrichment Analysis (GSEA) was performed on these data; selected significant gene sets are displayed. <strong>j, k</strong>, EdU analysis of mouse (<strong>j</strong>) and human (<strong>k</strong>) SCLC cells (three independent biological replicas with three technical replicas each; two-tailed paired Student’s <em>t</em>-test). *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001. Data are represented as mean ± s.d.</p></figcaption></figure></section><section id="SM"><h2 class="pmc_sec_title" data-anchor-id="SM"><a id="SM-anchor" data-anchor-id="SM" class="usa-anchor"></a>Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>Tables</span></div>
<div class="media p" id="d37e3463"><div class="caption">
<a href="/articles/instance/4861069/bin/NIHMS782739-supplement-Tables.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS782739-supplement-Tables.xlsx</a><sup> (31.5MB, xlsx) </sup>
</div></div></section></section><section id="S34" class="ack"><h2 class="pmc_sec_title" data-anchor-id="S34"><a id="S34-anchor" data-anchor-id="S34" class="usa-anchor"></a>Acknowledgments</h2>
<p id="P92">We are grateful to all the patients who contributed their tumour specimens. We thank the computing center of the University of Cologne (RRZK) for providing the CPU time on the DFG-funded supercomputer ‘CHEOPS’, as well as for the support. We thank S. Artavanis-Tsakonas and S. Fre for the gift of the mice with inducible NICD expression. We thank C. Nguyen, J. Berg, J. Heuckmann, F. Malchers, C. Lovely and A. Bernschein for scientific discussions and advice. We thank Genentech/gRED for providing raw sequencing data from a previously published study<sup><a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup>. Some tumors in these studies were provided by the LungBiobank Heidelberg, member of the NCT-Tissue bank, the biomaterial bank Heidelberg and the biobank platform of the German Center for Lung Research, Heidelberg, Germany. This work was supported by the German Cancer Aid (Deutsche Krebshilfe) as part of the small cell lung cancer genome sequencing consortium (grant ID: 109679 to R.K.T., M.P., R.B., P.N., M.V. and S.A.H.). Further support was provided by the Korea Research Foundation (KRF 2011-0030105; grant to S.J.J.). Additional funding was provided by the NIH (5R01CA114102-08 to J.S.), the German Ministry of Science and Education (BMBF) as part of the NGFNplus program (grant 01GS08101 to R.K.T., J.W. and P.N.) and as part of the e:Med program (grant no. 01ZX1303A to R.K.T., J.W., C.R., R.B. and M.P. and grant no. 01ZX1406 to M.P.), by the Deutsche Forschungsgemeinschaft (DFG; through TH1386/3-1 to R.K.T and KFO-286 to P.N.), by the German federal state North Rhine Westphalia (NRW), by the European Union (European Regional Development Fund: Investing In Your Future) as part of the PerMed NRW initiative (grant 005-1111-0025 to R.K.T., J.W. and R.B.), by SFB832 (TP6 to R.K.T., TP5 to L.C.H.), by the Deutsche Krebshilfe as part of the Oncology Centers of Excellence funding program (to R.B., R.K.T. and M.S.), by the EU-Framework program CURELUNG (HEALTH-F2-2010-258677 to R.K.T., J.W., J.K.F., L.R., M.S.C. and E.B.), by Stand Up To Cancer—American Association of Cancer Research Innovative Research Grant (SU2C-AACR-IR60109 to R.K.T.), by the German Consortium for Translational Cancer Research (DKTK) Joint Funding program, by the National Cancer Center Research and Development Fund (NCC Biobank: 23A-1, to T.K., J.Y. and R.I.), by the Italian Ministry of Health (Ricerca Corrente RC1303LO57 and GR program 2010-2316264 to L.A.M.), by the Roy Castle Lung Cancer Foundation UK (to J.K.F.), by the AIRC/MGAF grant 12983 (to L.A.M.) and by A*STAR in Singapore (scholarship to J.S.L.). J.S. is the Harriet and Mary Zelencik Scientist in Children’s Cancer and Blood Diseases.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN3"><p id="P88"><strong>Supplementary Information</strong> is available in the online version of the paper.</p></div>
<div class="fn p" id="FN4"><p id="P89"><strong>Author Contributions</strong> J.G., J.S.L., M.P., J.S. and R.K.T. conceived the project, analysed and interpreted the data, and wrote the manuscript. J.G., J.S.L., J.S., F.L., R.M., S.P., D.E., B.Pü, M.S.W., J.O.K., J.A., C.B., M.B. and P.S. designed experiments. J.G., J.S.L., I.D., C.M., A.T., R.M., S.-M.C., D.K., D.E., I.V., D.S., B.Pi, P.S., C.B., P.M.S. and M.Bog performed experiments. J.G., M.P., J.S.L., S.A.H., M.V., M.S.W., J.O.K., Y.C., X.L., D.V., M.W., N.H. and M.Bos performed data analysis. E.B., W.D.T., R.B., L.H., L.O., S.P., S.J.J. and G.K. performed pathology review. E.B., W.D.T., L.O., A.N.K., Y.Y. and V.T. conducted further immunohistochemistry studies. F.L., L.F.-C., G.B., S.M., D.S., V.A., U.L., T.Z., S.A., M.H., J.W., P.N. and C.R. helped with logistics. S.J.J., N.S.J., K.-S.P., D.Y., J.Y., T.K., R.I., K.T., M.N., T.M., H.H., P.A.S., I.P., Y.C., A.S., C.-M.C., Y.-H.K., P.P.M., Y.Z., D.J., M.K., G.M.W., P.A.R., B.S., I.K., M.L., L.A.M, A.l.T., J.K.F., M.J., J.Kn., E.C.-V., L.R., U.P., O.-T.B., M.L.-I., E.T., J.Kö., M.Sc, J.B., M.Sa, M.S.-C., H.B.S., Y.Y., S.P., L.H., R.B. and E.B. contributed with murine and human tissue samples.</p></div>
<div class="fn p" id="FN5"><p id="P90">Affymetrix SNP 6.0, whole-genome, and transcriptome sequencing on human specimen have been deposited at the European Genome-phenome Archive under the accession code EGAS00001000925. Whole-exome and whole-genome sequencing data of murine SCLC tumors can be accessed through (<a href="http://www.translational-genomics.uni-koeln.de/scientific-resources/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.translational-genomics.uni-koeln.de/scientific-resources/</a>). Microarray data on mouse cell lines is accessible through Gene Expression Omnibus (GEO) accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE69091</a>.</p></div>
<div class="fn p" id="FN6"><p id="P91">The authors declare competing financial interests: details are available in the online version of the paper. Readers are welcome to comment on the online version of the paper.</p></div>
</div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>van Meerbeeck JP, Fennell DA, De Ruysscher DK. Small-cell lung cancer. Lancet. 2011;378:1741–1755. doi: 10.1016/S0140-6736(11)60165-7.</cite> [<a href="https://doi.org/10.1016/S0140-6736(11)60165-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21565397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Small-cell%20lung%20cancer&amp;author=JP%20van%20Meerbeeck&amp;author=DA%20Fennell&amp;author=DK%20De%20Ruysscher&amp;volume=378&amp;publication_year=2011&amp;pages=1741-1755&amp;pmid=21565397&amp;doi=10.1016/S0140-6736(11)60165-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Peifer M, et al.  Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. Nature Genet. 2012;44:1104–1110. doi: 10.1038/ng.2396.</cite> [<a href="https://doi.org/10.1038/ng.2396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4915822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genet&amp;title=Integrative%20genome%20analyses%20identify%20key%20somatic%20driver%20mutations%20of%20small-cell%20lung%20cancer&amp;author=M%20Peifer&amp;volume=44&amp;publication_year=2012&amp;pages=1104-1110&amp;pmid=22941188&amp;doi=10.1038/ng.2396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Rudin CM, et al.  Comprehensive genomic analysis identifies SOX2 as a frequently amplified gene in small-cell lung cancer. Nature Genet. 2012;44:1111–1116. doi: 10.1038/ng.2405.</cite> [<a href="https://doi.org/10.1038/ng.2405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genet&amp;title=Comprehensive%20genomic%20analysis%20identifies%20SOX2%20as%20a%20frequently%20amplified%20gene%20in%20small-cell%20lung%20cancer&amp;author=CM%20Rudin&amp;volume=44&amp;publication_year=2012&amp;pages=1111-1116&amp;pmid=22941189&amp;doi=10.1038/ng.2405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Takahashi T, et al.  p53: a frequent target for genetic abnormalities in lung cancer. Science. 1989;246:491–494. doi: 10.1126/science.2554494.</cite> [<a href="https://doi.org/10.1126/science.2554494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2554494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=p53:%20a%20frequent%20target%20for%20genetic%20abnormalities%20in%20lung%20cancer&amp;author=T%20Takahashi&amp;volume=246&amp;publication_year=1989&amp;pages=491-494&amp;pmid=2554494&amp;doi=10.1126/science.2554494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Wistuba II, Gazdar AF, Minna JD. Molecular genetics of small cell lung carcinoma. Semin. Oncol. 2001;28:3–13.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11479891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin.%20Oncol&amp;title=Molecular%20genetics%20of%20small%20cell%20lung%20carcinoma&amp;author=II%20Wistuba&amp;author=AF%20Gazdar&amp;author=JD%20Minna&amp;volume=28&amp;publication_year=2001&amp;pages=3-13&amp;pmid=11479891&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Horowitz JM, et al.  Frequent inactivation of the retinoblastoma anti-oncogene is restricted to a subset of human tumor cells. Proc. Natl Acad. Sci. USA. 1990;87:2775–2779. doi: 10.1073/pnas.87.7.2775.</cite> [<a href="https://doi.org/10.1073/pnas.87.7.2775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC53773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2181449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Frequent%20inactivation%20of%20the%20retinoblastoma%20anti-oncogene%20is%20restricted%20to%20a%20subset%20of%20human%20tumor%20cells&amp;author=JM%20Horowitz&amp;volume=87&amp;publication_year=1990&amp;pages=2775-2779&amp;pmid=2181449&amp;doi=10.1073/pnas.87.7.2775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Mori N, et al.  Variable mutations of the RB gene in small-cell lung carcinoma. Oncogene. 1990;5:1713–1717.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2176283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Variable%20mutations%20of%20the%20RB%20gene%20in%20small-cell%20lung%20carcinoma&amp;author=N%20Mori&amp;volume=5&amp;publication_year=1990&amp;pages=1713-1717&amp;pmid=2176283&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Meuwissen R, et al.  Induction of small cell lung cancer by somatic inactivation of both Trp53 and Rb1 in a conditional mouse model. Cancer Cell. 2003;4:181–189. doi: 10.1016/s1535-6108(03)00220-4.</cite> [<a href="https://doi.org/10.1016/s1535-6108(03)00220-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14522252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Induction%20of%20small%20cell%20lung%20cancer%20by%20somatic%20inactivation%20of%20both%20Trp53%20and%20Rb1%20in%20a%20conditional%20mouse%20model&amp;author=R%20Meuwissen&amp;volume=4&amp;publication_year=2003&amp;pages=181-189&amp;pmid=14522252&amp;doi=10.1016/s1535-6108(03)00220-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Schaffer BE, et al.  Loss of p130 accelerates tumor development in a mouse model for human small-cell lung carcinoma. Cancer Res. 2010;70:3877–3883. doi: 10.1158/0008-5472.CAN-09-4228.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-09-4228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2873158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20406986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Loss%20of%20p130%20accelerates%20tumor%20development%20in%20a%20mouse%20model%20for%20human%20small-cell%20lung%20carcinoma&amp;author=BE%20Schaffer&amp;volume=70&amp;publication_year=2010&amp;pages=3877-3883&amp;pmid=20406986&amp;doi=10.1158/0008-5472.CAN-09-4228&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>McFadden DG, et al.  Genetic and clonal dissection of murine small cell lung carcinoma progression by genome sequencing. Cell. 2014;156:1298–1311. doi: 10.1016/j.cell.2014.02.031.</cite> [<a href="https://doi.org/10.1016/j.cell.2014.02.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4040459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24630729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Genetic%20and%20clonal%20dissection%20of%20murine%20small%20cell%20lung%20carcinoma%20progression%20by%20genome%20sequencing&amp;author=DG%20McFadden&amp;volume=156&amp;publication_year=2014&amp;pages=1298-1311&amp;pmid=24630729&amp;doi=10.1016/j.cell.2014.02.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Imielinski M, et al.  Mapping the hallmarks of lung adenocarcinoma with massively parallel sequencing. Cell. 2012;150:1107–1120. doi: 10.1016/j.cell.2012.08.029.</cite> [<a href="https://doi.org/10.1016/j.cell.2012.08.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22980975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mapping%20the%20hallmarks%20of%20lung%20adenocarcinoma%20with%20massively%20parallel%20sequencing&amp;author=M%20Imielinski&amp;volume=150&amp;publication_year=2012&amp;pages=1107-1120&amp;pmid=22980975&amp;doi=10.1016/j.cell.2012.08.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Zhang J, et al.  Intratumor heterogeneity in localized lung adenocarcinomas delineated by multiregion sequencing. Science. 2014;346:256–259. doi: 10.1126/science.1256930.</cite> [<a href="https://doi.org/10.1126/science.1256930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4354858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25301631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Intratumor%20heterogeneity%20in%20localized%20lung%20adenocarcinomas%20delineated%20by%20multiregion%20sequencing&amp;author=J%20Zhang&amp;volume=346&amp;publication_year=2014&amp;pages=256-259&amp;pmid=25301631&amp;doi=10.1126/science.1256930&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>de Bruin EC, et al.  Spatial and temporal diversity in genomic instability processes defines lung cancer evolution. Science. 2014;346:251–256. doi: 10.1126/science.1253462.</cite> [<a href="https://doi.org/10.1126/science.1253462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25301630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Spatial%20and%20temporal%20diversity%20in%20genomic%20instability%20processes%20defines%20lung%20cancer%20evolution&amp;author=EC%20de%20Bruin&amp;volume=346&amp;publication_year=2014&amp;pages=251-256&amp;pmid=25301630&amp;doi=10.1126/science.1253462&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Futreal PA, et al.  A census of human cancer genes. Nature Rev. Cancer. 2004;4:177–183. doi: 10.1038/nrc1299.</cite> [<a href="https://doi.org/10.1038/nrc1299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2665285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14993899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Rev.%20Cancer&amp;title=A%20census%20of%20human%20cancer%20genes&amp;author=PA%20Futreal&amp;volume=4&amp;publication_year=2004&amp;pages=177-183&amp;pmid=14993899&amp;doi=10.1038/nrc1299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Forbes SA, et al.  COSMIC: exploring the world’s knowledge of somatic mutations in human cancer. Nucleic Acids Res. 2015;43:D805–D811. doi: 10.1093/nar/gku1075.</cite> [<a href="https://doi.org/10.1093/nar/gku1075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25355519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=COSMIC:%20exploring%20the%20world%E2%80%99s%20knowledge%20of%20somatic%20mutations%20in%20human%20cancer&amp;author=SA%20Forbes&amp;volume=43&amp;publication_year=2015&amp;pages=D805-D811&amp;pmid=25355519&amp;doi=10.1093/nar/gku1075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Lawrence MS, et al.  Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature. 2013;499:214–218. doi: 10.1038/nature12213.</cite> [<a href="https://doi.org/10.1038/nature12213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3919509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23770567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutational%20heterogeneity%20in%20cancer%20and%20the%20search%20for%20new%20cancer-associated%20genes&amp;author=MS%20Lawrence&amp;volume=499&amp;publication_year=2013&amp;pages=214-218&amp;pmid=23770567&amp;doi=10.1038/nature12213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Yokomizo A, et al.  PTEN/MMAC1 mutations identified in small cell, but not in non-small cell lung cancers. Oncogene. 1998;17:475–479. doi: 10.1038/sj.onc.1201956.</cite> [<a href="https://doi.org/10.1038/sj.onc.1201956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9696041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=PTEN/MMAC1%20mutations%20identified%20in%20small%20cell,%20but%20not%20in%20non-small%20cell%20lung%20cancers&amp;author=A%20Yokomizo&amp;volume=17&amp;publication_year=1998&amp;pages=475-479&amp;pmid=9696041&amp;doi=10.1038/sj.onc.1201956&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Holderfield M, Deuker MM, McCormick F, McMahon M. Targeting RAF kinases for cancer therapy: BRAF-mutated melanoma and beyond. Nature Rev. Cancer. 2014;14:455–467. doi: 10.1038/nrc3760.</cite> [<a href="https://doi.org/10.1038/nrc3760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4250230/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24957944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Rev.%20Cancer&amp;title=Targeting%20RAF%20kinases%20for%20cancer%20therapy:%20BRAF-mutated%20melanoma%20and%20beyond&amp;author=M%20Holderfield&amp;author=MM%20Deuker&amp;author=F%20McCormick&amp;author=M%20McMahon&amp;volume=14&amp;publication_year=2014&amp;pages=455-467&amp;pmid=24957944&amp;doi=10.1038/nrc3760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Hibi K, et al.  Coexpression of the stem cell factor and the c-kit genes in small-cell lung cancer. Oncogene. 1991;6:2291–2296.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1722571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Coexpression%20of%20the%20stem%20cell%20factor%20and%20the%20c-kit%20genes%20in%20small-cell%20lung%20cancer&amp;author=K%20Hibi&amp;volume=6&amp;publication_year=1991&amp;pages=2291-2296&amp;pmid=1722571&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Hirota S, et al.  Gain-of-function mutations of c-kit in human gastrointestinal stromal tumors. Science. 1998;279:577–580. doi: 10.1126/science.279.5350.577.</cite> [<a href="https://doi.org/10.1126/science.279.5350.577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9438854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gain-of-function%20mutations%20of%20c-kit%20in%20human%20gastrointestinal%20stromal%20tumors&amp;author=S%20Hirota&amp;volume=279&amp;publication_year=1998&amp;pages=577-580&amp;pmid=9438854&amp;doi=10.1126/science.279.5350.577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Shibata T, Kokubu A, Tsuta K, Hirohashi S. Oncogenic mutation of PIK3CA in small cell lung carcinoma: a potential therapeutic target pathway for chemotherapy-resistant lung cancer. Cancer Lett. 2009;283:203–211. doi: 10.1016/j.canlet.2009.03.038.</cite> [<a href="https://doi.org/10.1016/j.canlet.2009.03.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19394761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&amp;title=Oncogenic%20mutation%20of%20PIK3CA%20in%20small%20cell%20lung%20carcinoma:%20a%20potential%20therapeutic%20target%20pathway%20for%20chemotherapy-resistant%20lung%20cancer&amp;author=T%20Shibata&amp;author=A%20Kokubu&amp;author=K%20Tsuta&amp;author=S%20Hirohashi&amp;volume=283&amp;publication_year=2009&amp;pages=203-211&amp;pmid=19394761&amp;doi=10.1016/j.canlet.2009.03.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Angeloni D, et al.  Analysis of a new homozygous deletion in the tumor suppressor region at 3p12.3 reveals two novel intronic noncoding RNA genes. Genes Chromosom. Cancer. 2006;45:676–691. doi: 10.1002/gcc.20332.</cite> [<a href="https://doi.org/10.1002/gcc.20332" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16607615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosom.%20Cancer&amp;title=Analysis%20of%20a%20new%20homozygous%20deletion%20in%20the%20tumor%20suppressor%20region%20at%203p12.3%20reveals%20two%20novel%20intronic%20noncoding%20RNA%20genes&amp;author=D%20Angeloni&amp;volume=45&amp;publication_year=2006&amp;pages=676-691&amp;pmid=16607615&amp;doi=10.1002/gcc.20332&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Kovatich A, et al.  Molecular alterations to human chromosome 3p loci in neuroendocrine lung tumors. Cancer. 1998;83:1109–1117. doi: 10.1002/(sici)1097-0142(19980915)83:6&lt;1109::aid-cncr9&gt;3.0.co;2-2.</cite> [<a href="https://doi.org/10.1002/(sici)1097-0142(19980915)83:6<1109::aid-cncr9>3.0.co;2-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9740075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Molecular%20alterations%20to%20human%20chromosome%203p%20loci%20in%20neuroendocrine%20lung%20tumors&amp;author=A%20Kovatich&amp;volume=83&amp;publication_year=1998&amp;pages=1109-1117&amp;pmid=9740075&amp;doi=10.1002/(sici)1097-0142(19980915)83:6<1109::aid-cncr9>3.0.co;2-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Weiss J, et al.  Frequent and focal FGFR1 amplification associates with therapeutically tractable FGFR1 dependency in squamous cell lung cancer. Sci. Transl. Med. 2010;2:62ra93. doi: 10.1126/scitranslmed.3001451.</cite> [<a href="https://doi.org/10.1126/scitranslmed.3001451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3990281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21160078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med&amp;title=Frequent%20and%20focal%20FGFR1%20amplification%20associates%20with%20therapeutically%20tractable%20FGFR1%20dependency%20in%20squamous%20cell%20lung%20cancer&amp;author=J%20Weiss&amp;volume=2&amp;publication_year=2010&amp;pages=62ra93&amp;pmid=21160078&amp;doi=10.1126/scitranslmed.3001451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Korbel JO, Campbell PJ. Criteria for inference of chromothripsis in cancer genomes. Cell. 2013;152:1226–1236. doi: 10.1016/j.cell.2013.02.023.</cite> [<a href="https://doi.org/10.1016/j.cell.2013.02.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23498933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Criteria%20for%20inference%20of%20chromothripsis%20in%20cancer%20genomes&amp;author=JO%20Korbel&amp;author=PJ%20Campbell&amp;volume=152&amp;publication_year=2013&amp;pages=1226-1236&amp;pmid=23498933&amp;doi=10.1016/j.cell.2013.02.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Knudson AG. Two genetic hits (more or less) to cancer. 1
. Nature Rev. Genet. 2001;1:157–162. doi: 10.1038/35101031.</cite> [<a href="https://doi.org/10.1038/35101031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11905807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Rev.%20Genet&amp;title=Two%20genetic%20hits%20(more%20or%20less)%20to%20cancer.%201&amp;author=AG%20Knudson&amp;volume=1&amp;publication_year=2001&amp;pages=157-162&amp;pmid=11905807&amp;doi=10.1038/35101031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Berger AH, Knudson AG, Pandolfi PP. A continuum model for tumour suppression. Nature. 2011;476:163–169. doi: 10.1038/nature10275.</cite> [<a href="https://doi.org/10.1038/nature10275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3206311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21833082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20continuum%20model%20for%20tumour%20suppression&amp;author=AH%20Berger&amp;author=AG%20Knudson&amp;author=PP%20Pandolfi&amp;volume=476&amp;publication_year=2011&amp;pages=163-169&amp;pmid=21833082&amp;doi=10.1038/nature10275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Beasley MB, et al.  The P16/cyclin D1/Rb pathway in neuroendocrine tumors of the lung. Hum. Pathol. 2003;34:136–142. doi: 10.1053/hupa.2003.8.</cite> [<a href="https://doi.org/10.1053/hupa.2003.8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12612881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Pathol&amp;title=The%20P16/cyclin%20D1/Rb%20pathway%20in%20neuroendocrine%20tumors%20of%20the%20lung&amp;author=MB%20Beasley&amp;volume=34&amp;publication_year=2003&amp;pages=136-142&amp;pmid=12612881&amp;doi=10.1053/hupa.2003.8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Northcott PA, et al.  Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma. Nature. 2014;511:428–434. doi: 10.1038/nature13379.</cite> [<a href="https://doi.org/10.1038/nature13379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4201514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25043047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Enhancer%20hijacking%20activates%20GFI1%20family%20oncogenes%20in%20medulloblastoma&amp;author=PA%20Northcott&amp;volume=511&amp;publication_year=2014&amp;pages=428-434&amp;pmid=25043047&amp;doi=10.1038/nature13379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>The Cancer Genome Atlas Network. Comprehensive molecular characterization of human colon and rectal cancer. Nature. 2012;487:330–337. doi: 10.1038/nature11252.</cite> [<a href="https://doi.org/10.1038/nature11252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3401966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20molecular%20characterization%20of%20human%20colon%20and%20rectal%20cancer&amp;volume=487&amp;publication_year=2012&amp;pages=330-337&amp;pmid=22810696&amp;doi=10.1038/nature11252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Pitkänen E, Cajuso T, Katainen R, Kaasinen E, Välimäki N. Frequent L1 retrotranspositions originating from TTC28 in colorectal cancer. Oncotarget. 2014;5:853–859. doi: 10.18632/oncotarget.1781.</cite> [<a href="https://doi.org/10.18632/oncotarget.1781" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3996660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24553397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Frequent%20L1%20retrotranspositions%20originating%20from%20TTC28%20in%20colorectal%20cancer&amp;author=E%20Pitk%C3%A4nen&amp;author=T%20Cajuso&amp;author=R%20Katainen&amp;author=E%20Kaasinen&amp;author=N%20V%C3%A4lim%C3%A4ki&amp;volume=5&amp;publication_year=2014&amp;pages=853-859&amp;pmid=24553397&amp;doi=10.18632/oncotarget.1781&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Helman E, Lawrence ML, Stewart C. Somatic retrotransposition in human cancer revealed by whole-genome and exome sequencing. Genome Res. 2014;24:1053–1063. doi: 10.1101/gr.163659.113.</cite> [<a href="https://doi.org/10.1101/gr.163659.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4079962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24823667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Somatic%20retrotransposition%20in%20human%20cancer%20revealed%20by%20whole-genome%20and%20exome%20sequencing&amp;author=E%20Helman&amp;author=ML%20Lawrence&amp;author=C%20Stewart&amp;volume=24&amp;publication_year=2014&amp;pages=1053-1063&amp;pmid=24823667&amp;doi=10.1101/gr.163659.113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Jancalek R. The role of the TP73 gene and its transcripts in neuro-oncology. Br. J. Neurosurg. 2014;28:598–605. doi: 10.3109/02688697.2014.908162.</cite> [<a href="https://doi.org/10.3109/02688697.2014.908162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24742294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Neurosurg&amp;title=The%20role%20of%20the%20TP73%20gene%20and%20its%20transcripts%20in%20neuro-oncology&amp;author=R%20Jancalek&amp;volume=28&amp;publication_year=2014&amp;pages=598-605&amp;pmid=24742294&amp;doi=10.3109/02688697.2014.908162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Tannapfel A, et al.  Autonomous growth and hepatocarcinogenesis in transgenic mice expressing the p53 family inhibitor DNp73. Carcinogenesis. 2008;29:211–218. doi: 10.1093/carcin/bgm236.</cite> [<a href="https://doi.org/10.1093/carcin/bgm236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17984115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Autonomous%20growth%20and%20hepatocarcinogenesis%20in%20transgenic%20mice%20expressing%20the%20p53%20family%20inhibitor%20DNp73&amp;author=A%20Tannapfel&amp;volume=29&amp;publication_year=2008&amp;pages=211-218&amp;pmid=17984115&amp;doi=10.1093/carcin/bgm236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Venkatanarayan A, et al.  IAPP-driven metabolic reprogramming induces regression of p53-deficient tumours in vivo. Nature. 2015;517:626–630. doi: 10.1038/nature13910.</cite> [<a href="https://doi.org/10.1038/nature13910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4312210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25409149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=IAPP-driven%20metabolic%20reprogramming%20induces%20regression%20of%20p53-deficient%20tumours%20in%20vivo&amp;author=A%20Venkatanarayan&amp;volume=517&amp;publication_year=2015&amp;pages=626-630&amp;pmid=25409149&amp;doi=10.1038/nature13910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Falix FA, Aronson DC, Lamers WH, Gaemers IC. Possible roles of DLK1 in the Notch pathway during development and disease. Biochim. Biophys. Acta. 2012;1822:988–995. doi: 10.1016/j.bbadis.2012.02.003.</cite> [<a href="https://doi.org/10.1016/j.bbadis.2012.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22353464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&amp;title=Possible%20roles%20of%20DLK1%20in%20the%20Notch%20pathway%20during%20development%20and%20disease&amp;author=FA%20Falix&amp;author=DC%20Aronson&amp;author=WH%20Lamers&amp;author=IC%20Gaemers&amp;volume=1822&amp;publication_year=2012&amp;pages=988-995&amp;pmid=22353464&amp;doi=10.1016/j.bbadis.2012.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Ball DW. Achaete-scute homolog-1 and Notch in lung neuroendocrine development and cancer. Cancer Lett. 2004;204:159–169. doi: 10.1016/S0304-3835(03)00452-X.</cite> [<a href="https://doi.org/10.1016/S0304-3835(03)00452-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15013215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&amp;title=Achaete-scute%20homolog-1%20and%20Notch%20in%20lung%20neuroendocrine%20development%20and%20cancer&amp;author=DW%20Ball&amp;volume=204&amp;publication_year=2004&amp;pages=159-169&amp;pmid=15013215&amp;doi=10.1016/S0304-3835(03)00452-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Augustyn A, et al.  ASCL1 is a lineage oncogene providing therapeutic targets for high-grade neuroendocrine lung cancers. Proc. Natl Acad. Sci. USA. 2014;111:14788–14793. doi: 10.1073/pnas.1410419111.</cite> [<a href="https://doi.org/10.1073/pnas.1410419111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4205603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25267614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=ASCL1%20is%20a%20lineage%20oncogene%20providing%20therapeutic%20targets%20for%20high-grade%20neuroendocrine%20lung%20cancers&amp;author=A%20Augustyn&amp;volume=111&amp;publication_year=2014&amp;pages=14788-14793&amp;pmid=25267614&amp;doi=10.1073/pnas.1410419111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Sriuranpong V, et al.  Notch signaling induces cell cycle arrest in small cell lung cancer cells. Cancer Res. 2001;1:3200–3205.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11306509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Notch%20signaling%20induces%20cell%20cycle%20arrest%20in%20small%20cell%20lung%20cancer%20cells&amp;author=V%20Sriuranpong&amp;volume=1&amp;publication_year=2001&amp;pages=3200-3205&amp;pmid=11306509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Wael H, et al.  Notch1 signaling controls cell proliferation, apoptosis and differentiation in lung carcinoma. Lung Cancer. 2014;85:131–140. doi: 10.1016/j.lungcan.2014.05.001.</cite> [<a href="https://doi.org/10.1016/j.lungcan.2014.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24888228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=Notch1%20signaling%20controls%20cell%20proliferation,%20apoptosis%20and%20differentiation%20in%20lung%20carcinoma&amp;author=H%20Wael&amp;volume=85&amp;publication_year=2014&amp;pages=131-140&amp;pmid=24888228&amp;doi=10.1016/j.lungcan.2014.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Rangarajan A, et al.  Notch signaling is a direct determinant of keratinocyte growth arrest and entry into differentiation. EMBO J. 2001;20:3427–3436. doi: 10.1093/emboj/20.13.3427.</cite> [<a href="https://doi.org/10.1093/emboj/20.13.3427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC125257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11432830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J&amp;title=Notch%20signaling%20is%20a%20direct%20determinant%20of%20keratinocyte%20growth%20arrest%20and%20entry%20into%20differentiation&amp;author=A%20Rangarajan&amp;volume=20&amp;publication_year=2001&amp;pages=3427-3436&amp;pmid=11432830&amp;doi=10.1093/emboj/20.13.3427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Qi R, et al.  Notch1 signaling inhibits growth of human hepatocellular carcinoma through induction of cell cycle arrest and apoptosis. Cancer Res. 2003;63:8323–8329.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/14678992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Notch1%20signaling%20inhibits%20growth%20of%20human%20hepatocellular%20carcinoma%20through%20induction%20of%20cell%20cycle%20arrest%20and%20apoptosis&amp;author=R%20Qi&amp;volume=63&amp;publication_year=2003&amp;pages=8323-8329&amp;pmid=14678992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="ref-list2" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list2"><a id="ref-list2-anchor" data-anchor-id="ref-list2" class="usa-anchor"></a>References</h2>
<section id="ref-list2_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R43">
<span class="label">43.</span><cite>Iwakawa R, et al.  Genome-wide identification of genes with amplification and/or fusion in small cell lung cancer. Genes Chromosomes Cancer. 2013;52:802–816. doi: 10.1002/gcc.22076.</cite> [<a href="https://doi.org/10.1002/gcc.22076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3806277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23716474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&amp;title=Genome-wide%20identification%20of%20genes%20with%20amplification%20and/or%20fusion%20in%20small%20cell%20lung%20cancer&amp;author=R%20Iwakawa&amp;volume=52&amp;publication_year=2013&amp;pages=802-816&amp;pmid=23716474&amp;doi=10.1002/gcc.22076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Seidel D. A genomics-based classification of human lung tumors. Sci. Transl. Med. 2013;5:209ra153. doi: 10.1126/scitranslmed.3006802.</cite> [<a href="https://doi.org/10.1126/scitranslmed.3006802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4006630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24174329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med&amp;title=A%20genomics-based%20classification%20of%20human%20lung%20tumors&amp;author=D%20Seidel&amp;volume=5&amp;publication_year=2013&amp;pages=209ra153&amp;pmid=24174329&amp;doi=10.1126/scitranslmed.3006802&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Lafkas D, et al.  Notch3 marks clonogenic mammary luminal progenitor cells in vivo. J. Cell Biol. 2013;203:47–56. doi: 10.1083/jcb.201307046.</cite> [<a href="https://doi.org/10.1083/jcb.201307046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3798243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24100291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=Notch3%20marks%20clonogenic%20mammary%20luminal%20progenitor%20cells%20in%20vivo&amp;author=D%20Lafkas&amp;volume=203&amp;publication_year=2013&amp;pages=47-56&amp;pmid=24100291&amp;doi=10.1083/jcb.201307046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Muzumdar MD, Tasic B, Miyamichi K, Li L, Luo L. A global double-fluorescent Cre reporter mouse. Genesis. 2007;45:593–605. doi: 10.1002/dvg.20335.</cite> [<a href="https://doi.org/10.1002/dvg.20335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17868096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genesis&amp;title=A%20global%20double-fluorescent%20Cre%20reporter%20mouse&amp;author=MD%20Muzumdar&amp;author=B%20Tasic&amp;author=K%20Miyamichi&amp;author=L%20Li&amp;author=L%20Luo&amp;volume=45&amp;publication_year=2007&amp;pages=593-605&amp;pmid=17868096&amp;doi=10.1002/dvg.20335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Lu X, Thomas RK, Peifer M. CGARS: cancer genome analysis by rank sums. Bioinformatics. 2014;30:1295–1296. doi: 10.1093/bioinformatics/btu011.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24413525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CGARS:%20cancer%20genome%20analysis%20by%20rank%20sums&amp;author=X%20Lu&amp;author=RK%20Thomas&amp;author=M%20Peifer&amp;volume=30&amp;publication_year=2014&amp;pages=1295-1296&amp;pmid=24413525&amp;doi=10.1093/bioinformatics/btu011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Li H, Durbin R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics. 2009;25:1754–1760. doi: 10.1093/bioinformatics/btp324.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H%20Li&amp;author=R%20Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Fernandez-Cuesta L, et al.  Frequent mutations in chromatin-remodelling genes in pulmonary carcinoids. Nat. Commun. 2014;5:3518. doi: 10.1038/ncomms4518.</cite> [<a href="https://doi.org/10.1038/ncomms4518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4132974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24670920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Frequent%20mutations%20in%20chromatin-remodelling%20genes%20in%20pulmonary%20carcinoids&amp;author=L%20Fernandez-Cuesta&amp;volume=5&amp;publication_year=2014&amp;pages=3518&amp;pmid=24670920&amp;doi=10.1038/ncomms4518&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Jahchan NS, et al.  A drug repositioning approach identifies tricyclic antidepressants as inhibitors of small cell lung cancer and other neuroendocrine tumors. Cancer Discov. 2013;3:1364–1377. doi: 10.1158/2159-8290.CD-13-0183.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-0183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3864571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24078773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=A%20drug%20repositioning%20approach%20identifies%20tricyclic%20antidepressants%20as%20inhibitors%20of%20small%20cell%20lung%20cancer%20and%20other%20neuroendocrine%20tumors&amp;author=NS%20Jahchan&amp;volume=3&amp;publication_year=2013&amp;pages=1364-1377&amp;pmid=24078773&amp;doi=10.1158/2159-8290.CD-13-0183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Tables</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/4861069/bin/NIHMS782739-supplement-Tables.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS782739-supplement-Tables.xlsx</a><sup> (31.5MB, xlsx) </sup>
</div></div></section></section></section></section></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1038/nature14664" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/nihms782739.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.8&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/4861069/" data-citation-style="nlm" data-download-format-link="/resources/citations/4861069/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC4861069%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC4861069/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC4861069/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC4861069/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/26168399/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC4861069/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/26168399/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC4861069/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/4861069/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="abstract1" href="#abstract1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S1" href="#S1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Samples and clinical data">Samples and clinical data</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S2" href="#S2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Recurrent somatic alterations in SCLC">Recurrent somatic alterations in SCLC</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S3" href="#S3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Universal inactivation of TP53 and RB1
">Universal inactivation of TP53 and RB1
</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S4" href="#S4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Oncogenic genomic events affecting TP73
">Oncogenic genomic events affecting TP73
</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S5" href="#S5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Tumour suppressive roles of Notch in SCLC">Tumour suppressive roles of Notch in SCLC</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S6" href="#S6" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Discussion">Discussion</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S7" href="#S7" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="METHODS">METHODS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S33" href="#S33" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Extended Data">Extended Data</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="SM" href="#SM" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary Material">Supplementary Material</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S34" href="#S34" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Acknowledgments">Acknowledgments</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list2" href="#ref-list2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="mVrUECvYedR0DbJtBuVhjs5cVBNsN4EmvxFjhuLjhl55CRQf5RotjnEP4zrtig13">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Amanuscript" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-3a787fba.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>