<html lang="en" class=""><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He5561v847342615za200&amp;tag_exp=101509157~103101750~103101752~103116026~103200004~103233427~103251618~103251620~103252644~103252646~103284320~103284322~103301114~103301116"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EAuthor Manuscript%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Proteogenomic insights suggest druggable pathways in endometrial carcinoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="5BB846BA81BE2B130746BA0027BD201F.m_1">
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov">
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="nihpa">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="manuscript">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Cancer cell">
<meta name="citation_title" content="Proteogenomic insights suggest druggable pathways in endometrial carcinoma">
<meta name="citation_author" content="Yongchao Dou">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Lizabeth Katsnelson">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Marina A Gritsenko">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Yingwei Hu">
<meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA">
<meta name="citation_author" content="Boris Reva">
<meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA">
<meta name="citation_author" content="Runyu Hong">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Yi-Ting Wang">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Iga Kolodziejczak">
<meta name="citation_author_institution" content="International Institute for Molecular Oncology, 20-203 Poznań, Poland">
<meta name="citation_author_institution" content="Postgraduate School of Molecular Medicine, Medical University of Warsaw, 02-091 Warsaw, Poland">
<meta name="citation_author" content="Rita Jui-Hsien Lu">
<meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA">
<meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA">
<meta name="citation_author" content="Chia-Feng Tsai">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Wen Bu">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Medicine, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Wenke Liu">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Xiaofang Guo">
<meta name="citation_author_institution" content="Division of Gynecologic Oncology, University of South Florida Morsani College of Medicine and Tampa General Hospital Cancer Institute, Tampa, FL 33606, USA">
<meta name="citation_author" content="Eunkyung An">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Rebecca C Arend">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA">
<meta name="citation_author" content="Jasmin Bavarva">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Lijun Chen">
<meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA">
<meta name="citation_author" content="Rosalie K Chu">
<meta name="citation_author_institution" content="Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Andrzej Czekański">
<meta name="citation_author_institution" content="Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland">
<meta name="citation_author" content="Teresa Davoli">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Elizabeth G Demicco">
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Mount Sinai Hospital and Laboratory Medicine and Pathobiology, University of Toronto, Toronto, M5G 1X5, Canada">
<meta name="citation_author" content="Deborah DeLair">
<meta name="citation_author_institution" content="Department of Pathology, NYU Grossman School of Medicine, New York, NY, 10016, USA">
<meta name="citation_author" content="Kelly Devereaux">
<meta name="citation_author_institution" content="Department of Pathology, NYU Grossman School of Medicine, New York, NY, 10016, USA">
<meta name="citation_author" content="Saravana M Dhanasekaran">
<meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author" content="Peter Dottino">
<meta name="citation_author_institution" content="Department of Obstetrics, Gynecology and Reproductive Science, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA">
<meta name="citation_author" content="Bailee Dover">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA">
<meta name="citation_author" content="Thomas L Fillmore">
<meta name="citation_author_institution" content="Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="McKenzie Foxall">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA">
<meta name="citation_author" content="Catherine E Hermann">
<meta name="citation_author_institution" content="Division of Gynecologic Oncology, Department of Obstetrics and Gynecology, NYU Grossman School of Medicine, New York, NY, 10016, USA">
<meta name="citation_author" content="Tara Hiltke">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Galen Hostetter">
<meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA">
<meta name="citation_author" content="Marcin Jędryka">
<meta name="citation_author_institution" content="Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland">
<meta name="citation_author" content="Scott D Jewell">
<meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA">
<meta name="citation_author" content="Isabelle Johnson">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Andrea G Kahn">
<meta name="citation_author_institution" content="Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA">
<meta name="citation_author" content="Amy T Ku">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Chandan Kumar-Sinha">
<meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author" content="Paweł Kurzawa">
<meta name="citation_author_institution" content="Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań">
<meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland">
<meta name="citation_author" content="Alexander J Lazar">
<meta name="citation_author_institution" content="Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author" content="Rossana Lazcano">
<meta name="citation_author_institution" content="Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author" content="Jonathan T Lei">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Yi Li">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Yuxing Liao">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Tung-Shing M Lih">
<meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA">
<meta name="citation_author" content="Tai-Tu Lin">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="John A Martignetti">
<meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA">
<meta name="citation_author" content="Ramya P Masand">
<meta name="citation_author_institution" content="Department of Pathology &amp; Immunology, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Rafał Matkowski">
<meta name="citation_author_institution" content="Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland">
<meta name="citation_author" content="Wilson McKerrow">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Mehdi Mesri">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Matthew E Monroe">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Jamie Moon">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Ronald J Moore">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Michael D Nestor">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Chelsea Newton">
<meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA">
<meta name="citation_author" content="Tatiana Omelchenko">
<meta name="citation_author_institution" content="The Rockefeller University, New York, NY 10065, USA">
<meta name="citation_author" content="Gilbert S Omenn">
<meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Department of Internal Medicine, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Department of Human Genetics, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author" content="Samuel H Payne">
<meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA">
<meta name="citation_author" content="Vladislav A Petyuk">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Ana I Robles">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Henry Rodriguez">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Kelly V Ruggles">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Division of Precision Medicine, Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Dmitry Rykunov">
<meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA">
<meta name="citation_author" content="Sara R Savage">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Athena A Schepmoes">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Tujin Shi">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Zhiao Shi">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Jimin Tan">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Mason Taylor">
<meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA">
<meta name="citation_author" content="Mathangi Thiagarajan">
<meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA">
<meta name="citation_author" content="Joshua M Wang">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Karl K Weitz">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Bo Wen">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="C M Williams">
<meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA">
<meta name="citation_author" content="Yige Wu">
<meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA">
<meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA">
<meta name="citation_author" content="Matthew A Wyczalkowski">
<meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA">
<meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA">
<meta name="citation_author" content="Xinpei Yi">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Xu Zhang">
<meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA">
<meta name="citation_author" content="Rui Zhao">
<meta name="citation_author_institution" content="Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="David Mutch">
<meta name="citation_author_institution" content="Washington University School of Medicine, Saint Louis, MO, 63110, USA">
<meta name="citation_author" content="Arul M Chinnaiyan">
<meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author" content="Richard D Smith">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="Alexey I Nesvizhskii">
<meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA">
<meta name="citation_author" content="Pei Wang">
<meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA">
<meta name="citation_author" content="Maciej Wiznerowicz">
<meta name="citation_author_institution" content="International Institute for Molecular Oncology, 60-203 Poznań, Poland">
<meta name="citation_author_institution" content="Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań, Poland">
<meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland">
<meta name="citation_author" content="Li Ding">
<meta name="citation_author_institution" content="Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA">
<meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA">
<meta name="citation_author" content="D R Mani">
<meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA">
<meta name="citation_author" content="Hui Zhang">
<meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA">
<meta name="citation_author" content="Matthew L Anderson">
<meta name="citation_author_institution" content="Division of Gynecologic Oncology, University of South Florida Morsani College of Medicine and Tampa General Hospital Cancer Institute, Tampa, FL 33606, USA">
<meta name="citation_author" content="Karin D Rodland">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author_institution" content="Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97221, USA">
<meta name="citation_author" content="Bing Zhang">
<meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA">
<meta name="citation_author" content="Tao Liu">
<meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA">
<meta name="citation_author" content="David Fenyö">
<meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA">
<meta name="citation_author" content="Clinical Proteomic Tumor Analysis Consortium">
<meta name="citation_publication_date" content="2023 Aug 10">
<meta name="citation_volume" content="41">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="1586">
<meta name="citation_doi" content="10.1016/j.ccell.2023.07.007">
<meta name="citation_pmid" content="37567170">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/pdf/nihms-1921121.pdf">
<meta name="description" content="We characterized a prospective endometrial carcinoma (EC) cohort containing 138 tumors and 20 enriched normal tissues using 10 different omics platforms. Targeted quantitation of two peptides can predict antigen processing and presentation machinery ...">
<meta name="og:title" content="Proteogenomic insights suggest druggable pathways in endometrial carcinoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="We characterized a prospective endometrial carcinoma (EC) cohort containing 138 tumors and 20 enriched normal tissues using 10 different omics platforms. Targeted quantitation of two peptides can predict antigen processing and presentation machinery ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/8.b8b768ed34bc7505f919.chunk.js?Q_CLIENTVERSION=2.28.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d3874cd52b47a77b4475.chunk.js?Q_CLIENTVERSION=2.28.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.faadef8bceb94e053f79.chunk.js?Q_CLIENTVERSION=2.28.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="10631452">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <span class="usa-tooltip"><a href="https://doi.org/10.1016/j.ccell.2023.07.007" class="usa-link display-flex usa-tooltip__trigger" role="button" target="_blank" rel="noreferrer noopener" data-position="bottom" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile" aria-describedby="tooltip-403480" tabindex="0">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a><span class="usa-tooltip__body" id="tooltip-403480" role="tooltip" aria-hidden="true">View on publisher site</span></span>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <span class="usa-tooltip"><a href="pdf/nihms-1921121.pdf" class="usa-link display-flex usa-tooltip__trigger" role="button" data-position="bottom" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile" aria-describedby="tooltip-377376" tabindex="0">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a><span class="usa-tooltip__body" id="tooltip-377376" role="tooltip" aria-hidden="true">Download PDF</span></span>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <span class="usa-tooltip"><button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty usa-tooltip__trigger" data-position="bottom" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC10631452%2F%23open-collections-dialog" data-in-collections="false" aria-describedby="tooltip-360900" tabindex="0">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button><span class="usa-tooltip__body" id="tooltip-360900" role="tooltip" aria-hidden="true">Add to Collections</span></span>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <span class="usa-tooltip"><button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex usa-tooltip__trigger" aria-label="Open dialog with citation text in different styles" data-position="bottom" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/10631452/" data-citation-style="nlm" data-download-format-link="/resources/citations/10631452/export/" aria-describedby="tooltip-939237" tabindex="0">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button><span class="usa-tooltip__body" id="tooltip-939237" role="tooltip" aria-hidden="true">Cite</span></span>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <span class="usa-tooltip"><button type="button" data-position="bottom" class="usa-button usa-button--unstyled display-flex usa-tooltip__trigger" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile" aria-describedby="tooltip-847756" tabindex="0">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button><span class="usa-tooltip__body" id="tooltip-847756" role="tooltip" aria-hidden="true">Permalink</span></span>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nihpa.png" alt="NIHPA Author Manuscripts logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NIHPA Author Manuscripts" title="Link to NIHPA Author Manuscripts" shape="default" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Cancer Cell</button></div>. Author manuscript; available in PMC: 2024 Sep 11.</div>
<div>
<em>Published in final edited form as: </em>Cancer Cell. 2023 Aug 10;41(9):1586–1605.e15. doi: <a href="https://doi.org/10.1016/j.ccell.2023.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1016/j.ccell.2023.07.007</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Cancer%20Cell%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cancer%20Cell%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cancer%20Cell%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Cancer%20Cell%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Proteogenomic insights suggest druggable pathways in endometrial carcinoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Yongchao Dou</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yongchao Dou</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongchao Dou</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Katsnelson%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Lizabeth Katsnelson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lizabeth Katsnelson</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Katsnelson%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizabeth Katsnelson</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gritsenko%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Marina A Gritsenko</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marina A Gritsenko</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gritsenko%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Marina A Gritsenko</span></a>
</div>
</div>
<sup>6,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Yingwei Hu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yingwei Hu</span></h3>
<div class="p">
<sup>7</sup>Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yingwei Hu</span></a>
</div>
</div>
<sup>7,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Boris Reva</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Boris Reva</span></h3>
<div class="p">
<sup>8</sup>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Boris Reva</span></a>
</div>
</div>
<sup>8,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Runyu Hong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Runyu Hong</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Runyu Hong</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20YT%22%5BAuthor%5D" class="usa-link" aria-describedby="id7" aria-expanded="false"><span class="name western">Yi-Ting Wang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yi-Ting Wang</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20YT%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi-Ting Wang</span></a>
</div>
</div>
<sup>6,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolodziejczak%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id8" aria-expanded="false"><span class="name western">Iga Kolodziejczak</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Iga Kolodziejczak</span></h3>
<div class="p">
<sup>9</sup>International Institute for Molecular Oncology, 20-203 Poznań, Poland</div>
<div class="p">
<sup>10</sup>Postgraduate School of Molecular Medicine, Medical University of Warsaw, 02-091 Warsaw, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolodziejczak%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Iga Kolodziejczak</span></a>
</div>
</div>
<sup>9,</sup><sup>10,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20RJH%22%5BAuthor%5D" class="usa-link" aria-describedby="id9" aria-expanded="false"><span class="name western">Rita Jui-Hsien Lu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Rita Jui-Hsien Lu</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA</div>
<div class="p">
<sup>12</sup>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20RJH%22%5BAuthor%5D" class="usa-link"><span class="name western">Rita Jui-Hsien Lu</span></a>
</div>
</div>
<sup>11,</sup><sup>12,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20CF%22%5BAuthor%5D" class="usa-link" aria-describedby="id10" aria-expanded="false"><span class="name western">Chia-Feng Tsai</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Chia-Feng Tsai</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20CF%22%5BAuthor%5D" class="usa-link"><span class="name western">Chia-Feng Tsai</span></a>
</div>
</div>
<sup>6,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id11" aria-expanded="false"><span class="name western">Wen Bu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Wen Bu</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>13</sup>Department of Medicine, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen Bu</span></a>
</div>
</div>
<sup>1,</sup><sup>13,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id12" aria-expanded="false"><span class="name western">Wenke Liu</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Wenke Liu</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenke Liu</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id13" aria-expanded="false"><span class="name western">Xiaofang Guo</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Xiaofang Guo</span></h3>
<div class="p">
<sup>14</sup>Division of Gynecologic Oncology, University of South Florida Morsani College of Medicine and Tampa General Hospital Cancer Institute, Tampa, FL 33606, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaofang Guo</span></a>
</div>
</div>
<sup>14,</sup><sup>$</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22An%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id14" aria-expanded="false"><span class="name western">Eunkyung An</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Eunkyung An</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22An%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eunkyung An</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arend%20RC%22%5BAuthor%5D" class="usa-link" aria-describedby="id15" aria-expanded="false"><span class="name western">Rebecca C Arend</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Rebecca C Arend</span></h3>
<div class="p">
<sup>16</sup>Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arend%20RC%22%5BAuthor%5D" class="usa-link"><span class="name western">Rebecca C Arend</span></a>
</div>
</div>
<sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bavarva%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id16" aria-expanded="false"><span class="name western">Jasmin Bavarva</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Jasmin Bavarva</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bavarva%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jasmin Bavarva</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id17" aria-expanded="false"><span class="name western">Lijun Chen</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Lijun Chen</span></h3>
<div class="p">
<sup>7</sup>Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lijun Chen</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id18" aria-expanded="false"><span class="name western">Rosalie K Chu</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Rosalie K Chu</span></h3>
<div class="p">
<sup>17</sup>Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rosalie K Chu</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Czeka%C5%84ski%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id19" aria-expanded="false"><span class="name western">Andrzej Czekański</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Andrzej Czekański</span></h3>
<div class="p">
<sup>18</sup>Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Czeka%C5%84ski%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrzej Czekański</span></a>
</div>
</div>
<sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davoli%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id20" aria-expanded="false"><span class="name western">Teresa Davoli</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Teresa Davoli</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davoli%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teresa Davoli</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demicco%20EG%22%5BAuthor%5D" class="usa-link" aria-describedby="id21" aria-expanded="false"><span class="name western">Elizabeth G Demicco</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Elizabeth G Demicco</span></h3>
<div class="p">
<sup>19</sup>Department of Pathology and Laboratory Medicine, Mount Sinai Hospital and Laboratory Medicine and Pathobiology, University of Toronto, Toronto, M5G 1X5, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demicco%20EG%22%5BAuthor%5D" class="usa-link"><span class="name western">Elizabeth G Demicco</span></a>
</div>
</div>
<sup>19</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22DeLair%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id22" aria-expanded="false"><span class="name western">Deborah DeLair</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Deborah DeLair</span></h3>
<div class="p">
<sup>20</sup>Department of Pathology, NYU Grossman School of Medicine, New York, NY, 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22DeLair%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Deborah DeLair</span></a>
</div>
</div>
<sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devereaux%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id23" aria-expanded="false"><span class="name western">Kelly Devereaux</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Kelly Devereaux</span></h3>
<div class="p">
<sup>20</sup>Department of Pathology, NYU Grossman School of Medicine, New York, NY, 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devereaux%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kelly Devereaux</span></a>
</div>
</div>
<sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id24" aria-expanded="false"><span class="name western">Saravana M Dhanasekaran</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Saravana M Dhanasekaran</span></h3>
<div class="p">
<sup>21</sup>Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>22</sup>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Saravana M Dhanasekaran</span></a>
</div>
</div>
<sup>21,</sup><sup>22</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dottino%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id25" aria-expanded="false"><span class="name western">Peter Dottino</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Peter Dottino</span></h3>
<div class="p">
<sup>23</sup>Department of Obstetrics, Gynecology and Reproductive Science, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dottino%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Dottino</span></a>
</div>
</div>
<sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dover%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id26" aria-expanded="false"><span class="name western">Bailee Dover</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Bailee Dover</span></h3>
<div class="p">
<sup>16</sup>Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dover%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bailee Dover</span></a>
</div>
</div>
<sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fillmore%20TL%22%5BAuthor%5D" class="usa-link" aria-describedby="id27" aria-expanded="false"><span class="name western">Thomas L Fillmore</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Thomas L Fillmore</span></h3>
<div class="p">
<sup>17</sup>Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fillmore%20TL%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas L Fillmore</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foxall%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id28" aria-expanded="false"><span class="name western">McKenzie Foxall</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">McKenzie Foxall</span></h3>
<div class="p">
<sup>16</sup>Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foxall%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">McKenzie Foxall</span></a>
</div>
</div>
<sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hermann%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id29" aria-expanded="false"><span class="name western">Catherine E Hermann</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Catherine E Hermann</span></h3>
<div class="p">
<sup>24</sup>Division of Gynecologic Oncology, Department of Obstetrics and Gynecology, NYU Grossman School of Medicine, New York, NY, 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hermann%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Catherine E Hermann</span></a>
</div>
</div>
<sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiltke%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id30" aria-expanded="false"><span class="name western">Tara Hiltke</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Tara Hiltke</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiltke%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tara Hiltke</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id31" aria-expanded="false"><span class="name western">Galen Hostetter</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Galen Hostetter</span></h3>
<div class="p">
<sup>25</sup>Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Galen Hostetter</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%C4%99dryka%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id32" aria-expanded="false"><span class="name western">Marcin Jędryka</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Marcin Jędryka</span></h3>
<div class="p">
<sup>18</sup>Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%C4%99dryka%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcin Jędryka</span></a>
</div>
</div>
<sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D" class="usa-link" aria-describedby="id33" aria-expanded="false"><span class="name western">Scott D Jewell</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Scott D Jewell</span></h3>
<div class="p">
<sup>25</sup>Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D" class="usa-link"><span class="name western">Scott D Jewell</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id34" aria-expanded="false"><span class="name western">Isabelle Johnson</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Isabelle Johnson</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Isabelle Johnson</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kahn%20AG%22%5BAuthor%5D" class="usa-link" aria-describedby="id35" aria-expanded="false"><span class="name western">Andrea G Kahn</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Andrea G Kahn</span></h3>
<div class="p">
<sup>26</sup>Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kahn%20AG%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrea G Kahn</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ku%20AT%22%5BAuthor%5D" class="usa-link" aria-describedby="id36" aria-expanded="false"><span class="name western">Amy T Ku</span></a><div hidden="hidden" id="id36">
<h3><span class="name western">Amy T Ku</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ku%20AT%22%5BAuthor%5D" class="usa-link"><span class="name western">Amy T Ku</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id37" aria-expanded="false"><span class="name western">Chandan Kumar-Sinha</span></a><div hidden="hidden" id="id37">
<h3><span class="name western">Chandan Kumar-Sinha</span></h3>
<div class="p">
<sup>21</sup>Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>22</sup>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chandan Kumar-Sinha</span></a>
</div>
</div>
<sup>21,</sup><sup>22</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurzawa%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id38" aria-expanded="false"><span class="name western">Paweł Kurzawa</span></a><div hidden="hidden" id="id38">
<h3><span class="name western">Paweł Kurzawa</span></h3>
<div class="p">
<sup>27</sup>Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań</div>
<div class="p">
<sup>28</sup>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurzawa%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paweł Kurzawa</span></a>
</div>
</div>
<sup>27,</sup><sup>28</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazar%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id39" aria-expanded="false"><span class="name western">Alexander J Lazar</span></a><div hidden="hidden" id="id39">
<h3><span class="name western">Alexander J Lazar</span></h3>
<div class="p">
<sup>29</sup>Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div class="p">
<sup>30</sup>Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazar%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander J Lazar</span></a>
</div>
</div>
<sup>29,</sup><sup>30</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazcano%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id40" aria-expanded="false"><span class="name western">Rossana Lazcano</span></a><div hidden="hidden" id="id40">
<h3><span class="name western">Rossana Lazcano</span></h3>
<div class="p">
<sup>30</sup>Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazcano%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rossana Lazcano</span></a>
</div>
</div>
<sup>30</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D" class="usa-link" aria-describedby="id41" aria-expanded="false"><span class="name western">Jonathan T Lei</span></a><div hidden="hidden" id="id41">
<h3><span class="name western">Jonathan T Lei</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D" class="usa-link"><span class="name western">Jonathan T Lei</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id42" aria-expanded="false"><span class="name western">Yi Li</span></a><div hidden="hidden" id="id42">
<h3><span class="name western">Yi Li</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>31</sup>Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Li</span></a>
</div>
</div>
<sup>1,</sup><sup>31</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id43" aria-expanded="false"><span class="name western">Yuxing Liao</span></a><div hidden="hidden" id="id43">
<h3><span class="name western">Yuxing Liao</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuxing Liao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lih%20TSM%22%5BAuthor%5D" class="usa-link" aria-describedby="id44" aria-expanded="false"><span class="name western">Tung-Shing M Lih</span></a><div hidden="hidden" id="id44">
<h3><span class="name western">Tung-Shing M Lih</span></h3>
<div class="p">
<sup>7</sup>Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lih%20TSM%22%5BAuthor%5D" class="usa-link"><span class="name western">Tung-Shing M Lih</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20TT%22%5BAuthor%5D" class="usa-link" aria-describedby="id45" aria-expanded="false"><span class="name western">Tai-Tu Lin</span></a><div hidden="hidden" id="id45">
<h3><span class="name western">Tai-Tu Lin</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20TT%22%5BAuthor%5D" class="usa-link"><span class="name western">Tai-Tu Lin</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martignetti%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id46" aria-expanded="false"><span class="name western">John A Martignetti</span></a><div hidden="hidden" id="id46">
<h3><span class="name western">John A Martignetti</span></h3>
<div class="p">
<sup>8</sup>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martignetti%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">John A Martignetti</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masand%20RP%22%5BAuthor%5D" class="usa-link" aria-describedby="id47" aria-expanded="false"><span class="name western">Ramya P Masand</span></a><div hidden="hidden" id="id47">
<h3><span class="name western">Ramya P Masand</span></h3>
<div class="p">
<sup>32</sup>Department of Pathology &amp; Immunology, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masand%20RP%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramya P Masand</span></a>
</div>
</div>
<sup>32</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matkowski%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id48" aria-expanded="false"><span class="name western">Rafał Matkowski</span></a><div hidden="hidden" id="id48">
<h3><span class="name western">Rafał Matkowski</span></h3>
<div class="p">
<sup>18</sup>Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matkowski%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rafał Matkowski</span></a>
</div>
</div>
<sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McKerrow%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id49" aria-expanded="false"><span class="name western">Wilson McKerrow</span></a><div hidden="hidden" id="id49">
<h3><span class="name western">Wilson McKerrow</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McKerrow%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wilson McKerrow</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id50" aria-expanded="false"><span class="name western">Mehdi Mesri</span></a><div hidden="hidden" id="id50">
<h3><span class="name western">Mehdi Mesri</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mehdi Mesri</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monroe%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id51" aria-expanded="false"><span class="name western">Matthew E Monroe</span></a><div hidden="hidden" id="id51">
<h3><span class="name western">Matthew E Monroe</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monroe%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew E Monroe</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moon%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id52" aria-expanded="false"><span class="name western">Jamie Moon</span></a><div hidden="hidden" id="id52">
<h3><span class="name western">Jamie Moon</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moon%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jamie Moon</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moore%20RJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id53" aria-expanded="false"><span class="name western">Ronald J Moore</span></a><div hidden="hidden" id="id53">
<h3><span class="name western">Ronald J Moore</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moore%20RJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Ronald J Moore</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nestor%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id54" aria-expanded="false"><span class="name western">Michael D Nestor</span></a><div hidden="hidden" id="id54">
<h3><span class="name western">Michael D Nestor</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nestor%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael D Nestor</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id55" aria-expanded="false"><span class="name western">Chelsea Newton</span></a><div hidden="hidden" id="id55">
<h3><span class="name western">Chelsea Newton</span></h3>
<div class="p">
<sup>25</sup>Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chelsea Newton</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id56" aria-expanded="false"><span class="name western">Tatiana Omelchenko</span></a><div hidden="hidden" id="id56">
<h3><span class="name western">Tatiana Omelchenko</span></h3>
<div class="p">
<sup>33</sup>The Rockefeller University, New York, NY 10065, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tatiana Omelchenko</span></a>
</div>
</div>
<sup>33</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D" class="usa-link" aria-describedby="id57" aria-expanded="false"><span class="name western">Gilbert S Omenn</span></a><div hidden="hidden" id="id57">
<h3><span class="name western">Gilbert S Omenn</span></h3>
<div class="p">
<sup>34</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>35</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>36</sup>Department of Human Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>37</sup>School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D" class="usa-link"><span class="name western">Gilbert S Omenn</span></a>
</div>
</div>
<sup>34,</sup><sup>35,</sup><sup>36,</sup><sup>37</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Payne%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id58" aria-expanded="false"><span class="name western">Samuel H Payne</span></a><div hidden="hidden" id="id58">
<h3><span class="name western">Samuel H Payne</span></h3>
<div class="p">
<sup>38</sup>Department of Biology, Brigham Young University, Provo, UT 84602, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Payne%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">Samuel H Payne</span></a>
</div>
</div>
<sup>38</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petyuk%20VA%22%5BAuthor%5D" class="usa-link" aria-describedby="id59" aria-expanded="false"><span class="name western">Vladislav A Petyuk</span></a><div hidden="hidden" id="id59">
<h3><span class="name western">Vladislav A Petyuk</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petyuk%20VA%22%5BAuthor%5D" class="usa-link"><span class="name western">Vladislav A Petyuk</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D" class="usa-link" aria-describedby="id60" aria-expanded="false"><span class="name western">Ana I Robles</span></a><div hidden="hidden" id="id60">
<h3><span class="name western">Ana I Robles</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D" class="usa-link"><span class="name western">Ana I Robles</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id61" aria-expanded="false"><span class="name western">Henry Rodriguez</span></a><div hidden="hidden" id="id61">
<h3><span class="name western">Henry Rodriguez</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Henry Rodriguez</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D" class="usa-link" aria-describedby="id62" aria-expanded="false"><span class="name western">Kelly V Ruggles</span></a><div hidden="hidden" id="id62">
<h3><span class="name western">Kelly V Ruggles</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>39</sup>Division of Precision Medicine, Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D" class="usa-link"><span class="name western">Kelly V Ruggles</span></a>
</div>
</div>
<sup>4,</sup><sup>39</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rykunov%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id63" aria-expanded="false"><span class="name western">Dmitry Rykunov</span></a><div hidden="hidden" id="id63">
<h3><span class="name western">Dmitry Rykunov</span></h3>
<div class="p">
<sup>8</sup>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rykunov%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dmitry Rykunov</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D" class="usa-link" aria-describedby="id64" aria-expanded="false"><span class="name western">Sara R Savage</span></a><div hidden="hidden" id="id64">
<h3><span class="name western">Sara R Savage</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara R Savage</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schepmoes%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id65" aria-expanded="false"><span class="name western">Athena A Schepmoes</span></a><div hidden="hidden" id="id65">
<h3><span class="name western">Athena A Schepmoes</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schepmoes%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Athena A Schepmoes</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id66" aria-expanded="false"><span class="name western">Tujin Shi</span></a><div hidden="hidden" id="id66">
<h3><span class="name western">Tujin Shi</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tujin Shi</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id67" aria-expanded="false"><span class="name western">Zhiao Shi</span></a><div hidden="hidden" id="id67">
<h3><span class="name western">Zhiao Shi</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiao Shi</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id68" aria-expanded="false"><span class="name western">Jimin Tan</span></a><div hidden="hidden" id="id68">
<h3><span class="name western">Jimin Tan</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jimin Tan</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id69" aria-expanded="false"><span class="name western">Mason Taylor</span></a><div hidden="hidden" id="id69">
<h3><span class="name western">Mason Taylor</span></h3>
<div class="p">
<sup>38</sup>Department of Biology, Brigham Young University, Provo, UT 84602, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mason Taylor</span></a>
</div>
</div>
<sup>38</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id70" aria-expanded="false"><span class="name western">Mathangi Thiagarajan</span></a><div hidden="hidden" id="id70">
<h3><span class="name western">Mathangi Thiagarajan</span></h3>
<div class="p">
<sup>40</sup>Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mathangi Thiagarajan</span></a>
</div>
</div>
<sup>40</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id71" aria-expanded="false"><span class="name western">Joshua M Wang</span></a><div hidden="hidden" id="id71">
<h3><span class="name western">Joshua M Wang</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">Joshua M Wang</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weitz%20KK%22%5BAuthor%5D" class="usa-link" aria-describedby="id72" aria-expanded="false"><span class="name western">Karl K Weitz</span></a><div hidden="hidden" id="id72">
<h3><span class="name western">Karl K Weitz</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weitz%20KK%22%5BAuthor%5D" class="usa-link"><span class="name western">Karl K Weitz</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id73" aria-expanded="false"><span class="name western">Bo Wen</span></a><div hidden="hidden" id="id73">
<h3><span class="name western">Bo Wen</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bo Wen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id74" aria-expanded="false"><span class="name western">C M Williams</span></a><div hidden="hidden" id="id74">
<h3><span class="name western">C M Williams</span></h3>
<div class="p">
<sup>41</sup>Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">C M Williams</span></a>
</div>
</div>
<sup>41</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id75" aria-expanded="false"><span class="name western">Yige Wu</span></a><div hidden="hidden" id="id75">
<h3><span class="name western">Yige Wu</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA</div>
<div class="p">
<sup>12</sup>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yige Wu</span></a>
</div>
</div>
<sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id76" aria-expanded="false"><span class="name western">Matthew A Wyczalkowski</span></a><div hidden="hidden" id="id76">
<h3><span class="name western">Matthew A Wyczalkowski</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA</div>
<div class="p">
<sup>12</sup>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew A Wyczalkowski</span></a>
</div>
</div>
<sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yi%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id77" aria-expanded="false"><span class="name western">Xinpei Yi</span></a><div hidden="hidden" id="id77">
<h3><span class="name western">Xinpei Yi</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yi%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinpei Yi</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id78" aria-expanded="false"><span class="name western">Xu Zhang</span></a><div hidden="hidden" id="id78">
<h3><span class="name western">Xu Zhang</span></h3>
<div class="p">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xu Zhang</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id79" aria-expanded="false"><span class="name western">Rui Zhao</span></a><div hidden="hidden" id="id79">
<h3><span class="name western">Rui Zhao</span></h3>
<div class="p">
<sup>17</sup>Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Zhao</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mutch%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id80" aria-expanded="false"><span class="name western">David Mutch</span></a><div hidden="hidden" id="id80">
<h3><span class="name western">David Mutch</span></h3>
<div class="p">
<sup>42</sup>Washington University School of Medicine, Saint Louis, MO, 63110, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mutch%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Mutch</span></a>
</div>
</div>
<sup>42</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id81" aria-expanded="false"><span class="name western">Arul M Chinnaiyan</span></a><div hidden="hidden" id="id81">
<h3><span class="name western">Arul M Chinnaiyan</span></h3>
<div class="p">
<sup>21</sup>Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>22</sup>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>34</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Arul M Chinnaiyan</span></a>
</div>
</div>
<sup>21,</sup><sup>22,</sup><sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20RD%22%5BAuthor%5D" class="usa-link" aria-describedby="id82" aria-expanded="false"><span class="name western">Richard D Smith</span></a><div hidden="hidden" id="id82">
<h3><span class="name western">Richard D Smith</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20RD%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard D Smith</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D" class="usa-link" aria-describedby="id83" aria-expanded="false"><span class="name western">Alexey I Nesvizhskii</span></a><div hidden="hidden" id="id83">
<h3><span class="name western">Alexey I Nesvizhskii</span></h3>
<div class="p">
<sup>21</sup>Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>22</sup>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">
<sup>34</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexey I Nesvizhskii</span></a>
</div>
</div>
<sup>21,</sup><sup>22,</sup><sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id84" aria-expanded="false"><span class="name western">Pei Wang</span></a><div hidden="hidden" id="id84">
<h3><span class="name western">Pei Wang</span></h3>
<div class="p">
<sup>8</sup>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pei Wang</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiznerowicz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id85" aria-expanded="false"><span class="name western">Maciej Wiznerowicz</span></a><div hidden="hidden" id="id85">
<h3><span class="name western">Maciej Wiznerowicz</span></h3>
<div class="p">
<sup>43</sup>International Institute for Molecular Oncology, 60-203 Poznań, Poland</div>
<div class="p">
<sup>44</sup>Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań, Poland</div>
<div class="p">
<sup>28</sup>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiznerowicz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maciej Wiznerowicz</span></a>
</div>
</div>
<sup>43,</sup><sup>44,</sup><sup>28</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id86" aria-expanded="false"><span class="name western">Li Ding</span></a><div hidden="hidden" id="id86">
<h3><span class="name western">Li Ding</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA</div>
<div class="p">
<sup>12</sup>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Li Ding</span></a>
</div>
</div>
<sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D" class="usa-link" aria-describedby="id87" aria-expanded="false"><span class="name western">D R Mani</span></a><div hidden="hidden" id="id87">
<h3><span class="name western">D R Mani</span></h3>
<div class="p">
<sup>41</sup>Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D" class="usa-link"><span class="name western">D R Mani</span></a>
</div>
</div>
<sup>41</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id88" aria-expanded="false"><span class="name western">Hui Zhang</span></a><div hidden="hidden" id="id88">
<h3><span class="name western">Hui Zhang</span></h3>
<div class="p">
<sup>7</sup>Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hui Zhang</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anderson%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id89" aria-expanded="false"><span class="name western">Matthew L Anderson</span></a><div hidden="hidden" id="id89">
<h3><span class="name western">Matthew L Anderson</span></h3>
<div class="p">
<sup>14</sup>Division of Gynecologic Oncology, University of South Florida Morsani College of Medicine and Tampa General Hospital Cancer Institute, Tampa, FL 33606, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anderson%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew L Anderson</span></a>
</div>
</div>
<sup>14,</sup><sup>&amp;,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodland%20KD%22%5BAuthor%5D" class="usa-link" aria-describedby="id90" aria-expanded="false"><span class="name western">Karin D Rodland</span></a><div hidden="hidden" id="id90">
<h3><span class="name western">Karin D Rodland</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">
<sup>45</sup>Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97221, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodland%20KD%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin D Rodland</span></a>
</div>
</div>
<sup>6,</sup><sup>45,</sup><sup>&amp;,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id91" aria-expanded="false"><span class="name western">Bing Zhang</span></a><div hidden="hidden" id="id91">
<h3><span class="name western">Bing Zhang</span></h3>
<div class="p">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bing Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>&amp;,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id92" aria-expanded="false"><span class="name western">Tao Liu</span></a><div hidden="hidden" id="id92">
<h3><span class="name western">Tao Liu</span></h3>
<div class="p">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tao Liu</span></a>
</div>
</div>
<sup>6,</sup><sup>&amp;,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id93" aria-expanded="false"><span class="name western">David Fenyö</span></a><div hidden="hidden" id="id93">
<h3><span class="name western">David Fenyö</span></h3>
<div class="p">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Fenyö</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>&amp;,</sup><sup>#,</sup><sup>¶</sup>; <span class="collab">Clinical Proteomic Tumor Analysis Consortium</span>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A2">
<sup>2</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A3">
<sup>3</sup>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A4">
<sup>4</sup>Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div id="A5">
<sup>5</sup>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div id="A6">
<sup>6</sup>Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div id="A7">
<sup>7</sup>Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD, 21231, USA</div>
<div id="A8">
<sup>8</sup>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div id="A9">
<sup>9</sup>International Institute for Molecular Oncology, 20-203 Poznań, Poland</div>
<div id="A10">
<sup>10</sup>Postgraduate School of Molecular Medicine, Medical University of Warsaw, 02-091 Warsaw, Poland</div>
<div id="A11">
<sup>11</sup>Department of Medicine and Genetics, Siteman Cancer Center, Washington University in St. Louis, St. Louis, Missouri, 63110, USA</div>
<div id="A12">
<sup>12</sup>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, Missouri, 63108, USA</div>
<div id="A13">
<sup>13</sup>Department of Medicine, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A14">
<sup>14</sup>Division of Gynecologic Oncology, University of South Florida Morsani College of Medicine and Tampa General Hospital Cancer Institute, Tampa, FL 33606, USA</div>
<div id="A15">
<sup>15</sup>Office of Cancer Clinical Proteomics Research, National Cancer Institute, Rockville, MD 20850, USA</div>
<div id="A16">
<sup>16</sup>Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div id="A17">
<sup>17</sup>Environmental Molecular Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div>
<div id="A18">
<sup>18</sup>Wroclaw Medical University and Lower Silesian Oncology, Pulmonology and Hematology Center (DCOPIH), Wrocław, Poland</div>
<div id="A19">
<sup>19</sup>Department of Pathology and Laboratory Medicine, Mount Sinai Hospital and Laboratory Medicine and Pathobiology, University of Toronto, Toronto, M5G 1X5, Canada</div>
<div id="A20">
<sup>20</sup>Department of Pathology, NYU Grossman School of Medicine, New York, NY, 10016, USA</div>
<div id="A21">
<sup>21</sup>Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A22">
<sup>22</sup>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A23">
<sup>23</sup>Department of Obstetrics, Gynecology and Reproductive Science, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div>
<div id="A24">
<sup>24</sup>Division of Gynecologic Oncology, Department of Obstetrics and Gynecology, NYU Grossman School of Medicine, New York, NY, 10016, USA</div>
<div id="A25">
<sup>25</sup>Van Andel Research Institute, Grand Rapids, Michigan, 49503, USA</div>
<div id="A26">
<sup>26</sup>Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35249, USA</div>
<div id="A27">
<sup>27</sup>Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań</div>
<div id="A28">
<sup>28</sup>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div>
<div id="A29">
<sup>29</sup>Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div id="A30">
<sup>30</sup>Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div id="A31">
<sup>31</sup>Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A32">
<sup>32</sup>Department of Pathology &amp; Immunology, Baylor College of Medicine, Houston, TX 77030, USA</div>
<div id="A33">
<sup>33</sup>The Rockefeller University, New York, NY 10065, USA</div>
<div id="A34">
<sup>34</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A35">
<sup>35</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A36">
<sup>36</sup>Department of Human Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A37">
<sup>37</sup>School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA</div>
<div id="A38">
<sup>38</sup>Department of Biology, Brigham Young University, Provo, UT 84602, USA</div>
<div id="A39">
<sup>39</sup>Division of Precision Medicine, Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div>
<div id="A40">
<sup>40</sup>Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div>
<div id="A41">
<sup>41</sup>Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div>
<div id="A42">
<sup>42</sup>Washington University School of Medicine, Saint Louis, MO, 63110, USA</div>
<div id="A43">
<sup>43</sup>International Institute for Molecular Oncology, 60-203 Poznań, Poland</div>
<div id="A44">
<sup>44</sup>Heliodor Swiecicki Clinical Hospital in Poznan ul. Przybyszewskiego 49, 60-355 Poznań, Poland</div>
<div id="A45">
<sup>45</sup>Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97221, USA</div>
<div class="author-notes p">
<div class="fn" id="FN1">
<p id="P1">Secondary Author List:</p>
<p id="P2">Andrzej Antczak, Meenakshi Anurag, Thomas Bauer, Chet Birger, Michael J. Birrer, Melissa Borucki, Shuang Cai, Anna Calinawan, Steven A. Carr, Patricia Castro, Sandra Cerda, Daniel W. Chan, David Chesla, Marcin P. Cieslik, Sandra Cottingham, Rajiv Dhir, Marcin J. Domagalski, Brian J. Druker, Elizabeth Duffy, Nathan J. Edwards, Robert Edwards, Matthew J. Ellis, Jennifer Eschbacher, Mina Fam, Brenda Fevrier-Sullivan, Jesse Francis, John Freymann, Stacey Gabriel, Gad Getz, Michael A. Gillette, Andrew K. Godwin, Charles A. Goldthwaite, Pamela Grady, Jason Hafron, Pushpa Hariharan, Barbara Hindenach, Katherine A. Hoadley, Jasmine Huang, Michael M. Ittmann, Ashlie Johnson, Corbin D. Jones, Karen A. Ketchum, Justin Kirby, Toan Le, Avi Ma’ayan, Rashna Madan, Sailaja Mareedu, Peter B. McGarvey, Francesmary Modugno, Rebecca Montgomery, Kristen Nyce, Amanda G. Paulovich, Barbara L. Pruetz, Liqun Qi, Shannon Richey, Eric E. Schadt, Yvonne Shutack, Shilpi Singh, Michael Smith, Darlene Tansil, Ratna R. Thangudu, Matt Tobin, Ki Sung Um, Negin Vatanian, Alex Webster, George D. Wilson, Jason Wright, Kakhaber Zaalishvili, Zhen Zhang, Grace Zhao</p>
</div>
<div class="fn" id="FN2">
<p id="P3">AUTHOR CONTRIBUTIONS</p>
<p id="P4">Study Conception &amp; Design: Y.D., L.K., M.A.G., M.L.A., K.D.R., B.Z., T.L., D.F.; Performed Experiment or Data Collection: M.A.G., Y.H., Y.W., I.K., C.T., W.B., X.G., R.C.A., L.C., B.D., M.F., A.G.K., A.T.K., Y. Li, T. Lin; Computation &amp; Statistical Analysis: Y.D., L.K., B.R., R.H., R.J.L., W.L., I.J., W.M., T.O., V.A.P., D.R., S.R.S., J.T., J.M.W., Y. Wu, M.A.W., D.R.M.; Data Interpretation &amp; Biological Analysis: Y.D., L.K., M.A.G., Y.H., B.R., R.H., Y.W., I.K., R.J.L., C.T., W.L., E.A., J.B., L.C., R.K.C., A.C., T.D., E.G.D., D.D., K.D., S.M.D., P.D., T.L.F., C.E.H., T.H., G.H., M.J., S.D.J., I.J., C.K., P.K., A.J.L., R.L., J.T.L., Y. Liao, T.M.L., J.A.M., R.P.M., R.M., W.M., M.M., M.E.M., J.M., R.J.M., M.D.N., C.N., T.O., G.S.O., S.H.P., V.A.P., A.I.R., H.R., K.V.R., D.R., S.R.S., A.A.S., T.S., Z.S., J.T., M.Ta., M.Th., J.M.W., K.K.W., B.W., C.M.W., Y. Wu, M.A.W., X.Y., X.Z., R.Z., D.M., A.M.C., R.D.S., A.I.N., P.W., M.W., L.D., D.R.M., H.Z., M.L.A., K.D.R., B.Z., T.L., D.F.; Writing – Original Drafts: Y.D., L.K., M.A.G., Y.H., B.R., R.H., Y.W., I.K., R.J.L., C.T., M.W., D.R.M., H.Z., M.L.A., K.D.R., B.Z., T.L., D.F.; Writing – Review &amp; Editing: Y.D., L.K., M.A.G., Y.H., B.R., R.H., Y.W., I.K., R.J.L., C.T., W.B., W.L., X.G., E.A., R.C.A., J.B., L.C., R.K.C., A.C., T.D., E.G.D., D.D., K.D., S.M.D., P.D., B.D., T.L.F., M.F., C.E.H., T.H., G.H., M.J., S.D.J., I.J., A.G.K., A.T.K., C.K., P.K., A.J.L., R.L., J.T.L., Y. Li, Y. Liao, T.M.L., T. Lin, J.A.M., R.P.M., R.M., W.M., M.M., M.E.M., J.M., R.J.M., M.D.N., C.N., T.O., G.S.O., S.H.P., V.A.P., A.I.R., H.R., K.V.R., D.R., S.R.S., A.A.S., T.S., Z.S., J.T., M. Ta., M. Th., J.M.W., K.K.W., B.W., C.M.W., Y. Wu, M.A.W., X.Y., X.Z., R.Z., D.M., A.M.C., R.D.S., A.I.N., P.W., M.W., L.D., D.R.M., H.Z., M.L.A., K.D.R., B.Z., T.L., D.F.; Supervision: M.W., D.R.M., H.Z., M.L.A., K.D.R., B.Z., T.L., D.F.; Administration: M.L.A., K.D.R., B.Z., T.L., D.F.</p>
</div>
<div class="fn" id="FN3">
<sup>*</sup><p class="display-inline" id="P5">These authors contributed equally</p>
</div>
<div class="fn" id="FN4">
<sup>$</sup><p class="display-inline" id="P6">These authors contributed equally</p>
</div>
<div class="fn" id="FN5">
<sup>&amp;</sup><p class="display-inline" id="P7">Senior author</p>
</div>
<div class="fn" id="FN6">
<sup>¶</sup><p class="display-inline" id="P8">Lead contact</p>
</div>
<div class="fn" id="CR1">
<sup>#</sup><p class="display-inline">Correspondence: <span>mlander5@usf.edu</span> (M.L.A.), <span>rodland@ohsu.edu</span> (K.D.R.), <span>bing.zhang@bcm.edu</span> (B.Z.), <span>tao.liu@pnnl.gov</span> (T. Liu), <span>david@fenyolab.org</span> (D.F.)</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Issue date 2023 Sep 11.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC10631452&nbsp;&nbsp;NIHMSID: NIHMS1921121&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/37567170/" class="usa-link">37567170</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>The publisher's version of this article is available at <a href="https://doi.org/10.1016/j.ccell.2023.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Cancer Cell</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2 data-anchor-id="ABS1"><a id="ABS1-anchor" data-anchor-id="ABS1" class="usa-anchor"></a>SUMMARY</h2>
<p id="P9">We characterized a prospective endometrial carcinoma (EC) cohort containing 138 tumors and 20 enriched normal tissues using 10 different omics platforms. Targeted quantitation of two peptides can predict antigen processing and presentation machinery activity, and may inform patient selection for immunotherapy. Association analysis between MYC activity and metformin treatment in both patients and cell lines suggests a potential role for metformin treatment in non-diabetic patients with elevated MYC activity. <em>PIK3R1</em> in-frame indels are associated with elevated AKT phosphorylation and increased sensitivity to AKT inhibitors. <em>CTNNB1</em> hotspot mutations are concentrated near phosphorylation sites mediating pS45-induced degradation of β-catenin, which may render Wnt-FZD antagonists ineffective. Deep learning accurately predicts EC subtypes and mutations from histopathology images, which may be useful for rapid diagnosis. Overall, this study identified molecular and imaging markers that can be further investigated to guide patient stratification for more precise treatment of EC.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Endometrial cancer, proteogenomics, target assays, deep learning, <em>PIK3R1</em>, <em>CTNNB1</em>, Metformin, CPTAC</p></section></section><section class="abstract" id="ABS2"><h2 data-anchor-id="ABS2"><a id="ABS2-anchor" data-anchor-id="ABS2" class="usa-anchor"></a>eTOC Blurb</h2>
<p id="P10">Dou et al. report the characterization of a prospective endometrial carcinoma (EC) cohort using 10 different omics platforms. They identify potential molecular and imaging markers for guiding patient stratification for more precise treatment of EC.</p></section><section class="abstract" id="ABS3"><h2 data-anchor-id="ABS3"><a id="ABS3-anchor" data-anchor-id="ABS3" class="usa-anchor"></a>Graphical Abstract</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/45db48794e6a/nihms-1921121-f0008.jpg" loading="lazy" height="530" width="640" alt="graphic file with name nihms-1921121-f0008.jpg"></p></section><section id="S1"><h2 class="pmc_sec_title" data-anchor-id="S1"><a id="S1-anchor" data-anchor-id="S1" class="usa-anchor"></a>INTRODUCTION</h2>
<p id="P12">Endometrial carcinoma (EC) is the most common gynecologic malignancy in developed nations<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a>–<a href="#R3" class="usa-link" aria-describedby="R3" aria-expanded="false">3</a></sup>. Over the past decade, the incidence of EC has steadily increased (~1% annually<sup><a href="#R1" class="usa-link" aria-describedby="R1" aria-expanded="false">1</a>,<a href="#R4" class="usa-link" aria-describedby="R4" aria-expanded="false">4</a></sup>). Curiously, EC-specific mortality has steadily worsened over the past 10 years, despite the fact that postmenopausal vaginal bleeding is a common early symptom and the majority of early stage, well-differentiated EC can be cured by hysterectomy<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup>. Worsening EC outcomes have been attributed to the increasing incidence of aggressive EC histotypes, particularly among Black and Hispanic women<sup><a href="#R5" class="usa-link" aria-describedby="R5" aria-expanded="false">5</a></sup>. For high-risk EC patients, adjuvant radiotherapy and cytotoxic chemotherapy have been shown to decrease the likelihood of disease recurrences and improve survival. However, failure rates associated with these treatments remain unacceptably high<sup><a href="#R6" class="usa-link" aria-describedby="R6" aria-expanded="false">6</a></sup>. In other situations, the ability of radiotherapy or cytotoxic chemotherapy to improve outcomes for EC patients at intermediate risk of disease progression or recurrence remains much less clear. In part, this may depend on the patterns of driver mutations present<sup><a href="#R7" class="usa-link" aria-describedby="R7" aria-expanded="false">7</a></sup>. Effective therapies capable of curing EC once widely metastatic are lacking.</p>
<p id="P13">Comprehensive genomic profiling by The Cancer Genome Atlas (TCGA) has led to a better understanding of key genomic drivers of EC, including the identification of four EC subtypes defined solely by their genomic signatures: POLE ultramutated, microsatellite instability hypermutated (MSI-H), copy-number low (CNV-L), and copy-number high (CNV-H)<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>. Associations between these subtypes and clinical outcomes are now being evaluated. Integrated genomic and proteomic characterization of EC by the Clinical Proteomic Tumor Analysis Consortium (CPTAC) revealed additional oncogenic signaling pathways and regulatory mechanisms, and identified specific defects in antigen presentation machinery (APM) which appear important for determining clinical responses to immune checkpoint inhibition (ICI) therapy<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>.</p>
<p id="P14">Here, we present the results of a comprehensive proteogenomic analysis of a prospectively curated set of 138 EC tumors and 20 specimens enriched for normal endometrium from healthy donors. Analysis of this independent cohort, incorporating pre-existing EC tumor and cell line cohorts, has not only confirmed published findings from our recent exploratory studies, but also provided biological insights relevant to potential therapeutic strategies. We also evaluated the utility of using machine learning for parsing clinically-relevant genomic features based solely on the analysis of histological slides routinely created for clinical evaluation.</p></section><section id="S2"><h2 class="pmc_sec_title" data-anchor-id="S2"><a id="S2-anchor" data-anchor-id="S2" class="usa-anchor"></a>RESULTS</h2>
<section id="S3"><h3 class="pmc_sec_title">Proteogenomic landscape of an independent EC cohort</h3>
<p id="P15">We characterized the proteogenomic landscape of 138 EC tumors (including 119 endometrioid, 13 serous, 3 clear cell) with matched blood normal samples and 20 enriched normal endometrium samples from healthy donors (<a href="#SD2" class="usa-link">Table S1</a>) using 10 omic platforms including whole-genome sequencing (WGS), whole-exome sequencing (WES), methylation array, total RNA sequencing (RNA-seq), microRNA sequencing, targeted proteomics, global proteomics, phosphoproteomics, acetylproteomics, and glycoproteomics (<a href="#S17" class="usa-link">STAR methods</a>). A total of 10,135 proteins and 25,300 phosphorylation, 5,556 acetylation, and 6,513 glycosylation sites were quantified (<a href="#F1" class="usa-link">Figure 1A</a>), using rigorous quality control metrics (<a href="#SD1" class="usa-link">Figure S1A</a>–<a href="#SD1" class="usa-link">I</a>). Principal component analysis (PCA) separated the tumors and normal samples based on global proteome, phosphoproteome, or glycoproteome data, but not on acetylproteomics data, with no batch effect observed in the tandem mass tag (TMT) plexes. (<a href="#SD1" class="usa-link">Figure S1E</a>–<a href="#SD1" class="usa-link">H</a>). Significant overlaps were observed for quantifiable global proteomics and phosphoproteomics features between the exploratory cohort<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup> and this independent cohort. However, more acetylation sites were quantified in this cohort due to platform improvement (<a href="#F1" class="usa-link">Figure 1B</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1: Proteogenomic landscape of the independent EC cohort.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/0dc244ef1336/nihms-1921121-f0001.jpg" loading="lazy" height="1050" width="803" alt="Figure 1:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P145">(A) Multi-omic data availability in the independent EC cohort. The main heatmap shows each patient per column and the side heatmap (right) shows each enriched normal sample per column.</p>
<p id="P146">(B) Identified and quantified proteomic features across the exploratory (Exp) and independent (Ind) cohorts.</p>
<p id="P147">(C) Oncoplot showing the most frequently mutated genes in the independent cohort identified by whole genome sequencing. Each column is a patient. Side table (right) compares mutation frequencies per gene for the two CPTAC cohorts and TCGA cohort.</p>
<p id="P148">(D) Focal-level copy number variations (CNVs) across the genome (x-axis) versus G-score (y-axis), which is the Frequency x Amplitude of the CNV. Significant amplifications are shown in red and deletions are shown in blue, with annotated peaks containing known tumor drivers.</p>
<p id="P149">(E) Prioritization of CNV drivers across the two CPTAC cohorts.</p>
<p id="P150">(F) Barplot showing pathways enriched from CNV drivers analysis (E).</p>
<p id="P151">(G) Scatter plot of protein level tumor/normal between the two CPTAC cohorts.</p>
<p id="P152">(H) Cis/trans effects of somatic mutations (y-axis) on protein expression (x-axis). See also <a href="#SD1" class="usa-link">Figure S1</a> and <a href="#SD2" class="usa-link">Tables S1</a> and <a href="#SD3" class="usa-link">S2</a>.</p></figcaption></figure><p id="P16">Top mutated genes in this cohort included <em>PTEN</em>, <em>ARID1A</em>, <em>PIK3CA</em>, <em>PIK3R1</em>, and <em>CTNNB1</em> whose mutation frequencies were similar to those in the exploratory cohort (<a href="#F1" class="usa-link">Figure 1C</a>). Mutation frequencies were lower in the TCGA EC cohort, due to a higher proportion of serous tumors<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>. The mutation frequencies were highly correlated across the three cohorts (<a href="#SD1" class="usa-link">Figure S1J</a>). Tumors were classified into four genomic subtypes: 6 POLE, 47 MSI-H, 66 CNV-L, 16 CNV-H tumors and 3 unclassified because of missing WGS-based CNV data (<a href="#F1" class="usa-link">Figure 1C</a> and <a href="#SD1" class="usa-link">Figure S1K</a>–<a href="#SD1" class="usa-link">L</a>; <a href="#SD2" class="usa-link">Table S1</a>). One tumor was identified as both POLE and CNV-H by genomic data, highlighting the high degree of heterogeneity in certain EC tumors that likely requires careful consideration in treatment decisions. In general, these results indicate that mutational features of this current independent cohort are similar to previous EC studies.</p>
<p id="P17">WGS-based CNV analysis identified significant focal amplifications and deletions. Several known tumor drivers were located within the amplified regions (e.g., <em>SETDB1</em>, <em>ECT2</em>, <em>ATAD2</em>, <em>GRB7</em>, and <em>CCNE1</em>) (<a href="#F1" class="usa-link">Figure 1D</a>; <a href="#SD3" class="usa-link">Table S2</a>). We prioritized CNV driver genes by their correlation with mRNA and protein levels (<a href="#F1" class="usa-link">Figure 1E</a>) and filtered them by comparing protein levels in tumors vs enriched normal tissues, which identified 351 and 237 potential CNV drivers from this and the exploratory cohorts, respectively, of which a significant overlap of 88 genes was found between them (<a href="#F1" class="usa-link">Figure 1E</a>; <a href="#SD3" class="usa-link">Table S2</a>; <strong>p&lt;2.2e-16</strong>), and these were significantly enriched in proliferation and cell cycle related processes (<a href="#F1" class="usa-link">Figure 1F</a>).</p>
<p id="P18">We identified 1,292 up- and 1,488 down-regulated proteins when tumors were compared to enriched normal tissues (FDR &lt; 0.01, |Log2FC| &gt; 0.25). Significantly differentially expressed proteins were highly overlapping between the independent cohort and the exploratory cohort (<a href="#F1" class="usa-link">Figure 1G</a>). Interestingly, two cancer/testis antigens, <em>PBK</em> and <em>KIF2C</em>, were found to be significantly upregulated in both cohorts<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>. The <em>cis-</em> and <em>trans-</em>effects on the proteome of somatic driver mutations were similar across cohorts (<a href="#F1" class="usa-link">Figure 1H</a>). For example, <em>CTNNB1</em> mutations were significantly positively correlated with higher CTNNB1 and LEF1 protein levels. We confirmed that <em>MLH1</em> and <em>HOX</em> family genes were silenced by DNA methylation in this independent cohort (<a href="#SD1" class="usa-link">Figure S1M</a>). circRNAs show stronger positive correlations than that between their host genes (<a href="#SD1" class="usa-link">Figure S1N</a>) and miR-200c-3p significantly negatively correlated with QKI, a circRNA regulator, protein level (<a href="#SD1" class="usa-link">Figure S1O</a>). These results suggest that many findings at the multi-omics level are highly consistent between the two CPTAC EC studies.</p></section><section id="S4"><h3 class="pmc_sec_title">
<em>PIK3R1</em> in-frame indels are potential markers of <em>AKT</em> inhibition response</h3>
<p id="P19">Previous studies have shown that the PI3K-AKT pathway is frequently altered in endometrioid tumors<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a>–<a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a></sup> with <em>PTEN</em>, <em>PIK3CA</em>, and <em>PIK3R1</em> being the most frequently mutated genes in the pathway. <em>PTEN</em> is mutated in ~80% of endometrioid tumors and mostly co-occurs with <em>PIK3CA</em> and <em>PIK3R1</em> mutations, which are mutually exclusive (<a href="#F2" class="usa-link">Figure 2A</a>). Although <em>PIK3CA</em> is commonly mutated in many cancer types<sup><a href="#R10" class="usa-link" aria-describedby="R10" aria-expanded="false">10</a></sup>, <em>PIK3R1</em> mutations, especially in-frame indels of <em>PIK3R1</em>, are more frequent in EC (<a href="#F1" class="usa-link">Figures 1C</a>, <a href="#SD1" class="usa-link">S2A</a>). Interestingly, these in-frame indels are tightly clustered at the two ends of the P85/iSH2 domain (<a href="#SD1" class="usa-link">Figure S2B</a>), which are structurally close to each other in a region of PIK3R1 that interacts with PIK3CA (<a href="#F2" class="usa-link">Figure 2B</a>). Of note, this clustering pattern is not observed for truncations and missense mutations (<a href="#SD1" class="usa-link">Figure S2C</a>–<a href="#SD1" class="usa-link">D</a>). This suggests a potentially distinct function of <em>PIK3R1</em> in-frame indels from other <em>PIK3R1</em> variants.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2: <em>PIK3R1</em> in-frame indels show induction of activating AKT phosphosites.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/9b31af5d0c0d/nihms-1921121-f0002.jpg" loading="lazy" height="1050" width="960" alt="Figure 2:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P153">(A) <em>PTEN</em>, <em>PIK3CA</em>, and <em>PIK3R1</em> mutations across independent (Ind), exploratory (Exp), and TCGA cohorts. P-values derived from Fisher’s exact test.</p>
<p id="P154">(B) 3D structure of PI3K complex. PIK3CA protein is colored in green, PIK3R1 protein is colored in blue, and location of <em>PIK3R1</em> in-frame variants is shown in red.</p>
<p id="P155">(C-D) Boxplots comparing <em>PIK3R1</em> mRNA (C) and protein levels (D) between <em>PIK3R1</em> variants across the independent (Ind), exploratory (Exp), and TCGA cohorts. P-values derived from Student’s t-test</p>
<p id="P156">(E) Survival analysis of TCGA <em>PTEN</em> mutated EC patients harboring <em>PIK3R1</em> and <em>PIK3CA</em> variants. P-values derived from log-rank test.</p>
<p id="P157">(F) Boxplots comparing AKT1-pT308 levels between <em>PIK3R1</em> and <em>PIK3CA</em> variants in the independent cohort. P-values derived from Student’s t-test.</p>
<p id="P158">(G-H) Boxplots comparing TCGA RPPA data for AKT-pT308 (G) and AKT-pS473 (H) levels between <em>PIK3R1</em> and <em>PIK3CA</em> variants. P-values derived from Student’s t-test.</p>
<p id="P159">(I) Western blot for AKT pT308 and pS473 in HEC-151 cells with CRISPR-Cas9 created T576 deletion.</p>
<p id="P160">(J) Boxplots comparing DepMap EC cell lines’ response to MK-2206. P-values derived from Student’s t-test.</p>
<p id="P161">(K) Schematic showing consequences of <em>PIK3R1</em> in-frame variants.</p>
<p id="P162">Boxplots: Box portion represents Interquartile range (IQR), midline corresponds to the median, and whiskers range from the minimum (bottom) and maximum (top) variability outside the first and third quartiles (Q1 and Q3). Outliers are shown as points above whiskers. See also <a href="#SD1" class="usa-link">Figure S2</a> and <a href="#SD4" class="usa-link">Table S3</a>.</p></figcaption></figure><p id="P20">To test this hypothesis, we examined <em>cis</em>-effects of different <em>PIK3R1</em> mutations on mRNA and protein levels (<a href="#F2" class="usa-link">Figure 2C</a>–<a href="#F2" class="usa-link">D</a>). In-frame indels were associated with comparable PIK3R1 protein (p85α subunit of PI3K) levels (<a href="#F2" class="usa-link">Figure 2D</a>; <a href="#SD4" class="usa-link">Table S3</a>), while truncating mutations were associated with lower protein levels compared to the WT <em>PIK3R1</em> group<a href="#R11" class="usa-link" aria-describedby="R11" aria-expanded="false">11</a>. Since PIK3R1 is a suppressor of AKT phosphorylation, we expected that lower levels of PIK3R1 by truncating mutations would result in activation of the PI3K-AKT pathway and worse clinical outcomes. Surprisingly, <em>PIK3R1</em> in-frame indels were associated with worse survival compared to both <em>PIK3R1</em> truncating and <em>PIK3CA</em> hotspot mutations in the TCGA cohort, despite our finding that they do not result in reduced levels of PIK3R1 protein (<a href="#F2" class="usa-link">Figure 2E</a>). The Both WT group (<em>PIK3CA</em> WT and <em>PIK3R1</em> WT) had similar survival with the <em>PIK3R1</em> in-frame indel group. This may be due to the fact that this cohort is significantly enriched for CNV-high ECs and contains a lower proportion of <em>POLE-</em>mutated cancers (<a href="#SD1" class="usa-link">Figure S2E</a>).</p>
<p id="P21">We hypothesized that PIK3R1 proteins with in-frame indels abrogate the ability of PIK3R1 to suppress AKT phosphorylation, leading to elevated levels of phosphorylated AKT1. To examine this hypothesis, we first examined the relationships between AKT1 phosphorylation level and <em>PTEN</em>, <em>PIK3CA</em>, and <em>PIK3R1</em> mutation status. As expected, <em>PTEN</em> WT samples have significantly lower AKT1-T308 phosphorylation levels than <em>PTEN</em> mutated samples<sup><a href="#R12" class="usa-link" aria-describedby="R12" aria-expanded="false">12</a></sup> (<a href="#F2" class="usa-link">Figure 2F</a>). Moreover, samples with in-frame indels had higher levels of AKT1-T308 phosphorylation compared with cancers carrying <em>PIK3CA</em> hotspot or <em>PIK3R1</em> truncating mutations (<a href="#F2" class="usa-link">Figure 2F</a>). Since this phosphosite was not identified in the exploratory cohort, TCGA Reverse Phase Protein Array (RPPA) data was used to further confirm our hypothesis that AKT-T308 phosphorylation is higher in tumors harboring <em>PIK3R1</em> in-frame indels and the lowest in <em>PTEN</em> WT tumors (<a href="#F2" class="usa-link">Figure 2G</a>). We observed the same trend for another well studied phosphorylation site, AKT-S473 (<a href="#F2" class="usa-link">Figure 2H</a>). Moreover, AKT protein levels were not consistently altered by <em>PIK3CA</em> and <em>PIK3R1</em> mutations in the independent cohort or TCGA (<a href="#SD1" class="usa-link">Figure S2F</a>–<a href="#SD1" class="usa-link">G</a>). These results are consistent with a previous study that showed <em>PIK3R1</em> in-frame indels were able to more efficiently promote AKT phosphorylation than other mutations in EC cell lines<sup><a href="#R13" class="usa-link" aria-describedby="R13" aria-expanded="false">13</a></sup>. To further confirm the relation between <em>PIK3R1</em> in-frame indels and AKT-T308 and AKT-S473 phosphorylation, we created an in-frame deletion, T576Del, in an EC cell line, HEC151, by CRISPR-Cas9 (<a href="#SD1" class="usa-link">Figure S2H</a>). As expected, both AKT-T308 and AKT-S473 were upregulated at phosphorylation level (<a href="#F2" class="usa-link">Figure 2I</a>). Taken together, these results suggest that <em>PIK3R1</em> in-frame indels are associated with worse outcomes in <em>PTEN-</em>mutated EC patients and contribute to altered PI3K-AKT pathway by increasing AKT1 phosphorylation.</p>
<p id="P22">Various drugs have been developed to target the PI3K-AKT pathway, but many have failed in clinical trials<sup><a href="#R14" class="usa-link" aria-describedby="R14" aria-expanded="false">14</a>,<a href="#R15" class="usa-link" aria-describedby="R15" aria-expanded="false">15</a></sup>. For example, results from a clinical trial with MK-2206, a well-established AKT inhibitor targeting AKT-S473, failed to improve survival in EC patients with <em>PIK3CA</em> mutated EC<sup><a href="#R16" class="usa-link" aria-describedby="R16" aria-expanded="false">16</a></sup>. In light of our analyses described above, we next evaluated whether <em>PIK3R1</em> in-frame indels may be a better marker of clinical responses to AKT inhibition than <em>PIK3CA</em> mutations using data from EC cell lines from DepMap<sup><a href="#R17" class="usa-link" aria-describedby="R17" aria-expanded="false">17</a></sup>. As expected, EC cell lines with <em>PIK3R1</em> in-frame indels and mutated <em>PTEN</em> were significantly more sensitive to MK-2206 compared to lines with <em>PIK3CA</em> hotspot mutation, <em>PIK3R1</em> truncation, or WT <em>PTEN</em> (<a href="#F2" class="usa-link">Figure 2J</a>; <a href="#SD4" class="usa-link">Table S3</a>). This result was also observed for two additional AKT inhibitors, GDC-0068 (<a href="https://clinicaltrials.gov/ct2/show/NCT02465060" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02465060</a>, a phase II study of multiple cancer types including EC with inclusion criteria for tumors with <em>AKT</em> mutations) and GSK2110183 (<a href="#SD1" class="usa-link">Figure S2I</a>–<a href="#SD1" class="usa-link">J</a>; <a href="#SD4" class="usa-link">Table S3</a>). In summary, our results suggest the <em>PIK3R1</em> in-frame indels with mutated <em>PTEN</em> could be a potential biomarker for response to AKT inhibition (<a href="#F2" class="usa-link">Figure 2K</a>).</p></section><section id="S5"><h3 class="pmc_sec_title">Selected Reaction Monitoring (SRM) assay accurately predicts APM status</h3>
<p id="P23">Effective cellular APM has been reported as an important factor of response to ICI<sup><a href="#R18" class="usa-link" aria-describedby="R18" aria-expanded="false">18</a>–<a href="#R20" class="usa-link" aria-describedby="R20" aria-expanded="false">20</a></sup> independent of tumor-mutation burden (TMB)<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>. By grouping tumors according to their TMB and APM levels, we confirmed the finding of high APM variation in TMB-H tumors<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup> (<a href="#SD1" class="usa-link">Figure S3A</a>;<a href="#SD5" class="usa-link">Table S4</a>). The TMB-H/APM-L group were enriched for <em>JAK1</em> mutations (p = 1.05e-3, Fisher’s test) and had lower JAK-STAT, APM, and HLA protein levels (<a href="#SD1" class="usa-link">Figure S3B</a>). TMB-H/APM-H tumors were immune hot with activated CD8+ T cells (<a href="#F3" class="usa-link">Figure 3A</a>). Moreover <em>JAK1</em> variants were very rare in TMB-L tumors (<a href="#SD5" class="usa-link">Table S4</a>) suggesting <em>JAK1</em> mutations are associated with TMB-H tumors. This potential interaction between <em>JAK1</em> mutations and TMB-H status warrants further examination in other EC cohorts.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3: Selective reaction monitoring (SRM) assay for antigen presentation machinery (APM) status.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/4ad1bd5d5f4a/nihms-1921121-f0003.jpg" loading="lazy" height="1050" width="732" alt="Figure 3:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P163">(A) Heatmap of immune subgroups and related pathways derived from ssGSEA pathway scores using protein level as input for the independent cohort. Each column corresponds to individual samples in the main heatmap and the mean score for each immune subgroup in the side (right) panel. Each row represents a pathway.</p>
<p id="P164">(B-C) Histograms showing the frequency of correlations between SRM peptides from the same genes (B) and SRM peptides with TMT-based protein levels (C) for the independent cohort.</p>
<p id="P165">(D-E) Heatmaps showing SRM-based peptide quantitation of JAK-STAT and selected APM proteins in the exploratory (D) and independent (E) cohorts. Columns correspond to individual samples and rows represent peptides. <em>JAK1</em> mutations enriched for TMB-H/APM-L groups indicated by asterisks (* p &lt; 0.05, ** p &lt; 0.01). P-values determined by Fisher’s exact test.</p>
<p id="P166">(F) ROC curves showing model performances of classifiers using two peptides, PSMB10-LPFTALGSGQDAALAVLEDR and PSMB9-VSAGEAVVNR.</p>
<p id="P167">(G) ROC curves showing model performances of the ORFlog classifier on the independent cohort, comparing top N peptides used per model.</p>
<p id="P168">See also <a href="#SD1" class="usa-link">Figure S3</a> and <a href="#SD5" class="usa-link">Table S4</a>.</p></figcaption></figure><p id="P24">Unlike TMB which is already used as a clinical biomarker<sup><a href="#R21" class="usa-link" aria-describedby="R21" aria-expanded="false">21</a>,<a href="#R22" class="usa-link" aria-describedby="R22" aria-expanded="false">22</a></sup>, there are no clinical biomarkers to determine APM status. The previous approach for computing APM scores relies on proteome-wide measurements, and is not suitable as a diagnostic assay<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a>,<a href="#R23" class="usa-link" aria-describedby="R23" aria-expanded="false">23</a></sup>. To develop targeted proteomic assays, we selected a panel comprising 17 JAK-STAT and APM proteins, and 51 other immune related proteins, including two peptides for each protein (<a href="#SD5" class="usa-link">Table S4</a>, <a href="#SD1" class="usa-link">Figure S3C</a>). In both exploratory and independent cohorts, we observed a strong correlation between peptides originating from the same protein, as well as between the targeted assay and TMT measurements, particularly for proteins with relatively higher levels (<a href="#F3" class="usa-link">Figures 3B</a>–<a href="#F3" class="usa-link">C</a>, <a href="#SD1" class="usa-link">S3D</a>–<a href="#SD1" class="usa-link">E</a>). The targeted assays verified downregulation of APM proteins that were previously identified as downregulated in the TMB-H/APM-L group through global proteomics (<a href="#F3" class="usa-link">Figures 3D</a>–<a href="#F3" class="usa-link">E</a>). By utilizing these peptide measurements, we constructed machine learning models that accurately predicted the APM status, achieving an AUC of 0.961 using only two peptides: PSMB9−VSAGEAVVNR and PSMB10−LPFTALGSGQDAALAVLEDR (<a href="#F3" class="usa-link">Figures 3F</a>–<a href="#F3" class="usa-link">G</a>, <a href="#SD1" class="usa-link">S3F</a>–<a href="#SD1" class="usa-link">J</a>). We applied this SRM assay to immunotherapy-treated MSI-H EC tumors (10 sensitive and 2 resistant) <strong>(</strong><a href="#SD5" class="usa-link">Table S4</a>). The Immunotherapy-sensitive group had higher peptide levels than the TMB-H/APM-L group (<a href="#SD1" class="usa-link">Figure S3K</a>–<a href="#SD1" class="usa-link">L</a>), without reaching statistical significance, possibly as a result of the small cohort size. Therefore, this simple SRM assay should be further validated to evaluate its suitabillity for predicting APM status in a clinical setting.</p></section><section id="S6"><h3 class="pmc_sec_title">MYC activity is the target and potential biomarker for metformin treatment</h3>
<p id="P25">Obesity and type 2 diabetes (T2D) are well recognized risk factors for EC<sup><a href="#R24" class="usa-link" aria-describedby="R24" aria-expanded="false">24</a>–<a href="#R26" class="usa-link" aria-describedby="R26" aria-expanded="false">26</a></sup>. In a recent metaanalysis, metformin treatment was shown to improve overall survival of EC patients with T2D<sup><a href="#R27" class="usa-link" aria-describedby="R27" aria-expanded="false">27</a></sup>. A number of mechanisms by which metformin potentially improves EC outcomes include its ability to modulate autophagy<sup><a href="#R28" class="usa-link" aria-describedby="R28" aria-expanded="false">28</a></sup>, inhibit mitochondrial respiration<sup><a href="#R29" class="usa-link" aria-describedby="R29" aria-expanded="false">29</a></sup>, modify epigenetic signatures<sup><a href="#R30" class="usa-link" aria-describedby="R30" aria-expanded="false">30</a></sup>, and directly suppress cell growth<sup><a href="#R31" class="usa-link" aria-describedby="R31" aria-expanded="false">31</a>,<a href="#R32" class="usa-link" aria-describedby="R32" aria-expanded="false">32</a></sup>. However, the mechanisms by which metformin impacts EC and potentially improves patient outcomes remain unclear.</p>
<p id="P26">To examine molecular determinants underlying metformin response, we integrated clinical phenotypes with multi-omics data. RNA-seq-based pathway analysis showed significantly lower proliferation-related MYC activity in metformin-treated patients (<a href="#F4" class="usa-link">Figures 4A</a>, <a href="#SD1" class="usa-link">S4A</a>; <a href="#SD2" class="usa-link">Table S1</a>). A similar relationship was observed where MYC protein level decreased when Ishikawa EC cell lines were treated with metformin<sup><a href="#R33" class="usa-link" aria-describedby="R33" aria-expanded="false">33</a></sup>. We also analyzed responses of 22 EC cell lines (8 sensitive, 14 insensitive) from DepMap<sup><a href="#R17" class="usa-link" aria-describedby="R17" aria-expanded="false">17</a></sup> (<a href="#SD1" class="usa-link">Figure S4B</a>) and found that metformin-sensitive cell lines had significantly higher MYC activity than resistant ones (<a href="#F4" class="usa-link">Figure 4B</a>). Collectively, we interpreted these observations to indicate that MYC potentially serves as a biomarker for metformin responses in EC. To further evaluate this hypothesis, we sought to determine whether downregulation of MYC activity could be a direct consequence of metformin treatment<sup><a href="#R34" class="usa-link" aria-describedby="R34" aria-expanded="false">34</a></sup>. We found MYC activity to be downregulated in several CMap signatures of metformin-treated cell lines (<a href="#F4" class="usa-link">Figure 4C</a>; <a href="#SD6" class="usa-link">Table S5</a>). Analyzing two metformin-sensitive and two insensitive EC cell lines from DepMap (<a href="#SD1" class="usa-link">Figure S4B</a>), we observed higher MYC protein levels in the metformin-sensitive cell lines (<a href="#F4" class="usa-link">Figure 4D</a>), but not mRNA levels (<a href="#SD1" class="usa-link">Figure S4C</a>). Use of a previously validated reporter assay confirmed higher levels of MYC activity in the two cell lines with greater MYC protein levels (<a href="#SD1" class="usa-link">Figure S4D</a>). Cell lines with higher MYC activity were more sensitive to metformin treatment (<a href="#F4" class="usa-link">Figure 4E</a>). Moreover, mRNA expression of <em>MYC</em> and multiple MYC targets were suppressed in response to metformin treatment (<a href="#SD1" class="usa-link">Figure S4E</a>; <a href="#SD6" class="usa-link">Table S5</a>). In general, decreases in putative MYC-regulated transcripts occurred earlier and to a greater degree in the two cell lines with greatest sensitivity to metformin. These experiments demonstrate that metformin treatment significantly decreases MYC activity, with a trend toward increased sensitivity in cell lines which express high levels of MYC. These results suggest that an important mechanism by which metformin decreases EC growth is by directly downregulating MYC activity.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4: Metformin may target MYC in EC.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/466d9624686c/nihms-1921121-f0004.jpg" loading="lazy" height="1050" width="685" alt="Figure 4:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P169"> (A) MYC Targets V2 enrichment plots from pathway analysis comparing metformin-treated versus untreated patients with Type2 Diabetes (T2D) in the independent cohort.</p>
<p id="P170">(B) MYC Targets V2 enrichment plot from pathway analysis comparing metformin-sensitive versus insensitive EC cell lines from DepMap.</p>
<p id="P171">(C) Volcano plot of MYC Targets V2 pathway scores (x-axis) versus -log10(FDR) (y-axis) for CMAP metformin treatment signatures.</p>
<p id="P172">(D) Western blot showing MYC expression in four EC cell lines.</p>
<p id="P173">(E) Dose-response curves of EC cell lines treated with metformin at increasing concentrations (x-axis).</p>
<p id="P174">(F-G) Survival analysis of TCGA MSI-H (F) and CNV-L (G) tumors with high and low MYC activity. (H) Scatter plot of MYC activity (y-axis) versus MYC IHC score (x-axis). P-values derived from log-rank test.</p>
<p id="P175">(I) Heatmap of all endometrioid tumors in the independent cohort, sorted by MYC activity (top panel) and grouped by diabetes and metformin treatment status. Side boxplots (right) compare MYC activity (top) and BMI (bottom) across the diabetes/treatment groups. MYC IHC scores (third panel from top) are shown from samples with IHC data available. P-values derived from Student’s t-test (boxplots) and Spearman correlation (left panel).</p>
<p id="P176">Boxplots: Box portion represents Interquartile range (IQR), midline corresponds to the median, and whiskers range from the minimum (bottom) and maximum (top) variability outside the first and third quartiles (Q1 and Q3). Outliers are shown as points above whiskers.</p>
<p id="P177">See also <a href="#SD1" class="usa-link">Figure S4</a> and <a href="#SD6" class="usa-link">Table S5</a>.</p></figcaption></figure><p id="P27">Next, we evaluated the relationship between T2D, EC outcomes, MYC expression and metformin exposure. We found EC patients with T2D whose tumors were characterized by high levels of MYC protein have significantly worse overall survival (OS) (<a href="#SD1" class="usa-link">Figure S4F</a>) than MYC-low patients (<a href="#SD6" class="usa-link">Table S5</a>). Interestingly, high levels of MYC were also associated with worse OS in both MSI-H and CNV-L subsets of EC (<a href="#F4" class="usa-link">Figure 4F</a>–<a href="#F4" class="usa-link">G</a>), but not in serous tumors (<a href="#SD1" class="usa-link">Figure S4G</a>). Although the MYC protein is not quantifiable by global proteomics, we found MYC activity to be significantly higher in MYC-high tumors in TCGA RPPA data (<a href="#SD1" class="usa-link">Figure S4H</a>). MYC activity also significantly correlated with MYC Immunohistochemistry (IHC) score (<a href="#F4" class="usa-link">Figure 4H</a>; <a href="#SD6" class="usa-link">Table S5</a>), but not <em>MYC</em> mRNA levels (<a href="#SD1" class="usa-link">Figure S4I</a>). Therefore, we used inferred MYC activity as a readout of MYC protein level to further examine relationships between MYC and EC genotypes/phenotypes. We divided patients into 3 groups, non-diabetic, diabetic untreated, and diabetic treated, and sorted patients by their MYC activity from high to low (<a href="#F4" class="usa-link">Figure 4I</a>). As predicted by our earlier results, metformin-treated tumors had significantly lower levels of MYC activity when compared to untreated tumors from T2D patients (p = 4.49e-2, t test). A similar trend was observed when we re-evaluated our earlier exploratory cohort (<a href="#F4" class="usa-link">Figure 4I</a> and <a href="#SD1" class="usa-link">S4J</a>). Of note, we found that MYC activity in ECs from non-diabetic patients was similar or even higher than those observed in diabetic patients not previously treated with metformin (<a href="#F4" class="usa-link">Figure 4I</a> and <a href="#SD1" class="usa-link">S4J</a>). We also found that MYC activity was negatively correlated with BMI in this cohort (p = 1.26e-2, R = −0.24).</p>
<p id="P28">Collectively, these observations suggest that high levels of MYC activity may be a biomarker for identifying EC patients most likely to benefit from metformin treatment, including non-diabetic and non-obese patients.</p></section><section id="S7"><h3 class="pmc_sec_title">Comprehensive pathway analysis reveals EC tumors separate by high and low transporter activity</h3>
<p id="P29">Calculating the activation score for 8,615 pathways using EC protein levels (<a href="#SD7" class="usa-link">Table S6</a>), revealed the transporters axis to be the most significant subtype across EC tumors (<a href="#SD1" class="usa-link">Figure S5A</a>–<a href="#SD1" class="usa-link">B</a>). The transporter axis is composed of transmembrane proteins including ATP-binding cassette transporters, ATPases, ion channels and solute carriers, all of which mediate exchange of ions, metabolites and nutrients between the intra- and extracellular space. EC cancers with high transporter scores featured elevated levels of transporters, biosynthetic and metabolic pathways for lipids and glycans, synthesis of fatty acyl-CoAs, insulin, triglyceride, steroid and cholesterol, and downregulation of proinflammatory cytokine signaling responsible for activation of CD4<sup>+</sup> and CD8+ T lymphocytes. The transporter-high tumors were associated with a higher <em>CTNNB1</em> mutation frequency (<a href="#SD1" class="usa-link">Figure S5A</a>, p = 0.004), resulting in stabilization of β-catenin and transcriptional activation of target genes involved in glycosylation and in molecular exchange required for lipid metabolic reprogramming and cancer progression. Among numerous β-catenin transcriptional targets were immune checkpoints<sup><a href="#R35" class="usa-link" aria-describedby="R35" aria-expanded="false">35</a>,<a href="#R36" class="usa-link" aria-describedby="R36" aria-expanded="false">36</a></sup> contributing to an immunosuppressive environment.</p></section><section id="S8"><h3 class="pmc_sec_title">
<em>CTNNB1</em> hotspot mutations inhibit DKK-induced degradation</h3>
<p id="P30">Hotspot mutations in exon 3 of <em>CTNNB1</em> (β-catenin) (<a href="#SD1" class="usa-link">Figure S5C</a>), potential drivers of tumorigenesis<sup><a href="#R37" class="usa-link" aria-describedby="R37" aria-expanded="false">37</a></sup>, were significantly enriched in CNV-L tumors (<a href="#F5" class="usa-link">Figure 5A</a>, <a href="#SD1" class="usa-link">S5D</a>–<a href="#SD1" class="usa-link">E</a>) and with a higher frequency in CNV-L than in TMB-H tumors (<a href="#SD1" class="usa-link">Figure S5F</a>). <em>CTNNB1</em> hotspot mutations led to significant upregulation of Wnt-β-catenin signaling at both RNA and protein levels and downregulation of several immune related pathways (<a href="#F5" class="usa-link">Figure 5B</a>), and these were also true across the whole cohort (<a href="#SD7" class="usa-link">Table S6</a>). Wnt signaling proteins including CTNNB1 and LEF1 were upregulated in <em>CTNNB1</em> hotspot-mutated tumors despite DKK4, an inhibitor of the Wnt pathway<sup><a href="#R38" class="usa-link" aria-describedby="R38" aria-expanded="false">38</a></sup>, being the most significantly upregulated protein (<a href="#F5" class="usa-link">Figure 5C</a>, <a href="#SD7" class="usa-link">Table S6</a>). Because <em>CTNNB1</em> hotspot mutations occur at key phosphorylation sites or neighboring amino acids, they may block phosphorylation-dependent β-catenin degradation<sup><a href="#R39" class="usa-link" aria-describedby="R39" aria-expanded="false">39</a></sup>. Consistent with this hypothesis, S45 phosphorylation was significantly downregulated in hotspot-mutated tumors, whereas phosphosites distant from the hotspot mutations were mostly upregulated (<a href="#F5" class="usa-link">Figure 5D</a>, <a href="#SD7" class="usa-link">Table S6</a>). We also found significantly upregulated cell proliferation and transporter activity and downregulated total immune score in tumors with <em>CTNNB1</em> hotspot mutations, consistent with existing knowledge<sup><a href="#R37" class="usa-link" aria-describedby="R37" aria-expanded="false">37</a></sup>.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5: <em>CTNNB1</em> hotspot mutations block DKK induced degradation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/f7c34d82f012/nihms-1921121-f0005.jpg" loading="lazy" height="1050" width="747" alt="Figure 5:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P178"> (A) Mosaic plot showing distribution of <em>CTNNB1</em> mutations across CNV-L tumors versus all other tumors in the independent cohort. P-value determined from Chi-Square Test.</p>
<p id="P179">(B) Scatter plot of pathway Normalized Enrichment Scores (NES) comparing <em>CTNNB1</em> hotspot mutant versus WT CNV-L tumors at the protein (x-axis) and RNA (y-axis) levels in the independent cohort. Points with FDR &lt; 0.01 at both protein and RNA levels are colored in red, RNA only are in green, protein only are in blue, and neither are in gray.</p>
<p id="P180">(C) Volcano plot of protein log2 fold change (x-axis) between <em>CTNNB1</em> hotspot and WT CNV-L tumors versus −log10 FDR (y-axis) determined by Student’s T-test. Points with FDR &lt; 0.01 and log2 fold change &lt; −0.5 or &gt; 0.5 are shown in blue and red, respectively.</p>
<p id="P181">(D) Heatmap showing mRNA, protein, and phosphoprotein values for CTNNB1, LEF1 protein, MYC activity score, Immune Score, and Transporters Score across CNV-L tumors with and without <em>CTNNB1</em> hotspot mutations. Side panel showing boxplots (right) compares mutants versus WT tumors. P-values determined by Wilcoxon Rank Sum Test.</p>
<p id="P182">(E) Schematic depicting proposed downstream implications of hotspot mutations in <em>CTNNB1</em>.</p>
<p id="P183">(F) ROC curves of Lasso regression models predicting <em>CTNNB1</em> hotspot mutation status using exploratory protein data as training and independent protein data as testing. Models vary by which samples (all tumors or just CNV-L tumors) and which proteins (all proteins or only Wnt- β-catenin pathway proteins) were used.</p>
<p id="P184">(G) Venn diagram showing top 10 proteins selected by regression analysis per model.</p>
<p id="P185">Boxplots: Box portion represents Interquartile range (IQR), midline corresponds to the median, and whiskers range from the minimum (bottom) and maximum (top) variability outside the first and third quartiles (Q1 and Q3). Outliers are shown as points above whiskers.</p>
<p id="P186">See also <a href="#SD1" class="usa-link">Figure S5</a> and <a href="#SD7" class="usa-link">Table S6</a>.</p></figcaption></figure><p id="P31"><em>CTNNB1</em> hotspot mutations are associated with higher risk of recurrence<sup><a href="#R40" class="usa-link" aria-describedby="R40" aria-expanded="false">40</a>,<a href="#R41" class="usa-link" aria-describedby="R41" aria-expanded="false">41</a></sup> and therefore useful for making treatment decisions. Currently, sequencing is used to identify these mutations, but sequencing is not always available or covered by insurance for low-grade tumors<sup><a href="#R42" class="usa-link" aria-describedby="R42" aria-expanded="false">42</a>,<a href="#R43" class="usa-link" aria-describedby="R43" aria-expanded="false">43</a></sup>, revealing an unmet need for a more accessible diagnostic tool. To screen for potential protein biomarkers, we used proteomic data from the exploratory cohort to train regression models and tested their performance using the independent cohort. The best performing model used CNV-L tumors and Wnt proteins (AUC = 0.99) (<a href="#F5" class="usa-link">Figure 5F</a>–<a href="#F5" class="usa-link">G</a>, <a href="#SD7" class="usa-link">Table S6</a>). Using IHC for CTNNB1, LEF1, and MYC, we validated elevated protein levels and MYC activity in hotspot-mutated tumors. Since EC tumors derive from the epithelium, membrane CTNNB1 stains positive in all cells. However, nuclear CTNNB1 was significantly higher in CNV-L hotspot-mutated tumors compared to WT (<a href="#SD1" class="usa-link">Figure S5G</a>), implying a higher occurrence of oncogenic activity in these tumors.</p>
<p id="P32">Together, our data suggest that <em>CTNNB1</em> hotspot mutations block phosphorylation induced β-catenin degradation and may render Wnt-FZD antagonists ineffective for ECs with the hotspot mutations (<a href="#F5" class="usa-link">Figure 5E</a>), and that protein-based assays, such as IHC, can be used to detect hotspot mutation status, suggesting possibilities beyond bulk sequencing for clinical testing.</p></section><section id="S9"><h3 class="pmc_sec_title">Deep learning models using histopathology slides predict subtypes and mutation status</h3>
<p id="P33">The CNV-H molecular subtype of EC, which includes some high-grade endometrioid (<a href="#SD1" class="usa-link">Figure S6A</a>) and all histologically serous (<a href="#SD1" class="usa-link">Figure S6B</a>) cases, has the worst patient outcomes<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>. In contrast, POLE tumors, typically ultra-mutated with favorable outcomes, are not currently identifiable by human pathologists without sequencing the tumors (<a href="#SD1" class="usa-link">Figure S6C</a>). Here, we trained convolutional neural network models on H&amp;E stained slides from TCGA and CPTAC exploratory cohorts to predict the molecular subtypes, histological subtypes, and common mutations of EC<sup><a href="#R44" class="usa-link" aria-describedby="R44" aria-expanded="false">44</a></sup>, and tested them on the independent cohort (<a href="#F6" class="usa-link">Figure 6A</a>). We qualitatively confirmed that our models were able to successfully distinguish CNV-H and non-CNV-H samples (<a href="#F6" class="usa-link">Figure 6B</a>–<a href="#F6" class="usa-link">C</a>). Notably, a rare dual CNV-H and POLE tumor was also picked out by the imaging model, which aligned to both molecular subtypes (<a href="#SD1" class="usa-link">Figure S6D</a>). tSNE plots showing CNV-H and POLE predictions of this sample show separation of tiles based on their subtype prediction, highlighting the heterogeneity in this tumor. Using similar tSNE plots for samples with either CNV-H or POLE subtype, but not both, the model does not predict high CNV-H scores for POLE tiles (not shown). The highest AUROC obtained from our POLE model was 0.925, which was higher than the 0.89 previously reported<sup><a href="#R44" class="usa-link" aria-describedby="R44" aria-expanded="false">44</a></sup>. We then tested and re-trained with an independent cohort from NYU, which resulted in AUROCs ranging 0.6–0.8 (<a href="#F6" class="usa-link">Figure 6A</a>, <a href="#SD8" class="usa-link">Table S7</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Figure 6: Deep learning models successfully classify molecular features.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/46609b2f5177/nihms-1921121-f0006.jpg" loading="lazy" height="1050" width="738" alt="Figure 6:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P187">(A) Barplots showing the mean AUROC per model from internal training data split tests (trained on TCGA and exploratory cohorts) and independent tests (tested on independent cohort plus NYU cohort for POLE predictions). Bar color is determined by AUROC value coming from internal or independent tests, and outlines denote if the top performing model architecture comes from the internal or independent test.</p>
<p id="P188">(B-C) tSNE plots where each point is a tile, colored by predicted CNV-H score (B) and true CNV-H label (C).</p>
<p id="P189">(D) Distribution of chromosome 1q copy number status across all tumors in the independent cohort, grouped by genomic and histologic subtypes.</p>
<p id="P190">(E) Boxplots of xCell immune scores (y-axis) comparing tumors with 1q gain versus no gain (xaxis). P-values determined by Wilcoxon Rank Sum Test.</p>
<p id="P191">(F) Volcano plot of differentially expressed glycopeptides in tumors with 1q gain versus no gain. X-axis shows log2 fold change and the y-axis shows -log10 FDR, determined by Student’s T-test.</p>
<p id="P192">(G) Heatmap of <em>PARP1</em> multi-omic levels in samples with and without 1q gain.</p>
<p id="P193">(H) Boxplots of olaparib (PARP-inhibitor) response in DepMap EC cell lines with and without <em>PARP1</em> amplification. P-values determined by Wilcoxon Rank Sum Test.</p>
<p id="P194">Boxplots: Box portion represents Interquartile range (IQR), midline corresponds to the median, and whiskers range from the minimum (bottom) and maximum (top) variability outside the first and third quartiles (Q1 and Q3). Outliers are shown as points above whiskers.</p>
<p id="P195">See also <a href="#SD1" class="usa-link">Figure S6</a> and <a href="#SD8" class="usa-link">Table S7</a>.</p></figcaption></figure></section><section id="S10"><h3 class="pmc_sec_title">1q amplification leads to increased glycoprotein levels and may be a biomarker for <em>PARP</em>-inhibition treatment</h3>
<p id="P34">Increased CNV, a hallmark of cancer, is correlated with poorer outcomes, increased immune evasion, and lowered response to immunotherapies<sup><a href="#R45" class="usa-link" aria-describedby="R45" aria-expanded="false">45</a>,<a href="#R46" class="usa-link" aria-describedby="R46" aria-expanded="false">46</a></sup>. While the CNV-H subtype shows the highest concentration of CNV (<a href="#SD1" class="usa-link">Figure S1K</a>), there are patterns of chromosome-specific alterations that occur across all EC subtypes, such as gains of 1q, the most common arm level CNV in EC (<a href="#F6" class="usa-link">Figure 6D</a>, <a href="#SD3" class="usa-link">Table S2</a>). Tumors with 1q gain have a significantly lowered overall immune score, CD8<sup>+</sup> naive T cell score, myeloid dendritic cell score, and microenvironment score (<a href="#F6" class="usa-link">Figure 6E</a>, <a href="#SD3" class="usa-link">Table S2</a>). Using TCGA data, we saw a significant decrease in disease-free survival among tumors with 1q gain (<a href="#SD1" class="usa-link">Figure S6G</a>, p=2.9e-3). Interestingly, pathways associated with protein glycosylation were significantly enriched in samples with 1q gain (<a href="#SD1" class="usa-link">Figure S6E</a>). Using glycoproteomics data, we observed upregulation of glycopeptides for polymeric immunoglobulin receptor (<em>PIGR</em>) and other glycosylated proteins in tumors with 1q gain (<a href="#F6" class="usa-link">Figure 6F</a>). Focusing on glycosyltransferases or glycosidases encoded on 1q, we found 7 proteins involved in glycosylation (<a href="#SD1" class="usa-link">Figure S6F</a>) including poly [ADP-ribose] polymerase 1 (<em>PARP1</em>), a key protein involved in DNA damage repair<sup><a href="#R47" class="usa-link" aria-describedby="R47" aria-expanded="false">47</a></sup>. We observed increased <em>PARP1</em> copy number, RNA, protein, phosphorylation, and acetylation levels, not only in tumors with 1q gain, but also in tumors with focal amplifications of <em>PARP1</em> (<a href="#F6" class="usa-link">Figure 6G</a>). To test if there is any correlation between <em>PARP1</em> amplification and drug response to PARP inhibitors (PARP-i), we used DepMap and observed that EC cell lines with <em>PARP1</em> amplification showed increased sensitivity to the PARP-i olaparib (p=0.04) (<a href="#F6" class="usa-link">Figure 6H</a>). Sensitivity to niraparib and talazoparib, two other PARP-i in clinical use, also appeared to be greater in samples with <em>PARP1</em> gain, although these differences were not statistically significant (<a href="#SD1" class="usa-link">Figure S6H</a>). Gain of <em>PARP1</em> may be an evolutionary advantage for EC tumors and could be a possible biomarker for treatment with PARP-i.</p></section><section id="S11"><h3 class="pmc_sec_title">Multi-omic clustering uncovers a <em>CTNNB1</em> hotspot mutation enriched CNV-L subgroup and a 1q gain MSI-H subgroup</h3>
<p id="P35">Multi-omic clustering identified four clusters based on CNV, mRNA, protein, phosphosite, and acetylation site levels, which loosely follow the traditional genomic subtypes (<a href="#F7" class="usa-link">Figure 7A</a>). All CNV-H tumors fall into Cluster 1 with significant enrichment of <em>TP53</em>-mutated tumors (p=4.54E-06) and a proliferation signature (<a href="#F7" class="usa-link">Figure 7B</a>, <a href="#SD8" class="usa-link">Table S7</a>). Clusters 2 and 3 were enriched for the CNV-L subtype (p=0.0006, p=0.001), with Cluster 3 characterized by <em>CTNNB1</em> hotspot mutations (p=4.53E-07), arm 1q gain (p=7.39E-05), upregulated transporter activity (p=0.005) and downregulated immune pathways. Cluster 4 was enriched for MSI-H (p=1.70E-08) and TMB-H, and includes a majority of POLE tumors. Clusters 2 and 3 have complementary pathway signatures with Cluster 2 showing upregulation of immune-related pathways. Cluster 4 has enhanced activity in cell-cycle, metabolic and DNA damage related pathways in addition to mTOR and TNFα signaling. Comparison of only MSI-H samples among three clusters reveals an enrichment of 1q gain in Cluster 3 (p=0.002) and enrichment of <em>ELMSAN1</em> mutation in Cluster 4 (p=7.05E-05), suggesting subgroups within the MSI-H subtype.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Figure 7: Multi-omic and glycoproteomic NMF clustering separates samples into 4 clusters.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10631452_nihms-1921121-f0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9170/10631452/d1653673ae11/nihms-1921121-f0007.jpg" loading="lazy" height="1050" width="734" alt="Figure 7:"></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P196">(A) Heatmap of all tumors in the independent cohort, separated by multi-omic NMF clusters. Panels show histologic subtypes, histologic grade, genomic subtypes, APM class, transporter status, mutation status of selected genes, 1q copy number status, immune score, and corresponding glyco-NMF cluster assignment.</p>
<p id="P197">(B) Heatmap of mean ssGSEA pathway enrichments per cluster.</p>
<p id="P198">(C) Heatmap of glycopeptide levels for all independent cohort tumors, separated by glyco-peptide derived NMF clusters. Side panel (left) annotates types of glycans.</p>
<p id="P199">(D) Dot plots of glyco-enzyme levels between tumor versus normal across glyco-clusters, separated by glyco-enzyme function: Precursor (left), Trimming (middle), and Capping (right). Red denotes positive log2 fold change (higher in tumor) while blue indicates negative log2 fold change (higher in normal). Size of dots is determined by -log10 p-value, which comes from Student’s T-test.</p>
<p id="P200">(E) Dot plot comparing tumor and normal samples’ glycosylated kinases in the PI3K-AKT pathway by glycans (x-axis) and corresponding peptide (y-axis). Red denotes positive log2 fold change (higher in tumor) while blue indicates negative log2 fold change (higher in normal). Size of dots is determined by -log10 p-value, which comes from Student’s T-test.</p>
<p id="P201">See also <a href="#SD1" class="usa-link">Figure S7</a>.</p></figcaption></figure></section><section id="S12"><h3 class="pmc_sec_title">Glycopeptides and the corresponding unmodified protein levels are significantly different between tumor and normal samples</h3>
<p id="P36">Altered glycosylation in tumors is known to be associated with increased metastatic potential and immune evasion<sup><a href="#R48" class="usa-link" aria-describedby="R48" aria-expanded="false">48</a></sup>. In our data, oligomannose (HM) occupies 19% of N-linked glycopeptides, and ~70% of the rest were complex glycans with fucose (Fuc) (26%), sialic acids (Sia) (17%), or both (27%) (<a href="#SD1" class="usa-link">Figure S7A</a>). Comparing tumor and normal samples, we observed 121 upregulated and 296 downregulated glycopeptides (<a href="#SD1" class="usa-link">Figure S7B</a>), and ~80% of the glycopeptides involved in the lysosome pathway were upregulated, while ~60% of glycopeptides involved in the PI3K-AKT pathway were downregulated. The change in levels between tumor and normal samples of N-linked glycopeptides and the corresponding unmodified protein levels were correlated (r=0.71) (<a href="#SD1" class="usa-link">Figure S7C</a>–<a href="#SD1" class="usa-link">D</a>). Interestingly, three of the downregulated genes involved in trimming were <em>MAN1C1</em> for trimming mannose and <em>MGAT4C</em> and <em>MGAT5B</em> for branching with GlcNAc, which may result in reduced synthesis of complex-type N-linked oligosaccharides, especially for bisected tetra-antennary complex N-glycans.</p></section><section id="S13"><h3 class="pmc_sec_title">NMF clustering of glycopeptides groups tumors into four subtypes that are distinct from the multi-omics clusters</h3>
<p id="P37">Unsupervised NMF clustering of glycopeptides levels<sup><a href="#R49" class="usa-link" aria-describedby="R49" aria-expanded="false">49</a></sup> divides the tumors into four subtypes (<a href="#F7" class="usa-link">Figure 7C</a>). Clusters 1 and 3 were mostly associated with the complex glycans Sia and Fuc, Cluster 2 with Sia, and Cluster 4 with HM. Chromosome 1q gain was most frequent in Cluster 1, <em>TP53</em> mutations were frequent in Clusters 2 and 4, and <em>CTNNB1</em> mutations were rare in Cluster 2. The levels of glycopeptides enriched in each cluster and the corresponding protein correlated with each other (<a href="#SD1" class="usa-link">Figure S7E</a>). Proteins associated with oligosaccharide precursor synthesis were downregulated in all four clusters (<a href="#F7" class="usa-link">Figure 7D</a>). The glyco-clusters were, in general, not related to genomic subtypes except for a slight enrichment of the CNV-L group in Cluster 3 (<a href="#SD1" class="usa-link">Figure S7F</a>), providing a complementary subtyping of EC tumors.</p>
<p id="P38">Focusing on the PI3K-AKT pathway differences between tumor and normal samples (<a href="#F7" class="usa-link">Figure 7E</a>), we observed 15 glycopeptides from 7 kinases. Most of these (12/15) were upregulated in tumors, including multiple glycosylation sites from FLT1. A significantly downregulated oligomannose structure was observed on EGFR, which indicates that glycosylation of EGFR could be altered in tumor samples in the trimming step, which may influence the function of EGFR in tumors.</p></section></section><section id="S14"><h2 class="pmc_sec_title" data-anchor-id="S14"><a id="S14-anchor" data-anchor-id="S14" class="usa-anchor"></a>DISCUSSION</h2>
<p id="P39">This study is a comprehensive proteogenomic investigation of 138 prospectively collected EC tumors and 20 enriched normal endometrium samples, and provides a valuable community resource that can be used for both hypothesis generation and testing. Findings initially reported in Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup> were confirmed in this independent cohort, including the functional validation of the effects of frequently observed gene mutations, proteomic markers of clinical and genomic tumor subgroups, and the impact of defects in antigen presentation on patterns of immune infiltrates in TMB-H tumors. The concordance between the two studies was remarkably high, demonstrating the robustness of our proteogenomic strategy, including its sample collection protocols and analytical methods. Of note, the correlation between measured transcript and protein levels was in the range typically seen for tumors (median correlation across all genes of 0.48), once again demonstrating that, unsurprisingly, transcript levels are often not accurate for predicting protein levels for a given gene, as transcripts and protein can have largely different degradation rates and regulatory mechanisms.</p>
<p id="P40">Our current study expands our previous work in EC in several ways, including the exploration of additional data types: glycoproteomics, contributing further insights into EC tumor biology, and targeted mass spectrometry-based assays, which provide a path to the development of clinical assays.</p>
<p id="P41">ICI therapy provides a promising treatment option for many tumor types including EC. High TMB has been shown to predict response to ICI treatment in some cases, and pembrolizumab has been approved by the FDA to treat MSI-H cancers. However, high failure rates are observed among EC patients triaged to ICI therapy by currently available biomarkers. For example, recent prospective studies evaluating the clinical efficacy of dostarlimab in EC have demonstrated response rates no higher than 45% in MSI-H ECs<sup><a href="#R50" class="usa-link" aria-describedby="R50" aria-expanded="false">50</a></sup>. Our previous study indicated that defects in antigen presentation may serve as a key rate-limiting factor which prevents variant peptide antigens from being processed and presented properly, and renders the host immune system unable to mount a tumoricidal response<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>. To address this potential clinical need, we developed an SRM assay for two peptides that can be used to predict APM status with high accuracy (AUC=0.96). Given its high accuracy, we believe this assay is amenable to being validated for use in clinical laboratories.</p>
<p id="P42">A second key finding of our current study is that MYC activity can potentially be used as a biomarker for triaging EC patients to metformin treatment. Over the past decade, a multitude of clinical studies have evaluated metformin as a strategy for EC prevention and treatment. For reasons that remain unclear, outcomes of these studies, which have typically focused on evaluating the impact of metformin in patients impacted by T2D and obesity, remain conflicted<sup><a href="#R51" class="usa-link" aria-describedby="R51" aria-expanded="false">51</a>–<a href="#R53" class="usa-link" aria-describedby="R53" aria-expanded="false">53</a></sup>. We found that high levels of MYC activity are associated with worse survival for both CNV-L and MSI-H ECs. Despite controversy whether sufficient circulating levels of metformin can be achieved clinically to biologically impact a cancer, our data clearly indicate that real-world metformin treatment results in lower levels of MYC activity in EC patients with T2D across both our current and previous EC cohorts. Lastly, we have uncovered an association between MYC activity in EC and patient BMI, with higher levels of MYC activity observed in non-obese patients. Thus, the response of ECs to metformin may be tempered by a complex relationship between MYC activity and BMI. In the future, it will be important to further parse these relationships as well as to directly test our therapeutic hypothesis that high MYC activity can serve as a biomarker for metformin response even in non-diabetic EC patients. Given the relatively limited correlation (cor = 0.38) we observed between MYC activity and MYC IHC score, this may require development of an alternative diagnostic.</p>
<p id="P43">Despite rapid clinical uptake of somatic tumor sequencing, traditional histology, being faster and cheaper, is still widely used to determine cancer treatments. In part, this is because the cost of genomic technologies has largely limited their use to advanced stage and metastatic disease, e.g., most insurances do not cover sequencing of low-grade EC. We therefore explored the possibility that features derived from traditional histopathology images could be used to predict the molecular features of a tumor. The high degree of accuracy achieved with our algorithm as a predictive model for some EC subtypes and mutations, specifically POLE-mutated EC, is promising but needs to be tested on a larger cohort.</p>
<p id="P44">Our study demonstrates the ability of proteogenomic analysis to increase our understanding of EC tumor biology and to generate new hypotheses. We have highlighted a few examples of integrative analysis across the omics data modalities that can provide insights with potential clinical applications.</p></section><section id="S15"><h2 class="pmc_sec_title" data-anchor-id="S15"><a id="S15-anchor" data-anchor-id="S15" class="usa-anchor"></a>STAR METHODS</h2>
<section id="S16"><h3 class="pmc_sec_title">RESOURCE AVAILABILITY</h3>
<section id="S17"><h4 class="pmc_sec_title">Lead Contact</h4>
<p id="P45">Further information and requests for resources should be directed to the <a href="#S19" class="usa-link">Lead Contact</a>, David Fenyö (<span>david@fenyolab.org</span>).</p></section><section id="S18"><h4 class="pmc_sec_title">Materials availability</h4>
<p id="P46">This study did not generate new unique reagents.</p></section></section><section id="S19"><h3 class="pmc_sec_title">DATA AND CODE AVAILABILITY</h3>
<p id="P47">Clinical and proteomic (raw MS files and processed data files) data reported in this paper, including both exploratory and independent CPTAC datasets, can be accessed via the Proteomic Data Commons (PDC) at: <a href="https://pdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pdc.cancer.gov/</a> (PDC000125, PDC000126, PDC000226). Genomic and transcriptomic data files for both CPTAC cohorts (phs001287) and TCGA cohort (phs000178) can be accessed via Genomic Data Commons (GDC) at: <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/projects/CPTAC-3</a>. The DepMap datasets can be accessed via the DepMap portal: <a href="https://depmap.org/portal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/</a>. Processed CPTAC data for both cohorts used in this publication can also be found in the PDC, the Python package called ‘cptac’ (<a href="https://pypi.org/project/cptac/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pypi.org/project/cptac/</a>, install via pip) to allow programmatic access and LinkedOmics via <a href="http://www.linkedomics.org/data_download/CPTAC-UCEC-independent/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.linkedomics.org/data_download/CPTAC-UCEC-independent/</a>
<sup><a href="#R64" class="usa-link" aria-describedby="R64" aria-expanded="false">64</a></sup>. Histological and radiological images for both CPTAC cohorts (10.7937/k9/tcia.2018.3r3juisw) and TCGA cohort (10.7937/k9/tcia.2016.gkj0zwac) can be accessed via Imaging Data Commons (IDC) at <a href="https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec</a>, and The Cancer Imaging Archive at <a href="https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=33948263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=33948263</a>. Deidentified digitized H&amp;E slides from NYU reported in this paper will be shared by the <a href="#S19" class="usa-link">lead contact</a> upon request.</p></section><section id="S20"><h3 class="pmc_sec_title">EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS</h3>
<section id="S21"><h4 class="pmc_sec_title">Patient Selection</h4>
<p id="P48">The tumor, normal tissue, and whole blood samples used in this manuscript were prospectively collected between May 2016 and May 2019 for the CPTAC program from 5 different countries. Informed consent was collected for each patient and all patients consented to donate their specimens to CPTAC. Each sample collection site has an internally approved IRB. None of the sites use any central IRB and none are involved in trials. The average age of the cohort was 64 yrs. Biospecimens were collected from newly diagnosed patients with endometrial cancer (EC) who were undergoing surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy, and were collected independent of grade or stage. EC cases were graded using the FIGO (International Federation of Gynecology and Obstetrics) system or the American Joint Committee on Cancer TNM staging system, which are functionally identical.</p></section><section id="S22"><h4 class="pmc_sec_title">Sample Collection</h4>
<p id="P49">The CPTAC Biospecimen Core Resource (BCR) at the Pathology and Biorepository Core of the Van Andel Research Institute in Grand Rapids, Michigan manufactured and distributed biospecimen kits to the Tissue Source Sites (TSS) located in the US, Europe, and Asia. Each kit contained a set of pre-manufactured labels for unique tracking of every specimen respective to TSS location, disease, and sample type, used to track the specimens through the BCR to the CPTAC proteomic and genomic characterization centers. Tissue specimens averaging 302 mg were snap-frozen by the TSS within a 30 min cold ischemic time (CIT) (CIT average = 17 min) and an adjacent segment was formalin-fixed paraffin-embedded (FFPE) and H&amp;E stained by the TSS for quality assessment to meet the CPTAC EC requirements. Routinely, several tissue segments for each case were collected. Tissues were flash frozen in liquid nitrogen (LN2) then transferred to a liquid nitrogen freezer for storage until approval for shipment to the BCR. Specimens were shipped using a cryoport that maintained an average temperature of under −140°C to the BCR with a time and temperature tracker to monitor the shipment. Receipt of specimens at the BCR included a physical inspection and review of the time and temperature tracker data for specimen integrity, followed by barcode entry into a biospecimen tracking database. Specimens were again placed in storage at LN2 temperatures until further processing. Acceptable EC tumor tissue segments were determined by TSS pathologists based on the percent viable tumor nuclei (&gt; 80%), total cellularity (&gt; 50%), and necrosis (&lt; 20%). Segments received at the BCR were verified by BCR and Leidos Biomedical Research (LBR) pathologists and the percent of total area of tumor in the segment was also documented. Additionally, disease-specific working group pathology experts reviewed the morphology to clarify or standardize specific disease classifications and correlation to the proteomic and genomic data. Specimens selected for the discovery set were determined on the maximal percent in the pathology criteria and best weight. Specimens were pulled from the biorepository using an LN2 cryocart to maintain specimen integrity and then cryopulverized. The cryopulverized specimen was divided into aliquots for DNA (30 mg) and RNA (30 mg) isolation and proteomics (50 mg) for molecular characterization. Nucleic acids were isolated and stored at −80°C until further processing and distribution; cryopulverized protein material was returned to the LN2 freezer until distribution. Shipment of the cryopulverized segments used cryoports for distribution to the proteomic characterization centers and shipment of the nucleic acids used dry ice shippers for distribution to the genomic characterization centers; a shipment manifest accompanied all distributions for the receipt and integrity inspection of the specimens at the destination. The DNA sequencing was performed at the Broad Institute, Cambridge, MA and RNA sequencing was performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses was sent to the Proteomic Characterization Center (PCC) at Pacific Northwest National Laboratory (PNNL), Richland, Washington.</p></section><section id="S23"><h4 class="pmc_sec_title">Enriched normal sample collection</h4>
<p id="P50">20 enriched normal endometrium tissue samples were collected according to CPTAC’s Standard Operating Procedure (SOP) for collection and preservation of research specimens. Enriched normal samples were obtained from healthy patients with an indication for surgery due to the presence of a non-cancerous lesion (hyperplasia, endometrial polyps, myoma) and served as an independent cohort of normal endometrial samples. The collected fragments were excised carefully to avoid any contamination of myometrium and other stromal components. After excision, the normal tissue samples were snap-frozen within a 30 min cold ischemic time, and next flash-frozen in liquid nitrogen, and transferred to a liquid nitrogen freezer for storage until approval for shipment to BCR. An adjacent segment of each sample was formalin-fixed paraffin-embedded and H&amp;E stained to perform quality assessment. To meet CPTAC pathology acceptance criteria, collected normal tissue samples had to include at least 80% of endometrial cells. Thus, enriched normal tissue means that the normal tissue includes at least 80% of endometrial cells.</p></section><section id="S24"><h4 class="pmc_sec_title">ICI treated sample collection</h4>
<p id="P51">Medical records were abstracted to identify uterine cancer patients who had received treatment with ICI therapy. Pre-ICI FFPE tumor specimens were retrieved from those with archived tissue stored at UAB. Medical records were analyzed to obtain basic demographic data, histology, stage, treatments used, and best response to ICI therapy. Best response to therapy was collected by reviewing all imaging reports obtained on therapy. Complete response was defined as no radiographic evidence of residual disease; partial response was defined as any reduction in lesion size by &gt; 1 cm; stable disease was defined as no growth or reduction of lesions &gt; 1 cm; progressive disease was defined as growth of lesions &gt; 1 cm or oncologist’s determination of clinical progression.</p></section><section id="S25"><h4 class="pmc_sec_title">Cell lines</h4>
<p id="P52">The EC cell line, HEC-151, was purchased from JCRB cell bank. Cells were cultivated using MEM containing 10% FBS and 1% penicillin–streptomycin in a 37 °C incubator with 5% CO<sub>2</sub>. Cell lines were routinely tested and confirmed negative for Mycoplasma contamination by Lonza Mycoalert Mycoplasma Detection Kit.</p>
<p id="P53">EC cell lines, HEC265, HEC-108, HEC-251, were obtained from either the JCRB Cell Bank. EC cell line, HEC1A, was obtained from the American Type Culture Collection. Cells were cultured in MEM media supplemented with 15% FBS. Identity of each cell line was verified by a PCR-based method for Short Tandem Repeat (STR) profiling prior to receipt. Once thawed, cell lines were cultured for fewer than 20 passages and regularly screening for mycoplasma infection by two independent methods: MycoStrip (Rep-mys-10; Invivogen) and RT-PCR (Venor GeM, Cat MP00025–1KT; Sigma-Millipore).</p></section></section><section id="S26"><h3 class="pmc_sec_title">METHOD DETAILS</h3>
<section id="S27"><h4 class="pmc_sec_title">Sample Processing for Genomic DNA and Total RNA Extraction</h4>
<p id="P54">Our study sampled a single site of the primary tumor from surgical resections, due to the internal requirement to process a minimum of 125 mg of tumor issue and 50 mg of adjacent normal tissue. DNA and RNA were extracted from tumor and adjacent normal specimens in a co-isolation protocol using QIAGEN’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3–5 mL) to serve as matched normal reference material. The Qubit dsDNA BR Assay Kit was used with the Qubit 2.0 Fluorometer to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through various genomic assays was sent for genomic characterization. RNA quality was quantified using both the NanoDrop 8000 and quality assessed using Agilent Bioanalyzer. A sample that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity match for germline, normal adjacent tissue, and tumor tissue was assayed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification.</p></section><section id="S28"><h4 class="pmc_sec_title">Whole Exome Sequencing</h4>
<section id="S29"><h5 class="pmc_sec_title">Whole Exome Sequencing Methods</h5>
<section id="S30"><h6 class="pmc_sec_title">Library Construction</h6>
<p id="P55">Library construction was performed as described in Fisher et al., with the following modifications: initial genomic DNA input into shearing was reduced from 3 μg to 20–250 ng in 5 0μL of solution. For adapter ligation, Illumina paired end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies, with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A- tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30 μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted.</p></section><section id="S31"><h6 class="pmc_sec_title">In-solution hybrid selection</h6>
<p id="P56">After library construction, libraries are pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions: first, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system.</p></section><section id="S32"><h6 class="pmc_sec_title">Preparation of libraries for cluster amplification and sequencing</h6>
<p id="P57">After post-capture enrichment, library pools were quantified using qPCR (automated assay on the Agilent Bravo), using a kit purchased from KAPA Biosystems with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM.</p></section><section id="S33"><h6 class="pmc_sec_title">Cluster amplification and sequencing</h6>
<p id="P58">Cluster amplification of DNA libraries was performed according to the manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flowcells were sequenced utilizing Sequencing-by-Synthesis chemistry. The flow cells are then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76 cycle runs with two 8 cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired end runs to achieve a minimum of 150x on target coverage per each sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files, adapter and low-quality sequences were trimmed. The raw reads were mapped to the HG38 human reference genome. The validated bams were used for downstream analysis and variant calling.</p></section><section id="S34"><h6 class="pmc_sec_title">PCR-Free Whole Genome Sequencing</h6></section></section></section></section><section id="S35"><h3 class="pmc_sec_title">Whole Genome Sequencing, PCR-Free</h3>
<section id="S36"><h4 class="pmc_sec_title">Preparation of libraries for cluster amplification and sequencing</h4>
<p id="P59">An aliquot of genomic DNA (350 ng in 50 μL) is used as the input into DNA fragmentation (aka shearing). Shearing is performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection is performed using a SPRI cleanup. Library preparation is performed using a commercially available kit provided by KAPA Biosystems (KAPA Hyper Prep without amplification module) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (purchased from IDT). Following sample preparation, libraries are quantified using quantitative PCR (kit purchased from KAPA Biosystems), with probes specific to the ends of the adapters. This assay is automated using Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries are normalized to 1.7 nM and pooled into 24-plexes.</p></section><section id="S37"><h4 class="pmc_sec_title">Cluster amplification and sequencing (HiSeq X)</h4>
<p id="P60">Sample pools are combined with HiSeqX Cluster Amp Regents EPX1, EPX2 and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates is performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flow Cells are sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software is processed by the Picard data-processing pipeline to yield BAM files containing demultiplexed, aggregated aligned reads.</p>
<p id="P61">All sample information tracking is performed by automated LIMS messaging.</p></section><section id="S38"><h4 class="pmc_sec_title">Illumina Infinium MethylationEPIC BeadChip Array</h4>
<p id="P62">The MethylationEPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt;850,000 methylation sites per sample. The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype calling pipeline. Data generated consists of raw idats and a sample sheet</p></section><section id="S39"><h4 class="pmc_sec_title">RNA Sequencing.</h4>
<section id="S40"><h5 class="pmc_sec_title">Total RNA Sequencing</h5>
<p id="P63">Indexed cDNA sequencing libraries were prepared from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags. Quality control was performed at every step, and the libraries were quantified. Indexed libraries were prepared and run on HiSeq4000 paired end 75 base pairs to generate a minimum of 120 million reads per sample library with a minimum of 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to fastq files, and adapter and low-quality sequences were trimmed for downstream analysis.</p></section><section id="S41"><h5 class="pmc_sec_title">miRNA Sequencing</h5>
<p id="P64">Indexed small RNA sequencing libraries were prepared from the RNA samples using the TruSeq Small Total RNA Sample Prep Kit and bar-coded with individual tags. Quality control was performed at every step, and the libraries were quantified. Indexed libraries were prepared and loaded on the Hiseq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to fastq files for downstream analysis.</p></section></section></section><section id="S42"><h3 class="pmc_sec_title">MS Sample Processing and Data Collection</h3>
<section id="S43"><h4 class="pmc_sec_title">Protein extraction and Lys-C/trypsin tandem digestion</h4>
<p id="P65">Approximately 50 mg of each of the pulverized OCT-embedded uterine tumor and normal tissues were homogenized separately in 200 μL of lysis buffer (8 M urea, 75 mM NaCl, 50 mM Tris, pH 8.0, 1 mM EDTA, 2 μg/mL aprotinin, 10 μg/mL leupeptin, 1 mM PMSF, 10 mM NaF, 1:100 v/v Sigma phosphatase inhibitor cocktail 2, 1:100 v/v Sigma phosphatase inhibitor cocktail 3, 20 μM PUGNAc, and 5 mM sodium butyrate). Lysates were precleared by centrifugation at 20,000 g for 10 min at 4 °C and protein concentrations were determined by BCA assay (ThermoFisher Scientific) and adjusted to 8 μg/μL with lysis buffer. Proteins were reduced with 5 mM dithiothreitol for 1 h at 37 °C, and subsequently alkylated with 10 mM iodoacetamide for 45 min at 25°C in the dark. Samples were diluted 1:4 with 50 mM Tris, pH 8.0 and digested with Lys-C (Wako) at 1:50 enzyme-to-substrate ratio sequencing grade modified trypsin (Promega, V5117) at 1:50 enzyme-to-substrate ratio. After 2 h of digestion at 25 °C, an aliquot of the same amount of sequencing grade modified trypsin (Promega, V5117) was added to the samples and further incubated at 25 °C overnight (~14 h). The digested samples were then acidified with 100% formic acid to 1% of the final concentration of formic acid. Tryptic peptides were desalted on C18 SPE (Waters tC18 SepPak, WAT054925) and dried using Speed-Vac.</p></section><section id="S44"><h4 class="pmc_sec_title">TMT-11 labeling</h4>
<p id="P66">Desalted peptides were labeled with 11-plex TMT reagents using conditions modified from the manufacturer’s instructions (ThermoFisher Scientific). Peptides (400 μg) from each of the tumors were dissolved in 80 μL of 50 mM HEPES, pH 8.5 solution, and mixed with 400 μg of TMT reagent that was dissolved freshly in 20 μL of anhydrous acetonitrile. Channel 126 was used for labeling the internal reference sample (pooled from all tumor and normal samples in the CPTAC UCEC Exploratory cohort – Cell. 2020, 180(4):729–748) throughout the sample analysis. After 1 h incubation at RT each sample was diluted with 60 μL 50 mM HEPES pH 8.5, 20% acetonitrile. Quench of the reaction was achieved by adding 12 μL of 5% hydroxylamine and incubation for 15 min at RT. Peptides labeled by different TMT reagents were then mixed, dried using Speed-Vac, reconstituted with 3% acetonitrile, 0.1% formic acid and were desalted on tC18 SepPak SPE columns.</p></section><section id="S45"><h4 class="pmc_sec_title">Fractionation</h4>
<p id="P67">Approximately 3.5 mg of 11-plex TMT labeled sample was separated on a reversed-phase Agilent Zorbax 300 Extend-C18 column (250 mm × 4.6 mm column containing 3.5-μm particles) using Agilent 1200 HPLC System. Solvent A was 5 mM ammonium formate, pH 10, 2% acetonitrile and solvent B was 5 mM ammonium formate, pH 10, 90% acetonitrile. The flow rate was 1 mL/min and the injection volume was 900 μL. The LC gradient started with a linear increase of solvent B to 16% in 6 min, then linearly increased to 40% B in 70 min, 4 min to 44% B, 5 min to 60% B and another 14 of 60% solvent B. A total of 96 fractions were collected into a 96 well plate throughout the LC gradient. These fractions were concatenated into 24 fractions by combining 4 fractions that are 24 fractions apart (i.e., combining fractions #1, #25, #49, and #73; #2, #26, #50, and #74; and so on). For proteome analysis, 5% of each concatenated fraction was dried down and resuspended in 2% acetonitrile, 0.1% formic acid to a peptide concentration of 0.1 mg/mL for LC-MS/MS analysis. The rest of the fractions (95%) were further concatenated into 12 fractions (i.e., by combining fractions #1 and #13; #3 and #15; and so on), dried down, and subjected to immobilized metal affinity chromatography (IMAC) for phosphopeptide enrichment.</p></section><section id="S46"><h4 class="pmc_sec_title">Phosphopeptide enrichment using IMAC</h4>
<p id="P68">Fe<sup>3+</sup>-NTA-agarose beads were freshly prepared using the Ni-NTA Superflow agarose beads (QIAGEN, #30410) for phosphopeptide enrichment. For each of the 12 fractions, peptides were reconstituted in 500 μL IMAC binding/wash buffer (80% acetonitrile, 0.1% trifluoroacetic acid) and incubated with 20 μL of the 50% bead suspension for 30 min at RT. After incubation, the beads were washed 2 times each with 50 μL of wash buffer and once with 50 μL of 1% formic acid on the stage tip packed with 2 discs of Empore C18 material (Empore Octadecyl C18, 47 mm; Supleco, 66883-U). Phosphopeptides were eluted from the beads on C18 using 70 μL of Elution Buffer (500 mM K<sub>2</sub>HPO<sub>4</sub>, pH 7.0). 50% acetonitrile, 0.1% formic acid was used for elution of phosphopeptides from the C18 stage tips. Samples were dried using Speed-Vac, and later reconstituted with 10 μL of 3% acetonitrile, 0.1% formic acid for LC-MS/MS analysis.</p></section><section id="S47"><h4 class="pmc_sec_title">Immunoaffinity purification of acetylated peptides</h4>
<p id="P69">Tryptic peptides from the flow-through of IMAC were combined into four samples follow concatenation scheme by combining 3 fractions that were 4 fractions apart (i.e., combining fractions #1, #5 and #9 as a new fraction) and dried down using Speed-Vac. The dried peptides were reconstituted in 1.4 mL of the immunoaffinity purification (IAP) buffer (50 mM MOPS/NaOH pH 7.2, 10 mM Na<sub>2</sub>HPO<sub>4</sub> and 50 mM NaCl). After dissolving the peptide, the pH of the peptide solution was checked using pH indicator paper. The antibody beads from PTMScan<sup>®</sup> Acetyl-Lysine Motif [Ac-K] Kit (Cell Signaling, #13416) were freshly prepared. Briefly, the antibody beads were centrifuged at 2,000 x g for 30 sec and all buffers from the beads were removed; the antibody beads were then washed with 1 mL of IAP buffer for four times and finally resuspend in 40 μL of IAP buffer. For each fraction, half of the antibody in each tube was transferred to the peptide solution and incubated on a rotator overnight at 4 °C. After removing the supernatant, the reacted beads were washed with 1 mL of PBS buffer five times. For the elution of acetylated peptides, the antibody beads were incubated 2 times each with 50 μL of 0.15% TFA at room temperature for 10 min. The eluted peptides were transferred to the stage tip packed with two discs of Empore C18 material. The C18 stage tips were washed by 1% formic acid and 50% acetonitrile, and 0.1% formic acid was used for elution of peptides from the C18 stage tips. The eluted peptides were dried using Speed-Vac, and reconstituted with 13 μL of 2% acetonitrile, 0.1% formic acid contained 0.01% DDM (n-Dodecyl β-D-maltoside) right before the LC-MS/MS analysis.</p>
<p id="P70">The acetylated peptides prepared by IP from the IMAC flow-through may very well miss those peptides that are both phosphorylated and acetylated. Splitting the samples for independent IP and IMAC may improve the chance of recovering such peptides, assuming having both PTMs on the same peptide does not impact the affinity of either the IP or IMAC process. However, acetylated peptides are estimated to be 10 times lower in abundance than the phosphopeptides, hence much larger input may be needed to recover the dual-modified peptides. Given the extremely low stoichiometry of these dual-modified peptides and the sample size limitations, it was not pursued in this work.</p></section><section id="S48"><h4 class="pmc_sec_title">Enrichment of glycopeptides</h4>
<p id="P71">Peptides from the flow-through of acetylated peptide enrichment were desalted on the reversed phase C18 SPE column (Waters). The glycopeptides were enriched with OASIS MAX solid-phase extraction (Waters). The MAX cartridge was conditioned with 3×1 mL of ACN, then 3×1 mL of 100 mM triethylammonium acetate buffer, followed by 3×1 mL of water, and finally 3×1 mL of 95% ACN / 1% TFA. The peptides were reconstituted in 95% ACN / 1% TFA and loaded twice. The cartridge was washed with 4×1 mL of 95% ACN / 1% TFA to remove non-glycosylated peptides. The glycopeptide fraction was eluted with 50% ACN / 0.1% FA, dried down, and reconstituted in 3% ACN / 0.1% FA prior to ESI-LC-MS/MS analysis.</p></section><section id="S49"><h4 class="pmc_sec_title">LC-MS/MS data acquisition</h4>
<p id="P72">Fractionated samples prepared for global proteome, phosphoproteome, and acetylome analysis were separated using a nanoACQUITY UPLC system (Waters) by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase (Dr. Maisch GmbH) and slurry packed into a 25-cm length of 360 μm o.d. x 75 μm i.d. fused silica picofrit capillary tubing (New Objective). The analytical column was heated to 50 °C using an AgileSLEEVE column heater (Analytical Sales and Services). The analytical column was equilibrated to 98% Mobile Phase A (MP A, 0.1% formic acid/3% acetonitrile) and 2% Mobile Phase B (MP B, 0.1% formic acid/90% acetonitrile) and maintained at a constant column flow of 200 nL/min. The sample was injected into a 5-μL loop placed in-line with the analytical column which initiated the gradient profile (min:%MP B): 0:2, 1:6, 85:30, 94:60, 95:90, 100:90, 101:50, 110:50 (for global proteome and phosphoproteome analysis); 0:2, 1:6, 235:30, 244:60, 245:90, 250:90, 251:50, 260:50 (for acetylome analysis). The column was allowed to equilibrate at start conditions for 30 minutes between analytical runs.</p>
<p id="P73">MS analysis was performed using an Orbitrap Fusion Lumos mass spectrometer (ThermoFisher Scientific). The global proteome and phosphoproteome samples were analyzed under identical conditions. Electrospray voltage (1.8 kV) was applied at a carbon composite union (Valco Instruments) coupling a 360 μm o.d. x 20 μm i.d. fused silica extension from the LC gradient pump to the analytical column and the ion transfer tube was set at 250 °C. Following a 25-min delay from the time of sample injection, Orbitrap precursor spectra (AGC 4 × 10<sup>5</sup>) were collected from 350–1800 m/z for 110 min at a resolution of 60K along with data dependent Orbitrap HCD MS/MS spectra (centroid) at a resolution of 50K (AGC 1 × 10<sup>5</sup>) and max ion time of 105 ms for a total duty cycle of 2 seconds. Masses selected for MS/MS were isolated (quadrupole) at a width of 0.7 m/z and fragmented using a collision energy of 30%. Peptide mode was selected for monoisotopic precursor scan and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 45 seconds to discriminate against previously analyzed ions between ±10 ppm. The acetylome samples were analyzed under similar conditions except that the max ion time was 125 ms.</p>
<p id="P74">The TMT-labeled glycoproteome fractions were analyzed using Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). Approximately 0.5 ug of peptides were separated on an in-house packed 28 cm x 75 um diameter C18 column (1.9 um Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10 um opening (New Objective)) lined up with an Easy nLC 1200 UHPLC system (Thermo Scientific). The column was heated to 50C using a column heater (Phoenix-ST). The flow rate was set at 200 nl/min. Buffer A and B were 3% ACN / 0.1% FA and 90% ACN / 0.1% FA, respectively. The peptides were separated with a 6%–30% B gradient in 84 min. Peptides were eluted from the column and nanosprayed directly into the mass spectrometer. The mass spectrometer was operated in a data-dependent mode. Parameters for glycoproteomic samples were set as follows: MS1 resolution – 60,000, mass range – 500 to 2000 m/z, RF Lens – 30%, AGC Target – 5.0e5, Max injection time – 50 ms, charge state include – 2–6, dynamic exclusion – 45 s. The cycle time was set to 2 s, and within this 2 s the most abundant ions per scan were selected for MS/MS in the orbitrap. MS2 resolution – 50,000, high-energy collision dissociation activation energy (HCD) – 35, isolation width (m/z) – 0.7, AGC Target – 1.0e5, Max injection time – 100 ms.</p></section><section id="S50"><h4 class="pmc_sec_title">Construction and utilization of the Comparative Reference samples</h4>
<p id="P75">As a quality control measure, two different types of “Comparative Reference” (“CompRef”) patient-derived xenograft (PDX) samples were generated as previously described (Li et al., 2013; Tabb et al., 2016) and used to monitor the longitudinal performance of the proteomics workflow throughout the course of this study. Briefly, the PDX tumors from established basal and luminal breast cancer intrinsic subtypes were raised subcutaneously in 8-week old NOD.Cg-Prkdc<sup>scid</sup> Il2rg<sup>tm1Wjl</sup>/SzJ mice (Jackson Laboratories, Bar Harbor, ME) using procedures reviewed and approved by the Institutional Animal Care and Use Committee at Washington University in St. Louis. Xenografts were grown in multiple mice, pooled, and cryopulverized to provide a sufficient amount of uniform material for the duration of the study. Full proteome, phosphoproteome and acetylome process replicates of each of the two types of CompRef samples were prepared and analyzed as standalone 11-plex TMT experiments alongside every four TMT-11 experiments of the study samples, using the same analysis protocol as the patient samples. These interstitially analyzed CompRef samples were evaluated for depth of proteome, phosphoproteome, and acetylome coverage and for consistency in quantitative comparison between the basal and luminal models.</p></section></section><section id="S51"><h3 class="pmc_sec_title">Genomic Data Analysis</h3>
<section id="S52"><h4 class="pmc_sec_title">Harmonized genome alignment</h4>
<p id="P76">WGS, WES, RNA-Seq sequence data were harmonized by NCI Genomic Data Commons (GDC) <a href="https://gdc.cancer.gov/about-data/gdc-data-harmonization" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gdc.cancer.gov/about-data/gdc-data-harmonization</a>, which included alignment to GDC’s hg38 human reference genome (GRCh38.d1.vd1) and additional quality checks. All the downstream genomic processing was based on the GDC-aligned BAMs to ensure reproducibility.</p></section><section id="S53"><h4 class="pmc_sec_title">Copy Number Calling</h4>
<p id="P77">Copy number variation was detected using BIC-seq2 <sup><a href="#R94" class="usa-link" aria-describedby="R94" aria-expanded="false">94</a></sup>, a read-depth-based CNV calling algorithm to detect somatic copy number variation (CNVs) from the WGS data of tumors. Briefly, BIC-seq2 divides genomic regions into disjoint bins and counts uniquely aligned reads in each bin. Then, it combines neighboring bins into genomic segments with similar copy numbers iteratively based on Bayesian Information Criteria (BIC), a statistical criterion measuring both the fitness and complexity of a statistical model. We used paired-sample CNV calling that takes a pair of samples as input and detects genomic regions with different copy numbers between the two samples. We used a bin size of ∼100 bp and a lambda of 3 (a smoothing parameter for CNV segmentation). We recommend calling segments as copy gain or loss when their log2 copy ratios were larger than 0.2 or smaller than −0.2, respectively (according to the BIC-seq publication).</p></section><section id="S54"><h4 class="pmc_sec_title">Somatic Variant Calling</h4>
<p id="P78">Somatic mutations were called by the Somaticwrapper pipeline v1.6 (<a href="https://github.com/ding-lab/somaticwrapper" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ding-lab/somaticwrapper</a>), which includes four different callers, i.e., Strelka v.2<sup><a href="#R95" class="usa-link" aria-describedby="R95" aria-expanded="false">95</a></sup>, MUTECT v1.7 <sup><a href="#R71" class="usa-link" aria-describedby="R71" aria-expanded="false">71</a></sup>, VarScan v.2.3.8 <sup><a href="#R84" class="usa-link" aria-describedby="R84" aria-expanded="false">84</a></sup>, and Pindel v.0.2.5 <sup><a href="#R76" class="usa-link" aria-describedby="R76" aria-expanded="false">76</a></sup> from WES. We kept the exonic SNVs called by any two callers among MUTECT v1.7, VarScan v.2.3.8, and Strelka v.2 and indels called by any two callers among VarScan v.2.3.8, Strelka v.2, and Pindel v.0.2.5. For the merged SNVs and indels, we applied a 14X and 8X coverage cutoff for tumor and normal, separately. We also filtered SNVs and indels by a minimal variant allele frequency (VAF) of 0.05 in tumors and a maximal VAF of 0.02 in normal samples. We filtered any SNV, which was within 10bp of an indel found in the same tumor sample. Finally, we rescued the rare mutations with VAF of [0.015, 0.05) in ccRCC driver genes based on the gene consensus list <sup><a href="#R96" class="usa-link" aria-describedby="R96" aria-expanded="false">96</a></sup></p></section><section id="S55"><h4 class="pmc_sec_title">Mutational Signatures</h4>
<p id="P79">The R package SigProfilerMatrixGeneratorR (version 1.0) was used to call mutation signatures from somatic mutation data (PMID: 31470794). The maximum number of signatures was set to 10 and nmf replicate was set to 100. The activity scores of SigProfilerMatricGenetator suggested decomposed solutions were used as signature scores. Only single base substitutions signatures were included in our analysis.</p></section><section id="S56"><h4 class="pmc_sec_title">DNA methylation analysis</h4>
<p id="P80">Raw methylation idat files were downloaded from CPTAC DCC and GDC. Beta values of CpG loci were reported after functional normalization, quality check, common SNP filtering, and probe annotation using Li Ding Lab’s methylation pipeline v1.1 <a href="https://github.com/ding-lab/cptac_methylation" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ding-lab/cptac_methylation</a>. For downstream integrated analysis, we focused only on the methylation levels (represented as beta values) of the probes located both in the CpG island and the promoter (including 5’UTR) regions. The gene-level methylation was derived by averaging these probe-level methylation values.</p></section><section id="S57"><h4 class="pmc_sec_title">Microsatellite Instability Prediction</h4>
<p id="P81">MSI scores were calculated by MSIsensor (<a href="https://github.com/ding-lab/msisensor" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ding-lab/msisensor</a>) and interpreted as the percentage of microsatellite sites (with deep enough sequencing coverage) that have a lesion. Samples with an MSIscore &gt; 5 are classified as “MSI-High” and the rest will be classified as “MSS” based on a bimodal distribution of MSIscore on this cohort.</p></section><section id="S58"><h4 class="pmc_sec_title">Copy Number Classification</h4>
<p id="P82">The copy number subtypes were characterized by CNV deletion events. A sample was defined as CNV-high if more than 10% of its genome was deleted, regardless of the number of CNV-independent events.</p></section><section id="S59"><h4 class="pmc_sec_title">TCGA Subtype Classification</h4>
<p id="P83">TCGA identified four subtypes of endometrial cancer: POLE, MSI, CNV-high, and CNV-low. We replicated this subtyping for the samples in this study.</p>
<p id="P84">The MSI subtype consists of all samples called MSI-H, as described in the Microsatellite Instability Prediction section. To identify the POLE subtype, we looked for samples with mutations in the POLE exonuclease domain. There were 6 samples carrying exonuclease domain mutations (EDM) including 5 P286R and 1 V411L. All of them passed the criteria [CA] signature &gt; 20% and [CG] signature &lt; 3%. Thus, six samples were classified as POLE. Samples identified as having high CNV, as described in the Copy Number Classification section, were assigned to the CNV-high subgroup. All remaining samples not classified as MSI, POLE, or CNV-high were classified as CNV-low. The genomic subtypes of three samples, C3L-00898, C3L-02802, and C3N-02298, could not be determined due to lack of WGS-based CNV data. One sample, C3L-02557, was identified as both POLE and CNV-H.</p></section><section id="S60"><h4 class="pmc_sec_title">RNA Quantification &amp; Analysis</h4>
<section id="S61"><h5 class="pmc_sec_title">RNA Quantification and Circular RNA Prediction</h5>
<p id="P85">The Hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis. They were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNA was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression <sup><a href="#R97" class="usa-link" aria-describedby="R97" aria-expanded="false">97</a></sup>. In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene and circular RNA expression based on the mixed transcripts. After quantification, the upper quantile method was applied for normalization. The normalized matrix was log2-transformed and separated into gene and circular RNA expression matrices.</p>
<p id="P86"><em>miRNA-Seq Data Analysis</em> miRNA-Seq FASTQ files were downloaded from GDC. We reported the mature miRNA and precursor miRNA expression in TPM (Transcripts Per Million) after adapter trimming, quality check, alignment, annotation, reads counting using Li Ding Lab’s miRNA pipeline <a href="https://github.com/ding-lab/CPTAC_miRNA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ding-lab/CPTAC_miRNA</a>. The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression from its precursor miRNAs.</p></section><section id="S62"><h5 class="pmc_sec_title">PROGENy Pathway Activity</h5>
<p id="P87">The PROGENy R package was applied to the log2 transformed RSEM mRNA matrix to estimate activity of 11 cancer related pathways: EGFR, Hypoxia, JAK-STAT, MAPK, NFkB, PI3K, TGFb, TNFa, Trail, VEGF, p53<sup><a href="#R77" class="usa-link" aria-describedby="R77" aria-expanded="false">77</a></sup>.</p></section></section></section><section id="S63"><h3 class="pmc_sec_title">MS Data Interpretation</h3>
<section id="S64"><h4 class="pmc_sec_title">Quantification of TMT global proteomics data</h4>
<p id="P88">LC-MS/MS analysis of the TMT11-labeled, bRPLC fractionated samples generated a total of 384 global proteomics data files. The Thermo RAW files were processed with mzRefinery to characterize and correct for any instrument calibration errors, and then with MS-GF+ v9881<sup><a href="#R67" class="usa-link" aria-describedby="R67" aria-expanded="false">67</a>,<a href="#R72" class="usa-link" aria-describedby="R72" aria-expanded="false">72</a>,<a href="#R98" class="usa-link" aria-describedby="R98" aria-expanded="false">98</a></sup> to match against the RefSeq human protein sequence database downloaded on June 29, 2018 (hg38; 41,734 proteins), combined with 264 contaminants (e.g., trypsin, keratin). The partially tryptic search used a ± 10 ppm parent ion tolerance, allowed for isotopic error in precursor ion selection, and searched a decoy database composed of the forward and reversed protein sequences. MS-GF+ considered static carbamidomethylation (+57.0215 Da) on Cys residues and TMT modification (+229.1629 Da) on the peptide N-terminus and Lys residues, and dynamic oxidation (+15.9949 Da) on Met residues for searching the global proteome data.</p>
<p id="P89">Peptide identification stringency was set at a maximum 1% FDR at peptide level using PepQValue &lt; 0.005 and parent ion mass deviation &lt; 7 ppm criteria. A minimum of 6 unique peptides per 1000 amino acids of protein length was then required for achieving 1% at the protein level within the full data set.</p>
<p id="P90">The intensities of all 11 TMT reporter ions were extracted using MASIC software (Monroe et al., 2008). Next, PSMs passing the confidence thresholds described above were linked to the extracted reporter ion intensities by scan number. The reporter ion intensities from different scans and different bRPLC fractions corresponding to the same gene were grouped. Relative protein level was calculated as the ratio of sample level to reference level using the summed reporter ion intensities from peptides that could be uniquely mapped to a gene. The pooled reference sample was labeled with TMT 126 reagent, allowing comparison of relative protein levels across different TMT-11 plexes. The relative levels were log<sub>2</sub> transformed and zero-centered for each gene to obtain final relative levels.</p>
<p id="P91">Small differences in laboratory conditions and sample handling can result in systematic, sample-specific bias in the quantification of protein levels. In order to mitigate these effects, we computed the median, log<sub>2</sub> relative protein level for each sample and re-centered to achieve a common median of 0.</p></section><section id="S65"><h4 class="pmc_sec_title">Quantification of phosphopeptides</h4>
<p id="P92">Phosphopeptide identification for the 192 phosphoproteomics data files were performed as in the global proteome data analysis described above (e.g., peptide level FDR &lt; 1%), with an additional dynamic phosphorylation (+79.9663 Da) on Ser, Thr, or Tyr residues. The phosphoproteome data were further processed by the Ascore algorithm (Beausoleil et al., 2006) for phosphorylation site localization, and the top-scoring sequences were reported. For phosphoproteomic datasets, the TMT-11 quantitative data were not summarized by protein but left at the phosphopeptide level. All peptides (phosphopeptides and global peptides) were labeled with TMT-11 reagent simultaneously. Separation into phospho- and non-phosphopeptides using IMAC was performed after the labeling. Thus, all the biases upstream of labeling are assumed to be identical between global and phosphoproteomic datasets. Therefore, to account for sample-specific biases in the phosphoproteome analysis, we applied the correction factors derived from median-centering the global proteomic dataset.</p></section><section id="S66"><h4 class="pmc_sec_title">Quantification and analysis of acetylated peptides</h4>
<p id="P93">Acetylated peptide identification for the 64 acetylome data files were performed as in the global proteome data analysis described above, with additional dynamic acetylation (+42.0105 Da) and carbamylation (+43.0058 Da) on Lys residues. The acetylation site localization, protein inference, and quantification of the acetylome data were performed in identical fashion as in the phosphoproteome data.</p></section><section id="S67"><h4 class="pmc_sec_title">Targeted selected reaction monitoring (SRM) analysis</h4>
<section id="S68"><h5 class="pmc_sec_title">Stable isotope-labeled peptides</h5>
<p id="P94">Proteotypic peptides for the target proteins were selected for targeted proteomics analysis based on the TMT identification results and well-accepted criteria <sup><a href="#R99" class="usa-link" aria-describedby="R99" aria-expanded="false">99</a></sup>, and the corresponding crude heavy stable isotope-labeled peptides were synthesized with <sup>13</sup>C/<sup>15</sup>N on C-terminal lysine or arginine (New England Peptide, Gardner, MA). The heavy peptides were dissolved individually in 15% acetonitrile (ACN) and 0.1% formic acid (FA) at a concentration of 2 mM and used for creating a peptide internal standard (IS) mixture with a final concentration of 5 μMfor each peptide.</p></section><section id="S69"><h5 class="pmc_sec_title">SRM assay development</h5>
<p id="P95">The heavy peptides in the peptide mixture were evaluated for peptide response and fragmentation pattern using LC-SRM. For each peptide, transition settings were as follows: (1) precursor charges: 2, 3 and 4; (2) fragment ion charges: 1, 2 and 3; (3) ion type: y ions; and (4) m/z window: 250–1500. Transition lists were generated with optimal collision energy values by Skyline software (Version 21.2) <sup><a href="#R81" class="usa-link" aria-describedby="R81" aria-expanded="false">81</a></sup>. LC-SRM was then used to evaluate all heavy peptides for stability of peptide retention time, reliable heavy peptides identification, transition interferences, and endogenous peptide detectability. In the end, 3 transitions per peptide were selected for the final assay configuration for targeted quantitation of a total of 108 peptides of the 62 target proteins and 11 phosphopeptides of RB1 protein.</p></section><section id="S70"><h5 class="pmc_sec_title">PRISM fractionation</h5>
<p id="P96">High-pressure, high-resolution separations coupled with intelligent selection and multiplexing (PRISM) fractionation <sup><a href="#R100" class="usa-link" aria-describedby="R100" aria-expanded="false">100</a></sup> was performed for 39 peptides to achieve higher sensitivity quantitation by SRM. A nanoACQUITY UPLC<sup>®</sup> system (Waters Corporation, Milford, MA) equipped with a reversed-phase capillary LC column (30-μm Jupiter C18 bonded particles packed in 200 μm i.d. × 50 cm capillary) was used. Separations were performed by reversed-phase LC fractionation at mobile phase flow rates of 2.2 μL/min on the binary pump systems using 10 mM ammonium formate (pH 7) in water as mobile phase A and 10 mM ammonium formate (pH 7) in 90% ACN as mobile phase B. 45 μL of sample with a peptide concentration of 0.5 μg/μL and 12 fmol/μL of heavy peptide standards was loaded onto the reversed-phase capillary column and separated into 96 fractions using a 190-min gradient of (min:%B): 35:1, 37:10, 52:15, 87:25, 112:35, 125:45, 150:90, 156:1. The eluent was automatically deposited every minute and concatenated onto 6 vials. Prior to peptide fraction collection, 20 μL of 0.1% FA was added to each well of the 96-well plate to avoid the loss of peptides. All the elute were then dried under vacuum.</p></section><section id="S71"><h5 class="pmc_sec_title">Phosphopeptides enrichment by IMAC</h5>
<p id="P97">The in-house-made IMAC tip was capped in a tip-end with a 20-μm polypropylene frits disk followed by packing with Ni-NTA silica resin (QIAGEN, Hilden, Germany). First, Ni<sup>2+</sup> ions were removed by adding 50 mM EDTA in 1 M NaCl. The tip was then activated with 100 mM FeCl<sub>3</sub> and equilibrated with 1% (v/v) acetic acid at pH 3.0 prior to sample loading. 1 μL of 4000 fmol/μL of each of the crude IS phosphopeptides spiked into 200 μL of 1 μg/μL tryptic peptides or into in 0.1% (v/v) trifluoroacetic acid, 80% ACN were subjected to IMAC. Followed by 1% (v/v) trifluoroacetic acid, 80% ACN, and 1% (v/v) acetic acid washing steps, the bound phosphopeptides were eluted by 200 mM NH<sub>4</sub>H<sub>2</sub>PO<sub>4</sub> onto the activated desalting SDB-XC StageTips for desalting and directly eluted to sample vials of LC-SRM then dried under vacuum.</p></section><section id="S72"><h5 class="pmc_sec_title">LC-SRM data acquisition</h5>
<p id="P98">The digested tissue samples were reconstituted in 2% ACN/0.1% FA and spiked with 50 fmol/μL heavy peptides for a final concentration of 0.25 μg/μL, and 2 μL of the resulting samples were analyzed by LC-SRM using a Waters nanoACQUITY UPLC system coupled to a Thermo Scientific TSQ Altis triple quadrupole mass spectrometer (Thermo Scientific, Waltham, MA). A 100 μm i.d. × 10 cm, BEH 1.7-μm C18 capillary column (Waters) was operated at a temperature of 44 °C. The mobile phases were (A) 0.1% FA in water and (B) 0.1% FA in ACN. The peptide samples were separated at a flow rate of 400 nL/min using a 110-min gradient profile as follows (min:%B): 7:1, 9:6, 40:13, 70:22, 80:40, 85:95, 93:50, 94:95 and 95:1. Data were acquired in timescheduled SRM mode (retention time window: 15 min). PRISM collected fractions were reconstituted in 20 μL 2% ACN/0.1% FA and 9 μL were loaded onto the column and separated at a flow rate of 400 nL/min using a 70-min gradient profile as follows (min:%B): 11:0.5, 14: 5, 30:20, 37:25, 48:60, 31:95, 54:0.5. Data were acquired in unscheduled SRM mode. For enriched phosphopeptide samples, all the eluent from IMAC was dissolved in 12 μL and 9 μL were loaded onto the column and separated at a flow rate of 400 nL/min using a 42-min gradient profile as follows (min:%B): 11:0.5, 13: 5, 18:20, 23:25, 53:95, 36:0.5. Data were acquired in unscheduled SRM mode. The parameters of the triple quadruple instrument were set with 0.7 fwhm Q1 and Q3 resolution, and 1.2-s cycle time.</p></section><section id="S73"><h5 class="pmc_sec_title">SRM data analysis</h5>
<p id="P99">SRM data were analyzed using the Skyline software (Version 21.2) <sup><a href="#R81" class="usa-link" aria-describedby="R81" aria-expanded="false">81</a></sup>. The total peak area ratios of endogenous light peptides and their heavy isotope-labeled internal standards (i.e., L/H peak area ratios) were exported for quantitation. Peak detection and integration were carried out according to two criteria: (1) same retention time and (2) similar peak area ratios for the transitions. All data were manually inspected to ensure correct retention time, peak detection and accurate integration. All SRM results including the assay characterization data are organized as Skyline files on the Panorama server <sup><a href="#R101" class="usa-link" aria-describedby="R101" aria-expanded="false">101</a></sup> and can be accessed via <a href="https://panoramaweb.org/CPTAC_UCEC.url" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://panoramaweb.org/CPTAC_UCEC.url</a> (the account for reviewer is: Email: <span>panorama+reviewer134@proteinms.net</span>; Password: ALZsnvJc).</p></section><section id="S74"><h5 class="pmc_sec_title">Quantification and analysis of glycopeptides</h5>
<p id="P100">Mass spectrometry data from each individual cohort was downloaded from the DCC website. The data sets were annotated following their metadata information. MzML files were searched using the GPQuest search engine version 2.1 (Hu et al., 2018; Mertins et al., 2018) against a reported glycopeptide database reported in Glycositeatlas (Sun et al, 2019 ) appended with an equal number of decoy sequences and an N-linked glycan database, which was collected from the public database of GlycomeDB (Ranzinger et al., 2011) (<a href="http://www.glycome-db.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.glycome-db.org</a>). The post-processing, statistical summarization, and reporting were done using MS-PyCloud (Li Chen, 2018, biorxiv). The consensus spectral library for each glycopeptide was constructed by using SpectraST and data visualization was done by OmicsOne (Hu et. al, 2021). Below are the individual search settings for glycoproteome analysis, including the necessary setting to account for individual experimental settings.</p>
<p id="P101">Searching N-linked glycopeptides using GPQuest. Prior to the database search, ProteoWizard 3.0 was used to convert the .RAW files to .mzML files with the “centroid all scans” option selected. The MS/MS spectra containing the oxonium ions (m/z 204.0966) in the top 10 abundant peaks after removing TMT reporter ions were considered as the potential glycopeptide candidates. Isotope error was set to (−1/0/1/2) for all searches. Cysteine carbamidomethylation (+57.0215) was specified as fixed modification. Methionine oxidation (+15.9949), and protein N-terminal acetylation (+42.01060) were specified as variable modifications. The minimum number of peaks used was set to 15, and the maximum set to 100. The precursor ion tolerance was set to 10 ppm and the fragment tolerance was set to 20 ppm. Lysine TMT labeling (+229.1629), and peptide N-terminal TMT labeling (+229.1629) were specified as fixed modifications. The best hits of all glycopeptide-spectrum matchings (GPSMs) were ranked by the Morpheus scores (Wenger and Coon, 2013) in descending order, in which those with FDR &lt; 1% and covering &gt; 10% total intensity of each tandem spectrum were reserved as qualified identifications.</p>
<p id="P102">Post-processing using MS-PyCloud. Glycopeptide-spectrum matchings (GPSMs) were filtered based on a user-defined PSM-level false discovery rate (FDR) cutoff and significant GPSMs from all sets of each cohort were grouped to infer the represented proteins parsimoniously using a bipartite graph analysis algorithm adopted in many protein inference tools <sup><a href="#R102" class="usa-link" aria-describedby="R102" aria-expanded="false">102</a>–<a href="#R104" class="usa-link" aria-describedby="R104" aria-expanded="false">104</a></sup>. The final FDRs are then estimated at N-glycopeptide-levels using the reversed decoy search. For isobaric labeled data, the TMT reporter ion intensities are extracted from the mzML file for MS2 scans corresponding to the identified glycopeptides. For TMT data, the TMT reagent lot correction factors were used to adjust the reported TMT intensities for interference between TMT channels. Log2 ratios are calculated at GPSM-level relative to the user-specified reference channel and are then rolled up by the median value to N-glycopeptide-level (intact glycopeptide enriched). Normalized log2 ratio matrices are generated using median normalization (MD norm), and median normalization plus median absolute deviation scaling (MD norm + MAD scaling). Normalised levels are generated from the log2 ratio matrices by summing the log2 ratios with the median log2 value of the reference channel summed MS2 intensities across all sets for each N-glycopeptide using the same approach used in the proteome and phosphoproteome processing. Additional details regarding these steps can be found in <sup><a href="#R105" class="usa-link" aria-describedby="R105" aria-expanded="false">105</a>,<a href="#R106" class="usa-link" aria-describedby="R106" aria-expanded="false">106</a></sup>.</p></section><section id="S75"><h5 class="pmc_sec_title">IHC analysis</h5>
<p id="P103">Immunohistochemical staining was performed on formalin-fixed paraffin-embedded 4-μm tumor sections obtained from 52 UCEC patients. Slides and appropriate positive and negative controls were then stained with primary and secondary antibodies using the automated platform, Dako Autostainer Link 48. The following primary antibodies were used: anti-c-MYC (1:100; rabbit monoclonal; clone Y69; cat. no. 790–4628; Ventana), anti-CTNNB1 (1:1000; rabbit polyclonal; cat. no. HPA029159; Atlas Antibodies), anti-LEF1 (pre-diluted, ready-to-use formulation; rabbit monoclonal; clone ZR336; cat. no. ZC-Z2642RL, Zeta Corporation). An EnVision FLEX kit (cat. no. K800221–2; Dako, Agilent Technologies, Inc.) was used to visualize antibody binding. Subsequently, IHC slides were quantified manually by H-score on a scale of 0 to 300 taking into account the percentage of positive tumor cells and staining intensity. Slides were also digitally scanned with automated ScanScope AT Turbo slide scanner (Aperio/Leica Microsystems) using 20X magnification and digital images were saved as.svs.</p></section><section id="S76"><h5 class="pmc_sec_title">CRISPR editing materials and methods</h5>
<section id="S77"><h6 class="pmc_sec_title">Cell line</h6>
<p id="P104">The endometrial carcinoma cell line HEC-151 was purchased from JCRB cell bank. Cells were cultivated using MEM containing 10% FBS and 1% penicillin–streptomycin in a 37 °C incubator with 5% CO<sub>2</sub>. Cell lines were routinely tested and confirmed negative for Mycoplasma contamination by Lonza Mycoalert Mycoplasma Detection Kit.</p></section><section id="S78"><h6 class="pmc_sec_title">PIK3R1T576del editing</h6>
<p id="P105">The SpCas9 2NLS nuclease and the CRISPRevolution sgRNA EZ kit were ordered from Synthego corporation. The sequence of the gRNA is: 5’ GGUCUCUCGUCUUUCUCAGC. The Homology-directed repair (HDR) donor ssODN (PAGE purified) and PCR primers was ordered from Sigma-Aldrich. The sequence of ssODN is: 5’ caattattcatgtataggattccatttcaaatacttacATCAAGTATTGGTCTCTtTTcCTgAGCTGGATAAGGTCT GGTTTAATGCTGTTCATACGTTTGTCAATTTCTCGATACT. The PCR primers used for screening edited alleles are: 5’ CCAGCTcAGgAAaAGACC (<em>PIK3R1</em>-T576del-F2) and 5’ gcaatcaccaattattcatg (<em>PIK3R1</em>-R4). The primers used to amplify the edited region for Sanger sequencing and deep sequencing are: 5’ AGAAGACTTGAAGAAGCAGG (<em>PIK3R1</em>-1) and 5’ aactcatcctgaattgtagc (<em>PIK3R1</em>-2).</p>
<p id="P106">2×10<sup>5</sup> HEC-151 cells resuspended in 5μl resuspension buffer (in Neon<sup>™</sup> Transfection System 10 μL Kit, ThermoFisher) were mixed with sgRNA (90pmol), Cas9 (10pmol), and ssODN (60pmol) in 7μl resuspension buffer. 10μl of the resulting mixture was electroporated using Neon Transfection System (ThermoFisher) with the setting of 1400 volts, 10ms, and 4 pulses. Cells were then transferred into one well in a 24-well-dish and recovered in full medium for 2 days. Cells were then seeded in a 96-well-dish with 10~20 cells/well. When cells reached confluency, a screening PCR was conducted by using 10% of the cells from each well in a direct PCR assay using primers specifically amplifying the edited allele and Terra<sup>™</sup> PCR Direct Red Dye Premix (TaKaRa). The remaining cells were expanded for further analyses later. After a positive sub-pool was identified, a Sanger sequencing and Deep sequencing further confirmed the editing. For these two assays, the 172bp fragment spanning the edited region was amplified by PCR using the cells from the positive wells, primers amplifying the 172bp spanning the edited region, and Terra<sup>™</sup> PCR Direct Red Dye Premix. The purified PCR product was sent to Azenta Life Sciences for Sanger sequencing and Amplicon-EZ Next Generation sequencing, respectively.</p></section><section id="S79"><h6 class="pmc_sec_title">Western blotting</h6>
<p id="P107">Western blotting has been previously described <sup><a href="#R107" class="usa-link" aria-describedby="R107" aria-expanded="false">107</a></sup> with a few modifications. Cell lysate protein (30μg) was loaded into each well of a 4–15% Mini-Protein TGX Precast gel (Bio-Rad). The first antibodies used were rabbit anti–phospho-AKT (Thr308) (Cell Signaling Technology, RRIRAB_329825, 1:500), rabbit anti-phospho-AKT (Ser473) (Cell Signaling Technology, RRID:AB 329825, 1:1000 dilution), rabbit anti-AKT (Cell Signaling Technology, RRID:AB_329827, 1:1000 dilution), and mouse anti-β-actin (Sigma-Aldrich, RRID:AB_476744, 1:5000 dilution).</p></section><section id="S80"><h6 class="pmc_sec_title">In vitro Metformin sensitivity assays</h6>
<p id="P108">Established UCEC cell lines were obtained from either the JCRB Cell Bank (HEC265, HEC-108, HEC-251) or the American Type Culture Collection (HEC1A) and cultured in MEM media supplemented with 15% FBS. Identity of each cell line was verified by a PCR-based method for Short Tandem Repeat (STR) profiling prior to receipt. Once thawed, cell lines were cultured for fewer than 20 passages and regularly screening for mycoplasma infection by two independent methods: MycoStrip (Rep-mys-10; Invivogen) and RT-PCR (Venor GeM, Cat MP00025–1KT; Sigma-Millipore). To assess metformin sensitivity, all cell lines were serum starved for 24 hours before being aliquoted in 96-well plates and incubated in MEM supplemented with 5% FBS and metformin at specified doses. Metformin (PHR 1085, Sigma Millipore) was prepared fresh in MEM media for each use. Cell counts were quantified using Cell Titer96 reagent (Promega; Cat# G3580) as previously described (Delaney MA, et al, 2017).</p>
<p id="P109">Gene expression was evaluated using validated Taqman assays specific for c-myc (Hs00153408_m1) as well as putative myc-regulated targets identified by our interrogation of DEPMAP data: NOP2 (Hs00999660_m1), NOP16 (Hs00212622_m1), NOLC-1 (Hs01102319_g1); WDR74 (Hs00276510_m1), PES1 (Hs00362795_g1), IMP4 (Hs00369187), GRWD1 (Hs00230365_m1) and TBRG4 (Hs01056260_g1). Total RNA was prepared from flash frozen pellets of washed cells using the MirVANA kit (AM1561; Fisher Scientific) as previously described (Delaney MA, et al, 2017). Relative levels of transcript were quantified by the delta-deltaC<sub>T</sub> method (Livak KJ and Schmittgen TD, 2001).</p>
<p id="P110">Western blotting was used to assess levels of expression for proteins of interest as described (Delaney MA, et al, 2017) with the following modifications. Thirty micrograms of cell lysate was loaded into each well of a 4–12% polyacrylamide gel (Invitrogen) and transferred to PVDF membranes. Membranes were then incubated with primary antibody for c-myc (Cat#E5Q6W, Cell Signaling; 1:1000 dilution;). Secondary HRP-conjugated anti-rabbit antibody (Cat#7974S; Cell Signaling) was used at 1:3000 dilution. Immunoreactivity was visualized using ECL Western PICO (Fisher Scientific). Specimen-specific immunoreactivity for GADPH (Cat#G9545; Sigma-Millipore; 1:3000 dilution), was used as a loading control. Results are reported as levels of luminescence normalized to loading control for each lane of interest. Results are reported as relative levels of expression normalized to baseline/controls.</p>
<p id="P111">Both a previously validated E-box luciferase reporter vector (pGL2, pGL2-E4) and CMV-beta-gal vector were obtained from Dr. Rosalie Sears (Oregon Health and Science University) and used to evaluate c-myc transcriptional activity <em>in vitro</em> as described (Hurlin, PJ, et al, 1997). In brief, 3 × 10<sup>5</sup> cells from each UCEC cell line were co-transfected with either 200 ng of pGL2 (non-expressing control) or pGL2-E4 and 100 ng b-galactosidase reporter plasmid using lipofectamine 2000 (Cat#: 11668027, ThermoFisher) in Opti-MEM media (Cat#31985062, ThermoFisher). After 5 minutes, media bathing transfected cells were replaced and cells incubated for 24 hours. Each transfected culture was divided and used to measure either luciferase (E1500, Promega) or b-galactosidase activity (E2000, Promega) using a manufacturer recommended protocol. After subtracting background, results are reported as luciferase activity normalized to b-galactosidase activity for each cell pellet.</p>
<p id="P112">For all studies, statistical significance was assessed using Student’s T-test, with significance set at p&lt;0.05.</p></section></section></section></section><section id="S81"><h3 class="pmc_sec_title">Other proteogenomic analyses</h3>
<section id="S82"><h4 class="pmc_sec_title">Mutation impact on the proteome</h4>
<p id="P113">To assess the impact of somatic mutations on protein and phosphorylation levels, we performed cis/trans analysis as detailed in <a href="https://github.com/PayneLab/GBM_confirmatory" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PayneLab/GBM_confirmatory</a>. Briefly, samples were segregated into wildtype or mutation for a specific gene. Cis analysis then performed a Wilcoxon rank sum test to determine whether the samples with a mutation had a significantly different protein level for the mutated gene. Trans analysis was similar, with samples separated into wildtype and mutated. Then all proteins were tested to see if a significantly different level was observed (with a BH corrected FDR of 5%). See the GitHub repository for explicit code used in the manuscript.</p></section><section id="S83"><h4 class="pmc_sec_title">APM subgroup prediction</h4>
<p id="P114">The R package caret (v6.0.88) <sup><a href="#R86" class="usa-link" aria-describedby="R86" aria-expanded="false">86</a></sup> was used to train and predict APM status using SRM data. There are 35 TMB high samples in the Exploratory cohort with 33 features in total. Among them, there are 20 positive samples (TMB_H/APM_H) and 15 negative samples (TMB_H/APM_L). The buil-tin backwards feature selection method, recursive feature elimination, was used to sort the 33 features based on their importance with 10-fold cross-validation and 5 times repeating. After that, 10 different classic machine learning methods with default parameters were used to train the classification model on the Exploratory cohort by adding peptides one at a time, starting with the peptide with the highest feature importance. These methods are AdaBoost, Bayesian Generalized Linear Model (Bayesglm), Logistic Model Trees (LMT), Boosted Logistic Regression (LogitBoost), Penalized Multinomial Regression (Multinom), Oblique Random Forest (ORFlog), Penalized Logistic Regression (plr), Random Forest (ranger, rf), and Regularized Random Forest (RRF). Among them, LMT, LogitBoost and plr are three different versions of logistic regression. ORFlog, parRF, ranger, rf and RRF are five different versions of random forest. Then, the trained models were applied to the independent cohort and AUC values were used to evaluate the performances of these methods.</p></section><section id="S84"><h4 class="pmc_sec_title">Inferred Immune, MYC, APM, and Wnt pathway scores</h4>
<p id="P115">All scores were inferred by single sample gene set enrichment analysis (ssGSEA) method from the GSVA R package <sup><a href="#R62" class="usa-link" aria-describedby="R62" aria-expanded="false">62</a>,<a href="#R108" class="usa-link" aria-describedby="R108" aria-expanded="false">108</a></sup>. The immune signatures are from <sup><a href="#R109" class="usa-link" aria-describedby="R109" aria-expanded="false">109</a></sup>, MYC hallmark gene sets were used as MYC signature, and the KEGG antigen processing and presentation pathway gene set is used as the APM signature. The KEGG Wnt Signaling Pathway gene set was used to analyze the Wnt pathway signatures for our CTNNB1-mutated and WT tumors. mRNA expression was used to infer MYC and immune scores and protein level was used to infer the APM score.</p></section><section id="S85"><h4 class="pmc_sec_title">CTNNB1 hotspot mutation prediction</h4>
<p id="P116">The glm.net package in R was used to train and test lasso regression models to classify hotspot CTNNB1 mutations. Protein data from the exploratory CPTAC UCEC cohort (Dou et al. Cell 2020) was used for training and the current independent dataset was used as testing. Four lasso regression models were run using (1) all samples and all available proteins, (2) CNV-low samples only and all available proteins, (3) all samples and Wnt-β-catenin signaling genes (<span class="text-underline">MSigDB</span>), and (4) CNV-low samples and Wnt-β-catenin signaling genes. Before training the models, feature selection was performed using a two-sided t-test to determine top 10 genes from either all available proteins or Wnt-β-catenin signaling geneset. The top ten genes were then used in training and testing models to classify hotspot mutation or WT tumors. ROC curves were plotted using the pROC package in R (<a href="https://cran.r-project.org/web/packages/pROC/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/pROC/index.html</a>).</p></section><section id="S86"><h4 class="pmc_sec_title">PANOPLY analysis</h4>
<p id="P117">Proteomics data – including global proteome, phosphoproteome, acetylome and glycoproteome – along with RNAseq gene expression and CNV data from the V3.0 data freeze were reformatted into GCT files (<a href="https://clue.io/connectopedia/gct_format" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://clue.io/connectopedia/gct_format</a>) and input to PANOPLY v1.1. PANOPLY <sup><a href="#R110" class="usa-link" aria-describedby="R110" aria-expanded="false">110</a></sup> is a cloud-based platform for automated and reproducible proteogenomic data analysis implementing a wide array of algorithms.</p>
<p id="P118">In the first round of analysis, we focused on <span class="text-underline">multi-omic clustering</span> using all the proteomics, RNAseq and CNV data as input. Based on preliminary exploration, the number of clusters <em>k</em><sub><em>min</em></sub> was set to 4. Clustering was performed using all features from all -omes (balance_omes=false), filtered to exclude 5% of features globally with the lowest standard deviation (sd_filt=0.05 with filt_mode=global).</p>
<p id="P119">Once the clusters were defined, over-representation analysis was performed to determine enrichments for various selected sample annotation terms in the clusters using Fisher’s exact test on the set of samples forming the cluster core. Functional characterization of the clusters was performed using gene set enrichment analysis (GSEA) on the matrix of signed multi-omic feature weights (W). Details are available in the PANOPLY documentation Wiki (<a href="https://github.com/broadinstitute/PANOPLY/wiki" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/broadinstitute/PANOPLY/wiki</a>).</p>
<p id="P120">Subsequently, the sample annotation was updated to include the multi-omic cluster ID for each sample. This updated sample annotation along with the proteogenomics datasets previously described are then used as input to a subsequent run of the PANOPLY pipeline to perform <span class="text-underline">immune analysis</span>, <span class="text-underline">outlier analysis</span>, <span class="text-underline">association analysis</span> and <span class="text-underline">connectivity map analysis</span> among others. Each analysis outputs a report summarizing results in addition to detailed plots and tables.</p></section><section id="S87"><h4 class="pmc_sec_title">Neural networks for subtype and mutation predictions</h4>
<p id="P121">Digital whole slide images (WSI) stained with hematoxylin and eosin (H&amp;E) were obtained as SVS files. TCGA, CPTAC Exploratory (Dou et al. 2020), and CPTAC independent cohort images were obtained from the The Cancer Imaging Archive (TCIA). NYU slides were obtained from the NYU Center for Biospecimen Research &amp; Development (CBRD, RRID:SCR_017930). Slides were scanned at either 40x (TCGA cohort) or 20x (CPTAC and NYU cohorts). Due to the size and the multi-resolution data structure of the WSI, the Python package, Openslide, was used to cut slides into 299 × 299 pixel tiles at 10x, 5x, and 2.5x equivalent magnification of the scanned WSI. 10x, 5x, and 2.5x tiles covering the same regions were then grouped into tile sets and were treated as 1 sample for multi-resolution models following the Panoptes sample preparation (Hong et al Cell Rep Med 2021, PMID: 34622237). In short, tiles were excluded if more than 40% of the pixels are of white background and irrelevant contaminants. Stain colors were normalized using the method described in Vahadane et al. 2016 (PMID: 27164577). Labels for each task were one-hot encoded at per-tile level. Multi-resolution and single-resolution convolutional neural network (CNN) imaging models were trained on TCGA and CPTAC exploratory cohorts, which were randomly split into training, validation, and testing at a per-patient ratio of 8:1:1. The CPTAC independent dataset was used for completely independent testing of the trained models.</p>
<p id="P122">For each task, 13 model architectures were applied to the data. These include Inception V1, V2, V3, InceptionResnet V1, V2, and Panoptes 1, 2, 3, 4 with and without a clinical variable branch (I1, I2, I3, I5, I6, X2, X1, X4, X3, X2CF, X1CF, X4CF, X3CF, respectively). Inception and InceptionResnet are established deep CNN classifiers (Szegedy et al. 2017, DOI: <a href="https://doi.org/10.1609/aaai.v31i1.11231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1609/aaai.v31i1.11231</a>). The Panoptes architecture is a multi-resolution InceptionResnet-based CNN architecture that takes in three tiles of the same region on each slide (Hong et al 2021, PMID: 34622237). Panoptes1 and Panoptes2 have three branches based on InceptionResnet1 and InceptionResnet1, respectively. Panoptes3 and Panoptes4 use the same architecture as Panoptes1 and 2, respectively, and contain an additional 1-by-1 convolutional layer between the concatenation of branches and the global average pooling. Models with an added clinical feature layer were fed patient age and BMI (Panoptes models + CF).</p>
<p id="P123">Receiver Operating Characteristic (ROC) curve, plotting true positive rate against false positive rate, and the area under the ROC curve (AUROC), were calculated using the R package PRROC (<a href="https://cran.r-project.org/web/packages/PRROC/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/PRROC/index.html</a>). 95% confidence intervals were calculated using a 2000 times bootstrap method by randomly sampling true labels and prediction probabilities. Mean AUROC with 95% confidence intervals were plotted for the top performing internal split test architectures per task, top performing internal test architecture for the independent test, and the top performing architecture for the independent test (<a href="#F6" class="usa-link">Figure 6A</a>).</p>
<p id="P124">The best performing models were selected based on the highest AUROC at the per-slide (patient) level. For visualization, activations of 20,000 randomly sampled tiles of the second-to-the-last layer of the test set were extracted and dimensionally reduced and plotted with tSNE for feature visualization. Example tiles were highlighted and sent to pathologists for secondary review. Selected whole slide cases from the test set were fed into the trained model and per-tile level predictions were aggregated into a heatmap layer to overlay onto the original slides for feature visualization and localization.</p>
<p id="P125">To correct for multiple testing, we adapted the method used by Kather et al. 2020<sup><a href="#R111" class="usa-link" aria-describedby="R111" aria-expanded="false">111</a></sup>, and included a list of p-values and false discovery rates (FDR, with Benjamini-Hochberg correction) for top performing models at the slide level (<a href="#SD8" class="usa-link">Table S7</a>). The p-values were obtained from one-sided ttests between the probability values of patients in the positive class and the negative class, for each corresponding model and test datasets.</p>
<p id="P126">To address the limited sample size for the POLE subtype, we obtained 13 positive POLE slides from the NYU Center for Biospecimen Research &amp; Development (CBRD), as well as 78 non-POLE slides to control for batch effect. We did not obtain clinical data for NYU samples other than EC genomic subtype, and therefore could not test or train with any of the CF model architectures. We first tested the X4 architecture trained on TCGA and CPTAC exploratory on the NYU POLE positive samples and CPTAC independent samples, labeled “POLE + NYU Test”. We then trained new models with TCGA, CPTAC exploratory, and all NYU samples, and then applied to the current independent cohort for testing across 4 Panoptes architectures, labeled “POLE + NYU Train”.</p>
<p id="P127">Antigen Presentation Machary (APM) status models were trained using only CPTAC cohorts due to the lack of proteomic data in the TCGA dataset. APM status was derived from a proteomic based scoring metric. We therefore tested two modeling methods for the APM classifier: (1) training on the exploratory cohort and testing on independent, labeled “APM-high”, and (2) training on the combined exploratory and independent cohorts and testing on the internal split, labeled “APM-high Mixed”.</p></section><section id="S88"><h4 class="pmc_sec_title">Transporters score</h4>
<p id="P128">Determining the most distinct separation of tumors into distinct classes along a given scoring axis was completed by Inferred Pathway Activity Score (IPAS) (<a href="https://doi.org/10.1101/2020.10.19.345629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2020.10.19.345629</a>). Pathway activity scores from proteome profiles of EC tumors were determined by using a collection of 8,600 pathways and cell type signatures from various databases: KEGG <sup><a href="#R112" class="usa-link" aria-describedby="R112" aria-expanded="false">112</a></sup>, Reactome <sup><a href="#R112" class="usa-link" aria-describedby="R112" aria-expanded="false">112</a>,<a href="#R113" class="usa-link" aria-describedby="R113" aria-expanded="false">113</a></sup>, HMDB <sup><a href="#R114" class="usa-link" aria-describedby="R114" aria-expanded="false">114</a></sup>, SMPDB <sup><a href="#R114" class="usa-link" aria-describedby="R114" aria-expanded="false">114</a>,<a href="#R115" class="usa-link" aria-describedby="R115" aria-expanded="false">115</a></sup>, Hallmark <sup><a href="#R116" class="usa-link" aria-describedby="R116" aria-expanded="false">116</a></sup>, WikiPathways <sup><a href="#R117" class="usa-link" aria-describedby="R117" aria-expanded="false">117</a></sup>, BioPlanet <sup><a href="#R118" class="usa-link" aria-describedby="R118" aria-expanded="false">118</a></sup>, BioCarta <sup><a href="#R119" class="usa-link" aria-describedby="R119" aria-expanded="false">119</a></sup>, xCell <sup><a href="#R120" class="usa-link" aria-describedby="R120" aria-expanded="false">120</a></sup>, Tabula Sapiens <sup><a href="#R121" class="usa-link" aria-describedby="R121" aria-expanded="false">121</a></sup>. Each pathway score was treated as a dimension and the tumors were ranked along the axis by increasing scoring value. At each point <em>i</em> of the axis, geometrically averaged P values were computed between distributions of individual protein levels in class [1,<em>i</em>] and class [<em>i</em>+1,N], (where N is a number of tumors) and determined the minimum of the geometrically averaged P value. It is assumed that the minimal value the averaged P determines the most distinct separation of given molecular profiles (tumors) along a given axis. Mann Whitney Wilcoxon test was used for computing individual P values.</p>
<p id="P129">To assess significance of the obtained optimal classes of tumors, 10,000 randomly formed tumor classes of the same sizes were generated. For each randomly formed tumor class, the combined P values <sup><a href="#R122" class="usa-link" aria-describedby="R122" aria-expanded="false">122</a>,<a href="#R123" class="usa-link" aria-describedby="R123" aria-expanded="false">123</a></sup> and the actual histograms of P values were computed and compared to combined P values and P value histograms of the optimal classes. Both the number of times combined P values of randomly selected formed tumor classes were less than the combined P values of original classes and the number of times individual P values of a certain power (eg. P &lt; 10<sup>−6</sup>) in random classes were less than P values of the same power in original classes were obtained. The obtained numbers were divided by the number of tests (10<sup>4</sup>), and significance was assessed of the obtained optimal classes <sup><a href="#R123" class="usa-link" aria-describedby="R123" aria-expanded="false">123</a></sup>. Mutation enrichment in a specific class (eg. Transporters) was performed by Fisher test.</p></section><section id="S89"><h4 class="pmc_sec_title">SRM data batch effect correction</h4>
<p id="P130">The ICI treated samples are formalin-fixed paraffin-embedded specimens which are different to the fresh frozen method used by CPTAC. Combat method implemented in the R package ez.combat was used to do the batch effect correction with default parameters <sup><a href="#R87" class="usa-link" aria-describedby="R87" aria-expanded="false">87</a></sup>. Peptides with more than 50% NAs were removed, and then remaining NAs were set to the minimum values of non-NA numbers. After batch effect correction, these values were reset to NAs.</p></section><section id="S90"><h4 class="pmc_sec_title">QUANTIFICATION AND STATISTICAL ANALYSIS</h4>
<p id="P131">Data analysis and visualization was done with a diverse set of methods depending on which data modality was analyzed. The methods are described in the methods details section. When multiple tests were performed p-values were corrected for multiple testing using the Benjamini-Hochberg method to control for the false discovery rate.</p></section></section></section><section id="SM1"><h2 class="pmc_sec_title" data-anchor-id="SM1"><a id="SM1-anchor" data-anchor-id="SM1" class="usa-anchor"></a>Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>1</span></div>
<div class="media p" id="d64e2829"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-1.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-1.pdf</a><sup> (8.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p">
<span>2</span><p id="P132">Table S1. Genotype, phenotype, immune scores, clinical, and pathological characteristics of tumor and normal samples. Related to <a href="#F1" class="usa-link">Figure 1</a>.</p>
</div>
<div class="media p" id="d64e2838"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-2.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-2.xlsx</a><sup> (270.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p">
<span>3</span><p id="P133">Table S2. Mutation signatures, CNV drivers, CNV arm immune score correlations, and PARP-i drug sensitivity data. Related to <a href="#F1" class="usa-link">Figures 1</a> and <a href="#F6" class="usa-link">6</a>.</p>
</div>
<div class="media p" id="d64e2850"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-3.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-3.xlsx</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD4"><div class="caption p">
<span>4</span><p id="P134">Table S3. <em>PIK3R1</em> mutation status, AKT inhibitor response of EC cell lines, and sequencing of CRISPR-Cas9 edited cells. Related to <a href="#F2" class="usa-link">Figure 2</a>.</p>
</div>
<div class="media p" id="d64e2862"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-4.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-4.xlsx</a><sup> (563.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD5"><div class="caption p">
<span>5</span><p id="P135">Table S4. Immune subgroup, immune scores and SRM peptide levels for ICI treated samples. Related to <a href="#F3" class="usa-link">Figure 3</a>.</p>
</div>
<div class="media p" id="d64e2871"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-5.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-5.xlsx</a><sup> (111KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD6"><div class="caption p">
<span>6</span><p id="P136">Table S5. Inferred MYC activity in CMAP, TCGA, Exploratory and Independent cohorts, total MYC IHC scores. Related to <a href="#F4" class="usa-link">Figure 4</a>.</p>
</div>
<div class="media p" id="d64e2880"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-6.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-6.xlsx</a><sup> (55.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD7"><div class="caption p">
<span>7</span><p id="P137">Table S6. Transporters scores and classes, <em>CTNNB1</em> mutation protein fold change, and top classifying genes for hotspot mutation. Related to <a href="#F5" class="usa-link">Figure 5</a>.</p>
</div>
<div class="media p" id="d64e2892"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-7.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-7.xlsx</a><sup> (625.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD8"><div class="caption p">
<span>8</span><p id="P138">Table S7. Deep learning models, multi-omic NMF clustering, and glycopeptide NMF clustering. Related to <a href="#F6" class="usa-link">Figures 6</a> and <a href="#F7" class="usa-link">7</a>.</p>
</div>
<div class="media p" id="d64e2904"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-8.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-8.xlsx</a><sup> (176.3KB, xlsx) </sup>
</div></div></section></section><section id="floats-group1" class="floats-group"><section class="tw xbox font-sm" id="T1"><div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">REAGENT or RESOURCE</th>
<th align="left" valign="top" rowspan="1" colspan="1">SOURCE</th>
<th align="left" valign="top" rowspan="1" colspan="1">IDENTIFIER</th>
</tr></thead>
<tbody>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Antibodies</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">anti–phospho-AKT (S473)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 9271</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">anti–phospho-AKT (T308)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 9275</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Anti- GAPDH</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: G9545</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">anti-myc</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: E5Q6W</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">rabbit anti-AKT</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 9272</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">anti–β-actin</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma-Aldrich</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: A5441</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HRP-conjugated anti-rabbit antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 7074S</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Biological Samples</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Primary tumor and normal tissue samples</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Patient-derived xenograft tissue samples</td>
<td align="left" valign="top" rowspan="1" colspan="1">Washington University in St. Louis</td>
<td align="left" valign="top" rowspan="1" colspan="1">Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>
</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Chemicals, peptides, and recombinant proteins</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: H3375</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ammonium bicarbonate</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 9830</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ammonium formate</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 9735</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Aprotinin</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: A6103</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Calcium chloride</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: C1016</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Dithiothreitol</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 20291</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ethylenediaminetetraacetic acid</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: E7889</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Fetal Bovine Serum</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: F7524</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Formic acid</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 33015</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Iodoacetamide</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: A3221</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Iron (III) chloride</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 451649</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Leupeptin</td>
<td align="left" valign="top" rowspan="1" colspan="1">Roche</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 11017101001</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lipofectamine 2000</td>
<td align="left" valign="top" rowspan="1" colspan="1">Invitrogen</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 52785</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lysyl Endopeptidase</td>
<td align="left" valign="top" rowspan="1" colspan="1">Wako Chemicals</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 129–02541</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Metformin</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: PHR 1084</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MOPS buffer</td>
<td align="left" valign="top" rowspan="1" colspan="1">Invitrogen</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: NP0001</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ni-NTA Superflow Agarose Beads</td>
<td align="left" valign="top" rowspan="1" colspan="1">Qiagen</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 30410</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">OptiMem Media</td>
<td align="left" valign="top" rowspan="1" colspan="1">ThermoFisher</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 31985062</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Phenylmethylsulfonyl fluoride</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 93482</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Phosphatase Inhibitor Cocktail 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: P5726</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Phosphatase Inhibitor Cocktail 3</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: P0044</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Polyvinylidene fluoride membrane (PVDF)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Invitrogen</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: IB401002</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequencing grade modified trypsin</td>
<td align="left" valign="top" rowspan="1" colspan="1">Promega</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: V517</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sodium butyrate</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 303410</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sodium chloride</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: S7653</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sodium fluoride</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: S7920</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Urea</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: U0631</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Acetonitrile, Optima LC/MS</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Chemical</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: A955–4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Water, Optima LC/MS</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Chemical</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: W6–4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Trifluoroacetic acid</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 302031</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Triethylammonium acetate buffer</td>
<td align="left" valign="top" rowspan="1" colspan="1">Sigma</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 90358</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MEM</td>
<td align="left" valign="top" rowspan="1" colspan="1">Corning</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 10–010-CV</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Critical Commercial Assays</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Beta-Galactosidase Assay System</td>
<td align="left" valign="top" rowspan="1" colspan="1">Promega</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: P2000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">BCA Protein Assay Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Scientific Pierce</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/A53225" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A53225</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Titer 96 Assay Reagents</td>
<td align="left" valign="top" rowspan="1" colspan="1">Promega</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: G3580</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Chemiluminescence PICO Assay Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Fisher</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 34578</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Luciferase Assay System</td>
<td align="left" valign="top" rowspan="1" colspan="1">Promega</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: E1500</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">PTMScan Acetyl-Lysine Motif [Ac-K] Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signaling</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 13416</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Reversed-phase tC18 SepPak</td>
<td align="left" valign="top" rowspan="1" colspan="1">Waters</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: WAT054925</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Tandem mass tags - 11plex</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: A34808</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - MYC</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00153408_m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - NOP2</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00099660_m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - NOP16</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00212612 _m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - NOLC1</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs01102319_g1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - WDR74</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00276510_m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - PES1</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00362795_g1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - IMP4</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00369187_m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - GRWD1</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs00230365_m1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Assay - TBRG4</td>
<td align="left" valign="top" rowspan="1" colspan="1">Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hs01056260_g1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Taqman Universal Master Mix</td>
<td align="left" valign="top" rowspan="1" colspan="1">Applied Biosystems</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 444038</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Oasis MAX 1 cc Vac Cartridge, 10 mg Sorbent per Cartridge</td>
<td align="left" valign="top" rowspan="1" colspan="1">Waters</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 186004649</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Mycoalert Mycoplasma Detection Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Lonza</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: LT07–318</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MycoStrip Test Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Invivogen</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: Rep-mys-10</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MycoPlasma RT-PCR Test Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Venor GEM</td>
<td align="left" valign="top" rowspan="1" colspan="1">MP00025</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SpCas9 nuclease 2NLS</td>
<td align="left" valign="top" rowspan="1" colspan="1">Synthego</td>
<td align="left" valign="top" rowspan="1" colspan="1">SpCas9 2NLS Nuclease (300 pmol)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CRISPRevolution sgRNA EZ kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">Synthego</td>
<td align="left" valign="top" rowspan="1" colspan="1">Customized</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Neon Transfection System 10 μL Kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">ThermoFisher</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: MPK1025</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Neon Transfection System</td>
<td align="left" valign="top" rowspan="1" colspan="1">ThermoFisher</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: MPK5000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Terra PCR Direct Red Dye Premix</td>
<td align="left" valign="top" rowspan="1" colspan="1">TaKaRa</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 639270</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Mini-Protein TGX Precast gel</td>
<td align="left" valign="top" rowspan="1" colspan="1">Bio-Rad</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog: 4561086</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Deposited data</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CPTAC Raw Genomic data</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper and Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">dbGaP phs001287</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CPTAC Raw Proteomic data</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper and Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">PDC000125, PDC000126, PDC000226</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CPTAC and TCGA Processed data tables</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper, Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>, and The Cancer Genome Atlas et al.<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://www.linkedomics.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.linkedomics.org/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CPTAC Image data</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper and Dou et al.<sup><a href="#R9" class="usa-link" aria-describedby="R9" aria-expanded="false">9</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">10.7937/k9/tcia.2018.3r3juisw <a href="https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TCGA Raw Genomic data</td>
<td align="left" valign="top" rowspan="1" colspan="1">The Cancer Genome Atlas et al.<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">dbGaP phs000178</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TCGA Image data</td>
<td align="left" valign="top" rowspan="1" colspan="1">The Cancer Genome Atlas et al.<sup><a href="#R8" class="usa-link" aria-describedby="R8" aria-expanded="false">8</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">10.7937/k9/tcia.2016.gkj0zwac <a href="https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=tcga_ucec" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=tcga_ucec</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CCLE DepMap</td>
<td align="left" valign="top" rowspan="1" colspan="1">Corsello et al.<sup><a href="#R17" class="usa-link" aria-describedby="R17" aria-expanded="false">17</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://depmap.org/portal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/</a>
</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Experimental models: Cell lines</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HEC151 UCEC cell line</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB cell bank</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB1122</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HEC251 UCEC cell line</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB cell bank</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB1141</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HEC265 UCEC cell line</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB cell bank</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB1142</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HEC108 UCEC cell line</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB cell bank</td>
<td align="left" valign="top" rowspan="1" colspan="1">JCRB1123</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HEC1A UCEC cell line</td>
<td align="left" valign="top" rowspan="1" colspan="1">American Type Culture Collection</td>
<td align="left" valign="top" rowspan="1" colspan="1">HTB-112</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Oligonucleotides</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CRISPRevolution sgRNA: 5’GGUCUCUCGUCUUUCUCAGC</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper, Synthego</td>
<td align="left" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Homology-directed repair (HDR) donor ssODN: 5’caattattcatgtataggattccatttcaaatac ttacATCAAGTATTGGTCTCTtTTcC TgAGCTGGATAAGGTCTGGTTTAATGCTGTTCATACGTTTGTCAATTTCTCGATACT</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper, Sigma-Aldrich</td>
<td align="left" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers for PIK3R1 edited alleles: 5’CCAGCTcAGgAAaAGACC (PIK3R1-T576del-F2) and 5’gcaatcaccaattattcatg (PIK3R1-R4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper, Sigma-Aldrich</td>
<td align="left" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sanger sequencing primers for PIK3R1 edited alleles: 5’AGAAGACTTGAAGAAGCAGG (PIK3R1–1) and 5’aactcatcctgaattgtagc (PIK3R1–2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper, Sigma-Aldrich</td>
<td align="left" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Software and Algorithms</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ascore (v1.0.6858)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Beausoleil et al.<sup><a href="#R54" class="usa-link" aria-describedby="R54" aria-expanded="false">54</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/PNNL-Comp-Mass-Spec/AScore/releases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PNNL-Comp-Mass-Spec/AScore/releases</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">BlackSheep</td>
<td align="left" valign="top" rowspan="1" colspan="1">Blumenberg et al.<sup><a href="#R55" class="usa-link" aria-describedby="R55" aria-expanded="false">55</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/ruggleslab/blackSheep" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ruggleslab/blackSheep</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Bowtie2 (v2.3.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Langmead et al.<sup><a href="#R56" class="usa-link" aria-describedby="R56" aria-expanded="false">56</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">BWA (v0.7.17-r1188)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Li et al.<sup><a href="#R57" class="usa-link" aria-describedby="R57" aria-expanded="false">57</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://bio-bwa.sourceforge.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bio-bwa.sourceforge.net/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CIRI (v2.0.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Gao et al.<sup><a href="#R58" class="usa-link" aria-describedby="R58" aria-expanded="false">58</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://sourceforge.net/projects/ciri/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sourceforge.net/projects/ciri/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">cptac (Python package)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Lindgren et al.<sup><a href="#R59" class="usa-link" aria-describedby="R59" aria-expanded="false">59</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://paynelab.github.io/cptac/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://paynelab.github.io/cptac/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">eriscript (v0.5.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Benelli et al.<sup><a href="#R60" class="usa-link" aria-describedby="R60" aria-expanded="false">60</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://sites.google.com/site/bioericscript/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sites.google.com/site/bioericscript/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GPQuest (v2.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Toghi Eshghi et al.<sup><a href="#R61" class="usa-link" aria-describedby="R61" aria-expanded="false">61</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/huizhanglabjhu/GPQuest" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/huizhanglabjhu/GPQuest</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GSVA (v1.30.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Hänzelmann et al.<sup><a href="#R62" class="usa-link" aria-describedby="R62" aria-expanded="false">62</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://bioconductor.org/packages/release/bioc/html/GSVA.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioconductor.org/packages/release/bioc/html/GSVA.html</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">INTEGRATE v(0.2.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Zhang et al.<sup><a href="#R63" class="usa-link" aria-describedby="R63" aria-expanded="false">63</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://sourceforge.net/p/integrate-fusion/wiki/Home/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sourceforge.net/p/integrate-fusion/wiki/Home/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">LinkedOmics</td>
<td align="left" valign="top" rowspan="1" colspan="1">Vasaikar et al.<sup><a href="#R64" class="usa-link" aria-describedby="R64" aria-expanded="false">64</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://www.linkedomics.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.linkedomics.org</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MASIC</td>
<td align="left" valign="top" rowspan="1" colspan="1">Monroe et al.<sup><a href="#R65" class="usa-link" aria-describedby="R65" aria-expanded="false">65</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/PNNL-Comp-Mass-Spec/MASIC/releases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PNNL-Comp-Mass-Spec/MASIC/releases</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">metaX</td>
<td align="left" valign="top" rowspan="1" colspan="1">Wen et al.<sup><a href="#R66" class="usa-link" aria-describedby="R66" aria-expanded="false">66</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/wenbostar/metaX" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/wenbostar/metaX</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MS-GF+</td>
<td align="left" valign="top" rowspan="1" colspan="1">Kim et al.<sup><a href="#R67" class="usa-link" aria-describedby="R67" aria-expanded="false">67</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/MSGFPlus/msgfplus/releases" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MSGFPlus/msgfplus/releases</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MSIsensor (v0.2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Niu et al.<sup><a href="#R68" class="usa-link" aria-describedby="R68" aria-expanded="false">68</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/ding-lab/msisensor" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ding-lab/msisensor</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MS-PyCloud</td>
<td align="left" valign="top" rowspan="1" colspan="1">Chen et al.<sup><a href="#R69" class="usa-link" aria-describedby="R69" aria-expanded="false">69</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://bitbucket.org/mschnau1/mspycloud/src/main/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bitbucket.org/mschnau1/mspycloud/src/main/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MuSiC (v0.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Dees et al.<sup><a href="#R70" class="usa-link" aria-describedby="R70" aria-expanded="false">70</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://gmt.genome.wustl.edu/packages/genome-music/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gmt.genome.wustl.edu/packages/genome-music/index.html</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Mutect (v1.1.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cibulskis et al.<sup><a href="#R71" class="usa-link" aria-describedby="R71" aria-expanded="false">71</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://software.broadinstitute.org/cancer/cga/mutect" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://software.broadinstitute.org/cancer/cga/mutect</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">mzRefinery</td>
<td align="left" valign="top" rowspan="1" colspan="1">Gibbons et al.<sup><a href="#R72" class="usa-link" aria-describedby="R72" aria-expanded="false">72</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://omics.pnl.gov/software/mzrefinery" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://omics.pnl.gov/software/mzrefinery</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Optitype</td>
<td align="left" valign="top" rowspan="1" colspan="1">Szolek et al.<sup><a href="#R73" class="usa-link" aria-describedby="R73" aria-expanded="false">73</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://github.com/FRED-2/OptiType" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://github.com/FRED-2/OptiType</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">PepQuery</td>
<td align="left" valign="top" rowspan="1" colspan="1">Wen et al.<sup><a href="#R74" class="usa-link" aria-describedby="R74" aria-expanded="false">74</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://pepquery.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pepquery.org</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Maftools (2.10.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Mayakonda et al.<sup><a href="#R75" class="usa-link" aria-describedby="R75" aria-expanded="false">75</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/PoisonAlien/maftools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PoisonAlien/maftools</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Pindel (v0.2.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Ye et al.<sup><a href="#R76" class="usa-link" aria-describedby="R76" aria-expanded="false">76</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://gmt.genome.wustl.edu/packages/pindel/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gmt.genome.wustl.edu/packages/pindel/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">PROGENy</td>
<td align="left" valign="top" rowspan="1" colspan="1">Schubert et al.<sup><a href="#R77" class="usa-link" aria-describedby="R77" aria-expanded="false">77</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://bioconductor.org/packages/release/bioc/html/progeny.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioconductor.org/packages/release/bioc/html/progeny.html</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">pyQUILTS (v1.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Ruggles et al.<sup><a href="#R78" class="usa-link" aria-describedby="R78" aria-expanded="false">78</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://quilts.fenyolab.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://quilts.fenyolab.org</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">RSEM (v1.3.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Li et al.<sup><a href="#R79" class="usa-link" aria-describedby="R79" aria-expanded="false">79</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://deweylab.github.io/RSEM/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://deweylab.github.io/RSEM/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SigProfilerMatrixGeneratorR (v1.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Tan et al.<sup><a href="#R80" class="usa-link" aria-describedby="R80" aria-expanded="false">80</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/AlexandrovLab/SigProfilerMatrixGeneratorR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/AlexandrovLab/SigProfilerMatrixGeneratorR</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Skyline (v 21.2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">MacLean et al.<sup><a href="#R81" class="usa-link" aria-describedby="R81" aria-expanded="false">81</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://skyline.ms/project/home/software/Skyline/begin.view" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://skyline.ms/project/home/software/Skyline/begin.view</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">STAR-Fusion (v1.6.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Haas et al.<sup><a href="#R82" class="usa-link" aria-describedby="R82" aria-expanded="false">82</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/STAR-Fusion/STAR-Fusion/wiki" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/STAR-Fusion/STAR-Fusion/wiki</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Strelka (v2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Saunders et al.<sup><a href="#R83" class="usa-link" aria-describedby="R83" aria-expanded="false">83</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/Illumina/strelka" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Illumina/strelka</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Varscan (v2.3.8)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Koboldt et al<sup><a href="#R84" class="usa-link" aria-describedby="R84" aria-expanded="false">84</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://varscan.sourceforge.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://varscan.sourceforge.net/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">XGboost (v0.81)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Chen et al.<sup><a href="#R85" class="usa-link" aria-describedby="R85" aria-expanded="false">85</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://xgboost.readthedocs.io/en/latest/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://xgboost.readthedocs.io/en/latest/</a>
</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Caret (v6.0.88)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Kuhn et al.<sup><a href="#R86" class="usa-link" aria-describedby="R86" aria-expanded="false">86</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://topepo.github.io/caret/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://topepo.github.io/caret/</a>
</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">ez.combat</td>
<td align="left" valign="top" rowspan="1" colspan="1">Johnson et al.<sup><a href="#R87" class="usa-link" aria-describedby="R87" aria-expanded="false">87</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/TKoscik/ez.combat" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TKoscik/ez.combat</a>
</td>
</tr>
<tr><td colspan="3" align="left" valign="top" rowspan="1">
<strong>Other</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CPTAC Data Portal</td>
<td align="left" valign="top" rowspan="1" colspan="1">Edwards et al.<sup><a href="#R88" class="usa-link" aria-describedby="R88" aria-expanded="false">88</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://cptac-data-portal.georgetown.edu/cptacPublic" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cptac-data-portal.georgetown.edu/cptacPublic</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CTdatabase</td>
<td align="left" valign="top" rowspan="1" colspan="1">Chen et al.<sup><a href="#R89" class="usa-link" aria-describedby="R89" aria-expanded="false">89</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://www.cta.lncc.br/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.cta.lncc.br</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">dbGaP</td>
<td align="left" valign="top" rowspan="1" colspan="1">Tryka et al.<sup><a href="#R90" class="usa-link" aria-describedby="R90" aria-expanded="false">90</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/gap/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/gap/</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Genomic Data Commons</td>
<td align="left" valign="top" rowspan="1" colspan="1">Grossman et al.<sup><a href="#R91" class="usa-link" aria-describedby="R91" aria-expanded="false">91</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gdc.cancer.gov</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">RefSeq (downloaded from UCSC Genome Browser 2018/06/29)</td>
<td align="left" valign="top" rowspan="1" colspan="1">O’Leary et al.<sup><a href="#R92" class="usa-link" aria-describedby="R92" aria-expanded="false">92</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/refseq/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/refseq/</a>; <a href="https://genome.ucsc.edu/cgibin/hgTables" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/cgibin/hgTables</a>; RRID:SCR_003496</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">UniProt (r2017.06, r2019.01)</td>
<td align="left" valign="top" rowspan="1" colspan="1">UniProt Consortium<sup><a href="#R93" class="usa-link" aria-describedby="R93" aria-expanded="false">93</a></sup>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>; RRID: SCR_002380</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">BioRender</td>
<td align="left" valign="top" rowspan="1" colspan="1">BioRender</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.biorender.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.biorender.com/</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="bt xbox font-sm" id="BX1"><h2 class="obj_head">HIGHLIGHTS.</h2>
<ul id="L1" class="list" style="list-style-type:disc">
<li><p id="P202">APM status can be accurately predicted using a targeted proteomic assay</p></li>
<li><p id="P203">MYC activity is a potential biomarker for metformin treatment</p></li>
<li><p id="P204"><em>PIK3R1</em> in-frame indels are associated with upregulated AKT1 phosphorylation</p></li>
<li><p id="P205"><em>CTNNB1</em> exon 3 hotspot mutations block pS45 induced <strong>𝛃</strong>-catenin degradation</p></li>
</ul></section></section><section id="S91" class="ack"><h2 class="pmc_sec_title" data-anchor-id="S91"><a id="S91-anchor" data-anchor-id="S91" class="usa-anchor"></a>ACKNOWLEDGEMENTS</h2>
<p id="P139">The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is supported by the National Cancer Institute of the National Institutes of Health under award numbers U24CA210955, U24CA210985, U24CA210986, U24CA210954, U24CA210967, U24CA210972, U24CA210979, U24CA210993, U01CA214114, U01CA214116, U01CA214125, U24CA271012, U24CA271076, U24CA271114, and contracts from Leidos (S21-167 and 21X164Q); by NCI grant T32CA203690; by grant RR160027 from the Cancer Prevention &amp; Research Institutes of Texas (CPRIT); and by funding from the McNair Medical Institute at The Robert and Janice McNair Foundation. B.Z. and M.E. are Cancer Prevention &amp; Research Institutes of Texas Scholars in Cancer Research and McNair Medical Institute Scholars. The Pacific Northwest National Laboratory (PNNL) proteomics work described herein was performed in the Environmental Molecular Sciences Laboratory (grid.436923.9), a US Department of Energy (DOE) National Scientific User Facility located at PNNL in Richland, WA. PNNL is a multi-program national laboratory operated by the Battelle Memorial Institute for the DOE under contract DE-AC05-76RL01830. This work was also supported by a US Department of Defence funding DOD BC220523.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN7">
<p id="P140">DECLARATION OF INTERESTS</p>
<p id="P141">The authors declare no competing interests.</p>
</div>
<div class="fn p" id="FN8">
<p id="P142">CONSORTIA</p>
<p id="P143">Eunkyung An, Matthew L. Anderson, Andrzej Antczak, Meenakshi Anurag, Rebecca C. Arend, Thomas Bauer, Jasmin Bavarva, Chet Birger, Michael J. Birrer, Melissa Borucki, Wen Bu, Shuang Cai, Anna Calinawan, Steven A. Carr, Patricia Castro, Sandra Cerda, Daniel W. Chan, Lijun Chen, David Chesla, Arul M. Chinnaiyan, Rosalie K. Chu, Marcin P. Cieslik, Sandra Cottingham, Andrzej Czekański, Teresa Davoli, Deborah DeLair, Elizabeth G. Demicco, Kelly Devereaux, Saravana M. Dhanasekaran, Rajiv Dhir, Li Ding, Marcin J. Domagalski, Peter Dottino, Yongchao Dou, Bailee Dover, Brian J. Druker, Elizabeth Duffy, Nathan J. Edwards, Robert Edwards, Matthew J. Ellis, Jennifer Eschbacher, Mina Fam, David Fenyö, Brenda Fevrier-Sullivan, Thomas L. Fillmore, McKenzie Foxall, Jesse Francis, John Freymann, Stacey Gabriel, Gad Getz, Michael A. Gillette, Andrew K. Godwin, Charles A. Goldthwaite, Pamela Grady, Marina A. Gritsenko, Xiaofang Guo, Jason Hafron, Pushpa Hariharan, Catherine E. Hermann, Tara Hiltke, Barbara Hindenach, Katherine A. Hoadley, Runyu Hong, Galen Hostetter, Yingwei Hu, Jasmine Huang, Michael M. Ittmann, Scott D. Jewell, Ashlie Johnson, Isabelle Johnson, Corbin D. Jones, Marcin Jędryka, Andrea G. Kahn, Lizabeth Katsnelson, Karen A. Ketchum, Justin Kirby, Iga Kolodziejczak, Amy T. Ku, Chandan Kumar-Sinha, Paweł Kurzawa, Alexander J. Lazar, Rossana Lazcano, Toan Le, Jonathan T. Lei, Yi Li, Yuxing Liao, Tung-Shing M. Lih, Tai-Tu Lin, Tao Liu, Wenke Liu, Rita Jui-Hsien Lu, Avi Ma’ayan, Rashna Madan, D. R. Mani, Sailaja Mareedu, John A. Martignetti, Ramya P. Masand, Rafał Matkowski, Peter B. McGarvey, Wilson McKerrow, Mehdi Mesri, Francesmary Modugno, Matthew E. Monroe, Rebecca Montgomery, Jamie Moon, Ronald J. Moore, David Mutch, Michael D. Nestor, Alexey I. Nesvizhskii, Chelsea Newton, Kristen Nyce, Tatiana Omelchenko, Gilbert S. Omenn, Amanda G. Paulovich, Samuel H. Payne, Vladislav A. Petyuk, Barbara L. Pruetz, Liqun Qi, Boris Reva, Shannon Richey, Ana I. Robles, Karin D. Rodland, Henry Rodriguez, Kelly V. Ruggles, Dmitry Rykunov, Sara R. Savage, Eric E. Schadt, Athena A. Schepmoes, Tujin Shi, Zhiao Shi, Yvonne Shutack, Shilpi Singh, Michael Smith, Richard D. Smith, Jimin Tan, Darlene Tansil, Mason Taylor, Ratna R. Thangudu, Mathangi Thiagarajan, Matt Tobin, Chia-Feng Tsai, Ki Sung Um, Negin Vatanian, Joshua M. Wang, Pei Wang, Yi-Ting Wang, Alex Webster, Karl K. Weitz, Bo Wen, C. M. Williams, George D. Wilson, Maciej Wiznerowicz, Jason Wright, Yige Wu, Matthew A. Wyczalkowski, Xinpei Yi, Kakhaber Zaalishvili, Bing Zhang, Hui Zhang, Xu Zhang, Zhen Zhang, Grace Zhao, Rui Zhao</p>
</div>
<div class="fn p" id="FN9"><p id="P144"><strong>Publisher's Disclaimer:</strong> This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p></div>
</div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Minihan AK, Patel AV, Flanders WD, Sauer AG, Jemal A, and Islami F. (2022). Proportion of Cancer Cases Attributable to Physical Inactivity by US State, 2013–2016. Med. Sci. Sports Exerc. 54, 417–423.</cite> [<a href="https://doi.org/10.1249/MSS.0000000000002801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34628449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Sci.%20Sports%20Exerc&amp;title=Proportion%20of%20Cancer%20Cases%20Attributable%20to%20Physical%20Inactivity%20by%20US%20State,%202013%E2%80%932016&amp;author=AK%20Minihan&amp;author=AV%20Patel&amp;author=WD%20Flanders&amp;author=AG%20Sauer&amp;author=A%20Jemal&amp;volume=54&amp;publication_year=2022&amp;pages=417-423&amp;pmid=34628449&amp;doi=10.1249/MSS.0000000000002801&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Crosbie EJ, Kitson SJ, McAlpine JN, Mukhopadhyay A, Powell ME, and Singh N. (2022). Endometrial cancer. Lancet
399, 1412–1428.</cite> [<a href="https://doi.org/10.1016/S0140-6736(22)00323-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35397864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Endometrial%20cancer&amp;author=EJ%20Crosbie&amp;author=SJ%20Kitson&amp;author=JN%20McAlpine&amp;author=A%20Mukhopadhyay&amp;author=ME%20Powell&amp;volume=399&amp;publication_year=2022&amp;pages=1412-1428&amp;pmid=35397864&amp;doi=10.1016/S0140-6736(22)00323-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Zhang S, Gong T-T, Liu F-H, Jiang Y-T, Sun H, Ma X-X, Zhao Y-H, and Wu Q-J (2019). Global, Regional, and National Burden of Endometrial Cancer, 1990–2017: Results From the Global Burden of Disease Study, 2017. Front. Oncol. 9, 1440.</cite> [<a href="https://doi.org/10.3389/fonc.2019.01440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6930915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31921687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol&amp;title=Global,%20Regional,%20and%20National%20Burden%20of%20Endometrial%20Cancer,%201990%E2%80%932017:%20Results%20From%20the%20Global%20Burden%20of%20Disease%20Study,%202017&amp;author=S%20Zhang&amp;author=T-T%20Gong&amp;author=F-H%20Liu&amp;author=Y-T%20Jiang&amp;author=H%20Sun&amp;volume=9&amp;publication_year=2019&amp;pages=1440&amp;pmid=31921687&amp;doi=10.3389/fonc.2019.01440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Temkin SM, Kohn EC, Penberthy L, Cronin KA, Rubinsak L, Dickie LA, Minasian L, and Noone A-M (2018). Hysterectomy-corrected rates of endometrial cancer among women younger than age 50 in the United States. Cancer Causes Control
29, 427–433.</cite> [<a href="https://doi.org/10.1007/s10552-018-1018-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29497884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Causes%20Control&amp;title=Hysterectomy-corrected%20rates%20of%20endometrial%20cancer%20among%20women%20younger%20than%20age%2050%20in%20the%20United%20States&amp;author=SM%20Temkin&amp;author=EC%20Kohn&amp;author=L%20Penberthy&amp;author=KA%20Cronin&amp;author=L%20Rubinsak&amp;volume=29&amp;publication_year=2018&amp;pages=427-433&amp;pmid=29497884&amp;doi=10.1007/s10552-018-1018-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Clarke MA, Devesa SS, Hammer A, and Wentzensen N. (2022). Racial and Ethnic Differences in Hysterectomy-Corrected Uterine Corpus Cancer Mortality by Stage and Histologic Subtype. JAMA Oncol
8, 895–903.</cite> [<a href="https://doi.org/10.1001/jamaoncol.2022.0009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9073658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35511145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&amp;title=Racial%20and%20Ethnic%20Differences%20in%20Hysterectomy-Corrected%20Uterine%20Corpus%20Cancer%20Mortality%20by%20Stage%20and%20Histologic%20Subtype&amp;author=MA%20Clarke&amp;author=SS%20Devesa&amp;author=A%20Hammer&amp;author=N%20Wentzensen&amp;volume=8&amp;publication_year=2022&amp;pages=895-903&amp;pmid=35511145&amp;doi=10.1001/jamaoncol.2022.0009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Khouri OR, Frey MK, Musa F, Muggia F, Lee J, Boyd L, Curtin JP, and Pothuri B. (2019). Neoadjuvant chemotherapy in patients with advanced endometrial cancer. Cancer Chemotherapy and Pharmacology
84, 281–285. 10.1007/s00280-019-03838-x.</cite> [<a href="https://doi.org/10.1007/s00280-019-03838-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30980132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemotherapy%20and%20Pharmacology&amp;title=Neoadjuvant%20chemotherapy%20in%20patients%20with%20advanced%20endometrial%20cancer&amp;author=OR%20Khouri&amp;author=MK%20Frey&amp;author=F%20Musa&amp;author=F%20Muggia&amp;author=J%20Lee&amp;volume=84&amp;publication_year=2019&amp;pages=281-285&amp;pmid=30980132&amp;doi=10.1007/s00280-019-03838-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>van den Heerik ASVM, Horeweg N, Nout RA, Lutgens LCHW, van der Steen-Banasik EM, Westerveld GH, van den Berg HA, Slot A, Koppe FLA, Kommoss S, et al.  (2020). PORTEC-4a: international randomized trial of molecular profile-based adjuvant treatment for women with high-intermediate risk endometrial cancer. Int. J. Gynecol. Cancer
30, 2002–2007.</cite> [<a href="https://doi.org/10.1136/ijgc-2020-001929" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7788476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33046573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Gynecol.%20Cancer&amp;title=PORTEC-4a:%20international%20randomized%20trial%20of%20molecular%20profile-based%20adjuvant%20treatment%20for%20women%20with%20high-intermediate%20risk%20endometrial%20cancer&amp;author=ASVM%20van%20den%20Heerik&amp;author=N%20Horeweg&amp;author=RA%20Nout&amp;author=LCHW%20Lutgens&amp;author=EM%20van%20der%20Steen-Banasik&amp;volume=30&amp;publication_year=2020&amp;pages=2002-2007&amp;pmid=33046573&amp;doi=10.1136/ijgc-2020-001929&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Cancer Genome Atlas Research Network, Kandoth C, Schultz N, Cherniack AD, Akbani R, Liu Y, Shen H, Robertson AG, Pashtan I, Shen R, et al.  (2013). Integrated genomic characterization of endometrial carcinoma. Nature
497, 67–73.</cite> [<a href="https://doi.org/10.1038/nature12113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3704730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23636398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrated%20genomic%20characterization%20of%20endometrial%20carcinoma&amp;author=C%20Kandoth&amp;author=N%20Schultz&amp;author=AD%20Cherniack&amp;author=R%20Akbani&amp;author=Y%20Liu&amp;volume=497&amp;publication_year=2013&amp;pages=67-73&amp;pmid=23636398&amp;doi=10.1038/nature12113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Dou Y, Kawaler EA, Cui Zhou D, Gritsenko MA, Huang C, Blumenberg L, Karpova A, Petyuk VA, Savage SR, Satpathy S, et al.  (2020). Proteogenomic Characterization of Endometrial Carcinoma. Cell
180, 729–748.e26.</cite> [<a href="https://doi.org/10.1016/j.cell.2020.01.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7233456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32059776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Characterization%20of%20Endometrial%20Carcinoma&amp;author=Y%20Dou&amp;author=EA%20Kawaler&amp;author=D%20Cui%20Zhou&amp;author=MA%20Gritsenko&amp;author=C%20Huang&amp;volume=180&amp;publication_year=2020&amp;pages=729-748&amp;pmid=32059776&amp;doi=10.1016/j.cell.2020.01.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Sanchez-Vega F, Mina M, Armenia J, Chatila WK, Luna A, La KC, Dimitriadoy S, Liu DL, Kantheti HS, Saghafinia S, et al.  (2018). Oncogenic Signaling Pathways in The Cancer Genome Atlas. Cell
173, 321–337.e10.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6070353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Oncogenic%20Signaling%20Pathways%20in%20The%20Cancer%20Genome%20Atlas&amp;author=F%20Sanchez-Vega&amp;author=M%20Mina&amp;author=J%20Armenia&amp;author=WK%20Chatila&amp;author=A%20Luna&amp;volume=173&amp;publication_year=2018&amp;pages=321-337&amp;pmid=29625050&amp;doi=10.1016/j.cell.2018.03.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Lindeboom RGH., Vermeulen M., Lehner B., and Supek F. (2019). The impact of nonsense-mediated mRNA decay on genetic disease, gene editing and cancer immunotherapy. Nat. Genet. 51, 1645–1651.</cite> [<a href="https://doi.org/10.1038/s41588-019-0517-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31659324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=The%20impact%20of%20nonsense-mediated%20mRNA%20decay%20on%20genetic%20disease,%20gene%20editing%20and%20cancer%20immunotherapy&amp;author=RGH%20Lindeboom&amp;author=M%20Vermeulen&amp;author=B%20Lehner&amp;author=F%20Supek&amp;volume=51&amp;publication_year=2019&amp;pages=1645-1651&amp;pmid=31659324&amp;doi=10.1038/s41588-019-0517-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Yuan TL, and Cantley LC (2008). PI3K pathway alterations in cancer: variations on a theme. Oncogene
27, 5497–5510.</cite> [<a href="https://doi.org/10.1038/onc.2008.245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3398461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18794884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=PI3K%20pathway%20alterations%20in%20cancer:%20variations%20on%20a%20theme&amp;author=TL%20Yuan&amp;author=LC%20Cantley&amp;volume=27&amp;publication_year=2008&amp;pages=5497-5510&amp;pmid=18794884&amp;doi=10.1038/onc.2008.245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Urick ME, Rudd ML, Godwin AK, Sgroi D, Merino M, and Bell DW (2011). PIK3R1 (p85α) is somatically mutated at high frequency in primary endometrial cancer. Cancer Res. 71, 4061–4067.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-11-0549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3117071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21478295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=PIK3R1%20(p85%CE%B1)%20is%20somatically%20mutated%20at%20high%20frequency%20in%20primary%20endometrial%20cancer&amp;author=ME%20Urick&amp;author=ML%20Rudd&amp;author=AK%20Godwin&amp;author=D%20Sgroi&amp;author=M%20Merino&amp;volume=71&amp;publication_year=2011&amp;pages=4061-4067&amp;pmid=21478295&amp;doi=10.1158/0008-5472.CAN-11-0549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Song M, Bode AM, Dong Z, and Lee M-H (2019). AKT as a Therapeutic Target for Cancer. Cancer Res. 79, 1019–1031.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-18-2738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30808672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=AKT%20as%20a%20Therapeutic%20Target%20for%20Cancer&amp;author=M%20Song&amp;author=AM%20Bode&amp;author=Z%20Dong&amp;author=M-H%20Lee&amp;volume=79&amp;publication_year=2019&amp;pages=1019-1031&amp;pmid=30808672&amp;doi=10.1158/0008-5472.CAN-18-2738&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Martorana F, Motta G, Pavone G, Motta L, Stella S, Vitale SR, Manzella L, and Vigneri P. (2021). AKT Inhibitors: New Weapons in the Fight Against Breast Cancer?
Front. Pharmacol. 12, 662232.</cite> [<a href="https://doi.org/10.3389/fphar.2021.662232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8118639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33995085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol&amp;title=AKT%20Inhibitors:%20New%20Weapons%20in%20the%20Fight%20Against%20Breast%20Cancer?&amp;author=F%20Martorana&amp;author=G%20Motta&amp;author=G%20Pavone&amp;author=L%20Motta&amp;author=S%20Stella&amp;volume=12&amp;publication_year=2021&amp;pages=662232&amp;pmid=33995085&amp;doi=10.3389/fphar.2021.662232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Myers AP, Konstantinopoulos PA, Barry WT, Luo W, Broaddus RR, Makker V, Drapkin R, Liu J, Doyle A, Horowitz NS, et al.  (2020). Phase II, 2-stage, 2-arm, PIK3CA mutation stratified trial of MK-2206 in recurrent endometrial cancer. Int. J. Cancer
147, 413–422.</cite> [<a href="https://doi.org/10.1002/ijc.32783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7214201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31714586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&amp;title=Phase%20II,%202-stage,%202-arm,%20PIK3CA%20mutation%20stratified%20trial%20of%20MK-2206%20in%20recurrent%20endometrial%20cancer&amp;author=AP%20Myers&amp;author=PA%20Konstantinopoulos&amp;author=WT%20Barry&amp;author=W%20Luo&amp;author=RR%20Broaddus&amp;volume=147&amp;publication_year=2020&amp;pages=413-422&amp;pmid=31714586&amp;doi=10.1002/ijc.32783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Corsello SM, Nagari RT, Spangler RD, Rossen J, Kocak M, Bryan JG, Humeidi R, Peck D, Wu X, Tang AA, et al.  (2020). Discovering the anti-cancer potential of non-oncology drugs by systematic viability profiling. Nat Cancer
1, 235–248.</cite> [<a href="https://doi.org/10.1038/s43018-019-0018-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7328899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32613204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=Discovering%20the%20anti-cancer%20potential%20of%20non-oncology%20drugs%20by%20systematic%20viability%20profiling&amp;author=SM%20Corsello&amp;author=RT%20Nagari&amp;author=RD%20Spangler&amp;author=J%20Rossen&amp;author=M%20Kocak&amp;volume=1&amp;publication_year=2020&amp;pages=235-248&amp;pmid=32613204&amp;doi=10.1038/s43018-019-0018-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>O’Donnell JS, Long GV, Scolyer RA, Teng MWL, and Smyth MJ (2017). Resistance to PD1/PDL1 checkpoint inhibition. Cancer Treat. Rev. 52, 71–81.</cite> [<a href="https://doi.org/10.1016/j.ctrv.2016.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27951441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat.%20Rev&amp;title=Resistance%20to%20PD1/PDL1%20checkpoint%20inhibition&amp;author=JS%20O%E2%80%99Donnell&amp;author=GV%20Long&amp;author=RA%20Scolyer&amp;author=MWL%20Teng&amp;author=MJ%20Smyth&amp;volume=52&amp;publication_year=2017&amp;pages=71-81&amp;pmid=27951441&amp;doi=10.1016/j.ctrv.2016.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Jenkins RW, Barbie DA, and Flaherty KT (2018). Mechanisms of resistance to immune checkpoint inhibitors. Br. J. Cancer
118, 9–16.</cite> [<a href="https://doi.org/10.1038/bjc.2017.434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5765236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29319049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&amp;title=Mechanisms%20of%20resistance%20to%20immune%20checkpoint%20inhibitors&amp;author=RW%20Jenkins&amp;author=DA%20Barbie&amp;author=KT%20Flaherty&amp;volume=118&amp;publication_year=2018&amp;pages=9-16&amp;pmid=29319049&amp;doi=10.1038/bjc.2017.434&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Wang S, He Z, Wang X, Li H, and Liu X-S (2019). Antigen presentation and tumor immunogenicity in cancer immunotherapy response prediction. Elife
8. 10.7554/eLife.49020.</cite> [<a href="https://doi.org/10.7554/eLife.49020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6879305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31767055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Antigen%20presentation%20and%20tumor%20immunogenicity%20in%20cancer%20immunotherapy%20response%20prediction&amp;author=S%20Wang&amp;author=Z%20He&amp;author=X%20Wang&amp;author=H%20Li&amp;author=X-S%20Liu&amp;volume=8&amp;publication_year=2019&amp;pmid=31767055&amp;doi=10.7554/eLife.49020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Goodman AM, Kato S, Bazhenova L, Patel SP, Frampton GM, Miller V, Stephens PJ, Daniels GA, and Kurzrock R. (2017). Tumor Mutational Burden as an Independent Predictor of Response to Immunotherapy in Diverse Cancers. Mol. Cancer Ther. 16, 2598–2608.</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-17-0386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5670009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28835386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer%20Ther&amp;title=Tumor%20Mutational%20Burden%20as%20an%20Independent%20Predictor%20of%20Response%20to%20Immunotherapy%20in%20Diverse%20Cancers&amp;author=AM%20Goodman&amp;author=S%20Kato&amp;author=L%20Bazhenova&amp;author=SP%20Patel&amp;author=GM%20Frampton&amp;volume=16&amp;publication_year=2017&amp;pages=2598-2608&amp;pmid=28835386&amp;doi=10.1158/1535-7163.MCT-17-0386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Samstein RM., Lee C-H., Shoushtari AN., Hellmann MD., Shen R., Janjigian YY., Barron DA., Zehir A., Jordan EJ., Omuro A., et al. (2019). Tumor mutational load predicts survival after immunotherapy across multiple cancer types. Nat. Genet. 51, 202–206.</cite> [<a href="https://doi.org/10.1038/s41588-018-0312-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6365097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30643254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Tumor%20mutational%20load%20predicts%20survival%20after%20immunotherapy%20across%20multiple%20cancer%20types&amp;author=RM%20Samstein&amp;author=C-H%20Lee&amp;author=AN%20Shoushtari&amp;author=MD%20Hellmann&amp;author=R%20Shen&amp;volume=51&amp;publication_year=2019&amp;pages=202-206&amp;pmid=30643254&amp;doi=10.1038/s41588-018-0312-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Huang C, Chen L, Savage SR, Eguez RV, Dou Y, Li Y, da Veiga Leprevost F, Jaehnig EJ, Lei JT, Wen B, et al.  (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. Cancer Cell
39, 361–379.e16.</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7946781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33417831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20insights%20into%20the%20biology%20and%20treatment%20of%20HPV-negative%20head%20and%20neck%20squamous%20cell%20carcinoma&amp;author=C%20Huang&amp;author=L%20Chen&amp;author=SR%20Savage&amp;author=RV%20Eguez&amp;author=Y%20Dou&amp;volume=39&amp;publication_year=2021&amp;pages=361-379&amp;pmid=33417831&amp;doi=10.1016/j.ccell.2020.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Lu KH, and Broaddus RR (2020). Endometrial Cancer. N. Engl. J. Med. 383, 2053–2064.</cite> [<a href="https://doi.org/10.1056/NEJMra1514010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33207095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&amp;title=Endometrial%20Cancer&amp;author=KH%20Lu&amp;author=RR%20Broaddus&amp;volume=383&amp;publication_year=2020&amp;pages=2053-2064&amp;pmid=33207095&amp;doi=10.1056/NEJMra1514010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Friberg E, Orsini N, Mantzoros CS, and Wolk A. (2007). Diabetes mellitus and risk of endometrial cancer: a meta-analysis. Diabetologia
50, 1365–1374.</cite> [<a href="https://doi.org/10.1007/s00125-007-0681-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17476474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Diabetes%20mellitus%20and%20risk%20of%20endometrial%20cancer:%20a%20meta-analysis&amp;author=E%20Friberg&amp;author=N%20Orsini&amp;author=CS%20Mantzoros&amp;author=A%20Wolk&amp;volume=50&amp;publication_year=2007&amp;pages=1365-1374&amp;pmid=17476474&amp;doi=10.1007/s00125-007-0681-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Saed L, Varse F, Baradaran HR, Moradi Y, Khateri S, Friberg E, Khazaei Z, Gharahjeh S, Tehrani S, Sioofy-Khojine A-B, et al.  (2019). The effect of diabetes on the risk of endometrial Cancer: an updated a systematic review and meta-analysis. BMC Cancer
19, 527.</cite> [<a href="https://doi.org/10.1186/s12885-019-5748-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31151429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&amp;title=The%20effect%20of%20diabetes%20on%20the%20risk%20of%20endometrial%20Cancer:%20an%20updated%20a%20systematic%20review%20and%20meta-analysis&amp;author=L%20Saed&amp;author=F%20Varse&amp;author=HR%20Baradaran&amp;author=Y%20Moradi&amp;author=S%20Khateri&amp;volume=19&amp;publication_year=2019&amp;pages=527&amp;pmid=31151429&amp;doi=10.1186/s12885-019-5748-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Meireles CG, Pereira SA, Valadares LP, Rêgo DF, Simeoni LA, Guerra ENS, and Lofrano-Porto A. (2017). Effects of metformin on endometrial cancer: Systematic review and meta-analysis. Gynecol. Oncol. 147, 167–180.</cite> [<a href="https://doi.org/10.1016/j.ygyno.2017.07.120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28760367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gynecol.%20Oncol&amp;title=Effects%20of%20metformin%20on%20endometrial%20cancer:%20Systematic%20review%20and%20meta-analysis&amp;author=CG%20Meireles&amp;author=SA%20Pereira&amp;author=LP%20Valadares&amp;author=DF%20R%C3%AAgo&amp;author=LA%20Simeoni&amp;volume=147&amp;publication_year=2017&amp;pages=167-180&amp;pmid=28760367&amp;doi=10.1016/j.ygyno.2017.07.120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Tong X-P, Chen Y, Zhang S-Y, Xie T, Tian M, Guo M-R, Kasimu R, Ouyang L, and Wang J-H (2015). Key autophagic targets and relevant small-molecule compounds in cancer therapy. Cell Prolif. 48, 7–16.</cite> [<a href="https://doi.org/10.1111/cpr.12154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6496147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25474301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Prolif&amp;title=Key%20autophagic%20targets%20and%20relevant%20small-molecule%20compounds%20in%20cancer%20therapy&amp;author=X-P%20Tong&amp;author=Y%20Chen&amp;author=S-Y%20Zhang&amp;author=T%20Xie&amp;author=M%20Tian&amp;volume=48&amp;publication_year=2015&amp;pages=7-16&amp;pmid=25474301&amp;doi=10.1111/cpr.12154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Kalyanaraman B, Cheng G, Hardy M, Ouari O, Lopez M, Joseph J, Zielonka J, and Dwinell MB (2018). A review of the basics of mitochondrial bioenergetics, metabolism, and related signaling pathways in cancer cells: Therapeutic targeting of tumor mitochondria with lipophilic cationic compounds. Redox Biol
14, 316–327.</cite> [<a href="https://doi.org/10.1016/j.redox.2017.09.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5633086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29017115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Redox%20Biol&amp;title=A%20review%20of%20the%20basics%20of%20mitochondrial%20bioenergetics,%20metabolism,%20and%20related%20signaling%20pathways%20in%20cancer%20cells:%20Therapeutic%20targeting%20of%20tumor%20mitochondria%20with%20lipophilic%20cationic%20compounds&amp;author=B%20Kalyanaraman&amp;author=G%20Cheng&amp;author=M%20Hardy&amp;author=O%20Ouari&amp;author=M%20Lopez&amp;volume=14&amp;publication_year=2018&amp;pages=316-327&amp;pmid=29017115&amp;doi=10.1016/j.redox.2017.09.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Bridgeman SC, Ellison GC, Melton PE, Newsholme P, and Mamotte CDS (2018). Epigenetic effects of metformin: From molecular mechanisms to clinical implications. Diabetes Obes. Metab. 20, 1553–1562.</cite> [<a href="https://doi.org/10.1111/dom.13262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29457866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes%20Obes.%20Metab&amp;title=Epigenetic%20effects%20of%20metformin:%20From%20molecular%20mechanisms%20to%20clinical%20implications&amp;author=SC%20Bridgeman&amp;author=GC%20Ellison&amp;author=PE%20Melton&amp;author=P%20Newsholme&amp;author=CDS%20Mamotte&amp;volume=20&amp;publication_year=2018&amp;pages=1553-1562&amp;pmid=29457866&amp;doi=10.1111/dom.13262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Tseng H-W, Li S-C, and Tsai K-W (2019). Metformin Treatment Suppresses Melanoma Cell Growth and Motility Through Modulation of microRNA Expression. Cancers
11. 10.3390/cancers11020209.</cite> [<a href="https://doi.org/10.3390/cancers11020209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6406940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30754729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Metformin%20Treatment%20Suppresses%20Melanoma%20Cell%20Growth%20and%20Motility%20Through%20Modulation%20of%20microRNA%20Expression&amp;author=H-W%20Tseng&amp;author=S-C%20Li&amp;author=K-W%20Tsai&amp;volume=11&amp;publication_year=2019&amp;pmid=30754729&amp;doi=10.3390/cancers11020209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Xue J, Li L, Li N, Li F, Qin X, Li T, and Liu M. (2019). Metformin suppresses cancer cell growth in endometrial carcinoma by inhibiting PD-L1. Eur. J. Pharmacol. 859, 172541.</cite> [<a href="https://doi.org/10.1016/j.ejphar.2019.172541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31319067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol&amp;title=Metformin%20suppresses%20cancer%20cell%20growth%20in%20endometrial%20carcinoma%20by%20inhibiting%20PD-L1&amp;author=J%20Xue&amp;author=L%20Li&amp;author=N%20Li&amp;author=F%20Li&amp;author=X%20Qin&amp;volume=859&amp;publication_year=2019&amp;pages=172541&amp;pmid=31319067&amp;doi=10.1016/j.ejphar.2019.172541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Wallbillich JJ, Josyula S, Saini U, Zingarelli RA, Dorayappan KDP, Riley MK, Wanner RA, Cohn DE, and Selvendiran K. (2017). High Glucose-Mediated STAT3 Activation in Endometrial Cancer Is Inhibited by Metformin: Therapeutic Implications for Endometrial Cancer. PLoS One
12, e0170318.</cite> [<a href="https://doi.org/10.1371/journal.pone.0170318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5256996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28114390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=High%20Glucose-Mediated%20STAT3%20Activation%20in%20Endometrial%20Cancer%20Is%20Inhibited%20by%20Metformin:%20Therapeutic%20Implications%20for%20Endometrial%20Cancer&amp;author=JJ%20Wallbillich&amp;author=S%20Josyula&amp;author=U%20Saini&amp;author=RA%20Zingarelli&amp;author=KDP%20Dorayappan&amp;volume=12&amp;publication_year=2017&amp;pages=e0170318&amp;pmid=28114390&amp;doi=10.1371/journal.pone.0170318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Lamb J., Crawford ED., Peck D., Modell JW., Blat IC., Wrobel MJ., Lerner J., Brunet J-P., Subramanian A., Ross KN., et al. (2006). The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science
313, 1929–1935.</cite> [<a href="https://doi.org/10.1126/science.1132939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17008526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Connectivity%20Map:%20using%20gene-expression%20signatures%20to%20connect%20small%20molecules,%20genes,%20and%20disease&amp;author=J%20Lamb&amp;author=ED%20Crawford&amp;author=D%20Peck&amp;author=JW%20Modell&amp;author=IC%20Blat&amp;volume=313&amp;publication_year=2006&amp;pages=1929-1935&amp;pmid=17008526&amp;doi=10.1126/science.1132939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Perry JM, Tao F, Roy A, Lin T, He XC, Chen S, Lu X, Nemechek J, Ruan L, Yu X, et al.  (2020). Overcoming Wnt-β-catenin dependent anticancer therapy resistance in leukaemia stem cells. Nat. Cell Biol. 22, 689–700.</cite> [<a href="https://doi.org/10.1038/s41556-020-0507-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8010717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32313104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=Overcoming%20Wnt-%CE%B2-catenin%20dependent%20anticancer%20therapy%20resistance%20in%20leukaemia%20stem%20cells&amp;author=JM%20Perry&amp;author=F%20Tao&amp;author=A%20Roy&amp;author=T%20Lin&amp;author=XC%20He&amp;volume=22&amp;publication_year=2020&amp;pages=689-700&amp;pmid=32313104&amp;doi=10.1038/s41556-020-0507-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Hsu J-M, Xia W, Hsu Y-H, Chan L-C, Yu W-H, Cha J-H, Chen C-T, Liao H-W, Kuo C-W, Khoo K-H, et al.  (2018). STT3-dependent PD-L1 accumulation on cancer stem cells promotes immune evasion. Nat. Commun. 9, 1908.</cite> [<a href="https://doi.org/10.1038/s41467-018-04313-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5954021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29765039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=STT3-dependent%20PD-L1%20accumulation%20on%20cancer%20stem%20cells%20promotes%20immune%20evasion&amp;author=J-M%20Hsu&amp;author=W%20Xia&amp;author=Y-H%20Hsu&amp;author=L-C%20Chan&amp;author=W-H%20Yu&amp;volume=9&amp;publication_year=2018&amp;pages=1908&amp;pmid=29765039&amp;doi=10.1038/s41467-018-04313-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Gao C, Wang Y, Broaddus R, Sun L, Xue F, and Zhang W. (2018). Exon 3 mutations of drive tumorigenesis: a review. Oncotarget
9, 5492–5508.</cite> [<a href="https://doi.org/10.18632/oncotarget.23695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5797067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29435196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Exon%203%20mutations%20of%20drive%20tumorigenesis:%20a%20review&amp;author=C%20Gao&amp;author=Y%20Wang&amp;author=R%20Broaddus&amp;author=L%20Sun&amp;author=F%20Xue&amp;volume=9&amp;publication_year=2018&amp;pages=5492-5508&amp;pmid=29435196&amp;doi=10.18632/oncotarget.23695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Niida A, Hiroko T, Kasai M, Furukawa Y, Nakamura Y, Suzuki Y, Sugano S, and Akiyama T. (2004). DKK1, a negative regulator of Wnt signaling, is a target of the beta-catenin/TCF pathway. Oncogene
23, 8520–8526.</cite> [<a href="https://doi.org/10.1038/sj.onc.1207892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15378020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=DKK1,%20a%20negative%20regulator%20of%20Wnt%20signaling,%20is%20a%20target%20of%20the%20beta-catenin/TCF%20pathway&amp;author=A%20Niida&amp;author=T%20Hiroko&amp;author=M%20Kasai&amp;author=Y%20Furukawa&amp;author=Y%20Nakamura&amp;volume=23&amp;publication_year=2004&amp;pages=8520-8526&amp;pmid=15378020&amp;doi=10.1038/sj.onc.1207892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Liu C, Li Y, Semenov M, Han C, Baeg GH, Tan Y, Zhang Z, Lin X, and He X. (2002). Control of beta-catenin phosphorylation/degradation by a dual-kinase mechanism. Cell
108, 837–847.</cite> [<a href="https://doi.org/10.1016/s0092-8674(02)00685-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11955436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Control%20of%20beta-catenin%20phosphorylation/degradation%20by%20a%20dual-kinase%20mechanism&amp;author=C%20Liu&amp;author=Y%20Li&amp;author=M%20Semenov&amp;author=C%20Han&amp;author=GH%20Baeg&amp;volume=108&amp;publication_year=2002&amp;pages=837-847&amp;pmid=11955436&amp;doi=10.1016/s0092-8674(02)00685-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Kurnit KC, Kim GN, Fellman BM, Urbauer DL, Mills GB, Zhang W, and Broaddus RR (2017). CTNNB1 (beta-catenin) mutation identifies low grade, early stage endometrial cancer patients at increased risk of recurrence. Mod. Pathol. 30, 1032–1041.</cite> [<a href="https://doi.org/10.1038/modpathol.2017.15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5493522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28281553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mod.%20Pathol&amp;title=CTNNB1%20(beta-catenin)%20mutation%20identifies%20low%20grade,%20early%20stage%20endometrial%20cancer%20patients%20at%20increased%20risk%20of%20recurrence&amp;author=KC%20Kurnit&amp;author=GN%20Kim&amp;author=BM%20Fellman&amp;author=DL%20Urbauer&amp;author=GB%20Mills&amp;volume=30&amp;publication_year=2017&amp;pages=1032-1041&amp;pmid=28281553&amp;doi=10.1038/modpathol.2017.15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Myers A, Barry WT, Hirsch MS, Matulonis U, and Lee L. (2014). β-Catenin mutations in recurrent FIGO IA grade I endometrioid endometrial cancers. Gynecol. Oncol. 134, 426–427.</cite> [<a href="https://doi.org/10.1016/j.ygyno.2014.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24952365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gynecol.%20Oncol&amp;title=%CE%B2-Catenin%20mutations%20in%20recurrent%20FIGO%20IA%20grade%20I%20endometrioid%20endometrial%20cancers&amp;author=A%20Myers&amp;author=WT%20Barry&amp;author=MS%20Hirsch&amp;author=U%20Matulonis&amp;author=L%20Lee&amp;volume=134&amp;publication_year=2014&amp;pages=426-427&amp;pmid=24952365&amp;doi=10.1016/j.ygyno.2014.06.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Phillips KA, Deverka PA, Trosman JR, Douglas MP, Chambers JD, Weldon CB, and Dervan AP (2017). Payer coverage policies for multigene tests. Nat. Biotechnol. 35, 614–617.</cite> [<a href="https://doi.org/10.1038/nbt.3912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5553867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28700544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Payer%20coverage%20policies%20for%20multigene%20tests&amp;author=KA%20Phillips&amp;author=PA%20Deverka&amp;author=JR%20Trosman&amp;author=MP%20Douglas&amp;author=JD%20Chambers&amp;volume=35&amp;publication_year=2017&amp;pages=614-617&amp;pmid=28700544&amp;doi=10.1038/nbt.3912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Trosman JR, Weldon CB, Gradishar WJ, Benson AB 3rd, Cristofanilli M, Kurian AW, Ford JM, Balch A, Watkins J, and Phillips KA (2018). From the Past to the Present: Insurer Coverage Frameworks for Next-Generation Tumor Sequencing. Value Health
21, 1062–1068.</cite> [<a href="https://doi.org/10.1016/j.jval.2018.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6374027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30224110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Value%20Health&amp;title=From%20the%20Past%20to%20the%20Present:%20Insurer%20Coverage%20Frameworks%20for%20Next-Generation%20Tumor%20Sequencing&amp;author=JR%20Trosman&amp;author=CB%20Weldon&amp;author=WJ%20Gradishar&amp;author=AB%20Benson&amp;author=M%20Cristofanilli&amp;volume=21&amp;publication_year=2018&amp;pages=1062-1068&amp;pmid=30224110&amp;doi=10.1016/j.jval.2018.06.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Hong R, Liu W, DeLair D, Razavian N, and Fenyö D. (2021). Predicting endometrial cancer subtypes and molecular features from histopathology images using multi-resolution deep learning models. Cell Rep Med
2, 100400.</cite> [<a href="https://doi.org/10.1016/j.xcrm.2021.100400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8484685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34622237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep%20Med&amp;title=Predicting%20endometrial%20cancer%20subtypes%20and%20molecular%20features%20from%20histopathology%20images%20using%20multi-resolution%20deep%20learning%20models&amp;author=R%20Hong&amp;author=W%20Liu&amp;author=D%20DeLair&amp;author=N%20Razavian&amp;author=D%20Feny%C3%B6&amp;volume=2&amp;publication_year=2021&amp;pages=100400&amp;pmid=34622237&amp;doi=10.1016/j.xcrm.2021.100400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Davoli T, Uno H, Wooten EC, and Elledge SJ (2017). Tumor aneuploidy correlates with markers of immune evasion and with reduced response to immunotherapy. Science
355. 10.1126/science.aaf8399.</cite> [<a href="https://doi.org/10.1126/science.aaf8399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5592794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28104840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Tumor%20aneuploidy%20correlates%20with%20markers%20of%20immune%20evasion%20and%20with%20reduced%20response%20to%20immunotherapy&amp;author=T%20Davoli&amp;author=H%20Uno&amp;author=EC%20Wooten&amp;author=SJ%20Elledge&amp;volume=355&amp;publication_year=2017&amp;pmid=28104840&amp;doi=10.1126/science.aaf8399&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Taylor AM, Shih J, Ha G, Gao GF, Zhang X, Berger AC, Schumacher SE, Wang C, Hu H, Liu J, et al.  (2018). Genomic and Functional Approaches to Understanding Cancer Aneuploidy. Cancer Cell
33, 676–689.e3.</cite> [<a href="https://doi.org/10.1016/j.ccell.2018.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6028190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29622463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Genomic%20and%20Functional%20Approaches%20to%20Understanding%20Cancer%20Aneuploidy&amp;author=AM%20Taylor&amp;author=J%20Shih&amp;author=G%20Ha&amp;author=GF%20Gao&amp;author=X%20Zhang&amp;volume=33&amp;publication_year=2018&amp;pages=676-689&amp;pmid=29622463&amp;doi=10.1016/j.ccell.2018.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Ray Chaudhuri A, and Nussenzweig A. (2017). The multifaceted roles of PARP1 in DNA repair and chromatin remodelling. Nat. Rev. Mol. Cell Biol. 18, 610–621.</cite> [<a href="https://doi.org/10.1038/nrm.2017.53" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6591728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28676700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;title=The%20multifaceted%20roles%20of%20PARP1%20in%20DNA%20repair%20and%20chromatin%20remodelling&amp;author=A%20Ray%20Chaudhuri&amp;author=A%20Nussenzweig&amp;volume=18&amp;publication_year=2017&amp;pages=610-621&amp;pmid=28676700&amp;doi=10.1038/nrm.2017.53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Pinho SS, and Reis CA (2015). Glycosylation in cancer: mechanisms and clinical implications. Nat. Rev. Cancer
15, 540–555.</cite> [<a href="https://doi.org/10.1038/nrc3982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26289314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Glycosylation%20in%20cancer:%20mechanisms%20and%20clinical%20implications&amp;author=SS%20Pinho&amp;author=CA%20Reis&amp;volume=15&amp;publication_year=2015&amp;pages=540-555&amp;pmid=26289314&amp;doi=10.1038/nrc3982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Cao L, Huang C, Cui Zhou D, Hu Y, Lih TM, Savage SR, Krug K, Clark DJ, Schnaubelt M, Chen L, et al.  (2021). Proteogenomic characterization of pancreatic ductal adenocarcinoma. Cell
184, 5031–5052.e26.</cite> [<a href="https://doi.org/10.1016/j.cell.2021.08.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8654574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34534465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20characterization%20of%20pancreatic%20ductal%20adenocarcinoma&amp;author=L%20Cao&amp;author=C%20Huang&amp;author=D%20Cui%20Zhou&amp;author=Y%20Hu&amp;author=TM%20Lih&amp;volume=184&amp;publication_year=2021&amp;pages=5031-5052&amp;pmid=34534465&amp;doi=10.1016/j.cell.2021.08.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Oaknin A., Gilbert L., Tinker AV., Brown J., Mathews C., Press J., Sabatier R., O’Malley DM., Samouelian V., Boni V., et al. (2022). Safety and antitumor activity of dostarlimab in patients with advanced or recurrent DNA mismatch repair deficient/microsatellite instability-high (dMMR/MSI-H) or proficient/stable (MMRp/MSS) endometrial cancer: interim results from GARNET-a phase I, single-arm study. J Immunother Cancer
10. 10.1136/jitc-2021-003777.</cite> [<a href="https://doi.org/10.1136/jitc-2021-003777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8785197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35064011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Safety%20and%20antitumor%20activity%20of%20dostarlimab%20in%20patients%20with%20advanced%20or%20recurrent%20DNA%20mismatch%20repair%20deficient/microsatellite%20instability-high%20(dMMR/MSI-H)%20or%20proficient/stable%20(MMRp/MSS)%20endometrial%20cancer:%20interim%20results%20from%20GARNET-a%20phase%20I,%20single-arm%20study&amp;author=A%20Oaknin&amp;author=L%20Gilbert&amp;author=AV%20Tinker&amp;author=J%20Brown&amp;author=C%20Mathews&amp;volume=10&amp;publication_year=2022&amp;pmid=35064011&amp;doi=10.1136/jitc-2021-003777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Soliman PT, Westin SN, Iglesias DA, Fellman BM, Yuan Y, Zhang Q, Yates MS, Broaddus RR, Slomovitz BM, Lu KH, et al.  (2020). Everolimus, Letrozole, and Metformin in Women with Advanced or Recurrent Endometrioid Endometrial Cancer: A Multi-Center, Single Arm, Phase II Study. Clin. Cancer Res. 26, 581–587.</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-19-0471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7002216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31628143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Everolimus,%20Letrozole,%20and%20Metformin%20in%20Women%20with%20Advanced%20or%20Recurrent%20Endometrioid%20Endometrial%20Cancer:%20A%20Multi-Center,%20Single%20Arm,%20Phase%20II%20Study&amp;author=PT%20Soliman&amp;author=SN%20Westin&amp;author=DA%20Iglesias&amp;author=BM%20Fellman&amp;author=Y%20Yuan&amp;volume=26&amp;publication_year=2020&amp;pages=581-587&amp;pmid=31628143&amp;doi=10.1158/1078-0432.CCR-19-0471&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Hall C, Stone RL, Gehlot A, Zorn KK, and Burnett AF (2016). Use of Metformin in Obese Women With Type I Endometrial Cancer Is Associated With a Reduced Incidence of Cancer Recurrence. Int. J. Gynecol. Cancer
26, 313–317.</cite> [<a href="https://doi.org/10.1097/IGC.0000000000000603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26588235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Gynecol.%20Cancer&amp;title=Use%20of%20Metformin%20in%20Obese%20Women%20With%20Type%20I%20Endometrial%20Cancer%20Is%20Associated%20With%20a%20Reduced%20Incidence%20of%20Cancer%20Recurrence&amp;author=C%20Hall&amp;author=RL%20Stone&amp;author=A%20Gehlot&amp;author=KK%20Zorn&amp;author=AF%20Burnett&amp;volume=26&amp;publication_year=2016&amp;pages=313-317&amp;pmid=26588235&amp;doi=10.1097/IGC.0000000000000603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Schuler KM, Rambally BS, DiFurio MJ, Sampey BP, Gehrig PA, Makowski L, and Bae-Jump VL (2015). Antiproliferative and metabolic effects of metformin in a preoperative window clinical trial for endometrial cancer. Cancer Med. 4, 161–173.</cite> [<a href="https://doi.org/10.1002/cam4.353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4329001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25417601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Med&amp;title=Antiproliferative%20and%20metabolic%20effects%20of%20metformin%20in%20a%20preoperative%20window%20clinical%20trial%20for%20endometrial%20cancer&amp;author=KM%20Schuler&amp;author=BS%20Rambally&amp;author=MJ%20DiFurio&amp;author=BP%20Sampey&amp;author=PA%20Gehrig&amp;volume=4&amp;publication_year=2015&amp;pages=161-173&amp;pmid=25417601&amp;doi=10.1002/cam4.353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Beausoleil SA, Villén J, Gerber SA, Rush J, and Gygi SP (2006). A probability-based approach for high-throughput protein phosphorylation analysis and site localization. Nat. Biotechnol. 24, 1285–1292.</cite> [<a href="https://doi.org/10.1038/nbt1240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16964243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=A%20probability-based%20approach%20for%20high-throughput%20protein%20phosphorylation%20analysis%20and%20site%20localization&amp;author=SA%20Beausoleil&amp;author=J%20Vill%C3%A9n&amp;author=SA%20Gerber&amp;author=J%20Rush&amp;author=SP%20Gygi&amp;volume=24&amp;publication_year=2006&amp;pages=1285-1292&amp;pmid=16964243&amp;doi=10.1038/nbt1240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Blumenberg L, Kawaler E, Cornwell M, Smith S, Ruggles K, and Fenyö D. (2019). BlackSheep: A Bioconductor and Bioconda package for differential extreme value analysis. bioRxiv, 825067. 10.1101/825067.</cite> [<a href="https://doi.org/10.1101/825067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8256816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34165986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=BlackSheep:%20A%20Bioconductor%20and%20Bioconda%20package%20for%20differential%20extreme%20value%20analysis&amp;author=L%20Blumenberg&amp;author=E%20Kawaler&amp;author=M%20Cornwell&amp;author=S%20Smith&amp;author=K%20Ruggles&amp;publication_year=2019&amp;pages=825067&amp;pmid=34165986&amp;doi=10.1101/825067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Langmead B, and Salzberg SL (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods
9, 357–359.</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B%20Langmead&amp;author=SL%20Salzberg&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Li H, and Durbin R. (2010). Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics
26, 589–595.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2828108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20080505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H%20Li&amp;author=R%20Durbin&amp;volume=26&amp;publication_year=2010&amp;pages=589-595&amp;pmid=20080505&amp;doi=10.1093/bioinformatics/btp698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Gao Y, Wang J, and Zhao F. (2015). CIRI: an efficient and unbiased algorithm for de novo circular RNA identification. Genome Biol. 16, 4.</cite> [<a href="https://doi.org/10.1186/s13059-014-0571-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4316645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25583365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=CIRI:%20an%20efficient%20and%20unbiased%20algorithm%20for%20de%20novo%20circular%20RNA%20identification&amp;author=Y%20Gao&amp;author=J%20Wang&amp;author=F%20Zhao&amp;volume=16&amp;publication_year=2015&amp;pages=4&amp;pmid=25583365&amp;doi=10.1186/s13059-014-0571-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Lindgren CM, Adams DW, Kimball B, Boekweg H, Tayler S, Pugh SL, and Payne SH (2021). Simplified and Unified Access to Cancer Proteogenomic Data. J. Proteome Res. 20, 1902–1910.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8022323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33560848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res&amp;title=Simplified%20and%20Unified%20Access%20to%20Cancer%20Proteogenomic%20Data&amp;author=CM%20Lindgren&amp;author=DW%20Adams&amp;author=B%20Kimball&amp;author=H%20Boekweg&amp;author=S%20Tayler&amp;volume=20&amp;publication_year=2021&amp;pages=1902-1910&amp;pmid=33560848&amp;doi=10.1021/acs.jproteome.0c00919&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Benelli M, Pescucci C, Marseglia G, Severgnini M, Torricelli F, and Magi A. (2012). Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript. Bioinformatics
28, 3232–3239.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23093608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Discovering%20chimeric%20transcripts%20in%20paired-end%20RNA-seq%20data%20by%20using%20EricScript&amp;author=M%20Benelli&amp;author=C%20Pescucci&amp;author=G%20Marseglia&amp;author=M%20Severgnini&amp;author=F%20Torricelli&amp;volume=28&amp;publication_year=2012&amp;pages=3232-3239&amp;pmid=23093608&amp;doi=10.1093/bioinformatics/bts617&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Toghi Eshghi S, Shah P, Yang W, Li X, and Zhang H. (2015). GPQuest: A Spectral Library Matching Algorithm for Site-Specific Assignment of Tandem Mass Spectra to Intact N-glycopeptides. Anal. Chem. 87, 5181–5188.</cite> [<a href="https://doi.org/10.1021/acs.analchem.5b00024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4721644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25945896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem&amp;title=GPQuest:%20A%20Spectral%20Library%20Matching%20Algorithm%20for%20Site-Specific%20Assignment%20of%20Tandem%20Mass%20Spectra%20to%20Intact%20N-glycopeptides&amp;author=S%20Toghi%20Eshghi&amp;author=P%20Shah&amp;author=W%20Yang&amp;author=X%20Li&amp;author=H%20Zhang&amp;volume=87&amp;publication_year=2015&amp;pages=5181-5188&amp;pmid=25945896&amp;doi=10.1021/acs.analchem.5b00024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Hänzelmann S, Castelo R, and Guinney J. (2013). GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics
14, 7.</cite> [<a href="https://doi.org/10.1186/1471-2105-14-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3618321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23323831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data&amp;author=S%20H%C3%A4nzelmann&amp;author=R%20Castelo&amp;author=J%20Guinney&amp;volume=14&amp;publication_year=2013&amp;pages=7&amp;pmid=23323831&amp;doi=10.1186/1471-2105-14-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Zhang J., White NM., Schmidt HK., Fulton RS., Tomlinson C., Warren WC., Wilson RK., and Maher CA. (2016). INTEGRATE: gene fusion discovery using whole genome and transcriptome data. Genome Res. 26, 108–118.</cite> [<a href="https://doi.org/10.1101/gr.186114.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4691743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26556708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=INTEGRATE:%20gene%20fusion%20discovery%20using%20whole%20genome%20and%20transcriptome%20data&amp;author=J%20Zhang&amp;author=NM%20White&amp;author=HK%20Schmidt&amp;author=RS%20Fulton&amp;author=C%20Tomlinson&amp;volume=26&amp;publication_year=2016&amp;pages=108-118&amp;pmid=26556708&amp;doi=10.1101/gr.186114.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Vasaikar SV, Straub P, Wang J, and Zhang B. (2018). LinkedOmics: analyzing multi-omics data within and across 32 cancer types. Nucleic Acids Res. 46, D956–D963.</cite> [<a href="https://doi.org/10.1093/nar/gkx1090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29136207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LinkedOmics:%20analyzing%20multi-omics%20data%20within%20and%20across%2032%20cancer%20types&amp;author=SV%20Vasaikar&amp;author=P%20Straub&amp;author=J%20Wang&amp;author=B%20Zhang&amp;volume=46&amp;publication_year=2018&amp;pages=D956-D963&amp;pmid=29136207&amp;doi=10.1093/nar/gkx1090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Monroe ME, Shaw JL, Daly DS, Adkins JN, and Smith RD (2008). MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features. Comput. Biol. Chem. 32, 215–217.</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2008.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2487672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18440872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Biol.%20Chem&amp;title=MASIC:%20a%20software%20program%20for%20fast%20quantitation%20and%20flexible%20visualization%20of%20chromatographic%20profiles%20from%20detected%20LC-MS(/MS)%20features&amp;author=ME%20Monroe&amp;author=JL%20Shaw&amp;author=DS%20Daly&amp;author=JN%20Adkins&amp;author=RD%20Smith&amp;volume=32&amp;publication_year=2008&amp;pages=215-217&amp;pmid=18440872&amp;doi=10.1016/j.compbiolchem.2008.02.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Wen B, Mei Z, Zeng C, and Liu S. (2017). metaX: a flexible and comprehensive software for processing metabolomics data. BMC Bioinformatics
18, 183.</cite> [<a href="https://doi.org/10.1186/s12859-017-1579-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5361702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28327092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=metaX:%20a%20flexible%20and%20comprehensive%20software%20for%20processing%20metabolomics%20data&amp;author=B%20Wen&amp;author=Z%20Mei&amp;author=C%20Zeng&amp;author=S%20Liu&amp;volume=18&amp;publication_year=2017&amp;pages=183&amp;pmid=28327092&amp;doi=10.1186/s12859-017-1579-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Kim S, and Pevzner PA (2014). MS-GF+ makes progress towards a universal database search tool for proteomics. Nat. Commun. 5, 5277.</cite> [<a href="https://doi.org/10.1038/ncomms6277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5036525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25358478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=MS-GF+%20makes%20progress%20towards%20a%20universal%20database%20search%20tool%20for%20proteomics&amp;author=S%20Kim&amp;author=PA%20Pevzner&amp;volume=5&amp;publication_year=2014&amp;pages=5277&amp;pmid=25358478&amp;doi=10.1038/ncomms6277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Niu B, Ye K, Zhang Q, Lu C, Xie M, McLellan MD, Wendl MC, and Ding L. (2014). MSIsensor: microsatellite instability detection using paired tumor-normal sequence data. Bioinformatics
30, 1015–1016.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3967115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24371154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MSIsensor:%20microsatellite%20instability%20detection%20using%20paired%20tumor-normal%20sequence%20data&amp;author=B%20Niu&amp;author=K%20Ye&amp;author=Q%20Zhang&amp;author=C%20Lu&amp;author=M%20Xie&amp;volume=30&amp;publication_year=2014&amp;pages=1015-1016&amp;pmid=24371154&amp;doi=10.1093/bioinformatics/btt755&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Chen L, Zhang B, Schnaubelt M, Shah P, Aiyetan P, Chan D, Zhang H, and Zhang Z. (2018). MS-PyCloud: An open-source, cloud computing-based pipeline for LC-MS/MS data analysis. bioRxiv, 320887. 10.1101/320887.</cite> [<a href="https://doi.org/10.1101/320887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=MS-PyCloud:%20An%20open-source,%20cloud%20computing-based%20pipeline%20for%20LC-MS/MS%20data%20analysis&amp;author=L%20Chen&amp;author=B%20Zhang&amp;author=M%20Schnaubelt&amp;author=P%20Shah&amp;author=P%20Aiyetan&amp;publication_year=2018&amp;pages=320887&amp;doi=10.1101/320887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Dees ND, Zhang Q, Kandoth C, Wendl MC, Schierding W, Koboldt DC, Mooney TB, Callaway MB, Dooling D, Mardis ER, et al.  (2012). MuSiC: identifying mutational significance in cancer genomes. Genome Res. 22, 1589–1598.</cite> [<a href="https://doi.org/10.1101/gr.134635.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3409272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22759861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=MuSiC:%20identifying%20mutational%20significance%20in%20cancer%20genomes&amp;author=ND%20Dees&amp;author=Q%20Zhang&amp;author=C%20Kandoth&amp;author=MC%20Wendl&amp;author=W%20Schierding&amp;volume=22&amp;publication_year=2012&amp;pages=1589-1598&amp;pmid=22759861&amp;doi=10.1101/gr.134635.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Cibulskis K, Lawrence MS, Carter SL, Sivachenko A, Jaffe D, Sougnez C, Gabriel S, Meyerson M, Lander ES, and Getz G. (2013). Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Nat. Biotechnol. 31, 213–219.</cite> [<a href="https://doi.org/10.1038/nbt.2514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3833702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23396013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Sensitive%20detection%20of%20somatic%20point%20mutations%20in%20impure%20and%20heterogeneous%20cancer%20samples&amp;author=K%20Cibulskis&amp;author=MS%20Lawrence&amp;author=SL%20Carter&amp;author=A%20Sivachenko&amp;author=D%20Jaffe&amp;volume=31&amp;publication_year=2013&amp;pages=213-219&amp;pmid=23396013&amp;doi=10.1038/nbt.2514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>Gibbons BC, Chambers MC, Monroe ME, Tabb DL, and Payne SH (2015). Correcting systematic bias and instrument measurement drift with mzRefinery. Bioinformatics
31, 3838–3840.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4653383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26243018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Correcting%20systematic%20bias%20and%20instrument%20measurement%20drift%20with%20mzRefinery&amp;author=BC%20Gibbons&amp;author=MC%20Chambers&amp;author=ME%20Monroe&amp;author=DL%20Tabb&amp;author=SH%20Payne&amp;volume=31&amp;publication_year=2015&amp;pages=3838-3840&amp;pmid=26243018&amp;doi=10.1093/bioinformatics/btv437&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R73">
<span class="label">73.</span><cite>Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, and Kohlbacher O. (2014). OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics
30, 3310–3316.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4441069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25143287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=OptiType:%20precision%20HLA%20typing%20from%20next-generation%20sequencing%20data&amp;author=A%20Szolek&amp;author=B%20Schubert&amp;author=C%20Mohr&amp;author=M%20Sturm&amp;author=M%20Feldhahn&amp;volume=30&amp;publication_year=2014&amp;pages=3310-3316&amp;pmid=25143287&amp;doi=10.1093/bioinformatics/btu548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Wen B, Wang X, and Zhang B. (2019). PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations. Genome Res. 29, 485–493.</cite> [<a href="https://doi.org/10.1101/gr.235028.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6396417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30610011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=PepQuery%20enables%20fast,%20accurate,%20and%20convenient%20proteomic%20validation%20of%20novel%20genomic%20alterations&amp;author=B%20Wen&amp;author=X%20Wang&amp;author=B%20Zhang&amp;volume=29&amp;publication_year=2019&amp;pages=485-493&amp;pmid=30610011&amp;doi=10.1101/gr.235028.118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Mayakonda A, Lin D-C, Assenov Y, Plass C, and Koeffler HP (2018). Maftools: efficient and comprehensive analysis of somatic variants in cancer. Genome Res. 28, 1747–1756.</cite> [<a href="https://doi.org/10.1101/gr.239244.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6211645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30341162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Maftools:%20efficient%20and%20comprehensive%20analysis%20of%20somatic%20variants%20in%20cancer&amp;author=A%20Mayakonda&amp;author=D-C%20Lin&amp;author=Y%20Assenov&amp;author=C%20Plass&amp;author=HP%20Koeffler&amp;volume=28&amp;publication_year=2018&amp;pages=1747-1756&amp;pmid=30341162&amp;doi=10.1101/gr.239244.118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>Ye K., Schulz MH., Long Q., Apweiler R., and Ning Z. (2009). Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics
25, 2865–2871.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19561018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&amp;author=K%20Ye&amp;author=MH%20Schulz&amp;author=Q%20Long&amp;author=R%20Apweiler&amp;author=Z%20Ning&amp;volume=25&amp;publication_year=2009&amp;pages=2865-2871&amp;pmid=19561018&amp;doi=10.1093/bioinformatics/btp394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Schubert M, Klinger B, Klünemann M, Sieber A, Uhlitz F, Sauer S, Garnett MJ, Blüthgen N, and Saez-Rodriguez J. (2018). Perturbation-response genes reveal signaling footprints in cancer gene expression. Nat. Commun. 9, 20.</cite> [<a href="https://doi.org/10.1038/s41467-017-02391-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5750219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29295995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Perturbation-response%20genes%20reveal%20signaling%20footprints%20in%20cancer%20gene%20expression&amp;author=M%20Schubert&amp;author=B%20Klinger&amp;author=M%20Kl%C3%BCnemann&amp;author=A%20Sieber&amp;author=F%20Uhlitz&amp;volume=9&amp;publication_year=2018&amp;pages=20&amp;pmid=29295995&amp;doi=10.1038/s41467-017-02391-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Ruggles KV, Tang Z, Wang X, Grover H, Askenazi M, Teubl J, Cao S, McLellan MD, Clauser KR, Tabb DL, et al.  (2016). An Analysis of the Sensitivity of Proteogenomic Mapping of Somatic Mutations and Novel Splicing Events in Cancer. Mol. Cell. Proteomics
15, 1060–1071.</cite> [<a href="https://doi.org/10.1074/mcp.M115.056226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4813688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26631509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=An%20Analysis%20of%20the%20Sensitivity%20of%20Proteogenomic%20Mapping%20of%20Somatic%20Mutations%20and%20Novel%20Splicing%20Events%20in%20Cancer&amp;author=KV%20Ruggles&amp;author=Z%20Tang&amp;author=X%20Wang&amp;author=H%20Grover&amp;author=M%20Askenazi&amp;volume=15&amp;publication_year=2016&amp;pages=1060-1071&amp;pmid=26631509&amp;doi=10.1074/mcp.M115.056226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Li B, and Dewey CN (2011). RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics
12, 323.</cite> [<a href="https://doi.org/10.1186/1471-2105-12-323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21816040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=RSEM:%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B%20Li&amp;author=CN%20Dewey&amp;volume=12&amp;publication_year=2011&amp;pages=323&amp;pmid=21816040&amp;doi=10.1186/1471-2105-12-323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R80">
<span class="label">80.</span><cite>Tan VYF, and Févotte C. (2013). Automatic relevance determination in nonnegative matrix factorization with the β-divergence. IEEE Trans. Pattern Anal. Mach. Intell. 35, 1592–1605.</cite> [<a href="https://doi.org/10.1109/TPAMI.2012.240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23681989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell&amp;title=Automatic%20relevance%20determination%20in%20nonnegative%20matrix%20factorization%20with%20the%20%CE%B2-divergence&amp;author=VYF%20Tan&amp;author=C%20F%C3%A9votte&amp;volume=35&amp;publication_year=2013&amp;pages=1592-1605&amp;pmid=23681989&amp;doi=10.1109/TPAMI.2012.240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>MacLean B, Tomazela DM, Shulman N, Chambers M, Finney GL, Frewen B, Kern R, Tabb DL, Liebler DC, and MacCoss MJ (2010). Skyline: an open source document editor for creating and analyzing targeted proteomics experiments. Bioinformatics
26, 966–968.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2844992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20147306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Skyline:%20an%20open%20source%20document%20editor%20for%20creating%20and%20analyzing%20targeted%20proteomics%20experiments&amp;author=B%20MacLean&amp;author=DM%20Tomazela&amp;author=N%20Shulman&amp;author=M%20Chambers&amp;author=GL%20Finney&amp;volume=26&amp;publication_year=2010&amp;pages=966-968&amp;pmid=20147306&amp;doi=10.1093/bioinformatics/btq054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R82">
<span class="label">82.</span><cite>Haas BJ, Dobin A, Stransky N, Li B, Yang X, Tickle T, Bankapur A, Ganote C, Doak TG, Pochet N, et al.  (2017). STAR-Fusion: Fast and Accurate Fusion Transcript Detection from RNA-Seq. bioRxiv, 120295. 10.1101/120295.</cite> [<a href="https://doi.org/10.1101/120295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=STAR-Fusion:%20Fast%20and%20Accurate%20Fusion%20Transcript%20Detection%20from%20RNA-Seq&amp;author=BJ%20Haas&amp;author=A%20Dobin&amp;author=N%20Stransky&amp;author=B%20Li&amp;author=X%20Yang&amp;publication_year=2017&amp;pages=120295&amp;doi=10.1101/120295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R83">
<span class="label">83.</span><cite>Saunders CT, Wong WSW, Swamy S, Becq J, Murray LJ, and Cheetham RK (2012). Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs. Bioinformatics
28, 1811–1817.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Strelka:%20accurate%20somatic%20small-variant%20calling%20from%20sequenced%20tumor-normal%20sample%20pairs&amp;author=CT%20Saunders&amp;author=WSW%20Wong&amp;author=S%20Swamy&amp;author=J%20Becq&amp;author=LJ%20Murray&amp;volume=28&amp;publication_year=2012&amp;pages=1811-1817&amp;pmid=22581179&amp;doi=10.1093/bioinformatics/bts271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R84">
<span class="label">84.</span><cite>Koboldt DC, Zhang Q, Larson DE, Shen D, McLellan MD, Lin L, Miller CA, Mardis ER, Ding L, and Wilson RK (2012). VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 22, 568–576.</cite> [<a href="https://doi.org/10.1101/gr.129684.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3290792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=VarScan%202:%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&amp;author=DC%20Koboldt&amp;author=Q%20Zhang&amp;author=DE%20Larson&amp;author=D%20Shen&amp;author=MD%20McLellan&amp;volume=22&amp;publication_year=2012&amp;pages=568-576&amp;pmid=22300766&amp;doi=10.1101/gr.129684.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R85">
<span class="label">85.</span><cite>Chen T, and Guestrin C. (2016). XGBoost: A Scalable Tree Boosting System. In Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, pp. 785–794.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=XGBoost:%20A%20Scalable%20Tree%20Boosting%20System&amp;author=T%20Chen&amp;author=C%20Guestrin&amp;publication_year=2016&amp;pages=785-794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R86">
<span class="label">86.</span><cite>Kuhn M. (2008). Building Predictive Models inRUsing thecaretPackage. Journal of Statistical Software 28. 10.18637/jss.v028.i05.</cite> [<a href="https://doi.org/10.18637/jss.v028.i05" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Building%20Predictive%20Models%20inRUsing%20thecaretPackage&amp;author=M%20Kuhn&amp;publication_year=2008&amp;doi=10.18637/jss.v028.i05&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R87">
<span class="label">87.</span><cite>Johnson WE, Li C, and Rabinovic A. (2007). Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostatistics
8, 118–127.</cite> [<a href="https://doi.org/10.1093/biostatistics/kxj037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16632515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Adjusting%20batch%20effects%20in%20microarray%20expression%20data%20using%20empirical%20Bayes%20methods&amp;author=WE%20Johnson&amp;author=C%20Li&amp;author=A%20Rabinovic&amp;volume=8&amp;publication_year=2007&amp;pages=118-127&amp;pmid=16632515&amp;doi=10.1093/biostatistics/kxj037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R88">
<span class="label">88.</span><cite>Edwards NJ, Oberti M, Thangudu RR, Cai S, McGarvey PB, Jacob S, Madhavan S, and Ketchum KA (2015). The CPTAC Data Portal: A Resource for Cancer Proteomics Research. J. Proteome Res. 14, 2707–2713.</cite> [<a href="https://doi.org/10.1021/pr501254j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25873244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res&amp;title=The%20CPTAC%20Data%20Portal:%20A%20Resource%20for%20Cancer%20Proteomics%20Research&amp;author=NJ%20Edwards&amp;author=M%20Oberti&amp;author=RR%20Thangudu&amp;author=S%20Cai&amp;author=PB%20McGarvey&amp;volume=14&amp;publication_year=2015&amp;pages=2707-2713&amp;pmid=25873244&amp;doi=10.1021/pr501254j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R89">
<span class="label">89.</span><cite>Chen YT., Scanlan MJ., Sahin U., Türeci O., Gure AO., Tsang S., Williamson B., Stockert E., Pfreundschuh M., and Old LJ. (1997). A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening. Proc. Natl. Acad. Sci. U. S. A. 94, 1914–1918.</cite> [<a href="https://doi.org/10.1073/pnas.94.5.1914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC20017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9050879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=A%20testicular%20antigen%20aberrantly%20expressed%20in%20human%20cancers%20detected%20by%20autologous%20antibody%20screening&amp;author=YT%20Chen&amp;author=MJ%20Scanlan&amp;author=U%20Sahin&amp;author=O%20T%C3%BCreci&amp;author=AO%20Gure&amp;volume=94&amp;publication_year=1997&amp;pages=1914-1918&amp;pmid=9050879&amp;doi=10.1073/pnas.94.5.1914&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R90">
<span class="label">90.</span><cite>Tryka KA, Hao L, Sturcke A, Jin Y, Wang ZY, Ziyabari L, Lee M, Popova N, Sharopova N, Kimura M, et al.  (2014). NCBI’s Database of Genotypes and Phenotypes: dbGaP. Nucleic Acids Res. 42, D975–D979.</cite> [<a href="https://doi.org/10.1093/nar/gkt1211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24297256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=NCBI%E2%80%99s%20Database%20of%20Genotypes%20and%20Phenotypes:%20dbGaP&amp;author=KA%20Tryka&amp;author=L%20Hao&amp;author=A%20Sturcke&amp;author=Y%20Jin&amp;author=ZY%20Wang&amp;volume=42&amp;publication_year=2014&amp;pages=D975-D979&amp;pmid=24297256&amp;doi=10.1093/nar/gkt1211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R91">
<span class="label">91.</span><cite>Grossman RL, Heath AP, Ferretti V, Varmus HE, Lowy DR, Kibbe WA, and Staudt LM (2016). Toward a Shared Vision for Cancer Genomic Data. New England Journal of Medicine
375, 1109–1112. 10.1056/nejmp1607591.</cite> [<a href="https://doi.org/10.1056/nejmp1607591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6309165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27653561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Toward%20a%20Shared%20Vision%20for%20Cancer%20Genomic%20Data&amp;author=RL%20Grossman&amp;author=AP%20Heath&amp;author=V%20Ferretti&amp;author=HE%20Varmus&amp;author=DR%20Lowy&amp;volume=375&amp;publication_year=2016&amp;pages=1109-1112&amp;pmid=27653561&amp;doi=10.1056/nejmp1607591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R92">
<span class="label">92.</span><cite>O’Leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, McVeigh R, Rajput B, Robbertse B, Smith-White B, Ako-Adjei D, et al.  (2016). Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res. 44, D733–D745.</cite> [<a href="https://doi.org/10.1093/nar/gkv1189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Reference%20sequence%20(RefSeq)%20database%20at%20NCBI:%20current%20status,%20taxonomic%20expansion,%20and%20functional%20annotation&amp;author=NA%20O%E2%80%99Leary&amp;author=MW%20Wright&amp;author=JR%20Brister&amp;author=S%20Ciufo&amp;author=D%20Haddad&amp;volume=44&amp;publication_year=2016&amp;pages=D733-D745&amp;pmid=26553804&amp;doi=10.1093/nar/gkv1189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R93">
<span class="label">93.</span><cite>Consortium UniProt (2019). UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515.</cite> [<a href="https://doi.org/10.1093/nar/gky1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20a%20worldwide%20hub%20of%20protein%20knowledge&amp;author=UniProt%20Consortium&amp;volume=47&amp;publication_year=2019&amp;pages=D506-D515&amp;pmid=30395287&amp;doi=10.1093/nar/gky1049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R94">
<span class="label">94.</span><cite>Xi R, Lee S, Xia Y, Kim T-M, and Park PJ (2016). Copy number analysis of whole-genome data using BIC-seq2 and its application to detection of cancer susceptibility variants. Nucleic Acids Res. 44, 6274–6286.</cite> [<a href="https://doi.org/10.1093/nar/gkw491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5772337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27260798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Copy%20number%20analysis%20of%20whole-genome%20data%20using%20BIC-seq2%20and%20its%20application%20to%20detection%20of%20cancer%20susceptibility%20variants&amp;author=R%20Xi&amp;author=S%20Lee&amp;author=Y%20Xia&amp;author=T-M%20Kim&amp;author=PJ%20Park&amp;volume=44&amp;publication_year=2016&amp;pages=6274-6286&amp;pmid=27260798&amp;doi=10.1093/nar/gkw491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R95">
<span class="label">95.</span><cite>Kim S, Scheffler K, Halpern AL, Bekritsky MA, Noh E, Källberg M, Chen X, Kim Y, Beyter D, Krusche P, et al.  (2018). Strelka2: fast and accurate calling of germline and somatic variants. Nat. Methods
15, 591–594.</cite> [<a href="https://doi.org/10.1038/s41592-018-0051-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30013048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Strelka2:%20fast%20and%20accurate%20calling%20of%20germline%20and%20somatic%20variants&amp;author=S%20Kim&amp;author=K%20Scheffler&amp;author=AL%20Halpern&amp;author=MA%20Bekritsky&amp;author=E%20Noh&amp;volume=15&amp;publication_year=2018&amp;pages=591-594&amp;pmid=30013048&amp;doi=10.1038/s41592-018-0051-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R96">
<span class="label">96.</span><cite>Bailey MH, Tokheim C, Porta-Pardo E, Sengupta S, Bertrand D, Weerasinghe A, Colaprico A, Wendl MC, Kim J, Reardon B, et al.  (2018). Comprehensive Characterization of Cancer Driver Genes and Mutations. Cell
173, 371–385.e18.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.02.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6029450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Characterization%20of%20Cancer%20Driver%20Genes%20and%20Mutations&amp;author=MH%20Bailey&amp;author=C%20Tokheim&amp;author=E%20Porta-Pardo&amp;author=S%20Sengupta&amp;author=D%20Bertrand&amp;volume=173&amp;publication_year=2018&amp;pages=371-385&amp;pmid=29625053&amp;doi=10.1016/j.cell.2018.02.060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R97">
<span class="label">97.</span><cite>Li M, Xie X, Zhou J, Sheng M, Yin X, Ko E-A, Zhou T, and Gu W. (2017). Quantifying circular RNA expression from RNA-seq data using model-based framework. Bioinformatics
33, 2131–2139.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28334396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quantifying%20circular%20RNA%20expression%20from%20RNA-seq%20data%20using%20model-based%20framework&amp;author=M%20Li&amp;author=X%20Xie&amp;author=J%20Zhou&amp;author=M%20Sheng&amp;author=X%20Yin&amp;volume=33&amp;publication_year=2017&amp;pages=2131-2139&amp;pmid=28334396&amp;doi=10.1093/bioinformatics/btx129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R98">
<span class="label">98.</span><cite>Kim S, Gupta N, and Pevzner PA (2008). Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases. J. Proteome Res. 7, 3354–3363.</cite> [<a href="https://doi.org/10.1021/pr8001244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2689316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18597511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res&amp;title=Spectral%20probabilities%20and%20generating%20functions%20of%20tandem%20mass%20spectra:%20a%20strike%20against%20decoy%20databases&amp;author=S%20Kim&amp;author=N%20Gupta&amp;author=PA%20Pevzner&amp;volume=7&amp;publication_year=2008&amp;pages=3354-3363&amp;pmid=18597511&amp;doi=10.1021/pr8001244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R99">
<span class="label">99.</span><cite>Lange V, Picotti P, Domon B, and Aebersold R. (2008). Selected reaction monitoring for quantitative proteomics: a tutorial. Mol. Syst. Biol. 4, 222.</cite> [<a href="https://doi.org/10.1038/msb.2008.61" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2583086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18854821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol&amp;title=Selected%20reaction%20monitoring%20for%20quantitative%20proteomics:%20a%20tutorial&amp;author=V%20Lange&amp;author=P%20Picotti&amp;author=B%20Domon&amp;author=R%20Aebersold&amp;volume=4&amp;publication_year=2008&amp;pages=222&amp;pmid=18854821&amp;doi=10.1038/msb.2008.61&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R100">
<span class="label">100.</span><cite>Shi T, Fillmore TL, Sun X, Zhao R, Schepmoes AA, Hossain M, Xie F, Wu S, Kim J-S, Jones N, et al.  (2012). Antibody-free, targeted mass-spectrometric approach for quantification of proteins at low picogram per milliliter levels in human plasma/serum. Proc. Natl. Acad. Sci. U. S. A. 109, 15395–15400.</cite> [<a href="https://doi.org/10.1073/pnas.1204366109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3458402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22949669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Antibody-free,%20targeted%20mass-spectrometric%20approach%20for%20quantification%20of%20proteins%20at%20low%20picogram%20per%20milliliter%20levels%20in%20human%20plasma/serum&amp;author=T%20Shi&amp;author=TL%20Fillmore&amp;author=X%20Sun&amp;author=R%20Zhao&amp;author=AA%20Schepmoes&amp;volume=109&amp;publication_year=2012&amp;pages=15395-15400&amp;pmid=22949669&amp;doi=10.1073/pnas.1204366109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R101">
<span class="label">101.</span><cite>Sharma V, Eckels J, Schilling B, Ludwig C, Jaffe JD, MacCoss MJ, and MacLean B. (2018). Panorama Public: A Public Repository for Quantitative Data Sets Processed in Skyline. Mol. Cell. Proteomics
17, 1239–1244.</cite> [<a href="https://doi.org/10.1074/mcp.RA117.000543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5986241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29487113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Panorama%20Public:%20A%20Public%20Repository%20for%20Quantitative%20Data%20Sets%20Processed%20in%20Skyline&amp;author=V%20Sharma&amp;author=J%20Eckels&amp;author=B%20Schilling&amp;author=C%20Ludwig&amp;author=JD%20Jaffe&amp;volume=17&amp;publication_year=2018&amp;pages=1239-1244&amp;pmid=29487113&amp;doi=10.1074/mcp.RA117.000543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R102">
<span class="label">102.</span><cite>Ma Z-Q., Dasari S., Chambers MC., Litton MD., Sobecki SM., Zimmerman LJ., Halvey PJ., Schilling B., Drake PM., Gibson BW., et al. (2009). IDPicker 2.0: Improved protein assembly with high discrimination peptide identification filtering. J. Proteome Res. 8, 3872–3881.</cite> [<a href="https://doi.org/10.1021/pr900360j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2810655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19522537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res&amp;title=IDPicker%202.0:%20Improved%20protein%20assembly%20with%20high%20discrimination%20peptide%20identification%20filtering&amp;author=Z-Q%20Ma&amp;author=S%20Dasari&amp;author=MC%20Chambers&amp;author=MD%20Litton&amp;author=SM%20Sobecki&amp;volume=8&amp;publication_year=2009&amp;pages=3872-3881&amp;pmid=19522537&amp;doi=10.1021/pr900360j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R103">
<span class="label">103.</span><cite>Patro R, and Kingsford C. (2013). Predicting protein interactions via parsimonious network history inference. Bioinformatics
29, i237–i246.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3694678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23812989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Predicting%20protein%20interactions%20via%20parsimonious%20network%20history%20inference&amp;author=R%20Patro&amp;author=C%20Kingsford&amp;volume=29&amp;publication_year=2013&amp;pages=i237-i246&amp;pmid=23812989&amp;doi=10.1093/bioinformatics/btt224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R104">
<span class="label">104.</span><cite>Zhang B, Chambers MC, and Tabb DL (2007). Proteomic parsimony through bipartite graph analysis improves accuracy and transparency. J. Proteome Res. 6, 3549–3557.</cite> [<a href="https://doi.org/10.1021/pr070230d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2810678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17676885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res&amp;title=Proteomic%20parsimony%20through%20bipartite%20graph%20analysis%20improves%20accuracy%20and%20transparency&amp;author=B%20Zhang&amp;author=MC%20Chambers&amp;author=DL%20Tabb&amp;volume=6&amp;publication_year=2007&amp;pages=3549-3557&amp;pmid=17676885&amp;doi=10.1021/pr070230d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R105">
<span class="label">105.</span><cite>Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B, Lih T-SM, Chang H-Y, et al.  (2019). Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma. Cell
179, 964–983.e31.</cite> [<a href="https://doi.org/10.1016/j.cell.2019.10.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7331093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31675502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated%20Proteogenomic%20Characterization%20of%20Clear%20Cell%20Renal%20Cell%20Carcinoma&amp;author=DJ%20Clark&amp;author=SM%20Dhanasekaran&amp;author=F%20Petralia&amp;author=J%20Pan&amp;author=X%20Song&amp;volume=179&amp;publication_year=2019&amp;pages=964-983&amp;pmid=31675502&amp;doi=10.1016/j.cell.2019.10.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R106">
<span class="label">106.</span><cite>Hu Y, Pan J, Shah P, Ao M, Thomas SN, Liu Y, Chen L, Schnaubelt M, Clark DJ, Rodriguez H, et al.  (2020). Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma. Cell Rep. 33, 108276.</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7970828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33086064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Integrated%20Proteomic%20and%20Glycoproteomic%20Characterization%20of%20Human%20High-Grade%20Serous%20Ovarian%20Carcinoma&amp;author=Y%20Hu&amp;author=J%20Pan&amp;author=P%20Shah&amp;author=M%20Ao&amp;author=SN%20Thomas&amp;volume=33&amp;publication_year=2020&amp;pages=108276&amp;pmid=33086064&amp;doi=10.1016/j.celrep.2020.108276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R107">
<span class="label">107.</span><cite>Ku AT, Young AIJ, Ibrahim AA, Bu W, Jiang W, Lin M, Williams LC, McCue BL, Miles G, Nagi C, et al.  (2023). Short-term PI3K Inhibition Prevents Breast Cancer in Preclinical Models. Cancer Prev. Res. 16, 65–73.</cite> [<a href="https://doi.org/10.1158/1940-6207.CAPR-22-0275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9905287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36343340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Prev.%20Res&amp;title=Short-term%20PI3K%20Inhibition%20Prevents%20Breast%20Cancer%20in%20Preclinical%20Models&amp;author=AT%20Ku&amp;author=AIJ%20Young&amp;author=AA%20Ibrahim&amp;author=W%20Bu&amp;author=W%20Jiang&amp;volume=16&amp;publication_year=2023&amp;pages=65-73&amp;pmid=36343340&amp;doi=10.1158/1940-6207.CAPR-22-0275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R108">
<span class="label">108.</span><cite>Barbie DA, Tamayo P, Boehm JS, Kim SY, Moody SE, Dunn IF, Schinzel AC, Sandy P, Meylan E, Scholl C, et al.  (2009). Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature
462, 108–112.</cite> [<a href="https://doi.org/10.1038/nature08460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2783335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19847166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;author=DA%20Barbie&amp;author=P%20Tamayo&amp;author=JS%20Boehm&amp;author=SY%20Kim&amp;author=SE%20Moody&amp;volume=462&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19847166&amp;doi=10.1038/nature08460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R109">
<span class="label">109.</span><cite>Charoentong P, Finotello F, Angelova M, Mayer C, Efremova M, Rieder D, Hackl H, and Trajanoski Z. (2017). Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade. Cell Rep. 18, 248–262.</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.12.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28052254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Pan-cancer%20Immunogenomic%20Analyses%20Reveal%20Genotype-Immunophenotype%20Relationships%20and%20Predictors%20of%20Response%20to%20Checkpoint%20Blockade&amp;author=P%20Charoentong&amp;author=F%20Finotello&amp;author=M%20Angelova&amp;author=C%20Mayer&amp;author=M%20Efremova&amp;volume=18&amp;publication_year=2017&amp;pages=248-262&amp;pmid=28052254&amp;doi=10.1016/j.celrep.2016.12.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R110">
<span class="label">110.</span><cite>Mani DR, Maynard M, Kothadia R, Krug K, Christianson KE, Heiman D, Clauser KR, Birger C, Getz G, and Carr SA (2021). PANOPLY: a cloud-based platform for automated and reproducible proteogenomic data analysis. Nat. Methods
18, 580–582.</cite> [<a href="https://doi.org/10.1038/s41592-021-01176-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8791030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34040252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=PANOPLY:%20a%20cloud-based%20platform%20for%20automated%20and%20reproducible%20proteogenomic%20data%20analysis&amp;author=DR%20Mani&amp;author=M%20Maynard&amp;author=R%20Kothadia&amp;author=K%20Krug&amp;author=KE%20Christianson&amp;volume=18&amp;publication_year=2021&amp;pages=580-582&amp;pmid=34040252&amp;doi=10.1038/s41592-021-01176-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R111">
<span class="label">111.</span><cite>Kather JN, Heij LR, Grabsch HI, Loeffler C, Echle A, Muti HS, Krause J, Niehues JM, Sommer KAJ, Bankhead P, et al.  (2020). Pan-cancer image-based detection of clinically actionable genetic alterations. Nat Cancer
1, 789–799.</cite> [<a href="https://doi.org/10.1038/s43018-020-0087-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33763651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=Pan-cancer%20image-based%20detection%20of%20clinically%20actionable%20genetic%20alterations&amp;author=JN%20Kather&amp;author=LR%20Heij&amp;author=HI%20Grabsch&amp;author=C%20Loeffler&amp;author=A%20Echle&amp;volume=1&amp;publication_year=2020&amp;pages=789-799&amp;pmid=33763651&amp;doi=10.1038/s43018-020-0087-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R112">
<span class="label">112.</span><cite>Kanehisa M, Furumichi M, Tanabe M, Sato Y, and Morishima K. (2017). KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353–D361.</cite> [<a href="https://doi.org/10.1093/nar/gkw1092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20new%20perspectives%20on%20genomes,%20pathways,%20diseases%20and%20drugs&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=M%20Tanabe&amp;author=Y%20Sato&amp;author=K%20Morishima&amp;volume=45&amp;publication_year=2017&amp;pages=D353-D361&amp;pmid=27899662&amp;doi=10.1093/nar/gkw1092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R113">
<span class="label">113.</span><cite>Jassal B., Matthews L., Viteri G., Gong C., Lorente P., Fabregat A., Sidiropoulos K., Cook J., Gillespie M., Haw R., et al. (2020). The reactome pathway knowledgebase. Nucleic Acids Res. 48, D498–D503.</cite> [<a href="https://doi.org/10.1093/nar/gkz1031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31691815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20reactome%20pathway%20knowledgebase&amp;author=B%20Jassal&amp;author=L%20Matthews&amp;author=G%20Viteri&amp;author=C%20Gong&amp;author=P%20Lorente&amp;volume=48&amp;publication_year=2020&amp;pages=D498-D503&amp;pmid=31691815&amp;doi=10.1093/nar/gkz1031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R114">
<span class="label">114.</span><cite>Wishart DS, Feunang YD, Marcu A, Guo AC, Liang K, Vázquez-Fresno R, Sajed T, Johnson D, Li C, Karu N, et al.  (2018). HMDB 4.0: the human metabolome database for 2018. Nucleic Acids Res. 46, D608–D617.</cite> [<a href="https://doi.org/10.1093/nar/gkx1089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29140435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=HMDB%204.0:%20the%20human%20metabolome%20database%20for%202018&amp;author=DS%20Wishart&amp;author=YD%20Feunang&amp;author=A%20Marcu&amp;author=AC%20Guo&amp;author=K%20Liang&amp;volume=46&amp;publication_year=2018&amp;pages=D608-D617&amp;pmid=29140435&amp;doi=10.1093/nar/gkx1089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R115">
<span class="label">115.</span><cite>Frolkis A, Knox C, Lim E, Jewison T, Law V, Hau DD, Liu P, Gautam B, Ly S, Guo AC, et al.  (2010). SMPDB: The Small Molecule Pathway Database. Nucleic Acids Res. 38, D480–D487.</cite> [<a href="https://doi.org/10.1093/nar/gkp1002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2808928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19948758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SMPDB:%20The%20Small%20Molecule%20Pathway%20Database&amp;author=A%20Frolkis&amp;author=C%20Knox&amp;author=E%20Lim&amp;author=T%20Jewison&amp;author=V%20Law&amp;volume=38&amp;publication_year=2010&amp;pages=D480-D487&amp;pmid=19948758&amp;doi=10.1093/nar/gkp1002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R116">
<span class="label">116.</span><cite>Liberzon A, Birger C, Thorvaldsdóttir H, Ghandi M, Mesirov JP, and Tamayo P. (2015). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst
1, 417–425.</cite> [<a href="https://doi.org/10.1016/j.cels.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20Molecular%20Signatures%20Database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;author=A%20Liberzon&amp;author=C%20Birger&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=M%20Ghandi&amp;author=JP%20Mesirov&amp;volume=1&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R117">
<span class="label">117.</span><cite>Martens M, Ammar A, Riutta A, Waagmeester A, Slenter DN, Hanspers K, A Miller R, Digles D, Lopes EN, Ehrhart F, et al.  (2021). WikiPathways: connecting communities. Nucleic Acids Res. 49, D613–D621.</cite> [<a href="https://doi.org/10.1093/nar/gkaa1024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33211851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WikiPathways:%20connecting%20communities&amp;author=M%20Martens&amp;author=A%20Ammar&amp;author=A%20Riutta&amp;author=A%20Waagmeester&amp;author=DN%20Slenter&amp;volume=49&amp;publication_year=2021&amp;pages=D613-D621&amp;pmid=33211851&amp;doi=10.1093/nar/gkaa1024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R118">
<span class="label">118.</span><cite>Huang R, Grishagin I, Wang Y, Zhao T, Greene J, Obenauer JC, Ngan D, Nguyen D-T, Guha R, Jadhav A, et al.  (2019). The NCATS BioPlanet - An Integrated Platform for Exploring the Universe of Cellular Signaling Pathways for Toxicology, Systems Biology, and Chemical Genomics. Front. Pharmacol. 10, 445.</cite> [<a href="https://doi.org/10.3389/fphar.2019.00445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6524730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol&amp;title=The%20NCATS%20BioPlanet%20-%20An%20Integrated%20Platform%20for%20Exploring%20the%20Universe%20of%20Cellular%20Signaling%20Pathways%20for%20Toxicology,%20Systems%20Biology,%20and%20Chemical%20Genomics&amp;author=R%20Huang&amp;author=I%20Grishagin&amp;author=Y%20Wang&amp;author=T%20Zhao&amp;author=J%20Greene&amp;volume=10&amp;publication_year=2019&amp;pages=445&amp;pmid=31133849&amp;doi=10.3389/fphar.2019.00445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R119">
<span class="label">119.</span><cite>Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, and Ma’ayan A. (2016). The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. Database 2016. 10.1093/database/baw100.</cite> [<a href="https://doi.org/10.1093/database/baw100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4930834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database%202016&amp;title=The%20harmonizome:%20a%20collection%20of%20processed%20datasets%20gathered%20to%20serve%20and%20mine%20knowledge%20about%20genes%20and%20proteins&amp;author=AD%20Rouillard&amp;author=GW%20Gundersen&amp;author=NF%20Fernandez&amp;author=Z%20Wang&amp;author=CD%20Monteiro&amp;publication_year=2016&amp;pmid=27374120&amp;doi=10.1093/database/baw100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R120">
<span class="label">120.</span><cite>Aran D, Hu Z, and Butte AJ (2017). xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 18, 220.</cite> [<a href="https://doi.org/10.1186/s13059-017-1349-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5688663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29141660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=xCell:%20digitally%20portraying%20the%20tissue%20cellular%20heterogeneity%20landscape&amp;author=D%20Aran&amp;author=Z%20Hu&amp;author=AJ%20Butte&amp;volume=18&amp;publication_year=2017&amp;pages=220&amp;pmid=29141660&amp;doi=10.1186/s13059-017-1349-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R121">
<span class="label">121.</span><cite>Tabula Sapiens Consortium*, Jones RC, Karkanias J, Krasnow MA, Pisco AO, Quake SR, Salzman J, Yosef N, Bulthaup B, Brown P, et al.  (2022). The Tabula Sapiens: A multiple-organ, single-cell transcriptomic atlas of humans. Science
376, eabl4896.</cite> [<a href="https://doi.org/10.1126/science.abl4896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9812260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35549404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Tabula%20Sapiens:%20A%20multiple-organ,%20single-cell%20transcriptomic%20atlas%20of%20humans&amp;author=RC%20Jones&amp;author=J%20Karkanias&amp;author=MA%20Krasnow&amp;author=AO%20Pisco&amp;author=SR%20Quake&amp;volume=376&amp;publication_year=2022&amp;pmid=35549404&amp;doi=10.1126/science.abl4896&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R122">
<span class="label">122.</span><cite>Poole W, Gibbs DL, Shmulevich I, Bernard B, and Knijnenburg TA (2016). Combining dependent P-values with an empirical adaptation of Brown’s method. Bioinformatics
32, i430–i436.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5013915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27587659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Combining%20dependent%20P-values%20with%20an%20empirical%20adaptation%20of%20Brown%E2%80%99s%20method&amp;author=W%20Poole&amp;author=DL%20Gibbs&amp;author=I%20Shmulevich&amp;author=B%20Bernard&amp;author=TA%20Knijnenburg&amp;volume=32&amp;publication_year=2016&amp;pages=i430-i436&amp;pmid=27587659&amp;doi=10.1093/bioinformatics/btw438&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R123">
<span class="label">123.</span><cite>Rykunov D, Beckmann ND, Li H, Uzilov A, Schadt EE, and Reva B. (2016). A new molecular signature method for prediction of driver cancer pathways from transcriptional data. Nucleic Acids Res. 44, e110.</cite> [<a href="https://doi.org/10.1093/nar/gkw269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4914110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27098033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=A%20new%20molecular%20signature%20method%20for%20prediction%20of%20driver%20cancer%20pathways%20from%20transcriptional%20data&amp;author=D%20Rykunov&amp;author=ND%20Beckmann&amp;author=H%20Li&amp;author=A%20Uzilov&amp;author=EE%20Schadt&amp;volume=44&amp;publication_year=2016&amp;pages=e110&amp;pmid=27098033&amp;doi=10.1093/nar/gkw269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>1</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-1.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-1.pdf</a><sup> (8.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>2</span><p>Table S1. Genotype, phenotype, immune scores, clinical, and pathological characteristics of tumor and normal samples. Related to <a href="#F1" class="usa-link">Figure 1</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-2.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-2.xlsx</a><sup> (270.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>3</span><p>Table S2. Mutation signatures, CNV drivers, CNV arm immune score correlations, and PARP-i drug sensitivity data. Related to <a href="#F1" class="usa-link">Figures 1</a> and <a href="#F6" class="usa-link">6</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-3.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-3.xlsx</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>4</span><p>Table S3. <em>PIK3R1</em> mutation status, AKT inhibitor response of EC cell lines, and sequencing of CRISPR-Cas9 edited cells. Related to <a href="#F2" class="usa-link">Figure 2</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-4.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-4.xlsx</a><sup> (563.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>5</span><p>Table S4. Immune subgroup, immune scores and SRM peptide levels for ICI treated samples. Related to <a href="#F3" class="usa-link">Figure 3</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-5.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-5.xlsx</a><sup> (111KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>6</span><p>Table S5. Inferred MYC activity in CMAP, TCGA, Exploratory and Independent cohorts, total MYC IHC scores. Related to <a href="#F4" class="usa-link">Figure 4</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-6.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-6.xlsx</a><sup> (55.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>7</span><p>Table S6. Transporters scores and classes, <em>CTNNB1</em> mutation protein fold change, and top classifying genes for hotspot mutation. Related to <a href="#F5" class="usa-link">Figure 5</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-7.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-7.xlsx</a><sup> (625.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>8</span><p>Table S7. Deep learning models, multi-omic NMF clustering, and glycopeptide NMF clustering. Related to <a href="#F6" class="usa-link">Figures 6</a> and <a href="#F7" class="usa-link">7</a>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/10631452/bin/NIHMS1921121-supplement-8.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS1921121-supplement-8.xlsx</a><sup> (176.3KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Clinical and proteomic (raw MS files and processed data files) data reported in this paper, including both exploratory and independent CPTAC datasets, can be accessed via the Proteomic Data Commons (PDC) at: <a href="https://pdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pdc.cancer.gov/</a> (PDC000125, PDC000126, PDC000226). Genomic and transcriptomic data files for both CPTAC cohorts (phs001287) and TCGA cohort (phs000178) can be accessed via Genomic Data Commons (GDC) at: <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/projects/CPTAC-3</a>. The DepMap datasets can be accessed via the DepMap portal: <a href="https://depmap.org/portal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/</a>. Processed CPTAC data for both cohorts used in this publication can also be found in the PDC, the Python package called ‘cptac’ (<a href="https://pypi.org/project/cptac/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pypi.org/project/cptac/</a>, install via pip) to allow programmatic access and LinkedOmics via <a href="http://www.linkedomics.org/data_download/CPTAC-UCEC-independent/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.linkedomics.org/data_download/CPTAC-UCEC-independent/</a>
<sup><a href="#R64" class="usa-link" aria-describedby="R64" aria-expanded="false">64</a></sup>. Histological and radiological images for both CPTAC cohorts (10.7937/k9/tcia.2018.3r3juisw) and TCGA cohort (10.7937/k9/tcia.2016.gkj0zwac) can be accessed via Imaging Data Commons (IDC) at <a href="https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ucec</a>, and The Cancer Imaging Archive at <a href="https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=33948263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=33948263</a>. Deidentified digitized H&amp;E slides from NYU reported in this paper will be shared by the <a href="#S19" class="usa-link">lead contact</a> upon request.</p></section></section></section></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1016/j.ccell.2023.07.007" class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/nihms-1921121.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/10631452/" data-citation-style="nlm" data-download-format-link="/resources/citations/10631452/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC10631452%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC10631452/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC10631452/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC10631452/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/37567170/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC10631452/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/37567170/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC10631452/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/10631452/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="ABS1" href="#ABS1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="SUMMARY">SUMMARY</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ABS2" href="#ABS2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="eTOC Blurb">eTOC Blurb</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ABS3" href="#ABS3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Graphical Abstract">Graphical Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S1" href="#S1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="INTRODUCTION">INTRODUCTION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S2" href="#S2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="RESULTS">RESULTS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S14" href="#S14" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="DISCUSSION">DISCUSSION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S15" href="#S15" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="STAR METHODS">STAR METHODS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="SM1" href="#SM1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary Material">Supplementary Material</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="S91" href="#S91" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="ACKNOWLEDGEMENTS">ACKNOWLEDGEMENTS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MBeOXDmkcA0UBHYkOsjDwcvjSHC4dTW21lBR3jgXNNaPxYAwIk8U46Ugakr6STvs">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Amanuscript" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.28.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.28.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 150px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>