{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "ad1baf88",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "from data_gatherer.llm.response_schema import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a0319a9e",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 329 - INFO - Setting up data fetcher...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 359 - INFO - Data fetcher setup completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 108 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.5-flash\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "data_gatherer = DataGatherer(llm_name=\"gemini-2.5-flash\", log_level=\"INFO\", process_entire_document=True, load_from_cache=False, save_to_cache=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "d47ab036",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization of Endometrial Carcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability.pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic and metabolomic characterization of human glioblastoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic characterization of pancreatic ductal adenocarcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/A proteogenomic portrait of lung squamous cell carcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma .pdf']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles = []\n",
    "\n",
    "for article in os.listdir('scripts/exp_input/bdf_paper_pdfs/'):\n",
    "    articles.append(os.path.join('scripts/exp_input/bdf_paper_pdfs/', article))\n",
    "\n",
    "articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "06d39d47",
   "metadata": {},
   "outputs": [],
   "source": [
    "articles = [\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7331093/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7233456/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8077737/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6768830/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8044053/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7946781/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7373300/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7289043/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8654574/\"\n",
    "]\n",
    "\n",
    "articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "665d9186",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 700 - INFO - #0 function call: self.process_url(scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 425 - INFO - Processing URL: scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 183 - WARNING - No valid root extracted from URL. This may cause issues with data gathering.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 365 - INFO - URL scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf ends with .pdf\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 268 - INFO - URL scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf is a PDF. Using PdfFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 433 - INFO - Type of data_fetcher PdfFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 454 - INFO - Local file scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf detected. Using it as raw data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 503 - INFO - Parsing Raw content from format: pdf\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 544 - INFO - Using PDFParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.5-flash\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 302 - INFO - Function call: parse_data(scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf, scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf, additional_data, pdf)\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 147 - INFO - Extracting text from PDF using PyMuPDF: scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 217 - INFO - Normalizing extracted text. Initial length: 133937 characters\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 221 - INFO - Number of pages detected: 35\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 236 - INFO - Removing frequent header/footer: 'NewPage'\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 249 - INFO - Normalized text length: 133238\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 311 - INFO - Extracting links from full content.\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 393 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Descr from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 401 - INFO - Content length: 133238\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 412 - INFO - Prompt messages total length: 34136 tokens\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 418 - INFO - Prompt ID: gemini-2.5-flash-0-8f67440f966ccc908b70e77ea912a68d8b3511549c29005c3bcc17a264e04d4b\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 437 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 34136 tokens, schema: {'type': 'json_schema', 'name': 'PaperMiner_dataset_provenance_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository where the dataset can be found.'}, 'dataset_context_from_paper': {'type': 'string', 'description': 'Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly.'}, 'citation_type': {'type': 'string', 'description': 'Type of citation used for this dataset. It can be either Primary (firsthand information collected by the researcher for a specific purpose) or Secondary (pre-existing information collected by someone else and then used by another researcher).'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference', 'dataset_context_from_paper', 'citation_type']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.5-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-0ae46253-a8e1-453a-b2d9-e4baca0cfe04\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n{\\n  \\\"datasets\\\": [\\n    {\\n      \\\"dataset_identifier\\\": \\\"PRJNA514017 / SRP178677\\\",\\n      \\\"repository_reference\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Raw genomics data from this study, including Whole-Exome Sequencing (WXS), RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) data, along with Illumina SNP array data, were collected from 110 prospectively collected colon cancer patients (tumor specimens, matched normal adjacent tissues, and blood samples). This data was used for somatic mutation detection (SNVs, INDELs, MS INDELs), somatic copy number alteration (SCNA) analysis, HLA genotyping, subtype classification, and neoantigen identification.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S045\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S045\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Raw and low-level processed proteomics data generated in this study, comprising label-free shotgun proteomic analyses, isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and normal adjacent tissues from 110 colon cancer patients (specifically 96 tumor and NAT pairs for TMT). Additionally, selected reaction monitoring (SRM) data for validation of specific proteins (CD8A, SLC2A3, PKM1, PKM2) in representative tumor samples is included. This data was instrumental in identifying colon cancer-associated proteins and phosphosites, investigating proteomic consequences of somatic mutations, predicting kinase activity, identifying tumor antigens (neoantigens, CT antigens), and enabling subtype classification.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"cptac-colon\\\",\\n      \\\"repository_reference\\\": \\\"http://linkedomics.org/cptac-colon/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"All final processed data matrices derived from the proteogenomic analysis of the colon cancer cohort (CPTAC Colon Cancer Study) are available through the LinkedOmics portal. This resource provides computational tools for further exploration and is intended for broad usage by other investigators for new biological discoveries and therapeutic hypothesis generation.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S037\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"This table summarizes the clinical and pathological characteristics of the 110 colon cancer tumors and outlines the specific numbers of samples analyzed by each omics platform within the study cohort. This clinical data was collected in strict accordance with the CPTAC-2 colon procurement protocol to ensure suitability for analysis.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S2\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S045\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"PCR-based microsatellite instability (MSI) testing results were obtained for 85 samples from the colon cancer cohort. This data was used to validate the WXS-based MSI assignments, confirming agreement between the two methods.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"The Cancer Genome Atlas (TCGA) Cohort\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"dbSNP150\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/COSMIC83 (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The dbSNP150 database was utilized for variant annotation as part of the process to identify protein-altering somatic mutations in the study's WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"COSMIC83\\\",\\n      \\\"repository_reference\\\": \\\"https://cancer.sanger.ac.uk/cosmicClinvar (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The COSMIC83 database was used for variant annotation during the identification of protein-altering somatic mutations from the whole-exome sequencing data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Clinvar (version 20170905)\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/dbNSFP33a (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"dbNSFP33a\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The dbNSFP33a database was used as a source for functional predictions during the annotation of human nonsynonymous SNPs and other protein-altering somatic mutations identified in the WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PhosphoSitePlus\\\",\\n      \\\"repository_reference\\\": \\\"https://www.phosphosite.org/SignorLo\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"PhosphoSitePlus was used to compile kinase-substrate interactions, which were then utilized in the phosphosite set enrichment analysis for kinase activity prediction. It also provided general phosphosite information.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Signor\\\",\\n      \\\"repository_reference\\\": \\\"https://signor.uniroma2.it/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The SIGNOR database was referenced to identify sites annotated as activating kinase activity for use in kinase activity prediction. It also contributed to the compilation of kinase-substrate interactions for phosphosite set enrichment analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"The Human Protein Atlas (HPA)\\\",\\n      \\\"repository_reference\\\": \\\"https://www.proteinatlas.org/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CTdatabase\\\",\\n      \\\"repository_reference\\\": \\\"http://www.cta.lncc.br/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The CTdatabase, a knowledge-base containing 269 carefully curated and annotated cancer-testis (CT) antigens, was the source from which CT antigens were downloaded for further analysis in this study.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"MSigDB (Molecular Signatures Database)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Liberzon et al., 2015.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Hallmark gene sets were collected from the Molecular Signatures Database (MSigDB) to perform gene set enrichment analyses, specifically for Rb phosphorylation quantification and correlation analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"ENCODE Project Consortium Data (2011)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but implied by the citation ENCODE Project Consortium (2011).\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Data from the ENCODE Project Consortium was utilized to collect transcriptional targets of E2F transcription factors, which were then used to infer E2F1 activity.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Synapse Platform metabolic gene sets\\\",\\n      \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706/wiki/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"RefSeq protein database\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"NCI-60 colorectal cell lines\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Gholami et al., 2013.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Proteome data from 60 NCI-60 colorectal cell lines were used for Pearson's correlation coefficient analysis of average gene-wise protein expression, enabling comparison with the prospective colon tumor samples.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Colorectal cell lines (Wang et al., 2017b)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Wang et al., 2017b.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"RNA-seq and proteomic data from 44 colorectal cell lines were utilized for Pearson's correlation coefficient analysis of average gene-wise RNA and protein expression, allowing for comparison with the prospective colon tumor samples.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    }\\n  ]\\n}\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n{\\n  \\\"datasets\\\": [\\n    {\\n      \\\"dataset_identifier\\\": \\\"PRJNA514017 / SRP178677\\\",\\n      \\\"repository_reference\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Raw genomics data from this study, including Whole-Exome Sequencing (WXS), RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) data, along with Illumina SNP array data, were collected from 110 prospectively collected colon cancer patients (tumor specimens, matched normal adjacent tissues, and blood samples). This data was used for somatic mutation detection (SNVs, INDELs, MS INDELs), somatic copy number alteration (SCNA) analysis, HLA genotyping, subtype classification, and neoantigen identification.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S045\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S045\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Raw and low-level processed proteomics data generated in this study, comprising label-free shotgun proteomic analyses, isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and normal adjacent tissues from 110 colon cancer patients (specifically 96 tumor and NAT pairs for TMT). Additionally, selected reaction monitoring (SRM) data for validation of specific proteins (CD8A, SLC2A3, PKM1, PKM2) in representative tumor samples is included. This data was instrumental in identifying colon cancer-associated proteins and phosphosites, investigating proteomic consequences of somatic mutations, predicting kinase activity, identifying tumor antigens (neoantigens, CT antigens), and enabling subtype classification.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"cptac-colon\\\",\\n      \\\"repository_reference\\\": \\\"http://linkedomics.org/cptac-colon/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"All final processed data matrices derived from the proteogenomic analysis of the colon cancer cohort (CPTAC Colon Cancer Study) are available through the LinkedOmics portal. This resource provides computational tools for further exploration and is intended for broad usage by other investigators for new biological discoveries and therapeutic hypothesis generation.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S037\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"This table summarizes the clinical and pathological characteristics of the 110 colon cancer tumors and outlines the specific numbers of samples analyzed by each omics platform within the study cohort. This clinical data was collected in strict accordance with the CPTAC-2 colon procurement protocol to ensure suitability for analysis.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S2\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/cptac/s/S045\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"PCR-based microsatellite instability (MSI) testing results were obtained for 85 samples from the colon cancer cohort. This data was used to validate the WXS-based MSI assignments, confirming agreement between the two methods.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"The Cancer Genome Atlas (TCGA) Cohort\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"dbSNP150\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/COSMIC83 (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The dbSNP150 database was utilized for variant annotation as part of the process to identify protein-altering somatic mutations in the study's WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"COSMIC83\\\",\\n      \\\"repository_reference\\\": \\\"https://cancer.sanger.ac.uk/cosmicClinvar (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The COSMIC83 database was used for variant annotation during the identification of protein-altering somatic mutations from the whole-exome sequencing data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Clinvar (version 20170905)\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/dbNSFP33a (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"dbNSFP33a\\\",\\n      \\\"repository_reference\\\": \\\"http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus (accessed via ANNOVAR)\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The dbNSFP33a database was used as a source for functional predictions during the annotation of human nonsynonymous SNPs and other protein-altering somatic mutations identified in the WXS data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PhosphoSitePlus\\\",\\n      \\\"repository_reference\\\": \\\"https://www.phosphosite.org/SignorLo\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"PhosphoSitePlus was used to compile kinase-substrate interactions, which were then utilized in the phosphosite set enrichment analysis for kinase activity prediction. It also provided general phosphosite information.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Signor\\\",\\n      \\\"repository_reference\\\": \\\"https://signor.uniroma2.it/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The SIGNOR database was referenced to identify sites annotated as activating kinase activity for use in kinase activity prediction. It also contributed to the compilation of kinase-substrate interactions for phosphosite set enrichment analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"The Human Protein Atlas (HPA)\\\",\\n      \\\"repository_reference\\\": \\\"https://www.proteinatlas.org/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CTdatabase\\\",\\n      \\\"repository_reference\\\": \\\"http://www.cta.lncc.br/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The CTdatabase, a knowledge-base containing 269 carefully curated and annotated cancer-testis (CT) antigens, was the source from which CT antigens were downloaded for further analysis in this study.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"MSigDB (Molecular Signatures Database)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Liberzon et al., 2015.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Hallmark gene sets were collected from the Molecular Signatures Database (MSigDB) to perform gene set enrichment analyses, specifically for Rb phosphorylation quantification and correlation analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"ENCODE Project Consortium Data (2011)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but implied by the citation ENCODE Project Consortium (2011).\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Data from the ENCODE Project Consortium was utilized to collect transcriptional targets of E2F transcription factors, which were then used to infer E2F1 activity.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Synapse Platform metabolic gene sets\\\",\\n      \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706/wiki/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"RefSeq protein database\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"NCI-60 colorectal cell lines\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Gholami et al., 2013.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Proteome data from 60 NCI-60 colorectal cell lines were used for Pearson's correlation coefficient analysis of average gene-wise protein expression, enabling comparison with the prospective colon tumor samples.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Colorectal cell lines (Wang et al., 2017b)\\\",\\n      \\\"repository_reference\\\": \\\"Not explicitly linked, but referenced via Wang et al., 2017b.\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"RNA-seq and proteomic data from 44 colorectal cell lines were utilized for Pearson's correlation coefficient analysis of average gene-wise RNA and protein expression, allowing for comparison with the prospective colon tumor samples.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    }\\n  ]\\n}\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1759423567,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 38452,\n",
      "        \"completion_tokens\": 2711,\n",
      "        \"total_tokens\": 48250,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 7087,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 20 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PRJNA514017 / SRP178677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: cptac-colon\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: The Cancer Genome Atlas (TCGA) Cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: dbSNP150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: COSMIC83\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Clinvar (version 20170905)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: dbNSFP33a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Signor\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: The Human Protein Atlas (HPA)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: MSigDB (Molecular Signatures Database)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: ENCODE Project Consortium Data (2011)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Synapse Platform metabolic gene sets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: RefSeq protein database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: NCI-60 colorectal cell lines\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Colorectal cell lines (Wang et al., 2017b)\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 468 - INFO - Response <class 'list'> not saved (caching disabled)\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'PRJNA514017 / SRP178677', 'repository_reference': 'ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a', 'dataset_context_from_paper': 'Raw genomics data from this study, including Whole-Exome Sequencing (WXS), RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) data, along with Illumina SNP array data, were collected from 110 prospectively collected colon cancer patients (tumor specimens, matched normal adjacent tissues, and blood samples). This data was used for somatic mutation detection (SNVs, INDELs, MS INDELs), somatic copy number alteration (SCNA) analysis, HLA genotyping, subtype classification, and neoantigen identification.', 'citation_type': 'Primary', 'dataset_id': 'PRJNA514017 / SRP178677'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA514017 / SRP178677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PRJNA514017 / SRP178677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a, identifier: PRJNA514017 / SRP178677, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA514017 / SRP178677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PRJNA514017 / SRP178677, unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PRJNA514017 / SRP178677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PRJNA514017 / SRP178677, unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'S045', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/cptac/s/S045', 'dataset_context_from_paper': 'Raw and low-level processed proteomics data generated in this study, comprising label-free shotgun proteomic analyses, isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and normal adjacent tissues from 110 colon cancer patients (specifically 96 tumor and NAT pairs for TMT). Additionally, selected reaction monitoring (SRM) data for validation of specific proteins (CD8A, SLC2A3, PKM1, PKM2) in representative tumor samples is included. This data was instrumental in identifying colon cancer-associated proteins and phosphosites, investigating proteomic consequences of somatic mutations, predicting kinase activity, identifying tumor antigens (neoantigens, CT antigens), and enabling subtype classification.', 'citation_type': 'Primary', 'dataset_id': 'S045'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID S045 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/cptac/s/S045, identifier: S045, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/cptac/s/S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/cptac/s/S045,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/cptac/s/S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: S045, cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'cptac-colon', 'repository_reference': 'http://linkedomics.org/cptac-colon/', 'dataset_context_from_paper': 'All final processed data matrices derived from the proteogenomic analysis of the colon cancer cohort (CPTAC Colon Cancer Study) are available through the LinkedOmics portal. This resource provides computational tools for further exploration and is intended for broad usage by other investigators for new biological discoveries and therapeutic hypothesis generation.', 'citation_type': 'Primary', 'dataset_id': 'cptac-colon'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: cptac-colon\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID cptac-colon is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: http://linkedomics.org/cptac-colon/, identifier: cptac-colon, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://linkedomics.org/cptac-colon/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(http://linkedomics.org/cptac-colon/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: http://linkedomics.org/cptac-colon/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: cptac-colon, linkedomics.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Table S1', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/cptac/s/S037', 'dataset_context_from_paper': 'This table summarizes the clinical and pathological characteristics of the 110 colon cancer tumors and outlines the specific numbers of samples analyzed by each omics platform within the study cohort. This clinical data was collected in strict accordance with the CPTAC-2 colon procurement protocol to ensure suitability for analysis.', 'citation_type': 'Primary', 'dataset_id': 'Table S1'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 853 - INFO - Accession ID Table S1 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/cptac/s/S037, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/cptac/s/S037\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/cptac/s/S037,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/cptac/s/S037\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: n/a, cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Table S2', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/cptac/s/S045', 'dataset_context_from_paper': 'PCR-based microsatellite instability (MSI) testing results were obtained for 85 samples from the colon cancer cohort. This data was used to validate the WXS-based MSI assignments, confirming agreement between the two methods.', 'citation_type': 'Primary', 'dataset_id': 'Table S2'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 853 - INFO - Accession ID Table S2 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/cptac/s/S045, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/cptac/s/S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/cptac/s/S045,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/cptac/s/S045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: n/a, cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'The Cancer Genome Atlas (TCGA) Cohort', 'repository_reference': 'Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).', 'dataset_context_from_paper': 'The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.', 'citation_type': 'Secondary', 'dataset_id': 'The Cancer Genome Atlas (TCGA) Cohort'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016)., identifier: The Cancer Genome Atlas (TCGA) Cohort, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks  Colaprico et al., 2016 .,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks  Colaprico et al., 2016 .\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks  Colaprico et al., 2016 .\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016)., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016)., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'dbSNP150', 'repository_reference': 'http://annovar.openbioinformatics.org/en/latest/COSMIC83 (accessed via ANNOVAR)', 'dataset_context_from_paper': \"The dbSNP150 database was utilized for variant annotation as part of the process to identify protein-altering somatic mutations in the study's WXS data.\", 'citation_type': 'Secondary', 'dataset_id': 'dbSNP150'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: dbSNP150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID dbSNP150 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: http://annovar.openbioinformatics.org/en/latest/COSMIC83 (accessed via ANNOVAR), identifier: dbSNP150, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://annovar.openbioinformatics.org/en/latest/COSMIC83 (accessed via ANNOVAR)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(http://annovar.openbioinformatics.org/en/latest/COSMIC83  accessed via ANNOVAR ,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: http://annovar.openbioinformatics.org/en/latest/COSMIC83  accessed via ANNOVAR \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: dbSNP150, annovar.openbioinformatics.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'COSMIC83', 'repository_reference': 'https://cancer.sanger.ac.uk/cosmicClinvar (accessed via ANNOVAR)', 'dataset_context_from_paper': 'The COSMIC83 database was used for variant annotation during the identification of protein-altering somatic mutations from the whole-exome sequencing data.', 'citation_type': 'Secondary', 'dataset_id': 'COSMIC83'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: COSMIC83\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID COSMIC83 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://cancer.sanger.ac.uk/cosmicClinvar (accessed via ANNOVAR), identifier: COSMIC83, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://cancer.sanger.ac.uk/cosmicClinvar (accessed via ANNOVAR)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://cancer.sanger.ac.uk/cosmicClinvar  accessed via ANNOVAR ,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://cancer.sanger.ac.uk/cosmicClinvar  accessed via ANNOVAR \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: COSMIC83, cancer.sanger.ac.uk, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Clinvar (version 20170905)', 'repository_reference': 'http://annovar.openbioinformatics.org/en/latest/dbNSFP33a (accessed via ANNOVAR)', 'dataset_context_from_paper': 'The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.', 'citation_type': 'Secondary', 'dataset_id': 'Clinvar (version 20170905)'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Clinvar (version 20170905)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Clinvar (version 20170905)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: http://annovar.openbioinformatics.org/en/latest/dbNSFP33a (accessed via ANNOVAR), identifier: Clinvar (version 20170905), dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://annovar.openbioinformatics.org/en/latest/dbNSFP33a (accessed via ANNOVAR)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(http://annovar.openbioinformatics.org/en/latest/dbNSFP33a  accessed via ANNOVAR ,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: http://annovar.openbioinformatics.org/en/latest/dbNSFP33a  accessed via ANNOVAR \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Clinvar (version 20170905)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Clinvar (version 20170905), annovar.openbioinformatics.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Clinvar (version 20170905)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: annovar.openbioinformatics.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Clinvar (version 20170905), annovar.openbioinformatics.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'dbNSFP33a', 'repository_reference': 'http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus (accessed via ANNOVAR)', 'dataset_context_from_paper': 'The dbNSFP33a database was used as a source for functional predictions during the annotation of human nonsynonymous SNPs and other protein-altering somatic mutations identified in the WXS data.', 'citation_type': 'Secondary', 'dataset_id': 'dbNSFP33a'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: dbNSFP33a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID dbNSFP33a is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus (accessed via ANNOVAR), identifier: dbNSFP33a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus (accessed via ANNOVAR)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus  accessed via ANNOVAR ,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: http://annovar.openbioinformatics.org/en/latest/PhosphoSitePlus  accessed via ANNOVAR \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: dbNSFP33a, annovar.openbioinformatics.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org/SignorLo', 'dataset_context_from_paper': 'PhosphoSitePlus was used to compile kinase-substrate interactions, which were then utilized in the phosphosite set enrichment analysis for kinase activity prediction. It also provided general phosphosite information.', 'citation_type': 'Secondary', 'dataset_id': 'PhosphoSitePlus'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID PhosphoSitePlus is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://www.phosphosite.org/SignorLo, identifier: PhosphoSitePlus, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.phosphosite.org/SignorLo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://www.phosphosite.org/SignorLo,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://www.phosphosite.org/SignorLo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PhosphoSitePlus, www.phosphosite.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Signor', 'repository_reference': 'https://signor.uniroma2.it/', 'dataset_context_from_paper': 'The SIGNOR database was referenced to identify sites annotated as activating kinase activity for use in kinase activity prediction. It also contributed to the compilation of kinase-substrate interactions for phosphosite set enrichment analysis.', 'citation_type': 'Secondary', 'dataset_id': 'Signor'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: Signor\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID Signor is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://signor.uniroma2.it/, identifier: Signor, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://signor.uniroma2.it/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://signor.uniroma2.it/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://signor.uniroma2.it/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Signor, signor.uniroma2.it, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'The Human Protein Atlas (HPA)', 'repository_reference': 'https://www.proteinatlas.org/', 'dataset_context_from_paper': 'The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.', 'citation_type': 'Secondary', 'dataset_id': 'The Human Protein Atlas (HPA)'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://www.proteinatlas.org/, identifier: The Human Protein Atlas (HPA), dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.proteinatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(https://www.proteinatlas.org/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: https://www.proteinatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer., www.proteinatlas.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.proteinatlas.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer., www.proteinatlas.org, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br/', 'dataset_context_from_paper': 'The CTdatabase, a knowledge-base containing 269 carefully curated and annotated cancer-testis (CT) antigens, was the source from which CT antigens were downloaded for further analysis in this study.', 'citation_type': 'Secondary', 'dataset_id': 'CTdatabase'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID CTdatabase is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: http://www.cta.lncc.br/, identifier: CTdatabase, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://www.cta.lncc.br/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(http://www.cta.lncc.br/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: http://www.cta.lncc.br/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: CTdatabase, www.cta.lncc.br, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'MSigDB (Molecular Signatures Database)', 'repository_reference': 'Not explicitly linked, but referenced via Liberzon et al., 2015.', 'dataset_context_from_paper': 'Hallmark gene sets were collected from the Molecular Signatures Database (MSigDB) to perform gene set enrichment analyses, specifically for Rb phosphorylation quantification and correlation analysis.', 'citation_type': 'Secondary', 'dataset_id': 'MSigDB (Molecular Signatures Database)'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but referenced via Liberzon et al., 2015.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Not explicitly linked, but referenced via Liberzon et al., 2015.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but referenced via Liberzon et al., 2015., identifier: MSigDB (Molecular Signatures Database), dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Not explicitly linked, but referenced via Liberzon et al., 2015.,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Not explicitly linked, but referenced via Liberzon et al., 2015.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: Not explicitly linked, but referenced via Liberzon et al., 2015.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Not explicitly linked, but referenced via Liberzon et al., 2015., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Not explicitly linked, but referenced via Liberzon et al., 2015.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Not explicitly linked, but referenced via Liberzon et al., 2015., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'ENCODE Project Consortium Data (2011)', 'repository_reference': 'Not explicitly linked, but implied by the citation ENCODE Project Consortium (2011).', 'dataset_context_from_paper': 'Data from the ENCODE Project Consortium was utilized to collect transcriptional targets of E2F transcription factors, which were then used to infer E2F1 activity.', 'citation_type': 'Secondary', 'dataset_id': 'ENCODE Project Consortium Data (2011)'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ENCODE Project Consortium Data (2011)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: ENCODE Project Consortium Data (2011)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but implied by the citation ENCODE Project Consortium (2011)., identifier: ENCODE Project Consortium Data (2011), dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Not explicitly linked, but implied by the citation ENCODE Project Consortium  2011 .,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Not explicitly linked, but implied by the citation ENCODE Project Consortium  2011 .\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: Not explicitly linked, but implied by the citation ENCODE Project Consortium  2011 .\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but implied by the citation ENCODE Project Consortium (2011).\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ENCODE Project Consortium Data (2011)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: ENCODE Project Consortium Data (2011), unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: ENCODE Project Consortium Data (2011)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: ENCODE Project Consortium Data (2011), unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Synapse Platform metabolic gene sets', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2623706/wiki/', 'dataset_context_from_paper': 'Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.', 'citation_type': 'Secondary', 'dataset_id': 'Synapse Platform metabolic gene sets'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn2623706/wiki/, identifier: Synapse Platform metabolic gene sets, dataset_page: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn2623706/wiki/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes., synapse.org, https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn2623706/wiki/, resolved_repo synapse.org, dataset_id Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.synapse.org/[#!]*Synapse:, resolved_dataset_page: https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.synapse.org/[#!]*Synapse: of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes., synapse.org, https://www.synapse.org/#!Synapse:syn2623706/wiki/\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'RefSeq protein database', 'repository_reference': \"Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\", 'dataset_context_from_paper': 'The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.', 'citation_type': 'Secondary', 'dataset_id': 'RefSeq protein database'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods., identifier: RefSeq protein database, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Not explicitly linked, but referenced as 'RefSeq protein database  03/29/2017, 45929 sequences + 245 contaminant sequences ' in Methods.,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Not explicitly linked, but referenced as 'RefSeq protein database  03/29/2017, 45929 sequences + 245 contaminant sequences ' in Methods.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: Not explicitly linked, but referenced as 'RefSeq protein database  03/29/2017, 45929 sequences + 245 contaminant sequences ' in Methods.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'NCI-60 colorectal cell lines', 'repository_reference': 'Not explicitly linked, but referenced via Gholami et al., 2013.', 'dataset_context_from_paper': \"Proteome data from 60 NCI-60 colorectal cell lines were used for Pearson's correlation coefficient analysis of average gene-wise protein expression, enabling comparison with the prospective colon tumor samples.\", 'citation_type': 'Secondary', 'dataset_id': 'NCI-60 colorectal cell lines'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but referenced via Gholami et al., 2013.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Not explicitly linked, but referenced via Gholami et al., 2013.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but referenced via Gholami et al., 2013., identifier: NCI-60 colorectal cell lines, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Not explicitly linked, but referenced via Gholami et al., 2013.,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Not explicitly linked, but referenced via Gholami et al., 2013.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 816 - ERROR - Error extracting domain from URL: Not explicitly linked, but referenced via Gholami et al., 2013.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Not explicitly linked, but referenced via Gholami et al., 2013., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Not explicitly linked, but referenced via Gholami et al., 2013.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: unknown_publisher\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Not explicitly linked, but referenced via Gholami et al., 2013., unknown_publisher, None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 476 - INFO - Processing dataset: {'dataset_identifier': 'Colorectal cell lines (Wang et al., 2017b)', 'repository_reference': 'Not explicitly linked, but referenced via Wang et al., 2017b.', 'dataset_context_from_paper': \"RNA-seq and proteomic data from 44 colorectal cell lines were utilized for Pearson's correlation coefficient analysis of average gene-wise RNA and protein expression, allowing for comparison with the prospective colon tumor samples.\", 'citation_type': 'Secondary', 'dataset_id': 'Colorectal cell lines (Wang et al., 2017b)'}\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 491 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Colorectal cell lines (Wang et al., 2017b)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: Colorectal cell lines (Wang et al., 2017b)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Not explicitly linked, but referenced via Wang et al., 2017b., identifier: Colorectal cell lines (Wang et al., 2017b), dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 981 - WARNING - Repository contains a comma: Not explicitly linked, but referenced via Wang et al., 2017b.. Same data may be in multiple repos.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Not explicitly linked, but referenced via Wang et al., 2017b.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Colorectal cell lines (Wang et al., 2017b)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: Colorectal cell lines (Wang et al., 2017b), [], None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: Colorectal cell lines (Wang et al., 2017b)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: []\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 513 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: Colorectal cell lines (Wang et al., 2017b), [], None\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 524 - INFO - Final result: [{'dataset_identifier': 'PRJNA514017 / SRP178677', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Raw genomics data from this study, including Whole-Exome Sequencing (WXS), RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) data, along with Illumina SNP array data, were collected from 110 prospectively collected colon cancer patients (tumor specimens, matched normal adjacent tissues, and blood samples). This data was used for somatic mutation detection (SNVs, INDELs, MS INDELs), somatic copy number alteration (SCNA) analysis, HLA genotyping, subtype classification, and neoantigen identification.'}, {'dataset_identifier': 'S045', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Raw and low-level processed proteomics data generated in this study, comprising label-free shotgun proteomic analyses, isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and normal adjacent tissues from 110 colon cancer patients (specifically 96 tumor and NAT pairs for TMT). Additionally, selected reaction monitoring (SRM) data for validation of specific proteins (CD8A, SLC2A3, PKM1, PKM2) in representative tumor samples is included. This data was instrumental in identifying colon cancer-associated proteins and phosphosites, investigating proteomic consequences of somatic mutations, predicting kinase activity, identifying tumor antigens (neoantigens, CT antigens), and enabling subtype classification.'}, {'dataset_identifier': 'cptac-colon', 'data_repository': 'linkedomics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'All final processed data matrices derived from the proteogenomic analysis of the colon cancer cohort (CPTAC Colon Cancer Study) are available through the LinkedOmics portal. This resource provides computational tools for further exploration and is intended for broad usage by other investigators for new biological discoveries and therapeutic hypothesis generation.'}, {'dataset_identifier': 'n/a', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'This table summarizes the clinical and pathological characteristics of the 110 colon cancer tumors and outlines the specific numbers of samples analyzed by each omics platform within the study cohort. This clinical data was collected in strict accordance with the CPTAC-2 colon procurement protocol to ensure suitability for analysis.'}, {'dataset_identifier': 'n/a', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'PCR-based microsatellite instability (MSI) testing results were obtained for 85 samples from the colon cancer cohort. This data was used to validate the WXS-based MSI assignments, confirming agreement between the two methods.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.'}, {'dataset_identifier': 'dbSNP150', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"The dbSNP150 database was utilized for variant annotation as part of the process to identify protein-altering somatic mutations in the study's WXS data.\"}, {'dataset_identifier': 'COSMIC83', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The COSMIC83 database was used for variant annotation during the identification of protein-altering somatic mutations from the whole-exome sequencing data.'}, {'dataset_identifier': 'Clinvar (version 20170905)', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.'}, {'dataset_identifier': 'dbNSFP33a', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The dbNSFP33a database was used as a source for functional predictions during the annotation of human nonsynonymous SNPs and other protein-altering somatic mutations identified in the WXS data.'}, {'dataset_identifier': 'PhosphoSitePlus', 'data_repository': 'www.phosphosite.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'PhosphoSitePlus was used to compile kinase-substrate interactions, which were then utilized in the phosphosite set enrichment analysis for kinase activity prediction. It also provided general phosphosite information.'}, {'dataset_identifier': 'Signor', 'data_repository': 'signor.uniroma2.it', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The SIGNOR database was referenced to identify sites annotated as activating kinase activity for use in kinase activity prediction. It also contributed to the compilation of kinase-substrate interactions for phosphosite set enrichment analysis.'}, {'dataset_identifier': 'The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.'}, {'dataset_identifier': 'CTdatabase', 'data_repository': 'www.cta.lncc.br', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The CTdatabase, a knowledge-base containing 269 carefully curated and annotated cancer-testis (CT) antigens, was the source from which CT antigens were downloaded for further analysis in this study.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via Liberzon et al., 2015.', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Hallmark gene sets were collected from the Molecular Signatures Database (MSigDB) to perform gene set enrichment analyses, specifically for Rb phosphorylation quantification and correlation analysis.'}, {'dataset_identifier': 'ENCODE Project Consortium Data (2011)', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Data from the ENCODE Project Consortium was utilized to collect transcriptional targets of E2F transcription factors, which were then used to infer E2F1 activity.'}, {'dataset_identifier': 'Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn2623706/wiki/', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.'}, {'dataset_identifier': \"Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\", 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via Gholami et al., 2013.', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"Proteome data from 60 NCI-60 colorectal cell lines were used for Pearson's correlation coefficient analysis of average gene-wise protein expression, enabling comparison with the prospective colon tumor samples.\"}, {'dataset_identifier': 'Colorectal cell lines (Wang et al., 2017b)', 'data_repository': [], 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"RNA-seq and proteomic data from 44 colorectal cell lines were utilized for Pearson's correlation coefficient analysis of average gene-wise RNA and protein expression, allowing for comparison with the prospective colon tumor samples.\"}]\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 321 - INFO - Augmented dataset links: [{'dataset_identifier': 'PRJNA514017 / SRP178677', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Raw genomics data from this study, including Whole-Exome Sequencing (WXS), RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) data, along with Illumina SNP array data, were collected from 110 prospectively collected colon cancer patients (tumor specimens, matched normal adjacent tissues, and blood samples). This data was used for somatic mutation detection (SNVs, INDELs, MS INDELs), somatic copy number alteration (SCNA) analysis, HLA genotyping, subtype classification, and neoantigen identification.'}, {'dataset_identifier': 'S045', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Raw and low-level processed proteomics data generated in this study, comprising label-free shotgun proteomic analyses, isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and normal adjacent tissues from 110 colon cancer patients (specifically 96 tumor and NAT pairs for TMT). Additionally, selected reaction monitoring (SRM) data for validation of specific proteins (CD8A, SLC2A3, PKM1, PKM2) in representative tumor samples is included. This data was instrumental in identifying colon cancer-associated proteins and phosphosites, investigating proteomic consequences of somatic mutations, predicting kinase activity, identifying tumor antigens (neoantigens, CT antigens), and enabling subtype classification.'}, {'dataset_identifier': 'cptac-colon', 'data_repository': 'linkedomics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'All final processed data matrices derived from the proteogenomic analysis of the colon cancer cohort (CPTAC Colon Cancer Study) are available through the LinkedOmics portal. This resource provides computational tools for further exploration and is intended for broad usage by other investigators for new biological discoveries and therapeutic hypothesis generation.'}, {'dataset_identifier': 'n/a', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'This table summarizes the clinical and pathological characteristics of the 110 colon cancer tumors and outlines the specific numbers of samples analyzed by each omics platform within the study cohort. This clinical data was collected in strict accordance with the CPTAC-2 colon procurement protocol to ensure suitability for analysis.'}, {'dataset_identifier': 'n/a', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'PCR-based microsatellite instability (MSI) testing results were obtained for 85 samples from the colon cancer cohort. This data was used to validate the WXS-based MSI assignments, confirming agreement between the two methods.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via various publications such as Cancer Genome Atlas Network, 2012; Zhang et al., 2014; Mertins et al., 2016; Zhang et al., 2016 and accessed via tools like TCGAbiolinks (Colaprico et al., 2016).', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The TCGA cohort, including data from colorectal, breast, and ovarian cancer studies, was used for several comparative analyses. Specifically, average mRNA and label-free protein profiles were correlated with TCGA data. Findings such as significantly mutated genes and arm-level SCNAs were compared to TCGA reports for validation. Additionally, TP53-S315 phosphorylation data from CPTAC breast (Mertins et al., 2016) and ovarian cancer (Zhang et al., 2016) studies, part of the broader TCGA ecosystem, were reviewed to strengthen associations. TCGA CRC cohort was also used to confirm associations in subtype analysis.'}, {'dataset_identifier': 'dbSNP150', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"The dbSNP150 database was utilized for variant annotation as part of the process to identify protein-altering somatic mutations in the study's WXS data.\"}, {'dataset_identifier': 'COSMIC83', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The COSMIC83 database was used for variant annotation during the identification of protein-altering somatic mutations from the whole-exome sequencing data.'}, {'dataset_identifier': 'Clinvar (version 20170905)', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The ClinVar database (version 20170905) was employed for variant annotation in the process of identifying protein-altering somatic mutations from the generated WXS data.'}, {'dataset_identifier': 'dbNSFP33a', 'data_repository': 'annovar.openbioinformatics.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The dbNSFP33a database was used as a source for functional predictions during the annotation of human nonsynonymous SNPs and other protein-altering somatic mutations identified in the WXS data.'}, {'dataset_identifier': 'PhosphoSitePlus', 'data_repository': 'www.phosphosite.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'PhosphoSitePlus was used to compile kinase-substrate interactions, which were then utilized in the phosphosite set enrichment analysis for kinase activity prediction. It also provided general phosphosite information.'}, {'dataset_identifier': 'Signor', 'data_repository': 'signor.uniroma2.it', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The SIGNOR database was referenced to identify sites annotated as activating kinase activity for use in kinase activity prediction. It also contributed to the compilation of kinase-substrate interactions for phosphosite set enrichment analysis.'}, {'dataset_identifier': 'The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The Human Protein Atlas (HPA) was consulted to retrieve immunohistochemistry (IHC) staining data for the 30 cancer-associated proteins identified in colorectal tumors, allowing for external validation of protein expression patterns. Additionally, HPA data was used to verify IGF2BP3 protein expression patterns in normal human tissues and colon cancer.'}, {'dataset_identifier': 'CTdatabase', 'data_repository': 'www.cta.lncc.br', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The CTdatabase, a knowledge-base containing 269 carefully curated and annotated cancer-testis (CT) antigens, was the source from which CT antigens were downloaded for further analysis in this study.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via Liberzon et al., 2015.', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Hallmark gene sets were collected from the Molecular Signatures Database (MSigDB) to perform gene set enrichment analyses, specifically for Rb phosphorylation quantification and correlation analysis.'}, {'dataset_identifier': 'ENCODE Project Consortium Data (2011)', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Data from the ENCODE Project Consortium was utilized to collect transcriptional targets of E2F transcription factors, which were then used to infer E2F1 activity.'}, {'dataset_identifier': 'Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn2623706/wiki/', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Pre-defined metabolic gene sets from the Synapse platform, attributed to Guinney et al., 2015, were downloaded and used for gene set variation analysis (ssGSEA) to infer RNA-based and protein-based pathway activities, specifically for studying metabolism differences between CMS3 and other CMS subtypes.'}, {'dataset_identifier': \"Not explicitly linked, but referenced as 'RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences)' in Methods.\", 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The RefSeq protein database (version 03/29/2017) was used as the primary database for matching raw MS data with the MS-GF+ search engine during proteomics quantification.'}, {'dataset_identifier': 'Not explicitly linked, but referenced via Gholami et al., 2013.', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"Proteome data from 60 NCI-60 colorectal cell lines were used for Pearson's correlation coefficient analysis of average gene-wise protein expression, enabling comparison with the prospective colon tumor samples.\"}, {'dataset_identifier': 'Colorectal cell lines (Wang et al., 2017b)', 'data_repository': [], 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': \"RNA-seq and proteomic data from 44 colorectal cell lines were utilized for Pearson's correlation coefficient analysis of average gene-wise RNA and protein expression, allowing for comparison with the prospective colon tumor samples.\"}]\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 351 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'dataset_context_from_paper'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mpdf_parser.py - line 534 - INFO - Extracting publication title from PDF\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 568 - INFO - PDF parsing completed. Elements collected: 20\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 574 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 587 - INFO - PDF element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 723 - INFO - \n",
      "Progress: 1/1 (100.00%) | Elapsed: 00:00:02 | ETA: 00:00:00\n",
      "\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "parsed_data = data_gatherer.process_articles(\n",
    "    articles[:1], \n",
    "    response_format=dataset_response_schema_with_use_description,\n",
    "    prompt_name=\"GPT_FDR_FewShot_Descr\",\n",
    "    grobid_for_pdf=False,\n",
    "    write_htmls_xmls=True,\n",
    "    article_file_dir='scripts/tmp/raw_files/'\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "961e2338",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.DataFrame()\n",
    "\n",
    "for k, v in parsed_data.items():\n",
    "    df = pd.concat([df, v], ignore_index=True)\n",
    "\n",
    "df.to_csv('scripts/output/bdf_parsed_data_gemini_flash.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "da823880",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "2025_arm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
