{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "ad1baf88",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "from data_gatherer.llm.response_schema import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a0319a9e",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 329 - INFO - Setting up data fetcher...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 91 - DEBUG - DataFetcher (WebScraper) initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 538 - DEBUG - WebScraper initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 359 - INFO - Data fetcher setup completed.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 1036 - DEBUG - DataCompletenessChecker initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 108 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.5-flash\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "data_gatherer = DataGatherer(llm_name=\"gemini-2.5-flash\", log_level=\"DEBUG\", process_entire_document=True, load_from_cache=False, save_to_cache=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "d47ab036",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['scripts/exp_input/bdf_paper_pdfs/Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization of Endometrial Carcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability.pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic and metabolomic characterization of human glioblastoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic characterization of pancreatic ductal adenocarcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/A proteogenomic portrait of lung squamous cell carcinoma .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy .pdf',\n",
       " 'scripts/exp_input/bdf_paper_pdfs/Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma .pdf']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles = []\n",
    "\n",
    "for article in os.listdir('scripts/exp_input/bdf_paper_pdfs/'):\n",
    "    articles.append(os.path.join('scripts/exp_input/bdf_paper_pdfs/', article))\n",
    "\n",
    "articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "06d39d47",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7331093/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7233456/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8077737/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6768830/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8044053/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7946781/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7373300/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7289043/',\n",
       " 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8654574/']"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles = [\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7331093/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7233456/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8077737/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6768830/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8044053/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7946781/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7373300/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7289043/\",\n",
    "    \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8654574/\"\n",
    "]\n",
    "\n",
    "articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "665d9186",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 699 - DEBUG - Starting to process URL list...\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 706 - INFO - #0 function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 425 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 158 - DEBUG - URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 261 - DEBUG - update_DataFetcher_settings for URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 363 - DEBUG - Checking if URL is a PDF: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 348 - DEBUG - Checking PMC with pattern https://www.ncbi.nlm.nih.gov/pmc/articles/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 348 - DEBUG - Checking PMC with pattern https://pmc.ncbi.nlm.nih.gov/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 351 - DEBUG - URL detected as PMC.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 278 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 286 - INFO - Creating EntrezFetcher with backup support\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 91 - DEBUG - DataFetcher (EntrezFetcher) initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 832 - INFO - Raw_data_format: XML\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 841 - DEBUG - EntrezFetcher initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 433 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 444 - DEBUG - Fetching Raw content...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 862 - INFO - Local data not found, fetching live from API for PMC8475722\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 873 - INFO - API request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8475722&retmode=xml&api_key=426d4afdbf84343737c3138818880bc88208\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 882 - DEBUG - Successfully fetched data for PMC8475722\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 55 - DEBUG - Checking XML completeness for https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 452 - DEBUG - type of raw_data: <class 'lxml.etree._Element'>, raw_data: <Element pmc-articleset at 0x1555f1600>\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 454 - INFO - ----Checking for data_availability_sections section in raw data.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 456 - DEBUG - Section patterns: [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 458 - DEBUG - Namespaces: {'xlink': 'http://www.w3.org/1999/xlink'}\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 461 - DEBUG - Checking pattern: .//sec[@sec-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 461 - DEBUG - Checking pattern: .//notes[@notes-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 461 - DEBUG - Checking pattern: .//notes[@notes-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h2[contains(text(), 'Data Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h2[contains(text(), 'Data and Code Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h3[contains(text(), 'Data Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h3[contains(text(), 'Data and Code Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h4[contains(text(), 'Data Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h4[contains(text(), 'Data and Code Availability')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: following-sibling::p[1]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 473 - DEBUG - Checking XPath: //section[h2[contains(text(), 'Footnotes')]]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 465 - INFO - Fallback from EntrezFetcher to HTML Fetchers.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 425 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 158 - DEBUG - URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 261 - DEBUG - update_DataFetcher_settings for URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 363 - DEBUG - Checking if URL is a PDF: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 278 - INFO - API detected: None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 291 - INFO - Using HttpGetRequest with backup support for URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 91 - DEBUG - DataFetcher (HttpGetRequest) initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 399 - DEBUG - HttpGetRequest initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 433 - INFO - Type of data_fetcher HttpGetRequest\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 444 - DEBUG - Fetching Raw content...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 420 - INFO - Local data not found, fetching live from https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 425 - INFO - HTTP GET attempt 1 of 3\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 483 - INFO - HTML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 522 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 487 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/ is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 491 - INFO - Raw Data is HTML.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 502 - WARNING - Unsupported raw data format: HTML.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 509 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 535 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['BIGD', 'European Genome-Phenome Archive', 'MassIVE', 'PRIDE', 'iProX', 'ProteomeCentral', 'ProteomeXchange', 'Proteomic Data Commons', 'Genomic Data Commons', 'The Cancer Imaging Archive', 'Imaging Data Commons', 'JPOST', 'Gene Expression Omnibus', 'database of Genotypes and Phenotypes', 'Sequence Read Archive', 'ArrayExpress', 'Nucleotide', 'Mendeley', 'Synapse', 'doi.org', 'Protein Data Bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 857 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'gene expression omnibus': 'geo', 'database of genotypes and phenotypes': 'dbgap', 'sequence read archive': 'sra', 'arrayexpress': 'arrayexpress', 'nucleotide': 'nuccore', 'mendeley': 'data.mendeley.com', 'synapse': 'synapse.org', 'doi.org': 'doi.org', 'protein data bank': 'pdb'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.5-flash\u001b[0m\n",
      "\u001b[97mllm_client.py - line 38 - DEBUG - _initialize_client called with model=gemini-2.5-flash, use_portkey=True\u001b[0m\n",
      "\u001b[97mllm_client.py - line 41 - DEBUG - Initializing Portkey client for Gemini model: gemini-2.5-flash\u001b[0m\n",
      "\u001b[97mllm_client.py - line 50 - DEBUG - Portkey client initialized: <portkey_ai.api_resources.client.Portkey object at 0x15580c740>\u001b[0m\n",
      "\u001b[97mllm_client.py - line 73 - DEBUG - Client initialization complete. self.client: <portkey_ai.api_resources.client.Portkey object at 0x15580c740>, self.portkey: <portkey_ai.api_resources.client.Portkey object at 0x15580c740>\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 353 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 451 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 459 - DEBUG - Found 641 anchors with data-ga-action='click_feat_suppl'.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1016/j.cell.2021.07.016, pmc-layout__citation font-secondary font-xs, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1016/j.cell.2021.07.016, pmc-layout__citation font-secondary font-xs, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1016/j.cell.2021.07.016, ra xbox p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1016/j.cell.2021.07.016, ra xbox p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clinicaltrials.gov/ct2/show/NCT04350463, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clinicaltrials.gov/ct2/show/NCT04350463, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S056, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S056, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://portal.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://portal.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.cancerimagingarchive.net/datascope/cptac/home/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.cancerimagingarchive.net/datascope/cptac/home/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.ncbi.nlm.nih.gov/nuccore/Ab182422, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.ncbi.nlm.nih.gov/nuccore/Ab182422, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1101/2020.05.22.110247, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1101/2020.05.22.110247, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.phosphosite.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.phosphosite.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.broadinstitute.org/connectivity-map-cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.broadinstitute.org/connectivity-map-cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.proteinatlas.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.proteinatlas.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.cta.lncc.br, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.cta.lncc.br, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/data, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/data, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ding-lab/cptac_methylation, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ding-lab/cptac_methylation, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ding-lab/CPTAC_miRNA, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ding-lab/CPTAC_miRNA, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/Ensembl/ensembl-vep/tags, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/Ensembl/ensembl-vep/tags, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://sentieon.com, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://sentieon.com, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/brentp/vcfanno, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/brentp/vcfanno, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen), n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen), n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://gmt.genome.wustl.edu/packages/pindel/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://gmt.genome.wustl.edu/packages/pindel/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://software.broadinstitute.org/cancer/cga/msp, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://software.broadinstitute.org/cancer/cga/msp, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/mcieslik-mctp/crisp-build, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/mcieslik-mctp/crisp-build, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://proteomics.broadinstitute.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://proteomics.broadinstitute.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/sva.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/sva.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/gPCA/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/gPCA/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/WangLab-MSSM/iProFun, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/WangLab-MSSM/iProFun, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioinformatics.mdanderson.org/public-software/estimate/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioinformatics.mdanderson.org/public-software/estimate/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.webgestalt.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.webgestalt.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/GSVA.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/GSVA.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/WangLab-MSSM/TSNet, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/WangLab-MSSM/TSNet, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://xcell.ucsf.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://xcell.ucsf.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/MSGFPlus/msgfplus, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/MSGFPlus/msgfplus, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.cbs.dtu.dk/services/NetMHCpan/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.cbs.dtu.dk/services/NetMHCpan/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/FRED-2/OptiType, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/FRED-2/OptiType, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/wenbostar/PDV, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/wenbostar/PDV, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://pepquery.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://pepquery.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/PANOPLY, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/PANOPLY, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/limma.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/limma.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/FactoMineR/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/FactoMineR/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/mclust/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/mclust/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://biit.cs.ut.ee/gprofiler/gost, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://biit.cs.ut.ee/gprofiler/gost, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cytoscape.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cytoscape.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pecan.stjude.doud/proteinpaint/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pecan.stjude.doud/proteinpaint/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://CRAN.R-project.org/package=ordinal, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://CRAN.R-project.org/package=ordinal, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://docs.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://docs.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.broadinstitute.org/proteomics, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.broadinstitute.org/proteomics, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://genome.ucsc.edu/cgi-bin/hgTables, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://genome.ucsc.edu/cgi-bin/hgTables, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.thermofisher.com/order/catalog/product/90406, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.thermofisher.com/order/catalog/product/90406, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://rdrr.io/bioc/limma/man/removeBatchEffect.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://rdrr.io/bioc/limma/man/removeBatchEffect.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://app.terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://app.terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/GEO-guide, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/GEO-guide, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://hub.docker.com/u/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://hub.docker.com/u/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cancer.sanger.ac.uk/census, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cancer.sanger.ac.uk/census, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.proteinatlas.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.proteinatlas.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://cibersort.stanford.edu, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://cibersort.stanford.edu, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.ncbi.nlm.nih.gov/protein/NP_000068.1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.ncbi.nlm.nih.gov/protein/NP_000068.1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.ncbi.nlm.nih.gov/protein/NP_478102.2, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.ncbi.nlm.nih.gov/protein/NP_478102.2, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.pepquery.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.pepquery.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.zhang-lab.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.zhang-lab.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/cmap, caption p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/cmap, caption p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-1.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-2.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-3.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-4.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-6.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-7.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-8.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-9.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-10.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-11.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-12.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-13.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-14.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-Figure_S5.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://proteomics.cancer.gov/programs/cptac, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://proteomics.cancer.gov/programs/cptac, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.linkedomics.org/, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.linkedomics.org/, fn p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clinicaltrials.gov/ct2/show/NCT02154490, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clinicaltrials.gov/ct2/show/NCT02154490, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 530 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/cmap, caption p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/cmap, caption p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-1.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-2.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-3.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-4.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-6.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-7.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-8.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-9.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-10.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-11.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-12.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-13.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-14.xlsx from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf, media p, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 131 - DEBUG - Extracted file name: NIHMS1730863-supplement-Figure_S5.pdf from href: /articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pdc.cancer.gov/pdc/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S056, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S056, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://portal.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://portal.gdc.cancer.gov/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.cancerimagingarchive.net/datascope/cptac/home/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.cancerimagingarchive.net/datascope/cptac/home/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.ncbi.nlm.nih.gov/nuccore/Ab182422, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.ncbi.nlm.nih.gov/nuccore/Ab182422, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.6084/m9.figshare.13681534.v1, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1101/2020.05.22.110247, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1101/2020.05.22.110247, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/download/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.phosphosite.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.phosphosite.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.broadinstitute.org/connectivity-map-cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.broadinstitute.org/connectivity-map-cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://www.proteinatlas.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://www.proteinatlas.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.cta.lncc.br, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.cta.lncc.br, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://depmap.org/portal/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://depmap.org/portal/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/data, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/data, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S054, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cptac-data-portal.georgetown.edu/study-summary/S063, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ding-lab/cptac_methylation, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ding-lab/cptac_methylation, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ding-lab/CPTAC_miRNA, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ding-lab/CPTAC_miRNA, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/Ensembl/ensembl-vep/tags, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/Ensembl/ensembl-vep/tags, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://sentieon.com, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://sentieon.com, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/brentp/vcfanno, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/brentp/vcfanno, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/suhrig/arriba/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen), n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen), n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://gmt.genome.wustl.edu/packages/pindel/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://gmt.genome.wustl.edu/packages/pindel/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://software.broadinstitute.org/cancer/cga/msp, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://software.broadinstitute.org/cancer/cga/msp, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/mctp/cnvex, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/mcieslik-mctp/crisp-build, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/mcieslik-mctp/crisp-build, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://proteomics.broadinstitute.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://proteomics.broadinstitute.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/sva.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/sva.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/gPCA/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/gPCA/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/WangLab-MSSM/iProFun, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/WangLab-MSSM/iProFun, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioinformatics.mdanderson.org/public-software/estimate/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioinformatics.mdanderson.org/public-software/estimate/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.webgestalt.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.webgestalt.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/GSVA.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/GSVA.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/WangLab-MSSM/TSNet, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/WangLab-MSSM/TSNet, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://xcell.ucsf.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://xcell.ucsf.edu/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/MSGFPlus/msgfplus, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/MSGFPlus/msgfplus, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/neoflow, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://www.cbs.dtu.dk/services/NetMHCpan/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://www.cbs.dtu.dk/services/NetMHCpan/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/FRED-2/OptiType, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/FRED-2/OptiType, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/bzhanglab/customprodbj, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/wenbostar/PDV, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/wenbostar/PDV, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://pepquery.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://pepquery.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/ssGSEA2.0, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://terra.bio/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/broadinstitute/PANOPLY, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/broadinstitute/PANOPLY, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://clue.io/cmap, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://bioconductor.org/packages/release/bioc/html/limma.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://bioconductor.org/packages/release/bioc/html/limma.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/FactoMineR/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/FactoMineR/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cran.r-project.org/web/packages/mclust/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cran.r-project.org/web/packages/mclust/index.html, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://biit.cs.ut.ee/gprofiler/gost, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://biit.cs.ut.ee/gprofiler/gost, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://cytoscape.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://cytoscape.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://pecan.stjude.doud/proteinpaint/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://pecan.stjude.doud/proteinpaint/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 122 - DEBUG - Function_call: reconstruct_download_link(https://CRAN.R-project.org/package=ordinal, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 125 - DEBUG - Inputs to reconstruct_download_link: https://CRAN.R-project.org/package=ordinal, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/, 8475722\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 553 - INFO - Extracted 106 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 563 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S11: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S20'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S20: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S20'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S20: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S21'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S21: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S22'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S22: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S22'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S22: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S22'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S22: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S22'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S22: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#T1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 159) for T1: Software and code used in this study are referenced in their corresponding STAR Method [href=#S14] sections and also the Key Resource Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S24: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S64'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S64: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S64'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S64: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S67'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S67: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S67'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S67: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S69'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S69: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S69'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S69: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S71'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S71: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 2 <a> elements with href='#S75'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 291) for S75: The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in STAR methods [href=#S14] (“ Quality control and Batch effect assessment [href=this file] ”).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S77'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S77: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S84'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S84: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S84'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S84: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S85'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S85: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S86'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S86: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 1 <a> elements with href='#S90'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 143) for S90: Differential expression analysis for all three cohorts was performed as described in the Differential marker analysis [href=this file] method.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 1 <a> elements with href='#S98'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 235) for S98: The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method Estimation of Tumor Purity, Stromal and Immune Scores [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S103'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S103: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S106'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S106: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S107'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S107: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S107'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S107: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S113'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S113: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#S113'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for S113: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 18 <a> elements with href='#SD1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2838) for SD1: We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( Figure 1A [href=#F1] , S1A [href=this file] , Table S1 [href=#SD7] – 3 [href=#SD9] ). Proteomics data were generated using TMT11-based multiplexing linked by a common reference ( Figure S1A [href=this file] ) and demonstrated high technical quality ( Figure S1B [href=this file] – C [href=this file] ). Most self-reported never-smokers displayed high genomic smoking scores ( Figure S1D [href=this file] ), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported ( Gillette et al., 2020 [href=#R51] ). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs ( Figure S1E [href=this file] ).\n",
      "See also Figure S1 [href=this file] and Table S1 [href=#SD7] – 3 [href=#SD9]\n",
      "Six CNAs ( , , , , , ) ( Table S4 [href=this file0] ) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher’s exact p<0.01) with clinical metadata. deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, ( Figure S1F [href=this file] (upper panel)), consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC ( Timperi et al., 2017 [href=#R175] ). amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score ( Figure S1F [href=this file] (lower panel)).\n",
      "LSCC tumor DNA showed overall hypermethylation relative to NATs ( Figure S1G [href=this file] ), with tumors separating into CIMP (CpG island methylator phenotype) clusters ( Figure S1H [href=this file] ). Of 90 genes that showed such significant (FDR<0.1) -effects in tumors ( Table S4 [href=this file0] ), 20 were also differentially expressed (FDR<0.01) between tumors and NATs ( Figure 1G [href=#F1] ).\n",
      "For each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as common denominator ( Figure S1C [href=this file] ).\n",
      "Table S1: Metadata Associated with LSCC Patients and Tumors, Related to Figures 1 [href=#F1] – 7 [href=#F7] , Figures S1 [href=this file] – 7 [href=#SD6]\n",
      "Table S2: Genomic and methylation datasets associated with this study, Related to Figures 1 [href=#F1] – 7 [href=#F7] , Figures S1 [href=this file] – 7 [href=#SD6]\n",
      "Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to Figures 1 [href=#F1] – 7 [href=#F7] and Figures S1 [href=this file] – 7 [href=#SD6]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 23 <a> elements with href='#SD2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 4019) for SD2: The five resulting multi-omic subtypes ( Figure 2A [href=#F2] – B [href=#F2] , Figure S2A [href=this file] ) were named based on their predominant pathway associations and similarities to previously defined RNA clusters ( Wilkerson et al., 2010 [href=#R196] ).\n",
      "See also Figure S2 [href=this file] and Table S1 [href=#SD7] – 4 [href=#SD10]\n",
      "We compared B-I ( Figure S2B [href=this file] ) and EMT-E ( Figure S2C [href=this file] ) to other subtypes to identify distinguishing features. Transcription activator PBX3, which is upregulated only at the protein level ( Figure S2B [href=this file] ), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, ( Lamprecht et al., 2018 [href=#R94] ) including in laryngeal squamous cell carcinoma ( Wu et al., 2020 [href=#R198] ). Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation ( Figure S2D [href=this file] ), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors ( Figure S2E [href=this file] ) and with the xCell-based deconvolution signature ( Figure S2F [href=this file] ).\n",
      "In addition to a strong EMT signature ( Figure 2B [href=#F2] ), the EMT-E subtype showed upregulated VCAN ( Figure S2C [href=this file] ), a tumor-promoting target of SNAIL ( Zhang et al., 2019a [href=#R207] ) expressed in cancer-associated fibroblasts (CAFs) ( Chida et al., 2016 [href=#R24] ), and FHL3, a protein that stabilizes EMT-associated transcription factors ( Li et al., 2020 [href=#R104] ) ( Figure S2C [href=this file] ). Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker α-smooth muscle actin (α-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT ( Figure S2G [href=this file] ), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either CK or α-SMA alone. TGFβ is a well-established and potent inducer of EMT ( Su et al., 2020 [href=#R166] ), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFβ inhibitors ( Figure S2H [href=this file] , Table S4H [href=#SD10] ).\n",
      "Integrating our global proteome data with a prior LSCC dataset ( Stewart et al., 2019 [href=#R165] ) resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study ( Figure S2I [href=this file] – J [href=this file] ).\n",
      "As summarized in Figure S2K [href=this file] , we found phosphosite-based evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR ( Tam et al., 2013 [href=#R169] ), both targetable ( Roskoski 2018 [href=#R150] ; Debebe and Rathmell 2015 [href=#R25] ) and recently implicated in the progression from a hybrid E/M to a highly mesenchymal state ( Kröger et al., 2019 [href=#R87] ).\n",
      "Our NMF-based pipeline applied to the integrated proteomics dataset ( STAR Methods [href=#S14] ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( Figure S2A [href=this file] ).\n",
      "Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in Figure S2I [href=this file] . At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( Figure S2I [href=this file] ). In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( Figure S2I [href=this file] ), known drivers of EMT ( Cannito et al., 2008 [href=#R22] ) and of Wnt signaling ( Hong et al., 2017 [href=#R62] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 10 <a> elements with href='#SD3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2137) for SD3: Mutations were associated with higher tumor mutational burden ( Shen et al., 2018 [href=#R156] ) ( Figure S3A [href=this file] ) and led to protein downregulation, associated with worse overall survival in NSCLC ( Hung et al., 2020 [href=#R68] ).\n",
      "See also Figure S3 [href=this file] and Table S4 [href=#SD10]\n",
      "Many of those /p16 mutations were unlikely to result in nonsense-mediated decay ( Figure S3B [href=this file] ), accounting for preserved expression in those samples. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC ( Hammerman et al., 2012 [href=#R54] ), there was no association between the amplification of , , , or and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels ( Table S4 [href=#SD10] ). amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping ( Figure S3C [href=this file] ). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in and copy number alterations in and ( Figure S3D [href=this file] ).\n",
      "NFE2L2 phosphorylation was observed in samples with any of these mutations ( Figure S3E [href=this file] ). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes ( Figure 3F [href=#F3] , Figure S3F [href=this file] – G [href=this file] , Table S4 [href=#SD10] ), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits ( Table S4 [href=#SD10] ) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases ( Figure S3H [href=this file] ).\n",
      "Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) ( Figure S3D [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 17 <a> elements with href='#SD4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2831) for SD4: Chromosome 3q, which harbors key squamous differentiation markers and ( Qian and Massion, 2008 [href=#R143] ), showed the most dramatic arm-level amplification in this cohort ( Figure S4A [href=this file] ). LSCC cell lines amplified for and were highly dependent upon them, supporting oncogene addiction ( Figure S4B [href=this file] ).\n",
      "See also Figure S4 [href=this file] and Table S4 [href=#SD10]\n",
      "As previously reported ( Campbell et al., 2018 [href=#R20] ; Lo Iacono et al., 2011 [href=#R113] ), ΔNp63α was the dominant TP63 isoform in this dataset ( Figure S4C [href=this file] ) and was highly correlated with TP63 expression ( Figure S4D [href=this file] ). Its amplification was associated with improved survival in the TCGA dataset ( Figure S4E [href=this file] ). Notably, a substantial number of LSCC cell lines also had low Np63 expression ( Figure S4F [href=this file] ), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 ( Figure S4G [href=this file] ). Survivin promotes cell proliferation and G2M checkpoints ( Wheatley and Altieri, 2019 [href=#R193] ), evident in Np63-low samples ( Figure S4H [href=this file] ).\n",
      "Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC ( Lebanony et al., 2009 [href=#R97] ) ( Figure S4I [href=this file] ), was associated with decreased promoter DNA methylation ( Figure S4J [href=this file] ) and elevation of ΔNp63α, a transcriptional regulator of this miRNA ( Tran et al., 2013 [href=#R176] ). miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples ( Figure 4D [href=#F4] , Figure S4K [href=this file] ). TP73 positively correlated with miR-205 at the transcript ( Figure 4E [href=#F4] ) and still more strongly at the protein level ( Figure 4F [href=#F4] ) and with TP63 protein but not RNA ( Figure S4L [href=this file] ).\n",
      "EZH2 dependency was observed in SOX2 amplified LSCC cell lines ( Figure S4M [href=this file] ).\n",
      "Figure S4B [href=this file] : To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from Figure 4A [href=#F4] with Log2FC>0 & FDR < 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified.\n",
      "Continuous log2 copy number data from DepMap was used in Figure S4M [href=this file] to correlate copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap).\n",
      "Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression ( Figure S4G [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 9 <a> elements with href='#SD5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 1616) for SD5: The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses ( Figures 6A [href=#F6] , C [href=#F6] , S6C [href=this file] , D [href=this file] ), and showed upregulation of immune-related pathways ( Figure 6A [href=#F6] – B [href=#F6] , Figure S6A [href=this file] , Table S6 [href=#SD12] ). Notably, NATs did not display consonant immune signaling ( Figure S6B [href=this file] ). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors ( Figure 6A [href=#F6] – B [href=#F6] , Figure S6A [href=this file] ). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors ( Table S6 [href=#SD12] , Figure S6E [href=this file] )\n",
      "See also Figure S6 [href=this file] and Table S6 [href=#SD12]\n",
      "In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators ( Figures 6F [href=#F6] , S6F [href=this file] , Table S6 [href=#SD12] ). Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells ( Figure S6G [href=this file] ), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas ( von Klot et al., 2017 [href=#R84] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 14 <a> elements with href='#SD6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2032) for SD6: The top 50 of 502 differential proteins are shown in Figure S7A [href=this file] . In addition to 206 upregulated oncogenic proteins (OncoKB, Table S7 [href=#SD13] ), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors ( Figure S7B [href=this file] ), with the I-S samples showing the fewest CT antigens. Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS ( Figure 7B [href=#F7] , Figure S7C [href=this file] ).\n",
      "See also Figure S7 [href=this file] and Table S7 [href=#SD13]\n",
      "CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell type of origin ( Figure S7D [href=this file] ). Immune-related deletions were less prominent in HNSCC and especially LUAD ( Figure S7E [href=this file] ), except for Chromosome 3p deletion, encompassing five chemokine receptors. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs ( Figure S7F [href=this file] ), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD ( Figure S7G [href=this file] ), although LUAD had many activating mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC.\n",
      "Table S1: Metadata Associated with LSCC Patients and Tumors, Related to Figures 1 [href=#F1] – 7 [href=#F7] , Figures S1 [href=#SD1] – 7 [href=this file]\n",
      "Table S2: Genomic and methylation datasets associated with this study, Related to Figures 1 [href=#F1] – 7 [href=#F7] , Figures S1 [href=#SD1] – 7 [href=this file]\n",
      "Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to Figures 1 [href=#F1] – 7 [href=#F7] and Figures S1 [href=#SD1] – 7 [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 12 <a> elements with href='#SD7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 1711) for SD7: We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( Figure 1A [href=#F1] , S1A [href=#SD1] , Table S1 [href=this file] – 3 [href=#SD9] ).\n",
      "See also Figure S1 [href=#SD1] and Table S1 [href=this file] – 3 [href=#SD9]\n",
      "Of 5,523 significant (FDR<0.01) CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 “cancer associated genes” (CAG) ( Figure 1E [href=#F1] , Table S1 [href=this file] ).\n",
      "Significant associations between NMF subtypes and cohort metadata are presented in Table S1 [href=this file] . Two NMF subtypes, “Basal-Inclusive” (B-I) and “Epithelial to Mesenchymal Transition-enriched” (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni correction (Fisher’s p<0.01, Table S1 [href=this file] ).\n",
      "See also Figure S2 [href=#SD2] and Table S1 [href=this file] – 4 [href=#SD10]\n",
      "Sample annotation, processed and normalized data files are provided as Tables S1 [href=this file] – S3 [href=#SD9] .\n",
      "The histologic, genotypic, clinical, geographical and other associated metadata is summarized in Table S1 [href=this file] .\n",
      "The list of genes is provided in Table S1 [href=this file] .\n",
      "For the ubiquitylproteome, a different TMT11 experimental design was used ( Table S1 [href=this file] ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 9 <a> elements with href='#SD8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 669) for SD8: A third subtype characterized by mutations in , and genes and high-level amplification (GISTIC threshold value = 2) of and ( Figure 2A [href=#F2] – B [href=#F2] , Table S2 [href=this file] ) was labeled the “Classical” subtype, consistent with TCGA nomenclature and with a previous publication ( Stewart et al., 2019 [href=#R165] ).\n",
      "Same as Table S2A [href=this file] but CNAs are represented as log2(CN)-1.\n",
      "Same as Table S2A [href=this file] but CNAs are represented as thresholded copy number level calls.\n",
      "Same as Table S2D [href=this file] but with FPKM values median-centered by gene.\n",
      "Same as Table S2F [href=this file] but with TPM values median-centered by gene.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 22 <a> elements with href='#SD9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 1255) for SD9: We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( Figure 1A [href=#F1] , S1A [href=#SD1] , Table S1 [href=#SD7] – 3 [href=this file] ).\n",
      "See also Figure S1 [href=#SD1] and Table S1 [href=#SD7] – 3 [href=this file]\n",
      "Sample annotation, processed and normalized data files are provided as Tables S1 [href=#SD7] – S3 [href=this file] .\n",
      "Same as Table S3A [href=this file] but after application of the filtering strategy described in the STAR methods [href=#S14]\n",
      "Gene-centric global proteome data matrix derived from Table S3B [href=this file] .\n",
      "Gene-centric phosphoproteomic data matrix derived from Table S3E [href=this file] .\n",
      "Gene-centric acetylproteomics data matrix derived from Table S3H [href=this file] .\n",
      "Acetylproteomic data matrix based on Table S3G [href=this file] corrected for global protein expression ( Table S3B [href=this file] ) as described in the STAR methods [href=#S14] .\n",
      "Gene-centric ubiquitylproteomic data matrix derived from Table S3L [href=this file] .\n",
      "Ubiquitylproteomic data matrix based on Table S3L [href=this file] corrected for global protein expression ( Table S3B [href=this file] ) as described in the STAR methods [href=#S14] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 14 <a> elements with href='#SD10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 3895) for SD10: We identified six amplified (including , and ) and 29 deleted (including , , and ) CAGs in significant focal events (Q<0.25) ( Figure 1E [href=#F1] , Table S4 [href=this file] ) ( Mermel et al., 2011 [href=#R121] ).\n",
      "Six CNAs ( , , , , , ) ( Table S4 [href=this file] ) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher’s exact p<0.01) with clinical metadata. deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, ( Figure S1F [href=#SD1] (upper panel)), consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC ( Timperi et al., 2017 [href=#R175] ). amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score ( Figure S1F [href=#SD1] (lower panel)).\n",
      "Of 90 genes that showed such significant (FDR<0.1) -effects in tumors ( Table S4 [href=this file] ), 20 were also differentially expressed (FDR<0.01) between tumors and NATs ( Figure 1G [href=#F1] ).\n",
      "See also Figure S2 [href=#SD2] and Table S1 [href=#SD7] – 4 [href=this file]\n",
      "TGFβ is a well-established and potent inducer of EMT ( Su et al., 2020 [href=#R166] ), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFβ inhibitors ( Figure S2H [href=#SD2] , Table S4H [href=this file] ).\n",
      "CNA data for and was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see Table S4 [href=this file] ). genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown.\n",
      "See also Figure S3 [href=#SD3] and Table S4 [href=this file]\n",
      "We analyzed the effects of genetic and epigenetic alterations of on both its major isoforms, p16INK4a (p16) and p14ARF (p14) ( Table S4 [href=this file] ). Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC ( Hammerman et al., 2012 [href=#R54] ), there was no association between the amplification of , , , or and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels ( Table S4 [href=this file] ). amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping ( Figure S3C [href=#SD3] ).\n",
      "Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes ( Figure 3F [href=#F3] , Figure S3F [href=#SD3] – G [href=#SD3] , Table S4 [href=this file] ), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits ( Table S4 [href=this file] ) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases ( Figure S3H [href=#SD3] ). Using the proteogenomic signature ( Table S4 [href=this file] ) that defined mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype ( Figure 3G [href=#F3] ).\n",
      "See also Figure S4 [href=#SD4] and Table S4 [href=this file]\n",
      "Genes that showed significant upregulation (up) in more than one dataset ( Table S4 [href=this file] ) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA ( https://github.com/broadinstitute/ssGSEA2.0 [href=https://github.com/broadinstitute/ssGSEA2.0] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 6 <a> elements with href='#SD11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 1084) for SD11: Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters ( Figure 5A [href=#F5] , Table S5 [href=this file] ) associated with country of origin and ethnicity (Fisher’s exact p<0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters ( Figure 5A [href=#F5] , Figure S5A [href=#SD14] – B [href=#SD14] , Table S5 [href=this file] ).\n",
      "See also Figure S5 [href=#SD14] and Table S5 [href=this file]\n",
      "To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites ( Table S5 [href=this file] ).\n",
      "Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p<0.05) ( Table S5 [href=this file] ). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos ( Table S5 [href=this file] ), supporting modulation of these metabolic pathways by ubiquitylation and acetylation.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 14 <a> elements with href='#SD12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2181) for SD12: Consensus clustering based on the xCell ( Aran et al., 2017 [href=#R6] ) signatures ( Table S6 [href=this file] ) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters ( Figure 6A [href=#F6] – C [href=#F6] ), as well as an NAT-enriched cluster. The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses ( Figures 6A [href=#F6] , C [href=#F6] , S6C [href=#SD5] , D [href=#SD5] ), and showed upregulation of immune-related pathways ( Figure 6A [href=#F6] – B [href=#F6] , Figure S6A [href=#SD5] , Table S6 [href=this file] ). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors ( Table S6 [href=this file] , Figure S6E [href=#SD5] )\n",
      "See also Figure S6 [href=#SD5] and Table S6 [href=this file]\n",
      "Rho GTPase signaling was upregulated in Hot tumors ( Figure 6A [href=#F6] , Table S6 [href=this file] ). In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators ( Figures 6F [href=#F6] , S6F [href=#SD5] , Table S6 [href=this file] ).\n",
      "Table S6 [href=this file] contains the final score computed by xCell of different cell types. This filtering resulted in 25 cell types annotated in Table S6 [href=this file] . Specifically, signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times ( Wilkerson and Hayes, 2010 [href=#R195] ) ( Figure 6A [href=#F6] , Table S6 [href=this file] ).\n",
      "Besides xCell, we utilized ESTIMATE ( Yoshihara et al., 2013 [href=#R204] ) to infer immune and stromal scores based on gene expression data ( Table S6 [href=this file] ). To infer tumor purity, TSNet was utilized ( Petralia et al., 2018 [href=#R141] ) ( Table S6 [href=this file] ).\n",
      "The pathway analysis results for different data types are contained in Table S6 [href=this file] .\n",
      "Among 319 -pQTLs ( -regulating the cognate protein) identified in this study, only 38 were also -eQTLs ( -regulating cognate RNA)( Table S6 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 4 <a> elements with href='#SD13'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 768) for SD13: Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment ( Figure 7A [href=#F7] , Table S7 [href=this file] ). In addition to 206 upregulated oncogenic proteins (OncoKB, Table S7 [href=this file] ), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors ( Figure S7B [href=#SD6] ), with the I-S samples showing the fewest CT antigens. Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines ( https://depmap.org/ [href=https://depmap.org/] ), suggesting critical roles in key cellular transformation and proliferation processes ( Figure 7C [href=#F7] , Table S7 [href=this file] ).\n",
      "See also Figure S7 [href=#SD6] and Table S7 [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 13 <a> elements with href='#SD14'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 2171) for SD14: Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters ( Figure 5A [href=#F5] , Figure S5A [href=this file] – B [href=this file] , Table S5 [href=#SD11] ).\n",
      "See also Figure S5 [href=this file] and Table S5 [href=#SD11]\n",
      "Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in Figure 5B [href=#F5] and Figure S5C [href=this file] . Consistent with this, we observed positive correlation between ISG15 and HERC5 abundance ( Figure S5D [href=this file] ). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations ( Figure S5E [href=this file] ). We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 ( Figure 5C [href=#F5] , Figure S5F [href=this file] ). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation ( Figure S5G [href=this file] , Figure S5H [href=this file] ). ISGylation can be induced by interferon in cancer ( Tecalco-Cruz and Cruz-Ramos, 2018 [href=#R171] ; Wong et al., 2006 [href=#R197] ); accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores ( Figure S5I [href=this file] , Figure S5J [href=this file] ), and HERC5 was increased in tumors of the IFN-gamma dominant subtype ( Figure 5D [href=#F5] ).\n",
      "In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket ( Wang et al., 2015 [href=#R188] ; Zhao et al., 2014 [href=#R209] )( Xiong et al., 2011 [href=#R200] ) ( Figure 5E [href=#F5] , Figure S5K [href=this file] ).\n",
      "We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors ( Figure 5E [href=#F5] , Figure S5K [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for fn-group1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for fn-group1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 574 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 582 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 97 - INFO - Length of original HTML: 819251\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 143 - INFO - Length of normalized HTML: 725869\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - DEBUG - Preprocessed data: <!DOCTYPE html> <html lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> A proteogenomic portrait of lung squamous cell carcinoma - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here's how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you've safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" role=\"button\" target=\"_blank\" title=\"View on publisher site\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip\" href=\"pdf/nihms-1730863.pdf\" role=\"button\" title=\"Download PDF\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"pmc-permalink display-flex\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to NIHPA Author Manuscripts\" href=\"https://www.hhs.gov/open/public-access-guiding-principles/index.html\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to NIHPA Author Manuscripts\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . Author manuscript; available in PMC: 2022 Aug 5. </div> <div> <em> Published in final edited form as: </em> Cell. 2021 Aug 5;184(16):4348–4371.e40. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1016/j.cell.2021.07.016 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> A proteogenomic portrait of lung squamous cell carcinoma </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Shankha Satpathy </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> </div> </div> <sup> 1, </sup> <sup> 19, </sup> <sup> * </sup> , <a aria-describedby=\"id2\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Karsten Krug </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id3\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Pierre M Jean Beltran </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id4\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Sara R Savage </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> </div> </div> <sup> 2, </sup> <sup> 19 </sup> , <a aria-describedby=\"id5\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Francesca Petralia </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id6\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> <div hidden=\"hidden\" id=\"id6\"> <h3> <span class=\"name western\"> Chandan Kumar-Sinha </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> </div> </div> <sup> 4, </sup> <sup> 20 </sup> , <a aria-describedby=\"id7\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> <div hidden=\"hidden\" id=\"id7\"> <h3> <span class=\"name western\"> Yongchao Dou </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> </div> </div> <sup> 2, </sup> <sup> 20 </sup> , <a aria-describedby=\"id8\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> <div hidden=\"hidden\" id=\"id8\"> <h3> <span class=\"name western\"> Boris Reva </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id9\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> <div hidden=\"hidden\" id=\"id9\"> <h3> <span class=\"name western\"> M Harry Kane </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id10\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> <div hidden=\"hidden\" id=\"id10\"> <h3> <span class=\"name western\"> Shayan C Avanessian </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id11\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> <div hidden=\"hidden\" id=\"id11\"> <h3> <span class=\"name western\"> Suhas V Vasaikar </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> </div> </div> <sup> 5, </sup> <sup> 20 </sup> , <a aria-describedby=\"id12\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> <div hidden=\"hidden\" id=\"id12\"> <h3> <span class=\"name western\"> Azra Krek </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id13\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> <div hidden=\"hidden\" id=\"id13\"> <h3> <span class=\"name western\"> Jonathan T Lei </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id14\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> <div hidden=\"hidden\" id=\"id14\"> <h3> <span class=\"name western\"> Eric J Jaehnig </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id15\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> <div hidden=\"hidden\" id=\"id15\"> <h3> <span class=\"name western\"> Tatiana Omelchenko </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id16\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> <div hidden=\"hidden\" id=\"id16\"> <h3> <span class=\"name western\"> Yifat Geffen </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id17\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> <div hidden=\"hidden\" id=\"id17\"> <h3> <span class=\"name western\"> Erik J Bergstrom </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id18\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> <div hidden=\"hidden\" id=\"id18\"> <h3> <span class=\"name western\"> Vasileios Stathias </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id19\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> <div hidden=\"hidden\" id=\"id19\"> <h3> <span class=\"name western\"> Karen E Christianson </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id20\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> <div hidden=\"hidden\" id=\"id20\"> <h3> <span class=\"name western\"> David I Heiman </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id21\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> <div hidden=\"hidden\" id=\"id21\"> <h3> <span class=\"name western\"> Marcin P Cieslik </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id22\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> <div hidden=\"hidden\" id=\"id22\"> <h3> <span class=\"name western\"> Song Cao </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id23\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> <div hidden=\"hidden\" id=\"id23\"> <h3> <span class=\"name western\"> Xiaoyu Song </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id24\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> <div hidden=\"hidden\" id=\"id24\"> <h3> <span class=\"name western\"> Jiayi Ji </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id25\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> <div hidden=\"hidden\" id=\"id25\"> <h3> <span class=\"name western\"> Wenke Liu </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id26\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> <div hidden=\"hidden\" id=\"id26\"> <h3> <span class=\"name western\"> Kai Li </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id27\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> <div hidden=\"hidden\" id=\"id27\"> <h3> <span class=\"name western\"> Bo Wen </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id28\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> <div hidden=\"hidden\" id=\"id28\"> <h3> <span class=\"name western\"> Yize Li </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id29\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> <div hidden=\"hidden\" id=\"id29\"> <h3> <span class=\"name western\"> Zeynep H Gümüş </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id30\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> <div hidden=\"hidden\" id=\"id30\"> <h3> <span class=\"name western\"> Myvizhi Esai Selvan </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id31\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> <div hidden=\"hidden\" id=\"id31\"> <h3> <span class=\"name western\"> Rama Soundararajan </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id32\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> <div hidden=\"hidden\" id=\"id32\"> <h3> <span class=\"name western\"> Tanvi H Visal </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id33\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> <div hidden=\"hidden\" id=\"id33\"> <h3> <span class=\"name western\"> Maria G Raso </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id34\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> <div hidden=\"hidden\" id=\"id34\"> <h3> <span class=\"name western\"> Edwin Roger Parra </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id35\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> <div hidden=\"hidden\" id=\"id35\"> <h3> <span class=\"name western\"> Özgün Babur </span> </h3> <div class=\"p\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> </div> </div> <sup> 11 </sup> , <a aria-describedby=\"id36\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> <div hidden=\"hidden\" id=\"id36\"> <h3> <span class=\"name western\"> Pankaj Vats </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id37\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> <div hidden=\"hidden\" id=\"id37\"> <h3> <span class=\"name western\"> Shankara Anand </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id38\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> <div hidden=\"hidden\" id=\"id38\"> <h3> <span class=\"name western\"> Tobias Schraink </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id39\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> <div hidden=\"hidden\" id=\"id39\"> <h3> <span class=\"name western\"> MacIntosh Cornwell </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id40\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> <div hidden=\"hidden\" id=\"id40\"> <h3> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id41\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> <div hidden=\"hidden\" id=\"id41\"> <h3> <span class=\"name western\"> Houxiang Zhu </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id42\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> <div hidden=\"hidden\" id=\"id42\"> <h3> <span class=\"name western\"> Chia-Kuei Mo </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id43\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> <div hidden=\"hidden\" id=\"id43\"> <h3> <span class=\"name western\"> Yuping Zhang </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id44\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> <div hidden=\"hidden\" id=\"id44\"> <h3> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id45\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> <div hidden=\"hidden\" id=\"id45\"> <h3> <span class=\"name western\"> Chen Huang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id46\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> <div hidden=\"hidden\" id=\"id46\"> <h3> <span class=\"name western\"> Arul M Chinnaiyan </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id47\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> <div hidden=\"hidden\" id=\"id47\"> <h3> <span class=\"name western\"> Matthew A Wyczalkowski </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id48\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> <div hidden=\"hidden\" id=\"id48\"> <h3> <span class=\"name western\"> Gilbert S Omenn </span> </h3> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> </div> </div> <sup> 13 </sup> , <a aria-describedby=\"id49\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> <div hidden=\"hidden\" id=\"id49\"> <h3> <span class=\"name western\"> Chelsea J Newton </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id50\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> <div hidden=\"hidden\" id=\"id50\"> <h3> <span class=\"name western\"> Stephan Schurer </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id51\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> <div hidden=\"hidden\" id=\"id51\"> <h3> <span class=\"name western\"> Kelly V Ruggles </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id52\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> <div hidden=\"hidden\" id=\"id52\"> <h3> <span class=\"name western\"> David Fenyö </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id53\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> <div hidden=\"hidden\" id=\"id53\"> <h3> <span class=\"name western\"> Scott D Jewell </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id54\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> <div hidden=\"hidden\" id=\"id54\"> <h3> <span class=\"name western\"> Mathangi Thiagarajan </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id55\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> <div hidden=\"hidden\" id=\"id55\"> <h3> <span class=\"name western\"> Mehdi Mesri </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id56\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> <div hidden=\"hidden\" id=\"id56\"> <h3> <span class=\"name western\"> Henry Rodriguez </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id57\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> <div hidden=\"hidden\" id=\"id57\"> <h3> <span class=\"name western\"> Sendurai A Mani </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id58\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> <div hidden=\"hidden\" id=\"id58\"> <h3> <span class=\"name western\"> Namrata D Udeshi </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id59\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> <div hidden=\"hidden\" id=\"id59\"> <h3> <span class=\"name western\"> Gad Getz </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id60\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> <div hidden=\"hidden\" id=\"id60\"> <h3> <span class=\"name western\"> James Suh </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id61\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> <div hidden=\"hidden\" id=\"id61\"> <h3> <span class=\"name western\"> Qing Kay Li </span> </h3> <div class=\"p\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> </div> </div> <sup> 17 </sup> , <a aria-describedby=\"id62\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> <div hidden=\"hidden\" id=\"id62\"> <h3> <span class=\"name western\"> Galen Hostetter </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id63\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> <div hidden=\"hidden\" id=\"id63\"> <h3> <span class=\"name western\"> Paul K Paik </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id64\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> <div hidden=\"hidden\" id=\"id64\"> <h3> <span class=\"name western\"> Saravana M Dhanasekaran </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id65\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> <div hidden=\"hidden\" id=\"id65\"> <h3> <span class=\"name western\"> Ramaswamy Govindan </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id66\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> <div hidden=\"hidden\" id=\"id66\"> <h3> <span class=\"name western\"> Li Ding </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id67\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> <div hidden=\"hidden\" id=\"id67\"> <h3> <span class=\"name western\"> Ana I Robles </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id68\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> <div hidden=\"hidden\" id=\"id68\"> <h3> <span class=\"name western\"> Karl R Clauser </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id69\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> <div hidden=\"hidden\" id=\"id69\"> <h3> <span class=\"name western\"> Alexey I Nesvizhskii </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id70\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> <div hidden=\"hidden\" id=\"id70\"> <h3> <span class=\"name western\"> Pei Wang </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id71\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> <div hidden=\"hidden\" id=\"id71\"> <h3> <span class=\"name western\"> Steven A Carr </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id72\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> <div hidden=\"hidden\" id=\"id72\"> <h3> <span class=\"name western\"> Bing Zhang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> </div> </div> <sup> 2, </sup> <sup> * </sup> , <a aria-describedby=\"id73\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> <div hidden=\"hidden\" id=\"id73\"> <h3> <span class=\"name western\"> D R Mani </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id74\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> <div hidden=\"hidden\" id=\"id74\"> <h3> <span class=\"name western\"> Michael A Gillette </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div class=\"p\"> <sup> 21 </sup> Lead author </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> </div> </div> <sup> 1, </sup> <sup> 18, </sup> <sup> 21, </sup> <sup> * </sup> , <span class=\"on-behalf-of\"> Clinical Proteomic Tumor Analysis Consortium </span> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"A1\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div id=\"A2\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div id=\"A3\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div id=\"A4\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A5\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div id=\"A6\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div id=\"A7\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div id=\"A8\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div id=\"A9\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div id=\"A10\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A11\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div id=\"A12\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A13\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A14\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div id=\"A15\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div id=\"A16\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div id=\"A17\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div id=\"A18\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div id=\"A19\"> <sup> 19 </sup> Equal contribution </div> <div id=\"A20\"> <sup> 20 </sup> Equal contribution </div> <div id=\"A21\"> <sup> 21 </sup> Lead author </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"FN1\"> <p id=\"P1\"> Author Contributions </p> <p id=\"P2\"> <strong> Conception </strong> : SS, SAC, MAG; <strong> Data Collection </strong> : SS, MHK, SCA, EJB; <strong> Data analysis </strong> : SS, KK, PMJB, SRS, FP, CKS, YD, BR,MHK, SVV, AK, JTL, EJJ, TO, YG, VS, KEC, DIH, MPC, SC, XS, JJ, WL, KL, BW, YL,ZHG, MES, OB, PV, SA, TS, MC, CKM, YZ, FVL, CH, MAW, SMD, KRC, DRM; <strong> QC </strong> : CJN, GH, JS, QKL, MT; <strong> IHC and analysis </strong> : RS, TV, MGR, ERP, SAM, CJN, GH, JS, KL; <strong> Manuscript writing and editing </strong> : SS and MAG with help from all authors <strong> Supervision </strong> : SS, LD, DF, KR, MM, HR, MT, AIR, PW, AIN, SAC, BZ, DRM, MAG </p> </div> <div class=\"fn\" id=\"FN2\"> <p id=\"P3\"> Secondary Authors </p> <p id=\"P4\"> Alex Green, Alfredo Molinolo, Alicia Francis, Amanda G. Paulovich, Andrii Karnuta, Antonio Colaprico, Barbara Hindenach, Barbara L. Pruetz, Bartosz Kubisa, Brian J. Druker, Carissa Huynh, Charles A. Goldthwaite Jr., Chet Birger, Christopher R. Kinsinger, Corbin D. Jones, Dan Rohrer, Dana R. Valley, Daniel W. Chan, David Chesla, Donna Hansel, Elena V. Ponomareva, Elizabeth Duffy, Eric Burks, Eric E. Schadt, Eugene S. Fedorov, Eunkyung An, Fei Ding, George D. Wilson, Harsh Batra, Hui Zhang, Jennifer E. Maas, Jennifer Eschbacher, Karen A. Ketchum, Karin D. Rodland, Katherine A. Hoadley, Kei Suzuki, Ki Sung Um, Liqun Qi, Lori Bernard, Maciej Wiznerowicz, Małgorzata Wojtyś, Marcin J. Domagalski, Matthew J. Ellis, Maureen A. Dyer, Melissa Borucki, Meenakshi Anurag, Michael J. Birrer, Midie Xu, Mikhail Krotevich, Nancy Roche, Nathan J. Edwards, Negin Vatanian, Neil R. Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Pamela Grady, Peter B. McGarvey, Pushpa Hariharan, Ratna R. Thangudu, Rebecca Montgomery, Renganayaki Pandurengan, Richard D. Smith, Robert J. Welsh, Sailaja Mareedu, Samuel H. Payne, Sandra Cottingham, Shilpi Singh, Shirley X. Tsang, Shuang Cai, Stacey Gabriel, Tao Liu, Tara Hiltke, Tanmayi Vashist, Thomas Bauer, Volodymyr Sovenko, Warren G. Tourtellotte, Weiping Ma, William Bocik, Wohaib Hasan, Xiaojun Jing, Ximing Tang, Yuxing Liao, Yvonne, Shutack, Zhen Zhang, Ziad Hanhan </p> </div> <div class=\"fn\" id=\"CR1\"> <sup> * </sup> <p class=\"display-inline\"> Corresponding authors: <span> shankha@broadinstitute.org </span> (S.S.), <span> scarr@broad.mit.edu </span> (S.A.C), <span> bing.zhang@bcm.edu </span> (B.Z.), <span> manidr@broadinstitute.org </span> (D.R.M.), <span> gillette@broadinstitute.org </span> (M.A.G.) </p> </div> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC8475722 NIHMSID: NIHMS1730863 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34358469/\"> 34358469 </a> </div> <div aria-label=\"Related or updated information about this article\" class=\"ra xbox p\" role=\"complementary\"> <div> The publisher's version of this article is available at <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> Cell </a> </div> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"ABS1\"> <h2> Summary </h2> <p id=\"P5\"> Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. </p> <section class=\"kwd-group\" id=\"kwd-group1\"> <p> <strong> Keywords: </strong> Lung Cancer, Squamous, Proteogenomics, Proteomics, Genomics, Protein, Phosphorylation, Acetylation, Ubiquitylation, CPTAC </p> </section> </section> <section class=\"abstract\" id=\"ABS2\"> <h2> In Brief </h2> <p id=\"P6\"> Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential therapeutic targets. </p> </section> <section class=\"abstract\" id=\"ABS3\"> <h2> Graphical Abstract </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> </section> <section id=\"S1\"> <h2 class=\"pmc_sec_title\"> Introduction </h2> <p id=\"P8\"> Lung cancer is the leading cause of cancer-associated mortality ( <a aria-describedby=\"R15\" class=\"usa-link\" href=\"#R15\"> Bray et al., 2018 </a> ; <a aria-describedby=\"R158\" class=\"usa-link\" href=\"#R158\"> Siegel et al., 2020 </a> ). Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ; <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as <em> CDKN2A </em> ) are found in upwards of 60% but targeting these clinically has largely failed ( <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). While other therapeutic regimens are being explored as part of the LungMAP consortium ( <a aria-describedby=\"R7\" class=\"usa-link\" href=\"#R7\"> Ardini-Poleske et al., 2017 </a> ), to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC ( <a aria-describedby=\"R76\" class=\"usa-link\" href=\"#R76\"> Karachaliou et al., 2018 </a> ; <a aria-describedby=\"R137\" class=\"usa-link\" href=\"#R137\"> Paz-Ares et al., 2018 </a> ). Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. </p> <p id=\"P9\"> We characterized 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. </p> </section> <section id=\"S2\"> <h2 class=\"pmc_sec_title\"> Results </h2> <section id=\"S3\"> <h3 class=\"pmc_sec_title\"> Proteogenomic landscape of LSCC </h3> <p id=\"P10\"> We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> , <a class=\"usa-link\" href=\"#SD1\"> S1A </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> ). Proteomics data were generated using TMT11-based multiplexing linked by a common reference ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1A </a> ) and demonstrated high technical quality ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1B </a> – <a class=\"usa-link\" href=\"#SD1\"> C </a> ). The cohort demographics are summarized in <a class=\"usa-link\" href=\"#F1\"> Figure 1B </a> . Most self-reported never-smokers displayed high genomic smoking scores ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1D </a> ), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). The genomic landscape reaffirmed previously reported somatic alterations in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1C </a> ). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1E </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F1\"> <h4 class=\"obj_head\"> Figure 1: Proteogenomic Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0002.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P373\"> A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. </p> <p id=\"P374\"> B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. </p> <p id=\"P375\"> C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value &lt; 0.1) in this dataset are highlighted in red font. </p> <p id=\"P376\"> D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR &lt;0.01) positive and negative correlations, respectively. </p> <p id=\"P377\"> E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q&lt;0.25) and <em> cis </em> -effects in both mRNA and protein (FDR&lt;0.05). </p> <p id=\"P378\"> F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the <em> FGFR1 </em> gene (GISTIC thresholded value =2). </p> <p id=\"P379\"> G. Genes whose DNA methylation was significantly associated with cascading <em> cis </em> regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the <em> cis </em> -effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. </p> <p id=\"P380\"> See also <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> </p> </figcaption> </figure> <p id=\"P11\"> Proteogenomic data helped annotate the impact of copy number alteration (CNA) events ( <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> , <a class=\"usa-link\" href=\"#F1\"> E </a> ). Of 5,523 significant (FDR&lt;0.01) <em> cis </em> CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 “cancer associated genes” (CAG) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). We identified six amplified (including <em> WHSC1L1, CCND1 </em> , and <em> SOX2 </em> ) and 29 deleted (including <em> NCOR1 </em> , <em> SETD2 </em> , and <em> CBL </em> ) CAGs in significant focal events (Q&lt;0.25) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ). Intriguingly, <em> WHSC1L1 </em> ( <em> NSD3 </em> ) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses <em> FGFR1 </em> . Therapies targeted against FGFR1 have been unsuccessful ( <a aria-describedby=\"R108\" class=\"usa-link\" href=\"#R108\"> Lim et al., 2016 </a> ; <a aria-describedby=\"R189\" class=\"usa-link\" href=\"#R189\"> Weeden et al., 2015 </a> ). Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon ( <a class=\"usa-link\" href=\"#F1\"> Figure 1F </a> ), nominating it as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes ( <a aria-describedby=\"R205\" class=\"usa-link\" href=\"#R205\"> Yuan et al., 2021 </a> ). </p> <p id=\"P12\"> To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ) to perturbation profiles from the Connectivity Map (CMap) ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ). Six CNAs ( <em> IKBKAP </em> , <em> PIN1 </em> , <em> DNAJB1 </em> , <em> IL18 </em> , <em> NR2F6 </em> , <em> AKAP </em> ) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher’s exact p&lt;0.01) with clinical metadata. <em> IL-18 </em> deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (upper panel)), consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC ( <a aria-describedby=\"R175\" class=\"usa-link\" href=\"#R175\"> Timperi et al., 2017 </a> ). <em> NR2F6 </em> amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade ( <a aria-describedby=\"R83\" class=\"usa-link\" href=\"#R83\"> Klepsch et al., 2018 </a> ). </p> <p id=\"P13\"> LSCC tumor DNA showed overall hypermethylation relative to NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1G </a> ), with tumors separating into CIMP (CpG island methylator phenotype) clusters ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1H </a> ). Multi-omic data allowed the identification of “cascading” promoter methylation <em> cis </em> -effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR&lt;0.1) <em> cis </em> -effects in tumors ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), 20 were also differentially expressed (FDR&lt;0.01) between tumors and NATs ( <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> ). While most were newly associated with LSCC, methylation-driven expression of <em> FAM110A </em> , <em> PTGES3 </em> , <em> PLAU </em> and <em> SLC16A3 </em> ( <a aria-describedby=\"R35\" class=\"usa-link\" href=\"#R35\"> Faubert et al., 2017 </a> ; <a aria-describedby=\"R44\" class=\"usa-link\" href=\"#R44\"> Gao et al., 2018 </a> ; <a aria-describedby=\"R79\" class=\"usa-link\" href=\"#R79\"> Kikuchi et al., 2012 </a> ; <a aria-describedby=\"R105\" class=\"usa-link\" href=\"#R105\"> Liang et al., 2013 </a> ; <a aria-describedby=\"R136\" class=\"usa-link\" href=\"#R136\"> Payen et al., 2020 </a> ; <a aria-describedby=\"R157\" class=\"usa-link\" href=\"#R157\"> Showe et al., 2009 </a> ) has been implicated in lung and other cancers. </p> </section> <section id=\"S4\"> <h3 class=\"pmc_sec_title\"> Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched </h3> <p id=\"P14\"> We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the entire cohort ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> ). The five resulting multi-omic subtypes ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ) were named based on their predominant pathway associations and similarities to previously defined RNA clusters ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ). Significant associations between NMF subtypes and cohort metadata are presented in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . Two NMF subtypes, “Basal-Inclusive” (B-I) and “Epithelial to Mesenchymal Transition-enriched” (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni correction (Fisher’s p&lt;0.01, <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in <em> KEAP1 </em> , <em> CUL3 </em> and <em> NFE2L2 </em> genes and high-level amplification (GISTIC threshold value = 2) of <em> SOX2 </em> and <em> TP63 </em> ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD8\"> Table S2 </a> ) was labeled the “Classical” subtype, consistent with TCGA nomenclature and with a previous publication ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ). These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> ). A fourth NMF subtype was designated “Inflamed-Secretory” (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, “Proliferative-Primitive” (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a “mixed” subgroup, since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival ( <a class=\"usa-link\" href=\"#F2\"> Figure 2C </a> ), an observation also made in a recent proteogenomic analysis of glioblastoma ( <a aria-describedby=\"R187\" class=\"usa-link\" href=\"#R187\"> Wang et al., 2021 </a> ). The mixed class was associated in the current cohort with increased frequency of <em> SOX2 </em> amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. </p> <figure class=\"fig xbox font-sm\" id=\"F2\"> <h4 class=\"obj_head\"> Figure 2: LSCC Molecular Subtypes and Associations. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0003.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P381\"> A. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). </p> <p id=\"P382\"> B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. </p> <p id=\"P383\"> C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). </p> <p id=\"P384\"> D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. </p> <p id=\"P385\"> E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8×10 <sup> −14 </sup> ). </p> <p id=\"P386\"> F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5×10 <sup> −6 </sup> for PDGFRB and 2.7×10 <sup> −7 </sup> for ROR2. </p> <p id=\"P387\"> G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. </p> <p id=\"P388\"> See also <a class=\"usa-link\" href=\"#SD2\"> Figure S2 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD10\"> 4 </a> </p> </figcaption> </figure> <p id=\"P15\"> We compared B-I ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ) and EMT-E ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 ( <a aria-describedby=\"R56\" class=\"usa-link\" href=\"#R56\"> Happel et al., 2009 </a> ) and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis ( <a aria-describedby=\"R64\" class=\"usa-link\" href=\"#R64\"> Hsu et al., 2020 </a> ), and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance ( <a aria-describedby=\"R67\" class=\"usa-link\" href=\"#R67\"> Hubaux et al., 2015 </a> ; <a aria-describedby=\"R100\" class=\"usa-link\" href=\"#R100\"> Lewandowski and Piwnica-Worms, 2014 </a> ). Transcription activator PBX3, which is upregulated only at the protein level ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, ( <a aria-describedby=\"R94\" class=\"usa-link\" href=\"#R94\"> Lamprecht et al., 2018 </a> ) including in laryngeal squamous cell carcinoma ( <a aria-describedby=\"R198\" class=\"usa-link\" href=\"#R198\"> Wu et al., 2020 </a> ). Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2D </a> ), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2E </a> ) and with the xCell-based deconvolution signature ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2F </a> ). This correlation was not observed in paired NATs, suggesting a tumor-specific role. </p> <p id=\"P16\"> In addition to a strong EMT signature ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), the EMT-E subtype showed upregulated VCAN ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ), a tumor-promoting target of SNAIL ( <a aria-describedby=\"R207\" class=\"usa-link\" href=\"#R207\"> Zhang et al., 2019a </a> ) expressed in cancer-associated fibroblasts (CAFs) ( <a aria-describedby=\"R24\" class=\"usa-link\" href=\"#R24\"> Chida et al., 2016 </a> ), and FHL3, a protein that stabilizes EMT-associated transcription factors ( <a aria-describedby=\"R104\" class=\"usa-link\" href=\"#R104\"> Li et al., 2020 </a> ) ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in <a class=\"usa-link\" href=\"#F2\"> Figure 2D </a> – <a class=\"usa-link\" href=\"#F2\"> E </a> , coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a “fibroblast proliferation” signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker α-smooth muscle actin (α-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2G </a> ), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either CK or α-SMA alone. TGFβ is a well-established and potent inducer of EMT ( <a aria-describedby=\"R166\" class=\"usa-link\" href=\"#R166\"> Su et al., 2020 </a> ), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFβ inhibitors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2H </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4H </a> ). </p> <p id=\"P17\"> Integrating our global proteome data with a prior LSCC dataset ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> – <a class=\"usa-link\" href=\"#SD2\"> J </a> ). </p> </section> <section id=\"S5\"> <h3 class=\"pmc_sec_title\"> NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. </h3> <p id=\"P18\"> Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p &lt;0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E ( <a class=\"usa-link\" href=\"#F2\"> Figure 2F </a> ), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state ( <a class=\"usa-link\" href=\"#F2\"> Figure 2G </a> ) ( <a aria-describedby=\"R60\" class=\"usa-link\" href=\"#R60\"> Herreño et al., 2019 </a> ; <a aria-describedby=\"R130\" class=\"usa-link\" href=\"#R130\"> Niu et al., 2012 </a> ). As summarized in <a class=\"usa-link\" href=\"#SD2\"> Figure S2K </a> , we found phosphosite-based evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR ( <a aria-describedby=\"R169\" class=\"usa-link\" href=\"#R169\"> Tam et al., 2013 </a> ), both targetable ( <a aria-describedby=\"R150\" class=\"usa-link\" href=\"#R150\"> Roskoski 2018 </a> ; <a aria-describedby=\"R25\" class=\"usa-link\" href=\"#R25\"> Debebe and Rathmell 2015 </a> ) and recently implicated in the progression from a hybrid E/M to a highly mesenchymal state ( <a aria-describedby=\"R87\" class=\"usa-link\" href=\"#R87\"> Kröger et al., 2019 </a> ). Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. </p> </section> <section id=\"S6\"> <h3 class=\"pmc_sec_title\"> Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable </h3> <p id=\"P19\"> We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs ( <em> cis </em> -effects) and on a set of cancer associated genes ( <em> trans </em> -effects) ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ). Significant (FDR&lt;0.05) <em> Cis/Trans </em> pairs are shown in <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> – <a class=\"usa-link\" href=\"#F3\"> D </a> . <em> RB1 </em> mutated tumors had upregulated cell cycle-related protein expression. <em> NOTCH1 </em> mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK ( <a aria-describedby=\"R69\" class=\"usa-link\" href=\"#R69\"> Ishitani et al., 2010 </a> ) and GSK3B ( <a aria-describedby=\"R38\" class=\"usa-link\" href=\"#R38\"> Foltz et al., 2002 </a> ; <a aria-describedby=\"R71\" class=\"usa-link\" href=\"#R71\"> Jin et al., 2009 </a> ; <a aria-describedby=\"R210\" class=\"usa-link\" href=\"#R210\"> Zheng and Conner, 2018 </a> ) inhibit NOTCH1, recently described as a tumor suppressor in LSCC ( <a aria-describedby=\"R160\" class=\"usa-link\" href=\"#R160\"> Sinicropi-Yao et al., 2019 </a> ), suggesting that both might inhibit NOTCH1 downstream signaling in tumors with <em> NOTCH1 </em> mutations. <em> ARID1A </em> is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden ( <a aria-describedby=\"R156\" class=\"usa-link\" href=\"#R156\"> Shen et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3A </a> ) and led to protein downregulation, associated with worse overall survival in NSCLC ( <a aria-describedby=\"R68\" class=\"usa-link\" href=\"#R68\"> Hung et al., 2020 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F3\"> <h4 class=\"obj_head\"> Figure 3: Impact of Somatic Mutations on Proteogenomic Features. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0004.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F3/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P389\"> A. Significant (Wilcoxon FDR&lt;0.05) <em> cis </em> - (circles) and <em> trans </em> -effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green. </p> <p id=\"P390\"> B. Similar to panel A but showing phosphosites. </p> <p id=\"P391\"> C. Similar to panel A but showing acetylsites. </p> <p id=\"P392\"> D. Similar to panel A but showing ubiquitylsites. </p> <p id=\"P393\"> E. CNA data for <em> CDKN2A </em> and <em> RB1 </em> was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CDKN2A </em> genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these <em> CDKN2A/p16 </em> aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of <em> CCND1–3 </em> , <em> CDK4 </em> , and <em> CDK6 </em> were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of <em> CDKN2A </em> are not included. </p> <p id=\"P394\"> F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in <em> KEAP1 </em> , <em> CUL3 </em> , or <em> NFE2L2 </em> ) versus NRF2 WT tumors (no NRF2 pathway aberration). </p> <p id=\"P395\"> G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype. </p> <p id=\"P396\"> H. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. </p> <p id=\"P397\"> I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR&lt;0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. </p> <p id=\"P398\"> See also <a class=\"usa-link\" href=\"#SD3\"> Figure S3 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P20\"> Paradoxically, mutations in <em> CDKN2A, </em> proposed as a potential biomarker for CDK4/6 inhibitors ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ), resulted in increased RNA expression ( <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> ). We analyzed the effects of genetic and epigenetic alterations of <em> CDKN2A </em> on both its major isoforms, p16INK4a (p16) and p14ARF (p14) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). As expected, its homozygous deletion resulted in the loss of expression of both isoforms ( <a class=\"usa-link\" href=\"#F3\"> Figure 3E </a> ). However, only eight of the 58 <em> CDKN2A </em> wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with <em> CDKN2A </em> /p16 missense mutations (log2(FPKM+1)&gt;2). Many of those <em> CDKN2A </em> /p16 mutations were unlikely to result in nonsense-mediated decay ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3B </a> ), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had <em> RB1 </em> mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes ( <a aria-describedby=\"R85\" class=\"usa-link\" href=\"#R85\"> Knudsen et al., 2020 </a> ). This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ), there was no association between the amplification of <em> CCND2 </em> , <em> CCND3 </em> , <em> CDK4 </em> , or <em> CDK6 </em> and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CCND1 </em> amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3C </a> ). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in <em> CDKN2A </em> and copy number alterations in <em> CCND1 </em> and <em> CDKN2A </em> ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib ( <a aria-describedby=\"R135\" class=\"usa-link\" href=\"#R135\"> Patnaik et al., 2016 </a> ) and palbociclib ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R31\" class=\"usa-link\" href=\"#R31\"> Edelman et al., 2019 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ). While we found that <em> CCND1 </em> amplification resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with <em> CCND1 </em> amplification provides a potential explanation for some patients’ lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 inhibition. </p> </section> <section id=\"S7\"> <h3 class=\"pmc_sec_title\"> NRF2 pathway activation in tumors with and without NRF2 pathway mutations </h3> <p id=\"P21\"> LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> . NFE2L2 phosphorylation was observed in samples with any of these mutations ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3E </a> ). LSCC tumors were annotated as “NRF2 pathway mutated” (n=33; with either one hit [n=28] or two hits [n=5] in <em> NFE2L2 </em> , <em> KEAP1 </em> , and <em> CUL3 </em> ), or “NRF2 WT” (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes ( <a class=\"usa-link\" href=\"#F3\"> Figure 3F </a> , <a class=\"usa-link\" href=\"#SD3\"> Figure S3F </a> – <a class=\"usa-link\" href=\"#SD3\"> G </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3H </a> ). Using the proteogenomic signature ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) that defined <em> NFE2L2 </em> mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype ( <a class=\"usa-link\" href=\"#F3\"> Figure 3G </a> ). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein ( <a class=\"usa-link\" href=\"#F3\"> Figure 3H </a> ) and PTM-SEA-derived activity ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) were significantly upregulated (FDR&lt;0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes ( <a aria-describedby=\"R70\" class=\"usa-link\" href=\"#R70\"> Jimenez-Blasco et al., 2015 </a> ) and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ), <em> KEAP1 </em> mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. </p> </section> <section id=\"S8\"> <h3 class=\"pmc_sec_title\"> Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC </h3> <p id=\"P22\"> Chromosome 3q, which harbors key squamous differentiation markers <em> SOX2 </em> and <em> TP63 </em> ( <a aria-describedby=\"R143\" class=\"usa-link\" href=\"#R143\"> Qian and Massion, 2008 </a> ), showed the most dramatic arm-level amplification in this cohort ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4A </a> ). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). LSCC cell lines amplified for <em> TP63 </em> and <em> SOX2 </em> were highly dependent upon them, supporting oncogene addiction ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F4\"> <h4 class=\"obj_head\"> Figure 4: Proteogenomic Impact of Chromosome 3q Amplification. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0005.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F4/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P399\"> A. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. </p> <p id=\"P400\"> B. Frequency distribution of ΔNp63α RNA expression in tumors and NATs. </p> <p id=\"P401\"> C. Differential protein expression in samples classified as ΔNp63α-low vs -high based on ΔNp63α transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. </p> <p id=\"P402\"> D. Differential expression of microRNAs in ΔNp63α-low vs -high samples. </p> <p id=\"P403\"> E. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. </p> <p id=\"P404\"> F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. </p> <p id=\"P405\"> G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3×10 <sup> −08 </sup> , Correlation = −0.53) and DNA replication (bottom, p=1.1×10 <sup> −06 </sup> , Correlation= 0.47) scores. </p> <p id=\"P406\"> H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR&lt;0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. </p> <p id=\"P407\"> I. Pearson correlation (p= 5.2×10 <sup> −07 </sup> , Correlation = −0.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES. </p> <p id=\"P408\"> See also <a class=\"usa-link\" href=\"#SD4\"> Figure S4 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P23\"> As previously reported ( <a aria-describedby=\"R20\" class=\"usa-link\" href=\"#R20\"> Campbell et al., 2018 </a> ; <a aria-describedby=\"R113\" class=\"usa-link\" href=\"#R113\"> Lo Iacono et al., 2011 </a> ), ΔNp63α was the dominant TP63 isoform in this dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4C </a> ) and was highly correlated with TP63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4D </a> ). Its amplification was associated with improved survival in the TCGA dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4E </a> ). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 “Np63-low” samples histologically confirmed to be LSCC showed no elevation at RNA or protein levels ( <a class=\"usa-link\" href=\"#F4\"> Figure 4B </a> ). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group ( <a class=\"usa-link\" href=\"#F4\"> Figure 4C </a> ). Notably, a substantial number of LSCC cell lines also had low Np63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4F </a> ), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Survivin promotes cell proliferation and G2M checkpoints ( <a aria-describedby=\"R193\" class=\"usa-link\" href=\"#R193\"> Wheatley and Altieri, 2019 </a> ), evident in Np63-low samples ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4H </a> ). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in NSCLC ( <a aria-describedby=\"R48\" class=\"usa-link\" href=\"#R48\"> Giaccone et al., 2009 </a> ; <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ; <a aria-describedby=\"R101\" class=\"usa-link\" href=\"#R101\"> Li et al., 2019a </a> ), but have shown modest or no improvement ( <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ). Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin inhibition. </p> <p id=\"P24\"> Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC ( <a aria-describedby=\"R97\" class=\"usa-link\" href=\"#R97\"> Lebanony et al., 2009 </a> ) ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4I </a> ), was associated with decreased promoter DNA methylation ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4J </a> ) and elevation of ΔNp63α, a transcriptional regulator of this miRNA ( <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4D </a> , <a class=\"usa-link\" href=\"#SD4\"> Figure S4K </a> ). As expected, the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN ( <a aria-describedby=\"R18\" class=\"usa-link\" href=\"#R18\"> Cai et al., 2013 </a> ; <a aria-describedby=\"R53\" class=\"usa-link\" href=\"#R53\"> Gregory et al., 2008 </a> ; <a aria-describedby=\"R172\" class=\"usa-link\" href=\"#R172\"> Tellez et al., 2011 </a> ; <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ). TP73 positively correlated with miR-205 at the transcript ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ) and still more strongly at the protein level ( <a class=\"usa-link\" href=\"#F4\"> Figure 4F </a> ) and with TP63 protein but not RNA ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4L </a> ). TP73 overexpression mediates ΔNp63α-dependent cell survival in squamous cell carcinoma ( <a aria-describedby=\"R28\" class=\"usa-link\" href=\"#R28\"> DeYoung et al., 2006 </a> ), suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC ( <a aria-describedby=\"R126\" class=\"usa-link\" href=\"#R126\"> Natan and Joerger, 2012 </a> ). Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication ( <a class=\"usa-link\" href=\"#F4\"> Figure 4G </a> ). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in previous studies ( <a aria-describedby=\"R110\" class=\"usa-link\" href=\"#R110\"> Lindsay et al., 2011 </a> ; <a aria-describedby=\"R164\" class=\"usa-link\" href=\"#R164\"> Srivastava et al., 2018 </a> ; <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). Of note, although ΔNp63α and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, ΔNp63α, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype ( <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ), and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. </p> <p id=\"P25\"> <em> SOX2 </em> , a recurrently amplified squamous cell marker often co-amplified with <em> TP63 </em> , was also overexpressed in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). <em> SOX2 </em> is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression ( <a aria-describedby=\"R206\" class=\"usa-link\" href=\"#R206\"> Zhang et al., 2013 </a> ) and its inhibition in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT04350463\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT04350463 </a> ). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer ( <a aria-describedby=\"R144\" class=\"usa-link\" href=\"#R144\"> Ramadoss et al., 2017 </a> ), and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression ( <a aria-describedby=\"R203\" class=\"usa-link\" href=\"#R203\"> Yomtoubian et al., 2020 </a> ). EZH2 dependency was observed in SOX2 amplified LSCC cell lines ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> ). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) suggest stemness-like features in these tumors ( <a aria-describedby=\"R78\" class=\"usa-link\" href=\"#R78\"> Keysar et al., 2017 </a> ; <a aria-describedby=\"R112\" class=\"usa-link\" href=\"#R112\"> Liu et al., 2016 </a> ; <a aria-describedby=\"R134\" class=\"usa-link\" href=\"#R134\"> Patel et al., 2008 </a> ). Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ), and negative correlation was seen between SOX2 and JAK-STAT signaling ( <a class=\"usa-link\" href=\"#F4\"> Figure 4I </a> ). </p> </section> <section id=\"S9\"> <h3 class=\"pmc_sec_title\"> Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism </h3> <p id=\"P26\"> Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ) associated with country of origin and ethnicity (Fisher’s exact p&lt;0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5A </a> – <a class=\"usa-link\" href=\"#SD14\"> B </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F5\"> <h4 class=\"obj_head\"> Figure 5: Ubiquitylation landscape in LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0006.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F5/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P409\"> A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR&lt;0.01). Enriched pathways and molecular and clinical annotations are indicated. </p> <p id=\"P410\"> B. Number of K-GG sites showing significant correlations (FDR&lt; 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. </p> <p id=\"P411\"> C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. </p> <p id=\"P412\"> D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal−Wallis, p = 2.8×10 <sup> −05 </sup> ) association is seen between HERC5 abundance and immune subtypes. </p> <p id=\"P413\"> E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). </p> <p id=\"P414\"> F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal−Wallis p-values are indicated in the respective plots. </p> <p id=\"P415\"> G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. </p> <p id=\"P416\"> H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)&gt;1 or Hypo: &lt;−1, FDR&lt;0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. “k” represents modified lysine. </p> <p id=\"P417\"> I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p&gt;0.05. </p> <p id=\"P418\"> See also <a class=\"usa-link\" href=\"#SD14\"> Figure S5 </a> and <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> </p> </figcaption> </figure> <p id=\"P27\"> To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in <a class=\"usa-link\" href=\"#F5\"> Figure 5B </a> and <a class=\"usa-link\" href=\"#SD14\"> Figure S5C </a> . HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in humans ( <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); hence the numerous positively correlated Ub/UbL sites (n=474; FDR&lt;0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 abundance ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5D </a> ). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5E </a> ). We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 ( <a class=\"usa-link\" href=\"#F5\"> Figure 5C </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5F </a> ). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5G </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5H </a> ). ISG15 modifications have been described in these glycolytic enzymes ( <a aria-describedby=\"R3\" class=\"usa-link\" href=\"#R3\"> Albert et al., 2018 </a> ; <a aria-describedby=\"R49\" class=\"usa-link\" href=\"#R49\"> Giannakopoulos et al., 2005 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ), and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis ( <a aria-describedby=\"R4\" class=\"usa-link\" href=\"#R4\"> Alcalá et al., 2020 </a> ). ISGylation can be induced by interferon in cancer ( <a aria-describedby=\"R171\" class=\"usa-link\" href=\"#R171\"> Tecalco-Cruz and Cruz-Ramos, 2018 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5I </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5J </a> ), and HERC5 was increased in tumors of the IFN-gamma dominant subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5D </a> ). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion ( <a aria-describedby=\"R208\" class=\"usa-link\" href=\"#R208\"> Zhang et al., 2019b </a> ). Collectively, our data supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. </p> <p id=\"P28\"> This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ) to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p&lt;0.05) ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket ( <a aria-describedby=\"R188\" class=\"usa-link\" href=\"#R188\"> Wang et al., 2015 </a> ; <a aria-describedby=\"R209\" class=\"usa-link\" href=\"#R209\"> Zhao et al., 2014 </a> )( <a aria-describedby=\"R200\" class=\"usa-link\" href=\"#R200\"> Xiong et al., 2011 </a> ) ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator ( <a class=\"usa-link\" href=\"#F5\"> Figures 5F </a> ). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37) ( <a aria-describedby=\"R11\" class=\"usa-link\" href=\"#R11\"> Bando et al., 2005 </a> ; <a aria-describedby=\"R201\" class=\"usa-link\" href=\"#R201\"> Yang and Lu, 2013 </a> ). Upregulation of a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs ( <a aria-describedby=\"R26\" class=\"usa-link\" href=\"#R26\"> DeBerardinis and Chandel, 2016 </a> ; <a aria-describedby=\"R61\" class=\"usa-link\" href=\"#R61\"> Hitosugi and Chen, 2014 </a> ). While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. </p> <p id=\"P29\"> Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR&lt;0.01). The TXN1 activator TXNRD1 ( <a aria-describedby=\"R17\" class=\"usa-link\" href=\"#R17\"> Cadenas et al., 2010 </a> ) was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1 ( <a aria-describedby=\"R124\" class=\"usa-link\" href=\"#R124\"> Morrison et al., 2014 </a> ), was decreased ( <a class=\"usa-link\" href=\"#F5\"> Figure 5F </a> ). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p&lt;2.2×10 <sup> −16 </sup> ), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial cancers ( <a aria-describedby=\"R42\" class=\"usa-link\" href=\"#R42\"> Fu et al., 2017 </a> ), including NSCLC ( <a aria-describedby=\"R36\" class=\"usa-link\" href=\"#R36\"> Fernandes et al., 2009 </a> ). The small proportion of glucose directed to the tumor mitochondria ( <a aria-describedby=\"R34\" class=\"usa-link\" href=\"#R34\"> Fan et al., 2019 </a> ) and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis) ( <a aria-describedby=\"R9\" class=\"usa-link\" href=\"#R9\"> Baeza et al., 2015 </a> ; <a aria-describedby=\"R43\" class=\"usa-link\" href=\"#R43\"> Gandhi and Das, 2019 </a> ; <a aria-describedby=\"R58\" class=\"usa-link\" href=\"#R58\"> Hebert et al., 2013 </a> ). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). PTM-based modulation of these key metabolic and ROS pathways is summarized in <a class=\"usa-link\" href=\"#F5\"> Figure 5G </a> . </p> <p id=\"P30\"> As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. <a class=\"usa-link\" href=\"#F5\"> Figure 5H </a> shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5I </a> ), named in part for its generally high proliferation ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. </p> </section> <section id=\"S10\"> <h3 class=\"pmc_sec_title\"> Immune landscape and regulation in LSCC </h3> <p id=\"P31\"> Consensus clustering based on the xCell ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) signatures ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> C </a> ), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher’s exact p=6.371×10 <sup> −09 </sup> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> ). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses ( <a class=\"usa-link\" href=\"#F6\"> Figures 6A </a> , <a class=\"usa-link\" href=\"#F6\"> C </a> , <a class=\"usa-link\" href=\"#SD5\"> S6C </a> , <a class=\"usa-link\" href=\"#SD5\"> D </a> ), and showed upregulation of immune-related pathways ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Notably, NATs did not display consonant immune signaling ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6B </a> ). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> ). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6B </a> ). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6E </a> ) </p> <figure class=\"fig xbox font-sm\" id=\"F6\"> <h4 class=\"obj_head\"> Figure 6: Immune Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0007.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F6/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P419\"> A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). </p> <p id=\"P420\"> B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. </p> <p id=\"P421\"> C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types’ distributions observed across the different immune clusters. </p> <p id=\"P422\"> D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR &lt; 0.1). </p> <p id=\"P423\"> E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. </p> <p id=\"P424\"> F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P425\"> G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) and the third pathway scores based on global protein abundance. </p> <p id=\"P426\"> H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. </p> <p id=\"P427\"> I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also <a class=\"usa-link\" href=\"#SD5\"> Figure S6 </a> and <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> </p> </figcaption> </figure> <p id=\"P32\"> Rho GTPase signaling was upregulated in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation ( <a aria-describedby=\"R45\" class=\"usa-link\" href=\"#R45\"> Garcia-Mata et al., 2011 </a> ), was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial enzymes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly ( <a aria-describedby=\"R90\" class=\"usa-link\" href=\"#R90\"> Kuhlmann et al., 2016a </a> ) by activating RhoA ( <a aria-describedby=\"R91\" class=\"usa-link\" href=\"#R91\"> Kuhlmann et al., 2016b </a> ). In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low ( <a aria-describedby=\"R52\" class=\"usa-link\" href=\"#R52\"> Gorvel et al., 1998 </a> ), and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead ( <a aria-describedby=\"R123\" class=\"usa-link\" href=\"#R123\"> Moissoglu et al., 2009 </a> ). In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators ( <a class=\"usa-link\" href=\"#F6\"> Figures 6F </a> , <a class=\"usa-link\" href=\"#SD5\"> S6F </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Notably, these proteins are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6G </a> ), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas ( <a aria-describedby=\"R84\" class=\"usa-link\" href=\"#R84\"> von Klot et al., 2017 </a> ). Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ) and immune-enriched networks ( <a aria-describedby=\"R140\" class=\"usa-link\" href=\"#R140\"> Peters et al., 2017 </a> ). </p> <p id=\"P33\"> We explored the immunological associations of the RTK CBPE scores described above ( <a class=\"usa-link\" href=\"#F2\"> Figure 2I </a> , <a class=\"usa-link\" href=\"#F2\"> J </a> ) in LSCC tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster ( <a class=\"usa-link\" href=\"#F6\"> Figure 6H </a> ) and consequently with immune-related pathways and cell types ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103 ( <a aria-describedby=\"R19\" class=\"usa-link\" href=\"#R19\"> Cammer et al., 2009 </a> ; <a aria-describedby=\"R30\" class=\"usa-link\" href=\"#R30\"> Dovas et al., 2009 </a> ; <a aria-describedby=\"R125\" class=\"usa-link\" href=\"#R125\"> Mouchemore et al., 2013 </a> ), and upregulation of AKAP13, among other immune-related proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6I </a> ). Tumor-associated macrophages (( <a aria-describedby=\"R23\" class=\"usa-link\" href=\"#R23\"> Cassetta and Pollard, 2018 </a> ), CSF1R signaling ( <a aria-describedby=\"R21\" class=\"usa-link\" href=\"#R21\"> Cannarile et al., 2017 </a> ), and AKAP13 ( <a aria-describedby=\"R29\" class=\"usa-link\" href=\"#R29\"> Diviani et al., 2016 </a> ) have each been proposed as anti-cancer targets. </p> </section> <section id=\"S11\"> <h3 class=\"pmc_sec_title\"> Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment </h3> <p id=\"P34\"> Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment ( <a class=\"usa-link\" href=\"#F7\"> Figure 7A </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). The top 50 of 502 differential proteins are shown in <a class=\"usa-link\" href=\"#SD6\"> Figure S7A </a> . In addition to 206 upregulated oncogenic proteins (OncoKB, <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7B </a> ), with the I-S samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS ( <a class=\"usa-link\" href=\"#F7\"> Figure 7B </a> , <a class=\"usa-link\" href=\"#SD6\"> Figure S7C </a> ). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/ </a> ), suggesting critical roles in key cellular transformation and proliferation processes ( <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F7\"> <h4 class=\"obj_head\"> Figure 7: Proteomic Features Related to Diagnosis, Prognosis, or Treatment. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0008.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F7/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P428\"> A. Differentially expressed proteins between tumors and NATs. </p> <p id=\"P429\"> B. Significantly increased proteins (larger font indicates &gt;4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. </p> <p id=\"P430\"> C. Genetic dependencies of 502 proteins (log2 FC &gt;2, FDR&lt;0.01 and NAs &lt;50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. </p> <p id=\"P431\"> D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. </p> <p id=\"P432\"> E. EGFR protein and tyrosine phosphorylation levels compared to <em> EGFR </em> copy number and an EGFR activity score (PROGENy). </p> <p id=\"P433\"> F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p&lt;0.05 </p> <p id=\"P434\"> G. GO Biological Process enrichment for proteins with increased phosphorylation in <em> EGFR </em> amplified samples compared to non-amplified samples. </p> <p id=\"P435\"> H. Summary roadmap figure partitioned into five major categories, indicated by different colors. </p> <p id=\"P436\"> See also <a class=\"usa-link\" href=\"#SD6\"> Figure S7 </a> and <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> </p> </figcaption> </figure> <p id=\"P35\"> There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell type of origin ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7D </a> ). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores ( <a class=\"usa-link\" href=\"#F7\"> Figure 7D </a> ). Immune-related deletions were less prominent in HNSCC and especially LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7E </a> ), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7F </a> ), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7G </a> ), although LUAD had many activating <em> EGFR </em> mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High <em> EGFR </em> amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy scoring ( <a class=\"usa-link\" href=\"#F7\"> Figure 7E </a> ) ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors ( <a class=\"usa-link\" href=\"#F7\"> Figure 7F </a> ). Furthermore, <em> EGFR </em> amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion ( <a class=\"usa-link\" href=\"#F7\"> Figure 7G </a> ). These findings and similar results recently reported in HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. <a class=\"usa-link\" href=\"#F7\"> Figure 7H </a> graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. </p> </section> </section> <section id=\"S12\"> <h2 class=\"pmc_sec_title\"> Discussion </h2> <p id=\"P36\"> In this study, we provide detailed proteogenomic profiles of treatment-naïve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. </p> <p id=\"P37\"> Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses ( <a aria-describedby=\"R57\" class=\"usa-link\" href=\"#R57\"> Haslam and Prasad, 2019 </a> ; <a aria-describedby=\"R59\" class=\"usa-link\" href=\"#R59\"> Herbst et al., 2018 </a> ); numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the downstream effects of <em> CDKN2A </em> mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. </p> <section id=\"S13\"> <h3 class=\"pmc_sec_title\"> Limitations of the study </h3> <p id=\"P38\"> Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity ( <a aria-describedby=\"R139\" class=\"usa-link\" href=\"#R139\"> Peng et al., 2020 </a> ; <a aria-describedby=\"R151\" class=\"usa-link\" href=\"#R151\"> Satpathy et al., 2020 </a> ), such as pairing microdissection with mass spectrometry ( <a aria-describedby=\"R99\" class=\"usa-link\" href=\"#R99\"> Le Large et al., 2020 </a> ) or using mass cytometry ( <a aria-describedby=\"R163\" class=\"usa-link\" href=\"#R163\"> Spitzer and Nolan, 2016 </a> ) will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the understanding and treatment of LSCC. </p> </section> </section> <section id=\"S14\"> <h2 class=\"pmc_sec_title\"> STAR★METHODS </h2> <section id=\"S16\"> <h3 class=\"pmc_sec_title\"> RESOURCE AVAILABILITY </h3> <section id=\"S17\"> <h4 class=\"pmc_sec_title\"> Lead contact </h4> <p id=\"P39\"> Further information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette ( <span> gillette@broadinstitute.org </span> ) </p> </section> <section id=\"S18\"> <h4 class=\"pmc_sec_title\"> Material availability </h4> <p id=\"P40\"> This study did not generate new unique reagents. </p> </section> <section id=\"S19\"> <h4 class=\"pmc_sec_title\"> Data and Code Availability </h4> <section id=\"S20\"> <h5 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h5> <ul class=\"list\" id=\"L2\"> <li> <p id=\"P41\"> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p id=\"P42\"> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"S21\"> <h5 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h5> <ul class=\"list\" id=\"L4\"> <li> <p id=\"P43\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p id=\"P44\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"S22\"> <h5 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h5> <ul class=\"list\" id=\"L6\"> <li> <p id=\"P45\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p id=\"P46\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p id=\"P47\"> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p id=\"P48\"> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p id=\"P49\"> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p id=\"P50\"> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"T1\"> <h6 class=\"obj_head\"> KEY RESOURCES TABLE. </h6> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/T1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> <section id=\"S23\"> <h3 class=\"pmc_sec_title\"> EXPERIMENTAL MODEL AND SUBJECT DETAILS </h3> <section id=\"S24\"> <h4 class=\"pmc_sec_title\"> Human Subjects </h4> <p id=\"P51\"> A total of 113 participants (90 males, 23 females, 40–88 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40–88. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC ( <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/ </a> ). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&amp;E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> </section> <section id=\"S25\"> <h3 class=\"pmc_sec_title\"> METHOD DETAILS </h3> <section id=\"S26\"> <h4 class=\"pmc_sec_title\"> Specimen Acquisition </h4> <p id=\"P52\"> The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than −140°C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. </p> </section> <section id=\"S27\"> <h4 class=\"pmc_sec_title\"> Sequencing sample preparation </h4> <p id=\"P53\"> Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol using Qiagen’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3–5mL) to serve as matched normal reference material. The Qubit™ dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification. </p> <section id=\"S28\"> <h5 class=\"pmc_sec_title\"> Whole exome Sequencing (WES) </h5> <section id=\"S29\"> <h6 class=\"pmc_sec_title\"> Library construction and Hybrid Selection </h6> <p id=\"P54\"> Library construction was performed as described in ( <a aria-describedby=\"R37\" class=\"usa-link\" href=\"#R37\"> Fisher et al., 2011 </a> ), with the following modifications: initial genomic DNA input into shearing was reduced from 3μg to 20–250ng in 50μL of solution. For adapter ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. </p> </section> <section id=\"S30\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P55\"> After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. </p> </section> </section> <section id=\"S31\"> <h5 class=\"pmc_sec_title\"> Whole genome sequencing (WGS) </h5> <section id=\"S32\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P56\"> An aliquot of genomic DNA (350ng in 50μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into 24-plexes. </p> <p id=\"P57\"> Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging. </p> </section> </section> </section> <section id=\"S33\"> <h4 class=\"pmc_sec_title\"> Array Based Methylation Analysis </h4> <p id=\"P58\"> The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt;850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. </p> </section> <section id=\"S34\"> <h4 class=\"pmc_sec_title\"> RNA and miRNA sequencing </h4> <section id=\"S35\"> <h5 class=\"pmc_sec_title\"> Quality Assurance and Control of RNA Analytes </h5> <p id=\"P59\"> All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt;7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer’s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. </p> </section> <section id=\"S36\"> <h5 class=\"pmc_sec_title\"> Total RNA Sequencing </h5> <p id=\"P60\"> Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. </p> </section> <section id=\"S37\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Library Construction </h5> <p id=\"P61\"> miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer’s instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). </p> </section> <section id=\"S38\"> <h5 class=\"pmc_sec_title\"> miRNA Sequencing </h5> <p id=\"P62\"> Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. </p> </section> </section> <section id=\"S39\"> <h4 class=\"pmc_sec_title\"> Mass Spectrometry methods </h4> <p id=\"P63\"> The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Acetyl-enrichment was performed as described before ( <a aria-describedby=\"R89\" class=\"usa-link\" href=\"#R89\"> Krug et al., 2020 </a> ) with modifications as indicated below. </p> <section id=\"S40\"> <h5 class=\"pmc_sec_title\"> Protein Extraction and Tryptic Digestion </h5> <p id=\"P64\"> Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 μg/mL aprotinin (Sigma, A6103), 10 μg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129–02541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. </p> </section> <section id=\"S41\"> <h5 class=\"pmc_sec_title\"> Common Reference (CR) Pool and Plex layout </h5> <section id=\"S42\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, and Acetyprotome: </h6> <p id=\"P65\"> The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. A common physical, rather than <em> in silico </em> reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data analyses. </p> <p id=\"P66\"> 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5 <sup> th </sup> tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. </p> <p id=\"P67\"> To ensure capacity for additional samples or experiments given a target input of 300 μg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 μg peptide / sample for individual sample analysis, an additional amount of 120 μg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 μg aliquots, dried down, and frozen at −80°C until use. </p> </section> <section id=\"S43\"> <h6 class=\"pmc_sec_title\"> Ubiquitylproteome: </h6> <p id=\"P68\"> 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: </p> <ol class=\"list\" id=\"L8\"> <li> <p id=\"P69\"> 104/147 samples included in ubiquitylproteome study and made full CR contribution, &gt;1000ug available. </p> </li> <li> <p id=\"P70\"> 42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug −1000 ug available. </p> </li> <li> <p id=\"P71\"> 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, &lt;750 ug available. </p> </li> </ol> <p id=\"P72\"> CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR’s proportion of 50:50), and aliquoted into 750 ug aliquots. </p> <p id=\"P73\"> In the ubiquitylproteome experimental design, 15 11-plex experiments contained <strong> 87 </strong> unique tumor samples in the C channels and <strong> 60 </strong> normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11 <sup> th </sup> channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). </p> </section> </section> <section id=\"S44\"> <h5 class=\"pmc_sec_title\"> TMT-11 Labeling of Peptides </h5> <section id=\"S45\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P74\"> 300 μg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a “reduced labeling” approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 μg peptide aliquot of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 μL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 μg aliquot from each sample and analyzing 0.25 μg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples were taken out, an additional 4 μg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/− 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample’s contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 × 300 μg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). </p> </section> </section> <section id=\"S46\"> <h5 class=\"pmc_sec_title\"> Peptide Fractionation </h5> <section id=\"S47\"> <h6 class=\"pmc_sec_title\"> For Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P75\"> To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was reconstituted in 935 μL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 μm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were made by 40–fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (&gt;95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701–5200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 μg/ μL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. </p> </section> </section> <section id=\"S48\"> <h5 class=\"pmc_sec_title\"> Phosphopeptide Enrichment </h5> <p id=\"P76\"> Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 μg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 μL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 μL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 μL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 μL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 × 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 μL 1% FA and phosphopeptides were eluted from the stage tips with 60 μL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 μL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 μL was injected per run. </p> </section> <section id=\"S49\"> <h5 class=\"pmc_sec_title\"> Acetylpeptide Enrichment </h5> <p id=\"P77\"> Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 μg peptides per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 × 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. </p> </section> <section id=\"S50\"> <h5 class=\"pmc_sec_title\"> Ubiquitylpeptide Enrichment with on-bead TMT labeling </h5> <p id=\"P78\"> Ubiquitin enrichment was performed based on the recently published UbiFast protocol ( <a aria-describedby=\"R178\" class=\"usa-link\" href=\"#R178\"> Udeshi et al., 2020 </a> ). Before enrichment, anti-K-ε-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4°C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4°C at a concentration of 0.5 ug/uL. </p> <p id=\"P79\"> For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4 <sup> o </sup> C for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30–40 s, left to settle on ice for 10–20 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20°C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. </p> </section> <section id=\"S51\"> <h5 class=\"pmc_sec_title\"> LC-MS/MS for Proteomics Analyses </h5> <p id=\"P80\"> Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross (360μm, IDEX Health &amp; Science, UH-906x). The column was heated to 50°C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 μm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360–75-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). </p> <p id=\"P81\"> For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). </p> <p id=\"P82\"> For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/− 10 ppm mass tolerance. “Perform dependent scan on single charge state per precursor only” was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. </p> </section> <section id=\"S52\"> <h5 class=\"pmc_sec_title\"> Immunohistochemistry (IHC) analysis. </h5> <section id=\"S53\"> <h6 class=\"pmc_sec_title\"> TMA construction </h6> <p id=\"P83\"> Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. </p> </section> <section id=\"S54\"> <h6 class=\"pmc_sec_title\"> Immune cell marker, Immunohistochemistry (IHC) </h6> <p id=\"P84\"> Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0–3 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the “ordinal” package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. </p> </section> <section id=\"S55\"> <h6 class=\"pmc_sec_title\"> Multiplex Immunofluorescence (mIF) staining </h6> <p id=\"P85\"> Development and optimization of the mIF platform has been previously described ( <a aria-describedby=\"R133\" class=\"usa-link\" href=\"#R133\"> Parra et al., 2020 </a> ). mIF staining was performed in a 4-μm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). </p> </section> <section id=\"S56\"> <h6 class=\"pmc_sec_title\"> IHC based assessment of TP63 low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> ) </h6> <p id=\"P86\"> For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66–2754) deemed LSCC by TCGA pathology. </p> </section> </section> </section> </section> <section id=\"S57\"> <h3 class=\"pmc_sec_title\"> QUANTIFICATION AND STATISTICAL ANALYSIS </h3> <section id=\"S58\"> <h4 class=\"pmc_sec_title\"> Genomic Data Analysis </h4> <section id=\"S59\"> <h5 class=\"pmc_sec_title\"> Copy Number Calling </h5> <p id=\"P87\"> Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> ). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3–0.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) ( <a aria-describedby=\"R47\" class=\"usa-link\" href=\"#R47\"> Gey and Lebarbier, 2008 </a> ), and dynamic programming ( <a aria-describedby=\"R13\" class=\"usa-link\" href=\"#R13\"> Bellman, 1961 </a> ), as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method ( <a aria-describedby=\"R98\" class=\"usa-link\" href=\"#R98\"> Lebarbier, 2005 </a> ). For the final set of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data. </p> </section> <section id=\"S60\"> <h5 class=\"pmc_sec_title\"> Somatic Variant Calling </h5> <p id=\"P88\"> Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels “GATK bundle known” and “Mills 1000G” somatic variants were called using Sentieon TNScope with the following settings “--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5”, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings “--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000”. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most ‘severe’ consequence occuring in a ‘reliable’ transcript was chosen. To accomplish this the computed consequences were ranked by ‘severity’ and ‘priority’ reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked ‘CANONICAL’ or ‘PICK’ by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD &gt; 5, NLOD &gt; 6, Fisher’s test p-value &lt; 0.05, NFLOD &gt; 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. </p> </section> <section id=\"S61\"> <h5 class=\"pmc_sec_title\"> Germline Variant Calling </h5> <p id=\"P89\"> Germline variant calling and annotation was carried out analogously using DNAScope using the following settings “--emit_conf 10 --call_conf 10” to generate a VCF file and “--emit_mode GVCF” to generate a gVCF file. The variants were custom-filtered based on the specific applications. </p> </section> <section id=\"S62\"> <h5 class=\"pmc_sec_title\"> GISTIC and MutSig analysis </h5> <p id=\"P90\"> The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value &lt;0.25 considered significant. The following parameters were used: </p> <ul class=\"list\" id=\"L10\"> <li> <p id=\"P91\"> Amplification Threshold = 0.1 </p> </li> <li> <p id=\"P92\"> Deletion Threshold = 0.1 </p> </li> <li> <p id=\"P93\"> Cap Values = 1.5 </p> </li> <li> <p id=\"P94\"> Broad Length Cutoff = 0.98 </p> </li> <li> <p id=\"P95\"> Remove X-Chromosome = 0 </p> </li> <li> <p id=\"P96\"> Confidence Level = 0.99 </p> </li> <li> <p id=\"P97\"> Join Segment Size = 4 </p> </li> <li> <p id=\"P98\"> Arm Level Peel Off = 1 </p> </li> <li> <p id=\"P99\"> Maximum Sample Segments = 2000 </p> </li> <li> <p id=\"P100\"> Gene GISTIC = 1 </p> </li> </ul> <p id=\"P101\"> Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from −2 to 2, where 0 means no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: −1 means deletion beyond the threshold; −2 means deletions greater than the minimum arm-level copy number observed in the sample. </p> <p id=\"P102\"> The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV ( <a aria-describedby=\"R95\" class=\"usa-link\" href=\"#R95\"> Lawrence et al., 2014 </a> ) was run on these filtered results to evaluate the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census ( <a aria-describedby=\"R161\" class=\"usa-link\" href=\"#R161\"> Sondka et al., 2018 </a> ), with false discovery rates (q values) recalculated. Genes of q value &lt; 0.1 were declared significant. </p> </section> </section> <section id=\"S63\"> <h4 class=\"pmc_sec_title\"> RNAseq and miRNAseq Quantification </h4> <section id=\"S64\"> <h5 class=\"pmc_sec_title\"> RNAseq Quantification </h5> <p id=\"P103\"> The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page ( <a class=\"usa-link usa-link--external\" href=\"https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files\" rel=\"noopener noreferrer\" target=\"_blank\"> https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files </a> ). The readcount is later converted to FPKM and FPKM-UQ using GDC’s formula ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/ </a> ). </p> </section> <section id=\"S65\"> <h5 class=\"pmc_sec_title\"> Isoform specific RNA quantification </h5> <p id=\"P104\"> For TP63 isoform centric analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> , the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression ( <a aria-describedby=\"R103\" class=\"usa-link\" href=\"#R103\"> Li et al., 2017 </a> ). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. </p> </section> <section id=\"S66\"> <h5 class=\"pmc_sec_title\"> DNA methylation data preprocessing </h5> <p id=\"P105\"> Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously ( <a aria-describedby=\"R39\" class=\"usa-link\" href=\"#R39\"> Fortin et al., 2014 </a> ). We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent analysis. </p> <p id=\"P106\"> Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5’ UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package ( <a aria-describedby=\"R40\" class=\"usa-link\" href=\"#R40\"> Fraley et al., 2016 </a> ) in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty &lt; 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. </p> </section> <section id=\"S67\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Data Analysis </h5> <p id=\"P107\"> miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov </a> ). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md </a> ). The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. </p> </section> </section> <section id=\"S68\"> <h4 class=\"pmc_sec_title\"> Proteomics Data Analysis </h4> <section id=\"S69\"> <h5 class=\"pmc_sec_title\"> Spectrum quality filtering and searching </h5> <p id=\"P108\"> All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory ( <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/proteomics\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/proteomics </a> ). Similar MS/MS spectra acquired on the same precursor m/z within +/− 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length &gt; 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800–6000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser ( <a class=\"usa-link usa-link--external\" href=\"https://genome.ucsc.edu/cgi-bin/hgTables\" rel=\"noopener noreferrer\" target=\"_blank\"> https://genome.ucsc.edu/cgi-bin/hgTables </a> ) on June 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/− 20 ppm mass tolerance for precursor and product ions, 30% minimum matched peak intensity, and “trypsin allow P” enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of −18 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −18 to 272 Da. For the acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −690 to 70 Da. </p> </section> <section id=\"S70\"> <h5 class=\"pmc_sec_title\"> Protein grouping, and localization of PTMs </h5> <p id=\"P109\"> Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. </p> <p id=\"P110\"> In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 – Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of &lt;0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5–6, thresholds were optimized to yield a PSM-level FDR estimate of &lt;0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher charge states. </p> <p id=\"P111\"> In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called “one-hit wonders.” After assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method “expand subgroups, top uses shared” to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide sequence of &gt;8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of &lt;0.6% at the peptide-spectrum match level and &lt;0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and &lt;0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. </p> <p id=\"P112\"> In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, &gt;1.1, essentially corresponded to at least 1 b or y ion located between two candidate sites that has a peak height &gt;10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, &gt;99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. </p> </section> <section id=\"S71\"> <h5 class=\"pmc_sec_title\"> Quantitation using TMT ratios </h5> <p id=\"P113\"> Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method “expand subgroups, top uses shared” (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method “unexpand subgroups”. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer’s Rule ( <a aria-describedby=\"R153\" class=\"usa-link\" href=\"#R153\"> Shadforth et al., 2005 </a> ) and correction factors obtained from the reagent manufacturer’s certificate of analysis ( <a class=\"usa-link usa-link--external\" href=\"https://www.thermofisher.com/order/catalog/product/90406\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.thermofisher.com/order/catalog/product/90406 </a> ) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. </p> </section> <section id=\"S72\"> <h5 class=\"pmc_sec_title\"> Two-component normalization of TMT ratios </h5> <p id=\"P114\"> It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians <em> N </em> ( <em> μi </em> 1, <em> σi </em> 1) and <em> N </em> ( <em> μi </em> 2, <em> σi </em> 2) for a sample <em> i </em> were fitted and used in the normalization process as follows: the mode <em> mi </em> of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians constrained to be <em> mi </em> , i.e., <em> μi </em> 1= <em> μi </em> 2= <em> mi </em> . The Gaussian with the smaller estimated standard deviation <span xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"> <math display=\"inline\" id=\"M1\"> <mrow> <mi> σ </mi> <mi> i </mi> <mo> = </mo> <mi> min </mi> <mo> ( </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 1 </mn> <mo> , </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 2 </mn> <mo> ) </mo> </mrow> </math> </span> was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using ( <em> mi </em> , <em> σi </em> ) by subtracting the mean <em> mi </em> from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation <em> σi </em> . </p> </section> <section id=\"S73\"> <h5 class=\"pmc_sec_title\"> Normalization of acetyl- and ubiquitylproteome </h5> <p id=\"P115\"> To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site’s main accession number did not yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. </p> </section> </section> <section id=\"S74\"> <h4 class=\"pmc_sec_title\"> Proteogenomic analysis </h4> <section id=\"S75\"> <h5 class=\"pmc_sec_title\"> Quality control and Batch effect assessment </h5> <p id=\"P116\"> We used guided principal component analysis (gPCA) ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. </p> <p id=\"P117\"> To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median pearson correlation &gt; 0.85. </p> <p id=\"P118\"> In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as common denominator ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1C </a> ). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value &lt; 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using <em> limma’s </em> removeBatchEffect ( <a class=\"usa-link usa-link--external\" href=\"https://rdrr.io/bioc/limma/man/removeBatchEffect.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rdrr.io/bioc/limma/man/removeBatchEffect.html </a> ) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. </p> </section> <section id=\"S76\"> <h5 class=\"pmc_sec_title\"> Dataset filtering </h5> <p id=\"P119\"> Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in &gt;70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at least two observed TMT ratios in &gt;25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio in &gt;25% of samples. </p> <p id=\"P120\"> Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. </p> </section> <section id=\"S77\"> <h5 class=\"pmc_sec_title\"> Unsupervised multi-omic clustering using NMF </h5> <p id=\"P121\"> We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Briefly, given a factorization rank <em> k </em> (where <em> k </em> is the number of clusters), NMF decomposes a <em> p x n </em> data matrix <em> V </em> into two matrices <em> W </em> and <em> H </em> such that multiplication of <em> W </em> and <em> H </em> approximates <em> V </em> . Matrix <em> H </em> is a <em> k x n </em> matrix whose entries represent weights for each sample (1 to <em> N </em> ) to contribute to each cluster (1 to <em> k </em> ), whereas matrix <em> W </em> is a <em> p x k </em> matrix representing weights for each feature (1 to <em> p </em> ) to contribute to each cluster (1 to <em> k </em> ). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em> H </em> . We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5. </p> <p id=\"P122\"> To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or −2) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by NMF as follows: </p> <ol class=\"list\" id=\"L12\"> <li> <p id=\"P123\"> Create one data matrix with all negative numbers zeroed. </p> </li> <li> <p id=\"P124\"> Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. </p> </li> <li> <p id=\"P125\"> Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. </p> </li> </ol> <p id=\"P126\"> The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ) and using the factorization method described in ( <a aria-describedby=\"R16\" class=\"usa-link\" href=\"#R16\"> Brunet et al., 2004 </a> ). To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters between <em> k </em> =2 and 8. For each k we factorized matrix <em> V </em> using 50 iterations with random initializations of <em> W </em> and <em> H </em> . To determine the optimal factorization rank we calculated two metrics for each <em> k </em> : 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix as defined in ( <a aria-describedby=\"R80\" class=\"usa-link\" href=\"#R80\"> Kim and Park, 2007 </a> ) measuring the reproducibility of the clustering across 50 iterations. The optimal <em> k </em> was defined as the maximum of the product of both metrics for cluster numbers between <em> k </em> =3 and 8 </p> <p id=\"P127\"> Having determined the optimal factorization rank <em> k </em> , in order to achieve robust factorization of the multi-omic data matrix <em> V </em> , we repeated the NMF analysis using 500 iterations with random initializations of <em> W </em> and <em> H </em> and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em> W </em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first normalized each row in matrix <em> W </em> by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix W <sub> signed </sub> contained signed cluster weights for each feature in the input matrix. </p> <p id=\"P128\"> For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights (W <sub> signed </sub> ) onto Hallmark pathway gene sets ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) using ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ). To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> using the following parameters: </p> <ul class=\"list\" id=\"L14\"> <li> <p id=\"P129\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P130\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P131\"> weight=1 </p> </li> <li> <p id=\"P132\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P133\"> output.score.type=“NES” </p> </li> <li> <p id=\"P134\"> nperm=1000 </p> </li> <li> <p id=\"P135\"> global.fdr=TRUE </p> </li> <li> <p id=\"P136\"> min.overlap=5 </p> </li> <li> <p id=\"P137\"> correl.type=”z.score” </p> </li> </ul> <p id=\"P138\"> To test the association of the resulting clusters to clinical variables we used Fisher’s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. </p> <p id=\"P139\"> The entire workflow described above has been implemented as a module for PANOPLY ( <a aria-describedby=\"R116\" class=\"usa-link\" href=\"#R116\"> Mani et al., 2020 </a> ) which runs on Broad’s Cloud platform Terra ( <a class=\"usa-link usa-link--external\" href=\"https://app.terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://app.terra.bio/ </a> ). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> . </p> </section> <section id=\"S78\"> <h5 class=\"pmc_sec_title\"> Integrative analysis with Stewart et al. </h5> <p id=\"P140\"> To enable integrative analysis of our protein data with the protein data in Stewart et al.( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. </p> </section> <section id=\"S79\"> <h5 class=\"pmc_sec_title\"> RNA subtyping </h5> <p id=\"P141\"> Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ) and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ). Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p &gt; 0.01) were marked to have ‘undecided’ subtype. </p> </section> <section id=\"S80\"> <h5 class=\"pmc_sec_title\"> Chromosomal instability index </h5> <p id=\"P142\"> The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. </p> </section> <section id=\"S81\"> <h5 class=\"pmc_sec_title\"> Fusion detection and analysis </h5> <p id=\"P143\"> Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline ( <a aria-describedby=\"R149\" class=\"usa-link\" href=\"#R149\"> Robinson et al., 2017 </a> ; <a aria-describedby=\"R199\" class=\"usa-link\" href=\"#R199\"> Wu et al., 2018 </a> ), fusioncatcher_v1.10 ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) and arriba_v1.1.0 ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> ). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described ( <a aria-describedby=\"R182\" class=\"usa-link\" href=\"#R182\"> Vats et al., 2020 </a> ). </p> </section> <section id=\"S82\"> <h5 class=\"pmc_sec_title\"> mRNA and Protein correlation </h5> <p id=\"P144\"> To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini–Hochberg procedure. </p> </section> <section id=\"S83\"> <h5 class=\"pmc_sec_title\"> CNA-driven cis and trans effects </h5> <p id=\"P145\"> Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR &lt; 0.05) positive or negative correlations. </p> </section> <section id=\"S84\"> <h5 class=\"pmc_sec_title\"> CMAP analysis </h5> <p id=\"P146\"> Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\" rel=\"noopener noreferrer\" target=\"_blank\"> GSE92742 </a> ). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See <a class=\"usa-link usa-link--external\" href=\"https://clue.io/GEO-guide\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/GEO-guide </a> for more information. </p> <p id=\"P147\"> To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in “CNA driven cis and trans effects” above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ). The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. </p> <p id=\"P148\"> For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were identified with Fisher’s test FDR &lt; 0.25, using this process. </p> <p id=\"P149\"> In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with rate λ, counting the number of identified candidate driver genes. Given the small n ( = 10) and λ, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with <em> both </em> CNA amplification and deletion. </p> <p id=\"P150\"> To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with α = 0.05, followed by counting the number of trans-genes in this list of regulated genes. </p> <p id=\"P151\"> To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. </p> </section> <section id=\"S85\"> <h5 class=\"pmc_sec_title\"> LINCS analysis </h5> <p id=\"P152\"> Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value &lt; 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. </p> <p id=\"P153\"> Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini &amp; Hochberg method. Subtype-specific differentially expressed proteins were identified based on their differential expression (adjusted p.value &lt; 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline ( <a class=\"usa-link usa-link--external\" href=\"https://hub.docker.com/u/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://hub.docker.com/u/cmap </a> ) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ). The corresponding compound metadata were obtained from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. </p> </section> <section id=\"S86\"> <h5 class=\"pmc_sec_title\"> Defining cancer-associated genes </h5> <p id=\"P154\"> Cancer-associated genes (CAG) were compiled using the Census website <a class=\"usa-link usa-link--external\" href=\"https://cancer.sanger.ac.uk/census\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cancer.sanger.ac.uk/census </a> . The list of genes is provided in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> <section id=\"S87\"> <h5 class=\"pmc_sec_title\"> CpG Island Methylator Phenotype </h5> <p id=\"P155\"> The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score &gt;0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value −0.041 and −0.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. </p> </section> <section id=\"S88\"> <h5 class=\"pmc_sec_title\"> iProFun Based Cis Association Analysis </h5> <p id=\"P156\"> We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR)&lt;10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities &gt; 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) &lt;10% by selecting a minimal cutoff value of alpha such that 75%&lt;alpha&lt;100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities &gt; alpha in permuted data) / (Averaged No. of genes with posterior probabilities &gt; alpha in original data). <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. </p> </section> <section id=\"S89\"> <h5 class=\"pmc_sec_title\"> EMT-specific cluster and fibroblast enrichment </h5> <p id=\"P157\"> NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). </p> </section> <section id=\"S90\"> <h5 class=\"pmc_sec_title\"> Differential marker analysis </h5> <p id=\"P158\"> A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having &lt; 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR ( <a aria-describedby=\"R106\" class=\"usa-link\" href=\"#R106\"> Liao et al., 2019 </a> ). The selected gene list was either significant proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted p-value &lt;0.01. </p> <p id=\"P159\"> Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> ), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. </p> </section> <section id=\"S91\"> <h5 class=\"pmc_sec_title\"> Overall and disease-free survival in TCGA lung squamous carcinoma </h5> <p id=\"P160\"> Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. </p> </section> <section id=\"S92\"> <h5 class=\"pmc_sec_title\"> Kaplan-Meier survival analysis </h5> <p id=\"P161\"> We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. </p> <p id=\"P162\"> We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. </p> </section> <section id=\"S93\"> <h5 class=\"pmc_sec_title\"> Continuous Smoking Score </h5> <p id=\"P163\"> Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC ( <a aria-describedby=\"R170\" class=\"usa-link\" href=\"#R170\"> Tate et al., 2019 </a> ) and cosine similarity was calculated to identify the best match. </p> <p id=\"P164\"> We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 &lt; S &lt; 1, to quantify the degree of confidence that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C ∩ T ∩ N, and used A and A’ to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A’ and D to the joint probability of A and D. For example, the latter can be written P(A)・P(C|A)・P(T|A)・P(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9 ( <a aria-describedby=\"R184\" class=\"usa-link\" href=\"#R184\"> Walser et al., 2008 </a> ). </p> <p id=\"P165\"> We consider <em> S </em> to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type <em> G </em> is P( <em> G </em> | <em> A </em> ) = [ <em> x </em> ・ <em> erf </em> ( <em> g/y </em> ) + 1] <em> / </em> ( <em> x </em> + 2) and P( <em> G </em> | <em> A </em> ’) = 1 - P( <em> G </em> | <em> A </em> ), where <em> erf </em> is the Gaussian error function, <em> g </em> is the observed value for the given data type, and <em> x </em> and <em> y </em> are empirically-determined weights. </p> <p id=\"P166\"> The shape function for total mutations, <em> T </em> , has two modifiers. First, it includes a simple expected-value correction for purity, <em> u </em> . Namely, assuming mutation-calling does not capture all mutations because of impurities, <em> t </em> is taken as the observed number of mutations divided by a purity shape function, <em> f </em> , where <em> f </em> ≤ 1. Although one might model <em> f </em> according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we use a line function, <em> f </em> = 0.2・ <em> u </em> + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce’s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score &gt; 0.6; samples with weak smoking evidence if smoking score &lt; 0.15. </p> </section> <section id=\"S94\"> <h5 class=\"pmc_sec_title\"> Identification of differentially regulated events in NRF2 mutant tumors. </h5> <p id=\"P167\"> For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value &lt;0.05 and logFC&gt;1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 ( <a aria-describedby=\"R115\" class=\"usa-link\" href=\"#R115\"> Malhotra et al., 2010 </a> ) and SINGH_NFE2L2_TARGETS (( <a aria-describedby=\"R159\" class=\"usa-link\" href=\"#R159\"> Singh et al., 2008 </a> ), BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ). Ten of the 54 genes were novel targets identified in this study. </p> </section> <section id=\"S95\"> <h5 class=\"pmc_sec_title\"> Immune cluster identification based on cell type composition </h5> <p id=\"P168\"> The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) based on 108 tumors and 94 NAT samples. <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> contains the final score computed by xCell of different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR &lt; 5%). This filtering resulted in 25 cell types annotated in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S96\"> <h5 class=\"pmc_sec_title\"> TCGA pan-cancer immune subtyping </h5> <p id=\"P169\"> Tumors were classified into the six distinct immune subtypes — wound healing, IFN-γ dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-β dominant — identified by a TCGA pan-cancer analysis presented in ( <a aria-describedby=\"R173\" class=\"usa-link\" href=\"#R173\"> Thorsson et al., 2018 </a> ). Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> ) ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) to assign an immune subtype to each of the tumors. </p> </section> <section id=\"S97\"> <h5 class=\"pmc_sec_title\"> Ranking tumors by inferred activity of IFN-γ pathway </h5> <p id=\"P170\"> We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above “top” and “bottom” probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the “top” and “bottom” probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher’s exact test, given protein’s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). The thermodynamic interpretation of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway’s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway’s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. </p> <p id=\"P171\"> In our analysis, we used a gene list of IFN-γ signaling pathway from ( <a aria-describedby=\"R1\" class=\"usa-link\" href=\"#R1\"> Abril-Rodriguez and Ribas, 2017 </a> ). We settled with this 15 genes signaling pathway because it has all necessary components – a cytokine mediator, receptors, signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. </p> </section> <section id=\"S98\"> <h5 class=\"pmc_sec_title\"> Estimation of Tumor Purity, Stromal and Immune Scores </h5> <p id=\"P172\"> Besides xCell, we utilized ESTIMATE ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) to infer immune and stromal scores based on gene expression data ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app ( <a class=\"usa-link usa-link--external\" href=\"http://cibersort.stanford.edu\" rel=\"noopener noreferrer\" target=\"_blank\"> cibersort.stanford.edu </a> ; ( <a aria-describedby=\"R127\" class=\"usa-link\" href=\"#R127\"> Newman et al., 2015 </a> )) in absolute mode. To infer tumor purity, TSNet was utilized ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S99\"> <h5 class=\"pmc_sec_title\"> Differentially Expressed Genes and Pathway Analysis </h5> <p id=\"P173\"> Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ) was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hänzelmann et al., 2013 </a> ). For this analysis, pathways from the Reactome ( <a aria-describedby=\"R33\" class=\"usa-link\" href=\"#R33\"> Fabregat et al., 2018 </a> ), KEGG ( <a aria-describedby=\"R75\" class=\"usa-link\" href=\"#R75\"> Kanehisa et al., 2017 </a> ) and Hallmark ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . </p> </section> <section id=\"S100\"> <h5 class=\"pmc_sec_title\"> Deriving RTK CBPE scores </h5> <p id=\"P174\"> A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ). Each association computation was carried out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. </p> </section> <section id=\"S101\"> <h5 class=\"pmc_sec_title\"> Independent component analysis </h5> <p id=\"P175\"> ICA was performed with a workflow modified from previously described ( <a aria-describedby=\"R111\" class=\"usa-link\" href=\"#R111\"> Liu et al., 2019 </a> ). Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm ( <a aria-describedby=\"R117\" class=\"usa-link\" href=\"#R117\"> McInnes et al., 2017 </a> ) with cosine distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 <em> P </em> -values. Significance was controlled for multiple testing at 0.01 level (log10 ( <em> p-value) </em> =−5.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. </p> </section> <section id=\"S102\"> <h5 class=\"pmc_sec_title\"> Mutation-based cis- and trans-effects </h5> <p id=\"P176\"> We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ) on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR &lt;0.05 cut-off were separated into two categories based on cis- and trans- effects. </p> </section> <section id=\"S103\"> <h5 class=\"pmc_sec_title\"> Germline quantitative trait loci (QTL) analysis </h5> <p id=\"P177\"> To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen </a> ). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count ≥ 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9 ( <a aria-describedby=\"R27\" class=\"usa-link\" href=\"#R27\"> Deelen et al., 2014 </a> ). To adjust for population stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07 ( <a aria-describedby=\"R142\" class=\"usa-link\" href=\"#R142\"> Purcell et al., 2007 </a> ). For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE &gt; 0.0001, MAF &gt; 0.01, and call rate &gt; 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at ≤ FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis ( <a aria-describedby=\"R86\" class=\"usa-link\" href=\"#R86\"> Krämer et al., 2014 </a> ) </p> </section> <section id=\"S104\"> <h5 class=\"pmc_sec_title\"> miRNA analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </h5> <p id=\"P178\"> Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained ( <a aria-describedby=\"R66\" class=\"usa-link\" href=\"#R66\"> Huang et al., 2020 </a> ). Spearman correlation was used to calculate the correlations between miRNA and its target genes. </p> </section> <section id=\"S105\"> <h5 class=\"pmc_sec_title\"> Pathway projection using ssGSEA </h5> <p id=\"P179\"> The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: </p> <ul class=\"list\" id=\"L16\"> <li> <p id=\"P180\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P181\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P182\"> weight=0.75 </p> </li> <li> <p id=\"P183\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P184\"> output.score.type=“NES” </p> </li> <li> <p id=\"P185\"> nperm=1000 </p> </li> <li> <p id=\"P186\"> global.fdr=TRUE </p> </li> <li> <p id=\"P187\"> min.overlap=10 </p> </li> <li> <p id=\"P188\"> correl.type=”z.score” </p> </li> </ul> </section> <section id=\"S106\"> <h5 class=\"pmc_sec_title\"> Phosphorylation-driven signature analysis </h5> <p id=\"P189\"> We performed phosphosite-specific signature enrichment analysis (PTM-SEA) ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed −log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) </p> <table class=\"disp-formula p\" id=\"FD1\"> <tr> <td class=\"formula\"> <math display=\"block\" id=\"M2\"> <mrow> <mtext> logPsite </mtext> <mspace width=\"0.25em\"> </mspace> <mo> = </mo> <mspace width=\"0.25em\"> </mspace> <mo> − </mo> <mtext> log </mtext> <mn> 10 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> p </mtext> <mo> − </mo> <mtext> valuesite </mtext> </mrow> <mo> ) </mo> </mrow> <mo> * </mo> <mtext> sign </mtext> <mrow> <mo> ( </mo> <mrow> <mtext> log </mtext> <mn> 2 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> fold </mtext> <mspace width=\"0.25em\"> </mspace> <mtext> changesite </mtext> </mrow> <mo> ) </mo> </mrow> </mrow> <mo> ) </mo> </mrow> </mrow> </math> </td> </tr> </table> <p> PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. </p> <p id=\"P190\"> We employed the heuristic method introduced by Krug et al. ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. </p> <p id=\"P191\"> We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb </a> using the flanking amino acid sequence (+/− 7 aa) as primary identifier. We used the implementation of PTM-SEA available on GitHub ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: </p> <p id=\"P192\"> weight: 1 </p> <p id=\"P193\"> statistic: “area.under.RES” </p> <p id=\"P194\"> output.score.type: “NES” </p> <p id=\"P195\"> nperm: 1000 </p> <p id=\"P196\"> min.overlap: 5 </p> <p id=\"P197\"> correl.type: “Z-score” </p> <p id=\"P198\"> The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini &amp; and Hochberg ( <a aria-describedby=\"R14\" class=\"usa-link\" href=\"#R14\"> Benjamini and Hochberg, 1995 </a> ). Signatures with FDR-corrected p-values &lt; 0.05 were considered to be differential between tumor and NAT. </p> </section> <section id=\"S107\"> <h5 class=\"pmc_sec_title\"> <em> CDKN2A </em> and <em> RB1 </em> annotations and pathway analysis </h5> <p id=\"P199\"> Comprehensive tumor annotation for <em> CDKN2A </em> and <em> RB1 </em> genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values &gt;= 0.2. Multi-gene proliferation scores were calculated as described previously ( <a aria-describedby=\"R32\" class=\"usa-link\" href=\"#R32\"> Ellis et al., 2017 </a> ) as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al ( <a aria-describedby=\"R194\" class=\"usa-link\" href=\"#R194\"> Whitfield et al., 2002 </a> ). p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_000068.1\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_000068.1 </a> for p16INK4a and Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_478102.2\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_478102.2 </a> for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ). Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. </p> </section> <section id=\"S108\"> <h5 class=\"pmc_sec_title\"> Association analysis between KGG-site abundances and E3 ligases and DUBs </h5> <p id=\"P200\"> A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from ( <a aria-describedby=\"R119\" class=\"usa-link\" href=\"#R119\"> Medvar et al., 2016 </a> ; <a aria-describedby=\"R129\" class=\"usa-link\" href=\"#R129\"> Nijman et al., 2005 </a> ). We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR &lt; 0.01). </p> </section> <section id=\"S109\"> <h5 class=\"pmc_sec_title\"> Cluster and pathway analysis of significantly modulated K-GG sites in tumors. </h5> <p id=\"P201\"> Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. </p> <p id=\"P202\"> Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere et al., 2019 </a> ). Pathway enrichment was performed using a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon et al., 2003 </a> ) using the Enrichment Map app ( <a aria-describedby=\"R120\" class=\"usa-link\" href=\"#R120\"> Merico et al., 2010 </a> ) for network analysis of pathways. Pathways were connected using the gene overlap and clustered (pathway cutoff q-val &lt; 0.1; jaccard overlap &gt; 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. </p> </section> <section id=\"S110\"> <h5 class=\"pmc_sec_title\"> PTM CLUMPS analysis </h5> <p id=\"P203\"> We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual “events” as done with mutations in CLUMPS v1 ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ). For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and Tumor samples ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ). Sites are selected with an FDR &lt; 0.1 (Benjamini-Hochberg) and LFC &gt; 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. </p> </section> <section id=\"S111\"> <h5 class=\"pmc_sec_title\"> DepMap genetic dependency and drug response analysis </h5> <p id=\"P204\"> Cell lines annotated as “NSCLC_squamous” from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. </p> <p id=\"P205\"> To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study’s LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from DepMap. </p> <p id=\"P206\"> <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> : For identifying if top protein biomarkers (502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. </p> <p id=\"P207\"> <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> : To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. </p> <p id=\"P208\"> Continuous log2 copy number data from DepMap was used in <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> to correlate <em> SOX2 </em> copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). </p> <p id=\"P209\"> Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). </p> </section> <section id=\"S112\"> <h5 class=\"pmc_sec_title\"> CausalPath analysis </h5> <p id=\"P210\"> CausalPath ( <a aria-describedby=\"R8\" class=\"usa-link\" href=\"#R8\"> Babur et al., 2018 </a> ) searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). </p> </section> <section id=\"S113\"> <h5 class=\"pmc_sec_title\"> Variant Peptide Identification </h5> <p id=\"P211\"> We used NeoFlow ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> ) for neoantigen prediction ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ). Specifically, Optitype ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) to predict HLA peptide binding affinity for somatic mutation–derived variant peptides with a length between 8–11 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). To identify variant peptides, we used a customized protein sequence database approach ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ). We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> ) for customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using PepQuery ( <a class=\"usa-link usa-link--external\" href=\"http://www.pepquery.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.pepquery.org </a> ) ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) with the p-value cutoff &lt;= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV ( <a class=\"usa-link usa-link--external\" href=\"http://www.zhang-lab.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.zhang-lab.org/ </a> ) ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ). </p> </section> <section id=\"S114\"> <h5 class=\"pmc_sec_title\"> Cancer/testis Antigen Prediction </h5> <p id=\"P212\"> Cancer/testis (CT) antigens were downloaded from the CTdatabase ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ). CT antigens with a median &gt;4-fold increase in tumor from NAT were highlighted. </p> </section> <section id=\"S115\"> <h5 class=\"pmc_sec_title\"> PROGENy Scores </h5> <p id=\"P213\"> PROGENy ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ) was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy. </p> </section> <section id=\"S116\"> <h5 class=\"pmc_sec_title\"> LSCC, HNSCC and LUAD integrative analysis </h5> <p id=\"P214\"> LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ). The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in the <a class=\"usa-link\" href=\"#S90\"> Differential marker analysis </a> method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. </p> <p id=\"P215\"> Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value &lt; .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value &lt;0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p value &lt;0.01). </p> <p id=\"P216\"> To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method <a class=\"usa-link\" href=\"#S98\"> Estimation of Tumor Purity, Stromal and Immune Scores </a> . To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p value &lt; 0.01 were considered significant. </p> </section> </section> </section> </section> <section id=\"SM1\"> <h2 class=\"pmc_sec_title\"> Supplementary Material </h2> <section class=\"sm xbox font-sm\" id=\"SD1\"> <div class=\"caption p\"> <span> 1 </span> <p id=\"P217\"> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p id=\"P218\"> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p id=\"P219\"> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p id=\"P220\"> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p id=\"P221\"> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p id=\"P222\"> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p id=\"P223\"> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p id=\"P224\"> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p id=\"P225\"> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\" id=\"d40e3501\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD2\"> <div class=\"caption p\"> <span> 2 </span> <p id=\"P226\"> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p id=\"P227\"> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p id=\"P228\"> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p id=\"P229\"> C. Same as B but in the EMT-enriched subtype. </p> <p id=\"P230\"> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p id=\"P231\"> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p id=\"P232\"> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P233\"> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p id=\"P234\"> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p id=\"P235\"> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P236\"> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p id=\"P237\"> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\" id=\"d40e3559\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD3\"> <div class=\"caption p\"> <span> 3 </span> <p id=\"P238\"> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p id=\"P239\"> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p id=\"P240\"> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p id=\"P241\"> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p id=\"P242\"> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p id=\"P243\"> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p id=\"P244\"> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p id=\"P245\"> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p id=\"P246\"> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\" id=\"d40e3627\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD4\"> <div class=\"caption p\"> <span> 4 </span> <p id=\"P247\"> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p id=\"P248\"> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p id=\"P249\"> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p id=\"P250\"> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p id=\"P251\"> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p id=\"P252\"> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p id=\"P253\"> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p id=\"P254\"> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p id=\"P255\"> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p id=\"P256\"> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p id=\"P257\"> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p id=\"P258\"> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p id=\"P259\"> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p id=\"P260\"> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\" id=\"d40e3671\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD5\"> <div class=\"caption p\"> <span> 6 </span> <p id=\"P261\"> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P262\"> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p id=\"P263\"> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p id=\"P264\"> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p id=\"P265\"> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p id=\"P266\"> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p id=\"P267\"> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P268\"> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\" id=\"d40e3727\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD6\"> <div class=\"caption p\"> <span> 7 </span> <p id=\"P269\"> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P270\"> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p id=\"P271\"> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p id=\"P272\"> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p id=\"P273\"> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p id=\"P274\"> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p id=\"P275\"> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p id=\"P276\"> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\" id=\"d40e3755\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD7\"> <div class=\"caption p\"> <span> 8 </span> <p id=\"P277\"> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P278\"> A. Metadata associated with this study </p> <p id=\"P279\"> B. Cancer Associated Genes (CAGs) </p> <p id=\"P280\"> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p id=\"P281\"> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p id=\"P282\"> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\" id=\"d40e3782\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD8\"> <div class=\"caption p\"> <span> 9 </span> <p id=\"P283\"> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P284\"> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p id=\"P285\"> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p id=\"P286\"> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p id=\"P287\"> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p id=\"P288\"> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p id=\"P289\"> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p id=\"P290\"> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p id=\"P291\"> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p id=\"P292\"> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\" id=\"d40e3829\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD9\"> <div class=\"caption p\"> <span> 10 </span> <p id=\"P293\"> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P294\"> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P295\"> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p id=\"P296\"> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p id=\"P297\"> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P298\"> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P299\"> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P300\"> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P301\"> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P302\"> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P303\"> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P304\"> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P305\"> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P306\"> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P307\"> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\" id=\"d40e3937\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD10\"> <div class=\"caption p\"> <span> 11 </span> <p id=\"P308\"> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p id=\"P309\"> A. Copy number amplified cytobands </p> <p id=\"P310\"> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p id=\"P311\"> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p id=\"P312\"> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p id=\"P313\"> F. CMAP tab- Connectivity Map (CMAP) results </p> <p id=\"P314\"> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p id=\"P315\"> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p id=\"P316\"> I. Summary of CDKN2A and RB1 annotations </p> <p id=\"P317\"> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p id=\"P318\"> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p id=\"P319\"> L. NRF2 signature </p> <p id=\"P320\"> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\" id=\"d40e4003\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD11\"> <div class=\"caption p\"> <span> 12 </span> <p id=\"P321\"> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P322\"> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p id=\"P323\"> B. Ub-sites that show differential abundance across Ub clusters </p> <p id=\"P324\"> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p id=\"P325\"> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p id=\"P326\"> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P327\"> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P328\"> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p id=\"P329\"> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P330\"> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P331\"> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\" id=\"d40e4032\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD12\"> <div class=\"caption p\"> <span> 13 </span> <p id=\"P332\"> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P333\"> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p id=\"P334\"> B. Immune clusters based on xCell signatures </p> <p id=\"P335\"> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p id=\"P336\"> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p id=\"P337\"> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p id=\"P338\"> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p id=\"P339\"> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p id=\"P340\"> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p id=\"P341\"> I. List of genes/proteins whose expression is associated with a SNP </p> <p id=\"P342\"> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p id=\"P343\"> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\" id=\"d40e4072\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD13\"> <div class=\"caption p\"> <span> 14 </span> <p id=\"P344\"> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P345\"> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p id=\"P346\"> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p id=\"P347\"> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p id=\"P348\"> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p id=\"P349\"> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p id=\"P350\"> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\" id=\"d40e4094\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD14\"> <div class=\"caption p\"> <span> Figure S5 </span> <p id=\"P351\"> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P352\"> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p id=\"P353\"> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p id=\"P354\"> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p id=\"P355\"> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p id=\"P356\"> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p id=\"P357\"> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p id=\"P358\"> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p id=\"P359\"> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p id=\"P360\"> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p id=\"P361\"> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p id=\"P362\"> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\" id=\"d40e4134\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"floats-group\" id=\"floats-group1\"> <section class=\"bt xbox font-sm\" id=\"BX1\"> <h2 class=\"obj_head\"> Highlights. </h2> <ul class=\"list\" id=\"L18\"> <li> <p id=\"P437\"> Unsupervised clustering revealed subtype with EMT and phosphoprotein signatures </p> </li> <li> <p id=\"P438\"> Potential therapeutic vulnerabilities included survivin, NSD3, LSD1 and EZH2 </p> </li> <li> <p id=\"P439\"> Rb phosphorylation nominated as a biomarker for trials with CDK4/6 inhibitors </p> </li> <li> <p id=\"P440\"> Detailed immune landscape analysis highlighted targetable points of immune regulation </p> </li> </ul> </section> </section> <section class=\"ack\" id=\"S117\"> <h2 class=\"pmc_sec_title\"> Acknowledgements </h2> <p id=\"P363\"> This work was supported by grants U24CA210955, U24CA210985, U24CA210986, U24CA210954, U24CA210967, U24CA210972, U24CA210979, U24CA210993, U01CA214114, U01CA214116, and U01CA214125 from the NCI-CPTAC, U54HL127624, P30CA240139, 1F32HL154711-01, and T32CA203690 from the NIH. We thank Dr. Lata Adnani for helping with graphics. </p> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN3\"> <p id=\"P364\"> Declaration of Interests </p> <p id=\"P365\"> The authors declare no competing interests </p> </div> <div class=\"fn p\" id=\"FN4\"> <p id=\"P366\"> ADDITIONAL RESOURCES </p> <p id=\"P367\"> The CPTAC program website, detailing program initiatives, investigators, and datasets, is found at <a class=\"usa-link usa-link--external\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.cancer.gov/programs/cptac </a> . </p> <p id=\"P368\"> A website for interactive visualization of the multi-omic dataset is available at: <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </p> <p id=\"P369\"> All processed data matrices are available at LinkedOmics ( <a aria-describedby=\"R181\" class=\"usa-link\" href=\"#R181\"> Vasaikar et al., 2018 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"http://www.linkedomics.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.linkedomics.org </a> ), where computational tools are available for further exploration of this dataset. </p> <p id=\"P370\"> Consortia </p> <p id=\"P371\"> Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Fenyö, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Małgorzata Wojtyś, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Özgün Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. </p> </div> <div class=\"fn p\" id=\"FN5\"> <p id=\"P372\"> <strong> Publisher's Disclaimer: </strong> This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. </p> </div> </div> </section> <section class=\"ref-list\" id=\"ref-list1\"> <h2 class=\"pmc_sec_title\"> REFERENCES: </h2> <section id=\"ref-list1_sec2\"> <ol class=\"ref-list font-sm\"> <li id=\"R1\"> <cite> Abril-Rodriguez G, and Ribas A (2017). SnapShot: Immune Checkpoint Inhibitors. Cancer Cell 31, 848–848.e1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2017.05.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28609660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=SnapShot:%20Immune%20Checkpoint%20Inhibitors&amp;author=G%20Abril-Rodriguez&amp;author=A%20Ribas&amp;volume=31&amp;publication_year=2017&amp;pages=848-848.e1&amp;pmid=28609660&amp;doi=10.1016/j.ccell.2017.05.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R2\"> <cite> Ahn ER, Mangat PK, Garrett-Mayer E, Halabi S, Dib EG, Haggstrom DE, Alguire KB, Calfa CJ, Cannon TL, Crilley PA, et al. (2020). Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study. JCO Precision Oncology 757–766. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/PO.20.00037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/35050752/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCO%20Precision%20Oncology&amp;title=Palbociclib%20in%20Patients%20With%20Non%E2%80%93Small-Cell%20Lung%20Cancer%20With%20CDKN2A%20Alterations:%20Results%20From%20the%20Targeted%20Agent%20and%20Profiling%20Utilization%20Registry%20Study&amp;author=ER%20Ahn&amp;author=PK%20Mangat&amp;author=E%20Garrett-Mayer&amp;author=S%20Halabi&amp;author=EG%20Dib&amp;publication_year=2020&amp;pages=757-766&amp;pmid=35050752&amp;doi=10.1200/PO.20.00037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R3\"> <cite> Albert M, Bécares M, Falqui M, Fernández-Lozano C, and Guerra S (2018). ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis. Viruses 10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/v10110629\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6265978/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30428561/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=ISG15,%20a%20Small%20Molecule%20with%20Huge%20Implications:%20Regulation%20of%20Mitochondrial%20Homeostasis&amp;author=M%20Albert&amp;author=M%20B%C3%A9cares&amp;author=M%20Falqui&amp;author=C%20Fern%C3%A1ndez-Lozano&amp;author=S%20Guerra&amp;volume=10&amp;publication_year=2018&amp;pmid=30428561&amp;doi=10.3390/v10110629&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R4\"> <cite> Alcalá S, Sancho P, Martinelli P, Navarro D, Pedrero C, Martín-Hijano L, Valle S, Earl J, Rodríguez-Serrano M, Ruiz-Cañas L, et al. (2020). ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity. Nat. Commun. 11, 2682. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-16395-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7260233/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32472071/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=ISG15%20and%20ISGylation%20is%20required%20for%20pancreatic%20cancer%20stem%20cell%20mitophagy%20and%20metabolic%20plasticity&amp;author=S%20Alcal%C3%A1&amp;author=P%20Sancho&amp;author=P%20Martinelli&amp;author=D%20Navarro&amp;author=C%20Pedrero&amp;volume=11&amp;publication_year=2020&amp;pages=2682&amp;pmid=32472071&amp;doi=10.1038/s41467-020-16395-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R5\"> <cite> Almeida LG, Sakabe NJ, deOliveira AR, Silva MCC, Mundstein AS, Cohen T, Chen Y-T, Chua R, Gurung S, Gnjatic S, et al. (2009). CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens. Nucleic Acids Res. 37, D816–D819. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkn673\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2686577/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18838390/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CTdatabase:%20a%20knowledge-base%20of%20high-throughput%20and%20curated%20data%20on%20cancer-testis%20antigens&amp;author=LG%20Almeida&amp;author=NJ%20Sakabe&amp;author=AR%20deOliveira&amp;author=MCC%20Silva&amp;author=AS%20Mundstein&amp;volume=37&amp;publication_year=2009&amp;pages=D816-D819&amp;pmid=18838390&amp;doi=10.1093/nar/gkn673&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R6\"> <cite> Aran D, Hu Z, and Butte AJ (2017). xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 18, 220. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-017-1349-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5688663/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29141660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=xCell:%20digitally%20portraying%20the%20tissue%20cellular%20heterogeneity%20landscape&amp;author=D%20Aran&amp;author=Z%20Hu&amp;author=AJ%20Butte&amp;volume=18&amp;publication_year=2017&amp;pages=220&amp;pmid=29141660&amp;doi=10.1186/s13059-017-1349-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R7\"> <cite> Ardini-Poleske ME, Clark RF, Ansong C, Carson JP, Corley RA, Deutsch GH, Hagood JS, Kaminski N, Mariani TJ, Potter SS, et al. (2017). LungMAP: The Molecular Atlas of Lung Development Program. Am. J. Physiol. Lung Cell. Mol. Physiol. 313, L733–L740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/ajplung.00139.2017\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5792185/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28798251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Lung%20Cell.%20Mol.%20Physiol&amp;title=LungMAP:%20The%20Molecular%20Atlas%20of%20Lung%20Development%20Program&amp;author=ME%20Ardini-Poleske&amp;author=RF%20Clark&amp;author=C%20Ansong&amp;author=JP%20Carson&amp;author=RA%20Corley&amp;volume=313&amp;publication_year=2017&amp;pages=L733-L740&amp;pmid=28798251&amp;doi=10.1152/ajplung.00139.2017&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R8\"> <cite> Babur Ö, Luna A, Korkut A, Durupinar F, Siper MC, Dogrusoz U, Aslan JE, Sander C, and Demir E (2018). Causal interactions from proteomic profiles: molecular data meets pathway knowledge. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.patter.2021.100257\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8212145/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34179843/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Causal%20interactions%20from%20proteomic%20profiles:%20molecular%20data%20meets%20pathway%20knowledge&amp;author=%C3%96%20Babur&amp;author=A%20Luna&amp;author=A%20Korkut&amp;author=F%20Durupinar&amp;author=MC%20Siper&amp;publication_year=2018&amp;pmid=34179843&amp;doi=10.1016/j.patter.2021.100257&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R9\"> <cite> Baeza J, Smallegan MJ, and Denu JM (2015). Site-specific reactivity of nonenzymatic lysine acetylation. ACS Chem. Biol. 10, 122–128. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/cb500848p\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4301072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25555129/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol&amp;title=Site-specific%20reactivity%20of%20nonenzymatic%20lysine%20acetylation&amp;author=J%20Baeza&amp;author=MJ%20Smallegan&amp;author=JM%20Denu&amp;volume=10&amp;publication_year=2015&amp;pages=122-128&amp;pmid=25555129&amp;doi=10.1021/cb500848p&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R10\"> <cite> Bailey MH, Tokheim C, Porta-Pardo E, Sengupta S, Bertrand D, Weerasinghe A, Colaprico A, Wendl MC, Kim J, Reardon B, et al. (2018). Comprehensive Characterization of Cancer Driver Genes and Mutations. Cell 173, 371–385.e18. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.02.060\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6029450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29625053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Characterization%20of%20Cancer%20Driver%20Genes%20and%20Mutations&amp;author=MH%20Bailey&amp;author=C%20Tokheim&amp;author=E%20Porta-Pardo&amp;author=S%20Sengupta&amp;author=D%20Bertrand&amp;volume=173&amp;publication_year=2018&amp;pages=371-385.e18&amp;pmid=29625053&amp;doi=10.1016/j.cell.2018.02.060&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R11\"> <cite> Bando H, Atsumi T, Nishio T, Niwa H, Mishima S, Shimizu C, Yoshioka N, Bucala R, and Koike T (2005). Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer. Clin. Cancer Res. 11, 5784–5792. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-05-0149\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16115917/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Phosphorylation%20of%20the%206-phosphofructo-2-kinase/fructose%202,6-bisphosphatase/PFKFB3%20family%20of%20glycolytic%20regulators%20in%20human%20cancer&amp;author=H%20Bando&amp;author=T%20Atsumi&amp;author=T%20Nishio&amp;author=H%20Niwa&amp;author=S%20Mishima&amp;volume=11&amp;publication_year=2005&amp;pages=5784-5792&amp;pmid=16115917&amp;doi=10.1158/1078-0432.CCR-05-0149&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R12\"> <cite> Barbie DA, Tamayo P, Boehm JS, Kim SY, Moody SE, Dunn IF, Schinzel AC, Sandy P, Meylan E, Scholl C, et al. (2009). Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108–112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature08460\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2783335/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19847166/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;author=DA%20Barbie&amp;author=P%20Tamayo&amp;author=JS%20Boehm&amp;author=SY%20Kim&amp;author=SE%20Moody&amp;volume=462&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19847166&amp;doi=10.1038/nature08460&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R13\"> <cite> Bellman R (1961). On the approximation of curves by line segments using dynamic programming. Commun. ACM 4, 284. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun.%20ACM&amp;title=On%20the%20approximation%20of%20curves%20by%20line%20segments%20using%20dynamic%20programming&amp;author=R%20Bellman&amp;volume=4&amp;publication_year=1961&amp;pages=284&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R14\"> <cite> Benjamini Y, and Hochberg Y (1995). Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat. Soc. Series B Stat. Methodol. 57, 289–300. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol&amp;title=Controlling%20the%20False%20Discovery%20Rate:%20A%20Practical%20and%20Powerful%20Approach%20to%20Multiple%20Testing&amp;author=Y%20Benjamini&amp;author=Y%20Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R15\"> <cite> Bray F, Ferlay J, Soerjomataram I, Siegel RL, Torre LA, and Jemal A (2018). Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 68, 394–424. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30207593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&amp;author=F%20Bray&amp;author=J%20Ferlay&amp;author=I%20Soerjomataram&amp;author=RL%20Siegel&amp;author=LA%20Torre&amp;volume=68&amp;publication_year=2018&amp;pages=394-424&amp;pmid=30207593&amp;doi=10.3322/caac.21492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R16\"> <cite> Brunet J-P, Tamayo P, Golub TR, and Mesirov JP (2004). Metagenes and molecular pattern discovery using matrix factorization. Proc. Natl. Acad. Sci. U. S. A. 101, 4164–4169. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0308531101\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC384712/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/15016911/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Metagenes%20and%20molecular%20pattern%20discovery%20using%20matrix%20factorization&amp;author=J-P%20Brunet&amp;author=P%20Tamayo&amp;author=TR%20Golub&amp;author=JP%20Mesirov&amp;volume=101&amp;publication_year=2004&amp;pages=4164-4169&amp;pmid=15016911&amp;doi=10.1073/pnas.0308531101&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R17\"> <cite> Cadenas C, Franckenstein D, Schmidt M, Gehrmann M, Hermes M, Geppert B, Schormann W, Maccoux LJ, Schug M, Schumann A, et al. (2010). Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer. Breast Cancer Res. 12, R44. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/bcr2599\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2917039/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20584310/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&amp;title=Role%20of%20thioredoxin%20reductase%201%20and%20thioredoxin%20interacting%20protein%20in%20prognosis%20of%20breast%20cancer&amp;author=C%20Cadenas&amp;author=D%20Franckenstein&amp;author=M%20Schmidt&amp;author=M%20Gehrmann&amp;author=M%20Hermes&amp;volume=12&amp;publication_year=2010&amp;pages=R44&amp;pmid=20584310&amp;doi=10.1186/bcr2599&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R18\"> <cite> Cai J, Fang L, Huang Y, Li R, Yuan J, Yang Y, Zhu X, Chen B, Wu J, and Li M (2013). miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer. Cancer Res. 73, 5402–5415. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-13-0297\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23856247/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=miR-205%20targets%20PTEN%20and%20PHLPP2%20to%20augment%20AKT%20signaling%20and%20drive%20malignant%20phenotypes%20in%20non-small%20cell%20lung%20cancer&amp;author=J%20Cai&amp;author=L%20Fang&amp;author=Y%20Huang&amp;author=R%20Li&amp;author=J%20Yuan&amp;volume=73&amp;publication_year=2013&amp;pages=5402-5415&amp;pmid=23856247&amp;doi=10.1158/0008-5472.CAN-13-0297&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R19\"> <cite> Cammer M, Gevrey J-C, Lorenz M, Dovas A, Condeelis J, and Cox D (2009). The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42. J. Biol. Chem. 284, 23302–23311. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M109.036384\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2749104/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561083/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20mechanism%20of%20CSF-1-induced%20Wiskott-Aldrich%20syndrome%20protein%20activation%20in%20vivo:%20a%20role%20for%20phosphatidylinositol%203-kinase%20and%20Cdc42&amp;author=M%20Cammer&amp;author=J-C%20Gevrey&amp;author=M%20Lorenz&amp;author=A%20Dovas&amp;author=J%20Condeelis&amp;volume=284&amp;publication_year=2009&amp;pages=23302-23311&amp;pmid=19561083&amp;doi=10.1074/jbc.M109.036384&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R20\"> <cite> Campbell JD, Yau C, Bowlby R, Liu Y, Brennan K, Fan H, Taylor AM, Wang C, Walter V, Akbani R, et al. (2018). Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas. Cell Rep. 23, 194–212.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2018.03.063\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6002769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29617660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Genomic,%20Pathway%20Network,%20and%20Immunologic%20Features%20Distinguishing%20Squamous%20Carcinomas&amp;author=JD%20Campbell&amp;author=C%20Yau&amp;author=R%20Bowlby&amp;author=Y%20Liu&amp;author=K%20Brennan&amp;volume=23&amp;publication_year=2018&amp;pages=194-212.e6&amp;pmid=29617660&amp;doi=10.1016/j.celrep.2018.03.063&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R21\"> <cite> Cannarile MA, Weisser M, Jacob W, Jegg A-M, Ries CH, and Rüttinger D (2017). Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy. J Immunother Cancer 5, 53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s40425-017-0257-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5514481/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28716061/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Colony-stimulating%20factor%201%20receptor%20(CSF1R)%20inhibitors%20in%20cancer%20therapy&amp;author=MA%20Cannarile&amp;author=M%20Weisser&amp;author=W%20Jacob&amp;author=A-M%20Jegg&amp;author=CH%20Ries&amp;volume=5&amp;publication_year=2017&amp;pages=53&amp;pmid=28716061&amp;doi=10.1186/s40425-017-0257-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R22\"> <cite> Cannito S, Novo E, Compagnone A, Valfrè di Bonzo L, Busletta C, Zamara E, Paternostro C, Povero D, Bandino A, Bozzo F, et al. (2008). Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells. Carcinogenesis 29, 2267–2278. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgn216\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18791199/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Redox%20mechanisms%20switch%20on%20hypoxia-dependent%20epithelial-mesenchymal%20transition%20in%20cancer%20cells&amp;author=S%20Cannito&amp;author=E%20Novo&amp;author=A%20Compagnone&amp;author=L%20Valfr%C3%A8%20di%20Bonzo&amp;author=C%20Busletta&amp;volume=29&amp;publication_year=2008&amp;pages=2267-2278&amp;pmid=18791199&amp;doi=10.1093/carcin/bgn216&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R23\"> <cite> Cassetta L, and Pollard JW (2018). Targeting macrophages: therapeutic approaches in cancer. Nat. Rev. Drug Discov. 17, 887–904. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrd.2018.169\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30361552/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov&amp;title=Targeting%20macrophages:%20therapeutic%20approaches%20in%20cancer&amp;author=L%20Cassetta&amp;author=JW%20Pollard&amp;volume=17&amp;publication_year=2018&amp;pages=887-904&amp;pmid=30361552&amp;doi=10.1038/nrd.2018.169&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R24\"> <cite> Chida S, Okayama H, Noda M, Saito K, Nakajima T, Aoto K, Hayase S, Momma T, Ohki S, Kono K, et al. (2016). Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer. Carcinogenesis 37, 878–887. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgw069\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27287872/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Stromal%20VCAN%20expression%20as%20a%20potential%20prognostic%20biomarker%20for%20disease%20recurrence%20in%20stage%20II-III%20colon%20cancer&amp;author=S%20Chida&amp;author=H%20Okayama&amp;author=M%20Noda&amp;author=K%20Saito&amp;author=T%20Nakajima&amp;volume=37&amp;publication_year=2016&amp;pages=878-887&amp;pmid=27287872&amp;doi=10.1093/carcin/bgw069&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R25\"> <cite> Debebe Z, and Rathmell WK (2015). Ror2 as a therapeutic target in cancer. Pharmacol. Ther. 150, 143–148. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.pharmthera.2015.01.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC9236825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25614331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&amp;title=Ror2%20as%20a%20therapeutic%20target%20in%20cancer&amp;author=Z%20Debebe&amp;author=WK%20Rathmell&amp;volume=150&amp;publication_year=2015&amp;pages=143-148&amp;pmid=25614331&amp;doi=10.1016/j.pharmthera.2015.01.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R26\"> <cite> DeBerardinis RJ, and Chandel NS (2016). Fundamentals of cancer metabolism. Sci Adv 2, e1600200. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/sciadv.1600200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4928883/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27386546/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&amp;title=Fundamentals%20of%20cancer%20metabolism&amp;author=RJ%20DeBerardinis&amp;author=NS%20Chandel&amp;volume=2&amp;publication_year=2016&amp;pages=e1600200&amp;pmid=27386546&amp;doi=10.1126/sciadv.1600200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R27\"> <cite> Deelen P, Bonder MJ, van der Velde KJ, Westra H-J, Winder E, Hendriksen D, Franke L, and Swertz MA (2014). Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration. BMC Res. Notes 7, 901. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1756-0500-7-901\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4307387/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25495213/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&amp;title=Genotype%20harmonizer:%20automatic%20strand%20alignment%20and%20format%20conversion%20for%20genotype%20data%20integration&amp;author=P%20Deelen&amp;author=MJ%20Bonder&amp;author=KJ%20van%20der%20Velde&amp;author=H-J%20Westra&amp;author=E%20Winder&amp;volume=7&amp;publication_year=2014&amp;pages=901&amp;pmid=25495213&amp;doi=10.1186/1756-0500-7-901&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R28\"> <cite> DeYoung MP, Johannessen CM, Leong C-O, Faquin W, Rocco JW, and Ellisen LW (2006). Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma. Cancer Res. 66, 9362–9368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-06-1619\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17018588/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Tumor-specific%20p73%20up-regulation%20mediates%20p63%20dependence%20in%20squamous%20cell%20carcinoma&amp;author=MP%20DeYoung&amp;author=CM%20Johannessen&amp;author=C-O%20Leong&amp;author=W%20Faquin&amp;author=JW%20Rocco&amp;volume=66&amp;publication_year=2006&amp;pages=9362-9368&amp;pmid=17018588&amp;doi=10.1158/0008-5472.CAN-06-1619&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R29\"> <cite> Diviani D, Raimondi F, Del Vescovo CD, Dreyer E, Reggi E, Osman H, Ruggieri L, Gonano C, Cavin S, Box CL, et al. (2016). Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling. Cell Chem Biol 23, 1135–1146. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.chembiol.2016.07.015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27593112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&amp;title=Small-Molecule%20Protein-Protein%20Interaction%20Inhibitor%20of%20Oncogenic%20Rho%20Signaling&amp;author=D%20Diviani&amp;author=F%20Raimondi&amp;author=CD%20Del%20Vescovo&amp;author=E%20Dreyer&amp;author=E%20Reggi&amp;volume=23&amp;publication_year=2016&amp;pages=1135-1146&amp;pmid=27593112&amp;doi=10.1016/j.chembiol.2016.07.015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R30\"> <cite> Dovas A, Gevrey J-C, Grossi A, Park H, Abou-Kheir W, and Cox D (2009). Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages. J. Cell Sci. 122, 3873–3882. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.051755\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2773189/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19808890/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Regulation%20of%20podosome%20dynamics%20by%20WASp%20phosphorylation:%20implication%20in%20matrix%20degradation%20and%20chemotaxis%20in%20macrophages&amp;author=A%20Dovas&amp;author=J-C%20Gevrey&amp;author=A%20Grossi&amp;author=H%20Park&amp;author=W%20Abou-Kheir&amp;volume=122&amp;publication_year=2009&amp;pages=3873-3882&amp;pmid=19808890&amp;doi=10.1242/jcs.051755&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R31\"> <cite> Edelman MJ, Redman MW, Albain KS, McGary EC, Rafique NM, Petro D, Waqar SN, Minichiello K, Miao J, Papadimitrakopoulou VA, et al. (2019). SWOG S1400C ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT02154490\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT02154490 </a> )-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy). J. Thorac. Oncol. 14, 1853–1859. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2019.06.027\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6764876/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31302234/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=SWOG%20S1400C%20(NCT02154490)-A%20Phase%20II%20Study%20of%20Palbociclib%20for%20Previously%20Treated%20Cell%20Cycle%20Gene%20Alteration-Positive%20Patients%20with%20Stage%20IV%20Squamous%20Cell%20Lung%20Cancer%20(Lung-MAP%20Substudy)&amp;author=MJ%20Edelman&amp;author=MW%20Redman&amp;author=KS%20Albain&amp;author=EC%20McGary&amp;author=NM%20Rafique&amp;volume=14&amp;publication_year=2019&amp;pages=1853-1859&amp;pmid=31302234&amp;doi=10.1016/j.jtho.2019.06.027&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R32\"> <cite> Ellis MJ, Suman VJ, Hoog J, Goncalves R, Sanati S, Creighton CJ, DeSchryver K, Crouch E, Brink A, Watson M, et al. (2017). Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance). J. Clin. Oncol. 35, 1061–1069. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2016.69.4406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5455353/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28045625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Ki67%20Proliferation%20Index%20as%20a%20Tool%20for%20Chemotherapy%20Decisions%20During%20and%20After%20Neoadjuvant%20Aromatase%20Inhibitor%20Treatment%20of%20Breast%20Cancer:%20Results%20From%20the%20American%20College%20of%20Surgeons%20Oncology%20Group%20Z1031%20Trial%20(Alliance)&amp;author=MJ%20Ellis&amp;author=VJ%20Suman&amp;author=J%20Hoog&amp;author=R%20Goncalves&amp;author=S%20Sanati&amp;volume=35&amp;publication_year=2017&amp;pages=1061-1069&amp;pmid=28045625&amp;doi=10.1200/JCO.2016.69.4406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R33\"> <cite> Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B, et al. (2018). The Reactome Pathway Knowledgebase. Nucleic Acids Res. 46, D649–D655. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29145629/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Reactome%20Pathway%20Knowledgebase&amp;author=A%20Fabregat&amp;author=S%20Jupe&amp;author=L%20Matthews&amp;author=K%20Sidiropoulos&amp;author=M%20Gillespie&amp;volume=46&amp;publication_year=2018&amp;pages=D649-D655&amp;pmid=29145629&amp;doi=10.1093/nar/gkx1132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R34\"> <cite> Fan T, Sun G, Sun X, Zhao L, Zhong R, and Peng Y (2019). Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment. Cancers 11. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers11030317\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6468516/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30845728/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Tumor%20Energy%20Metabolism%20and%20Potential%20of%203-Bromopyruvate%20as%20an%20Inhibitor%20of%20Aerobic%20Glycolysis:%20Implications%20in%20Tumor%20Treatment&amp;author=T%20Fan&amp;author=G%20Sun&amp;author=X%20Sun&amp;author=L%20Zhao&amp;author=R%20Zhong&amp;volume=11&amp;publication_year=2019&amp;pmid=30845728&amp;doi=10.3390/cancers11030317&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R35\"> <cite> Faubert B, Li KY, Cai L, Hensley CT, Kim J, Zacharias LG, Yang C, Do QN, Doucette S, Burguete D, et al. (2017). Lactate Metabolism in Human Lung Tumors. Cell 171, 358–371.e9. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.09.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5684706/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28985563/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Lactate%20Metabolism%20in%20Human%20Lung%20Tumors&amp;author=B%20Faubert&amp;author=KY%20Li&amp;author=L%20Cai&amp;author=CT%20Hensley&amp;author=J%20Kim&amp;volume=171&amp;publication_year=2017&amp;pages=358-371.e9&amp;pmid=28985563&amp;doi=10.1016/j.cell.2017.09.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R36\"> <cite> Fernandes AP, Capitanio A, Selenius M, Brodin O, Rundlöf A-K, and Björnstedt M (2009). Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation. Histopathology 55, 313–320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1365-2559.2009.03381.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19723146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Histopathology&amp;title=Expression%20profiles%20of%20thioredoxin%20family%20proteins%20in%20human%20lung%20cancer%20tissue:%20correlation%20with%20proliferation%20and%20differentiation&amp;author=AP%20Fernandes&amp;author=A%20Capitanio&amp;author=M%20Selenius&amp;author=O%20Brodin&amp;author=A-K%20Rundl%C3%B6f&amp;volume=55&amp;publication_year=2009&amp;pages=313-320&amp;pmid=19723146&amp;doi=10.1111/j.1365-2559.2009.03381.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R37\"> <cite> Fisher S, Barry A, Abreu J, Minie B, Nolan J, Delorey TM, Young G, Fennell TJ, Allen A, Ambrogio L, et al. (2011). A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries. Genome Biol. 12, R1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-1-r1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3091298/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21205303/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=A%20scalable,%20fully%20automated%20process%20for%20construction%20of%20sequence-ready%20human%20exome%20targeted%20capture%20libraries&amp;author=S%20Fisher&amp;author=A%20Barry&amp;author=J%20Abreu&amp;author=B%20Minie&amp;author=J%20Nolan&amp;volume=12&amp;publication_year=2011&amp;pages=R1&amp;pmid=21205303&amp;doi=10.1186/gb-2011-12-1-r1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R38\"> <cite> Foltz DR, Santiago MC, Berechid BE, and Nye JS (2002). Glycogen synthase kinase-3beta modulates notch signaling and stability. Curr. Biol. 12, 1006–1011. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0960-9822(02)00888-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12123574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol&amp;title=Glycogen%20synthase%20kinase-3beta%20modulates%20notch%20signaling%20and%20stability&amp;author=DR%20Foltz&amp;author=MC%20Santiago&amp;author=BE%20Berechid&amp;author=JS%20Nye&amp;volume=12&amp;publication_year=2002&amp;pages=1006-1011&amp;pmid=12123574&amp;doi=10.1016/s0960-9822(02)00888-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R39\"> <cite> Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CM, and Hansen KD (2014). Functional normalization of 450k methylation array data improves replication in large cancer studies. Genome Biol. 15, 503. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0503-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4283580/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25599564/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Functional%20normalization%20of%20450k%20methylation%20array%20data%20improves%20replication%20in%20large%20cancer%20studies&amp;author=J-P%20Fortin&amp;author=A%20Labbe&amp;author=M%20Lemire&amp;author=BW%20Zanke&amp;author=TJ%20Hudson&amp;volume=15&amp;publication_year=2014&amp;pages=503&amp;pmid=25599564&amp;doi=10.1186/s13059-014-0503-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R40\"> <cite> Fraley C, Raftery AE, Scrucca L, Murphy TB, and Fop M (2016). mclust: Gaussian mixture modelling for model-based clustering, classification, and density estimation. R Package Version 5. </cite> [ <a class=\"usa-link\" href=\"/articles/PMC5096736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27818791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=R%20Package%20Version&amp;title=mclust:%20Gaussian%20mixture%20modelling%20for%20model-based%20clustering,%20classification,%20and%20density%20estimation&amp;author=C%20Fraley&amp;author=AE%20Raftery&amp;author=L%20Scrucca&amp;author=TB%20Murphy&amp;author=M%20Fop&amp;volume=5&amp;publication_year=2016&amp;pmid=27818791&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R41\"> <cite> Freed D, Pan R, and Aldana R TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=TNscope:%20Accurate%20Detection%20of%20Somatic%20Mutations%20with%20Haplotype-based%20Variant%20Candidate%20Detection%20and%20Machine%20Learning%20Filtering&amp;author=D%20Freed&amp;author=R%20Pan&amp;author=R%20Aldana&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R42\"> <cite> Fu B, Meng W, Zeng X, Zhao H, Liu W, and Zhang T (2017). TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma. Biomed Res. Int. 2017, 4698167. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1155/2017/4698167\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5425838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28536696/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biomed%20Res.%20Int&amp;title=TXNRD1%20Is%20an%20Unfavorable%20Prognostic%20Factor%20for%20Patients%20with%20Hepatocellular%20Carcinoma&amp;author=B%20Fu&amp;author=W%20Meng&amp;author=X%20Zeng&amp;author=H%20Zhao&amp;author=W%20Liu&amp;volume=2017&amp;publication_year=2017&amp;pages=4698167&amp;pmid=28536696&amp;doi=10.1155/2017/4698167&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R43\"> <cite> Gandhi N, and Das GM (2019). Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications. Cells 8. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cells8020089\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6406734/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30691108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Metabolic%20Reprogramming%20in%20Breast%20Cancer%20and%20Its%20Therapeutic%20Implications&amp;author=N%20Gandhi&amp;author=GM%20Das&amp;volume=8&amp;publication_year=2019&amp;pmid=30691108&amp;doi=10.3390/cells8020089&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R44\"> <cite> Gao C, Zhuang J, Li H, Liu C, Zhou C, Liu L, and Sun C (2018). Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma. Cancer Cell Int. 18, 194. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12935-018-0691-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6258452/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30498398/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&amp;title=Exploration%20of%20methylation-driven%20genes%20for%20monitoring%20and%20prognosis%20of%20patients%20with%20lung%20adenocarcinoma&amp;author=C%20Gao&amp;author=J%20Zhuang&amp;author=H%20Li&amp;author=C%20Liu&amp;author=C%20Zhou&amp;volume=18&amp;publication_year=2018&amp;pages=194&amp;pmid=30498398&amp;doi=10.1186/s12935-018-0691-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R45\"> <cite> Garcia-Mata R, Boulter E, and Burridge K (2011). The “invisible hand”: regulation of RHO GTPases by RHOGDIs. Nat. Rev. Mol. Cell Biol. 12, 493–504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrm3153\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3260518/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21779026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;title=The%20%E2%80%9Cinvisible%20hand%E2%80%9D:%20regulation%20of%20RHO%20GTPases%20by%20RHOGDIs&amp;author=R%20Garcia-Mata&amp;author=E%20Boulter&amp;author=K%20Burridge&amp;volume=12&amp;publication_year=2011&amp;pages=493-504&amp;pmid=21779026&amp;doi=10.1038/nrm3153&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R46\"> <cite> Gaujoux R, and Seoighe C (2010). A flexible R package for nonnegative matrix factorization. BMC Bioinformatics 11, 367. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-11-367\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2912887/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20598126/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=A%20flexible%20R%20package%20for%20nonnegative%20matrix%20factorization&amp;author=R%20Gaujoux&amp;author=C%20Seoighe&amp;volume=11&amp;publication_year=2010&amp;pages=367&amp;pmid=20598126&amp;doi=10.1186/1471-2105-11-367&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R47\"> <cite> Gey S, and Lebarbier E (2008). Using CART to detect multiple change points in the mean for large sample. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Using%20CART%20to%20detect%20multiple%20change%20points%20in%20the%20mean%20for%20large%20sample&amp;author=S%20Gey&amp;author=E%20Lebarbier&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R48\"> <cite> Giaccone G, Zatloukal P, Roubec J, Floor K, Musil J, Kuta M, van Klaveren RJ, Chaudhary S, Gunther A, and Shamsili S (2009). Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer. J. Clin. Oncol. 27, 4481–4486. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.21.1862\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19687333/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Multicenter%20phase%20II%20trial%20of%20YM155,%20a%20small-molecule%20suppressor%20of%20survivin,%20in%20patients%20with%20advanced,%20refractory,%20non-small-cell%20lung%20cancer&amp;author=G%20Giaccone&amp;author=P%20Zatloukal&amp;author=J%20Roubec&amp;author=K%20Floor&amp;author=J%20Musil&amp;volume=27&amp;publication_year=2009&amp;pages=4481-4486&amp;pmid=19687333&amp;doi=10.1200/JCO.2008.21.1862&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R49\"> <cite> Giannakopoulos NV, Luo J-K, Papov V, Zou W, Lenschow DJ, Jacobs BS, Borden EC, Li J, Virgin HW, and Zhang D-E (2005). Proteomic identification of proteins conjugated to ISG15 in mouse and human cells. Biochem. Biophys. Res. Commun. 336, 496–506. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2005.08.132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16139798/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Proteomic%20identification%20of%20proteins%20conjugated%20to%20ISG15%20in%20mouse%20and%20human%20cells&amp;author=NV%20Giannakopoulos&amp;author=J-K%20Luo&amp;author=V%20Papov&amp;author=W%20Zou&amp;author=DJ%20Lenschow&amp;volume=336&amp;publication_year=2005&amp;pages=496-506&amp;pmid=16139798&amp;doi=10.1016/j.bbrc.2005.08.132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R50\"> <cite> Gibbs DL (2020). Robust classification of Immune Subtypes in Cancer. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Robust%20classification%20of%20Immune%20Subtypes%20in%20Cancer&amp;author=DL%20Gibbs&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R51\"> <cite> Gillette MA, Satpathy S, Cao S, Dhanasekaran SM, Vasaikar SV, Krug K, Petralia F, Li Y, Liang W-W, Reva B, et al. (2020). Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma. Cell 182, 200–225.e35. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.06.013\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7373300/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32649874/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Characterization%20Reveals%20Therapeutic%20Vulnerabilities%20in%20Lung%20Adenocarcinoma&amp;author=MA%20Gillette&amp;author=S%20Satpathy&amp;author=S%20Cao&amp;author=SM%20Dhanasekaran&amp;author=SV%20Vasaikar&amp;volume=182&amp;publication_year=2020&amp;pages=200-225.e35&amp;pmid=32649874&amp;doi=10.1016/j.cell.2020.06.013&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R52\"> <cite> Gorvel JP, Chang TC, Boretto J, Azuma T, and Chavrier P (1998). Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity. FEBS Lett. 422, 269–273. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0014-5793(98)00020-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/9490022/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Differential%20properties%20of%20D4/LyGDI%20versus%20RhoGDI:%20phosphorylation%20and%20rho%20GTPase%20selectivity&amp;author=JP%20Gorvel&amp;author=TC%20Chang&amp;author=J%20Boretto&amp;author=T%20Azuma&amp;author=P%20Chavrier&amp;volume=422&amp;publication_year=1998&amp;pages=269-273&amp;pmid=9490022&amp;doi=10.1016/s0014-5793(98)00020-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R53\"> <cite> Gregory PA, Bert AG, Paterson EL, Barry SC, Tsykin A, Farshid G, Vadas MA, Khew-Goodall Y, and Goodall GJ (2008). The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. Nat. Cell Biol. 10, 593–601. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb1722\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18376396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=The%20miR-200%20family%20and%20miR-205%20regulate%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20ZEB1%20and%20SIP1&amp;author=PA%20Gregory&amp;author=AG%20Bert&amp;author=EL%20Paterson&amp;author=SC%20Barry&amp;author=A%20Tsykin&amp;volume=10&amp;publication_year=2008&amp;pages=593-601&amp;pmid=18376396&amp;doi=10.1038/ncb1722&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R54\"> <cite> Hammerman PS, Lawrence MS, Voet D, Jing R, Cibulskis K, Sivachenko A, Stojanov P, McKenna A, Lander ES, Gabriel S, et al. (2012). Comprehensive genomic characterization of squamous cell lung cancers. Nature 489, 519–525. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature11404\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3466113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22960745/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20characterization%20of%20squamous%20cell%20lung%20cancers&amp;author=PS%20Hammerman&amp;author=MS%20Lawrence&amp;author=D%20Voet&amp;author=R%20Jing&amp;author=K%20Cibulskis&amp;volume=489&amp;publication_year=2012&amp;pages=519-525&amp;pmid=22960745&amp;doi=10.1038/nature11404&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R55\"> <cite> Hänzelmann S, Castelo R, and Guinney J (2013). GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 14, 7. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3618321/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23323831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data&amp;author=S%20H%C3%A4nzelmann&amp;author=R%20Castelo&amp;author=J%20Guinney&amp;volume=14&amp;publication_year=2013&amp;pages=7&amp;pmid=23323831&amp;doi=10.1186/1471-2105-14-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R56\"> <cite> Happel N, Stoldt S, Schmidt B, and Doenecke D (2009). M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3. J. Mol. Biol. 386, 339–350. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2008.12.047\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19136008/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=M%20phase-specific%20phosphorylation%20of%20histone%20H1.5%20at%20threonine%2010%20by%20GSK-3&amp;author=N%20Happel&amp;author=S%20Stoldt&amp;author=B%20Schmidt&amp;author=D%20Doenecke&amp;volume=386&amp;publication_year=2009&amp;pages=339-350&amp;pmid=19136008&amp;doi=10.1016/j.jmb.2008.12.047&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R57\"> <cite> Haslam A, and Prasad V (2019). Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs. JAMA Netw Open 2, e192535. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1001/jamanetworkopen.2019.2535\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6503493/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31050774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw%20Open&amp;title=Estimation%20of%20the%20Percentage%20of%20US%20Patients%20With%20Cancer%20Who%20Are%20Eligible%20for%20and%20Respond%20to%20Checkpoint%20Inhibitor%20Immunotherapy%20Drugs&amp;author=A%20Haslam&amp;author=V%20Prasad&amp;volume=2&amp;publication_year=2019&amp;pages=e192535&amp;pmid=31050774&amp;doi=10.1001/jamanetworkopen.2019.2535&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R58\"> <cite> Hebert AS, Dittenhafer-Reed KE, Yu W, Bailey DJ, Selen ES, Boersma MD, Carson JJ, Tonelli M, Balloon AJ, Higbee AJ, et al. (2013). Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome. Mol. Cell 49, 186–199. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molcel.2012.10.024\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3704155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23201123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Calorie%20restriction%20and%20SIRT3%20trigger%20global%20reprogramming%20of%20the%20mitochondrial%20protein%20acetylome&amp;author=AS%20Hebert&amp;author=KE%20Dittenhafer-Reed&amp;author=W%20Yu&amp;author=DJ%20Bailey&amp;author=ES%20Selen&amp;volume=49&amp;publication_year=2013&amp;pages=186-199&amp;pmid=23201123&amp;doi=10.1016/j.molcel.2012.10.024&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R59\"> <cite> Herbst RS, Morgensztern D, and Boshoff C (2018). The biology and management of non-small cell lung cancer. Nature 553, 446–454. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature25183\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29364287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20biology%20and%20management%20of%20non-small%20cell%20lung%20cancer&amp;author=RS%20Herbst&amp;author=D%20Morgensztern&amp;author=C%20Boshoff&amp;volume=553&amp;publication_year=2018&amp;pages=446-454&amp;pmid=29364287&amp;doi=10.1038/nature25183&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R60\"> <cite> Herreño AM, Ramírez AC, Chaparro VP, Fernandez MJ, Cañas A, Morantes CF, Moreno OM, Brugés RE, Mejía JA, Bustos FJ, et al. (2019). Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter. Tumour Biol. 41, 1010428319851014. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1177/1010428319851014\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31109257/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&amp;title=Role%20of%20RUNX2%20transcription%20factor%20in%20epithelial%20mesenchymal%20transition%20in%20non-small%20cell%20lung%20cancer%20lung%20cancer:%20Epigenetic%20control%20of%20the%20RUNX2%20P1%20promoter&amp;author=AM%20Herre%C3%B1o&amp;author=AC%20Ram%C3%ADrez&amp;author=VP%20Chaparro&amp;author=MJ%20Fernandez&amp;author=A%20Ca%C3%B1as&amp;volume=41&amp;publication_year=2019&amp;pages=1010428319851014&amp;pmid=31109257&amp;doi=10.1177/1010428319851014&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R61\"> <cite> Hitosugi T, and Chen J (2014). Post-translational modifications and the Warburg effect. Oncogene 33, 4279–4285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2013.406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24096483/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Post-translational%20modifications%20and%20the%20Warburg%20effect&amp;author=T%20Hitosugi&amp;author=J%20Chen&amp;volume=33&amp;publication_year=2014&amp;pages=4279-4285&amp;pmid=24096483&amp;doi=10.1038/onc.2013.406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R62\"> <cite> Hong C-F, Chen W-Y, and Wu C-W (2017). Upregulation of Wnt signaling under hypoxia promotes lung cancer progression. Oncol. Rep. 38, 1706–1714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/or.2017.5807\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28713928/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep&amp;title=Upregulation%20of%20Wnt%20signaling%20under%20hypoxia%20promotes%20lung%20cancer%20progression&amp;author=C-F%20Hong&amp;author=W-Y%20Chen&amp;author=C-W%20Wu&amp;volume=38&amp;publication_year=2017&amp;pages=1706-1714&amp;pmid=28713928&amp;doi=10.3892/or.2017.5807&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R63\"> <cite> Hornbeck PV, Kornhauser JM, Tkachev S, Zhang B, Skrzypek E, Murray B, Latham V, and Sullivan M (2012). PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse. Nucleic Acids Res. 40, D261–D270. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkr1122\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3245126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22135298/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PhosphoSitePlus:%20a%20comprehensive%20resource%20for%20investigating%20the%20structure%20and%20function%20of%20experimentally%20determined%20post-translational%20modifications%20in%20man%20and%20mouse&amp;author=PV%20Hornbeck&amp;author=JM%20Kornhauser&amp;author=S%20Tkachev&amp;author=B%20Zhang&amp;author=E%20Skrzypek&amp;volume=40&amp;publication_year=2012&amp;pages=D261-D270&amp;pmid=22135298&amp;doi=10.1093/nar/gkr1122&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R64\"> <cite> Hsu E-C, Rice MA, Bermudez A, Marques FJG, Aslan M, Liu S, Ghoochani A, Zhang CA, Chen Y-S, Zlitni A, et al. (2020). Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1. Proc. Natl. Acad. Sci. U. S. A. 117, 2032–2042. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1905384117\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6994991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31932422/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Trop2%20is%20a%20driver%20of%20metastatic%20prostate%20cancer%20with%20neuroendocrine%20phenotype%20via%20PARP1&amp;author=E-C%20Hsu&amp;author=MA%20Rice&amp;author=A%20Bermudez&amp;author=FJG%20Marques&amp;author=M%20Aslan&amp;volume=117&amp;publication_year=2020&amp;pages=2032-2042&amp;pmid=31932422&amp;doi=10.1073/pnas.1905384117&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R65\"> <cite> Huang C, Chen L, Savage SR, Eguez RV, Dou Y, Li Y, da Veiga Leprevost F, Jaehnig EJ, Lei JT, Wen B, et al. (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2020.12.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7946781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33417831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20insights%20into%20the%20biology%20and%20treatment%20of%20HPV-negative%20head%20and%20neck%20squamous%20cell%20carcinoma&amp;author=C%20Huang&amp;author=L%20Chen&amp;author=SR%20Savage&amp;author=RV%20Eguez&amp;author=Y%20Dou&amp;publication_year=2021&amp;pmid=33417831&amp;doi=10.1016/j.ccell.2020.12.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R66\"> <cite> Huang H-Y, Lin Y-C-D, Li J, Huang K-Y, Shrestha S, Hong H-C, Tang Y, Chen Y-G, Jin C-N, Yu Y, et al. (2020). miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database. Nucleic Acids Res. 48, D148–D154. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz896\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145596/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31647101/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRTarBase%202020:%20updates%20to%20the%20experimentally%20validated%20microRNA-target%20interaction%20database&amp;author=H-Y%20Huang&amp;author=Y-C-D%20Lin&amp;author=J%20Li&amp;author=K-Y%20Huang&amp;author=S%20Shrestha&amp;volume=48&amp;publication_year=2020&amp;pages=D148-D154&amp;pmid=31647101&amp;doi=10.1093/nar/gkz896&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R67\"> <cite> Hubaux R, Thu KL, Vucic EA, Pikor LA, Kung SHY, Martinez VD, Mosslemi M, Becker-Santos DD, Gazdar AF, Lam S, et al. (2015). Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer. Int. J. Cancer 137, 2072–2082. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/ijc.29577\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4537683/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25907283/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&amp;title=Microtubule%20affinity-regulating%20kinase%202%20is%20associated%20with%20DNA%20damage%20response%20and%20cisplatin%20resistance%20in%20non-small%20cell%20lung%20cancer&amp;author=R%20Hubaux&amp;author=KL%20Thu&amp;author=EA%20Vucic&amp;author=LA%20Pikor&amp;author=SHY%20Kung&amp;volume=137&amp;publication_year=2015&amp;pages=2072-2082&amp;pmid=25907283&amp;doi=10.1002/ijc.29577&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R68\"> <cite> Hung YP, Redig A, Hornick JL, and Sholl LM (2020). ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis. Mod. Pathol. 33, 2256–2268. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41379-020-0592-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32572156/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mod.%20Pathol&amp;title=ARID1A%20mutations%20and%20expression%20loss%20in%20non-small%20cell%20lung%20carcinomas:%20clinicopathologic%20and%20molecular%20analysis&amp;author=YP%20Hung&amp;author=A%20Redig&amp;author=JL%20Hornick&amp;author=LM%20Sholl&amp;volume=33&amp;publication_year=2020&amp;pages=2256-2268&amp;pmid=32572156&amp;doi=10.1038/s41379-020-0592-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R69\"> <cite> Ishitani T, Hirao T, Suzuki M, Isoda M, Ishitani S, Harigaya K, Kitagawa M, Matsumoto K, and Itoh M (2010). Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex. Nat. Cell Biol. 12, 278–285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb2028\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20118921/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=Nemo-like%20kinase%20suppresses%20Notch%20signalling%20by%20interfering%20with%20formation%20of%20the%20Notch%20active%20transcriptional%20complex&amp;author=T%20Ishitani&amp;author=T%20Hirao&amp;author=M%20Suzuki&amp;author=M%20Isoda&amp;author=S%20Ishitani&amp;volume=12&amp;publication_year=2010&amp;pages=278-285&amp;pmid=20118921&amp;doi=10.1038/ncb2028&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R70\"> <cite> Jimenez-Blasco D, Santofimia-Castaño P, Gonzalez A, Almeida A, and Bolaños JP (2015). Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway. Cell Death Differ. 22, 1877–1889. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cdd.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4648333/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25909891/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Astrocyte%20NMDA%20receptors%E2%80%99%20activity%20sustains%20neuronal%20survival%20through%20a%20Cdk5-Nrf2%20pathway&amp;author=D%20Jimenez-Blasco&amp;author=P%20Santofimia-Casta%C3%B1o&amp;author=A%20Gonzalez&amp;author=A%20Almeida&amp;author=JP%20Bola%C3%B1os&amp;volume=22&amp;publication_year=2015&amp;pages=1877-1889&amp;pmid=25909891&amp;doi=10.1038/cdd.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R71\"> <cite> Jin YH, Kim H, Oh M, Ki H, and Kim K (2009). Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta. Mol. Cells 27, 15–19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/s10059-009-0001-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19214430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cells&amp;title=Regulation%20of%20Notch1/NICD%20and%20Hes1%20expressions%20by%20GSK-3alpha/beta&amp;author=YH%20Jin&amp;author=H%20Kim&amp;author=M%20Oh&amp;author=H%20Ki&amp;author=K%20Kim&amp;volume=27&amp;publication_year=2009&amp;pages=15-19&amp;pmid=19214430&amp;doi=10.1007/s10059-009-0001-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R72\"> <cite> Johnson WE, Li C, and Rabinovic A (2007). Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostatistics 8, 118–127. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/biostatistics/kxj037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16632515/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Adjusting%20batch%20effects%20in%20microarray%20expression%20data%20using%20empirical%20Bayes%20methods&amp;author=WE%20Johnson&amp;author=C%20Li&amp;author=A%20Rabinovic&amp;volume=8&amp;publication_year=2007&amp;pages=118-127&amp;pmid=16632515&amp;doi=10.1093/biostatistics/kxj037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R73\"> <cite> Jurtz V, Paul S, Andreatta M, Marcatili P, Peters B, and Nielsen M (2017). NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data. J. Immunol. 199, 3360–3368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4049/jimmunol.1700893\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5679736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28978689/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Immunol&amp;title=NetMHCpan-4.0:%20Improved%20Peptide-MHC%20Class%20I%20Interaction%20Predictions%20Integrating%20Eluted%20Ligand%20and%20Peptide%20Binding%20Affinity%20Data&amp;author=V%20Jurtz&amp;author=S%20Paul&amp;author=M%20Andreatta&amp;author=P%20Marcatili&amp;author=B%20Peters&amp;volume=199&amp;publication_year=2017&amp;pages=3360-3368&amp;pmid=28978689&amp;doi=10.4049/jimmunol.1700893&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R74\"> <cite> Kamburov A, Lawrence MS, Polak P, Leshchiner I, Lage K, Golub TR, Lander ES, and Getz G (2015). Comprehensive assessment of cancer missense mutation clustering in protein structures. Proc. Natl. Acad. Sci. U. S. A. 112, E5486–E5495. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1516373112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4603469/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26392535/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Comprehensive%20assessment%20of%20cancer%20missense%20mutation%20clustering%20in%20protein%20structures&amp;author=A%20Kamburov&amp;author=MS%20Lawrence&amp;author=P%20Polak&amp;author=I%20Leshchiner&amp;author=K%20Lage&amp;volume=112&amp;publication_year=2015&amp;pages=E5486-E5495&amp;pmid=26392535&amp;doi=10.1073/pnas.1516373112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R75\"> <cite> Kanehisa M, Furumichi M, Tanabe M, Sato Y, and Morishima K (2017). KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353–D361. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkw1092\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5210567/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27899662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20new%20perspectives%20on%20genomes,%20pathways,%20diseases%20and%20drugs&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=M%20Tanabe&amp;author=Y%20Sato&amp;author=K%20Morishima&amp;volume=45&amp;publication_year=2017&amp;pages=D353-D361&amp;pmid=27899662&amp;doi=10.1093/nar/gkw1092&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R76\"> <cite> Karachaliou N, Fernandez-Bruno M, and Rosell R (2018). Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer? Transl Lung Cancer Res 7, S198–S201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.21037/tlcr.2018.07.02\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6193917/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30393600/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Transl%20Lung%20Cancer%20Res&amp;title=Strategies%20for%20first-line%20immunotherapy%20in%20squamous%20cell%20lung%20cancer:%20are%20combinations%20a%20game%20changer?&amp;author=N%20Karachaliou&amp;author=M%20Fernandez-Bruno&amp;author=R%20Rosell&amp;volume=7&amp;publication_year=2018&amp;pages=S198-S201&amp;pmid=30393600&amp;doi=10.21037/tlcr.2018.07.02&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R77\"> <cite> Kelly RJ, Thomas A, Rajan A, Chun G, Lopez-Chavez A, Szabo E, Spencer S, Carter CA, Guha U, Khozin S, et al. (2013). A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer. Ann. Oncol. 24, 2601–2606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/annonc/mdt249\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3784336/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23857959/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=A%20phase%20I/II%20study%20of%20sepantronium%20bromide%20(YM155,%20survivin%20suppressor)%20with%20paclitaxel%20and%20carboplatin%20in%20patients%20with%20advanced%20non-small-cell%20lung%20cancer&amp;author=RJ%20Kelly&amp;author=A%20Thomas&amp;author=A%20Rajan&amp;author=G%20Chun&amp;author=A%20Lopez-Chavez&amp;volume=24&amp;publication_year=2013&amp;pages=2601-2606&amp;pmid=23857959&amp;doi=10.1093/annonc/mdt249&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R78\"> <cite> Keysar SB, Le PN, Miller B, Jackson BC, Eagles JR, Nieto C, Kim J, Tang B, Glogowska MJ, Morton JJ, et al. (2017). Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2. J. Natl. Cancer Inst. 109. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/jnci/djw189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5025278/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27634934/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&amp;title=Regulation%20of%20Head%20and%20Neck%20Squamous%20Cancer%20Stem%20Cells%20by%20PI3K%20and%20SOX2&amp;author=SB%20Keysar&amp;author=PN%20Le&amp;author=B%20Miller&amp;author=BC%20Jackson&amp;author=JR%20Eagles&amp;volume=109&amp;publication_year=2017&amp;pmid=27634934&amp;doi=10.1093/jnci/djw189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R79\"> <cite> Kikuchi T, Hassanein M, Amann JM, Liu Q, Slebos RJC, Rahman SMJ, Kaufman JM, Zhang X, Hoeksema MD, Harris BK, et al. (2012). In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers. Mol. Cell. Proteomics 11, 916–932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M111.015370\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3494148/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22761400/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=In-depth%20proteomic%20analysis%20of%20nonsmall%20cell%20lung%20cancer%20to%20discover%20molecular%20targets%20and%20candidate%20biomarkers&amp;author=T%20Kikuchi&amp;author=M%20Hassanein&amp;author=JM%20Amann&amp;author=Q%20Liu&amp;author=RJC%20Slebos&amp;volume=11&amp;publication_year=2012&amp;pages=916-932&amp;pmid=22761400&amp;doi=10.1074/mcp.M111.015370&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R80\"> <cite> Kim H, and Park H (2007). Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis. Bioinformatics 23, 1495–1502. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17483501/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Sparse%20non-negative%20matrix%20factorizations%20via%20alternating%20non-negativity-constrained%20least%20squares%20for%20microarray%20data%20analysis&amp;author=H%20Kim&amp;author=H%20Park&amp;volume=23&amp;publication_year=2007&amp;pages=1495-1502&amp;pmid=17483501&amp;doi=10.1093/bioinformatics/btm134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R81\"> <cite> Kim S, and Pevzner PA (2014). MS-GF+ makes progress towards a universal database search tool for proteomics. Nat. Commun. 5, 5277. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms6277\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5036525/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25358478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=MS-GF+%20makes%20progress%20towards%20a%20universal%20database%20search%20tool%20for%20proteomics&amp;author=S%20Kim&amp;author=PA%20Pevzner&amp;volume=5&amp;publication_year=2014&amp;pages=5277&amp;pmid=25358478&amp;doi=10.1038/ncomms6277&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R82\"> <cite> Kim J, Mouw KW, Polak P, Braunstein LZ, Kamburov A, Kwiatkowski DJ, Rosenberg JE, Van Allen EM, D’Andrea A, and Getz G (2016). Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. Nat. Genet. 48, 600–606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3557\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4936490/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27111033/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&amp;author=J%20Kim&amp;author=KW%20Mouw&amp;author=P%20Polak&amp;author=LZ%20Braunstein&amp;author=A%20Kamburov&amp;volume=48&amp;publication_year=2016&amp;pages=600-606&amp;pmid=27111033&amp;doi=10.1038/ng.3557&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R83\"> <cite> Klepsch V, Hermann-Kleiter N, Do-Dinh P, Jakic B, Offermann A, Efremova M, Sopper S, Rieder D, Krogsdam A, Gamerith G, et al. (2018). Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade. Nat. Commun. 9, 1538. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-018-04004-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5906604/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29670099/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Nuclear%20receptor%20NR2F6%20inhibition%20potentiates%20responses%20to%20PD-L1/PD-1%20cancer%20immune%20checkpoint%20blockade&amp;author=V%20Klepsch&amp;author=N%20Hermann-Kleiter&amp;author=P%20Do-Dinh&amp;author=B%20Jakic&amp;author=A%20Offermann&amp;volume=9&amp;publication_year=2018&amp;pages=1538&amp;pmid=29670099&amp;doi=10.1038/s41467-018-04004-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R84\"> <cite> von Klot C-A, Dubrowinskaja N, Peters I, Hennenlotter J, Merseburger AS, Stenzl A, Kuczyk MA, and Serth J (2017). Rho GDP dissociation inhibitor-β in renal cell carcinoma. Oncol. Lett. 14, 8190–8196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/ol.2017.7233\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5727609/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29250194/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett&amp;title=Rho%20GDP%20dissociation%20inhibitor-%CE%B2%20in%20renal%20cell%20carcinoma&amp;author=C-A%20von%20Klot&amp;author=N%20Dubrowinskaja&amp;author=I%20Peters&amp;author=J%20Hennenlotter&amp;author=AS%20Merseburger&amp;volume=14&amp;publication_year=2017&amp;pages=8190-8196&amp;pmid=29250194&amp;doi=10.3892/ol.2017.7233&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R85\"> <cite> Knudsen ES, Nambiar R, Rosario SR, Smiraglia DJ, Goodrich DW, and Witkiewicz AK (2020). Pan-cancer molecular analysis of the RB tumor suppressor pathway. Commun Biol 3, 158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s42003-020-0873-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7118159/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32242058/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Pan-cancer%20molecular%20analysis%20of%20the%20RB%20tumor%20suppressor%20pathway&amp;author=ES%20Knudsen&amp;author=R%20Nambiar&amp;author=SR%20Rosario&amp;author=DJ%20Smiraglia&amp;author=DW%20Goodrich&amp;volume=3&amp;publication_year=2020&amp;pages=158&amp;pmid=32242058&amp;doi=10.1038/s42003-020-0873-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R86\"> <cite> Krämer A, Green J, Pollard J Jr, and Tugendreich S (2014). Causal analysis approaches in Ingenuity Pathway Analysis. Bioinformatics 30, 523–530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt703\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3928520/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24336805/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Causal%20analysis%20approaches%20in%20Ingenuity%20Pathway%20Analysis&amp;author=A%20Kr%C3%A4mer&amp;author=J%20Green&amp;author=J%20Pollard&amp;author=S%20Tugendreich&amp;volume=30&amp;publication_year=2014&amp;pages=523-530&amp;pmid=24336805&amp;doi=10.1093/bioinformatics/btt703&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R87\"> <cite> Kröger C, Afeyan A, Mraz J, Eaton EN, Reinhardt F, Khodor YL, Thiru P, Bierie B, Ye X, Burge CB, et al. (2019). Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells. Proc. Natl. Acad. Sci. U. S. A. 116, 7353–7362. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1812876116\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6462070/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30910979/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Acquisition%20of%20a%20hybrid%20E/M%20state%20is%20essential%20for%20tumorigenicity%20of%20basal%20breast%20cancer%20cells&amp;author=C%20Kr%C3%B6ger&amp;author=A%20Afeyan&amp;author=J%20Mraz&amp;author=EN%20Eaton&amp;author=F%20Reinhardt&amp;volume=116&amp;publication_year=2019&amp;pages=7353-7362&amp;pmid=30910979&amp;doi=10.1073/pnas.1812876116&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R88\"> <cite> Krug K, Mertins P, Zhang B, Hornbeck P, Raju R, Ahmad R, Szucs M, Mundt F, Forestier D, Jane-Valbuena J, et al. (2018). A curated resource for phosphosite-specific signature analysis. Mol. Cell. Proteomics. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR118.000943\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6398202/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30563849/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20curated%20resource%20for%20phosphosite-specific%20signature%20analysis&amp;author=K%20Krug&amp;author=P%20Mertins&amp;author=B%20Zhang&amp;author=P%20Hornbeck&amp;author=R%20Raju&amp;publication_year=2018&amp;pmid=30563849&amp;doi=10.1074/mcp.TIR118.000943&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R89\"> <cite> Krug K, Jaehnig EJ, Satpathy S, Blumenberg L, Karpova A, Anurag M, Miles G, Mertins P, Geffen Y, Tang LC, et al. (2020). Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy. Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.10.036\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8077737/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33212010/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Landscape%20of%20Breast%20Cancer%20Tumorigenesis%20and%20Targeted%20Therapy&amp;author=K%20Krug&amp;author=EJ%20Jaehnig&amp;author=S%20Satpathy&amp;author=L%20Blumenberg&amp;author=A%20Karpova&amp;publication_year=2020&amp;pmid=33212010&amp;doi=10.1016/j.cell.2020.10.036&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R90\"> <cite> Kuhlmann N, Wroblowski S, Knyphausen P, de Boor S, Brenig J, Zienert AY, Meyer-Teschendorf K, Praefcke GJK, Nolte H, Krüger M, et al. (2016a). Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation. J. Biol. Chem. 291, 5484–5499. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M115.707091\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4786691/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26719334/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Structural%20and%20Mechanistic%20Insights%20into%20the%20Regulation%20of%20the%20Fundamental%20Rho%20Regulator%20RhoGDI%CE%B1%20by%20Lysine%20Acetylation&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=P%20Knyphausen&amp;author=S%20de%20Boor&amp;author=J%20Brenig&amp;volume=291&amp;publication_year=2016a&amp;pages=5484-5499&amp;pmid=26719334&amp;doi=10.1074/jbc.M115.707091&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R91\"> <cite> Kuhlmann N, Wroblowski S, Scislowski L, and Lammers M (2016b). RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA. Biochemistry 55, 304–312. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/acs.biochem.5b01242\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26695096/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=RhoGDI%CE%B1%20Acetylation%20at%20K127%20and%20K141%20Affects%20Binding%20toward%20Nonprenylated%20RhoA&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=L%20Scislowski&amp;author=M%20Lammers&amp;volume=55&amp;publication_year=2016b&amp;pages=304-312&amp;pmid=26695096&amp;doi=10.1021/acs.biochem.5b01242&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R92\"> <cite> Lacher SE, and Slattery M (2016). Gene regulatory effects of disease-associated variation in the NRF2 network. Curr Opin Toxicol 1, 71–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cotox.2016.09.001\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5305174/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28203648/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Toxicol&amp;title=Gene%20regulatory%20effects%20of%20disease-associated%20variation%20in%20the%20NRF2%20network&amp;author=SE%20Lacher&amp;author=M%20Slattery&amp;volume=1&amp;publication_year=2016&amp;pages=71-79&amp;pmid=28203648&amp;doi=10.1016/j.cotox.2016.09.001&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R93\"> <cite> Lamb J, Crawford ED, Peck D, Modell JW, Blat IC, Wrobel MJ, Lerner J, Brunet J-P, Subramanian A, Ross KN, et al. (2006). The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 313, 1929–1935. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1132939\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17008526/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Connectivity%20Map:%20using%20gene-expression%20signatures%20to%20connect%20small%20molecules,%20genes,%20and%20disease&amp;author=J%20Lamb&amp;author=ED%20Crawford&amp;author=D%20Peck&amp;author=JW%20Modell&amp;author=IC%20Blat&amp;volume=313&amp;publication_year=2006&amp;pages=1929-1935&amp;pmid=17008526&amp;doi=10.1126/science.1132939&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R94\"> <cite> Lamprecht S, Kaller M, Schmidt EM, Blaj C, Schiergens TS, Engel J, Jung A, Hermeking H, Grünewald TGP, Kirchner T, et al. (2018). PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer. Clin. Cancer Res. 24, 1974–1986. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-2572\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29391352/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=PBX3%20Is%20Part%20of%20an%20EMT%20Regulatory%20Network%20and%20Indicates%20Poor%20Outcome%20in%20Colorectal%20Cancer&amp;author=S%20Lamprecht&amp;author=M%20Kaller&amp;author=EM%20Schmidt&amp;author=C%20Blaj&amp;author=TS%20Schiergens&amp;volume=24&amp;publication_year=2018&amp;pages=1974-1986&amp;pmid=29391352&amp;doi=10.1158/1078-0432.CCR-17-2572&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R95\"> <cite> Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, Meyerson M, Gabriel SB, Lander ES, and Getz G (2014). Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature12912\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4048962/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24390350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Discovery%20and%20saturation%20analysis%20of%20cancer%20genes%20across%2021%20tumour%20types&amp;author=MS%20Lawrence&amp;author=P%20Stojanov&amp;author=CH%20Mermel&amp;author=JT%20Robinson&amp;author=LA%20Garraway&amp;volume=505&amp;publication_year=2014&amp;pages=495-501&amp;pmid=24390350&amp;doi=10.1038/nature12912&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R96\"> <cite> Lê S, Josse J, and Husson F (2008). FactoMineR: AnRPackage for Multivariate Analysis. Journal of Statistical Software 25. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=FactoMineR:%20AnRPackage%20for%20Multivariate%20Analysis&amp;author=S%20L%C3%AA&amp;author=J%20Josse&amp;author=F%20Husson&amp;volume=25&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R97\"> <cite> Lebanony D, Benjamin H, Gilad S, Ezagouri M, Dov A, Ashkenazi K, Gefen N, Izraeli S, Rechavi G, Pass H, et al. (2009). Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma. J. Clin. Oncol. 27, 2030–2037. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.19.4134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19273703/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Diagnostic%20assay%20based%20on%20hsa-miR-205%20expression%20distinguishes%20squamous%20from%20nonsquamous%20non-small-cell%20lung%20carcinoma&amp;author=D%20Lebanony&amp;author=H%20Benjamin&amp;author=S%20Gilad&amp;author=M%20Ezagouri&amp;author=A%20Dov&amp;volume=27&amp;publication_year=2009&amp;pages=2030-2037&amp;pmid=19273703&amp;doi=10.1200/JCO.2008.19.4134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R98\"> <cite> Lebarbier E (2005). Detecting multiple change-points in the mean of Gaussian process by model selection. Signal Processing 85, 717–736. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Signal%20Processing&amp;title=Detecting%20multiple%20change-points%20in%20the%20mean%20of%20Gaussian%20process%20by%20model%20selection&amp;author=E%20Lebarbier&amp;volume=85&amp;publication_year=2005&amp;pages=717-736&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R99\"> <cite> Le Large TY, Mantini G, Meijer LL, Pham TV, Funel N, van Grieken NC, Kok B, Knol J, van Laarhoven HW, Piersma SR, et al. (2020). Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets. JCI Insight 5. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1172/jci.insight.138290\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7455080/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32634123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCI%20Insight&amp;title=Microdissected%20pancreatic%20cancer%20proteomes%20reveal%20tumor%20heterogeneity%20and%20therapeutic%20targets&amp;author=TY%20Le%20Large&amp;author=G%20Mantini&amp;author=LL%20Meijer&amp;author=TV%20Pham&amp;author=N%20Funel&amp;volume=5&amp;publication_year=2020&amp;pmid=32634123&amp;doi=10.1172/jci.insight.138290&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R100\"> <cite> Lewandowski KT, and Piwnica-Worms H (2014). Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity. J. Cell Sci. 127, 315–327. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.129148\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3889397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24259665/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Phosphorylation%20of%20the%20E3%20ubiquitin%20ligase%20RNF41%20by%20the%20kinase%20Par-1b%20is%20required%20for%20epithelial%20cell%20polarity&amp;author=KT%20Lewandowski&amp;author=H%20Piwnica-Worms&amp;volume=127&amp;publication_year=2014&amp;pages=315-327&amp;pmid=24259665&amp;doi=10.1242/jcs.129148&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R101\"> <cite> Li F, Aljahdali I, and Ling X (2019a). Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study? J. Exp. Clin. Cancer Res. 38, 368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13046-019-1362-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6704566/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31439015/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Clin.%20Cancer%20Res&amp;title=Cancer%20therapeutics%20using%20survivin%20BIRC5%20as%20a%20target:%20what%20can%20we%20do%20after%20over%20two%20decades%20of%20study?&amp;author=F%20Li&amp;author=I%20Aljahdali&amp;author=X%20Ling&amp;volume=38&amp;publication_year=2019a&amp;pages=368&amp;pmid=31439015&amp;doi=10.1186/s13046-019-1362-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R102\"> <cite> Li K, Vaudel M, Zhang B, Ren Y, and Wen B (2019b). PDV: an integrative proteomics data viewer. Bioinformatics 35, 1249–1251. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty770\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6821182/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30169737/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PDV:%20an%20integrative%20proteomics%20data%20viewer&amp;author=K%20Li&amp;author=M%20Vaudel&amp;author=B%20Zhang&amp;author=Y%20Ren&amp;author=B%20Wen&amp;volume=35&amp;publication_year=2019b&amp;pages=1249-1251&amp;pmid=30169737&amp;doi=10.1093/bioinformatics/bty770&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R103\"> <cite> Li M, Xie X, Zhou J, Sheng M, Yin X, Ko E-A, Zhou T, and Gu W (2017). Quantifying circular RNA expression from RNA-seq data using model-based framework. Bioinformatics 33, 2131–2139. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btx129\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28334396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quantifying%20circular%20RNA%20expression%20from%20RNA-seq%20data%20using%20model-based%20framework&amp;author=M%20Li&amp;author=X%20Xie&amp;author=J%20Zhou&amp;author=M%20Sheng&amp;author=X%20Yin&amp;volume=33&amp;publication_year=2017&amp;pages=2131-2139&amp;pmid=28334396&amp;doi=10.1093/bioinformatics/btx129&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R104\"> <cite> Li P, Cao G, Zhang Y, Shi J, Cai K, Zhen L, He X, Zhou Y, Li Y, Zhu Y, et al. (2020). FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors. Aging 12, 53–69. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/aging.102564\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6977653/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31935687/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Aging&amp;title=FHL3%20promotes%20pancreatic%20cancer%20invasion%20and%20metastasis%20through%20preventing%20the%20ubiquitination%20degradation%20of%20EMT%20associated%20transcription%20factors&amp;author=P%20Li&amp;author=G%20Cao&amp;author=Y%20Zhang&amp;author=J%20Shi&amp;author=K%20Cai&amp;volume=12&amp;publication_year=2020&amp;pages=53-69&amp;pmid=31935687&amp;doi=10.18632/aging.102564&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R105\"> <cite> Liang Y, Liu M, Wang P, Ding X, and Cao Y (2013). Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer. Genes Chromosomes Cancer 52, 305–315. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/gcc.22030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23225332/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&amp;title=Analysis%20of%2020%20genes%20at%20chromosome%20band%2012q13:%20RACGAP1%20and%20MCRS1%20overexpression%20in%20nonsmall-cell%20lung%20cancer&amp;author=Y%20Liang&amp;author=M%20Liu&amp;author=P%20Wang&amp;author=X%20Ding&amp;author=Y%20Cao&amp;volume=52&amp;publication_year=2013&amp;pages=305-315&amp;pmid=23225332&amp;doi=10.1002/gcc.22030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R106\"> <cite> Liao Y, Wang J, Jaehnig EJ, Shi Z, and Zhang B (2019). WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Nucleic Acids Res. 47, W199–W205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602449/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31114916/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202019:%20gene%20set%20analysis%20toolkit%20with%20revamped%20UIs%20and%20APIs&amp;author=Y%20Liao&amp;author=J%20Wang&amp;author=EJ%20Jaehnig&amp;author=Z%20Shi&amp;author=B%20Zhang&amp;volume=47&amp;publication_year=2019&amp;pages=W199-W205&amp;pmid=31114916&amp;doi=10.1093/nar/gkz401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R107\"> <cite> Liberzon A, Birger C, Thorvaldsdóttir H, Ghandi M, Mesirov JP, and Tamayo P (2015). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst 1, 417–425. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cels.2015.12.004\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4707969/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26771021/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20Molecular%20Signatures%20Database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;author=A%20Liberzon&amp;author=C%20Birger&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=M%20Ghandi&amp;author=JP%20Mesirov&amp;volume=1&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R108\"> <cite> Lim SH, Sun J-M, Choi Y-L, Kim HR, Ahn S, Lee JY, Lee S-H, Ahn JS, Park K, Kim JH, et al. (2016). Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study. Cancer 122, 3024–3031. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/cncr.30135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27315356/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Efficacy%20and%20safety%20of%20dovitinib%20in%20pretreated%20patients%20with%20advanced%20squamous%20non-small%20cell%20lung%20cancer%20with%20FGFR1%20amplification:%20A%20single-arm,%20phase%202%20study&amp;author=SH%20Lim&amp;author=J-M%20Sun&amp;author=Y-L%20Choi&amp;author=HR%20Kim&amp;author=S%20Ahn&amp;volume=122&amp;publication_year=2016&amp;pages=3024-3031&amp;pmid=27315356&amp;doi=10.1002/cncr.30135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R109\"> <cite> Lim YW, Chen-Harris H, Mayba O, Lianoglou S, Wuster A, Bhangale T, Khan Z, Mariathasan S, Daemen A, Reeder J, et al. (2018). Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration. Proc. Natl. Acad. Sci. U. S. A. 115, E11701–E11710. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1804506115\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6294879/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30463956/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Germline%20genetic%20polymorphisms%20influence%20tumor%20gene%20expression%20and%20immune%20cell%20infiltration&amp;author=YW%20Lim&amp;author=H%20Chen-Harris&amp;author=O%20Mayba&amp;author=S%20Lianoglou&amp;author=A%20Wuster&amp;volume=115&amp;publication_year=2018&amp;pages=E11701-E11710&amp;pmid=30463956&amp;doi=10.1073/pnas.1804506115&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R110\"> <cite> Lindsay J, McDade SS, Pickard A, McCloskey KD, and McCance DJ (2011). Role of DeltaNp63gamma in epithelial to mesenchymal transition. J. Biol. Chem. 286, 3915–3924. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M110.162511\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3030392/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21127042/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Role%20of%20DeltaNp63gamma%20in%20epithelial%20to%20mesenchymal%20transition&amp;author=J%20Lindsay&amp;author=SS%20McDade&amp;author=A%20Pickard&amp;author=KD%20McCloskey&amp;author=DJ%20McCance&amp;volume=286&amp;publication_year=2011&amp;pages=3915-3924&amp;pmid=21127042&amp;doi=10.1074/jbc.M110.162511&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R111\"> <cite> Liu W, Payne SH, Ma S, and Fenyö D (2019). Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis. Mol. Cell. Proteomics 18, S169–S182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR119.001442\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692784/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31213479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Extracting%20Pathway-level%20Signatures%20from%20Proteogenomic%20Data%20in%20Breast%20Cancer%20Using%20Independent%20Component%20Analysis&amp;author=W%20Liu&amp;author=SH%20Payne&amp;author=S%20Ma&amp;author=D%20Feny%C3%B6&amp;volume=18&amp;publication_year=2019&amp;pages=S169-S182&amp;pmid=31213479&amp;doi=10.1074/mcp.TIR119.001442&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R112\"> <cite> Liu X, Wang L, Cui W, Yuan X, Lin L, Cao Q, Wang N, Li Y, Guo W, Zhang X, et al. (2016). Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells. Oncotarget 7, 58516–58530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/oncotarget.11305\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5295448/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27542268/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Targeting%20ALDH1A1%20by%20disulfiram/copper%20complex%20inhibits%20non-small%20cell%20lung%20cancer%20recurrence%20driven%20by%20ALDH-positive%20cancer%20stem%20cells&amp;author=X%20Liu&amp;author=L%20Wang&amp;author=W%20Cui&amp;author=X%20Yuan&amp;author=L%20Lin&amp;volume=7&amp;publication_year=2016&amp;pages=58516-58530&amp;pmid=27542268&amp;doi=10.18632/oncotarget.11305&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R113\"> <cite> Lo Iacono M, Monica V, Saviozzi S, Ceppi P, Bracco E, Papotti M, and Scagliotti GV (2011). p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue. J. Thorac. Oncol. 6, 473–481. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e31820b86b0\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21289519/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=p63%20and%20p73%20isoform%20expression%20in%20non-small%20cell%20lung%20cancer%20and%20corresponding%20morphological%20normal%20lung%20tissue&amp;author=M%20Lo%20Iacono&amp;author=V%20Monica&amp;author=S%20Saviozzi&amp;author=P%20Ceppi&amp;author=E%20Bracco&amp;volume=6&amp;publication_year=2011&amp;pages=473-481&amp;pmid=21289519&amp;doi=10.1097/JTO.0b013e31820b86b0&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R114\"> <cite> Maksimovic J, Phipson B, and Oshlack A (2016). A cross-package Bioconductor workflow for analysing methylation array data. F1000Res. 5, 1281. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.12688/f1000research.8839.1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4916993/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27347385/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=A%20cross-package%20Bioconductor%20workflow%20for%20analysing%20methylation%20array%20data&amp;author=J%20Maksimovic&amp;author=B%20Phipson&amp;author=A%20Oshlack&amp;volume=5&amp;publication_year=2016&amp;pages=1281&amp;pmid=27347385&amp;doi=10.12688/f1000research.8839.1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R115\"> <cite> Malhotra D, Portales-Casamar E, Singh A, Srivastava S, Arenillas D, Happel C, Shyr C, Wakabayashi N, Kensler TW, Wasserman WW, et al. (2010). Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis. Nucleic Acids Res. 38, 5718–5734. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkq212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2943601/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20460467/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Global%20mapping%20of%20binding%20sites%20for%20Nrf2%20identifies%20novel%20targets%20in%20cell%20survival%20response%20through%20ChIP-Seq%20profiling%20and%20network%20analysis&amp;author=D%20Malhotra&amp;author=E%20Portales-Casamar&amp;author=A%20Singh&amp;author=S%20Srivastava&amp;author=D%20Arenillas&amp;volume=38&amp;publication_year=2010&amp;pages=5718-5734&amp;pmid=20460467&amp;doi=10.1093/nar/gkq212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R116\"> <cite> Mani DR, Maynard M, Kothadia R, Krug K, Christianson KE, Heiman D, Clauser KR, Birger C, Getz G, and Carr SA (2020). PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41592-021-01176-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8791030/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34040252/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PANOPLY:%20A%20cloud-based%20platform%20for%20automated%20and%20reproducible%20proteogenomic%20data%20analysis&amp;author=DR%20Mani&amp;author=M%20Maynard&amp;author=R%20Kothadia&amp;author=K%20Krug&amp;author=KE%20Christianson&amp;publication_year=2020&amp;pmid=34040252&amp;doi=10.1038/s41592-021-01176-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R117\"> <cite> McInnes L, Healy J, and Astels S (2017). hdbscan: Hierarchical density based clustering. Journal of Open Source Software 2, 205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Open%20Source%20Software&amp;title=hdbscan:%20Hierarchical%20density%20based%20clustering&amp;author=L%20McInnes&amp;author=J%20Healy&amp;author=S%20Astels&amp;volume=2&amp;publication_year=2017&amp;pages=205&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R118\"> <cite> McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, Flicek P, and Cunningham F (2016). The Ensembl Variant Effect Predictor. Genome Biol. 17, 122. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0974-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4893825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27268795/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=The%20Ensembl%20Variant%20Effect%20Predictor&amp;author=W%20McLaren&amp;author=L%20Gil&amp;author=SE%20Hunt&amp;author=HS%20Riat&amp;author=GRS%20Ritchie&amp;volume=17&amp;publication_year=2016&amp;pages=122&amp;pmid=27268795&amp;doi=10.1186/s13059-016-0974-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R119\"> <cite> Medvar B, Raghuram V, Pisitkun T, Sarkar A, and Knepper MA (2016). Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation. Physiol. Genomics 48, 502–512. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/physiolgenomics.00031.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4967219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27199454/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Physiol.%20Genomics&amp;title=Comprehensive%20database%20of%20human%20E3%20ubiquitin%20ligases:%20application%20to%20aquaporin-2%20regulation&amp;author=B%20Medvar&amp;author=V%20Raghuram&amp;author=T%20Pisitkun&amp;author=A%20Sarkar&amp;author=MA%20Knepper&amp;volume=48&amp;publication_year=2016&amp;pages=502-512&amp;pmid=27199454&amp;doi=10.1152/physiolgenomics.00031.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R120\"> <cite> Merico D, Isserlin R, Stueker O, Emili A, and Bader GD (2010). Enrichment map: a network-based method for gene-set enrichment visualization and interpretation. PLoS One 5, e13984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0013984\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2981572/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21085593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Enrichment%20map:%20a%20network-based%20method%20for%20gene-set%20enrichment%20visualization%20and%20interpretation&amp;author=D%20Merico&amp;author=R%20Isserlin&amp;author=O%20Stueker&amp;author=A%20Emili&amp;author=GD%20Bader&amp;volume=5&amp;publication_year=2010&amp;pages=e13984&amp;pmid=21085593&amp;doi=10.1371/journal.pone.0013984&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R121\"> <cite> Mermel CH, Schumacher SE, Hill B, Meyerson ML, Beroukhim R, and Getz G (2011). GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-4-r41\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3218867/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21527027/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers&amp;author=CH%20Mermel&amp;author=SE%20Schumacher&amp;author=B%20Hill&amp;author=ML%20Meyerson&amp;author=R%20Beroukhim&amp;volume=12&amp;publication_year=2011&amp;pages=R41&amp;pmid=21527027&amp;doi=10.1186/gb-2011-12-4-r41&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R122\"> <cite> Middleton G, Fletcher P, Popat S, Savage J, Summers Y, Greystoke A, Gilligan D, Cave J, O’Rourke N, Brewster A, et al. (2020). The National Lung Matrix Trial of personalized therapy in lung cancer. Nature 583, 807–812. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-2481-8\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7116732/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32669708/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20National%20Lung%20Matrix%20Trial%20of%20personalized%20therapy%20in%20lung%20cancer&amp;author=G%20Middleton&amp;author=P%20Fletcher&amp;author=S%20Popat&amp;author=J%20Savage&amp;author=Y%20Summers&amp;volume=583&amp;publication_year=2020&amp;pages=807-812&amp;pmid=32669708&amp;doi=10.1038/s41586-020-2481-8&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R123\"> <cite> Moissoglu K, McRoberts KS, Meier JA, Theodorescu D, and Schwartz MA (2009). Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases. Cancer Res. 69, 2838–2844. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1397\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2701105/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19276387/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Rho%20GDP%20dissociation%20inhibitor%202%20suppresses%20metastasis%20via%20unconventional%20regulation%20of%20RhoGTPases&amp;author=K%20Moissoglu&amp;author=KS%20McRoberts&amp;author=JA%20Meier&amp;author=D%20Theodorescu&amp;author=MA%20Schwartz&amp;volume=69&amp;publication_year=2009&amp;pages=2838-2844&amp;pmid=19276387&amp;doi=10.1158/0008-5472.CAN-08-1397&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R124\"> <cite> Morrison JA, Pike LA, Sams SB, Sharma V, Zhou Q, Severson JJ, Tan A-C, Wood WM, and Haugen BR (2014). Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer. Mol. Cancer 13, 62. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1476-4598-13-62\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3995095/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24645981/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Thioredoxin%20interacting%20protein%20(TXNIP)%20is%20a%20novel%20tumor%20suppressor%20in%20thyroid%20cancer&amp;author=JA%20Morrison&amp;author=LA%20Pike&amp;author=SB%20Sams&amp;author=V%20Sharma&amp;author=Q%20Zhou&amp;volume=13&amp;publication_year=2014&amp;pages=62&amp;pmid=24645981&amp;doi=10.1186/1476-4598-13-62&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R125\"> <cite> Mouchemore KA, Sampaio NG, Murrey MW, Stanley ER, Lannutti BJ, and Pixley FJ (2013). Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity. FEBS J. 280, 5228–5236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/febs.12316\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3773002/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23648053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Specific%20inhibition%20of%20PI3K%20p110%CE%B4%20inhibits%20CSF-1-induced%20macrophage%20spreading%20and%20invasive%20capacity&amp;author=KA%20Mouchemore&amp;author=NG%20Sampaio&amp;author=MW%20Murrey&amp;author=ER%20Stanley&amp;author=BJ%20Lannutti&amp;volume=280&amp;publication_year=2013&amp;pages=5228-5236&amp;pmid=23648053&amp;doi=10.1111/febs.12316&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R126\"> <cite> Natan E, and Joerger AC (2012). Structure and kinetic stability of the p63 tetramerization domain. J. Mol. Biol. 415, 503–513. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2011.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3277882/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22100306/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=Structure%20and%20kinetic%20stability%20of%20the%20p63%20tetramerization%20domain&amp;author=E%20Natan&amp;author=AC%20Joerger&amp;volume=415&amp;publication_year=2012&amp;pages=503-513&amp;pmid=22100306&amp;doi=10.1016/j.jmb.2011.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R127\"> <cite> Newman AM, Liu CL, Green MR, Gentles AJ, Feng W, Xu Y, Hoang CD, Diehn M, and Alizadeh AA (2015). Robust enumeration of cell subsets from tissue expression profiles. Nat. Methods 12, 453–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.3337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4739640/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25822800/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Robust%20enumeration%20of%20cell%20subsets%20from%20tissue%20expression%20profiles&amp;author=AM%20Newman&amp;author=CL%20Liu&amp;author=MR%20Green&amp;author=AJ%20Gentles&amp;author=W%20Feng&amp;volume=12&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;doi=10.1038/nmeth.3337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R128\"> <cite> Nicorici D, Satalan M, Edgren H, Kangaspeska S, Murumagi A, Kallioniemi O, Virtanen S, and Kilkku O FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FusionCatcher%20-%20a%20tool%20for%20finding%20somatic%20fusion%20genes%20in%20paired-end%20RNA-sequencing%20data&amp;author=D%20Nicorici&amp;author=M%20Satalan&amp;author=H%20Edgren&amp;author=S%20Kangaspeska&amp;author=A%20Murumagi&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R129\"> <cite> Nijman SMB, Luna-Vargas MPA, Velds A, Brummelkamp TR, Dirac AMG, Sixma TK, and Bernards R (2005). A genomic and functional inventory of deubiquitinating enzymes. Cell 123, 773–786. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2005.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16325574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20genomic%20and%20functional%20inventory%20of%20deubiquitinating%20enzymes&amp;author=SMB%20Nijman&amp;author=MPA%20Luna-Vargas&amp;author=A%20Velds&amp;author=TR%20Brummelkamp&amp;author=AMG%20Dirac&amp;volume=123&amp;publication_year=2005&amp;pages=773-786&amp;pmid=16325574&amp;doi=10.1016/j.cell.2005.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R130\"> <cite> Niu D-F, Kondo T, Nakazawa T, Oishi N, Kawasaki T, Mochizuki K, Yamane T, and Katoh R (2012). Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas. Lab. Invest. 92, 1181–1190. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/labinvest.2012.84\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22641097/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lab.%20Invest&amp;title=Transcription%20factor%20Runx2%20is%20a%20regulator%20of%20epithelial-mesenchymal%20transition%20and%20invasion%20in%20thyroid%20carcinomas&amp;author=D-F%20Niu&amp;author=T%20Kondo&amp;author=T%20Nakazawa&amp;author=N%20Oishi&amp;author=T%20Kawasaki&amp;volume=92&amp;publication_year=2012&amp;pages=1181-1190&amp;pmid=22641097&amp;doi=10.1038/labinvest.2012.84&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R131\"> <cite> Obenchain V, Lawrence M, Carey V, Gogarten S, Shannon P, and Morgan M (2014). VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants. Bioinformatics 30, 2076–2078. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu168\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080743/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24681907/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=VariantAnnotation:%20a%20Bioconductor%20package%20for%20exploration%20and%20annotation%20of%20genetic%20variants&amp;author=V%20Obenchain&amp;author=M%20Lawrence&amp;author=V%20Carey&amp;author=S%20Gogarten&amp;author=P%20Shannon&amp;volume=30&amp;publication_year=2014&amp;pages=2076-2078&amp;pmid=24681907&amp;doi=10.1093/bioinformatics/btu168&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R132\"> <cite> Paik PK, Pillai RN, Lathan CS, Velasco SA, and Papadimitrakopoulou V (2019). New Treatment Options in Advanced Squamous Cell Lung Cancer. Am Soc Clin Oncol Educ Book 39, e198–e206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/EDBK_237829\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31099625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am%20Soc%20Clin%20Oncol%20Educ%20Book&amp;title=New%20Treatment%20Options%20in%20Advanced%20Squamous%20Cell%20Lung%20Cancer&amp;author=PK%20Paik&amp;author=RN%20Pillai&amp;author=CS%20Lathan&amp;author=SA%20Velasco&amp;author=V%20Papadimitrakopoulou&amp;volume=39&amp;publication_year=2019&amp;pages=e198-e206&amp;pmid=31099625&amp;doi=10.1200/EDBK_237829&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R133\"> <cite> Parra ER, Jiang M, Solis L, Mino B, Laberiano C, Hernandez S, Gite S, Verma A, Tetzlaff M, Haymaker C, et al. (2020). Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies. Cancers 12. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers12020255\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7072187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31972974/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Procedural%20Requirements%20and%20Recommendations%20for%20Multiplex%20Immunofluorescence%20Tyramide%20Signal%20Amplification%20Assays%20to%20Support%20Translational%20Oncology%20Studies&amp;author=ER%20Parra&amp;author=M%20Jiang&amp;author=L%20Solis&amp;author=B%20Mino&amp;author=C%20Laberiano&amp;volume=12&amp;publication_year=2020&amp;pmid=31972974&amp;doi=10.3390/cancers12020255&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R134\"> <cite> Patel M, Lu L, Zander DS, Sreerama L, Coco D, and Moreb JS (2008). ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors. Lung Cancer 59, 340–349. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.lungcan.2007.08.033\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17920722/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=ALDH1A1%20and%20ALDH3A1%20expression%20in%20lung%20cancers:%20correlation%20with%20histologic%20type%20and%20potential%20precursors&amp;author=M%20Patel&amp;author=L%20Lu&amp;author=DS%20Zander&amp;author=L%20Sreerama&amp;author=D%20Coco&amp;volume=59&amp;publication_year=2008&amp;pages=340-349&amp;pmid=17920722&amp;doi=10.1016/j.lungcan.2007.08.033&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R135\"> <cite> Patnaik A, Rosen LS, Tolaney SM, Tolcher AW, Goldman JW, Gandhi L, Papadopoulos KP, Beeram M, Rasco DW, Hilton JF, et al. (2016). Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors. Cancer Discov. 6, 740–753. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/2159-8290.CD-16-0095\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27217383/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Efficacy%20and%20Safety%20of%20Abemaciclib,%20an%20Inhibitor%20of%20CDK4%20and%20CDK6,%20for%20Patients%20with%20Breast%20Cancer,%20Non-Small%20Cell%20Lung%20Cancer,%20and%20Other%20Solid%20Tumors&amp;author=A%20Patnaik&amp;author=LS%20Rosen&amp;author=SM%20Tolaney&amp;author=AW%20Tolcher&amp;author=JW%20Goldman&amp;volume=6&amp;publication_year=2016&amp;pages=740-753&amp;pmid=27217383&amp;doi=10.1158/2159-8290.CD-16-0095&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R136\"> <cite> Payen VL, Mina E, Van Hée VF, Porporato PE, and Sonveaux P (2020). Monocarboxylate transporters in cancer. Mol Metab 33, 48–66. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molmet.2019.07.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7056923/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395464/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Metab&amp;title=Monocarboxylate%20transporters%20in%20cancer&amp;author=VL%20Payen&amp;author=E%20Mina&amp;author=VF%20Van%20H%C3%A9e&amp;author=PE%20Porporato&amp;author=P%20Sonveaux&amp;volume=33&amp;publication_year=2020&amp;pages=48-66&amp;pmid=31395464&amp;doi=10.1016/j.molmet.2019.07.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R137\"> <cite> Paz-Ares L, Luft A, Vicente D, Tafreshi A, Gümüş M, Mazières J, Hermes B, Çay Şenler F, Csőszi T, Fülöp A, et al. (2018). Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer. N. Engl. J. Med. 379, 2040–2051. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1056/NEJMoa1810865\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30280635/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&amp;title=Pembrolizumab%20plus%20Chemotherapy%20for%20Squamous%20Non-Small-Cell%20Lung%20Cancer&amp;author=L%20Paz-Ares&amp;author=A%20Luft&amp;author=D%20Vicente&amp;author=A%20Tafreshi&amp;author=M%20G%C3%BCm%C3%BC%C5%9F&amp;volume=379&amp;publication_year=2018&amp;pages=2040-2051&amp;pmid=30280635&amp;doi=10.1056/NEJMoa1810865&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R138\"> <cite> Pedersen BS, Layer RM, and Quinlan AR (2016). Vcfanno: fast, flexible annotation of genetic variants. Genome Biol. 17, 118. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0973-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4888505/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27250555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Vcfanno:%20fast,%20flexible%20annotation%20of%20genetic%20variants&amp;author=BS%20Pedersen&amp;author=RM%20Layer&amp;author=AR%20Quinlan&amp;volume=17&amp;publication_year=2016&amp;pages=118&amp;pmid=27250555&amp;doi=10.1186/s13059-016-0973-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R139\"> <cite> Peng A, Mao X, Zhong J, Fan S, and Hu Y (2020). Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology. Proteomics 20, e1900271. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/pmic.201900271\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32223079/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Single-Cell%20Multi-Omics%20and%20Its%20Prospective%20Application%20in%20Cancer%20Biology&amp;author=A%20Peng&amp;author=X%20Mao&amp;author=J%20Zhong&amp;author=S%20Fan&amp;author=Y%20Hu&amp;volume=20&amp;publication_year=2020&amp;pages=e1900271&amp;pmid=32223079&amp;doi=10.1002/pmic.201900271&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R140\"> <cite> Peters LA, Perrigoue J, Mortha A, Iuga A, Song W-M, Neiman EM, Llewellyn SR, Di Narzo A, Kidd BA, Telesco SE, et al. (2017). A functional genomics predictive network model identifies regulators of inflammatory bowel disease. Nat. Genet. 49, 1437–1449. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3947\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5660607/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28892060/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=A%20functional%20genomics%20predictive%20network%20model%20identifies%20regulators%20of%20inflammatory%20bowel%20disease&amp;author=LA%20Peters&amp;author=J%20Perrigoue&amp;author=A%20Mortha&amp;author=A%20Iuga&amp;author=W-M%20Song&amp;volume=49&amp;publication_year=2017&amp;pages=1437-1449&amp;pmid=28892060&amp;doi=10.1038/ng.3947&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R141\"> <cite> Petralia F, Wang L, Peng J, Yan A, Zhu J, and Wang P (2018). A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity. Bioinformatics 34, i528–i536. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty280\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6022554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29949994/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20method%20for%20constructing%20tumor%20specific%20gene%20co-expression%20networks%20based%20on%20samples%20with%20tumor%20purity%20heterogeneity&amp;author=F%20Petralia&amp;author=L%20Wang&amp;author=J%20Peng&amp;author=A%20Yan&amp;author=J%20Zhu&amp;volume=34&amp;publication_year=2018&amp;pages=i528-i536&amp;pmid=29949994&amp;doi=10.1093/bioinformatics/bty280&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R142\"> <cite> Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, de Bakker PIW, Daly MJ, et al. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1086/519795\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1950838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17701901/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&amp;title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S%20Purcell&amp;author=B%20Neale&amp;author=K%20Todd-Brown&amp;author=L%20Thomas&amp;author=MAR%20Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R143\"> <cite> Qian J, and Massion PP (2008). Role of chromosome 3q amplification in lung cancer. J. Thorac. Oncol. 3, 212–215. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e3181663544\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18317062/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Role%20of%20chromosome%203q%20amplification%20in%20lung%20cancer&amp;author=J%20Qian&amp;author=PP%20Massion&amp;volume=3&amp;publication_year=2008&amp;pages=212-215&amp;pmid=18317062&amp;doi=10.1097/JTO.0b013e3181663544&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R144\"> <cite> Ramadoss S, Sen S, Ramachandran I, Roy S, Chaudhuri G, and Farias-Eisner R (2017). Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance. Oncogene 36, 6508. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2017.331\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5701089/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28925393/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Lysine-specific%20demethylase%20KDM3A%20regulates%20ovarian%20cancer%20stemness%20and%20chemoresistance&amp;author=S%20Ramadoss&amp;author=S%20Sen&amp;author=I%20Ramachandran&amp;author=S%20Roy&amp;author=G%20Chaudhuri&amp;volume=36&amp;publication_year=2017&amp;pages=6508&amp;pmid=28925393&amp;doi=10.1038/onc.2017.331&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R145\"> <cite> Raudvere U, Kolberg L, Kuzmin I, Arak T, Adler P, Peterson H, and Vilo J (2019). g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). Nucleic Acids Res. 47, W191–W198. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz369\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31066453/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=g:Profiler:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20(2019%20update)&amp;author=U%20Raudvere&amp;author=L%20Kolberg&amp;author=I%20Kuzmin&amp;author=T%20Arak&amp;author=P%20Adler&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;doi=10.1093/nar/gkz369&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R146\"> <cite> Reese SE, Archer KJ, Therneau TM, Atkinson EJ, Vachon CM, de Andrade M, Kocher J-PA, and Eckel-Passow JE (2013). A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis. Bioinformatics 29, 2877–2883. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt480\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810845/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23958724/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20statistic%20for%20identifying%20batch%20effects%20in%20high-throughput%20genomic%20data%20that%20uses%20guided%20principal%20component%20analysis&amp;author=SE%20Reese&amp;author=KJ%20Archer&amp;author=TM%20Therneau&amp;author=EJ%20Atkinson&amp;author=CM%20Vachon&amp;volume=29&amp;publication_year=2013&amp;pages=2877-2883&amp;pmid=23958724&amp;doi=10.1093/bioinformatics/btt480&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R147\"> <cite> Reva B, Omelchenko T, Calinawan A, Nair S, Schadt E, and Tewari A (2020). Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-γ axis and identification of immune resistance mechanisms. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Prioritization%20prostate%20cancer%20to%20immune%20checkpoint%20therapy%20by%20ranking%20tumors%20along%20IFN-%CE%B3%20axis%20and%20identification%20of%20immune%20resistance%20mechanisms&amp;author=B%20Reva&amp;author=T%20Omelchenko&amp;author=A%20Calinawan&amp;author=S%20Nair&amp;author=E%20Schadt&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R148\"> <cite> Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, and Smyth GK (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkv007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4402510/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25605792/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;author=ME%20Ritchie&amp;author=B%20Phipson&amp;author=D%20Wu&amp;author=Y%20Hu&amp;author=CW%20Law&amp;volume=43&amp;publication_year=2015&amp;pages=e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R149\"> <cite> Robinson DR, Wu Y-M, Lonigro RJ, Vats P, Cobain E, Everett J, Cao X, Rabban E, Kumar-Sinha C, Raymond V, et al. (2017). Integrative clinical genomics of metastatic cancer. Nature 548, 297–303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature23306\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5995337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28783718/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative%20clinical%20genomics%20of%20metastatic%20cancer&amp;author=DR%20Robinson&amp;author=Y-M%20Wu&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=E%20Cobain&amp;volume=548&amp;publication_year=2017&amp;pages=297-303&amp;pmid=28783718&amp;doi=10.1038/nature23306&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R150\"> <cite> Roskoski R Jr (2018). The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders. Pharmacol. Res. 129, 65–83. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.phrs.2018.01.021\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29408302/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res&amp;title=The%20role%20of%20small%20molecule%20platelet-derived%20growth%20factor%20receptor%20(PDGFR)%20inhibitors%20in%20the%20treatment%20of%20neoplastic%20disorders&amp;author=R%20Roskoski&amp;volume=129&amp;publication_year=2018&amp;pages=65-83&amp;pmid=29408302&amp;doi=10.1016/j.phrs.2018.01.021&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R151\"> <cite> Satpathy S, Jaehnig EJ, Krug K, Kim B-J, Saltzman AB, Chan DW, Holloway KR, Anurag M, Huang C, Singh P, et al. (2020). Microscaled proteogenomic methods for precision oncology. Nat. Commun. 11, 532. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-14381-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6985126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31988290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Microscaled%20proteogenomic%20methods%20for%20precision%20oncology&amp;author=S%20Satpathy&amp;author=EJ%20Jaehnig&amp;author=K%20Krug&amp;author=B-J%20Kim&amp;author=AB%20Saltzman&amp;volume=11&amp;publication_year=2020&amp;pages=532&amp;pmid=31988290&amp;doi=10.1038/s41467-020-14381-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R152\"> <cite> Schubert M, Klinger B, Klünemann M, Sieber A, Uhlitz F, Sauer S, Garnett MJ, Blüthgen N, and Saez-Rodriguez J (2018). Perturbation-response genes reveal signaling footprints in cancer gene expression. Nat. Commun. 9, 20. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-017-02391-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5750219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29295995/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Perturbation-response%20genes%20reveal%20signaling%20footprints%20in%20cancer%20gene%20expression&amp;author=M%20Schubert&amp;author=B%20Klinger&amp;author=M%20Kl%C3%BCnemann&amp;author=A%20Sieber&amp;author=F%20Uhlitz&amp;volume=9&amp;publication_year=2018&amp;pages=20&amp;pmid=29295995&amp;doi=10.1038/s41467-017-02391-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R153\"> <cite> Shadforth IP, Dunkley TPJ, Lilley KS, and Bessant C (2005). i-Tracker: for quantitative proteomics using iTRAQ. BMC Genomics 6, 145. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2164-6-145\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1276793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16242023/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=i-Tracker:%20for%20quantitative%20proteomics%20using%20iTRAQ&amp;author=IP%20Shadforth&amp;author=TPJ%20Dunkley&amp;author=KS%20Lilley&amp;author=C%20Bessant&amp;volume=6&amp;publication_year=2005&amp;pages=145&amp;pmid=16242023&amp;doi=10.1186/1471-2164-6-145&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R154\"> <cite> Shahamatdar S, He MX, Reyna MA, Gusev A, AlDubayan SH, Van Allen EM, and Ramachandran S (2020). Germline Features Associated with Immune Infiltration in Solid Tumors. Cell Rep. 30, 2900–2908.e4. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2020.02.039\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7082123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32130895/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Germline%20Features%20Associated%20with%20Immune%20Infiltration%20in%20Solid%20Tumors&amp;author=S%20Shahamatdar&amp;author=MX%20He&amp;author=MA%20Reyna&amp;author=A%20Gusev&amp;author=SH%20AlDubayan&amp;volume=30&amp;publication_year=2020&amp;pages=2900-2908.e4&amp;pmid=32130895&amp;doi=10.1016/j.celrep.2020.02.039&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R155\"> <cite> Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, and Ideker T (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–2504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.1239303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC403769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14597658/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R156\"> <cite> Shen J, Ju Z, Zhao W, Wang L, Peng Y, Ge Z, Nagel ZD, Zou J, Wang C, Kapoor P, et al. (2018). ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade. Nat. Med. 24, 556–562. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41591-018-0012-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6076433/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29736026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=ARID1A%20deficiency%20promotes%20mutability%20and%20potentiates%20therapeutic%20antitumor%20immunity%20unleashed%20by%20immune%20checkpoint%20blockade&amp;author=J%20Shen&amp;author=Z%20Ju&amp;author=W%20Zhao&amp;author=L%20Wang&amp;author=Y%20Peng&amp;volume=24&amp;publication_year=2018&amp;pages=556-562&amp;pmid=29736026&amp;doi=10.1038/s41591-018-0012-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R157\"> <cite> Showe MK, Vachani A, Kossenkov AV, Yousef M, Nichols C, Nikonova EV, Chang C, Kucharczuk J, Tran B, Wakeam E, et al. (2009). Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease. Cancer Res. 69, 9202–9210. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-09-1378\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2798582/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19951989/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Gene%20expression%20profiles%20in%20peripheral%20blood%20mononuclear%20cells%20can%20distinguish%20patients%20with%20non-small%20cell%20lung%20cancer%20from%20patients%20with%20nonmalignant%20lung%20disease&amp;author=MK%20Showe&amp;author=A%20Vachani&amp;author=AV%20Kossenkov&amp;author=M%20Yousef&amp;author=C%20Nichols&amp;volume=69&amp;publication_year=2009&amp;pages=9202-9210&amp;pmid=19951989&amp;doi=10.1158/0008-5472.CAN-09-1378&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R158\"> <cite> Siegel RL, Miller KD, and Jemal A (2020). Cancer statistics, 2020. CA Cancer J. Clin. 70, 7–30. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21590\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31912902/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Cancer%20statistics,%202020&amp;author=RL%20Siegel&amp;author=KD%20Miller&amp;author=A%20Jemal&amp;volume=70&amp;publication_year=2020&amp;pages=7-30&amp;pmid=31912902&amp;doi=10.3322/caac.21590&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R159\"> <cite> Singh A, Boldin-Adamsky S, Thimmulappa RK, Rath SK, Ashush H, Coulter J, Blackford A, Goodman SN, Bunz F, Watson WH, et al. (2008). RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy. Cancer Res. 68, 7975–7984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3070411/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18829555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=RNAi-mediated%20silencing%20of%20nuclear%20factor%20erythroid-2-related%20factor%202%20gene%20expression%20in%20non-small%20cell%20lung%20cancer%20inhibits%20tumor%20growth%20and%20increases%20efficacy%20of%20chemotherapy&amp;author=A%20Singh&amp;author=S%20Boldin-Adamsky&amp;author=RK%20Thimmulappa&amp;author=SK%20Rath&amp;author=H%20Ashush&amp;volume=68&amp;publication_year=2008&amp;pages=7975-7984&amp;pmid=18829555&amp;doi=10.1158/0008-5472.CAN-08-1401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R160\"> <cite> Sinicropi-Yao SL, Amann JM, Lopez DLY, Cerciello F, Coombes KR, and Carbone DP (2019). Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC. J. Thorac. Oncol. 14, 223–236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2018.10.162\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6348021/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30408569/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Co-Expression%20Analysis%20Reveals%20Mechanisms%20Underlying%20the%20Varied%20Roles%20of%20NOTCH1%20in%20NSCLC&amp;author=SL%20Sinicropi-Yao&amp;author=JM%20Amann&amp;author=DLY%20Lopez&amp;author=F%20Cerciello&amp;author=KR%20Coombes&amp;volume=14&amp;publication_year=2019&amp;pages=223-236&amp;pmid=30408569&amp;doi=10.1016/j.jtho.2018.10.162&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R161\"> <cite> Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, and Forbes SA (2018). The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat. Rev. Cancer 18, 696–705. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41568-018-0060-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6450507/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30293088/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=The%20COSMIC%20Cancer%20Gene%20Census:%20describing%20genetic%20dysfunction%20across%20all%20human%20cancers&amp;author=Z%20Sondka&amp;author=S%20Bamford&amp;author=CG%20Cole&amp;author=SA%20Ward&amp;author=I%20Dunham&amp;volume=18&amp;publication_year=2018&amp;pages=696-705&amp;pmid=30293088&amp;doi=10.1038/s41568-018-0060-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R162\"> <cite> Song X, Ji J, Gleason KJ, Yang F, Martignetti JA, Chen LS, and Wang P (2019). Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis. Mol. Cell. Proteomics 18, S52–S65. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.RA118.001220\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692782/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31227599/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Insights%20into%20Impact%20of%20DNA%20Copy%20Number%20Alteration%20and%20Methylation%20on%20the%20Proteogenomic%20Landscape%20of%20Human%20Ovarian%20Cancer%20via%20a%20Multi-omics%20Integrative%20Analysis&amp;author=X%20Song&amp;author=J%20Ji&amp;author=KJ%20Gleason&amp;author=F%20Yang&amp;author=JA%20Martignetti&amp;volume=18&amp;publication_year=2019&amp;pages=S52-S65&amp;pmid=31227599&amp;doi=10.1074/mcp.RA118.001220&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R163\"> <cite> Spitzer MH, and Nolan GP (2016). Mass Cytometry: Single Cells, Many Features. Cell 165, 780–791. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2016.04.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4860251/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27153492/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Many%20Features.%20Cell&amp;title=Mass%20Cytometry:%20Single%20Cells&amp;author=MH%20Spitzer&amp;author=GP%20Nolan&amp;volume=165&amp;publication_year=2016&amp;pages=780-791&amp;pmid=27153492&amp;doi=10.1016/j.cell.2016.04.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R164\"> <cite> Srivastava K, Pickard A, Craig SG, Quinn GP, Lambe SM, James JA, McDade SS, and McCance DJ (2018). ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers. Clin. Cancer Res. 24, 3917–3927. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-3775\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6098695/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29739791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=%CE%94Np63%CE%B3/SRC/Slug%20Signaling%20Axis%20Promotes%20Epithelial-to-Mesenchymal%20Transition%20in%20Squamous%20Cancers&amp;author=K%20Srivastava&amp;author=A%20Pickard&amp;author=SG%20Craig&amp;author=GP%20Quinn&amp;author=SM%20Lambe&amp;volume=24&amp;publication_year=2018&amp;pages=3917-3927&amp;pmid=29739791&amp;doi=10.1158/1078-0432.CCR-17-3775&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R165\"> <cite> Stewart PA, Welsh EA, Slebos RJC, Fang B, Izumi V, Chambers M, Zhang G, Cen L, Pettersson F, Zhang Y, et al. (2019). Proteogenomic landscape of squamous cell lung cancer. Nat. Commun. 10, 3578. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-11452-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6687710/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395880/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Proteogenomic%20landscape%20of%20squamous%20cell%20lung%20cancer&amp;author=PA%20Stewart&amp;author=EA%20Welsh&amp;author=RJC%20Slebos&amp;author=B%20Fang&amp;author=V%20Izumi&amp;volume=10&amp;publication_year=2019&amp;pages=3578&amp;pmid=31395880&amp;doi=10.1038/s41467-019-11452-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R166\"> <cite> Su J, Morgani SM, David CJ, Wang Q, Er EE, Huang Y-H, Basnet H, Zou Y, Shu W, Soni RK, et al. (2020). TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1. Nature 577, 566–571. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-019-1897-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7450666/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31915377/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=TGF-%CE%B2%20orchestrates%20fibrogenic%20and%20developmental%20EMTs%20via%20the%20RAS%20effector%20RREB1&amp;author=J%20Su&amp;author=SM%20Morgani&amp;author=CJ%20David&amp;author=Q%20Wang&amp;author=EE%20Er&amp;volume=577&amp;publication_year=2020&amp;pages=566-571&amp;pmid=31915377&amp;doi=10.1038/s41586-019-1897-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R167\"> <cite> Subramanian A, Narayan R, Corsello SM, Peck DD, Natoli TE, Lu X, Gould J, Davis JF, Tubelli AA, Asiedu JK, et al. (2017). A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles. Cell 171, 1437–1452.e17. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.10.049\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5990023/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29195078/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Next%20Generation%20Connectivity%20Map:%20L1000%20Platform%20and%20the%20First%201,000,000%20Profiles&amp;author=A%20Subramanian&amp;author=R%20Narayan&amp;author=SM%20Corsello&amp;author=DD%20Peck&amp;author=TE%20Natoli&amp;volume=171&amp;publication_year=2017&amp;pages=1437-1452.e17&amp;pmid=29195078&amp;doi=10.1016/j.cell.2017.10.049&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R168\"> <cite> Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, and Kohlbacher O (2014). OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics 30, 3310–3316. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu548\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4441069/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25143287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=OptiType:%20precision%20HLA%20typing%20from%20next-generation%20sequencing%20data&amp;author=A%20Szolek&amp;author=B%20Schubert&amp;author=C%20Mohr&amp;author=M%20Sturm&amp;author=M%20Feldhahn&amp;volume=30&amp;publication_year=2014&amp;pages=3310-3316&amp;pmid=25143287&amp;doi=10.1093/bioinformatics/btu548&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R169\"> <cite> Tam WL, Lu H, Buikhuisen J, Soh BS, Lim E, Reinhardt F, Wu ZJ, Krall JA, Bierie B, Guo W, et al. (2013). Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells. Cancer Cell 24, 347–364. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccr.2013.08.005\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4001722/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24029232/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Protein%20kinase%20C%20%CE%B1%20is%20a%20central%20signaling%20node%20and%20therapeutic%20target%20for%20breast%20cancer%20stem%20cells&amp;author=WL%20Tam&amp;author=H%20Lu&amp;author=J%20Buikhuisen&amp;author=BS%20Soh&amp;author=E%20Lim&amp;volume=24&amp;publication_year=2013&amp;pages=347-364&amp;pmid=24029232&amp;doi=10.1016/j.ccr.2013.08.005&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R170\"> <cite> Tate JG, Bamford S, Jubb HC, Sondka Z, Beare DM, Bindal N, Boutselakis H, Cole CG, Creatore C, Dawson E, et al. (2019). COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gky1015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6323903/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30371878/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=COSMIC:%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;author=JG%20Tate&amp;author=S%20Bamford&amp;author=HC%20Jubb&amp;author=Z%20Sondka&amp;author=DM%20Beare&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;pmid=30371878&amp;doi=10.1093/nar/gky1015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R171\"> <cite> Tecalco-Cruz AC, and Cruz-Ramos E (2018). Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells. Biochem. Biophys. Res. Commun. 499, 973–978. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2018.04.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29626479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Protein%20ISGylation%20and%20free%20ISG15%20levels%20are%20increased%20by%20interferon%20gamma%20in%20breast%20cancer%20cells&amp;author=AC%20Tecalco-Cruz&amp;author=E%20Cruz-Ramos&amp;volume=499&amp;publication_year=2018&amp;pages=973-978&amp;pmid=29626479&amp;doi=10.1016/j.bbrc.2018.04.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R172\"> <cite> Tellez CS, Juri DE, Do K, Bernauer AM, Thomas CL, Damiani LA, Tessema M, Leng S, and Belinsky SA (2011). EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells. Cancer Res. 71, 3087–3097. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-10-3035\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3078195/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21363915/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=EMT%20and%20stem%20cell-like%20properties%20associated%20with%20miR-205%20and%20miR-200%20epigenetic%20silencing%20are%20early%20manifestations%20during%20carcinogen-induced%20transformation%20of%20human%20lung%20epithelial%20cells&amp;author=CS%20Tellez&amp;author=DE%20Juri&amp;author=K%20Do&amp;author=AM%20Bernauer&amp;author=CL%20Thomas&amp;volume=71&amp;publication_year=2011&amp;pages=3087-3097&amp;pmid=21363915&amp;doi=10.1158/0008-5472.CAN-10-3035&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R173\"> <cite> Thorsson V, Gibbs DL, Brown SD, Wolf D, Bortone DS, Ou Yang T-H, Porta-Pardo E, Gao GF, Plaisier CL, Eddy JA, et al. (2018). The Immune Landscape of Cancer. Immunity 48, 812–830.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.immuni.2018.03.023\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5982584/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29628290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20Immune%20Landscape%20of%20Cancer&amp;author=V%20Thorsson&amp;author=DL%20Gibbs&amp;author=SD%20Brown&amp;author=D%20Wolf&amp;author=DS%20Bortone&amp;volume=48&amp;publication_year=2018&amp;pages=812-830.e14&amp;pmid=29628290&amp;doi=10.1016/j.immuni.2018.03.023&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R174\"> <cite> Tian J, Cai Y, Li Y, Lu Z, Huang J, Deng Y, Yang N, Wang X, Ying P, Zhang S, et al. (2021). CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer. Nucleic Acids Res. 49, D1065–D1073. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkaa805\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7778991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33010176/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CancerImmunityQTL:%20a%20database%20to%20systematically%20evaluate%20the%20impact%20of%20genetic%20variants%20on%20immune%20infiltration%20in%20human%20cancer&amp;author=J%20Tian&amp;author=Y%20Cai&amp;author=Y%20Li&amp;author=Z%20Lu&amp;author=J%20Huang&amp;volume=49&amp;publication_year=2021&amp;pages=D1065-D1073&amp;pmid=33010176&amp;doi=10.1093/nar/gkaa805&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R175\"> <cite> Timperi E, Focaccetti C, Gallerano D, Panetta M, Spada S, Gallo E, Visca P, Venuta F, Diso D, Prelaj A, et al. (2017). IL-18 receptor marks functional CD8+ T cells in non-small cell lung cancer. Oncoimmunology 6, e1328337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1080/2162402X.2017.1328337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5543928/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28811967/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncoimmunology&amp;title=IL-18%20receptor%20marks%20functional%20CD8+%20T%20cells%20in%20non-small%20cell%20lung%20cancer&amp;author=E%20Timperi&amp;author=C%20Focaccetti&amp;author=D%20Gallerano&amp;author=M%20Panetta&amp;author=S%20Spada&amp;volume=6&amp;publication_year=2017&amp;pages=e1328337&amp;pmid=28811967&amp;doi=10.1080/2162402X.2017.1328337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R176\"> <cite> Tran MN, Choi W, Wszolek MF, Navai N, Lee I-LC, Nitti G, Wen S, Flores ER, Siefker-Radtke A, Czerniak B, et al. (2013). The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205. J. Biol. Chem. 288, 3275–3288. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M112.408104\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3561548/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23239884/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20p63%20protein%20isoform%20%CE%94Np63%CE%B1%20inhibits%20epithelial-mesenchymal%20transition%20in%20human%20bladder%20cancer%20cells:%20role%20of%20MIR-205&amp;author=MN%20Tran&amp;author=W%20Choi&amp;author=MF%20Wszolek&amp;author=N%20Navai&amp;author=I-LC%20Lee&amp;volume=288&amp;publication_year=2013&amp;pages=3275-3288&amp;pmid=23239884&amp;doi=10.1074/jbc.M112.408104&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R177\"> <cite> Tsherniak A, Vazquez F, Montgomery PG, Weir BA, Kryukov G, Cowley GS, Gill S, Harrington WF, Pantel S, Krill-Burger JM, et al. (2017). Defining a Cancer Dependency Map. Cell 170, 564–576.e16. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.06.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5667678/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28753430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Defining%20a%20Cancer%20Dependency%20Map&amp;author=A%20Tsherniak&amp;author=F%20Vazquez&amp;author=PG%20Montgomery&amp;author=BA%20Weir&amp;author=G%20Kryukov&amp;volume=170&amp;publication_year=2017&amp;pages=564-576.e16&amp;pmid=28753430&amp;doi=10.1016/j.cell.2017.06.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R178\"> <cite> Udeshi ND, Mani DC, Satpathy S, Fereshetian S, Gasser JA, Svinkina T, Olive ME, Ebert BL, Mertins P, and Carr SA (2020). Rapid and deep-scale ubiquitylation profiling for biology and translational research. Nat. Commun. 11, 359. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-14175-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6969155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31953384/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Rapid%20and%20deep-scale%20ubiquitylation%20profiling%20for%20biology%20and%20translational%20research&amp;author=ND%20Udeshi&amp;author=DC%20Mani&amp;author=S%20Satpathy&amp;author=S%20Fereshetian&amp;author=JA%20Gasser&amp;volume=11&amp;publication_year=2020&amp;pages=359&amp;pmid=31953384&amp;doi=10.1038/s41467-019-14175-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R179\"> <cite> Uhlén M, Björling E, Agaton C, Szigyarto CA-K, Amini B, Andersen E, Andersson A-C, Angelidou P, Asplund A, Asplund C, et al. (2005). A human protein atlas for normal and cancer tissues based on antibody proteomics. Mol. Cell. Proteomics 4, 1920–1932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M500279-MCP200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16127175/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20human%20protein%20atlas%20for%20normal%20and%20cancer%20tissues%20based%20on%20antibody%20proteomics&amp;author=M%20Uhl%C3%A9n&amp;author=E%20Bj%C3%B6rling&amp;author=C%20Agaton&amp;author=CA-K%20Szigyarto&amp;author=B%20Amini&amp;volume=4&amp;publication_year=2005&amp;pages=1920-1932&amp;pmid=16127175&amp;doi=10.1074/mcp.M500279-MCP200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R180\"> <cite> Vasaikar S, Huang C, Wang X, Petyuk VA, Savage SR, Wen B, Dou Y, Zhang Y, Shi Z, Arshad OA, et al. (2019). Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities. Cell 177, 1035–1049.e19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2019.03.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6768830/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31031003/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Analysis%20of%20Human%20Colon%20Cancer%20Reveals%20New%20Therapeutic%20Opportunities&amp;author=S%20Vasaikar&amp;author=C%20Huang&amp;author=X%20Wang&amp;author=VA%20Petyuk&amp;author=SR%20Savage&amp;volume=177&amp;publication_year=2019&amp;pages=1035-1049.e19&amp;pmid=31031003&amp;doi=10.1016/j.cell.2019.03.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R181\"> <cite> Vasaikar SV, Straub P, Wang J, and Zhang B (2018). LinkedOmics: analyzing multi-omics data within and across 32 cancer types. Nucleic Acids Res. 46, D956–D963. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1090\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753188/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29136207/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LinkedOmics:%20analyzing%20multi-omics%20data%20within%20and%20across%2032%20cancer%20types&amp;author=SV%20Vasaikar&amp;author=P%20Straub&amp;author=J%20Wang&amp;author=B%20Zhang&amp;volume=46&amp;publication_year=2018&amp;pages=D956-D963&amp;pmid=29136207&amp;doi=10.1093/nar/gkx1090&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R182\"> <cite> Vats P, Chinnaiyan AM, and Kumar-Sinha C (2020). Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit. Methods Mol. Biol. 2079, 69–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/978-1-4939-9904-0_5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31728962/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&amp;title=Case%20Study:%20Systematic%20Detection%20and%20Prioritization%20of%20Gene%20Fusions%20in%20Cancer%20by%20RNA-Seq:%20A%20DIY%20Toolkit&amp;author=P%20Vats&amp;author=AM%20Chinnaiyan&amp;author=C%20Kumar-Sinha&amp;volume=2079&amp;publication_year=2020&amp;pages=69-79&amp;pmid=31728962&amp;doi=10.1007/978-1-4939-9904-0_5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R183\"> <cite> Vosgha H, Ariana A, Smith RA, and Lam AK-Y (2018). miR-205 targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma. Endocr. Relat. Cancer 25, 323–337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1530/ERC-17-0497\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29317480/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Endocr.%20Relat.%20Cancer&amp;title=miR-205%20targets%20angiogenesis%20and%20EMT%20concurrently%20in%20anaplastic%20thyroid%20carcinoma&amp;author=H%20Vosgha&amp;author=A%20Ariana&amp;author=RA%20Smith&amp;author=AK-Y%20Lam&amp;volume=25&amp;publication_year=2018&amp;pages=323-337&amp;pmid=29317480&amp;doi=10.1530/ERC-17-0497&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R184\"> <cite> Walser T, Cui X, Yanagawa J, Lee JM, Heinrich E, Lee G, Sharma S, and Dubinett SM (2008). Smoking and lung cancer: the role of inflammation. Proc. Am. Thorac. Soc. 5, 811–815. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1513/pats.200809-100TH\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080902/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19017734/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Am.%20Thorac.%20Soc&amp;title=Smoking%20and%20lung%20cancer:%20the%20role%20of%20inflammation&amp;author=T%20Walser&amp;author=X%20Cui&amp;author=J%20Yanagawa&amp;author=JM%20Lee&amp;author=E%20Heinrich&amp;volume=5&amp;publication_year=2008&amp;pages=811-815&amp;pmid=19017734&amp;doi=10.1513/pats.200809-100TH&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R185\"> <cite> Wang I-M, Zhang B, Yang X, Zhu J, Stepaniants S, Zhang C, Meng Q, Peters M, He Y, Ni C, et al. (2012). Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers. Mol. Syst. Biol. 8, 594. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2012.24\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3421440/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22806142/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol&amp;title=Systems%20analysis%20of%20eleven%20rodent%20disease%20models%20reveals%20an%20inflammatome%20signature%20and%20key%20drivers&amp;author=I-M%20Wang&amp;author=B%20Zhang&amp;author=X%20Yang&amp;author=J%20Zhu&amp;author=S%20Stepaniants&amp;volume=8&amp;publication_year=2012&amp;pages=594&amp;pmid=22806142&amp;doi=10.1038/msb.2012.24&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R186\"> <cite> Wang J, Vasaikar S, Shi Z, Greer M, and Zhang B (2017). WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. Nucleic Acids Res. 45, W130–W137. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx356\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5570149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28472511/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202017:%20a%20more%20comprehensive,%20powerful,%20flexible%20and%20interactive%20gene%20set%20enrichment%20analysis%20toolkit&amp;author=J%20Wang&amp;author=S%20Vasaikar&amp;author=Z%20Shi&amp;author=M%20Greer&amp;author=B%20Zhang&amp;volume=45&amp;publication_year=2017&amp;pages=W130-W137&amp;pmid=28472511&amp;doi=10.1093/nar/gkx356&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R187\"> <cite> Wang L-B, Karpova A, Gritsenko MA, Kyle JE, Cao S, Li Y, Rykunov D, Colaprico A, Rothstein JH, Hong R, et al. (2021). Proteogenomic and metabolomic characterization of human glioblastoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2021.01.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8044053/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33577785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20and%20metabolomic%20characterization%20of%20human%20glioblastoma&amp;author=L-B%20Wang&amp;author=A%20Karpova&amp;author=MA%20Gritsenko&amp;author=JE%20Kyle&amp;author=S%20Cao&amp;publication_year=2021&amp;pmid=33577785&amp;doi=10.1016/j.ccell.2021.01.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R188\"> <cite> Wang S, Jiang B, Zhang T, Liu L, Wang Y, Wang Y, Chen X, Lin H, Zhou L, Xia Y, et al. (2015). Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1. PLoS Biol. 13, e1002243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pbio.1002243\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4565669/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26356530/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=Insulin%20and%20mTOR%20Pathway%20Regulate%20HDAC3-Mediated%20Deacetylation%20and%20Activation%20of%20PGK1&amp;author=S%20Wang&amp;author=B%20Jiang&amp;author=T%20Zhang&amp;author=L%20Liu&amp;author=Y%20Wang&amp;volume=13&amp;publication_year=2015&amp;pages=e1002243&amp;pmid=26356530&amp;doi=10.1371/journal.pbio.1002243&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R189\"> <cite> Weeden CE, Solomon B, and Asselin-Labat M-L (2015). FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies. Cell Death Discov 1, 15049. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cddiscovery.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4979434/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27551478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discov&amp;title=FGFR1%20inhibition%20in%20lung%20squamous%20cell%20carcinoma:%20questions%20and%20controversies&amp;author=CE%20Weeden&amp;author=B%20Solomon&amp;author=M-L%20Asselin-Labat&amp;volume=1&amp;publication_year=2015&amp;pages=15049&amp;pmid=27551478&amp;doi=10.1038/cddiscovery.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R190\"> <cite> Wen B, Wang X, and Zhang B (2019). PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations. Genome Res. 29, 485–493. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.235028.118\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6396417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30610011/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=PepQuery%20enables%20fast,%20accurate,%20and%20convenient%20proteomic%20validation%20of%20novel%20genomic%20alterations&amp;author=B%20Wen&amp;author=X%20Wang&amp;author=B%20Zhang&amp;volume=29&amp;publication_year=2019&amp;pages=485-493&amp;pmid=30610011&amp;doi=10.1101/gr.235028.118&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R191\"> <cite> Wen B, Li K, Zhang Y, and Zhang B (2020). Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat. Commun. 11, 1759. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-15456-w\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145864/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32273506/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Cancer%20neoantigen%20prioritization%20through%20sensitive%20and%20reliable%20proteogenomics%20analysis&amp;author=B%20Wen&amp;author=K%20Li&amp;author=Y%20Zhang&amp;author=B%20Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=1759&amp;pmid=32273506&amp;doi=10.1038/s41467-020-15456-w&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R192\"> <cite> Westra H-J, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, et al. (2013). Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat. Genet. 45, 1238–1243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2756\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3991562/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013639/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&amp;author=H-J%20Westra&amp;author=MJ%20Peters&amp;author=T%20Esko&amp;author=H%20Yaghootkar&amp;author=C%20Schurmann&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;doi=10.1038/ng.2756&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R193\"> <cite> Wheatley SP, and Altieri DC (2019). Survivin at a glance. J. Cell Sci. 132. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.223826\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6467487/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30948431/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Survivin%20at%20a%20glance&amp;author=SP%20Wheatley&amp;author=DC%20Altieri&amp;volume=132&amp;publication_year=2019&amp;pmid=30948431&amp;doi=10.1242/jcs.223826&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R194\"> <cite> Whitfield ML, Sherlock G, Saldanha AJ, Murray JI, Ball CA, Alexander KE, Matese JC, Perou CM, Hurt MM, Brown PO, et al. (2002). Identification of genes periodically expressed in the human cell cycle and their expression in tumors. Mol. Biol. Cell 13, 1977–2000. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.02-02-0030.\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC117619/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12058064/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Identification%20of%20genes%20periodically%20expressed%20in%20the%20human%20cell%20cycle%20and%20their%20expression%20in%20tumors&amp;author=ML%20Whitfield&amp;author=G%20Sherlock&amp;author=AJ%20Saldanha&amp;author=JI%20Murray&amp;author=CA%20Ball&amp;volume=13&amp;publication_year=2002&amp;pages=1977-2000&amp;pmid=12058064&amp;doi=10.1091/mbc.02-02-0030.&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R195\"> <cite> Wilkerson MD, and Hayes DN (2010). ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. Bioinformatics 26, 1572–1573. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq170\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881355/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20427518/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConsensusClusterPlus:%20a%20class%20discovery%20tool%20with%20confidence%20assessments%20and%20item%20tracking&amp;author=MD%20Wilkerson&amp;author=DN%20Hayes&amp;volume=26&amp;publication_year=2010&amp;pages=1572-1573&amp;pmid=20427518&amp;doi=10.1093/bioinformatics/btq170&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R196\"> <cite> Wilkerson MD, Yin X, Hoadley KA, Liu Y, Hayward MC, Cabanski CR, Muldrew K, Miller CR, Randell SH, Socinski MA, et al. (2010). Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types. Clin. Cancer Res. 16, 4864–4875. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0199\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2953768/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20643781/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Lung%20squamous%20cell%20carcinoma%20mRNA%20expression%20subtypes%20are%20reproducible,%20clinically%20important,%20and%20correspond%20to%20normal%20cell%20types&amp;author=MD%20Wilkerson&amp;author=X%20Yin&amp;author=KA%20Hoadley&amp;author=Y%20Liu&amp;author=MC%20Hayward&amp;volume=16&amp;publication_year=2010&amp;pages=4864-4875&amp;pmid=20643781&amp;doi=10.1158/1078-0432.CCR-10-0199&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R197\"> <cite> Wong JJY, Pung YF, Sze NS-K, and Chin K-C (2006). HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets. Proc. Natl. Acad. Sci. U. S. A. 103, 10735–10740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0600397103\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1484417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16815975/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=HERC5%20is%20an%20IFN-induced%20HECT-type%20E3%20protein%20ligase%20that%20mediates%20type%20I%20IFN-induced%20ISGylation%20of%20protein%20targets&amp;author=JJY%20Wong&amp;author=YF%20Pung&amp;author=NS-K%20Sze&amp;author=K-C%20Chin&amp;volume=103&amp;publication_year=2006&amp;pages=10735-10740&amp;pmid=16815975&amp;doi=10.1073/pnas.0600397103&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R198\"> <cite> Wu Y, Zhang Y, Zheng X, Dai F, Lu Y, Dai L, Niu M, Guo H, Li W, Xue X, et al. (2020). Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis. Mol. Cancer 19, 99. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12943-020-01215-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7265647/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32487167/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Circular%20RNA%20circCORO1C%20promotes%20laryngeal%20squamous%20cell%20carcinoma%20progression%20by%20modulating%20the%20let-7c-5p/PBX3%20axis&amp;author=Y%20Wu&amp;author=Y%20Zhang&amp;author=X%20Zheng&amp;author=F%20Dai&amp;author=Y%20Lu&amp;volume=19&amp;publication_year=2020&amp;pages=99&amp;pmid=32487167&amp;doi=10.1186/s12943-020-01215-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R199\"> <cite> Wu Y-M, Cieślik M, Lonigro RJ, Vats P, Reimers MA, Cao X, Ning Y, Wang L, Kunju LP, de Sarkar N, et al. (2018). Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer. Cell 173, 1770–1782.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.04.034\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6084431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29906450/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Inactivation%20of%20CDK12%20Delineates%20a%20Distinct%20Immunogenic%20Class%20of%20Advanced%20Prostate%20Cancer&amp;author=Y-M%20Wu&amp;author=M%20Cie%C5%9Blik&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=MA%20Reimers&amp;volume=173&amp;publication_year=2018&amp;pages=1770-1782.e14&amp;pmid=29906450&amp;doi=10.1016/j.cell.2018.04.034&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R200\"> <cite> Xiong Y, Lei Q-Y, Zhao S, and Guan K-L (2011). Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK. Cold Spring Harb. Symp. Quant. Biol. 76, 285–289. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/sqb.2011.76.010942\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4876600/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22096030/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol&amp;title=Regulation%20of%20glycolysis%20and%20gluconeogenesis%20by%20acetylation%20of%20PKM%20and%20PEPCK&amp;author=Y%20Xiong&amp;author=Q-Y%20Lei&amp;author=S%20Zhao&amp;author=K-L%20Guan&amp;volume=76&amp;publication_year=2011&amp;pages=285-289&amp;pmid=22096030&amp;doi=10.1101/sqb.2011.76.010942&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R201\"> <cite> Yang W, and Lu Z (2013). Nuclear PKM2 regulates the Warburg effect. Cell Cycle 12, 3154–3158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/cc.26182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3865010/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013426/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&amp;title=Nuclear%20PKM2%20regulates%20the%20Warburg%20effect&amp;author=W%20Yang&amp;author=Z%20Lu&amp;volume=12&amp;publication_year=2013&amp;pages=3154-3158&amp;pmid=24013426&amp;doi=10.4161/cc.26182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R202\"> <cite> Ye K, Schulz MH, Long Q, Apweiler R, and Ning Z (2009). Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics 25, 2865–2871. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btp394\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2781750/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561018/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&amp;author=K%20Ye&amp;author=MH%20Schulz&amp;author=Q%20Long&amp;author=R%20Apweiler&amp;author=Z%20Ning&amp;volume=25&amp;publication_year=2009&amp;pages=2865-2871&amp;pmid=19561018&amp;doi=10.1093/bioinformatics/btp394&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R203\"> <cite> Yomtoubian S, Lee SB, Verma A, Izzo F, Markowitz G, Choi H, Cerchietti L, Vahdat L, Brown KA, Andreopoulou E, et al. (2020). Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer. Cell Rep. 30, 755–770.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2019.12.056\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31968251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Inhibition%20of%20EZH2%20Catalytic%20Activity%20Selectively%20Targets%20a%20Metastatic%20Subpopulation%20in%20Triple-Negative%20Breast%20Cancer&amp;author=S%20Yomtoubian&amp;author=SB%20Lee&amp;author=A%20Verma&amp;author=F%20Izzo&amp;author=G%20Markowitz&amp;volume=30&amp;publication_year=2020&amp;pages=755-770.e6&amp;pmid=31968251&amp;doi=10.1016/j.celrep.2019.12.056&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R204\"> <cite> Yoshihara K, Shahmoradgoli M, Martínez E, Vegesna R, Kim H, Torres-Garcia W, Treviño V, Shen H, Laird PW, Levine DA, et al. (2013). Inferring tumour purity and stromal and immune cell admixture from expression data. Nat. Commun. 4, 2612. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms3612\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3826632/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24113773/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data&amp;author=K%20Yoshihara&amp;author=M%20Shahmoradgoli&amp;author=E%20Mart%C3%ADnez&amp;author=R%20Vegesna&amp;author=H%20Kim&amp;volume=4&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;doi=10.1038/ncomms3612&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R205\"> <cite> Yuan G, Flores NM, Hausmann S, Lofgren SM, Kharchenko V, Angulo-Ibanez M, Sengupta D, Lu X, Czaban I, Azhibek D, et al. (2021). Elevated NSD3 histone methylation activity drives squamous cell lung cancer. Nature. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-03170-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7895461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33536620/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Elevated%20NSD3%20histone%20methylation%20activity%20drives%20squamous%20cell%20lung%20cancer&amp;author=G%20Yuan&amp;author=NM%20Flores&amp;author=S%20Hausmann&amp;author=SM%20Lofgren&amp;author=V%20Kharchenko&amp;publication_year=2021&amp;pmid=33536620&amp;doi=10.1038/s41586-020-03170-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R206\"> <cite> Zhang X, Lu F, Wang J, Yin F, Xu Z, Qi D, Wu X, Cao Y, Liang W, Liu Y, et al. (2013). Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells. Cell Rep. 5, 445–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2013.09.018\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC11162149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24139802/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Pluripotent%20stem%20cell%20protein%20Sox2%20confers%20sensitivity%20to%20LSD1%20inhibition%20in%20cancer%20cells&amp;author=X%20Zhang&amp;author=F%20Lu&amp;author=J%20Wang&amp;author=F%20Yin&amp;author=Z%20Xu&amp;volume=5&amp;publication_year=2013&amp;pages=445-457&amp;pmid=24139802&amp;doi=10.1016/j.celrep.2013.09.018&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R207\"> <cite> Zhang Y, Zou X, Qian W, Weng X, Zhang L, Zhang L, Wang S, Cao X, Ma L, Wei G, et al. (2019a). Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis. Cell Death Differ. 26, 565–579. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41418-018-0147-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6370781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29955124/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Enhanced%20PAPSS2/VCAN%20sulfation%20axis%20is%20essential%20for%20Snail-mediated%20breast%20cancer%20cell%20migration%20and%20metastasis&amp;author=Y%20Zhang&amp;author=X%20Zou&amp;author=W%20Qian&amp;author=X%20Weng&amp;author=L%20Zhang&amp;volume=26&amp;publication_year=2019a&amp;pages=565-579&amp;pmid=29955124&amp;doi=10.1038/s41418-018-0147-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R208\"> <cite> Zhang Y, Thery F, Wu NC, Luhmann EK, Dussurget O, Foecke M, Bredow C, Jiménez-Fernández D, Leandro K, Beling A, et al. (2019b). The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection. Nat. Commun. 10, 5383. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-13393-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6879477/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31772204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=The%20in%20vivo%20ISGylome%20links%20ISG15%20to%20metabolic%20pathways%20and%20autophagy%20upon%20Listeria%20monocytogenes%20infection&amp;author=Y%20Zhang&amp;author=F%20Thery&amp;author=NC%20Wu&amp;author=EK%20Luhmann&amp;author=O%20Dussurget&amp;volume=10&amp;publication_year=2019b&amp;pages=5383&amp;pmid=31772204&amp;doi=10.1038/s41467-019-13393-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R209\"> <cite> Zhao D, Li F-L, Cheng Z-L, and Lei Q-Y (2014). Impact of acetylation on tumor metabolism. Mol Cell Oncol 1, e963452. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/23723548.2014.963452\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4905055/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27308346/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Oncol&amp;title=Impact%20of%20acetylation%20on%20tumor%20metabolism&amp;author=D%20Zhao&amp;author=F-L%20Li&amp;author=Z-L%20Cheng&amp;author=Q-Y%20Lei&amp;volume=1&amp;publication_year=2014&amp;pages=e963452&amp;pmid=27308346&amp;doi=10.4161/23723548.2014.963452&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R210\"> <cite> Zheng L, and Conner SD (2018). Glycogen synthase kinase 3β inhibition enhances Notch1 recycling. Mol. Biol. Cell 29, 389–395. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.E17-07-0474\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6014177/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29237816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Glycogen%20synthase%20kinase%203%CE%B2%20inhibition%20enhances%20Notch1%20recycling&amp;author=L%20Zheng&amp;author=SD%20Conner&amp;volume=29&amp;publication_year=2018&amp;pages=389-395&amp;pmid=29237816&amp;doi=10.1091/mbc.E17-07-0474&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R211\"> <cite> Zhou X, Edmonson MN, Wilkinson MR, Patel A, Wu G, Liu Y, Li Y, Zhang Z, Rusch MC, Parker M, et al. (2016). Exploring genomic alteration in pediatric cancer using ProteinPaint. Nat. Genet. 48, 4–6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3466\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4892362/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26711108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Exploring%20genomic%20alteration%20in%20pediatric%20cancer%20using%20ProteinPaint&amp;author=X%20Zhou&amp;author=MN%20Edmonson&amp;author=MR%20Wilkinson&amp;author=A%20Patel&amp;author=G%20Wu&amp;volume=48&amp;publication_year=2016&amp;pages=4-6&amp;pmid=26711108&amp;doi=10.1038/ng.3466&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ol> </section> </section> <section class=\"associated-data\" id=\"_ad93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> Associated Data </h2> <p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> <section class=\"supplementary-materials\" id=\"_adsm93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Supplementary Materials </h3> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material1_reqid_\"> <div class=\"caption p\"> <span> 1 </span> <p> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material2_reqid_\"> <div class=\"caption p\"> <span> 2 </span> <p> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p> C. Same as B but in the EMT-enriched subtype. </p> <p> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material3_reqid_\"> <div class=\"caption p\"> <span> 3 </span> <p> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material4_reqid_\"> <div class=\"caption p\"> <span> 4 </span> <p> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material5_reqid_\"> <div class=\"caption p\"> <span> 6 </span> <p> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material6_reqid_\"> <div class=\"caption p\"> <span> 7 </span> <p> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material7_reqid_\"> <div class=\"caption p\"> <span> 8 </span> <p> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Metadata associated with this study </p> <p> B. Cancer Associated Genes (CAGs) </p> <p> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material8_reqid_\"> <div class=\"caption p\"> <span> 9 </span> <p> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material9_reqid_\"> <div class=\"caption p\"> <span> 10 </span> <p> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material10_reqid_\"> <div class=\"caption p\"> <span> 11 </span> <p> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p> A. Copy number amplified cytobands </p> <p> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p> F. CMAP tab- Connectivity Map (CMAP) results </p> <p> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p> I. Summary of CDKN2A and RB1 annotations </p> <p> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p> L. NRF2 signature </p> <p> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material11_reqid_\"> <div class=\"caption p\"> <span> 12 </span> <p> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p> B. Ub-sites that show differential abundance across Ub clusters </p> <p> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material12_reqid_\"> <div class=\"caption p\"> <span> 13 </span> <p> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p> B. Immune clusters based on xCell signatures </p> <p> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p> I. List of genes/proteins whose expression is associated with a SNP </p> <p> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material13_reqid_\"> <div class=\"caption p\"> <span> 14 </span> <p> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material14_reqid_\"> <div class=\"caption p\"> <span> Figure S5 </span> <p> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"data-availability-statement\" id=\"_adda93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Data Availability Statement </h3> <section id=\"db_da_sec19_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"db_da_sec20_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"db_da_sec21_reqid_\"> <h3 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"db_da_table-wrap1_reqid_\"> <h4 class=\"obj_head\"> KEY RESOURCES TABLE. </h4> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/db_da_table-wrap1_reqid_/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> </section> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/nihms-1730863.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (3.6 MB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\" hidden=\"\"> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> </div> </div> </div> <div class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br> 8600 Rockville Pike <br> Bethesda, MD 20894 </br> </br> </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> </body> </html>\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 371 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 259 - DEBUG - Loading prompt: GPT_FDR_FewShot_Descr for model gemini-2.5-flash\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Descr from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1221 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1228 - DEBUG - Counting tokens for model: gemini-2.5-flash, prompt length: 2989 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1237 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.5-flash': 747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 725868\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 274 - DEBUG - static_prompt: [{'role': 'system', 'content': 'You are PaperMiner, a specialized assistant that extracts datasets from research publications to enable data reuse. Your mission is to help future researchers discover and properly reuse datasets by understanding how they were originally used in scientific papers. In 2025, the abundance of data is unprecedented, research velocity is accelerating, and proper dataset provenance from publications is crucial for scientific progress.'}, {'role': 'user', 'content': 'Your task: Extract datasets mentioned in research papers and capture their usage context to enable data reuse.\\n\\nWhy this matters: When researchers want to reuse data, they need to understand how it was originally used, what methodology was applied, and what insights it provided. This provenance information from the source publication is essential for proper data reuse.\\n\\nOutput format: JSON object with a \"datasets\" array containing objects with these required keys:\\n- \"dataset_identifier\": A unique identifier or accession code for the dataset\\n- \"repository_reference\": A valid URI or string referring to the repository where the dataset can be found\\n- \"dataset_context_from_paper\": Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly\\n- \"citation_type\": Type of citation used for this dataset. Must be either \"Primary\" (firsthand information collected by the researcher for a specific purpose) or \"Secondary\" (pre-existing information collected by someone else and then used by another researcher)\\n\\nAnalysis approach:\\n1. Find dataset identifiers and their repositories\\n2. Extract relevant text passages that describe the dataset\\'s use\\n3. Determine if this is Primary (authors collected the data) or Secondary (authors used existing data)\\n4. Include direct quotes or paraphrased descriptions from the paper\\n\\nExample output:\\n{{\\n  \"datasets\": [\\n    {{\\n      \"dataset_identifier\": \"GSE69091\",\\n      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\",\\n      \"dataset_context_from_paper\": \"Gene expression dataset from 150 breast cancer patients used to train machine learning models for predicting chemotherapy response. Authors applied random forest classification achieving 85% accuracy. Dataset includes clinical outcomes and 20,000 gene expression values per patient.\",\\n      \"citation_type\": \"Primary\"\\n    }},\\n    {{\\n      \"dataset_identifier\": \"EGAS00001000925\", \\n      \"repository_reference\": \"https://ega-archive.org/datasets/EGAS00001000925\",\\n      \"dataset_context_from_paper\": \"Whole genome sequencing data from 1000 European individuals used as reference population for variant calling pipeline validation. Authors used this dataset to establish baseline allele frequencies and validate their new variant detection algorithm.\",\\n      \"citation_type\": \"Secondary\"\\n    }}\\n  ]\\n}}\\n\\nNow analyze this publication content: {content}\\n\\nKnown repositories: {repos}\\n\\nReturn only the JSON object with the datasets array - no additional text.'}]\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 38 - DEBUG - Rendering prompt with static_prompt:[{'role': 'system', 'content': 'You are PaperMiner, a specialized assistant that extracts datasets from research publications to enable data reuse. Your mission is to help future researchers discover and properly reuse datasets by understanding how they were originally used in scientific papers. In 2025, the abundance of data is unprecedented, research velocity is accelerating, and proper dataset provenance from publications is crucial for scientific progress.'}, {'role': 'user', 'content': 'Your task: Extract datasets mentioned in research papers and capture their usage context to enable data reuse.\\n\\nWhy this matters: When researchers want to reuse data, they need to understand how it was originally used, what methodology was applied, and what insights it provided. This provenance information from the source publication is essential for proper data reuse.\\n\\nOutput format: JSON object with a \"datasets\" array containing objects with these required keys:\\n- \"dataset_identifier\": A unique identifier or accession code for the dataset\\n- \"repository_reference\": A valid URI or string referring to the repository where the dataset can be found\\n- \"dataset_context_from_paper\": Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly\\n- \"citation_type\": Type of citation used for this dataset. Must be either \"Primary\" (firsthand information collected by the researcher for a specific purpose) or \"Secondary\" (pre-existing information collected by someone else and then used by another researcher)\\n\\nAnalysis approach:\\n1. Find dataset identifiers and their repositories\\n2. Extract relevant text passages that describe the dataset\\'s use\\n3. Determine if this is Primary (authors collected the data) or Secondary (authors used existing data)\\n4. Include direct quotes or paraphrased descriptions from the paper\\n\\nExample output:\\n{{\\n  \"datasets\": [\\n    {{\\n      \"dataset_identifier\": \"GSE69091\",\\n      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\",\\n      \"dataset_context_from_paper\": \"Gene expression dataset from 150 breast cancer patients used to train machine learning models for predicting chemotherapy response. Authors applied random forest classification achieving 85% accuracy. Dataset includes clinical outcomes and 20,000 gene expression values per patient.\",\\n      \"citation_type\": \"Primary\"\\n    }},\\n    {{\\n      \"dataset_identifier\": \"EGAS00001000925\", \\n      \"repository_reference\": \"https://ega-archive.org/datasets/EGAS00001000925\",\\n      \"dataset_context_from_paper\": \"Whole genome sequencing data from 1000 European individuals used as reference population for variant calling pipeline validation. Authors used this dataset to establish baseline allele frequencies and validate their new variant detection algorithm.\",\\n      \"citation_type\": \"Secondary\"\\n    }}\\n  ]\\n}}\\n\\nNow analyze this publication content: {content}\\n\\nKnown repositories: {repos}\\n\\nReturn only the JSON object with the datasets array - no additional text.'}], entire_doc: True, dynamic parts(<class 'dict'>): {'content': '<!DOCTYPE html> <html lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> A proteogenomic portrait of lung squamous cell carcinoma - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here\\'s how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you\\'ve safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" role=\"button\" target=\"_blank\" title=\"View on publisher site\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip\" href=\"pdf/nihms-1730863.pdf\" role=\"button\" title=\"Download PDF\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"pmc-permalink display-flex\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to NIHPA Author Manuscripts\" href=\"https://www.hhs.gov/open/public-access-guiding-principles/index.html\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to NIHPA Author Manuscripts\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . Author manuscript; available in PMC: 2022 Aug 5. </div> <div> <em> Published in final edited form as: </em> Cell. 2021 Aug 5;184(16):4348–4371.e40. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1016/j.cell.2021.07.016 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> A proteogenomic portrait of lung squamous cell carcinoma </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Shankha Satpathy </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> </div> </div> <sup> 1, </sup> <sup> 19, </sup> <sup> * </sup> , <a aria-describedby=\"id2\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Karsten Krug </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id3\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Pierre M Jean Beltran </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id4\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Sara R Savage </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> </div> </div> <sup> 2, </sup> <sup> 19 </sup> , <a aria-describedby=\"id5\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Francesca Petralia </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id6\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> <div hidden=\"hidden\" id=\"id6\"> <h3> <span class=\"name western\"> Chandan Kumar-Sinha </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> </div> </div> <sup> 4, </sup> <sup> 20 </sup> , <a aria-describedby=\"id7\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> <div hidden=\"hidden\" id=\"id7\"> <h3> <span class=\"name western\"> Yongchao Dou </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> </div> </div> <sup> 2, </sup> <sup> 20 </sup> , <a aria-describedby=\"id8\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> <div hidden=\"hidden\" id=\"id8\"> <h3> <span class=\"name western\"> Boris Reva </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id9\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> <div hidden=\"hidden\" id=\"id9\"> <h3> <span class=\"name western\"> M Harry Kane </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id10\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> <div hidden=\"hidden\" id=\"id10\"> <h3> <span class=\"name western\"> Shayan C Avanessian </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id11\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> <div hidden=\"hidden\" id=\"id11\"> <h3> <span class=\"name western\"> Suhas V Vasaikar </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> </div> </div> <sup> 5, </sup> <sup> 20 </sup> , <a aria-describedby=\"id12\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> <div hidden=\"hidden\" id=\"id12\"> <h3> <span class=\"name western\"> Azra Krek </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id13\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> <div hidden=\"hidden\" id=\"id13\"> <h3> <span class=\"name western\"> Jonathan T Lei </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id14\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> <div hidden=\"hidden\" id=\"id14\"> <h3> <span class=\"name western\"> Eric J Jaehnig </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id15\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> <div hidden=\"hidden\" id=\"id15\"> <h3> <span class=\"name western\"> Tatiana Omelchenko </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id16\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> <div hidden=\"hidden\" id=\"id16\"> <h3> <span class=\"name western\"> Yifat Geffen </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id17\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> <div hidden=\"hidden\" id=\"id17\"> <h3> <span class=\"name western\"> Erik J Bergstrom </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id18\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> <div hidden=\"hidden\" id=\"id18\"> <h3> <span class=\"name western\"> Vasileios Stathias </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id19\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> <div hidden=\"hidden\" id=\"id19\"> <h3> <span class=\"name western\"> Karen E Christianson </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id20\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> <div hidden=\"hidden\" id=\"id20\"> <h3> <span class=\"name western\"> David I Heiman </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id21\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> <div hidden=\"hidden\" id=\"id21\"> <h3> <span class=\"name western\"> Marcin P Cieslik </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id22\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> <div hidden=\"hidden\" id=\"id22\"> <h3> <span class=\"name western\"> Song Cao </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id23\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> <div hidden=\"hidden\" id=\"id23\"> <h3> <span class=\"name western\"> Xiaoyu Song </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id24\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> <div hidden=\"hidden\" id=\"id24\"> <h3> <span class=\"name western\"> Jiayi Ji </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id25\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> <div hidden=\"hidden\" id=\"id25\"> <h3> <span class=\"name western\"> Wenke Liu </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id26\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> <div hidden=\"hidden\" id=\"id26\"> <h3> <span class=\"name western\"> Kai Li </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id27\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> <div hidden=\"hidden\" id=\"id27\"> <h3> <span class=\"name western\"> Bo Wen </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id28\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> <div hidden=\"hidden\" id=\"id28\"> <h3> <span class=\"name western\"> Yize Li </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id29\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> <div hidden=\"hidden\" id=\"id29\"> <h3> <span class=\"name western\"> Zeynep H Gümüş </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id30\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> <div hidden=\"hidden\" id=\"id30\"> <h3> <span class=\"name western\"> Myvizhi Esai Selvan </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id31\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> <div hidden=\"hidden\" id=\"id31\"> <h3> <span class=\"name western\"> Rama Soundararajan </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id32\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> <div hidden=\"hidden\" id=\"id32\"> <h3> <span class=\"name western\"> Tanvi H Visal </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id33\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> <div hidden=\"hidden\" id=\"id33\"> <h3> <span class=\"name western\"> Maria G Raso </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id34\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> <div hidden=\"hidden\" id=\"id34\"> <h3> <span class=\"name western\"> Edwin Roger Parra </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id35\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> <div hidden=\"hidden\" id=\"id35\"> <h3> <span class=\"name western\"> Özgün Babur </span> </h3> <div class=\"p\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> </div> </div> <sup> 11 </sup> , <a aria-describedby=\"id36\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> <div hidden=\"hidden\" id=\"id36\"> <h3> <span class=\"name western\"> Pankaj Vats </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id37\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> <div hidden=\"hidden\" id=\"id37\"> <h3> <span class=\"name western\"> Shankara Anand </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id38\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> <div hidden=\"hidden\" id=\"id38\"> <h3> <span class=\"name western\"> Tobias Schraink </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id39\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> <div hidden=\"hidden\" id=\"id39\"> <h3> <span class=\"name western\"> MacIntosh Cornwell </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id40\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> <div hidden=\"hidden\" id=\"id40\"> <h3> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id41\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> <div hidden=\"hidden\" id=\"id41\"> <h3> <span class=\"name western\"> Houxiang Zhu </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id42\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> <div hidden=\"hidden\" id=\"id42\"> <h3> <span class=\"name western\"> Chia-Kuei Mo </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id43\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> <div hidden=\"hidden\" id=\"id43\"> <h3> <span class=\"name western\"> Yuping Zhang </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id44\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> <div hidden=\"hidden\" id=\"id44\"> <h3> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id45\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> <div hidden=\"hidden\" id=\"id45\"> <h3> <span class=\"name western\"> Chen Huang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id46\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> <div hidden=\"hidden\" id=\"id46\"> <h3> <span class=\"name western\"> Arul M Chinnaiyan </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id47\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> <div hidden=\"hidden\" id=\"id47\"> <h3> <span class=\"name western\"> Matthew A Wyczalkowski </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id48\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> <div hidden=\"hidden\" id=\"id48\"> <h3> <span class=\"name western\"> Gilbert S Omenn </span> </h3> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> </div> </div> <sup> 13 </sup> , <a aria-describedby=\"id49\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> <div hidden=\"hidden\" id=\"id49\"> <h3> <span class=\"name western\"> Chelsea J Newton </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id50\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> <div hidden=\"hidden\" id=\"id50\"> <h3> <span class=\"name western\"> Stephan Schurer </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id51\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> <div hidden=\"hidden\" id=\"id51\"> <h3> <span class=\"name western\"> Kelly V Ruggles </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id52\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> <div hidden=\"hidden\" id=\"id52\"> <h3> <span class=\"name western\"> David Fenyö </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id53\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> <div hidden=\"hidden\" id=\"id53\"> <h3> <span class=\"name western\"> Scott D Jewell </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id54\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> <div hidden=\"hidden\" id=\"id54\"> <h3> <span class=\"name western\"> Mathangi Thiagarajan </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id55\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> <div hidden=\"hidden\" id=\"id55\"> <h3> <span class=\"name western\"> Mehdi Mesri </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id56\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> <div hidden=\"hidden\" id=\"id56\"> <h3> <span class=\"name western\"> Henry Rodriguez </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id57\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> <div hidden=\"hidden\" id=\"id57\"> <h3> <span class=\"name western\"> Sendurai A Mani </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id58\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> <div hidden=\"hidden\" id=\"id58\"> <h3> <span class=\"name western\"> Namrata D Udeshi </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id59\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> <div hidden=\"hidden\" id=\"id59\"> <h3> <span class=\"name western\"> Gad Getz </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id60\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> <div hidden=\"hidden\" id=\"id60\"> <h3> <span class=\"name western\"> James Suh </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id61\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> <div hidden=\"hidden\" id=\"id61\"> <h3> <span class=\"name western\"> Qing Kay Li </span> </h3> <div class=\"p\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> </div> </div> <sup> 17 </sup> , <a aria-describedby=\"id62\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> <div hidden=\"hidden\" id=\"id62\"> <h3> <span class=\"name western\"> Galen Hostetter </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id63\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> <div hidden=\"hidden\" id=\"id63\"> <h3> <span class=\"name western\"> Paul K Paik </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id64\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> <div hidden=\"hidden\" id=\"id64\"> <h3> <span class=\"name western\"> Saravana M Dhanasekaran </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id65\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> <div hidden=\"hidden\" id=\"id65\"> <h3> <span class=\"name western\"> Ramaswamy Govindan </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id66\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> <div hidden=\"hidden\" id=\"id66\"> <h3> <span class=\"name western\"> Li Ding </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id67\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> <div hidden=\"hidden\" id=\"id67\"> <h3> <span class=\"name western\"> Ana I Robles </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id68\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> <div hidden=\"hidden\" id=\"id68\"> <h3> <span class=\"name western\"> Karl R Clauser </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id69\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> <div hidden=\"hidden\" id=\"id69\"> <h3> <span class=\"name western\"> Alexey I Nesvizhskii </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id70\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> <div hidden=\"hidden\" id=\"id70\"> <h3> <span class=\"name western\"> Pei Wang </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id71\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> <div hidden=\"hidden\" id=\"id71\"> <h3> <span class=\"name western\"> Steven A Carr </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id72\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> <div hidden=\"hidden\" id=\"id72\"> <h3> <span class=\"name western\"> Bing Zhang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> </div> </div> <sup> 2, </sup> <sup> * </sup> , <a aria-describedby=\"id73\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> <div hidden=\"hidden\" id=\"id73\"> <h3> <span class=\"name western\"> D R Mani </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id74\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> <div hidden=\"hidden\" id=\"id74\"> <h3> <span class=\"name western\"> Michael A Gillette </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div class=\"p\"> <sup> 21 </sup> Lead author </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> </div> </div> <sup> 1, </sup> <sup> 18, </sup> <sup> 21, </sup> <sup> * </sup> , <span class=\"on-behalf-of\"> Clinical Proteomic Tumor Analysis Consortium </span> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"A1\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div id=\"A2\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div id=\"A3\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div id=\"A4\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A5\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div id=\"A6\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div id=\"A7\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div id=\"A8\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div id=\"A9\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div id=\"A10\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A11\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div id=\"A12\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A13\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A14\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div id=\"A15\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div id=\"A16\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div id=\"A17\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div id=\"A18\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div id=\"A19\"> <sup> 19 </sup> Equal contribution </div> <div id=\"A20\"> <sup> 20 </sup> Equal contribution </div> <div id=\"A21\"> <sup> 21 </sup> Lead author </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"FN1\"> <p id=\"P1\"> Author Contributions </p> <p id=\"P2\"> <strong> Conception </strong> : SS, SAC, MAG; <strong> Data Collection </strong> : SS, MHK, SCA, EJB; <strong> Data analysis </strong> : SS, KK, PMJB, SRS, FP, CKS, YD, BR,MHK, SVV, AK, JTL, EJJ, TO, YG, VS, KEC, DIH, MPC, SC, XS, JJ, WL, KL, BW, YL,ZHG, MES, OB, PV, SA, TS, MC, CKM, YZ, FVL, CH, MAW, SMD, KRC, DRM; <strong> QC </strong> : CJN, GH, JS, QKL, MT; <strong> IHC and analysis </strong> : RS, TV, MGR, ERP, SAM, CJN, GH, JS, KL; <strong> Manuscript writing and editing </strong> : SS and MAG with help from all authors <strong> Supervision </strong> : SS, LD, DF, KR, MM, HR, MT, AIR, PW, AIN, SAC, BZ, DRM, MAG </p> </div> <div class=\"fn\" id=\"FN2\"> <p id=\"P3\"> Secondary Authors </p> <p id=\"P4\"> Alex Green, Alfredo Molinolo, Alicia Francis, Amanda G. Paulovich, Andrii Karnuta, Antonio Colaprico, Barbara Hindenach, Barbara L. Pruetz, Bartosz Kubisa, Brian J. Druker, Carissa Huynh, Charles A. Goldthwaite Jr., Chet Birger, Christopher R. Kinsinger, Corbin D. Jones, Dan Rohrer, Dana R. Valley, Daniel W. Chan, David Chesla, Donna Hansel, Elena V. Ponomareva, Elizabeth Duffy, Eric Burks, Eric E. Schadt, Eugene S. Fedorov, Eunkyung An, Fei Ding, George D. Wilson, Harsh Batra, Hui Zhang, Jennifer E. Maas, Jennifer Eschbacher, Karen A. Ketchum, Karin D. Rodland, Katherine A. Hoadley, Kei Suzuki, Ki Sung Um, Liqun Qi, Lori Bernard, Maciej Wiznerowicz, Małgorzata Wojtyś, Marcin J. Domagalski, Matthew J. Ellis, Maureen A. Dyer, Melissa Borucki, Meenakshi Anurag, Michael J. Birrer, Midie Xu, Mikhail Krotevich, Nancy Roche, Nathan J. Edwards, Negin Vatanian, Neil R. Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Pamela Grady, Peter B. McGarvey, Pushpa Hariharan, Ratna R. Thangudu, Rebecca Montgomery, Renganayaki Pandurengan, Richard D. Smith, Robert J. Welsh, Sailaja Mareedu, Samuel H. Payne, Sandra Cottingham, Shilpi Singh, Shirley X. Tsang, Shuang Cai, Stacey Gabriel, Tao Liu, Tara Hiltke, Tanmayi Vashist, Thomas Bauer, Volodymyr Sovenko, Warren G. Tourtellotte, Weiping Ma, William Bocik, Wohaib Hasan, Xiaojun Jing, Ximing Tang, Yuxing Liao, Yvonne, Shutack, Zhen Zhang, Ziad Hanhan </p> </div> <div class=\"fn\" id=\"CR1\"> <sup> * </sup> <p class=\"display-inline\"> Corresponding authors: <span> shankha@broadinstitute.org </span> (S.S.), <span> scarr@broad.mit.edu </span> (S.A.C), <span> bing.zhang@bcm.edu </span> (B.Z.), <span> manidr@broadinstitute.org </span> (D.R.M.), <span> gillette@broadinstitute.org </span> (M.A.G.) </p> </div> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC8475722 NIHMSID: NIHMS1730863 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34358469/\"> 34358469 </a> </div> <div aria-label=\"Related or updated information about this article\" class=\"ra xbox p\" role=\"complementary\"> <div> The publisher\\'s version of this article is available at <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> Cell </a> </div> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"ABS1\"> <h2> Summary </h2> <p id=\"P5\"> Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. </p> <section class=\"kwd-group\" id=\"kwd-group1\"> <p> <strong> Keywords: </strong> Lung Cancer, Squamous, Proteogenomics, Proteomics, Genomics, Protein, Phosphorylation, Acetylation, Ubiquitylation, CPTAC </p> </section> </section> <section class=\"abstract\" id=\"ABS2\"> <h2> In Brief </h2> <p id=\"P6\"> Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential therapeutic targets. </p> </section> <section class=\"abstract\" id=\"ABS3\"> <h2> Graphical Abstract </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> </section> <section id=\"S1\"> <h2 class=\"pmc_sec_title\"> Introduction </h2> <p id=\"P8\"> Lung cancer is the leading cause of cancer-associated mortality ( <a aria-describedby=\"R15\" class=\"usa-link\" href=\"#R15\"> Bray et al., 2018 </a> ; <a aria-describedby=\"R158\" class=\"usa-link\" href=\"#R158\"> Siegel et al., 2020 </a> ). Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ; <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as <em> CDKN2A </em> ) are found in upwards of 60% but targeting these clinically has largely failed ( <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). While other therapeutic regimens are being explored as part of the LungMAP consortium ( <a aria-describedby=\"R7\" class=\"usa-link\" href=\"#R7\"> Ardini-Poleske et al., 2017 </a> ), to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC ( <a aria-describedby=\"R76\" class=\"usa-link\" href=\"#R76\"> Karachaliou et al., 2018 </a> ; <a aria-describedby=\"R137\" class=\"usa-link\" href=\"#R137\"> Paz-Ares et al., 2018 </a> ). Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. </p> <p id=\"P9\"> We characterized 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. </p> </section> <section id=\"S2\"> <h2 class=\"pmc_sec_title\"> Results </h2> <section id=\"S3\"> <h3 class=\"pmc_sec_title\"> Proteogenomic landscape of LSCC </h3> <p id=\"P10\"> We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> , <a class=\"usa-link\" href=\"#SD1\"> S1A </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> ). Proteomics data were generated using TMT11-based multiplexing linked by a common reference ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1A </a> ) and demonstrated high technical quality ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1B </a> – <a class=\"usa-link\" href=\"#SD1\"> C </a> ). The cohort demographics are summarized in <a class=\"usa-link\" href=\"#F1\"> Figure 1B </a> . Most self-reported never-smokers displayed high genomic smoking scores ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1D </a> ), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). The genomic landscape reaffirmed previously reported somatic alterations in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1C </a> ). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1E </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F1\"> <h4 class=\"obj_head\"> Figure 1: Proteogenomic Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0002.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P373\"> A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. </p> <p id=\"P374\"> B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. </p> <p id=\"P375\"> C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value &lt; 0.1) in this dataset are highlighted in red font. </p> <p id=\"P376\"> D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR &lt;0.01) positive and negative correlations, respectively. </p> <p id=\"P377\"> E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q&lt;0.25) and <em> cis </em> -effects in both mRNA and protein (FDR&lt;0.05). </p> <p id=\"P378\"> F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the <em> FGFR1 </em> gene (GISTIC thresholded value =2). </p> <p id=\"P379\"> G. Genes whose DNA methylation was significantly associated with cascading <em> cis </em> regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the <em> cis </em> -effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. </p> <p id=\"P380\"> See also <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> </p> </figcaption> </figure> <p id=\"P11\"> Proteogenomic data helped annotate the impact of copy number alteration (CNA) events ( <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> , <a class=\"usa-link\" href=\"#F1\"> E </a> ). Of 5,523 significant (FDR&lt;0.01) <em> cis </em> CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 “cancer associated genes” (CAG) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). We identified six amplified (including <em> WHSC1L1, CCND1 </em> , and <em> SOX2 </em> ) and 29 deleted (including <em> NCOR1 </em> , <em> SETD2 </em> , and <em> CBL </em> ) CAGs in significant focal events (Q&lt;0.25) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ). Intriguingly, <em> WHSC1L1 </em> ( <em> NSD3 </em> ) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses <em> FGFR1 </em> . Therapies targeted against FGFR1 have been unsuccessful ( <a aria-describedby=\"R108\" class=\"usa-link\" href=\"#R108\"> Lim et al., 2016 </a> ; <a aria-describedby=\"R189\" class=\"usa-link\" href=\"#R189\"> Weeden et al., 2015 </a> ). Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon ( <a class=\"usa-link\" href=\"#F1\"> Figure 1F </a> ), nominating it as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes ( <a aria-describedby=\"R205\" class=\"usa-link\" href=\"#R205\"> Yuan et al., 2021 </a> ). </p> <p id=\"P12\"> To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ) to perturbation profiles from the Connectivity Map (CMap) ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ). Six CNAs ( <em> IKBKAP </em> , <em> PIN1 </em> , <em> DNAJB1 </em> , <em> IL18 </em> , <em> NR2F6 </em> , <em> AKAP </em> ) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher’s exact p&lt;0.01) with clinical metadata. <em> IL-18 </em> deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (upper panel)), consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC ( <a aria-describedby=\"R175\" class=\"usa-link\" href=\"#R175\"> Timperi et al., 2017 </a> ). <em> NR2F6 </em> amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade ( <a aria-describedby=\"R83\" class=\"usa-link\" href=\"#R83\"> Klepsch et al., 2018 </a> ). </p> <p id=\"P13\"> LSCC tumor DNA showed overall hypermethylation relative to NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1G </a> ), with tumors separating into CIMP (CpG island methylator phenotype) clusters ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1H </a> ). Multi-omic data allowed the identification of “cascading” promoter methylation <em> cis </em> -effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR&lt;0.1) <em> cis </em> -effects in tumors ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), 20 were also differentially expressed (FDR&lt;0.01) between tumors and NATs ( <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> ). While most were newly associated with LSCC, methylation-driven expression of <em> FAM110A </em> , <em> PTGES3 </em> , <em> PLAU </em> and <em> SLC16A3 </em> ( <a aria-describedby=\"R35\" class=\"usa-link\" href=\"#R35\"> Faubert et al., 2017 </a> ; <a aria-describedby=\"R44\" class=\"usa-link\" href=\"#R44\"> Gao et al., 2018 </a> ; <a aria-describedby=\"R79\" class=\"usa-link\" href=\"#R79\"> Kikuchi et al., 2012 </a> ; <a aria-describedby=\"R105\" class=\"usa-link\" href=\"#R105\"> Liang et al., 2013 </a> ; <a aria-describedby=\"R136\" class=\"usa-link\" href=\"#R136\"> Payen et al., 2020 </a> ; <a aria-describedby=\"R157\" class=\"usa-link\" href=\"#R157\"> Showe et al., 2009 </a> ) has been implicated in lung and other cancers. </p> </section> <section id=\"S4\"> <h3 class=\"pmc_sec_title\"> Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched </h3> <p id=\"P14\"> We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the entire cohort ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> ). The five resulting multi-omic subtypes ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ) were named based on their predominant pathway associations and similarities to previously defined RNA clusters ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ). Significant associations between NMF subtypes and cohort metadata are presented in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . Two NMF subtypes, “Basal-Inclusive” (B-I) and “Epithelial to Mesenchymal Transition-enriched” (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni correction (Fisher’s p&lt;0.01, <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in <em> KEAP1 </em> , <em> CUL3 </em> and <em> NFE2L2 </em> genes and high-level amplification (GISTIC threshold value = 2) of <em> SOX2 </em> and <em> TP63 </em> ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD8\"> Table S2 </a> ) was labeled the “Classical” subtype, consistent with TCGA nomenclature and with a previous publication ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ). These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> ). A fourth NMF subtype was designated “Inflamed-Secretory” (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, “Proliferative-Primitive” (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a “mixed” subgroup, since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival ( <a class=\"usa-link\" href=\"#F2\"> Figure 2C </a> ), an observation also made in a recent proteogenomic analysis of glioblastoma ( <a aria-describedby=\"R187\" class=\"usa-link\" href=\"#R187\"> Wang et al., 2021 </a> ). The mixed class was associated in the current cohort with increased frequency of <em> SOX2 </em> amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. </p> <figure class=\"fig xbox font-sm\" id=\"F2\"> <h4 class=\"obj_head\"> Figure 2: LSCC Molecular Subtypes and Associations. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0003.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P381\"> A. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). </p> <p id=\"P382\"> B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. </p> <p id=\"P383\"> C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). </p> <p id=\"P384\"> D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. </p> <p id=\"P385\"> E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8×10 <sup> −14 </sup> ). </p> <p id=\"P386\"> F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5×10 <sup> −6 </sup> for PDGFRB and 2.7×10 <sup> −7 </sup> for ROR2. </p> <p id=\"P387\"> G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. </p> <p id=\"P388\"> See also <a class=\"usa-link\" href=\"#SD2\"> Figure S2 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD10\"> 4 </a> </p> </figcaption> </figure> <p id=\"P15\"> We compared B-I ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ) and EMT-E ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 ( <a aria-describedby=\"R56\" class=\"usa-link\" href=\"#R56\"> Happel et al., 2009 </a> ) and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis ( <a aria-describedby=\"R64\" class=\"usa-link\" href=\"#R64\"> Hsu et al., 2020 </a> ), and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance ( <a aria-describedby=\"R67\" class=\"usa-link\" href=\"#R67\"> Hubaux et al., 2015 </a> ; <a aria-describedby=\"R100\" class=\"usa-link\" href=\"#R100\"> Lewandowski and Piwnica-Worms, 2014 </a> ). Transcription activator PBX3, which is upregulated only at the protein level ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, ( <a aria-describedby=\"R94\" class=\"usa-link\" href=\"#R94\"> Lamprecht et al., 2018 </a> ) including in laryngeal squamous cell carcinoma ( <a aria-describedby=\"R198\" class=\"usa-link\" href=\"#R198\"> Wu et al., 2020 </a> ). Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2D </a> ), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2E </a> ) and with the xCell-based deconvolution signature ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2F </a> ). This correlation was not observed in paired NATs, suggesting a tumor-specific role. </p> <p id=\"P16\"> In addition to a strong EMT signature ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), the EMT-E subtype showed upregulated VCAN ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ), a tumor-promoting target of SNAIL ( <a aria-describedby=\"R207\" class=\"usa-link\" href=\"#R207\"> Zhang et al., 2019a </a> ) expressed in cancer-associated fibroblasts (CAFs) ( <a aria-describedby=\"R24\" class=\"usa-link\" href=\"#R24\"> Chida et al., 2016 </a> ), and FHL3, a protein that stabilizes EMT-associated transcription factors ( <a aria-describedby=\"R104\" class=\"usa-link\" href=\"#R104\"> Li et al., 2020 </a> ) ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in <a class=\"usa-link\" href=\"#F2\"> Figure 2D </a> – <a class=\"usa-link\" href=\"#F2\"> E </a> , coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a “fibroblast proliferation” signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker α-smooth muscle actin (α-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2G </a> ), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either CK or α-SMA alone. TGFβ is a well-established and potent inducer of EMT ( <a aria-describedby=\"R166\" class=\"usa-link\" href=\"#R166\"> Su et al., 2020 </a> ), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFβ inhibitors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2H </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4H </a> ). </p> <p id=\"P17\"> Integrating our global proteome data with a prior LSCC dataset ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> – <a class=\"usa-link\" href=\"#SD2\"> J </a> ). </p> </section> <section id=\"S5\"> <h3 class=\"pmc_sec_title\"> NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. </h3> <p id=\"P18\"> Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p &lt;0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E ( <a class=\"usa-link\" href=\"#F2\"> Figure 2F </a> ), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state ( <a class=\"usa-link\" href=\"#F2\"> Figure 2G </a> ) ( <a aria-describedby=\"R60\" class=\"usa-link\" href=\"#R60\"> Herreño et al., 2019 </a> ; <a aria-describedby=\"R130\" class=\"usa-link\" href=\"#R130\"> Niu et al., 2012 </a> ). As summarized in <a class=\"usa-link\" href=\"#SD2\"> Figure S2K </a> , we found phosphosite-based evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR ( <a aria-describedby=\"R169\" class=\"usa-link\" href=\"#R169\"> Tam et al., 2013 </a> ), both targetable ( <a aria-describedby=\"R150\" class=\"usa-link\" href=\"#R150\"> Roskoski 2018 </a> ; <a aria-describedby=\"R25\" class=\"usa-link\" href=\"#R25\"> Debebe and Rathmell 2015 </a> ) and recently implicated in the progression from a hybrid E/M to a highly mesenchymal state ( <a aria-describedby=\"R87\" class=\"usa-link\" href=\"#R87\"> Kröger et al., 2019 </a> ). Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. </p> </section> <section id=\"S6\"> <h3 class=\"pmc_sec_title\"> Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable </h3> <p id=\"P19\"> We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs ( <em> cis </em> -effects) and on a set of cancer associated genes ( <em> trans </em> -effects) ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ). Significant (FDR&lt;0.05) <em> Cis/Trans </em> pairs are shown in <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> – <a class=\"usa-link\" href=\"#F3\"> D </a> . <em> RB1 </em> mutated tumors had upregulated cell cycle-related protein expression. <em> NOTCH1 </em> mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK ( <a aria-describedby=\"R69\" class=\"usa-link\" href=\"#R69\"> Ishitani et al., 2010 </a> ) and GSK3B ( <a aria-describedby=\"R38\" class=\"usa-link\" href=\"#R38\"> Foltz et al., 2002 </a> ; <a aria-describedby=\"R71\" class=\"usa-link\" href=\"#R71\"> Jin et al., 2009 </a> ; <a aria-describedby=\"R210\" class=\"usa-link\" href=\"#R210\"> Zheng and Conner, 2018 </a> ) inhibit NOTCH1, recently described as a tumor suppressor in LSCC ( <a aria-describedby=\"R160\" class=\"usa-link\" href=\"#R160\"> Sinicropi-Yao et al., 2019 </a> ), suggesting that both might inhibit NOTCH1 downstream signaling in tumors with <em> NOTCH1 </em> mutations. <em> ARID1A </em> is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden ( <a aria-describedby=\"R156\" class=\"usa-link\" href=\"#R156\"> Shen et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3A </a> ) and led to protein downregulation, associated with worse overall survival in NSCLC ( <a aria-describedby=\"R68\" class=\"usa-link\" href=\"#R68\"> Hung et al., 2020 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F3\"> <h4 class=\"obj_head\"> Figure 3: Impact of Somatic Mutations on Proteogenomic Features. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0004.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F3/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P389\"> A. Significant (Wilcoxon FDR&lt;0.05) <em> cis </em> - (circles) and <em> trans </em> -effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green. </p> <p id=\"P390\"> B. Similar to panel A but showing phosphosites. </p> <p id=\"P391\"> C. Similar to panel A but showing acetylsites. </p> <p id=\"P392\"> D. Similar to panel A but showing ubiquitylsites. </p> <p id=\"P393\"> E. CNA data for <em> CDKN2A </em> and <em> RB1 </em> was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CDKN2A </em> genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these <em> CDKN2A/p16 </em> aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of <em> CCND1–3 </em> , <em> CDK4 </em> , and <em> CDK6 </em> were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of <em> CDKN2A </em> are not included. </p> <p id=\"P394\"> F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in <em> KEAP1 </em> , <em> CUL3 </em> , or <em> NFE2L2 </em> ) versus NRF2 WT tumors (no NRF2 pathway aberration). </p> <p id=\"P395\"> G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype. </p> <p id=\"P396\"> H. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. </p> <p id=\"P397\"> I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR&lt;0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. </p> <p id=\"P398\"> See also <a class=\"usa-link\" href=\"#SD3\"> Figure S3 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P20\"> Paradoxically, mutations in <em> CDKN2A, </em> proposed as a potential biomarker for CDK4/6 inhibitors ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ), resulted in increased RNA expression ( <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> ). We analyzed the effects of genetic and epigenetic alterations of <em> CDKN2A </em> on both its major isoforms, p16INK4a (p16) and p14ARF (p14) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). As expected, its homozygous deletion resulted in the loss of expression of both isoforms ( <a class=\"usa-link\" href=\"#F3\"> Figure 3E </a> ). However, only eight of the 58 <em> CDKN2A </em> wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with <em> CDKN2A </em> /p16 missense mutations (log2(FPKM+1)&gt;2). Many of those <em> CDKN2A </em> /p16 mutations were unlikely to result in nonsense-mediated decay ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3B </a> ), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had <em> RB1 </em> mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes ( <a aria-describedby=\"R85\" class=\"usa-link\" href=\"#R85\"> Knudsen et al., 2020 </a> ). This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ), there was no association between the amplification of <em> CCND2 </em> , <em> CCND3 </em> , <em> CDK4 </em> , or <em> CDK6 </em> and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CCND1 </em> amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3C </a> ). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in <em> CDKN2A </em> and copy number alterations in <em> CCND1 </em> and <em> CDKN2A </em> ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib ( <a aria-describedby=\"R135\" class=\"usa-link\" href=\"#R135\"> Patnaik et al., 2016 </a> ) and palbociclib ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R31\" class=\"usa-link\" href=\"#R31\"> Edelman et al., 2019 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ). While we found that <em> CCND1 </em> amplification resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with <em> CCND1 </em> amplification provides a potential explanation for some patients’ lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 inhibition. </p> </section> <section id=\"S7\"> <h3 class=\"pmc_sec_title\"> NRF2 pathway activation in tumors with and without NRF2 pathway mutations </h3> <p id=\"P21\"> LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> . NFE2L2 phosphorylation was observed in samples with any of these mutations ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3E </a> ). LSCC tumors were annotated as “NRF2 pathway mutated” (n=33; with either one hit [n=28] or two hits [n=5] in <em> NFE2L2 </em> , <em> KEAP1 </em> , and <em> CUL3 </em> ), or “NRF2 WT” (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes ( <a class=\"usa-link\" href=\"#F3\"> Figure 3F </a> , <a class=\"usa-link\" href=\"#SD3\"> Figure S3F </a> – <a class=\"usa-link\" href=\"#SD3\"> G </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3H </a> ). Using the proteogenomic signature ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) that defined <em> NFE2L2 </em> mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype ( <a class=\"usa-link\" href=\"#F3\"> Figure 3G </a> ). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein ( <a class=\"usa-link\" href=\"#F3\"> Figure 3H </a> ) and PTM-SEA-derived activity ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) were significantly upregulated (FDR&lt;0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes ( <a aria-describedby=\"R70\" class=\"usa-link\" href=\"#R70\"> Jimenez-Blasco et al., 2015 </a> ) and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ), <em> KEAP1 </em> mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. </p> </section> <section id=\"S8\"> <h3 class=\"pmc_sec_title\"> Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC </h3> <p id=\"P22\"> Chromosome 3q, which harbors key squamous differentiation markers <em> SOX2 </em> and <em> TP63 </em> ( <a aria-describedby=\"R143\" class=\"usa-link\" href=\"#R143\"> Qian and Massion, 2008 </a> ), showed the most dramatic arm-level amplification in this cohort ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4A </a> ). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). LSCC cell lines amplified for <em> TP63 </em> and <em> SOX2 </em> were highly dependent upon them, supporting oncogene addiction ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F4\"> <h4 class=\"obj_head\"> Figure 4: Proteogenomic Impact of Chromosome 3q Amplification. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0005.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F4/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P399\"> A. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. </p> <p id=\"P400\"> B. Frequency distribution of ΔNp63α RNA expression in tumors and NATs. </p> <p id=\"P401\"> C. Differential protein expression in samples classified as ΔNp63α-low vs -high based on ΔNp63α transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. </p> <p id=\"P402\"> D. Differential expression of microRNAs in ΔNp63α-low vs -high samples. </p> <p id=\"P403\"> E. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. </p> <p id=\"P404\"> F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. </p> <p id=\"P405\"> G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3×10 <sup> −08 </sup> , Correlation = −0.53) and DNA replication (bottom, p=1.1×10 <sup> −06 </sup> , Correlation= 0.47) scores. </p> <p id=\"P406\"> H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR&lt;0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. </p> <p id=\"P407\"> I. Pearson correlation (p= 5.2×10 <sup> −07 </sup> , Correlation = −0.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES. </p> <p id=\"P408\"> See also <a class=\"usa-link\" href=\"#SD4\"> Figure S4 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P23\"> As previously reported ( <a aria-describedby=\"R20\" class=\"usa-link\" href=\"#R20\"> Campbell et al., 2018 </a> ; <a aria-describedby=\"R113\" class=\"usa-link\" href=\"#R113\"> Lo Iacono et al., 2011 </a> ), ΔNp63α was the dominant TP63 isoform in this dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4C </a> ) and was highly correlated with TP63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4D </a> ). Its amplification was associated with improved survival in the TCGA dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4E </a> ). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 “Np63-low” samples histologically confirmed to be LSCC showed no elevation at RNA or protein levels ( <a class=\"usa-link\" href=\"#F4\"> Figure 4B </a> ). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group ( <a class=\"usa-link\" href=\"#F4\"> Figure 4C </a> ). Notably, a substantial number of LSCC cell lines also had low Np63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4F </a> ), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Survivin promotes cell proliferation and G2M checkpoints ( <a aria-describedby=\"R193\" class=\"usa-link\" href=\"#R193\"> Wheatley and Altieri, 2019 </a> ), evident in Np63-low samples ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4H </a> ). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in NSCLC ( <a aria-describedby=\"R48\" class=\"usa-link\" href=\"#R48\"> Giaccone et al., 2009 </a> ; <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ; <a aria-describedby=\"R101\" class=\"usa-link\" href=\"#R101\"> Li et al., 2019a </a> ), but have shown modest or no improvement ( <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ). Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin inhibition. </p> <p id=\"P24\"> Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC ( <a aria-describedby=\"R97\" class=\"usa-link\" href=\"#R97\"> Lebanony et al., 2009 </a> ) ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4I </a> ), was associated with decreased promoter DNA methylation ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4J </a> ) and elevation of ΔNp63α, a transcriptional regulator of this miRNA ( <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4D </a> , <a class=\"usa-link\" href=\"#SD4\"> Figure S4K </a> ). As expected, the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN ( <a aria-describedby=\"R18\" class=\"usa-link\" href=\"#R18\"> Cai et al., 2013 </a> ; <a aria-describedby=\"R53\" class=\"usa-link\" href=\"#R53\"> Gregory et al., 2008 </a> ; <a aria-describedby=\"R172\" class=\"usa-link\" href=\"#R172\"> Tellez et al., 2011 </a> ; <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ). TP73 positively correlated with miR-205 at the transcript ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ) and still more strongly at the protein level ( <a class=\"usa-link\" href=\"#F4\"> Figure 4F </a> ) and with TP63 protein but not RNA ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4L </a> ). TP73 overexpression mediates ΔNp63α-dependent cell survival in squamous cell carcinoma ( <a aria-describedby=\"R28\" class=\"usa-link\" href=\"#R28\"> DeYoung et al., 2006 </a> ), suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC ( <a aria-describedby=\"R126\" class=\"usa-link\" href=\"#R126\"> Natan and Joerger, 2012 </a> ). Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication ( <a class=\"usa-link\" href=\"#F4\"> Figure 4G </a> ). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in previous studies ( <a aria-describedby=\"R110\" class=\"usa-link\" href=\"#R110\"> Lindsay et al., 2011 </a> ; <a aria-describedby=\"R164\" class=\"usa-link\" href=\"#R164\"> Srivastava et al., 2018 </a> ; <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). Of note, although ΔNp63α and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, ΔNp63α, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype ( <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ), and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. </p> <p id=\"P25\"> <em> SOX2 </em> , a recurrently amplified squamous cell marker often co-amplified with <em> TP63 </em> , was also overexpressed in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). <em> SOX2 </em> is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression ( <a aria-describedby=\"R206\" class=\"usa-link\" href=\"#R206\"> Zhang et al., 2013 </a> ) and its inhibition in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT04350463\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT04350463 </a> ). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer ( <a aria-describedby=\"R144\" class=\"usa-link\" href=\"#R144\"> Ramadoss et al., 2017 </a> ), and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression ( <a aria-describedby=\"R203\" class=\"usa-link\" href=\"#R203\"> Yomtoubian et al., 2020 </a> ). EZH2 dependency was observed in SOX2 amplified LSCC cell lines ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> ). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) suggest stemness-like features in these tumors ( <a aria-describedby=\"R78\" class=\"usa-link\" href=\"#R78\"> Keysar et al., 2017 </a> ; <a aria-describedby=\"R112\" class=\"usa-link\" href=\"#R112\"> Liu et al., 2016 </a> ; <a aria-describedby=\"R134\" class=\"usa-link\" href=\"#R134\"> Patel et al., 2008 </a> ). Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ), and negative correlation was seen between SOX2 and JAK-STAT signaling ( <a class=\"usa-link\" href=\"#F4\"> Figure 4I </a> ). </p> </section> <section id=\"S9\"> <h3 class=\"pmc_sec_title\"> Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism </h3> <p id=\"P26\"> Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ) associated with country of origin and ethnicity (Fisher’s exact p&lt;0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5A </a> – <a class=\"usa-link\" href=\"#SD14\"> B </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F5\"> <h4 class=\"obj_head\"> Figure 5: Ubiquitylation landscape in LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0006.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F5/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P409\"> A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR&lt;0.01). Enriched pathways and molecular and clinical annotations are indicated. </p> <p id=\"P410\"> B. Number of K-GG sites showing significant correlations (FDR&lt; 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. </p> <p id=\"P411\"> C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. </p> <p id=\"P412\"> D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal−Wallis, p = 2.8×10 <sup> −05 </sup> ) association is seen between HERC5 abundance and immune subtypes. </p> <p id=\"P413\"> E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). </p> <p id=\"P414\"> F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal−Wallis p-values are indicated in the respective plots. </p> <p id=\"P415\"> G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. </p> <p id=\"P416\"> H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)&gt;1 or Hypo: &lt;−1, FDR&lt;0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. “k” represents modified lysine. </p> <p id=\"P417\"> I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p&gt;0.05. </p> <p id=\"P418\"> See also <a class=\"usa-link\" href=\"#SD14\"> Figure S5 </a> and <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> </p> </figcaption> </figure> <p id=\"P27\"> To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in <a class=\"usa-link\" href=\"#F5\"> Figure 5B </a> and <a class=\"usa-link\" href=\"#SD14\"> Figure S5C </a> . HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in humans ( <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); hence the numerous positively correlated Ub/UbL sites (n=474; FDR&lt;0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 abundance ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5D </a> ). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5E </a> ). We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 ( <a class=\"usa-link\" href=\"#F5\"> Figure 5C </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5F </a> ). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5G </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5H </a> ). ISG15 modifications have been described in these glycolytic enzymes ( <a aria-describedby=\"R3\" class=\"usa-link\" href=\"#R3\"> Albert et al., 2018 </a> ; <a aria-describedby=\"R49\" class=\"usa-link\" href=\"#R49\"> Giannakopoulos et al., 2005 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ), and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis ( <a aria-describedby=\"R4\" class=\"usa-link\" href=\"#R4\"> Alcalá et al., 2020 </a> ). ISGylation can be induced by interferon in cancer ( <a aria-describedby=\"R171\" class=\"usa-link\" href=\"#R171\"> Tecalco-Cruz and Cruz-Ramos, 2018 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5I </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5J </a> ), and HERC5 was increased in tumors of the IFN-gamma dominant subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5D </a> ). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion ( <a aria-describedby=\"R208\" class=\"usa-link\" href=\"#R208\"> Zhang et al., 2019b </a> ). Collectively, our data supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. </p> <p id=\"P28\"> This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ) to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p&lt;0.05) ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket ( <a aria-describedby=\"R188\" class=\"usa-link\" href=\"#R188\"> Wang et al., 2015 </a> ; <a aria-describedby=\"R209\" class=\"usa-link\" href=\"#R209\"> Zhao et al., 2014 </a> )( <a aria-describedby=\"R200\" class=\"usa-link\" href=\"#R200\"> Xiong et al., 2011 </a> ) ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator ( <a class=\"usa-link\" href=\"#F5\"> Figures 5F </a> ). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37) ( <a aria-describedby=\"R11\" class=\"usa-link\" href=\"#R11\"> Bando et al., 2005 </a> ; <a aria-describedby=\"R201\" class=\"usa-link\" href=\"#R201\"> Yang and Lu, 2013 </a> ). Upregulation of a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs ( <a aria-describedby=\"R26\" class=\"usa-link\" href=\"#R26\"> DeBerardinis and Chandel, 2016 </a> ; <a aria-describedby=\"R61\" class=\"usa-link\" href=\"#R61\"> Hitosugi and Chen, 2014 </a> ). While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. </p> <p id=\"P29\"> Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR&lt;0.01). The TXN1 activator TXNRD1 ( <a aria-describedby=\"R17\" class=\"usa-link\" href=\"#R17\"> Cadenas et al., 2010 </a> ) was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1 ( <a aria-describedby=\"R124\" class=\"usa-link\" href=\"#R124\"> Morrison et al., 2014 </a> ), was decreased ( <a class=\"usa-link\" href=\"#F5\"> Figure 5F </a> ). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p&lt;2.2×10 <sup> −16 </sup> ), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial cancers ( <a aria-describedby=\"R42\" class=\"usa-link\" href=\"#R42\"> Fu et al., 2017 </a> ), including NSCLC ( <a aria-describedby=\"R36\" class=\"usa-link\" href=\"#R36\"> Fernandes et al., 2009 </a> ). The small proportion of glucose directed to the tumor mitochondria ( <a aria-describedby=\"R34\" class=\"usa-link\" href=\"#R34\"> Fan et al., 2019 </a> ) and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis) ( <a aria-describedby=\"R9\" class=\"usa-link\" href=\"#R9\"> Baeza et al., 2015 </a> ; <a aria-describedby=\"R43\" class=\"usa-link\" href=\"#R43\"> Gandhi and Das, 2019 </a> ; <a aria-describedby=\"R58\" class=\"usa-link\" href=\"#R58\"> Hebert et al., 2013 </a> ). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). PTM-based modulation of these key metabolic and ROS pathways is summarized in <a class=\"usa-link\" href=\"#F5\"> Figure 5G </a> . </p> <p id=\"P30\"> As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. <a class=\"usa-link\" href=\"#F5\"> Figure 5H </a> shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5I </a> ), named in part for its generally high proliferation ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. </p> </section> <section id=\"S10\"> <h3 class=\"pmc_sec_title\"> Immune landscape and regulation in LSCC </h3> <p id=\"P31\"> Consensus clustering based on the xCell ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) signatures ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> C </a> ), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher’s exact p=6.371×10 <sup> −09 </sup> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> ). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses ( <a class=\"usa-link\" href=\"#F6\"> Figures 6A </a> , <a class=\"usa-link\" href=\"#F6\"> C </a> , <a class=\"usa-link\" href=\"#SD5\"> S6C </a> , <a class=\"usa-link\" href=\"#SD5\"> D </a> ), and showed upregulation of immune-related pathways ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Notably, NATs did not display consonant immune signaling ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6B </a> ). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> ). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6B </a> ). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6E </a> ) </p> <figure class=\"fig xbox font-sm\" id=\"F6\"> <h4 class=\"obj_head\"> Figure 6: Immune Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0007.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F6/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P419\"> A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). </p> <p id=\"P420\"> B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. </p> <p id=\"P421\"> C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types’ distributions observed across the different immune clusters. </p> <p id=\"P422\"> D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR &lt; 0.1). </p> <p id=\"P423\"> E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. </p> <p id=\"P424\"> F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P425\"> G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) and the third pathway scores based on global protein abundance. </p> <p id=\"P426\"> H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. </p> <p id=\"P427\"> I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also <a class=\"usa-link\" href=\"#SD5\"> Figure S6 </a> and <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> </p> </figcaption> </figure> <p id=\"P32\"> Rho GTPase signaling was upregulated in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation ( <a aria-describedby=\"R45\" class=\"usa-link\" href=\"#R45\"> Garcia-Mata et al., 2011 </a> ), was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial enzymes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly ( <a aria-describedby=\"R90\" class=\"usa-link\" href=\"#R90\"> Kuhlmann et al., 2016a </a> ) by activating RhoA ( <a aria-describedby=\"R91\" class=\"usa-link\" href=\"#R91\"> Kuhlmann et al., 2016b </a> ). In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low ( <a aria-describedby=\"R52\" class=\"usa-link\" href=\"#R52\"> Gorvel et al., 1998 </a> ), and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead ( <a aria-describedby=\"R123\" class=\"usa-link\" href=\"#R123\"> Moissoglu et al., 2009 </a> ). In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators ( <a class=\"usa-link\" href=\"#F6\"> Figures 6F </a> , <a class=\"usa-link\" href=\"#SD5\"> S6F </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Notably, these proteins are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6G </a> ), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas ( <a aria-describedby=\"R84\" class=\"usa-link\" href=\"#R84\"> von Klot et al., 2017 </a> ). Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ) and immune-enriched networks ( <a aria-describedby=\"R140\" class=\"usa-link\" href=\"#R140\"> Peters et al., 2017 </a> ). </p> <p id=\"P33\"> We explored the immunological associations of the RTK CBPE scores described above ( <a class=\"usa-link\" href=\"#F2\"> Figure 2I </a> , <a class=\"usa-link\" href=\"#F2\"> J </a> ) in LSCC tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster ( <a class=\"usa-link\" href=\"#F6\"> Figure 6H </a> ) and consequently with immune-related pathways and cell types ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103 ( <a aria-describedby=\"R19\" class=\"usa-link\" href=\"#R19\"> Cammer et al., 2009 </a> ; <a aria-describedby=\"R30\" class=\"usa-link\" href=\"#R30\"> Dovas et al., 2009 </a> ; <a aria-describedby=\"R125\" class=\"usa-link\" href=\"#R125\"> Mouchemore et al., 2013 </a> ), and upregulation of AKAP13, among other immune-related proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6I </a> ). Tumor-associated macrophages (( <a aria-describedby=\"R23\" class=\"usa-link\" href=\"#R23\"> Cassetta and Pollard, 2018 </a> ), CSF1R signaling ( <a aria-describedby=\"R21\" class=\"usa-link\" href=\"#R21\"> Cannarile et al., 2017 </a> ), and AKAP13 ( <a aria-describedby=\"R29\" class=\"usa-link\" href=\"#R29\"> Diviani et al., 2016 </a> ) have each been proposed as anti-cancer targets. </p> </section> <section id=\"S11\"> <h3 class=\"pmc_sec_title\"> Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment </h3> <p id=\"P34\"> Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment ( <a class=\"usa-link\" href=\"#F7\"> Figure 7A </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). The top 50 of 502 differential proteins are shown in <a class=\"usa-link\" href=\"#SD6\"> Figure S7A </a> . In addition to 206 upregulated oncogenic proteins (OncoKB, <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7B </a> ), with the I-S samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS ( <a class=\"usa-link\" href=\"#F7\"> Figure 7B </a> , <a class=\"usa-link\" href=\"#SD6\"> Figure S7C </a> ). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/ </a> ), suggesting critical roles in key cellular transformation and proliferation processes ( <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F7\"> <h4 class=\"obj_head\"> Figure 7: Proteomic Features Related to Diagnosis, Prognosis, or Treatment. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0008.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F7/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P428\"> A. Differentially expressed proteins between tumors and NATs. </p> <p id=\"P429\"> B. Significantly increased proteins (larger font indicates &gt;4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. </p> <p id=\"P430\"> C. Genetic dependencies of 502 proteins (log2 FC &gt;2, FDR&lt;0.01 and NAs &lt;50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. </p> <p id=\"P431\"> D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. </p> <p id=\"P432\"> E. EGFR protein and tyrosine phosphorylation levels compared to <em> EGFR </em> copy number and an EGFR activity score (PROGENy). </p> <p id=\"P433\"> F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p&lt;0.05 </p> <p id=\"P434\"> G. GO Biological Process enrichment for proteins with increased phosphorylation in <em> EGFR </em> amplified samples compared to non-amplified samples. </p> <p id=\"P435\"> H. Summary roadmap figure partitioned into five major categories, indicated by different colors. </p> <p id=\"P436\"> See also <a class=\"usa-link\" href=\"#SD6\"> Figure S7 </a> and <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> </p> </figcaption> </figure> <p id=\"P35\"> There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell type of origin ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7D </a> ). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores ( <a class=\"usa-link\" href=\"#F7\"> Figure 7D </a> ). Immune-related deletions were less prominent in HNSCC and especially LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7E </a> ), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7F </a> ), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7G </a> ), although LUAD had many activating <em> EGFR </em> mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High <em> EGFR </em> amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy scoring ( <a class=\"usa-link\" href=\"#F7\"> Figure 7E </a> ) ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors ( <a class=\"usa-link\" href=\"#F7\"> Figure 7F </a> ). Furthermore, <em> EGFR </em> amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion ( <a class=\"usa-link\" href=\"#F7\"> Figure 7G </a> ). These findings and similar results recently reported in HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. <a class=\"usa-link\" href=\"#F7\"> Figure 7H </a> graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. </p> </section> </section> <section id=\"S12\"> <h2 class=\"pmc_sec_title\"> Discussion </h2> <p id=\"P36\"> In this study, we provide detailed proteogenomic profiles of treatment-naïve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. </p> <p id=\"P37\"> Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses ( <a aria-describedby=\"R57\" class=\"usa-link\" href=\"#R57\"> Haslam and Prasad, 2019 </a> ; <a aria-describedby=\"R59\" class=\"usa-link\" href=\"#R59\"> Herbst et al., 2018 </a> ); numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the downstream effects of <em> CDKN2A </em> mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. </p> <section id=\"S13\"> <h3 class=\"pmc_sec_title\"> Limitations of the study </h3> <p id=\"P38\"> Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity ( <a aria-describedby=\"R139\" class=\"usa-link\" href=\"#R139\"> Peng et al., 2020 </a> ; <a aria-describedby=\"R151\" class=\"usa-link\" href=\"#R151\"> Satpathy et al., 2020 </a> ), such as pairing microdissection with mass spectrometry ( <a aria-describedby=\"R99\" class=\"usa-link\" href=\"#R99\"> Le Large et al., 2020 </a> ) or using mass cytometry ( <a aria-describedby=\"R163\" class=\"usa-link\" href=\"#R163\"> Spitzer and Nolan, 2016 </a> ) will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the understanding and treatment of LSCC. </p> </section> </section> <section id=\"S14\"> <h2 class=\"pmc_sec_title\"> STAR★METHODS </h2> <section id=\"S16\"> <h3 class=\"pmc_sec_title\"> RESOURCE AVAILABILITY </h3> <section id=\"S17\"> <h4 class=\"pmc_sec_title\"> Lead contact </h4> <p id=\"P39\"> Further information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette ( <span> gillette@broadinstitute.org </span> ) </p> </section> <section id=\"S18\"> <h4 class=\"pmc_sec_title\"> Material availability </h4> <p id=\"P40\"> This study did not generate new unique reagents. </p> </section> <section id=\"S19\"> <h4 class=\"pmc_sec_title\"> Data and Code Availability </h4> <section id=\"S20\"> <h5 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h5> <ul class=\"list\" id=\"L2\"> <li> <p id=\"P41\"> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p id=\"P42\"> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"S21\"> <h5 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h5> <ul class=\"list\" id=\"L4\"> <li> <p id=\"P43\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p id=\"P44\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"S22\"> <h5 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h5> <ul class=\"list\" id=\"L6\"> <li> <p id=\"P45\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p id=\"P46\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p id=\"P47\"> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p id=\"P48\"> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p id=\"P49\"> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p id=\"P50\"> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"T1\"> <h6 class=\"obj_head\"> KEY RESOURCES TABLE. </h6> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/T1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> <section id=\"S23\"> <h3 class=\"pmc_sec_title\"> EXPERIMENTAL MODEL AND SUBJECT DETAILS </h3> <section id=\"S24\"> <h4 class=\"pmc_sec_title\"> Human Subjects </h4> <p id=\"P51\"> A total of 113 participants (90 males, 23 females, 40–88 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40–88. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC ( <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/ </a> ). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&amp;E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> </section> <section id=\"S25\"> <h3 class=\"pmc_sec_title\"> METHOD DETAILS </h3> <section id=\"S26\"> <h4 class=\"pmc_sec_title\"> Specimen Acquisition </h4> <p id=\"P52\"> The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than −140°C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. </p> </section> <section id=\"S27\"> <h4 class=\"pmc_sec_title\"> Sequencing sample preparation </h4> <p id=\"P53\"> Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol using Qiagen’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3–5mL) to serve as matched normal reference material. The Qubit™ dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification. </p> <section id=\"S28\"> <h5 class=\"pmc_sec_title\"> Whole exome Sequencing (WES) </h5> <section id=\"S29\"> <h6 class=\"pmc_sec_title\"> Library construction and Hybrid Selection </h6> <p id=\"P54\"> Library construction was performed as described in ( <a aria-describedby=\"R37\" class=\"usa-link\" href=\"#R37\"> Fisher et al., 2011 </a> ), with the following modifications: initial genomic DNA input into shearing was reduced from 3μg to 20–250ng in 50μL of solution. For adapter ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. </p> </section> <section id=\"S30\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P55\"> After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. </p> </section> </section> <section id=\"S31\"> <h5 class=\"pmc_sec_title\"> Whole genome sequencing (WGS) </h5> <section id=\"S32\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P56\"> An aliquot of genomic DNA (350ng in 50μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into 24-plexes. </p> <p id=\"P57\"> Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging. </p> </section> </section> </section> <section id=\"S33\"> <h4 class=\"pmc_sec_title\"> Array Based Methylation Analysis </h4> <p id=\"P58\"> The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt;850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. </p> </section> <section id=\"S34\"> <h4 class=\"pmc_sec_title\"> RNA and miRNA sequencing </h4> <section id=\"S35\"> <h5 class=\"pmc_sec_title\"> Quality Assurance and Control of RNA Analytes </h5> <p id=\"P59\"> All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt;7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer’s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. </p> </section> <section id=\"S36\"> <h5 class=\"pmc_sec_title\"> Total RNA Sequencing </h5> <p id=\"P60\"> Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. </p> </section> <section id=\"S37\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Library Construction </h5> <p id=\"P61\"> miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer’s instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). </p> </section> <section id=\"S38\"> <h5 class=\"pmc_sec_title\"> miRNA Sequencing </h5> <p id=\"P62\"> Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. </p> </section> </section> <section id=\"S39\"> <h4 class=\"pmc_sec_title\"> Mass Spectrometry methods </h4> <p id=\"P63\"> The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Acetyl-enrichment was performed as described before ( <a aria-describedby=\"R89\" class=\"usa-link\" href=\"#R89\"> Krug et al., 2020 </a> ) with modifications as indicated below. </p> <section id=\"S40\"> <h5 class=\"pmc_sec_title\"> Protein Extraction and Tryptic Digestion </h5> <p id=\"P64\"> Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 μg/mL aprotinin (Sigma, A6103), 10 μg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129–02541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. </p> </section> <section id=\"S41\"> <h5 class=\"pmc_sec_title\"> Common Reference (CR) Pool and Plex layout </h5> <section id=\"S42\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, and Acetyprotome: </h6> <p id=\"P65\"> The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. A common physical, rather than <em> in silico </em> reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data analyses. </p> <p id=\"P66\"> 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5 <sup> th </sup> tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. </p> <p id=\"P67\"> To ensure capacity for additional samples or experiments given a target input of 300 μg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 μg peptide / sample for individual sample analysis, an additional amount of 120 μg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 μg aliquots, dried down, and frozen at −80°C until use. </p> </section> <section id=\"S43\"> <h6 class=\"pmc_sec_title\"> Ubiquitylproteome: </h6> <p id=\"P68\"> 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: </p> <ol class=\"list\" id=\"L8\"> <li> <p id=\"P69\"> 104/147 samples included in ubiquitylproteome study and made full CR contribution, &gt;1000ug available. </p> </li> <li> <p id=\"P70\"> 42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug −1000 ug available. </p> </li> <li> <p id=\"P71\"> 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, &lt;750 ug available. </p> </li> </ol> <p id=\"P72\"> CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR’s proportion of 50:50), and aliquoted into 750 ug aliquots. </p> <p id=\"P73\"> In the ubiquitylproteome experimental design, 15 11-plex experiments contained <strong> 87 </strong> unique tumor samples in the C channels and <strong> 60 </strong> normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11 <sup> th </sup> channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). </p> </section> </section> <section id=\"S44\"> <h5 class=\"pmc_sec_title\"> TMT-11 Labeling of Peptides </h5> <section id=\"S45\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P74\"> 300 μg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a “reduced labeling” approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 μg peptide aliquot of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 μL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 μg aliquot from each sample and analyzing 0.25 μg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples were taken out, an additional 4 μg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/− 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample’s contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 × 300 μg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). </p> </section> </section> <section id=\"S46\"> <h5 class=\"pmc_sec_title\"> Peptide Fractionation </h5> <section id=\"S47\"> <h6 class=\"pmc_sec_title\"> For Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P75\"> To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was reconstituted in 935 μL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 μm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were made by 40–fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (&gt;95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701–5200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 μg/ μL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. </p> </section> </section> <section id=\"S48\"> <h5 class=\"pmc_sec_title\"> Phosphopeptide Enrichment </h5> <p id=\"P76\"> Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 μg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 μL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 μL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 μL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 μL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 × 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 μL 1% FA and phosphopeptides were eluted from the stage tips with 60 μL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 μL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 μL was injected per run. </p> </section> <section id=\"S49\"> <h5 class=\"pmc_sec_title\"> Acetylpeptide Enrichment </h5> <p id=\"P77\"> Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 μg peptides per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 × 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. </p> </section> <section id=\"S50\"> <h5 class=\"pmc_sec_title\"> Ubiquitylpeptide Enrichment with on-bead TMT labeling </h5> <p id=\"P78\"> Ubiquitin enrichment was performed based on the recently published UbiFast protocol ( <a aria-describedby=\"R178\" class=\"usa-link\" href=\"#R178\"> Udeshi et al., 2020 </a> ). Before enrichment, anti-K-ε-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4°C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4°C at a concentration of 0.5 ug/uL. </p> <p id=\"P79\"> For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4 <sup> o </sup> C for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30–40 s, left to settle on ice for 10–20 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20°C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. </p> </section> <section id=\"S51\"> <h5 class=\"pmc_sec_title\"> LC-MS/MS for Proteomics Analyses </h5> <p id=\"P80\"> Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross (360μm, IDEX Health &amp; Science, UH-906x). The column was heated to 50°C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 μm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360–75-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). </p> <p id=\"P81\"> For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). </p> <p id=\"P82\"> For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/− 10 ppm mass tolerance. “Perform dependent scan on single charge state per precursor only” was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. </p> </section> <section id=\"S52\"> <h5 class=\"pmc_sec_title\"> Immunohistochemistry (IHC) analysis. </h5> <section id=\"S53\"> <h6 class=\"pmc_sec_title\"> TMA construction </h6> <p id=\"P83\"> Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. </p> </section> <section id=\"S54\"> <h6 class=\"pmc_sec_title\"> Immune cell marker, Immunohistochemistry (IHC) </h6> <p id=\"P84\"> Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0–3 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the “ordinal” package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. </p> </section> <section id=\"S55\"> <h6 class=\"pmc_sec_title\"> Multiplex Immunofluorescence (mIF) staining </h6> <p id=\"P85\"> Development and optimization of the mIF platform has been previously described ( <a aria-describedby=\"R133\" class=\"usa-link\" href=\"#R133\"> Parra et al., 2020 </a> ). mIF staining was performed in a 4-μm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). </p> </section> <section id=\"S56\"> <h6 class=\"pmc_sec_title\"> IHC based assessment of TP63 low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> ) </h6> <p id=\"P86\"> For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66–2754) deemed LSCC by TCGA pathology. </p> </section> </section> </section> </section> <section id=\"S57\"> <h3 class=\"pmc_sec_title\"> QUANTIFICATION AND STATISTICAL ANALYSIS </h3> <section id=\"S58\"> <h4 class=\"pmc_sec_title\"> Genomic Data Analysis </h4> <section id=\"S59\"> <h5 class=\"pmc_sec_title\"> Copy Number Calling </h5> <p id=\"P87\"> Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> ). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3–0.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) ( <a aria-describedby=\"R47\" class=\"usa-link\" href=\"#R47\"> Gey and Lebarbier, 2008 </a> ), and dynamic programming ( <a aria-describedby=\"R13\" class=\"usa-link\" href=\"#R13\"> Bellman, 1961 </a> ), as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method ( <a aria-describedby=\"R98\" class=\"usa-link\" href=\"#R98\"> Lebarbier, 2005 </a> ). For the final set of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data. </p> </section> <section id=\"S60\"> <h5 class=\"pmc_sec_title\"> Somatic Variant Calling </h5> <p id=\"P88\"> Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels “GATK bundle known” and “Mills 1000G” somatic variants were called using Sentieon TNScope with the following settings “--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5”, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings “--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000”. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most ‘severe’ consequence occuring in a ‘reliable’ transcript was chosen. To accomplish this the computed consequences were ranked by ‘severity’ and ‘priority’ reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked ‘CANONICAL’ or ‘PICK’ by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD &gt; 5, NLOD &gt; 6, Fisher’s test p-value &lt; 0.05, NFLOD &gt; 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. </p> </section> <section id=\"S61\"> <h5 class=\"pmc_sec_title\"> Germline Variant Calling </h5> <p id=\"P89\"> Germline variant calling and annotation was carried out analogously using DNAScope using the following settings “--emit_conf 10 --call_conf 10” to generate a VCF file and “--emit_mode GVCF” to generate a gVCF file. The variants were custom-filtered based on the specific applications. </p> </section> <section id=\"S62\"> <h5 class=\"pmc_sec_title\"> GISTIC and MutSig analysis </h5> <p id=\"P90\"> The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value &lt;0.25 considered significant. The following parameters were used: </p> <ul class=\"list\" id=\"L10\"> <li> <p id=\"P91\"> Amplification Threshold = 0.1 </p> </li> <li> <p id=\"P92\"> Deletion Threshold = 0.1 </p> </li> <li> <p id=\"P93\"> Cap Values = 1.5 </p> </li> <li> <p id=\"P94\"> Broad Length Cutoff = 0.98 </p> </li> <li> <p id=\"P95\"> Remove X-Chromosome = 0 </p> </li> <li> <p id=\"P96\"> Confidence Level = 0.99 </p> </li> <li> <p id=\"P97\"> Join Segment Size = 4 </p> </li> <li> <p id=\"P98\"> Arm Level Peel Off = 1 </p> </li> <li> <p id=\"P99\"> Maximum Sample Segments = 2000 </p> </li> <li> <p id=\"P100\"> Gene GISTIC = 1 </p> </li> </ul> <p id=\"P101\"> Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from −2 to 2, where 0 means no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: −1 means deletion beyond the threshold; −2 means deletions greater than the minimum arm-level copy number observed in the sample. </p> <p id=\"P102\"> The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV ( <a aria-describedby=\"R95\" class=\"usa-link\" href=\"#R95\"> Lawrence et al., 2014 </a> ) was run on these filtered results to evaluate the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census ( <a aria-describedby=\"R161\" class=\"usa-link\" href=\"#R161\"> Sondka et al., 2018 </a> ), with false discovery rates (q values) recalculated. Genes of q value &lt; 0.1 were declared significant. </p> </section> </section> <section id=\"S63\"> <h4 class=\"pmc_sec_title\"> RNAseq and miRNAseq Quantification </h4> <section id=\"S64\"> <h5 class=\"pmc_sec_title\"> RNAseq Quantification </h5> <p id=\"P103\"> The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page ( <a class=\"usa-link usa-link--external\" href=\"https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files\" rel=\"noopener noreferrer\" target=\"_blank\"> https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files </a> ). The readcount is later converted to FPKM and FPKM-UQ using GDC’s formula ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/ </a> ). </p> </section> <section id=\"S65\"> <h5 class=\"pmc_sec_title\"> Isoform specific RNA quantification </h5> <p id=\"P104\"> For TP63 isoform centric analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> , the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression ( <a aria-describedby=\"R103\" class=\"usa-link\" href=\"#R103\"> Li et al., 2017 </a> ). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. </p> </section> <section id=\"S66\"> <h5 class=\"pmc_sec_title\"> DNA methylation data preprocessing </h5> <p id=\"P105\"> Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously ( <a aria-describedby=\"R39\" class=\"usa-link\" href=\"#R39\"> Fortin et al., 2014 </a> ). We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent analysis. </p> <p id=\"P106\"> Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5’ UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package ( <a aria-describedby=\"R40\" class=\"usa-link\" href=\"#R40\"> Fraley et al., 2016 </a> ) in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty &lt; 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. </p> </section> <section id=\"S67\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Data Analysis </h5> <p id=\"P107\"> miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov </a> ). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md </a> ). The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. </p> </section> </section> <section id=\"S68\"> <h4 class=\"pmc_sec_title\"> Proteomics Data Analysis </h4> <section id=\"S69\"> <h5 class=\"pmc_sec_title\"> Spectrum quality filtering and searching </h5> <p id=\"P108\"> All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory ( <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/proteomics\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/proteomics </a> ). Similar MS/MS spectra acquired on the same precursor m/z within +/− 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length &gt; 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800–6000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser ( <a class=\"usa-link usa-link--external\" href=\"https://genome.ucsc.edu/cgi-bin/hgTables\" rel=\"noopener noreferrer\" target=\"_blank\"> https://genome.ucsc.edu/cgi-bin/hgTables </a> ) on June 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/− 20 ppm mass tolerance for precursor and product ions, 30% minimum matched peak intensity, and “trypsin allow P” enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of −18 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −18 to 272 Da. For the acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −690 to 70 Da. </p> </section> <section id=\"S70\"> <h5 class=\"pmc_sec_title\"> Protein grouping, and localization of PTMs </h5> <p id=\"P109\"> Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. </p> <p id=\"P110\"> In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 – Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of &lt;0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5–6, thresholds were optimized to yield a PSM-level FDR estimate of &lt;0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher charge states. </p> <p id=\"P111\"> In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called “one-hit wonders.” After assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method “expand subgroups, top uses shared” to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide sequence of &gt;8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of &lt;0.6% at the peptide-spectrum match level and &lt;0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and &lt;0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. </p> <p id=\"P112\"> In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, &gt;1.1, essentially corresponded to at least 1 b or y ion located between two candidate sites that has a peak height &gt;10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, &gt;99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. </p> </section> <section id=\"S71\"> <h5 class=\"pmc_sec_title\"> Quantitation using TMT ratios </h5> <p id=\"P113\"> Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method “expand subgroups, top uses shared” (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method “unexpand subgroups”. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer’s Rule ( <a aria-describedby=\"R153\" class=\"usa-link\" href=\"#R153\"> Shadforth et al., 2005 </a> ) and correction factors obtained from the reagent manufacturer’s certificate of analysis ( <a class=\"usa-link usa-link--external\" href=\"https://www.thermofisher.com/order/catalog/product/90406\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.thermofisher.com/order/catalog/product/90406 </a> ) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. </p> </section> <section id=\"S72\"> <h5 class=\"pmc_sec_title\"> Two-component normalization of TMT ratios </h5> <p id=\"P114\"> It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians <em> N </em> ( <em> μi </em> 1, <em> σi </em> 1) and <em> N </em> ( <em> μi </em> 2, <em> σi </em> 2) for a sample <em> i </em> were fitted and used in the normalization process as follows: the mode <em> mi </em> of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians constrained to be <em> mi </em> , i.e., <em> μi </em> 1= <em> μi </em> 2= <em> mi </em> . The Gaussian with the smaller estimated standard deviation <span xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"> <math display=\"inline\" id=\"M1\"> <mrow> <mi> σ </mi> <mi> i </mi> <mo> = </mo> <mi> min </mi> <mo> ( </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 1 </mn> <mo> , </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 2 </mn> <mo> ) </mo> </mrow> </math> </span> was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using ( <em> mi </em> , <em> σi </em> ) by subtracting the mean <em> mi </em> from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation <em> σi </em> . </p> </section> <section id=\"S73\"> <h5 class=\"pmc_sec_title\"> Normalization of acetyl- and ubiquitylproteome </h5> <p id=\"P115\"> To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site’s main accession number did not yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. </p> </section> </section> <section id=\"S74\"> <h4 class=\"pmc_sec_title\"> Proteogenomic analysis </h4> <section id=\"S75\"> <h5 class=\"pmc_sec_title\"> Quality control and Batch effect assessment </h5> <p id=\"P116\"> We used guided principal component analysis (gPCA) ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. </p> <p id=\"P117\"> To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median pearson correlation &gt; 0.85. </p> <p id=\"P118\"> In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as common denominator ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1C </a> ). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value &lt; 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using <em> limma’s </em> removeBatchEffect ( <a class=\"usa-link usa-link--external\" href=\"https://rdrr.io/bioc/limma/man/removeBatchEffect.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rdrr.io/bioc/limma/man/removeBatchEffect.html </a> ) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. </p> </section> <section id=\"S76\"> <h5 class=\"pmc_sec_title\"> Dataset filtering </h5> <p id=\"P119\"> Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in &gt;70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at least two observed TMT ratios in &gt;25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio in &gt;25% of samples. </p> <p id=\"P120\"> Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. </p> </section> <section id=\"S77\"> <h5 class=\"pmc_sec_title\"> Unsupervised multi-omic clustering using NMF </h5> <p id=\"P121\"> We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Briefly, given a factorization rank <em> k </em> (where <em> k </em> is the number of clusters), NMF decomposes a <em> p x n </em> data matrix <em> V </em> into two matrices <em> W </em> and <em> H </em> such that multiplication of <em> W </em> and <em> H </em> approximates <em> V </em> . Matrix <em> H </em> is a <em> k x n </em> matrix whose entries represent weights for each sample (1 to <em> N </em> ) to contribute to each cluster (1 to <em> k </em> ), whereas matrix <em> W </em> is a <em> p x k </em> matrix representing weights for each feature (1 to <em> p </em> ) to contribute to each cluster (1 to <em> k </em> ). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em> H </em> . We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5. </p> <p id=\"P122\"> To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or −2) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by NMF as follows: </p> <ol class=\"list\" id=\"L12\"> <li> <p id=\"P123\"> Create one data matrix with all negative numbers zeroed. </p> </li> <li> <p id=\"P124\"> Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. </p> </li> <li> <p id=\"P125\"> Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. </p> </li> </ol> <p id=\"P126\"> The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ) and using the factorization method described in ( <a aria-describedby=\"R16\" class=\"usa-link\" href=\"#R16\"> Brunet et al., 2004 </a> ). To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters between <em> k </em> =2 and 8. For each k we factorized matrix <em> V </em> using 50 iterations with random initializations of <em> W </em> and <em> H </em> . To determine the optimal factorization rank we calculated two metrics for each <em> k </em> : 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix as defined in ( <a aria-describedby=\"R80\" class=\"usa-link\" href=\"#R80\"> Kim and Park, 2007 </a> ) measuring the reproducibility of the clustering across 50 iterations. The optimal <em> k </em> was defined as the maximum of the product of both metrics for cluster numbers between <em> k </em> =3 and 8 </p> <p id=\"P127\"> Having determined the optimal factorization rank <em> k </em> , in order to achieve robust factorization of the multi-omic data matrix <em> V </em> , we repeated the NMF analysis using 500 iterations with random initializations of <em> W </em> and <em> H </em> and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em> W </em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first normalized each row in matrix <em> W </em> by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix W <sub> signed </sub> contained signed cluster weights for each feature in the input matrix. </p> <p id=\"P128\"> For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights (W <sub> signed </sub> ) onto Hallmark pathway gene sets ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) using ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ). To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> using the following parameters: </p> <ul class=\"list\" id=\"L14\"> <li> <p id=\"P129\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P130\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P131\"> weight=1 </p> </li> <li> <p id=\"P132\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P133\"> output.score.type=“NES” </p> </li> <li> <p id=\"P134\"> nperm=1000 </p> </li> <li> <p id=\"P135\"> global.fdr=TRUE </p> </li> <li> <p id=\"P136\"> min.overlap=5 </p> </li> <li> <p id=\"P137\"> correl.type=”z.score” </p> </li> </ul> <p id=\"P138\"> To test the association of the resulting clusters to clinical variables we used Fisher’s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. </p> <p id=\"P139\"> The entire workflow described above has been implemented as a module for PANOPLY ( <a aria-describedby=\"R116\" class=\"usa-link\" href=\"#R116\"> Mani et al., 2020 </a> ) which runs on Broad’s Cloud platform Terra ( <a class=\"usa-link usa-link--external\" href=\"https://app.terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://app.terra.bio/ </a> ). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> . </p> </section> <section id=\"S78\"> <h5 class=\"pmc_sec_title\"> Integrative analysis with Stewart et al. </h5> <p id=\"P140\"> To enable integrative analysis of our protein data with the protein data in Stewart et al.( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. </p> </section> <section id=\"S79\"> <h5 class=\"pmc_sec_title\"> RNA subtyping </h5> <p id=\"P141\"> Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ) and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ). Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p &gt; 0.01) were marked to have ‘undecided’ subtype. </p> </section> <section id=\"S80\"> <h5 class=\"pmc_sec_title\"> Chromosomal instability index </h5> <p id=\"P142\"> The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. </p> </section> <section id=\"S81\"> <h5 class=\"pmc_sec_title\"> Fusion detection and analysis </h5> <p id=\"P143\"> Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline ( <a aria-describedby=\"R149\" class=\"usa-link\" href=\"#R149\"> Robinson et al., 2017 </a> ; <a aria-describedby=\"R199\" class=\"usa-link\" href=\"#R199\"> Wu et al., 2018 </a> ), fusioncatcher_v1.10 ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) and arriba_v1.1.0 ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> ). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described ( <a aria-describedby=\"R182\" class=\"usa-link\" href=\"#R182\"> Vats et al., 2020 </a> ). </p> </section> <section id=\"S82\"> <h5 class=\"pmc_sec_title\"> mRNA and Protein correlation </h5> <p id=\"P144\"> To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini–Hochberg procedure. </p> </section> <section id=\"S83\"> <h5 class=\"pmc_sec_title\"> CNA-driven cis and trans effects </h5> <p id=\"P145\"> Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR &lt; 0.05) positive or negative correlations. </p> </section> <section id=\"S84\"> <h5 class=\"pmc_sec_title\"> CMAP analysis </h5> <p id=\"P146\"> Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\" rel=\"noopener noreferrer\" target=\"_blank\"> GSE92742 </a> ). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See <a class=\"usa-link usa-link--external\" href=\"https://clue.io/GEO-guide\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/GEO-guide </a> for more information. </p> <p id=\"P147\"> To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in “CNA driven cis and trans effects” above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ). The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. </p> <p id=\"P148\"> For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were identified with Fisher’s test FDR &lt; 0.25, using this process. </p> <p id=\"P149\"> In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with rate λ, counting the number of identified candidate driver genes. Given the small n ( = 10) and λ, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with <em> both </em> CNA amplification and deletion. </p> <p id=\"P150\"> To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with α = 0.05, followed by counting the number of trans-genes in this list of regulated genes. </p> <p id=\"P151\"> To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. </p> </section> <section id=\"S85\"> <h5 class=\"pmc_sec_title\"> LINCS analysis </h5> <p id=\"P152\"> Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value &lt; 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. </p> <p id=\"P153\"> Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini &amp; Hochberg method. Subtype-specific differentially expressed proteins were identified based on their differential expression (adjusted p.value &lt; 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline ( <a class=\"usa-link usa-link--external\" href=\"https://hub.docker.com/u/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://hub.docker.com/u/cmap </a> ) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ). The corresponding compound metadata were obtained from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. </p> </section> <section id=\"S86\"> <h5 class=\"pmc_sec_title\"> Defining cancer-associated genes </h5> <p id=\"P154\"> Cancer-associated genes (CAG) were compiled using the Census website <a class=\"usa-link usa-link--external\" href=\"https://cancer.sanger.ac.uk/census\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cancer.sanger.ac.uk/census </a> . The list of genes is provided in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> <section id=\"S87\"> <h5 class=\"pmc_sec_title\"> CpG Island Methylator Phenotype </h5> <p id=\"P155\"> The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score &gt;0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value −0.041 and −0.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. </p> </section> <section id=\"S88\"> <h5 class=\"pmc_sec_title\"> iProFun Based Cis Association Analysis </h5> <p id=\"P156\"> We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR)&lt;10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities &gt; 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) &lt;10% by selecting a minimal cutoff value of alpha such that 75%&lt;alpha&lt;100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities &gt; alpha in permuted data) / (Averaged No. of genes with posterior probabilities &gt; alpha in original data). <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. </p> </section> <section id=\"S89\"> <h5 class=\"pmc_sec_title\"> EMT-specific cluster and fibroblast enrichment </h5> <p id=\"P157\"> NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). </p> </section> <section id=\"S90\"> <h5 class=\"pmc_sec_title\"> Differential marker analysis </h5> <p id=\"P158\"> A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having &lt; 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR ( <a aria-describedby=\"R106\" class=\"usa-link\" href=\"#R106\"> Liao et al., 2019 </a> ). The selected gene list was either significant proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted p-value &lt;0.01. </p> <p id=\"P159\"> Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> ), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. </p> </section> <section id=\"S91\"> <h5 class=\"pmc_sec_title\"> Overall and disease-free survival in TCGA lung squamous carcinoma </h5> <p id=\"P160\"> Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. </p> </section> <section id=\"S92\"> <h5 class=\"pmc_sec_title\"> Kaplan-Meier survival analysis </h5> <p id=\"P161\"> We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. </p> <p id=\"P162\"> We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. </p> </section> <section id=\"S93\"> <h5 class=\"pmc_sec_title\"> Continuous Smoking Score </h5> <p id=\"P163\"> Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC ( <a aria-describedby=\"R170\" class=\"usa-link\" href=\"#R170\"> Tate et al., 2019 </a> ) and cosine similarity was calculated to identify the best match. </p> <p id=\"P164\"> We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 &lt; S &lt; 1, to quantify the degree of confidence that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C ∩ T ∩ N, and used A and A’ to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A’ and D to the joint probability of A and D. For example, the latter can be written P(A)・P(C|A)・P(T|A)・P(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9 ( <a aria-describedby=\"R184\" class=\"usa-link\" href=\"#R184\"> Walser et al., 2008 </a> ). </p> <p id=\"P165\"> We consider <em> S </em> to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type <em> G </em> is P( <em> G </em> | <em> A </em> ) = [ <em> x </em> ・ <em> erf </em> ( <em> g/y </em> ) + 1] <em> / </em> ( <em> x </em> + 2) and P( <em> G </em> | <em> A </em> ’) = 1 - P( <em> G </em> | <em> A </em> ), where <em> erf </em> is the Gaussian error function, <em> g </em> is the observed value for the given data type, and <em> x </em> and <em> y </em> are empirically-determined weights. </p> <p id=\"P166\"> The shape function for total mutations, <em> T </em> , has two modifiers. First, it includes a simple expected-value correction for purity, <em> u </em> . Namely, assuming mutation-calling does not capture all mutations because of impurities, <em> t </em> is taken as the observed number of mutations divided by a purity shape function, <em> f </em> , where <em> f </em> ≤ 1. Although one might model <em> f </em> according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we use a line function, <em> f </em> = 0.2・ <em> u </em> + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce’s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score &gt; 0.6; samples with weak smoking evidence if smoking score &lt; 0.15. </p> </section> <section id=\"S94\"> <h5 class=\"pmc_sec_title\"> Identification of differentially regulated events in NRF2 mutant tumors. </h5> <p id=\"P167\"> For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value &lt;0.05 and logFC&gt;1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 ( <a aria-describedby=\"R115\" class=\"usa-link\" href=\"#R115\"> Malhotra et al., 2010 </a> ) and SINGH_NFE2L2_TARGETS (( <a aria-describedby=\"R159\" class=\"usa-link\" href=\"#R159\"> Singh et al., 2008 </a> ), BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ). Ten of the 54 genes were novel targets identified in this study. </p> </section> <section id=\"S95\"> <h5 class=\"pmc_sec_title\"> Immune cluster identification based on cell type composition </h5> <p id=\"P168\"> The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) based on 108 tumors and 94 NAT samples. <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> contains the final score computed by xCell of different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR &lt; 5%). This filtering resulted in 25 cell types annotated in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S96\"> <h5 class=\"pmc_sec_title\"> TCGA pan-cancer immune subtyping </h5> <p id=\"P169\"> Tumors were classified into the six distinct immune subtypes — wound healing, IFN-γ dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-β dominant — identified by a TCGA pan-cancer analysis presented in ( <a aria-describedby=\"R173\" class=\"usa-link\" href=\"#R173\"> Thorsson et al., 2018 </a> ). Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> ) ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) to assign an immune subtype to each of the tumors. </p> </section> <section id=\"S97\"> <h5 class=\"pmc_sec_title\"> Ranking tumors by inferred activity of IFN-γ pathway </h5> <p id=\"P170\"> We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above “top” and “bottom” probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the “top” and “bottom” probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher’s exact test, given protein’s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). The thermodynamic interpretation of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway’s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway’s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. </p> <p id=\"P171\"> In our analysis, we used a gene list of IFN-γ signaling pathway from ( <a aria-describedby=\"R1\" class=\"usa-link\" href=\"#R1\"> Abril-Rodriguez and Ribas, 2017 </a> ). We settled with this 15 genes signaling pathway because it has all necessary components – a cytokine mediator, receptors, signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. </p> </section> <section id=\"S98\"> <h5 class=\"pmc_sec_title\"> Estimation of Tumor Purity, Stromal and Immune Scores </h5> <p id=\"P172\"> Besides xCell, we utilized ESTIMATE ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) to infer immune and stromal scores based on gene expression data ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app ( <a class=\"usa-link usa-link--external\" href=\"http://cibersort.stanford.edu\" rel=\"noopener noreferrer\" target=\"_blank\"> cibersort.stanford.edu </a> ; ( <a aria-describedby=\"R127\" class=\"usa-link\" href=\"#R127\"> Newman et al., 2015 </a> )) in absolute mode. To infer tumor purity, TSNet was utilized ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S99\"> <h5 class=\"pmc_sec_title\"> Differentially Expressed Genes and Pathway Analysis </h5> <p id=\"P173\"> Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ) was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hänzelmann et al., 2013 </a> ). For this analysis, pathways from the Reactome ( <a aria-describedby=\"R33\" class=\"usa-link\" href=\"#R33\"> Fabregat et al., 2018 </a> ), KEGG ( <a aria-describedby=\"R75\" class=\"usa-link\" href=\"#R75\"> Kanehisa et al., 2017 </a> ) and Hallmark ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . </p> </section> <section id=\"S100\"> <h5 class=\"pmc_sec_title\"> Deriving RTK CBPE scores </h5> <p id=\"P174\"> A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ). Each association computation was carried out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. </p> </section> <section id=\"S101\"> <h5 class=\"pmc_sec_title\"> Independent component analysis </h5> <p id=\"P175\"> ICA was performed with a workflow modified from previously described ( <a aria-describedby=\"R111\" class=\"usa-link\" href=\"#R111\"> Liu et al., 2019 </a> ). Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm ( <a aria-describedby=\"R117\" class=\"usa-link\" href=\"#R117\"> McInnes et al., 2017 </a> ) with cosine distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 <em> P </em> -values. Significance was controlled for multiple testing at 0.01 level (log10 ( <em> p-value) </em> =−5.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. </p> </section> <section id=\"S102\"> <h5 class=\"pmc_sec_title\"> Mutation-based cis- and trans-effects </h5> <p id=\"P176\"> We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ) on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR &lt;0.05 cut-off were separated into two categories based on cis- and trans- effects. </p> </section> <section id=\"S103\"> <h5 class=\"pmc_sec_title\"> Germline quantitative trait loci (QTL) analysis </h5> <p id=\"P177\"> To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen </a> ). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count ≥ 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9 ( <a aria-describedby=\"R27\" class=\"usa-link\" href=\"#R27\"> Deelen et al., 2014 </a> ). To adjust for population stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07 ( <a aria-describedby=\"R142\" class=\"usa-link\" href=\"#R142\"> Purcell et al., 2007 </a> ). For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE &gt; 0.0001, MAF &gt; 0.01, and call rate &gt; 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at ≤ FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis ( <a aria-describedby=\"R86\" class=\"usa-link\" href=\"#R86\"> Krämer et al., 2014 </a> ) </p> </section> <section id=\"S104\"> <h5 class=\"pmc_sec_title\"> miRNA analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </h5> <p id=\"P178\"> Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained ( <a aria-describedby=\"R66\" class=\"usa-link\" href=\"#R66\"> Huang et al., 2020 </a> ). Spearman correlation was used to calculate the correlations between miRNA and its target genes. </p> </section> <section id=\"S105\"> <h5 class=\"pmc_sec_title\"> Pathway projection using ssGSEA </h5> <p id=\"P179\"> The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: </p> <ul class=\"list\" id=\"L16\"> <li> <p id=\"P180\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P181\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P182\"> weight=0.75 </p> </li> <li> <p id=\"P183\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P184\"> output.score.type=“NES” </p> </li> <li> <p id=\"P185\"> nperm=1000 </p> </li> <li> <p id=\"P186\"> global.fdr=TRUE </p> </li> <li> <p id=\"P187\"> min.overlap=10 </p> </li> <li> <p id=\"P188\"> correl.type=”z.score” </p> </li> </ul> </section> <section id=\"S106\"> <h5 class=\"pmc_sec_title\"> Phosphorylation-driven signature analysis </h5> <p id=\"P189\"> We performed phosphosite-specific signature enrichment analysis (PTM-SEA) ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed −log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) </p> <table class=\"disp-formula p\" id=\"FD1\"> <tr> <td class=\"formula\"> <math display=\"block\" id=\"M2\"> <mrow> <mtext> logPsite </mtext> <mspace width=\"0.25em\"> </mspace> <mo> = </mo> <mspace width=\"0.25em\"> </mspace> <mo> − </mo> <mtext> log </mtext> <mn> 10 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> p </mtext> <mo> − </mo> <mtext> valuesite </mtext> </mrow> <mo> ) </mo> </mrow> <mo> * </mo> <mtext> sign </mtext> <mrow> <mo> ( </mo> <mrow> <mtext> log </mtext> <mn> 2 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> fold </mtext> <mspace width=\"0.25em\"> </mspace> <mtext> changesite </mtext> </mrow> <mo> ) </mo> </mrow> </mrow> <mo> ) </mo> </mrow> </mrow> </math> </td> </tr> </table> <p> PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. </p> <p id=\"P190\"> We employed the heuristic method introduced by Krug et al. ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. </p> <p id=\"P191\"> We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb </a> using the flanking amino acid sequence (+/− 7 aa) as primary identifier. We used the implementation of PTM-SEA available on GitHub ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: </p> <p id=\"P192\"> weight: 1 </p> <p id=\"P193\"> statistic: “area.under.RES” </p> <p id=\"P194\"> output.score.type: “NES” </p> <p id=\"P195\"> nperm: 1000 </p> <p id=\"P196\"> min.overlap: 5 </p> <p id=\"P197\"> correl.type: “Z-score” </p> <p id=\"P198\"> The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini &amp; and Hochberg ( <a aria-describedby=\"R14\" class=\"usa-link\" href=\"#R14\"> Benjamini and Hochberg, 1995 </a> ). Signatures with FDR-corrected p-values &lt; 0.05 were considered to be differential between tumor and NAT. </p> </section> <section id=\"S107\"> <h5 class=\"pmc_sec_title\"> <em> CDKN2A </em> and <em> RB1 </em> annotations and pathway analysis </h5> <p id=\"P199\"> Comprehensive tumor annotation for <em> CDKN2A </em> and <em> RB1 </em> genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values &gt;= 0.2. Multi-gene proliferation scores were calculated as described previously ( <a aria-describedby=\"R32\" class=\"usa-link\" href=\"#R32\"> Ellis et al., 2017 </a> ) as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al ( <a aria-describedby=\"R194\" class=\"usa-link\" href=\"#R194\"> Whitfield et al., 2002 </a> ). p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_000068.1\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_000068.1 </a> for p16INK4a and Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_478102.2\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_478102.2 </a> for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ). Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. </p> </section> <section id=\"S108\"> <h5 class=\"pmc_sec_title\"> Association analysis between KGG-site abundances and E3 ligases and DUBs </h5> <p id=\"P200\"> A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from ( <a aria-describedby=\"R119\" class=\"usa-link\" href=\"#R119\"> Medvar et al., 2016 </a> ; <a aria-describedby=\"R129\" class=\"usa-link\" href=\"#R129\"> Nijman et al., 2005 </a> ). We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR &lt; 0.01). </p> </section> <section id=\"S109\"> <h5 class=\"pmc_sec_title\"> Cluster and pathway analysis of significantly modulated K-GG sites in tumors. </h5> <p id=\"P201\"> Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. </p> <p id=\"P202\"> Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere et al., 2019 </a> ). Pathway enrichment was performed using a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon et al., 2003 </a> ) using the Enrichment Map app ( <a aria-describedby=\"R120\" class=\"usa-link\" href=\"#R120\"> Merico et al., 2010 </a> ) for network analysis of pathways. Pathways were connected using the gene overlap and clustered (pathway cutoff q-val &lt; 0.1; jaccard overlap &gt; 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. </p> </section> <section id=\"S110\"> <h5 class=\"pmc_sec_title\"> PTM CLUMPS analysis </h5> <p id=\"P203\"> We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual “events” as done with mutations in CLUMPS v1 ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ). For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and Tumor samples ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ). Sites are selected with an FDR &lt; 0.1 (Benjamini-Hochberg) and LFC &gt; 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. </p> </section> <section id=\"S111\"> <h5 class=\"pmc_sec_title\"> DepMap genetic dependency and drug response analysis </h5> <p id=\"P204\"> Cell lines annotated as “NSCLC_squamous” from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. </p> <p id=\"P205\"> To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study’s LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from DepMap. </p> <p id=\"P206\"> <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> : For identifying if top protein biomarkers (502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. </p> <p id=\"P207\"> <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> : To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. </p> <p id=\"P208\"> Continuous log2 copy number data from DepMap was used in <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> to correlate <em> SOX2 </em> copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). </p> <p id=\"P209\"> Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). </p> </section> <section id=\"S112\"> <h5 class=\"pmc_sec_title\"> CausalPath analysis </h5> <p id=\"P210\"> CausalPath ( <a aria-describedby=\"R8\" class=\"usa-link\" href=\"#R8\"> Babur et al., 2018 </a> ) searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). </p> </section> <section id=\"S113\"> <h5 class=\"pmc_sec_title\"> Variant Peptide Identification </h5> <p id=\"P211\"> We used NeoFlow ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> ) for neoantigen prediction ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ). Specifically, Optitype ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) to predict HLA peptide binding affinity for somatic mutation–derived variant peptides with a length between 8–11 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). To identify variant peptides, we used a customized protein sequence database approach ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ). We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> ) for customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using PepQuery ( <a class=\"usa-link usa-link--external\" href=\"http://www.pepquery.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.pepquery.org </a> ) ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) with the p-value cutoff &lt;= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV ( <a class=\"usa-link usa-link--external\" href=\"http://www.zhang-lab.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.zhang-lab.org/ </a> ) ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ). </p> </section> <section id=\"S114\"> <h5 class=\"pmc_sec_title\"> Cancer/testis Antigen Prediction </h5> <p id=\"P212\"> Cancer/testis (CT) antigens were downloaded from the CTdatabase ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ). CT antigens with a median &gt;4-fold increase in tumor from NAT were highlighted. </p> </section> <section id=\"S115\"> <h5 class=\"pmc_sec_title\"> PROGENy Scores </h5> <p id=\"P213\"> PROGENy ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ) was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy. </p> </section> <section id=\"S116\"> <h5 class=\"pmc_sec_title\"> LSCC, HNSCC and LUAD integrative analysis </h5> <p id=\"P214\"> LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ). The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in the <a class=\"usa-link\" href=\"#S90\"> Differential marker analysis </a> method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. </p> <p id=\"P215\"> Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value &lt; .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value &lt;0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p value &lt;0.01). </p> <p id=\"P216\"> To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method <a class=\"usa-link\" href=\"#S98\"> Estimation of Tumor Purity, Stromal and Immune Scores </a> . To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p value &lt; 0.01 were considered significant. </p> </section> </section> </section> </section> <section id=\"SM1\"> <h2 class=\"pmc_sec_title\"> Supplementary Material </h2> <section class=\"sm xbox font-sm\" id=\"SD1\"> <div class=\"caption p\"> <span> 1 </span> <p id=\"P217\"> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p id=\"P218\"> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p id=\"P219\"> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p id=\"P220\"> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p id=\"P221\"> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p id=\"P222\"> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p id=\"P223\"> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p id=\"P224\"> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p id=\"P225\"> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\" id=\"d40e3501\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD2\"> <div class=\"caption p\"> <span> 2 </span> <p id=\"P226\"> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p id=\"P227\"> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p id=\"P228\"> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p id=\"P229\"> C. Same as B but in the EMT-enriched subtype. </p> <p id=\"P230\"> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p id=\"P231\"> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p id=\"P232\"> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P233\"> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p id=\"P234\"> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p id=\"P235\"> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P236\"> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p id=\"P237\"> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\" id=\"d40e3559\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD3\"> <div class=\"caption p\"> <span> 3 </span> <p id=\"P238\"> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p id=\"P239\"> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p id=\"P240\"> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p id=\"P241\"> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p id=\"P242\"> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p id=\"P243\"> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p id=\"P244\"> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p id=\"P245\"> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p id=\"P246\"> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\" id=\"d40e3627\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD4\"> <div class=\"caption p\"> <span> 4 </span> <p id=\"P247\"> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p id=\"P248\"> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p id=\"P249\"> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p id=\"P250\"> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p id=\"P251\"> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p id=\"P252\"> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p id=\"P253\"> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p id=\"P254\"> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p id=\"P255\"> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p id=\"P256\"> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p id=\"P257\"> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p id=\"P258\"> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p id=\"P259\"> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p id=\"P260\"> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\" id=\"d40e3671\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD5\"> <div class=\"caption p\"> <span> 6 </span> <p id=\"P261\"> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P262\"> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p id=\"P263\"> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p id=\"P264\"> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p id=\"P265\"> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p id=\"P266\"> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p id=\"P267\"> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P268\"> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\" id=\"d40e3727\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD6\"> <div class=\"caption p\"> <span> 7 </span> <p id=\"P269\"> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P270\"> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p id=\"P271\"> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p id=\"P272\"> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p id=\"P273\"> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p id=\"P274\"> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p id=\"P275\"> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p id=\"P276\"> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\" id=\"d40e3755\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD7\"> <div class=\"caption p\"> <span> 8 </span> <p id=\"P277\"> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P278\"> A. Metadata associated with this study </p> <p id=\"P279\"> B. Cancer Associated Genes (CAGs) </p> <p id=\"P280\"> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p id=\"P281\"> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p id=\"P282\"> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\" id=\"d40e3782\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD8\"> <div class=\"caption p\"> <span> 9 </span> <p id=\"P283\"> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P284\"> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p id=\"P285\"> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p id=\"P286\"> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p id=\"P287\"> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p id=\"P288\"> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p id=\"P289\"> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p id=\"P290\"> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p id=\"P291\"> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p id=\"P292\"> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\" id=\"d40e3829\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD9\"> <div class=\"caption p\"> <span> 10 </span> <p id=\"P293\"> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P294\"> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P295\"> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p id=\"P296\"> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p id=\"P297\"> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P298\"> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P299\"> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P300\"> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P301\"> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P302\"> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P303\"> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P304\"> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P305\"> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P306\"> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P307\"> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\" id=\"d40e3937\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD10\"> <div class=\"caption p\"> <span> 11 </span> <p id=\"P308\"> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p id=\"P309\"> A. Copy number amplified cytobands </p> <p id=\"P310\"> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p id=\"P311\"> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p id=\"P312\"> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p id=\"P313\"> F. CMAP tab- Connectivity Map (CMAP) results </p> <p id=\"P314\"> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p id=\"P315\"> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p id=\"P316\"> I. Summary of CDKN2A and RB1 annotations </p> <p id=\"P317\"> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p id=\"P318\"> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p id=\"P319\"> L. NRF2 signature </p> <p id=\"P320\"> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\" id=\"d40e4003\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD11\"> <div class=\"caption p\"> <span> 12 </span> <p id=\"P321\"> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P322\"> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p id=\"P323\"> B. Ub-sites that show differential abundance across Ub clusters </p> <p id=\"P324\"> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p id=\"P325\"> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p id=\"P326\"> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P327\"> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P328\"> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p id=\"P329\"> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P330\"> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P331\"> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\" id=\"d40e4032\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD12\"> <div class=\"caption p\"> <span> 13 </span> <p id=\"P332\"> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P333\"> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p id=\"P334\"> B. Immune clusters based on xCell signatures </p> <p id=\"P335\"> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p id=\"P336\"> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p id=\"P337\"> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p id=\"P338\"> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p id=\"P339\"> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p id=\"P340\"> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p id=\"P341\"> I. List of genes/proteins whose expression is associated with a SNP </p> <p id=\"P342\"> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p id=\"P343\"> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\" id=\"d40e4072\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD13\"> <div class=\"caption p\"> <span> 14 </span> <p id=\"P344\"> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P345\"> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p id=\"P346\"> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p id=\"P347\"> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p id=\"P348\"> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p id=\"P349\"> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p id=\"P350\"> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\" id=\"d40e4094\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD14\"> <div class=\"caption p\"> <span> Figure S5 </span> <p id=\"P351\"> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P352\"> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p id=\"P353\"> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p id=\"P354\"> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p id=\"P355\"> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p id=\"P356\"> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p id=\"P357\"> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p id=\"P358\"> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p id=\"P359\"> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p id=\"P360\"> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p id=\"P361\"> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p id=\"P362\"> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\" id=\"d40e4134\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"floats-group\" id=\"floats-group1\"> <section class=\"bt xbox font-sm\" id=\"BX1\"> <h2 class=\"obj_head\"> Highlights. </h2> <ul class=\"list\" id=\"L18\"> <li> <p id=\"P437\"> Unsupervised clustering revealed subtype with EMT and phosphoprotein signatures </p> </li> <li> <p id=\"P438\"> Potential therapeutic vulnerabilities included survivin, NSD3, LSD1 and EZH2 </p> </li> <li> <p id=\"P439\"> Rb phosphorylation nominated as a biomarker for trials with CDK4/6 inhibitors </p> </li> <li> <p id=\"P440\"> Detailed immune landscape analysis highlighted targetable points of immune regulation </p> </li> </ul> </section> </section> <section class=\"ack\" id=\"S117\"> <h2 class=\"pmc_sec_title\"> Acknowledgements </h2> <p id=\"P363\"> This work was supported by grants U24CA210955, U24CA210985, U24CA210986, U24CA210954, U24CA210967, U24CA210972, U24CA210979, U24CA210993, U01CA214114, U01CA214116, and U01CA214125 from the NCI-CPTAC, U54HL127624, P30CA240139, 1F32HL154711-01, and T32CA203690 from the NIH. We thank Dr. Lata Adnani for helping with graphics. </p> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN3\"> <p id=\"P364\"> Declaration of Interests </p> <p id=\"P365\"> The authors declare no competing interests </p> </div> <div class=\"fn p\" id=\"FN4\"> <p id=\"P366\"> ADDITIONAL RESOURCES </p> <p id=\"P367\"> The CPTAC program website, detailing program initiatives, investigators, and datasets, is found at <a class=\"usa-link usa-link--external\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.cancer.gov/programs/cptac </a> . </p> <p id=\"P368\"> A website for interactive visualization of the multi-omic dataset is available at: <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </p> <p id=\"P369\"> All processed data matrices are available at LinkedOmics ( <a aria-describedby=\"R181\" class=\"usa-link\" href=\"#R181\"> Vasaikar et al., 2018 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"http://www.linkedomics.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.linkedomics.org </a> ), where computational tools are available for further exploration of this dataset. </p> <p id=\"P370\"> Consortia </p> <p id=\"P371\"> Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Fenyö, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Małgorzata Wojtyś, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Özgün Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. </p> </div> <div class=\"fn p\" id=\"FN5\"> <p id=\"P372\"> <strong> Publisher\\'s Disclaimer: </strong> This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. </p> </div> </div> </section> <section class=\"ref-list\" id=\"ref-list1\"> <h2 class=\"pmc_sec_title\"> REFERENCES: </h2> <section id=\"ref-list1_sec2\"> <ol class=\"ref-list font-sm\"> <li id=\"R1\"> <cite> Abril-Rodriguez G, and Ribas A (2017). SnapShot: Immune Checkpoint Inhibitors. Cancer Cell 31, 848–848.e1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2017.05.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28609660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=SnapShot:%20Immune%20Checkpoint%20Inhibitors&amp;author=G%20Abril-Rodriguez&amp;author=A%20Ribas&amp;volume=31&amp;publication_year=2017&amp;pages=848-848.e1&amp;pmid=28609660&amp;doi=10.1016/j.ccell.2017.05.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R2\"> <cite> Ahn ER, Mangat PK, Garrett-Mayer E, Halabi S, Dib EG, Haggstrom DE, Alguire KB, Calfa CJ, Cannon TL, Crilley PA, et al. (2020). Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study. JCO Precision Oncology 757–766. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/PO.20.00037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/35050752/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCO%20Precision%20Oncology&amp;title=Palbociclib%20in%20Patients%20With%20Non%E2%80%93Small-Cell%20Lung%20Cancer%20With%20CDKN2A%20Alterations:%20Results%20From%20the%20Targeted%20Agent%20and%20Profiling%20Utilization%20Registry%20Study&amp;author=ER%20Ahn&amp;author=PK%20Mangat&amp;author=E%20Garrett-Mayer&amp;author=S%20Halabi&amp;author=EG%20Dib&amp;publication_year=2020&amp;pages=757-766&amp;pmid=35050752&amp;doi=10.1200/PO.20.00037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R3\"> <cite> Albert M, Bécares M, Falqui M, Fernández-Lozano C, and Guerra S (2018). ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis. Viruses 10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/v10110629\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6265978/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30428561/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=ISG15,%20a%20Small%20Molecule%20with%20Huge%20Implications:%20Regulation%20of%20Mitochondrial%20Homeostasis&amp;author=M%20Albert&amp;author=M%20B%C3%A9cares&amp;author=M%20Falqui&amp;author=C%20Fern%C3%A1ndez-Lozano&amp;author=S%20Guerra&amp;volume=10&amp;publication_year=2018&amp;pmid=30428561&amp;doi=10.3390/v10110629&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R4\"> <cite> Alcalá S, Sancho P, Martinelli P, Navarro D, Pedrero C, Martín-Hijano L, Valle S, Earl J, Rodríguez-Serrano M, Ruiz-Cañas L, et al. (2020). ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity. Nat. Commun. 11, 2682. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-16395-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7260233/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32472071/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=ISG15%20and%20ISGylation%20is%20required%20for%20pancreatic%20cancer%20stem%20cell%20mitophagy%20and%20metabolic%20plasticity&amp;author=S%20Alcal%C3%A1&amp;author=P%20Sancho&amp;author=P%20Martinelli&amp;author=D%20Navarro&amp;author=C%20Pedrero&amp;volume=11&amp;publication_year=2020&amp;pages=2682&amp;pmid=32472071&amp;doi=10.1038/s41467-020-16395-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R5\"> <cite> Almeida LG, Sakabe NJ, deOliveira AR, Silva MCC, Mundstein AS, Cohen T, Chen Y-T, Chua R, Gurung S, Gnjatic S, et al. (2009). CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens. Nucleic Acids Res. 37, D816–D819. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkn673\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2686577/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18838390/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CTdatabase:%20a%20knowledge-base%20of%20high-throughput%20and%20curated%20data%20on%20cancer-testis%20antigens&amp;author=LG%20Almeida&amp;author=NJ%20Sakabe&amp;author=AR%20deOliveira&amp;author=MCC%20Silva&amp;author=AS%20Mundstein&amp;volume=37&amp;publication_year=2009&amp;pages=D816-D819&amp;pmid=18838390&amp;doi=10.1093/nar/gkn673&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R6\"> <cite> Aran D, Hu Z, and Butte AJ (2017). xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 18, 220. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-017-1349-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5688663/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29141660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=xCell:%20digitally%20portraying%20the%20tissue%20cellular%20heterogeneity%20landscape&amp;author=D%20Aran&amp;author=Z%20Hu&amp;author=AJ%20Butte&amp;volume=18&amp;publication_year=2017&amp;pages=220&amp;pmid=29141660&amp;doi=10.1186/s13059-017-1349-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R7\"> <cite> Ardini-Poleske ME, Clark RF, Ansong C, Carson JP, Corley RA, Deutsch GH, Hagood JS, Kaminski N, Mariani TJ, Potter SS, et al. (2017). LungMAP: The Molecular Atlas of Lung Development Program. Am. J. Physiol. Lung Cell. Mol. Physiol. 313, L733–L740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/ajplung.00139.2017\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5792185/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28798251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Lung%20Cell.%20Mol.%20Physiol&amp;title=LungMAP:%20The%20Molecular%20Atlas%20of%20Lung%20Development%20Program&amp;author=ME%20Ardini-Poleske&amp;author=RF%20Clark&amp;author=C%20Ansong&amp;author=JP%20Carson&amp;author=RA%20Corley&amp;volume=313&amp;publication_year=2017&amp;pages=L733-L740&amp;pmid=28798251&amp;doi=10.1152/ajplung.00139.2017&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R8\"> <cite> Babur Ö, Luna A, Korkut A, Durupinar F, Siper MC, Dogrusoz U, Aslan JE, Sander C, and Demir E (2018). Causal interactions from proteomic profiles: molecular data meets pathway knowledge. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.patter.2021.100257\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8212145/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34179843/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Causal%20interactions%20from%20proteomic%20profiles:%20molecular%20data%20meets%20pathway%20knowledge&amp;author=%C3%96%20Babur&amp;author=A%20Luna&amp;author=A%20Korkut&amp;author=F%20Durupinar&amp;author=MC%20Siper&amp;publication_year=2018&amp;pmid=34179843&amp;doi=10.1016/j.patter.2021.100257&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R9\"> <cite> Baeza J, Smallegan MJ, and Denu JM (2015). Site-specific reactivity of nonenzymatic lysine acetylation. ACS Chem. Biol. 10, 122–128. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/cb500848p\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4301072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25555129/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol&amp;title=Site-specific%20reactivity%20of%20nonenzymatic%20lysine%20acetylation&amp;author=J%20Baeza&amp;author=MJ%20Smallegan&amp;author=JM%20Denu&amp;volume=10&amp;publication_year=2015&amp;pages=122-128&amp;pmid=25555129&amp;doi=10.1021/cb500848p&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R10\"> <cite> Bailey MH, Tokheim C, Porta-Pardo E, Sengupta S, Bertrand D, Weerasinghe A, Colaprico A, Wendl MC, Kim J, Reardon B, et al. (2018). Comprehensive Characterization of Cancer Driver Genes and Mutations. Cell 173, 371–385.e18. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.02.060\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6029450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29625053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Characterization%20of%20Cancer%20Driver%20Genes%20and%20Mutations&amp;author=MH%20Bailey&amp;author=C%20Tokheim&amp;author=E%20Porta-Pardo&amp;author=S%20Sengupta&amp;author=D%20Bertrand&amp;volume=173&amp;publication_year=2018&amp;pages=371-385.e18&amp;pmid=29625053&amp;doi=10.1016/j.cell.2018.02.060&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R11\"> <cite> Bando H, Atsumi T, Nishio T, Niwa H, Mishima S, Shimizu C, Yoshioka N, Bucala R, and Koike T (2005). Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer. Clin. Cancer Res. 11, 5784–5792. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-05-0149\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16115917/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Phosphorylation%20of%20the%206-phosphofructo-2-kinase/fructose%202,6-bisphosphatase/PFKFB3%20family%20of%20glycolytic%20regulators%20in%20human%20cancer&amp;author=H%20Bando&amp;author=T%20Atsumi&amp;author=T%20Nishio&amp;author=H%20Niwa&amp;author=S%20Mishima&amp;volume=11&amp;publication_year=2005&amp;pages=5784-5792&amp;pmid=16115917&amp;doi=10.1158/1078-0432.CCR-05-0149&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R12\"> <cite> Barbie DA, Tamayo P, Boehm JS, Kim SY, Moody SE, Dunn IF, Schinzel AC, Sandy P, Meylan E, Scholl C, et al. (2009). Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108–112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature08460\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2783335/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19847166/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;author=DA%20Barbie&amp;author=P%20Tamayo&amp;author=JS%20Boehm&amp;author=SY%20Kim&amp;author=SE%20Moody&amp;volume=462&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19847166&amp;doi=10.1038/nature08460&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R13\"> <cite> Bellman R (1961). On the approximation of curves by line segments using dynamic programming. Commun. ACM 4, 284. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun.%20ACM&amp;title=On%20the%20approximation%20of%20curves%20by%20line%20segments%20using%20dynamic%20programming&amp;author=R%20Bellman&amp;volume=4&amp;publication_year=1961&amp;pages=284&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R14\"> <cite> Benjamini Y, and Hochberg Y (1995). Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat. Soc. Series B Stat. Methodol. 57, 289–300. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol&amp;title=Controlling%20the%20False%20Discovery%20Rate:%20A%20Practical%20and%20Powerful%20Approach%20to%20Multiple%20Testing&amp;author=Y%20Benjamini&amp;author=Y%20Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R15\"> <cite> Bray F, Ferlay J, Soerjomataram I, Siegel RL, Torre LA, and Jemal A (2018). Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 68, 394–424. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30207593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&amp;author=F%20Bray&amp;author=J%20Ferlay&amp;author=I%20Soerjomataram&amp;author=RL%20Siegel&amp;author=LA%20Torre&amp;volume=68&amp;publication_year=2018&amp;pages=394-424&amp;pmid=30207593&amp;doi=10.3322/caac.21492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R16\"> <cite> Brunet J-P, Tamayo P, Golub TR, and Mesirov JP (2004). Metagenes and molecular pattern discovery using matrix factorization. Proc. Natl. Acad. Sci. U. S. A. 101, 4164–4169. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0308531101\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC384712/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/15016911/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Metagenes%20and%20molecular%20pattern%20discovery%20using%20matrix%20factorization&amp;author=J-P%20Brunet&amp;author=P%20Tamayo&amp;author=TR%20Golub&amp;author=JP%20Mesirov&amp;volume=101&amp;publication_year=2004&amp;pages=4164-4169&amp;pmid=15016911&amp;doi=10.1073/pnas.0308531101&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R17\"> <cite> Cadenas C, Franckenstein D, Schmidt M, Gehrmann M, Hermes M, Geppert B, Schormann W, Maccoux LJ, Schug M, Schumann A, et al. (2010). Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer. Breast Cancer Res. 12, R44. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/bcr2599\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2917039/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20584310/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&amp;title=Role%20of%20thioredoxin%20reductase%201%20and%20thioredoxin%20interacting%20protein%20in%20prognosis%20of%20breast%20cancer&amp;author=C%20Cadenas&amp;author=D%20Franckenstein&amp;author=M%20Schmidt&amp;author=M%20Gehrmann&amp;author=M%20Hermes&amp;volume=12&amp;publication_year=2010&amp;pages=R44&amp;pmid=20584310&amp;doi=10.1186/bcr2599&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R18\"> <cite> Cai J, Fang L, Huang Y, Li R, Yuan J, Yang Y, Zhu X, Chen B, Wu J, and Li M (2013). miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer. Cancer Res. 73, 5402–5415. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-13-0297\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23856247/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=miR-205%20targets%20PTEN%20and%20PHLPP2%20to%20augment%20AKT%20signaling%20and%20drive%20malignant%20phenotypes%20in%20non-small%20cell%20lung%20cancer&amp;author=J%20Cai&amp;author=L%20Fang&amp;author=Y%20Huang&amp;author=R%20Li&amp;author=J%20Yuan&amp;volume=73&amp;publication_year=2013&amp;pages=5402-5415&amp;pmid=23856247&amp;doi=10.1158/0008-5472.CAN-13-0297&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R19\"> <cite> Cammer M, Gevrey J-C, Lorenz M, Dovas A, Condeelis J, and Cox D (2009). The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42. J. Biol. Chem. 284, 23302–23311. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M109.036384\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2749104/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561083/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20mechanism%20of%20CSF-1-induced%20Wiskott-Aldrich%20syndrome%20protein%20activation%20in%20vivo:%20a%20role%20for%20phosphatidylinositol%203-kinase%20and%20Cdc42&amp;author=M%20Cammer&amp;author=J-C%20Gevrey&amp;author=M%20Lorenz&amp;author=A%20Dovas&amp;author=J%20Condeelis&amp;volume=284&amp;publication_year=2009&amp;pages=23302-23311&amp;pmid=19561083&amp;doi=10.1074/jbc.M109.036384&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R20\"> <cite> Campbell JD, Yau C, Bowlby R, Liu Y, Brennan K, Fan H, Taylor AM, Wang C, Walter V, Akbani R, et al. (2018). Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas. Cell Rep. 23, 194–212.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2018.03.063\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6002769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29617660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Genomic,%20Pathway%20Network,%20and%20Immunologic%20Features%20Distinguishing%20Squamous%20Carcinomas&amp;author=JD%20Campbell&amp;author=C%20Yau&amp;author=R%20Bowlby&amp;author=Y%20Liu&amp;author=K%20Brennan&amp;volume=23&amp;publication_year=2018&amp;pages=194-212.e6&amp;pmid=29617660&amp;doi=10.1016/j.celrep.2018.03.063&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R21\"> <cite> Cannarile MA, Weisser M, Jacob W, Jegg A-M, Ries CH, and Rüttinger D (2017). Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy. J Immunother Cancer 5, 53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s40425-017-0257-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5514481/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28716061/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Colony-stimulating%20factor%201%20receptor%20(CSF1R)%20inhibitors%20in%20cancer%20therapy&amp;author=MA%20Cannarile&amp;author=M%20Weisser&amp;author=W%20Jacob&amp;author=A-M%20Jegg&amp;author=CH%20Ries&amp;volume=5&amp;publication_year=2017&amp;pages=53&amp;pmid=28716061&amp;doi=10.1186/s40425-017-0257-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R22\"> <cite> Cannito S, Novo E, Compagnone A, Valfrè di Bonzo L, Busletta C, Zamara E, Paternostro C, Povero D, Bandino A, Bozzo F, et al. (2008). Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells. Carcinogenesis 29, 2267–2278. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgn216\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18791199/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Redox%20mechanisms%20switch%20on%20hypoxia-dependent%20epithelial-mesenchymal%20transition%20in%20cancer%20cells&amp;author=S%20Cannito&amp;author=E%20Novo&amp;author=A%20Compagnone&amp;author=L%20Valfr%C3%A8%20di%20Bonzo&amp;author=C%20Busletta&amp;volume=29&amp;publication_year=2008&amp;pages=2267-2278&amp;pmid=18791199&amp;doi=10.1093/carcin/bgn216&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R23\"> <cite> Cassetta L, and Pollard JW (2018). Targeting macrophages: therapeutic approaches in cancer. Nat. Rev. Drug Discov. 17, 887–904. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrd.2018.169\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30361552/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov&amp;title=Targeting%20macrophages:%20therapeutic%20approaches%20in%20cancer&amp;author=L%20Cassetta&amp;author=JW%20Pollard&amp;volume=17&amp;publication_year=2018&amp;pages=887-904&amp;pmid=30361552&amp;doi=10.1038/nrd.2018.169&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R24\"> <cite> Chida S, Okayama H, Noda M, Saito K, Nakajima T, Aoto K, Hayase S, Momma T, Ohki S, Kono K, et al. (2016). Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer. Carcinogenesis 37, 878–887. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgw069\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27287872/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Stromal%20VCAN%20expression%20as%20a%20potential%20prognostic%20biomarker%20for%20disease%20recurrence%20in%20stage%20II-III%20colon%20cancer&amp;author=S%20Chida&amp;author=H%20Okayama&amp;author=M%20Noda&amp;author=K%20Saito&amp;author=T%20Nakajima&amp;volume=37&amp;publication_year=2016&amp;pages=878-887&amp;pmid=27287872&amp;doi=10.1093/carcin/bgw069&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R25\"> <cite> Debebe Z, and Rathmell WK (2015). Ror2 as a therapeutic target in cancer. Pharmacol. Ther. 150, 143–148. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.pharmthera.2015.01.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC9236825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25614331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&amp;title=Ror2%20as%20a%20therapeutic%20target%20in%20cancer&amp;author=Z%20Debebe&amp;author=WK%20Rathmell&amp;volume=150&amp;publication_year=2015&amp;pages=143-148&amp;pmid=25614331&amp;doi=10.1016/j.pharmthera.2015.01.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R26\"> <cite> DeBerardinis RJ, and Chandel NS (2016). Fundamentals of cancer metabolism. Sci Adv 2, e1600200. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/sciadv.1600200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4928883/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27386546/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&amp;title=Fundamentals%20of%20cancer%20metabolism&amp;author=RJ%20DeBerardinis&amp;author=NS%20Chandel&amp;volume=2&amp;publication_year=2016&amp;pages=e1600200&amp;pmid=27386546&amp;doi=10.1126/sciadv.1600200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R27\"> <cite> Deelen P, Bonder MJ, van der Velde KJ, Westra H-J, Winder E, Hendriksen D, Franke L, and Swertz MA (2014). Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration. BMC Res. Notes 7, 901. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1756-0500-7-901\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4307387/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25495213/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&amp;title=Genotype%20harmonizer:%20automatic%20strand%20alignment%20and%20format%20conversion%20for%20genotype%20data%20integration&amp;author=P%20Deelen&amp;author=MJ%20Bonder&amp;author=KJ%20van%20der%20Velde&amp;author=H-J%20Westra&amp;author=E%20Winder&amp;volume=7&amp;publication_year=2014&amp;pages=901&amp;pmid=25495213&amp;doi=10.1186/1756-0500-7-901&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R28\"> <cite> DeYoung MP, Johannessen CM, Leong C-O, Faquin W, Rocco JW, and Ellisen LW (2006). Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma. Cancer Res. 66, 9362–9368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-06-1619\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17018588/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Tumor-specific%20p73%20up-regulation%20mediates%20p63%20dependence%20in%20squamous%20cell%20carcinoma&amp;author=MP%20DeYoung&amp;author=CM%20Johannessen&amp;author=C-O%20Leong&amp;author=W%20Faquin&amp;author=JW%20Rocco&amp;volume=66&amp;publication_year=2006&amp;pages=9362-9368&amp;pmid=17018588&amp;doi=10.1158/0008-5472.CAN-06-1619&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R29\"> <cite> Diviani D, Raimondi F, Del Vescovo CD, Dreyer E, Reggi E, Osman H, Ruggieri L, Gonano C, Cavin S, Box CL, et al. (2016). Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling. Cell Chem Biol 23, 1135–1146. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.chembiol.2016.07.015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27593112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&amp;title=Small-Molecule%20Protein-Protein%20Interaction%20Inhibitor%20of%20Oncogenic%20Rho%20Signaling&amp;author=D%20Diviani&amp;author=F%20Raimondi&amp;author=CD%20Del%20Vescovo&amp;author=E%20Dreyer&amp;author=E%20Reggi&amp;volume=23&amp;publication_year=2016&amp;pages=1135-1146&amp;pmid=27593112&amp;doi=10.1016/j.chembiol.2016.07.015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R30\"> <cite> Dovas A, Gevrey J-C, Grossi A, Park H, Abou-Kheir W, and Cox D (2009). Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages. J. Cell Sci. 122, 3873–3882. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.051755\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2773189/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19808890/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Regulation%20of%20podosome%20dynamics%20by%20WASp%20phosphorylation:%20implication%20in%20matrix%20degradation%20and%20chemotaxis%20in%20macrophages&amp;author=A%20Dovas&amp;author=J-C%20Gevrey&amp;author=A%20Grossi&amp;author=H%20Park&amp;author=W%20Abou-Kheir&amp;volume=122&amp;publication_year=2009&amp;pages=3873-3882&amp;pmid=19808890&amp;doi=10.1242/jcs.051755&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R31\"> <cite> Edelman MJ, Redman MW, Albain KS, McGary EC, Rafique NM, Petro D, Waqar SN, Minichiello K, Miao J, Papadimitrakopoulou VA, et al. (2019). SWOG S1400C ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT02154490\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT02154490 </a> )-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy). J. Thorac. Oncol. 14, 1853–1859. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2019.06.027\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6764876/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31302234/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=SWOG%20S1400C%20(NCT02154490)-A%20Phase%20II%20Study%20of%20Palbociclib%20for%20Previously%20Treated%20Cell%20Cycle%20Gene%20Alteration-Positive%20Patients%20with%20Stage%20IV%20Squamous%20Cell%20Lung%20Cancer%20(Lung-MAP%20Substudy)&amp;author=MJ%20Edelman&amp;author=MW%20Redman&amp;author=KS%20Albain&amp;author=EC%20McGary&amp;author=NM%20Rafique&amp;volume=14&amp;publication_year=2019&amp;pages=1853-1859&amp;pmid=31302234&amp;doi=10.1016/j.jtho.2019.06.027&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R32\"> <cite> Ellis MJ, Suman VJ, Hoog J, Goncalves R, Sanati S, Creighton CJ, DeSchryver K, Crouch E, Brink A, Watson M, et al. (2017). Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance). J. Clin. Oncol. 35, 1061–1069. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2016.69.4406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5455353/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28045625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Ki67%20Proliferation%20Index%20as%20a%20Tool%20for%20Chemotherapy%20Decisions%20During%20and%20After%20Neoadjuvant%20Aromatase%20Inhibitor%20Treatment%20of%20Breast%20Cancer:%20Results%20From%20the%20American%20College%20of%20Surgeons%20Oncology%20Group%20Z1031%20Trial%20(Alliance)&amp;author=MJ%20Ellis&amp;author=VJ%20Suman&amp;author=J%20Hoog&amp;author=R%20Goncalves&amp;author=S%20Sanati&amp;volume=35&amp;publication_year=2017&amp;pages=1061-1069&amp;pmid=28045625&amp;doi=10.1200/JCO.2016.69.4406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R33\"> <cite> Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B, et al. (2018). The Reactome Pathway Knowledgebase. Nucleic Acids Res. 46, D649–D655. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29145629/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Reactome%20Pathway%20Knowledgebase&amp;author=A%20Fabregat&amp;author=S%20Jupe&amp;author=L%20Matthews&amp;author=K%20Sidiropoulos&amp;author=M%20Gillespie&amp;volume=46&amp;publication_year=2018&amp;pages=D649-D655&amp;pmid=29145629&amp;doi=10.1093/nar/gkx1132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R34\"> <cite> Fan T, Sun G, Sun X, Zhao L, Zhong R, and Peng Y (2019). Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment. Cancers 11. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers11030317\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6468516/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30845728/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Tumor%20Energy%20Metabolism%20and%20Potential%20of%203-Bromopyruvate%20as%20an%20Inhibitor%20of%20Aerobic%20Glycolysis:%20Implications%20in%20Tumor%20Treatment&amp;author=T%20Fan&amp;author=G%20Sun&amp;author=X%20Sun&amp;author=L%20Zhao&amp;author=R%20Zhong&amp;volume=11&amp;publication_year=2019&amp;pmid=30845728&amp;doi=10.3390/cancers11030317&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R35\"> <cite> Faubert B, Li KY, Cai L, Hensley CT, Kim J, Zacharias LG, Yang C, Do QN, Doucette S, Burguete D, et al. (2017). Lactate Metabolism in Human Lung Tumors. Cell 171, 358–371.e9. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.09.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5684706/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28985563/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Lactate%20Metabolism%20in%20Human%20Lung%20Tumors&amp;author=B%20Faubert&amp;author=KY%20Li&amp;author=L%20Cai&amp;author=CT%20Hensley&amp;author=J%20Kim&amp;volume=171&amp;publication_year=2017&amp;pages=358-371.e9&amp;pmid=28985563&amp;doi=10.1016/j.cell.2017.09.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R36\"> <cite> Fernandes AP, Capitanio A, Selenius M, Brodin O, Rundlöf A-K, and Björnstedt M (2009). Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation. Histopathology 55, 313–320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1365-2559.2009.03381.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19723146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Histopathology&amp;title=Expression%20profiles%20of%20thioredoxin%20family%20proteins%20in%20human%20lung%20cancer%20tissue:%20correlation%20with%20proliferation%20and%20differentiation&amp;author=AP%20Fernandes&amp;author=A%20Capitanio&amp;author=M%20Selenius&amp;author=O%20Brodin&amp;author=A-K%20Rundl%C3%B6f&amp;volume=55&amp;publication_year=2009&amp;pages=313-320&amp;pmid=19723146&amp;doi=10.1111/j.1365-2559.2009.03381.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R37\"> <cite> Fisher S, Barry A, Abreu J, Minie B, Nolan J, Delorey TM, Young G, Fennell TJ, Allen A, Ambrogio L, et al. (2011). A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries. Genome Biol. 12, R1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-1-r1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3091298/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21205303/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=A%20scalable,%20fully%20automated%20process%20for%20construction%20of%20sequence-ready%20human%20exome%20targeted%20capture%20libraries&amp;author=S%20Fisher&amp;author=A%20Barry&amp;author=J%20Abreu&amp;author=B%20Minie&amp;author=J%20Nolan&amp;volume=12&amp;publication_year=2011&amp;pages=R1&amp;pmid=21205303&amp;doi=10.1186/gb-2011-12-1-r1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R38\"> <cite> Foltz DR, Santiago MC, Berechid BE, and Nye JS (2002). Glycogen synthase kinase-3beta modulates notch signaling and stability. Curr. Biol. 12, 1006–1011. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0960-9822(02)00888-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12123574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol&amp;title=Glycogen%20synthase%20kinase-3beta%20modulates%20notch%20signaling%20and%20stability&amp;author=DR%20Foltz&amp;author=MC%20Santiago&amp;author=BE%20Berechid&amp;author=JS%20Nye&amp;volume=12&amp;publication_year=2002&amp;pages=1006-1011&amp;pmid=12123574&amp;doi=10.1016/s0960-9822(02)00888-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R39\"> <cite> Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CM, and Hansen KD (2014). Functional normalization of 450k methylation array data improves replication in large cancer studies. Genome Biol. 15, 503. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0503-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4283580/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25599564/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Functional%20normalization%20of%20450k%20methylation%20array%20data%20improves%20replication%20in%20large%20cancer%20studies&amp;author=J-P%20Fortin&amp;author=A%20Labbe&amp;author=M%20Lemire&amp;author=BW%20Zanke&amp;author=TJ%20Hudson&amp;volume=15&amp;publication_year=2014&amp;pages=503&amp;pmid=25599564&amp;doi=10.1186/s13059-014-0503-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R40\"> <cite> Fraley C, Raftery AE, Scrucca L, Murphy TB, and Fop M (2016). mclust: Gaussian mixture modelling for model-based clustering, classification, and density estimation. R Package Version 5. </cite> [ <a class=\"usa-link\" href=\"/articles/PMC5096736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27818791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=R%20Package%20Version&amp;title=mclust:%20Gaussian%20mixture%20modelling%20for%20model-based%20clustering,%20classification,%20and%20density%20estimation&amp;author=C%20Fraley&amp;author=AE%20Raftery&amp;author=L%20Scrucca&amp;author=TB%20Murphy&amp;author=M%20Fop&amp;volume=5&amp;publication_year=2016&amp;pmid=27818791&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R41\"> <cite> Freed D, Pan R, and Aldana R TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=TNscope:%20Accurate%20Detection%20of%20Somatic%20Mutations%20with%20Haplotype-based%20Variant%20Candidate%20Detection%20and%20Machine%20Learning%20Filtering&amp;author=D%20Freed&amp;author=R%20Pan&amp;author=R%20Aldana&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R42\"> <cite> Fu B, Meng W, Zeng X, Zhao H, Liu W, and Zhang T (2017). TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma. Biomed Res. Int. 2017, 4698167. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1155/2017/4698167\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5425838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28536696/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biomed%20Res.%20Int&amp;title=TXNRD1%20Is%20an%20Unfavorable%20Prognostic%20Factor%20for%20Patients%20with%20Hepatocellular%20Carcinoma&amp;author=B%20Fu&amp;author=W%20Meng&amp;author=X%20Zeng&amp;author=H%20Zhao&amp;author=W%20Liu&amp;volume=2017&amp;publication_year=2017&amp;pages=4698167&amp;pmid=28536696&amp;doi=10.1155/2017/4698167&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R43\"> <cite> Gandhi N, and Das GM (2019). Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications. Cells 8. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cells8020089\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6406734/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30691108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Metabolic%20Reprogramming%20in%20Breast%20Cancer%20and%20Its%20Therapeutic%20Implications&amp;author=N%20Gandhi&amp;author=GM%20Das&amp;volume=8&amp;publication_year=2019&amp;pmid=30691108&amp;doi=10.3390/cells8020089&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R44\"> <cite> Gao C, Zhuang J, Li H, Liu C, Zhou C, Liu L, and Sun C (2018). Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma. Cancer Cell Int. 18, 194. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12935-018-0691-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6258452/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30498398/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&amp;title=Exploration%20of%20methylation-driven%20genes%20for%20monitoring%20and%20prognosis%20of%20patients%20with%20lung%20adenocarcinoma&amp;author=C%20Gao&amp;author=J%20Zhuang&amp;author=H%20Li&amp;author=C%20Liu&amp;author=C%20Zhou&amp;volume=18&amp;publication_year=2018&amp;pages=194&amp;pmid=30498398&amp;doi=10.1186/s12935-018-0691-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R45\"> <cite> Garcia-Mata R, Boulter E, and Burridge K (2011). The “invisible hand”: regulation of RHO GTPases by RHOGDIs. Nat. Rev. Mol. Cell Biol. 12, 493–504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrm3153\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3260518/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21779026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;title=The%20%E2%80%9Cinvisible%20hand%E2%80%9D:%20regulation%20of%20RHO%20GTPases%20by%20RHOGDIs&amp;author=R%20Garcia-Mata&amp;author=E%20Boulter&amp;author=K%20Burridge&amp;volume=12&amp;publication_year=2011&amp;pages=493-504&amp;pmid=21779026&amp;doi=10.1038/nrm3153&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R46\"> <cite> Gaujoux R, and Seoighe C (2010). A flexible R package for nonnegative matrix factorization. BMC Bioinformatics 11, 367. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-11-367\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2912887/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20598126/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=A%20flexible%20R%20package%20for%20nonnegative%20matrix%20factorization&amp;author=R%20Gaujoux&amp;author=C%20Seoighe&amp;volume=11&amp;publication_year=2010&amp;pages=367&amp;pmid=20598126&amp;doi=10.1186/1471-2105-11-367&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R47\"> <cite> Gey S, and Lebarbier E (2008). Using CART to detect multiple change points in the mean for large sample. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Using%20CART%20to%20detect%20multiple%20change%20points%20in%20the%20mean%20for%20large%20sample&amp;author=S%20Gey&amp;author=E%20Lebarbier&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R48\"> <cite> Giaccone G, Zatloukal P, Roubec J, Floor K, Musil J, Kuta M, van Klaveren RJ, Chaudhary S, Gunther A, and Shamsili S (2009). Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer. J. Clin. Oncol. 27, 4481–4486. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.21.1862\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19687333/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Multicenter%20phase%20II%20trial%20of%20YM155,%20a%20small-molecule%20suppressor%20of%20survivin,%20in%20patients%20with%20advanced,%20refractory,%20non-small-cell%20lung%20cancer&amp;author=G%20Giaccone&amp;author=P%20Zatloukal&amp;author=J%20Roubec&amp;author=K%20Floor&amp;author=J%20Musil&amp;volume=27&amp;publication_year=2009&amp;pages=4481-4486&amp;pmid=19687333&amp;doi=10.1200/JCO.2008.21.1862&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R49\"> <cite> Giannakopoulos NV, Luo J-K, Papov V, Zou W, Lenschow DJ, Jacobs BS, Borden EC, Li J, Virgin HW, and Zhang D-E (2005). Proteomic identification of proteins conjugated to ISG15 in mouse and human cells. Biochem. Biophys. Res. Commun. 336, 496–506. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2005.08.132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16139798/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Proteomic%20identification%20of%20proteins%20conjugated%20to%20ISG15%20in%20mouse%20and%20human%20cells&amp;author=NV%20Giannakopoulos&amp;author=J-K%20Luo&amp;author=V%20Papov&amp;author=W%20Zou&amp;author=DJ%20Lenschow&amp;volume=336&amp;publication_year=2005&amp;pages=496-506&amp;pmid=16139798&amp;doi=10.1016/j.bbrc.2005.08.132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R50\"> <cite> Gibbs DL (2020). Robust classification of Immune Subtypes in Cancer. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Robust%20classification%20of%20Immune%20Subtypes%20in%20Cancer&amp;author=DL%20Gibbs&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R51\"> <cite> Gillette MA, Satpathy S, Cao S, Dhanasekaran SM, Vasaikar SV, Krug K, Petralia F, Li Y, Liang W-W, Reva B, et al. (2020). Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma. Cell 182, 200–225.e35. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.06.013\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7373300/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32649874/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Characterization%20Reveals%20Therapeutic%20Vulnerabilities%20in%20Lung%20Adenocarcinoma&amp;author=MA%20Gillette&amp;author=S%20Satpathy&amp;author=S%20Cao&amp;author=SM%20Dhanasekaran&amp;author=SV%20Vasaikar&amp;volume=182&amp;publication_year=2020&amp;pages=200-225.e35&amp;pmid=32649874&amp;doi=10.1016/j.cell.2020.06.013&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R52\"> <cite> Gorvel JP, Chang TC, Boretto J, Azuma T, and Chavrier P (1998). Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity. FEBS Lett. 422, 269–273. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0014-5793(98)00020-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/9490022/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Differential%20properties%20of%20D4/LyGDI%20versus%20RhoGDI:%20phosphorylation%20and%20rho%20GTPase%20selectivity&amp;author=JP%20Gorvel&amp;author=TC%20Chang&amp;author=J%20Boretto&amp;author=T%20Azuma&amp;author=P%20Chavrier&amp;volume=422&amp;publication_year=1998&amp;pages=269-273&amp;pmid=9490022&amp;doi=10.1016/s0014-5793(98)00020-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R53\"> <cite> Gregory PA, Bert AG, Paterson EL, Barry SC, Tsykin A, Farshid G, Vadas MA, Khew-Goodall Y, and Goodall GJ (2008). The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. Nat. Cell Biol. 10, 593–601. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb1722\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18376396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=The%20miR-200%20family%20and%20miR-205%20regulate%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20ZEB1%20and%20SIP1&amp;author=PA%20Gregory&amp;author=AG%20Bert&amp;author=EL%20Paterson&amp;author=SC%20Barry&amp;author=A%20Tsykin&amp;volume=10&amp;publication_year=2008&amp;pages=593-601&amp;pmid=18376396&amp;doi=10.1038/ncb1722&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R54\"> <cite> Hammerman PS, Lawrence MS, Voet D, Jing R, Cibulskis K, Sivachenko A, Stojanov P, McKenna A, Lander ES, Gabriel S, et al. (2012). Comprehensive genomic characterization of squamous cell lung cancers. Nature 489, 519–525. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature11404\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3466113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22960745/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20characterization%20of%20squamous%20cell%20lung%20cancers&amp;author=PS%20Hammerman&amp;author=MS%20Lawrence&amp;author=D%20Voet&amp;author=R%20Jing&amp;author=K%20Cibulskis&amp;volume=489&amp;publication_year=2012&amp;pages=519-525&amp;pmid=22960745&amp;doi=10.1038/nature11404&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R55\"> <cite> Hänzelmann S, Castelo R, and Guinney J (2013). GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 14, 7. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3618321/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23323831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data&amp;author=S%20H%C3%A4nzelmann&amp;author=R%20Castelo&amp;author=J%20Guinney&amp;volume=14&amp;publication_year=2013&amp;pages=7&amp;pmid=23323831&amp;doi=10.1186/1471-2105-14-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R56\"> <cite> Happel N, Stoldt S, Schmidt B, and Doenecke D (2009). M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3. J. Mol. Biol. 386, 339–350. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2008.12.047\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19136008/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=M%20phase-specific%20phosphorylation%20of%20histone%20H1.5%20at%20threonine%2010%20by%20GSK-3&amp;author=N%20Happel&amp;author=S%20Stoldt&amp;author=B%20Schmidt&amp;author=D%20Doenecke&amp;volume=386&amp;publication_year=2009&amp;pages=339-350&amp;pmid=19136008&amp;doi=10.1016/j.jmb.2008.12.047&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R57\"> <cite> Haslam A, and Prasad V (2019). Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs. JAMA Netw Open 2, e192535. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1001/jamanetworkopen.2019.2535\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6503493/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31050774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw%20Open&amp;title=Estimation%20of%20the%20Percentage%20of%20US%20Patients%20With%20Cancer%20Who%20Are%20Eligible%20for%20and%20Respond%20to%20Checkpoint%20Inhibitor%20Immunotherapy%20Drugs&amp;author=A%20Haslam&amp;author=V%20Prasad&amp;volume=2&amp;publication_year=2019&amp;pages=e192535&amp;pmid=31050774&amp;doi=10.1001/jamanetworkopen.2019.2535&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R58\"> <cite> Hebert AS, Dittenhafer-Reed KE, Yu W, Bailey DJ, Selen ES, Boersma MD, Carson JJ, Tonelli M, Balloon AJ, Higbee AJ, et al. (2013). Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome. Mol. Cell 49, 186–199. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molcel.2012.10.024\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3704155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23201123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Calorie%20restriction%20and%20SIRT3%20trigger%20global%20reprogramming%20of%20the%20mitochondrial%20protein%20acetylome&amp;author=AS%20Hebert&amp;author=KE%20Dittenhafer-Reed&amp;author=W%20Yu&amp;author=DJ%20Bailey&amp;author=ES%20Selen&amp;volume=49&amp;publication_year=2013&amp;pages=186-199&amp;pmid=23201123&amp;doi=10.1016/j.molcel.2012.10.024&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R59\"> <cite> Herbst RS, Morgensztern D, and Boshoff C (2018). The biology and management of non-small cell lung cancer. Nature 553, 446–454. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature25183\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29364287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20biology%20and%20management%20of%20non-small%20cell%20lung%20cancer&amp;author=RS%20Herbst&amp;author=D%20Morgensztern&amp;author=C%20Boshoff&amp;volume=553&amp;publication_year=2018&amp;pages=446-454&amp;pmid=29364287&amp;doi=10.1038/nature25183&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R60\"> <cite> Herreño AM, Ramírez AC, Chaparro VP, Fernandez MJ, Cañas A, Morantes CF, Moreno OM, Brugés RE, Mejía JA, Bustos FJ, et al. (2019). Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter. Tumour Biol. 41, 1010428319851014. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1177/1010428319851014\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31109257/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&amp;title=Role%20of%20RUNX2%20transcription%20factor%20in%20epithelial%20mesenchymal%20transition%20in%20non-small%20cell%20lung%20cancer%20lung%20cancer:%20Epigenetic%20control%20of%20the%20RUNX2%20P1%20promoter&amp;author=AM%20Herre%C3%B1o&amp;author=AC%20Ram%C3%ADrez&amp;author=VP%20Chaparro&amp;author=MJ%20Fernandez&amp;author=A%20Ca%C3%B1as&amp;volume=41&amp;publication_year=2019&amp;pages=1010428319851014&amp;pmid=31109257&amp;doi=10.1177/1010428319851014&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R61\"> <cite> Hitosugi T, and Chen J (2014). Post-translational modifications and the Warburg effect. Oncogene 33, 4279–4285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2013.406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24096483/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Post-translational%20modifications%20and%20the%20Warburg%20effect&amp;author=T%20Hitosugi&amp;author=J%20Chen&amp;volume=33&amp;publication_year=2014&amp;pages=4279-4285&amp;pmid=24096483&amp;doi=10.1038/onc.2013.406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R62\"> <cite> Hong C-F, Chen W-Y, and Wu C-W (2017). Upregulation of Wnt signaling under hypoxia promotes lung cancer progression. Oncol. Rep. 38, 1706–1714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/or.2017.5807\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28713928/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep&amp;title=Upregulation%20of%20Wnt%20signaling%20under%20hypoxia%20promotes%20lung%20cancer%20progression&amp;author=C-F%20Hong&amp;author=W-Y%20Chen&amp;author=C-W%20Wu&amp;volume=38&amp;publication_year=2017&amp;pages=1706-1714&amp;pmid=28713928&amp;doi=10.3892/or.2017.5807&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R63\"> <cite> Hornbeck PV, Kornhauser JM, Tkachev S, Zhang B, Skrzypek E, Murray B, Latham V, and Sullivan M (2012). PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse. Nucleic Acids Res. 40, D261–D270. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkr1122\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3245126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22135298/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PhosphoSitePlus:%20a%20comprehensive%20resource%20for%20investigating%20the%20structure%20and%20function%20of%20experimentally%20determined%20post-translational%20modifications%20in%20man%20and%20mouse&amp;author=PV%20Hornbeck&amp;author=JM%20Kornhauser&amp;author=S%20Tkachev&amp;author=B%20Zhang&amp;author=E%20Skrzypek&amp;volume=40&amp;publication_year=2012&amp;pages=D261-D270&amp;pmid=22135298&amp;doi=10.1093/nar/gkr1122&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R64\"> <cite> Hsu E-C, Rice MA, Bermudez A, Marques FJG, Aslan M, Liu S, Ghoochani A, Zhang CA, Chen Y-S, Zlitni A, et al. (2020). Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1. Proc. Natl. Acad. Sci. U. S. A. 117, 2032–2042. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1905384117\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6994991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31932422/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Trop2%20is%20a%20driver%20of%20metastatic%20prostate%20cancer%20with%20neuroendocrine%20phenotype%20via%20PARP1&amp;author=E-C%20Hsu&amp;author=MA%20Rice&amp;author=A%20Bermudez&amp;author=FJG%20Marques&amp;author=M%20Aslan&amp;volume=117&amp;publication_year=2020&amp;pages=2032-2042&amp;pmid=31932422&amp;doi=10.1073/pnas.1905384117&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R65\"> <cite> Huang C, Chen L, Savage SR, Eguez RV, Dou Y, Li Y, da Veiga Leprevost F, Jaehnig EJ, Lei JT, Wen B, et al. (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2020.12.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7946781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33417831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20insights%20into%20the%20biology%20and%20treatment%20of%20HPV-negative%20head%20and%20neck%20squamous%20cell%20carcinoma&amp;author=C%20Huang&amp;author=L%20Chen&amp;author=SR%20Savage&amp;author=RV%20Eguez&amp;author=Y%20Dou&amp;publication_year=2021&amp;pmid=33417831&amp;doi=10.1016/j.ccell.2020.12.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R66\"> <cite> Huang H-Y, Lin Y-C-D, Li J, Huang K-Y, Shrestha S, Hong H-C, Tang Y, Chen Y-G, Jin C-N, Yu Y, et al. (2020). miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database. Nucleic Acids Res. 48, D148–D154. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz896\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145596/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31647101/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRTarBase%202020:%20updates%20to%20the%20experimentally%20validated%20microRNA-target%20interaction%20database&amp;author=H-Y%20Huang&amp;author=Y-C-D%20Lin&amp;author=J%20Li&amp;author=K-Y%20Huang&amp;author=S%20Shrestha&amp;volume=48&amp;publication_year=2020&amp;pages=D148-D154&amp;pmid=31647101&amp;doi=10.1093/nar/gkz896&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R67\"> <cite> Hubaux R, Thu KL, Vucic EA, Pikor LA, Kung SHY, Martinez VD, Mosslemi M, Becker-Santos DD, Gazdar AF, Lam S, et al. (2015). Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer. Int. J. Cancer 137, 2072–2082. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/ijc.29577\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4537683/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25907283/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&amp;title=Microtubule%20affinity-regulating%20kinase%202%20is%20associated%20with%20DNA%20damage%20response%20and%20cisplatin%20resistance%20in%20non-small%20cell%20lung%20cancer&amp;author=R%20Hubaux&amp;author=KL%20Thu&amp;author=EA%20Vucic&amp;author=LA%20Pikor&amp;author=SHY%20Kung&amp;volume=137&amp;publication_year=2015&amp;pages=2072-2082&amp;pmid=25907283&amp;doi=10.1002/ijc.29577&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R68\"> <cite> Hung YP, Redig A, Hornick JL, and Sholl LM (2020). ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis. Mod. Pathol. 33, 2256–2268. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41379-020-0592-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32572156/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mod.%20Pathol&amp;title=ARID1A%20mutations%20and%20expression%20loss%20in%20non-small%20cell%20lung%20carcinomas:%20clinicopathologic%20and%20molecular%20analysis&amp;author=YP%20Hung&amp;author=A%20Redig&amp;author=JL%20Hornick&amp;author=LM%20Sholl&amp;volume=33&amp;publication_year=2020&amp;pages=2256-2268&amp;pmid=32572156&amp;doi=10.1038/s41379-020-0592-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R69\"> <cite> Ishitani T, Hirao T, Suzuki M, Isoda M, Ishitani S, Harigaya K, Kitagawa M, Matsumoto K, and Itoh M (2010). Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex. Nat. Cell Biol. 12, 278–285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb2028\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20118921/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=Nemo-like%20kinase%20suppresses%20Notch%20signalling%20by%20interfering%20with%20formation%20of%20the%20Notch%20active%20transcriptional%20complex&amp;author=T%20Ishitani&amp;author=T%20Hirao&amp;author=M%20Suzuki&amp;author=M%20Isoda&amp;author=S%20Ishitani&amp;volume=12&amp;publication_year=2010&amp;pages=278-285&amp;pmid=20118921&amp;doi=10.1038/ncb2028&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R70\"> <cite> Jimenez-Blasco D, Santofimia-Castaño P, Gonzalez A, Almeida A, and Bolaños JP (2015). Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway. Cell Death Differ. 22, 1877–1889. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cdd.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4648333/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25909891/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Astrocyte%20NMDA%20receptors%E2%80%99%20activity%20sustains%20neuronal%20survival%20through%20a%20Cdk5-Nrf2%20pathway&amp;author=D%20Jimenez-Blasco&amp;author=P%20Santofimia-Casta%C3%B1o&amp;author=A%20Gonzalez&amp;author=A%20Almeida&amp;author=JP%20Bola%C3%B1os&amp;volume=22&amp;publication_year=2015&amp;pages=1877-1889&amp;pmid=25909891&amp;doi=10.1038/cdd.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R71\"> <cite> Jin YH, Kim H, Oh M, Ki H, and Kim K (2009). Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta. Mol. Cells 27, 15–19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/s10059-009-0001-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19214430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cells&amp;title=Regulation%20of%20Notch1/NICD%20and%20Hes1%20expressions%20by%20GSK-3alpha/beta&amp;author=YH%20Jin&amp;author=H%20Kim&amp;author=M%20Oh&amp;author=H%20Ki&amp;author=K%20Kim&amp;volume=27&amp;publication_year=2009&amp;pages=15-19&amp;pmid=19214430&amp;doi=10.1007/s10059-009-0001-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R72\"> <cite> Johnson WE, Li C, and Rabinovic A (2007). Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostatistics 8, 118–127. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/biostatistics/kxj037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16632515/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Adjusting%20batch%20effects%20in%20microarray%20expression%20data%20using%20empirical%20Bayes%20methods&amp;author=WE%20Johnson&amp;author=C%20Li&amp;author=A%20Rabinovic&amp;volume=8&amp;publication_year=2007&amp;pages=118-127&amp;pmid=16632515&amp;doi=10.1093/biostatistics/kxj037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R73\"> <cite> Jurtz V, Paul S, Andreatta M, Marcatili P, Peters B, and Nielsen M (2017). NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data. J. Immunol. 199, 3360–3368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4049/jimmunol.1700893\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5679736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28978689/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Immunol&amp;title=NetMHCpan-4.0:%20Improved%20Peptide-MHC%20Class%20I%20Interaction%20Predictions%20Integrating%20Eluted%20Ligand%20and%20Peptide%20Binding%20Affinity%20Data&amp;author=V%20Jurtz&amp;author=S%20Paul&amp;author=M%20Andreatta&amp;author=P%20Marcatili&amp;author=B%20Peters&amp;volume=199&amp;publication_year=2017&amp;pages=3360-3368&amp;pmid=28978689&amp;doi=10.4049/jimmunol.1700893&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R74\"> <cite> Kamburov A, Lawrence MS, Polak P, Leshchiner I, Lage K, Golub TR, Lander ES, and Getz G (2015). Comprehensive assessment of cancer missense mutation clustering in protein structures. Proc. Natl. Acad. Sci. U. S. A. 112, E5486–E5495. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1516373112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4603469/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26392535/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Comprehensive%20assessment%20of%20cancer%20missense%20mutation%20clustering%20in%20protein%20structures&amp;author=A%20Kamburov&amp;author=MS%20Lawrence&amp;author=P%20Polak&amp;author=I%20Leshchiner&amp;author=K%20Lage&amp;volume=112&amp;publication_year=2015&amp;pages=E5486-E5495&amp;pmid=26392535&amp;doi=10.1073/pnas.1516373112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R75\"> <cite> Kanehisa M, Furumichi M, Tanabe M, Sato Y, and Morishima K (2017). KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353–D361. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkw1092\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5210567/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27899662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20new%20perspectives%20on%20genomes,%20pathways,%20diseases%20and%20drugs&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=M%20Tanabe&amp;author=Y%20Sato&amp;author=K%20Morishima&amp;volume=45&amp;publication_year=2017&amp;pages=D353-D361&amp;pmid=27899662&amp;doi=10.1093/nar/gkw1092&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R76\"> <cite> Karachaliou N, Fernandez-Bruno M, and Rosell R (2018). Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer? Transl Lung Cancer Res 7, S198–S201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.21037/tlcr.2018.07.02\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6193917/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30393600/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Transl%20Lung%20Cancer%20Res&amp;title=Strategies%20for%20first-line%20immunotherapy%20in%20squamous%20cell%20lung%20cancer:%20are%20combinations%20a%20game%20changer?&amp;author=N%20Karachaliou&amp;author=M%20Fernandez-Bruno&amp;author=R%20Rosell&amp;volume=7&amp;publication_year=2018&amp;pages=S198-S201&amp;pmid=30393600&amp;doi=10.21037/tlcr.2018.07.02&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R77\"> <cite> Kelly RJ, Thomas A, Rajan A, Chun G, Lopez-Chavez A, Szabo E, Spencer S, Carter CA, Guha U, Khozin S, et al. (2013). A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer. Ann. Oncol. 24, 2601–2606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/annonc/mdt249\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3784336/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23857959/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=A%20phase%20I/II%20study%20of%20sepantronium%20bromide%20(YM155,%20survivin%20suppressor)%20with%20paclitaxel%20and%20carboplatin%20in%20patients%20with%20advanced%20non-small-cell%20lung%20cancer&amp;author=RJ%20Kelly&amp;author=A%20Thomas&amp;author=A%20Rajan&amp;author=G%20Chun&amp;author=A%20Lopez-Chavez&amp;volume=24&amp;publication_year=2013&amp;pages=2601-2606&amp;pmid=23857959&amp;doi=10.1093/annonc/mdt249&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R78\"> <cite> Keysar SB, Le PN, Miller B, Jackson BC, Eagles JR, Nieto C, Kim J, Tang B, Glogowska MJ, Morton JJ, et al. (2017). Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2. J. Natl. Cancer Inst. 109. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/jnci/djw189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5025278/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27634934/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&amp;title=Regulation%20of%20Head%20and%20Neck%20Squamous%20Cancer%20Stem%20Cells%20by%20PI3K%20and%20SOX2&amp;author=SB%20Keysar&amp;author=PN%20Le&amp;author=B%20Miller&amp;author=BC%20Jackson&amp;author=JR%20Eagles&amp;volume=109&amp;publication_year=2017&amp;pmid=27634934&amp;doi=10.1093/jnci/djw189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R79\"> <cite> Kikuchi T, Hassanein M, Amann JM, Liu Q, Slebos RJC, Rahman SMJ, Kaufman JM, Zhang X, Hoeksema MD, Harris BK, et al. (2012). In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers. Mol. Cell. Proteomics 11, 916–932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M111.015370\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3494148/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22761400/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=In-depth%20proteomic%20analysis%20of%20nonsmall%20cell%20lung%20cancer%20to%20discover%20molecular%20targets%20and%20candidate%20biomarkers&amp;author=T%20Kikuchi&amp;author=M%20Hassanein&amp;author=JM%20Amann&amp;author=Q%20Liu&amp;author=RJC%20Slebos&amp;volume=11&amp;publication_year=2012&amp;pages=916-932&amp;pmid=22761400&amp;doi=10.1074/mcp.M111.015370&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R80\"> <cite> Kim H, and Park H (2007). Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis. Bioinformatics 23, 1495–1502. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17483501/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Sparse%20non-negative%20matrix%20factorizations%20via%20alternating%20non-negativity-constrained%20least%20squares%20for%20microarray%20data%20analysis&amp;author=H%20Kim&amp;author=H%20Park&amp;volume=23&amp;publication_year=2007&amp;pages=1495-1502&amp;pmid=17483501&amp;doi=10.1093/bioinformatics/btm134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R81\"> <cite> Kim S, and Pevzner PA (2014). MS-GF+ makes progress towards a universal database search tool for proteomics. Nat. Commun. 5, 5277. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms6277\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5036525/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25358478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=MS-GF+%20makes%20progress%20towards%20a%20universal%20database%20search%20tool%20for%20proteomics&amp;author=S%20Kim&amp;author=PA%20Pevzner&amp;volume=5&amp;publication_year=2014&amp;pages=5277&amp;pmid=25358478&amp;doi=10.1038/ncomms6277&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R82\"> <cite> Kim J, Mouw KW, Polak P, Braunstein LZ, Kamburov A, Kwiatkowski DJ, Rosenberg JE, Van Allen EM, D’Andrea A, and Getz G (2016). Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. Nat. Genet. 48, 600–606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3557\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4936490/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27111033/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&amp;author=J%20Kim&amp;author=KW%20Mouw&amp;author=P%20Polak&amp;author=LZ%20Braunstein&amp;author=A%20Kamburov&amp;volume=48&amp;publication_year=2016&amp;pages=600-606&amp;pmid=27111033&amp;doi=10.1038/ng.3557&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R83\"> <cite> Klepsch V, Hermann-Kleiter N, Do-Dinh P, Jakic B, Offermann A, Efremova M, Sopper S, Rieder D, Krogsdam A, Gamerith G, et al. (2018). Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade. Nat. Commun. 9, 1538. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-018-04004-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5906604/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29670099/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Nuclear%20receptor%20NR2F6%20inhibition%20potentiates%20responses%20to%20PD-L1/PD-1%20cancer%20immune%20checkpoint%20blockade&amp;author=V%20Klepsch&amp;author=N%20Hermann-Kleiter&amp;author=P%20Do-Dinh&amp;author=B%20Jakic&amp;author=A%20Offermann&amp;volume=9&amp;publication_year=2018&amp;pages=1538&amp;pmid=29670099&amp;doi=10.1038/s41467-018-04004-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R84\"> <cite> von Klot C-A, Dubrowinskaja N, Peters I, Hennenlotter J, Merseburger AS, Stenzl A, Kuczyk MA, and Serth J (2017). Rho GDP dissociation inhibitor-β in renal cell carcinoma. Oncol. Lett. 14, 8190–8196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/ol.2017.7233\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5727609/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29250194/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett&amp;title=Rho%20GDP%20dissociation%20inhibitor-%CE%B2%20in%20renal%20cell%20carcinoma&amp;author=C-A%20von%20Klot&amp;author=N%20Dubrowinskaja&amp;author=I%20Peters&amp;author=J%20Hennenlotter&amp;author=AS%20Merseburger&amp;volume=14&amp;publication_year=2017&amp;pages=8190-8196&amp;pmid=29250194&amp;doi=10.3892/ol.2017.7233&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R85\"> <cite> Knudsen ES, Nambiar R, Rosario SR, Smiraglia DJ, Goodrich DW, and Witkiewicz AK (2020). Pan-cancer molecular analysis of the RB tumor suppressor pathway. Commun Biol 3, 158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s42003-020-0873-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7118159/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32242058/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Pan-cancer%20molecular%20analysis%20of%20the%20RB%20tumor%20suppressor%20pathway&amp;author=ES%20Knudsen&amp;author=R%20Nambiar&amp;author=SR%20Rosario&amp;author=DJ%20Smiraglia&amp;author=DW%20Goodrich&amp;volume=3&amp;publication_year=2020&amp;pages=158&amp;pmid=32242058&amp;doi=10.1038/s42003-020-0873-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R86\"> <cite> Krämer A, Green J, Pollard J Jr, and Tugendreich S (2014). Causal analysis approaches in Ingenuity Pathway Analysis. Bioinformatics 30, 523–530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt703\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3928520/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24336805/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Causal%20analysis%20approaches%20in%20Ingenuity%20Pathway%20Analysis&amp;author=A%20Kr%C3%A4mer&amp;author=J%20Green&amp;author=J%20Pollard&amp;author=S%20Tugendreich&amp;volume=30&amp;publication_year=2014&amp;pages=523-530&amp;pmid=24336805&amp;doi=10.1093/bioinformatics/btt703&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R87\"> <cite> Kröger C, Afeyan A, Mraz J, Eaton EN, Reinhardt F, Khodor YL, Thiru P, Bierie B, Ye X, Burge CB, et al. (2019). Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells. Proc. Natl. Acad. Sci. U. S. A. 116, 7353–7362. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1812876116\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6462070/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30910979/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Acquisition%20of%20a%20hybrid%20E/M%20state%20is%20essential%20for%20tumorigenicity%20of%20basal%20breast%20cancer%20cells&amp;author=C%20Kr%C3%B6ger&amp;author=A%20Afeyan&amp;author=J%20Mraz&amp;author=EN%20Eaton&amp;author=F%20Reinhardt&amp;volume=116&amp;publication_year=2019&amp;pages=7353-7362&amp;pmid=30910979&amp;doi=10.1073/pnas.1812876116&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R88\"> <cite> Krug K, Mertins P, Zhang B, Hornbeck P, Raju R, Ahmad R, Szucs M, Mundt F, Forestier D, Jane-Valbuena J, et al. (2018). A curated resource for phosphosite-specific signature analysis. Mol. Cell. Proteomics. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR118.000943\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6398202/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30563849/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20curated%20resource%20for%20phosphosite-specific%20signature%20analysis&amp;author=K%20Krug&amp;author=P%20Mertins&amp;author=B%20Zhang&amp;author=P%20Hornbeck&amp;author=R%20Raju&amp;publication_year=2018&amp;pmid=30563849&amp;doi=10.1074/mcp.TIR118.000943&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R89\"> <cite> Krug K, Jaehnig EJ, Satpathy S, Blumenberg L, Karpova A, Anurag M, Miles G, Mertins P, Geffen Y, Tang LC, et al. (2020). Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy. Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.10.036\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8077737/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33212010/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Landscape%20of%20Breast%20Cancer%20Tumorigenesis%20and%20Targeted%20Therapy&amp;author=K%20Krug&amp;author=EJ%20Jaehnig&amp;author=S%20Satpathy&amp;author=L%20Blumenberg&amp;author=A%20Karpova&amp;publication_year=2020&amp;pmid=33212010&amp;doi=10.1016/j.cell.2020.10.036&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R90\"> <cite> Kuhlmann N, Wroblowski S, Knyphausen P, de Boor S, Brenig J, Zienert AY, Meyer-Teschendorf K, Praefcke GJK, Nolte H, Krüger M, et al. (2016a). Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation. J. Biol. Chem. 291, 5484–5499. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M115.707091\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4786691/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26719334/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Structural%20and%20Mechanistic%20Insights%20into%20the%20Regulation%20of%20the%20Fundamental%20Rho%20Regulator%20RhoGDI%CE%B1%20by%20Lysine%20Acetylation&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=P%20Knyphausen&amp;author=S%20de%20Boor&amp;author=J%20Brenig&amp;volume=291&amp;publication_year=2016a&amp;pages=5484-5499&amp;pmid=26719334&amp;doi=10.1074/jbc.M115.707091&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R91\"> <cite> Kuhlmann N, Wroblowski S, Scislowski L, and Lammers M (2016b). RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA. Biochemistry 55, 304–312. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/acs.biochem.5b01242\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26695096/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=RhoGDI%CE%B1%20Acetylation%20at%20K127%20and%20K141%20Affects%20Binding%20toward%20Nonprenylated%20RhoA&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=L%20Scislowski&amp;author=M%20Lammers&amp;volume=55&amp;publication_year=2016b&amp;pages=304-312&amp;pmid=26695096&amp;doi=10.1021/acs.biochem.5b01242&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R92\"> <cite> Lacher SE, and Slattery M (2016). Gene regulatory effects of disease-associated variation in the NRF2 network. Curr Opin Toxicol 1, 71–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cotox.2016.09.001\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5305174/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28203648/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Toxicol&amp;title=Gene%20regulatory%20effects%20of%20disease-associated%20variation%20in%20the%20NRF2%20network&amp;author=SE%20Lacher&amp;author=M%20Slattery&amp;volume=1&amp;publication_year=2016&amp;pages=71-79&amp;pmid=28203648&amp;doi=10.1016/j.cotox.2016.09.001&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R93\"> <cite> Lamb J, Crawford ED, Peck D, Modell JW, Blat IC, Wrobel MJ, Lerner J, Brunet J-P, Subramanian A, Ross KN, et al. (2006). The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 313, 1929–1935. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1132939\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17008526/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Connectivity%20Map:%20using%20gene-expression%20signatures%20to%20connect%20small%20molecules,%20genes,%20and%20disease&amp;author=J%20Lamb&amp;author=ED%20Crawford&amp;author=D%20Peck&amp;author=JW%20Modell&amp;author=IC%20Blat&amp;volume=313&amp;publication_year=2006&amp;pages=1929-1935&amp;pmid=17008526&amp;doi=10.1126/science.1132939&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R94\"> <cite> Lamprecht S, Kaller M, Schmidt EM, Blaj C, Schiergens TS, Engel J, Jung A, Hermeking H, Grünewald TGP, Kirchner T, et al. (2018). PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer. Clin. Cancer Res. 24, 1974–1986. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-2572\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29391352/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=PBX3%20Is%20Part%20of%20an%20EMT%20Regulatory%20Network%20and%20Indicates%20Poor%20Outcome%20in%20Colorectal%20Cancer&amp;author=S%20Lamprecht&amp;author=M%20Kaller&amp;author=EM%20Schmidt&amp;author=C%20Blaj&amp;author=TS%20Schiergens&amp;volume=24&amp;publication_year=2018&amp;pages=1974-1986&amp;pmid=29391352&amp;doi=10.1158/1078-0432.CCR-17-2572&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R95\"> <cite> Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, Meyerson M, Gabriel SB, Lander ES, and Getz G (2014). Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature12912\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4048962/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24390350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Discovery%20and%20saturation%20analysis%20of%20cancer%20genes%20across%2021%20tumour%20types&amp;author=MS%20Lawrence&amp;author=P%20Stojanov&amp;author=CH%20Mermel&amp;author=JT%20Robinson&amp;author=LA%20Garraway&amp;volume=505&amp;publication_year=2014&amp;pages=495-501&amp;pmid=24390350&amp;doi=10.1038/nature12912&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R96\"> <cite> Lê S, Josse J, and Husson F (2008). FactoMineR: AnRPackage for Multivariate Analysis. Journal of Statistical Software 25. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=FactoMineR:%20AnRPackage%20for%20Multivariate%20Analysis&amp;author=S%20L%C3%AA&amp;author=J%20Josse&amp;author=F%20Husson&amp;volume=25&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R97\"> <cite> Lebanony D, Benjamin H, Gilad S, Ezagouri M, Dov A, Ashkenazi K, Gefen N, Izraeli S, Rechavi G, Pass H, et al. (2009). Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma. J. Clin. Oncol. 27, 2030–2037. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.19.4134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19273703/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Diagnostic%20assay%20based%20on%20hsa-miR-205%20expression%20distinguishes%20squamous%20from%20nonsquamous%20non-small-cell%20lung%20carcinoma&amp;author=D%20Lebanony&amp;author=H%20Benjamin&amp;author=S%20Gilad&amp;author=M%20Ezagouri&amp;author=A%20Dov&amp;volume=27&amp;publication_year=2009&amp;pages=2030-2037&amp;pmid=19273703&amp;doi=10.1200/JCO.2008.19.4134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R98\"> <cite> Lebarbier E (2005). Detecting multiple change-points in the mean of Gaussian process by model selection. Signal Processing 85, 717–736. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Signal%20Processing&amp;title=Detecting%20multiple%20change-points%20in%20the%20mean%20of%20Gaussian%20process%20by%20model%20selection&amp;author=E%20Lebarbier&amp;volume=85&amp;publication_year=2005&amp;pages=717-736&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R99\"> <cite> Le Large TY, Mantini G, Meijer LL, Pham TV, Funel N, van Grieken NC, Kok B, Knol J, van Laarhoven HW, Piersma SR, et al. (2020). Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets. JCI Insight 5. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1172/jci.insight.138290\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7455080/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32634123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCI%20Insight&amp;title=Microdissected%20pancreatic%20cancer%20proteomes%20reveal%20tumor%20heterogeneity%20and%20therapeutic%20targets&amp;author=TY%20Le%20Large&amp;author=G%20Mantini&amp;author=LL%20Meijer&amp;author=TV%20Pham&amp;author=N%20Funel&amp;volume=5&amp;publication_year=2020&amp;pmid=32634123&amp;doi=10.1172/jci.insight.138290&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R100\"> <cite> Lewandowski KT, and Piwnica-Worms H (2014). Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity. J. Cell Sci. 127, 315–327. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.129148\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3889397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24259665/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Phosphorylation%20of%20the%20E3%20ubiquitin%20ligase%20RNF41%20by%20the%20kinase%20Par-1b%20is%20required%20for%20epithelial%20cell%20polarity&amp;author=KT%20Lewandowski&amp;author=H%20Piwnica-Worms&amp;volume=127&amp;publication_year=2014&amp;pages=315-327&amp;pmid=24259665&amp;doi=10.1242/jcs.129148&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R101\"> <cite> Li F, Aljahdali I, and Ling X (2019a). Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study? J. Exp. Clin. Cancer Res. 38, 368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13046-019-1362-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6704566/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31439015/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Clin.%20Cancer%20Res&amp;title=Cancer%20therapeutics%20using%20survivin%20BIRC5%20as%20a%20target:%20what%20can%20we%20do%20after%20over%20two%20decades%20of%20study?&amp;author=F%20Li&amp;author=I%20Aljahdali&amp;author=X%20Ling&amp;volume=38&amp;publication_year=2019a&amp;pages=368&amp;pmid=31439015&amp;doi=10.1186/s13046-019-1362-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R102\"> <cite> Li K, Vaudel M, Zhang B, Ren Y, and Wen B (2019b). PDV: an integrative proteomics data viewer. Bioinformatics 35, 1249–1251. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty770\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6821182/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30169737/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PDV:%20an%20integrative%20proteomics%20data%20viewer&amp;author=K%20Li&amp;author=M%20Vaudel&amp;author=B%20Zhang&amp;author=Y%20Ren&amp;author=B%20Wen&amp;volume=35&amp;publication_year=2019b&amp;pages=1249-1251&amp;pmid=30169737&amp;doi=10.1093/bioinformatics/bty770&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R103\"> <cite> Li M, Xie X, Zhou J, Sheng M, Yin X, Ko E-A, Zhou T, and Gu W (2017). Quantifying circular RNA expression from RNA-seq data using model-based framework. Bioinformatics 33, 2131–2139. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btx129\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28334396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quantifying%20circular%20RNA%20expression%20from%20RNA-seq%20data%20using%20model-based%20framework&amp;author=M%20Li&amp;author=X%20Xie&amp;author=J%20Zhou&amp;author=M%20Sheng&amp;author=X%20Yin&amp;volume=33&amp;publication_year=2017&amp;pages=2131-2139&amp;pmid=28334396&amp;doi=10.1093/bioinformatics/btx129&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R104\"> <cite> Li P, Cao G, Zhang Y, Shi J, Cai K, Zhen L, He X, Zhou Y, Li Y, Zhu Y, et al. (2020). FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors. Aging 12, 53–69. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/aging.102564\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6977653/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31935687/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Aging&amp;title=FHL3%20promotes%20pancreatic%20cancer%20invasion%20and%20metastasis%20through%20preventing%20the%20ubiquitination%20degradation%20of%20EMT%20associated%20transcription%20factors&amp;author=P%20Li&amp;author=G%20Cao&amp;author=Y%20Zhang&amp;author=J%20Shi&amp;author=K%20Cai&amp;volume=12&amp;publication_year=2020&amp;pages=53-69&amp;pmid=31935687&amp;doi=10.18632/aging.102564&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R105\"> <cite> Liang Y, Liu M, Wang P, Ding X, and Cao Y (2013). Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer. Genes Chromosomes Cancer 52, 305–315. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/gcc.22030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23225332/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&amp;title=Analysis%20of%2020%20genes%20at%20chromosome%20band%2012q13:%20RACGAP1%20and%20MCRS1%20overexpression%20in%20nonsmall-cell%20lung%20cancer&amp;author=Y%20Liang&amp;author=M%20Liu&amp;author=P%20Wang&amp;author=X%20Ding&amp;author=Y%20Cao&amp;volume=52&amp;publication_year=2013&amp;pages=305-315&amp;pmid=23225332&amp;doi=10.1002/gcc.22030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R106\"> <cite> Liao Y, Wang J, Jaehnig EJ, Shi Z, and Zhang B (2019). WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Nucleic Acids Res. 47, W199–W205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602449/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31114916/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202019:%20gene%20set%20analysis%20toolkit%20with%20revamped%20UIs%20and%20APIs&amp;author=Y%20Liao&amp;author=J%20Wang&amp;author=EJ%20Jaehnig&amp;author=Z%20Shi&amp;author=B%20Zhang&amp;volume=47&amp;publication_year=2019&amp;pages=W199-W205&amp;pmid=31114916&amp;doi=10.1093/nar/gkz401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R107\"> <cite> Liberzon A, Birger C, Thorvaldsdóttir H, Ghandi M, Mesirov JP, and Tamayo P (2015). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst 1, 417–425. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cels.2015.12.004\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4707969/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26771021/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20Molecular%20Signatures%20Database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;author=A%20Liberzon&amp;author=C%20Birger&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=M%20Ghandi&amp;author=JP%20Mesirov&amp;volume=1&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R108\"> <cite> Lim SH, Sun J-M, Choi Y-L, Kim HR, Ahn S, Lee JY, Lee S-H, Ahn JS, Park K, Kim JH, et al. (2016). Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study. Cancer 122, 3024–3031. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/cncr.30135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27315356/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Efficacy%20and%20safety%20of%20dovitinib%20in%20pretreated%20patients%20with%20advanced%20squamous%20non-small%20cell%20lung%20cancer%20with%20FGFR1%20amplification:%20A%20single-arm,%20phase%202%20study&amp;author=SH%20Lim&amp;author=J-M%20Sun&amp;author=Y-L%20Choi&amp;author=HR%20Kim&amp;author=S%20Ahn&amp;volume=122&amp;publication_year=2016&amp;pages=3024-3031&amp;pmid=27315356&amp;doi=10.1002/cncr.30135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R109\"> <cite> Lim YW, Chen-Harris H, Mayba O, Lianoglou S, Wuster A, Bhangale T, Khan Z, Mariathasan S, Daemen A, Reeder J, et al. (2018). Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration. Proc. Natl. Acad. Sci. U. S. A. 115, E11701–E11710. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1804506115\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6294879/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30463956/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Germline%20genetic%20polymorphisms%20influence%20tumor%20gene%20expression%20and%20immune%20cell%20infiltration&amp;author=YW%20Lim&amp;author=H%20Chen-Harris&amp;author=O%20Mayba&amp;author=S%20Lianoglou&amp;author=A%20Wuster&amp;volume=115&amp;publication_year=2018&amp;pages=E11701-E11710&amp;pmid=30463956&amp;doi=10.1073/pnas.1804506115&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R110\"> <cite> Lindsay J, McDade SS, Pickard A, McCloskey KD, and McCance DJ (2011). Role of DeltaNp63gamma in epithelial to mesenchymal transition. J. Biol. Chem. 286, 3915–3924. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M110.162511\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3030392/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21127042/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Role%20of%20DeltaNp63gamma%20in%20epithelial%20to%20mesenchymal%20transition&amp;author=J%20Lindsay&amp;author=SS%20McDade&amp;author=A%20Pickard&amp;author=KD%20McCloskey&amp;author=DJ%20McCance&amp;volume=286&amp;publication_year=2011&amp;pages=3915-3924&amp;pmid=21127042&amp;doi=10.1074/jbc.M110.162511&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R111\"> <cite> Liu W, Payne SH, Ma S, and Fenyö D (2019). Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis. Mol. Cell. Proteomics 18, S169–S182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR119.001442\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692784/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31213479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Extracting%20Pathway-level%20Signatures%20from%20Proteogenomic%20Data%20in%20Breast%20Cancer%20Using%20Independent%20Component%20Analysis&amp;author=W%20Liu&amp;author=SH%20Payne&amp;author=S%20Ma&amp;author=D%20Feny%C3%B6&amp;volume=18&amp;publication_year=2019&amp;pages=S169-S182&amp;pmid=31213479&amp;doi=10.1074/mcp.TIR119.001442&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R112\"> <cite> Liu X, Wang L, Cui W, Yuan X, Lin L, Cao Q, Wang N, Li Y, Guo W, Zhang X, et al. (2016). Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells. Oncotarget 7, 58516–58530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/oncotarget.11305\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5295448/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27542268/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Targeting%20ALDH1A1%20by%20disulfiram/copper%20complex%20inhibits%20non-small%20cell%20lung%20cancer%20recurrence%20driven%20by%20ALDH-positive%20cancer%20stem%20cells&amp;author=X%20Liu&amp;author=L%20Wang&amp;author=W%20Cui&amp;author=X%20Yuan&amp;author=L%20Lin&amp;volume=7&amp;publication_year=2016&amp;pages=58516-58530&amp;pmid=27542268&amp;doi=10.18632/oncotarget.11305&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R113\"> <cite> Lo Iacono M, Monica V, Saviozzi S, Ceppi P, Bracco E, Papotti M, and Scagliotti GV (2011). p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue. J. Thorac. Oncol. 6, 473–481. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e31820b86b0\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21289519/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=p63%20and%20p73%20isoform%20expression%20in%20non-small%20cell%20lung%20cancer%20and%20corresponding%20morphological%20normal%20lung%20tissue&amp;author=M%20Lo%20Iacono&amp;author=V%20Monica&amp;author=S%20Saviozzi&amp;author=P%20Ceppi&amp;author=E%20Bracco&amp;volume=6&amp;publication_year=2011&amp;pages=473-481&amp;pmid=21289519&amp;doi=10.1097/JTO.0b013e31820b86b0&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R114\"> <cite> Maksimovic J, Phipson B, and Oshlack A (2016). A cross-package Bioconductor workflow for analysing methylation array data. F1000Res. 5, 1281. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.12688/f1000research.8839.1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4916993/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27347385/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=A%20cross-package%20Bioconductor%20workflow%20for%20analysing%20methylation%20array%20data&amp;author=J%20Maksimovic&amp;author=B%20Phipson&amp;author=A%20Oshlack&amp;volume=5&amp;publication_year=2016&amp;pages=1281&amp;pmid=27347385&amp;doi=10.12688/f1000research.8839.1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R115\"> <cite> Malhotra D, Portales-Casamar E, Singh A, Srivastava S, Arenillas D, Happel C, Shyr C, Wakabayashi N, Kensler TW, Wasserman WW, et al. (2010). Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis. Nucleic Acids Res. 38, 5718–5734. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkq212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2943601/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20460467/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Global%20mapping%20of%20binding%20sites%20for%20Nrf2%20identifies%20novel%20targets%20in%20cell%20survival%20response%20through%20ChIP-Seq%20profiling%20and%20network%20analysis&amp;author=D%20Malhotra&amp;author=E%20Portales-Casamar&amp;author=A%20Singh&amp;author=S%20Srivastava&amp;author=D%20Arenillas&amp;volume=38&amp;publication_year=2010&amp;pages=5718-5734&amp;pmid=20460467&amp;doi=10.1093/nar/gkq212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R116\"> <cite> Mani DR, Maynard M, Kothadia R, Krug K, Christianson KE, Heiman D, Clauser KR, Birger C, Getz G, and Carr SA (2020). PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41592-021-01176-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8791030/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34040252/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PANOPLY:%20A%20cloud-based%20platform%20for%20automated%20and%20reproducible%20proteogenomic%20data%20analysis&amp;author=DR%20Mani&amp;author=M%20Maynard&amp;author=R%20Kothadia&amp;author=K%20Krug&amp;author=KE%20Christianson&amp;publication_year=2020&amp;pmid=34040252&amp;doi=10.1038/s41592-021-01176-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R117\"> <cite> McInnes L, Healy J, and Astels S (2017). hdbscan: Hierarchical density based clustering. Journal of Open Source Software 2, 205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Open%20Source%20Software&amp;title=hdbscan:%20Hierarchical%20density%20based%20clustering&amp;author=L%20McInnes&amp;author=J%20Healy&amp;author=S%20Astels&amp;volume=2&amp;publication_year=2017&amp;pages=205&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R118\"> <cite> McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, Flicek P, and Cunningham F (2016). The Ensembl Variant Effect Predictor. Genome Biol. 17, 122. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0974-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4893825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27268795/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=The%20Ensembl%20Variant%20Effect%20Predictor&amp;author=W%20McLaren&amp;author=L%20Gil&amp;author=SE%20Hunt&amp;author=HS%20Riat&amp;author=GRS%20Ritchie&amp;volume=17&amp;publication_year=2016&amp;pages=122&amp;pmid=27268795&amp;doi=10.1186/s13059-016-0974-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R119\"> <cite> Medvar B, Raghuram V, Pisitkun T, Sarkar A, and Knepper MA (2016). Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation. Physiol. Genomics 48, 502–512. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/physiolgenomics.00031.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4967219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27199454/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Physiol.%20Genomics&amp;title=Comprehensive%20database%20of%20human%20E3%20ubiquitin%20ligases:%20application%20to%20aquaporin-2%20regulation&amp;author=B%20Medvar&amp;author=V%20Raghuram&amp;author=T%20Pisitkun&amp;author=A%20Sarkar&amp;author=MA%20Knepper&amp;volume=48&amp;publication_year=2016&amp;pages=502-512&amp;pmid=27199454&amp;doi=10.1152/physiolgenomics.00031.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R120\"> <cite> Merico D, Isserlin R, Stueker O, Emili A, and Bader GD (2010). Enrichment map: a network-based method for gene-set enrichment visualization and interpretation. PLoS One 5, e13984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0013984\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2981572/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21085593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Enrichment%20map:%20a%20network-based%20method%20for%20gene-set%20enrichment%20visualization%20and%20interpretation&amp;author=D%20Merico&amp;author=R%20Isserlin&amp;author=O%20Stueker&amp;author=A%20Emili&amp;author=GD%20Bader&amp;volume=5&amp;publication_year=2010&amp;pages=e13984&amp;pmid=21085593&amp;doi=10.1371/journal.pone.0013984&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R121\"> <cite> Mermel CH, Schumacher SE, Hill B, Meyerson ML, Beroukhim R, and Getz G (2011). GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-4-r41\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3218867/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21527027/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers&amp;author=CH%20Mermel&amp;author=SE%20Schumacher&amp;author=B%20Hill&amp;author=ML%20Meyerson&amp;author=R%20Beroukhim&amp;volume=12&amp;publication_year=2011&amp;pages=R41&amp;pmid=21527027&amp;doi=10.1186/gb-2011-12-4-r41&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R122\"> <cite> Middleton G, Fletcher P, Popat S, Savage J, Summers Y, Greystoke A, Gilligan D, Cave J, O’Rourke N, Brewster A, et al. (2020). The National Lung Matrix Trial of personalized therapy in lung cancer. Nature 583, 807–812. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-2481-8\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7116732/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32669708/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20National%20Lung%20Matrix%20Trial%20of%20personalized%20therapy%20in%20lung%20cancer&amp;author=G%20Middleton&amp;author=P%20Fletcher&amp;author=S%20Popat&amp;author=J%20Savage&amp;author=Y%20Summers&amp;volume=583&amp;publication_year=2020&amp;pages=807-812&amp;pmid=32669708&amp;doi=10.1038/s41586-020-2481-8&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R123\"> <cite> Moissoglu K, McRoberts KS, Meier JA, Theodorescu D, and Schwartz MA (2009). Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases. Cancer Res. 69, 2838–2844. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1397\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2701105/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19276387/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Rho%20GDP%20dissociation%20inhibitor%202%20suppresses%20metastasis%20via%20unconventional%20regulation%20of%20RhoGTPases&amp;author=K%20Moissoglu&amp;author=KS%20McRoberts&amp;author=JA%20Meier&amp;author=D%20Theodorescu&amp;author=MA%20Schwartz&amp;volume=69&amp;publication_year=2009&amp;pages=2838-2844&amp;pmid=19276387&amp;doi=10.1158/0008-5472.CAN-08-1397&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R124\"> <cite> Morrison JA, Pike LA, Sams SB, Sharma V, Zhou Q, Severson JJ, Tan A-C, Wood WM, and Haugen BR (2014). Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer. Mol. Cancer 13, 62. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1476-4598-13-62\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3995095/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24645981/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Thioredoxin%20interacting%20protein%20(TXNIP)%20is%20a%20novel%20tumor%20suppressor%20in%20thyroid%20cancer&amp;author=JA%20Morrison&amp;author=LA%20Pike&amp;author=SB%20Sams&amp;author=V%20Sharma&amp;author=Q%20Zhou&amp;volume=13&amp;publication_year=2014&amp;pages=62&amp;pmid=24645981&amp;doi=10.1186/1476-4598-13-62&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R125\"> <cite> Mouchemore KA, Sampaio NG, Murrey MW, Stanley ER, Lannutti BJ, and Pixley FJ (2013). Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity. FEBS J. 280, 5228–5236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/febs.12316\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3773002/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23648053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Specific%20inhibition%20of%20PI3K%20p110%CE%B4%20inhibits%20CSF-1-induced%20macrophage%20spreading%20and%20invasive%20capacity&amp;author=KA%20Mouchemore&amp;author=NG%20Sampaio&amp;author=MW%20Murrey&amp;author=ER%20Stanley&amp;author=BJ%20Lannutti&amp;volume=280&amp;publication_year=2013&amp;pages=5228-5236&amp;pmid=23648053&amp;doi=10.1111/febs.12316&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R126\"> <cite> Natan E, and Joerger AC (2012). Structure and kinetic stability of the p63 tetramerization domain. J. Mol. Biol. 415, 503–513. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2011.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3277882/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22100306/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=Structure%20and%20kinetic%20stability%20of%20the%20p63%20tetramerization%20domain&amp;author=E%20Natan&amp;author=AC%20Joerger&amp;volume=415&amp;publication_year=2012&amp;pages=503-513&amp;pmid=22100306&amp;doi=10.1016/j.jmb.2011.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R127\"> <cite> Newman AM, Liu CL, Green MR, Gentles AJ, Feng W, Xu Y, Hoang CD, Diehn M, and Alizadeh AA (2015). Robust enumeration of cell subsets from tissue expression profiles. Nat. Methods 12, 453–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.3337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4739640/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25822800/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Robust%20enumeration%20of%20cell%20subsets%20from%20tissue%20expression%20profiles&amp;author=AM%20Newman&amp;author=CL%20Liu&amp;author=MR%20Green&amp;author=AJ%20Gentles&amp;author=W%20Feng&amp;volume=12&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;doi=10.1038/nmeth.3337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R128\"> <cite> Nicorici D, Satalan M, Edgren H, Kangaspeska S, Murumagi A, Kallioniemi O, Virtanen S, and Kilkku O FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FusionCatcher%20-%20a%20tool%20for%20finding%20somatic%20fusion%20genes%20in%20paired-end%20RNA-sequencing%20data&amp;author=D%20Nicorici&amp;author=M%20Satalan&amp;author=H%20Edgren&amp;author=S%20Kangaspeska&amp;author=A%20Murumagi&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R129\"> <cite> Nijman SMB, Luna-Vargas MPA, Velds A, Brummelkamp TR, Dirac AMG, Sixma TK, and Bernards R (2005). A genomic and functional inventory of deubiquitinating enzymes. Cell 123, 773–786. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2005.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16325574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20genomic%20and%20functional%20inventory%20of%20deubiquitinating%20enzymes&amp;author=SMB%20Nijman&amp;author=MPA%20Luna-Vargas&amp;author=A%20Velds&amp;author=TR%20Brummelkamp&amp;author=AMG%20Dirac&amp;volume=123&amp;publication_year=2005&amp;pages=773-786&amp;pmid=16325574&amp;doi=10.1016/j.cell.2005.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R130\"> <cite> Niu D-F, Kondo T, Nakazawa T, Oishi N, Kawasaki T, Mochizuki K, Yamane T, and Katoh R (2012). Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas. Lab. Invest. 92, 1181–1190. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/labinvest.2012.84\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22641097/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lab.%20Invest&amp;title=Transcription%20factor%20Runx2%20is%20a%20regulator%20of%20epithelial-mesenchymal%20transition%20and%20invasion%20in%20thyroid%20carcinomas&amp;author=D-F%20Niu&amp;author=T%20Kondo&amp;author=T%20Nakazawa&amp;author=N%20Oishi&amp;author=T%20Kawasaki&amp;volume=92&amp;publication_year=2012&amp;pages=1181-1190&amp;pmid=22641097&amp;doi=10.1038/labinvest.2012.84&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R131\"> <cite> Obenchain V, Lawrence M, Carey V, Gogarten S, Shannon P, and Morgan M (2014). VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants. Bioinformatics 30, 2076–2078. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu168\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080743/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24681907/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=VariantAnnotation:%20a%20Bioconductor%20package%20for%20exploration%20and%20annotation%20of%20genetic%20variants&amp;author=V%20Obenchain&amp;author=M%20Lawrence&amp;author=V%20Carey&amp;author=S%20Gogarten&amp;author=P%20Shannon&amp;volume=30&amp;publication_year=2014&amp;pages=2076-2078&amp;pmid=24681907&amp;doi=10.1093/bioinformatics/btu168&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R132\"> <cite> Paik PK, Pillai RN, Lathan CS, Velasco SA, and Papadimitrakopoulou V (2019). New Treatment Options in Advanced Squamous Cell Lung Cancer. Am Soc Clin Oncol Educ Book 39, e198–e206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/EDBK_237829\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31099625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am%20Soc%20Clin%20Oncol%20Educ%20Book&amp;title=New%20Treatment%20Options%20in%20Advanced%20Squamous%20Cell%20Lung%20Cancer&amp;author=PK%20Paik&amp;author=RN%20Pillai&amp;author=CS%20Lathan&amp;author=SA%20Velasco&amp;author=V%20Papadimitrakopoulou&amp;volume=39&amp;publication_year=2019&amp;pages=e198-e206&amp;pmid=31099625&amp;doi=10.1200/EDBK_237829&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R133\"> <cite> Parra ER, Jiang M, Solis L, Mino B, Laberiano C, Hernandez S, Gite S, Verma A, Tetzlaff M, Haymaker C, et al. (2020). Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies. Cancers 12. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers12020255\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7072187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31972974/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Procedural%20Requirements%20and%20Recommendations%20for%20Multiplex%20Immunofluorescence%20Tyramide%20Signal%20Amplification%20Assays%20to%20Support%20Translational%20Oncology%20Studies&amp;author=ER%20Parra&amp;author=M%20Jiang&amp;author=L%20Solis&amp;author=B%20Mino&amp;author=C%20Laberiano&amp;volume=12&amp;publication_year=2020&amp;pmid=31972974&amp;doi=10.3390/cancers12020255&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R134\"> <cite> Patel M, Lu L, Zander DS, Sreerama L, Coco D, and Moreb JS (2008). ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors. Lung Cancer 59, 340–349. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.lungcan.2007.08.033\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17920722/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=ALDH1A1%20and%20ALDH3A1%20expression%20in%20lung%20cancers:%20correlation%20with%20histologic%20type%20and%20potential%20precursors&amp;author=M%20Patel&amp;author=L%20Lu&amp;author=DS%20Zander&amp;author=L%20Sreerama&amp;author=D%20Coco&amp;volume=59&amp;publication_year=2008&amp;pages=340-349&amp;pmid=17920722&amp;doi=10.1016/j.lungcan.2007.08.033&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R135\"> <cite> Patnaik A, Rosen LS, Tolaney SM, Tolcher AW, Goldman JW, Gandhi L, Papadopoulos KP, Beeram M, Rasco DW, Hilton JF, et al. (2016). Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors. Cancer Discov. 6, 740–753. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/2159-8290.CD-16-0095\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27217383/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Efficacy%20and%20Safety%20of%20Abemaciclib,%20an%20Inhibitor%20of%20CDK4%20and%20CDK6,%20for%20Patients%20with%20Breast%20Cancer,%20Non-Small%20Cell%20Lung%20Cancer,%20and%20Other%20Solid%20Tumors&amp;author=A%20Patnaik&amp;author=LS%20Rosen&amp;author=SM%20Tolaney&amp;author=AW%20Tolcher&amp;author=JW%20Goldman&amp;volume=6&amp;publication_year=2016&amp;pages=740-753&amp;pmid=27217383&amp;doi=10.1158/2159-8290.CD-16-0095&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R136\"> <cite> Payen VL, Mina E, Van Hée VF, Porporato PE, and Sonveaux P (2020). Monocarboxylate transporters in cancer. Mol Metab 33, 48–66. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molmet.2019.07.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7056923/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395464/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Metab&amp;title=Monocarboxylate%20transporters%20in%20cancer&amp;author=VL%20Payen&amp;author=E%20Mina&amp;author=VF%20Van%20H%C3%A9e&amp;author=PE%20Porporato&amp;author=P%20Sonveaux&amp;volume=33&amp;publication_year=2020&amp;pages=48-66&amp;pmid=31395464&amp;doi=10.1016/j.molmet.2019.07.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R137\"> <cite> Paz-Ares L, Luft A, Vicente D, Tafreshi A, Gümüş M, Mazières J, Hermes B, Çay Şenler F, Csőszi T, Fülöp A, et al. (2018). Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer. N. Engl. J. Med. 379, 2040–2051. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1056/NEJMoa1810865\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30280635/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&amp;title=Pembrolizumab%20plus%20Chemotherapy%20for%20Squamous%20Non-Small-Cell%20Lung%20Cancer&amp;author=L%20Paz-Ares&amp;author=A%20Luft&amp;author=D%20Vicente&amp;author=A%20Tafreshi&amp;author=M%20G%C3%BCm%C3%BC%C5%9F&amp;volume=379&amp;publication_year=2018&amp;pages=2040-2051&amp;pmid=30280635&amp;doi=10.1056/NEJMoa1810865&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R138\"> <cite> Pedersen BS, Layer RM, and Quinlan AR (2016). Vcfanno: fast, flexible annotation of genetic variants. Genome Biol. 17, 118. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0973-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4888505/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27250555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Vcfanno:%20fast,%20flexible%20annotation%20of%20genetic%20variants&amp;author=BS%20Pedersen&amp;author=RM%20Layer&amp;author=AR%20Quinlan&amp;volume=17&amp;publication_year=2016&amp;pages=118&amp;pmid=27250555&amp;doi=10.1186/s13059-016-0973-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R139\"> <cite> Peng A, Mao X, Zhong J, Fan S, and Hu Y (2020). Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology. Proteomics 20, e1900271. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/pmic.201900271\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32223079/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Single-Cell%20Multi-Omics%20and%20Its%20Prospective%20Application%20in%20Cancer%20Biology&amp;author=A%20Peng&amp;author=X%20Mao&amp;author=J%20Zhong&amp;author=S%20Fan&amp;author=Y%20Hu&amp;volume=20&amp;publication_year=2020&amp;pages=e1900271&amp;pmid=32223079&amp;doi=10.1002/pmic.201900271&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R140\"> <cite> Peters LA, Perrigoue J, Mortha A, Iuga A, Song W-M, Neiman EM, Llewellyn SR, Di Narzo A, Kidd BA, Telesco SE, et al. (2017). A functional genomics predictive network model identifies regulators of inflammatory bowel disease. Nat. Genet. 49, 1437–1449. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3947\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5660607/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28892060/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=A%20functional%20genomics%20predictive%20network%20model%20identifies%20regulators%20of%20inflammatory%20bowel%20disease&amp;author=LA%20Peters&amp;author=J%20Perrigoue&amp;author=A%20Mortha&amp;author=A%20Iuga&amp;author=W-M%20Song&amp;volume=49&amp;publication_year=2017&amp;pages=1437-1449&amp;pmid=28892060&amp;doi=10.1038/ng.3947&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R141\"> <cite> Petralia F, Wang L, Peng J, Yan A, Zhu J, and Wang P (2018). A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity. Bioinformatics 34, i528–i536. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty280\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6022554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29949994/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20method%20for%20constructing%20tumor%20specific%20gene%20co-expression%20networks%20based%20on%20samples%20with%20tumor%20purity%20heterogeneity&amp;author=F%20Petralia&amp;author=L%20Wang&amp;author=J%20Peng&amp;author=A%20Yan&amp;author=J%20Zhu&amp;volume=34&amp;publication_year=2018&amp;pages=i528-i536&amp;pmid=29949994&amp;doi=10.1093/bioinformatics/bty280&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R142\"> <cite> Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, de Bakker PIW, Daly MJ, et al. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1086/519795\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1950838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17701901/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&amp;title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S%20Purcell&amp;author=B%20Neale&amp;author=K%20Todd-Brown&amp;author=L%20Thomas&amp;author=MAR%20Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R143\"> <cite> Qian J, and Massion PP (2008). Role of chromosome 3q amplification in lung cancer. J. Thorac. Oncol. 3, 212–215. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e3181663544\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18317062/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Role%20of%20chromosome%203q%20amplification%20in%20lung%20cancer&amp;author=J%20Qian&amp;author=PP%20Massion&amp;volume=3&amp;publication_year=2008&amp;pages=212-215&amp;pmid=18317062&amp;doi=10.1097/JTO.0b013e3181663544&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R144\"> <cite> Ramadoss S, Sen S, Ramachandran I, Roy S, Chaudhuri G, and Farias-Eisner R (2017). Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance. Oncogene 36, 6508. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2017.331\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5701089/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28925393/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Lysine-specific%20demethylase%20KDM3A%20regulates%20ovarian%20cancer%20stemness%20and%20chemoresistance&amp;author=S%20Ramadoss&amp;author=S%20Sen&amp;author=I%20Ramachandran&amp;author=S%20Roy&amp;author=G%20Chaudhuri&amp;volume=36&amp;publication_year=2017&amp;pages=6508&amp;pmid=28925393&amp;doi=10.1038/onc.2017.331&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R145\"> <cite> Raudvere U, Kolberg L, Kuzmin I, Arak T, Adler P, Peterson H, and Vilo J (2019). g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). Nucleic Acids Res. 47, W191–W198. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz369\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31066453/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=g:Profiler:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20(2019%20update)&amp;author=U%20Raudvere&amp;author=L%20Kolberg&amp;author=I%20Kuzmin&amp;author=T%20Arak&amp;author=P%20Adler&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;doi=10.1093/nar/gkz369&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R146\"> <cite> Reese SE, Archer KJ, Therneau TM, Atkinson EJ, Vachon CM, de Andrade M, Kocher J-PA, and Eckel-Passow JE (2013). A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis. Bioinformatics 29, 2877–2883. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt480\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810845/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23958724/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20statistic%20for%20identifying%20batch%20effects%20in%20high-throughput%20genomic%20data%20that%20uses%20guided%20principal%20component%20analysis&amp;author=SE%20Reese&amp;author=KJ%20Archer&amp;author=TM%20Therneau&amp;author=EJ%20Atkinson&amp;author=CM%20Vachon&amp;volume=29&amp;publication_year=2013&amp;pages=2877-2883&amp;pmid=23958724&amp;doi=10.1093/bioinformatics/btt480&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R147\"> <cite> Reva B, Omelchenko T, Calinawan A, Nair S, Schadt E, and Tewari A (2020). Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-γ axis and identification of immune resistance mechanisms. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Prioritization%20prostate%20cancer%20to%20immune%20checkpoint%20therapy%20by%20ranking%20tumors%20along%20IFN-%CE%B3%20axis%20and%20identification%20of%20immune%20resistance%20mechanisms&amp;author=B%20Reva&amp;author=T%20Omelchenko&amp;author=A%20Calinawan&amp;author=S%20Nair&amp;author=E%20Schadt&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R148\"> <cite> Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, and Smyth GK (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkv007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4402510/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25605792/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;author=ME%20Ritchie&amp;author=B%20Phipson&amp;author=D%20Wu&amp;author=Y%20Hu&amp;author=CW%20Law&amp;volume=43&amp;publication_year=2015&amp;pages=e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R149\"> <cite> Robinson DR, Wu Y-M, Lonigro RJ, Vats P, Cobain E, Everett J, Cao X, Rabban E, Kumar-Sinha C, Raymond V, et al. (2017). Integrative clinical genomics of metastatic cancer. Nature 548, 297–303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature23306\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5995337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28783718/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative%20clinical%20genomics%20of%20metastatic%20cancer&amp;author=DR%20Robinson&amp;author=Y-M%20Wu&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=E%20Cobain&amp;volume=548&amp;publication_year=2017&amp;pages=297-303&amp;pmid=28783718&amp;doi=10.1038/nature23306&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R150\"> <cite> Roskoski R Jr (2018). The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders. Pharmacol. Res. 129, 65–83. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.phrs.2018.01.021\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29408302/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res&amp;title=The%20role%20of%20small%20molecule%20platelet-derived%20growth%20factor%20receptor%20(PDGFR)%20inhibitors%20in%20the%20treatment%20of%20neoplastic%20disorders&amp;author=R%20Roskoski&amp;volume=129&amp;publication_year=2018&amp;pages=65-83&amp;pmid=29408302&amp;doi=10.1016/j.phrs.2018.01.021&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R151\"> <cite> Satpathy S, Jaehnig EJ, Krug K, Kim B-J, Saltzman AB, Chan DW, Holloway KR, Anurag M, Huang C, Singh P, et al. (2020). Microscaled proteogenomic methods for precision oncology. Nat. Commun. 11, 532. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-14381-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6985126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31988290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Microscaled%20proteogenomic%20methods%20for%20precision%20oncology&amp;author=S%20Satpathy&amp;author=EJ%20Jaehnig&amp;author=K%20Krug&amp;author=B-J%20Kim&amp;author=AB%20Saltzman&amp;volume=11&amp;publication_year=2020&amp;pages=532&amp;pmid=31988290&amp;doi=10.1038/s41467-020-14381-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R152\"> <cite> Schubert M, Klinger B, Klünemann M, Sieber A, Uhlitz F, Sauer S, Garnett MJ, Blüthgen N, and Saez-Rodriguez J (2018). Perturbation-response genes reveal signaling footprints in cancer gene expression. Nat. Commun. 9, 20. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-017-02391-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5750219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29295995/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Perturbation-response%20genes%20reveal%20signaling%20footprints%20in%20cancer%20gene%20expression&amp;author=M%20Schubert&amp;author=B%20Klinger&amp;author=M%20Kl%C3%BCnemann&amp;author=A%20Sieber&amp;author=F%20Uhlitz&amp;volume=9&amp;publication_year=2018&amp;pages=20&amp;pmid=29295995&amp;doi=10.1038/s41467-017-02391-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R153\"> <cite> Shadforth IP, Dunkley TPJ, Lilley KS, and Bessant C (2005). i-Tracker: for quantitative proteomics using iTRAQ. BMC Genomics 6, 145. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2164-6-145\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1276793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16242023/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=i-Tracker:%20for%20quantitative%20proteomics%20using%20iTRAQ&amp;author=IP%20Shadforth&amp;author=TPJ%20Dunkley&amp;author=KS%20Lilley&amp;author=C%20Bessant&amp;volume=6&amp;publication_year=2005&amp;pages=145&amp;pmid=16242023&amp;doi=10.1186/1471-2164-6-145&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R154\"> <cite> Shahamatdar S, He MX, Reyna MA, Gusev A, AlDubayan SH, Van Allen EM, and Ramachandran S (2020). Germline Features Associated with Immune Infiltration in Solid Tumors. Cell Rep. 30, 2900–2908.e4. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2020.02.039\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7082123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32130895/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Germline%20Features%20Associated%20with%20Immune%20Infiltration%20in%20Solid%20Tumors&amp;author=S%20Shahamatdar&amp;author=MX%20He&amp;author=MA%20Reyna&amp;author=A%20Gusev&amp;author=SH%20AlDubayan&amp;volume=30&amp;publication_year=2020&amp;pages=2900-2908.e4&amp;pmid=32130895&amp;doi=10.1016/j.celrep.2020.02.039&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R155\"> <cite> Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, and Ideker T (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–2504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.1239303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC403769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14597658/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R156\"> <cite> Shen J, Ju Z, Zhao W, Wang L, Peng Y, Ge Z, Nagel ZD, Zou J, Wang C, Kapoor P, et al. (2018). ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade. Nat. Med. 24, 556–562. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41591-018-0012-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6076433/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29736026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=ARID1A%20deficiency%20promotes%20mutability%20and%20potentiates%20therapeutic%20antitumor%20immunity%20unleashed%20by%20immune%20checkpoint%20blockade&amp;author=J%20Shen&amp;author=Z%20Ju&amp;author=W%20Zhao&amp;author=L%20Wang&amp;author=Y%20Peng&amp;volume=24&amp;publication_year=2018&amp;pages=556-562&amp;pmid=29736026&amp;doi=10.1038/s41591-018-0012-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R157\"> <cite> Showe MK, Vachani A, Kossenkov AV, Yousef M, Nichols C, Nikonova EV, Chang C, Kucharczuk J, Tran B, Wakeam E, et al. (2009). Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease. Cancer Res. 69, 9202–9210. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-09-1378\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2798582/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19951989/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Gene%20expression%20profiles%20in%20peripheral%20blood%20mononuclear%20cells%20can%20distinguish%20patients%20with%20non-small%20cell%20lung%20cancer%20from%20patients%20with%20nonmalignant%20lung%20disease&amp;author=MK%20Showe&amp;author=A%20Vachani&amp;author=AV%20Kossenkov&amp;author=M%20Yousef&amp;author=C%20Nichols&amp;volume=69&amp;publication_year=2009&amp;pages=9202-9210&amp;pmid=19951989&amp;doi=10.1158/0008-5472.CAN-09-1378&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R158\"> <cite> Siegel RL, Miller KD, and Jemal A (2020). Cancer statistics, 2020. CA Cancer J. Clin. 70, 7–30. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21590\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31912902/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Cancer%20statistics,%202020&amp;author=RL%20Siegel&amp;author=KD%20Miller&amp;author=A%20Jemal&amp;volume=70&amp;publication_year=2020&amp;pages=7-30&amp;pmid=31912902&amp;doi=10.3322/caac.21590&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R159\"> <cite> Singh A, Boldin-Adamsky S, Thimmulappa RK, Rath SK, Ashush H, Coulter J, Blackford A, Goodman SN, Bunz F, Watson WH, et al. (2008). RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy. Cancer Res. 68, 7975–7984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3070411/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18829555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=RNAi-mediated%20silencing%20of%20nuclear%20factor%20erythroid-2-related%20factor%202%20gene%20expression%20in%20non-small%20cell%20lung%20cancer%20inhibits%20tumor%20growth%20and%20increases%20efficacy%20of%20chemotherapy&amp;author=A%20Singh&amp;author=S%20Boldin-Adamsky&amp;author=RK%20Thimmulappa&amp;author=SK%20Rath&amp;author=H%20Ashush&amp;volume=68&amp;publication_year=2008&amp;pages=7975-7984&amp;pmid=18829555&amp;doi=10.1158/0008-5472.CAN-08-1401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R160\"> <cite> Sinicropi-Yao SL, Amann JM, Lopez DLY, Cerciello F, Coombes KR, and Carbone DP (2019). Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC. J. Thorac. Oncol. 14, 223–236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2018.10.162\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6348021/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30408569/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Co-Expression%20Analysis%20Reveals%20Mechanisms%20Underlying%20the%20Varied%20Roles%20of%20NOTCH1%20in%20NSCLC&amp;author=SL%20Sinicropi-Yao&amp;author=JM%20Amann&amp;author=DLY%20Lopez&amp;author=F%20Cerciello&amp;author=KR%20Coombes&amp;volume=14&amp;publication_year=2019&amp;pages=223-236&amp;pmid=30408569&amp;doi=10.1016/j.jtho.2018.10.162&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R161\"> <cite> Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, and Forbes SA (2018). The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat. Rev. Cancer 18, 696–705. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41568-018-0060-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6450507/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30293088/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=The%20COSMIC%20Cancer%20Gene%20Census:%20describing%20genetic%20dysfunction%20across%20all%20human%20cancers&amp;author=Z%20Sondka&amp;author=S%20Bamford&amp;author=CG%20Cole&amp;author=SA%20Ward&amp;author=I%20Dunham&amp;volume=18&amp;publication_year=2018&amp;pages=696-705&amp;pmid=30293088&amp;doi=10.1038/s41568-018-0060-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R162\"> <cite> Song X, Ji J, Gleason KJ, Yang F, Martignetti JA, Chen LS, and Wang P (2019). Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis. Mol. Cell. Proteomics 18, S52–S65. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.RA118.001220\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692782/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31227599/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Insights%20into%20Impact%20of%20DNA%20Copy%20Number%20Alteration%20and%20Methylation%20on%20the%20Proteogenomic%20Landscape%20of%20Human%20Ovarian%20Cancer%20via%20a%20Multi-omics%20Integrative%20Analysis&amp;author=X%20Song&amp;author=J%20Ji&amp;author=KJ%20Gleason&amp;author=F%20Yang&amp;author=JA%20Martignetti&amp;volume=18&amp;publication_year=2019&amp;pages=S52-S65&amp;pmid=31227599&amp;doi=10.1074/mcp.RA118.001220&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R163\"> <cite> Spitzer MH, and Nolan GP (2016). Mass Cytometry: Single Cells, Many Features. Cell 165, 780–791. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2016.04.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4860251/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27153492/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Many%20Features.%20Cell&amp;title=Mass%20Cytometry:%20Single%20Cells&amp;author=MH%20Spitzer&amp;author=GP%20Nolan&amp;volume=165&amp;publication_year=2016&amp;pages=780-791&amp;pmid=27153492&amp;doi=10.1016/j.cell.2016.04.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R164\"> <cite> Srivastava K, Pickard A, Craig SG, Quinn GP, Lambe SM, James JA, McDade SS, and McCance DJ (2018). ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers. Clin. Cancer Res. 24, 3917–3927. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-3775\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6098695/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29739791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=%CE%94Np63%CE%B3/SRC/Slug%20Signaling%20Axis%20Promotes%20Epithelial-to-Mesenchymal%20Transition%20in%20Squamous%20Cancers&amp;author=K%20Srivastava&amp;author=A%20Pickard&amp;author=SG%20Craig&amp;author=GP%20Quinn&amp;author=SM%20Lambe&amp;volume=24&amp;publication_year=2018&amp;pages=3917-3927&amp;pmid=29739791&amp;doi=10.1158/1078-0432.CCR-17-3775&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R165\"> <cite> Stewart PA, Welsh EA, Slebos RJC, Fang B, Izumi V, Chambers M, Zhang G, Cen L, Pettersson F, Zhang Y, et al. (2019). Proteogenomic landscape of squamous cell lung cancer. Nat. Commun. 10, 3578. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-11452-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6687710/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395880/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Proteogenomic%20landscape%20of%20squamous%20cell%20lung%20cancer&amp;author=PA%20Stewart&amp;author=EA%20Welsh&amp;author=RJC%20Slebos&amp;author=B%20Fang&amp;author=V%20Izumi&amp;volume=10&amp;publication_year=2019&amp;pages=3578&amp;pmid=31395880&amp;doi=10.1038/s41467-019-11452-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R166\"> <cite> Su J, Morgani SM, David CJ, Wang Q, Er EE, Huang Y-H, Basnet H, Zou Y, Shu W, Soni RK, et al. (2020). TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1. Nature 577, 566–571. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-019-1897-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7450666/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31915377/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=TGF-%CE%B2%20orchestrates%20fibrogenic%20and%20developmental%20EMTs%20via%20the%20RAS%20effector%20RREB1&amp;author=J%20Su&amp;author=SM%20Morgani&amp;author=CJ%20David&amp;author=Q%20Wang&amp;author=EE%20Er&amp;volume=577&amp;publication_year=2020&amp;pages=566-571&amp;pmid=31915377&amp;doi=10.1038/s41586-019-1897-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R167\"> <cite> Subramanian A, Narayan R, Corsello SM, Peck DD, Natoli TE, Lu X, Gould J, Davis JF, Tubelli AA, Asiedu JK, et al. (2017). A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles. Cell 171, 1437–1452.e17. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.10.049\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5990023/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29195078/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Next%20Generation%20Connectivity%20Map:%20L1000%20Platform%20and%20the%20First%201,000,000%20Profiles&amp;author=A%20Subramanian&amp;author=R%20Narayan&amp;author=SM%20Corsello&amp;author=DD%20Peck&amp;author=TE%20Natoli&amp;volume=171&amp;publication_year=2017&amp;pages=1437-1452.e17&amp;pmid=29195078&amp;doi=10.1016/j.cell.2017.10.049&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R168\"> <cite> Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, and Kohlbacher O (2014). OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics 30, 3310–3316. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu548\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4441069/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25143287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=OptiType:%20precision%20HLA%20typing%20from%20next-generation%20sequencing%20data&amp;author=A%20Szolek&amp;author=B%20Schubert&amp;author=C%20Mohr&amp;author=M%20Sturm&amp;author=M%20Feldhahn&amp;volume=30&amp;publication_year=2014&amp;pages=3310-3316&amp;pmid=25143287&amp;doi=10.1093/bioinformatics/btu548&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R169\"> <cite> Tam WL, Lu H, Buikhuisen J, Soh BS, Lim E, Reinhardt F, Wu ZJ, Krall JA, Bierie B, Guo W, et al. (2013). Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells. Cancer Cell 24, 347–364. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccr.2013.08.005\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4001722/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24029232/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Protein%20kinase%20C%20%CE%B1%20is%20a%20central%20signaling%20node%20and%20therapeutic%20target%20for%20breast%20cancer%20stem%20cells&amp;author=WL%20Tam&amp;author=H%20Lu&amp;author=J%20Buikhuisen&amp;author=BS%20Soh&amp;author=E%20Lim&amp;volume=24&amp;publication_year=2013&amp;pages=347-364&amp;pmid=24029232&amp;doi=10.1016/j.ccr.2013.08.005&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R170\"> <cite> Tate JG, Bamford S, Jubb HC, Sondka Z, Beare DM, Bindal N, Boutselakis H, Cole CG, Creatore C, Dawson E, et al. (2019). COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gky1015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6323903/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30371878/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=COSMIC:%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;author=JG%20Tate&amp;author=S%20Bamford&amp;author=HC%20Jubb&amp;author=Z%20Sondka&amp;author=DM%20Beare&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;pmid=30371878&amp;doi=10.1093/nar/gky1015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R171\"> <cite> Tecalco-Cruz AC, and Cruz-Ramos E (2018). Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells. Biochem. Biophys. Res. Commun. 499, 973–978. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2018.04.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29626479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Protein%20ISGylation%20and%20free%20ISG15%20levels%20are%20increased%20by%20interferon%20gamma%20in%20breast%20cancer%20cells&amp;author=AC%20Tecalco-Cruz&amp;author=E%20Cruz-Ramos&amp;volume=499&amp;publication_year=2018&amp;pages=973-978&amp;pmid=29626479&amp;doi=10.1016/j.bbrc.2018.04.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R172\"> <cite> Tellez CS, Juri DE, Do K, Bernauer AM, Thomas CL, Damiani LA, Tessema M, Leng S, and Belinsky SA (2011). EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells. Cancer Res. 71, 3087–3097. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-10-3035\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3078195/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21363915/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=EMT%20and%20stem%20cell-like%20properties%20associated%20with%20miR-205%20and%20miR-200%20epigenetic%20silencing%20are%20early%20manifestations%20during%20carcinogen-induced%20transformation%20of%20human%20lung%20epithelial%20cells&amp;author=CS%20Tellez&amp;author=DE%20Juri&amp;author=K%20Do&amp;author=AM%20Bernauer&amp;author=CL%20Thomas&amp;volume=71&amp;publication_year=2011&amp;pages=3087-3097&amp;pmid=21363915&amp;doi=10.1158/0008-5472.CAN-10-3035&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R173\"> <cite> Thorsson V, Gibbs DL, Brown SD, Wolf D, Bortone DS, Ou Yang T-H, Porta-Pardo E, Gao GF, Plaisier CL, Eddy JA, et al. (2018). The Immune Landscape of Cancer. Immunity 48, 812–830.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.immuni.2018.03.023\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5982584/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29628290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20Immune%20Landscape%20of%20Cancer&amp;author=V%20Thorsson&amp;author=DL%20Gibbs&amp;author=SD%20Brown&amp;author=D%20Wolf&amp;author=DS%20Bortone&amp;volume=48&amp;publication_year=2018&amp;pages=812-830.e14&amp;pmid=29628290&amp;doi=10.1016/j.immuni.2018.03.023&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R174\"> <cite> Tian J, Cai Y, Li Y, Lu Z, Huang J, Deng Y, Yang N, Wang X, Ying P, Zhang S, et al. (2021). CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer. Nucleic Acids Res. 49, D1065–D1073. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkaa805\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7778991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33010176/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CancerImmunityQTL:%20a%20database%20to%20systematically%20evaluate%20the%20impact%20of%20genetic%20variants%20on%20immune%20infiltration%20in%20human%20cancer&amp;author=J%20Tian&amp;author=Y%20Cai&amp;author=Y%20Li&amp;author=Z%20Lu&amp;author=J%20Huang&amp;volume=49&amp;publication_year=2021&amp;pages=D1065-D1073&amp;pmid=33010176&amp;doi=10.1093/nar/gkaa805&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R175\"> <cite> Timperi E, Focaccetti C, Gallerano D, Panetta M, Spada S, Gallo E, Visca P, Venuta F, Diso D, Prelaj A, et al. (2017). IL-18 receptor marks functional CD8+ T cells in non-small cell lung cancer. Oncoimmunology 6, e1328337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1080/2162402X.2017.1328337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5543928/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28811967/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncoimmunology&amp;title=IL-18%20receptor%20marks%20functional%20CD8+%20T%20cells%20in%20non-small%20cell%20lung%20cancer&amp;author=E%20Timperi&amp;author=C%20Focaccetti&amp;author=D%20Gallerano&amp;author=M%20Panetta&amp;author=S%20Spada&amp;volume=6&amp;publication_year=2017&amp;pages=e1328337&amp;pmid=28811967&amp;doi=10.1080/2162402X.2017.1328337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R176\"> <cite> Tran MN, Choi W, Wszolek MF, Navai N, Lee I-LC, Nitti G, Wen S, Flores ER, Siefker-Radtke A, Czerniak B, et al. (2013). The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205. J. Biol. Chem. 288, 3275–3288. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M112.408104\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3561548/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23239884/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20p63%20protein%20isoform%20%CE%94Np63%CE%B1%20inhibits%20epithelial-mesenchymal%20transition%20in%20human%20bladder%20cancer%20cells:%20role%20of%20MIR-205&amp;author=MN%20Tran&amp;author=W%20Choi&amp;author=MF%20Wszolek&amp;author=N%20Navai&amp;author=I-LC%20Lee&amp;volume=288&amp;publication_year=2013&amp;pages=3275-3288&amp;pmid=23239884&amp;doi=10.1074/jbc.M112.408104&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R177\"> <cite> Tsherniak A, Vazquez F, Montgomery PG, Weir BA, Kryukov G, Cowley GS, Gill S, Harrington WF, Pantel S, Krill-Burger JM, et al. (2017). Defining a Cancer Dependency Map. Cell 170, 564–576.e16. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.06.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5667678/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28753430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Defining%20a%20Cancer%20Dependency%20Map&amp;author=A%20Tsherniak&amp;author=F%20Vazquez&amp;author=PG%20Montgomery&amp;author=BA%20Weir&amp;author=G%20Kryukov&amp;volume=170&amp;publication_year=2017&amp;pages=564-576.e16&amp;pmid=28753430&amp;doi=10.1016/j.cell.2017.06.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R178\"> <cite> Udeshi ND, Mani DC, Satpathy S, Fereshetian S, Gasser JA, Svinkina T, Olive ME, Ebert BL, Mertins P, and Carr SA (2020). Rapid and deep-scale ubiquitylation profiling for biology and translational research. Nat. Commun. 11, 359. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-14175-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6969155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31953384/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Rapid%20and%20deep-scale%20ubiquitylation%20profiling%20for%20biology%20and%20translational%20research&amp;author=ND%20Udeshi&amp;author=DC%20Mani&amp;author=S%20Satpathy&amp;author=S%20Fereshetian&amp;author=JA%20Gasser&amp;volume=11&amp;publication_year=2020&amp;pages=359&amp;pmid=31953384&amp;doi=10.1038/s41467-019-14175-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R179\"> <cite> Uhlén M, Björling E, Agaton C, Szigyarto CA-K, Amini B, Andersen E, Andersson A-C, Angelidou P, Asplund A, Asplund C, et al. (2005). A human protein atlas for normal and cancer tissues based on antibody proteomics. Mol. Cell. Proteomics 4, 1920–1932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M500279-MCP200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16127175/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20human%20protein%20atlas%20for%20normal%20and%20cancer%20tissues%20based%20on%20antibody%20proteomics&amp;author=M%20Uhl%C3%A9n&amp;author=E%20Bj%C3%B6rling&amp;author=C%20Agaton&amp;author=CA-K%20Szigyarto&amp;author=B%20Amini&amp;volume=4&amp;publication_year=2005&amp;pages=1920-1932&amp;pmid=16127175&amp;doi=10.1074/mcp.M500279-MCP200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R180\"> <cite> Vasaikar S, Huang C, Wang X, Petyuk VA, Savage SR, Wen B, Dou Y, Zhang Y, Shi Z, Arshad OA, et al. (2019). Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities. Cell 177, 1035–1049.e19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2019.03.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6768830/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31031003/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Analysis%20of%20Human%20Colon%20Cancer%20Reveals%20New%20Therapeutic%20Opportunities&amp;author=S%20Vasaikar&amp;author=C%20Huang&amp;author=X%20Wang&amp;author=VA%20Petyuk&amp;author=SR%20Savage&amp;volume=177&amp;publication_year=2019&amp;pages=1035-1049.e19&amp;pmid=31031003&amp;doi=10.1016/j.cell.2019.03.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R181\"> <cite> Vasaikar SV, Straub P, Wang J, and Zhang B (2018). LinkedOmics: analyzing multi-omics data within and across 32 cancer types. Nucleic Acids Res. 46, D956–D963. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1090\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753188/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29136207/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LinkedOmics:%20analyzing%20multi-omics%20data%20within%20and%20across%2032%20cancer%20types&amp;author=SV%20Vasaikar&amp;author=P%20Straub&amp;author=J%20Wang&amp;author=B%20Zhang&amp;volume=46&amp;publication_year=2018&amp;pages=D956-D963&amp;pmid=29136207&amp;doi=10.1093/nar/gkx1090&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R182\"> <cite> Vats P, Chinnaiyan AM, and Kumar-Sinha C (2020). Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit. Methods Mol. Biol. 2079, 69–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/978-1-4939-9904-0_5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31728962/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&amp;title=Case%20Study:%20Systematic%20Detection%20and%20Prioritization%20of%20Gene%20Fusions%20in%20Cancer%20by%20RNA-Seq:%20A%20DIY%20Toolkit&amp;author=P%20Vats&amp;author=AM%20Chinnaiyan&amp;author=C%20Kumar-Sinha&amp;volume=2079&amp;publication_year=2020&amp;pages=69-79&amp;pmid=31728962&amp;doi=10.1007/978-1-4939-9904-0_5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R183\"> <cite> Vosgha H, Ariana A, Smith RA, and Lam AK-Y (2018). miR-205 targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma. Endocr. Relat. Cancer 25, 323–337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1530/ERC-17-0497\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29317480/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Endocr.%20Relat.%20Cancer&amp;title=miR-205%20targets%20angiogenesis%20and%20EMT%20concurrently%20in%20anaplastic%20thyroid%20carcinoma&amp;author=H%20Vosgha&amp;author=A%20Ariana&amp;author=RA%20Smith&amp;author=AK-Y%20Lam&amp;volume=25&amp;publication_year=2018&amp;pages=323-337&amp;pmid=29317480&amp;doi=10.1530/ERC-17-0497&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R184\"> <cite> Walser T, Cui X, Yanagawa J, Lee JM, Heinrich E, Lee G, Sharma S, and Dubinett SM (2008). Smoking and lung cancer: the role of inflammation. Proc. Am. Thorac. Soc. 5, 811–815. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1513/pats.200809-100TH\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080902/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19017734/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Am.%20Thorac.%20Soc&amp;title=Smoking%20and%20lung%20cancer:%20the%20role%20of%20inflammation&amp;author=T%20Walser&amp;author=X%20Cui&amp;author=J%20Yanagawa&amp;author=JM%20Lee&amp;author=E%20Heinrich&amp;volume=5&amp;publication_year=2008&amp;pages=811-815&amp;pmid=19017734&amp;doi=10.1513/pats.200809-100TH&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R185\"> <cite> Wang I-M, Zhang B, Yang X, Zhu J, Stepaniants S, Zhang C, Meng Q, Peters M, He Y, Ni C, et al. (2012). Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers. Mol. Syst. Biol. 8, 594. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2012.24\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3421440/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22806142/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol&amp;title=Systems%20analysis%20of%20eleven%20rodent%20disease%20models%20reveals%20an%20inflammatome%20signature%20and%20key%20drivers&amp;author=I-M%20Wang&amp;author=B%20Zhang&amp;author=X%20Yang&amp;author=J%20Zhu&amp;author=S%20Stepaniants&amp;volume=8&amp;publication_year=2012&amp;pages=594&amp;pmid=22806142&amp;doi=10.1038/msb.2012.24&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R186\"> <cite> Wang J, Vasaikar S, Shi Z, Greer M, and Zhang B (2017). WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. Nucleic Acids Res. 45, W130–W137. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx356\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5570149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28472511/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202017:%20a%20more%20comprehensive,%20powerful,%20flexible%20and%20interactive%20gene%20set%20enrichment%20analysis%20toolkit&amp;author=J%20Wang&amp;author=S%20Vasaikar&amp;author=Z%20Shi&amp;author=M%20Greer&amp;author=B%20Zhang&amp;volume=45&amp;publication_year=2017&amp;pages=W130-W137&amp;pmid=28472511&amp;doi=10.1093/nar/gkx356&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R187\"> <cite> Wang L-B, Karpova A, Gritsenko MA, Kyle JE, Cao S, Li Y, Rykunov D, Colaprico A, Rothstein JH, Hong R, et al. (2021). Proteogenomic and metabolomic characterization of human glioblastoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2021.01.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8044053/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33577785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20and%20metabolomic%20characterization%20of%20human%20glioblastoma&amp;author=L-B%20Wang&amp;author=A%20Karpova&amp;author=MA%20Gritsenko&amp;author=JE%20Kyle&amp;author=S%20Cao&amp;publication_year=2021&amp;pmid=33577785&amp;doi=10.1016/j.ccell.2021.01.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R188\"> <cite> Wang S, Jiang B, Zhang T, Liu L, Wang Y, Wang Y, Chen X, Lin H, Zhou L, Xia Y, et al. (2015). Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1. PLoS Biol. 13, e1002243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pbio.1002243\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4565669/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26356530/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=Insulin%20and%20mTOR%20Pathway%20Regulate%20HDAC3-Mediated%20Deacetylation%20and%20Activation%20of%20PGK1&amp;author=S%20Wang&amp;author=B%20Jiang&amp;author=T%20Zhang&amp;author=L%20Liu&amp;author=Y%20Wang&amp;volume=13&amp;publication_year=2015&amp;pages=e1002243&amp;pmid=26356530&amp;doi=10.1371/journal.pbio.1002243&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R189\"> <cite> Weeden CE, Solomon B, and Asselin-Labat M-L (2015). FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies. Cell Death Discov 1, 15049. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cddiscovery.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4979434/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27551478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discov&amp;title=FGFR1%20inhibition%20in%20lung%20squamous%20cell%20carcinoma:%20questions%20and%20controversies&amp;author=CE%20Weeden&amp;author=B%20Solomon&amp;author=M-L%20Asselin-Labat&amp;volume=1&amp;publication_year=2015&amp;pages=15049&amp;pmid=27551478&amp;doi=10.1038/cddiscovery.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R190\"> <cite> Wen B, Wang X, and Zhang B (2019). PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations. Genome Res. 29, 485–493. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.235028.118\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6396417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30610011/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=PepQuery%20enables%20fast,%20accurate,%20and%20convenient%20proteomic%20validation%20of%20novel%20genomic%20alterations&amp;author=B%20Wen&amp;author=X%20Wang&amp;author=B%20Zhang&amp;volume=29&amp;publication_year=2019&amp;pages=485-493&amp;pmid=30610011&amp;doi=10.1101/gr.235028.118&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R191\"> <cite> Wen B, Li K, Zhang Y, and Zhang B (2020). Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat. Commun. 11, 1759. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-15456-w\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145864/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32273506/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Cancer%20neoantigen%20prioritization%20through%20sensitive%20and%20reliable%20proteogenomics%20analysis&amp;author=B%20Wen&amp;author=K%20Li&amp;author=Y%20Zhang&amp;author=B%20Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=1759&amp;pmid=32273506&amp;doi=10.1038/s41467-020-15456-w&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R192\"> <cite> Westra H-J, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, et al. (2013). Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat. Genet. 45, 1238–1243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2756\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3991562/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013639/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&amp;author=H-J%20Westra&amp;author=MJ%20Peters&amp;author=T%20Esko&amp;author=H%20Yaghootkar&amp;author=C%20Schurmann&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;doi=10.1038/ng.2756&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R193\"> <cite> Wheatley SP, and Altieri DC (2019). Survivin at a glance. J. Cell Sci. 132. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.223826\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6467487/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30948431/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Survivin%20at%20a%20glance&amp;author=SP%20Wheatley&amp;author=DC%20Altieri&amp;volume=132&amp;publication_year=2019&amp;pmid=30948431&amp;doi=10.1242/jcs.223826&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R194\"> <cite> Whitfield ML, Sherlock G, Saldanha AJ, Murray JI, Ball CA, Alexander KE, Matese JC, Perou CM, Hurt MM, Brown PO, et al. (2002). Identification of genes periodically expressed in the human cell cycle and their expression in tumors. Mol. Biol. Cell 13, 1977–2000. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.02-02-0030.\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC117619/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12058064/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Identification%20of%20genes%20periodically%20expressed%20in%20the%20human%20cell%20cycle%20and%20their%20expression%20in%20tumors&amp;author=ML%20Whitfield&amp;author=G%20Sherlock&amp;author=AJ%20Saldanha&amp;author=JI%20Murray&amp;author=CA%20Ball&amp;volume=13&amp;publication_year=2002&amp;pages=1977-2000&amp;pmid=12058064&amp;doi=10.1091/mbc.02-02-0030.&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R195\"> <cite> Wilkerson MD, and Hayes DN (2010). ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. Bioinformatics 26, 1572–1573. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq170\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881355/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20427518/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConsensusClusterPlus:%20a%20class%20discovery%20tool%20with%20confidence%20assessments%20and%20item%20tracking&amp;author=MD%20Wilkerson&amp;author=DN%20Hayes&amp;volume=26&amp;publication_year=2010&amp;pages=1572-1573&amp;pmid=20427518&amp;doi=10.1093/bioinformatics/btq170&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R196\"> <cite> Wilkerson MD, Yin X, Hoadley KA, Liu Y, Hayward MC, Cabanski CR, Muldrew K, Miller CR, Randell SH, Socinski MA, et al. (2010). Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types. Clin. Cancer Res. 16, 4864–4875. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0199\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2953768/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20643781/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Lung%20squamous%20cell%20carcinoma%20mRNA%20expression%20subtypes%20are%20reproducible,%20clinically%20important,%20and%20correspond%20to%20normal%20cell%20types&amp;author=MD%20Wilkerson&amp;author=X%20Yin&amp;author=KA%20Hoadley&amp;author=Y%20Liu&amp;author=MC%20Hayward&amp;volume=16&amp;publication_year=2010&amp;pages=4864-4875&amp;pmid=20643781&amp;doi=10.1158/1078-0432.CCR-10-0199&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R197\"> <cite> Wong JJY, Pung YF, Sze NS-K, and Chin K-C (2006). HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets. Proc. Natl. Acad. Sci. U. S. A. 103, 10735–10740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0600397103\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1484417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16815975/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=HERC5%20is%20an%20IFN-induced%20HECT-type%20E3%20protein%20ligase%20that%20mediates%20type%20I%20IFN-induced%20ISGylation%20of%20protein%20targets&amp;author=JJY%20Wong&amp;author=YF%20Pung&amp;author=NS-K%20Sze&amp;author=K-C%20Chin&amp;volume=103&amp;publication_year=2006&amp;pages=10735-10740&amp;pmid=16815975&amp;doi=10.1073/pnas.0600397103&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R198\"> <cite> Wu Y, Zhang Y, Zheng X, Dai F, Lu Y, Dai L, Niu M, Guo H, Li W, Xue X, et al. (2020). Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis. Mol. Cancer 19, 99. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12943-020-01215-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7265647/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32487167/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Circular%20RNA%20circCORO1C%20promotes%20laryngeal%20squamous%20cell%20carcinoma%20progression%20by%20modulating%20the%20let-7c-5p/PBX3%20axis&amp;author=Y%20Wu&amp;author=Y%20Zhang&amp;author=X%20Zheng&amp;author=F%20Dai&amp;author=Y%20Lu&amp;volume=19&amp;publication_year=2020&amp;pages=99&amp;pmid=32487167&amp;doi=10.1186/s12943-020-01215-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R199\"> <cite> Wu Y-M, Cieślik M, Lonigro RJ, Vats P, Reimers MA, Cao X, Ning Y, Wang L, Kunju LP, de Sarkar N, et al. (2018). Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer. Cell 173, 1770–1782.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.04.034\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6084431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29906450/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Inactivation%20of%20CDK12%20Delineates%20a%20Distinct%20Immunogenic%20Class%20of%20Advanced%20Prostate%20Cancer&amp;author=Y-M%20Wu&amp;author=M%20Cie%C5%9Blik&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=MA%20Reimers&amp;volume=173&amp;publication_year=2018&amp;pages=1770-1782.e14&amp;pmid=29906450&amp;doi=10.1016/j.cell.2018.04.034&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R200\"> <cite> Xiong Y, Lei Q-Y, Zhao S, and Guan K-L (2011). Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK. Cold Spring Harb. Symp. Quant. Biol. 76, 285–289. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/sqb.2011.76.010942\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4876600/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22096030/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol&amp;title=Regulation%20of%20glycolysis%20and%20gluconeogenesis%20by%20acetylation%20of%20PKM%20and%20PEPCK&amp;author=Y%20Xiong&amp;author=Q-Y%20Lei&amp;author=S%20Zhao&amp;author=K-L%20Guan&amp;volume=76&amp;publication_year=2011&amp;pages=285-289&amp;pmid=22096030&amp;doi=10.1101/sqb.2011.76.010942&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R201\"> <cite> Yang W, and Lu Z (2013). Nuclear PKM2 regulates the Warburg effect. Cell Cycle 12, 3154–3158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/cc.26182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3865010/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013426/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&amp;title=Nuclear%20PKM2%20regulates%20the%20Warburg%20effect&amp;author=W%20Yang&amp;author=Z%20Lu&amp;volume=12&amp;publication_year=2013&amp;pages=3154-3158&amp;pmid=24013426&amp;doi=10.4161/cc.26182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R202\"> <cite> Ye K, Schulz MH, Long Q, Apweiler R, and Ning Z (2009). Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics 25, 2865–2871. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btp394\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2781750/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561018/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&amp;author=K%20Ye&amp;author=MH%20Schulz&amp;author=Q%20Long&amp;author=R%20Apweiler&amp;author=Z%20Ning&amp;volume=25&amp;publication_year=2009&amp;pages=2865-2871&amp;pmid=19561018&amp;doi=10.1093/bioinformatics/btp394&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R203\"> <cite> Yomtoubian S, Lee SB, Verma A, Izzo F, Markowitz G, Choi H, Cerchietti L, Vahdat L, Brown KA, Andreopoulou E, et al. (2020). Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer. Cell Rep. 30, 755–770.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2019.12.056\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31968251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Inhibition%20of%20EZH2%20Catalytic%20Activity%20Selectively%20Targets%20a%20Metastatic%20Subpopulation%20in%20Triple-Negative%20Breast%20Cancer&amp;author=S%20Yomtoubian&amp;author=SB%20Lee&amp;author=A%20Verma&amp;author=F%20Izzo&amp;author=G%20Markowitz&amp;volume=30&amp;publication_year=2020&amp;pages=755-770.e6&amp;pmid=31968251&amp;doi=10.1016/j.celrep.2019.12.056&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R204\"> <cite> Yoshihara K, Shahmoradgoli M, Martínez E, Vegesna R, Kim H, Torres-Garcia W, Treviño V, Shen H, Laird PW, Levine DA, et al. (2013). Inferring tumour purity and stromal and immune cell admixture from expression data. Nat. Commun. 4, 2612. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms3612\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3826632/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24113773/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data&amp;author=K%20Yoshihara&amp;author=M%20Shahmoradgoli&amp;author=E%20Mart%C3%ADnez&amp;author=R%20Vegesna&amp;author=H%20Kim&amp;volume=4&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;doi=10.1038/ncomms3612&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R205\"> <cite> Yuan G, Flores NM, Hausmann S, Lofgren SM, Kharchenko V, Angulo-Ibanez M, Sengupta D, Lu X, Czaban I, Azhibek D, et al. (2021). Elevated NSD3 histone methylation activity drives squamous cell lung cancer. Nature. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-03170-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7895461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33536620/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Elevated%20NSD3%20histone%20methylation%20activity%20drives%20squamous%20cell%20lung%20cancer&amp;author=G%20Yuan&amp;author=NM%20Flores&amp;author=S%20Hausmann&amp;author=SM%20Lofgren&amp;author=V%20Kharchenko&amp;publication_year=2021&amp;pmid=33536620&amp;doi=10.1038/s41586-020-03170-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R206\"> <cite> Zhang X, Lu F, Wang J, Yin F, Xu Z, Qi D, Wu X, Cao Y, Liang W, Liu Y, et al. (2013). Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells. Cell Rep. 5, 445–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2013.09.018\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC11162149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24139802/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Pluripotent%20stem%20cell%20protein%20Sox2%20confers%20sensitivity%20to%20LSD1%20inhibition%20in%20cancer%20cells&amp;author=X%20Zhang&amp;author=F%20Lu&amp;author=J%20Wang&amp;author=F%20Yin&amp;author=Z%20Xu&amp;volume=5&amp;publication_year=2013&amp;pages=445-457&amp;pmid=24139802&amp;doi=10.1016/j.celrep.2013.09.018&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R207\"> <cite> Zhang Y, Zou X, Qian W, Weng X, Zhang L, Zhang L, Wang S, Cao X, Ma L, Wei G, et al. (2019a). Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis. Cell Death Differ. 26, 565–579. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41418-018-0147-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6370781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29955124/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Enhanced%20PAPSS2/VCAN%20sulfation%20axis%20is%20essential%20for%20Snail-mediated%20breast%20cancer%20cell%20migration%20and%20metastasis&amp;author=Y%20Zhang&amp;author=X%20Zou&amp;author=W%20Qian&amp;author=X%20Weng&amp;author=L%20Zhang&amp;volume=26&amp;publication_year=2019a&amp;pages=565-579&amp;pmid=29955124&amp;doi=10.1038/s41418-018-0147-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R208\"> <cite> Zhang Y, Thery F, Wu NC, Luhmann EK, Dussurget O, Foecke M, Bredow C, Jiménez-Fernández D, Leandro K, Beling A, et al. (2019b). The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection. Nat. Commun. 10, 5383. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-13393-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6879477/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31772204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=The%20in%20vivo%20ISGylome%20links%20ISG15%20to%20metabolic%20pathways%20and%20autophagy%20upon%20Listeria%20monocytogenes%20infection&amp;author=Y%20Zhang&amp;author=F%20Thery&amp;author=NC%20Wu&amp;author=EK%20Luhmann&amp;author=O%20Dussurget&amp;volume=10&amp;publication_year=2019b&amp;pages=5383&amp;pmid=31772204&amp;doi=10.1038/s41467-019-13393-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R209\"> <cite> Zhao D, Li F-L, Cheng Z-L, and Lei Q-Y (2014). Impact of acetylation on tumor metabolism. Mol Cell Oncol 1, e963452. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/23723548.2014.963452\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4905055/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27308346/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Oncol&amp;title=Impact%20of%20acetylation%20on%20tumor%20metabolism&amp;author=D%20Zhao&amp;author=F-L%20Li&amp;author=Z-L%20Cheng&amp;author=Q-Y%20Lei&amp;volume=1&amp;publication_year=2014&amp;pages=e963452&amp;pmid=27308346&amp;doi=10.4161/23723548.2014.963452&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R210\"> <cite> Zheng L, and Conner SD (2018). Glycogen synthase kinase 3β inhibition enhances Notch1 recycling. Mol. Biol. Cell 29, 389–395. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.E17-07-0474\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6014177/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29237816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Glycogen%20synthase%20kinase%203%CE%B2%20inhibition%20enhances%20Notch1%20recycling&amp;author=L%20Zheng&amp;author=SD%20Conner&amp;volume=29&amp;publication_year=2018&amp;pages=389-395&amp;pmid=29237816&amp;doi=10.1091/mbc.E17-07-0474&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R211\"> <cite> Zhou X, Edmonson MN, Wilkinson MR, Patel A, Wu G, Liu Y, Li Y, Zhang Z, Rusch MC, Parker M, et al. (2016). Exploring genomic alteration in pediatric cancer using ProteinPaint. Nat. Genet. 48, 4–6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3466\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4892362/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26711108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Exploring%20genomic%20alteration%20in%20pediatric%20cancer%20using%20ProteinPaint&amp;author=X%20Zhou&amp;author=MN%20Edmonson&amp;author=MR%20Wilkinson&amp;author=A%20Patel&amp;author=G%20Wu&amp;volume=48&amp;publication_year=2016&amp;pages=4-6&amp;pmid=26711108&amp;doi=10.1038/ng.3466&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ol> </section> </section> <section class=\"associated-data\" id=\"_ad93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> Associated Data </h2> <p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> <section class=\"supplementary-materials\" id=\"_adsm93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Supplementary Materials </h3> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material1_reqid_\"> <div class=\"caption p\"> <span> 1 </span> <p> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material2_reqid_\"> <div class=\"caption p\"> <span> 2 </span> <p> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p> C. Same as B but in the EMT-enriched subtype. </p> <p> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material3_reqid_\"> <div class=\"caption p\"> <span> 3 </span> <p> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material4_reqid_\"> <div class=\"caption p\"> <span> 4 </span> <p> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material5_reqid_\"> <div class=\"caption p\"> <span> 6 </span> <p> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material6_reqid_\"> <div class=\"caption p\"> <span> 7 </span> <p> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material7_reqid_\"> <div class=\"caption p\"> <span> 8 </span> <p> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Metadata associated with this study </p> <p> B. Cancer Associated Genes (CAGs) </p> <p> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material8_reqid_\"> <div class=\"caption p\"> <span> 9 </span> <p> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material9_reqid_\"> <div class=\"caption p\"> <span> 10 </span> <p> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material10_reqid_\"> <div class=\"caption p\"> <span> 11 </span> <p> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p> A. Copy number amplified cytobands </p> <p> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p> F. CMAP tab- Connectivity Map (CMAP) results </p> <p> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p> I. Summary of CDKN2A and RB1 annotations </p> <p> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p> L. NRF2 signature </p> <p> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material11_reqid_\"> <div class=\"caption p\"> <span> 12 </span> <p> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p> B. Ub-sites that show differential abundance across Ub clusters </p> <p> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material12_reqid_\"> <div class=\"caption p\"> <span> 13 </span> <p> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p> B. Immune clusters based on xCell signatures </p> <p> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p> I. List of genes/proteins whose expression is associated with a SNP </p> <p> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material13_reqid_\"> <div class=\"caption p\"> <span> 14 </span> <p> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material14_reqid_\"> <div class=\"caption p\"> <span> Figure S5 </span> <p> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"data-availability-statement\" id=\"_adda93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Data Availability Statement </h3> <section id=\"db_da_sec19_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"db_da_sec20_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"db_da_sec21_reqid_\"> <h3 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"db_da_table-wrap1_reqid_\"> <h4 class=\"obj_head\"> KEY RESOURCES TABLE. </h4> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/db_da_table-wrap1_reqid_/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> </section> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/nihms-1730863.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (3.6 MB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\" hidden=\"\"> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> </div> </div> </div> <div class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br> 8600 Rockville Pike <br> Bethesda, MD 20894 </br> </br> </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> </body> </html>', 'repos': 'bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, geo, dbgap, sra, arrayexpress, nuccore, data.mendeley.com, synapse.org, doi.org, pdb'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1221 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1228 - DEBUG - Counting tokens for model: gemini-2.5-flash, prompt length: 729175 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1237 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.5-flash': 182293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 182293 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 284 - DEBUG - Prompt messages: [{'role': 'system', 'content': 'You are PaperMiner, a specialized assistant that extracts datasets from research publications to enable data reuse. Your mission is to help future researchers discover and properly reuse datasets by understanding how they were originally used in scientific papers. In 2025, the abundance of data is unprecedented, research velocity is accelerating, and proper dataset provenance from publications is crucial for scientific progress.'}, {'role': 'user', 'content': 'Your task: Extract datasets mentioned in research papers and capture their usage context to enable data reuse.\\n\\nWhy this matters: When researchers want to reuse data, they need to understand how it was originally used, what methodology was applied, and what insights it provided. This provenance information from the source publication is essential for proper data reuse.\\n\\nOutput format: JSON object with a \"datasets\" array containing objects with these required keys:\\n- \"dataset_identifier\": A unique identifier or accession code for the dataset\\n- \"repository_reference\": A valid URI or string referring to the repository where the dataset can be found\\n- \"dataset_context_from_paper\": Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly\\n- \"citation_type\": Type of citation used for this dataset. Must be either \"Primary\" (firsthand information collected by the researcher for a specific purpose) or \"Secondary\" (pre-existing information collected by someone else and then used by another researcher)\\n\\nAnalysis approach:\\n1. Find dataset identifiers and their repositories\\n2. Extract relevant text passages that describe the dataset\\'s use\\n3. Determine if this is Primary (authors collected the data) or Secondary (authors used existing data)\\n4. Include direct quotes or paraphrased descriptions from the paper\\n\\nExample output:\\n{\\n  \"datasets\": [\\n    {\\n      \"dataset_identifier\": \"GSE69091\",\\n      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\",\\n      \"dataset_context_from_paper\": \"Gene expression dataset from 150 breast cancer patients used to train machine learning models for predicting chemotherapy response. Authors applied random forest classification achieving 85% accuracy. Dataset includes clinical outcomes and 20,000 gene expression values per patient.\",\\n      \"citation_type\": \"Primary\"\\n    },\\n    {\\n      \"dataset_identifier\": \"EGAS00001000925\", \\n      \"repository_reference\": \"https://ega-archive.org/datasets/EGAS00001000925\",\\n      \"dataset_context_from_paper\": \"Whole genome sequencing data from 1000 European individuals used as reference population for variant calling pipeline validation. Authors used this dataset to establish baseline allele frequencies and validate their new variant detection algorithm.\",\\n      \"citation_type\": \"Secondary\"\\n    }\\n  ]\\n}\\n\\nNow analyze this publication content: <!DOCTYPE html> <html lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> A proteogenomic portrait of lung squamous cell carcinoma - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here\\'s how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you\\'ve safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" role=\"button\" target=\"_blank\" title=\"View on publisher site\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip\" href=\"pdf/nihms-1730863.pdf\" role=\"button\" title=\"Download PDF\"> </a> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> </li> <li class=\"pmc-permalink display-flex\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to NIHPA Author Manuscripts\" href=\"https://www.hhs.gov/open/public-access-guiding-principles/index.html\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to NIHPA Author Manuscripts\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . Author manuscript; available in PMC: 2022 Aug 5. </div> <div> <em> Published in final edited form as: </em> Cell. 2021 Aug 5;184(16):4348–4371.e40. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1016/j.cell.2021.07.016 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Cell%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> A proteogenomic portrait of lung squamous cell carcinoma </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Shankha Satpathy </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Satpathy%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankha Satpathy </span> </a> </div> </div> <sup> 1, </sup> <sup> 19, </sup> <sup> * </sup> , <a aria-describedby=\"id2\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Karsten Krug </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krug%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Karsten Krug </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id3\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Pierre M Jean Beltran </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%20Beltran%20PM%22%5BAuthor%5D\"> <span class=\"name western\"> Pierre M Jean Beltran </span> </a> </div> </div> <sup> 1, </sup> <sup> 19 </sup> , <a aria-describedby=\"id4\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Sara R Savage </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 19 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Savage%20SR%22%5BAuthor%5D\"> <span class=\"name western\"> Sara R Savage </span> </a> </div> </div> <sup> 2, </sup> <sup> 19 </sup> , <a aria-describedby=\"id5\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Francesca Petralia </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Petralia%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Francesca Petralia </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id6\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> <div hidden=\"hidden\" id=\"id6\"> <h3> <span class=\"name western\"> Chandan Kumar-Sinha </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar-Sinha%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chandan Kumar-Sinha </span> </a> </div> </div> <sup> 4, </sup> <sup> 20 </sup> , <a aria-describedby=\"id7\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> <div hidden=\"hidden\" id=\"id7\"> <h3> <span class=\"name western\"> Yongchao Dou </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dou%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yongchao Dou </span> </a> </div> </div> <sup> 2, </sup> <sup> 20 </sup> , <a aria-describedby=\"id8\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> <div hidden=\"hidden\" id=\"id8\"> <h3> <span class=\"name western\"> Boris Reva </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Reva%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Boris Reva </span> </a> </div> </div> <sup> 3, </sup> <sup> 20 </sup> , <a aria-describedby=\"id9\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> <div hidden=\"hidden\" id=\"id9\"> <h3> <span class=\"name western\"> M Harry Kane </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Kane%20MH%22%5BAuthor%5D\"> <span class=\"name western\"> M Harry Kane </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id10\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> <div hidden=\"hidden\" id=\"id10\"> <h3> <span class=\"name western\"> Shayan C Avanessian </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Avanessian%20SC%22%5BAuthor%5D\"> <span class=\"name western\"> Shayan C Avanessian </span> </a> </div> </div> <sup> 1, </sup> <sup> 20 </sup> , <a aria-describedby=\"id11\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> <div hidden=\"hidden\" id=\"id11\"> <h3> <span class=\"name western\"> Suhas V Vasaikar </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> <sup> 20 </sup> Equal contribution </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasaikar%20SV%22%5BAuthor%5D\"> <span class=\"name western\"> Suhas V Vasaikar </span> </a> </div> </div> <sup> 5, </sup> <sup> 20 </sup> , <a aria-describedby=\"id12\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> <div hidden=\"hidden\" id=\"id12\"> <h3> <span class=\"name western\"> Azra Krek </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Krek%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Azra Krek </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id13\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> <div hidden=\"hidden\" id=\"id13\"> <h3> <span class=\"name western\"> Jonathan T Lei </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Lei%20JT%22%5BAuthor%5D\"> <span class=\"name western\"> Jonathan T Lei </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id14\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> <div hidden=\"hidden\" id=\"id14\"> <h3> <span class=\"name western\"> Eric J Jaehnig </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaehnig%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Eric J Jaehnig </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id15\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> <div hidden=\"hidden\" id=\"id15\"> <h3> <span class=\"name western\"> Tatiana Omelchenko </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omelchenko%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tatiana Omelchenko </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id16\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> <div hidden=\"hidden\" id=\"id16\"> <h3> <span class=\"name western\"> Yifat Geffen </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Geffen%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yifat Geffen </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id17\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> <div hidden=\"hidden\" id=\"id17\"> <h3> <span class=\"name western\"> Erik J Bergstrom </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergstrom%20EJ%22%5BAuthor%5D\"> <span class=\"name western\"> Erik J Bergstrom </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id18\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> <div hidden=\"hidden\" id=\"id18\"> <h3> <span class=\"name western\"> Vasileios Stathias </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stathias%20V%22%5BAuthor%5D\"> <span class=\"name western\"> Vasileios Stathias </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id19\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> <div hidden=\"hidden\" id=\"id19\"> <h3> <span class=\"name western\"> Karen E Christianson </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Christianson%20KE%22%5BAuthor%5D\"> <span class=\"name western\"> Karen E Christianson </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id20\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> <div hidden=\"hidden\" id=\"id20\"> <h3> <span class=\"name western\"> David I Heiman </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Heiman%20DI%22%5BAuthor%5D\"> <span class=\"name western\"> David I Heiman </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id21\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> <div hidden=\"hidden\" id=\"id21\"> <h3> <span class=\"name western\"> Marcin P Cieslik </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cieslik%20MP%22%5BAuthor%5D\"> <span class=\"name western\"> Marcin P Cieslik </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id22\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> <div hidden=\"hidden\" id=\"id22\"> <h3> <span class=\"name western\"> Song Cao </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Song Cao </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id23\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> <div hidden=\"hidden\" id=\"id23\"> <h3> <span class=\"name western\"> Xiaoyu Song </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D\"> <span class=\"name western\"> Xiaoyu Song </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id24\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> <div hidden=\"hidden\" id=\"id24\"> <h3> <span class=\"name western\"> Jiayi Ji </span> </h3> <div class=\"p\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20J%22%5BAuthor%5D\"> <span class=\"name western\"> Jiayi Ji </span> </a> </div> </div> <sup> 9 </sup> , <a aria-describedby=\"id25\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> <div hidden=\"hidden\" id=\"id25\"> <h3> <span class=\"name western\"> Wenke Liu </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D\"> <span class=\"name western\"> Wenke Liu </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id26\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> <div hidden=\"hidden\" id=\"id26\"> <h3> <span class=\"name western\"> Kai Li </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D\"> <span class=\"name western\"> Kai Li </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id27\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> <div hidden=\"hidden\" id=\"id27\"> <h3> <span class=\"name western\"> Bo Wen </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bo Wen </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id28\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> <div hidden=\"hidden\" id=\"id28\"> <h3> <span class=\"name western\"> Yize Li </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yize Li </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id29\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> <div hidden=\"hidden\" id=\"id29\"> <h3> <span class=\"name western\"> Zeynep H Gümüş </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%BCm%C3%BC%C5%9F%20ZH%22%5BAuthor%5D\"> <span class=\"name western\"> Zeynep H Gümüş </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id30\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> <div hidden=\"hidden\" id=\"id30\"> <h3> <span class=\"name western\"> Myvizhi Esai Selvan </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Selvan%20ME%22%5BAuthor%5D\"> <span class=\"name western\"> Myvizhi Esai Selvan </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id31\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> <div hidden=\"hidden\" id=\"id31\"> <h3> <span class=\"name western\"> Rama Soundararajan </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Soundararajan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Rama Soundararajan </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id32\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> <div hidden=\"hidden\" id=\"id32\"> <h3> <span class=\"name western\"> Tanvi H Visal </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Visal%20TH%22%5BAuthor%5D\"> <span class=\"name western\"> Tanvi H Visal </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id33\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> <div hidden=\"hidden\" id=\"id33\"> <h3> <span class=\"name western\"> Maria G Raso </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Raso%20MG%22%5BAuthor%5D\"> <span class=\"name western\"> Maria G Raso </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id34\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> <div hidden=\"hidden\" id=\"id34\"> <h3> <span class=\"name western\"> Edwin Roger Parra </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20ER%22%5BAuthor%5D\"> <span class=\"name western\"> Edwin Roger Parra </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id35\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> <div hidden=\"hidden\" id=\"id35\"> <h3> <span class=\"name western\"> Özgün Babur </span> </h3> <div class=\"p\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Babur%20%C3%96%22%5BAuthor%5D\"> <span class=\"name western\"> Özgün Babur </span> </a> </div> </div> <sup> 11 </sup> , <a aria-describedby=\"id36\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> <div hidden=\"hidden\" id=\"id36\"> <h3> <span class=\"name western\"> Pankaj Vats </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Vats%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pankaj Vats </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id37\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> <div hidden=\"hidden\" id=\"id37\"> <h3> <span class=\"name western\"> Shankara Anand </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Shankara Anand </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id38\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> <div hidden=\"hidden\" id=\"id38\"> <h3> <span class=\"name western\"> Tobias Schraink </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schraink%20T%22%5BAuthor%5D\"> <span class=\"name western\"> Tobias Schraink </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id39\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> <div hidden=\"hidden\" id=\"id39\"> <h3> <span class=\"name western\"> MacIntosh Cornwell </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Cornwell%20M%22%5BAuthor%5D\"> <span class=\"name western\"> MacIntosh Cornwell </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id40\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> <div hidden=\"hidden\" id=\"id40\"> <h3> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20FM%22%5BAuthor%5D\"> <span class=\"name western\"> Fernanda Martins Rodrigues </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id41\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> <div hidden=\"hidden\" id=\"id41\"> <h3> <span class=\"name western\"> Houxiang Zhu </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Houxiang Zhu </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id42\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> <div hidden=\"hidden\" id=\"id42\"> <h3> <span class=\"name western\"> Chia-Kuei Mo </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mo%20CK%22%5BAuthor%5D\"> <span class=\"name western\"> Chia-Kuei Mo </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id43\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> <div hidden=\"hidden\" id=\"id43\"> <h3> <span class=\"name western\"> Yuping Zhang </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D\"> <span class=\"name western\"> Yuping Zhang </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id44\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> <div hidden=\"hidden\" id=\"id44\"> <h3> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22da%20Veiga%20Leprevost%20F%22%5BAuthor%5D\"> <span class=\"name western\"> Felipe da Veiga Leprevost </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id45\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> <div hidden=\"hidden\" id=\"id45\"> <h3> <span class=\"name western\"> Chen Huang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20C%22%5BAuthor%5D\"> <span class=\"name western\"> Chen Huang </span> </a> </div> </div> <sup> 2 </sup> , <a aria-describedby=\"id46\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> <div hidden=\"hidden\" id=\"id46\"> <h3> <span class=\"name western\"> Arul M Chinnaiyan </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinnaiyan%20AM%22%5BAuthor%5D\"> <span class=\"name western\"> Arul M Chinnaiyan </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id47\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> <div hidden=\"hidden\" id=\"id47\"> <h3> <span class=\"name western\"> Matthew A Wyczalkowski </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyczalkowski%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Matthew A Wyczalkowski </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id48\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> <div hidden=\"hidden\" id=\"id48\"> <h3> <span class=\"name western\"> Gilbert S Omenn </span> </h3> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Omenn%20GS%22%5BAuthor%5D\"> <span class=\"name western\"> Gilbert S Omenn </span> </a> </div> </div> <sup> 13 </sup> , <a aria-describedby=\"id49\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> <div hidden=\"hidden\" id=\"id49\"> <h3> <span class=\"name western\"> Chelsea J Newton </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20CJ%22%5BAuthor%5D\"> <span class=\"name western\"> Chelsea J Newton </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id50\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> <div hidden=\"hidden\" id=\"id50\"> <h3> <span class=\"name western\"> Stephan Schurer </span> </h3> <div class=\"p\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Schurer%20S%22%5BAuthor%5D\"> <span class=\"name western\"> Stephan Schurer </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id51\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> <div hidden=\"hidden\" id=\"id51\"> <h3> <span class=\"name western\"> Kelly V Ruggles </span> </h3> <div class=\"p\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruggles%20KV%22%5BAuthor%5D\"> <span class=\"name western\"> Kelly V Ruggles </span> </a> </div> </div> <sup> 12 </sup> , <a aria-describedby=\"id52\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> <div hidden=\"hidden\" id=\"id52\"> <h3> <span class=\"name western\"> David Fenyö </span> </h3> <div class=\"p\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Feny%C3%B6%20D%22%5BAuthor%5D\"> <span class=\"name western\"> David Fenyö </span> </a> </div> </div> <sup> 10 </sup> , <a aria-describedby=\"id53\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> <div hidden=\"hidden\" id=\"id53\"> <h3> <span class=\"name western\"> Scott D Jewell </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewell%20SD%22%5BAuthor%5D\"> <span class=\"name western\"> Scott D Jewell </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id54\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> <div hidden=\"hidden\" id=\"id54\"> <h3> <span class=\"name western\"> Mathangi Thiagarajan </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiagarajan%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mathangi Thiagarajan </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id55\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> <div hidden=\"hidden\" id=\"id55\"> <h3> <span class=\"name western\"> Mehdi Mesri </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mesri%20M%22%5BAuthor%5D\"> <span class=\"name western\"> Mehdi Mesri </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id56\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> <div hidden=\"hidden\" id=\"id56\"> <h3> <span class=\"name western\"> Henry Rodriguez </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20H%22%5BAuthor%5D\"> <span class=\"name western\"> Henry Rodriguez </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id57\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> <div hidden=\"hidden\" id=\"id57\"> <h3> <span class=\"name western\"> Sendurai A Mani </span> </h3> <div class=\"p\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Sendurai A Mani </span> </a> </div> </div> <sup> 5 </sup> , <a aria-describedby=\"id58\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> <div hidden=\"hidden\" id=\"id58\"> <h3> <span class=\"name western\"> Namrata D Udeshi </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Udeshi%20ND%22%5BAuthor%5D\"> <span class=\"name western\"> Namrata D Udeshi </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id59\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> <div hidden=\"hidden\" id=\"id59\"> <h3> <span class=\"name western\"> Gad Getz </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Getz%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gad Getz </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id60\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> <div hidden=\"hidden\" id=\"id60\"> <h3> <span class=\"name western\"> James Suh </span> </h3> <div class=\"p\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Suh%20J%22%5BAuthor%5D\"> <span class=\"name western\"> James Suh </span> </a> </div> </div> <sup> 15 </sup> , <a aria-describedby=\"id61\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> <div hidden=\"hidden\" id=\"id61\"> <h3> <span class=\"name western\"> Qing Kay Li </span> </h3> <div class=\"p\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20QK%22%5BAuthor%5D\"> <span class=\"name western\"> Qing Kay Li </span> </a> </div> </div> <sup> 17 </sup> , <a aria-describedby=\"id62\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> <div hidden=\"hidden\" id=\"id62\"> <h3> <span class=\"name western\"> Galen Hostetter </span> </h3> <div class=\"p\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Hostetter%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Galen Hostetter </span> </a> </div> </div> <sup> 14 </sup> , <a aria-describedby=\"id63\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> <div hidden=\"hidden\" id=\"id63\"> <h3> <span class=\"name western\"> Paul K Paik </span> </h3> <div class=\"p\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Paik%20PK%22%5BAuthor%5D\"> <span class=\"name western\"> Paul K Paik </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id64\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> <div hidden=\"hidden\" id=\"id64\"> <h3> <span class=\"name western\"> Saravana M Dhanasekaran </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhanasekaran%20SM%22%5BAuthor%5D\"> <span class=\"name western\"> Saravana M Dhanasekaran </span> </a> </div> </div> <sup> 4 </sup> , <a aria-describedby=\"id65\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> <div hidden=\"hidden\" id=\"id65\"> <h3> <span class=\"name western\"> Ramaswamy Govindan </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Govindan%20R%22%5BAuthor%5D\"> <span class=\"name western\"> Ramaswamy Govindan </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id66\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> <div hidden=\"hidden\" id=\"id66\"> <h3> <span class=\"name western\"> Li Ding </span> </h3> <div class=\"p\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20L%22%5BAuthor%5D\"> <span class=\"name western\"> Li Ding </span> </a> </div> </div> <sup> 8 </sup> , <a aria-describedby=\"id67\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> <div hidden=\"hidden\" id=\"id67\"> <h3> <span class=\"name western\"> Ana I Robles </span> </h3> <div class=\"p\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Robles%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Ana I Robles </span> </a> </div> </div> <sup> 16 </sup> , <a aria-describedby=\"id68\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> <div hidden=\"hidden\" id=\"id68\"> <h3> <span class=\"name western\"> Karl R Clauser </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Clauser%20KR%22%5BAuthor%5D\"> <span class=\"name western\"> Karl R Clauser </span> </a> </div> </div> <sup> 1 </sup> , <a aria-describedby=\"id69\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> <div hidden=\"hidden\" id=\"id69\"> <h3> <span class=\"name western\"> Alexey I Nesvizhskii </span> </h3> <div class=\"p\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Nesvizhskii%20AI%22%5BAuthor%5D\"> <span class=\"name western\"> Alexey I Nesvizhskii </span> </a> </div> </div> <sup> 4, </sup> <sup> 13 </sup> , <a aria-describedby=\"id70\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> <div hidden=\"hidden\" id=\"id70\"> <h3> <span class=\"name western\"> Pei Wang </span> </h3> <div class=\"p\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D\"> <span class=\"name western\"> Pei Wang </span> </a> </div> </div> <sup> 3 </sup> , <a aria-describedby=\"id71\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> <div hidden=\"hidden\" id=\"id71\"> <h3> <span class=\"name western\"> Steven A Carr </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20SA%22%5BAuthor%5D\"> <span class=\"name western\"> Steven A Carr </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id72\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> <div hidden=\"hidden\" id=\"id72\"> <h3> <span class=\"name western\"> Bing Zhang </span> </h3> <div class=\"p\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D\"> <span class=\"name western\"> Bing Zhang </span> </a> </div> </div> <sup> 2, </sup> <sup> * </sup> , <a aria-describedby=\"id73\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> <div hidden=\"hidden\" id=\"id73\"> <h3> <span class=\"name western\"> D R Mani </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Mani%20DR%22%5BAuthor%5D\"> <span class=\"name western\"> D R Mani </span> </a> </div> </div> <sup> 1, </sup> <sup> * </sup> , <a aria-describedby=\"id74\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> <div hidden=\"hidden\" id=\"id74\"> <h3> <span class=\"name western\"> Michael A Gillette </span> </h3> <div class=\"p\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div class=\"p\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div class=\"p\"> <sup> 21 </sup> Lead author </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Gillette%20MA%22%5BAuthor%5D\"> <span class=\"name western\"> Michael A Gillette </span> </a> </div> </div> <sup> 1, </sup> <sup> 18, </sup> <sup> 21, </sup> <sup> * </sup> , <span class=\"on-behalf-of\"> Clinical Proteomic Tumor Analysis Consortium </span> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"A1\"> <sup> 1 </sup> Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, 02142, USA </div> <div id=\"A2\"> <sup> 2 </sup> Lester and Sue Smith Breast Center; Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA </div> <div id=\"A3\"> <sup> 3 </sup> Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA. </div> <div id=\"A4\"> <sup> 4 </sup> Department of Pathology, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A5\"> <sup> 5 </sup> Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA. </div> <div id=\"A6\"> <sup> 6 </sup> Memorial Sloan Kettering Cancer Center, New York, NY, 10065, USA </div> <div id=\"A7\"> <sup> 7 </sup> Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA </div> <div id=\"A8\"> <sup> 8 </sup> Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA </div> <div id=\"A9\"> <sup> 9 </sup> Department of Population Health Science and Policy; Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, 10029, USA </div> <div id=\"A10\"> <sup> 10 </sup> Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A11\"> <sup> 11 </sup> Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA </div> <div id=\"A12\"> <sup> 12 </sup> Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA. </div> <div id=\"A13\"> <sup> 13 </sup> Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI, 48109, USA </div> <div id=\"A14\"> <sup> 14 </sup> Van Andel Research Institute, Grand Rapids, MI, 49503, USA </div> <div id=\"A15\"> <sup> 15 </sup> Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD, 21702, USA. </div> <div id=\"A16\"> <sup> 16 </sup> Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD, 20892, USA. </div> <div id=\"A17\"> <sup> 17 </sup> Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD, 21224, USA </div> <div id=\"A18\"> <sup> 18 </sup> Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA, 02115, USA </div> <div id=\"A19\"> <sup> 19 </sup> Equal contribution </div> <div id=\"A20\"> <sup> 20 </sup> Equal contribution </div> <div id=\"A21\"> <sup> 21 </sup> Lead author </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"FN1\"> <p id=\"P1\"> Author Contributions </p> <p id=\"P2\"> <strong> Conception </strong> : SS, SAC, MAG; <strong> Data Collection </strong> : SS, MHK, SCA, EJB; <strong> Data analysis </strong> : SS, KK, PMJB, SRS, FP, CKS, YD, BR,MHK, SVV, AK, JTL, EJJ, TO, YG, VS, KEC, DIH, MPC, SC, XS, JJ, WL, KL, BW, YL,ZHG, MES, OB, PV, SA, TS, MC, CKM, YZ, FVL, CH, MAW, SMD, KRC, DRM; <strong> QC </strong> : CJN, GH, JS, QKL, MT; <strong> IHC and analysis </strong> : RS, TV, MGR, ERP, SAM, CJN, GH, JS, KL; <strong> Manuscript writing and editing </strong> : SS and MAG with help from all authors <strong> Supervision </strong> : SS, LD, DF, KR, MM, HR, MT, AIR, PW, AIN, SAC, BZ, DRM, MAG </p> </div> <div class=\"fn\" id=\"FN2\"> <p id=\"P3\"> Secondary Authors </p> <p id=\"P4\"> Alex Green, Alfredo Molinolo, Alicia Francis, Amanda G. Paulovich, Andrii Karnuta, Antonio Colaprico, Barbara Hindenach, Barbara L. Pruetz, Bartosz Kubisa, Brian J. Druker, Carissa Huynh, Charles A. Goldthwaite Jr., Chet Birger, Christopher R. Kinsinger, Corbin D. Jones, Dan Rohrer, Dana R. Valley, Daniel W. Chan, David Chesla, Donna Hansel, Elena V. Ponomareva, Elizabeth Duffy, Eric Burks, Eric E. Schadt, Eugene S. Fedorov, Eunkyung An, Fei Ding, George D. Wilson, Harsh Batra, Hui Zhang, Jennifer E. Maas, Jennifer Eschbacher, Karen A. Ketchum, Karin D. Rodland, Katherine A. Hoadley, Kei Suzuki, Ki Sung Um, Liqun Qi, Lori Bernard, Maciej Wiznerowicz, Małgorzata Wojtyś, Marcin J. Domagalski, Matthew J. Ellis, Maureen A. Dyer, Melissa Borucki, Meenakshi Anurag, Michael J. Birrer, Midie Xu, Mikhail Krotevich, Nancy Roche, Nathan J. Edwards, Negin Vatanian, Neil R. Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Pamela Grady, Peter B. McGarvey, Pushpa Hariharan, Ratna R. Thangudu, Rebecca Montgomery, Renganayaki Pandurengan, Richard D. Smith, Robert J. Welsh, Sailaja Mareedu, Samuel H. Payne, Sandra Cottingham, Shilpi Singh, Shirley X. Tsang, Shuang Cai, Stacey Gabriel, Tao Liu, Tara Hiltke, Tanmayi Vashist, Thomas Bauer, Volodymyr Sovenko, Warren G. Tourtellotte, Weiping Ma, William Bocik, Wohaib Hasan, Xiaojun Jing, Ximing Tang, Yuxing Liao, Yvonne, Shutack, Zhen Zhang, Ziad Hanhan </p> </div> <div class=\"fn\" id=\"CR1\"> <sup> * </sup> <p class=\"display-inline\"> Corresponding authors: <span> shankha@broadinstitute.org </span> (S.S.), <span> scarr@broad.mit.edu </span> (S.A.C), <span> bing.zhang@bcm.edu </span> (B.Z.), <span> manidr@broadinstitute.org </span> (D.R.M.), <span> gillette@broadinstitute.org </span> (M.A.G.) </p> </div> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC8475722 NIHMSID: NIHMS1730863 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34358469/\"> 34358469 </a> </div> <div aria-label=\"Related or updated information about this article\" class=\"ra xbox p\" role=\"complementary\"> <div> The publisher\\'s version of this article is available at <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noopener noreferrer\" target=\"_blank\"> Cell </a> </div> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"ABS1\"> <h2> Summary </h2> <p id=\"P5\"> Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. </p> <section class=\"kwd-group\" id=\"kwd-group1\"> <p> <strong> Keywords: </strong> Lung Cancer, Squamous, Proteogenomics, Proteomics, Genomics, Protein, Phosphorylation, Acetylation, Ubiquitylation, CPTAC </p> </section> </section> <section class=\"abstract\" id=\"ABS2\"> <h2> In Brief </h2> <p id=\"P6\"> Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential therapeutic targets. </p> </section> <section class=\"abstract\" id=\"ABS3\"> <h2> Graphical Abstract </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> </section> <section id=\"S1\"> <h2 class=\"pmc_sec_title\"> Introduction </h2> <p id=\"P8\"> Lung cancer is the leading cause of cancer-associated mortality ( <a aria-describedby=\"R15\" class=\"usa-link\" href=\"#R15\"> Bray et al., 2018 </a> ; <a aria-describedby=\"R158\" class=\"usa-link\" href=\"#R158\"> Siegel et al., 2020 </a> ). Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ; <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as <em> CDKN2A </em> ) are found in upwards of 60% but targeting these clinically has largely failed ( <a aria-describedby=\"R132\" class=\"usa-link\" href=\"#R132\"> Paik et al., 2019 </a> ). While other therapeutic regimens are being explored as part of the LungMAP consortium ( <a aria-describedby=\"R7\" class=\"usa-link\" href=\"#R7\"> Ardini-Poleske et al., 2017 </a> ), to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC ( <a aria-describedby=\"R76\" class=\"usa-link\" href=\"#R76\"> Karachaliou et al., 2018 </a> ; <a aria-describedby=\"R137\" class=\"usa-link\" href=\"#R137\"> Paz-Ares et al., 2018 </a> ). Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. </p> <p id=\"P9\"> We characterized 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. </p> </section> <section id=\"S2\"> <h2 class=\"pmc_sec_title\"> Results </h2> <section id=\"S3\"> <h3 class=\"pmc_sec_title\"> Proteogenomic landscape of LSCC </h3> <p id=\"P10\"> We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> , <a class=\"usa-link\" href=\"#SD1\"> S1A </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> ). Proteomics data were generated using TMT11-based multiplexing linked by a common reference ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1A </a> ) and demonstrated high technical quality ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1B </a> – <a class=\"usa-link\" href=\"#SD1\"> C </a> ). The cohort demographics are summarized in <a class=\"usa-link\" href=\"#F1\"> Figure 1B </a> . Most self-reported never-smokers displayed high genomic smoking scores ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1D </a> ), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). The genomic landscape reaffirmed previously reported somatic alterations in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1C </a> ). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1E </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F1\"> <h4 class=\"obj_head\"> Figure 1: Proteogenomic Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0002.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P373\"> A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. </p> <p id=\"P374\"> B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. </p> <p id=\"P375\"> C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value &lt; 0.1) in this dataset are highlighted in red font. </p> <p id=\"P376\"> D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR &lt;0.01) positive and negative correlations, respectively. </p> <p id=\"P377\"> E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q&lt;0.25) and <em> cis </em> -effects in both mRNA and protein (FDR&lt;0.05). </p> <p id=\"P378\"> F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the <em> FGFR1 </em> gene (GISTIC thresholded value =2). </p> <p id=\"P379\"> G. Genes whose DNA methylation was significantly associated with cascading <em> cis </em> regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the <em> cis </em> -effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. </p> <p id=\"P380\"> See also <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> 3 </a> </p> </figcaption> </figure> <p id=\"P11\"> Proteogenomic data helped annotate the impact of copy number alteration (CNA) events ( <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> , <a class=\"usa-link\" href=\"#F1\"> E </a> ). Of 5,523 significant (FDR&lt;0.01) <em> cis </em> CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 “cancer associated genes” (CAG) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). We identified six amplified (including <em> WHSC1L1, CCND1 </em> , and <em> SOX2 </em> ) and 29 deleted (including <em> NCOR1 </em> , <em> SETD2 </em> , and <em> CBL </em> ) CAGs in significant focal events (Q&lt;0.25) ( <a class=\"usa-link\" href=\"#F1\"> Figure 1E </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ). Intriguingly, <em> WHSC1L1 </em> ( <em> NSD3 </em> ) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses <em> FGFR1 </em> . Therapies targeted against FGFR1 have been unsuccessful ( <a aria-describedby=\"R108\" class=\"usa-link\" href=\"#R108\"> Lim et al., 2016 </a> ; <a aria-describedby=\"R189\" class=\"usa-link\" href=\"#R189\"> Weeden et al., 2015 </a> ). Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon ( <a class=\"usa-link\" href=\"#F1\"> Figure 1F </a> ), nominating it as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes ( <a aria-describedby=\"R205\" class=\"usa-link\" href=\"#R205\"> Yuan et al., 2021 </a> ). </p> <p id=\"P12\"> To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ) to perturbation profiles from the Connectivity Map (CMap) ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ). Six CNAs ( <em> IKBKAP </em> , <em> PIN1 </em> , <em> DNAJB1 </em> , <em> IL18 </em> , <em> NR2F6 </em> , <em> AKAP </em> ) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher’s exact p&lt;0.01) with clinical metadata. <em> IL-18 </em> deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (upper panel)), consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC ( <a aria-describedby=\"R175\" class=\"usa-link\" href=\"#R175\"> Timperi et al., 2017 </a> ). <em> NR2F6 </em> amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1F </a> (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade ( <a aria-describedby=\"R83\" class=\"usa-link\" href=\"#R83\"> Klepsch et al., 2018 </a> ). </p> <p id=\"P13\"> LSCC tumor DNA showed overall hypermethylation relative to NATs ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1G </a> ), with tumors separating into CIMP (CpG island methylator phenotype) clusters ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1H </a> ). Multi-omic data allowed the identification of “cascading” promoter methylation <em> cis </em> -effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR&lt;0.1) <em> cis </em> -effects in tumors ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), 20 were also differentially expressed (FDR&lt;0.01) between tumors and NATs ( <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> ). While most were newly associated with LSCC, methylation-driven expression of <em> FAM110A </em> , <em> PTGES3 </em> , <em> PLAU </em> and <em> SLC16A3 </em> ( <a aria-describedby=\"R35\" class=\"usa-link\" href=\"#R35\"> Faubert et al., 2017 </a> ; <a aria-describedby=\"R44\" class=\"usa-link\" href=\"#R44\"> Gao et al., 2018 </a> ; <a aria-describedby=\"R79\" class=\"usa-link\" href=\"#R79\"> Kikuchi et al., 2012 </a> ; <a aria-describedby=\"R105\" class=\"usa-link\" href=\"#R105\"> Liang et al., 2013 </a> ; <a aria-describedby=\"R136\" class=\"usa-link\" href=\"#R136\"> Payen et al., 2020 </a> ; <a aria-describedby=\"R157\" class=\"usa-link\" href=\"#R157\"> Showe et al., 2009 </a> ) has been implicated in lung and other cancers. </p> </section> <section id=\"S4\"> <h3 class=\"pmc_sec_title\"> Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched </h3> <p id=\"P14\"> We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the entire cohort ( <a class=\"usa-link\" href=\"#F1\"> Figure 1A </a> ). The five resulting multi-omic subtypes ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ) were named based on their predominant pathway associations and similarities to previously defined RNA clusters ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ). Significant associations between NMF subtypes and cohort metadata are presented in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . Two NMF subtypes, “Basal-Inclusive” (B-I) and “Epithelial to Mesenchymal Transition-enriched” (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni correction (Fisher’s p&lt;0.01, <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in <em> KEAP1 </em> , <em> CUL3 </em> and <em> NFE2L2 </em> genes and high-level amplification (GISTIC threshold value = 2) of <em> SOX2 </em> and <em> TP63 </em> ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> , <a class=\"usa-link\" href=\"#SD8\"> Table S2 </a> ) was labeled the “Classical” subtype, consistent with TCGA nomenclature and with a previous publication ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ). These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling ( <a class=\"usa-link\" href=\"#F2\"> Figure 2A </a> – <a class=\"usa-link\" href=\"#F2\"> B </a> ). A fourth NMF subtype was designated “Inflamed-Secretory” (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, “Proliferative-Primitive” (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a “mixed” subgroup, since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival ( <a class=\"usa-link\" href=\"#F2\"> Figure 2C </a> ), an observation also made in a recent proteogenomic analysis of glioblastoma ( <a aria-describedby=\"R187\" class=\"usa-link\" href=\"#R187\"> Wang et al., 2021 </a> ). The mixed class was associated in the current cohort with increased frequency of <em> SOX2 </em> amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. </p> <figure class=\"fig xbox font-sm\" id=\"F2\"> <h4 class=\"obj_head\"> Figure 2: LSCC Molecular Subtypes and Associations. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0003.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P381\"> A. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). </p> <p id=\"P382\"> B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. </p> <p id=\"P383\"> C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). </p> <p id=\"P384\"> D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. </p> <p id=\"P385\"> E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8×10 <sup> −14 </sup> ). </p> <p id=\"P386\"> F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5×10 <sup> −6 </sup> for PDGFRB and 2.7×10 <sup> −7 </sup> for ROR2. </p> <p id=\"P387\"> G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. </p> <p id=\"P388\"> See also <a class=\"usa-link\" href=\"#SD2\"> Figure S2 </a> and <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> – <a class=\"usa-link\" href=\"#SD10\"> 4 </a> </p> </figcaption> </figure> <p id=\"P15\"> We compared B-I ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ) and EMT-E ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 ( <a aria-describedby=\"R56\" class=\"usa-link\" href=\"#R56\"> Happel et al., 2009 </a> ) and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis ( <a aria-describedby=\"R64\" class=\"usa-link\" href=\"#R64\"> Hsu et al., 2020 </a> ), and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance ( <a aria-describedby=\"R67\" class=\"usa-link\" href=\"#R67\"> Hubaux et al., 2015 </a> ; <a aria-describedby=\"R100\" class=\"usa-link\" href=\"#R100\"> Lewandowski and Piwnica-Worms, 2014 </a> ). Transcription activator PBX3, which is upregulated only at the protein level ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2B </a> ), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, ( <a aria-describedby=\"R94\" class=\"usa-link\" href=\"#R94\"> Lamprecht et al., 2018 </a> ) including in laryngeal squamous cell carcinoma ( <a aria-describedby=\"R198\" class=\"usa-link\" href=\"#R198\"> Wu et al., 2020 </a> ). Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2D </a> ), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2E </a> ) and with the xCell-based deconvolution signature ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2F </a> ). This correlation was not observed in paired NATs, suggesting a tumor-specific role. </p> <p id=\"P16\"> In addition to a strong EMT signature ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ), the EMT-E subtype showed upregulated VCAN ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ), a tumor-promoting target of SNAIL ( <a aria-describedby=\"R207\" class=\"usa-link\" href=\"#R207\"> Zhang et al., 2019a </a> ) expressed in cancer-associated fibroblasts (CAFs) ( <a aria-describedby=\"R24\" class=\"usa-link\" href=\"#R24\"> Chida et al., 2016 </a> ), and FHL3, a protein that stabilizes EMT-associated transcription factors ( <a aria-describedby=\"R104\" class=\"usa-link\" href=\"#R104\"> Li et al., 2020 </a> ) ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2C </a> ). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in <a class=\"usa-link\" href=\"#F2\"> Figure 2D </a> – <a class=\"usa-link\" href=\"#F2\"> E </a> , coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a “fibroblast proliferation” signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker α-smooth muscle actin (α-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2G </a> ), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either CK or α-SMA alone. TGFβ is a well-established and potent inducer of EMT ( <a aria-describedby=\"R166\" class=\"usa-link\" href=\"#R166\"> Su et al., 2020 </a> ), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFβ inhibitors ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2H </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4H </a> ). </p> <p id=\"P17\"> Integrating our global proteome data with a prior LSCC dataset ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> – <a class=\"usa-link\" href=\"#SD2\"> J </a> ). </p> </section> <section id=\"S5\"> <h3 class=\"pmc_sec_title\"> NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. </h3> <p id=\"P18\"> Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p &lt;0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E ( <a class=\"usa-link\" href=\"#F2\"> Figure 2F </a> ), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state ( <a class=\"usa-link\" href=\"#F2\"> Figure 2G </a> ) ( <a aria-describedby=\"R60\" class=\"usa-link\" href=\"#R60\"> Herreño et al., 2019 </a> ; <a aria-describedby=\"R130\" class=\"usa-link\" href=\"#R130\"> Niu et al., 2012 </a> ). As summarized in <a class=\"usa-link\" href=\"#SD2\"> Figure S2K </a> , we found phosphosite-based evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR ( <a aria-describedby=\"R169\" class=\"usa-link\" href=\"#R169\"> Tam et al., 2013 </a> ), both targetable ( <a aria-describedby=\"R150\" class=\"usa-link\" href=\"#R150\"> Roskoski 2018 </a> ; <a aria-describedby=\"R25\" class=\"usa-link\" href=\"#R25\"> Debebe and Rathmell 2015 </a> ) and recently implicated in the progression from a hybrid E/M to a highly mesenchymal state ( <a aria-describedby=\"R87\" class=\"usa-link\" href=\"#R87\"> Kröger et al., 2019 </a> ). Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. </p> </section> <section id=\"S6\"> <h3 class=\"pmc_sec_title\"> Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable </h3> <p id=\"P19\"> We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs ( <em> cis </em> -effects) and on a set of cancer associated genes ( <em> trans </em> -effects) ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ). Significant (FDR&lt;0.05) <em> Cis/Trans </em> pairs are shown in <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> – <a class=\"usa-link\" href=\"#F3\"> D </a> . <em> RB1 </em> mutated tumors had upregulated cell cycle-related protein expression. <em> NOTCH1 </em> mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK ( <a aria-describedby=\"R69\" class=\"usa-link\" href=\"#R69\"> Ishitani et al., 2010 </a> ) and GSK3B ( <a aria-describedby=\"R38\" class=\"usa-link\" href=\"#R38\"> Foltz et al., 2002 </a> ; <a aria-describedby=\"R71\" class=\"usa-link\" href=\"#R71\"> Jin et al., 2009 </a> ; <a aria-describedby=\"R210\" class=\"usa-link\" href=\"#R210\"> Zheng and Conner, 2018 </a> ) inhibit NOTCH1, recently described as a tumor suppressor in LSCC ( <a aria-describedby=\"R160\" class=\"usa-link\" href=\"#R160\"> Sinicropi-Yao et al., 2019 </a> ), suggesting that both might inhibit NOTCH1 downstream signaling in tumors with <em> NOTCH1 </em> mutations. <em> ARID1A </em> is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden ( <a aria-describedby=\"R156\" class=\"usa-link\" href=\"#R156\"> Shen et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3A </a> ) and led to protein downregulation, associated with worse overall survival in NSCLC ( <a aria-describedby=\"R68\" class=\"usa-link\" href=\"#R68\"> Hung et al., 2020 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F3\"> <h4 class=\"obj_head\"> Figure 3: Impact of Somatic Mutations on Proteogenomic Features. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0004.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F3/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P389\"> A. Significant (Wilcoxon FDR&lt;0.05) <em> cis </em> - (circles) and <em> trans </em> -effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green. </p> <p id=\"P390\"> B. Similar to panel A but showing phosphosites. </p> <p id=\"P391\"> C. Similar to panel A but showing acetylsites. </p> <p id=\"P392\"> D. Similar to panel A but showing ubiquitylsites. </p> <p id=\"P393\"> E. CNA data for <em> CDKN2A </em> and <em> RB1 </em> was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CDKN2A </em> genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these <em> CDKN2A/p16 </em> aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of <em> CCND1–3 </em> , <em> CDK4 </em> , and <em> CDK6 </em> were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of <em> CDKN2A </em> are not included. </p> <p id=\"P394\"> F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in <em> KEAP1 </em> , <em> CUL3 </em> , or <em> NFE2L2 </em> ) versus NRF2 WT tumors (no NRF2 pathway aberration). </p> <p id=\"P395\"> G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype. </p> <p id=\"P396\"> H. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. </p> <p id=\"P397\"> I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR&lt;0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. </p> <p id=\"P398\"> See also <a class=\"usa-link\" href=\"#SD3\"> Figure S3 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P20\"> Paradoxically, mutations in <em> CDKN2A, </em> proposed as a potential biomarker for CDK4/6 inhibitors ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ), resulted in increased RNA expression ( <a class=\"usa-link\" href=\"#F3\"> Figure 3A </a> ). We analyzed the effects of genetic and epigenetic alterations of <em> CDKN2A </em> on both its major isoforms, p16INK4a (p16) and p14ARF (p14) ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). As expected, its homozygous deletion resulted in the loss of expression of both isoforms ( <a class=\"usa-link\" href=\"#F3\"> Figure 3E </a> ). However, only eight of the 58 <em> CDKN2A </em> wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with <em> CDKN2A </em> /p16 missense mutations (log2(FPKM+1)&gt;2). Many of those <em> CDKN2A </em> /p16 mutations were unlikely to result in nonsense-mediated decay ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3B </a> ), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had <em> RB1 </em> mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes ( <a aria-describedby=\"R85\" class=\"usa-link\" href=\"#R85\"> Knudsen et al., 2020 </a> ). This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ), there was no association between the amplification of <em> CCND2 </em> , <em> CCND3 </em> , <em> CDK4 </em> , or <em> CDK6 </em> and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ). <em> CCND1 </em> amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3C </a> ). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in <em> CDKN2A </em> and copy number alterations in <em> CCND1 </em> and <em> CDKN2A </em> ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib ( <a aria-describedby=\"R135\" class=\"usa-link\" href=\"#R135\"> Patnaik et al., 2016 </a> ) and palbociclib ( <a aria-describedby=\"R2\" class=\"usa-link\" href=\"#R2\"> Ahn et al., 2020 </a> ; <a aria-describedby=\"R31\" class=\"usa-link\" href=\"#R31\"> Edelman et al., 2019 </a> ; <a aria-describedby=\"R122\" class=\"usa-link\" href=\"#R122\"> Middleton et al., 2020 </a> ). While we found that <em> CCND1 </em> amplification resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with <em> CCND1 </em> amplification provides a potential explanation for some patients’ lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 inhibition. </p> </section> <section id=\"S7\"> <h3 class=\"pmc_sec_title\"> NRF2 pathway activation in tumors with and without NRF2 pathway mutations </h3> <p id=\"P21\"> LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> . NFE2L2 phosphorylation was observed in samples with any of these mutations ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3E </a> ). LSCC tumors were annotated as “NRF2 pathway mutated” (n=33; with either one hit [n=28] or two hits [n=5] in <em> NFE2L2 </em> , <em> KEAP1 </em> , and <em> CUL3 </em> ), or “NRF2 WT” (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes ( <a class=\"usa-link\" href=\"#F3\"> Figure 3F </a> , <a class=\"usa-link\" href=\"#SD3\"> Figure S3F </a> – <a class=\"usa-link\" href=\"#SD3\"> G </a> , <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3H </a> ). Using the proteogenomic signature ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) that defined <em> NFE2L2 </em> mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype ( <a class=\"usa-link\" href=\"#F3\"> Figure 3G </a> ). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein ( <a class=\"usa-link\" href=\"#F3\"> Figure 3H </a> ) and PTM-SEA-derived activity ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) were significantly upregulated (FDR&lt;0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes ( <a aria-describedby=\"R70\" class=\"usa-link\" href=\"#R70\"> Jimenez-Blasco et al., 2015 </a> ) and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ), <em> KEAP1 </em> mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. </p> </section> <section id=\"S8\"> <h3 class=\"pmc_sec_title\"> Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC </h3> <p id=\"P22\"> Chromosome 3q, which harbors key squamous differentiation markers <em> SOX2 </em> and <em> TP63 </em> ( <a aria-describedby=\"R143\" class=\"usa-link\" href=\"#R143\"> Qian and Massion, 2008 </a> ), showed the most dramatic arm-level amplification in this cohort ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4A </a> ). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). LSCC cell lines amplified for <em> TP63 </em> and <em> SOX2 </em> were highly dependent upon them, supporting oncogene addiction ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F4\"> <h4 class=\"obj_head\"> Figure 4: Proteogenomic Impact of Chromosome 3q Amplification. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0005.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F4/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P399\"> A. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. </p> <p id=\"P400\"> B. Frequency distribution of ΔNp63α RNA expression in tumors and NATs. </p> <p id=\"P401\"> C. Differential protein expression in samples classified as ΔNp63α-low vs -high based on ΔNp63α transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. </p> <p id=\"P402\"> D. Differential expression of microRNAs in ΔNp63α-low vs -high samples. </p> <p id=\"P403\"> E. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. </p> <p id=\"P404\"> F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. </p> <p id=\"P405\"> G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3×10 <sup> −08 </sup> , Correlation = −0.53) and DNA replication (bottom, p=1.1×10 <sup> −06 </sup> , Correlation= 0.47) scores. </p> <p id=\"P406\"> H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR&lt;0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. </p> <p id=\"P407\"> I. Pearson correlation (p= 5.2×10 <sup> −07 </sup> , Correlation = −0.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES. </p> <p id=\"P408\"> See also <a class=\"usa-link\" href=\"#SD4\"> Figure S4 </a> and <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> </p> </figcaption> </figure> <p id=\"P23\"> As previously reported ( <a aria-describedby=\"R20\" class=\"usa-link\" href=\"#R20\"> Campbell et al., 2018 </a> ; <a aria-describedby=\"R113\" class=\"usa-link\" href=\"#R113\"> Lo Iacono et al., 2011 </a> ), ΔNp63α was the dominant TP63 isoform in this dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4C </a> ) and was highly correlated with TP63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4D </a> ). Its amplification was associated with improved survival in the TCGA dataset ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4E </a> ). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 “Np63-low” samples histologically confirmed to be LSCC showed no elevation at RNA or protein levels ( <a class=\"usa-link\" href=\"#F4\"> Figure 4B </a> ). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group ( <a class=\"usa-link\" href=\"#F4\"> Figure 4C </a> ). Notably, a substantial number of LSCC cell lines also had low Np63 expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4F </a> ), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Survivin promotes cell proliferation and G2M checkpoints ( <a aria-describedby=\"R193\" class=\"usa-link\" href=\"#R193\"> Wheatley and Altieri, 2019 </a> ), evident in Np63-low samples ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4H </a> ). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in NSCLC ( <a aria-describedby=\"R48\" class=\"usa-link\" href=\"#R48\"> Giaccone et al., 2009 </a> ; <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ; <a aria-describedby=\"R101\" class=\"usa-link\" href=\"#R101\"> Li et al., 2019a </a> ), but have shown modest or no improvement ( <a aria-describedby=\"R77\" class=\"usa-link\" href=\"#R77\"> Kelly et al., 2013 </a> ). Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin inhibition. </p> <p id=\"P24\"> Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC ( <a aria-describedby=\"R97\" class=\"usa-link\" href=\"#R97\"> Lebanony et al., 2009 </a> ) ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4I </a> ), was associated with decreased promoter DNA methylation ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4J </a> ) and elevation of ΔNp63α, a transcriptional regulator of this miRNA ( <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4D </a> , <a class=\"usa-link\" href=\"#SD4\"> Figure S4K </a> ). As expected, the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN ( <a aria-describedby=\"R18\" class=\"usa-link\" href=\"#R18\"> Cai et al., 2013 </a> ; <a aria-describedby=\"R53\" class=\"usa-link\" href=\"#R53\"> Gregory et al., 2008 </a> ; <a aria-describedby=\"R172\" class=\"usa-link\" href=\"#R172\"> Tellez et al., 2011 </a> ; <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ). TP73 positively correlated with miR-205 at the transcript ( <a class=\"usa-link\" href=\"#F4\"> Figure 4E </a> ) and still more strongly at the protein level ( <a class=\"usa-link\" href=\"#F4\"> Figure 4F </a> ) and with TP63 protein but not RNA ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4L </a> ). TP73 overexpression mediates ΔNp63α-dependent cell survival in squamous cell carcinoma ( <a aria-describedby=\"R28\" class=\"usa-link\" href=\"#R28\"> DeYoung et al., 2006 </a> ), suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC ( <a aria-describedby=\"R126\" class=\"usa-link\" href=\"#R126\"> Natan and Joerger, 2012 </a> ). Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication ( <a class=\"usa-link\" href=\"#F4\"> Figure 4G </a> ). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in previous studies ( <a aria-describedby=\"R110\" class=\"usa-link\" href=\"#R110\"> Lindsay et al., 2011 </a> ; <a aria-describedby=\"R164\" class=\"usa-link\" href=\"#R164\"> Srivastava et al., 2018 </a> ; <a aria-describedby=\"R176\" class=\"usa-link\" href=\"#R176\"> Tran et al., 2013 </a> ). Of note, although ΔNp63α and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, ΔNp63α, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype ( <a aria-describedby=\"R183\" class=\"usa-link\" href=\"#R183\"> Vosgha et al., 2018 </a> ), and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. </p> <p id=\"P25\"> <em> SOX2 </em> , a recurrently amplified squamous cell marker often co-amplified with <em> TP63 </em> , was also overexpressed in tumors ( <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> ). <em> SOX2 </em> is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression ( <a aria-describedby=\"R206\" class=\"usa-link\" href=\"#R206\"> Zhang et al., 2013 </a> ) and its inhibition in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT04350463\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT04350463 </a> ). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer ( <a aria-describedby=\"R144\" class=\"usa-link\" href=\"#R144\"> Ramadoss et al., 2017 </a> ), and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression ( <a aria-describedby=\"R203\" class=\"usa-link\" href=\"#R203\"> Yomtoubian et al., 2020 </a> ). EZH2 dependency was observed in SOX2 amplified LSCC cell lines ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> ). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ) suggest stemness-like features in these tumors ( <a aria-describedby=\"R78\" class=\"usa-link\" href=\"#R78\"> Keysar et al., 2017 </a> ; <a aria-describedby=\"R112\" class=\"usa-link\" href=\"#R112\"> Liu et al., 2016 </a> ; <a aria-describedby=\"R134\" class=\"usa-link\" href=\"#R134\"> Patel et al., 2008 </a> ). Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 ( <a class=\"usa-link\" href=\"#F4\"> Figure 4H </a> ), and negative correlation was seen between SOX2 and JAK-STAT signaling ( <a class=\"usa-link\" href=\"#F4\"> Figure 4I </a> ). </p> </section> <section id=\"S9\"> <h3 class=\"pmc_sec_title\"> Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism </h3> <p id=\"P26\"> Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ) associated with country of origin and ethnicity (Fisher’s exact p&lt;0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5A </a> – <a class=\"usa-link\" href=\"#SD14\"> B </a> , <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F5\"> <h4 class=\"obj_head\"> Figure 5: Ubiquitylation landscape in LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0006.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F5/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P409\"> A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR&lt;0.01). Enriched pathways and molecular and clinical annotations are indicated. </p> <p id=\"P410\"> B. Number of K-GG sites showing significant correlations (FDR&lt; 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. </p> <p id=\"P411\"> C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. </p> <p id=\"P412\"> D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal−Wallis, p = 2.8×10 <sup> −05 </sup> ) association is seen between HERC5 abundance and immune subtypes. </p> <p id=\"P413\"> E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). </p> <p id=\"P414\"> F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal−Wallis p-values are indicated in the respective plots. </p> <p id=\"P415\"> G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. </p> <p id=\"P416\"> H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)&gt;1 or Hypo: &lt;−1, FDR&lt;0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. “k” represents modified lysine. </p> <p id=\"P417\"> I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p&gt;0.05. </p> <p id=\"P418\"> See also <a class=\"usa-link\" href=\"#SD14\"> Figure S5 </a> and <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> </p> </figcaption> </figure> <p id=\"P27\"> To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in <a class=\"usa-link\" href=\"#F5\"> Figure 5B </a> and <a class=\"usa-link\" href=\"#SD14\"> Figure S5C </a> . HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in humans ( <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); hence the numerous positively correlated Ub/UbL sites (n=474; FDR&lt;0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 abundance ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5D </a> ). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5E </a> ). We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 ( <a class=\"usa-link\" href=\"#F5\"> Figure 5C </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5F </a> ). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5G </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5H </a> ). ISG15 modifications have been described in these glycolytic enzymes ( <a aria-describedby=\"R3\" class=\"usa-link\" href=\"#R3\"> Albert et al., 2018 </a> ; <a aria-describedby=\"R49\" class=\"usa-link\" href=\"#R49\"> Giannakopoulos et al., 2005 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ), and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis ( <a aria-describedby=\"R4\" class=\"usa-link\" href=\"#R4\"> Alcalá et al., 2020 </a> ). ISGylation can be induced by interferon in cancer ( <a aria-describedby=\"R171\" class=\"usa-link\" href=\"#R171\"> Tecalco-Cruz and Cruz-Ramos, 2018 </a> ; <a aria-describedby=\"R197\" class=\"usa-link\" href=\"#R197\"> Wong et al., 2006 </a> ); accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores ( <a class=\"usa-link\" href=\"#SD14\"> Figure S5I </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5J </a> ), and HERC5 was increased in tumors of the IFN-gamma dominant subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5D </a> ). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion ( <a aria-describedby=\"R208\" class=\"usa-link\" href=\"#R208\"> Zhang et al., 2019b </a> ). Collectively, our data supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. </p> <p id=\"P28\"> This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ) to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p&lt;0.05) ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos ( <a class=\"usa-link\" href=\"#SD11\"> Table S5 </a> ), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket ( <a aria-describedby=\"R188\" class=\"usa-link\" href=\"#R188\"> Wang et al., 2015 </a> ; <a aria-describedby=\"R209\" class=\"usa-link\" href=\"#R209\"> Zhao et al., 2014 </a> )( <a aria-describedby=\"R200\" class=\"usa-link\" href=\"#R200\"> Xiong et al., 2011 </a> ) ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator ( <a class=\"usa-link\" href=\"#F5\"> Figures 5F </a> ). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37) ( <a aria-describedby=\"R11\" class=\"usa-link\" href=\"#R11\"> Bando et al., 2005 </a> ; <a aria-describedby=\"R201\" class=\"usa-link\" href=\"#R201\"> Yang and Lu, 2013 </a> ). Upregulation of a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs ( <a aria-describedby=\"R26\" class=\"usa-link\" href=\"#R26\"> DeBerardinis and Chandel, 2016 </a> ; <a aria-describedby=\"R61\" class=\"usa-link\" href=\"#R61\"> Hitosugi and Chen, 2014 </a> ). While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. </p> <p id=\"P29\"> Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR&lt;0.01). The TXN1 activator TXNRD1 ( <a aria-describedby=\"R17\" class=\"usa-link\" href=\"#R17\"> Cadenas et al., 2010 </a> ) was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1 ( <a aria-describedby=\"R124\" class=\"usa-link\" href=\"#R124\"> Morrison et al., 2014 </a> ), was decreased ( <a class=\"usa-link\" href=\"#F5\"> Figure 5F </a> ). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p&lt;2.2×10 <sup> −16 </sup> ), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial cancers ( <a aria-describedby=\"R42\" class=\"usa-link\" href=\"#R42\"> Fu et al., 2017 </a> ), including NSCLC ( <a aria-describedby=\"R36\" class=\"usa-link\" href=\"#R36\"> Fernandes et al., 2009 </a> ). The small proportion of glucose directed to the tumor mitochondria ( <a aria-describedby=\"R34\" class=\"usa-link\" href=\"#R34\"> Fan et al., 2019 </a> ) and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis) ( <a aria-describedby=\"R9\" class=\"usa-link\" href=\"#R9\"> Baeza et al., 2015 </a> ; <a aria-describedby=\"R43\" class=\"usa-link\" href=\"#R43\"> Gandhi and Das, 2019 </a> ; <a aria-describedby=\"R58\" class=\"usa-link\" href=\"#R58\"> Hebert et al., 2013 </a> ). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors ( <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> , <a class=\"usa-link\" href=\"#SD14\"> Figure S5K </a> ). PTM-based modulation of these key metabolic and ROS pathways is summarized in <a class=\"usa-link\" href=\"#F5\"> Figure 5G </a> . </p> <p id=\"P30\"> As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. <a class=\"usa-link\" href=\"#F5\"> Figure 5H </a> shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype ( <a class=\"usa-link\" href=\"#F5\"> Figure 5I </a> ), named in part for its generally high proliferation ( <a class=\"usa-link\" href=\"#F2\"> Figure 2B </a> ). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. </p> </section> <section id=\"S10\"> <h3 class=\"pmc_sec_title\"> Immune landscape and regulation in LSCC </h3> <p id=\"P31\"> Consensus clustering based on the xCell ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) signatures ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> C </a> ), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher’s exact p=6.371×10 <sup> −09 </sup> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> ). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses ( <a class=\"usa-link\" href=\"#F6\"> Figures 6A </a> , <a class=\"usa-link\" href=\"#F6\"> C </a> , <a class=\"usa-link\" href=\"#SD5\"> S6C </a> , <a class=\"usa-link\" href=\"#SD5\"> D </a> ), and showed upregulation of immune-related pathways ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Notably, NATs did not display consonant immune signaling ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6B </a> ). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> – <a class=\"usa-link\" href=\"#F6\"> B </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6A </a> ). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6B </a> ). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> , <a class=\"usa-link\" href=\"#SD5\"> Figure S6E </a> ) </p> <figure class=\"fig xbox font-sm\" id=\"F6\"> <h4 class=\"obj_head\"> Figure 6: Immune Landscape of LSCC. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0007.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F6/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P419\"> A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). </p> <p id=\"P420\"> B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. </p> <p id=\"P421\"> C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types’ distributions observed across the different immune clusters. </p> <p id=\"P422\"> D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR &lt; 0.1). </p> <p id=\"P423\"> E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. </p> <p id=\"P424\"> F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P425\"> G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) and the third pathway scores based on global protein abundance. </p> <p id=\"P426\"> H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. </p> <p id=\"P427\"> I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also <a class=\"usa-link\" href=\"#SD5\"> Figure S6 </a> and <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> </p> </figcaption> </figure> <p id=\"P32\"> Rho GTPase signaling was upregulated in Hot tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation ( <a aria-describedby=\"R45\" class=\"usa-link\" href=\"#R45\"> Garcia-Mata et al., 2011 </a> ), was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial enzymes ( <a class=\"usa-link\" href=\"#F6\"> Figure 6D </a> ). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly ( <a aria-describedby=\"R90\" class=\"usa-link\" href=\"#R90\"> Kuhlmann et al., 2016a </a> ) by activating RhoA ( <a aria-describedby=\"R91\" class=\"usa-link\" href=\"#R91\"> Kuhlmann et al., 2016b </a> ). In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low ( <a aria-describedby=\"R52\" class=\"usa-link\" href=\"#R52\"> Gorvel et al., 1998 </a> ), and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead ( <a aria-describedby=\"R123\" class=\"usa-link\" href=\"#R123\"> Moissoglu et al., 2009 </a> ). In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators ( <a class=\"usa-link\" href=\"#F6\"> Figures 6F </a> , <a class=\"usa-link\" href=\"#SD5\"> S6F </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Notably, these proteins are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells ( <a class=\"usa-link\" href=\"#SD5\"> Figure S6G </a> ), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas ( <a aria-describedby=\"R84\" class=\"usa-link\" href=\"#R84\"> von Klot et al., 2017 </a> ). Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ) and immune-enriched networks ( <a aria-describedby=\"R140\" class=\"usa-link\" href=\"#R140\"> Peters et al., 2017 </a> ). </p> <p id=\"P33\"> We explored the immunological associations of the RTK CBPE scores described above ( <a class=\"usa-link\" href=\"#F2\"> Figure 2I </a> , <a class=\"usa-link\" href=\"#F2\"> J </a> ) in LSCC tumors ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster ( <a class=\"usa-link\" href=\"#F6\"> Figure 6H </a> ) and consequently with immune-related pathways and cell types ( <a class=\"usa-link\" href=\"#F6\"> Figure 6G </a> ). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103 ( <a aria-describedby=\"R19\" class=\"usa-link\" href=\"#R19\"> Cammer et al., 2009 </a> ; <a aria-describedby=\"R30\" class=\"usa-link\" href=\"#R30\"> Dovas et al., 2009 </a> ; <a aria-describedby=\"R125\" class=\"usa-link\" href=\"#R125\"> Mouchemore et al., 2013 </a> ), and upregulation of AKAP13, among other immune-related proteins ( <a class=\"usa-link\" href=\"#F6\"> Figure 6I </a> ). Tumor-associated macrophages (( <a aria-describedby=\"R23\" class=\"usa-link\" href=\"#R23\"> Cassetta and Pollard, 2018 </a> ), CSF1R signaling ( <a aria-describedby=\"R21\" class=\"usa-link\" href=\"#R21\"> Cannarile et al., 2017 </a> ), and AKAP13 ( <a aria-describedby=\"R29\" class=\"usa-link\" href=\"#R29\"> Diviani et al., 2016 </a> ) have each been proposed as anti-cancer targets. </p> </section> <section id=\"S11\"> <h3 class=\"pmc_sec_title\"> Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment </h3> <p id=\"P34\"> Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment ( <a class=\"usa-link\" href=\"#F7\"> Figure 7A </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). The top 50 of 502 differential proteins are shown in <a class=\"usa-link\" href=\"#SD6\"> Figure S7A </a> . In addition to 206 upregulated oncogenic proteins (OncoKB, <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7B </a> ), with the I-S samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ) to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS ( <a class=\"usa-link\" href=\"#F7\"> Figure 7B </a> , <a class=\"usa-link\" href=\"#SD6\"> Figure S7C </a> ). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/ </a> ), suggesting critical roles in key cellular transformation and proliferation processes ( <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"F7\"> <h4 class=\"obj_head\"> Figure 7: Proteomic Features Related to Diagnosis, Prognosis, or Treatment. </h4> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> <a class=\"tileshop\" href=\"https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8475722_nihms-1730863-f0008.jpg\" target=\"_blank\"> </a> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/F7/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p id=\"P428\"> A. Differentially expressed proteins between tumors and NATs. </p> <p id=\"P429\"> B. Significantly increased proteins (larger font indicates &gt;4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. </p> <p id=\"P430\"> C. Genetic dependencies of 502 proteins (log2 FC &gt;2, FDR&lt;0.01 and NAs &lt;50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. </p> <p id=\"P431\"> D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. </p> <p id=\"P432\"> E. EGFR protein and tyrosine phosphorylation levels compared to <em> EGFR </em> copy number and an EGFR activity score (PROGENy). </p> <p id=\"P433\"> F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p&lt;0.05 </p> <p id=\"P434\"> G. GO Biological Process enrichment for proteins with increased phosphorylation in <em> EGFR </em> amplified samples compared to non-amplified samples. </p> <p id=\"P435\"> H. Summary roadmap figure partitioned into five major categories, indicated by different colors. </p> <p id=\"P436\"> See also <a class=\"usa-link\" href=\"#SD6\"> Figure S7 </a> and <a class=\"usa-link\" href=\"#SD13\"> Table S7 </a> </p> </figcaption> </figure> <p id=\"P35\"> There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell type of origin ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7D </a> ). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores ( <a class=\"usa-link\" href=\"#F7\"> Figure 7D </a> ). Immune-related deletions were less prominent in HNSCC and especially LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7E </a> ), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7F </a> ), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD ( <a class=\"usa-link\" href=\"#SD6\"> Figure S7G </a> ), although LUAD had many activating <em> EGFR </em> mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High <em> EGFR </em> amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy scoring ( <a class=\"usa-link\" href=\"#F7\"> Figure 7E </a> ) ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors ( <a class=\"usa-link\" href=\"#F7\"> Figure 7F </a> ). Furthermore, <em> EGFR </em> amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion ( <a class=\"usa-link\" href=\"#F7\"> Figure 7G </a> ). These findings and similar results recently reported in HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. <a class=\"usa-link\" href=\"#F7\"> Figure 7H </a> graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. </p> </section> </section> <section id=\"S12\"> <h2 class=\"pmc_sec_title\"> Discussion </h2> <p id=\"P36\"> In this study, we provide detailed proteogenomic profiles of treatment-naïve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. </p> <p id=\"P37\"> Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses ( <a aria-describedby=\"R57\" class=\"usa-link\" href=\"#R57\"> Haslam and Prasad, 2019 </a> ; <a aria-describedby=\"R59\" class=\"usa-link\" href=\"#R59\"> Herbst et al., 2018 </a> ); numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the downstream effects of <em> CDKN2A </em> mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. </p> <section id=\"S13\"> <h3 class=\"pmc_sec_title\"> Limitations of the study </h3> <p id=\"P38\"> Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity ( <a aria-describedby=\"R139\" class=\"usa-link\" href=\"#R139\"> Peng et al., 2020 </a> ; <a aria-describedby=\"R151\" class=\"usa-link\" href=\"#R151\"> Satpathy et al., 2020 </a> ), such as pairing microdissection with mass spectrometry ( <a aria-describedby=\"R99\" class=\"usa-link\" href=\"#R99\"> Le Large et al., 2020 </a> ) or using mass cytometry ( <a aria-describedby=\"R163\" class=\"usa-link\" href=\"#R163\"> Spitzer and Nolan, 2016 </a> ) will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the understanding and treatment of LSCC. </p> </section> </section> <section id=\"S14\"> <h2 class=\"pmc_sec_title\"> STAR★METHODS </h2> <section id=\"S16\"> <h3 class=\"pmc_sec_title\"> RESOURCE AVAILABILITY </h3> <section id=\"S17\"> <h4 class=\"pmc_sec_title\"> Lead contact </h4> <p id=\"P39\"> Further information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette ( <span> gillette@broadinstitute.org </span> ) </p> </section> <section id=\"S18\"> <h4 class=\"pmc_sec_title\"> Material availability </h4> <p id=\"P40\"> This study did not generate new unique reagents. </p> </section> <section id=\"S19\"> <h4 class=\"pmc_sec_title\"> Data and Code Availability </h4> <section id=\"S20\"> <h5 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h5> <ul class=\"list\" id=\"L2\"> <li> <p id=\"P41\"> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p id=\"P42\"> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"S21\"> <h5 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h5> <ul class=\"list\" id=\"L4\"> <li> <p id=\"P43\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p id=\"P44\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"S22\"> <h5 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h5> <ul class=\"list\" id=\"L6\"> <li> <p id=\"P45\"> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p id=\"P46\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p id=\"P47\"> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p id=\"P48\"> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p id=\"P49\"> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p id=\"P50\"> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"T1\"> <h6 class=\"obj_head\"> KEY RESOURCES TABLE. </h6> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/T1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> <section id=\"S23\"> <h3 class=\"pmc_sec_title\"> EXPERIMENTAL MODEL AND SUBJECT DETAILS </h3> <section id=\"S24\"> <h4 class=\"pmc_sec_title\"> Human Subjects </h4> <p id=\"P51\"> A total of 113 participants (90 males, 23 females, 40–88 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40–88. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC ( <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/ </a> ). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&amp;E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> </section> <section id=\"S25\"> <h3 class=\"pmc_sec_title\"> METHOD DETAILS </h3> <section id=\"S26\"> <h4 class=\"pmc_sec_title\"> Specimen Acquisition </h4> <p id=\"P52\"> The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than −140°C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. </p> </section> <section id=\"S27\"> <h4 class=\"pmc_sec_title\"> Sequencing sample preparation </h4> <p id=\"P53\"> Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol using Qiagen’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3–5mL) to serve as matched normal reference material. The Qubit™ dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification. </p> <section id=\"S28\"> <h5 class=\"pmc_sec_title\"> Whole exome Sequencing (WES) </h5> <section id=\"S29\"> <h6 class=\"pmc_sec_title\"> Library construction and Hybrid Selection </h6> <p id=\"P54\"> Library construction was performed as described in ( <a aria-describedby=\"R37\" class=\"usa-link\" href=\"#R37\"> Fisher et al., 2011 </a> ), with the following modifications: initial genomic DNA input into shearing was reduced from 3μg to 20–250ng in 50μL of solution. For adapter ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. </p> </section> <section id=\"S30\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P55\"> After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. </p> </section> </section> <section id=\"S31\"> <h5 class=\"pmc_sec_title\"> Whole genome sequencing (WGS) </h5> <section id=\"S32\"> <h6 class=\"pmc_sec_title\"> Cluster Amplification and Sequencing </h6> <p id=\"P56\"> An aliquot of genomic DNA (350ng in 50μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into 24-plexes. </p> <p id=\"P57\"> Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging. </p> </section> </section> </section> <section id=\"S33\"> <h4 class=\"pmc_sec_title\"> Array Based Methylation Analysis </h4> <p id=\"P58\"> The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt;850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. </p> </section> <section id=\"S34\"> <h4 class=\"pmc_sec_title\"> RNA and miRNA sequencing </h4> <section id=\"S35\"> <h5 class=\"pmc_sec_title\"> Quality Assurance and Control of RNA Analytes </h5> <p id=\"P59\"> All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt;7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer’s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. </p> </section> <section id=\"S36\"> <h5 class=\"pmc_sec_title\"> Total RNA Sequencing </h5> <p id=\"P60\"> Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. </p> </section> <section id=\"S37\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Library Construction </h5> <p id=\"P61\"> miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer’s instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). </p> </section> <section id=\"S38\"> <h5 class=\"pmc_sec_title\"> miRNA Sequencing </h5> <p id=\"P62\"> Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. </p> </section> </section> <section id=\"S39\"> <h4 class=\"pmc_sec_title\"> Mass Spectrometry methods </h4> <p id=\"P63\"> The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Acetyl-enrichment was performed as described before ( <a aria-describedby=\"R89\" class=\"usa-link\" href=\"#R89\"> Krug et al., 2020 </a> ) with modifications as indicated below. </p> <section id=\"S40\"> <h5 class=\"pmc_sec_title\"> Protein Extraction and Tryptic Digestion </h5> <p id=\"P64\"> Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 μg/mL aprotinin (Sigma, A6103), 10 μg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129–02541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. </p> </section> <section id=\"S41\"> <h5 class=\"pmc_sec_title\"> Common Reference (CR) Pool and Plex layout </h5> <section id=\"S42\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, and Acetyprotome: </h6> <p id=\"P65\"> The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. A common physical, rather than <em> in silico </em> reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data analyses. </p> <p id=\"P66\"> 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5 <sup> th </sup> tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. </p> <p id=\"P67\"> To ensure capacity for additional samples or experiments given a target input of 300 μg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 μg peptide / sample for individual sample analysis, an additional amount of 120 μg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 μg aliquots, dried down, and frozen at −80°C until use. </p> </section> <section id=\"S43\"> <h6 class=\"pmc_sec_title\"> Ubiquitylproteome: </h6> <p id=\"P68\"> 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: </p> <ol class=\"list\" id=\"L8\"> <li> <p id=\"P69\"> 104/147 samples included in ubiquitylproteome study and made full CR contribution, &gt;1000ug available. </p> </li> <li> <p id=\"P70\"> 42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug −1000 ug available. </p> </li> <li> <p id=\"P71\"> 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, &lt;750 ug available. </p> </li> </ol> <p id=\"P72\"> CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR’s proportion of 50:50), and aliquoted into 750 ug aliquots. </p> <p id=\"P73\"> In the ubiquitylproteome experimental design, 15 11-plex experiments contained <strong> 87 </strong> unique tumor samples in the C channels and <strong> 60 </strong> normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11 <sup> th </sup> channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). </p> </section> </section> <section id=\"S44\"> <h5 class=\"pmc_sec_title\"> TMT-11 Labeling of Peptides </h5> <section id=\"S45\"> <h6 class=\"pmc_sec_title\"> Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P74\"> 300 μg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a “reduced labeling” approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 μg peptide aliquot of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 μL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 μg aliquot from each sample and analyzing 0.25 μg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples were taken out, an additional 4 μg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/− 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample’s contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 × 300 μg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). </p> </section> </section> <section id=\"S46\"> <h5 class=\"pmc_sec_title\"> Peptide Fractionation </h5> <section id=\"S47\"> <h6 class=\"pmc_sec_title\"> For Proteome, Phosphoproteome, Acetylproteome: </h6> <p id=\"P75\"> To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was reconstituted in 935 μL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 μm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were made by 40–fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (&gt;95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701–5200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 μg/ μL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. </p> </section> </section> <section id=\"S48\"> <h5 class=\"pmc_sec_title\"> Phosphopeptide Enrichment </h5> <p id=\"P76\"> Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 μg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 μL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 μL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 μL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 μL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 × 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 μL 1% FA and phosphopeptides were eluted from the stage tips with 60 μL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 μL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 μL was injected per run. </p> </section> <section id=\"S49\"> <h5 class=\"pmc_sec_title\"> Acetylpeptide Enrichment </h5> <p id=\"P77\"> Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 μg peptides per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 × 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. </p> </section> <section id=\"S50\"> <h5 class=\"pmc_sec_title\"> Ubiquitylpeptide Enrichment with on-bead TMT labeling </h5> <p id=\"P78\"> Ubiquitin enrichment was performed based on the recently published UbiFast protocol ( <a aria-describedby=\"R178\" class=\"usa-link\" href=\"#R178\"> Udeshi et al., 2020 </a> ). Before enrichment, anti-K-ε-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4°C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4°C at a concentration of 0.5 ug/uL. </p> <p id=\"P79\"> For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4 <sup> o </sup> C for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30–40 s, left to settle on ice for 10–20 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20°C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. </p> </section> <section id=\"S51\"> <h5 class=\"pmc_sec_title\"> LC-MS/MS for Proteomics Analyses </h5> <p id=\"P80\"> Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross (360μm, IDEX Health &amp; Science, UH-906x). The column was heated to 50°C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 μm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360–75-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). </p> <p id=\"P81\"> For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). </p> <p id=\"P82\"> For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2–6, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/− 10 ppm mass tolerance. “Perform dependent scan on single charge state per precursor only” was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. </p> </section> <section id=\"S52\"> <h5 class=\"pmc_sec_title\"> Immunohistochemistry (IHC) analysis. </h5> <section id=\"S53\"> <h6 class=\"pmc_sec_title\"> TMA construction </h6> <p id=\"P83\"> Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. </p> </section> <section id=\"S54\"> <h6 class=\"pmc_sec_title\"> Immune cell marker, Immunohistochemistry (IHC) </h6> <p id=\"P84\"> Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0–3 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the “ordinal” package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. </p> </section> <section id=\"S55\"> <h6 class=\"pmc_sec_title\"> Multiplex Immunofluorescence (mIF) staining </h6> <p id=\"P85\"> Development and optimization of the mIF platform has been previously described ( <a aria-describedby=\"R133\" class=\"usa-link\" href=\"#R133\"> Parra et al., 2020 </a> ). mIF staining was performed in a 4-μm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). </p> </section> <section id=\"S56\"> <h6 class=\"pmc_sec_title\"> IHC based assessment of TP63 low samples ( <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> ) </h6> <p id=\"P86\"> For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66–2754) deemed LSCC by TCGA pathology. </p> </section> </section> </section> </section> <section id=\"S57\"> <h3 class=\"pmc_sec_title\"> QUANTIFICATION AND STATISTICAL ANALYSIS </h3> <section id=\"S58\"> <h4 class=\"pmc_sec_title\"> Genomic Data Analysis </h4> <section id=\"S59\"> <h5 class=\"pmc_sec_title\"> Copy Number Calling </h5> <p id=\"P87\"> Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> ). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3–0.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) ( <a aria-describedby=\"R47\" class=\"usa-link\" href=\"#R47\"> Gey and Lebarbier, 2008 </a> ), and dynamic programming ( <a aria-describedby=\"R13\" class=\"usa-link\" href=\"#R13\"> Bellman, 1961 </a> ), as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method ( <a aria-describedby=\"R98\" class=\"usa-link\" href=\"#R98\"> Lebarbier, 2005 </a> ). For the final set of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data. </p> </section> <section id=\"S60\"> <h5 class=\"pmc_sec_title\"> Somatic Variant Calling </h5> <p id=\"P88\"> Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels “GATK bundle known” and “Mills 1000G” somatic variants were called using Sentieon TNScope with the following settings “--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5”, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings “--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000”. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most ‘severe’ consequence occuring in a ‘reliable’ transcript was chosen. To accomplish this the computed consequences were ranked by ‘severity’ and ‘priority’ reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked ‘CANONICAL’ or ‘PICK’ by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD &gt; 5, NLOD &gt; 6, Fisher’s test p-value &lt; 0.05, NFLOD &gt; 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. </p> </section> <section id=\"S61\"> <h5 class=\"pmc_sec_title\"> Germline Variant Calling </h5> <p id=\"P89\"> Germline variant calling and annotation was carried out analogously using DNAScope using the following settings “--emit_conf 10 --call_conf 10” to generate a VCF file and “--emit_mode GVCF” to generate a gVCF file. The variants were custom-filtered based on the specific applications. </p> </section> <section id=\"S62\"> <h5 class=\"pmc_sec_title\"> GISTIC and MutSig analysis </h5> <p id=\"P90\"> The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value &lt;0.25 considered significant. The following parameters were used: </p> <ul class=\"list\" id=\"L10\"> <li> <p id=\"P91\"> Amplification Threshold = 0.1 </p> </li> <li> <p id=\"P92\"> Deletion Threshold = 0.1 </p> </li> <li> <p id=\"P93\"> Cap Values = 1.5 </p> </li> <li> <p id=\"P94\"> Broad Length Cutoff = 0.98 </p> </li> <li> <p id=\"P95\"> Remove X-Chromosome = 0 </p> </li> <li> <p id=\"P96\"> Confidence Level = 0.99 </p> </li> <li> <p id=\"P97\"> Join Segment Size = 4 </p> </li> <li> <p id=\"P98\"> Arm Level Peel Off = 1 </p> </li> <li> <p id=\"P99\"> Maximum Sample Segments = 2000 </p> </li> <li> <p id=\"P100\"> Gene GISTIC = 1 </p> </li> </ul> <p id=\"P101\"> Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from −2 to 2, where 0 means no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: −1 means deletion beyond the threshold; −2 means deletions greater than the minimum arm-level copy number observed in the sample. </p> <p id=\"P102\"> The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV ( <a aria-describedby=\"R95\" class=\"usa-link\" href=\"#R95\"> Lawrence et al., 2014 </a> ) was run on these filtered results to evaluate the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census ( <a aria-describedby=\"R161\" class=\"usa-link\" href=\"#R161\"> Sondka et al., 2018 </a> ), with false discovery rates (q values) recalculated. Genes of q value &lt; 0.1 were declared significant. </p> </section> </section> <section id=\"S63\"> <h4 class=\"pmc_sec_title\"> RNAseq and miRNAseq Quantification </h4> <section id=\"S64\"> <h5 class=\"pmc_sec_title\"> RNAseq Quantification </h5> <p id=\"P103\"> The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page ( <a class=\"usa-link usa-link--external\" href=\"https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files\" rel=\"noopener noreferrer\" target=\"_blank\"> https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files </a> ). The readcount is later converted to FPKM and FPKM-UQ using GDC’s formula ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/ </a> ). </p> </section> <section id=\"S65\"> <h5 class=\"pmc_sec_title\"> Isoform specific RNA quantification </h5> <p id=\"P104\"> For TP63 isoform centric analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> , the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression ( <a aria-describedby=\"R103\" class=\"usa-link\" href=\"#R103\"> Li et al., 2017 </a> ). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. </p> </section> <section id=\"S66\"> <h5 class=\"pmc_sec_title\"> DNA methylation data preprocessing </h5> <p id=\"P105\"> Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously ( <a aria-describedby=\"R39\" class=\"usa-link\" href=\"#R39\"> Fortin et al., 2014 </a> ). We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent analysis. </p> <p id=\"P106\"> Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5’ UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package ( <a aria-describedby=\"R40\" class=\"usa-link\" href=\"#R40\"> Fraley et al., 2016 </a> ) in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty &lt; 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. </p> </section> <section id=\"S67\"> <h5 class=\"pmc_sec_title\"> miRNA-seq Data Analysis </h5> <p id=\"P107\"> miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API ( <a class=\"usa-link usa-link--external\" href=\"https://docs.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://docs.gdc.cancer.gov </a> ). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md </a> ). The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. </p> </section> </section> <section id=\"S68\"> <h4 class=\"pmc_sec_title\"> Proteomics Data Analysis </h4> <section id=\"S69\"> <h5 class=\"pmc_sec_title\"> Spectrum quality filtering and searching </h5> <p id=\"P108\"> All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory ( <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/proteomics\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/proteomics </a> ). Similar MS/MS spectra acquired on the same precursor m/z within +/− 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length &gt; 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800–6000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser ( <a class=\"usa-link usa-link--external\" href=\"https://genome.ucsc.edu/cgi-bin/hgTables\" rel=\"noopener noreferrer\" target=\"_blank\"> https://genome.ucsc.edu/cgi-bin/hgTables </a> ) on June 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/− 20 ppm mass tolerance for precursor and product ions, 30% minimum matched peak intensity, and “trypsin allow P” enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of −18 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −18 to 272 Da. For the acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −690 to 70 Da. </p> </section> <section id=\"S70\"> <h5 class=\"pmc_sec_title\"> Protein grouping, and localization of PTMs </h5> <p id=\"P109\"> Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. </p> <p id=\"P110\"> In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 – Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of &lt;0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5–6, thresholds were optimized to yield a PSM-level FDR estimate of &lt;0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher charge states. </p> <p id=\"P111\"> In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called “one-hit wonders.” After assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method “expand subgroups, top uses shared” to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide sequence of &gt;8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of &lt;0.6% at the peptide-spectrum match level and &lt;0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and &lt;0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. </p> <p id=\"P112\"> In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, &gt;1.1, essentially corresponded to at least 1 b or y ion located between two candidate sites that has a peak height &gt;10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, &gt;99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. </p> </section> <section id=\"S71\"> <h5 class=\"pmc_sec_title\"> Quantitation using TMT ratios </h5> <p id=\"P113\"> Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method “expand subgroups, top uses shared” (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method “unexpand subgroups”. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer’s Rule ( <a aria-describedby=\"R153\" class=\"usa-link\" href=\"#R153\"> Shadforth et al., 2005 </a> ) and correction factors obtained from the reagent manufacturer’s certificate of analysis ( <a class=\"usa-link usa-link--external\" href=\"https://www.thermofisher.com/order/catalog/product/90406\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.thermofisher.com/order/catalog/product/90406 </a> ) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. </p> </section> <section id=\"S72\"> <h5 class=\"pmc_sec_title\"> Two-component normalization of TMT ratios </h5> <p id=\"P114\"> It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians <em> N </em> ( <em> μi </em> 1, <em> σi </em> 1) and <em> N </em> ( <em> μi </em> 2, <em> σi </em> 2) for a sample <em> i </em> were fitted and used in the normalization process as follows: the mode <em> mi </em> of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians constrained to be <em> mi </em> , i.e., <em> μi </em> 1= <em> μi </em> 2= <em> mi </em> . The Gaussian with the smaller estimated standard deviation <span xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"> <math display=\"inline\" id=\"M1\"> <mrow> <mi> σ </mi> <mi> i </mi> <mo> = </mo> <mi> min </mi> <mo> ( </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 1 </mn> <mo> , </mo> <mover> <mrow> <mi> σ </mi> </mrow> <mrow> <mo> ^ </mo> </mrow> </mover> <mi> i </mi> <mn> 2 </mn> <mo> ) </mo> </mrow> </math> </span> was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using ( <em> mi </em> , <em> σi </em> ) by subtracting the mean <em> mi </em> from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation <em> σi </em> . </p> </section> <section id=\"S73\"> <h5 class=\"pmc_sec_title\"> Normalization of acetyl- and ubiquitylproteome </h5> <p id=\"P115\"> To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site’s main accession number did not yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. </p> </section> </section> <section id=\"S74\"> <h4 class=\"pmc_sec_title\"> Proteogenomic analysis </h4> <section id=\"S75\"> <h5 class=\"pmc_sec_title\"> Quality control and Batch effect assessment </h5> <p id=\"P116\"> We used guided principal component analysis (gPCA) ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. </p> <p id=\"P117\"> To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median pearson correlation &gt; 0.85. </p> <p id=\"P118\"> In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as common denominator ( <a class=\"usa-link\" href=\"#SD1\"> Figure S1C </a> ). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value &lt; 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using <em> limma’s </em> removeBatchEffect ( <a class=\"usa-link usa-link--external\" href=\"https://rdrr.io/bioc/limma/man/removeBatchEffect.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rdrr.io/bioc/limma/man/removeBatchEffect.html </a> ) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. </p> </section> <section id=\"S76\"> <h5 class=\"pmc_sec_title\"> Dataset filtering </h5> <p id=\"P119\"> Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in &gt;70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at least two observed TMT ratios in &gt;25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio in &gt;25% of samples. </p> <p id=\"P120\"> Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. </p> </section> <section id=\"S77\"> <h5 class=\"pmc_sec_title\"> Unsupervised multi-omic clustering using NMF </h5> <p id=\"P121\"> We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). Briefly, given a factorization rank <em> k </em> (where <em> k </em> is the number of clusters), NMF decomposes a <em> p x n </em> data matrix <em> V </em> into two matrices <em> W </em> and <em> H </em> such that multiplication of <em> W </em> and <em> H </em> approximates <em> V </em> . Matrix <em> H </em> is a <em> k x n </em> matrix whose entries represent weights for each sample (1 to <em> N </em> ) to contribute to each cluster (1 to <em> k </em> ), whereas matrix <em> W </em> is a <em> p x k </em> matrix representing weights for each feature (1 to <em> p </em> ) to contribute to each cluster (1 to <em> k </em> ). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em> H </em> . We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5. </p> <p id=\"P122\"> To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or −2) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by NMF as follows: </p> <ol class=\"list\" id=\"L12\"> <li> <p id=\"P123\"> Create one data matrix with all negative numbers zeroed. </p> </li> <li> <p id=\"P124\"> Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. </p> </li> <li> <p id=\"P125\"> Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. </p> </li> </ol> <p id=\"P126\"> The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ) and using the factorization method described in ( <a aria-describedby=\"R16\" class=\"usa-link\" href=\"#R16\"> Brunet et al., 2004 </a> ). To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters between <em> k </em> =2 and 8. For each k we factorized matrix <em> V </em> using 50 iterations with random initializations of <em> W </em> and <em> H </em> . To determine the optimal factorization rank we calculated two metrics for each <em> k </em> : 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix as defined in ( <a aria-describedby=\"R80\" class=\"usa-link\" href=\"#R80\"> Kim and Park, 2007 </a> ) measuring the reproducibility of the clustering across 50 iterations. The optimal <em> k </em> was defined as the maximum of the product of both metrics for cluster numbers between <em> k </em> =3 and 8 </p> <p id=\"P127\"> Having determined the optimal factorization rank <em> k </em> , in order to achieve robust factorization of the multi-omic data matrix <em> V </em> , we repeated the NMF analysis using 500 iterations with random initializations of <em> W </em> and <em> H </em> and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em> W </em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first normalized each row in matrix <em> W </em> by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix W <sub> signed </sub> contained signed cluster weights for each feature in the input matrix. </p> <p id=\"P128\"> For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights (W <sub> signed </sub> ) onto Hallmark pathway gene sets ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) using ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ). To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> using the following parameters: </p> <ul class=\"list\" id=\"L14\"> <li> <p id=\"P129\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P130\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P131\"> weight=1 </p> </li> <li> <p id=\"P132\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P133\"> output.score.type=“NES” </p> </li> <li> <p id=\"P134\"> nperm=1000 </p> </li> <li> <p id=\"P135\"> global.fdr=TRUE </p> </li> <li> <p id=\"P136\"> min.overlap=5 </p> </li> <li> <p id=\"P137\"> correl.type=”z.score” </p> </li> </ul> <p id=\"P138\"> To test the association of the resulting clusters to clinical variables we used Fisher’s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. </p> <p id=\"P139\"> The entire workflow described above has been implemented as a module for PANOPLY ( <a aria-describedby=\"R116\" class=\"usa-link\" href=\"#R116\"> Mani et al., 2020 </a> ) which runs on Broad’s Cloud platform Terra ( <a class=\"usa-link usa-link--external\" href=\"https://app.terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://app.terra.bio/ </a> ). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> . </p> </section> <section id=\"S78\"> <h5 class=\"pmc_sec_title\"> Integrative analysis with Stewart et al. </h5> <p id=\"P140\"> To enable integrative analysis of our protein data with the protein data in Stewart et al.( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ) we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. </p> </section> <section id=\"S79\"> <h5 class=\"pmc_sec_title\"> RNA subtyping </h5> <p id=\"P141\"> Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication ( <a aria-describedby=\"R196\" class=\"usa-link\" href=\"#R196\"> Wilkerson et al., 2010 </a> ) and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study ( <a aria-describedby=\"R54\" class=\"usa-link\" href=\"#R54\"> Hammerman et al., 2012 </a> ). Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p &gt; 0.01) were marked to have ‘undecided’ subtype. </p> </section> <section id=\"S80\"> <h5 class=\"pmc_sec_title\"> Chromosomal instability index </h5> <p id=\"P142\"> The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. </p> </section> <section id=\"S81\"> <h5 class=\"pmc_sec_title\"> Fusion detection and analysis </h5> <p id=\"P143\"> Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline ( <a aria-describedby=\"R149\" class=\"usa-link\" href=\"#R149\"> Robinson et al., 2017 </a> ; <a aria-describedby=\"R199\" class=\"usa-link\" href=\"#R199\"> Wu et al., 2018 </a> ), fusioncatcher_v1.10 ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) and arriba_v1.1.0 ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> ). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described ( <a aria-describedby=\"R182\" class=\"usa-link\" href=\"#R182\"> Vats et al., 2020 </a> ). </p> </section> <section id=\"S82\"> <h5 class=\"pmc_sec_title\"> mRNA and Protein correlation </h5> <p id=\"P144\"> To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini–Hochberg procedure. </p> </section> <section id=\"S83\"> <h5 class=\"pmc_sec_title\"> CNA-driven cis and trans effects </h5> <p id=\"P145\"> Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR &lt; 0.05) positive or negative correlations. </p> </section> <section id=\"S84\"> <h5 class=\"pmc_sec_title\"> CMAP analysis </h5> <p id=\"P146\"> Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\" rel=\"noopener noreferrer\" target=\"_blank\"> GSE92742 </a> ). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See <a class=\"usa-link usa-link--external\" href=\"https://clue.io/GEO-guide\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/GEO-guide </a> for more information. </p> <p id=\"P147\"> To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in “CNA driven cis and trans effects” above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ). The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. </p> <p id=\"P148\"> For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were identified with Fisher’s test FDR &lt; 0.25, using this process. </p> <p id=\"P149\"> In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with rate λ, counting the number of identified candidate driver genes. Given the small n ( = 10) and λ, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with <em> both </em> CNA amplification and deletion. </p> <p id=\"P150\"> To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with α = 0.05, followed by counting the number of trans-genes in this list of regulated genes. </p> <p id=\"P151\"> To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. </p> </section> <section id=\"S85\"> <h5 class=\"pmc_sec_title\"> LINCS analysis </h5> <p id=\"P152\"> Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value &lt; 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. </p> <p id=\"P153\"> Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini &amp; Hochberg method. Subtype-specific differentially expressed proteins were identified based on their differential expression (adjusted p.value &lt; 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline ( <a class=\"usa-link usa-link--external\" href=\"https://hub.docker.com/u/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://hub.docker.com/u/cmap </a> ) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described ( <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ). The corresponding compound metadata were obtained from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. </p> </section> <section id=\"S86\"> <h5 class=\"pmc_sec_title\"> Defining cancer-associated genes </h5> <p id=\"P154\"> Cancer-associated genes (CAG) were compiled using the Census website <a class=\"usa-link usa-link--external\" href=\"https://cancer.sanger.ac.uk/census\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cancer.sanger.ac.uk/census </a> . The list of genes is provided in <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> . </p> </section> <section id=\"S87\"> <h5 class=\"pmc_sec_title\"> CpG Island Methylator Phenotype </h5> <p id=\"P155\"> The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score &gt;0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value −0.041 and −0.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. </p> </section> <section id=\"S88\"> <h5 class=\"pmc_sec_title\"> iProFun Based Cis Association Analysis </h5> <p id=\"P156\"> We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR)&lt;10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities &gt; 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) &lt;10% by selecting a minimal cutoff value of alpha such that 75%&lt;alpha&lt;100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities &gt; alpha in permuted data) / (Averaged No. of genes with posterior probabilities &gt; alpha in original data). <a class=\"usa-link\" href=\"#F1\"> Figure 1G </a> annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. </p> </section> <section id=\"S89\"> <h5 class=\"pmc_sec_title\"> EMT-specific cluster and fibroblast enrichment </h5> <p id=\"P157\"> NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). </p> </section> <section id=\"S90\"> <h5 class=\"pmc_sec_title\"> Differential marker analysis </h5> <p id=\"P158\"> A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having &lt; 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR ( <a aria-describedby=\"R106\" class=\"usa-link\" href=\"#R106\"> Liao et al., 2019 </a> ). The selected gene list was either significant proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted p-value &lt;0.01. </p> <p id=\"P159\"> Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> ), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. </p> </section> <section id=\"S91\"> <h5 class=\"pmc_sec_title\"> Overall and disease-free survival in TCGA lung squamous carcinoma </h5> <p id=\"P160\"> Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. </p> </section> <section id=\"S92\"> <h5 class=\"pmc_sec_title\"> Kaplan-Meier survival analysis </h5> <p id=\"P161\"> We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. </p> <p id=\"P162\"> We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. </p> </section> <section id=\"S93\"> <h5 class=\"pmc_sec_title\"> Continuous Smoking Score </h5> <p id=\"P163\"> Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC ( <a aria-describedby=\"R170\" class=\"usa-link\" href=\"#R170\"> Tate et al., 2019 </a> ) and cosine similarity was calculated to identify the best match. </p> <p id=\"P164\"> We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 &lt; S &lt; 1, to quantify the degree of confidence that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C ∩ T ∩ N, and used A and A’ to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A’ and D to the joint probability of A and D. For example, the latter can be written P(A)・P(C|A)・P(T|A)・P(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9 ( <a aria-describedby=\"R184\" class=\"usa-link\" href=\"#R184\"> Walser et al., 2008 </a> ). </p> <p id=\"P165\"> We consider <em> S </em> to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type <em> G </em> is P( <em> G </em> | <em> A </em> ) = [ <em> x </em> ・ <em> erf </em> ( <em> g/y </em> ) + 1] <em> / </em> ( <em> x </em> + 2) and P( <em> G </em> | <em> A </em> ’) = 1 - P( <em> G </em> | <em> A </em> ), where <em> erf </em> is the Gaussian error function, <em> g </em> is the observed value for the given data type, and <em> x </em> and <em> y </em> are empirically-determined weights. </p> <p id=\"P166\"> The shape function for total mutations, <em> T </em> , has two modifiers. First, it includes a simple expected-value correction for purity, <em> u </em> . Namely, assuming mutation-calling does not capture all mutations because of impurities, <em> t </em> is taken as the observed number of mutations divided by a purity shape function, <em> f </em> , where <em> f </em> ≤ 1. Although one might model <em> f </em> according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we use a line function, <em> f </em> = 0.2・ <em> u </em> + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce’s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score &gt; 0.6; samples with weak smoking evidence if smoking score &lt; 0.15. </p> </section> <section id=\"S94\"> <h5 class=\"pmc_sec_title\"> Identification of differentially regulated events in NRF2 mutant tumors. </h5> <p id=\"P167\"> For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset ( <a class=\"usa-link\" href=\"#SD10\"> Table S4 </a> ) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value &lt;0.05 and logFC&gt;1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 ( <a aria-describedby=\"R115\" class=\"usa-link\" href=\"#R115\"> Malhotra et al., 2010 </a> ) and SINGH_NFE2L2_TARGETS (( <a aria-describedby=\"R159\" class=\"usa-link\" href=\"#R159\"> Singh et al., 2008 </a> ), BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ). Ten of the 54 genes were novel targets identified in this study. </p> </section> <section id=\"S95\"> <h5 class=\"pmc_sec_title\"> Immune cluster identification based on cell type composition </h5> <p id=\"P168\"> The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) based on 108 tumors and 94 NAT samples. <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> contains the final score computed by xCell of different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR &lt; 5%). This filtering resulted in 25 cell types annotated in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) ( <a class=\"usa-link\" href=\"#F6\"> Figure 6A </a> , <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S96\"> <h5 class=\"pmc_sec_title\"> TCGA pan-cancer immune subtyping </h5> <p id=\"P169\"> Tumors were classified into the six distinct immune subtypes — wound healing, IFN-γ dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-β dominant — identified by a TCGA pan-cancer analysis presented in ( <a aria-describedby=\"R173\" class=\"usa-link\" href=\"#R173\"> Thorsson et al., 2018 </a> ). Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> ) ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) to assign an immune subtype to each of the tumors. </p> </section> <section id=\"S97\"> <h5 class=\"pmc_sec_title\"> Ranking tumors by inferred activity of IFN-γ pathway </h5> <p id=\"P170\"> We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above “top” and “bottom” probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the “top” and “bottom” probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher’s exact test, given protein’s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor ( <a aria-describedby=\"R147\" class=\"usa-link\" href=\"#R147\"> Reva et al., 2020 </a> ). The thermodynamic interpretation of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway’s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway’s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. </p> <p id=\"P171\"> In our analysis, we used a gene list of IFN-γ signaling pathway from ( <a aria-describedby=\"R1\" class=\"usa-link\" href=\"#R1\"> Abril-Rodriguez and Ribas, 2017 </a> ). We settled with this 15 genes signaling pathway because it has all necessary components – a cytokine mediator, receptors, signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. </p> </section> <section id=\"S98\"> <h5 class=\"pmc_sec_title\"> Estimation of Tumor Purity, Stromal and Immune Scores </h5> <p id=\"P172\"> Besides xCell, we utilized ESTIMATE ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) to infer immune and stromal scores based on gene expression data ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app ( <a class=\"usa-link usa-link--external\" href=\"http://cibersort.stanford.edu\" rel=\"noopener noreferrer\" target=\"_blank\"> cibersort.stanford.edu </a> ; ( <a aria-describedby=\"R127\" class=\"usa-link\" href=\"#R127\"> Newman et al., 2015 </a> )) in absolute mode. To infer tumor purity, TSNet was utilized ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) ( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). </p> </section> <section id=\"S99\"> <h5 class=\"pmc_sec_title\"> Differentially Expressed Genes and Pathway Analysis </h5> <p id=\"P173\"> Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA ( <a aria-describedby=\"R12\" class=\"usa-link\" href=\"#R12\"> Barbie et al., 2009 </a> ) was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hänzelmann et al., 2013 </a> ). For this analysis, pathways from the Reactome ( <a aria-describedby=\"R33\" class=\"usa-link\" href=\"#R33\"> Fabregat et al., 2018 </a> ), KEGG ( <a aria-describedby=\"R75\" class=\"usa-link\" href=\"#R75\"> Kanehisa et al., 2017 </a> ) and Hallmark ( <a aria-describedby=\"R107\" class=\"usa-link\" href=\"#R107\"> Liberzon et al., 2015 </a> ) databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> . </p> </section> <section id=\"S100\"> <h5 class=\"pmc_sec_title\"> Deriving RTK CBPE scores </h5> <p id=\"P174\"> A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ). Each association computation was carried out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. </p> </section> <section id=\"S101\"> <h5 class=\"pmc_sec_title\"> Independent component analysis </h5> <p id=\"P175\"> ICA was performed with a workflow modified from previously described ( <a aria-describedby=\"R111\" class=\"usa-link\" href=\"#R111\"> Liu et al., 2019 </a> ). Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm ( <a aria-describedby=\"R117\" class=\"usa-link\" href=\"#R117\"> McInnes et al., 2017 </a> ) with cosine distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 <em> P </em> -values. Significance was controlled for multiple testing at 0.01 level (log10 ( <em> p-value) </em> =−5.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. </p> </section> <section id=\"S102\"> <h5 class=\"pmc_sec_title\"> Mutation-based cis- and trans-effects </h5> <p id=\"P176\"> We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study ( <a aria-describedby=\"R10\" class=\"usa-link\" href=\"#R10\"> Bailey et al., 2018 </a> ) on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR &lt;0.05 cut-off were separated into two categories based on cis- and trans- effects. </p> </section> <section id=\"S103\"> <h5 class=\"pmc_sec_title\"> Germline quantitative trait loci (QTL) analysis </h5> <p id=\"P177\"> To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen </a> ). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count ≥ 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9 ( <a aria-describedby=\"R27\" class=\"usa-link\" href=\"#R27\"> Deelen et al., 2014 </a> ). To adjust for population stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07 ( <a aria-describedby=\"R142\" class=\"usa-link\" href=\"#R142\"> Purcell et al., 2007 </a> ). For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE &gt; 0.0001, MAF &gt; 0.01, and call rate &gt; 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at ≤ FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis ( <a aria-describedby=\"R86\" class=\"usa-link\" href=\"#R86\"> Krämer et al., 2014 </a> ) </p> </section> <section id=\"S104\"> <h5 class=\"pmc_sec_title\"> miRNA analysis presented in <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </h5> <p id=\"P178\"> Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained ( <a aria-describedby=\"R66\" class=\"usa-link\" href=\"#R66\"> Huang et al., 2020 </a> ). Spearman correlation was used to calculate the correlations between miRNA and its target genes. </p> </section> <section id=\"S105\"> <h5 class=\"pmc_sec_title\"> Pathway projection using ssGSEA </h5> <p id=\"P179\"> The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: </p> <ul class=\"list\" id=\"L16\"> <li> <p id=\"P180\"> gene.set.database=“h.all.v6.2.symbols.gmt” </p> </li> <li> <p id=\"P181\"> sample.norm.type=“rank” </p> </li> <li> <p id=\"P182\"> weight=0.75 </p> </li> <li> <p id=\"P183\"> statistic=”area.under.RES” </p> </li> <li> <p id=\"P184\"> output.score.type=“NES” </p> </li> <li> <p id=\"P185\"> nperm=1000 </p> </li> <li> <p id=\"P186\"> global.fdr=TRUE </p> </li> <li> <p id=\"P187\"> min.overlap=10 </p> </li> <li> <p id=\"P188\"> correl.type=”z.score” </p> </li> </ul> </section> <section id=\"S106\"> <h5 class=\"pmc_sec_title\"> Phosphorylation-driven signature analysis </h5> <p id=\"P189\"> We performed phosphosite-specific signature enrichment analysis (PTM-SEA) ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed −log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in <a class=\"usa-link\" href=\"#F3\"> Figure 3I </a> ) </p> <table class=\"disp-formula p\" id=\"FD1\"> <tr> <td class=\"formula\"> <math display=\"block\" id=\"M2\"> <mrow> <mtext> logPsite </mtext> <mspace width=\"0.25em\"> </mspace> <mo> = </mo> <mspace width=\"0.25em\"> </mspace> <mo> − </mo> <mtext> log </mtext> <mn> 10 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> p </mtext> <mo> − </mo> <mtext> valuesite </mtext> </mrow> <mo> ) </mo> </mrow> <mo> * </mo> <mtext> sign </mtext> <mrow> <mo> ( </mo> <mrow> <mtext> log </mtext> <mn> 2 </mn> <mrow> <mo> ( </mo> <mrow> <mtext> fold </mtext> <mspace width=\"0.25em\"> </mspace> <mtext> changesite </mtext> </mrow> <mo> ) </mo> </mrow> </mrow> <mo> ) </mo> </mrow> </mrow> </math> </td> </tr> </table> <p> PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. </p> <p id=\"P190\"> We employed the heuristic method introduced by Krug et al. ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> ) to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. </p> <p id=\"P191\"> We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb </a> using the flanking amino acid sequence (+/− 7 aa) as primary identifier. We used the implementation of PTM-SEA available on GitHub ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> ) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: </p> <p id=\"P192\"> weight: 1 </p> <p id=\"P193\"> statistic: “area.under.RES” </p> <p id=\"P194\"> output.score.type: “NES” </p> <p id=\"P195\"> nperm: 1000 </p> <p id=\"P196\"> min.overlap: 5 </p> <p id=\"P197\"> correl.type: “Z-score” </p> <p id=\"P198\"> The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini &amp; and Hochberg ( <a aria-describedby=\"R14\" class=\"usa-link\" href=\"#R14\"> Benjamini and Hochberg, 1995 </a> ). Signatures with FDR-corrected p-values &lt; 0.05 were considered to be differential between tumor and NAT. </p> </section> <section id=\"S107\"> <h5 class=\"pmc_sec_title\"> <em> CDKN2A </em> and <em> RB1 </em> annotations and pathway analysis </h5> <p id=\"P199\"> Comprehensive tumor annotation for <em> CDKN2A </em> and <em> RB1 </em> genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values &gt;= 0.2. Multi-gene proliferation scores were calculated as described previously ( <a aria-describedby=\"R32\" class=\"usa-link\" href=\"#R32\"> Ellis et al., 2017 </a> ) as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al ( <a aria-describedby=\"R194\" class=\"usa-link\" href=\"#R194\"> Whitfield et al., 2002 </a> ). p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_000068.1\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_000068.1 </a> for p16INK4a and Refseq <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/protein/NP_478102.2\" rel=\"noopener noreferrer\" target=\"_blank\"> NP_478102.2 </a> for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ). Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. </p> </section> <section id=\"S108\"> <h5 class=\"pmc_sec_title\"> Association analysis between KGG-site abundances and E3 ligases and DUBs </h5> <p id=\"P200\"> A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from ( <a aria-describedby=\"R119\" class=\"usa-link\" href=\"#R119\"> Medvar et al., 2016 </a> ; <a aria-describedby=\"R129\" class=\"usa-link\" href=\"#R129\"> Nijman et al., 2005 </a> ). We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR &lt; 0.01). </p> </section> <section id=\"S109\"> <h5 class=\"pmc_sec_title\"> Cluster and pathway analysis of significantly modulated K-GG sites in tumors. </h5> <p id=\"P201\"> Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. </p> <p id=\"P202\"> Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere et al., 2019 </a> ). Pathway enrichment was performed using a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon et al., 2003 </a> ) using the Enrichment Map app ( <a aria-describedby=\"R120\" class=\"usa-link\" href=\"#R120\"> Merico et al., 2010 </a> ) for network analysis of pathways. Pathways were connected using the gene overlap and clustered (pathway cutoff q-val &lt; 0.1; jaccard overlap &gt; 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. </p> </section> <section id=\"S110\"> <h5 class=\"pmc_sec_title\"> PTM CLUMPS analysis </h5> <p id=\"P203\"> We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual “events” as done with mutations in CLUMPS v1 ( <a aria-describedby=\"R74\" class=\"usa-link\" href=\"#R74\"> Kamburov et al., 2015 </a> ). For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and Tumor samples ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ). Sites are selected with an FDR &lt; 0.1 (Benjamini-Hochberg) and LFC &gt; 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. </p> </section> <section id=\"S111\"> <h5 class=\"pmc_sec_title\"> DepMap genetic dependency and drug response analysis </h5> <p id=\"P204\"> Cell lines annotated as “NSCLC_squamous” from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. </p> <p id=\"P205\"> To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study’s LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from DepMap. </p> <p id=\"P206\"> <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> : For identifying if top protein biomarkers (502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines ( <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. </p> <p id=\"P207\"> <a class=\"usa-link\" href=\"#SD4\"> Figure S4B </a> : To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. </p> <p id=\"P208\"> Continuous log2 copy number data from DepMap was used in <a class=\"usa-link\" href=\"#SD4\"> Figure S4M </a> to correlate <em> SOX2 </em> copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). </p> <p id=\"P209\"> Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression ( <a class=\"usa-link\" href=\"#SD4\"> Figure S4G </a> ). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) ( <a class=\"usa-link\" href=\"#SD3\"> Figure S3D </a> ). </p> </section> <section id=\"S112\"> <h5 class=\"pmc_sec_title\"> CausalPath analysis </h5> <p id=\"P210\"> CausalPath ( <a aria-describedby=\"R8\" class=\"usa-link\" href=\"#R8\"> Babur et al., 2018 </a> ) searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of <a class=\"usa-link\" href=\"#F7\"> Figure 7C </a> , identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). </p> </section> <section id=\"S113\"> <h5 class=\"pmc_sec_title\"> Variant Peptide Identification </h5> <p id=\"P211\"> We used NeoFlow ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> ) for neoantigen prediction ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ). Specifically, Optitype ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) to predict HLA peptide binding affinity for somatic mutation–derived variant peptides with a length between 8–11 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study ( <a aria-describedby=\"R180\" class=\"usa-link\" href=\"#R180\"> Vasaikar et al., 2019 </a> ). To identify variant peptides, we used a customized protein sequence database approach ( <a aria-describedby=\"R185\" class=\"usa-link\" href=\"#R185\"> Wang et al., 2012 </a> ). We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> ) for customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using PepQuery ( <a class=\"usa-link usa-link--external\" href=\"http://www.pepquery.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.pepquery.org </a> ) ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) with the p-value cutoff &lt;= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV ( <a class=\"usa-link usa-link--external\" href=\"http://www.zhang-lab.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.zhang-lab.org/ </a> ) ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ). </p> </section> <section id=\"S114\"> <h5 class=\"pmc_sec_title\"> Cancer/testis Antigen Prediction </h5> <p id=\"P212\"> Cancer/testis (CT) antigens were downloaded from the CTdatabase ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ). CT antigens with a median &gt;4-fold increase in tumor from NAT were highlighted. </p> </section> <section id=\"S115\"> <h5 class=\"pmc_sec_title\"> PROGENy Scores </h5> <p id=\"P213\"> PROGENy ( <a aria-describedby=\"R152\" class=\"usa-link\" href=\"#R152\"> Schubert et al., 2018 </a> ) was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy. </p> </section> <section id=\"S116\"> <h5 class=\"pmc_sec_title\"> LSCC, HNSCC and LUAD integrative analysis </h5> <p id=\"P214\"> LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ). HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ). The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in the <a class=\"usa-link\" href=\"#S90\"> Differential marker analysis </a> method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. </p> <p id=\"P215\"> Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value &lt; .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value &lt;0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p value &lt;0.01). </p> <p id=\"P216\"> To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method <a class=\"usa-link\" href=\"#S98\"> Estimation of Tumor Purity, Stromal and Immune Scores </a> . To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p value &lt; 0.01 were considered significant. </p> </section> </section> </section> </section> <section id=\"SM1\"> <h2 class=\"pmc_sec_title\"> Supplementary Material </h2> <section class=\"sm xbox font-sm\" id=\"SD1\"> <div class=\"caption p\"> <span> 1 </span> <p id=\"P217\"> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p id=\"P218\"> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p id=\"P219\"> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p id=\"P220\"> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p id=\"P221\"> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p id=\"P222\"> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p id=\"P223\"> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p id=\"P224\"> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p id=\"P225\"> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\" id=\"d40e3501\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD2\"> <div class=\"caption p\"> <span> 2 </span> <p id=\"P226\"> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p id=\"P227\"> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p id=\"P228\"> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p id=\"P229\"> C. Same as B but in the EMT-enriched subtype. </p> <p id=\"P230\"> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p id=\"P231\"> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p id=\"P232\"> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P233\"> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p id=\"P234\"> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p id=\"P235\"> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p id=\"P236\"> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p id=\"P237\"> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\" id=\"d40e3559\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD3\"> <div class=\"caption p\"> <span> 3 </span> <p id=\"P238\"> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p id=\"P239\"> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p id=\"P240\"> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p id=\"P241\"> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p id=\"P242\"> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p id=\"P243\"> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p id=\"P244\"> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p id=\"P245\"> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p id=\"P246\"> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\" id=\"d40e3627\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD4\"> <div class=\"caption p\"> <span> 4 </span> <p id=\"P247\"> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p id=\"P248\"> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p id=\"P249\"> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p id=\"P250\"> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p id=\"P251\"> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p id=\"P252\"> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p id=\"P253\"> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p id=\"P254\"> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p id=\"P255\"> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p id=\"P256\"> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p id=\"P257\"> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p id=\"P258\"> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p id=\"P259\"> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p id=\"P260\"> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\" id=\"d40e3671\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD5\"> <div class=\"caption p\"> <span> 6 </span> <p id=\"P261\"> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P262\"> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p id=\"P263\"> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p id=\"P264\"> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p id=\"P265\"> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p id=\"P266\"> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p id=\"P267\"> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p id=\"P268\"> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\" id=\"d40e3727\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD6\"> <div class=\"caption p\"> <span> 7 </span> <p id=\"P269\"> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P270\"> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p id=\"P271\"> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p id=\"P272\"> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p id=\"P273\"> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p id=\"P274\"> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p id=\"P275\"> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p id=\"P276\"> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\" id=\"d40e3755\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD7\"> <div class=\"caption p\"> <span> 8 </span> <p id=\"P277\"> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P278\"> A. Metadata associated with this study </p> <p id=\"P279\"> B. Cancer Associated Genes (CAGs) </p> <p id=\"P280\"> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p id=\"P281\"> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p id=\"P282\"> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\" id=\"d40e3782\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD8\"> <div class=\"caption p\"> <span> 9 </span> <p id=\"P283\"> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P284\"> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p id=\"P285\"> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p id=\"P286\"> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p id=\"P287\"> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p id=\"P288\"> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p id=\"P289\"> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p id=\"P290\"> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p id=\"P291\"> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p id=\"P292\"> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\" id=\"d40e3829\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD9\"> <div class=\"caption p\"> <span> 10 </span> <p id=\"P293\"> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p id=\"P294\"> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P295\"> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p id=\"P296\"> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p id=\"P297\"> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P298\"> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P299\"> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P300\"> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P301\"> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P302\"> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P303\"> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p id=\"P304\"> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P305\"> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p id=\"P306\"> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p id=\"P307\"> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\" id=\"d40e3937\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD10\"> <div class=\"caption p\"> <span> 11 </span> <p id=\"P308\"> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p id=\"P309\"> A. Copy number amplified cytobands </p> <p id=\"P310\"> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p id=\"P311\"> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p id=\"P312\"> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p id=\"P313\"> F. CMAP tab- Connectivity Map (CMAP) results </p> <p id=\"P314\"> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p id=\"P315\"> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p id=\"P316\"> I. Summary of CDKN2A and RB1 annotations </p> <p id=\"P317\"> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p id=\"P318\"> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p id=\"P319\"> L. NRF2 signature </p> <p id=\"P320\"> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\" id=\"d40e4003\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD11\"> <div class=\"caption p\"> <span> 12 </span> <p id=\"P321\"> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P322\"> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p id=\"P323\"> B. Ub-sites that show differential abundance across Ub clusters </p> <p id=\"P324\"> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p id=\"P325\"> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p id=\"P326\"> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P327\"> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p id=\"P328\"> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p id=\"P329\"> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P330\"> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p id=\"P331\"> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\" id=\"d40e4032\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD12\"> <div class=\"caption p\"> <span> 13 </span> <p id=\"P332\"> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p id=\"P333\"> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p id=\"P334\"> B. Immune clusters based on xCell signatures </p> <p id=\"P335\"> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p id=\"P336\"> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p id=\"P337\"> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p id=\"P338\"> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p id=\"P339\"> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p id=\"P340\"> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p id=\"P341\"> I. List of genes/proteins whose expression is associated with a SNP </p> <p id=\"P342\"> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p id=\"P343\"> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\" id=\"d40e4072\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD13\"> <div class=\"caption p\"> <span> 14 </span> <p id=\"P344\"> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p id=\"P345\"> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p id=\"P346\"> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p id=\"P347\"> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p id=\"P348\"> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p id=\"P349\"> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p id=\"P350\"> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\" id=\"d40e4094\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"SD14\"> <div class=\"caption p\"> <span> Figure S5 </span> <p id=\"P351\"> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p id=\"P352\"> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p id=\"P353\"> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p id=\"P354\"> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p id=\"P355\"> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p id=\"P356\"> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p id=\"P357\"> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p id=\"P358\"> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p id=\"P359\"> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p id=\"P360\"> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p id=\"P361\"> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p id=\"P362\"> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\" id=\"d40e4134\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"floats-group\" id=\"floats-group1\"> <section class=\"bt xbox font-sm\" id=\"BX1\"> <h2 class=\"obj_head\"> Highlights. </h2> <ul class=\"list\" id=\"L18\"> <li> <p id=\"P437\"> Unsupervised clustering revealed subtype with EMT and phosphoprotein signatures </p> </li> <li> <p id=\"P438\"> Potential therapeutic vulnerabilities included survivin, NSD3, LSD1 and EZH2 </p> </li> <li> <p id=\"P439\"> Rb phosphorylation nominated as a biomarker for trials with CDK4/6 inhibitors </p> </li> <li> <p id=\"P440\"> Detailed immune landscape analysis highlighted targetable points of immune regulation </p> </li> </ul> </section> </section> <section class=\"ack\" id=\"S117\"> <h2 class=\"pmc_sec_title\"> Acknowledgements </h2> <p id=\"P363\"> This work was supported by grants U24CA210955, U24CA210985, U24CA210986, U24CA210954, U24CA210967, U24CA210972, U24CA210979, U24CA210993, U01CA214114, U01CA214116, and U01CA214125 from the NCI-CPTAC, U54HL127624, P30CA240139, 1F32HL154711-01, and T32CA203690 from the NIH. We thank Dr. Lata Adnani for helping with graphics. </p> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN3\"> <p id=\"P364\"> Declaration of Interests </p> <p id=\"P365\"> The authors declare no competing interests </p> </div> <div class=\"fn p\" id=\"FN4\"> <p id=\"P366\"> ADDITIONAL RESOURCES </p> <p id=\"P367\"> The CPTAC program website, detailing program initiatives, investigators, and datasets, is found at <a class=\"usa-link usa-link--external\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.cancer.gov/programs/cptac </a> . </p> <p id=\"P368\"> A website for interactive visualization of the multi-omic dataset is available at: <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </p> <p id=\"P369\"> All processed data matrices are available at LinkedOmics ( <a aria-describedby=\"R181\" class=\"usa-link\" href=\"#R181\"> Vasaikar et al., 2018 </a> ) ( <a class=\"usa-link usa-link--external\" href=\"http://www.linkedomics.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.linkedomics.org </a> ), where computational tools are available for further exploration of this dataset. </p> <p id=\"P370\"> Consortia </p> <p id=\"P371\"> Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Fenyö, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Małgorzata Wojtyś, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Özgün Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. </p> </div> <div class=\"fn p\" id=\"FN5\"> <p id=\"P372\"> <strong> Publisher\\'s Disclaimer: </strong> This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. </p> </div> </div> </section> <section class=\"ref-list\" id=\"ref-list1\"> <h2 class=\"pmc_sec_title\"> REFERENCES: </h2> <section id=\"ref-list1_sec2\"> <ol class=\"ref-list font-sm\"> <li id=\"R1\"> <cite> Abril-Rodriguez G, and Ribas A (2017). SnapShot: Immune Checkpoint Inhibitors. Cancer Cell 31, 848–848.e1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2017.05.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28609660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=SnapShot:%20Immune%20Checkpoint%20Inhibitors&amp;author=G%20Abril-Rodriguez&amp;author=A%20Ribas&amp;volume=31&amp;publication_year=2017&amp;pages=848-848.e1&amp;pmid=28609660&amp;doi=10.1016/j.ccell.2017.05.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R2\"> <cite> Ahn ER, Mangat PK, Garrett-Mayer E, Halabi S, Dib EG, Haggstrom DE, Alguire KB, Calfa CJ, Cannon TL, Crilley PA, et al. (2020). Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study. JCO Precision Oncology 757–766. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/PO.20.00037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/35050752/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCO%20Precision%20Oncology&amp;title=Palbociclib%20in%20Patients%20With%20Non%E2%80%93Small-Cell%20Lung%20Cancer%20With%20CDKN2A%20Alterations:%20Results%20From%20the%20Targeted%20Agent%20and%20Profiling%20Utilization%20Registry%20Study&amp;author=ER%20Ahn&amp;author=PK%20Mangat&amp;author=E%20Garrett-Mayer&amp;author=S%20Halabi&amp;author=EG%20Dib&amp;publication_year=2020&amp;pages=757-766&amp;pmid=35050752&amp;doi=10.1200/PO.20.00037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R3\"> <cite> Albert M, Bécares M, Falqui M, Fernández-Lozano C, and Guerra S (2018). ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis. Viruses 10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/v10110629\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6265978/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30428561/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=ISG15,%20a%20Small%20Molecule%20with%20Huge%20Implications:%20Regulation%20of%20Mitochondrial%20Homeostasis&amp;author=M%20Albert&amp;author=M%20B%C3%A9cares&amp;author=M%20Falqui&amp;author=C%20Fern%C3%A1ndez-Lozano&amp;author=S%20Guerra&amp;volume=10&amp;publication_year=2018&amp;pmid=30428561&amp;doi=10.3390/v10110629&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R4\"> <cite> Alcalá S, Sancho P, Martinelli P, Navarro D, Pedrero C, Martín-Hijano L, Valle S, Earl J, Rodríguez-Serrano M, Ruiz-Cañas L, et al. (2020). ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity. Nat. Commun. 11, 2682. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-16395-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7260233/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32472071/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=ISG15%20and%20ISGylation%20is%20required%20for%20pancreatic%20cancer%20stem%20cell%20mitophagy%20and%20metabolic%20plasticity&amp;author=S%20Alcal%C3%A1&amp;author=P%20Sancho&amp;author=P%20Martinelli&amp;author=D%20Navarro&amp;author=C%20Pedrero&amp;volume=11&amp;publication_year=2020&amp;pages=2682&amp;pmid=32472071&amp;doi=10.1038/s41467-020-16395-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R5\"> <cite> Almeida LG, Sakabe NJ, deOliveira AR, Silva MCC, Mundstein AS, Cohen T, Chen Y-T, Chua R, Gurung S, Gnjatic S, et al. (2009). CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens. Nucleic Acids Res. 37, D816–D819. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkn673\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2686577/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18838390/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CTdatabase:%20a%20knowledge-base%20of%20high-throughput%20and%20curated%20data%20on%20cancer-testis%20antigens&amp;author=LG%20Almeida&amp;author=NJ%20Sakabe&amp;author=AR%20deOliveira&amp;author=MCC%20Silva&amp;author=AS%20Mundstein&amp;volume=37&amp;publication_year=2009&amp;pages=D816-D819&amp;pmid=18838390&amp;doi=10.1093/nar/gkn673&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R6\"> <cite> Aran D, Hu Z, and Butte AJ (2017). xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 18, 220. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-017-1349-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5688663/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29141660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=xCell:%20digitally%20portraying%20the%20tissue%20cellular%20heterogeneity%20landscape&amp;author=D%20Aran&amp;author=Z%20Hu&amp;author=AJ%20Butte&amp;volume=18&amp;publication_year=2017&amp;pages=220&amp;pmid=29141660&amp;doi=10.1186/s13059-017-1349-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R7\"> <cite> Ardini-Poleske ME, Clark RF, Ansong C, Carson JP, Corley RA, Deutsch GH, Hagood JS, Kaminski N, Mariani TJ, Potter SS, et al. (2017). LungMAP: The Molecular Atlas of Lung Development Program. Am. J. Physiol. Lung Cell. Mol. Physiol. 313, L733–L740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/ajplung.00139.2017\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5792185/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28798251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Lung%20Cell.%20Mol.%20Physiol&amp;title=LungMAP:%20The%20Molecular%20Atlas%20of%20Lung%20Development%20Program&amp;author=ME%20Ardini-Poleske&amp;author=RF%20Clark&amp;author=C%20Ansong&amp;author=JP%20Carson&amp;author=RA%20Corley&amp;volume=313&amp;publication_year=2017&amp;pages=L733-L740&amp;pmid=28798251&amp;doi=10.1152/ajplung.00139.2017&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R8\"> <cite> Babur Ö, Luna A, Korkut A, Durupinar F, Siper MC, Dogrusoz U, Aslan JE, Sander C, and Demir E (2018). Causal interactions from proteomic profiles: molecular data meets pathway knowledge. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.patter.2021.100257\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8212145/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34179843/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Causal%20interactions%20from%20proteomic%20profiles:%20molecular%20data%20meets%20pathway%20knowledge&amp;author=%C3%96%20Babur&amp;author=A%20Luna&amp;author=A%20Korkut&amp;author=F%20Durupinar&amp;author=MC%20Siper&amp;publication_year=2018&amp;pmid=34179843&amp;doi=10.1016/j.patter.2021.100257&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R9\"> <cite> Baeza J, Smallegan MJ, and Denu JM (2015). Site-specific reactivity of nonenzymatic lysine acetylation. ACS Chem. Biol. 10, 122–128. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/cb500848p\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4301072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25555129/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol&amp;title=Site-specific%20reactivity%20of%20nonenzymatic%20lysine%20acetylation&amp;author=J%20Baeza&amp;author=MJ%20Smallegan&amp;author=JM%20Denu&amp;volume=10&amp;publication_year=2015&amp;pages=122-128&amp;pmid=25555129&amp;doi=10.1021/cb500848p&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R10\"> <cite> Bailey MH, Tokheim C, Porta-Pardo E, Sengupta S, Bertrand D, Weerasinghe A, Colaprico A, Wendl MC, Kim J, Reardon B, et al. (2018). Comprehensive Characterization of Cancer Driver Genes and Mutations. Cell 173, 371–385.e18. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.02.060\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6029450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29625053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Characterization%20of%20Cancer%20Driver%20Genes%20and%20Mutations&amp;author=MH%20Bailey&amp;author=C%20Tokheim&amp;author=E%20Porta-Pardo&amp;author=S%20Sengupta&amp;author=D%20Bertrand&amp;volume=173&amp;publication_year=2018&amp;pages=371-385.e18&amp;pmid=29625053&amp;doi=10.1016/j.cell.2018.02.060&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R11\"> <cite> Bando H, Atsumi T, Nishio T, Niwa H, Mishima S, Shimizu C, Yoshioka N, Bucala R, and Koike T (2005). Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer. Clin. Cancer Res. 11, 5784–5792. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-05-0149\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16115917/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Phosphorylation%20of%20the%206-phosphofructo-2-kinase/fructose%202,6-bisphosphatase/PFKFB3%20family%20of%20glycolytic%20regulators%20in%20human%20cancer&amp;author=H%20Bando&amp;author=T%20Atsumi&amp;author=T%20Nishio&amp;author=H%20Niwa&amp;author=S%20Mishima&amp;volume=11&amp;publication_year=2005&amp;pages=5784-5792&amp;pmid=16115917&amp;doi=10.1158/1078-0432.CCR-05-0149&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R12\"> <cite> Barbie DA, Tamayo P, Boehm JS, Kim SY, Moody SE, Dunn IF, Schinzel AC, Sandy P, Meylan E, Scholl C, et al. (2009). Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108–112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature08460\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2783335/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19847166/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;author=DA%20Barbie&amp;author=P%20Tamayo&amp;author=JS%20Boehm&amp;author=SY%20Kim&amp;author=SE%20Moody&amp;volume=462&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19847166&amp;doi=10.1038/nature08460&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R13\"> <cite> Bellman R (1961). On the approximation of curves by line segments using dynamic programming. Commun. ACM 4, 284. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun.%20ACM&amp;title=On%20the%20approximation%20of%20curves%20by%20line%20segments%20using%20dynamic%20programming&amp;author=R%20Bellman&amp;volume=4&amp;publication_year=1961&amp;pages=284&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R14\"> <cite> Benjamini Y, and Hochberg Y (1995). Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat. Soc. Series B Stat. Methodol. 57, 289–300. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol&amp;title=Controlling%20the%20False%20Discovery%20Rate:%20A%20Practical%20and%20Powerful%20Approach%20to%20Multiple%20Testing&amp;author=Y%20Benjamini&amp;author=Y%20Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R15\"> <cite> Bray F, Ferlay J, Soerjomataram I, Siegel RL, Torre LA, and Jemal A (2018). Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 68, 394–424. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30207593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&amp;author=F%20Bray&amp;author=J%20Ferlay&amp;author=I%20Soerjomataram&amp;author=RL%20Siegel&amp;author=LA%20Torre&amp;volume=68&amp;publication_year=2018&amp;pages=394-424&amp;pmid=30207593&amp;doi=10.3322/caac.21492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R16\"> <cite> Brunet J-P, Tamayo P, Golub TR, and Mesirov JP (2004). Metagenes and molecular pattern discovery using matrix factorization. Proc. Natl. Acad. Sci. U. S. A. 101, 4164–4169. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0308531101\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC384712/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/15016911/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Metagenes%20and%20molecular%20pattern%20discovery%20using%20matrix%20factorization&amp;author=J-P%20Brunet&amp;author=P%20Tamayo&amp;author=TR%20Golub&amp;author=JP%20Mesirov&amp;volume=101&amp;publication_year=2004&amp;pages=4164-4169&amp;pmid=15016911&amp;doi=10.1073/pnas.0308531101&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R17\"> <cite> Cadenas C, Franckenstein D, Schmidt M, Gehrmann M, Hermes M, Geppert B, Schormann W, Maccoux LJ, Schug M, Schumann A, et al. (2010). Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer. Breast Cancer Res. 12, R44. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/bcr2599\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2917039/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20584310/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&amp;title=Role%20of%20thioredoxin%20reductase%201%20and%20thioredoxin%20interacting%20protein%20in%20prognosis%20of%20breast%20cancer&amp;author=C%20Cadenas&amp;author=D%20Franckenstein&amp;author=M%20Schmidt&amp;author=M%20Gehrmann&amp;author=M%20Hermes&amp;volume=12&amp;publication_year=2010&amp;pages=R44&amp;pmid=20584310&amp;doi=10.1186/bcr2599&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R18\"> <cite> Cai J, Fang L, Huang Y, Li R, Yuan J, Yang Y, Zhu X, Chen B, Wu J, and Li M (2013). miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer. Cancer Res. 73, 5402–5415. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-13-0297\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23856247/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=miR-205%20targets%20PTEN%20and%20PHLPP2%20to%20augment%20AKT%20signaling%20and%20drive%20malignant%20phenotypes%20in%20non-small%20cell%20lung%20cancer&amp;author=J%20Cai&amp;author=L%20Fang&amp;author=Y%20Huang&amp;author=R%20Li&amp;author=J%20Yuan&amp;volume=73&amp;publication_year=2013&amp;pages=5402-5415&amp;pmid=23856247&amp;doi=10.1158/0008-5472.CAN-13-0297&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R19\"> <cite> Cammer M, Gevrey J-C, Lorenz M, Dovas A, Condeelis J, and Cox D (2009). The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42. J. Biol. Chem. 284, 23302–23311. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M109.036384\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2749104/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561083/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20mechanism%20of%20CSF-1-induced%20Wiskott-Aldrich%20syndrome%20protein%20activation%20in%20vivo:%20a%20role%20for%20phosphatidylinositol%203-kinase%20and%20Cdc42&amp;author=M%20Cammer&amp;author=J-C%20Gevrey&amp;author=M%20Lorenz&amp;author=A%20Dovas&amp;author=J%20Condeelis&amp;volume=284&amp;publication_year=2009&amp;pages=23302-23311&amp;pmid=19561083&amp;doi=10.1074/jbc.M109.036384&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R20\"> <cite> Campbell JD, Yau C, Bowlby R, Liu Y, Brennan K, Fan H, Taylor AM, Wang C, Walter V, Akbani R, et al. (2018). Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas. Cell Rep. 23, 194–212.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2018.03.063\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6002769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29617660/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Genomic,%20Pathway%20Network,%20and%20Immunologic%20Features%20Distinguishing%20Squamous%20Carcinomas&amp;author=JD%20Campbell&amp;author=C%20Yau&amp;author=R%20Bowlby&amp;author=Y%20Liu&amp;author=K%20Brennan&amp;volume=23&amp;publication_year=2018&amp;pages=194-212.e6&amp;pmid=29617660&amp;doi=10.1016/j.celrep.2018.03.063&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R21\"> <cite> Cannarile MA, Weisser M, Jacob W, Jegg A-M, Ries CH, and Rüttinger D (2017). Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy. J Immunother Cancer 5, 53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s40425-017-0257-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5514481/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28716061/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Colony-stimulating%20factor%201%20receptor%20(CSF1R)%20inhibitors%20in%20cancer%20therapy&amp;author=MA%20Cannarile&amp;author=M%20Weisser&amp;author=W%20Jacob&amp;author=A-M%20Jegg&amp;author=CH%20Ries&amp;volume=5&amp;publication_year=2017&amp;pages=53&amp;pmid=28716061&amp;doi=10.1186/s40425-017-0257-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R22\"> <cite> Cannito S, Novo E, Compagnone A, Valfrè di Bonzo L, Busletta C, Zamara E, Paternostro C, Povero D, Bandino A, Bozzo F, et al. (2008). Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells. Carcinogenesis 29, 2267–2278. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgn216\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18791199/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Redox%20mechanisms%20switch%20on%20hypoxia-dependent%20epithelial-mesenchymal%20transition%20in%20cancer%20cells&amp;author=S%20Cannito&amp;author=E%20Novo&amp;author=A%20Compagnone&amp;author=L%20Valfr%C3%A8%20di%20Bonzo&amp;author=C%20Busletta&amp;volume=29&amp;publication_year=2008&amp;pages=2267-2278&amp;pmid=18791199&amp;doi=10.1093/carcin/bgn216&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R23\"> <cite> Cassetta L, and Pollard JW (2018). Targeting macrophages: therapeutic approaches in cancer. Nat. Rev. Drug Discov. 17, 887–904. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrd.2018.169\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30361552/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov&amp;title=Targeting%20macrophages:%20therapeutic%20approaches%20in%20cancer&amp;author=L%20Cassetta&amp;author=JW%20Pollard&amp;volume=17&amp;publication_year=2018&amp;pages=887-904&amp;pmid=30361552&amp;doi=10.1038/nrd.2018.169&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R24\"> <cite> Chida S, Okayama H, Noda M, Saito K, Nakajima T, Aoto K, Hayase S, Momma T, Ohki S, Kono K, et al. (2016). Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer. Carcinogenesis 37, 878–887. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/carcin/bgw069\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27287872/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Stromal%20VCAN%20expression%20as%20a%20potential%20prognostic%20biomarker%20for%20disease%20recurrence%20in%20stage%20II-III%20colon%20cancer&amp;author=S%20Chida&amp;author=H%20Okayama&amp;author=M%20Noda&amp;author=K%20Saito&amp;author=T%20Nakajima&amp;volume=37&amp;publication_year=2016&amp;pages=878-887&amp;pmid=27287872&amp;doi=10.1093/carcin/bgw069&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R25\"> <cite> Debebe Z, and Rathmell WK (2015). Ror2 as a therapeutic target in cancer. Pharmacol. Ther. 150, 143–148. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.pharmthera.2015.01.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC9236825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25614331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther&amp;title=Ror2%20as%20a%20therapeutic%20target%20in%20cancer&amp;author=Z%20Debebe&amp;author=WK%20Rathmell&amp;volume=150&amp;publication_year=2015&amp;pages=143-148&amp;pmid=25614331&amp;doi=10.1016/j.pharmthera.2015.01.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R26\"> <cite> DeBerardinis RJ, and Chandel NS (2016). Fundamentals of cancer metabolism. Sci Adv 2, e1600200. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/sciadv.1600200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4928883/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27386546/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&amp;title=Fundamentals%20of%20cancer%20metabolism&amp;author=RJ%20DeBerardinis&amp;author=NS%20Chandel&amp;volume=2&amp;publication_year=2016&amp;pages=e1600200&amp;pmid=27386546&amp;doi=10.1126/sciadv.1600200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R27\"> <cite> Deelen P, Bonder MJ, van der Velde KJ, Westra H-J, Winder E, Hendriksen D, Franke L, and Swertz MA (2014). Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration. BMC Res. Notes 7, 901. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1756-0500-7-901\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4307387/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25495213/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&amp;title=Genotype%20harmonizer:%20automatic%20strand%20alignment%20and%20format%20conversion%20for%20genotype%20data%20integration&amp;author=P%20Deelen&amp;author=MJ%20Bonder&amp;author=KJ%20van%20der%20Velde&amp;author=H-J%20Westra&amp;author=E%20Winder&amp;volume=7&amp;publication_year=2014&amp;pages=901&amp;pmid=25495213&amp;doi=10.1186/1756-0500-7-901&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R28\"> <cite> DeYoung MP, Johannessen CM, Leong C-O, Faquin W, Rocco JW, and Ellisen LW (2006). Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma. Cancer Res. 66, 9362–9368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-06-1619\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17018588/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Tumor-specific%20p73%20up-regulation%20mediates%20p63%20dependence%20in%20squamous%20cell%20carcinoma&amp;author=MP%20DeYoung&amp;author=CM%20Johannessen&amp;author=C-O%20Leong&amp;author=W%20Faquin&amp;author=JW%20Rocco&amp;volume=66&amp;publication_year=2006&amp;pages=9362-9368&amp;pmid=17018588&amp;doi=10.1158/0008-5472.CAN-06-1619&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R29\"> <cite> Diviani D, Raimondi F, Del Vescovo CD, Dreyer E, Reggi E, Osman H, Ruggieri L, Gonano C, Cavin S, Box CL, et al. (2016). Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling. Cell Chem Biol 23, 1135–1146. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.chembiol.2016.07.015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27593112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&amp;title=Small-Molecule%20Protein-Protein%20Interaction%20Inhibitor%20of%20Oncogenic%20Rho%20Signaling&amp;author=D%20Diviani&amp;author=F%20Raimondi&amp;author=CD%20Del%20Vescovo&amp;author=E%20Dreyer&amp;author=E%20Reggi&amp;volume=23&amp;publication_year=2016&amp;pages=1135-1146&amp;pmid=27593112&amp;doi=10.1016/j.chembiol.2016.07.015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R30\"> <cite> Dovas A, Gevrey J-C, Grossi A, Park H, Abou-Kheir W, and Cox D (2009). Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages. J. Cell Sci. 122, 3873–3882. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.051755\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2773189/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19808890/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Regulation%20of%20podosome%20dynamics%20by%20WASp%20phosphorylation:%20implication%20in%20matrix%20degradation%20and%20chemotaxis%20in%20macrophages&amp;author=A%20Dovas&amp;author=J-C%20Gevrey&amp;author=A%20Grossi&amp;author=H%20Park&amp;author=W%20Abou-Kheir&amp;volume=122&amp;publication_year=2009&amp;pages=3873-3882&amp;pmid=19808890&amp;doi=10.1242/jcs.051755&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R31\"> <cite> Edelman MJ, Redman MW, Albain KS, McGary EC, Rafique NM, Petro D, Waqar SN, Minichiello K, Miao J, Papadimitrakopoulou VA, et al. (2019). SWOG S1400C ( <a class=\"usa-link\" href=\"https://clinicaltrials.gov/ct2/show/NCT02154490\" rel=\"noopener noreferrer\" target=\"_blank\"> NCT02154490 </a> )-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy). J. Thorac. Oncol. 14, 1853–1859. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2019.06.027\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6764876/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31302234/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=SWOG%20S1400C%20(NCT02154490)-A%20Phase%20II%20Study%20of%20Palbociclib%20for%20Previously%20Treated%20Cell%20Cycle%20Gene%20Alteration-Positive%20Patients%20with%20Stage%20IV%20Squamous%20Cell%20Lung%20Cancer%20(Lung-MAP%20Substudy)&amp;author=MJ%20Edelman&amp;author=MW%20Redman&amp;author=KS%20Albain&amp;author=EC%20McGary&amp;author=NM%20Rafique&amp;volume=14&amp;publication_year=2019&amp;pages=1853-1859&amp;pmid=31302234&amp;doi=10.1016/j.jtho.2019.06.027&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R32\"> <cite> Ellis MJ, Suman VJ, Hoog J, Goncalves R, Sanati S, Creighton CJ, DeSchryver K, Crouch E, Brink A, Watson M, et al. (2017). Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance). J. Clin. Oncol. 35, 1061–1069. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2016.69.4406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5455353/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28045625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Ki67%20Proliferation%20Index%20as%20a%20Tool%20for%20Chemotherapy%20Decisions%20During%20and%20After%20Neoadjuvant%20Aromatase%20Inhibitor%20Treatment%20of%20Breast%20Cancer:%20Results%20From%20the%20American%20College%20of%20Surgeons%20Oncology%20Group%20Z1031%20Trial%20(Alliance)&amp;author=MJ%20Ellis&amp;author=VJ%20Suman&amp;author=J%20Hoog&amp;author=R%20Goncalves&amp;author=S%20Sanati&amp;volume=35&amp;publication_year=2017&amp;pages=1061-1069&amp;pmid=28045625&amp;doi=10.1200/JCO.2016.69.4406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R33\"> <cite> Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B, et al. (2018). The Reactome Pathway Knowledgebase. Nucleic Acids Res. 46, D649–D655. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29145629/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Reactome%20Pathway%20Knowledgebase&amp;author=A%20Fabregat&amp;author=S%20Jupe&amp;author=L%20Matthews&amp;author=K%20Sidiropoulos&amp;author=M%20Gillespie&amp;volume=46&amp;publication_year=2018&amp;pages=D649-D655&amp;pmid=29145629&amp;doi=10.1093/nar/gkx1132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R34\"> <cite> Fan T, Sun G, Sun X, Zhao L, Zhong R, and Peng Y (2019). Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment. Cancers 11. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers11030317\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6468516/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30845728/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Tumor%20Energy%20Metabolism%20and%20Potential%20of%203-Bromopyruvate%20as%20an%20Inhibitor%20of%20Aerobic%20Glycolysis:%20Implications%20in%20Tumor%20Treatment&amp;author=T%20Fan&amp;author=G%20Sun&amp;author=X%20Sun&amp;author=L%20Zhao&amp;author=R%20Zhong&amp;volume=11&amp;publication_year=2019&amp;pmid=30845728&amp;doi=10.3390/cancers11030317&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R35\"> <cite> Faubert B, Li KY, Cai L, Hensley CT, Kim J, Zacharias LG, Yang C, Do QN, Doucette S, Burguete D, et al. (2017). Lactate Metabolism in Human Lung Tumors. Cell 171, 358–371.e9. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.09.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5684706/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28985563/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Lactate%20Metabolism%20in%20Human%20Lung%20Tumors&amp;author=B%20Faubert&amp;author=KY%20Li&amp;author=L%20Cai&amp;author=CT%20Hensley&amp;author=J%20Kim&amp;volume=171&amp;publication_year=2017&amp;pages=358-371.e9&amp;pmid=28985563&amp;doi=10.1016/j.cell.2017.09.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R36\"> <cite> Fernandes AP, Capitanio A, Selenius M, Brodin O, Rundlöf A-K, and Björnstedt M (2009). Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation. Histopathology 55, 313–320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1365-2559.2009.03381.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19723146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Histopathology&amp;title=Expression%20profiles%20of%20thioredoxin%20family%20proteins%20in%20human%20lung%20cancer%20tissue:%20correlation%20with%20proliferation%20and%20differentiation&amp;author=AP%20Fernandes&amp;author=A%20Capitanio&amp;author=M%20Selenius&amp;author=O%20Brodin&amp;author=A-K%20Rundl%C3%B6f&amp;volume=55&amp;publication_year=2009&amp;pages=313-320&amp;pmid=19723146&amp;doi=10.1111/j.1365-2559.2009.03381.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R37\"> <cite> Fisher S, Barry A, Abreu J, Minie B, Nolan J, Delorey TM, Young G, Fennell TJ, Allen A, Ambrogio L, et al. (2011). A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries. Genome Biol. 12, R1. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-1-r1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3091298/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21205303/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=A%20scalable,%20fully%20automated%20process%20for%20construction%20of%20sequence-ready%20human%20exome%20targeted%20capture%20libraries&amp;author=S%20Fisher&amp;author=A%20Barry&amp;author=J%20Abreu&amp;author=B%20Minie&amp;author=J%20Nolan&amp;volume=12&amp;publication_year=2011&amp;pages=R1&amp;pmid=21205303&amp;doi=10.1186/gb-2011-12-1-r1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R38\"> <cite> Foltz DR, Santiago MC, Berechid BE, and Nye JS (2002). Glycogen synthase kinase-3beta modulates notch signaling and stability. Curr. Biol. 12, 1006–1011. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0960-9822(02)00888-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12123574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol&amp;title=Glycogen%20synthase%20kinase-3beta%20modulates%20notch%20signaling%20and%20stability&amp;author=DR%20Foltz&amp;author=MC%20Santiago&amp;author=BE%20Berechid&amp;author=JS%20Nye&amp;volume=12&amp;publication_year=2002&amp;pages=1006-1011&amp;pmid=12123574&amp;doi=10.1016/s0960-9822(02)00888-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R39\"> <cite> Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CM, and Hansen KD (2014). Functional normalization of 450k methylation array data improves replication in large cancer studies. Genome Biol. 15, 503. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0503-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4283580/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25599564/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Functional%20normalization%20of%20450k%20methylation%20array%20data%20improves%20replication%20in%20large%20cancer%20studies&amp;author=J-P%20Fortin&amp;author=A%20Labbe&amp;author=M%20Lemire&amp;author=BW%20Zanke&amp;author=TJ%20Hudson&amp;volume=15&amp;publication_year=2014&amp;pages=503&amp;pmid=25599564&amp;doi=10.1186/s13059-014-0503-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R40\"> <cite> Fraley C, Raftery AE, Scrucca L, Murphy TB, and Fop M (2016). mclust: Gaussian mixture modelling for model-based clustering, classification, and density estimation. R Package Version 5. </cite> [ <a class=\"usa-link\" href=\"/articles/PMC5096736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27818791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=R%20Package%20Version&amp;title=mclust:%20Gaussian%20mixture%20modelling%20for%20model-based%20clustering,%20classification,%20and%20density%20estimation&amp;author=C%20Fraley&amp;author=AE%20Raftery&amp;author=L%20Scrucca&amp;author=TB%20Murphy&amp;author=M%20Fop&amp;volume=5&amp;publication_year=2016&amp;pmid=27818791&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R41\"> <cite> Freed D, Pan R, and Aldana R TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=TNscope:%20Accurate%20Detection%20of%20Somatic%20Mutations%20with%20Haplotype-based%20Variant%20Candidate%20Detection%20and%20Machine%20Learning%20Filtering&amp;author=D%20Freed&amp;author=R%20Pan&amp;author=R%20Aldana&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R42\"> <cite> Fu B, Meng W, Zeng X, Zhao H, Liu W, and Zhang T (2017). TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma. Biomed Res. Int. 2017, 4698167. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1155/2017/4698167\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5425838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28536696/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biomed%20Res.%20Int&amp;title=TXNRD1%20Is%20an%20Unfavorable%20Prognostic%20Factor%20for%20Patients%20with%20Hepatocellular%20Carcinoma&amp;author=B%20Fu&amp;author=W%20Meng&amp;author=X%20Zeng&amp;author=H%20Zhao&amp;author=W%20Liu&amp;volume=2017&amp;publication_year=2017&amp;pages=4698167&amp;pmid=28536696&amp;doi=10.1155/2017/4698167&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R43\"> <cite> Gandhi N, and Das GM (2019). Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications. Cells 8. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cells8020089\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6406734/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30691108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Metabolic%20Reprogramming%20in%20Breast%20Cancer%20and%20Its%20Therapeutic%20Implications&amp;author=N%20Gandhi&amp;author=GM%20Das&amp;volume=8&amp;publication_year=2019&amp;pmid=30691108&amp;doi=10.3390/cells8020089&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R44\"> <cite> Gao C, Zhuang J, Li H, Liu C, Zhou C, Liu L, and Sun C (2018). Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma. Cancer Cell Int. 18, 194. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12935-018-0691-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6258452/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30498398/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&amp;title=Exploration%20of%20methylation-driven%20genes%20for%20monitoring%20and%20prognosis%20of%20patients%20with%20lung%20adenocarcinoma&amp;author=C%20Gao&amp;author=J%20Zhuang&amp;author=H%20Li&amp;author=C%20Liu&amp;author=C%20Zhou&amp;volume=18&amp;publication_year=2018&amp;pages=194&amp;pmid=30498398&amp;doi=10.1186/s12935-018-0691-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R45\"> <cite> Garcia-Mata R, Boulter E, and Burridge K (2011). The “invisible hand”: regulation of RHO GTPases by RHOGDIs. Nat. Rev. Mol. Cell Biol. 12, 493–504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrm3153\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3260518/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21779026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;title=The%20%E2%80%9Cinvisible%20hand%E2%80%9D:%20regulation%20of%20RHO%20GTPases%20by%20RHOGDIs&amp;author=R%20Garcia-Mata&amp;author=E%20Boulter&amp;author=K%20Burridge&amp;volume=12&amp;publication_year=2011&amp;pages=493-504&amp;pmid=21779026&amp;doi=10.1038/nrm3153&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R46\"> <cite> Gaujoux R, and Seoighe C (2010). A flexible R package for nonnegative matrix factorization. BMC Bioinformatics 11, 367. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-11-367\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2912887/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20598126/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=A%20flexible%20R%20package%20for%20nonnegative%20matrix%20factorization&amp;author=R%20Gaujoux&amp;author=C%20Seoighe&amp;volume=11&amp;publication_year=2010&amp;pages=367&amp;pmid=20598126&amp;doi=10.1186/1471-2105-11-367&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R47\"> <cite> Gey S, and Lebarbier E (2008). Using CART to detect multiple change points in the mean for large sample. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Using%20CART%20to%20detect%20multiple%20change%20points%20in%20the%20mean%20for%20large%20sample&amp;author=S%20Gey&amp;author=E%20Lebarbier&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R48\"> <cite> Giaccone G, Zatloukal P, Roubec J, Floor K, Musil J, Kuta M, van Klaveren RJ, Chaudhary S, Gunther A, and Shamsili S (2009). Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer. J. Clin. Oncol. 27, 4481–4486. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.21.1862\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19687333/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Multicenter%20phase%20II%20trial%20of%20YM155,%20a%20small-molecule%20suppressor%20of%20survivin,%20in%20patients%20with%20advanced,%20refractory,%20non-small-cell%20lung%20cancer&amp;author=G%20Giaccone&amp;author=P%20Zatloukal&amp;author=J%20Roubec&amp;author=K%20Floor&amp;author=J%20Musil&amp;volume=27&amp;publication_year=2009&amp;pages=4481-4486&amp;pmid=19687333&amp;doi=10.1200/JCO.2008.21.1862&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R49\"> <cite> Giannakopoulos NV, Luo J-K, Papov V, Zou W, Lenschow DJ, Jacobs BS, Borden EC, Li J, Virgin HW, and Zhang D-E (2005). Proteomic identification of proteins conjugated to ISG15 in mouse and human cells. Biochem. Biophys. Res. Commun. 336, 496–506. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2005.08.132\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16139798/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Proteomic%20identification%20of%20proteins%20conjugated%20to%20ISG15%20in%20mouse%20and%20human%20cells&amp;author=NV%20Giannakopoulos&amp;author=J-K%20Luo&amp;author=V%20Papov&amp;author=W%20Zou&amp;author=DJ%20Lenschow&amp;volume=336&amp;publication_year=2005&amp;pages=496-506&amp;pmid=16139798&amp;doi=10.1016/j.bbrc.2005.08.132&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R50\"> <cite> Gibbs DL (2020). Robust classification of Immune Subtypes in Cancer. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Robust%20classification%20of%20Immune%20Subtypes%20in%20Cancer&amp;author=DL%20Gibbs&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R51\"> <cite> Gillette MA, Satpathy S, Cao S, Dhanasekaran SM, Vasaikar SV, Krug K, Petralia F, Li Y, Liang W-W, Reva B, et al. (2020). Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma. Cell 182, 200–225.e35. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.06.013\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7373300/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32649874/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Characterization%20Reveals%20Therapeutic%20Vulnerabilities%20in%20Lung%20Adenocarcinoma&amp;author=MA%20Gillette&amp;author=S%20Satpathy&amp;author=S%20Cao&amp;author=SM%20Dhanasekaran&amp;author=SV%20Vasaikar&amp;volume=182&amp;publication_year=2020&amp;pages=200-225.e35&amp;pmid=32649874&amp;doi=10.1016/j.cell.2020.06.013&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R52\"> <cite> Gorvel JP, Chang TC, Boretto J, Azuma T, and Chavrier P (1998). Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity. FEBS Lett. 422, 269–273. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/s0014-5793(98)00020-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/9490022/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Differential%20properties%20of%20D4/LyGDI%20versus%20RhoGDI:%20phosphorylation%20and%20rho%20GTPase%20selectivity&amp;author=JP%20Gorvel&amp;author=TC%20Chang&amp;author=J%20Boretto&amp;author=T%20Azuma&amp;author=P%20Chavrier&amp;volume=422&amp;publication_year=1998&amp;pages=269-273&amp;pmid=9490022&amp;doi=10.1016/s0014-5793(98)00020-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R53\"> <cite> Gregory PA, Bert AG, Paterson EL, Barry SC, Tsykin A, Farshid G, Vadas MA, Khew-Goodall Y, and Goodall GJ (2008). The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. Nat. Cell Biol. 10, 593–601. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb1722\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18376396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=The%20miR-200%20family%20and%20miR-205%20regulate%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20ZEB1%20and%20SIP1&amp;author=PA%20Gregory&amp;author=AG%20Bert&amp;author=EL%20Paterson&amp;author=SC%20Barry&amp;author=A%20Tsykin&amp;volume=10&amp;publication_year=2008&amp;pages=593-601&amp;pmid=18376396&amp;doi=10.1038/ncb1722&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R54\"> <cite> Hammerman PS, Lawrence MS, Voet D, Jing R, Cibulskis K, Sivachenko A, Stojanov P, McKenna A, Lander ES, Gabriel S, et al. (2012). Comprehensive genomic characterization of squamous cell lung cancers. Nature 489, 519–525. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature11404\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3466113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22960745/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20characterization%20of%20squamous%20cell%20lung%20cancers&amp;author=PS%20Hammerman&amp;author=MS%20Lawrence&amp;author=D%20Voet&amp;author=R%20Jing&amp;author=K%20Cibulskis&amp;volume=489&amp;publication_year=2012&amp;pages=519-525&amp;pmid=22960745&amp;doi=10.1038/nature11404&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R55\"> <cite> Hänzelmann S, Castelo R, and Guinney J (2013). GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 14, 7. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3618321/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23323831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data&amp;author=S%20H%C3%A4nzelmann&amp;author=R%20Castelo&amp;author=J%20Guinney&amp;volume=14&amp;publication_year=2013&amp;pages=7&amp;pmid=23323831&amp;doi=10.1186/1471-2105-14-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R56\"> <cite> Happel N, Stoldt S, Schmidt B, and Doenecke D (2009). M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3. J. Mol. Biol. 386, 339–350. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2008.12.047\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19136008/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=M%20phase-specific%20phosphorylation%20of%20histone%20H1.5%20at%20threonine%2010%20by%20GSK-3&amp;author=N%20Happel&amp;author=S%20Stoldt&amp;author=B%20Schmidt&amp;author=D%20Doenecke&amp;volume=386&amp;publication_year=2009&amp;pages=339-350&amp;pmid=19136008&amp;doi=10.1016/j.jmb.2008.12.047&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R57\"> <cite> Haslam A, and Prasad V (2019). Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs. JAMA Netw Open 2, e192535. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1001/jamanetworkopen.2019.2535\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6503493/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31050774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw%20Open&amp;title=Estimation%20of%20the%20Percentage%20of%20US%20Patients%20With%20Cancer%20Who%20Are%20Eligible%20for%20and%20Respond%20to%20Checkpoint%20Inhibitor%20Immunotherapy%20Drugs&amp;author=A%20Haslam&amp;author=V%20Prasad&amp;volume=2&amp;publication_year=2019&amp;pages=e192535&amp;pmid=31050774&amp;doi=10.1001/jamanetworkopen.2019.2535&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R58\"> <cite> Hebert AS, Dittenhafer-Reed KE, Yu W, Bailey DJ, Selen ES, Boersma MD, Carson JJ, Tonelli M, Balloon AJ, Higbee AJ, et al. (2013). Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome. Mol. Cell 49, 186–199. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molcel.2012.10.024\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3704155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23201123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Calorie%20restriction%20and%20SIRT3%20trigger%20global%20reprogramming%20of%20the%20mitochondrial%20protein%20acetylome&amp;author=AS%20Hebert&amp;author=KE%20Dittenhafer-Reed&amp;author=W%20Yu&amp;author=DJ%20Bailey&amp;author=ES%20Selen&amp;volume=49&amp;publication_year=2013&amp;pages=186-199&amp;pmid=23201123&amp;doi=10.1016/j.molcel.2012.10.024&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R59\"> <cite> Herbst RS, Morgensztern D, and Boshoff C (2018). The biology and management of non-small cell lung cancer. Nature 553, 446–454. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature25183\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29364287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20biology%20and%20management%20of%20non-small%20cell%20lung%20cancer&amp;author=RS%20Herbst&amp;author=D%20Morgensztern&amp;author=C%20Boshoff&amp;volume=553&amp;publication_year=2018&amp;pages=446-454&amp;pmid=29364287&amp;doi=10.1038/nature25183&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R60\"> <cite> Herreño AM, Ramírez AC, Chaparro VP, Fernandez MJ, Cañas A, Morantes CF, Moreno OM, Brugés RE, Mejía JA, Bustos FJ, et al. (2019). Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter. Tumour Biol. 41, 1010428319851014. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1177/1010428319851014\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31109257/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&amp;title=Role%20of%20RUNX2%20transcription%20factor%20in%20epithelial%20mesenchymal%20transition%20in%20non-small%20cell%20lung%20cancer%20lung%20cancer:%20Epigenetic%20control%20of%20the%20RUNX2%20P1%20promoter&amp;author=AM%20Herre%C3%B1o&amp;author=AC%20Ram%C3%ADrez&amp;author=VP%20Chaparro&amp;author=MJ%20Fernandez&amp;author=A%20Ca%C3%B1as&amp;volume=41&amp;publication_year=2019&amp;pages=1010428319851014&amp;pmid=31109257&amp;doi=10.1177/1010428319851014&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R61\"> <cite> Hitosugi T, and Chen J (2014). Post-translational modifications and the Warburg effect. Oncogene 33, 4279–4285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2013.406\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24096483/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Post-translational%20modifications%20and%20the%20Warburg%20effect&amp;author=T%20Hitosugi&amp;author=J%20Chen&amp;volume=33&amp;publication_year=2014&amp;pages=4279-4285&amp;pmid=24096483&amp;doi=10.1038/onc.2013.406&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R62\"> <cite> Hong C-F, Chen W-Y, and Wu C-W (2017). Upregulation of Wnt signaling under hypoxia promotes lung cancer progression. Oncol. Rep. 38, 1706–1714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/or.2017.5807\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28713928/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Rep&amp;title=Upregulation%20of%20Wnt%20signaling%20under%20hypoxia%20promotes%20lung%20cancer%20progression&amp;author=C-F%20Hong&amp;author=W-Y%20Chen&amp;author=C-W%20Wu&amp;volume=38&amp;publication_year=2017&amp;pages=1706-1714&amp;pmid=28713928&amp;doi=10.3892/or.2017.5807&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R63\"> <cite> Hornbeck PV, Kornhauser JM, Tkachev S, Zhang B, Skrzypek E, Murray B, Latham V, and Sullivan M (2012). PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse. Nucleic Acids Res. 40, D261–D270. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkr1122\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3245126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22135298/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PhosphoSitePlus:%20a%20comprehensive%20resource%20for%20investigating%20the%20structure%20and%20function%20of%20experimentally%20determined%20post-translational%20modifications%20in%20man%20and%20mouse&amp;author=PV%20Hornbeck&amp;author=JM%20Kornhauser&amp;author=S%20Tkachev&amp;author=B%20Zhang&amp;author=E%20Skrzypek&amp;volume=40&amp;publication_year=2012&amp;pages=D261-D270&amp;pmid=22135298&amp;doi=10.1093/nar/gkr1122&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R64\"> <cite> Hsu E-C, Rice MA, Bermudez A, Marques FJG, Aslan M, Liu S, Ghoochani A, Zhang CA, Chen Y-S, Zlitni A, et al. (2020). Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1. Proc. Natl. Acad. Sci. U. S. A. 117, 2032–2042. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1905384117\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6994991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31932422/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Trop2%20is%20a%20driver%20of%20metastatic%20prostate%20cancer%20with%20neuroendocrine%20phenotype%20via%20PARP1&amp;author=E-C%20Hsu&amp;author=MA%20Rice&amp;author=A%20Bermudez&amp;author=FJG%20Marques&amp;author=M%20Aslan&amp;volume=117&amp;publication_year=2020&amp;pages=2032-2042&amp;pmid=31932422&amp;doi=10.1073/pnas.1905384117&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R65\"> <cite> Huang C, Chen L, Savage SR, Eguez RV, Dou Y, Li Y, da Veiga Leprevost F, Jaehnig EJ, Lei JT, Wen B, et al. (2021). Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2020.12.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7946781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33417831/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20insights%20into%20the%20biology%20and%20treatment%20of%20HPV-negative%20head%20and%20neck%20squamous%20cell%20carcinoma&amp;author=C%20Huang&amp;author=L%20Chen&amp;author=SR%20Savage&amp;author=RV%20Eguez&amp;author=Y%20Dou&amp;publication_year=2021&amp;pmid=33417831&amp;doi=10.1016/j.ccell.2020.12.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R66\"> <cite> Huang H-Y, Lin Y-C-D, Li J, Huang K-Y, Shrestha S, Hong H-C, Tang Y, Chen Y-G, Jin C-N, Yu Y, et al. (2020). miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database. Nucleic Acids Res. 48, D148–D154. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz896\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145596/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31647101/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRTarBase%202020:%20updates%20to%20the%20experimentally%20validated%20microRNA-target%20interaction%20database&amp;author=H-Y%20Huang&amp;author=Y-C-D%20Lin&amp;author=J%20Li&amp;author=K-Y%20Huang&amp;author=S%20Shrestha&amp;volume=48&amp;publication_year=2020&amp;pages=D148-D154&amp;pmid=31647101&amp;doi=10.1093/nar/gkz896&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R67\"> <cite> Hubaux R, Thu KL, Vucic EA, Pikor LA, Kung SHY, Martinez VD, Mosslemi M, Becker-Santos DD, Gazdar AF, Lam S, et al. (2015). Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer. Int. J. Cancer 137, 2072–2082. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/ijc.29577\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4537683/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25907283/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&amp;title=Microtubule%20affinity-regulating%20kinase%202%20is%20associated%20with%20DNA%20damage%20response%20and%20cisplatin%20resistance%20in%20non-small%20cell%20lung%20cancer&amp;author=R%20Hubaux&amp;author=KL%20Thu&amp;author=EA%20Vucic&amp;author=LA%20Pikor&amp;author=SHY%20Kung&amp;volume=137&amp;publication_year=2015&amp;pages=2072-2082&amp;pmid=25907283&amp;doi=10.1002/ijc.29577&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R68\"> <cite> Hung YP, Redig A, Hornick JL, and Sholl LM (2020). ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis. Mod. Pathol. 33, 2256–2268. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41379-020-0592-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32572156/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mod.%20Pathol&amp;title=ARID1A%20mutations%20and%20expression%20loss%20in%20non-small%20cell%20lung%20carcinomas:%20clinicopathologic%20and%20molecular%20analysis&amp;author=YP%20Hung&amp;author=A%20Redig&amp;author=JL%20Hornick&amp;author=LM%20Sholl&amp;volume=33&amp;publication_year=2020&amp;pages=2256-2268&amp;pmid=32572156&amp;doi=10.1038/s41379-020-0592-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R69\"> <cite> Ishitani T, Hirao T, Suzuki M, Isoda M, Ishitani S, Harigaya K, Kitagawa M, Matsumoto K, and Itoh M (2010). Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex. Nat. Cell Biol. 12, 278–285. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncb2028\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20118921/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol&amp;title=Nemo-like%20kinase%20suppresses%20Notch%20signalling%20by%20interfering%20with%20formation%20of%20the%20Notch%20active%20transcriptional%20complex&amp;author=T%20Ishitani&amp;author=T%20Hirao&amp;author=M%20Suzuki&amp;author=M%20Isoda&amp;author=S%20Ishitani&amp;volume=12&amp;publication_year=2010&amp;pages=278-285&amp;pmid=20118921&amp;doi=10.1038/ncb2028&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R70\"> <cite> Jimenez-Blasco D, Santofimia-Castaño P, Gonzalez A, Almeida A, and Bolaños JP (2015). Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway. Cell Death Differ. 22, 1877–1889. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cdd.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4648333/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25909891/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Astrocyte%20NMDA%20receptors%E2%80%99%20activity%20sustains%20neuronal%20survival%20through%20a%20Cdk5-Nrf2%20pathway&amp;author=D%20Jimenez-Blasco&amp;author=P%20Santofimia-Casta%C3%B1o&amp;author=A%20Gonzalez&amp;author=A%20Almeida&amp;author=JP%20Bola%C3%B1os&amp;volume=22&amp;publication_year=2015&amp;pages=1877-1889&amp;pmid=25909891&amp;doi=10.1038/cdd.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R71\"> <cite> Jin YH, Kim H, Oh M, Ki H, and Kim K (2009). Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta. Mol. Cells 27, 15–19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/s10059-009-0001-7\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19214430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cells&amp;title=Regulation%20of%20Notch1/NICD%20and%20Hes1%20expressions%20by%20GSK-3alpha/beta&amp;author=YH%20Jin&amp;author=H%20Kim&amp;author=M%20Oh&amp;author=H%20Ki&amp;author=K%20Kim&amp;volume=27&amp;publication_year=2009&amp;pages=15-19&amp;pmid=19214430&amp;doi=10.1007/s10059-009-0001-7&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R72\"> <cite> Johnson WE, Li C, and Rabinovic A (2007). Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostatistics 8, 118–127. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/biostatistics/kxj037\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16632515/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Adjusting%20batch%20effects%20in%20microarray%20expression%20data%20using%20empirical%20Bayes%20methods&amp;author=WE%20Johnson&amp;author=C%20Li&amp;author=A%20Rabinovic&amp;volume=8&amp;publication_year=2007&amp;pages=118-127&amp;pmid=16632515&amp;doi=10.1093/biostatistics/kxj037&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R73\"> <cite> Jurtz V, Paul S, Andreatta M, Marcatili P, Peters B, and Nielsen M (2017). NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data. J. Immunol. 199, 3360–3368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4049/jimmunol.1700893\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5679736/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28978689/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Immunol&amp;title=NetMHCpan-4.0:%20Improved%20Peptide-MHC%20Class%20I%20Interaction%20Predictions%20Integrating%20Eluted%20Ligand%20and%20Peptide%20Binding%20Affinity%20Data&amp;author=V%20Jurtz&amp;author=S%20Paul&amp;author=M%20Andreatta&amp;author=P%20Marcatili&amp;author=B%20Peters&amp;volume=199&amp;publication_year=2017&amp;pages=3360-3368&amp;pmid=28978689&amp;doi=10.4049/jimmunol.1700893&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R74\"> <cite> Kamburov A, Lawrence MS, Polak P, Leshchiner I, Lage K, Golub TR, Lander ES, and Getz G (2015). Comprehensive assessment of cancer missense mutation clustering in protein structures. Proc. Natl. Acad. Sci. U. S. A. 112, E5486–E5495. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1516373112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4603469/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26392535/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Comprehensive%20assessment%20of%20cancer%20missense%20mutation%20clustering%20in%20protein%20structures&amp;author=A%20Kamburov&amp;author=MS%20Lawrence&amp;author=P%20Polak&amp;author=I%20Leshchiner&amp;author=K%20Lage&amp;volume=112&amp;publication_year=2015&amp;pages=E5486-E5495&amp;pmid=26392535&amp;doi=10.1073/pnas.1516373112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R75\"> <cite> Kanehisa M, Furumichi M, Tanabe M, Sato Y, and Morishima K (2017). KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res. 45, D353–D361. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkw1092\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5210567/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27899662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20new%20perspectives%20on%20genomes,%20pathways,%20diseases%20and%20drugs&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=M%20Tanabe&amp;author=Y%20Sato&amp;author=K%20Morishima&amp;volume=45&amp;publication_year=2017&amp;pages=D353-D361&amp;pmid=27899662&amp;doi=10.1093/nar/gkw1092&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R76\"> <cite> Karachaliou N, Fernandez-Bruno M, and Rosell R (2018). Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer? Transl Lung Cancer Res 7, S198–S201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.21037/tlcr.2018.07.02\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6193917/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30393600/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Transl%20Lung%20Cancer%20Res&amp;title=Strategies%20for%20first-line%20immunotherapy%20in%20squamous%20cell%20lung%20cancer:%20are%20combinations%20a%20game%20changer?&amp;author=N%20Karachaliou&amp;author=M%20Fernandez-Bruno&amp;author=R%20Rosell&amp;volume=7&amp;publication_year=2018&amp;pages=S198-S201&amp;pmid=30393600&amp;doi=10.21037/tlcr.2018.07.02&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R77\"> <cite> Kelly RJ, Thomas A, Rajan A, Chun G, Lopez-Chavez A, Szabo E, Spencer S, Carter CA, Guha U, Khozin S, et al. (2013). A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer. Ann. Oncol. 24, 2601–2606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/annonc/mdt249\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3784336/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23857959/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=A%20phase%20I/II%20study%20of%20sepantronium%20bromide%20(YM155,%20survivin%20suppressor)%20with%20paclitaxel%20and%20carboplatin%20in%20patients%20with%20advanced%20non-small-cell%20lung%20cancer&amp;author=RJ%20Kelly&amp;author=A%20Thomas&amp;author=A%20Rajan&amp;author=G%20Chun&amp;author=A%20Lopez-Chavez&amp;volume=24&amp;publication_year=2013&amp;pages=2601-2606&amp;pmid=23857959&amp;doi=10.1093/annonc/mdt249&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R78\"> <cite> Keysar SB, Le PN, Miller B, Jackson BC, Eagles JR, Nieto C, Kim J, Tang B, Glogowska MJ, Morton JJ, et al. (2017). Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2. J. Natl. Cancer Inst. 109. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/jnci/djw189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5025278/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27634934/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&amp;title=Regulation%20of%20Head%20and%20Neck%20Squamous%20Cancer%20Stem%20Cells%20by%20PI3K%20and%20SOX2&amp;author=SB%20Keysar&amp;author=PN%20Le&amp;author=B%20Miller&amp;author=BC%20Jackson&amp;author=JR%20Eagles&amp;volume=109&amp;publication_year=2017&amp;pmid=27634934&amp;doi=10.1093/jnci/djw189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R79\"> <cite> Kikuchi T, Hassanein M, Amann JM, Liu Q, Slebos RJC, Rahman SMJ, Kaufman JM, Zhang X, Hoeksema MD, Harris BK, et al. (2012). In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers. Mol. Cell. Proteomics 11, 916–932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M111.015370\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3494148/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22761400/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=In-depth%20proteomic%20analysis%20of%20nonsmall%20cell%20lung%20cancer%20to%20discover%20molecular%20targets%20and%20candidate%20biomarkers&amp;author=T%20Kikuchi&amp;author=M%20Hassanein&amp;author=JM%20Amann&amp;author=Q%20Liu&amp;author=RJC%20Slebos&amp;volume=11&amp;publication_year=2012&amp;pages=916-932&amp;pmid=22761400&amp;doi=10.1074/mcp.M111.015370&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R80\"> <cite> Kim H, and Park H (2007). Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis. Bioinformatics 23, 1495–1502. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17483501/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Sparse%20non-negative%20matrix%20factorizations%20via%20alternating%20non-negativity-constrained%20least%20squares%20for%20microarray%20data%20analysis&amp;author=H%20Kim&amp;author=H%20Park&amp;volume=23&amp;publication_year=2007&amp;pages=1495-1502&amp;pmid=17483501&amp;doi=10.1093/bioinformatics/btm134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R81\"> <cite> Kim S, and Pevzner PA (2014). MS-GF+ makes progress towards a universal database search tool for proteomics. Nat. Commun. 5, 5277. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms6277\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5036525/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25358478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=MS-GF+%20makes%20progress%20towards%20a%20universal%20database%20search%20tool%20for%20proteomics&amp;author=S%20Kim&amp;author=PA%20Pevzner&amp;volume=5&amp;publication_year=2014&amp;pages=5277&amp;pmid=25358478&amp;doi=10.1038/ncomms6277&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R82\"> <cite> Kim J, Mouw KW, Polak P, Braunstein LZ, Kamburov A, Kwiatkowski DJ, Rosenberg JE, Van Allen EM, D’Andrea A, and Getz G (2016). Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. Nat. Genet. 48, 600–606. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3557\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4936490/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27111033/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&amp;author=J%20Kim&amp;author=KW%20Mouw&amp;author=P%20Polak&amp;author=LZ%20Braunstein&amp;author=A%20Kamburov&amp;volume=48&amp;publication_year=2016&amp;pages=600-606&amp;pmid=27111033&amp;doi=10.1038/ng.3557&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R83\"> <cite> Klepsch V, Hermann-Kleiter N, Do-Dinh P, Jakic B, Offermann A, Efremova M, Sopper S, Rieder D, Krogsdam A, Gamerith G, et al. (2018). Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade. Nat. Commun. 9, 1538. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-018-04004-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5906604/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29670099/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Nuclear%20receptor%20NR2F6%20inhibition%20potentiates%20responses%20to%20PD-L1/PD-1%20cancer%20immune%20checkpoint%20blockade&amp;author=V%20Klepsch&amp;author=N%20Hermann-Kleiter&amp;author=P%20Do-Dinh&amp;author=B%20Jakic&amp;author=A%20Offermann&amp;volume=9&amp;publication_year=2018&amp;pages=1538&amp;pmid=29670099&amp;doi=10.1038/s41467-018-04004-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R84\"> <cite> von Klot C-A, Dubrowinskaja N, Peters I, Hennenlotter J, Merseburger AS, Stenzl A, Kuczyk MA, and Serth J (2017). Rho GDP dissociation inhibitor-β in renal cell carcinoma. Oncol. Lett. 14, 8190–8196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3892/ol.2017.7233\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5727609/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29250194/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett&amp;title=Rho%20GDP%20dissociation%20inhibitor-%CE%B2%20in%20renal%20cell%20carcinoma&amp;author=C-A%20von%20Klot&amp;author=N%20Dubrowinskaja&amp;author=I%20Peters&amp;author=J%20Hennenlotter&amp;author=AS%20Merseburger&amp;volume=14&amp;publication_year=2017&amp;pages=8190-8196&amp;pmid=29250194&amp;doi=10.3892/ol.2017.7233&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R85\"> <cite> Knudsen ES, Nambiar R, Rosario SR, Smiraglia DJ, Goodrich DW, and Witkiewicz AK (2020). Pan-cancer molecular analysis of the RB tumor suppressor pathway. Commun Biol 3, 158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s42003-020-0873-9\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7118159/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32242058/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Pan-cancer%20molecular%20analysis%20of%20the%20RB%20tumor%20suppressor%20pathway&amp;author=ES%20Knudsen&amp;author=R%20Nambiar&amp;author=SR%20Rosario&amp;author=DJ%20Smiraglia&amp;author=DW%20Goodrich&amp;volume=3&amp;publication_year=2020&amp;pages=158&amp;pmid=32242058&amp;doi=10.1038/s42003-020-0873-9&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R86\"> <cite> Krämer A, Green J, Pollard J Jr, and Tugendreich S (2014). Causal analysis approaches in Ingenuity Pathway Analysis. Bioinformatics 30, 523–530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt703\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3928520/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24336805/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Causal%20analysis%20approaches%20in%20Ingenuity%20Pathway%20Analysis&amp;author=A%20Kr%C3%A4mer&amp;author=J%20Green&amp;author=J%20Pollard&amp;author=S%20Tugendreich&amp;volume=30&amp;publication_year=2014&amp;pages=523-530&amp;pmid=24336805&amp;doi=10.1093/bioinformatics/btt703&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R87\"> <cite> Kröger C, Afeyan A, Mraz J, Eaton EN, Reinhardt F, Khodor YL, Thiru P, Bierie B, Ye X, Burge CB, et al. (2019). Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells. Proc. Natl. Acad. Sci. U. S. A. 116, 7353–7362. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1812876116\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6462070/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30910979/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Acquisition%20of%20a%20hybrid%20E/M%20state%20is%20essential%20for%20tumorigenicity%20of%20basal%20breast%20cancer%20cells&amp;author=C%20Kr%C3%B6ger&amp;author=A%20Afeyan&amp;author=J%20Mraz&amp;author=EN%20Eaton&amp;author=F%20Reinhardt&amp;volume=116&amp;publication_year=2019&amp;pages=7353-7362&amp;pmid=30910979&amp;doi=10.1073/pnas.1812876116&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R88\"> <cite> Krug K, Mertins P, Zhang B, Hornbeck P, Raju R, Ahmad R, Szucs M, Mundt F, Forestier D, Jane-Valbuena J, et al. (2018). A curated resource for phosphosite-specific signature analysis. Mol. Cell. Proteomics. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR118.000943\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6398202/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30563849/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20curated%20resource%20for%20phosphosite-specific%20signature%20analysis&amp;author=K%20Krug&amp;author=P%20Mertins&amp;author=B%20Zhang&amp;author=P%20Hornbeck&amp;author=R%20Raju&amp;publication_year=2018&amp;pmid=30563849&amp;doi=10.1074/mcp.TIR118.000943&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R89\"> <cite> Krug K, Jaehnig EJ, Satpathy S, Blumenberg L, Karpova A, Anurag M, Miles G, Mertins P, Geffen Y, Tang LC, et al. (2020). Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy. Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2020.10.036\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8077737/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33212010/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Landscape%20of%20Breast%20Cancer%20Tumorigenesis%20and%20Targeted%20Therapy&amp;author=K%20Krug&amp;author=EJ%20Jaehnig&amp;author=S%20Satpathy&amp;author=L%20Blumenberg&amp;author=A%20Karpova&amp;publication_year=2020&amp;pmid=33212010&amp;doi=10.1016/j.cell.2020.10.036&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R90\"> <cite> Kuhlmann N, Wroblowski S, Knyphausen P, de Boor S, Brenig J, Zienert AY, Meyer-Teschendorf K, Praefcke GJK, Nolte H, Krüger M, et al. (2016a). Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation. J. Biol. Chem. 291, 5484–5499. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M115.707091\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4786691/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26719334/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Structural%20and%20Mechanistic%20Insights%20into%20the%20Regulation%20of%20the%20Fundamental%20Rho%20Regulator%20RhoGDI%CE%B1%20by%20Lysine%20Acetylation&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=P%20Knyphausen&amp;author=S%20de%20Boor&amp;author=J%20Brenig&amp;volume=291&amp;publication_year=2016a&amp;pages=5484-5499&amp;pmid=26719334&amp;doi=10.1074/jbc.M115.707091&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R91\"> <cite> Kuhlmann N, Wroblowski S, Scislowski L, and Lammers M (2016b). RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA. Biochemistry 55, 304–312. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1021/acs.biochem.5b01242\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26695096/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=RhoGDI%CE%B1%20Acetylation%20at%20K127%20and%20K141%20Affects%20Binding%20toward%20Nonprenylated%20RhoA&amp;author=N%20Kuhlmann&amp;author=S%20Wroblowski&amp;author=L%20Scislowski&amp;author=M%20Lammers&amp;volume=55&amp;publication_year=2016b&amp;pages=304-312&amp;pmid=26695096&amp;doi=10.1021/acs.biochem.5b01242&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R92\"> <cite> Lacher SE, and Slattery M (2016). Gene regulatory effects of disease-associated variation in the NRF2 network. Curr Opin Toxicol 1, 71–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cotox.2016.09.001\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5305174/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28203648/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Toxicol&amp;title=Gene%20regulatory%20effects%20of%20disease-associated%20variation%20in%20the%20NRF2%20network&amp;author=SE%20Lacher&amp;author=M%20Slattery&amp;volume=1&amp;publication_year=2016&amp;pages=71-79&amp;pmid=28203648&amp;doi=10.1016/j.cotox.2016.09.001&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R93\"> <cite> Lamb J, Crawford ED, Peck D, Modell JW, Blat IC, Wrobel MJ, Lerner J, Brunet J-P, Subramanian A, Ross KN, et al. (2006). The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 313, 1929–1935. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1132939\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17008526/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Connectivity%20Map:%20using%20gene-expression%20signatures%20to%20connect%20small%20molecules,%20genes,%20and%20disease&amp;author=J%20Lamb&amp;author=ED%20Crawford&amp;author=D%20Peck&amp;author=JW%20Modell&amp;author=IC%20Blat&amp;volume=313&amp;publication_year=2006&amp;pages=1929-1935&amp;pmid=17008526&amp;doi=10.1126/science.1132939&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R94\"> <cite> Lamprecht S, Kaller M, Schmidt EM, Blaj C, Schiergens TS, Engel J, Jung A, Hermeking H, Grünewald TGP, Kirchner T, et al. (2018). PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer. Clin. Cancer Res. 24, 1974–1986. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-2572\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29391352/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=PBX3%20Is%20Part%20of%20an%20EMT%20Regulatory%20Network%20and%20Indicates%20Poor%20Outcome%20in%20Colorectal%20Cancer&amp;author=S%20Lamprecht&amp;author=M%20Kaller&amp;author=EM%20Schmidt&amp;author=C%20Blaj&amp;author=TS%20Schiergens&amp;volume=24&amp;publication_year=2018&amp;pages=1974-1986&amp;pmid=29391352&amp;doi=10.1158/1078-0432.CCR-17-2572&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R95\"> <cite> Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, Meyerson M, Gabriel SB, Lander ES, and Getz G (2014). Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature12912\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4048962/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24390350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Discovery%20and%20saturation%20analysis%20of%20cancer%20genes%20across%2021%20tumour%20types&amp;author=MS%20Lawrence&amp;author=P%20Stojanov&amp;author=CH%20Mermel&amp;author=JT%20Robinson&amp;author=LA%20Garraway&amp;volume=505&amp;publication_year=2014&amp;pages=495-501&amp;pmid=24390350&amp;doi=10.1038/nature12912&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R96\"> <cite> Lê S, Josse J, and Husson F (2008). FactoMineR: AnRPackage for Multivariate Analysis. Journal of Statistical Software 25. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=FactoMineR:%20AnRPackage%20for%20Multivariate%20Analysis&amp;author=S%20L%C3%AA&amp;author=J%20Josse&amp;author=F%20Husson&amp;volume=25&amp;publication_year=2008&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R97\"> <cite> Lebanony D, Benjamin H, Gilad S, Ezagouri M, Dov A, Ashkenazi K, Gefen N, Izraeli S, Rechavi G, Pass H, et al. (2009). Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma. J. Clin. Oncol. 27, 2030–2037. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/JCO.2008.19.4134\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19273703/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&amp;title=Diagnostic%20assay%20based%20on%20hsa-miR-205%20expression%20distinguishes%20squamous%20from%20nonsquamous%20non-small-cell%20lung%20carcinoma&amp;author=D%20Lebanony&amp;author=H%20Benjamin&amp;author=S%20Gilad&amp;author=M%20Ezagouri&amp;author=A%20Dov&amp;volume=27&amp;publication_year=2009&amp;pages=2030-2037&amp;pmid=19273703&amp;doi=10.1200/JCO.2008.19.4134&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R98\"> <cite> Lebarbier E (2005). Detecting multiple change-points in the mean of Gaussian process by model selection. Signal Processing 85, 717–736. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Signal%20Processing&amp;title=Detecting%20multiple%20change-points%20in%20the%20mean%20of%20Gaussian%20process%20by%20model%20selection&amp;author=E%20Lebarbier&amp;volume=85&amp;publication_year=2005&amp;pages=717-736&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R99\"> <cite> Le Large TY, Mantini G, Meijer LL, Pham TV, Funel N, van Grieken NC, Kok B, Knol J, van Laarhoven HW, Piersma SR, et al. (2020). Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets. JCI Insight 5. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1172/jci.insight.138290\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7455080/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32634123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=JCI%20Insight&amp;title=Microdissected%20pancreatic%20cancer%20proteomes%20reveal%20tumor%20heterogeneity%20and%20therapeutic%20targets&amp;author=TY%20Le%20Large&amp;author=G%20Mantini&amp;author=LL%20Meijer&amp;author=TV%20Pham&amp;author=N%20Funel&amp;volume=5&amp;publication_year=2020&amp;pmid=32634123&amp;doi=10.1172/jci.insight.138290&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R100\"> <cite> Lewandowski KT, and Piwnica-Worms H (2014). Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity. J. Cell Sci. 127, 315–327. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.129148\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3889397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24259665/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Phosphorylation%20of%20the%20E3%20ubiquitin%20ligase%20RNF41%20by%20the%20kinase%20Par-1b%20is%20required%20for%20epithelial%20cell%20polarity&amp;author=KT%20Lewandowski&amp;author=H%20Piwnica-Worms&amp;volume=127&amp;publication_year=2014&amp;pages=315-327&amp;pmid=24259665&amp;doi=10.1242/jcs.129148&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R101\"> <cite> Li F, Aljahdali I, and Ling X (2019a). Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study? J. Exp. Clin. Cancer Res. 38, 368. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13046-019-1362-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6704566/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31439015/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Clin.%20Cancer%20Res&amp;title=Cancer%20therapeutics%20using%20survivin%20BIRC5%20as%20a%20target:%20what%20can%20we%20do%20after%20over%20two%20decades%20of%20study?&amp;author=F%20Li&amp;author=I%20Aljahdali&amp;author=X%20Ling&amp;volume=38&amp;publication_year=2019a&amp;pages=368&amp;pmid=31439015&amp;doi=10.1186/s13046-019-1362-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R102\"> <cite> Li K, Vaudel M, Zhang B, Ren Y, and Wen B (2019b). PDV: an integrative proteomics data viewer. Bioinformatics 35, 1249–1251. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty770\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6821182/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30169737/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PDV:%20an%20integrative%20proteomics%20data%20viewer&amp;author=K%20Li&amp;author=M%20Vaudel&amp;author=B%20Zhang&amp;author=Y%20Ren&amp;author=B%20Wen&amp;volume=35&amp;publication_year=2019b&amp;pages=1249-1251&amp;pmid=30169737&amp;doi=10.1093/bioinformatics/bty770&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R103\"> <cite> Li M, Xie X, Zhou J, Sheng M, Yin X, Ko E-A, Zhou T, and Gu W (2017). Quantifying circular RNA expression from RNA-seq data using model-based framework. Bioinformatics 33, 2131–2139. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btx129\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28334396/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quantifying%20circular%20RNA%20expression%20from%20RNA-seq%20data%20using%20model-based%20framework&amp;author=M%20Li&amp;author=X%20Xie&amp;author=J%20Zhou&amp;author=M%20Sheng&amp;author=X%20Yin&amp;volume=33&amp;publication_year=2017&amp;pages=2131-2139&amp;pmid=28334396&amp;doi=10.1093/bioinformatics/btx129&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R104\"> <cite> Li P, Cao G, Zhang Y, Shi J, Cai K, Zhen L, He X, Zhou Y, Li Y, Zhu Y, et al. (2020). FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors. Aging 12, 53–69. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/aging.102564\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6977653/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31935687/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Aging&amp;title=FHL3%20promotes%20pancreatic%20cancer%20invasion%20and%20metastasis%20through%20preventing%20the%20ubiquitination%20degradation%20of%20EMT%20associated%20transcription%20factors&amp;author=P%20Li&amp;author=G%20Cao&amp;author=Y%20Zhang&amp;author=J%20Shi&amp;author=K%20Cai&amp;volume=12&amp;publication_year=2020&amp;pages=53-69&amp;pmid=31935687&amp;doi=10.18632/aging.102564&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R105\"> <cite> Liang Y, Liu M, Wang P, Ding X, and Cao Y (2013). Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer. Genes Chromosomes Cancer 52, 305–315. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/gcc.22030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23225332/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&amp;title=Analysis%20of%2020%20genes%20at%20chromosome%20band%2012q13:%20RACGAP1%20and%20MCRS1%20overexpression%20in%20nonsmall-cell%20lung%20cancer&amp;author=Y%20Liang&amp;author=M%20Liu&amp;author=P%20Wang&amp;author=X%20Ding&amp;author=Y%20Cao&amp;volume=52&amp;publication_year=2013&amp;pages=305-315&amp;pmid=23225332&amp;doi=10.1002/gcc.22030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R106\"> <cite> Liao Y, Wang J, Jaehnig EJ, Shi Z, and Zhang B (2019). WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Nucleic Acids Res. 47, W199–W205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602449/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31114916/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202019:%20gene%20set%20analysis%20toolkit%20with%20revamped%20UIs%20and%20APIs&amp;author=Y%20Liao&amp;author=J%20Wang&amp;author=EJ%20Jaehnig&amp;author=Z%20Shi&amp;author=B%20Zhang&amp;volume=47&amp;publication_year=2019&amp;pages=W199-W205&amp;pmid=31114916&amp;doi=10.1093/nar/gkz401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R107\"> <cite> Liberzon A, Birger C, Thorvaldsdóttir H, Ghandi M, Mesirov JP, and Tamayo P (2015). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst 1, 417–425. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cels.2015.12.004\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4707969/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26771021/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20Molecular%20Signatures%20Database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;author=A%20Liberzon&amp;author=C%20Birger&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=M%20Ghandi&amp;author=JP%20Mesirov&amp;volume=1&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R108\"> <cite> Lim SH, Sun J-M, Choi Y-L, Kim HR, Ahn S, Lee JY, Lee S-H, Ahn JS, Park K, Kim JH, et al. (2016). Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study. Cancer 122, 3024–3031. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/cncr.30135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27315356/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Efficacy%20and%20safety%20of%20dovitinib%20in%20pretreated%20patients%20with%20advanced%20squamous%20non-small%20cell%20lung%20cancer%20with%20FGFR1%20amplification:%20A%20single-arm,%20phase%202%20study&amp;author=SH%20Lim&amp;author=J-M%20Sun&amp;author=Y-L%20Choi&amp;author=HR%20Kim&amp;author=S%20Ahn&amp;volume=122&amp;publication_year=2016&amp;pages=3024-3031&amp;pmid=27315356&amp;doi=10.1002/cncr.30135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R109\"> <cite> Lim YW, Chen-Harris H, Mayba O, Lianoglou S, Wuster A, Bhangale T, Khan Z, Mariathasan S, Daemen A, Reeder J, et al. (2018). Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration. Proc. Natl. Acad. Sci. U. S. A. 115, E11701–E11710. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1804506115\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6294879/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30463956/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=Germline%20genetic%20polymorphisms%20influence%20tumor%20gene%20expression%20and%20immune%20cell%20infiltration&amp;author=YW%20Lim&amp;author=H%20Chen-Harris&amp;author=O%20Mayba&amp;author=S%20Lianoglou&amp;author=A%20Wuster&amp;volume=115&amp;publication_year=2018&amp;pages=E11701-E11710&amp;pmid=30463956&amp;doi=10.1073/pnas.1804506115&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R110\"> <cite> Lindsay J, McDade SS, Pickard A, McCloskey KD, and McCance DJ (2011). Role of DeltaNp63gamma in epithelial to mesenchymal transition. J. Biol. Chem. 286, 3915–3924. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M110.162511\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3030392/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21127042/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Role%20of%20DeltaNp63gamma%20in%20epithelial%20to%20mesenchymal%20transition&amp;author=J%20Lindsay&amp;author=SS%20McDade&amp;author=A%20Pickard&amp;author=KD%20McCloskey&amp;author=DJ%20McCance&amp;volume=286&amp;publication_year=2011&amp;pages=3915-3924&amp;pmid=21127042&amp;doi=10.1074/jbc.M110.162511&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R111\"> <cite> Liu W, Payne SH, Ma S, and Fenyö D (2019). Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis. Mol. Cell. Proteomics 18, S169–S182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.TIR119.001442\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692784/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31213479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Extracting%20Pathway-level%20Signatures%20from%20Proteogenomic%20Data%20in%20Breast%20Cancer%20Using%20Independent%20Component%20Analysis&amp;author=W%20Liu&amp;author=SH%20Payne&amp;author=S%20Ma&amp;author=D%20Feny%C3%B6&amp;volume=18&amp;publication_year=2019&amp;pages=S169-S182&amp;pmid=31213479&amp;doi=10.1074/mcp.TIR119.001442&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R112\"> <cite> Liu X, Wang L, Cui W, Yuan X, Lin L, Cao Q, Wang N, Li Y, Guo W, Zhang X, et al. (2016). Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells. Oncotarget 7, 58516–58530. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.18632/oncotarget.11305\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5295448/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27542268/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Targeting%20ALDH1A1%20by%20disulfiram/copper%20complex%20inhibits%20non-small%20cell%20lung%20cancer%20recurrence%20driven%20by%20ALDH-positive%20cancer%20stem%20cells&amp;author=X%20Liu&amp;author=L%20Wang&amp;author=W%20Cui&amp;author=X%20Yuan&amp;author=L%20Lin&amp;volume=7&amp;publication_year=2016&amp;pages=58516-58530&amp;pmid=27542268&amp;doi=10.18632/oncotarget.11305&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R113\"> <cite> Lo Iacono M, Monica V, Saviozzi S, Ceppi P, Bracco E, Papotti M, and Scagliotti GV (2011). p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue. J. Thorac. Oncol. 6, 473–481. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e31820b86b0\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21289519/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=p63%20and%20p73%20isoform%20expression%20in%20non-small%20cell%20lung%20cancer%20and%20corresponding%20morphological%20normal%20lung%20tissue&amp;author=M%20Lo%20Iacono&amp;author=V%20Monica&amp;author=S%20Saviozzi&amp;author=P%20Ceppi&amp;author=E%20Bracco&amp;volume=6&amp;publication_year=2011&amp;pages=473-481&amp;pmid=21289519&amp;doi=10.1097/JTO.0b013e31820b86b0&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R114\"> <cite> Maksimovic J, Phipson B, and Oshlack A (2016). A cross-package Bioconductor workflow for analysing methylation array data. F1000Res. 5, 1281. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.12688/f1000research.8839.1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4916993/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27347385/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=A%20cross-package%20Bioconductor%20workflow%20for%20analysing%20methylation%20array%20data&amp;author=J%20Maksimovic&amp;author=B%20Phipson&amp;author=A%20Oshlack&amp;volume=5&amp;publication_year=2016&amp;pages=1281&amp;pmid=27347385&amp;doi=10.12688/f1000research.8839.1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R115\"> <cite> Malhotra D, Portales-Casamar E, Singh A, Srivastava S, Arenillas D, Happel C, Shyr C, Wakabayashi N, Kensler TW, Wasserman WW, et al. (2010). Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis. Nucleic Acids Res. 38, 5718–5734. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkq212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2943601/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20460467/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Global%20mapping%20of%20binding%20sites%20for%20Nrf2%20identifies%20novel%20targets%20in%20cell%20survival%20response%20through%20ChIP-Seq%20profiling%20and%20network%20analysis&amp;author=D%20Malhotra&amp;author=E%20Portales-Casamar&amp;author=A%20Singh&amp;author=S%20Srivastava&amp;author=D%20Arenillas&amp;volume=38&amp;publication_year=2010&amp;pages=5718-5734&amp;pmid=20460467&amp;doi=10.1093/nar/gkq212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R116\"> <cite> Mani DR, Maynard M, Kothadia R, Krug K, Christianson KE, Heiman D, Clauser KR, Birger C, Getz G, and Carr SA (2020). PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41592-021-01176-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8791030/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/34040252/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PANOPLY:%20A%20cloud-based%20platform%20for%20automated%20and%20reproducible%20proteogenomic%20data%20analysis&amp;author=DR%20Mani&amp;author=M%20Maynard&amp;author=R%20Kothadia&amp;author=K%20Krug&amp;author=KE%20Christianson&amp;publication_year=2020&amp;pmid=34040252&amp;doi=10.1038/s41592-021-01176-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R117\"> <cite> McInnes L, Healy J, and Astels S (2017). hdbscan: Hierarchical density based clustering. Journal of Open Source Software 2, 205. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Open%20Source%20Software&amp;title=hdbscan:%20Hierarchical%20density%20based%20clustering&amp;author=L%20McInnes&amp;author=J%20Healy&amp;author=S%20Astels&amp;volume=2&amp;publication_year=2017&amp;pages=205&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R118\"> <cite> McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, Flicek P, and Cunningham F (2016). The Ensembl Variant Effect Predictor. Genome Biol. 17, 122. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0974-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4893825/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27268795/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=The%20Ensembl%20Variant%20Effect%20Predictor&amp;author=W%20McLaren&amp;author=L%20Gil&amp;author=SE%20Hunt&amp;author=HS%20Riat&amp;author=GRS%20Ritchie&amp;volume=17&amp;publication_year=2016&amp;pages=122&amp;pmid=27268795&amp;doi=10.1186/s13059-016-0974-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R119\"> <cite> Medvar B, Raghuram V, Pisitkun T, Sarkar A, and Knepper MA (2016). Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation. Physiol. Genomics 48, 502–512. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1152/physiolgenomics.00031.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4967219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27199454/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Physiol.%20Genomics&amp;title=Comprehensive%20database%20of%20human%20E3%20ubiquitin%20ligases:%20application%20to%20aquaporin-2%20regulation&amp;author=B%20Medvar&amp;author=V%20Raghuram&amp;author=T%20Pisitkun&amp;author=A%20Sarkar&amp;author=MA%20Knepper&amp;volume=48&amp;publication_year=2016&amp;pages=502-512&amp;pmid=27199454&amp;doi=10.1152/physiolgenomics.00031.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R120\"> <cite> Merico D, Isserlin R, Stueker O, Emili A, and Bader GD (2010). Enrichment map: a network-based method for gene-set enrichment visualization and interpretation. PLoS One 5, e13984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0013984\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2981572/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21085593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Enrichment%20map:%20a%20network-based%20method%20for%20gene-set%20enrichment%20visualization%20and%20interpretation&amp;author=D%20Merico&amp;author=R%20Isserlin&amp;author=O%20Stueker&amp;author=A%20Emili&amp;author=GD%20Bader&amp;volume=5&amp;publication_year=2010&amp;pages=e13984&amp;pmid=21085593&amp;doi=10.1371/journal.pone.0013984&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R121\"> <cite> Mermel CH, Schumacher SE, Hill B, Meyerson ML, Beroukhim R, and Getz G (2011). GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-4-r41\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3218867/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21527027/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers&amp;author=CH%20Mermel&amp;author=SE%20Schumacher&amp;author=B%20Hill&amp;author=ML%20Meyerson&amp;author=R%20Beroukhim&amp;volume=12&amp;publication_year=2011&amp;pages=R41&amp;pmid=21527027&amp;doi=10.1186/gb-2011-12-4-r41&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R122\"> <cite> Middleton G, Fletcher P, Popat S, Savage J, Summers Y, Greystoke A, Gilligan D, Cave J, O’Rourke N, Brewster A, et al. (2020). The National Lung Matrix Trial of personalized therapy in lung cancer. Nature 583, 807–812. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-2481-8\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7116732/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32669708/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20National%20Lung%20Matrix%20Trial%20of%20personalized%20therapy%20in%20lung%20cancer&amp;author=G%20Middleton&amp;author=P%20Fletcher&amp;author=S%20Popat&amp;author=J%20Savage&amp;author=Y%20Summers&amp;volume=583&amp;publication_year=2020&amp;pages=807-812&amp;pmid=32669708&amp;doi=10.1038/s41586-020-2481-8&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R123\"> <cite> Moissoglu K, McRoberts KS, Meier JA, Theodorescu D, and Schwartz MA (2009). Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases. Cancer Res. 69, 2838–2844. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1397\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2701105/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19276387/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Rho%20GDP%20dissociation%20inhibitor%202%20suppresses%20metastasis%20via%20unconventional%20regulation%20of%20RhoGTPases&amp;author=K%20Moissoglu&amp;author=KS%20McRoberts&amp;author=JA%20Meier&amp;author=D%20Theodorescu&amp;author=MA%20Schwartz&amp;volume=69&amp;publication_year=2009&amp;pages=2838-2844&amp;pmid=19276387&amp;doi=10.1158/0008-5472.CAN-08-1397&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R124\"> <cite> Morrison JA, Pike LA, Sams SB, Sharma V, Zhou Q, Severson JJ, Tan A-C, Wood WM, and Haugen BR (2014). Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer. Mol. Cancer 13, 62. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1476-4598-13-62\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3995095/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24645981/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Thioredoxin%20interacting%20protein%20(TXNIP)%20is%20a%20novel%20tumor%20suppressor%20in%20thyroid%20cancer&amp;author=JA%20Morrison&amp;author=LA%20Pike&amp;author=SB%20Sams&amp;author=V%20Sharma&amp;author=Q%20Zhou&amp;volume=13&amp;publication_year=2014&amp;pages=62&amp;pmid=24645981&amp;doi=10.1186/1476-4598-13-62&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R125\"> <cite> Mouchemore KA, Sampaio NG, Murrey MW, Stanley ER, Lannutti BJ, and Pixley FJ (2013). Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity. FEBS J. 280, 5228–5236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/febs.12316\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3773002/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23648053/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Specific%20inhibition%20of%20PI3K%20p110%CE%B4%20inhibits%20CSF-1-induced%20macrophage%20spreading%20and%20invasive%20capacity&amp;author=KA%20Mouchemore&amp;author=NG%20Sampaio&amp;author=MW%20Murrey&amp;author=ER%20Stanley&amp;author=BJ%20Lannutti&amp;volume=280&amp;publication_year=2013&amp;pages=5228-5236&amp;pmid=23648053&amp;doi=10.1111/febs.12316&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R126\"> <cite> Natan E, and Joerger AC (2012). Structure and kinetic stability of the p63 tetramerization domain. J. Mol. Biol. 415, 503–513. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jmb.2011.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3277882/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22100306/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=Structure%20and%20kinetic%20stability%20of%20the%20p63%20tetramerization%20domain&amp;author=E%20Natan&amp;author=AC%20Joerger&amp;volume=415&amp;publication_year=2012&amp;pages=503-513&amp;pmid=22100306&amp;doi=10.1016/j.jmb.2011.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R127\"> <cite> Newman AM, Liu CL, Green MR, Gentles AJ, Feng W, Xu Y, Hoang CD, Diehn M, and Alizadeh AA (2015). Robust enumeration of cell subsets from tissue expression profiles. Nat. Methods 12, 453–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.3337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4739640/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25822800/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Robust%20enumeration%20of%20cell%20subsets%20from%20tissue%20expression%20profiles&amp;author=AM%20Newman&amp;author=CL%20Liu&amp;author=MR%20Green&amp;author=AJ%20Gentles&amp;author=W%20Feng&amp;volume=12&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;doi=10.1038/nmeth.3337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R128\"> <cite> Nicorici D, Satalan M, Edgren H, Kangaspeska S, Murumagi A, Kallioniemi O, Virtanen S, and Kilkku O FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=FusionCatcher%20-%20a%20tool%20for%20finding%20somatic%20fusion%20genes%20in%20paired-end%20RNA-sequencing%20data&amp;author=D%20Nicorici&amp;author=M%20Satalan&amp;author=H%20Edgren&amp;author=S%20Kangaspeska&amp;author=A%20Murumagi&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R129\"> <cite> Nijman SMB, Luna-Vargas MPA, Velds A, Brummelkamp TR, Dirac AMG, Sixma TK, and Bernards R (2005). A genomic and functional inventory of deubiquitinating enzymes. Cell 123, 773–786. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2005.11.007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16325574/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20genomic%20and%20functional%20inventory%20of%20deubiquitinating%20enzymes&amp;author=SMB%20Nijman&amp;author=MPA%20Luna-Vargas&amp;author=A%20Velds&amp;author=TR%20Brummelkamp&amp;author=AMG%20Dirac&amp;volume=123&amp;publication_year=2005&amp;pages=773-786&amp;pmid=16325574&amp;doi=10.1016/j.cell.2005.11.007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R130\"> <cite> Niu D-F, Kondo T, Nakazawa T, Oishi N, Kawasaki T, Mochizuki K, Yamane T, and Katoh R (2012). Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas. Lab. Invest. 92, 1181–1190. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/labinvest.2012.84\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22641097/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lab.%20Invest&amp;title=Transcription%20factor%20Runx2%20is%20a%20regulator%20of%20epithelial-mesenchymal%20transition%20and%20invasion%20in%20thyroid%20carcinomas&amp;author=D-F%20Niu&amp;author=T%20Kondo&amp;author=T%20Nakazawa&amp;author=N%20Oishi&amp;author=T%20Kawasaki&amp;volume=92&amp;publication_year=2012&amp;pages=1181-1190&amp;pmid=22641097&amp;doi=10.1038/labinvest.2012.84&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R131\"> <cite> Obenchain V, Lawrence M, Carey V, Gogarten S, Shannon P, and Morgan M (2014). VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants. Bioinformatics 30, 2076–2078. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu168\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080743/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24681907/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=VariantAnnotation:%20a%20Bioconductor%20package%20for%20exploration%20and%20annotation%20of%20genetic%20variants&amp;author=V%20Obenchain&amp;author=M%20Lawrence&amp;author=V%20Carey&amp;author=S%20Gogarten&amp;author=P%20Shannon&amp;volume=30&amp;publication_year=2014&amp;pages=2076-2078&amp;pmid=24681907&amp;doi=10.1093/bioinformatics/btu168&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R132\"> <cite> Paik PK, Pillai RN, Lathan CS, Velasco SA, and Papadimitrakopoulou V (2019). New Treatment Options in Advanced Squamous Cell Lung Cancer. Am Soc Clin Oncol Educ Book 39, e198–e206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1200/EDBK_237829\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31099625/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am%20Soc%20Clin%20Oncol%20Educ%20Book&amp;title=New%20Treatment%20Options%20in%20Advanced%20Squamous%20Cell%20Lung%20Cancer&amp;author=PK%20Paik&amp;author=RN%20Pillai&amp;author=CS%20Lathan&amp;author=SA%20Velasco&amp;author=V%20Papadimitrakopoulou&amp;volume=39&amp;publication_year=2019&amp;pages=e198-e206&amp;pmid=31099625&amp;doi=10.1200/EDBK_237829&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R133\"> <cite> Parra ER, Jiang M, Solis L, Mino B, Laberiano C, Hernandez S, Gite S, Verma A, Tetzlaff M, Haymaker C, et al. (2020). Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies. Cancers 12. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3390/cancers12020255\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7072187/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31972974/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Procedural%20Requirements%20and%20Recommendations%20for%20Multiplex%20Immunofluorescence%20Tyramide%20Signal%20Amplification%20Assays%20to%20Support%20Translational%20Oncology%20Studies&amp;author=ER%20Parra&amp;author=M%20Jiang&amp;author=L%20Solis&amp;author=B%20Mino&amp;author=C%20Laberiano&amp;volume=12&amp;publication_year=2020&amp;pmid=31972974&amp;doi=10.3390/cancers12020255&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R134\"> <cite> Patel M, Lu L, Zander DS, Sreerama L, Coco D, and Moreb JS (2008). ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors. Lung Cancer 59, 340–349. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.lungcan.2007.08.033\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17920722/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=ALDH1A1%20and%20ALDH3A1%20expression%20in%20lung%20cancers:%20correlation%20with%20histologic%20type%20and%20potential%20precursors&amp;author=M%20Patel&amp;author=L%20Lu&amp;author=DS%20Zander&amp;author=L%20Sreerama&amp;author=D%20Coco&amp;volume=59&amp;publication_year=2008&amp;pages=340-349&amp;pmid=17920722&amp;doi=10.1016/j.lungcan.2007.08.033&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R135\"> <cite> Patnaik A, Rosen LS, Tolaney SM, Tolcher AW, Goldman JW, Gandhi L, Papadopoulos KP, Beeram M, Rasco DW, Hilton JF, et al. (2016). Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors. Cancer Discov. 6, 740–753. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/2159-8290.CD-16-0095\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27217383/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Efficacy%20and%20Safety%20of%20Abemaciclib,%20an%20Inhibitor%20of%20CDK4%20and%20CDK6,%20for%20Patients%20with%20Breast%20Cancer,%20Non-Small%20Cell%20Lung%20Cancer,%20and%20Other%20Solid%20Tumors&amp;author=A%20Patnaik&amp;author=LS%20Rosen&amp;author=SM%20Tolaney&amp;author=AW%20Tolcher&amp;author=JW%20Goldman&amp;volume=6&amp;publication_year=2016&amp;pages=740-753&amp;pmid=27217383&amp;doi=10.1158/2159-8290.CD-16-0095&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R136\"> <cite> Payen VL, Mina E, Van Hée VF, Porporato PE, and Sonveaux P (2020). Monocarboxylate transporters in cancer. Mol Metab 33, 48–66. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.molmet.2019.07.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7056923/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395464/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Metab&amp;title=Monocarboxylate%20transporters%20in%20cancer&amp;author=VL%20Payen&amp;author=E%20Mina&amp;author=VF%20Van%20H%C3%A9e&amp;author=PE%20Porporato&amp;author=P%20Sonveaux&amp;volume=33&amp;publication_year=2020&amp;pages=48-66&amp;pmid=31395464&amp;doi=10.1016/j.molmet.2019.07.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R137\"> <cite> Paz-Ares L, Luft A, Vicente D, Tafreshi A, Gümüş M, Mazières J, Hermes B, Çay Şenler F, Csőszi T, Fülöp A, et al. (2018). Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer. N. Engl. J. Med. 379, 2040–2051. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1056/NEJMoa1810865\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30280635/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&amp;title=Pembrolizumab%20plus%20Chemotherapy%20for%20Squamous%20Non-Small-Cell%20Lung%20Cancer&amp;author=L%20Paz-Ares&amp;author=A%20Luft&amp;author=D%20Vicente&amp;author=A%20Tafreshi&amp;author=M%20G%C3%BCm%C3%BC%C5%9F&amp;volume=379&amp;publication_year=2018&amp;pages=2040-2051&amp;pmid=30280635&amp;doi=10.1056/NEJMoa1810865&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R138\"> <cite> Pedersen BS, Layer RM, and Quinlan AR (2016). Vcfanno: fast, flexible annotation of genetic variants. Genome Biol. 17, 118. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-016-0973-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4888505/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27250555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Vcfanno:%20fast,%20flexible%20annotation%20of%20genetic%20variants&amp;author=BS%20Pedersen&amp;author=RM%20Layer&amp;author=AR%20Quinlan&amp;volume=17&amp;publication_year=2016&amp;pages=118&amp;pmid=27250555&amp;doi=10.1186/s13059-016-0973-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R139\"> <cite> Peng A, Mao X, Zhong J, Fan S, and Hu Y (2020). Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology. Proteomics 20, e1900271. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/pmic.201900271\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32223079/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Single-Cell%20Multi-Omics%20and%20Its%20Prospective%20Application%20in%20Cancer%20Biology&amp;author=A%20Peng&amp;author=X%20Mao&amp;author=J%20Zhong&amp;author=S%20Fan&amp;author=Y%20Hu&amp;volume=20&amp;publication_year=2020&amp;pages=e1900271&amp;pmid=32223079&amp;doi=10.1002/pmic.201900271&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R140\"> <cite> Peters LA, Perrigoue J, Mortha A, Iuga A, Song W-M, Neiman EM, Llewellyn SR, Di Narzo A, Kidd BA, Telesco SE, et al. (2017). A functional genomics predictive network model identifies regulators of inflammatory bowel disease. Nat. Genet. 49, 1437–1449. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3947\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5660607/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28892060/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=A%20functional%20genomics%20predictive%20network%20model%20identifies%20regulators%20of%20inflammatory%20bowel%20disease&amp;author=LA%20Peters&amp;author=J%20Perrigoue&amp;author=A%20Mortha&amp;author=A%20Iuga&amp;author=W-M%20Song&amp;volume=49&amp;publication_year=2017&amp;pages=1437-1449&amp;pmid=28892060&amp;doi=10.1038/ng.3947&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R141\"> <cite> Petralia F, Wang L, Peng J, Yan A, Zhu J, and Wang P (2018). A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity. Bioinformatics 34, i528–i536. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/bty280\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6022554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29949994/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20method%20for%20constructing%20tumor%20specific%20gene%20co-expression%20networks%20based%20on%20samples%20with%20tumor%20purity%20heterogeneity&amp;author=F%20Petralia&amp;author=L%20Wang&amp;author=J%20Peng&amp;author=A%20Yan&amp;author=J%20Zhu&amp;volume=34&amp;publication_year=2018&amp;pages=i528-i536&amp;pmid=29949994&amp;doi=10.1093/bioinformatics/bty280&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R142\"> <cite> Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, de Bakker PIW, Daly MJ, et al. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1086/519795\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1950838/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17701901/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&amp;title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S%20Purcell&amp;author=B%20Neale&amp;author=K%20Todd-Brown&amp;author=L%20Thomas&amp;author=MAR%20Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R143\"> <cite> Qian J, and Massion PP (2008). Role of chromosome 3q amplification in lung cancer. J. Thorac. Oncol. 3, 212–215. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1097/JTO.0b013e3181663544\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18317062/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Role%20of%20chromosome%203q%20amplification%20in%20lung%20cancer&amp;author=J%20Qian&amp;author=PP%20Massion&amp;volume=3&amp;publication_year=2008&amp;pages=212-215&amp;pmid=18317062&amp;doi=10.1097/JTO.0b013e3181663544&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R144\"> <cite> Ramadoss S, Sen S, Ramachandran I, Roy S, Chaudhuri G, and Farias-Eisner R (2017). Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance. Oncogene 36, 6508. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/onc.2017.331\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5701089/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28925393/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Lysine-specific%20demethylase%20KDM3A%20regulates%20ovarian%20cancer%20stemness%20and%20chemoresistance&amp;author=S%20Ramadoss&amp;author=S%20Sen&amp;author=I%20Ramachandran&amp;author=S%20Roy&amp;author=G%20Chaudhuri&amp;volume=36&amp;publication_year=2017&amp;pages=6508&amp;pmid=28925393&amp;doi=10.1038/onc.2017.331&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R145\"> <cite> Raudvere U, Kolberg L, Kuzmin I, Arak T, Adler P, Peterson H, and Vilo J (2019). g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). Nucleic Acids Res. 47, W191–W198. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkz369\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6602461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31066453/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=g:Profiler:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20(2019%20update)&amp;author=U%20Raudvere&amp;author=L%20Kolberg&amp;author=I%20Kuzmin&amp;author=T%20Arak&amp;author=P%20Adler&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;doi=10.1093/nar/gkz369&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R146\"> <cite> Reese SE, Archer KJ, Therneau TM, Atkinson EJ, Vachon CM, de Andrade M, Kocher J-PA, and Eckel-Passow JE (2013). A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis. Bioinformatics 29, 2877–2883. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt480\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810845/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23958724/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20statistic%20for%20identifying%20batch%20effects%20in%20high-throughput%20genomic%20data%20that%20uses%20guided%20principal%20component%20analysis&amp;author=SE%20Reese&amp;author=KJ%20Archer&amp;author=TM%20Therneau&amp;author=EJ%20Atkinson&amp;author=CM%20Vachon&amp;volume=29&amp;publication_year=2013&amp;pages=2877-2883&amp;pmid=23958724&amp;doi=10.1093/bioinformatics/btt480&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R147\"> <cite> Reva B, Omelchenko T, Calinawan A, Nair S, Schadt E, and Tewari A (2020). Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-γ axis and identification of immune resistance mechanisms. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Prioritization%20prostate%20cancer%20to%20immune%20checkpoint%20therapy%20by%20ranking%20tumors%20along%20IFN-%CE%B3%20axis%20and%20identification%20of%20immune%20resistance%20mechanisms&amp;author=B%20Reva&amp;author=T%20Omelchenko&amp;author=A%20Calinawan&amp;author=S%20Nair&amp;author=E%20Schadt&amp;publication_year=2020&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R148\"> <cite> Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, and Smyth GK (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkv007\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4402510/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25605792/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;author=ME%20Ritchie&amp;author=B%20Phipson&amp;author=D%20Wu&amp;author=Y%20Hu&amp;author=CW%20Law&amp;volume=43&amp;publication_year=2015&amp;pages=e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R149\"> <cite> Robinson DR, Wu Y-M, Lonigro RJ, Vats P, Cobain E, Everett J, Cao X, Rabban E, Kumar-Sinha C, Raymond V, et al. (2017). Integrative clinical genomics of metastatic cancer. Nature 548, 297–303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature23306\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5995337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28783718/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative%20clinical%20genomics%20of%20metastatic%20cancer&amp;author=DR%20Robinson&amp;author=Y-M%20Wu&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=E%20Cobain&amp;volume=548&amp;publication_year=2017&amp;pages=297-303&amp;pmid=28783718&amp;doi=10.1038/nature23306&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R150\"> <cite> Roskoski R Jr (2018). The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders. Pharmacol. Res. 129, 65–83. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.phrs.2018.01.021\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29408302/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res&amp;title=The%20role%20of%20small%20molecule%20platelet-derived%20growth%20factor%20receptor%20(PDGFR)%20inhibitors%20in%20the%20treatment%20of%20neoplastic%20disorders&amp;author=R%20Roskoski&amp;volume=129&amp;publication_year=2018&amp;pages=65-83&amp;pmid=29408302&amp;doi=10.1016/j.phrs.2018.01.021&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R151\"> <cite> Satpathy S, Jaehnig EJ, Krug K, Kim B-J, Saltzman AB, Chan DW, Holloway KR, Anurag M, Huang C, Singh P, et al. (2020). Microscaled proteogenomic methods for precision oncology. Nat. Commun. 11, 532. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-14381-2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6985126/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31988290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Microscaled%20proteogenomic%20methods%20for%20precision%20oncology&amp;author=S%20Satpathy&amp;author=EJ%20Jaehnig&amp;author=K%20Krug&amp;author=B-J%20Kim&amp;author=AB%20Saltzman&amp;volume=11&amp;publication_year=2020&amp;pages=532&amp;pmid=31988290&amp;doi=10.1038/s41467-020-14381-2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R152\"> <cite> Schubert M, Klinger B, Klünemann M, Sieber A, Uhlitz F, Sauer S, Garnett MJ, Blüthgen N, and Saez-Rodriguez J (2018). Perturbation-response genes reveal signaling footprints in cancer gene expression. Nat. Commun. 9, 20. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-017-02391-6\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5750219/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29295995/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Perturbation-response%20genes%20reveal%20signaling%20footprints%20in%20cancer%20gene%20expression&amp;author=M%20Schubert&amp;author=B%20Klinger&amp;author=M%20Kl%C3%BCnemann&amp;author=A%20Sieber&amp;author=F%20Uhlitz&amp;volume=9&amp;publication_year=2018&amp;pages=20&amp;pmid=29295995&amp;doi=10.1038/s41467-017-02391-6&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R153\"> <cite> Shadforth IP, Dunkley TPJ, Lilley KS, and Bessant C (2005). i-Tracker: for quantitative proteomics using iTRAQ. BMC Genomics 6, 145. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2164-6-145\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1276793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16242023/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=i-Tracker:%20for%20quantitative%20proteomics%20using%20iTRAQ&amp;author=IP%20Shadforth&amp;author=TPJ%20Dunkley&amp;author=KS%20Lilley&amp;author=C%20Bessant&amp;volume=6&amp;publication_year=2005&amp;pages=145&amp;pmid=16242023&amp;doi=10.1186/1471-2164-6-145&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R154\"> <cite> Shahamatdar S, He MX, Reyna MA, Gusev A, AlDubayan SH, Van Allen EM, and Ramachandran S (2020). Germline Features Associated with Immune Infiltration in Solid Tumors. Cell Rep. 30, 2900–2908.e4. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2020.02.039\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7082123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32130895/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Germline%20Features%20Associated%20with%20Immune%20Infiltration%20in%20Solid%20Tumors&amp;author=S%20Shahamatdar&amp;author=MX%20He&amp;author=MA%20Reyna&amp;author=A%20Gusev&amp;author=SH%20AlDubayan&amp;volume=30&amp;publication_year=2020&amp;pages=2900-2908.e4&amp;pmid=32130895&amp;doi=10.1016/j.celrep.2020.02.039&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R155\"> <cite> Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, and Ideker T (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–2504. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.1239303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC403769/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14597658/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R156\"> <cite> Shen J, Ju Z, Zhao W, Wang L, Peng Y, Ge Z, Nagel ZD, Zou J, Wang C, Kapoor P, et al. (2018). ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade. Nat. Med. 24, 556–562. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41591-018-0012-z\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6076433/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29736026/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=ARID1A%20deficiency%20promotes%20mutability%20and%20potentiates%20therapeutic%20antitumor%20immunity%20unleashed%20by%20immune%20checkpoint%20blockade&amp;author=J%20Shen&amp;author=Z%20Ju&amp;author=W%20Zhao&amp;author=L%20Wang&amp;author=Y%20Peng&amp;volume=24&amp;publication_year=2018&amp;pages=556-562&amp;pmid=29736026&amp;doi=10.1038/s41591-018-0012-z&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R157\"> <cite> Showe MK, Vachani A, Kossenkov AV, Yousef M, Nichols C, Nikonova EV, Chang C, Kucharczuk J, Tran B, Wakeam E, et al. (2009). Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease. Cancer Res. 69, 9202–9210. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-09-1378\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2798582/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19951989/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Gene%20expression%20profiles%20in%20peripheral%20blood%20mononuclear%20cells%20can%20distinguish%20patients%20with%20non-small%20cell%20lung%20cancer%20from%20patients%20with%20nonmalignant%20lung%20disease&amp;author=MK%20Showe&amp;author=A%20Vachani&amp;author=AV%20Kossenkov&amp;author=M%20Yousef&amp;author=C%20Nichols&amp;volume=69&amp;publication_year=2009&amp;pages=9202-9210&amp;pmid=19951989&amp;doi=10.1158/0008-5472.CAN-09-1378&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R158\"> <cite> Siegel RL, Miller KD, and Jemal A (2020). Cancer statistics, 2020. CA Cancer J. Clin. 70, 7–30. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.3322/caac.21590\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31912902/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin&amp;title=Cancer%20statistics,%202020&amp;author=RL%20Siegel&amp;author=KD%20Miller&amp;author=A%20Jemal&amp;volume=70&amp;publication_year=2020&amp;pages=7-30&amp;pmid=31912902&amp;doi=10.3322/caac.21590&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R159\"> <cite> Singh A, Boldin-Adamsky S, Thimmulappa RK, Rath SK, Ashush H, Coulter J, Blackford A, Goodman SN, Bunz F, Watson WH, et al. (2008). RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy. Cancer Res. 68, 7975–7984. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-08-1401\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3070411/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18829555/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=RNAi-mediated%20silencing%20of%20nuclear%20factor%20erythroid-2-related%20factor%202%20gene%20expression%20in%20non-small%20cell%20lung%20cancer%20inhibits%20tumor%20growth%20and%20increases%20efficacy%20of%20chemotherapy&amp;author=A%20Singh&amp;author=S%20Boldin-Adamsky&amp;author=RK%20Thimmulappa&amp;author=SK%20Rath&amp;author=H%20Ashush&amp;volume=68&amp;publication_year=2008&amp;pages=7975-7984&amp;pmid=18829555&amp;doi=10.1158/0008-5472.CAN-08-1401&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R160\"> <cite> Sinicropi-Yao SL, Amann JM, Lopez DLY, Cerciello F, Coombes KR, and Carbone DP (2019). Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC. J. Thorac. Oncol. 14, 223–236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.jtho.2018.10.162\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6348021/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30408569/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol&amp;title=Co-Expression%20Analysis%20Reveals%20Mechanisms%20Underlying%20the%20Varied%20Roles%20of%20NOTCH1%20in%20NSCLC&amp;author=SL%20Sinicropi-Yao&amp;author=JM%20Amann&amp;author=DLY%20Lopez&amp;author=F%20Cerciello&amp;author=KR%20Coombes&amp;volume=14&amp;publication_year=2019&amp;pages=223-236&amp;pmid=30408569&amp;doi=10.1016/j.jtho.2018.10.162&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R161\"> <cite> Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, and Forbes SA (2018). The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat. Rev. Cancer 18, 696–705. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41568-018-0060-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6450507/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30293088/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=The%20COSMIC%20Cancer%20Gene%20Census:%20describing%20genetic%20dysfunction%20across%20all%20human%20cancers&amp;author=Z%20Sondka&amp;author=S%20Bamford&amp;author=CG%20Cole&amp;author=SA%20Ward&amp;author=I%20Dunham&amp;volume=18&amp;publication_year=2018&amp;pages=696-705&amp;pmid=30293088&amp;doi=10.1038/s41568-018-0060-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R162\"> <cite> Song X, Ji J, Gleason KJ, Yang F, Martignetti JA, Chen LS, and Wang P (2019). Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis. Mol. Cell. Proteomics 18, S52–S65. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.RA118.001220\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6692782/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31227599/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Insights%20into%20Impact%20of%20DNA%20Copy%20Number%20Alteration%20and%20Methylation%20on%20the%20Proteogenomic%20Landscape%20of%20Human%20Ovarian%20Cancer%20via%20a%20Multi-omics%20Integrative%20Analysis&amp;author=X%20Song&amp;author=J%20Ji&amp;author=KJ%20Gleason&amp;author=F%20Yang&amp;author=JA%20Martignetti&amp;volume=18&amp;publication_year=2019&amp;pages=S52-S65&amp;pmid=31227599&amp;doi=10.1074/mcp.RA118.001220&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R163\"> <cite> Spitzer MH, and Nolan GP (2016). Mass Cytometry: Single Cells, Many Features. Cell 165, 780–791. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2016.04.019\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4860251/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27153492/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Many%20Features.%20Cell&amp;title=Mass%20Cytometry:%20Single%20Cells&amp;author=MH%20Spitzer&amp;author=GP%20Nolan&amp;volume=165&amp;publication_year=2016&amp;pages=780-791&amp;pmid=27153492&amp;doi=10.1016/j.cell.2016.04.019&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R164\"> <cite> Srivastava K, Pickard A, Craig SG, Quinn GP, Lambe SM, James JA, McDade SS, and McCance DJ (2018). ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers. Clin. Cancer Res. 24, 3917–3927. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-17-3775\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6098695/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29739791/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=%CE%94Np63%CE%B3/SRC/Slug%20Signaling%20Axis%20Promotes%20Epithelial-to-Mesenchymal%20Transition%20in%20Squamous%20Cancers&amp;author=K%20Srivastava&amp;author=A%20Pickard&amp;author=SG%20Craig&amp;author=GP%20Quinn&amp;author=SM%20Lambe&amp;volume=24&amp;publication_year=2018&amp;pages=3917-3927&amp;pmid=29739791&amp;doi=10.1158/1078-0432.CCR-17-3775&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R165\"> <cite> Stewart PA, Welsh EA, Slebos RJC, Fang B, Izumi V, Chambers M, Zhang G, Cen L, Pettersson F, Zhang Y, et al. (2019). Proteogenomic landscape of squamous cell lung cancer. Nat. Commun. 10, 3578. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-11452-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6687710/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31395880/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Proteogenomic%20landscape%20of%20squamous%20cell%20lung%20cancer&amp;author=PA%20Stewart&amp;author=EA%20Welsh&amp;author=RJC%20Slebos&amp;author=B%20Fang&amp;author=V%20Izumi&amp;volume=10&amp;publication_year=2019&amp;pages=3578&amp;pmid=31395880&amp;doi=10.1038/s41467-019-11452-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R166\"> <cite> Su J, Morgani SM, David CJ, Wang Q, Er EE, Huang Y-H, Basnet H, Zou Y, Shu W, Soni RK, et al. (2020). TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1. Nature 577, 566–571. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-019-1897-5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7450666/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31915377/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=TGF-%CE%B2%20orchestrates%20fibrogenic%20and%20developmental%20EMTs%20via%20the%20RAS%20effector%20RREB1&amp;author=J%20Su&amp;author=SM%20Morgani&amp;author=CJ%20David&amp;author=Q%20Wang&amp;author=EE%20Er&amp;volume=577&amp;publication_year=2020&amp;pages=566-571&amp;pmid=31915377&amp;doi=10.1038/s41586-019-1897-5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R167\"> <cite> Subramanian A, Narayan R, Corsello SM, Peck DD, Natoli TE, Lu X, Gould J, Davis JF, Tubelli AA, Asiedu JK, et al. (2017). A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles. Cell 171, 1437–1452.e17. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.10.049\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5990023/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29195078/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Next%20Generation%20Connectivity%20Map:%20L1000%20Platform%20and%20the%20First%201,000,000%20Profiles&amp;author=A%20Subramanian&amp;author=R%20Narayan&amp;author=SM%20Corsello&amp;author=DD%20Peck&amp;author=TE%20Natoli&amp;volume=171&amp;publication_year=2017&amp;pages=1437-1452.e17&amp;pmid=29195078&amp;doi=10.1016/j.cell.2017.10.049&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R168\"> <cite> Szolek A, Schubert B, Mohr C, Sturm M, Feldhahn M, and Kohlbacher O (2014). OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics 30, 3310–3316. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btu548\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4441069/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25143287/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=OptiType:%20precision%20HLA%20typing%20from%20next-generation%20sequencing%20data&amp;author=A%20Szolek&amp;author=B%20Schubert&amp;author=C%20Mohr&amp;author=M%20Sturm&amp;author=M%20Feldhahn&amp;volume=30&amp;publication_year=2014&amp;pages=3310-3316&amp;pmid=25143287&amp;doi=10.1093/bioinformatics/btu548&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R169\"> <cite> Tam WL, Lu H, Buikhuisen J, Soh BS, Lim E, Reinhardt F, Wu ZJ, Krall JA, Bierie B, Guo W, et al. (2013). Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells. Cancer Cell 24, 347–364. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccr.2013.08.005\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4001722/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24029232/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Protein%20kinase%20C%20%CE%B1%20is%20a%20central%20signaling%20node%20and%20therapeutic%20target%20for%20breast%20cancer%20stem%20cells&amp;author=WL%20Tam&amp;author=H%20Lu&amp;author=J%20Buikhuisen&amp;author=BS%20Soh&amp;author=E%20Lim&amp;volume=24&amp;publication_year=2013&amp;pages=347-364&amp;pmid=24029232&amp;doi=10.1016/j.ccr.2013.08.005&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R170\"> <cite> Tate JG, Bamford S, Jubb HC, Sondka Z, Beare DM, Bindal N, Boutselakis H, Cole CG, Creatore C, Dawson E, et al. (2019). COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gky1015\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6323903/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30371878/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=COSMIC:%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;author=JG%20Tate&amp;author=S%20Bamford&amp;author=HC%20Jubb&amp;author=Z%20Sondka&amp;author=DM%20Beare&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;pmid=30371878&amp;doi=10.1093/nar/gky1015&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R171\"> <cite> Tecalco-Cruz AC, and Cruz-Ramos E (2018). Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells. Biochem. Biophys. Res. Commun. 499, 973–978. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.bbrc.2018.04.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29626479/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun&amp;title=Protein%20ISGylation%20and%20free%20ISG15%20levels%20are%20increased%20by%20interferon%20gamma%20in%20breast%20cancer%20cells&amp;author=AC%20Tecalco-Cruz&amp;author=E%20Cruz-Ramos&amp;volume=499&amp;publication_year=2018&amp;pages=973-978&amp;pmid=29626479&amp;doi=10.1016/j.bbrc.2018.04.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R172\"> <cite> Tellez CS, Juri DE, Do K, Bernauer AM, Thomas CL, Damiani LA, Tessema M, Leng S, and Belinsky SA (2011). EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells. Cancer Res. 71, 3087–3097. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/0008-5472.CAN-10-3035\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3078195/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21363915/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=EMT%20and%20stem%20cell-like%20properties%20associated%20with%20miR-205%20and%20miR-200%20epigenetic%20silencing%20are%20early%20manifestations%20during%20carcinogen-induced%20transformation%20of%20human%20lung%20epithelial%20cells&amp;author=CS%20Tellez&amp;author=DE%20Juri&amp;author=K%20Do&amp;author=AM%20Bernauer&amp;author=CL%20Thomas&amp;volume=71&amp;publication_year=2011&amp;pages=3087-3097&amp;pmid=21363915&amp;doi=10.1158/0008-5472.CAN-10-3035&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R173\"> <cite> Thorsson V, Gibbs DL, Brown SD, Wolf D, Bortone DS, Ou Yang T-H, Porta-Pardo E, Gao GF, Plaisier CL, Eddy JA, et al. (2018). The Immune Landscape of Cancer. Immunity 48, 812–830.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.immuni.2018.03.023\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5982584/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29628290/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20Immune%20Landscape%20of%20Cancer&amp;author=V%20Thorsson&amp;author=DL%20Gibbs&amp;author=SD%20Brown&amp;author=D%20Wolf&amp;author=DS%20Bortone&amp;volume=48&amp;publication_year=2018&amp;pages=812-830.e14&amp;pmid=29628290&amp;doi=10.1016/j.immuni.2018.03.023&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R174\"> <cite> Tian J, Cai Y, Li Y, Lu Z, Huang J, Deng Y, Yang N, Wang X, Ying P, Zhang S, et al. (2021). CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer. Nucleic Acids Res. 49, D1065–D1073. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkaa805\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7778991/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33010176/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CancerImmunityQTL:%20a%20database%20to%20systematically%20evaluate%20the%20impact%20of%20genetic%20variants%20on%20immune%20infiltration%20in%20human%20cancer&amp;author=J%20Tian&amp;author=Y%20Cai&amp;author=Y%20Li&amp;author=Z%20Lu&amp;author=J%20Huang&amp;volume=49&amp;publication_year=2021&amp;pages=D1065-D1073&amp;pmid=33010176&amp;doi=10.1093/nar/gkaa805&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R175\"> <cite> Timperi E, Focaccetti C, Gallerano D, Panetta M, Spada S, Gallo E, Visca P, Venuta F, Diso D, Prelaj A, et al. (2017). IL-18 receptor marks functional CD8+ T cells in non-small cell lung cancer. Oncoimmunology 6, e1328337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1080/2162402X.2017.1328337\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5543928/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28811967/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Oncoimmunology&amp;title=IL-18%20receptor%20marks%20functional%20CD8+%20T%20cells%20in%20non-small%20cell%20lung%20cancer&amp;author=E%20Timperi&amp;author=C%20Focaccetti&amp;author=D%20Gallerano&amp;author=M%20Panetta&amp;author=S%20Spada&amp;volume=6&amp;publication_year=2017&amp;pages=e1328337&amp;pmid=28811967&amp;doi=10.1080/2162402X.2017.1328337&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R176\"> <cite> Tran MN, Choi W, Wszolek MF, Navai N, Lee I-LC, Nitti G, Wen S, Flores ER, Siefker-Radtke A, Czerniak B, et al. (2013). The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205. J. Biol. Chem. 288, 3275–3288. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/jbc.M112.408104\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3561548/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23239884/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=The%20p63%20protein%20isoform%20%CE%94Np63%CE%B1%20inhibits%20epithelial-mesenchymal%20transition%20in%20human%20bladder%20cancer%20cells:%20role%20of%20MIR-205&amp;author=MN%20Tran&amp;author=W%20Choi&amp;author=MF%20Wszolek&amp;author=N%20Navai&amp;author=I-LC%20Lee&amp;volume=288&amp;publication_year=2013&amp;pages=3275-3288&amp;pmid=23239884&amp;doi=10.1074/jbc.M112.408104&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R177\"> <cite> Tsherniak A, Vazquez F, Montgomery PG, Weir BA, Kryukov G, Cowley GS, Gill S, Harrington WF, Pantel S, Krill-Burger JM, et al. (2017). Defining a Cancer Dependency Map. Cell 170, 564–576.e16. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2017.06.010\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5667678/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28753430/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Defining%20a%20Cancer%20Dependency%20Map&amp;author=A%20Tsherniak&amp;author=F%20Vazquez&amp;author=PG%20Montgomery&amp;author=BA%20Weir&amp;author=G%20Kryukov&amp;volume=170&amp;publication_year=2017&amp;pages=564-576.e16&amp;pmid=28753430&amp;doi=10.1016/j.cell.2017.06.010&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R178\"> <cite> Udeshi ND, Mani DC, Satpathy S, Fereshetian S, Gasser JA, Svinkina T, Olive ME, Ebert BL, Mertins P, and Carr SA (2020). Rapid and deep-scale ubiquitylation profiling for biology and translational research. Nat. Commun. 11, 359. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-14175-1\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6969155/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31953384/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Rapid%20and%20deep-scale%20ubiquitylation%20profiling%20for%20biology%20and%20translational%20research&amp;author=ND%20Udeshi&amp;author=DC%20Mani&amp;author=S%20Satpathy&amp;author=S%20Fereshetian&amp;author=JA%20Gasser&amp;volume=11&amp;publication_year=2020&amp;pages=359&amp;pmid=31953384&amp;doi=10.1038/s41467-019-14175-1&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R179\"> <cite> Uhlén M, Björling E, Agaton C, Szigyarto CA-K, Amini B, Andersen E, Andersson A-C, Angelidou P, Asplund A, Asplund C, et al. (2005). A human protein atlas for normal and cancer tissues based on antibody proteomics. Mol. Cell. Proteomics 4, 1920–1932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1074/mcp.M500279-MCP200\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16127175/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20human%20protein%20atlas%20for%20normal%20and%20cancer%20tissues%20based%20on%20antibody%20proteomics&amp;author=M%20Uhl%C3%A9n&amp;author=E%20Bj%C3%B6rling&amp;author=C%20Agaton&amp;author=CA-K%20Szigyarto&amp;author=B%20Amini&amp;volume=4&amp;publication_year=2005&amp;pages=1920-1932&amp;pmid=16127175&amp;doi=10.1074/mcp.M500279-MCP200&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R180\"> <cite> Vasaikar S, Huang C, Wang X, Petyuk VA, Savage SR, Wen B, Dou Y, Zhang Y, Shi Z, Arshad OA, et al. (2019). Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities. Cell 177, 1035–1049.e19. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2019.03.030\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6768830/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31031003/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20Analysis%20of%20Human%20Colon%20Cancer%20Reveals%20New%20Therapeutic%20Opportunities&amp;author=S%20Vasaikar&amp;author=C%20Huang&amp;author=X%20Wang&amp;author=VA%20Petyuk&amp;author=SR%20Savage&amp;volume=177&amp;publication_year=2019&amp;pages=1035-1049.e19&amp;pmid=31031003&amp;doi=10.1016/j.cell.2019.03.030&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R181\"> <cite> Vasaikar SV, Straub P, Wang J, and Zhang B (2018). LinkedOmics: analyzing multi-omics data within and across 32 cancer types. Nucleic Acids Res. 46, D956–D963. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx1090\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5753188/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29136207/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LinkedOmics:%20analyzing%20multi-omics%20data%20within%20and%20across%2032%20cancer%20types&amp;author=SV%20Vasaikar&amp;author=P%20Straub&amp;author=J%20Wang&amp;author=B%20Zhang&amp;volume=46&amp;publication_year=2018&amp;pages=D956-D963&amp;pmid=29136207&amp;doi=10.1093/nar/gkx1090&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R182\"> <cite> Vats P, Chinnaiyan AM, and Kumar-Sinha C (2020). Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit. Methods Mol. Biol. 2079, 69–79. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1007/978-1-4939-9904-0_5\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31728962/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&amp;title=Case%20Study:%20Systematic%20Detection%20and%20Prioritization%20of%20Gene%20Fusions%20in%20Cancer%20by%20RNA-Seq:%20A%20DIY%20Toolkit&amp;author=P%20Vats&amp;author=AM%20Chinnaiyan&amp;author=C%20Kumar-Sinha&amp;volume=2079&amp;publication_year=2020&amp;pages=69-79&amp;pmid=31728962&amp;doi=10.1007/978-1-4939-9904-0_5&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R183\"> <cite> Vosgha H, Ariana A, Smith RA, and Lam AK-Y (2018). miR-205 targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma. Endocr. Relat. Cancer 25, 323–337. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1530/ERC-17-0497\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29317480/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Endocr.%20Relat.%20Cancer&amp;title=miR-205%20targets%20angiogenesis%20and%20EMT%20concurrently%20in%20anaplastic%20thyroid%20carcinoma&amp;author=H%20Vosgha&amp;author=A%20Ariana&amp;author=RA%20Smith&amp;author=AK-Y%20Lam&amp;volume=25&amp;publication_year=2018&amp;pages=323-337&amp;pmid=29317480&amp;doi=10.1530/ERC-17-0497&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R184\"> <cite> Walser T, Cui X, Yanagawa J, Lee JM, Heinrich E, Lee G, Sharma S, and Dubinett SM (2008). Smoking and lung cancer: the role of inflammation. Proc. Am. Thorac. Soc. 5, 811–815. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1513/pats.200809-100TH\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4080902/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19017734/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Am.%20Thorac.%20Soc&amp;title=Smoking%20and%20lung%20cancer:%20the%20role%20of%20inflammation&amp;author=T%20Walser&amp;author=X%20Cui&amp;author=J%20Yanagawa&amp;author=JM%20Lee&amp;author=E%20Heinrich&amp;volume=5&amp;publication_year=2008&amp;pages=811-815&amp;pmid=19017734&amp;doi=10.1513/pats.200809-100TH&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R185\"> <cite> Wang I-M, Zhang B, Yang X, Zhu J, Stepaniants S, Zhang C, Meng Q, Peters M, He Y, Ni C, et al. (2012). Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers. Mol. Syst. Biol. 8, 594. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2012.24\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3421440/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22806142/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol&amp;title=Systems%20analysis%20of%20eleven%20rodent%20disease%20models%20reveals%20an%20inflammatome%20signature%20and%20key%20drivers&amp;author=I-M%20Wang&amp;author=B%20Zhang&amp;author=X%20Yang&amp;author=J%20Zhu&amp;author=S%20Stepaniants&amp;volume=8&amp;publication_year=2012&amp;pages=594&amp;pmid=22806142&amp;doi=10.1038/msb.2012.24&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R186\"> <cite> Wang J, Vasaikar S, Shi Z, Greer M, and Zhang B (2017). WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. Nucleic Acids Res. 45, W130–W137. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/nar/gkx356\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC5570149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/28472511/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=WebGestalt%202017:%20a%20more%20comprehensive,%20powerful,%20flexible%20and%20interactive%20gene%20set%20enrichment%20analysis%20toolkit&amp;author=J%20Wang&amp;author=S%20Vasaikar&amp;author=Z%20Shi&amp;author=M%20Greer&amp;author=B%20Zhang&amp;volume=45&amp;publication_year=2017&amp;pages=W130-W137&amp;pmid=28472511&amp;doi=10.1093/nar/gkx356&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R187\"> <cite> Wang L-B, Karpova A, Gritsenko MA, Kyle JE, Cao S, Li Y, Rykunov D, Colaprico A, Rothstein JH, Hong R, et al. (2021). Proteogenomic and metabolomic characterization of human glioblastoma. Cancer Cell. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ccell.2021.01.006\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC8044053/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33577785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Proteogenomic%20and%20metabolomic%20characterization%20of%20human%20glioblastoma&amp;author=L-B%20Wang&amp;author=A%20Karpova&amp;author=MA%20Gritsenko&amp;author=JE%20Kyle&amp;author=S%20Cao&amp;publication_year=2021&amp;pmid=33577785&amp;doi=10.1016/j.ccell.2021.01.006&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R188\"> <cite> Wang S, Jiang B, Zhang T, Liu L, Wang Y, Wang Y, Chen X, Lin H, Zhou L, Xia Y, et al. (2015). Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1. PLoS Biol. 13, e1002243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pbio.1002243\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4565669/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26356530/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=Insulin%20and%20mTOR%20Pathway%20Regulate%20HDAC3-Mediated%20Deacetylation%20and%20Activation%20of%20PGK1&amp;author=S%20Wang&amp;author=B%20Jiang&amp;author=T%20Zhang&amp;author=L%20Liu&amp;author=Y%20Wang&amp;volume=13&amp;publication_year=2015&amp;pages=e1002243&amp;pmid=26356530&amp;doi=10.1371/journal.pbio.1002243&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R189\"> <cite> Weeden CE, Solomon B, and Asselin-Labat M-L (2015). FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies. Cell Death Discov 1, 15049. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/cddiscovery.2015.49\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4979434/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27551478/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discov&amp;title=FGFR1%20inhibition%20in%20lung%20squamous%20cell%20carcinoma:%20questions%20and%20controversies&amp;author=CE%20Weeden&amp;author=B%20Solomon&amp;author=M-L%20Asselin-Labat&amp;volume=1&amp;publication_year=2015&amp;pages=15049&amp;pmid=27551478&amp;doi=10.1038/cddiscovery.2015.49&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R190\"> <cite> Wen B, Wang X, and Zhang B (2019). PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations. Genome Res. 29, 485–493. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/gr.235028.118\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6396417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30610011/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=PepQuery%20enables%20fast,%20accurate,%20and%20convenient%20proteomic%20validation%20of%20novel%20genomic%20alterations&amp;author=B%20Wen&amp;author=X%20Wang&amp;author=B%20Zhang&amp;volume=29&amp;publication_year=2019&amp;pages=485-493&amp;pmid=30610011&amp;doi=10.1101/gr.235028.118&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R191\"> <cite> Wen B, Li K, Zhang Y, and Zhang B (2020). Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis. Nat. Commun. 11, 1759. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-020-15456-w\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7145864/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32273506/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Cancer%20neoantigen%20prioritization%20through%20sensitive%20and%20reliable%20proteogenomics%20analysis&amp;author=B%20Wen&amp;author=K%20Li&amp;author=Y%20Zhang&amp;author=B%20Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=1759&amp;pmid=32273506&amp;doi=10.1038/s41467-020-15456-w&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R192\"> <cite> Westra H-J, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, et al. (2013). Systematic identification of trans eQTLs as putative drivers of known disease associations. Nat. Genet. 45, 1238–1243. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2756\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3991562/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013639/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Systematic%20identification%20of%20trans%20eQTLs%20as%20putative%20drivers%20of%20known%20disease%20associations&amp;author=H-J%20Westra&amp;author=MJ%20Peters&amp;author=T%20Esko&amp;author=H%20Yaghootkar&amp;author=C%20Schurmann&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;doi=10.1038/ng.2756&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R193\"> <cite> Wheatley SP, and Altieri DC (2019). Survivin at a glance. J. Cell Sci. 132. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1242/jcs.223826\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6467487/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/30948431/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Survivin%20at%20a%20glance&amp;author=SP%20Wheatley&amp;author=DC%20Altieri&amp;volume=132&amp;publication_year=2019&amp;pmid=30948431&amp;doi=10.1242/jcs.223826&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R194\"> <cite> Whitfield ML, Sherlock G, Saldanha AJ, Murray JI, Ball CA, Alexander KE, Matese JC, Perou CM, Hurt MM, Brown PO, et al. (2002). Identification of genes periodically expressed in the human cell cycle and their expression in tumors. Mol. Biol. Cell 13, 1977–2000. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.02-02-0030.\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC117619/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/12058064/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Identification%20of%20genes%20periodically%20expressed%20in%20the%20human%20cell%20cycle%20and%20their%20expression%20in%20tumors&amp;author=ML%20Whitfield&amp;author=G%20Sherlock&amp;author=AJ%20Saldanha&amp;author=JI%20Murray&amp;author=CA%20Ball&amp;volume=13&amp;publication_year=2002&amp;pages=1977-2000&amp;pmid=12058064&amp;doi=10.1091/mbc.02-02-0030.&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R195\"> <cite> Wilkerson MD, and Hayes DN (2010). ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. Bioinformatics 26, 1572–1573. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq170\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881355/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20427518/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConsensusClusterPlus:%20a%20class%20discovery%20tool%20with%20confidence%20assessments%20and%20item%20tracking&amp;author=MD%20Wilkerson&amp;author=DN%20Hayes&amp;volume=26&amp;publication_year=2010&amp;pages=1572-1573&amp;pmid=20427518&amp;doi=10.1093/bioinformatics/btq170&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R196\"> <cite> Wilkerson MD, Yin X, Hoadley KA, Liu Y, Hayward MC, Cabanski CR, Muldrew K, Miller CR, Randell SH, Socinski MA, et al. (2010). Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types. Clin. Cancer Res. 16, 4864–4875. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0199\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2953768/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20643781/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=Lung%20squamous%20cell%20carcinoma%20mRNA%20expression%20subtypes%20are%20reproducible,%20clinically%20important,%20and%20correspond%20to%20normal%20cell%20types&amp;author=MD%20Wilkerson&amp;author=X%20Yin&amp;author=KA%20Hoadley&amp;author=Y%20Liu&amp;author=MC%20Hayward&amp;volume=16&amp;publication_year=2010&amp;pages=4864-4875&amp;pmid=20643781&amp;doi=10.1158/1078-0432.CCR-10-0199&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R197\"> <cite> Wong JJY, Pung YF, Sze NS-K, and Chin K-C (2006). HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets. Proc. Natl. Acad. Sci. U. S. A. 103, 10735–10740. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0600397103\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC1484417/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16815975/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;title=HERC5%20is%20an%20IFN-induced%20HECT-type%20E3%20protein%20ligase%20that%20mediates%20type%20I%20IFN-induced%20ISGylation%20of%20protein%20targets&amp;author=JJY%20Wong&amp;author=YF%20Pung&amp;author=NS-K%20Sze&amp;author=K-C%20Chin&amp;volume=103&amp;publication_year=2006&amp;pages=10735-10740&amp;pmid=16815975&amp;doi=10.1073/pnas.0600397103&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R198\"> <cite> Wu Y, Zhang Y, Zheng X, Dai F, Lu Y, Dai L, Niu M, Guo H, Li W, Xue X, et al. (2020). Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis. Mol. Cancer 19, 99. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s12943-020-01215-4\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7265647/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/32487167/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Circular%20RNA%20circCORO1C%20promotes%20laryngeal%20squamous%20cell%20carcinoma%20progression%20by%20modulating%20the%20let-7c-5p/PBX3%20axis&amp;author=Y%20Wu&amp;author=Y%20Zhang&amp;author=X%20Zheng&amp;author=F%20Dai&amp;author=Y%20Lu&amp;volume=19&amp;publication_year=2020&amp;pages=99&amp;pmid=32487167&amp;doi=10.1186/s12943-020-01215-4&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R199\"> <cite> Wu Y-M, Cieślik M, Lonigro RJ, Vats P, Reimers MA, Cao X, Ning Y, Wang L, Kunju LP, de Sarkar N, et al. (2018). Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer. Cell 173, 1770–1782.e14. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.cell.2018.04.034\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6084431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29906450/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Inactivation%20of%20CDK12%20Delineates%20a%20Distinct%20Immunogenic%20Class%20of%20Advanced%20Prostate%20Cancer&amp;author=Y-M%20Wu&amp;author=M%20Cie%C5%9Blik&amp;author=RJ%20Lonigro&amp;author=P%20Vats&amp;author=MA%20Reimers&amp;volume=173&amp;publication_year=2018&amp;pages=1770-1782.e14&amp;pmid=29906450&amp;doi=10.1016/j.cell.2018.04.034&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R200\"> <cite> Xiong Y, Lei Q-Y, Zhao S, and Guan K-L (2011). Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK. Cold Spring Harb. Symp. Quant. Biol. 76, 285–289. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/sqb.2011.76.010942\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4876600/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22096030/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol&amp;title=Regulation%20of%20glycolysis%20and%20gluconeogenesis%20by%20acetylation%20of%20PKM%20and%20PEPCK&amp;author=Y%20Xiong&amp;author=Q-Y%20Lei&amp;author=S%20Zhao&amp;author=K-L%20Guan&amp;volume=76&amp;publication_year=2011&amp;pages=285-289&amp;pmid=22096030&amp;doi=10.1101/sqb.2011.76.010942&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R201\"> <cite> Yang W, and Lu Z (2013). Nuclear PKM2 regulates the Warburg effect. Cell Cycle 12, 3154–3158. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/cc.26182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3865010/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24013426/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&amp;title=Nuclear%20PKM2%20regulates%20the%20Warburg%20effect&amp;author=W%20Yang&amp;author=Z%20Lu&amp;volume=12&amp;publication_year=2013&amp;pages=3154-3158&amp;pmid=24013426&amp;doi=10.4161/cc.26182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R202\"> <cite> Ye K, Schulz MH, Long Q, Apweiler R, and Ning Z (2009). Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics 25, 2865–2871. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btp394\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2781750/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19561018/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&amp;author=K%20Ye&amp;author=MH%20Schulz&amp;author=Q%20Long&amp;author=R%20Apweiler&amp;author=Z%20Ning&amp;volume=25&amp;publication_year=2009&amp;pages=2865-2871&amp;pmid=19561018&amp;doi=10.1093/bioinformatics/btp394&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R203\"> <cite> Yomtoubian S, Lee SB, Verma A, Izzo F, Markowitz G, Choi H, Cerchietti L, Vahdat L, Brown KA, Andreopoulou E, et al. (2020). Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer. Cell Rep. 30, 755–770.e6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2019.12.056\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31968251/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Inhibition%20of%20EZH2%20Catalytic%20Activity%20Selectively%20Targets%20a%20Metastatic%20Subpopulation%20in%20Triple-Negative%20Breast%20Cancer&amp;author=S%20Yomtoubian&amp;author=SB%20Lee&amp;author=A%20Verma&amp;author=F%20Izzo&amp;author=G%20Markowitz&amp;volume=30&amp;publication_year=2020&amp;pages=755-770.e6&amp;pmid=31968251&amp;doi=10.1016/j.celrep.2019.12.056&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R204\"> <cite> Yoshihara K, Shahmoradgoli M, Martínez E, Vegesna R, Kim H, Torres-Garcia W, Treviño V, Shen H, Laird PW, Levine DA, et al. (2013). Inferring tumour purity and stromal and immune cell admixture from expression data. Nat. Commun. 4, 2612. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ncomms3612\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3826632/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24113773/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data&amp;author=K%20Yoshihara&amp;author=M%20Shahmoradgoli&amp;author=E%20Mart%C3%ADnez&amp;author=R%20Vegesna&amp;author=H%20Kim&amp;volume=4&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;doi=10.1038/ncomms3612&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R205\"> <cite> Yuan G, Flores NM, Hausmann S, Lofgren SM, Kharchenko V, Angulo-Ibanez M, Sengupta D, Lu X, Czaban I, Azhibek D, et al. (2021). Elevated NSD3 histone methylation activity drives squamous cell lung cancer. Nature. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41586-020-03170-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC7895461/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/33536620/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Elevated%20NSD3%20histone%20methylation%20activity%20drives%20squamous%20cell%20lung%20cancer&amp;author=G%20Yuan&amp;author=NM%20Flores&amp;author=S%20Hausmann&amp;author=SM%20Lofgren&amp;author=V%20Kharchenko&amp;publication_year=2021&amp;pmid=33536620&amp;doi=10.1038/s41586-020-03170-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R206\"> <cite> Zhang X, Lu F, Wang J, Yin F, Xu Z, Qi D, Wu X, Cao Y, Liang W, Liu Y, et al. (2013). Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells. Cell Rep. 5, 445–457. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.celrep.2013.09.018\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC11162149/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24139802/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Pluripotent%20stem%20cell%20protein%20Sox2%20confers%20sensitivity%20to%20LSD1%20inhibition%20in%20cancer%20cells&amp;author=X%20Zhang&amp;author=F%20Lu&amp;author=J%20Wang&amp;author=F%20Yin&amp;author=Z%20Xu&amp;volume=5&amp;publication_year=2013&amp;pages=445-457&amp;pmid=24139802&amp;doi=10.1016/j.celrep.2013.09.018&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R207\"> <cite> Zhang Y, Zou X, Qian W, Weng X, Zhang L, Zhang L, Wang S, Cao X, Ma L, Wei G, et al. (2019a). Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis. Cell Death Differ. 26, 565–579. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41418-018-0147-y\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6370781/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29955124/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Enhanced%20PAPSS2/VCAN%20sulfation%20axis%20is%20essential%20for%20Snail-mediated%20breast%20cancer%20cell%20migration%20and%20metastasis&amp;author=Y%20Zhang&amp;author=X%20Zou&amp;author=W%20Qian&amp;author=X%20Weng&amp;author=L%20Zhang&amp;volume=26&amp;publication_year=2019a&amp;pages=565-579&amp;pmid=29955124&amp;doi=10.1038/s41418-018-0147-y&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R208\"> <cite> Zhang Y, Thery F, Wu NC, Luhmann EK, Dussurget O, Foecke M, Bredow C, Jiménez-Fernández D, Leandro K, Beling A, et al. (2019b). The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection. Nat. Commun. 10, 5383. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/s41467-019-13393-x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6879477/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/31772204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=The%20in%20vivo%20ISGylome%20links%20ISG15%20to%20metabolic%20pathways%20and%20autophagy%20upon%20Listeria%20monocytogenes%20infection&amp;author=Y%20Zhang&amp;author=F%20Thery&amp;author=NC%20Wu&amp;author=EK%20Luhmann&amp;author=O%20Dussurget&amp;volume=10&amp;publication_year=2019b&amp;pages=5383&amp;pmid=31772204&amp;doi=10.1038/s41467-019-13393-x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R209\"> <cite> Zhao D, Li F-L, Cheng Z-L, and Lei Q-Y (2014). Impact of acetylation on tumor metabolism. Mol Cell Oncol 1, e963452. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.4161/23723548.2014.963452\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4905055/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/27308346/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Oncol&amp;title=Impact%20of%20acetylation%20on%20tumor%20metabolism&amp;author=D%20Zhao&amp;author=F-L%20Li&amp;author=Z-L%20Cheng&amp;author=Q-Y%20Lei&amp;volume=1&amp;publication_year=2014&amp;pages=e963452&amp;pmid=27308346&amp;doi=10.4161/23723548.2014.963452&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R210\"> <cite> Zheng L, and Conner SD (2018). Glycogen synthase kinase 3β inhibition enhances Notch1 recycling. Mol. Biol. Cell 29, 389–395. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1091/mbc.E17-07-0474\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC6014177/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/29237816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Glycogen%20synthase%20kinase%203%CE%B2%20inhibition%20enhances%20Notch1%20recycling&amp;author=L%20Zheng&amp;author=SD%20Conner&amp;volume=29&amp;publication_year=2018&amp;pages=389-395&amp;pmid=29237816&amp;doi=10.1091/mbc.E17-07-0474&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"R211\"> <cite> Zhou X, Edmonson MN, Wilkinson MR, Patel A, Wu G, Liu Y, Li Y, Zhang Z, Rusch MC, Parker M, et al. (2016). Exploring genomic alteration in pediatric cancer using ProteinPaint. Nat. Genet. 48, 4–6. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.3466\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4892362/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/26711108/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Exploring%20genomic%20alteration%20in%20pediatric%20cancer%20using%20ProteinPaint&amp;author=X%20Zhou&amp;author=MN%20Edmonson&amp;author=MR%20Wilkinson&amp;author=A%20Patel&amp;author=G%20Wu&amp;volume=48&amp;publication_year=2016&amp;pages=4-6&amp;pmid=26711108&amp;doi=10.1038/ng.3466&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ol> </section> </section> <section class=\"associated-data\" id=\"_ad93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> Associated Data </h2> <p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> <section class=\"supplementary-materials\" id=\"_adsm93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Supplementary Materials </h3> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material1_reqid_\"> <div class=\"caption p\"> <span> 1 </span> <p> Figure S1: Experimental Design and Proteogenomic Data Quality Metrics, Related to <a class=\"usa-link\" href=\"#F1\"> Figure 1 </a> </p> <p> A. Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to 5 tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment ( <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> ). For the ubiquitylproteome, a different TMT11 experimental design was used ( <a class=\"usa-link\" href=\"#SD7\"> Table S1 </a> ) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses. </p> <p> B. Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median. </p> <p> C. Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> (“ <a class=\"usa-link\" href=\"#S75\"> Quality control and Batch effect assessment </a> ”). Top panel: Box plots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR. </p> <p> D. Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers. </p> <p> E. PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type. </p> <p> F. To identify specific events associated with CNA <em> trans </em> -effects (vertical stripes in <a class=\"usa-link\" href=\"#F1\"> Figure 1D </a> ), we leveraged CMAP data ( <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> ), which includes over 1.3 million gene expression profiles from ~20,000 small-molecule compounds and ~4,300 genetic reagents tested in multiple cell types. Patterns of significant <em> trans </em> -effects associated with CNAs that also had significant <em> cis </em> -effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p-values are indicated. </p> <p> G. Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p&lt;2×10−16). </p> <p> H. Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf\"> NIHMS1730863-supplement-1.pdf </a> <sup> (5.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material2_reqid_\"> <div class=\"caption p\"> <span> 2 </span> <p> Figure S2: Characteristics of Multi-omic NMF Subtypes, Related to <a class=\"usa-link\" href=\"#F2\"> Figure 2 </a> </p> <p> A. Upper panel: Line plot representing cophenetic correlation coefficients (y-axis) calculated for a range of factorization ranks (x-axis). The maximal cophenetic correlation coefficient was observed for rank K=5 as highlighted by the red box. Lower panel: silhouette plot for K=5. This plot indicates the quality of cluster separation. </p> <p> B. Directionality and significance (signed Wilcoxon p-value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-Inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box. </p> <p> C. Same as B but in the EMT-enriched subtype. </p> <p> D. Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt;0.05 and Pearson correlation &gt;0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways. </p> <p> E. Scatter plot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed −log10 (p-value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels. </p> <p> F. Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the box plots. P-values are derived from the Wilcoxon test. B.I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> G. Representative dual immunohistochemical staining of alpha-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes. </p> <p> H. Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g. drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-β inhibitors in the negatively-connected compounds of the EMT-E cluster. </p> <p> I. Integrative analysis of the cumulative 216 treatment-naïve LSCC tumors between this and the Stewart study ( <a aria-describedby=\"R165\" class=\"usa-link\" href=\"#R165\"> Stewart et al., 2019 </a> ), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset ( <a class=\"usa-link\" href=\"#S14\"> STAR Methods </a> ) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2A </a> ). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em> TP53 </em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors. B.I - Basal-inclusive, C - Classical, E-E – EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive. </p> <p> J. Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> . Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR&lt;0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation ( <a class=\"usa-link\" href=\"#SD2\"> Figure S2I </a> ), known drivers of EMT ( <a aria-describedby=\"R22\" class=\"usa-link\" href=\"#R22\"> Cannito et al., 2008 </a> ) and of Wnt signaling ( <a aria-describedby=\"R62\" class=\"usa-link\" href=\"#R62\"> Hong et al., 2017 </a> ). </p> <p> K. A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca2+ pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf\"> NIHMS1730863-supplement-2.pdf </a> <sup> (22.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material3_reqid_\"> <div class=\"caption p\"> <span> 3 </span> <p> Figure S3: Proteogenomic Consequences of CDKN2A and NRF2 Pathway Aberrations, Related to <a class=\"usa-link\" href=\"#F3\"> Figure 3 </a> </p> <p> A. Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em> ARID1A </em> gene somatic mutation. </p> <p> B. Schematic diagram showing amino acid alterations resulting from <em> CDKN2A/p16INK4a </em> nonsense and frameshift mutations in the CPTAC LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em> p16INK4a (p16) </em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5’ end (E10*) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein. </p> <p> C. Box plots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em> CCND1 </em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P-values are from Welch’s two-sample T-tests. </p> <p> D. Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatter plots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em> CCND1 </em> , <em> CDK2NA </em> , and <em> RB1 </em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em> CCND1 </em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em> CDKN2A </em> deletion (GISTIC threshold ≤ −1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines. </p> <p> E. Box plots showing NFE2L2 S215 and S433 abundance in samples with <em> CUL3 </em> , <em> KEAP1 </em> and <em> NFE2L2 </em> mutations. **** &lt;=0.0001, *** &lt; 0.01, **&lt; 0.01, *&lt; 0.05, ns = not significant. P-values are from the Wilcoxon test. </p> <p> F. Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) vs NRF2 pathway WT tumors. Significant events (&gt;2 fold and p-value&lt;0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues ( <a aria-describedby=\"R92\" class=\"usa-link\" href=\"#R92\"> Lacher and Slattery, 2016 </a> ) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes. </p> <p> G. Overlap between NRF2-related genesets (upper left), and up- (upper right) and down-regulated (bottom) events in NRF2 pathway mutant samples across mRNA, proteomics and phosphoproteomics. </p> <p> H. Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway ( <em> NF2L2, KEAP1, CUL3 </em> ) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup> # </sup> no phosphosite levels changes were detected for NR0B1 between the groups. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf\"> NIHMS1730863-supplement-3.pdf </a> <sup> (4.6MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material4_reqid_\"> <div class=\"caption p\"> <span> 4 </span> <p> Figure S4: Association Between TP63 and miR205 Expression in LSCC, Related to <a class=\"usa-link\" href=\"#F4\"> Figure 4 </a> </p> <p> A. Focal amplification and deletion landscape of the study LSCC cohort. The top five significant focal peaks are labeled in the plot </p> <p> B. Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in <a class=\"usa-link\" href=\"#S14\"> STAR METHODS </a> ). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors vs NATs (from <a class=\"usa-link\" href=\"#F4\"> Figure 4A </a> with Log2FC&gt;0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t-test of a given gene in amplified vs non-amplified LSCC cell lines. X-axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification vs without amplification of that gene. Y-axis depicts the −log10(p-value) from t-test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots. </p> <p> C. mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort. </p> <p> D. Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel). </p> <p> E. Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LSCC cohort. Samples with low ΔNp63α expression have significantly worse outcomes. </p> <p> F. Np63 mRNA transcript isoform expression analysis (RNAseq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63. </p> <p> G. Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P-value derived from t-test. </p> <p> H. Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples. </p> <p> I. Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors. </p> <p> J. Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors. </p> <p> K. Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205. </p> <p> L. Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73. </p> <p> M. Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em> SOX2 </em> Log2 copy number (Copy number (Gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf\"> NIHMS1730863-supplement-4.pdf </a> <sup> (4MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material5_reqid_\"> <div class=\"caption p\"> <span> 6 </span> <p> Figure S6: Differential pathway, CBPE scores and impact of germline mutations on immune signaling, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p-values are reported. </p> <p> B. Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p-values are reported. </p> <p> C. Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P-values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists. </p> <p> D. Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated. </p> <p> E. Overlap between differentially regulated mRNA and proteins between Hot vs Cold and Warm vs Cold tumors, and eGenes downstream of <em> cis </em> -eQTLs and <em> cis- </em> pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ; <a aria-describedby=\"R154\" class=\"usa-link\" href=\"#R154\"> Shahamatdar et al., 2020 </a> ; <a aria-describedby=\"R174\" class=\"usa-link\" href=\"#R174\"> Tian et al., 2021 </a> ). Among 319 <em> cis </em> -pQTLs ( <em> cis </em> -regulating the cognate protein) identified in this study, only 38 were also <em> cis </em> -eQTLs ( <em> cis </em> -regulating cognate RNA)( <a class=\"usa-link\" href=\"#SD12\"> Table S6 </a> ). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported ( <a aria-describedby=\"R109\" class=\"usa-link\" href=\"#R109\"> Lim et al., 2018 </a> ). These results imply a potentially important role of germline variants in shaping immune processes in LSCC. </p> <p> F. Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p-values are reported. </p> <p> G. Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf\"> NIHMS1730863-supplement-6.pdf </a> <sup> (6.5MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material6_reqid_\"> <div class=\"caption p\"> <span> 7 </span> <p> Figure S7: Differential tumor and NAT analysis, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR&lt;0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs. </p> <p> B. Fold changes of CT antigens and their relationship to NMF subtype and immune clusters. </p> <p> C. Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort. </p> <p> D. Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts. </p> <p> E. To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p&lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple. </p> <p> F. Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD ( <a aria-describedby=\"R51\" class=\"usa-link\" href=\"#R51\"> Gillette et al., 2020 </a> ) and the HNSCC ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) cohorts. </p> <p> G. EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC). </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf\"> NIHMS1730863-supplement-7.pdf </a> <sup> (2.7MB, pdf) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material7_reqid_\"> <div class=\"caption p\"> <span> 8 </span> <p> Table S1: Metadata Associated with LSCC Patients and Tumors, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Metadata associated with this study </p> <p> B. Cancer Associated Genes (CAGs) </p> <p> C. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. </p> <p> D. Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. </p> <p> E. Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx\"> NIHMS1730863-supplement-8.xlsx </a> <sup> (327.4KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material8_reqid_\"> <div class=\"caption p\"> <span> 9 </span> <p> Table S2: Genomic and methylation datasets associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> , <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as log2(CN)-1. </p> <p> C. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2A </a> but CNAs are represented as thresholded copy number level calls. </p> <p> D. RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. </p> <p> E. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2D </a> but with FPKM values median-centered by gene. </p> <p> F. miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. </p> <p> G. Same as <a class=\"usa-link\" href=\"#SD8\"> Table S2F </a> but with TPM values median-centered by gene. </p> <p> H. Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5’ UTR regions of the gene. </p> <p> I. Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx\"> NIHMS1730863-supplement-9.xlsx </a> <sup> (218.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material9_reqid_\"> <div class=\"caption p\"> <span> 10 </span> <p> Table S3: Global proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics data associated with this study, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F7\"> 7 </a> and <a class=\"usa-link\" href=\"#SD1\"> Figures S1 </a> – <a class=\"usa-link\" href=\"#SD6\"> 7 </a> </p> <p> A. Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> B. Same as <a class=\"usa-link\" href=\"#SD9\"> Table S3A </a> but after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> </p> <p> C. Gene-centric global proteome data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> . Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). </p> <p> D. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> E. Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> F. Gene-centric phosphoproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3E </a> . Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> G. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> H. Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> I. Gene-centric acetylproteomics data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3H </a> . Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> J. Acetylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3G </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> <p> K. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> L. Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . See <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> for details on batch correction. </p> <p> M. Gene-centric ubiquitylproteomic data matrix derived from <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> . Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. </p> <p> N. Ubiquitylproteomic data matrix based on <a class=\"usa-link\" href=\"#SD9\"> Table S3L </a> corrected for global protein expression ( <a class=\"usa-link\" href=\"#SD9\"> Table S3B </a> ) as described in the <a class=\"usa-link\" href=\"#S14\"> STAR methods </a> . </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx\"> NIHMS1730863-supplement-10.xlsx </a> <sup> (372.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material10_reqid_\"> <div class=\"caption p\"> <span> 11 </span> <p> Table S4: CNA <em> cis </em> and <em> trans </em> effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, <em> cis </em> -eQTL from RNA and proteomics datasets, eGenes, Related to <a class=\"usa-link\" href=\"#F1\"> Figures 1 </a> – <a class=\"usa-link\" href=\"#F4\"> 4 </a> </p> <p> A. Copy number amplified cytobands </p> <p> B. Copy number deleted cytobands C. CNA <em> cis- </em> and <em> trans- </em> effects </p> <p> D. Significant focal amplicons that show significant <em> cis </em> -effect and are cancer associated genes </p> <p> E. iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes=1/No=0) with FDR&lt;10% and identified phenotype association analyses (Yes=1/No=0). </p> <p> F. CMAP tab- Connectivity Map (CMAP) results </p> <p> G. Association between genes enriched in the CMAP analysis and sample metadata </p> <p> H. Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. </p> <p> I. Summary of CDKN2A and RB1 annotations </p> <p> J. T test p-values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 </p> <p> K. Differential proteogenomic features in samples with and without <em> NFE2L2 </em> ( <em> NRF2 </em> ), <em> CUL3 </em> and <em> KEAP1 </em> mutations </p> <p> L. NRF2 signature </p> <p> M. Differential expression of RNA and proteins between TP63-low vsTP63-high samples. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx\"> NIHMS1730863-supplement-11.xlsx </a> <sup> (7.9MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material11_reqid_\"> <div class=\"caption p\"> <span> 12 </span> <p> Table S5: Tables associated with ubiquitylproteome analysis, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Clustering of tumor samples using protein-corrected ubiquitin data </p> <p> B. Ub-sites that show differential abundance across Ub clusters </p> <p> C. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 </p> <p> D. Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 </p> <p> E. Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) </p> <p> F. Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) </p> <p> G. Pathway enrichment results for Ub-sites positively associated with HERC5 abundance </p> <p> H. Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS </p> <p> I. Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS </p> <p> J. Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx\"> NIHMS1730863-supplement-12.xlsx </a> <sup> (451KB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material12_reqid_\"> <div class=\"caption p\"> <span> 13 </span> <p> Table S6: Tables associated with immune-landscape analysis, Related to <a class=\"usa-link\" href=\"#F6\"> Figure 6 </a> </p> <p> A. Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. </p> <p> B. Immune clusters based on xCell signatures </p> <p> C. Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) </p> <p> D. Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) </p> <p> E. RTK Correlation-Based Enrichment (CBPE) Scores </p> <p> F. Pathway scores in Tumor samples based on global protein abundance via ssGSEA </p> <p> G. <em> cis </em> -eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) </p> <p> H. <em> cis </em> -pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) </p> <p> I. List of genes/proteins whose expression is associated with a SNP </p> <p> J. Ingenuity pathway analysis of the eGenes from QTL analysis </p> <p> K. <em> cis </em> -QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4) </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx\"> NIHMS1730863-supplement-13.xlsx </a> <sup> (3.4MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material13_reqid_\"> <div class=\"caption p\"> <span> 14 </span> <p> Table S7: Tables associated with tumor-associated proteins, Related to <a class=\"usa-link\" href=\"#F7\"> Figure 7 </a> </p> <p> A. Differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD </p> <p> B. 502 proteins significantly overexpressed (log <sub> 2 </sub> (FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis </p> <p> C. Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. </p> <p> D. DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines </p> <p> E. Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD </p> <p> F. Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx\"> NIHMS1730863-supplement-14.xlsx </a> <sup> (9.8MB, xlsx) </sup> </div> </div> </section> <section class=\"sm xbox font-sm\" id=\"db_ds_supplementary-material14_reqid_\"> <div class=\"caption p\"> <span> Figure S5 </span> <p> Figure S5: Pathway analysis and differential ubiquitylation and acetylation, Related to <a class=\"usa-link\" href=\"#F5\"> Figure 5 </a> </p> <p> A. Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters ( <a class=\"usa-link\" href=\"#F5\"> Figure 5A </a> ). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text. </p> <p> B. STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways. </p> <p> C. Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations. </p> <p> D. HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation. </p> <p> E. Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression. </p> <p> F. HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors. </p> <p> G. PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction. </p> <p> H. HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors. </p> <p> I. HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation. </p> <p> J. ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation. </p> <p> K. PGK1 acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a class=\"usa-link\" href=\"#F5\"> Figure 5E </a> (lower right) are decreased in tumors compared to NATs. Wilcoxon p-values are indicated. </p> </div> <div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf\"> NIHMS1730863-supplement-Figure_S5.pdf </a> <sup> (2MB, pdf) </sup> </div> </div> </section> </section> <section class=\"data-availability-statement\" id=\"_adda93_\" lang=\"en\"> <h3 class=\"pmc_sec_title\"> Data Availability Statement </h3> <section id=\"db_da_sec19_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC LSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> Proteomic Data Commons (PDC); <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 </p> </li> <li> <p> CPTAC data portal LSCC: <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </p> </li> </ul> </section> <section id=\"db_da_sec20_reqid_\"> <h3 class=\"pmc_sec_title\"> CPTAC HNSCC proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000221 and PDC000222 </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </p> </li> </ul> </section> <section id=\"db_da_sec21_reqid_\"> <h3 class=\"pmc_sec_title\"> CPATC LUAD proteomics data: </h3> <ul class=\"list\"> <li> <p> PDC: <a class=\"usa-link usa-link--external\" href=\"https://pdc.cancer.gov/pdc/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pdc.cancer.gov/pdc/ </a> with identifiers PDC000153, PDC000149, PDC000224) </p> </li> <li> <p> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S056\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S056 </a> </p> </li> </ul> <p> Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class=\"usa-link usa-link--external\" href=\"https://portal.gdc.cancer.gov/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://portal.gdc.cancer.gov/ </a> , via dbGaP Study Accession: phs001287.v10.p5 </p> <p> All histologic <a class=\"usa-link usa-link--external\" href=\"https://www.cancerimagingarchive.net/datascope/cptac/home/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.cancerimagingarchive.net/datascope/cptac/home/ </a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access <a class=\"usa-link usa-link--external\" href=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\" rel=\"noopener noreferrer\" target=\"_blank\"> https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC </a> . </p> <p> Sample annotation, processed and normalized data files are provided as <a class=\"usa-link\" href=\"#SD7\"> Tables S1 </a> – <a class=\"usa-link\" href=\"#SD9\"> S3 </a> . </p> <p> Software and code used in this study are referenced in their corresponding <a class=\"usa-link\" href=\"#S14\"> STAR Method </a> sections and also the <a class=\"usa-link\" href=\"#T1\"> Key Resource Table </a> . </p> <section class=\"tw xbox font-sm\" id=\"db_da_table-wrap1_reqid_\"> <h4 class=\"obj_head\"> KEY RESOURCES TABLE. </h4> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"box\" rules=\"all\"> <colgroup span=\"1\"> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> <col align=\"left\" span=\"1\" valign=\"middle\"/> </colgroup> <thead> <tr> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> REAGENT or RESOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SOURCE </th> <th align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> IDENTIFIER </th> </tr> </thead> <tbody> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Antibodies </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan® Ubiquitin Remnant Motif (K-ε-GG) Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog 5562 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMScan Acetyllysine Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cell Signaling Technology </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 13416 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD4 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B12, <strong> RRID:AB_2728838 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD8 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Bio-Rad </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Clone 4B11, <strong> RRID:AB_322868 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CD163 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nuccore/Ab182422\" rel=\"noopener noreferrer\" target=\"_blank\"> Ab182422 </a> , <strong> RRID:AB_2753196 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ARHGDIB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma-Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPA051235, <strong> RRID:AB_2681398 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CK (pan-Cytokeratin) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dako </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> clone AE1/AE3, <strong> RRID:AB_2132885 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> alpha-SMA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Abcam </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ab5694, <strong> RRID:AB_2223021 </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Biological Samples </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Primary tumor samples </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> See <a class=\"usa-link\" href=\"#S23\"> Experimental Model and Subject Details </a> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> N/A </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Chemicals and Reagents </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> HPLC-grade water </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> J.T. Baker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 4218-03 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Urea </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: U0631 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 71376 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1M Tris, pH 8.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Invitrogen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM9855G </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ethylenediaminetetraacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: E7889 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aprotinin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A6103 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Leupeptin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 11017101001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phenylmethylsulfonyl fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 78830 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium fluoride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S7920 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 2 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P5726 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphatase inhibitor cocktail 3 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0044 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dithiothretiol, No-Weigh Format </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoScientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20291 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iodoacetamide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A3221 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Lysyl endopeptidase </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Wako Chemicals </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 129-02541 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sequencing-grade modified trypsin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Promega </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: V511X </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Formic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: F0507 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, LC-MS grade </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34967 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetonitrile, anhydrous </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 271004 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Trifluoroacetic acid </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 302031 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Tandem Mass Tag reagent kit – 11plex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: A34808 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 0.5M HEPES, pH 8.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Alfa Aesar </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: J63218 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hydroxylamine solution, 50% (vol/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Aldrich </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 467804 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methanol </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Honeywell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 34966 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ammonium hydroxide solution, 28% (wt/vol) in H <sub> 2 </sub> O </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 338818 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ni-NTA agarose beads </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Qiagen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 30410 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Iron (III) chloride </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 451649 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Acetic acid, glacial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AX0073 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, monobasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P0662 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Potassium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: P3786 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MOPS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M5162 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium hydroxide </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: BDH7225 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sodium phosphate, dibasic </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sigma </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: S9763 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phosphate-buffered saline </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Fisher Scientific </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 10010023 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iVIEW DAB Detection Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 760-091 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Equipment </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Reversed-phase tC18 SepPak, 3cc 200mg </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WAT054925 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Solid-phase C18 disk, for Stage-tips </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Empore </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 66883-U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip needle </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cadence </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 7928 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Stage-tip puncher, PEEK tubing </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Idex Health &amp; Science </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 1581 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PicoFrit LC-MS column </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> New Objective </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PF360-75-10-N-5 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dr. Maisch </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: r119.aq </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nanospray column heater </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CH-20U </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Column heater controller </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Phoenix S&amp;T </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PST-CHC </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 300 μL LC-MS autosampler vial and cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Waters </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 186002639 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: Custom order </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline 96-well fractionation plate </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Whatman </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 77015200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler vial </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-14 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 700 μL bRP fractionation autosampler cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: C4010-55A </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 96-well microplate for BCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Greiner </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 655101 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate foil cover </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: PCR-AS-200 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Vacuum centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SPD121P-115 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Eppendorf </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 5427 R </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop mini centrifuge </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 6765 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Benchtop vortex </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Scientific Industries </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: SI-0236 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Incubating shaker </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VWR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 12620-942 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 15 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352097 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 50 mL centrifuge tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Corning </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 352070 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.607 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL microtube w/o cap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.608 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microtube caps </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sarstedt </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 72.692 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 1.5 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12450 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> 2.0 mL snapcap tube </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: AM12475 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Instrumentation </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Microplate Reader </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Molecular Devices </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: M2 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Offline HPLC System for bRP fractionation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Agilent 1260 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: G1380-90000 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Online LC for LC-MS </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: LC140 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive Plus Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFALGMBDK </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Q Exactive HF-X Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 0726042 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Orbitrap Fusion Lumos Tribrid Mass Spectrometer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: IQLAAEGAAPFADBMBHQ </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> <strong> Critical Commercial Assays </strong> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: RS-122-2301 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Infinium MethylationEPIC Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: WG-317-1003 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Nextera DNA Exosome Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 20020617 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> KAPA Hyper Prep Kit, PCR-free </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Roche </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 07962371001 </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> BCA Protein Assay Kit </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ThermoFisher </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Catalog: 23225 </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Deposited Data </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Mutation </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Segmented copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Gene level copy number </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: RNAseq (transcript isoform) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (PMID: 31068700; Dataset doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.6084/m9.figshare.13681534.v1\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.6084/m9.figshare.13681534.v1 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: Proteomics (RPPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CCLE 2019 (PMID: 31068700) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: CRISPR KO screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1101/2020.05.22.110247\" rel=\"noopener noreferrer\" target=\"_blank\"> https://doi.org/10.1101/2020.05.22.110247 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: shRNA screen (combined) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DEMETER2 Data v6 (PMID: 30389920) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: GDSC drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Sanger GDSC 1 (PMID: 27397505) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> DepMap: PRISM drug screen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/download/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/download/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PhosphoSitePlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R63\" class=\"usa-link\" href=\"#R63\"> Hornbeck et al., 2012 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.phosphosite.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.phosphosite.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Connectivity Map (CMAP) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.broadinstitute.org/connectivity-map-cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.broadinstitute.org/connectivity-map-cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Human Protein Atlas (HPA) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R179\" class=\"usa-link\" href=\"#R179\"> Uhlén et al., 2005 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://www.proteinatlas.org\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.proteinatlas.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CT Antigen database </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R5\" class=\"usa-link\" href=\"#R5\"> Almeida et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cta.lncc.br\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cta.lncc.br </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Dependency map (DepMap) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R177\" class=\"usa-link\" href=\"#R177\"> Tsherniak et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://depmap.org/portal/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://depmap.org/portal/ </a> <br/> v3.3.8 is a GECKOv2 Achilles dataset </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Library of Integrated Network-based Cellular Signatures (LINCS) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/data\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/data </a> <br/> Expanded CMap LINCS Resource 2020 (1/28/2021 update) </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC HNSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R65\" class=\"usa-link\" href=\"#R65\"> Huang et al., 2021 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S054\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S054 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC cohort </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> This study </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cptac-data-portal.georgetown.edu/study-summary/S063\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cptac-data-portal.georgetown.edu/study-summary/S063 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"3\" rowspan=\"1\" valign=\"top\"> Software and Algorithms </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> methylationArrayAnalysis (version 3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R114\" class=\"usa-link\" href=\"#R114\"> Maksimovic et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Illumina EPIC methylation array (3.9) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Hansen KD, 2019 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Methylation array analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/cptac_methylation\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/cptac_methylation </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> miRNA-Seq analysis pipeline for CPTAC </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Li Ding Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ding-lab/CPTAC_miRNA\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ding-lab/CPTAC_miRNA </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VEP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R118\" class=\"usa-link\" href=\"#R118\"> McLaren et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/Ensembl/ensembl-vep/tags\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/Ensembl/ensembl-vep/tags </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TNScope / DNAScope (Sentieon) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R41\" class=\"usa-link\" href=\"#R41\"> Freed et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://sentieon.com\" rel=\"noopener noreferrer\" target=\"_blank\"> sentieon.com </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> vcfAnno </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R138\" class=\"usa-link\" href=\"#R138\"> Pedersen et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/brentp/vcfanno\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/brentp/vcfanno </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> VariantAnnotation (Bioconductor) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R131\" class=\"usa-link\" href=\"#R131\"> Obenchain et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> arriba_v1.1.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/suhrig/arriba/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/suhrig/arriba/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> fusioncatcher_v1.10 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R128\" class=\"usa-link\" href=\"#R128\"> Nicorici et al. </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> eQTLGen </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R192\" class=\"usa-link\" href=\"#R192\"> Westra et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Pindel0.2.5 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R202\" class=\"usa-link\" href=\"#R202\"> Ye et al., 2009 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://gmt.genome.wustl.edu/packages/pindel/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://gmt.genome.wustl.edu/packages/pindel/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> SignatureAnalyzer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R82\" class=\"usa-link\" href=\"#R82\"> Kim et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://software.broadinstitute.org/cancer/cga/msp\" rel=\"noopener noreferrer\" target=\"_blank\"> https://software.broadinstitute.org/cancer/cga/msp </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CNVEX </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mctp/cnvex\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mctp/cnvex </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CRISP </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Marcin Cieslik Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/mcieslik-mctp/crisp-build\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/mcieslik-mctp/crisp-build </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Spectrum Mill </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Karl R. Clauser, Steven Carr Lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://proteomics.broadinstitute.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://proteomics.broadinstitute.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ComBat (v3.20.0) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R72\" class=\"usa-link\" href=\"#R72\"> Johnson et al., 2007 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/sva.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/sva.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> gPCA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R146\" class=\"usa-link\" href=\"#R146\"> Reese et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/gPCA/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/gPCA/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GISTIC2.0 </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R121\" class=\"usa-link\" href=\"#R121\"> Mermel et al., 2011 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\" rel=\"noopener noreferrer\" target=\"_blank\"> ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> iProFun </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R162\" class=\"usa-link\" href=\"#R162\"> Song et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/iProFun\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/iProFun </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ESTIMATE </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R204\" class=\"usa-link\" href=\"#R204\"> Yoshihara et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioinformatics.mdanderson.org/public-software/estimate/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioinformatics.mdanderson.org/public-software/estimate/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> WebGestaltR </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R186\" class=\"usa-link\" href=\"#R186\"> Wang et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.webgestalt.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.webgestalt.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> GSVA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R55\" class=\"usa-link\" href=\"#R55\"> Hanzelmann et al., 2013 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/GSVA.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/GSVA.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> TSNet </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R141\" class=\"usa-link\" href=\"#R141\"> Petralia et al., 2018 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/WangLab-MSSM/TSNet\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/WangLab-MSSM/TSNet </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> xCell </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R6\" class=\"usa-link\" href=\"#R6\"> Aran et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://xcell.ucsf.edu/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://xcell.ucsf.edu/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CPTAC LSCC Data Viewer </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Steven Carr lab </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ConsensusClusterPlus </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"bottom\"> ( <a aria-describedby=\"R195\" class=\"usa-link\" href=\"#R195\"> Wilkerson and Hayes, 2010 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\" rel=\"noopener noreferrer\" target=\"_blank\"> http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MS-GF+ </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R81\" class=\"usa-link\" href=\"#R81\"> Kim and Pevzner, 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/MSGFPlus/msgfplus\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/MSGFPlus/msgfplus </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> NeoFlow </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/neoflow\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/neoflow </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> netMHCpan </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R73\" class=\"usa-link\" href=\"#R73\"> Jurtz et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://www.cbs.dtu.dk/services/NetMHCpan/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.cbs.dtu.dk/services/NetMHCpan/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Optitype </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R168\" class=\"usa-link\" href=\"#R168\"> Szolek et al., 2014 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/FRED-2/OptiType\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/FRED-2/OptiType </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Customprodbj </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R191\" class=\"usa-link\" href=\"#R191\"> Wen et al., 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/bzhanglab/customprodbj\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/bzhanglab/customprodbj </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PDV </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R102\" class=\"usa-link\" href=\"#R102\"> Li et al., 2019b </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/wenbostar/PDV\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/wenbostar/PDV </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PepQuery </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R190\" class=\"usa-link\" href=\"#R190\"> Wen et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://pepquery.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://pepquery.org </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTM-SEA </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/ssGSEA2.0\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/ssGSEA2.0 </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> PTMsigDB </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R88\" class=\"usa-link\" href=\"#R88\"> Krug et al., 2018 </a> )) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Terra </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute data science platform. </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://terra.bio/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://terra.bio/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Panoply </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Broad Institute Proteomics Platform </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/broadinstitute/PANOPLY\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/broadinstitute/PANOPLY </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> CMap </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R93\" class=\"usa-link\" href=\"#R93\"> Lamb et al., 2006 </a> ; <a aria-describedby=\"R167\" class=\"usa-link\" href=\"#R167\"> Subramanian et al., 2017 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://clue.io/cmap\" rel=\"noopener noreferrer\" target=\"_blank\"> https://clue.io/cmap </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> LIMMA v3.36 (R Package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R148\" class=\"usa-link\" href=\"#R148\"> Ritchie et al., 2015 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://bioconductor.org/packages/release/bioc/html/limma.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://bioconductor.org/packages/release/bioc/html/limma.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> FactoMineR v1.41NMF(R - package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R46\" class=\"usa-link\" href=\"#R46\"> Gaujoux and Seoighe, 2010 </a> ; <a aria-describedby=\"R96\" class=\"usa-link\" href=\"#R96\"> Lê et al., 2008 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/FactoMineR/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/FactoMineR/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> MClust v5.4 (R package) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> (Scrucca, Fop, Murphy and Raftery, 2017) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cran.r-project.org/web/packages/mclust/index.html\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cran.r-project.org/web/packages/mclust/index.html </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> g:Profiler </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R145\" class=\"usa-link\" href=\"#R145\"> Raudvere U, et al., 2019 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://biit.cs.ut.ee/gprofiler/gost\" rel=\"noopener noreferrer\" target=\"_blank\"> https://biit.cs.ut.ee/gprofiler/gost </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Cytoscape </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R155\" class=\"usa-link\" href=\"#R155\"> Shannon P, et al., 2003 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://cytoscape.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://cytoscape.org/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ImmuneSubtypeClassifier </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R50\" class=\"usa-link\" href=\"#R50\"> Gibbs, 2020 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\" rel=\"noopener noreferrer\" target=\"_blank\"> https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ProteinPaint </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> ( <a aria-describedby=\"R211\" class=\"usa-link\" href=\"#R211\"> Zhou et al., 2016 </a> ) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://pecan.stjude.doud/proteinpaint/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pecan.stjude.doud/proteinpaint/ </a> </td> </tr> <tr> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Ordinal </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> Christensen RHB (2019) </td> <td align=\"left\" colspan=\"1\" rowspan=\"1\" valign=\"top\"> <a class=\"usa-link usa-link--external\" href=\"https://CRAN.R-project.org/package=ordinal\" rel=\"noopener noreferrer\" target=\"_blank\"> https://CRAN.R-project.org/package=ordinal </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/db_da_table-wrap1_reqid_/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> </section> </section> </section> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1016/j.cell.2021.07.016\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/nihms-1730863.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (3.6 MB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\" hidden=\"\"> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> </div> </div> </div> <div class=\"overlay\" hidden=\"\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br> 8600 Rockville Pike <br> Bethesda, MD 20894 </br> </br> </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> </body> </html>\\n\\nKnown repositories: bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, geo, dbgap, sra, arrayexpress, nuccore, data.mendeley.com, synapse.org, doi.org, pdb\\n\\nReturn only the JSON object with the datasets array - no additional text.'}]\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 132 - DEBUG - Calculating checksum for content: prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.5-flash-0-578cdfabcd621d13b48bd19282421c51e4344b4ffeae069ba0e2c13ea7185bf8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1221 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1228 - DEBUG - Counting tokens for model: gemini-2.5-flash, prompt length: 729175 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1237 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.5-flash': 182293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 182293 tokens, schema: {'type': 'json_schema', 'name': 'PaperMiner_dataset_provenance_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository where the dataset can be found.'}, 'dataset_context_from_paper': {'type': 'string', 'description': 'Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly.'}, 'citation_type': {'type': 'string', 'description': 'Type of citation used for this dataset. It can be either Primary (firsthand information collected by the researcher for a specific purpose) or Secondary (pre-existing information collected by someone else and then used by another researcher).'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference', 'dataset_context_from_paper', 'citation_type']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.5-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c4956ee5-2a91-4610-8a9f-0198c7c48910\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n{\\n  \\\"datasets\\\": [\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000232, PDC000233, PDC000234, PDC000237\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S063\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S063\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000221, PDC000222\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S054\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S054\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000153, PDC000149, PDC000224\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S056\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S056\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"phs001287.v10.p5\\\",\\n      \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CPTAC-LSCC\\\",\\n      \\\"repository_reference\\\": \\\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"GSE92742\\\",\\n      \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"DepMapPublic 21Q1\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CCLE 2019\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"DEMETER2 Data v6\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Sanger GDSC 1\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PRISM Repurposing 19Q4 Secondary Screen\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PhosphoSitePlus\\\",\\n      \\\"repository_reference\\\": \\\"https://www.phosphosite.org\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Human Protein Atlas\\\",\\n      \\\"repository_reference\\\": \\\"https://www.proteinatlas.org\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CTdatabase\\\",\\n      \\\"repository_reference\\\": \\\"http://www.cta.lncc.br\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"LINCS L1000 resource\\\",\\n      \\\"repository_reference\\\": \\\"https://clue.io/data\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"miRTarBase database\\\",\\n      \\\"repository_reference\\\": \\\"Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"TCGA LSCC cohort\\\",\\n      \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S2\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S3\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S4\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S5\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S6\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S7\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    }\\n  ]\\n}\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n{\\n  \\\"datasets\\\": [\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000232, PDC000233, PDC000234, PDC000237\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S063\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S063\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000221, PDC000222\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S054\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S054\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PDC000153, PDC000149, PDC000224\\\",\\n      \\\"repository_reference\\\": \\\"https://pdc.cancer.gov/pdc/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"S056\\\",\\n      \\\"repository_reference\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S056\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"phs001287.v10.p5\\\",\\n      \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CPTAC-LSCC\\\",\\n      \\\"repository_reference\\\": \\\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"GSE92742\\\",\\n      \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"DepMapPublic 21Q1\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CCLE 2019\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"DEMETER2 Data v6\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Sanger GDSC 1\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PRISM Repurposing 19Q4 Secondary Screen\\\",\\n      \\\"repository_reference\\\": \\\"https://depmap.org/portal/download/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"PhosphoSitePlus\\\",\\n      \\\"repository_reference\\\": \\\"https://www.phosphosite.org\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Human Protein Atlas\\\",\\n      \\\"repository_reference\\\": \\\"https://www.proteinatlas.org\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"CTdatabase\\\",\\n      \\\"repository_reference\\\": \\\"http://www.cta.lncc.br\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"LINCS L1000 resource\\\",\\n      \\\"repository_reference\\\": \\\"https://clue.io/data\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"miRTarBase database\\\",\\n      \\\"repository_reference\\\": \\\"Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"TCGA LSCC cohort\\\",\\n      \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).\\\",\\n      \\\"citation_type\\\": \\\"Secondary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S2\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S3\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S4\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S5\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S6\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    },\\n    {\\n      \\\"dataset_identifier\\\": \\\"Table S7\\\",\\n      \\\"repository_reference\\\": \\\"Supplementary Materials of this publication\\\",\\n      \\\"dataset_context_from_paper\\\": \\\"Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.\\\",\\n      \\\"citation_type\\\": \\\"Primary\\\"\\n    }\\n  ]\\n}\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1759538118,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 244549,\n",
      "        \"completion_tokens\": 3209,\n",
      "        \"total_tokens\": 251872,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4114,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - DEBUG - Calling process_llm_response with raw_response type: <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>\u001b[0m\n",
      "\u001b[97mllm_client.py - line 274 - DEBUG - process_llm_response called with model: gemini-2.5-flash\u001b[0m\n",
      "\u001b[97mllm_client.py - line 276 - DEBUG - response_format: {'type': 'json_schema', 'name': 'PaperMiner_dataset_provenance_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository where the dataset can be found.'}, 'dataset_context_from_paper': {'type': 'string', 'description': 'Relevant text passages from the paper that either describe this dataset and provide context of its use or refer to it more implicitly.'}, 'citation_type': {'type': 'string', 'description': 'Type of citation used for this dataset. It can be either Primary (firsthand information collected by the researcher for a specific purpose) or Secondary (pre-existing information collected by someone else and then used by another researcher).'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference', 'dataset_context_from_paper', 'citation_type']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 277 - DEBUG - expected_key: datasets\u001b[0m\n",
      "\u001b[97mllm_client.py - line 315 - DEBUG - Processing Gemini model response, use_portkey: True\u001b[0m\n",
      "\u001b[97mllm_client.py - line 384 - DEBUG - safe_parse_json called with type: <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 406 - DEBUG - Extracted content from response object, type: <class 'str'>\u001b[0m\n",
      "\u001b[97mllm_client.py - line 437 - DEBUG - Cleaned response text for JSON parsing: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"PDC000232, PDC000233, PDC000234, PDC000237\",\n",
      "      \"repository_reference\": \"https://pdc.cancer.gov/pdc/\",\n",
      "      \"dataset_context_from_paper\": \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionab\u001b[0m\n",
      "\u001b[97mllm_client.py - line 387 - DEBUG - safe_parse_json result: {'datasets': [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 319 - DEBUG - Gemini Portkey parsed response: {'datasets': [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 324 - DEBUG - Gemini Portkey result: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]\u001b[0m\n",
      "\u001b[97mllm_client.py - line 348 - DEBUG - normalize_response_format called with response type: <class 'list'>\u001b[0m\n",
      "\u001b[97mllm_client.py - line 349 - DEBUG - normalize_response_format input: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 0: {'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 1: {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 2: {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 3: {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 4: {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 5: {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 6: {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 7: {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 8: {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 9: {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 10: {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 11: {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 12: {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 13: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 14: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 15: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 16: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 17: {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 18: {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 19: {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 20: {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 21: {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 22: {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 23: {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 24: {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 25: {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 366 - DEBUG - Processing item 26: {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 373 - DEBUG - Adding item to normalized: {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 376 - DEBUG - normalize_response_format final result: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - DEBUG - process_llm_response returned: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}] (type: <class 'list'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 335 - DEBUG - Applying normalize_response_type to: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - DEBUG - normalize_response_type called with response type: <class 'list'>, length: 27\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 558 - DEBUG - normalize_response_type input: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 560 - INFO - Deduplicating response with 27 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 0: {'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 1: {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 2: {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 3: {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 4: {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 5: {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 6: {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 7: {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 8: {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 9: {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 10: {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 11: {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: DEMETER2 Data v6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: DEMETER2 Data v6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: DEMETER2 Data v6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: DEMETER2 Data v6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 12: {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Sanger GDSC 1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Sanger GDSC 1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Sanger GDSC 1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Sanger GDSC 1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 13: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 14: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 15: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://www.proteinatlas.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 16: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 17: {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 18: {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: miRTarBase database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: miRTarBase database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: miRTarBase database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: miRTarBase database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 19: {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: TCGA LSCC cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: TCGA LSCC cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: TCGA LSCC cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: TCGA LSCC cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 20: {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 21: {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 22: {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 23: {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 24: {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 25: {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 572 - DEBUG - Processing item 26: {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'} (type: <class 'dict'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 573 - DEBUG - Processing item: {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - DEBUG - Extracted dataset_id: Table S7\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - DEBUG - Extracted repo: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - DEBUG - Normalized clean_id: Table S7\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 601 - DEBUG - Adding unique item: Table S7\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 602 - INFO - Adding unique item: Table S7\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 614 - DEBUG - normalize_response_type final result: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'PDC000232, PDC000233, PDC000234, PDC000237'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'S063'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000221, PDC000222'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S054'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000153, PDC000149, PDC000224'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S056'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary', 'dataset_id': 'phs001287.v10.p5'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary', 'dataset_id': 'CPTAC-LSCC'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary', 'dataset_id': 'GSE92742'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary', 'dataset_id': 'DepMapPublic 21Q1'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary', 'dataset_id': 'CCLE 2019'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'DEMETER2 Data v6'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'Sanger GDSC 1'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'PRISM Repurposing 19Q4 Secondary Screen'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary', 'dataset_id': 'PhosphoSitePlus'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary', 'dataset_id': 'Human Protein Atlas'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary', 'dataset_id': 'CTdatabase'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary', 'dataset_id': 'LINCS L1000 resource'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary', 'dataset_id': 'miRTarBase database'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary', 'dataset_id': 'TCGA LSCC cohort'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary', 'dataset_id': 'Table S1'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary', 'dataset_id': 'Table S2'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary', 'dataset_id': 'Table S3'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary', 'dataset_id': 'Table S4'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary', 'dataset_id': 'Table S5'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary', 'dataset_id': 'Table S6'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary', 'dataset_id': 'Table S7'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 337 - DEBUG - normalize_response_type returned: [{'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'PDC000232, PDC000233, PDC000234, PDC000237'}, {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'S063'}, {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000221, PDC000222'}, {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S054'}, {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000153, PDC000149, PDC000224'}, {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S056'}, {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary', 'dataset_id': 'phs001287.v10.p5'}, {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary', 'dataset_id': 'CPTAC-LSCC'}, {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary', 'dataset_id': 'GSE92742'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary', 'dataset_id': 'DepMapPublic 21Q1'}, {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary', 'dataset_id': 'CCLE 2019'}, {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'DEMETER2 Data v6'}, {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'Sanger GDSC 1'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'PRISM Repurposing 19Q4 Secondary Screen'}, {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary', 'dataset_id': 'PhosphoSitePlus'}, {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary', 'dataset_id': 'Human Protein Atlas'}, {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary', 'dataset_id': 'CTdatabase'}, {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary', 'dataset_id': 'LINCS L1000 resource'}, {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary', 'dataset_id': 'miRTarBase database'}, {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary', 'dataset_id': 'TCGA LSCC cohort'}, {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary', 'dataset_id': 'Table S1'}, {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary', 'dataset_id': 'Table S2'}, {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary', 'dataset_id': 'Table S3'}, {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary', 'dataset_id': 'Table S4'}, {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary', 'dataset_id': 'Table S5'}, {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary', 'dataset_id': 'Table S6'}, {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary', 'dataset_id': 'Table S7'}] (type: <class 'list'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PDC000232, PDC000233, PDC000234, PDC000237', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'PDC000232, PDC000233, PDC000234, PDC000237'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://pdc.cancer.gov/pdc/, identifier: PDC000232, PDC000233, PDC000234, PDC000237, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier PDC000232, PDC000233, PDC000234, PDC000237 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier PDC000232, PDC000233, PDC000234, PDC000237 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1049 - INFO - Found id_pattern match for https://pdc.cancer.gov/pdc/ in PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: PDC000232, PDC000233, PDC000234, PDC000237, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: PDC000232, PDC000233, PDC000234, PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: PDC000232, PDC000233, PDC000234, PDC000237, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 415 - INFO - Multiple dataset IDs possibly found in dataset_id: PDC000232, PDC000233, PDC000234, PDC000237. Splitting and processing each.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'S063', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S063', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\", 'citation_type': 'Primary', 'dataset_id': 'S063'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 882 - INFO - Accession ID S063 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S063, identifier: S063, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S063 == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier S063 for https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/study-summary/S063,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S063\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: cptac-data-portal.georgetown.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: S063, cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'S063', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PDC000221, PDC000222', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000221, PDC000222'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://pdc.cancer.gov/pdc/, identifier: PDC000221, PDC000222, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier PDC000221, PDC000222 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier PDC000221, PDC000222 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1049 - INFO - Found id_pattern match for https://pdc.cancer.gov/pdc/ in PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: PDC000221, PDC000222, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: PDC000221, PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: PDC000221, PDC000222, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 415 - INFO - Multiple dataset IDs possibly found in dataset_id: PDC000221, PDC000222. Splitting and processing each.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'S054', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S054', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S054'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S054, identifier: S054, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S054 == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier S054 for https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/study-summary/S054,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S054\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: cptac-data-portal.georgetown.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data., cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: cptac-data-portal.georgetown.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data., cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PDC000153, PDC000149, PDC000224', 'repository_reference': 'https://pdc.cancer.gov/pdc/', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'PDC000153, PDC000149, PDC000224'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://pdc.cancer.gov/pdc/, identifier: PDC000153, PDC000149, PDC000224, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier PDC000153, PDC000149, PDC000224 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://pdc.cancer.gov/pdc/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier PDC000153, PDC000149, PDC000224 for https://pdc.cancer.gov/pdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1049 - INFO - Found id_pattern match for https://pdc.cancer.gov/pdc/ in PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: PDC[0-9]+ to value: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: PDC000153, PDC000149, PDC000224, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: PDC000153, PDC000149, PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: pdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: PDC000153, PDC000149, PDC000224, pdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 415 - INFO - Multiple dataset IDs possibly found in dataset_id: PDC000153, PDC000149, PDC000224. Splitting and processing each.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'S056', 'repository_reference': 'https://cptac-data-portal.georgetown.edu/study-summary/S056', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'citation_type': 'Secondary', 'dataset_id': 'S056'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S056, identifier: S056, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://cptac-data-portal.georgetown.edu/study-summary/S056 == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier S056 for https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://cptac-data-portal.georgetown.edu/study-summary/S056,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S056\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: cptac-data-portal.georgetown.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data., cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: cptac-data-portal.georgetown.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data., cptac-data-portal.georgetown.edu, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'phs001287.v10.p5', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.', 'citation_type': 'Primary', 'dataset_id': 'phs001287.v10.p5'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ to value: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: phs001287.v10.p5, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier phs001287.v10.p5 for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier phs001287.v10.p5 for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier phs001287.v10.p5 for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier phs001287.v10.p5 for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1049 - INFO - Found id_pattern match for https://portal.gdc.cancer.gov/ in phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ to value: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: phs001287.v10.p5, dbgap, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: phs001287.v10.p5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: phs001287.v10.p5, dbgap, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'phs001287.v10.p5', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'CPTAC-LSCC', 'repository_reference': 'https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.', 'citation_type': 'Primary', 'dataset_id': 'CPTAC-LSCC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 882 - INFO - Accession ID CPTAC-LSCC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC, identifier: CPTAC-LSCC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CPTAC-LSCC for https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: wiki.cancerimagingarchive.net\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: CPTAC-LSCC, wiki.cancerimagingarchive.net, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'CPTAC-LSCC', 'data_repository': 'wiki.cancerimagingarchive.net', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE92742', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.', 'citation_type': 'Secondary', 'dataset_id': 'GSE92742'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742, identifier: GSE92742, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier GSE92742 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier GSE92742 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE92742 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1049 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 483 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 486 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 491 - INFO - Extracted ID: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 486 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 491 - INFO - Extracted ID: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: GSE92742, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 531 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 894 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742, resolved_repo geo, dataset_id GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 947 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 901 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 903 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: GSE92742, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'GSE92742', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'DepMapPublic 21Q1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.', 'citation_type': 'Secondary', 'dataset_id': 'DepMapPublic 21Q1'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/, identifier: DepMapPublic 21Q1, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier DepMapPublic 21Q1 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://depmap.org/portal/download/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: DepMapPublic 21Q1, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: DepMapPublic 21Q1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: DepMapPublic 21Q1, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'DepMapPublic 21Q1', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'CCLE 2019', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.', 'citation_type': 'Secondary', 'dataset_id': 'CCLE 2019'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/, identifier: CCLE 2019, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CCLE 2019 for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://depmap.org/portal/download/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: CCLE 2019, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: CCLE 2019\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: CCLE 2019, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'CCLE 2019', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'DEMETER2 Data v6', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'DEMETER2 Data v6'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: DEMETER2 Data v6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID DEMETER2 Data v6 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://depmap.org/portal/download/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Sanger GDSC 1', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'Sanger GDSC 1'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Sanger GDSC 1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Sanger GDSC 1 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://depmap.org/portal/download/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'repository_reference': 'https://depmap.org/portal/download/', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.', 'citation_type': 'Secondary', 'dataset_id': 'PRISM Repurposing 19Q4 Secondary Screen'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/, identifier: PRISM Repurposing 19Q4 Secondary Screen, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://depmap.org/portal/download/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PRISM Repurposing 19Q4 Secondary Screen for https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://depmap.org/portal/download/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: PRISM Repurposing 19Q4 Secondary Screen, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: PRISM Repurposing 19Q4 Secondary Screen\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: depmap.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: PRISM Repurposing 19Q4 Secondary Screen, depmap.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PhosphoSitePlus', 'repository_reference': 'https://www.phosphosite.org', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.', 'citation_type': 'Secondary', 'dataset_id': 'PhosphoSitePlus'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: PhosphoSitePlus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 882 - INFO - Accession ID PhosphoSitePlus is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://www.phosphosite.org, identifier: PhosphoSitePlus, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.phosphosite.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PhosphoSitePlus for https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://www.phosphosite.org,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: www.phosphosite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: PhosphoSitePlus, www.phosphosite.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'PhosphoSitePlus', 'data_repository': 'www.phosphosite.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.', 'citation_type': 'Secondary', 'dataset_id': 'Human Protein Atlas'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Human Protein Atlas is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://www.proteinatlas.org, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://www.proteinatlas.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://www.proteinatlas.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://www.proteinatlas.org,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://www.proteinatlas.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: www.proteinatlas.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, www.proteinatlas.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'CTdatabase', 'repository_reference': 'http://www.cta.lncc.br', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.', 'citation_type': 'Secondary', 'dataset_id': 'CTdatabase'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: CTdatabase\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 882 - INFO - Accession ID CTdatabase is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: http://www.cta.lncc.br, identifier: CTdatabase, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if http://www.cta.lncc.br == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CTdatabase for http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(http://www.cta.lncc.br,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: http://www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: www.cta.lncc.br\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: CTdatabase, www.cta.lncc.br, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'CTdatabase', 'data_repository': 'www.cta.lncc.br', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'LINCS L1000 resource', 'repository_reference': 'https://clue.io/data', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).', 'citation_type': 'Secondary', 'dataset_id': 'LINCS L1000 resource'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 500 - INFO - Setting dataset_id to value: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 503 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://clue.io/data, identifier: LINCS L1000 resource, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern MSV\\d+ match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern PDC[0-9]+ match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern syn\\d{7,12} match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://clue.io/data == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1047 - DEBUG - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier LINCS L1000 resource for https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://clue.io/data,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://clue.io/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: clue.io\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 480 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: LINCS L1000 resource, clue.io, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset ID found via pattern matching: LINCS L1000 resource\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 523 - INFO - Data repository found via pattern matching: clue.io\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: LINCS L1000 resource, clue.io, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'LINCS L1000 resource', 'data_repository': 'clue.io', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'miRTarBase database', 'repository_reference': 'Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.', 'citation_type': 'Secondary', 'dataset_id': 'miRTarBase database'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: miRTarBase database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID miRTarBase database is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 832 - ERROR - Error extracting domain from URL: Not specified in text, commonly accessible via https://mirtarbase.cuhk.edu.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, unknown_publisher, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'TCGA LSCC cohort', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).', 'citation_type': 'Secondary', 'dataset_id': 'TCGA LSCC cohort'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: TCGA LSCC cohort\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID TCGA LSCC cohort is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 739 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if https://portal.gdc.cancer.gov/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(https://portal.gdc.cancer.gov/,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 821 - DEBUG - Repo Domain: portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, portal.gdc.cancer.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S1', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.', 'citation_type': 'Primary', 'dataset_id': 'Table S1'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S1 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S2', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.', 'citation_type': 'Primary', 'dataset_id': 'Table S2'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S2 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S3', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.', 'citation_type': 'Primary', 'dataset_id': 'Table S3'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S3 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S4', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.', 'citation_type': 'Primary', 'dataset_id': 'Table S4'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S4 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S5', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.', 'citation_type': 'Primary', 'dataset_id': 'Table S5'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S5 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S6', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.', 'citation_type': 'Primary', 'dataset_id': 'Table S6'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S6 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table S7', 'repository_reference': 'Supplementary Materials of this publication', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.', 'citation_type': 'Primary', 'dataset_id': 'Table S7'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 510 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 864 - INFO - Validating accession ID: Table S7\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 844 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 869 - INFO - Accession ID Table S7 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 990 - INFO - Resolving data repository for candidate: Supplementary Materials of this publication, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == ega-archive.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - DEBUG - Checking if Supplementary Materials of this publication == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 809 - INFO - Function call: url_to_repo_domain(Supplementary Materials of this publication,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 817 - INFO - Extracting repo domain from URL: Supplementary Materials of this publication\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1056 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 529 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 534 - INFO - Final schema validation vals: n/a, supplementary materials of this publication, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 445 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 447 - DEBUG - Final result: [{'dataset_identifier': 'PDC000232', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000233', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000234', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000237', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'S063', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000221', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000222', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000153', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000149', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000224', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'phs001287.v10.p5', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.'}, {'dataset_identifier': 'CPTAC-LSCC', 'data_repository': 'wiki.cancerimagingarchive.net', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.'}, {'dataset_identifier': 'GSE92742', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.'}, {'dataset_identifier': 'CCLE 2019', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.'}, {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.'}, {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.'}, {'dataset_identifier': 'PhosphoSitePlus', 'data_repository': 'www.phosphosite.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.'}, {'dataset_identifier': 'CTdatabase', 'data_repository': 'www.cta.lncc.br', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.'}, {'dataset_identifier': 'LINCS L1000 resource', 'data_repository': 'clue.io', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).'}, {'dataset_identifier': 'n/a', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.'}]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 381 - INFO - Augmented dataset links: [{'dataset_identifier': 'PDC000232', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000233', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000234', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000237', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"Proteomic data (global protein, phosphoproteome, acetylproteome, and ubiquitylproteome) generated from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'S063', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': \"CPTAC LSCC proteomics data from 108 prospectively-collected, treatment-naïve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs). This data was used to characterize the proteogenomic landscape of LSCC, identify actionable therapeutic protein targets, and elucidate cellular signaling pathways and crosstalk between multiple PTMs. This study's proteogenomic profiles provide an exceptional resource to guide further research.\"}, {'dataset_identifier': 'PDC000221', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000222', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000153', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000149', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'PDC000224', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.', 'data_repository': 'cptac-data-portal.georgetown.edu', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data.'}, {'dataset_identifier': 'phs001287.v10.p5', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Genomic (whole exome sequencing, WES; whole genome sequencing, WGS) and transcriptomic (total RNA-seq, miRNA-seq) data files for LSCC tumors and paired normal adjacent tissues (NATs) collected from 108 patients in this study. Used for genomic landscape analysis, somatic variant calling, gene expression quantification, and DNA methylation analysis.'}, {'dataset_identifier': 'CPTAC-LSCC', 'data_repository': 'wiki.cancerimagingarchive.net', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Histologic and radiologic details for the 108 LSCC tumors and 99 paired normal adjacent tissues (NATs) samples characterized in this study. Used for pathological assessment and confirmation of LSCC cases.'}, {'dataset_identifier': 'GSE92742', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Connectivity Map (CMAP) dataset, which is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. Used to identify candidate driver genes by correlating proteome profiles of copy number-altered samples with gene knockdown mRNA profiles.'}, {'dataset_identifier': 'DepMapPublic 21Q1', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing mutation, segmented copy number, gene-level copy number, and RNAseq (transcript isoform) data. Used to determine copy number status in LSCC cell lines, investigate copy number and dependency associations, and assess genetic dependencies for genes like SOX2.'}, {'dataset_identifier': 'CCLE 2019', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing Proteomics (RPPA) data. Used for phospho-Rb S807/811 RPPA abundance in LSCC cell lines to correlate with drug response.'}, {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing shRNA screen (combined) data. Used to correlate EZH2 dependency by shRNA with SOX2 copy number in LSCC cell lines.'}, {'dataset_identifier': 'n/a', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing GDSC drug screen data. Used for palbociclib drug response analysis in LSCC cell lines.'}, {'dataset_identifier': 'PRISM Repurposing 19Q4 Secondary Screen', 'data_repository': 'depmap.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'DepMap dataset containing PRISM drug screen data. Used for abemaciclib drug response analysis and response to survivin (BIRC5) inhibitor YM-155 in LSCC cell lines.'}, {'dataset_identifier': 'PhosphoSitePlus', 'data_repository': 'www.phosphosite.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A comprehensive resource used for site-specific annotation in PTM-SEA analysis and for activating/inhibiting classification of phosphorylation sites in CausalPath analysis.'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Used to obtain immunohistochemistry-based antibody-specific staining scores in lung tumors.'}, {'dataset_identifier': 'CTdatabase', 'data_repository': 'www.cta.lncc.br', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'A knowledge-base of high-throughput and curated data on cancer-testis antigens. Used to download cancer/testis (CT) antigens for analysis.'}, {'dataset_identifier': 'LINCS L1000 resource', 'data_repository': 'clue.io', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'Library of Integrated Network-based Cellular Signatures (LINCS) L1000 perturbation-response signatures. Used to query for compounds that reversed specific NMF multi-omic subtype signatures (e.g., EMT-E signature).'}, {'dataset_identifier': 'n/a', 'data_repository': 'unknown_publisher', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'miRNA targets database from which targets of miRNAs were downloaded. Only miRNA/target pairs with strong experimental evidence were retained and used to calculate correlations between miRNA and its target genes.'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'Secondary', 'dataset_context_from_paper': 'The TCGA LSCC dataset was leveraged to examine the association of 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS).'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains metadata associated with LSCC patients and tumors, including demographic, histopathologic, and treatment details, as well as Cancer Associated Genes (CAGs) and results from association analyses between metadata and NMF multi-omic subtyping/immune clusters.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains genomic and methylation datasets associated with this study, including gene-level data matrices for somatic copy number alterations (CNA), RNA expression (FPKM and median-centered), miRNA expression (TPM and median-centered), gene level DNA methylation, and somatic mutations in MAF format.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study. This includes two-component-normalized log2-transformed data matrices (sample/CR ratios) before and after filtering, and gene-centric versions derived from these.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains data related to CNA cis and trans effects, iProFun results, CMAP results, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, and CDKN2A/RB1 annotations.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with ubiquitylproteome analysis, including clustering of tumor samples, Ub-sites with differential abundance, pathway enrichment results for Ub-sites, and associations between Ub-sites and E3 ligases/DUBs, and significant spatial clustering data.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with immune-landscape analysis, including immune composition based on xCell, immune clusters, pathways/genes upregulated in immune groups (RNA-Seq, proteomics, phosphoproteomics, acetylproteomics), RTK Correlation-Based Enrichment (CBPE) Scores, cis-eQTLs/pQTLs, and Ingenuity pathway analysis of eGenes.'}, {'dataset_identifier': 'n/a', 'data_repository': 'supplementary materials of this publication', 'dataset_webpage': 'n/a', 'citation_type': 'Primary', 'dataset_context_from_paper': 'Contains tables associated with tumor-associated proteins, including differential abundance of proteins in tumor vs paired NAT in LSCC, HNSCC, and LUAD, significantly overexpressed proteins, tumor suppressors/oncogenes regulated at protein/phosphoproteome level, DepMap dependency scores, genes correlated to immune score, and predicted tumor neoantigens.'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1083 - INFO - Enhancing dataset pages for 33 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 1 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000232 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000232\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 1: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000232\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 1: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 2 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000233 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000233\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 2: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000233\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 2: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 3 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000234 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000234\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 3: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000234\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 3: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 4 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000237 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 4: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000237\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 4: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 5 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for S063 in cptac-data-portal.georgetown.edu, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 6 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000221 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000221\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 6: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000221\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 6: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 7 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000222 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 7: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000222\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 7: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 8 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for CPTAC HNSCC proteomics data for 108 head and neck squamous cell carcinoma samples and 66 normal adjacent tissues (NAT) that were acquired from a previously submitted manuscript (Huang et al., 2021). This data was used for integrative analysis alongside the LSCC data. in cptac-data-portal.georgetown.edu, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 9 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000153 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000153\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 9: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000153\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 9: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 10 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000149 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000149\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 10: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000149\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 10: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 11 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PDC000224 in pdc.cancer.gov, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for pdc.cancer.gov in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern PDC[0-9]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 11: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000224\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 11: Complex download or Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 12 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for CPTAC LUAD proteomics data for 110 lung adenocarcinoma samples and 102 normal adjacent tissues (NAT) that were acquired from the published manuscript (Gillette et al., 2020). This data was used for integrative analysis alongside the LSCC data. in cptac-data-portal.georgetown.edu, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository cptac-data-portal.georgetown.edu not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 12\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 13 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for phs001287.v10.p5 in dbgap, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 965 - WARNING - id_pattern not defined for dbgap in ontology. Add definition to improve validation.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 968 - INFO - Extracting ID from dataset page n/a using pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1153 - INFO - Constructed webpage URL: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001287.v10.p5.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1125 - INFO - Added dataset webpage for 13: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001287.v10.p5.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 13: Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 14 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for CPTAC-LSCC in wiki.cancerimagingarchive.net, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository wiki.cancerimagingarchive.net not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository wiki.cancerimagingarchive.net not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 14\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1093 - DEBUG - Dataset 15 already has valid webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1177 - INFO - Added access mode for dataset 15: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 16 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for DepMapPublic 21Q1 in depmap.org, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 16\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 17 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for CCLE 2019 in depmap.org, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 17\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 18 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 18: missing repo (depmap.org) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 19 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 19: missing repo (depmap.org) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 20 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PRISM Repurposing 19Q4 Secondary Screen in depmap.org, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository depmap.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 20\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 21 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for PhosphoSitePlus in www.phosphosite.org, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository www.phosphosite.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository www.phosphosite.org not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 21\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 22 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 22: missing repo (www.proteinatlas.org) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 23 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for CTdatabase in www.cta.lncc.br, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository www.cta.lncc.br not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository www.cta.lncc.br not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 23\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 24 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Resolving accession ID for LINCS L1000 resource in clue.io, resolved page: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 978 - WARNING - Repository clue.io not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1165 - WARNING - Repository clue.io not found in ontology\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1127 - WARNING - Could not construct valid webpage for dataset 24\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 25 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 25: missing repo (unknown_publisher) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 26 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 26: missing repo (portal.gdc.cancer.gov) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 27 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 27: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 28 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 28: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 29 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 29: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 30 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 30: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 31 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 31: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 32 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 32: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Processing dataset 33 - missing or invalid webpage\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1105 - INFO - Skipping dataset 33: missing repo (supplementary materials of this publication) or accession_id (n/a)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1133 - INFO - Dataset enhancement completed: 33 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 417 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 139, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'dataset_context_from_paper', 'access_mode', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-1.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-2.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-3.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-4.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-6.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-7.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-8.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-9.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-10.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-11.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-12.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-13.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-14.xlsx)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8475722/bin/NIHMS1730863-supplement-Figure_S5.pdf)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 92 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 646 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 547 - INFO - Parsed data extraction completed. Elements collected: 37\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 590 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 729 - INFO - \n",
      "Progress: 1/1 (100.00%) | Elapsed: 00:03:49 | ETA: 00:00:00\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 734 - DEBUG - Completed processing all URLs.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "parsed_data = data_gatherer.process_articles(\n",
    "    articles[:1], \n",
    "    response_format=dataset_response_schema_with_use_description,\n",
    "    prompt_name=\"GPT_FDR_FewShot_Descr\",\n",
    "    grobid_for_pdf=False,\n",
    "    write_htmls_xmls=True,\n",
    "    article_file_dir='scripts/tmp/raw_files/'\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "961e2338",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.DataFrame()\n",
    "\n",
    "for k, v in parsed_data.items():\n",
    "    df = pd.concat([df, v], ignore_index=True)\n",
    "\n",
    "df.to_csv('scripts/output/bdf_parsed_data_gemini_flash_xml.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "da823880",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>citation_type</th>\n",
       "      <th>dataset_context_from_paper</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>context_description</th>\n",
       "      <th>source_url</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>raw_data_format</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PDC000232</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Proteomic data (global protein, phosphoproteom...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PDC000233</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Proteomic data (global protein, phosphoproteom...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PDC000234</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Proteomic data (global protein, phosphoproteom...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PDC000237</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Proteomic data (global protein, phosphoproteom...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>S063</td>\n",
       "      <td>cptac-data-portal.georgetown.edu</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Primary</td>\n",
       "      <td>CPTAC LSCC proteomics data from 108 prospectiv...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>PDC000221</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC HNSCC proteomics data for 108 head and n...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>PDC000222</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC HNSCC proteomics data for 108 head and n...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>CPTAC HNSCC proteomics data for 108 head and n...</td>\n",
       "      <td>cptac-data-portal.georgetown.edu</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC HNSCC proteomics data for 108 head and n...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>PDC000153</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC LUAD proteomics data for 110 lung adenoc...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>PDC000149</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC LUAD proteomics data for 110 lung adenoc...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>PDC000224</td>\n",
       "      <td>pdc.cancer.gov</td>\n",
       "      <td>https://proteomic.datacommons.cancer.gov/pdc/s...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC LUAD proteomics data for 110 lung adenoc...</td>\n",
       "      <td>Complex download or Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>CPTAC LUAD proteomics data for 110 lung adenoc...</td>\n",
       "      <td>cptac-data-portal.georgetown.edu</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>CPTAC LUAD proteomics data for 110 lung adenoc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>phs001287.v10.p5</td>\n",
       "      <td>dbgap</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Genomic (whole exome sequencing, WES; whole ge...</td>\n",
       "      <td>Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>CPTAC-LSCC</td>\n",
       "      <td>wiki.cancerimagingarchive.net</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Primary</td>\n",
       "      <td>Histologic and radiologic details for the 108 ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>GSE92742</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>Connectivity Map (CMAP) dataset, which is a co...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>DepMapPublic 21Q1</td>\n",
       "      <td>depmap.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>DepMap dataset containing mutation, segmented ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>CCLE 2019</td>\n",
       "      <td>depmap.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>DepMap dataset containing Proteomics (RPPA) da...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>n/a</td>\n",
       "      <td>depmap.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>DepMap dataset containing shRNA screen (combin...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>PRISM Repurposing 19Q4 Secondary Screen</td>\n",
       "      <td>depmap.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>DepMap dataset containing PRISM drug screen da...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>PhosphoSitePlus</td>\n",
       "      <td>www.phosphosite.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>A comprehensive resource used for site-specifi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>CTdatabase</td>\n",
       "      <td>www.cta.lncc.br</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>A knowledge-base of high-throughput and curate...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>LINCS L1000 resource</td>\n",
       "      <td>clue.io</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Secondary</td>\n",
       "      <td>Library of Integrated Network-based Cellular S...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.1016/j.cell.2021.07.016</td>\n",
       "      <td>10.1016/j.cell.2021.07.016</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>https://doi.org/10.1016/j.cell.2021.07.016</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "      <td></td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-1.pdf</td>\n",
       "      <td>(5.6MB, pdf)</td>\n",
       "      <td>Figure S1: Experimental Design and Proteogenom...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>We performed deep-scale molecular analysis spa...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-2.pdf</td>\n",
       "      <td>(22.7MB, pdf)</td>\n",
       "      <td>Figure S2: Characteristics of Multi-omic NMF S...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The five resulting multi-omic subtypes ( Figur...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-3.pdf</td>\n",
       "      <td>(4.6MB, pdf)</td>\n",
       "      <td>Figure S3: Proteogenomic Consequences of CDKN2...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Mutations were associated with higher tumor mu...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-4.pdf</td>\n",
       "      <td>(4MB, pdf)</td>\n",
       "      <td>Figure S4: Association Between TP63 and miR205...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Chromosome 3q, which harbors key squamous diff...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-6.pdf</td>\n",
       "      <td>(6.5MB, pdf)</td>\n",
       "      <td>Figure S6: Differential pathway, CBPE scores a...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The Hot cluster was characterized by increased...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-7.pdf</td>\n",
       "      <td>(2.7MB, pdf)</td>\n",
       "      <td>Figure S7: Differential tumor and NAT analysis...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The top 50 of 502 differential proteins are sh...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-8.xlsx</td>\n",
       "      <td>(327.4KB, xlsx)</td>\n",
       "      <td>Table S1: Metadata Associated with LSCC Patien...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>We performed deep-scale molecular analysis spa...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-9.xlsx</td>\n",
       "      <td>(218.8MB, xlsx)</td>\n",
       "      <td>Table S2: Genomic and methylation datasets ass...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A third subtype characterized by mutations in ...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-10.xlsx</td>\n",
       "      <td>(372.9MB, xlsx)</td>\n",
       "      <td>Table S3: Global proteomics, phosphoproteomics...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>We performed deep-scale molecular analysis spa...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-11.xlsx</td>\n",
       "      <td>(7.9MB, xlsx)</td>\n",
       "      <td>Table S4: CNA cis and trans effects, iProFun, ...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>We identified six amplified (including , and )...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-12.xlsx</td>\n",
       "      <td>(451KB, xlsx)</td>\n",
       "      <td>Table S5: Tables associated with ubiquitylprot...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Consensus clustering of K-GG peptide abundance...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-13.xlsx</td>\n",
       "      <td>(3.4MB, xlsx)</td>\n",
       "      <td>Table S6: Tables associated with immune-landsc...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Consensus clustering based on the xCell ( Aran...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-14.xlsx</td>\n",
       "      <td>(9.8MB, xlsx)</td>\n",
       "      <td>Table S7: Tables associated with tumor-associa...</td>\n",
       "      <td>...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Tumors and paired NATs revealed remarkable dif...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>NIHMS1730863-supplement-Figure_S5.pdf</td>\n",
       "      <td>(2MB, pdf)</td>\n",
       "      <td>Figure S5: Pathway analysis and differential u...</td>\n",
       "      <td>...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/8475722/bin/NIHMS1730863-su...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Proteins in pathways such as glycolysis, JAK-S...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/</td>\n",
       "      <td>A proteogenomic portrait of lung squamous cell...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>37 rows × 23 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   dataset_identifier  \\\n",
       "0                                           PDC000232   \n",
       "1                                           PDC000233   \n",
       "2                                           PDC000234   \n",
       "3                                           PDC000237   \n",
       "4                                                S063   \n",
       "5                                           PDC000221   \n",
       "6                                           PDC000222   \n",
       "7   CPTAC HNSCC proteomics data for 108 head and n...   \n",
       "8                                           PDC000153   \n",
       "9                                           PDC000149   \n",
       "10                                          PDC000224   \n",
       "11  CPTAC LUAD proteomics data for 110 lung adenoc...   \n",
       "12                                   phs001287.v10.p5   \n",
       "13                                         CPTAC-LSCC   \n",
       "14                                           GSE92742   \n",
       "15                                  DepMapPublic 21Q1   \n",
       "16                                          CCLE 2019   \n",
       "17                                                n/a   \n",
       "18            PRISM Repurposing 19Q4 Secondary Screen   \n",
       "19                                    PhosphoSitePlus   \n",
       "20                                         CTdatabase   \n",
       "21                               LINCS L1000 resource   \n",
       "22                                                NaN   \n",
       "23                                                NaN   \n",
       "24                                                NaN   \n",
       "25                                                NaN   \n",
       "26                                                NaN   \n",
       "27                                                NaN   \n",
       "28                                                NaN   \n",
       "29                                                NaN   \n",
       "30                                                NaN   \n",
       "31                                                NaN   \n",
       "32                                                NaN   \n",
       "33                                                NaN   \n",
       "34                                                NaN   \n",
       "35                                                NaN   \n",
       "36                                                NaN   \n",
       "\n",
       "                     data_repository  \\\n",
       "0                     pdc.cancer.gov   \n",
       "1                     pdc.cancer.gov   \n",
       "2                     pdc.cancer.gov   \n",
       "3                     pdc.cancer.gov   \n",
       "4   cptac-data-portal.georgetown.edu   \n",
       "5                     pdc.cancer.gov   \n",
       "6                     pdc.cancer.gov   \n",
       "7   cptac-data-portal.georgetown.edu   \n",
       "8                     pdc.cancer.gov   \n",
       "9                     pdc.cancer.gov   \n",
       "10                    pdc.cancer.gov   \n",
       "11  cptac-data-portal.georgetown.edu   \n",
       "12                             dbgap   \n",
       "13     wiki.cancerimagingarchive.net   \n",
       "14                               geo   \n",
       "15                        depmap.org   \n",
       "16                        depmap.org   \n",
       "17                        depmap.org   \n",
       "18                        depmap.org   \n",
       "19               www.phosphosite.org   \n",
       "20                   www.cta.lncc.br   \n",
       "21                           clue.io   \n",
       "22                               NaN   \n",
       "23                               NaN   \n",
       "24                               NaN   \n",
       "25                               NaN   \n",
       "26                               NaN   \n",
       "27                               NaN   \n",
       "28                               NaN   \n",
       "29                               NaN   \n",
       "30                               NaN   \n",
       "31                               NaN   \n",
       "32                               NaN   \n",
       "33                               NaN   \n",
       "34                               NaN   \n",
       "35                               NaN   \n",
       "36                               NaN   \n",
       "\n",
       "                                      dataset_webpage citation_type  \\\n",
       "0   https://proteomic.datacommons.cancer.gov/pdc/s...       Primary   \n",
       "1   https://proteomic.datacommons.cancer.gov/pdc/s...       Primary   \n",
       "2   https://proteomic.datacommons.cancer.gov/pdc/s...       Primary   \n",
       "3   https://proteomic.datacommons.cancer.gov/pdc/s...       Primary   \n",
       "4                                                 n/a       Primary   \n",
       "5   https://proteomic.datacommons.cancer.gov/pdc/s...     Secondary   \n",
       "6   https://proteomic.datacommons.cancer.gov/pdc/s...     Secondary   \n",
       "7                                                 n/a     Secondary   \n",
       "8   https://proteomic.datacommons.cancer.gov/pdc/s...     Secondary   \n",
       "9   https://proteomic.datacommons.cancer.gov/pdc/s...     Secondary   \n",
       "10  https://proteomic.datacommons.cancer.gov/pdc/s...     Secondary   \n",
       "11                                                n/a     Secondary   \n",
       "12  https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...       Primary   \n",
       "13                                                n/a       Primary   \n",
       "14  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...     Secondary   \n",
       "15                                                n/a     Secondary   \n",
       "16                                                n/a     Secondary   \n",
       "17                                                n/a     Secondary   \n",
       "18                                                n/a     Secondary   \n",
       "19                                                n/a     Secondary   \n",
       "20                                                n/a     Secondary   \n",
       "21                                                n/a     Secondary   \n",
       "22                                                NaN           NaN   \n",
       "23                                                NaN           NaN   \n",
       "24                                                NaN           NaN   \n",
       "25                                                NaN           NaN   \n",
       "26                                                NaN           NaN   \n",
       "27                                                NaN           NaN   \n",
       "28                                                NaN           NaN   \n",
       "29                                                NaN           NaN   \n",
       "30                                                NaN           NaN   \n",
       "31                                                NaN           NaN   \n",
       "32                                                NaN           NaN   \n",
       "33                                                NaN           NaN   \n",
       "34                                                NaN           NaN   \n",
       "35                                                NaN           NaN   \n",
       "36                                                NaN           NaN   \n",
       "\n",
       "                           dataset_context_from_paper  \\\n",
       "0   Proteomic data (global protein, phosphoproteom...   \n",
       "1   Proteomic data (global protein, phosphoproteom...   \n",
       "2   Proteomic data (global protein, phosphoproteom...   \n",
       "3   Proteomic data (global protein, phosphoproteom...   \n",
       "4   CPTAC LSCC proteomics data from 108 prospectiv...   \n",
       "5   CPTAC HNSCC proteomics data for 108 head and n...   \n",
       "6   CPTAC HNSCC proteomics data for 108 head and n...   \n",
       "7   CPTAC HNSCC proteomics data for 108 head and n...   \n",
       "8   CPTAC LUAD proteomics data for 110 lung adenoc...   \n",
       "9   CPTAC LUAD proteomics data for 110 lung adenoc...   \n",
       "10  CPTAC LUAD proteomics data for 110 lung adenoc...   \n",
       "11  CPTAC LUAD proteomics data for 110 lung adenoc...   \n",
       "12  Genomic (whole exome sequencing, WES; whole ge...   \n",
       "13  Histologic and radiologic details for the 108 ...   \n",
       "14  Connectivity Map (CMAP) dataset, which is a co...   \n",
       "15  DepMap dataset containing mutation, segmented ...   \n",
       "16  DepMap dataset containing Proteomics (RPPA) da...   \n",
       "17  DepMap dataset containing shRNA screen (combin...   \n",
       "18  DepMap dataset containing PRISM drug screen da...   \n",
       "19  A comprehensive resource used for site-specifi...   \n",
       "20  A knowledge-base of high-throughput and curate...   \n",
       "21  Library of Integrated Network-based Cellular S...   \n",
       "22                                                NaN   \n",
       "23                                                NaN   \n",
       "24                                                NaN   \n",
       "25                                                NaN   \n",
       "26                                                NaN   \n",
       "27                                                NaN   \n",
       "28                                                NaN   \n",
       "29                                                NaN   \n",
       "30                                                NaN   \n",
       "31                                                NaN   \n",
       "32                                                NaN   \n",
       "33                                                NaN   \n",
       "34                                                NaN   \n",
       "35                                                NaN   \n",
       "36                                                NaN   \n",
       "\n",
       "                                  access_mode  \\\n",
       "0   Complex download or Application to access   \n",
       "1   Complex download or Application to access   \n",
       "2   Complex download or Application to access   \n",
       "3   Complex download or Application to access   \n",
       "4                                         NaN   \n",
       "5   Complex download or Application to access   \n",
       "6   Complex download or Application to access   \n",
       "7                                         NaN   \n",
       "8   Complex download or Application to access   \n",
       "9   Complex download or Application to access   \n",
       "10  Complex download or Application to access   \n",
       "11                                        NaN   \n",
       "12                      Application to access   \n",
       "13                                        NaN   \n",
       "14                           Complex download   \n",
       "15                                        NaN   \n",
       "16                                        NaN   \n",
       "17                                        NaN   \n",
       "18                                        NaN   \n",
       "19                                        NaN   \n",
       "20                                        NaN   \n",
       "21                                        NaN   \n",
       "22                                        NaN   \n",
       "23                                        NaN   \n",
       "24                                        NaN   \n",
       "25                                        NaN   \n",
       "26                                        NaN   \n",
       "27                                        NaN   \n",
       "28                                        NaN   \n",
       "29                                        NaN   \n",
       "30                                        NaN   \n",
       "31                                        NaN   \n",
       "32                                        NaN   \n",
       "33                                        NaN   \n",
       "34                                        NaN   \n",
       "35                                        NaN   \n",
       "36                                        NaN   \n",
       "\n",
       "                                                 link  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "5                                                 NaN   \n",
       "6                                                 NaN   \n",
       "7                                                 NaN   \n",
       "8                                                 NaN   \n",
       "9                                                 NaN   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "12                                                NaN   \n",
       "13                                                NaN   \n",
       "14                                                NaN   \n",
       "15                                                NaN   \n",
       "16                                                NaN   \n",
       "17                                                NaN   \n",
       "18                                                NaN   \n",
       "19                                                NaN   \n",
       "20                                                NaN   \n",
       "21                                                NaN   \n",
       "22         https://doi.org/10.1016/j.cell.2021.07.016   \n",
       "23  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "24  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "25  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "26  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "27  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "28  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "29  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "30  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "31  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "32  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "33  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "34  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "35  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "36  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "\n",
       "                                    title        file_info  \\\n",
       "0                                     NaN              NaN   \n",
       "1                                     NaN              NaN   \n",
       "2                                     NaN              NaN   \n",
       "3                                     NaN              NaN   \n",
       "4                                     NaN              NaN   \n",
       "5                                     NaN              NaN   \n",
       "6                                     NaN              NaN   \n",
       "7                                     NaN              NaN   \n",
       "8                                     NaN              NaN   \n",
       "9                                     NaN              NaN   \n",
       "10                                    NaN              NaN   \n",
       "11                                    NaN              NaN   \n",
       "12                                    NaN              NaN   \n",
       "13                                    NaN              NaN   \n",
       "14                                    NaN              NaN   \n",
       "15                                    NaN              NaN   \n",
       "16                                    NaN              NaN   \n",
       "17                                    NaN              NaN   \n",
       "18                                    NaN              NaN   \n",
       "19                                    NaN              NaN   \n",
       "20                                    NaN              NaN   \n",
       "21                                    NaN              NaN   \n",
       "22             10.1016/j.cell.2021.07.016              n/a   \n",
       "23          NIHMS1730863-supplement-1.pdf     (5.6MB, pdf)   \n",
       "24          NIHMS1730863-supplement-2.pdf    (22.7MB, pdf)   \n",
       "25          NIHMS1730863-supplement-3.pdf     (4.6MB, pdf)   \n",
       "26          NIHMS1730863-supplement-4.pdf       (4MB, pdf)   \n",
       "27          NIHMS1730863-supplement-6.pdf     (6.5MB, pdf)   \n",
       "28          NIHMS1730863-supplement-7.pdf     (2.7MB, pdf)   \n",
       "29         NIHMS1730863-supplement-8.xlsx  (327.4KB, xlsx)   \n",
       "30         NIHMS1730863-supplement-9.xlsx  (218.8MB, xlsx)   \n",
       "31        NIHMS1730863-supplement-10.xlsx  (372.9MB, xlsx)   \n",
       "32        NIHMS1730863-supplement-11.xlsx    (7.9MB, xlsx)   \n",
       "33        NIHMS1730863-supplement-12.xlsx    (451KB, xlsx)   \n",
       "34        NIHMS1730863-supplement-13.xlsx    (3.4MB, xlsx)   \n",
       "35        NIHMS1730863-supplement-14.xlsx    (9.8MB, xlsx)   \n",
       "36  NIHMS1730863-supplement-Figure_S5.pdf       (2MB, pdf)   \n",
       "\n",
       "                                          description  ... file_extension  \\\n",
       "0                                                 NaN  ...           None   \n",
       "1                                                 NaN  ...           None   \n",
       "2                                                 NaN  ...           None   \n",
       "3                                                 NaN  ...           None   \n",
       "4                                                 NaN  ...           None   \n",
       "5                                                 NaN  ...           None   \n",
       "6                                                 NaN  ...           None   \n",
       "7                                                 NaN  ...           None   \n",
       "8                                                 NaN  ...           None   \n",
       "9                                                 NaN  ...           None   \n",
       "10                                                NaN  ...           None   \n",
       "11                                                NaN  ...           None   \n",
       "12                                                NaN  ...           None   \n",
       "13                                                NaN  ...           None   \n",
       "14                                                NaN  ...           None   \n",
       "15                                                NaN  ...           None   \n",
       "16                                                NaN  ...           None   \n",
       "17                                                NaN  ...           None   \n",
       "18                                                NaN  ...           None   \n",
       "19                                                NaN  ...           None   \n",
       "20                                                NaN  ...           None   \n",
       "21                                                NaN  ...           None   \n",
       "22                                                n/a  ...           None   \n",
       "23  Figure S1: Experimental Design and Proteogenom...  ...            pdf   \n",
       "24  Figure S2: Characteristics of Multi-omic NMF S...  ...            pdf   \n",
       "25  Figure S3: Proteogenomic Consequences of CDKN2...  ...            pdf   \n",
       "26  Figure S4: Association Between TP63 and miR205...  ...            pdf   \n",
       "27  Figure S6: Differential pathway, CBPE scores a...  ...            pdf   \n",
       "28  Figure S7: Differential tumor and NAT analysis...  ...            pdf   \n",
       "29  Table S1: Metadata Associated with LSCC Patien...  ...           xlsx   \n",
       "30  Table S2: Genomic and methylation datasets ass...  ...           xlsx   \n",
       "31  Table S3: Global proteomics, phosphoproteomics...  ...           xlsx   \n",
       "32  Table S4: CNA cis and trans effects, iProFun, ...  ...           xlsx   \n",
       "33  Table S5: Tables associated with ubiquitylprot...  ...           xlsx   \n",
       "34  Table S6: Tables associated with immune-landsc...  ...           xlsx   \n",
       "35  Table S7: Tables associated with tumor-associa...  ...           xlsx   \n",
       "36  Figure S5: Pathway analysis and differential u...  ...            pdf   \n",
       "\n",
       "                                          a_attr_href  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "5                                                 NaN   \n",
       "6                                                 NaN   \n",
       "7                                                 NaN   \n",
       "8                                                 NaN   \n",
       "9                                                 NaN   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "12                                                NaN   \n",
       "13                                                NaN   \n",
       "14                                                NaN   \n",
       "15                                                NaN   \n",
       "16                                                NaN   \n",
       "17                                                NaN   \n",
       "18                                                NaN   \n",
       "19                                                NaN   \n",
       "20                                                NaN   \n",
       "21                                                NaN   \n",
       "22         https://doi.org/10.1016/j.cell.2021.07.016   \n",
       "23  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "24  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "25  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "26  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "27  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "28  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "29  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "30  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "31  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "32  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "33  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "34  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "35  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "36  /articles/instance/8475722/bin/NIHMS1730863-su...   \n",
       "\n",
       "                   a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0                           NaN                   NaN           NaN   \n",
       "1                           NaN                   NaN           NaN   \n",
       "2                           NaN                   NaN           NaN   \n",
       "3                           NaN                   NaN           NaN   \n",
       "4                           NaN                   NaN           NaN   \n",
       "5                           NaN                   NaN           NaN   \n",
       "6                           NaN                   NaN           NaN   \n",
       "7                           NaN                   NaN           NaN   \n",
       "8                           NaN                   NaN           NaN   \n",
       "9                           NaN                   NaN           NaN   \n",
       "10                          NaN                   NaN           NaN   \n",
       "11                          NaN                   NaN           NaN   \n",
       "12                          NaN                   NaN           NaN   \n",
       "13                          NaN                   NaN           NaN   \n",
       "14                          NaN                   NaN           NaN   \n",
       "15                          NaN                   NaN           NaN   \n",
       "16                          NaN                   NaN           NaN   \n",
       "17                          NaN                   NaN           NaN   \n",
       "18                          NaN                   NaN           NaN   \n",
       "19                          NaN                   NaN           NaN   \n",
       "20                          NaN                   NaN           NaN   \n",
       "21                          NaN                   NaN           NaN   \n",
       "22  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       "23                     usa-link      click_feat_suppl           NaN   \n",
       "24                     usa-link      click_feat_suppl           NaN   \n",
       "25                     usa-link      click_feat_suppl           NaN   \n",
       "26                     usa-link      click_feat_suppl           NaN   \n",
       "27                     usa-link      click_feat_suppl           NaN   \n",
       "28                     usa-link      click_feat_suppl           NaN   \n",
       "29                     usa-link      click_feat_suppl           NaN   \n",
       "30                     usa-link      click_feat_suppl           NaN   \n",
       "31                     usa-link      click_feat_suppl           NaN   \n",
       "32                     usa-link      click_feat_suppl           NaN   \n",
       "33                     usa-link      click_feat_suppl           NaN   \n",
       "34                     usa-link      click_feat_suppl           NaN   \n",
       "35                     usa-link      click_feat_suppl           NaN   \n",
       "36                     usa-link      click_feat_suppl           NaN   \n",
       "\n",
       "             a_attr_rel                                context_description  \\\n",
       "0                   NaN                                                NaN   \n",
       "1                   NaN                                                NaN   \n",
       "2                   NaN                                                NaN   \n",
       "3                   NaN                                                NaN   \n",
       "4                   NaN                                                NaN   \n",
       "5                   NaN                                                NaN   \n",
       "6                   NaN                                                NaN   \n",
       "7                   NaN                                                NaN   \n",
       "8                   NaN                                                NaN   \n",
       "9                   NaN                                                NaN   \n",
       "10                  NaN                                                NaN   \n",
       "11                  NaN                                                NaN   \n",
       "12                  NaN                                                NaN   \n",
       "13                  NaN                                                NaN   \n",
       "14                  NaN                                                NaN   \n",
       "15                  NaN                                                NaN   \n",
       "16                  NaN                                                NaN   \n",
       "17                  NaN                                                NaN   \n",
       "18                  NaN                                                NaN   \n",
       "19                  NaN                                                NaN   \n",
       "20                  NaN                                                NaN   \n",
       "21                  NaN                                                NaN   \n",
       "22  noopener noreferrer                                                      \n",
       "23                  NaN  We performed deep-scale molecular analysis spa...   \n",
       "24                  NaN  The five resulting multi-omic subtypes ( Figur...   \n",
       "25                  NaN  Mutations were associated with higher tumor mu...   \n",
       "26                  NaN  Chromosome 3q, which harbors key squamous diff...   \n",
       "27                  NaN  The Hot cluster was characterized by increased...   \n",
       "28                  NaN  The top 50 of 502 differential proteins are sh...   \n",
       "29                  NaN  We performed deep-scale molecular analysis spa...   \n",
       "30                  NaN  A third subtype characterized by mutations in ...   \n",
       "31                  NaN  We performed deep-scale molecular analysis spa...   \n",
       "32                  NaN  We identified six amplified (including , and )...   \n",
       "33                  NaN  Consensus clustering of K-GG peptide abundance...   \n",
       "34                  NaN  Consensus clustering based on the xCell ( Aran...   \n",
       "35                  NaN  Tumors and paired NATs revealed remarkable dif...   \n",
       "36                  NaN  Proteins in pathways such as glycolysis, JAK-S...   \n",
       "\n",
       "                                           source_url  \\\n",
       "0   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "1   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "2   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "3   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "4   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "5   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "6   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "7   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "8   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "9   https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "10  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "11  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "12  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "13  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "14  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "15  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "16  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "17  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "18  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "19  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "20  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "21  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "22  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "23  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "24  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "25  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "26  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "27  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "28  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "29  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "30  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "31  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "32  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "33  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "34  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "35  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "36  https://pmc.ncbi.nlm.nih.gov/articles/PMC8475722/   \n",
       "\n",
       "                                            pub_title raw_data_format  \n",
       "0   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "1   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "2   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "3   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "4   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "5   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "6   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "7   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "8   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "9   A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "10  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "11  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "12  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "13  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "14  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "15  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "16  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "17  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "18  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "19  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "20  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "21  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "22  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "23  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "24  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "25  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "26  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "27  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "28  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "29  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "30  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "31  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "32  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "33  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "34  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "35  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "36  A proteogenomic portrait of lung squamous cell...            HTML  \n",
       "\n",
       "[37 rows x 23 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a89d09bc",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "2025_arm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
