{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-08-27T20:37:40.250800Z",
     "start_time": "2025-08-27T20:37:28.866958Z"
    }
   },
   "source": [
    "import os\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "import pandas as pd"
   ],
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T20:37:40.264366Z",
     "start_time": "2025-08-27T20:37:40.252679Z"
    }
   },
   "cell_type": "code",
   "source": [
    "df = pd.read_csv(\"exp_input/SAGE_groundtruth.csv\")\n",
    "input_file = \"exp_input/SAGE_input.txt\""
   ],
   "id": "78d2b2bc4106d2ef",
   "outputs": [],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T20:37:40.268816Z",
     "start_time": "2025-08-27T20:37:40.265794Z"
    }
   },
   "cell_type": "code",
   "source": [
    "model_name = \"gemini-2.0-flash\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"GPT_FDR_FewShot_Syn\"  # \"GPT_from_full_input_Examples\" or \"retrieve_datasets_simple_JSON\"\n",
    "FDR = True\n",
    "semantic_retrieval = False\n",
    "section_filter= None"
   ],
   "id": "cca05a29b7c100d1",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T20:37:40.275480Z",
     "start_time": "2025-08-27T20:37:40.271286Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ],
   "id": "a4ae25cbd54d4fe1",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 100\n"
     ]
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T20:37:40.285512Z",
     "start_time": "2025-08-27T20:37:40.276629Z"
    }
   },
   "cell_type": "code",
   "source": "dg = DataGatherer(llm_name=model_name, log_level='INFO', process_entire_document=FDR) #, driver_path='../../Firefox/geckodriver'",
   "id": "b79cbd7265b497e1",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 266 - INFO - Setting up data fetcher...\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 296 - INFO - Data fetcher setup completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 107 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.0-flash\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T20:46:00.752279Z",
     "start_time": "2025-08-27T20:45:45.837799Z"
    }
   },
   "cell_type": "code",
   "source": [
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335/\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917/\"]\n",
    "\n",
    "#out = dg.process_url(\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\", semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
    "\n",
    "#out\n"
   ],
   "id": "5e0e0500474a6c34",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7029360&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b56a8c10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/ is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 121372\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31095 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cf2e39145e7f4ec8866cf6cff9793434c9a0c4e8d6169f93ff574af22b603c60\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 29 - INFO - Prompt saved to prompts/prompt_evals/gemini-2.0-flash-0-cf2e39145e7f4ec8866cf6cff9793434c9a0c4e8d6169f93ff574af22b603c60.json\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31095 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-45515bbb-43e0-4f5c-9733-c27493b8f1f1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC7029360\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"31561387\\\",\\n    \\\"repository_reference\\\": \\\"https://pubmed.ncbi.nlm.nih.gov/31561387/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8717496\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8717496\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC7029360\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"31561387\\\",\\n    \\\"repository_reference\\\": \\\"https://pubmed.ncbi.nlm.nih.gov/31561387/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8717496\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8717496\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322382,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41545,\n",
      "        \"completion_tokens\": 148,\n",
      "        \"total_tokens\": 41693,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PMC7029360', 'repository_reference': 'n/a', 'dataset_id': 'PMC7029360'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PMC7029360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PMC7029360 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: PMC7029360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'PMC7029360', 'repository_reference': 'n/a', 'dataset_id': 'PMC7029360'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '31561387', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/31561387/', 'dataset_id': '31561387'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID 31561387 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://pubmed.ncbi.nlm.nih.gov/31561387/, identifier: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '31561387', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn8717496', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8717496', 'dataset_id': 'syn8717496'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn8717496, synapse.org, https://www.synapse.org/#!Synapse:syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn8717496', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '31561387', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn8717496', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: pubmed.ncbi.nlm.nih.gov, identifier: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL pubmed.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 31561387 in pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: pubmed.ncbi.nlm.nih.gov and accession_id: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository pubmed.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn8717496 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "  dataset_identifier          data_repository  \\\n",
       "0           31561387  pubmed.ncbi.nlm.nih.gov   \n",
       "1         syn8717496              synapse.org   \n",
       "0                NaN                      NaN   \n",
       "\n",
       "                                   dataset_webpage  \\\n",
       "0                                              n/a   \n",
       "1  https://www.synapse.org/[#!]*Synapse:syn8717496   \n",
       "0                                              NaN   \n",
       "\n",
       "                                 access_mode  \\\n",
       "0                                        NaN   \n",
       "1  Complex download or application to access   \n",
       "0                                        NaN   \n",
       "\n",
       "                                 link               title file_info  \\\n",
       "0                                 NaN                 NaN       NaN   \n",
       "1                                 NaN                 NaN       NaN   \n",
       "0  https://doi.org/10.3233/JPD-191712  10.3233/JPD-191712       n/a   \n",
       "\n",
       "  description source_section                                section_class  \\\n",
       "0         NaN            NaN                                          NaN   \n",
       "1         NaN            NaN                                          NaN   \n",
       "0         n/a            n/a  pmc-layout__citation font-secondary font-xs   \n",
       "\n",
       "  download_link file_extension                         a_attr_href  \\\n",
       "0           NaN           None                                 NaN   \n",
       "1           NaN           None                                 NaN   \n",
       "0          None           None  https://doi.org/10.3233/JPD-191712   \n",
       "\n",
       "                  a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0                          NaN                   NaN           NaN   \n",
       "1                          NaN                   NaN           NaN   \n",
       "0  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       "\n",
       "            a_attr_rel context_description  \\\n",
       "0                  NaN                 NaN   \n",
       "1                  NaN                 NaN   \n",
       "0  noopener noreferrer                       \n",
       "\n",
       "                                          source_url  \\\n",
       "0  https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/   \n",
       "1  https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/   \n",
       "0  https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/   \n",
       "\n",
       "                                           pub_title  \n",
       "0  Quadruple Decision Making for Parkinson’s Dise...  \n",
       "1  Quadruple Decision Making for Parkinson’s Dise...  \n",
       "0  Quadruple Decision Making for Parkinson’s Dise...  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>section_class</th>\n",
       "      <th>download_link</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>context_description</th>\n",
       "      <th>source_url</th>\n",
       "      <th>pub_title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>31561387</td>\n",
       "      <td>pubmed.ncbi.nlm.nih.gov</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/</td>\n",
       "      <td>Quadruple Decision Making for Parkinson’s Dise...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>syn8717496</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn8717496</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/</td>\n",
       "      <td>Quadruple Decision Making for Parkinson’s Dise...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.3233/JPD-191712</td>\n",
       "      <td>10.3233/JPD-191712</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>pmc-layout__citation font-secondary font-xs</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>https://doi.org/10.3233/JPD-191712</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "      <td></td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/</td>\n",
       "      <td>Quadruple Decision Making for Parkinson’s Dise...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:27.508937Z",
     "start_time": "2025-08-27T19:09:31.536631Z"
    }
   },
   "cell_type": "code",
   "source": [
    "input_file = \"exp_input/SAGE_input.txt\"\n",
    "combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ],
   "id": "bfd0572331e32fe2",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 627 - INFO - Loaded 100 URLs from file.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 0th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11792374&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0858400>, <Element supplementary-material at 0x1ac67ef80>, <Element supplementary-material at 0x1b09a4d40>, <Element supplementary-material at 0x1b09a4cc0>, <Element supplementary-material at 0x1b09a4d80>, <Element supplementary-material at 0x1b09a4d00>, <Element supplementary-material at 0x1b09a4f40>, <Element supplementary-material at 0x1b09a4f00>, <Element supplementary-material at 0x1b09a4f80>, <Element supplementary-material at 0x1b09a4fc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0858400>, <Element supplementary-material at 0x1ac67ef80>, <Element supplementary-material at 0x1b09a4d40>, <Element supplementary-material at 0x1b09a4cc0>, <Element supplementary-material at 0x1b09a4d80>, <Element supplementary-material at 0x1b09a4d00>, <Element supplementary-material at 0x1b09a4f40>, <Element supplementary-material at 0x1b09a4f00>, <Element supplementary-material at 0x1b09a4f80>, <Element supplementary-material at 0x1b09a4fc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: ABCA1 expression also correlated with SASP scores in the same human brain samples (Supplementary Fig. 1 [this file] A). Sn-RNA sequencing analysis also indicated that mTOR activation scores were positively associated with ABCA1 expression in astrocytes, excitatory and inhibitory neurons, microglia, oligodendrocytes, and oligodendrocyte progenitor cells (Supplementary Fig. 1 [this file] B). AD characteristics, such as Braak stage, NFT burden, and neuritic plaque burden, were also positively correlated with mTOR activation in excitatory neurons (Supplementary Fig. 1 [this file] B), where APOE4 was associated with increased mTOR expression in excitatory neurons, oligodendrocytes, and pericytes (Supplementary Fig. 1 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Using SenTraGor (STG, a reagent for detection of lipofuscin) [ 50 [rid=CR50] ] staining in postmortem human brain slices (mid-frontal lobe region) from ROS, we observed higher STG intensity in GFAP-positive astrocytes in AD participants compared to NCI participants with the APOE3/4 genotype (Fig. 2 [rid=Fig2] A, the STG/GFAP staining test is shown in Supplementary Fig. 2 [this file] A), corroborating the increased astrocyte senescence markers in AD observed in the transcriptomic data. \n",
      "Given that ABCA1 is predominantly localized in cell membranes [ 51 [rid=CR51] ], cytosolic proteins were removed (Fig. 2 [rid=Fig2] B, Supplementary Fig. 2 [this file] B). Total membrane ABCA1 levels were higher in APOE3/3 and APOE3/4 carriers with AD dementia than in APOE3/3 and APOE3/4 carriers in the NCI group (Fig. 2 [rid=Fig2] B; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "In human postmortem middle frontal lobe tissues, caveolin-1 protein levels were higher in both APOE3/3 and APOE3/4 carriers with AD dementia than in the NCI group, although there was no difference between the APOE3/3 and APOE3/4 genotypes in either group (Fig. 3 [rid=Fig3] M; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: ABCA1 was more abundant in lysosome-enriched fractions in the AD group than in the NCI group within the APOE3/4 genotype but not in the APOE3/3 genotype (Fig. 2 [rid=Fig2] C, full blots are shown in Supplementary Fig. 3 [this file] ), supporting that more ABCA1 was trapped in lysosomes in APOE4 AD. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: To explore the underlying mechanism, we treated ABCA1-GFP overexpressing HeLa cells with recombinant ApoE3 (rE3) or ApoE4 (rE4). rE4 treatment resulted in lower total ABCA1 protein levels than rE3 treatment, indicating greater ABCA1 degradation (Supplementary Fig. 4 [this file] A). ABCA1-binding complexes were enriched for proteomic analysis (Fig. 3 [rid=Fig3] A and Supplementary Fig. 4 [this file] B). Proteomic analysis revealed 768 shared proteins between rE3 and rE4 treatments, with 314 proteins unique to rE4 (Supplementary Fig. 4 [this file] C). Nine of these proteins (Supplementary Table 4) showed higher expression in the rE4 treatment group than in the rE3 group (Supplementary Fig. 4 [this file] D). Reducing caveolin-1, but not, AP2B1 expression by siRNA increased plasma membrane ABCA1 levels in mouse primary astrocytes (Fig. 3 [rid=Fig3] F, Supplementary Fig. 4 [this file] E), supporting that caveolae-mediated endocytosis regulated ABCA1 degradation. Surprisingly, reducing caveolin-1 expression significantly decreased ABCA1-mediated cholesterol efflux (Fig. 3 [rid=Fig3] G) despite an increase in total ABCA1 protein levels (Supplementary Fig. 4 [this file] F). Furthermore, overexpression of caveolin-1 in astrocytes enhanced cholesterol efflux (Supplementary Fig. 4 [this file] G). \n",
      "Cholesterol loading, by adding LDL, led to increased caveolin-1 expression (Fig. 3 [rid=Fig3] H and I) and showed a dose-dependent increase in baby hamster kidney (BHK) cells (Supplementary Fig. 4 [this file] H). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Mouse cortices were fractionated using TBSX and guanidine HCl to isolate soluble membrane and insoluble aggregated proteins (Supplementary Fig. 5 [this file] A). \n",
      "To identify the main cell type in which ABCA1 and caveolin-1 interact, we isolated microglia, astrocytes and axon-enriched fraction from the brains of adult mice (Supplementary Fig. 5 [this file] B). Astrocytes were enriched in the CD11b fraction (Supplementary Fig. 5 [this file] C). ABCA1 was mainly expressed in astrocytes and axon-enriched fractions, and lower ABCA1 expression was detected in microglial cells (Supplementary Fig. 5 [this file] D). Caveolin-1 was mainly expressed in astrocytes and microglia but not in axon-enriched fractions (Supplementary Fig. 5 [this file] E). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: In agreement, ABCA1 brain knockout mice showed higher p21 intensity in the brain than ABCA1 WT mice (Fig. 4 [rid=Fig4] G, p21 staining test is shown in Supplementary Fig. 6 [this file] A). To indirectly assess lysosomal ABCA1 localization in APOE4-TR brains, tissue slides were double stained with LAMP1 and ABCA1 (Supplementary Fig. 6 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Body weight was not affected by HPCD treatment (Supplementary Fig. 7 [this file] A). Total cholesterol levels in the brain (Fig. 5 [rid=Fig5] B) or plasma (Supplementary Fig. 7 [this file] B) were also no different between HPCD and PBS groups. Lysosomal ABCA1 levels were lower in the HPCD treatment group, as verified by lysosome isolation and immunostaining (Fig. 5 [rid=Fig5] G and H; validation of lysosome isolation is shown in Supplementary Fig. 7 [this file] C). Treatment with HPCD was associated with more efficient cholesterol efflux, as evidenced by increased trends of small HDL particle formation in cerebrospinal fluid (CSF) and plasma, although these changes did not reach statistical significance due to the small sample size (Supplementary Fig. 7 [this file] D-F). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: HPCD also diminished the activated astrocytes and microglia (Fig. 6 [rid=Fig6] E), and several inflammatory cytokines (Supplementary Fig. 8 [this file] A), but increased synaptophysin, PSD95, and vGLuT1 levels (Supplementary Fig. 8 [this file] B and 8 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: To validate these findings in human cells, we cultured human induced pluripotent stem cell (hiPSC)-derived astrocytes with APOE4/4 genotype (Supplementary Fig. 9 [this file] A). When these human iPSc-derived astrocytes were treated with methyl-β-cyclodextrin (MβCD) after LDL loading, cellular cholesterol accumulation (Supplementary Fig. 9 [this file] B) and IL-1β and CCL2 mRNA levels were decreased (Supplementary Fig. 9 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM10: Clinical and pathological characteristics of the samples are detailed in Supplementarys Table ( 1 [this file] ) and Table ( 2 [this file] ). \n",
      "Detailed information regarding all samples is provided in Supplementary Table 1 [this file] . \n",
      "Detailed information about the samples is provided in Supplementary Table 2 [this file] . \n",
      "Detailed information on these samples is presented in Supplementary Table 3 [this file] . \n",
      "The antibodies used are listed in Supplementary Table 5 [this file] . \n",
      "The primers were listed in Supplementary Table 6 [this file] . \n",
      "Cellular senescence pathway enrichment scores for each sample were calculated using single-sample Gene Set Enrichment Analysis (ssGSEA) (the cellular senescence genes are listed in Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 311300\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 78577 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cc110dec616e46d5f29216109f6e010433781281a12901cbb67fe9d9ecf71c15\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 78577 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bb09d479-f66d-472b-8d03-b2deb1d7374e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9702085\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9702085\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8456637\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9702085\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27188\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293417\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27188\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27187\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9702085\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9702085\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8456637\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9702085\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27188\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293417\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27188\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"M27187\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321774,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 91959,\n",
      "        \"completion_tokens\": 319,\n",
      "        \"total_tokens\": 92278,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn9702085', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9702085', 'dataset_id': 'syn9702085'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn9702085, synapse.org, https://www.synapse.org/#!Synapse:syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn8456637', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9702085', 'dataset_id': 'syn8456637'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn8456637, synapse.org, https://www.synapse.org/#!Synapse:syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn8456637', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'M27188', 'repository_reference': 'nuccore', 'dataset_id': 'M27188'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID M27188 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'M27188', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn52293417', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52293417', 'dataset_id': 'syn52293417'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn52293417, synapse.org, https://www.synapse.org/#!Synapse:syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn52293417', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52293417', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2580853, synapse.org, https://www.synapse.org/#!Synapse:syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'M27187', 'repository_reference': 'nuccore', 'dataset_id': 'M27187'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID M27187 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'M27187', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn8456637', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'M27188', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn52293417', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'M27187', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 6 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn9702085 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn8456637 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier M27188 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for M27188 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: nuccore and accession_id: M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/M27188\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52293417 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52293417 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52293417 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52293417 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn52293417 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52293417\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier M27187 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for M27187 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: nuccore and accession_id: M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/M27187\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 6\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 1/100 (1.00%) | Elapsed: 00:00:02 | ETA: 00:04:47\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 1th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4318527&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1abba02d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 23 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 138300\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35327 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0f6c423d0baadb32493bbdbe1bd5d76895511b0ed89ca930a80cb38206ca84a0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35327 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3968fc88-eafb-444d-b9a0-42828de6662e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn312572\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn312572\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2455683\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2455683\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1720047\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1720047\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn312572\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn312572\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2455683\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2455683\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1720047\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1720047\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321790,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 45929,\n",
      "        \"completion_tokens\": 165,\n",
      "        \"total_tokens\": 46094,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn312572', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572', 'dataset_id': 'syn312572'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID syn312572 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn312572, identifier: syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn312572 for https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn312572 for https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn312572 for https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn312572 for https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn312572 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn312572', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2455683', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2455683', 'dataset_id': 'syn2455683'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2455683, synapse.org, https://www.synapse.org/#!Synapse:syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2455683', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1720047', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047', 'dataset_id': 'syn1720047'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1720047, synapse.org, https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1720047', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn312572', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2455683', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1720047', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn312572 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn312572 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn312572 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn312572 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn312572 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier syn312572 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2455683 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2455683 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2455683 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2455683 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2455683 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2455683\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1720047 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 2th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4339277&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0b30690>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 18 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 4802) for SD1: Moreover, samples in the paired-end group had many more mappable reads than those in the single-end group ( Table 1 [href=#T1] , Supplementary Fig. 1 [href=this file] ). Indeed, the two groups showed distinct global patterns of pseudogene expression ( Supplementary Fig. 2 [href=this file] ). For each cancer type, we observed generally weak correlations between the expression level of pseudogenes and their WT genes, which is consistent with the previous study ( Supplementary Fig. 3 [href=this file] ). As observed in Kalyana-Sundaram et al. (2012) , we detected some tumor-lineage-specific pseudogenes (296 from the paired-end group and 41 from the single-end group, Supplementary Fig. 4 [href=this file] ). In addition, for three cancer types with available RNA-seq data from nontumor tissue samples, we identified differentially expressed pseudogenes between tumor and nontumor samples (54 in BRCA, 110 in KIRC and 138 in LUSC, Supplementary Fig. 5 [href=this file] ).\n",
      "For several cancer types with established tumor subtypes, we performed the supervised analysis and revealed substantial numbers of pseudogenes with significant differential expression: 48 in UCEC (endometrioid vs. serous) , 138 in LUSC (basal, classical, primitive and secretory) , 71 in GBM (classical, mesenchymal, neural and proneural) and 547 in BRCA (PAM50 subtypes: luminal A, luminal B, basal-like, Her2-enriched and normal-like) (Methods, Fig. 2a [href=#F2] , Supplementary Data 1 [href=this file] ).\n",
      "The predictive power of pseudogene expression is comparable to those achieved by the mRNA expression profiles, suggesting that both pseudogene and mRNA expression can classify the UCEC subtypes independently ( Supplementary Fig. 6 [href=this file] ).\n",
      "Based on the NMF consensus clustering, 837 BRCA samples can be classified into three distinct subtypes: subtype 1 ( = 144), subtype 2 ( = 390), and subtype 3 ( = 303) ( Fig. 4b [href=#F4] , Supplementary Data 2 [href=this file] ).\n",
      "Based on the 500 pseudogenes with the most variable expression, we were able to classify 446 KIRC samples into two distinct subtypes ( Fig. 5a [href=#F5] , Supplementary Data 3 [href=this file] ). We observed an enrichment of pseudogenes (115 out of 500) with a statistically significant P-value ( < 0.05) ( Fig. 5c [href=#F5] , Supplementary Data 4 [href=this file] ).\n",
      "To further assess the clinical utility of the observed pseudogene-expression subtypes, we classified the KIRC samples into four risk quartiles based on the risk scores (in terms of overall survival) calculated from the multivariate Cox model, employing only clinical variables: low risk group (Q1, = 110), low-medium risk group (Q2, = 111), medium-high risk group (Q3, = 112), and high risk group (Q4, = 112) (Methods, Supplementary Data 3 [href=this file] ).\n",
      "To gain some mechanistic insight into how expressed pseudogenes contribute to the observed KIRC pseudogene-expression subtypes, we performed a systematic analysis ( Supplementary Fig. 9a and Supplementary Data 5 [href=this file] ). Among the WT cognate genes with strong positive correlations with their pseudogenes, we noticed that the survival correlations of individual WT genes with prognostic value match the survival pattern of the pseudogene-expression subtypes: WT genes with better prognosis (potentially tumor suppressors, hazard ratio < 1) show higher expression levels in subtype 1 (the better survival group) and the genes with worse prognosis (potentially oncogenes, hazard ratio > 1) show higher expression levels in subtype 2 (the worse survival group, Supplementary Fig. 8b [href=this file] ). Finally, we examined the classic miRNA decoy model as proposed in Poliseno et al. (2010) and identified 38 such candidates ( Methods and Supplementary Data 5 [href=this file] ). One candidate of interest is the potential regulation of a putative tumor suppressor α-catenin (CTNNA1) by the pseudogene PGOHUM00000257111 through competition for up to 9 shared miRNA regulators ( Supplementary Data 5 [href=this file] ). Indeed, the expression levels of PGOHUM00000257111 were significantly higher in cluster 1 (t-test P = 1.48×10-7), which may lead to the elevated levels of CTNNA1 in subtype 1 ( Supplementary Fig. 8b [href=this file] ) and therefore better survival.\n",
      "Finally, we performed feature selection ( Supplementary Data 6 [href=this file] ) and built the classifier from the whole training set using the best-performing algorithm (with the highest AUC) identified through the cross-validation, and applied it to the test set in order to independently validate the predictive power.\n",
      "We gratefully acknowledge contributions from the TCGA Research Network and its TCGA Pan-Cancer Analysis Working Group (contributing consortium members are listed in Supplementary Note 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 122627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31409 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-166e5bd9ed9e36e2ef0319219b850adc7cd78ba95d7a3969f6f157849740a226\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31409 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6a57d663-5c52-488c-b328-0b44989de05e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1732077\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1732077\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321805,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41120,\n",
      "        \"completion_tokens\": 48,\n",
      "        \"total_tokens\": 41168,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1732077', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn1732077'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1732077, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1732077', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1732077', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1732077 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 3th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8452114&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0ee2250>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 8 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 343) for SD1: We obtained a total of six coexpression modules ( Figure S1 [href=this file] ) in amygdala with the number of genes in each module ranging from 204 (red module) to 3,049 (turquoise). After eigengene extraction and differential analysis, no modules were differentially expressed between COVID-19+ and CTL in AMY ( Figure S1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 224) for SD2: Relationships between these modules are reported in Figure S2 [href=this file] . Interestingly, the two synaptic modules were not related to each other ( Figure S2 [href=this file] ) despite a similar functional enrichment.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 75) for SD3: The top genes for AMY and OBT are reported in Figure S3 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 10 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1320) for SD4: The top 9 genes for the two regions are reported in Figure 2A [href=#F2] and 2B [href=#F2] , whereas the complete list of DEGs is reported in Table S1 [href=this file] and S2 [href=this file] . Across downregulated genes we observed an enrichment of synaptic and neuronal pathways ( Figure 3 [href=#F3] ; Table S3 [href=this file] ). Finally, synaptic pathways were enriched within downregulated genes ( Figure 3 [href=#F3] ; Table S4 [href=this file] ).\n",
      "Other modules were enriched for development/angiogenesis (green, upregulated, and enriched for End and Per), immune system (tan, upregulated), and synaptic signaling (green-yellow and turquoise, both downregulated and enriched for neuronal cell genes) ( Table S5 [href=this file] ).\n",
      "The only differential pathways we detected were related to platelet activation processes/hemostasis, which was only detected in the olfactory bulb ( Figure 7 [href=#F7] ; Table S6 [href=this file] and S7 [href=this file] ).\n",
      "We detected a total of 4,275 genes only associated with pseudotime in amygdala and 3,102 only associated with pseudotime in olfactory bulb ( Table S8 [href=this file] and Table S9 [href=this file] , respectively).\n",
      "It is important to stress that the two modules are not related (see Table S2 [href=this file] ) representing two distinct biological processes.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 262949\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 66489 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-06999945398567aaae4f655f2676f49b61814f9b787779d7839d61c146c6c06a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 66489 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6d903c59-a816-4b28-acfc-d2e932abd086\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321819,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 97189,\n",
      "        \"completion_tokens\": 49,\n",
      "        \"total_tokens\": 97238,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18485175', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn18485175'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn18485175, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn18485175 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 4th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9641692&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b09c2540>, <Element supplementary-material at 0x1b12ea300>, <Element supplementary-material at 0x1b12e9180>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b09c2540>, <Element supplementary-material at 0x1b12ea300>, <Element supplementary-material at 0x1b12e9180>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: When evaluated on cell-type level, TAPE has the best performance of MAE, and comparable CCC with relatively small variance on real datasets (Supplementary Table 8 [this file] ). \n",
      "Meanwhile, we find that deep learning methods are robust for the cross-protocol prediction, while the performance of some statistical methods will drop if the reference and pseudo-bulk data type (UMI to counts or counts to UMI) are exchanged (Supplementary Figs. 1 [this file] and 2 [this file] ). \n",
      "Detailed comparison results are available in Supplementary Table 8 [this file] . For the “similar” scenario, we investigated TAPE’s performance on distinguishing similar cell types (CD4 T cell and CD8 T cell) in all the three PBMC datasets (Supplementary Fig. 3 [this file] ). With 13 defined cell subtypes (Supplementary Table 1 [this file] ), all the methods can not achieve satisfying results (Supplementary Fig. 4 [this file] , median CCC < 0.1), which clearly shows the common limitation of current methods. \n",
      "In Supplementary Fig. 6 [this file] , the two colors stand for the different methods, and the two coordinates of each dot represent overall MAE and CCC for all cell types respectively. \n",
      "From the results (Supplementary Fig. 10 [this file] ), we can have four conclusions: (1) TAPE’s predictive power is better than CIBERSORTx when the randomly selected DEGs are less than 1000; (2) both methods can achieve good performance when the DEGs are signature genes and there are not any similar cell types; (3) both methods can not distinguish DEGs from CD8 T cell rather than CD4 T cell if the DEGs are randomly selected; (4) CIBERSORTx is better than TAPE if the DEGs are signature genes and there exist similar cell types. \n",
      "The results show that TAPE only predicts and as DEGs in NK cell and no DEGs for other cell types (Supplementary Fig. 8 [this file] ). \n",
      "Firstly, when we study the correlation at the gene level using “overall” mode (Supplementary Fig. 7 [this file] ), about 30% of the predicted genes have the negative correlation. However, this phenomenon can also be observed in CIBERSORTx (Supplementary Fig. 8 [this file] ), which indicates the information loss between bulk samples and inferred cell-type-specific GEP is hard to be reconstructed. Thirdly, we notice that both CIBERSORTx and our method can not distinguish DEGs from similar cell subtypes correctly if the DEGs are not signature genes (Supplementary Fig. 9 [this file] ) which means that their resolution is still limited. But CIBERSORTx has displayed its advantages in distinguishing signature DEGs from similar cell types because of the incorporation of the signature matrix (Supplementary Fig. 10 [this file] ). \n",
      "When we used it to test whether deconvolution methods can achieve good performance with immune cell subtypes, we merged all 30 cell types into 13 cell subtypes (Supplementary Table 1 [this file] ). \n",
      "This step is very important, because TAPE will fail in predicting test bulk data proportions properly without proper filtering (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ) . If the less variable genes are not filtered out, TAPE can not predict a good result because of the noises (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ). \n",
      "Hyperparameter tuning file is available in Supplementary Tables 3 [this file] – 7 [this file] . \n",
      "All other relevant data supporting the key findings of this study are available within the article and its Supplementary Information [this file] files or from the corresponding author upon reasonable request. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Peer reviewer reports [this file] are available. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 198358\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 50342 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a6bb0c838d77070c809c70e4974c562e30f3340fe80200a4ffc1306c81a336ff\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 50342 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-80780826-9191-4904-85e4-2753c4921168\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/poseidonchan/TAPE\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/poseidonchan/TAPE\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://sctape.readthedocs.io/datasets/\\\",\\n    \\\"repository_reference\\\": \\\"https://sctape.readthedocs.io/datasets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/6-k-pbm-cs-from-a-healthy-donor-1-standard-1-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/10-k-pbm-cs-from-a-healthy-donor-v-3-chemistry-3-standard-3-0-0\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/poseidonchan/TAPE\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/poseidonchan/TAPE\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://sctape.readthedocs.io/datasets/\\\",\\n    \\\"repository_reference\\\": \\\"https://sctape.readthedocs.io/datasets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/6-k-pbm-cs-from-a-healthy-donor-1-standard-1-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/10-k-pbm-cs-from-a-healthy-donor-v-3-chemistry-3-standard-3-0-0\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321825,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58326,\n",
      "        \"completion_tokens\": 770,\n",
      "        \"total_tokens\": 59096,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE107011', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'dataset_id': 'GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE107011, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi', 'dataset_id': 'GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE65133, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE87544', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'dataset_id': 'GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE87544, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE67835', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'dataset_id': 'GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE67835, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'dataset_id': 'GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE84133, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE115449', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449', 'dataset_id': 'GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE115449, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE119117', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'dataset_id': 'GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE119117, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3219045', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3219045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3219045, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/poseidonchan/TAPE', 'repository_reference': 'https://github.com/poseidonchan/TAPE', 'dataset_id': 'https://github.com/poseidonchan/TAPE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/poseidonchan/TAPE, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/poseidonchan/TAPE is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/poseidonchan/TAPE, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://sctape.readthedocs.io/datasets/', 'repository_reference': 'https://sctape.readthedocs.io/datasets/', 'dataset_id': 'https://sctape.readthedocs.io/datasets/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://sctape.readthedocs.io/datasets/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: sctape.readthedocs.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://sctape.readthedocs.io/datasets/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://sctape.readthedocs.io/datasets/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'sctape.readthedocs.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets/8-k-pbm-cs-from-a-healthy-donor-2-standard-2-1-0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sctape.readthedocs.io', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 11 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE107011 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE65133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE87544 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE67835 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE84133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE115449 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE119117 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3219045 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 11: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 11\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 5th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5161470&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6591990>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 19 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s001'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 95) for pone.0167994.s001: In both results the tagging SNPs are spread across the chromosome ( S1 Fig [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s002'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 207) for pone.0167994.s002: To compare the minimum numbers of the tag SNPs that were identified by the two methods to reach >95% coverage, we tested the difference of the tag SNP numbers using the t-test ( S1 Table [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 87) for pone.0167994.s003: The comparison of the discovered tag SNP sets are given in S2 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#pone.0167994.s004'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 284) for pone.0167994.s004: An example output corresponding to ENr113 genotypes results is given in S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#pone.0167994.s005'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for pone.0167994.s005: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#pone.0167994.s006'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 284) for pone.0167994.s006: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s007'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 178) for pone.0167994.s007: The association of these three SNPs and the phenotype are significant ( S3 Table [href=this file] ), which suggests that the metaSNP method correctly tagged the designated SNPs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s008'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 205) for pone.0167994.s008: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=this file] ) as well.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#pone.0167994.s009'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 49) for pone.0167994.s009: Details are given in S5 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s010'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 92) for pone.0167994.s010: The details of the mutual information estimation is given in the S1 File [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 158274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 40321 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-85610190b9166c893a0362a5e6c766e8cc4c6c55f8c71e541bf6e375d84702b6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 40321 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f3626bfe-7ba9-43b0-bb55-9bc4916b9b47\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1446295\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HapMap\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"1000 Genomes Project\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENm013\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENm014\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENr112\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENr113\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IMPUTE2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"1KGP\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1446295\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HapMap\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"1000 Genomes Project\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENm013\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENm014\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENr112\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"ENr113\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IMPUTE2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"1KGP\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321844,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 55525,\n",
      "        \"completion_tokens\": 296,\n",
      "        \"total_tokens\": 55821,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1446295', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1446295'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1446295, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1446295\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1446295', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'HapMap', 'repository_reference': 'n/a', 'dataset_id': 'HapMap'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID HapMap is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HapMap for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HapMap for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HapMap for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HapMap for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier HapMap for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'HapMap', 'repository_reference': 'n/a', 'dataset_id': 'HapMap'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '1000 Genomes Project', 'repository_reference': 'n/a', 'dataset_id': '1000 Genomes Project'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID 1000 Genomes Project is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': '1000 Genomes Project', 'repository_reference': 'n/a', 'dataset_id': '1000 Genomes Project'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENm013', 'repository_reference': 'n/a', 'dataset_id': 'ENm013'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENm013\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENm013 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: ENm013\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENm013 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENm013 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENm013 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENm013 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENm013 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'ENm013', 'repository_reference': 'n/a', 'dataset_id': 'ENm013'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENm014', 'repository_reference': 'n/a', 'dataset_id': 'ENm014'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENm014\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENm014 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: ENm014\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENm014 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENm014 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENm014 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENm014 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENm014 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'ENm014', 'repository_reference': 'n/a', 'dataset_id': 'ENm014'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENr112', 'repository_reference': 'n/a', 'dataset_id': 'ENr112'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENr112\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENr112 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: ENr112\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENr112 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENr112 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENr112 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENr112 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENr112 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'ENr112', 'repository_reference': 'n/a', 'dataset_id': 'ENr112'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENr113', 'repository_reference': 'n/a', 'dataset_id': 'ENr113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENr113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENr113 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: ENr113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENr113 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENr113 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENr113 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENr113 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENr113 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'ENr113', 'repository_reference': 'n/a', 'dataset_id': 'ENr113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'IMPUTE2', 'repository_reference': 'n/a', 'dataset_id': 'IMPUTE2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: IMPUTE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID IMPUTE2 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: IMPUTE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IMPUTE2 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier IMPUTE2 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier IMPUTE2 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IMPUTE2 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IMPUTE2 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'IMPUTE2', 'repository_reference': 'n/a', 'dataset_id': 'IMPUTE2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '1KGP', 'repository_reference': 'n/a', 'dataset_id': '1KGP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID 1KGP is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 1KGP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 1KGP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 1KGP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 1KGP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 1KGP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': '1KGP', 'repository_reference': 'n/a', 'dataset_id': '1KGP'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1446295', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1446295\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1446295\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1446295 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1446295 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1446295 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1446295 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1446295 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1446295\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1446295\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 12\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 6th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9925742&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6934dc0>, <Element supplementary-material at 0x1b6b95980>, <Element supplementary-material at 0x1b6b96c80>, <Element supplementary-material at 0x1b6b97600>, <Element supplementary-material at 0x1b6b97d40>, <Element supplementary-material at 0x1b6b96740>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6934dc0>, <Element supplementary-material at 0x1b6b95980>, <Element supplementary-material at 0x1b6b96c80>, <Element supplementary-material at 0x1b6b97600>, <Element supplementary-material at 0x1b6b97d40>, <Element supplementary-material at 0x1b6b96740>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: After clustering (Supplementary Fig. 1a–e [this file] ), a total of 37,943 cells were annotated into 23 clusters using a predefined set of marker genes (Supplementary Fig. 1c [this file] ). \n",
      "To reveal heterogeneity within oligodendrocytes, we grouped the aforementioned oligodendrocytes into six classes described in a published database , as follows (Fig. 1a [rid=Fig1] and Supplementary Fig. 2a [this file] –d and 3a [this file] ): (1) Differentiation-committed oligodendrocyte precursors (COPs) and newly formed oligodendrocytes (NFOL) that express Sox6, Pdgfra, and Cspg4. (2) Myelin-forming oligodendrocytes (MFOL) that express Ctps, Ablims, and Thbs3. (3) Mature oligodendrocytes 1, 2 (MOL 1, 2) that express Il33, Ptgds, and Opalin. (4) Mature oligodendrocytes 3, 4 (MOL 3, 4) that express Klk6, S100b, Anxa5, Cd59a and Mgst3. Moreover, we discovered an additional oligodendrocyte cluster, which gradually increased in the 3-, 6-month-old App KI mice (Fig. 1b–d [rid=Fig1] and Supplementary Fig. 2e [this file] ). This cluster was present in the UMAP plot even with down-sampled data (400, 1,200, and 2,400 cells) (Supplementary Fig. 2f [this file] ). Remarkably, we found that this cluster shared some signatures with MOL2, while Alzheimer’s disease risk factors such as C4b, Apoe and Cd74 are strongly expressed (Supplementary Fig. 2c [this file] , 3a [this file] , 3b [this file] , and Source data file). \n",
      "We further confirmed the lineage transitions of DAOs and homeostatic MOLs from COP and NFOL in control and AD brains (Fig. 1e–g [rid=Fig1] and Supplementary Fig. 4 [this file] ), but density from the DAO cluster exclusively appeared in 3 and 6-month-old AD brains (Fig. 1 [rid=Fig1] g). \n",
      "We reanalyzed previously published single-cell transcriptome datasets from postmortem AD brains (Supplementary Fig. 5a–c [this file] ) . \n",
      "Notably, consistent with scRNA-seq results, we observed a significant increase in Mbp + , Cd74+ oligodendrocytes in 3- and 6-month-old App KI and 6-month-old 5xFAD mice, whereas the number of NeuroD1+ post-mitotic neurons significantly decreased (Fig. 2 [rid=Fig2] a, b [rid=Fig2] , Supplementary Fig. 6a–c [this file] , and Supplementary Fig. 7a [this file] , b [this file] ). We further confirmed an increase in Cd74+ and Oligo2+ AD-associated oligodendrocytes in App KI mice using flow cytometry (Fig. 2 [rid=Fig2] c, d [rid=Fig2] , and Supplementary Fig. 6d [this file] ). \n",
      "Transmission electron microscopy (TEM) analysis revealed that 6-month-old App KI mice exhibited a thinner myelin layer relative to the axon diameter (Fig. 2e [rid=Fig2] and supplementary fig. 8 [this file] ). \n",
      "Consistently, we found a significant increase in Mbp+ and Cd74+ DAOs in the hippocampus of premature progeria (6-month-old) and 13-month-old mice (Supplementary Fig. 9a–d [this file] ), whereas the number of NeuroD1-positive post-mitotic neurons decreased in the hippocampus of 13-month-old mice (Supplementary Fig. 9e and f [this file] ). We further confirmed that the genes enriched in DAOs were highly correlated with gene signatures shown in the aged human brain (Supplementary Fig. 10a, b [this file] ), indicating that Cd74-positive DAOs increased in the aged brains and had transcriptional profiles more similar to the aged human brain. \n",
      "In particular, we found significant alternations in the transcriptional profiles between DAOs and MOL 3/4 clusters and GO-term of up-regulated genes in DAO versus MOL3/4 showed the interleukin & chemokine production and ERK signaling (Fig. 3 [rid=Fig3] a and c [rid=Fig3] ), which was also confirmed during the disease progression in AD mice brain (Supplementary Fig. 11a, b [this file] ). \n",
      "We first identified microglia with a high DAM module score based on previously published DAM signatures (Supplementary Fig. 1e [this file] , 12a, b [this file] ), that gradually increased during AD progression (Fig. 3 [rid=Fig3] e, f [rid=Fig3] , and Supplementary Fig. 12c [this file] ). \n",
      "Supplementary Fig. 13a–c [this file] shows efficient inhibition of phosphor 44/42 MAPK signaling in primary oligodendrocytes upon treatment with 200 ng/ml SCH772984, a specific Erk1 and Erk2 inhibitor. Remarkably, oligodendrocytes treated with SCH772984 exhibited a significant increase in Mbp+ branching compared with the controls, whereas the number of Cd74+ cells significantly decreased (Supplementary Fig. 13d [this file] , 13e [this file] , and Fig. 4a–c [rid=Fig4] ). \n",
      "First, in order to examine the potential interactions between DAOs and other neuronal cell types, AD DAOs were co-cultured with AD neurons, astrocytes, and microglia treated with SCH772984 (Supplementary Fig. 14a [this file] ). We found that AD DAOs with other neuronal cell types did not increase the number of Mbp+ branches in the presence and absence of SCH772984 treatment (Supplementary Fig. 14b–f [this file] ), suggesting that inhibition of ERK/MAPK signaling in other neuronal cell types did not contribute to the phenotypic changes in DAOs (Fig. 4d–f [rid=Fig4] ). Moreover, to investigate whether other neural cell types can contribute to the AD-associated phenotypes via ERK signaling, each of the neurons, microglia, astrocytes and oligodendrocytes isolated from AD mice were subjected to SCH772984 treatment, and AD pathology was determined after co-culture with the remaining untreated AD neuronal cell types (Supplementary Fig. 15a [this file] ). Remarkably, we found that ERK1/2-inhibited oligodendrocytes significantly diminished the Aβ42/40 ratio in the AD primary neuronal cultures (Supplementary Fig. 15b [this file] ). Also, we observed a decrease in the amyloidogenic processing marker C99 in the AD neuronal cultures with SCH772984 treatment (Supplementary Fig. 15c–e [this file] ), indicating that AD oligodendrocytes predominantly contribute to the pathophysiology of AD via ERK signaling. \n",
      "Consistent with previous results, systemic administration of SCH772984 significantly reduced the occurrence of DAOs in the hippocampus of App KI mice (Fig. 5b, c [rid=Fig5] , and Supplementary Fig. 16a–d [this file] ) but not Cd74 + Iba1+ microglia (Supplementary Fig. 16c, d [this file] ). \n",
      "Moreover, we confirmed a decrease in the number of Aβ plaques (Fig. 5m, n [rid=Fig5] ), coupled with an increase in the number of NeuroD1+ cells in the SCH772984-treated AD mouse hippocampus (Supplementary Fig. 16e, f [this file] ). \n",
      "Figure 5a, b, c [this file] obtained from the AD Knowledge Portal ( https://adknowledgeportal.org (https://adknowledgeportal.org) ) under study 557 snRNAseqAD_TREM2(syn21126450). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: To identify transcriptional patterns of this cluster, we conducted differential expression gene analysis by using the FindAllMarkers function in the Seurat package (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: We next explored the unique molecular signatures of DAOs during AD development (Supplementary Data 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Hippocampus tissues from 1-, 3-, and 6-month-old wild-type and App-KI mice were then dissected (Supplementary Data 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 140680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35922 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ab2eddcbb5045521ceb83c4eb0ee5755172566e371a2421a5a3fd903a3020517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35922 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3ff84f75-fae1-477e-9866-792d8a0d1ca5\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE224398\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079269\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079266\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079265\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079270\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079268\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079267\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21126450\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE224398\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079269\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079266\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079265\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079270\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079268\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079267\\\",\\n    \\\"repository_reference\\\": \\\"SRA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21126450\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321847,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 42683,\n",
      "        \"completion_tokens\": 320,\n",
      "        \"total_tokens\": 43003,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE224398', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398', 'dataset_id': 'GSE224398'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE224398, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079269', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079269'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079269 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079266', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079266'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079266 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079265', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079265'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079265 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079270', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079270'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079270 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079268', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079268'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079268 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079267', 'repository_reference': 'SRA', 'dataset_id': 'SRR21079267'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRR21079267 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21126450', 'repository_reference': 'https://adknowledgeportal.org', 'dataset_id': 'syn21126450'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21126450, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE224398 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079269 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: sra and accession_id: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079266 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: sra and accession_id: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079265 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: sra and accession_id: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079270 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: sra and accession_id: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079268 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: sra and accession_id: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRR21079267 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: sra and accession_id: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21126450 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 7th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9235070&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1777a50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 13 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 215213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 54555 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e513543272f8284bd1a4c4c783ec0e974626bb97ba91fcf770e1f5f7f7fc25f9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 54555 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9b323f43-274c-4dcb-8027-7c54373ce3b3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE81861\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75688\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE81383\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81383\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103322\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103322\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE76312\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE70630\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70630\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi.org/10.1242/bio.059256\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn30243143\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn213309\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE81861\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75688\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE81383\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81383\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103322\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103322\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE76312\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE70630\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70630\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi.org/10.1242/bio.059256\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn30243143\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn213309\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321865,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 72149,\n",
      "        \"completion_tokens\": 489,\n",
      "        \"total_tokens\": 72638,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE81861', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861', 'dataset_id': 'GSE81861'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE81861, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE81861', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE75688', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688', 'dataset_id': 'GSE75688'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE75688, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75688', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE81383', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81383', 'dataset_id': 'GSE81383'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE81383, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE81383', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE103322', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103322', 'dataset_id': 'GSE103322'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE103322, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE103322', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE76312', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312', 'dataset_id': 'GSE76312'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE76312, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE76312', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE70630', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70630', 'dataset_id': 'GSE70630'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE70630, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE70630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'doi.org/10.1242/bio.059256', 'repository_reference': 'doi.org', 'dataset_id': 'doi.org/10.1242/bio.059256'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: doi.org/10.1242/bio.059256, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: doi.org/10.1242/bio.059256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'doi.org/10.1242/bio.059256', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn30243143', 'repository_reference': 'synapse.org', 'dataset_id': 'syn30243143'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn30243143, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn30243143', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn213309', 'repository_reference': 'synapse.org', 'dataset_id': 'syn213309'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID syn213309 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn213309', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE81861', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE75688', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE81383', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE103322', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE76312', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE70630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'doi.org/10.1242/bio.059256', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn30243143', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn213309', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81861 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81861 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE81861 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75688 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75688 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE75688 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81383 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81383 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE81383 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE103322 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE103322 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE103322 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE76312 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE76312 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE76312 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE70630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE70630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE70630 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: doi.org/10.1242/bio.059256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: doi.org/10.1242/bio.059256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier doi.org/10.1242/bio.059256 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for doi.org/10.1242/bio.059256 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier doi.org/10.1242/bio.059256 does not match pattern for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: doi.org and accession_id: doi.org/10.1242/bio.059256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/doi.org/10.1242/bio.059256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn30243143 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn213309 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn213309 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier syn213309 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 22, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 11\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 8th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11661334&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b66dd480>, <Element supplementary-material at 0x1b66dd640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b66dd480>, <Element supplementary-material at 0x1b66dd640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We also observed a clear linear trend between the measured correlation of protein expression within well-imputed pairs and their imputed co-expression, further supporting the robustness of COWAS imputation models ( Supplementary Figure 1 [this file] ). \n",
      "For LDL cholesterol, which had the best-powered GWAS of the three traits we considered, we identified 38 protein pairs with a significant co-expression effect ( Supplementary Figure 2 [this file] ). \n",
      "By visualizing significant COWAS pairs as a network, we identified complexes of mutually interacting proteins that modulate LDL cholesterol levels ( Supplementary Figure 3 [this file] ). For example, we found that APOE and PLTP both decrease LDL cholesterol levels, while their co-expression is associated with increased LDL cholesterol levels ( Supplementary Figure 4 [this file] ). \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 3 [rid=F3] are also summarized in Supplementary Table 1 [this file] . \n",
      "Results for protein pairs identified as significant using at least one of these three tests with IGAP GWAS data are shown in Supplementary Figure 5 [this file] and Supplementary Table 2 [this file] . \n",
      "Finally, we considered an alternative, product-based formulation of COWAS (see Supplementary Note 1 [this file] for details). The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [this file] and Supplementary Data 1 [rid=SD2] ). \n",
      "We also discovered 22 pairs with an overall effect on AD using the product-based COWAS global test and EADB summary statistics ( Supplementary Figure 7 [this file] and Supplementary Table 3 [this file] ). Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [this file] , Supplementary Table 3 [this file] , and Supplementary Data 18 [rid=SD2] – 19 [rid=SD2] . \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 4 [rid=F4] are also summarized in Supplementary Table 4 [this file] . \n",
      "In Supplementary Note 1 [this file] we define an alternative, product-based version of COWAS in which is replaced with . \n",
      "In Supplementary Note 2 [this file] we derive a summary-level version of the second stage following the approach introduced in the multivariate imaging-wide association study (MV-IWAS) method [ 14 [rid=R14] , 16 [rid=R16] ], which is related to other methods in the field of Mendelian randomization [ 87 [rid=R87] , 88 [rid=R88] ]. We also provide full derivations for both individual-level and summary-level versions of standard TWAS/PWAS in Supplementary Note 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Across all of the model types we considered, the median out-of-sample correlation between estimated and imputed co-expression was always lower than the median out-of-sample correlation between measured and imputed single-protein expression ( Figure 2a [rid=F2] , Figure 2c [rid=F2] , and Supplementary Data 1 [this file] ). \n",
      "Among these well-imputed pairs, lasso regression with -pQTLs pre-screened by their effect sizes achieved the highest mean out-of-sample for predicting co-expression (mean , Supplementary Data 1 [this file] ). On the other hand, ridge regression with both -pQTLs and -pQTLs pre-screened by their values yielded the greatest number of well-imputed protein pairs ( Figure 2d [rid=F2] and Supplementary Data 1 [this file] ). \n",
      "Model performance metrics for every combination of protein pair and model type are provided in Supplementary Data 2 [this file] – 13 [this file] . \n",
      "Full imputation performance metrics for all analyzed proteins and outcome traits are provided in Supplementary Data 14 [this file] – 20 [this file] . \n",
      "Full results for LDL cholesterol, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 14 [this file] . \n",
      "Full results from our validation study for LDL cholesterol are provided in Supplementary Data 15 [this file] . \n",
      "Full results for AD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 16 [this file] . \n",
      "Full results for all protein pairs passing quality control are provided in Supplementary Data 17 [this file] . \n",
      "The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [rid=SD1] and Supplementary Data 1 [this file] ). \n",
      "Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [rid=SD1] , Supplementary Table 3 [rid=SD1] , and Supplementary Data 18 [this file] – 19 [this file] . \n",
      "Full results for PD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 20 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 248046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 62764 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-8f22b04815960b365b0a6062a4b2b27331ffe37d0ccee4c652d5a73851a98dfe\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 62764 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-cbadb78f-6a29-4ed7-b390-fb24f05dabd1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://synapse.org/cowas\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://csg.sph.umich.edu/willer/public/glgc-lipids2021\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST007511\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://synapse.org/cowas\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://csg.sph.umich.edu/willer/public/glgc-lipids2021\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST007511\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"repository_reference\\\": \\\"www.ebi.ac.uk\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321868,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 82963,\n",
      "        \"completion_tokens\": 364,\n",
      "        \"total_tokens\": 83327,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://synapse.org/cowas', 'repository_reference': 'synapse.org', 'dataset_id': 'https://synapse.org/cowas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://synapse.org/cowas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://synapse.org/cowas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://synapse.org/cowas, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://synapse.org/cowas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://synapse.org/cowas in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://synapse.org/cowas could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.synapse.org/[#!]*Synapse:__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 914 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 918 - INFO - Extracted ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://synapse.org/cowas is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://synapse.org/cowas', 'repository_reference': 'synapse.org', 'dataset_id': 'https://synapse.org/cowas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.ukbiobank.ac.uk/enable-your-research', 'repository_reference': 'www.ebi.ac.uk', 'dataset_id': 'https://www.ukbiobank.ac.uk/enable-your-research'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ukbiobank.ac.uk/enable-your-research, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.ukbiobank.ac.uk/enable-your-research is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.ukbiobank.ac.uk/enable-your-research', 'repository_reference': 'www.ebi.ac.uk', 'dataset_id': 'https://www.ukbiobank.ac.uk/enable-your-research'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/Synapse:syn51364943', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/Synapse:syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn51364943, None, https://www.synapse.org/Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie', 'repository_reference': 'n/a', 'dataset_id': 'https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cbdm-01.zdv.uni-mainz.de\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie', 'repository_reference': 'n/a', 'dataset_id': 'https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://csg.sph.umich.edu/willer/public/glgc-lipids2021', 'repository_reference': 'n/a', 'dataset_id': 'https://csg.sph.umich.edu/willer/public/glgc-lipids2021'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://csg.sph.umich.edu/willer/public/glgc-lipids2021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://csg.sph.umich.edu/willer/public/glgc-lipids2021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://csg.sph.umich.edu/willer/public/glgc-lipids2021, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://csg.sph.umich.edu/willer/public/glgc-lipids2021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://csg.sph.umich.edu/willer/public/glgc-lipids2021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: csg.sph.umich.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://csg.sph.umich.edu/willer/public/glgc-lipids2021 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://csg.sph.umich.edu/willer/public/glgc-lipids2021', 'repository_reference': 'n/a', 'dataset_id': 'https://csg.sph.umich.edu/willer/public/glgc-lipids2021'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST90027158', 'repository_reference': 'www.ebi.ac.uk', 'dataset_id': 'GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GCST90027158 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST007511', 'repository_reference': 'www.ebi.ac.uk', 'dataset_id': 'GCST007511'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GCST007511 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST009325', 'repository_reference': 'www.ebi.ac.uk', 'dataset_id': 'GCST009325'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GCST009325 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn51364943 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GCST90027158 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: www.ebi.ac.uk and accession_id: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GCST007511 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GCST009325 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: www.ebi.ac.uk and accession_id: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 9th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11929800&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 40 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1003880>, <Element supplementary-material at 0x1b1999380>, <Element supplementary-material at 0x1b66d9240>, <Element supplementary-material at 0x1b66dc580>, <Element supplementary-material at 0x1b66de280>, <Element supplementary-material at 0x1ac685f80>, <Element supplementary-material at 0x1b18cf800>, <Element supplementary-material at 0x1b18cfa00>, <Element supplementary-material at 0x1b18ce000>, <Element supplementary-material at 0x1b18cc4c0>, <Element supplementary-material at 0x1b6b9c380>, <Element supplementary-material at 0x1b18cde80>, <Element supplementary-material at 0x1b18cec00>, <Element supplementary-material at 0x1b18ce2c0>, <Element supplementary-material at 0x1b18cdd40>, <Element supplementary-material at 0x1b18ce680>, <Element supplementary-material at 0x1b18cdb00>, <Element supplementary-material at 0x1b18ce500>, <Element supplementary-material at 0x1b18cffc0>, <Element supplementary-material at 0x1b18cd100>, <Element supplementary-material at 0x1b18cce80>, <Element supplementary-material at 0x1b18cd440>, <Element supplementary-material at 0x1b18ce0c0>, <Element supplementary-material at 0x1b18cfb80>, <Element supplementary-material at 0x1b18cf0c0>, <Element supplementary-material at 0x1b18cc500>, <Element supplementary-material at 0x1b18cdbc0>, <Element supplementary-material at 0x1b18cfa80>, <Element supplementary-material at 0x1b18cddc0>, <Element supplementary-material at 0x1b18cf240>, <Element supplementary-material at 0x1b10194c0>, <Element supplementary-material at 0x1b101b040>, <Element supplementary-material at 0x1b1018200>, <Element supplementary-material at 0x1b101be00>, <Element supplementary-material at 0x1b101af40>, <Element supplementary-material at 0x1b1018700>, <Element supplementary-material at 0x1b101b240>, <Element supplementary-material at 0x1b101b480>, <Element supplementary-material at 0x1b101a8c0>, <Element supplementary-material at 0x1b1019dc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1003880>, <Element supplementary-material at 0x1b1999380>, <Element supplementary-material at 0x1b66d9240>, <Element supplementary-material at 0x1b66dc580>, <Element supplementary-material at 0x1b66de280>, <Element supplementary-material at 0x1ac685f80>, <Element supplementary-material at 0x1b18cf800>, <Element supplementary-material at 0x1b18cfa00>, <Element supplementary-material at 0x1b18ce000>, <Element supplementary-material at 0x1b18cc4c0>, <Element supplementary-material at 0x1b6b9c380>, <Element supplementary-material at 0x1b18cde80>, <Element supplementary-material at 0x1b18cec00>, <Element supplementary-material at 0x1b18ce2c0>, <Element supplementary-material at 0x1b18cdd40>, <Element supplementary-material at 0x1b18ce680>, <Element supplementary-material at 0x1b18cdb00>, <Element supplementary-material at 0x1b18ce500>, <Element supplementary-material at 0x1b18cffc0>, <Element supplementary-material at 0x1b18cd100>, <Element supplementary-material at 0x1b18cce80>, <Element supplementary-material at 0x1b18cd440>, <Element supplementary-material at 0x1b18ce0c0>, <Element supplementary-material at 0x1b18cfb80>, <Element supplementary-material at 0x1b18cf0c0>, <Element supplementary-material at 0x1b18cc500>, <Element supplementary-material at 0x1b18cdbc0>, <Element supplementary-material at 0x1b18cfa80>, <Element supplementary-material at 0x1b18cddc0>, <Element supplementary-material at 0x1b18cf240>, <Element supplementary-material at 0x1b10194c0>, <Element supplementary-material at 0x1b101b040>, <Element supplementary-material at 0x1b1018200>, <Element supplementary-material at 0x1b101be00>, <Element supplementary-material at 0x1b101af40>, <Element supplementary-material at 0x1b1018700>, <Element supplementary-material at 0x1b101b240>, <Element supplementary-material at 0x1b101b480>, <Element supplementary-material at 0x1b101a8c0>, <Element supplementary-material at 0x1b1019dc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The procedures for sample selection are shown in Supplementary Fig. 1 [this file] . More details for the distribution of brain 5hmC data can be found in Supplementary Fig. 2 [this file] – 3 [this file] . \n",
      "Of the 2821 DhMRs (606 associated with AD diagnosis, 1542 associated with Aβ load, 1399 associated with PHFtau tangles, annotated to 2642 distinct genes) identified in stage II (replication), about 69% reside in exons, ~15% in intergenic regions, and ~17% in introns (Supplementary Fig. 4a [this file] ). Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [this file] , Supplementary Data 20 [rid=MOESM22] ). \n",
      "Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [this file] , Supplementary Data 22 [rid=MOESM24] ). These genes were clustered into multiple protein-protein interaction (PPI) networks (Supplementary Fig. 6 [this file] ). The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [this file] , Supplementary Data 24 [rid=MOESM26] ). \n",
      "At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [this file] – 9 [this file] ; Supplementary Data 30 [rid=MOESM32] – 32 [rid=MOESM34] ). \n",
      "We also performed colocalization analysis using summary statistics from a cell-type specific -eQTL analysis of eight human brain cell types and found that one of the colocalized genes, , is expressed in both astrocytes (Supplementary Fig. 10 [this file] ) and microglia (Supplementary Fig. 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: After data pre-processing and quality control, our dataset included 197,765 5hmC regions (Supplementary Data 1 [this file] ) in 1050 older individuals (mean age: 89.5, age range: 65.9–108.3, Table 1 [rid=Tab1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [this file] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [this file] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [this file] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [this file] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [this file] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM10: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [this file] – 10 [rid=MOESM12] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM11: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM12: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [rid=MOESM10] – 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM13: Of the 2821 DhMRs replicated in ROS, 93 regions were significantly associated with all three neuropathological phenotypes: AD diagnosis, Aβ,and PHFtau (Supplementary Data 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM14: Notably, 43 DhMRs (out of 2821) also harbor known AD variants (Supplementary Data 12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM15: However, the association between 5hmC and AD pathology appeared not to be mediated by genetic variants because additionally adjusting for the genotype of lead SNPs in the candidate regions did not substantially attenuate the observed associations (Supplementary Data 13 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM16: It showed that the observed associations on chromosome 19 remained significant after further controlling for genotype (Supplementary Data 14 [this file] ). \n",
      "The observed associations of 5hmC regions on chromosome 19 with AD pathology are unlikely to be driven by because additional adjustment for genotype did not change the results (Supplementary Data 14 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM17: The effect of an individual 5hmC region on AD pathology was in general small to moderate (Supplementary Data 15 [this file] ). \n",
      "Results for the variance explained in Aβ load and PHFtau tangles are shown in Supplementary Data 15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM18: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [this file] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM19: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [this file] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM20: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [this file] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM21: Of these, 118 regions were associated with all three phenotypes (Supplementary Data 19 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM22: Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [rid=MOESM1] , Supplementary Data 20 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM23: Focusing on the 2821 DhMRs associated with AD pathologies (606 regions associated with AD diagnosis, 1542 regions associated with Aβ, 1399 regions associated with PHFtau, annotated to 2642 distinct genes), we evaluated whether altered DNA hydroxymethylation affects the expression of genes located in the vicinity ( ± 50 kb) of each DhMR using brain RNA-seq data (DLPFC) from the same participants (Supplementary Data 21 [this file] ). \n",
      "To further functionally annotate the AD-associated 5hmC regions, we calculated the correlation between the level of 5hmC and the level of H3K9ac generated in the dorsolateral prefrontal cortex from the same individuals (Supplementary Data 21 [this file] ) in ROSMAP . \n",
      "In a subset of individuals (Supplementary Data 21 [this file] ) with both 5hmC and brain proteomics data (DLPFC) , we calculated the correlation of AD-associated 5hmC with protein abundances (Methods). \n",
      "A total of 731 unique individuals with both 5hmC and proteomic data were included in this analysis (Supplementary Data 21 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM24: At FDR < 0.05, the levels of 886 DhMRs (annotated to 818 distinct genes) were correlated with gene expression (Supplementary Data 22 [this file] ). Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [rid=MOESM1] , Supplementary Data 22 [this file] ). \n",
      "A total of 819 samples have complete data for both 5hmC and RNA-seq data (Supplementary Data 22 [this file] ). \n",
      "A total of 513 ROSMAP participants with both 5hmC and H3K9ac data were included in this analysis (Supplementary Data 22 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM25: Moreover, the expression level for most of these genes (83%) was also significantly associated with AD diagnosis, Aβ or PHFtau (Supplementary Data 23 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM26: The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [rid=MOESM1] , Supplementary Data 24 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM27: At FDR < 0.05, none of the 5hmC regions associated with AD diagnosis (out of 606) was correlated with H3K9ac, despite that eight regions did show nominal correlation with H3K9ac at FDR < 0.1 (Supplementary Data 25 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM28: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [this file] – 27 [rid=MOESM29] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM29: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [rid=MOESM28] – 27 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM30: At FDR < 0.05, the expression levels of 91 proteins were significantly correlated with the abundance of 5hmC regions that were significantly associated with AD pathologies (Supplementary Data 28 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM31: These proteins are significantly enriched in biological processes related to amyloid precursor protein catabolic process, regulation of Aβ formation, regulation of signal transduction, neurogenesis, phosphorylation, synapse, protein binding, proteolysis, neuronal ion channel clustering, regulation of protein modification process, etc. (Supplementary Data 29 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM32: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [this file] – 32 [rid=MOESM34] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM33: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM34: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [rid=MOESM32] – 32 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM35: At PP4 > 0.8 (i.e., a high posterior probability that a single shared variant is responsible for two signals), three 5hmC regions, including one associated with Aβ (C12P044736) and two associated with PHFtau (C12P044736, C14P057817), colocalize with known AD genes ( and ) near the variants rs12582221 and rs12590654 (Supplementary Data 33 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM36: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [this file] – 35 [rid=MOESM37] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM37: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [rid=MOESM36] – 35 [this file] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM38: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM39: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM40: Source data are provided as a Source Data [this file] file. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 255203\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 64553 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b875e602855d24d57e26ba2ba1041c9900bad5d3ddd6046ee0ea5ed36c70c160\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 64553 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-359a2091-0c20-466d-954a-338b476792c6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE291402\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn4896408\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/studies/GCST90027158\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.14804357\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE291402\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn4896408\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/studies/GCST90027158\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.14804357\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321872,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 80814,\n",
      "        \"completion_tokens\": 443,\n",
      "        \"total_tokens\": 81257,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE291402', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402', 'dataset_id': 'GSE291402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE291402, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/Synapse:syn11724057', 'dataset_id': 'syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn11724057, synapse.org, https://www.synapse.org/Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3388564, synapse.org, https://www.synapse.org/#!Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org/Synapse:syn17015098', 'dataset_id': 'syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn17015098, synapse.org, https://www.synapse.org/Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4896408', 'repository_reference': 'https://www.synapse.org/Synapse:syn4896408', 'dataset_id': 'syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4896408, synapse.org, https://www.synapse.org/Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST90027158', 'repository_reference': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158', 'dataset_id': 'GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GCST90027158 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/gwas/studies/GCST90027158, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST90027158 for https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GCST90027158 for https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GCST90027158 for https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST90027158 for https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GCST90027158 for https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18485175', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn18485175', 'dataset_id': 'syn18485175'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn18485175, synapse.org, https://www.synapse.org/#!Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.14804357', 'repository_reference': 'doi.org', 'dataset_id': '10.5281/zenodo.14804357'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/zenodo.14804357, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.14804357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.14804357', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.14804357', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE291402 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4896408 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GCST90027158 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: www.ebi.ac.uk and accession_id: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn18485175 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/zenodo.14804357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.14804357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.14804357 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/zenodo.14804357 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: doi.org and accession_id: 10.5281/zenodo.14804357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/zenodo.14804357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 48, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 10th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7113080&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0987d10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 25 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for s1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 28) for s13: DISCUSSION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 83686\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 21674 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-aae4b5d78e6b3b0b114047580f2ee6bbe16f0258920f593a128aa3f72e3d111e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 21674 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-664181f9-1b2a-46e9-bedd-7b0b1714e36d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17394041\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17394041\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17394041\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17394041\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321890,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 26390,\n",
      "        \"completion_tokens\": 63,\n",
      "        \"total_tokens\": 26453,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17394041', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17394041', 'dataset_id': 'syn17394041'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn17394041, synapse.org, https://www.synapse.org/#!Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn17394041', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn17394041', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn17394041 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 11/100 (11.00%) | Elapsed: 00:01:59 | ETA: 00:16:04\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 11th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8847344&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x142c8c800>, <Element supplementary-material at 0x1b1a79440>, <Element supplementary-material at 0x1b67fb6c0>, <Element supplementary-material at 0x1b101bd00>, <Element supplementary-material at 0x1b101aa00>, <Element supplementary-material at 0x1b101b800>, <Element supplementary-material at 0x1b1019780>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x142c8c800>, <Element supplementary-material at 0x1b1a79440>, <Element supplementary-material at 0x1b67fb6c0>, <Element supplementary-material at 0x1b101bd00>, <Element supplementary-material at 0x1b101aa00>, <Element supplementary-material at 0x1b101b800>, <Element supplementary-material at 0x1b1019780>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The data set is released in its entirety as Supplementary Materials [this file] to this paper, via an interactive website ( https://cancertrials.io/ (https://cancertrials.io/) ), and through the online Synapse repository (ID: syn25813713) . \n",
      "For trials reporting event-free survival data (e.g., PFS; Supplementary Fig. 1 [this file] ) median = 0.950 (Q1 = 0.909, mean: 0.929, Q3 = 0.970) as compared to a theoretical maximal = 0.996, which corresponds to 5% of observed variance not explained by the Weibull model. This is illustrated in Supplementary Fig. 2a, b [this file] for Weibull fits to OS and PFS data for panitumumab in combination with FOLFIRI and for FOLFIRI alone in wild-type and mutant metastatic colorectal cancer (trial 20050181 ; average = 0.99 for OS curves and 0.92 for PFS curves). \n",
      "Across the entire data set, some of the worst fits for two-parameter Weibull forms were observed for trials with relatively few events, for example, the Chronicle trial ( NCT00427713 (NCT00427713) ) with only eight deaths in the treatment arm and 16 progression events in the observation arm (Fig. 2c [rid=Fig2] ; Supplementary Fig. 1 [this file] ). \n",
      "For a small number of trials in which the asymptote of the fitted survival curve was greater than zero (i.e., patients were expected to be alive at the end of the longest follow-up), a three-parameter Weibull distribution consisting of the traditional two-parameter distribution with an additional “cure rate” term had an improved fit (Supplementary Fig. 3 [this file] ). \n",
      "This is potentially consistent with a two-population hypothesis (Fig. 3a [rid=Fig3] ; Supplementary Fig. 4 [this file] ). Moreover, a mixture model exhibited no meaningful improvement in fit as compared to a single-distribution for OS data from ICI or control arms in these trials (Fig. 3b [rid=Fig3] ; Supplementary Fig. 4 [this file] ). This generated the steep decline in PFS values observable in the control and experimental arms of actual ICI trials, and improved fit to PFS distributions, raising mean from 0.93 to 0.98 (Fig. 3c [rid=Fig3] ; Supplementary Fig. 4 [this file] ; “Methods”). \n",
      "A challenge in the analysis of such data is that, when too few events have occurred, nonparametric numeric confidence estimates return non-informative values (usually reported as a value “not reached” or “indeterminate” as illustrated for two different scenarios in Supplementary Fig. 5a [this file] ). We found that 12-month survival estimates were non-informative for 20–40% of OS trial arms, and 23–61% of event-free survival trial arms, with higher failure rates occurring when the sample size was smaller (Supplementary Fig. 5b, c [this file] ). \n",
      "For event-free survival (for which 99 event-free survival curves could be compared), the precision of a Weibull-based approach was also greater than a nonparametric approach across all sample sizes, while accuracy was equivalent for small sample sizes (fewer than 40 patients) (Supplementary Fig. 6a [this file] ). \n",
      "Thus, parametric fitting of data from 15 patients made a comparably precise and accurate estimate of median PFS as nonparametric analysis of 60 patients (Supplementary Fig. 6b [this file] ). \n",
      "We observed that for a representative simulation, a two-parameter Weibull form accurately described the synthetic trial data ( = 0.98; Supplementary Fig. 7 [this file] ). \n",
      "Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [this file] and Supplementary Data 2 [rid=MOESM5] ). \n",
      "A similar pattern was observed for event-free survival data, with control and experimental values differing by 1.0% for unsuccessful trials and 36% for successful trials (Supplementary Fig. 9 [this file] ). \n",
      "For event-free survival data, 155 comparisons of experimental and control arms from 146 trial figures revealed a range of Δ values from +0.55 to −0.85 (median |Δ | = 0.08) (Supplementary Fig. 9 [this file] ). \n",
      "All data generated or analyzed during this study are included in this published article (and its supplementary information [this file] files). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: The accuracy of IPD imputation is discussed further in Supplementary Data 1 [this file] and “Methods”. \n",
      "Trials with a hazard ratio difference greater than 0.1, or with perceptible visual differences, were removed from the final data set and not analyzed further (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Best-fit Weibull parameters were compared across cancer types and metastatic status using an ANOVA test with a Bonferroni correction for multiple hypothesis testing at a two-tailed significance level of 0.05 (see Supplementary Data 2 [this file] ). Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [rid=MOESM1] and Supplementary Data 2 [this file] ). \n",
      "All code used in this study is included in Supplementary Data 2 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Finally, when we examined PFS data from an additional 25 ICI trials, we found that the drop in survival at early event times (which we identified as the time corresponding to the greatest change in the slope of the survival curve) was strongly correlated with the time of the first radiological scan (as reported in trials’ methods sections; Pearson correlation 0.982, < 10 ; Supplementary Data 3 [this file] ). \n",
      "All extracted values can be found in Supplementary Data 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 157076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 40021 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4513df5cdc741cdd63a5037e0135c1720b3a69c257b4beb439ccd4fab2c7116e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 40021 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ec64ed3e-04a9-4a8a-91fb-f48bc984dbd9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00973609\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00982111\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01377376\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00601900\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00427713\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01229813\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01673867\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01057810\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT02130466\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01584830\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00110162\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01607957\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00861614\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00973609\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00982111\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01377376\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00601900\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00427713\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01229813\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01673867\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01057810\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT02130466\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01584830\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00110162\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT01607957\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT00861614\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321893,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44563,\n",
      "        \"completion_tokens\": 513,\n",
      "        \"total_tokens\": 45076,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25813713', 'repository_reference': 'synapse.org', 'dataset_id': 'syn25813713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25813713, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00973609', 'repository_reference': 'n/a', 'dataset_id': 'NCT00973609'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00973609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00973609 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00973609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00973609 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00973609 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00973609 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00973609 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00973609 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00973609', 'repository_reference': 'n/a', 'dataset_id': 'NCT00973609'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00982111', 'repository_reference': 'n/a', 'dataset_id': 'NCT00982111'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00982111\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00982111 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00982111\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00982111 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00982111 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00982111 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00982111 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00982111 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00982111', 'repository_reference': 'n/a', 'dataset_id': 'NCT00982111'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01377376', 'repository_reference': 'n/a', 'dataset_id': 'NCT01377376'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01377376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01377376 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01377376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01377376 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01377376 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01377376 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01377376 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01377376 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01377376', 'repository_reference': 'n/a', 'dataset_id': 'NCT01377376'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00601900', 'repository_reference': 'n/a', 'dataset_id': 'NCT00601900'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00601900\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00601900 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00601900\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00601900 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00601900 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00601900 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00601900 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00601900 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00601900', 'repository_reference': 'n/a', 'dataset_id': 'NCT00601900'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00427713', 'repository_reference': 'n/a', 'dataset_id': 'NCT00427713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00427713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00427713 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00427713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00427713 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00427713 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00427713 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00427713 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00427713 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00427713', 'repository_reference': 'n/a', 'dataset_id': 'NCT00427713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01229813', 'repository_reference': 'n/a', 'dataset_id': 'NCT01229813'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01229813\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01229813 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01229813\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01229813 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01229813 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01229813 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01229813 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01229813 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01229813', 'repository_reference': 'n/a', 'dataset_id': 'NCT01229813'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01673867', 'repository_reference': 'n/a', 'dataset_id': 'NCT01673867'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01673867\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01673867 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01673867\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01673867 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01673867 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01673867 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01673867 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01673867 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01673867', 'repository_reference': 'n/a', 'dataset_id': 'NCT01673867'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01057810', 'repository_reference': 'n/a', 'dataset_id': 'NCT01057810'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01057810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01057810 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01057810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01057810 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01057810 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01057810 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01057810 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01057810 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01057810', 'repository_reference': 'n/a', 'dataset_id': 'NCT01057810'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT02130466', 'repository_reference': 'n/a', 'dataset_id': 'NCT02130466'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT02130466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT02130466 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT02130466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT02130466 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT02130466 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT02130466 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT02130466 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT02130466 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT02130466', 'repository_reference': 'n/a', 'dataset_id': 'NCT02130466'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01584830', 'repository_reference': 'n/a', 'dataset_id': 'NCT01584830'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01584830\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01584830 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01584830\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01584830 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01584830 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01584830 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01584830 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01584830 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01584830', 'repository_reference': 'n/a', 'dataset_id': 'NCT01584830'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00110162', 'repository_reference': 'n/a', 'dataset_id': 'NCT00110162'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00110162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00110162 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00110162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00110162 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00110162 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00110162 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00110162 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00110162 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00110162', 'repository_reference': 'n/a', 'dataset_id': 'NCT00110162'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT01607957', 'repository_reference': 'n/a', 'dataset_id': 'NCT01607957'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT01607957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT01607957 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT01607957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT01607957 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT01607957 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT01607957 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT01607957 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT01607957 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT01607957', 'repository_reference': 'n/a', 'dataset_id': 'NCT01607957'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NCT00861614', 'repository_reference': 'n/a', 'dataset_id': 'NCT00861614'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NCT00861614\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NCT00861614 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: NCT00861614\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT00861614 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NCT00861614 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NCT00861614 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT00861614 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT00861614 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'NCT00861614', 'repository_reference': 'n/a', 'dataset_id': 'NCT00861614'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25813713 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 12th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9280291&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b68c8b00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b68c8b00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref DS1: The IPP gene set contains 26 immune-related genes and 3 genes used for normalization, and we used those the total of 29 as input in the IPP models ( Supplementary Table 1 [this file] ). \n",
      "In the 7 studies ( 43 [rid=B43] , 45 [rid=B45] , 46 [rid=B46] , 51 [rid=B51] – 53 [rid=B53] , 58 [rid=B58] ) with GE data collected at time points 3 to 7, there were 270 arrays sampled on 173 patients, of whom 134 were alive and 39 (22.5%) deceased at day 30; 122 were used for training and 51 for testing models ( Supplementary Tables 2 [this file] , 3 [this file] ). \n",
      "We ran the COCONUT algorithm on the 17 studies selected for analysis and assessed the effect of co-normalization: (1) on patient-level GE data across studies ( Figure 1 [rid=F1] and Supplementary Figure 1 [this file] ); (2) at the gene level in controls and cases ( Supplementary Figure 2 [this file] presenting data for CD3D); (3) for 2 genes in controls and cases, here with CLDN8 (a housekeeping gene, with minimal difference in mean GE between controls and cases and minimal overall GE variance) and CEACAM1, up-regulated during sepsis ( Supplementary Figure 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 177152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 45040 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-46a7a1b251ec287eb789d938d9d47502bc6f2bbcd11132b8ef674a8d864066d6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 45040 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c74a5bd7-5a5b-43b6-b4e4-c1c7ad9cb63d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE27131\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE32707\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE40586\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE66099\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE21802\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE54514\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE20346\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE40012\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57065\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60244\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65682\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE95233\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MEXP-3589\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-1548\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5273/5274\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13015\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE25504\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn5612563\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"DS1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE27131\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE32707\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE40586\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE66099\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE21802\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE54514\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE20346\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE40012\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57065\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60244\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65682\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE95233\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MEXP-3589\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-1548\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5273/5274\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13015\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE25504\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn5612563\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"DS1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321897,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 53352,\n",
      "        \"completion_tokens\": 639,\n",
      "        \"total_tokens\": 53991,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE27131', 'repository_reference': 'geo', 'dataset_id': 'GSE27131'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE27131, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE27131', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE32707', 'repository_reference': 'geo', 'dataset_id': 'GSE32707'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE32707, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE32707', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE40586', 'repository_reference': 'geo', 'dataset_id': 'GSE40586'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE40586, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE40586', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE66099', 'repository_reference': 'geo', 'dataset_id': 'GSE66099'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE66099, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE66099', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE21802', 'repository_reference': 'geo', 'dataset_id': 'GSE21802'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE21802, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE21802', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE54514', 'repository_reference': 'geo', 'dataset_id': 'GSE54514'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE54514, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE54514', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE20346', 'repository_reference': 'geo', 'dataset_id': 'GSE20346'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE20346, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE20346', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE40012', 'repository_reference': 'geo', 'dataset_id': 'GSE40012'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE40012, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE40012', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE57065', 'repository_reference': 'geo', 'dataset_id': 'GSE57065'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE57065, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE57065', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE60244', 'repository_reference': 'geo', 'dataset_id': 'GSE60244'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE60244, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60244', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65682', 'repository_reference': 'geo', 'dataset_id': 'GSE65682'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE65682, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65682', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE95233', 'repository_reference': 'geo', 'dataset_id': 'GSE95233'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE95233, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE95233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MEXP-3589', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MEXP-3589'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MEXP-3589 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MEXP-3589', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-1548', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-1548'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-1548 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-1548', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5273/5274', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-5273/5274'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-5273/5274 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5273/5274', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE13015', 'repository_reference': 'geo', 'dataset_id': 'GSE13015'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE13015, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13015', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE25504', 'repository_reference': 'geo', 'dataset_id': 'GSE25504'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE25504, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE25504', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn5612563', 'repository_reference': 'synapse.org', 'dataset_id': '10.7303/syn5612563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn5612563, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'DS1', 'repository_reference': 'n/a', 'dataset_id': 'DS1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: DS1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID DS1 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: DS1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier DS1 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier DS1 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier DS1 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DS1 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier DS1 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'DS1', 'repository_reference': 'n/a', 'dataset_id': 'DS1'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE27131', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE32707', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE40586', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE66099', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE21802', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE54514', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE20346', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE40012', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE57065', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE60244', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65682', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE95233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MEXP-3589', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-1548', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5273/5274', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13015', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE25504', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 18 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE27131 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE27131 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE27131 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE32707 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE32707 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE32707 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE40586 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE40586 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE40586 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE66099 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE66099 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE66099 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE21802 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE21802 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE21802 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE54514 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE54514 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE54514 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE20346 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE20346 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE20346 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE40012 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE40012 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE40012 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE57065 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE57065 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE57065 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE60244 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE60244 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE60244 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65682 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65682 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE65682 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE95233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE95233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE95233 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MEXP-3589 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: arrayexpress and accession_id: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-1548 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: arrayexpress and accession_id: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-5273/5274 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: arrayexpress and accession_id: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13015 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13015 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE13015 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE25504 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE25504 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE25504 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 17: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn5612563 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 18 with repo: doi.org and accession_id: 10.7303/syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 18\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 13th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10029021&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6bb95c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6bb95c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: S1 [this file] ) comprising 672 distinct donors. S2 [this file] ; Methods [this file] ), we retained 654 samples with matched genotype and cortical RNA-seq data spanning 4-39 PCW, the majority falling within the first two trimesters (Tri1, n=216; Tri2, n=433; Table S1 [this file] ). S3 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "Compared with adult brain eQTL datasets from PsychENCODE ( 13 [rid=R13] , 14 [rid=R14] ) and GTEx ( 16 [rid=R16] ), we identified 2,488 unique “fetal-specific” eGenes in this dataset ( Fig. 1B [rid=F1] ; Table S1 [this file] ). S3 [this file] ). S3 [this file] ), suggesting that these context-specific annotations may be more relevant for interpretation of neuropsychiatric GWAS signals, which are known to be under negative selection. S3 [this file] ; Table S1 [this file] ). \n",
      "S4 [this file] ; Fig. S5 [this file] ; Methods [this file] ), identifying 11,845 and 7,490 genes with a significant isoQTL or sQTL (termed “isoGenes” and “sGenes”; Table S1 [this file] ), respectively. As expected, -xQTL are strongly clustered near their target genes’ transcription start site (TSS) ( Fig. 1E [rid=F1] ) and are correspondingly enriched in functional regions of the genome ( Fig. 1F [rid=F1] ; Table S1 [this file] ). To investigate the variant-level overlap among -e/iso/sQTLs, we calculated Storey’s pi1 statistic ( 40 [rid=R40] ) and found that genetic variants directly associated with RNA splicing (especially isoQTL) capture independent and additional signals from eQTLs ( Fig. 1H [rid=F1] ; Methods [this file] ). \n",
      "S3 [this file] ; Table S1 [this file] ; Methods [this file] ), we identified multiple independent regulatory signals for 3,570 eGenes in the fetal brain, with some exhibiting up to 10 groups of conditionally independent -eQTL signals. Each CS prioritized a median of 5 SNPs, with 2,423 containing exactly one SNP, which are strong candidates for functional validation ( Data S1 [this file] ). \n",
      "S6 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "S2 [this file] ; Methods [this file] ), we inferred genetic ancestries in the fetal brain samples in reference to the 1000 Genomes ( 44 [rid=R44] ). S1 [this file] ; Table S1 [this file] ). Following the multi-ancestry QTL mapping pipeline, we independently mapped xQTLs in the three largest ancestry groupings: EUR, AMR, and AFR. 986 eGenes were shared between the multi-ancestry dataset and the three ancestry groupings ( Fig. 2B [rid=F2] ; Table S2 [this file] ). We then performed statistical fine-mapping separately in the ancestry groupings ( 42 [rid=R42] ), and show that AFR, despite of the smallest sample size, has on average the smallest credible sets (CS) of causal variants ( Fig. 2D [rid=F2] ; Data S2 [this file] , S3 [this file] , S4 [this file] ). \n",
      "S7 [this file] ; Table S3 [this file] ). Mirroring eQTL results, we found that heritability drops significantly from Tri1 to Tri2 ( Fig. 3C [rid=F3] ; Wilcoxon p < 2.2e-16; Table S3 [this file] ; Data S5 [this file] , S6 [this file] ). \n",
      "S7 [this file] ), gene expression levels, or phenotypic variance across Tri1 and Tri2 that could be driving these results ( Fig. S8 [this file] ). S7 [this file] ; Table S3 [this file] ; Data S7 [this file] , S8 [this file] ). \n",
      "We generated continuous annotations for genetic variants based on fine-mapping posterior probabilities ( 51 [rid=R51] ) and found SCZ GWAS signals to be highly enriched among fetal brain regulatory elements -- more so than in the adult brain ( Fig. 4B [rid=F4] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ). \n",
      "Across these traits, we consistently observed greater heritability mediated by isoform regulation (isoform h /h 6.45±1.92%) than that of gene expression (3.15±1.25%) or splicing (2.13±1.74%), although the overall extent of mediation remains small ( Fig. 4C [rid=F4] ; Table S4 [this file] ; Methods [this file] ). Extending these analyses across trimester-specific annotations, although less well powered, pointed to isoform-regulation in Tri2 as being particularly important, especially for BIP ( Fig. 4D [rid=F4] ; Table S4 [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ; Methods [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ). S10 [this file] ) ( 10 [rid=R10] ). S10 [this file] ). \n",
      "Other novel disease mechanisms prioritized by these analyses include: downregulation of the GABA -receptor subunit in SCZ (eQTL: rs514733, CLPP=0.32; Fig. 5B [rid=F5] ; Table S5 [this file] ), providing genetic evidence for GABAergic dysfunction hypothesized to contribute to disease pathophysiology ( 60 [rid=R60] ); splicing dysregulation of in BIP (top colocalization sQTL: rs11750832, CLPP=0.97; Fig. S10 [this file] ; Table S5 [this file] ) which encodes adenylate cyclase, a cell membrane-bound enzyme regulating cAMP signaling and a target of the mood stabilizer lithium ( 61 [rid=R61] ); splicing dysregulation in BIP of (sQTL: rs17183814, CLPP=0.70; Fig. S10 [this file] ; Table S5 [this file] ), a voltage gated sodium channel subunit with rare variant associations in ASD and epilepsy ( 62 [rid=R62] , 63 [rid=R63] ); and in MDD, the dysregulated splicing of (CLPP=0.43; Fig. S10 [this file] ; Table S5 [this file] ), a highly constrained E3 ubiquitin-protein ligase and regulator of N6-methyladenosine RNA modification involved in neural-immune activation ( 64 [rid=R64] ). \n",
      "We next conducted an isoform-centric transcriptome-wide association study (isoTWAS) ( 65 [rid=R65] ), to identify genes and their isoforms whose regulated expression is associated with SCZ risk. isoTWAS identified 536 isoforms across 271 unique genes with significant isoTWAS associations (Bonferroni-corrected P < 0.05 across genes, FWER-corrected P < 0.05 across transcripts of the same gene, permutation P < 0.05; Methods [this file] ). To leverage local LD and to account for SNP weight correlations, we performed fine-mapping on isoTWAS associations that passed permutation testing ( Fig. 5D [rid=F5] ; Table S5 [this file] ). \n",
      "Among 14 TWAS-identified genes with >3 conditionally independent eQTLs, the MRLocus framework provided additional support for 7 genes with FDR<0.2 ( Table S5 [this file] ; Methods [this file] ). S11 [this file] ). \n",
      "In total, we identified 124 gene and isoform co-expression modules demonstrating enrichment for all major developmental cell types ( 36 [rid=R36] ) and recapitulating early biochemical processes and developmental pathways ( Fig. 6A [rid=F6] ; Methods [this file] ). \n",
      "Overlaying enrichment for common and rare variation within the hierarchical structure defined by co-expression pinpointed groups of modules that harbor most of neurodevelopmental disease risk ( Fig. 6B [rid=F6] ; Table S6 [this file] ; Methods [this file] ). S12 [this file] ). \n",
      "S12 [this file] ), which exhibits strong enrichment for cross-disorder common variation and rare variation risk for ASD, epilepsy, and SCZ. S12 [this file] ), which strongly enriched for synaptic gene ontologies and excitatory neuron marker genes. S12 [this file] ). \n",
      "S12 [this file] ) displayed robust BIP, education attainment (EA), and SCZ enrichment while harboring high risk hub genes like and – similar to the prenatally-enriched M37 module described in Li et al ( 18 [rid=R18] ). S12 [this file] ). \n",
      "CellWalker mapped 21.7% of eQTL-containing genes to a specific cellular context in the developing human brain, corresponding to 3,739 of the bulk-derived eQTLs ( Fig. 7A [rid=F7] ; Table S7 [this file] ; Methods [this file] ). To identify such ieQTLs, in which SNP-gene associations are modulated by the levels of a given module, we tested the interaction effect between genotype and module eigengenes on gene and isoform expression ( Fig. 7B [rid=F7] ; Methods [this file] ). Across all gene and isoform modules, 8,008 ieQTLs were identified after the permutation test (BH-corrected FDR<0.05; Methods [this file] ), in which 3,960 ieQTLs were specific to a single module ( Fig. 7C [rid=F7] ; Data S9 [this file] ). Finally, integrating disease GWAS, we identified significant colocalizations (CLPP>0.01) between ieQTLs and 22 SCZ GWAS loci, 13 of which are not found with bulk cA-eQTLs ( Table S5 [this file] ). \n",
      "See Methods [this file] for details. ( ) A schematic of module interaction ieQTL mapping and validation in cultured neurons and progenitors. ( ) Results from module ieQTL mapping. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 565584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 142148 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-060cbf013ee685aa3fe8adfab5d5f7e5ab71d78a4bd555556be35c90de1046a1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 142148 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-166018c4-de04-4790-9707-97eac70f9e7f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/gandallab/Fetal_metaQTL\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/gandallab/Fetal_metaQTL\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321900,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 176561,\n",
      "        \"completion_tokens\": 112,\n",
      "        \"total_tokens\": 176673,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn50897018.3', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/', 'dataset_id': 'syn50897018.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn50897018.3, synapse.org, https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/gandallab/Fetal_metaQTL', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/gandallab/Fetal_metaQTL, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/gandallab/Fetal_metaQTL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/gandallab/Fetal_metaQTL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn50897018.3 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 14th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11525589&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6ba7180>, <Element supplementary-material at 0x1b1019200>, <Element supplementary-material at 0x1b1c85380>, <Element supplementary-material at 0x1b18b2100>, <Element supplementary-material at 0x1b1475ac0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6ba7180>, <Element supplementary-material at 0x1b1019200>, <Element supplementary-material at 0x1b1c85380>, <Element supplementary-material at 0x1b18b2100>, <Element supplementary-material at 0x1b1475ac0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 98364\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 25343 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-48109e8ba18b33ab103b143b0755d2a4195ce7ebb9aac2ec1c7df60c2f12b12f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 25343 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-07e5e838-3f25-4aa5-8e2a-78d60ca0b53c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321902,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 30288,\n",
      "        \"completion_tokens\": 93,\n",
      "        \"total_tokens\": 30381,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51514106', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn51514106'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn51514106, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn51514106', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn51514106'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn51514106, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn51514106 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn51514106 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 15th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10836119&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b18abe00>, <Element supplementary-material at 0x1b101c3c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b18abe00>, <Element supplementary-material at 0x1b101c3c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: Study characteristics and acylcarnitine descriptions can be found in Supplementary Tables 1 [this file] and 2 [this file] , respectively. \n",
      "The analysis split ADNI participants into two clusters, 2 and 3, at the top of the tree, with significantly different levels of p-tau in cerebrospinal fluid (CSF), glucose uptake measured by [ F] fluorodeoxyglucose-positron emission tomography (FDG-PET), and the 13-item AD assessment scale-cognitive subscale (ADAS-Cog. 13; Fig. 2b [rid=F2] - d [rid=F2] , Supplementary Table 3 [this file] ). All associations remained significant ( Supplementary Table 4 [this file] ). \n",
      "Specifically, we examined acylcarnitine-related SNPs reported by Shin et al. as non-modifiable factors (SNP list is provided in Supplementary Table 5 [this file] , replication in ADNI in Supplementary Table 6 [this file] ) and modifiable acylcarnitine levels (i.e., residuals adjusted for genetic and other non-modifiable factors) to better understand their impact on subgroup separation. \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [this file] - 9 [this file] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [rid=SD2] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [rid=SD2] and Supplementary Table 10 [this file] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [rid=SD2] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [rid=SD2] and Supplementary Table 11 [this file] ). When applied to the ADNI cohort, correlation dropped to 28% (cognitively normal participants only), which was very similar to what we observed when replicating the bioenergetic age computation process in AGES-RS (r = 29%, Supplementary Table 12 [this file] ). \n",
      "Bioenergetic age highlighted interesting relationships between subgroups and AD pathology throughout the tree ( Supplementary Table 13 [this file] ). This was further confirmed by significant associations of bioenergetic age with AD biomarkers across the A-T-(N)-(C) spectrum ( Supplementary Table 14 [this file] ), and consistent findings for cognitive function and grey matter volume in AGES-RS ( Supplementary Table 15 [this file] ). \n",
      "Specifically, bioenergetically younger individuals demonstrated a significantly slower decline over five years ( Supplementary Table 16 [this file] ). This result was replicated using longitudinal diagnosis data from the AGES-RS study ( Fig. 4d [rid=F4] , Supplementary Table 15 [this file] ), providing further evidence that younger bioenergetic age is protective against cognitive decline. \n",
      "We selected the grouping that had the most significant associations across three cognitive measures: ADAS-Cog 13, memory, and executive function ( Supplementary Table 17 [this file] ). \n",
      "The final dataset contained 140 metabolite measurements, covering 23 acylcarnitine species ( Supplementary Table 2 [this file] ), for 1,531 individuals. \n",
      "Data for 22 out of 23 acylcarnitines investigated in ADNI ( Supplementary Table 2 [this file] ) were available here, with the exception being levels of C4:1, which was not reliably measured in KORA. \n",
      "We obtained a list of 32 acylcarnitine-associated SNPs ( Supplementary Table 5 [this file] ) from a large GWAS study performed on a total of 7,824 individuals from the KORA and TwinsUK studies . \n",
      "To investigate robustness of the model, we performed 10-fold, 3 times repeated cross-validation ( Supplementary Table 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Notably, the most substantial contributions were observed for medium- and long-chain acylcarnitines, highlighting a likely modifiable role for beta-oxidation function ( Supplementary Fig. 1 [this file] ). \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [rid=SD1] - 9 [rid=SD1] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [this file] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [this file] and Supplementary Table 10 [rid=SD1] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [this file] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [this file] and Supplementary Table 11 [rid=SD1] ). \n",
      "Furthermore, this analysis revealed a strong correlation between bioenergetic age and the first principal component of acylcarnitine levels, indicating that the primary axis of variation in the data and the clustering is associated with bioenergetic age ( Supplementary Fig. 6 [this file] ). \n",
      "Rather, we found that individuals within cluster 7 consistently exhibit the same chronological but significantly younger bioenergetic age than other participants irrespective of their diagnostic group ( Supplementary Fig. 7 [this file] ). \n",
      "We attempted to replicate this grouping, which had similar genotype frequencies as in ADNI (59% vs. 58% in ROS/MAP; Supplementary Fig. 8 [this file] ), using trajectories of global cognition over 13 years in the ROS/MAP cohort. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 146240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 37312 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-035e9150a5ed76f322759673909475d9babf1baa0a9a159181d6cf99b86b0629\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 37312 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a8ae494e-be82-403c-942a-8648d5fd6494\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG024904\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"W81XWH-12-2-0012\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG046171\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG051550\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"3U01AG024904-09S4\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P30AG10161\\\",\\n    \\\"repository_reference\\\": \\\"ROS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG15819\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG17917\\\",\\n    \\\"repository_reference\\\": \\\"MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG30146\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36042\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RC2AG036547\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36836\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG48015\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG57473\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG32984\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46152\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46161\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG61356\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P50 AG016574\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG032990\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG046139\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG018023\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG006576\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG006786\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG025711\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG017216\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG003949\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 NS080820\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"N01-AG-12100\\\",\\n    \\\"repository_reference\\\": \\\"AGES-RS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HHSN271201200022C\\\",\\n    \\\"repository_reference\\\": \\\"AGES-RS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn5592519\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn9705278\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157329\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157322\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157238\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3205821.6\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG024904\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"W81XWH-12-2-0012\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG046171\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG051550\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"3U01AG024904-09S4\\\",\\n    \\\"repository_reference\\\": \\\"ADNI\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P30AG10161\\\",\\n    \\\"repository_reference\\\": \\\"ROS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG15819\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG17917\\\",\\n    \\\"repository_reference\\\": \\\"MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG30146\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36042\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RC2AG036547\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36836\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG48015\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG57473\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG32984\\\",\\n    \\\"repository_reference\\\": \\\"ROS/MAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46152\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46161\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG61356\\\",\\n    \\\"repository_reference\\\": \\\"ROSMAP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P50 AG016574\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG032990\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG046139\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG018023\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG006576\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01 AG006786\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG025711\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG017216\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 AG003949\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01 NS080820\\\",\\n    \\\"repository_reference\\\": \\\"MayoLOAD\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"N01-AG-12100\\\",\\n    \\\"repository_reference\\\": \\\"AGES-RS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HHSN271201200022C\\\",\\n    \\\"repository_reference\\\": \\\"AGES-RS\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn5592519\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn9705278\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157329\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157322\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157238\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3205821.6\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321908,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 43101,\n",
      "        \"completion_tokens\": 1386,\n",
      "        \"total_tokens\": 44487,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01 AG024904', 'repository_reference': 'ADNI', 'dataset_id': 'U01 AG024904'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01 AG024904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID U01 AG024904 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'W81XWH-12-2-0012', 'repository_reference': 'ADNI', 'dataset_id': 'W81XWH-12-2-0012'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: W81XWH-12-2-0012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID W81XWH-12-2-0012 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: W81XWH-12-2-0012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'W81XWH-12-2-0012', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG046171', 'repository_reference': 'ADNI', 'dataset_id': 'R01AG046171'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG046171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG046171 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: R01AG046171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG046171', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RF1AG051550', 'repository_reference': 'ADNI', 'dataset_id': 'RF1AG051550'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RF1AG051550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID RF1AG051550 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: RF1AG051550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'RF1AG051550', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '3U01AG024904-09S4', 'repository_reference': 'ADNI', 'dataset_id': '3U01AG024904-09S4'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 3U01AG024904-09S4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID 3U01AG024904-09S4 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: 3U01AG024904-09S4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '3U01AG024904-09S4', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'P30AG10161', 'repository_reference': 'ROS', 'dataset_id': 'P30AG10161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID P30AG10161 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS, identifier: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'P30AG10161', 'data_repository': 'ROS', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG15819', 'repository_reference': 'ROSMAP', 'dataset_id': 'R01AG15819'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG15819 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG15819', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG17917', 'repository_reference': 'MAP', 'dataset_id': 'R01AG17917'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG17917 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MAP, identifier: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG17917', 'data_repository': 'MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG30146', 'repository_reference': 'ROS/MAP', 'dataset_id': 'R01AG30146'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG30146 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG30146', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG36042', 'repository_reference': 'ROS/MAP', 'dataset_id': 'R01AG36042'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG36042 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG36042', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RC2AG036547', 'repository_reference': 'ROS/MAP', 'dataset_id': 'RC2AG036547'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID RC2AG036547 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'RC2AG036547', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG36836', 'repository_reference': 'ROS/MAP', 'dataset_id': 'R01AG36836'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG36836 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG36836', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG48015', 'repository_reference': 'ROS/MAP', 'dataset_id': 'R01AG48015'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG48015 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'R01AG48015', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RF1AG57473', 'repository_reference': 'ROS/MAP', 'dataset_id': 'RF1AG57473'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID RF1AG57473 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'RF1AG57473', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG32984', 'repository_reference': 'ROS/MAP', 'dataset_id': 'U01AG32984'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG32984 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'U01AG32984', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG46152', 'repository_reference': 'ROSMAP', 'dataset_id': 'U01AG46152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG46152 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'U01AG46152', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG46161', 'repository_reference': 'ROSMAP', 'dataset_id': 'U01AG46161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG46161 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'U01AG46161', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG61356', 'repository_reference': 'ROSMAP', 'dataset_id': 'U01AG61356'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG61356 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'U01AG61356', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'P50 AG016574', 'repository_reference': 'MayoLOAD', 'dataset_id': 'P50 AG016574'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: P50 AG016574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID P50 AG016574 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 AG032990', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 AG032990'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 AG032990\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 AG032990 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01 AG046139', 'repository_reference': 'MayoLOAD', 'dataset_id': 'U01 AG046139'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01 AG046139\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID U01 AG046139 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 AG018023', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 AG018023'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 AG018023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 AG018023 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01 AG006576', 'repository_reference': 'MayoLOAD', 'dataset_id': 'U01 AG006576'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01 AG006576\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID U01 AG006576 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01 AG006786', 'repository_reference': 'MayoLOAD', 'dataset_id': 'U01 AG006786'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01 AG006786\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID U01 AG006786 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 AG025711', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 AG025711'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 AG025711\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 AG025711 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 AG017216', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 AG017216'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 AG017216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 AG017216 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 AG003949', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 AG003949'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 AG003949\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 AG003949 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01 NS080820', 'repository_reference': 'MayoLOAD', 'dataset_id': 'R01 NS080820'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01 NS080820\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID R01 NS080820 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MayoLOAD, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'N01-AG-12100', 'repository_reference': 'AGES-RS', 'dataset_id': 'N01-AG-12100'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: N01-AG-12100\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID N01-AG-12100 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: AGES-RS, identifier: N01-AG-12100\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'N01-AG-12100', 'data_repository': 'AGES-RS', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'HHSN271201200022C', 'repository_reference': 'AGES-RS', 'dataset_id': 'HHSN271201200022C'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: HHSN271201200022C\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID HHSN271201200022C is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: AGES-RS, identifier: HHSN271201200022C\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'HHSN271201200022C', 'data_repository': 'AGES-RS', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn5592519', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn5592519'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn5592519, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn9705278', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn9705278'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn9705278, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn3157329', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn3157329'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn3157329, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn3157322', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn3157322'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn3157322, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn3157238', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn3157238'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn3157238, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn3205821.6', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': '10.7303/syn3205821.6'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn3205821.6, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn3205821.6', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'W81XWH-12-2-0012', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG046171', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'RF1AG051550', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '3U01AG024904-09S4', 'data_repository': 'ADNI', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'P30AG10161', 'data_repository': 'ROS', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG15819', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG17917', 'data_repository': 'MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG30146', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG36042', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'RC2AG036547', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG36836', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'R01AG48015', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'RF1AG57473', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'U01AG32984', 'data_repository': 'ROS/MAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'U01AG46152', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'U01AG46161', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'U01AG61356', 'data_repository': 'ROSMAP', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'MayoLOAD', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'N01-AG-12100', 'data_repository': 'AGES-RS', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'HHSN271201200022C', 'data_repository': 'AGES-RS', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn3205821.6', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 36 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: W81XWH-12-2-0012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: W81XWH-12-2-0012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier W81XWH-12-2-0012 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for W81XWH-12-2-0012 in adni\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: adni and accession_id: W81XWH-12-2-0012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository adni unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG046171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: R01AG046171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG046171 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG046171 in adni\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: adni and accession_id: R01AG046171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository adni unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: RF1AG051550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: RF1AG051550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RF1AG051550 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for RF1AG051550 in adni\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: adni and accession_id: RF1AG051550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository adni unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 3U01AG024904-09S4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ADNI, identifier: 3U01AG024904-09S4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 3U01AG024904-09S4 for ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 3U01AG024904-09S4 in adni\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: adni and accession_id: 3U01AG024904-09S4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository adni unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS, identifier: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier P30AG10161 for ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for P30AG10161 in ros\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: ros and accession_id: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros unknown in Ontology. Skipping dataset page 6.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG15819 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG15819 in rosmap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: rosmap and accession_id: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository rosmap unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: MAP, identifier: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG17917 for MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG17917 in map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: map and accession_id: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository map unknown in Ontology. Skipping dataset page 8.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG30146 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG30146 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: ros/map and accession_id: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36042 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG36042 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: ros/map and accession_id: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 10.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RC2AG036547 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for RC2AG036547 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: ros/map and accession_id: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36836 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG36836 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: ros/map and accession_id: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 12.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG48015 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for R01AG48015 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: ros/map and accession_id: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 13.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RF1AG57473 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for RF1AG57473 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: ros/map and accession_id: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 14.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROS/MAP, identifier: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG32984 for ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for U01AG32984 in ros/map\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: ros/map and accession_id: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ros/map unknown in Ontology. Skipping dataset page 15.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46152 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for U01AG46152 in rosmap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: rosmap and accession_id: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository rosmap unknown in Ontology. Skipping dataset page 16.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46161 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for U01AG46161 in rosmap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 17 with repo: rosmap and accession_id: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository rosmap unknown in Ontology. Skipping dataset page 17.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ROSMAP, identifier: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG61356 for ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ROSMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for U01AG61356 in rosmap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 18 with repo: rosmap and accession_id: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository rosmap unknown in Ontology. Skipping dataset page 18.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 19 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 19: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 20 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 20: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 21 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 21: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 22 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 22: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 23 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 23: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 24 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 24: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 25 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 25: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 26 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 26: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 27 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 27: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 28 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 28: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 29 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: N01-AG-12100\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: AGES-RS, identifier: N01-AG-12100\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier N01-AG-12100 for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for N01-AG-12100 in ages-rs\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 29 with repo: ages-rs and accession_id: N01-AG-12100\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ages-rs unknown in Ontology. Skipping dataset page 29.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 30 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: HHSN271201200022C\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: AGES-RS, identifier: HHSN271201200022C\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier HHSN271201200022C for AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for HHSN271201200022C in ages-rs\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 30 with repo: ages-rs and accession_id: HHSN271201200022C\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository ages-rs unknown in Ontology. Skipping dataset page 30.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 31 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn5592519 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 31 with repo: doi.org and accession_id: 10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 32 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn9705278 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 32 with repo: doi.org and accession_id: 10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 33 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn3157329 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 33 with repo: doi.org and accession_id: 10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 34 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn3157322 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 34 with repo: doi.org and accession_id: 10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 35 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn3157238 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 35 with repo: doi.org and accession_id: 10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 36 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn3205821.6 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn3205821.6 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn3205821.6 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 36 with repo: doi.org and accession_id: 10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 36\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 38, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 16th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6646046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b18cf700>, <Element supplementary-material at 0x1b18cd700>, <Element supplementary-material at 0x1b18cef80>, <Element supplementary-material at 0x1b18f2140>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b18cf700>, <Element supplementary-material at 0x1b18cd700>, <Element supplementary-material at 0x1b18cef80>, <Element supplementary-material at 0x1b18f2140>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [this file] , Methods [rid=S15] , and Supplementary Note [rid=SD3] ). \n",
      "As expected, no difference was observed for WBGs between prenatal and postnatal stages ( = 0.27, Supplementary Fig. 2 [this file] ). \n",
      "Interestingly, in GTEx data, we observed that nearest non-HRG genes are highly expressed in a majority of brain tissues while non-nearest HRGs are not ( Supplementary Fig. 3 [this file] ). \n",
      "Specifically, and both encoding voltage-gated calcium channel subunits, are involved in CCS and contribute to risk for SCZ . is differentially expressed in SCZ patients ( = 0.03), and both and capture multiple Hi-C links in brain Hi-C data ( Supplementary Fig. 4 [this file] ), contributing to the high PP of both genes in iRIGS. We also predicted two DNA binding proteins, SOX2 and SATB2 ( Supplementary Fig. 4 [this file] ), which have important roles in neurogenesis and have been widely implicated in SCZ. Several of our predicted HRGs are miR-137 target genes, including the aforementioned , and three other genes, and . , a glutamate-gated ion channel protein and a key mediator of synaptic plasticity , has a pdDNM and multiple regulatory connections ( Supplementary Fig. 4 [this file] ). , encoding Transcription Factor 4, participates in the initiation of neuronal differentiation by regulating the intrinsic excitability of prefrontal cortical neurons , and knockdown of alters the expression of genes important for developing prefrontal neocortex . is also linked with numerous DREs ( Supplementary Fig. 4 [this file] ) in iRIGS. \n",
      "We took a closer look at this region to gain more insights. captures 55 Hi-C links in brain Hi-C data ( Supplementary Fig. 5 [this file] ), while there is only one for . In GTEx data, is highly expressed in basal ganglia caudate, hypothalamus, basal ganglia nucleus accumbens, basal ganglia putamen and substantia nigra, but the expression in cortex and frontal cortex is rather low ( Supplementary Fig. 6 [this file] ). is uniformly and highly expressed in all brain tissues ( Supplementary Fig. 6 [this file] ). In BrainSpan data, shows constitutively high expression across all stages, and particularly higher expression at prenatal stages with a trajectory that peaks at the early-mid fetal stage, while shows lower expression across all developmental stages and no obvious pattern of transition between prenatal and postnatal stages ( Supplementary Fig. 7 [this file] ). \n",
      "One pdDNM and a high number of regulatory links were observed for ( Supplementary Fig. 8 [this file] ), promoting it as the top gene predicted by iRIGS. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [this file] and Supplementary Table 7 [rid=SD2] ). In particular, we found that 5 HRGs ( and ) are targets of nervous system drugs ( Supplementary Fig. 9 [this file] ), corresponding to a statistically significant enrichment ( = 0.01, OR = 3.78). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Specifically, we collected 278 gene sets relevant to CNS and behavior/neurological phenotypes from Mouse Genome Informatics (MGI) Mammalian Phenotype Ontology (MPO) database ( Methods [rid=S15] ) and observed significant enrichment of HRGs in 33 gene sets after Bonferroni correction ( Supplementary Table 4 [this file] ). \n",
      "In addition, we also observed that the 65 non-nearest HRGs are significantly enriched in 19 MPO gene sets ( Supplementary Table 5 [this file] ), while no significantly enriched gene sets were observed for the 65 nearest non-HRG genes ( Supplementary Table 6 [this file] ), providing strong and orthogonal support for iRIGS identified risk genes that are beyond the proximity to the GWAS index SNPs. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [rid=SD1] and Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: Details of iRIGS can be found in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "We performed iRIGS on these loci to identify risk genes of SCZ ( Supplementary Table 1 [this file] ). \n",
      "We also tried other thresholds to define LBGs, and the enrichment patterns are similar ( Supplementary Table 2 [this file] ). \n",
      "We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [rid=SD1] , Methods [rid=S15] , and Supplementary Note [this file] ). \n",
      "We therefore integrated them into iRIGS ( Methods [rid=S15] ) and predicted 104 high-confident risk genes (HRGs) in total ( Supplementary Table 1 [this file] ). \n",
      "We found that non-nearest HRGs are more significantly enriched in the gene sets compared to nearest non-HRG genes ( Supplementary Table 3 [this file] ), suggesting that the 65 non-nearest HRGs identified by iRIGS are more likely to be true risk genes than their nearest counterparts. \n",
      "We highlighted these genes with extended supporting evidence and description in Supplementary Table 1 [this file] . \n",
      "The distance of or to in the network is calculated using the random walk with restart (RWR) algorithm ( Supplemental Note [this file] ). We employed the Mahalanobis transformation to de-correlate the integrated multi-dimensional data so that any supportive genomic features can be properly incorporated ( Supplementary Note [this file] ). \n",
      "We adopted the Jensen-Shannon divergence to measure the tissue-specificity of each gene in each tissue ( Supplementary Note [this file] ). \n",
      "All the data used in this study are from public resources that were specified in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "See main text and Supplementary Note [this file] for details. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 164921\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 41982 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a5278a46cd5f6c333de2bc2509c439f2066e8ae5d69139cf0b32415dbc0b855b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 41982 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9d7309cf-667b-4e7b-90c4-fc22757e15b3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5609493\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn5609493\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-2323\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.gtexportal.org/home/datasets\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://help.brain-map.org/display/devhumanbrain/Documentation\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://enhancer.binf.ku.dk/presets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.vumc.org/cgg\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5609493\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn5609493\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-2323\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.gtexportal.org/home/datasets\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://help.brain-map.org/display/devhumanbrain/Documentation\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://enhancer.binf.ku.dk/presets/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.vumc.org/cgg\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321913,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51971,\n",
      "        \"completion_tokens\": 280,\n",
      "        \"total_tokens\": 52251,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5609493', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn5609493', 'dataset_id': 'syn5609493'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5609493, synapse.org, https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5609493', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-2323', 'repository_reference': 'http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/', 'dataset_id': 'E-MTAB-2323'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-2323 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/, identifier: E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-2323 for http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-2323 for http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-2323 for http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-2323 for http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-2323', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gtexportal.org/home/datasets', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.gtexportal.org/home/datasets, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.gtexportal.org/home/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.gtexportal.org/home/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn5609493', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-2323', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5609493 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-2323 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-2323 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-2323 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-2323 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-2323 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-2323\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 17th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10387117&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 9 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1002700>, <Element supplementary-material at 0x1b101bb40>, <Element supplementary-material at 0x1b1c60d00>, <Element supplementary-material at 0x1b1c61180>, <Element supplementary-material at 0x1b1c607c0>, <Element supplementary-material at 0x1b1c62540>, <Element supplementary-material at 0x1b1c62b00>, <Element supplementary-material at 0x1b1c620c0>, <Element supplementary-material at 0x1b1c62800>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1002700>, <Element supplementary-material at 0x1b101bb40>, <Element supplementary-material at 0x1b1c60d00>, <Element supplementary-material at 0x1b1c61180>, <Element supplementary-material at 0x1b1c607c0>, <Element supplementary-material at 0x1b1c62540>, <Element supplementary-material at 0x1b1c62b00>, <Element supplementary-material at 0x1b1c620c0>, <Element supplementary-material at 0x1b1c62800>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: MNN outperforms RPCA and Harmony on both the UMAP inspection and cluster stability metrics measurement, we therefore set the mutual nearest neighbor (MNN) matching method in Monocle 3 as default, and Seurat’s reciprocal principal component analysis (RPCA) based integration and Harmony as alternatives (Supplementary Fig. 1 [this file] ). \n",
      "Cells with consistent cell type annotations were combined with the CellRef Seed to form the LungMAP Human Lung CellRef (347,970 cells) (Fig. 2B [rid=Fig2] , Supplementary Figs. 1 [this file] – 3 [this file] , “Methods”). \n",
      "Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [this file] , Supplementary Data 3 [rid=MOESM5] , 4 [rid=MOESM6] ). \n",
      "These tools provide highly interactive search, analyzing, and visualization functionalities for users to explore and reanalyze cell type and gene expression patterns provided by the LungMAP CellRefs (Fig. 4 [rid=Fig4] , Supplementary Fig. 6 [this file] ). Users can also select and highlight cells of interest (Supplementary Fig. 6 [this file] ). \n",
      "Evaluation of the other three test data samples were shown in Supplementary Fig. 7 [this file] . \n",
      "As shown in Fig. 5D–I [rid=Fig5] and Supplementary Fig. 7 [this file] , predicted cell types were well separated and formed clusters. Cell-type-specific marker genes from CellCards were selectively expressed in each predicted cell type, supporting the concordance of the cell identities (Fig. 5E, F [rid=Fig5] , Supplementary Fig. 7 [this file] ). The median values of all metrics are greater than 0.92 (Supplementary Fig. 8 [this file] ), supporting the high consistency of the automated CellRef cell type prediction with the original cell annotations. Both the LungMAP human and mouse lung CellRefs achieved high cross validation performance with ~0.92 median F1 and MCC scores for the human lung CellRef and 0.98 median F1 and MCC scores for the mouse lung CellRef, respectively (Supplementary Fig. 9 [this file] ). \n",
      "We summarized the one-to-one mapped cell types and the unique cell types/states in the Supplementary Fig. 10 [this file] . \n",
      "Similarly, when applying these stability metrics to CellRef and HLCA using their own data collections, CellRef produced slightly higher scores on SCCAF and TF-IDF; as well as a significantly higher score on cell re-assignment ( = 0.011) as compared to HLCA (Supplementary Fig. 11 [this file] ). scTriangulate leverages the computed stability scores to further assess the marginal importance of each annotation (CellRef and HLCA) using a cooperative game theory framework. scTriangulate predicted that 50% ( = 24/48) of the CellRef cell types to be of higher confidence (AEC, AF1, AF2, AM, ASMC, CD8 T, cDC1, cDC2, Ciliated, Goblet, Ionocyte, LEC, Platelet, pMON, Suprabasal, SVEC, VSMC), compared to 46% ( = 27/58) of the HCLA cell types (AT2, aCAP, Pericyte, Mesothelium, Peribronchial fibroblasts and Adventitial fibroblasts, etc.) (Fig. 8D [rid=Fig8] ). \n",
      "The overall distributions of iLISI scores in both CellRef and HLCA are not ideal, on average about 15% for CellRef, and 5% for HLCA, respectively (Supplementary Fig. 11 [this file] ), likely due to that both atlases were constructed using large collections of heterogeneous data from different biological regions and conditions rather than a well-designed balanced cohort. Nevertheless, CellRef consistently outperforms HLCA on the measurements of iLISI, and has the cLISI scores close to 1, indicating a well separation of the CellRef cell types (Supplementary Fig. 11 [this file] ). \n",
      "Results supported the high accuracy, cluster stability, and marker specificity of our mouse lung reference cell types (Supplementary Fig. 12 [this file] ; mean AUC 95.8, mean SCCAF score: 0.970, mean Reassign score: 0.881, mean TFID10 score: 0.892). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: This collection contains data from similar numbers of female and male donors ( = 48 and 55, respectively; 1 unannotated) (Fig. 1A [rid=Fig1] ; Supplementary Data 1 [this file] ). In total, 505,256 lung cells from 148 sc/snRNA-seq of normal human lung samples from 104 donors were used for LungMAP human lung CellRef construction (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Using this guided approach and a cell type dictionary derived from LungMAP CellCards (Supplementary Data 2 [this file] ), we identified 8,080 seed cells representing 48 normal human lung cell types, termed LungMAP Human Lung CellRef Seed (Fig. 2A [rid=Fig2] ). \n",
      "Evaluation of cell type annotations using CellRef cell type markers from Supplementary Data 2 [this file] . Percentages of cell type markers (Supplementary Data 2 [this file] ) that are differentially expressed in their corresponding cell type predictions ( = 34 cell types) in ( ). \n",
      "The human lung CellRef cell type selective markers were generated by including dictionary marker genes (Supplementary Data 2 [this file] ) and top selectively expressed markers for each cell type. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [this file] , 4 [rid=MOESM6] ). \n",
      "In total, gene expression in 17 Drop-seq samples from eight time points (Supplementary Data 3 [this file] ) of mouse lung development were generated. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [rid=MOESM5] , 4 [this file] ). In total, 40 mouse lung cell types have been identified with the guidance of the mouse lung cell type dictionary (Fig. 3 [rid=Fig3] , Supplementary Data 4 [this file] ). \n",
      "We constructed a cell type dictionary for mouse lung (Supplementary Data 4 [this file] ) based on our constructed dictionary derived from the LungMAP CellCards. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: AUCs values for each of the mapped cell types ( = 42) in the CellRef (orange) and HLCA (blue) calculated using the cell type selective marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Next, we performed the same analysis using the cell type marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Top-ranked cell type specific DEGs (ranked by fold change in average expression) were combined with the known markers genes to form the CellRef cell type selective marker gene list (Supplementary Data 5 [this file] , up to 10 genes for each of the 48 CellRef human lung cell types). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: Last, we evaluated the LungMAP mouse lung CellRef using the three stability metrics calculated by scTriangulate and the marker-based AUC analysis (“Methods” and Supplementary Data 6 [this file] ). \n",
      "Using the same approach, the mouse lung CellRef cell type selective marker genes were selected (Supplementary Data 6 [this file] , up to 10 marker genes per cell type) and used for the AUC based assessment of mouse lung CellRef. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 247377\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 62596 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-2128c5890b033d4b2bc15929c82ea5968d3d2bec3659a8928b16bdabb94ff17d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 62596 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b25b1528-3a63-47b4-a5e2-1f1d2e747273\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n    \\\"repository_reference\\\": \\\"ega-archive.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135851\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE136831\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE136832\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE149563\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178360\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178362\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122332\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"LMEX0000004396\\\",\\n    \\\"repository_reference\\\": \\\"lungmap.net\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"LMEX0000004397\\\",\\n    \\\"repository_reference\\\": \\\"lungmap.net\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n    \\\"repository_reference\\\": \\\"ega-archive.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135851\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE136831\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE136832\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE149563\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178360\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178362\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122332\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"LMEX0000004396\\\",\\n    \\\"repository_reference\\\": \\\"lungmap.net\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"LMEX0000004397\\\",\\n    \\\"repository_reference\\\": \\\"lungmap.net\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321916,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 76576,\n",
      "        \"completion_tokens\": 564,\n",
      "        \"total_tokens\": 77140,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001004082', 'repository_reference': 'ega-archive.org', 'dataset_id': 'EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EGAS00001004082 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122960', 'repository_reference': 'geo', 'dataset_id': 'GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE122960, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE134174', 'repository_reference': 'geo', 'dataset_id': 'GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE134174, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135851', 'repository_reference': 'geo', 'dataset_id': 'GSE135851'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE135851, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135893', 'repository_reference': 'geo', 'dataset_id': 'GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE135893, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE136831', 'repository_reference': 'geo', 'dataset_id': 'GSE136831'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE136831, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE136832', 'repository_reference': 'geo', 'dataset_id': 'GSE136832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE136832, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE149563', 'repository_reference': 'geo', 'dataset_id': 'GSE149563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE149563, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE161382', 'repository_reference': 'geo', 'dataset_id': 'GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE161382, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE171524', 'repository_reference': 'geo', 'dataset_id': 'GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE171524, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE178360', 'repository_reference': 'geo', 'dataset_id': 'GSE178360'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE178360, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE178360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178360', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE178362', 'repository_reference': 'geo', 'dataset_id': 'GSE178362'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE178362, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122332', 'repository_reference': 'geo', 'dataset_id': 'GSE122332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE122332, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'repository_reference': 'synapse.org', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21041850, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004396', 'repository_reference': 'lungmap.net', 'dataset_id': 'LMEX0000004396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID LMEX0000004396 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004397', 'repository_reference': 'lungmap.net', 'dataset_id': 'LMEX0000004397'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID LMEX0000004397 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE178360', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 16 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EGAS00001004082 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: ega-archive.org and accession_id: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE122960 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE134174 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135851 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135851 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE135851 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE135893 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE136831 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE136832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE149563 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE149563 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE149563 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE161382 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE171524 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE178360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE178360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178360 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178360 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE178360 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE178360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE178362 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE122332 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for LMEX0000004396 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: lungmap.net and accession_id: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 15.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for LMEX0000004397 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: lungmap.net and accession_id: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 16.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 16\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 25, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 18th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4444046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6b8e310>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 11 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 51 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 9833) for SD1: We assembled a TCGA Pan-Cancer dataset of exome sequencing, array copy number, and RNA-seq data from 3,281 samples from 12 cancer types, analyzing single nucleotide variants (SNVs), small indels, and copy number aberrations (CNAs) in 19,424 transcripts ( Figure 1a [href=#F1] and Supplementary Figure 1 [href=this file] ). The resulting “star subnetworks” centered on the hot node ( Supplementary Figure 2 [href=this file] ; Online Methods) contain many neighboring genes that are not mutated at appreciable frequency and are of limited biological interest.\n",
      "We introduce the HotNet2 (HotNet iffusion riented ubnetworks) algorithm to address the problem of finding significantly mutated subnetworks on large, broad mutation frequency spectrum datasets like Pan-Cancer ( Figure 1c [href=#F1] and Supplementary Figure 3 [href=this file] ). HotNet2 uses a modified diffusion process and considers the source, or directionality, of heat flow in the identification of subnetworks ( Supplementary Figure 4 [href=this file] ).\n",
      "We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=#SD2] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=this file] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=#SD2] ). We developed an online interactive viewer (see URLs and Supplementary Figure 6 [href=this file] ) for Pan-Cancer HotNet2 subnetworks.\n",
      "The subnetworks and linker genes ( Figure 2a [href=#F2] ) include: portions of well-known cancer pathways such as TP53, PI3K, NOTCH, and receptor tyrosine kinases (RTKs; Supplementary Figure 7 [href=this file] ), as well as pathways and complexes that have more recently been observed to be important in cancer such as SWI/SNF complex, BAP1 complex, NFE2L2-KEAP1 ( Supplementary Figures 8,9 [href=this file] ), and RUNX1-CBFB core binding complex ( Supplementary Figure 10 [href=this file] ). The fifth most mutated subnetwork (16.9% of samples) consists of MLL2 and MLL3 and the putative interacting protein KDM6A ( Supplementary Figure 11 [href=this file] ), and was highly mutated (28.9% of samples) in TCGA Pan-Cancer squamous integrated subtype . The MHC Class I subnetwork ( Supplementary Figure 12 [href=this file] ) is an example of the ability of HotNet2 ability to identify rarely mutated cancer genes; all of the genes in the subnetwork are mutated in fewer than 35 samples (1.1%), yet four of the five genes have recently been proposed as novel cancer genes . Additional analyses are in the Supplementary Note [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=this file] and Supplementary Tables 21,22 [href=#SD2] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=#SD2] and Supplementary Note Section 5.1 [href=this file] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=this file] and Supplementary Table 23 [href=#SD2] ).\n",
      "This exclusivity was largely due to LUAD samples with mutually exclusive and mutations ( Supplementary Figure 15 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=#SD2] and Supplementary Figures 16,17 [href=this file] ).\n",
      "The subnetwork also includes , which has a significant ( < 8×10 ) cluster of missense mutations ( Supplementary Figure 18 [href=this file] ).\n",
      "We find that mutations in the SWI/SNF and BAP1 complexes show even greater mutual exclusivity ( =9.4×10 ) in KIRC because of mutations in additional genes in these complexes besides and , respectively ( Supplementary Note Section 5.8.1 [href=this file] ).\n",
      "Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=this file] and Supplementary Table 39 [href=#SD2] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=this file] and Supplementary Table 27 [href=#SD2] ).\n",
      "We selected the threshold of 400 aberrations per sample as the derivative of the number of mutations per sample starts increasing rapidly beyond this value ( Supplementary Figure 19 [href=this file] ). Finally, we used only those genes that had at least 3 reads from RNA-seq data in at least 70% of samples of at least one of the cancer types, as described in The resulting dataset contained aberrations in 11,565 genes and 3110 samples ( Supplementary Figure 1 [href=this file] ). Additional details of the datasets are in the Supplementary Note [href=this file] .\n",
      "While interaction networks have proven useful in analyzing various types of genomic data , statistically robust identification of significantly mutated subnetworks is a difficult problem with several major challenges ( Supplemental Note Section 1.1 [href=this file] ). HotNet2 addresses these challenges and identifies significantly mutated subnetworks of a genome-scale interaction network, using an insulated heat diffusion process that considers both the scores on individual genes/proteins as well as the topology of interactions between genes/proteins ( Supplementary Figure 3 [href=this file] ).\n",
      "The insulated heat model can also described in terms of a random walk with restart ( Supplementary Note Section 1.2 [href=this file] ).\n",
      "Since the diffusion matrix is not symmetric and in general ( ) ≠ ( ), the similarity ( , ) is also not symmetric ( Supplementary Note Section 1.2.1 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=this file] , Supplementary Figures 20–23 [href=this file] and Supplementary Table 28 [href=#SD2] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=this file] , Supplementary Figure 24 [href=this file] , and Supplementary Table 29 [href=#SD2] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=this file] , Supplementary Figure 25 [href=this file] , and Supplementary Table 30 [href=#SD2] ).\n",
      "See Supplementary Figure 2 [href=this file] .\n",
      "Expanding on this third point, when undirected diffusion algorithms like HotNet or related network propagation algorithms are run on large datasets containing a wide range of gene scores (e.g. the Pan-Cancer dataset), many of the resulting subnetworks are “hot” determined by a single high-scoring node and the immediate neighbors of this node ( Supplementary Figure 2 [href=this file] ).\n",
      "The HotNet2 subnetworks also have a higher fraction of interactions with proteins other than a hot central node ( Supplementary Note Section 7.1 [href=this file] ). In short, the results show that HotNet2 achieves higher sensitivity and specificity than HotNet ( Supplementary Note Section 7.2 [href=this file] and Supplementary Figure 26 [href=this file] ).\n",
      "We find that HotNet2 provides both new insights and a simpler summary of groups of interacting genes, and is a useful complement (or arguably a replacement for) other pathway tests ( Supplementary Note Section 8.1 [href=this file] ). We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=this file] , Supplementary Figure 27 [href=this file] , and Supplementary Tables 34–36 [href=#SD2] ). Finally, we find that HotNet2 was more stable than HotNet in identifying 20/20 genes using cross-validation ( Supplementary Note Section 7.3 [href=this file] and Supplementary Figure 28 [href=this file] ).\n",
      "We found that MEMo was unable to run on the Pan-Cancer mutation frequency dataset, consistent with the authors’ recommendation that MEMo should be run only on a small number of significant mutations (details in Supplementary Note Section 8.2 [href=this file] ).\n",
      "We ran HotNet2 on each combination of gene scores (mutation frequency and MutSigCV -values; see Supplementary Note Section 2.2 [href=this file] ) and interaction networks (HINT+HI2012 , iRefIndex , and Multinet ; Supplementary Note Section 2.5 [href=this file] and Supplementary Figure 29 [href=this file] ). This procedure is described in Supplementary Note Section 1.5 [href=this file] .\n",
      "Additional details of the statistical procedure are in Supplementary Note Section 1.3 [href=this file] .\n",
      "Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=#SD2] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=this file] and Supplementary Table 38 [href=#SD2] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 34 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 7301) for SD2: HotNet2 identified a significant number of subnetworks ( < 0.01 Supplementary Tables 1–2 [href=this file] ) for each of the two gene scores and three networks. We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=this file] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=#SD1] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=this file] ).\n",
      "Many of the subnetworks exhibit a significant enrichment for mutations in a subset of cancer types, including many previously unreported associations ( Supplementary Tables 6–18 [href=this file] ). In addition, the HotNet2 Pan-Cancer analysis provides a clearer and more robust summary of subnetworks and novel genes than HotNet2 analysis of individual cancer types ( Supplementary Table 19 [href=this file] ).\n",
      "These subnetworks and linkers include a total of 147 genes, including many well-known cancer genes and pathways, but also including genes with mutations that are too rare to be significant by the single-gene tests ( Supplementary Table 20 [href=this file] ). Table 1 [href=#T1] lists a subset of promising candidates, with the full list and associated references in Supplementary Table 20 [href=this file] .\n",
      "More information on these genes – as well as other candidate driver genes – is in Supplementary Table 20 [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=#SD1] and Supplementary Tables 21,22 [href=this file] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=this file] and Supplementary Note Section 5.1 [href=#SD1] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 23 [href=this file] ).\n",
      "Multiple pairs of genes within these subnetworks show co-occurring mutations ( Supplementary Table 24 [href=this file] ). We observe significant mutual exclusivity within 4 of our subnetworks ( Supplementary Table 25 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=this file] and Supplementary Figures 16,17 [href=#SD1] ).\n",
      "HotNet2 Pan-Cancer analyses also revealed a number of novel genes ( Supplementary Table 20 [href=this file] ) within these three subnetworks.\n",
      "The sixth most mutated HotNet2 Pan-Cancer subnetwork (16.8% of samples) includes multiple members of the SWI/SNF chromatin-remodeling complex ( Figure 3a [href=#F3] and Supplementary Table 12 [href=this file] ).\n",
      "Another HotNet2 Pan-Cancer subnetwork (mutated in 7.1% of samples) overlaps the BAP1 complex ( Figure 3b [href=#F3] and Supplementary Table 13 [href=this file] ).\n",
      "HotNet2 Pan-Cancer analysis identifies 4/5 members of the cohesin complex as a significantly mutated subnetwork (7.3% of samples, Figure 4a [href=#F4] and Supplementary Table 15 [href=this file] ).\n",
      "The combined subnetwork is mutated in 4.2% of samples ( Figure 4b [href=#F4] and Supplementary Table 6 [href=this file] ). Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 39 [href=this file] ).\n",
      "Some of these rarely mutated genes are inevitably false positive predictions of the analysis, but many (including , , , , , and others) exhibit either mutational clustering or protein interactions that support a role for the observed somatic aberrations ( Supplementary Tables 6–18 [href=this file] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=#SD1] and Supplementary Table 27 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=#SD1] , Supplementary Figures 20–23 [href=#SD1] and Supplementary Table 28 [href=this file] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=#SD1] , Supplementary Figure 24 [href=#SD1] , and Supplementary Table 29 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of β and found that varying β ±10% has only a minor effect on the results, with at most 7 genes (3.8% of total) added/removed from the subnetworks ( Supplementary Table 28 [href=this file] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=#SD1] , Supplementary Figure 25 [href=#SD1] , and Supplementary Table 30 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of δ, and found that varying δ ±5% changed at most 35 genes (12.3% of total) in the subnetworks ( Supplementary Table 29 [href=this file] ).\n",
      "We found that HotNet2 returns >80% fewer hot stars/spiders than HotNet on the Pan-Cancer datasets ( Supplementary Table 31 [href=this file] ). This is a major difference between the algorithms and is one of the reasons why HotNet fails to find statistically significant results ( ≤ 0.01 for any subnetwork size ) on three of six runs ( Supplementary Table 32,33 [href=this file] ), while HotNet2 finds statistically significant results on all six runs.\n",
      "We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=#SD1] , Supplementary Figure 27 [href=#SD1] , and Supplementary Tables 34–36 [href=this file] ).\n",
      "Most of the subnetworks (12/14) identified by HotNet2 were also found when we remove the requirement for RNA-Seq expression ( Supplementary Table 37 [href=this file] ). Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=this file] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=#SD1] and Supplementary Table 38 [href=this file] ). We found only minor differences in the HotNet2 subnetworks ( Supplementary Table 39 [href=this file] ), demonstrating that our reported subnetworks are altered by somatic aberrations in these samples.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 186016\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 47256 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cdd099228dd8e7d51cdee353db0dc797a5a7de9c5e2ac11ed8922f40892550ad\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 47256 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-640353b0-2334-49d7-9053-b46cfcc01171\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1710680\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1734155\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1710680\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1734155\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321934,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 63686,\n",
      "        \"completion_tokens\": 116,\n",
      "        \"total_tokens\": 63802,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1710680', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1710680'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1710680, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1710680', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1729383', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1729383'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1729383, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1734155', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1734155'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1734155, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1734155', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1710680', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1734155', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1710680 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1729383 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1734155 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 19th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10329279&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b636e490>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 1 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 162786\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 41449 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b5c1601bc48fa8e3573be4b0b5b8ae9a1bc7afc2eb74edeab521e6276cd0323a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 41449 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f48e78ea-80bf-4f27-ad7f-2139af2dae14\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"RA2 DREAM Challenge\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"RA2 DREAM Challenge\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321948,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 61479,\n",
      "        \"completion_tokens\": 39,\n",
      "        \"total_tokens\": 61518,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RA2 DREAM Challenge', 'repository_reference': 'n/a', 'dataset_id': 'RA2 DREAM Challenge'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RA2 DREAM Challenge\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID RA2 DREAM Challenge is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'RA2 DREAM Challenge', 'repository_reference': 'n/a', 'dataset_id': 'RA2 DREAM Challenge'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: []\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 1, with cols: Index(['link', 'title', 'file_info', 'description', 'source_section',\n",
      "       'section_class', 'download_link', 'file_extension', 'a_attr_href',\n",
      "       'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 1\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 20th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11145390&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b730d800>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b730d800>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 149110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 38030 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cb67bec01d870746bf5901cf996838fb3f121083a33cb7eb309d2393f0827b7e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 38030 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-546c340f-8db3-457b-998f-ab33772db504\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21788425\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/skannan4/cm-entropy-score#readme\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.r-project.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.rstudio.com/products/rstudio/download/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://CRAN.R-project.org/package=ggplot2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=reshape2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=Matrix\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=grid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=stringr\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=dplyr\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://pcahan1.github.io/singleCellNet/\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.10971926\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21788425\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/skannan4/cm-entropy-score#readme\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.r-project.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.rstudio.com/products/rstudio/download/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://CRAN.R-project.org/package=ggplot2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=reshape2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=Matrix\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=grid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=stringr\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/package=dplyr\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://pcahan1.github.io/singleCellNet/\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.10971926\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321952,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 46966,\n",
      "        \"completion_tokens\": 574,\n",
      "        \"total_tokens\": 47540,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE147807', 'repository_reference': 'geo', 'dataset_id': 'GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE147807, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21788425', 'repository_reference': 'synapse.org', 'dataset_id': 'syn21788425'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21788425, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21788425', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/skannan4/cm-entropy-score#readme', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/skannan4/cm-entropy-score#readme, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/skannan4/cm-entropy-score#readme\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/skannan4/cm-entropy-score#readme\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21788425', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE147807 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21788425 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 21/100 (21.00%) | Elapsed: 00:03:00 | ETA: 00:11:20\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 21th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11032436&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6791780>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6791780>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 137085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35023 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-7d51b5d294bf4baa311680946081018e25258ddb5ffdac2061e9225ebe0a84a4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35023 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c9022ffd-c013-4af1-b4be-b94f1dc606a9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.atcc.org/products/crl-3216\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://stemcells.nindsgenetics.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://zzz.bwh.harvard.edu/plink/ld.shtml\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/web/packages/susieR/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//#!Synapse:syn2759792/wiki/69613\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.targetscan.org/vert_61/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://microrna-atlas.omics.kitchen/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://hdsa.org/wp-content/uploads/2022/12/Novartis-VIBRANT-HD-Community-Letter-FINAL-PDF.pdf\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.atcc.org/products/crl-3216\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://stemcells.nindsgenetics.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://zzz.bwh.harvard.edu/plink/ld.shtml\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cran.r-project.org/web/packages/susieR/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//#!Synapse:syn2759792/wiki/69613\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.targetscan.org/vert_61/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://microrna-atlas.omics.kitchen/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://hdsa.org/wp-content/uploads/2022/12/Novartis-VIBRANT-HD-Community-Letter-FINAL-PDF.pdf\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321954,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 43754,\n",
      "        \"completion_tokens\": 489,\n",
      "        \"total_tokens\": 44243,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org//', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org//, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org//\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org// in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org//', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: []\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 1, with cols: Index(['link', 'source_url', 'download_link', 'title', 'content_type', 'id',\n",
      "       'caption', 'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 22th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9984184&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b631ffd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 17 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec33'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec33: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#supplementary-material1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for supplementary-material1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec40'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec40: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 231605\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 58653 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-972e3b95746b5a7b3dddbc09eb760e3dde7a59c0747febb2dde7280e430efe9f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 58653 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-28716bc7-1037-4680-ba4f-bfc346ce9227\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE147205\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE169185\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25576878\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn39456106\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"E-GEOD-44770\\\",\\n   \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"GSE95587\\\",\\n   \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"GSE15222\\\",\\n   \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE147205\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE169185\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25576878\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn39456106\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"E-GEOD-44770\\\",\\n   \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"GSE95587\\\",\\n   \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"GSE15222\\\",\\n   \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321970,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 78244,\n",
      "        \"completion_tokens\": 314,\n",
      "        \"total_tokens\": 78558,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE147205', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205', 'dataset_id': 'GSE147205'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE147205, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147205', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE169185', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185', 'dataset_id': 'GSE169185'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE169185, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE169185', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25576878', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn25576878'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25576878, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25576878', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn39456106', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn39456106'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn39456106, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn39456106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-GEOD-44770', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-GEOD-44770'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-GEOD-44770 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-GEOD-44770', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE95587', 'repository_reference': 'geo', 'dataset_id': 'GSE95587'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE95587, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE95587', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE15222', 'repository_reference': 'geo', 'dataset_id': 'GSE15222'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE15222, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE15222', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE147205', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE169185', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25576878', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn39456106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-GEOD-44770', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE95587', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE15222', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 7 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147205 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147205 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE147205 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE169185 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE169185 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE169185 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25576878 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn39456106 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-GEOD-44770 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: arrayexpress and accession_id: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE95587 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE95587 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE95587 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE15222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE15222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE15222 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 7\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 24, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 9\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 23th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7347884&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b756ff80>, <Element supplementary-material at 0x1b0a5ef00>, <Element supplementary-material at 0x1b0ff5780>, <Element supplementary-material at 0x1b1007e40>, <Element supplementary-material at 0x1b1005f40>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b756ff80>, <Element supplementary-material at 0x1b0a5ef00>, <Element supplementary-material at 0x1b0ff5780>, <Element supplementary-material at 0x1b1007e40>, <Element supplementary-material at 0x1b1005f40>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: As gene expression analyses do not allow to accurately discriminate mesenchymal cancer cells from normal stromal cells, we used the allele-specific copy number analysis of tumors (ASCAT) copy number-based tumor purity estimation method to assess tumor cell fraction (Supplementary Fig. 1a [this file] ). To avoid any substantial bias due to non-tumor cell contamination, a stringent purity threshold was determined by applying Wilcoxon test, for which the degree of contamination of claudin-low tumors was not statistically different from that of non-claudin-low tumors, thus enabling a comparable distribution of tumors (Supplementary Fig. 1b [this file] ). Demonstrating an unexpected degree of diversity, only 35.6% of these tumors were classified as TNBCs (Supplementary Fig. 1d [this file] ). This result was not due to the purity selection, as the percentage of TNBCs within all claudin-low tumors was similar before applying the purity threshold (Supplementary Fig. 1c [this file] ). \n",
      "Only 17.8% were found in cluster 10, whereas 48.8% and 17.8% were found in integrative clusters 4 and 3, respectively, regardless of tumor purity when compared with non-claudin-low tumors (Supplementary Fig. 2 [this file] ). After having previously removed germline copy number variations (CNVs), we analyzed B allele frequency (BAF) and log R ratio (LRR) profiles of single-nucleotide polymorphisms (SNPs) localized in genomic regions of deletion (LRR segment mean cutoff value < −0.4) (Fig. 1b [rid=Fig1] and Supplementary Fig. 3a, c [this file] ). Consistent with these data, the analysis of somatic mutations highlighted the presence of several point mutations with a variant allele frequency higher than 0.4 (Fig. 1c [rid=Fig1] and Supplementary Fig. 3b [this file] ). \n",
      "Of note, although these analyses were performed on tumors selected for their high tumor purity, similar data were obtained when studying the whole cohort of claudin-low tumors (Supplementary Fig. 4 [this file] ), further strengthening the characterization of the three claudin-low subgroups. \n",
      "Of note, global gene expression signatures demonstrated that the CL2 and CL3 subgroups showed lower luminal- and basal-related signature scores, respectively, compared with luminal- and basal-like breast tumors, while exhibiting a significant enrichment in invasiveness-related signatures (Supplementary Fig. 5a, b [this file] ). \n",
      "A claudin-low gene list of 137 genes discriminating the 3 claudin-low subgroups was thus created (Supplementary Fig. 6a, b [this file] ). This expression-based classifier showed an accuracy of 91% when compared with the FGA-based classification using Gaussian finite mixture models (Supplementary Fig. 5c, d [this file] ). Validating the accuracy of our classifier, CL1, CL2, and CL3 tumors and cell lines showed low, moderate, and elevated FGA levels, respectively (Supplementary Fig. 6e, f [this file] ). Of note, we performed a pathway analysis of CL1 discriminant genes that revealed an enrichment in stem cell and pediatric cancer markers (Supplementary Fig. 6g [this file] ). \n",
      "Differentially methylated gene analysis was next performed in claudin-low subgroups followed by pathway-enrichment analysis, using over 15,000 pathways available in MSigDB (HALLMARKS, C2, and C5 gene sets) (Supplementary Fig. 7 [this file] ). The CL1 subgroup had the highest score for stemness- ( , ) and EMT- (mesenchymal markers: , , ; epithelial markers: , ) related genes, whereas CL2 and CL3 displayed an intermediate score compared with CL1 tumors and with their relative luminal/basal counterparts (Supplementary Fig. 8 [this file] ). \n",
      "CL1 tumors mainly display hypomethylated genes that are related to a stemness phenotype, CL2 tumors are overall hypermethylated for genes associated with basal-like features, and CL3 tumors show more heterogeneous methylation profiles with both hypermethylated genes (linked to luminal phenotype) and hypomethylated genes (relative to basal-like characteristics) (see Supplementary Fig. 5 [this file] for methodology). \n",
      "Additional gene expression analyses performed on METABRIC and TCGA cohorts using two different cellular signature deconvolution algorithms confirmed that these tumors were highly infiltrated by immune cells (Supplementary Fig. 9a–d [this file] ). Nevertheless, unsupervised clustering of microenvironment signatures (immune and non-immune cell subsets) did not allow to discriminate the different claudin-low subgroups (Supplementary Fig. 9e–h [this file] ). \n",
      "Supporting our previous results, CL2 and CL3 showed, among the three claudin-low subgroups, the highest score for luminal- (estrogen-related response) and basal-related (proliferation and DDR) signatures, respectively (Supplementary Fig. 10 [this file] ). Furthermore, both CL2 and CL3 showed an activation of the EMT process (Fig. 6a, b, g [rid=Fig6] ), coherent with the downregulation of hormone (in CL2) and proliferation/DDR (in CL3) signatures, compared with luminal and basal tumors, respectively (Supplementary Fig. 10 [this file] and Fig. 6g [rid=Fig6] ). Moreover, CL1 displayed a low proliferation activity and a low DDR (Supplementary Fig. 10a, b, e, f [this file] ), as well as a strong TP53 signaling pathway signature associated with a low frequency of mutations (Fig. 6c, d, g [rid=Fig6] ). \n",
      "When stratified by CL subgroups, variations in survival outcome were found, consistent with their presumed origin (Supplementary Fig. 11 [this file] ). \n",
      "These traits include (i) a high expression of the ZEB1 EMT-inducing transcription factor and of its target, the methionine sulfoxide reductase MSRB3; (ii) a frequent activation of the RAS-MAPK signaling pathway; (iii) a low activation of the DDR; and (iv) a low frequency of mutations (Supplementary Figs. 8 [this file] and 10 [this file] , and Fig. 6 [rid=Fig6] ). \n",
      "The use of a high-purity threshold for tumor selection allowed to circumvent the contamination bias related to the tumor-related centroid (Supplementary Fig. 1 [this file] ). \n",
      "Finally, the nearest shrunken centroid classifier encompassed 137 genes whose expression discriminate the three FGA-related claudin-low subgroups in METABRIC cohort (Supplementary Fig. 6 [this file] ). \n",
      "As previously described for genome-wide expression–methylation quantitative trait loci analysis , genes for which methylation -values were negatively correlated with expression values (Spearman’s < 0 and FDR -value < 0.05) were selected for differential methylation analysis (Supplementary Fig. 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Gene expression signatures for MaSCs, LPs, and mL cells were generated based on the transcription profiles of subpopulations of normal human mammary epithelial cells isolated from reduction mammoplasty tissues, as shown in Morel et al. (Supplementary Data 1 [this file] ). \n",
      "The top 500 false discovery rate (FDR) -value ranked genes differentially expressed between one subpopulation (MaSC or LP, or mL cells) vs. the two others (FDR -value < 0.05) were used as transcriptomic signature for each of the three cell subpopulations (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 149516\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 38131 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-9ef5a7287dd54f4ab3bf87c0a0a1eb7d6ed8299ed6f3abc808ff4752965cd64d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 38131 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5d4201ff-71f5-462b-aa97-a792e1e8f003\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n    \\\"repository_reference\\\": \\\"ega-archive.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE56031\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-4145\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1757053\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n    \\\"repository_reference\\\": \\\"ega-archive.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE56031\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-4145\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1757053\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321972,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44924,\n",
      "        \"completion_tokens\": 154,\n",
      "        \"total_tokens\": 45078,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00000000083', 'repository_reference': 'ega-archive.org', 'dataset_id': 'EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EGAS00000000083 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE56031', 'repository_reference': 'geo', 'dataset_id': 'GSE56031'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE56031, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-4145', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-4145'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-4145 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1757053', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1757053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1757053, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'EGAS00000000083', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EGAS00000000083 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: ega-archive.org and accession_id: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE56031 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-4145 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1757053 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 24th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10802452&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b09b5a00>, <Element supplementary-material at 0x1b1caddc0>, <Element supplementary-material at 0x1b0b81840>, <Element supplementary-material at 0x1b66d13c0>, <Element supplementary-material at 0x1b1c62080>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b09b5a00>, <Element supplementary-material at 0x1b1caddc0>, <Element supplementary-material at 0x1b0b81840>, <Element supplementary-material at 0x1b66d13c0>, <Element supplementary-material at 0x1b1c62080>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: This cohort (n=104) comprised neurotypical (healthy) controls (HC, n = 38), and individuals with SCZ (ISCZ, n=38), BD (n = 20) and MDD (n = 8) ( Supplementary Table 1 [this file] ) with a mean age of onset in the late 20s adolescence (mean 28.5/27.1/29.1 SCZ/BD/MDD). \n",
      "From the latter, we also included a cohort of Amish individuals suffering from BD/MDD and respective controls, see Supplementary Table 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Importantly, 55.4% of the identified eQTLs overlapped with eQTLs identified in adult human tissue from the prefrontal cortex (PFC) , supporting their relevance ( Fig. 1f [rid=F1] and Supplementary Table 2 [this file] ). Nevertheless, 21.7% of the identified open chromatin peaks associated with caQTLs overlapped with eQTLs detected in bulk PFC. ( Fig. 1g [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "Interestingly, the overlap of QTLs across molecular features in iNs was limited, with 24.9% of eQTL overlapping with caQTLs ( Fig. 1l [rid=F1] and Supplementary Table 2 [this file] ). Moreover, 28 genes, 78 peaks and no microRNA with e/ca/miQTLs overlapped with GWAS risk loci for SCZ ( Fig. 1f [rid=F1] - h [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: Following a detailed power evaluation ( Extended Data Fig. 3b [rid=F9] - d [rid=F9] ), we performed differential expression analysis and identified 467 transcripts (365 protein coding and 102 non-coding transcripts) (FDR≤0.01, ∣log FC∣≥0.4; Fig. 2a [rid=F2] and Extended Data Fig. 3e [rid=F9] , and Supplementary Table 3 [this file] ), henceforth referred to as DEGs as well as 67 micro RNAs (FDR≤0.05; Fig. 2b [rid=F2] and Supplementary Table 3 [this file] ). \n",
      "This analysis revealed widespread differences in APA, identifying 628 differential APA (dAPA) events ( Fig. 3c [rid=F3] and Supplementary Table 3 [this file] ) that resulted from an increased usage of either the distal or the proximal PAS to similar degree (14.29% of all genes with more than 1 polyadenylation site). \n",
      "Consistent with the observations in iNs, this analysis confirmed widespread dAPA in postmortem PFC from ISCZ ( Fig. 3e [rid=F3] and Supplementary Table 3 [this file] ) as well as substantial overlap with dAPA in iNs ( Fig. 3f [rid=F3] ). \n",
      "Strikingly, genes subject to dAPA in ISCZ were strongly enriched for synapse related processes such as glutamatergic neurotransmission, synaptic membrane components, synaptic plasticity or vesicle biology in both iNs and human PFC ( Fig. 4a [rid=F4] , b [rid=F4] , and Supplementary Table 3 [this file] ). \n",
      "RBP motif analysis in the UTRs of transcripts subject to dAPA identified numerous expressed RBPs with critical roles for neuronal biology such as QKI, ELAVL and RBM proteins ( Fig. 5c [rid=F5] and Supplementary Table 3 [this file] ). \n",
      "Similarly, we also corrected the R values for gender, batch and TechnicalPC1 using beta regression and used the residuals to compute the average difference in R values between the groups (ΔAPA, Supplementary Table 3 [this file] ). \n",
      "Resulting enrichment scores and p-values for the set of transcripts with increased usage of the distal PAS are depicted in Fig. 5c [rid=F5] with p-values corrected for multiple testing using the q-value method and listed in Supplementary Table 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: MAP2, RTF2 or Distal/Proximal UTR, Supplementary Table 4 [this file] ) were used to calculate target specific relative expression as 2^(-(CT -CT )) giving rise to normalized transcript levels. \n",
      "Statistical testing of NRIs ( Supplementary Table 4 [this file] ) was conducted using by linear mixed models as implemented in the lme4 R package modeling diagnosis or treatment as fixed effect and batch as well as donor (for PSD95 analysis) as random effect. \n",
      "All well level density values used can be found in Supplementary Table 4 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 267302\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 67578 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0511effdf3d94503c351ff87932e6c4268d0f4b5edc485677f4b880ed196aec5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 67578 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f83f53c2-5c79-44e8-922a-4f3e257b5f17\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Data 1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Data 2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 3\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 5\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 4\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TBA\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TBA\\\",\\n    \\\"repository_reference\\\": \\\"European genome phenome archive\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Data 1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Data 2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 2\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 3\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 5\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplementary Table 4\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TBA\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TBA\\\",\\n    \\\"repository_reference\\\": \\\"European genome phenome archive\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321974,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 78989,\n",
      "        \"completion_tokens\": 281,\n",
      "        \"total_tokens\": 79270,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Table 1', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Table 1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Table 1 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Table 1', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Data 1', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Data 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Data 1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Data 1 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Data 1', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Data 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Data 2', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Data 2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Data 2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Data 2 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Data 2', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Data 2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Table 2', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Table 2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Table 2 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Table 2', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Table 3', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Table 3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Table 3 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Table 3', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Table 5', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Table 5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Table 5 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Table 5', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Supplementary Table 4', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 4'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Supplementary Table 4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Supplementary Table 4 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Supplementary Table 4', 'repository_reference': 'n/a', 'dataset_id': 'Supplementary Table 4'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TBA', 'repository_reference': 'GEO', 'dataset_id': 'TBA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TBA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GEO, identifier: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TBA for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TBA for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TBA for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TBA for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TBA in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier TBA does not match pattern for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 25th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8907344&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0986e90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 12 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec18: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec18: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 31 <a> elements with href='#MOESM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 2279) for MOESM1: S1 [href=this file] a,b).\n",
      "S2 [href=this file] shows the GO classifications of the acetylated proteins enriched in changed lists after SIRT2 knockdown with Supplementary Fig. S2 [href=this file] a indicating proteins with a decrease in acetylation and Supplementary Fig. S2 [href=this file] b listing those with an increase in acetylation. S3 [href=this file] shows the GO classification of the acetylated proteins after SIRT2 overexpression, with Supplementary Fig. S3 [href=this file] a indicating proteins with a decrease in acetylation and Supplementary Fig. S3 [href=this file] b listing those with an increase in acetylation.\n",
      "S4 [href=this file] a shows the correlation between up-regulated acetyl peptides in SIRT2 knockdown and down-regulated acetyl peptides in SIRT2 overexpression with identical corresponding peptides. S4 [href=this file] b shows the overlap between down-regulated acetyl peptides in SIRT2 knockdown and up-regulated acetyl peptides in SIRT2 overexpression. S4 [href=this file] a), suggesting that these acetylation sites are direct SIRT2 deacetylation targets. S4 [href=this file] b), suggesting that their acetylation may be indirectly dependent on regulation downstream of SIRT2. S2 [href=this file] d), providing validation for BCL9 as a novel SIRT2 substrate.\n",
      "S4 [href=this file] a shown in Supplementary Fig. S5 [href=this file] a. S5 [href=this file] a and Fig. 3 [href=#Fig3] a, red bars), with both comprising terms for actomyosin, actin filament bundles, histone acetyltransferase complex, and protein-DNA complex. S5 [href=this file] a and Fig. 3 [href=#Fig3] a, green bars). S5 [href=this file] a, blue bars). S5 [href=this file] b).\n",
      "S6 [href=this file] ).\n",
      "S7 [href=this file] ).\n",
      "S8 [href=this file] a). S8 [href=this file] b).\n",
      "S8 [href=this file] a). S9 [href=this file] a). S9 [href=this file] b). S8 [href=this file] a), suggesting that proteins important in tumorigenesis are key SIRT2 targets. S5 [href=this file] b), which discovered a significant enrichment of tumor-related GO terms. S9 [href=this file] c, an interaction network with these proteins revealed ubiquitin and p300 as two central hubs, with PARP1 and CBP being secondary hubs.\n",
      "S10 [href=this file] a), and protein-level FPR (Supplementary Fig. S10 [href=this file] b).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 199) for MOESM2: Using a false discovery rate (FDR) of < 1%, we identified a total of 2846 unique lysine acetylation sites from 1414 protein isoforms of 1404 gene products (Supplementary Table S1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#MOESM3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for MOESM3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#MOESM4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for MOESM4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#MOESM5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 471) for MOESM5: To identify how many of the direct SIRT2 targets are validated known SIRT2 substrates, we conducted an extensive literature review to curate experimentally validated SIRT2 substrates (Supplementary Table S4 [href=this file] ). More importantly, we found that 13 proteins are among our high-confidence direct SIRT2 target list and the known SIRT2 substrates, accounting for 23% of the total known list (Fig. 2 [href=#Fig2] c and Supplementary Table S4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 506) for MOESM6: To determine the overall specificity of the enrichment PSMs for acetylated lysine-containing peptides/total peptide PSMs for each of the four samples that underwent Ac-Lys enrichment with the Cell Signaling antibody prior to LC–MS/MS, we also conducted enrichment calculations from the alternate Proteome Discoverer (Thermo Scientific) software (PD) search for up to 3 Acetyl [K] per peptide, fully tryptic and 1% FDR enforced (high confidence peptide PSMs) (see Supplementary Table S5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 148722\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 37933 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0bc5e0a2613bd3541c035904b75767cfb4a19998b225d0a519c283a913e296df\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 37933 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-fca91efe-2461-4f18-b069-e7d66425f391\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26134616\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26134616\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi.org/10.7303/syn26134616\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26134616\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26134616\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi.org/10.7303/syn26134616\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321988,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51142,\n",
      "        \"completion_tokens\": 111,\n",
      "        \"total_tokens\": 51253,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn26134616', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26134616', 'dataset_id': 'syn26134616'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn26134616, synapse.org, https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'doi.org/10.7303/syn26134616', 'repository_reference': 'doi.org', 'dataset_id': 'doi.org/10.7303/syn26134616'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: doi.org/10.7303/syn26134616, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: doi.org/10.7303/syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'doi.org/10.7303/syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'doi.org/10.7303/syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn26134616 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: doi.org/10.7303/syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: doi.org/10.7303/syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier doi.org/10.7303/syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier doi.org/10.7303/syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier doi.org/10.7303/syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier doi.org/10.7303/syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for doi.org/10.7303/syn26134616 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier doi.org/10.7303/syn26134616 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: doi.org/10.7303/syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:doi.org/10.7303/syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 9\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 26th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10238095&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b636e480>, <Element supplementary-material at 0x1b1af56c0>, <Element supplementary-material at 0x1b64b9b00>, <Element supplementary-material at 0x1b64f5f80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b636e480>, <Element supplementary-material at 0x1b1af56c0>, <Element supplementary-material at 0x1b64b9b00>, <Element supplementary-material at 0x1b64f5f80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref supp1: According to the above biological framework, a total of 61 TME-related signatures were collected to form the TME panel ( Supplementary file 1a [this file] , Supplementary file 1b [this file] ), which ultimately established a holistic approach to thoroughly dissect the TME of CRC. \n",
      "Principal component analysis (PCA) indicated that after using the ‘ComBat’ function to remove batch effects, there was no significant batch effect in the merged cohorts of eight microarray datasets ( Figure 1—figure supplement 1A, B [rid=fig1s1] ) and two RNA sequencing datasets ( Figure 1—figure supplement 1C, D [rid=fig1s1] , Supplementary file 1c [this file] ). \n",
      "The stromal and tumor components significantly were correlated with decreased survival, particularly in the case of mesenchymal cells, endothelial cells, metastasis, differentiation, and EMT signatures ( Figure 1—figure supplement 1L [rid=fig1s1] , Supplementary file 1d [this file] ). \n",
      "Differences in the TME components between the CCCRC subtypes were also observed in the analysis of previously reported immune and stromal signatures obtained by the microenvironment cell populations (MCP)-counter, CIBERSORT, and ESTIMATE algorithm, and 10 classical oncogenic pathway activities and 86 metabolic pathway enrichment scores calculated by GSVA ( Figure 1—figure supplement 3A–E [rid=fig1s3] , Supplementary file 1e [this file] ). \n",
      "We analyzed the 86 metabolic pathways obtained from the Kyoto Encyclopedia of Genes and Genomes (KEGG) database ( Supplementary file 1e [this file] ) and observed that the number of upregulated metabolic pathways of the C3 subtype was the lowest. \n",
      "We further analyzed the association of CCCRC subtypes with clinicopathological characteristics ( Supplementary file 1f [this file] , Supplementary file 1g [this file] ). Multivariate Cox proportional hazard regression analysis showed that the C4 subtype was an independent predictor of the best OS and DFS, whereas the C3 subtype was an independent predictor of the worst OS and DFS after adjustment for age, gender, tumor site, TNM stage, grade, adjuvant chemotherapy or not, MSI status, BRAF and KRAS mutations, and the CMS classification system in the combined cohort (the CRC-AFFY and CRC-RNAseq cohorts) ( Supplementary file 1h [this file] ). \n",
      "The 254 CRC samples in the TCGA-CRC dataset were categorized as these three phenotypes based on the abundance of lymphocytes and their spatial location with malignant epithelial cells ( Supplementary file 1i [this file] ). \n",
      "A total of 2196 samples were obtained from ten publicly available datasets ( Supplementary file 1c [this file] ). Detailed information on the sample size and the corresponding clinicopathological data of the CRC-AFFY and CRC-RNAseq cohorts are summarized in Supplementary file 1c [this file] and Supplementary file 1f [this file] . \n",
      "The 4525 genes from each of the 61 TME-related signatures are listed in Supplementary file 1a [this file] , as well as the source of each signature. \n",
      "A total of 254 WSIs were included after removing the WSIs with poor quality and without views of the IM ( Supplementary file 1i [this file] ). \n",
      "The markers for the six cell types are listed in the Supplementary file 1a [this file] and Supplementary file 3a [rid=supp3] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref supp2: Among the frequently mutated genes (>5%), the mutation frequencies of (85.8%), (64.9%), and (46.7%) were the highest in the C1 subtype compared to the other subtypes (all p-value <0.05), followed by the C3, C2, and C4 subtypes, which are closely associated with the occurrence of CRC ( Figure 3A [rid=fig3] , Supplementary file 2a [this file] ). In terms of the differences in SCNA, the C1 subtype with the highest CIN level harbored significantly more amplified chromosomal regions (20q12, 20q13.12, 20q11.21, and 20q13.32) and deleted chromosomal regions (18q21.2, 18q22.1, and 18q12.3) (all p-value <0.05) ( Figure 3A, B [rid=fig3] , Supplementary file 2b [this file] ). The C4 subtype had the highest frequency of mutations in the cell cycle, HIPPO, MYC, NOTCH, PI3K, TGFB, and RTK–RAS pathways (all p-value <0.05) ( Figure 3C [rid=fig3] , Supplementary file 2c [this file] ). \n",
      "The lists of differentially expressed genes (DEGs) between CCCRC subtypes were compiled in Supplementary file 2d [this file] . In addition, we also analyzed the correlations between all genes and the exclusion and dysfunction scores obtained from the Tumor Immune Dysfunction and Exclusion (TIDE) website, as well as the MSI expression signature, which provided a useful reference for subsequent research in the discovery of new therapeutic targets ( Supplementary file 2e [this file] ). \n",
      "Furthermore, we intersected all the positives for increasing gene expression from C1 to C4, C2, and C3 subtypes in the CRC-AFFY and CRC-RNAseq cohorts and obtained 20 CCCRC genes ( Figure 4E [rid=fig4] , Supplementary file 2f [this file] ), which were associated with TGF-beta signaling and infiltrating nerves. \n",
      "We obtained the GEP for 7766T cells from 12 patients with CRC, including four patients with the C1 subtype, one patient with the C2 subtype, two patients with the C3 subtype, and four patients with the C4 subtype ( Supplementary file 2g [this file] ). The markers of the T cell clusters were summarized in Supplementary file 2h [this file] . \n",
      "For instance, KDM1A and RAD21 are specific genes for C1 subtype, and it has been reported that interfering with them could activate interferon signaling and induce tumor intrinsic immunogenicity ( Deng et al., 2022 [rid=bib26] ; Nguyen et al., 2022 [rid=bib65] ; Supplementary file 2d [this file] ). In this study, the expression levels of SPP1 and SERPINB9 were found to be significantly positively correlated with the exclusion and dysfunction scores (TIDE score), respectively ( Supplementary file 2e [this file] ). \n",
      "A total of 12 CRC samples with bulk RNAseq and scRNAseq data were obtained from the GSE108989 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989) dataset ( Zhang et al., 2018 [rid=bib96] ; Supplementary file 2g [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref supp3: To further explore the treatment strategies of the CCCRC subtypes, we trained a pre-clinical model based on a filtered gene set comprised of 335 CCCRC subtype-specific and cancer cell-intrinsic gene markers ( Supplementary file 3a [this file] ). The 71 human CRC cell lines were classified into four CCCRC subtypes ( Supplementary file 3b [this file] ). \n",
      "We established the CCCRC classifier on the training set by utilizing multiple machine learning algorithms based on the GEP of 80 upregulated subtype-specific genes (Materials and methods, Supplementary file 3c [this file] ). Notably, the CCCRC classifier based on the xgboost algorithm displayed robust performance across gene expression platforms, Affymetrix and RNA-sequencing platforms, exhibiting a balanced accuracy of >80% for all subtypes ( Supplementary file 3d [this file] ). \n",
      "The 1660 genes assigned to 86 human metabolism-related pathways are listed in Supplementary file 3e [this file] . Ten oncogenic signatures containing 331 genes and the terminally exhausted T cell signature were retrieved from a previously published study ( Beltra et al., 2020 [rid=bib11] ; Sanchez-Vega et al., 2018 [rid=bib73] ; Supplementary file 3e [this file] ). \n",
      "The R package ‘RTN’ was used to reconstruct the transcriptional regulatory networks of regulons ( Robertson et al., 2017 [rid=bib69] ), including 31 transcription factors and 82 chromatin remodeling genes, that were associated with CRC ( Vymetalkova et al., 2019 [rid=bib85] ; Xu et al., 2021 [rid=bib91] ; Supplementary file 3f [this file] ). \n",
      "To classify CRC samples into different CRC subtypes according to the previously published gene classifier, gene lists for the four classifiers were extracted from relevant publications and summarized ( Supplementary file 3g [this file] ). \n",
      "Detailed information on the sample size and the corresponding treatment data of the therapy-associated datasets are summarized in Supplementary file 3h [this file] . \n",
      "The markers for the six cell types are listed in the Supplementary file 1a [rid=supp1] and Supplementary file 3a [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mdar: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 553018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 139007 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ec9d5ecec8844d02c4dd824330d5264a7f2654d9121e9f7928721989123c2f66\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 139007 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-de6d6f75-1430-455c-99db-c9a952e8d3f5\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13067\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE14333\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE17536\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33113\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE37892\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38832\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiangkunWu/PAMR_classifier\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103479\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE104645\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.genome.jp/kegg/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.biolegend.com/en-us/cell-markers\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://depmap.org/portal/download/all/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.cbioportal.org/study/summary?id=ccle_broad_2019\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://qupath.github.io/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://tide.dfci.harvard.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://rdrr.io/github/lasseignelab/CINmetrics/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.genepattern.org/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE35144\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35144\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE108989\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB23709\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78220\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78220\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135222\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135222\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiangkunWu/CCCRC_classifier\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13067\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE14333\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE17536\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33113\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE37892\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38832\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiangkunWu/PAMR_classifier\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103479\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE104645\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.genome.jp/kegg/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.biolegend.com/en-us/cell-markers\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://depmap.org/portal/download/all/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.cbioportal.org/study/summary?id=ccle_broad_2019\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://qupath.github.io/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://tide.dfci.harvard.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://rdrr.io/github/lasseignelab/CINmetrics/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.genepattern.org/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE35144\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35144\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE108989\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB23709\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78220\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78220\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE135222\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135222\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiangkunWu/CCCRC_classifier\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756321996,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 150769,\n",
      "        \"completion_tokens\": 1696,\n",
      "        \"total_tokens\": 152465,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582', 'dataset_id': 'GSE39582'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE39582, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE13067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE13067'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE13067, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE13294'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE13294, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE14333'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE14333, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE17536', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE17536'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE17536, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE17536', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE33113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE33113, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE37892', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE37892'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE37892, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE37892', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE38832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE38832, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://portal.gdc.cancer.gov/', 'repository_reference': 'n/a', 'dataset_id': 'https://portal.gdc.cancer.gov/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://portal.gdc.cancer.gov/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://portal.gdc.cancer.gov/', 'repository_reference': 'n/a', 'dataset_id': 'https://portal.gdc.cancer.gov/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/XiangkunWu/PAMR_classifier', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiangkunWu/PAMR_classifier'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/XiangkunWu/PAMR_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/XiangkunWu/PAMR_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/XiangkunWu/PAMR_classifier, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/XiangkunWu/PAMR_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/XiangkunWu/PAMR_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/XiangkunWu/PAMR_classifier is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/XiangkunWu/PAMR_classifier', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiangkunWu/PAMR_classifier'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE103479', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE103479'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE103479, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE103479', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE104645', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE104645'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE104645, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE104645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE104645', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.genome.jp/kegg/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.genome.jp/kegg/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.genome.jp/kegg/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.genome.jp/kegg/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.genome.jp/kegg/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.genome.jp/kegg/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.genome.jp/kegg/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.genome.jp\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.genome.jp/kegg/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.genome.jp/kegg/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.genome.jp/kegg/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/#!Synapse:syn2623706', 'repository_reference': 'n/a', 'dataset_id': 'https://www.synapse.org/#!Synapse:syn2623706'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2623706, None, https://www.synapse.org/#!Synapse:syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2623706 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2623706 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2623706 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2623706 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn2623706 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for n/a in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2623706', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.biolegend.com/en-us/cell-markers', 'repository_reference': 'n/a', 'dataset_id': 'https://www.biolegend.com/en-us/cell-markers'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.biolegend.com/en-us/cell-markers\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.biolegend.com/en-us/cell-markers\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.biolegend.com/en-us/cell-markers, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.biolegend.com/en-us/cell-markers\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.biolegend.com/en-us/cell-markers\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.biolegend.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.biolegend.com/en-us/cell-markers is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.biolegend.com/en-us/cell-markers', 'repository_reference': 'n/a', 'dataset_id': 'https://www.biolegend.com/en-us/cell-markers'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ngdc.cncb.ac.cn/databasecommons/database/id/6917', 'repository_reference': 'n/a', 'dataset_id': 'https://ngdc.cncb.ac.cn/databasecommons/database/id/6917'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ngdc.cncb.ac.cn/databasecommons/database/id/6917, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ngdc.cncb.ac.cn/databasecommons/database/id/6917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ngdc.cncb.ac.cn\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ngdc.cncb.ac.cn/databasecommons/database/id/6917 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ngdc.cncb.ac.cn/databasecommons/database/id/6917', 'repository_reference': 'n/a', 'dataset_id': 'https://ngdc.cncb.ac.cn/databasecommons/database/id/6917'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://depmap.org/portal/download/all/', 'repository_reference': 'n/a', 'dataset_id': 'https://depmap.org/portal/download/all/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://depmap.org/portal/download/all/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://depmap.org/portal/download/all/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://depmap.org/portal/download/all/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://depmap.org/portal/download/all/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://depmap.org/portal/download/all/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: depmap.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://depmap.org/portal/download/all/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://depmap.org/portal/download/all/', 'repository_reference': 'n/a', 'dataset_id': 'https://depmap.org/portal/download/all/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.cbioportal.org/study/summary?id=ccle_broad_2019', 'repository_reference': 'n/a', 'dataset_id': 'https://www.cbioportal.org/study/summary?id=ccle_broad_2019'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.cbioportal.org/study/summary?id=ccle_broad_2019\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.cbioportal.org/study/summary?id=ccle_broad_2019\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.cbioportal.org/study/summary?id=ccle_broad_2019, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.cbioportal.org/study/summary?id=ccle_broad_2019\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.cbioportal.org/study/summary?id=ccle_broad_2019\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.cbioportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.cbioportal.org/study/summary?id=ccle_broad_2019 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.cbioportal.org/study/summary?id=ccle_broad_2019', 'repository_reference': 'n/a', 'dataset_id': 'https://www.cbioportal.org/study/summary?id=ccle_broad_2019'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.10xgenomics.com/resources/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.10xgenomics.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.10xgenomics.com/resources/datasets is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://qupath.github.io/', 'repository_reference': 'n/a', 'dataset_id': 'https://qupath.github.io/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://qupath.github.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://qupath.github.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://qupath.github.io/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://qupath.github.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://qupath.github.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: qupath.github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://qupath.github.io/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://qupath.github.io/', 'repository_reference': 'n/a', 'dataset_id': 'https://qupath.github.io/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://tide.dfci.harvard.edu/', 'repository_reference': 'n/a', 'dataset_id': 'http://tide.dfci.harvard.edu/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://tide.dfci.harvard.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://tide.dfci.harvard.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://tide.dfci.harvard.edu/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://tide.dfci.harvard.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://tide.dfci.harvard.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: tide.dfci.harvard.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://tide.dfci.harvard.edu/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://tide.dfci.harvard.edu/', 'repository_reference': 'n/a', 'dataset_id': 'http://tide.dfci.harvard.edu/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://rdrr.io/github/lasseignelab/CINmetrics/', 'repository_reference': 'n/a', 'dataset_id': 'https://rdrr.io/github/lasseignelab/CINmetrics/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://rdrr.io/github/lasseignelab/CINmetrics/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://rdrr.io/github/lasseignelab/CINmetrics/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://rdrr.io/github/lasseignelab/CINmetrics/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://rdrr.io/github/lasseignelab/CINmetrics/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://rdrr.io/github/lasseignelab/CINmetrics/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: rdrr.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://rdrr.io/github/lasseignelab/CINmetrics/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://rdrr.io/github/lasseignelab/CINmetrics/', 'repository_reference': 'n/a', 'dataset_id': 'https://rdrr.io/github/lasseignelab/CINmetrics/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.genepattern.org/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.genepattern.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.genepattern.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.genepattern.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.genepattern.org/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.genepattern.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.genepattern.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.genepattern.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.genepattern.org/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.genepattern.org/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.genepattern.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE35144', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35144', 'dataset_id': 'GSE35144'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE35144, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE35144', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE108989', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989', 'dataset_id': 'GSE108989'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE108989, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE108989', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB23709', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709', 'dataset_id': 'PRJEB23709'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PRJEB23709 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709, identifier: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB23709', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE78220', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78220', 'dataset_id': 'GSE78220'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE78220, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE78220', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135222', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135222', 'dataset_id': 'GSE135222'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE135222, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135222', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.10xgenomics.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets/human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.10xgenomics.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard', 'repository_reference': 'n/a', 'dataset_id': 'https://www.10xgenomics.com/resources/datasets/human-intestine-cancer-1-standard'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/XiangkunWu/CCCRC_classifier', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiangkunWu/CCCRC_classifier'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/XiangkunWu/CCCRC_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/XiangkunWu/CCCRC_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/XiangkunWu/CCCRC_classifier, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/XiangkunWu/CCCRC_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/XiangkunWu/CCCRC_classifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/XiangkunWu/CCCRC_classifier is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/XiangkunWu/CCCRC_classifier', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiangkunWu/CCCRC_classifier'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f', 'repository_reference': 'n/a', 'dataset_id': 'https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: archive.softwareheritage.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f', 'repository_reference': 'n/a', 'dataset_id': 'https://archive.softwareheritage.org/swh:1:dir:ab578816e920631223538b5fbbf5fab46608d10d;origin=https://github.com/XiangkunWu/CCCRC_classifier;visit=swh:1:snp:62f1de5d883ba11c4abb1b33df4c406601efd177;anchor=swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE17536', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE37892', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE103479', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE104645', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2623706', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE35144', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE108989', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB23709', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE78220', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE135222', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 16 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE39582 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE13067 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE13294 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE14333 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE17536 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE17536 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE17536 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE33113 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE37892 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE37892 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE37892 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE38832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE103479 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE103479 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE103479 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE104645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE104645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE104645 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE104645 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE104645 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE104645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE104645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2623706 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: synapse.org and accession_id: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE35144 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE35144 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE35144 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35144\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE108989 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE108989 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE108989 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PRJEB23709 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: www.ncbi.nlm.nih.gov and accession_id: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 14.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE78220 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE78220 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE78220 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE135222 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 16\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 27th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9710693&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1abb290>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 9 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for ack1: Acknowledgements [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 131750\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 33690 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0713452435a5ca02bfc125d5f11b626a409a54bc80ec3955c85ca111864b914d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 33690 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-45c1b067-0503-4a2f-8e4e-90104cbb11b6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75748\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE72056\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5061\\\",\\n    \\\"repository_reference\\\": \\\"ArrayExpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5060\\\",\\n    \\\"repository_reference\\\": \\\"ArrayExpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23554292\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3505720\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3191087\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75748\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE72056\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5061\\\",\\n    \\\"repository_reference\\\": \\\"ArrayExpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5060\\\",\\n    \\\"repository_reference\\\": \\\"ArrayExpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23554292\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3505720\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3191087\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322014,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44242,\n",
      "        \"completion_tokens\": 343,\n",
      "        \"total_tokens\": 44585,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE75748', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748', 'dataset_id': 'GSE75748'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE75748, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75748', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE72056', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056', 'dataset_id': 'GSE72056'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE72056, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE72056', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5061', 'repository_reference': 'ArrayExpress', 'dataset_id': 'E-MTAB-5061'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-5061 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ArrayExpress, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5060', 'repository_reference': 'ArrayExpress', 'dataset_id': 'E-MTAB-5060'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-5060 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ArrayExpress, identifier: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5060 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5060 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5060 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5060 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5060', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3219045', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3219045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3219045, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn23554292', 'repository_reference': 'synapse.org', 'dataset_id': 'syn23554292'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn23554292, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn23554292', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3505720', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3505720'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3505720, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3191087', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3191087'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3191087, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3191087', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE75748', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE72056', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5060', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23554292', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3191087', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75748 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75748 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE75748 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE72056 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE72056 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE72056 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-5061 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-5060 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: arrayexpress and accession_id: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3219045 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn23554292 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3505720 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3191087 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 9\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 28th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9915613&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1a74500>, <Element supplementary-material at 0x1b0bdd240>, <Element supplementary-material at 0x1b117e280>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1a74500>, <Element supplementary-material at 0x1b0bdd240>, <Element supplementary-material at 0x1b117e280>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: S1A [this file] ), suggesting that there was not a genomic location bias associated with the biological variation. \n",
      "S1B [this file] ) (i.e., annotating genes from upstream and downstream of a certain distance from each peak). \n",
      "S2A [this file] ). S2B [this file] ), indicating that the vast majority of genes do not bear platform-based bias. S2C [this file] ). S2D [this file] ). S2E [this file] ). S2F [this file] ). S2G [this file] ). \n",
      "S3A [this file] ). S3B [this file] ). S3C [this file] ). S3D [this file] ). \n",
      "S3E [this file] ). \n",
      "S4A [this file] ). S5B [this file] and Supplementary Table S6 [rid=SD3] ). \n",
      "S4C [this file] ), suggesting the unique gene induction program was due to context-dependent transcriptional regulomes rather than epigenetic changes. \n",
      "S5A [this file] ). S5B [this file] ). \n",
      "S5C [this file] ), likely due to a large number of genes which were not uniformly regulated. \n",
      "S5D [this file] ). \n",
      "S5E [this file] ) and we further extracted and re-clustered cancer cells by corresponding epithelial markers ( Supplementary Fig. S5F [this file] , G [this file] and Fig. 7H [rid=F7] ). \n",
      "S5H [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: To extend the database, we are further crowdsourcing datasets via social media and a google form ( https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0 (https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0) ) ( Fig. 1A [rid=F1] and Supplementary Table S1 [this file] ). \n",
      "Data-set level documentations such as source publications, experimental conditions and sequencing parameters are presented in Supplementary Data Table S1 [this file] . \n",
      "All the relevant information is summarized in Supplementary Table S1 [this file] . \n",
      "Details of all the curated 136 data sets are summarized in Supplementary Table S1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: This was to be expected as some minor technical variations in cell culture, hormone deprivation and sequencing platform were noted ( Supplementary Table S2 [this file] ). \n",
      "The intersection of up- and down-regulated DEGs from four data sets (|log2FC|>1, padj<0.05) showed that greater than 70% of DEGs were unique to one experiment, whereas only ~3% genes were regulated in all four data sets ( Fig. 3C [rid=F3] and 3D [rid=F3] , Supplementary Table S3 [this file] ). We classified the DEGs into four classes based upon the number of data sets they shared and compared the fold change regulation ( Supplementary Table S3 [this file] ). \n",
      "Importantly, the same ER antibody (Santa Cruz sc543) was utilized in all experiments to pull down ER ( Supplementary Table S4 [this file] ). We also examined the quality parameters (e.g., percentage of reads within peaks and percentage of reads within Blacklist regions) of derived peaks to examine technical variation ( Supplementary Table S5 [this file] ). The number of peaks in the vehicle groups showed a large variation ranging from 332 (C42) to 23,189 (C42) ( Supplementary Table S5 [this file] ). As expected, E2 stimulation resulted in a 1.6-to-27.5-fold gain of ER binding ( Supplementary Table S5 [this file] ). \n",
      "We identified 165, 59 and 136 genes representing early, mid, and late estrogen response signatures respectively ( Fig. 5B [rid=F5] and Supplementary Table S6 [this file] ). \n",
      "Regulatory factor prediction by Lisa( 38 [rid=R38] ) confirmed the ER/FOXA1/GATA3 as the central axis of gene regulation regardless of treatment duration ( Fig. 5D [rid=F5] and Supplementary Table S7 [this file] ). \n",
      "S5B [rid=SD1] and Supplementary Table S6 [this file] ). \n",
      "Mechanistically, MCF7 and T47D unique response genes were enriched for contextual regulatory factors ( Fig. 6D [rid=F6] and Supplementary Table S7 [this file] ) but showed equivalent levels of chromatin accessibility at each other’s open chromatin regions ( Supplementary Fig. \n",
      "We also identified a subset of bidirectionally regulated genes (n=101) that are present in the top 10% of both up- and down-regulated targets and in at least 10% of comparisons ( Fig. 7A [rid=F7] and Supplementary Table S6 [this file] ), such as , and ( Supplementary Fig. \n",
      "For the inter-study concordance analysis, detailed information of four RNA-seq and four ER ChIP-seq data sets are summarized in Supplementary Table S2 [this file] and S4 [this file] respectively.RNA-seq data and clinical information from TCGA and METABRIC were obtained from the GSE62944 (GSE62944) and Synapse software platform under accession number syn1688369, respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 200173\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 50795 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-391568a96eb7f28c915a70032797c7de3876ddabaea7456e87654e713df158c8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 50795 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-aa40ec27-b19a-4c94-89ab-430b5266e712\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE73663\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE51403\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE56066\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78167\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE62944\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE99542\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE102441\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84515\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE25710\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78913\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57436\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96517\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE154873\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://estrogene.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE73663\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE51403\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE56066\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78167\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE62944\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE99542\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE102441\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84515\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE25710\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE78913\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57436\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96517\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE154873\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://estrogene.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322017,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 61681,\n",
      "        \"completion_tokens\": 568,\n",
      "        \"total_tokens\": 62249,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE73663', 'repository_reference': 'geo', 'dataset_id': 'GSE73663'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE73663, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE73663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE73663', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE51403', 'repository_reference': 'geo', 'dataset_id': 'GSE51403'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE51403, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE51403\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE51403', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE56066', 'repository_reference': 'geo', 'dataset_id': 'GSE56066'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE56066, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE56066\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE56066', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE78167', 'repository_reference': 'geo', 'dataset_id': 'GSE78167'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE78167, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE78167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE78167', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1688369', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1688369'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1688369, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE62944', 'repository_reference': 'geo', 'dataset_id': 'GSE62944'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE62944, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE62944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE62944', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE99542', 'repository_reference': 'geo', 'dataset_id': 'GSE99542'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE99542, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE99542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE99542', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE102441', 'repository_reference': 'geo', 'dataset_id': 'GSE102441'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE102441, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE102441\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE102441', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84515', 'repository_reference': 'geo', 'dataset_id': 'GSE84515'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE84515, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE84515\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84515', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE25710', 'repository_reference': 'geo', 'dataset_id': 'GSE25710'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE25710, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE25710\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE25710', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE78913', 'repository_reference': 'geo', 'dataset_id': 'GSE78913'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE78913, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE78913\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE78913', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE57436', 'repository_reference': 'geo', 'dataset_id': 'GSE57436'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE57436, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE57436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE57436', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE96517', 'repository_reference': 'geo', 'dataset_id': 'GSE96517'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE96517, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE96517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE96517', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE154873', 'repository_reference': 'geo', 'dataset_id': 'GSE154873'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE154873, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE154873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE154873', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://estrogene.org/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://estrogene.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://estrogene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://estrogene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'estrogene.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE73663', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE51403', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE56066', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE78167', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE62944', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE99542', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE102441', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84515', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE25710', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE78913', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE57436', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE96517', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE154873', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'estrogene.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 15 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE73663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE73663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE73663 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE73663 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE73663 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE73663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE51403\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE51403\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE51403 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE51403 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE51403 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE51403\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51403\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE56066\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE56066\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE56066 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE56066 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE56066 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE56066\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56066\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE78167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE78167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE78167 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE78167 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE78167 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE78167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1688369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE62944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE62944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE62944 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE62944 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE62944 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE62944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE99542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE99542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE99542 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE99542 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE99542 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE99542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE102441\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE102441\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE102441 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE102441 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE102441 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE102441\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102441\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE84515\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE84515\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84515 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84515 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE84515 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE84515\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84515\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE25710\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE25710\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE25710 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE25710 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE25710 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE25710\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25710\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE78913\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE78913\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE78913 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE78913 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE78913 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE78913\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78913\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE57436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE57436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE57436 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE57436 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE57436 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE57436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE96517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE96517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE96517 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE96517 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE96517 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE96517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE154873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE154873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE154873 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE154873 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE154873 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE154873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 15: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 15\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 18, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 29th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11420198&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 14 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1531000>, <Element supplementary-material at 0x1b0fe9bc0>, <Element supplementary-material at 0x1b0dcb880>, <Element supplementary-material at 0x1b0dca600>, <Element supplementary-material at 0x1b0dc8e00>, <Element supplementary-material at 0x1b0dc8b00>, <Element supplementary-material at 0x1b0dca1c0>, <Element supplementary-material at 0x1b0dc9900>, <Element supplementary-material at 0x1b0dc9100>, <Element supplementary-material at 0x1b1560100>, <Element supplementary-material at 0x1b7307bc0>, <Element supplementary-material at 0x1b1562140>, <Element supplementary-material at 0x1b1562480>, <Element supplementary-material at 0x1b0ecba00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1531000>, <Element supplementary-material at 0x1b0fe9bc0>, <Element supplementary-material at 0x1b0dcb880>, <Element supplementary-material at 0x1b0dca600>, <Element supplementary-material at 0x1b0dc8e00>, <Element supplementary-material at 0x1b0dc8b00>, <Element supplementary-material at 0x1b0dca1c0>, <Element supplementary-material at 0x1b0dc9900>, <Element supplementary-material at 0x1b0dc9100>, <Element supplementary-material at 0x1b1560100>, <Element supplementary-material at 0x1b7307bc0>, <Element supplementary-material at 0x1b1562140>, <Element supplementary-material at 0x1b1562480>, <Element supplementary-material at 0x1b0ecba00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 148881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 37972 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e58234537e8da23b3a122cede8eb617b07882384c6789550dc65b3cad7d7fe96\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 37972 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ce2e9306-7053-47cf-bfc8-3f7c82efa016\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S1\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S2\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S3\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S4\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S5\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S6\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S7\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S8\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S9\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S10\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S11\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S12\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S13\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE249687\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84176\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59119\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96107\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE126957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE142881\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"syn4921369\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn4921369\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"https://github.com/zwang-bioinformatics/RAD21-2024/\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/zwang-bioinformatics/RAD21-2024/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S1\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S2\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S3\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S4\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S5\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S6\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S7\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S8\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S9\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S10\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S11\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S12\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset S13\\\",\\n    \\\"repository_reference\\\": \\\"http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE249687\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84176\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59119\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96107\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE126957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE142881\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"syn4921369\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn4921369\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"https://github.com/zwang-bioinformatics/RAD21-2024/\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/zwang-bioinformatics/RAD21-2024/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322023,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 48081,\n",
      "        \"completion_tokens\": 1352,\n",
      "        \"total_tokens\": 49433,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S1', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S2', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S3', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S4', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S4'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S5', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S6', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S6'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S7', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S7'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S8', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S8'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S9', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S9'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S10', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S10'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S11', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S11'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S12', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S12'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset S13', 'repository_reference': 'http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials', 'dataset_id': 'Dataset S13'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1073/, None, http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1073/ for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://www.pnas.org/lookup/doi/10.1073/pnas.2402518121#supplementary-materials in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE249687', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687', 'dataset_id': 'GSE249687'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE249687, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE249687', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84176', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176', 'dataset_id': 'GSE84176'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE84176, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84176', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE59119', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119', 'dataset_id': 'GSE59119'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE59119, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE96107, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE126957', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957', 'dataset_id': 'GSE126957'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE126957, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE126957', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE142881', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881', 'dataset_id': 'GSE142881'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE142881, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE142881', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4921369', 'repository_reference': 'https://www.synapse.org/Synapse:syn4921369', 'dataset_id': 'syn4921369'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4921369, synapse.org, https://www.synapse.org/Synapse:syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/zwang-bioinformatics/RAD21-2024/', 'repository_reference': 'https://github.com/zwang-bioinformatics/RAD21-2024/', 'dataset_id': 'https://github.com/zwang-bioinformatics/RAD21-2024/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/zwang-bioinformatics/RAD21-2024/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/zwang-bioinformatics/RAD21-2024/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/zwang-bioinformatics/RAD21-2024/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/zwang-bioinformatics/RAD21-2024/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE249687', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84176', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE126957', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE142881', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 21 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1073/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: doi.org and accession_id: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE249687 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE249687 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE249687 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84176 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84176 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE84176 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE59119 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE96107 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 17: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE126957 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE126957 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE126957 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 18 with repo: geo and accession_id: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 18: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 19 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE142881 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE142881 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE142881 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 19 with repo: geo and accession_id: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 19: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 20 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4921369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 20 with repo: synapse.org and accession_id: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 20: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 21 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 21: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 21\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 35, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 30th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7865022&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b66cd9d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 29) for Sec13: Usage Notes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 162080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 41272 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-55b0eaa14eae79d04a7c625e6eb482906afd65d1f733ee891b5a0a2c58616e6a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 41272 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d2c76744-666b-4349-8298-5288d19cbd7a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.13342055\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20681023\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn20681023\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.13342055\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20681023\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn20681023\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322040,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 55305,\n",
      "        \"completion_tokens\": 138,\n",
      "        \"total_tokens\": 55443,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.6084/m9.figshare.13342055', 'repository_reference': 'doi.org', 'dataset_id': '10.6084/m9.figshare.13342055'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.6084/m9.figshare.13342055, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.6084/m9.figshare.13342055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.6084/m9.figshare.13342055', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20681023', 'repository_reference': 'synapse.org', 'dataset_id': 'syn20681023'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20681023, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn20681023', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn20681023'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn20681023, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.6084/m9.figshare.13342055', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.6084/m9.figshare.13342055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.6084/m9.figshare.13342055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.6084/m9.figshare.13342055 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.6084/m9.figshare.13342055 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.6084/m9.figshare.13342055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.6084/m9.figshare.13342055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20681023 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn20681023 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 31/100 (31.00%) | Elapsed: 00:04:29 | ETA: 00:09:59\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 31th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8131595&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6a25f00>, <Element supplementary-material at 0x1b71cac40>, <Element supplementary-material at 0x1b71ca900>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6a25f00>, <Element supplementary-material at 0x1b71cac40>, <Element supplementary-material at 0x1b71ca900>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: We sourced frozen post-mortem prefrontal cortex from four CdLS patients aged 19–48 years and six age-matched controls (Supplementary Fig. 1a [this file] ). Three of the patients had heterozygous mutations in , the gene most frequently mutated in CdLS (ref. , Fig. 1a [rid=Fig1] , Supplementary Fig. 1a, b [this file] ). \n",
      "To characterise gene expression in CdLS patient neurons we isolated nuclei from the prefrontal cortex, stained with the neuronal marker NeuN (ref. ), and sorted NeuN positive and NeuN negative nuclei by flow cytometry (Fig. 1b [rid=Fig1] , Supplementary Fig. 1c [this file] ). However, CAGE analysis of promoters showed separation of CdLS patient and control samples along the major principle component PC1, and revealed significant deregulation of 766 gene promoters in CdLS neurons (adj < 0.05; 358 upregulated, 408 downregulated; Supplementary Fig. 2a–c [this file] , Supplementary Data 1 [this file] ). Gene promoters that were downregulated in CdLS neurons were enriched for the gene ontology (GO) terms synaptic signalling and organization, ion and neurotransmitter transport, axon development, behavior and cognition, while upregulated promoters showed no striking functional enrichment (Supplementary Fig. 2d [this file] , Supplementary Data 2 [this file] ). RNA-seq identified 617 differentially expressed genes (adj < 0.05) in CdLS neurons, 310 genes were up- and 307 were downregulated (Fig. 1c [rid=Fig1] , Supplementary Fig. 3a [this file] ). In close agreement with CAGE, genes that were downregulated by RNA-seq were related to synaptic transmission, signalling processes, learning and behaviour, and sphingolipid metabolism, while upregulated genes showed no striking functional enrichment (Fig. 1d [rid=Fig1] , Supplementary Data 4 [this file] ). \n",
      "The direction of gene deregulation by RNA-seq and CAGE was highly correlated ( = 0.68, = < 2.2e-16 for genes found deregulated by RNA-seq or CAGE, and = 0.95, = < 2.2e-16 for genes found deregulated by both RNA-seq and CAGE; Supplementary Fig. 2e [this file] ). \n",
      "RNA-seq of NeuN-negative nuclei showed upregulation of immune response genes that were not seen in NeuN-positive nuclei, including interferon response, JAK-STAT signalling, and the complement pathway (Supplementary Fig. 4 [this file] , Supplementary Data 5 [this file] , 6 [this file] ). There was significant overlap between changes in gene expression in human CdLS with published gene expression in heterozygous mouse brain at embryonic day 13.5 (e13.5; whole brain versus human CdLS NeuN positive and NeuN negative nuclei combined; = 1.32e-14, Odds ratio = 2.05) and with gene expression in cortical neurons in explant culture ( cortical neurons versus human CdLS NeuN positive neurons; Supplementary Data 7 [this file] ; = 8.38e-06, Odds ratio = 3.03). \n",
      "We established explant cultures from cortices at e14.5 under conditions that enrich for post-mitotic neurons and promote neuronal maturation (Fig. 2a [rid=Fig2] , Supplementary Fig. 5a, b [this file] ). \n",
      "Constructs were tagged by V5 and t2a-fused GFP (ERt2-TEV) for identification of ERt2-TEV-expressing cells (Fig. 2b [rid=Fig2] , Supplementary Fig. 5c [this file] ). ERt2-TEV remained cytoplasmic until nuclear translocation of ERt2-TEV was triggered, which occurred within minutes of 4-hydroxy tamoxifen (4-OHT) addition (Fig. 2b [rid=Fig2] , Supplementary Fig. 5c [this file] ). Addition of doxycycline leads to the expression of TEV protease with an exogeneous nuclear localization sequence (NLS-TEV, Supplementary Fig. 6a, b [this file] ). Induction of NLS-TEV depleted ~70% of RAD21-TEV within 24 h of doxycycline addition (Supplementary Fig. 6c [this file] ). \n",
      "RNA-seq 24 h after 4-OHT-induced nuclear translocation of ERt2-TEV identified 750 deregulated (adj < 0.05) genes, of which 463 were down- and 287 were upregulated (Fig. 2d [rid=Fig2] , Supplementary Data 8 [this file] ). As observed in CdLS neurons, no striking enrichment was observed for upregulated genes (Fig. 2e [rid=Fig2] , Supplementary Data 9 [this file] ). \n",
      "As observed for the nuclear translocation of ERt2-TEV, downregulated genes were enriched for adhesion, signalling, synaptic function and development (Supplementary Fig. 5d, e [this file] , Supplementary Data 10 [this file] , 11 [this file] ). There was extensive overlap between genes deregulated by acute cohesin depletion with genes deregulated in neurons (this study) and embryonic brain , but less so with genes deregulated in other mouse models of neuronal dysfunction (Supplementary Fig. 7 [this file] ). \n",
      "Reduced mRNA expression translated into reduced levels of protein in RAD21-TEV neurons, as illustrated for NLGN1, encoded by (Supplementary Fig. 8 [this file] , Supplementary Data 12 [this file] ). \n",
      "These shared ASD genes mediate important neuronal functions in cell adhesion ( ) signalling ( ), ion channels ( ), synaptic transmission ( ), transcription ( ) and sphingolipid metabolism ( ) (Fig. 3b [rid=Fig3] , Supplementary Table 1 [this file] ). \n",
      "Gene expression changes in Rett syndrome, but not other human neurological diseases , showed significant overlap with neurons (Odds ratio = 2.06, = 4.64e-10; Supplementary Fig. 9 [this file] ). \n",
      "Remarkably, comparison of RNA-seq during RAD21-TEV depletion (24 h) and after RAD21-TEV rescue (day 7) showed that the vast majority of genes that were found deregulated after 24 h (695 of 720 or 96.5%) were fully rescued upon restoration of cohesin levels (Fig. 4c, e [rid=Fig4] ; Supplementary Data 13 [this file] ). These experiments confirmed that rescue of gene expression required the restoration of RAD21 expression and was not due to the adaptation of neurons to reduced cohesin expression (Supplementary Fig 10 [this file] ; Supplementary Data 14 [this file] ). \n",
      "These included 14 genes implicated in ASD and shared between CdLS and NLS-TEV (Supplementary Table 1 [this file] ). \n",
      "Sequence data was analysed using the 4 Cseqpipe software suite with the setting of “-stat_type mean -trend_resolution 40,000”. 4C-seq primers are listed in Supplementary Data 15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 195469\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 49619 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-345978c01db62f57cf3b51263fe48c0f2219b0f59282deefe1b6f292cdeefa2f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 49619 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6859ab10-9833-4fe8-8e7e-c1c139300adb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE150130\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-9045\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96107\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRX3381923\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/SRX3381923\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR677HXC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR677HXC/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60192\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR198ZYJ\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR198ZYJ/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR822CEA\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR822CEA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001373.v2.p2\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84245\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE113673\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSEA 81912\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65776\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59318\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE113673\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0143563#sec021\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59630\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE150130\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-9045\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE96107\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRX3381923\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/SRX3381923\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR677HXC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR677HXC/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60192\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR198ZYJ\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR198ZYJ/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR822CEA\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org/experiments/ENCSR822CEA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001373.v2.p2\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84245\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE113673\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSEA 81912\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65776\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59318\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE113673\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0143563#sec021\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE59630\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322046,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 60655,\n",
      "        \"completion_tokens\": 1248,\n",
      "        \"total_tokens\": 61903,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE150130', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130', 'dataset_id': 'GSE150130'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE150130, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE150130', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-9045', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/', 'dataset_id': 'E-MTAB-9045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-9045 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/, identifier: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-9045', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4587609', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4587609', 'dataset_id': 'syn4587609'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4587609, synapse.org, https://www.synapse.org/#!Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE96107, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRX3381923', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/SRX3381923', 'dataset_id': 'SRX3381923'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRX3381923 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/sra/SRX3381923, identifier: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRX3381923', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR677HXC', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR677HXC/', 'dataset_id': 'ENCSR677HXC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCSR677HXC is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR677HXC/, identifier: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR677HXC', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE60192', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192', 'dataset_id': 'GSE60192'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE60192, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60192', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR198ZYJ', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR198ZYJ/', 'dataset_id': 'ENCSR198ZYJ'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCSR198ZYJ is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR198ZYJ/, identifier: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR198ZYJ', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR822CEA', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR822CEA/', 'dataset_id': 'ENCSR822CEA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCSR822CEA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR822CEA/, identifier: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR822CEA', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'phs001373.v2.p2', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2', 'dataset_id': 'phs001373.v2.p2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: phs001373.v2.p2, dbgap, https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'phs001373.v2.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1371/, None, https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4, identifier: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1371/ for https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1371/ for https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000650#s4 in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1371/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84245', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245', 'dataset_id': 'GSE84245'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE84245, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84245', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE113673', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673', 'dataset_id': 'GSE113673'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE113673, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE113673', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSEA 81912', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912', 'dataset_id': 'GSEA 81912'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE81912, None, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912, identifier: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81912 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81912 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GSE81912 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912 in \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE81912', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65776', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776', 'dataset_id': 'GSE65776'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE65776, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65776', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE59318', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318', 'dataset_id': 'GSE59318'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE59318, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59318', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18485175', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn18485175', 'dataset_id': 'syn18485175'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn18485175, synapse.org, https://www.synapse.org/#!Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE59630', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630', 'dataset_id': 'GSE59630'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE59630, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE150130', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-9045', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRX3381923', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR677HXC', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE60192', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR198ZYJ', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR822CEA', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'phs001373.v2.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1371/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84245', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE113673', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE81912', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65776', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59318', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 18 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE150130 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE150130 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE150130 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-9045 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE96107 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRX3381923 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: www.ncbi.nlm.nih.gov and accession_id: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCSR677HXC in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: www.encodeproject.org and accession_id: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 6.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE60192 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE60192 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE60192 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCSR198ZYJ in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: www.encodeproject.org and accession_id: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 8.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCSR822CEA in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: www.encodeproject.org and accession_id: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: dbgap, identifier: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for phs001373.v2.p2 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: dbgap and accession_id: phs001373.v2.p2.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1371/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1371/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: doi.org and accession_id: 10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1371/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84245 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84245 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE84245 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE113673 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE113673 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE113673 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81912 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81912 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE81912 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65776 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65776 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE65776 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59318 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59318 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE59318 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn18485175 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 17 with repo: synapse.org and accession_id: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 17: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE59630 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 18 with repo: geo and accession_id: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 18: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 18\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 32th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10828232&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6a44c40>, <Element supplementary-material at 0x1b6aa7440>, <Element supplementary-material at 0x1b6aa7300>, <Element supplementary-material at 0x1b6a25880>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6a44c40>, <Element supplementary-material at 0x1b6aa7440>, <Element supplementary-material at 0x1b6aa7300>, <Element supplementary-material at 0x1b6a25880>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The sequence of the specific primers used is provided in Supplemental Table 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: RNA-sequencing analysis of i-astrocyte cultures 72 h after transduction identified a total of 19,778 unique transcripts (Supplemental Table 2 [this file] ). Statistical analysis identified 500 transcripts with at least a 2-fold change in expression (adjusted ≤ 0.05) (Supplemental Table 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Representative images obtained from sections where the primary antibodies were omitted are shown in Supplementary Figure 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Differentiation and maturation of i-astrocyte cultures were monitored by real-time PCR and immunostaining for selected markers (Supplementary Fig. 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 200008\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 50754 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-709e77ba7d524fe92e008211c1802855e09b6a1483b999bd97e2811d89920be6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 50754 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-38ef6c82-6d5b-4355-b0c7-27c90b1148ec\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE214628\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn13363290-v49\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NM_021272.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE214628\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn13363290-v49\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NM_021272.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322048,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58336,\n",
      "        \"completion_tokens\": 118,\n",
      "        \"total_tokens\": 58454,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE214628', 'repository_reference': 'geo', 'dataset_id': 'GSE214628'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE214628, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE214628', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn13363290-v49', 'repository_reference': 'synapse.org', 'dataset_id': 'syn13363290-v49'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn13363290-v49, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn13363290-v49', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NM_021272.3', 'repository_reference': 'nuccore', 'dataset_id': 'NM_021272.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NM_021272.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'NM_021272.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE214628', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn13363290-v49', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'NM_021272.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE214628 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE214628 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE214628 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn13363290-v49 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_021272.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for NM_021272.3 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: nuccore and accession_id: NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/NM_021272.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 33th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11852220&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <supplementary-material xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.or...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6d97c00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6d97c00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref bioengineering-12-00126-s001: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 179116\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 45531 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-617644b85257980404bb8216bc45f7a2cf6a34cef4edf2e14a8a98e19fcfa592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 45531 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-851e7b7a-a9ee-49ba-9c34-cd949b2e430b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51365301\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn51365301\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"r9\\\",\\n    \\\"repository_reference\\\": \\\"https://r9.finngen.fi\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.genecards.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://dsigdb.tanlab.org/DSigDBv1.0/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://pubchem.ncbi.nlm.nih.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.rcsb.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51365301\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn51365301\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"r9\\\",\\n    \\\"repository_reference\\\": \\\"https://r9.finngen.fi\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.genecards.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://dsigdb.tanlab.org/DSigDBv1.0/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://pubchem.ncbi.nlm.nih.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.rcsb.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322051,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51005,\n",
      "        \"completion_tokens\": 253,\n",
      "        \"total_tokens\": 51258,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51365301', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn51365301', 'dataset_id': 'syn51365301'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn51365301, synapse.org, https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'r9', 'repository_reference': 'https://r9.finngen.fi', 'dataset_id': 'r9'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: r9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID r9 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://r9.finngen.fi, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'r9.finngen.fi', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.genecards.org/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.genecards.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.genecards.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.genecards.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.genecards.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'r9.finngen.fi', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.genecards.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn51365301 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 34th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9314356&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b654d200>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b654d200>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The parameters of each analysis step are summarized in Supplementary Table 3 [this file] . \n",
      "Summary statistics of age-at-onset (AAO) for LOAD cases and age-at-last-examination (AAE) for normal controls are presented in Supplementary Table 1 [this file] . The full name and accessibility of each study can be found in Supplementary Table 2 [this file] . \n",
      "The variance in AD explained by PRS maximizes at a -value threshold of 0.5 in participants from GERAD (Genetic and Environmental Risk for Alzheimer’s disease) [ 23 [rid=CR23] ] and 22 locally available ADGC (Alzheimer’s Disease Genetics Consortium) studies (Supplementary Fig. 4 [this file] ). \n",
      "Because our analytic approaches used only subsets of all available LOAD case–control GWAS data, we neither had nor anticipated having sufficient power to detect individual SNPs with genome-wide significant association with resilience (Supplementary Fig. 3 [this file] ). \n",
      "In the full samples from three locally downloaded replication studies (Alzheimer Disease Centers Wave 7 [ADC7], AddNeuroMed, and Alzheimer’s Disease Neuroimaging Initiative stage GO/2/3 [ADNI-GO/2/3]; normal controls, = 1321; LOAD cases, = 943) (Table 1 [rid=Tab1] and Supplementary Table 1 [this file] ), we tested for correlations between PRSs and polygenic resilience scores. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 245965\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 62243 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fbbbbd043a2f5c97cb3e7ae24d08429cb63baca7fbd2c73fc2bc56393b5c6f34\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 62243 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e25acd7f-9804-481f-8626-21a63a525b5f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"adni.loni.usc.edu\\\",\\n    \\\"repository_reference\\\": \\\"http://adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4907804\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4907804\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"adni.loni.usc.edu\\\",\\n    \\\"repository_reference\\\": \\\"http://adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4907804\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4907804\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322053,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 72091,\n",
      "        \"completion_tokens\": 139,\n",
      "        \"total_tokens\": 72230,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'adni.loni.usc.edu', 'repository_reference': 'http://adni.loni.usc.edu', 'dataset_id': 'adni.loni.usc.edu'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID adni.loni.usc.edu is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://adni.loni.usc.edu, identifier: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier adni.loni.usc.edu for http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier adni.loni.usc.edu for http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier adni.loni.usc.edu for http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier adni.loni.usc.edu for http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier adni.loni.usc.edu for http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'adni.loni.usc.edu', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4907804', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4907804', 'dataset_id': 'syn4907804'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4907804, synapse.org, https://www.synapse.org/#!Synapse:syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4907804', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.niagads.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'adni.loni.usc.edu', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4907804', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: adni.loni.usc.edu, identifier: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier adni.loni.usc.edu for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier adni.loni.usc.edu for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier adni.loni.usc.edu for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier adni.loni.usc.edu for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier adni.loni.usc.edu for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL adni.loni.usc.edu may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for adni.loni.usc.edu in adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: adni.loni.usc.edu and accession_id: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository adni.loni.usc.edu unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4907804 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 35th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11439699&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6e8cd90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 16 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for s3: 3 Transcriptomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for s3: 3 Transcriptomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for s4: 4 Proteomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 32) for s6: 6 Metabolomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 40) for s12: Supplementary material [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 190) for SM1: A brief meta-analysis of the topics studied concerning Alzheimer's disease using ROSMAP data, as well as outside of the cohort, can be found in the Supplementary material [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 296276\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 74821 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-9b80dc07b91609ab3da74ffee2ca950a5eec4c8b00222759003b4a2353b5b0c9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 74821 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-29012a4b-1885-417d-9928-6dd0eb54dc6b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20068543\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn20068543\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn22264775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157325\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn338856\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn338856\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26720676\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn26720676\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3157275\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17008935\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn17008935\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"syn10235592\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn10235592\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P30AG10161\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG15819\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG17917\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG30146\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"R01AG36042\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RC2AG036547\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36836\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG48015\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG57473\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG32984\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46152\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"U01AG46161\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG61356\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.radc.rush.edu\\\",\\n    \\\"repository_reference\\\": \\\"http://www.radc.rush.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.synapse.org\\\",\\n    \\\"repository_reference\\\": \\\"http://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.niagads.org\\\",\\n    \\\"repository_reference\\\": \\\"http://www.niagads.org\\\"\\n  },\\n     {\\n    \\\"dataset_identifier\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20068543\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn20068543\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn22264775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn3157325\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn338856\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn338856\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26720676\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn26720676\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3157275\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17008935\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn17008935\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"syn10235592\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn10235592\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"P30AG10161\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG15819\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG17917\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG30146\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"R01AG36042\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RC2AG036547\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG36836\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"R01AG48015\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RF1AG57473\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG32984\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG46152\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"U01AG46161\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"U01AG61356\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.radc.rush.edu\\\",\\n    \\\"repository_reference\\\": \\\"http://www.radc.rush.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.synapse.org\\\",\\n    \\\"repository_reference\\\": \\\"http://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.niagads.org\\\",\\n    \\\"repository_reference\\\": \\\"http://www.niagads.org\\\"\\n  },\\n     {\\n    \\\"dataset_identifier\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322075,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 107888,\n",
      "        \"completion_tokens\": 1247,\n",
      "        \"total_tokens\": 109135,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'synapse.org', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2580853, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20068543', 'repository_reference': 'https://www.synapse.org/Synapse:syn20068543', 'dataset_id': 'syn20068543'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20068543, synapse.org, https://www.synapse.org/Synapse:syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20068543', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn22264775', 'repository_reference': 'https://www.synapse.org/Synapse:syn22264775', 'dataset_id': 'syn22264775'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn22264775, synapse.org, https://www.synapse.org/Synapse:syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn3157325', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn3157325'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn3157325, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn3157325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn338856', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn338856', 'dataset_id': 'syn338856'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID syn338856 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn338856, identifier: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn338856 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn338856', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn26720676', 'repository_reference': 'https://www.synapse.org/Synapse:syn26720676', 'dataset_id': 'syn26720676'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn26720676, synapse.org, https://www.synapse.org/Synapse:syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn26720676', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4896408', 'repository_reference': 'synapse.org', 'dataset_id': 'syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4896408, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3157275', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3157275'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3157275, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17008935', 'repository_reference': 'https://www.synapse.org/Synapse:syn17008935', 'dataset_id': 'syn17008935'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn17008935, synapse.org, https://www.synapse.org/Synapse:syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn17008935', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn10235592', 'repository_reference': 'https://www.synapse.org/Synapse:syn10235592', 'dataset_id': 'syn10235592'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn10235592, synapse.org, https://www.synapse.org/Synapse:syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn10235592', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'P30AG10161', 'repository_reference': 'n/a', 'dataset_id': 'P30AG10161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID P30AG10161 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: P30AG10161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier P30AG10161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier P30AG10161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier P30AG10161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier P30AG10161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier P30AG10161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'P30AG10161', 'repository_reference': 'n/a', 'dataset_id': 'P30AG10161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG15819', 'repository_reference': 'n/a', 'dataset_id': 'R01AG15819'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG15819 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG15819\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG15819 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG15819 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG15819 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG15819 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG15819 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG15819', 'repository_reference': 'n/a', 'dataset_id': 'R01AG15819'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG17917', 'repository_reference': 'n/a', 'dataset_id': 'R01AG17917'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG17917 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG17917\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG17917 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG17917 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG17917 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG17917 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG17917 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG17917', 'repository_reference': 'n/a', 'dataset_id': 'R01AG17917'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG30146', 'repository_reference': 'n/a', 'dataset_id': 'R01AG30146'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG30146 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG30146\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG30146 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG30146 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG30146 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG30146 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG30146 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG30146', 'repository_reference': 'n/a', 'dataset_id': 'R01AG30146'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG36042', 'repository_reference': 'n/a', 'dataset_id': 'R01AG36042'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG36042 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG36042\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36042 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36042 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36042 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36042 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36042 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG36042', 'repository_reference': 'n/a', 'dataset_id': 'R01AG36042'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RC2AG036547', 'repository_reference': 'n/a', 'dataset_id': 'RC2AG036547'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID RC2AG036547 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: RC2AG036547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RC2AG036547 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RC2AG036547 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RC2AG036547 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RC2AG036547 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RC2AG036547 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'RC2AG036547', 'repository_reference': 'n/a', 'dataset_id': 'RC2AG036547'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG36836', 'repository_reference': 'n/a', 'dataset_id': 'R01AG36836'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG36836 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG36836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG36836 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG36836 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG36836 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG36836 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG36836 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG36836', 'repository_reference': 'n/a', 'dataset_id': 'R01AG36836'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'R01AG48015', 'repository_reference': 'n/a', 'dataset_id': 'R01AG48015'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID R01AG48015 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: R01AG48015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier R01AG48015 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier R01AG48015 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier R01AG48015 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier R01AG48015 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier R01AG48015 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'R01AG48015', 'repository_reference': 'n/a', 'dataset_id': 'R01AG48015'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'RF1AG57473', 'repository_reference': 'n/a', 'dataset_id': 'RF1AG57473'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID RF1AG57473 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: RF1AG57473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RF1AG57473 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RF1AG57473 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RF1AG57473 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RF1AG57473 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RF1AG57473 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'RF1AG57473', 'repository_reference': 'n/a', 'dataset_id': 'RF1AG57473'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG32984', 'repository_reference': 'n/a', 'dataset_id': 'U01AG32984'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG32984 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: U01AG32984\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG32984 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG32984 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG32984 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG32984 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG32984 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'U01AG32984', 'repository_reference': 'n/a', 'dataset_id': 'U01AG32984'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG46152', 'repository_reference': 'n/a', 'dataset_id': 'U01AG46152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG46152 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: U01AG46152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46152 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46152 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46152 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46152 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46152 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'U01AG46152', 'repository_reference': 'n/a', 'dataset_id': 'U01AG46152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG46161', 'repository_reference': 'n/a', 'dataset_id': 'U01AG46161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG46161 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: U01AG46161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG46161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG46161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG46161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG46161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG46161 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'U01AG46161', 'repository_reference': 'n/a', 'dataset_id': 'U01AG46161'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'U01AG61356', 'repository_reference': 'n/a', 'dataset_id': 'U01AG61356'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID U01AG61356 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: U01AG61356\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier U01AG61356 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier U01AG61356 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier U01AG61356 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier U01AG61356 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier U01AG61356 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'U01AG61356', 'repository_reference': 'n/a', 'dataset_id': 'U01AG61356'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.radc.rush.edu', 'repository_reference': 'http://www.radc.rush.edu', 'dataset_id': 'http://www.radc.rush.edu'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.radc.rush.edu, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.radc.rush.edu is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.radc.rush.edu, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.synapse.org', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'http://www.synapse.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.synapse.org, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://www.synapse.org in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link http://www.synapse.org could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.synapse.org/[#!]*Synapse:__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 914 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 918 - INFO - Extracted ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.synapse.org is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.synapse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://www.synapse.org in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://www.synapse.org', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'http://www.synapse.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.niagads.org', 'repository_reference': 'http://www.niagads.org', 'dataset_id': 'http://www.niagads.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.niagads.org, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.niagads.org is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.niagads.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775', 'dataset_id': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn22264775, None, https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775, identifier: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22264775 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22264775 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085', 'dataset_id': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn9702085, None, https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085, identifier: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9702085 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9702085 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9702085 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9702085 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20068543', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn3157325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn338856', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn26720676', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17008935', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn10235592', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 14 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20068543 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20068543 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20068543 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20068543 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20068543 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20068543\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn22264775 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn3157325 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: doi.org and accession_id: 10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn338856 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier syn338856 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn26720676 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4896408 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3157275 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn17008935 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn10235592 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn10235592 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn10235592 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn10235592 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn10235592 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn10235592\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 11: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 12: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn22264775 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: synapse.org and accession_id: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn9702085 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: synapse.org and accession_id: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 14\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 30, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 14\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 36th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6323985&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1b99910>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 23 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#abstract1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 26) for abstract1: Abstract [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-1-4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-1-4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#sup1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 206) for sup1: The details of the extended description and metric advantages are given in the supplementary materials [href=this file] .\n",
      "Supplementary Table S1 [href=this file] summarizes the GTRD content and statistics.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SEC4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 36) for SEC4: SUPPLEMENTARY DATA [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 107422\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 27608 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-06812ba8f36d7a03e73103ea03ea08dc9461ca796ddb841a808e8ac24b34ffde\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 27608 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7e960b96-7d45-4419-bc8f-274fcef02a95\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ncbi.nlm.nih.gov/sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCODE\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GEO\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn6131484/wiki/\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ncbi.nlm.nih.gov/sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCODE\\\",\\n    \\\"repository_reference\\\": \\\"https://www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GEO\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn6131484/wiki/\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322095,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 37660,\n",
      "        \"completion_tokens\": 170,\n",
      "        \"total_tokens\": 37830,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRA', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/sra', 'dataset_id': 'SRA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.ncbi.nlm.nih.gov/sra, identifier: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRA', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCODE', 'repository_reference': 'https://www.encodeproject.org', 'dataset_id': 'ENCODE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCODE is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.encodeproject.org, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCODE', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GEO', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GEO'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GEO is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.ncbi.nlm.nih.gov/geo/, identifier: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GEO', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/#!Synapse:syn6131484/wiki/', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/#!Synapse:syn6131484/wiki/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn6131484, None, https://www.synapse.org/#!Synapse:syn6131484/wiki/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn6131484', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'SRA', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCODE', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GEO', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn6131484', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRA in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: www.ncbi.nlm.nih.gov and accession_id: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCODE in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: www.encodeproject.org and accession_id: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GEO in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: www.ncbi.nlm.nih.gov and accession_id: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn6131484 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn6131484 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn6131484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 27, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 37th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10680627&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 21 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b682c380>, <Element supplementary-material at 0x1b71c8d00>, <Element supplementary-material at 0x1b0e85040>, <Element supplementary-material at 0x1b0e78180>, <Element supplementary-material at 0x1b0e34900>, <Element supplementary-material at 0x1b190a700>, <Element supplementary-material at 0x1b66f9d80>, <Element supplementary-material at 0x1b66f9e00>, <Element supplementary-material at 0x1b66fa780>, <Element supplementary-material at 0x1b66f9900>, <Element supplementary-material at 0x1b0c588c0>, <Element supplementary-material at 0x1b66f9b80>, <Element supplementary-material at 0x1b66fb080>, <Element supplementary-material at 0x1b66f9580>, <Element supplementary-material at 0x1b66fa3c0>, <Element supplementary-material at 0x1b66f9640>, <Element supplementary-material at 0x1b66f9bc0>, <Element supplementary-material at 0x1b66f9840>, <Element supplementary-material at 0x1b66f99c0>, <Element supplementary-material at 0x1b66f91c0>, <Element supplementary-material at 0x1b66f9040>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b682c380>, <Element supplementary-material at 0x1b71c8d00>, <Element supplementary-material at 0x1b0e85040>, <Element supplementary-material at 0x1b0e78180>, <Element supplementary-material at 0x1b0e34900>, <Element supplementary-material at 0x1b190a700>, <Element supplementary-material at 0x1b66f9d80>, <Element supplementary-material at 0x1b66f9e00>, <Element supplementary-material at 0x1b66fa780>, <Element supplementary-material at 0x1b66f9900>, <Element supplementary-material at 0x1b0c588c0>, <Element supplementary-material at 0x1b66f9b80>, <Element supplementary-material at 0x1b66fb080>, <Element supplementary-material at 0x1b66f9580>, <Element supplementary-material at 0x1b66fa3c0>, <Element supplementary-material at 0x1b66f9640>, <Element supplementary-material at 0x1b66f9bc0>, <Element supplementary-material at 0x1b66f9840>, <Element supplementary-material at 0x1b66f99c0>, <Element supplementary-material at 0x1b66f91c0>, <Element supplementary-material at 0x1b66f9040>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We reanalyzed and harmonized data from previous studies that used genetic perturbations (mostly CRISPR interference (CRISPRi)) to inhibit candidate enhancers and measure effects on gene expression (see Note S1 [this file] ). S1.1f [this file] ) and 9,938 “negative” element-gene pairs where no significant reduction in expression was observed despite the experiment having good power to detect >15–25% effects on gene expression ( Note S1 [this file] ). \n",
      "S10 [this file] ). \n",
      "S8c [this file] ), and the correlation between the predictive scores and the measured quantitative effect on expression ( Extended Data Fig. 2f [this file] , Fig. S9 [this file] ), using hold-one-chromosome-out cross-validation for supervised models. S8c [this file] , Extended Data Fig. 2f [this file] ), including at different distance thresholds ( Fig. 2c [rid=F2] ). S8c [this file] , Extended Data Fig. 2f [this file] ). We describe detailed evaluations of these other predictors and baseline methods in Note S2 [this file] , Note S3 [this file] , Note S5 [this file] , and Note S7 [this file] . \n",
      "ENCODE-rE2G and ENCODE-rE2G perform comparably to other wellperforming models (both with enrichment = 28 at a recall of 15%) ( Fig. 2d [rid=F2] , Table S15 [this file7] ). We also examined fine-mapped eQTL variant-gene links from 11 other GTEx tissues that were represented in ENCODE biosamples ( Table S14 [this file6] ). Using the score threshold corresponding to 70% in the K562 CRISPR dataset, ENCODE-rE2G achieved an average enrichment of 6.5 across tissues (range: 3.5–11.4), similar than other predictive and baseline models, while achieving a stronger recall, identifying an average of 16% (range: 10–22%) of fine-mapped eQTL variants and linking them to the correct eGene ( Extended Data Fig. 3c [this file] , d [this file] ). \n",
      "These conclusions were supported by additional benchmarking analyses ( Extended Data Fig. 4 [this file] ). \n",
      "S2.1a [this file] ). \n",
      "Many ENCODE-rE2G enhancers show elevated levels of active chromatin marks, such as H3K27ac and DNase-seq (mean DNase-seq signal for candidate elements across biosamples = 3.27; mean for enhancers linked to at least one gene = 7.52) — but importantly, many do not (15.6% of enhancers have DNase-seq scores below the global median) ( Extended Data Fig 5h [this file] ). S2.1d [this file] ). In part for this reason, annotations of enhancers based on 1D chromatin state alone miss 33% (chromHMM) or 61% (cCRE Catalog) of ENCODE-rE2G enhancers, as well as 30% or 12%, respectively, of the validated enhancergene pairs from the Nasser ., 2021 unbiased CRISPR DHS tiling dataset ( Extended Data Fig. 6 [this file] ). \n",
      "As expected, many element-gene regulatory interactions were highly cell-type specific: 26.4% were present in only one biosample (mean = 34.49, median = 7 biosamples) ( Fig. 3e [rid=F3] ), and predicted regulatory interactions were much more specific than the activity of either the promoters or enhancers alone ( Extended Data Fig 5g [this file] , h [this file] ). \n",
      "As previously observed, fine-mapped variants underlying enhancers were often predicted to regulate multiple genes (average=3.4, max=56), including 62% that are linked to different genes in different cell types; this appears to be a fundamental property of enhancer-promoter regulation , and motivates combining locus-specific variant-to-gene information with orthogonal information about gene function ( Note S7 [this file] , Fig S7.1 [this file] ). \n",
      "This combination of ENCODE-rE2G and PoPS predicted a target gene for 3,528 out of 13,012 non-coding credible sets with a PIP > 0.1 fine-mapped variant spanning all traits; this included 5,851 out of 29.2K fine-mapped GWAS variants ( Table S13 [this file5] ), representing a 3.7-fold excess overlap compared to control variants (all distal noncoding variants detected in the 1000 Genomes project, =9.2 million) ( Fig. 4b [rid=F4] ). Other interesting examples at the and loci are reported in Note S7 [this file] ( Fig S7.2 [this file] ). \n",
      "We provide further analysis and guidelines regarding the value of adding new cell types to the ENCODE resource, assessing tissue-specificity of the ENCODE-rE2G biosample predictions, evaluating other combinations of ENCODE-rE2G and PoPS, and restricting our ENCODE-rE2G analyses to top disease-enriched biosamples for identifying possible cell types and target genes for GWAS loci in Note S7 [this file] ( Fig S7.1 [this file] . \n",
      "S4.1 [this file] - 2 [this file] , Note S4 [this file] , Note S5 [this file] ). \n",
      "These results are consistent with the importance of enhancer activity and 3D contact in enhancer regulation and highlight the utility of combining them as in the ABC model ( Fig. 2b [rid=F2] , c [rid=F2] , Fig S4.3 [this file] ). \n",
      "S11 [this file] ). S11a [this file] ). \n",
      "Out of 6 methods for estimating 3D enhancer-promoter contact frequency in the ABC model, we found that the best dataset was cell-type-specific ENCODE Hi-C (precision at 70% recall = 53.8%; Hi-C depth = 2 billion reads, 5-kb resolution), which outperformed other datasets including Hi-C (1 billion reads) or an inverse function of distance (precision at 70% recall = 50.4% or 31.8%, respectively) ( Fig. 5g [rid=F5] , Note S2 [this file] ). S2.1f [this file] , g [this file] ). \n",
      "Key additional features were related to promoter class and enhancerenhancer interactions ( Fig. 5b [rid=F5] , h [rid=F5] , Fig S4.4 [this file] ). \n",
      "S6.1b [this file] ) to have a fine-mapped eQTL variant in a distal accessible site ( Fig. S6.1a [this file] ). S6.1c [this file] - f [this file] ). Together, these observations support a model in which sequence-intrinsic features of the promoters of ubiquitously expressed genes make them less sensitive to distal enhancers (see Note S6 [this file] ). \n",
      "In each dataset, perturbations to individual enhancers on average reduced H3K27ac signal at other nearby enhancers, and the magnitude of the effect decreased as enhancer-enhancer distance increased ( Fig. 6g [rid=F6] , h [rid=F6] , Extended Data Fig. 8 [this file] ). \n",
      "S6.1b [this file] ), possibly because they contain built-in motifs for activating transcription factors that buffer them against distal enhancers . \n",
      "The CRISPR datasets still have a relatively small number of positive examples; were designed with certain biases in enhancer and gene selection; do not distinguish from effects; are conducted largely in a single cell type (K562); and are underpowered to detect regulatory interactions with small effect sizes ( ., <25%) (see Note S1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: For 197 non-coding credible sets corresponding to 11 blood-related traits that are linked to exactly one “putatively causal” gene with a coding fine-mapped variant within 2Mb on either side of the lead variant, we compute the precision and recall in linking it to this gene for each of the element-gene predictions in blood biosamples (see Methods [this file] ). \n",
      "See Methods [this file] , ‘CRISPRi benchmark’ section for description of statistical test. \n",
      "See Methods [this file] , ‘Data visualization’ section for definition of box plot elements. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: We constructed two logistic regression classifiers to predict CRISPR-validated element-gene pairs using different combinations of molecular features ( Tables S1 [this file] , S3 [rid=SD5] ). \n",
      "We used this pipeline to evaluate ENCODE-rE2G and 574 other models ( Table S1 [this file] ), including (i) the ABC model, which we re-computed here natively in hg38 using various combinations of ENCODE assays to estimate enhancer activity and enhancer-promoter 3D contact; (ii) previous machine learning models that infer enhancer-gene regulation, including EpiMap , GraphReg , CIA , Enformer , and EPIraction ; and (iii) simple baselines such as linking elements to the closest expressed transcription start site (TSS) or gene, linking elements to genes solely on the basis of loops or domains from 3D contact measurements, or correlating distal element and promoter accessibility across cell types. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: ENCODE file accessions are listed in Table S2 [this file] and Table S10 [rid=SD12] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD5: We constructed two logistic regression classifiers to predict CRISPR-validated element-gene pairs using different combinations of molecular features ( Tables S1 [rid=SD3] , S3 [this file] ). Specifically, we included features related to (i) chromatin state (quantitative DNase-seq signals at the element and promoter), (ii) 3D contact frequency (averaged from ENCODE Hi-C in 35 diploid biosamples to capture cell-type-invariant features of genome organization); (iii) the Activity-by-Contact (ABC) model , which multiplies enhancer activity and 3D contact frequency; (iv) genomic position ( ., distance from element to promoter); (v) promoter class ( ., whether the gene is uniformly and ubiquitously expressed across cell types ( ., a “housekeeping” gene)); and (vi) information about nearby enhancers ( ., the activity of all other elements within 5 kb of the perturbed element) ( Table S3 [this file] ). In the ENCODE-rE2G model, we sought to evaluate the utility of additional assays available in Tier 1 ENCODE cell types, and expanded to include a total of 47 features derived from DNase-seq, histone ChIP-seq, cell-type specific ENCODE Hi-C, and ChIA-PET experiments, as well as features from additional predictive models such as EpiMap and GraphReg that jointly analyze many datasets ( Table S3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD6: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD7: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD8: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD9: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD10: ENCODE portal accession ids and public links for GraphReg and EPIraction predictions are available in Table S8 [this file] and Table S9 [rid=SD11] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD11: ENCODE portal accession ids and public links for GraphReg and EPIraction predictions are available in Table S8 [rid=SD10] and Table S9 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD12: ENCODE file accessions are listed in Table S2 [rid=SD4] and Table S10 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD13: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD14: We next applied ENCODE-rE2G across 352 biosamples with DNase-seq data generated by the ENCODE Project, and identified a total of 13,455,443 element-gene regulatory interactions using the threshold at 70% recall in the CRISPR benchmark ( Fig. 3a [rid=F3] , Table S12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD15: This combination of ENCODE-rE2G and PoPS predicted a target gene for 3,528 out of 13,012 non-coding credible sets with a PIP > 0.1 fine-mapped variant spanning all traits; this included 5,851 out of 29.2K fine-mapped GWAS variants ( Table S13 [this file] ), representing a 3.7-fold excess overlap compared to control variants (all distal noncoding variants detected in the 1000 Genomes project, =9.2 million) ( Fig. 4b [rid=F4] ). \n",
      "Numerical results are reported in Table S13 [this file] . \n",
      "Numerical results for each of the Figure panels are reported in Table S13 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD16: We also examined fine-mapped eQTL variant-gene links from 11 other GTEx tissues that were represented in ENCODE biosamples ( Table S14 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD17: ENCODE-rE2G and ENCODE-rE2G perform comparably to other wellperforming models (both with enrichment = 28 at a recall of 15%) ( Fig. 2d [rid=F2] , Table S15 [this file] ). \n",
      "Numerical results are reported in Table S15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD18: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD19: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD20: Summary statistics available in Table S18 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD21: Summary statistics available in Table S19 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 161219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 41057 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4e0c6a99105ac6d7aad28e31ca14632733f93a2b16746c57269035b3b2941686\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 41057 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5ffe58c8-fe5f-45f2-9b28-9bb3b38f2613\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR998YDI\\\",\\n    \\\"repository_reference\\\": \\\"www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52264240\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.finucanelab.org/data/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52234396\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR443VTK\\\",\\n    \\\"repository_reference\\\": \\\"www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE225157\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR998YDI\\\",\\n    \\\"repository_reference\\\": \\\"www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52264240\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.finucanelab.org/data/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52234396\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ENCSR443VTK\\\",\\n    \\\"repository_reference\\\": \\\"www.encodeproject.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE225157\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322097,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 50588,\n",
      "        \"completion_tokens\": 228,\n",
      "        \"total_tokens\": 50816,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR998YDI', 'repository_reference': 'www.encodeproject.org', 'dataset_id': 'ENCSR998YDI'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCSR998YDI is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR998YDI', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn52264240', 'repository_reference': 'synapse.org', 'dataset_id': 'syn52264240'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn52264240, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn52264240', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.finucanelab.org/data/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.finucanelab.org/data/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.finucanelab.org/data/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.finucanelab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.finucanelab.org/data/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.finucanelab.org/data/', 'repository_reference': 'n/a', 'dataset_id': 'https://www.finucanelab.org/data/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn52234396', 'repository_reference': 'synapse.org', 'dataset_id': 'syn52234396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn52234396, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn52234396', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR443VTK', 'repository_reference': 'www.encodeproject.org', 'dataset_id': 'ENCSR443VTK'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ENCSR443VTK is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR443VTK', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE225157', 'repository_reference': 'geo', 'dataset_id': 'GSE225157'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE225157, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE225157', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'ENCSR998YDI', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn52264240', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn52234396', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR443VTK', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE225157', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCSR998YDI in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: www.encodeproject.org and accession_id: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn52264240 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn52234396 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ENCSR443VTK in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: www.encodeproject.org and accession_id: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE225157 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE225157 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE225157 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 38th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5192600&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1cb0b10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec11'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec11: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#FPar2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for FPar2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#FPar2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for FPar2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 128541\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 32887 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ca51ada6aa43672e59d7054d67da7d34d7c8b231c2882eafb93e327497ac6645\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 32887 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2477ba34-9c4c-4748-afb5-bdd1bae4c16e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3049712\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3049712/files/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2787209\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2787209/files/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3049712\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3049712/files/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2787209\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2787209/files/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322112,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44790,\n",
      "        \"completion_tokens\": 118,\n",
      "        \"total_tokens\": 44908,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3049712', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3049712/files/', 'dataset_id': 'syn3049712'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3049712, synapse.org, https://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3049712', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2787209', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2787209/files/', 'dataset_id': 'syn2787209'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2787209, synapse.org, https://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2787209', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3049712', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2787209', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3049712 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2787209 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 39th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7752919&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b14c9c40>, <Element supplementary-material at 0x1b1202f40>, <Element supplementary-material at 0x1b639e000>, <Element supplementary-material at 0x1b0dbb780>, <Element supplementary-material at 0x1b0dbb740>, <Element supplementary-material at 0x1b0dbbe40>, <Element supplementary-material at 0x1b0db9640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b14c9c40>, <Element supplementary-material at 0x1b1202f40>, <Element supplementary-material at 0x1b639e000>, <Element supplementary-material at 0x1b0dbb780>, <Element supplementary-material at 0x1b0dbb740>, <Element supplementary-material at 0x1b0dbbe40>, <Element supplementary-material at 0x1b0db9640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: Colon carcinomas showed substantial MLL1 expression at all tumor stages (T0–T4), scored from weak to moderate and strong (Fig. 1a [rid=Fig1] and Supplementary Fig. 1a [this file] ). These analyses showed no significant changes in expression across tumor stages (Supplementary Fig. 1b [this file] ). Patients of stages I-III and stage IV were stratified in three groups based on the level of expression (MLL1 low, medium, and high, Supplementary Fig. 1c [this file] ). High levels were associated with shorter survival and increased risk of disease relapse (Supplementary Fig. 1c, d [this file] ). Immuno-staining for β-catenin revealed high levels of nuclear β-catenin, indicative of high Wnt signaling, in a major portion of the analyzed tumors (Supplementary Fig. 1e [this file] ). Across all tumor stages, carcinomas with high levels of nuclear β-catenin showed strong expression of MLL1 (Fig. 1b [rid=Fig1] and Supplementary Fig. 1f, g [this file] ). \n",
      "Mll1 is gradually lost towards the differentiated compartment (Fig. 1d [rid=Fig1] and Supplementary Fig. 1h [this file] ). A crypt–villus gradient was also observed for H3K4 tri-methylation (H3K4me3) (Supplementary Fig. 1i [this file] ). We observed a similar distribution of Mll1, Ki67, and H3K4me3 in the mouse colon (Supplementary Fig. 1j–l [this file] ). \n",
      "The dysplastic tissue in the β-cat ; Mll1 intestines showed an expansion of Lgr5-GFP stem cells (Fig. 2c [rid=Fig2] , upper picture, Supplementary Fig. 2a [this file] ) and an increase in the stem cell genes , , and (Supplementary Fig. 2b [this file] ). The stem cell expansion in β-cat tumorous intestines was associated with an increase in Mll1 protein levels (Supplementary Fig. 2c [this file] , the mutant β-catenin allele is marked by an arrow, quantification is shown below). Specifically, tumor cells with high levels of β-catenin expressed high nuclear levels of Mll1 (Supplementary Fig. 2d [this file] ; green arrows mark β-catenin cells, white arrowheads indicate β-catenin cells). Equal recombination was observed at day 5 in 25% of crypts (Supplementary Fig. 2e [this file] , upper row, quantification in f). Subsequently, the mutant cells in β-cat ; Mll1 mice expanded at day 10 and formed LacZ dysplasia at day 30 (Supplementary Fig. 2e [this file] , middle and lower left pictures, quantification in f). In contrast, β-cat ; Mll1 mice showed a gradual reduction of LacZ cells down to 10% over time (Supplementary Fig. 2e [this file] , lower right picture, quantification in f). At 30 days, the β-cat ; Mll1 mice exhibited crypts with LacZ long-lived Paneth cells interspersed with non-mutant (LacZ ) stem cells (Supplementary Fig. 2g [this file] , marked by an arrow). Furthermore, we observed crypts that were normally shaped and depleted of Mll1 but with nuclear β-catenin (β-cat ) and positive for Lgr5-GFP; they exhibited a slight reduction in proliferation, as seen by a decrease in the incorporation of BrdU (Fig. 2c [rid=Fig2] , lower picture, Supplementary Fig. 2h, i [this file] ). Apoptosis was not detected in β-cat ; Mll1 crypts (Supplementary Fig. 2j [this file] ). \n",
      "All β-cat organoids grew independently of R-spondin1 (Supplementary Fig. 2k [this file] ). β-cat induced a strong expansion of Lgr5-GFP stem cells (Fig. 2d, e [rid=Fig2] , second panel). Single β-cat ; Mll1 cells re-formed spheroid-shaped organoids over many passages with high efficiency (Supplementary Fig. 2l [this file] ), demonstrating the proliferative potential and self-renewal capacity of Mll1-deficient β-cat cells. From the seventh passage on, β-cat ; Mll1 cells formed less organoids, which were smaller and acquired a thick-walled shape (Supplementary Fig. 2m [this file] ), indicating differentiation and progressive exhaustion of stemness upon loss of Mll1. \n",
      "Under non-oncogenic conditions without genetic stabilization of β-catenin, mutant Mll1 stem cells produced progeny up to 20 days after the induction of mutagenesis, as seen by LacZ lineage tracing in Mll1 mice (Supplementary Fig. 3a [this file] , quantification on the right). From 20 days onwards, Mll1 crypts were progressively lost, but a substantial number of Mll1 crypts persisted beyond 50 days after induction (Supplementary Fig. 3b [this file] , quantification on the right). \n",
      "Genes specific for enteroendocrine and tuft cells did not undergo major changes (Supplementary Fig. 3c [this file] ). β-cat ; Mll1 stem cells exhibited a strongly increased expression of , a transcription factor essential for secretory lineage determination , and its target gene , which instructs secretory cell differentiation in Paneth and goblet cells (Fig. 3b [rid=Fig3] ). \n",
      "To investigate the essential role of Mll1 in the stemness of human colon tumors, we established stable, doxycycline-inducible shMLL1 single-cell clones of the human colon adenocarcinoma cell lines Ls174T and DLD1, which exhibited a strong reduction of MLL1 (Supplementary Fig. 4a–d [this file] ). The ablation of MLL1 did not affect the expression of other MLL family members (Supplementary Fig. 4e [this file] ). A significant growth reduction was detected only after four passages and a total of 10–14 days of culture (Supplementary Fig. 4f, g [this file] ). We found that MLL1 protein was strongly increased in 3D (non-adherent, self-renewing) sphere cultures of human Ls174T cells compared to adherent cultures, while the expression of the family member SETD1A was not changed (Supplementary Fig. 4h [this file] , quantification on the right). In contrast to the 2D culture, the ablation of MLL1 in 3D sphere cultures had a rapid effect on the growth and self-renewal of the tumor cells: secondary sphere formation of shMLL1 cells was reduced by 80% in Ls174T and by 60% in DLD1 cells, compared to control cells (Fig. 4a [rid=Fig4] and Supplementary Fig. 4i [this file] , quantifications on the right). Residual shMLL1 spheres exhibited minimal MLL1 expression and reduced proliferation, as shown by staining for Ki67 (Supplementary Fig. 4j [this file] ). shMLL1 spheres were negative for the apoptosis marker cleaved Caspase-3 (Supplementary Fig. 4k [this file] , western blot on the right). \n",
      "The residual xenografts showed a strong reduction of MLL1 (Supplementary Fig. 4l [this file] , quantification below) and H3K4me3 (Supplementary Fig. 4m [this file] , upper panel), and significantly fewer proliferating cells (Supplementary Fig. 4m [this file] , middle panel, quantifications on the right). The apoptosis marker cleaved Caspase-3 did not increase in shMLL1 Ls174T tumors (Supplementary Fig. 4m [this file] , lower panel, western blot on the right). The shMLL1 human Ls174T xenografts and sphere cells showed enhanced differentiation with an increase in the expression of the Paneth cell-specific genes and and the goblet cell markers and (Supplementary Fig. 4n [this file] ), as was also observed in the β-cat mouse tumor model. An increase in expression was also observed in DLD1 sphere cells (Supplementary Fig. 4o [this file] ). The expression of the differentiation-associated cell cycle inhibitor was upregulated in shMLL1 Ls174T xenografts and spheres (Supplementary Fig. 4p, q [this file] , quantifications on the right). \n",
      "Comparison of the differentially expressed genes with a signature of Mll1-ablated fibroblasts revealed deregulation of MLL1 target genes such as and (Supplementary Fig. 4r [this file] , Supplementary Data 2 [rid=MOESM5] ). A volcano plot of the deregulated genes overlaid with this signature showed a predominant downregulation of stem cell genes upon knockdown of MLL1; these included , , , and the Wnt-regulated gene (Supplementary Data 3 [rid=MOESM6] and Fig. 4d [this file] ). The selective regulation of Wnt-dependent stem cell genes was also observed in DLD1 human colon cancer cells (Supplementary Fig. 4s [this file] ). \n",
      "Chromatin immunoprecipitation (ChIP) revealed high H3K4me3 and low H3K27me3 levels at the transcription start sites (TSS) of the MLL1-regulated stem cell genes , , and (Fig. 5a [rid=Fig5] and Supplementary Fig. 5a [this file] ). Ablation of MLL1 shifted the epigenetic state of the stem cell genes from transcriptionally permissive (H3K4me3) to repressed (H3K27me3) (Fig. 5a [rid=Fig5] and Supplementary Fig. 5a [this file] ). Global levels of H3K4 and H3K27 methylation were not changed (Supplementary Fig. 5b [this file] ). We identified MLL1 binding to the promoters of the intestinal stem cell genes , , and (Fig. 5b [rid=Fig5] and Supplementary Fig. 5c, d [this file] ). Strongly reduced binding of MLL1 was observed in shMLL1 Ls174T and DLD1 cells (Fig. 5b [rid=Fig5] and Supplementary Fig. 5c–e [this file] ). \n",
      "Primer sequences used for ChIP-qPCR are given in Supplementary Information (Supplementary Table 1) [this file] . \n",
      "Primer sequences used for qRT-PCR are listed in Supplementary Information (Supplementary Table 2) [this file] . \n",
      "Transcriptome analyses in human CRC were carried out on 2207 samples that were available in two public repositories (Supplementary Table 3 [this file] ): (i) RNA seq (version 2) data from The Cancer Genome Atlas (TCGA) project ( http://cancergenome.nih.gov/ (http://cancergenome.nih.gov/) ) were downloaded from the legacy version of the NCI Genomic Data Commons (GDC) repository . \n",
      "The RNA sequencing data from FACS-isolated Lgr5-GFP intestinal stem cells (Fig. 3b, d [rid=Fig3] and Supplementary Fig. 3c [this file] , Supplementary Data 1 [rid=MOESM4] ) have been deposited in the Gene expression omnibus under accession number GSE148394 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394) . The RNA sequencing data from shMLL1 Ls174T colon cancer cells (Fig. 4c, d [rid=Fig4] and Supplementary Fig. 4r [this file] , Supplementary Data 2 [rid=MOESM5] ) are available in the ArrayExpress database under accession number E-MTAB-8152 (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/) . The human colon cancer data referenced during the study (Supplementary Table 3 [this file] ) are available in public repositories from the TCGA website ( TCGA-COAD (https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga) , TCGA-READ (https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga) ) and in the Gene Expression Omnibus (GEO) at NCBI ( GSE14333 (https://www.ncbi.nlm.nih.gov/gds/?term=gse14333) , GSE33113 (https://www.ncbi.nlm.nih.gov/gds/?term=gse33113) , GSE39582 (https://www.ncbi.nlm.nih.gov/gds/?term=gse39582) , GSE38832 (https://www.ncbi.nlm.nih.gov/gds/?term=gse38832) , GSE44076 (https://www.ncbi.nlm.nih.gov/gds/?term=gse44076) , GSE13294 (https://www.ncbi.nlm.nih.gov/gds/?term=gse13294) , GSE18088 (https://www.ncbi.nlm.nih.gov/gds/?term=gse18088) and GSE2109 (https://www.ncbi.nlm.nih.gov/gds/?term=gse2109) ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: A volcano plot of the differentially expressed genes in β cat ; Mll1 versus β-cat ; Mll1 stem cells showed both up- and downregulation of genes (Fig. 3b [rid=Fig3] and Supplementary Data 1 [this file] ). There were also no decreases in other Notch target genes such as and (Supplementary Data 1 [this file] ), which means that the upregulation of occurs independently of a global decrease in Notch signaling. \n",
      "RNA seq differential expression data can be found in Supplementary Data 1 [this file] . \n",
      "The RNA sequencing data from FACS-isolated Lgr5-GFP intestinal stem cells (Fig. 3b, d [rid=Fig3] and Supplementary Fig. 3c [rid=MOESM1] , Supplementary Data 1 [this file] ) have been deposited in the Gene expression omnibus under accession number GSE148394 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394) . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Comparison of the differentially expressed genes with a signature of Mll1-ablated fibroblasts revealed deregulation of MLL1 target genes such as and (Supplementary Fig. 4r [rid=MOESM1] , Supplementary Data 2 [this file] ). \n",
      "RNA seq differential expression data can be found in Supplementary Data 2 [this file] . \n",
      "The RNA sequencing data from shMLL1 Ls174T colon cancer cells (Fig. 4c, d [rid=Fig4] and Supplementary Fig. 4r [rid=MOESM1] , Supplementary Data 2 [this file] ) are available in the ArrayExpress database under accession number E-MTAB-8152 (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/) . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Volcano plot of differentially expressed genes in shMLL1 human Ls174T cells (blue dots), overlaid with MLL1-regulated colon cancer stem cell signature (Supplementary Data 3 [this file] ), marked as green dots and indicated by name. mRNA expression of intestinal stem cell genes and classical Wnt targets in control and 6d doxycycline-induced shMLL1 human Ls174T spheres, = 4 samples derived from three biologically independent cell clones (  = 3), two-tailed unpaired -test, *** = 0.001, ** = 0.0018, * = 0.015, * = 0.03. \n",
      "We compiled 132 deregulated stem cell genes to establish an MLL1-regulated colon cancer stem cell signature (Supplementary Data 3 [this file] ). A volcano plot of the deregulated genes overlaid with this signature showed a predominant downregulation of stem cell genes upon knockdown of MLL1; these included , , , and the Wnt-regulated gene (Supplementary Data 3 [this file] and Fig. 4d [rid=MOESM1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 193485\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 49123 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-08b2321bc30a4c37131460f450e1a68fe2bbd01a6630c046a0cd87f8f9ecfa73\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 49123 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3d3cbeb6-7fce-46cc-a5b9-f0ee324eb05d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148394\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-8152\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-COAD\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-READ\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE14333\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse14333\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33113\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse33113\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse39582\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38832\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse38832\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE44076\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse44076\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse13294\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE18088\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse18088\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE2109\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse2109\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2623706\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706/wiki/s\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148394\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-8152\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-COAD\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-READ\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE14333\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse14333\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33113\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse33113\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE39582\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse39582\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38832\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse38832\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE44076\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse44076\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse13294\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE18088\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse18088\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE2109\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/gds/?term=gse2109\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2623706\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2623706/wiki/s\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322116,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58867,\n",
      "        \"completion_tokens\": 723,\n",
      "        \"total_tokens\": 59590,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE148394', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394', 'dataset_id': 'GSE148394'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE148394, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE148394', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-8152', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/', 'dataset_id': 'E-MTAB-8152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-8152 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/, identifier: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-8152', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-COAD', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-COAD'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TCGA-COAD is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga, identifier: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-COAD', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-READ', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-READ'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TCGA-READ is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga, identifier: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-READ', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse14333', 'dataset_id': 'GSE14333'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE14333, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse33113', 'dataset_id': 'GSE33113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE33113, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse39582', 'dataset_id': 'GSE39582'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE39582, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse38832', 'dataset_id': 'GSE38832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE38832, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE44076', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse44076', 'dataset_id': 'GSE44076'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE44076, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE44076', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse13294', 'dataset_id': 'GSE13294'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE13294, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE18088', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse18088', 'dataset_id': 'GSE18088'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE18088, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE18088', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE2109', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse2109', 'dataset_id': 'GSE2109'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: gse2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE2109, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE2109', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2623706', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2623706/wiki/s', 'dataset_id': 'syn2623706'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2623706, synapse.org, https://www.synapse.org/#!Synapse:syn2623706/wiki/s\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2623706', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE148394', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-8152', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TCGA-COAD', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TCGA-READ', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE44076', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE18088', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE2109', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2623706', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 13 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE148394 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE148394 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE148394 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-8152 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.cancer.gov, identifier: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.cancer.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TCGA-COAD in www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: www.cancer.gov and accession_id: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.cancer.gov unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.cancer.gov, identifier: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.cancer.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TCGA-READ in www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: www.cancer.gov and accession_id: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.cancer.gov unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE14333 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE33113 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE39582 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE38832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE44076 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE44076 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE44076 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE13294 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE18088 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE18088 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE18088 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE2109 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE2109 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE2109 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2623706 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2623706 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: synapse.org and accession_id: syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2623706\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 13\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 40th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8859891&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b157e190>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 161432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 41110 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c365fb84733ba34089cb4f274b5b81c22247c5317d481e841f6be5f266191221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 41110 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-0775d455-2460-473a-b4a7-9fae6b5678dd\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4224222\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4224222\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322132,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 55501,\n",
      "        \"completion_tokens\": 48,\n",
      "        \"total_tokens\": 55549,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4224222', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn4224222'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4224222, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4224222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4224222', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn4224222', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4224222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4224222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4224222 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4224222 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4224222 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4224222 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4224222 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn4224222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4224222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 41/100 (41.00%) | Elapsed: 00:06:00 | ETA: 00:08:39\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 41th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9853518&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b10f1550>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 3 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 45) for s6: Data availability statement [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 140004\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35753 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-781edf38f99e5d3a94a68989da0b55c0270be04b3050460ce45a9cd6ab2cb3c2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35753 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2cdec86f-7987-4261-bbde-4aa2734c7194\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://zenodo.org/record/3757476#.Y1NGmy1h1B1\\\",\\n    \\\"repository_reference\\\": \\\"zenodo.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://zenodo.org/record/3757476#.Y1NGmy1h1B1\\\",\\n    \\\"repository_reference\\\": \\\"zenodo.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322149,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 47387,\n",
      "        \"completion_tokens\": 98,\n",
      "        \"total_tokens\": 47485,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://zenodo.org/record/3757476#.Y1NGmy1h1B1', 'repository_reference': 'zenodo.org', 'dataset_id': 'https://zenodo.org/record/3757476#.Y1NGmy1h1B1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://zenodo.org/record/3757476#.Y1NGmy1h1B1, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://zenodo.org/record/3757476#.Y1NGmy1h1B1 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: zenodo.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL zenodo.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'zenodo.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3193805', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3193805'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3193805, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'zenodo.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 42th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11852190&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b737aa90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 3 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#notes4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 45) for notes4: Data Availability Statement [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 132773\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 33945 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-669a570fd929b53c38cd5d9d9fdbdef2efda8a42cdff5882861b32db220a098c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 33945 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5f28cc38-4331-4fc2-a85d-9b80534a2fa6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322166,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41799,\n",
      "        \"completion_tokens\": 70,\n",
      "        \"total_tokens\": 41869,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'syn3193805'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3193805, synapse.org, https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 43th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10652415&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b16fcb00>, <Element supplementary-material at 0x1b734b080>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b16fcb00>, <Element supplementary-material at 0x1b734b080>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SM1: We constructed a linear regression model to investigate the correlation between CNVs and the age of death (AOD) of AD cases based on copy number profiles and meta-information of 1,127 North American white individuals from the ROSMAP AD cohort ( Chibnik et al., 2018 [rid=ref12] ; De Jager et al., 2018 [rid=ref15] ; Mostafavi et al., 2018 [rid=ref42] ; Lee et al., 2023 [rid=ref32] ) ( Supplementary Table S1 [this file] and Supplementary Figure S1 [rid=SM2] ), while sex and disease status were treated as covariates (Methods 4.5, Figure 2 [rid=fig2] ). \n",
      "We found all 10 AOD-correlated CNVs with low frequency in both AD and healthy control groups, and they showed no significant frequency difference in both groups ( Supplementary Table S2 [this file] ). They also showed rare frequency in human populations ( Supplementary Table S2 [this file] ). \n",
      "Based on the linear additive model with sex and disease status as covariates, there were 110 genes identified with their expression level significantly correlated with copy number dosage of the 10 AOD-correlated CNVs under the FDR threshold at 5%, forming 125 CNV-Gene pairs ( Supplementary Table S3 [this file] ). \n",
      "There were 2,056, 1,294, and 2,774 AOD-correlated genes identified based on cogdx, braaksc, and ceradsc respectively, under the genome-wide FDR threshold of 0.05 ( Supplementary Table S4 [this file] ). \n",
      "We further constructed correlation network for above CNV-gene-AOD pairs. ( Figure 4 [rid=fig4] ; Supplementary Table S5 [this file] ; Methods 4.9). \n",
      "Differential expression analysis showed and had no difference between the AD and normal groups indicating that and ’s contribution to longevity was independent of AD status ( Supplementary Table S8 [this file] ). \n",
      "The original contributions presented in the study are included in the article/ Supplementary material [this file] , further inquiries can be directed to the corresponding author. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SM2: We constructed a linear regression model to investigate the correlation between CNVs and the age of death (AOD) of AD cases based on copy number profiles and meta-information of 1,127 North American white individuals from the ROSMAP AD cohort ( Chibnik et al., 2018 [rid=ref12] ; De Jager et al., 2018 [rid=ref15] ; Mostafavi et al., 2018 [rid=ref42] ; Lee et al., 2023 [rid=ref32] ) ( Supplementary Table S1 [rid=SM1] and Supplementary Figure S1 [this file] ), while sex and disease status were treated as covariates (Methods 4.5, Figure 2 [rid=fig2] ). These findings each maintained a false discovery rate (FDR) of less than 0.05 ( Supplementary Figure S2 [this file] ). \n",
      "Copy number dosage had a significant influence on AOD and that was consistent with our correlation result between these 10 CNVs and AOD, except mCNV62126 ( Supplementary Figure S3 [this file] ). \n",
      "The transcriptomic profile was further adjusted for AD pathological traits using limma R package ( Ritchie et al., 2015 [rid=ref46] ) to exclude bias from disease status ( Supplementary Figure S4 [this file] , Methods 4.8). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 141409\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 36104 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d0e594c73bbc89f78748ca81fc4bbd5b7fe48991746dc5003ca7972c97eadc4a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 36104 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2e098f4f-09fa-4edf-a115-72cbee69180a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3157322\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26254632\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3157322\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26254632\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322168,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 43674,\n",
      "        \"completion_tokens\": 116,\n",
      "        \"total_tokens\": 43790,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3388564, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3157322', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3157322'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3157322, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn26254632', 'repository_reference': 'doi.org', 'dataset_id': 'syn26254632'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn26254632, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn26254632', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn26254632', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3157322 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn26254632 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 44th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7842189&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b784fbc0>, <Element supplementary-material at 0x1b735a840>, <Element supplementary-material at 0x1b6493800>, <Element supplementary-material at 0x1b6490680>, <Element supplementary-material at 0x1b6490300>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b784fbc0>, <Element supplementary-material at 0x1b735a840>, <Element supplementary-material at 0x1b6493800>, <Element supplementary-material at 0x1b6490680>, <Element supplementary-material at 0x1b6490300>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We identified 13 distinct, robust co-expression modules, including 7 modules enriched for microglial genes, but varying in their relationship to disease, cell type, and temporal trajectories; we call these microglial neurodegeneration-associated modules, MNMs ( Figures 1 [rid=F1] , S1 [this file] , and S2 [this file] ; Table S1 [rid=SD2] ). We compared them with six independent microglia-specific gene expression datasets from transgenic mouse models and humans with AD pathology to identify overlapping patterns of gene expression ( Figures 1B [rid=F1] , S1C [this file] – S1E [this file] , and S2F [this file] ; Table S2 [rid=SD3] ), supporting their generalizability. \n",
      "We confirmed that the MNMs include expected traits indicative of neuroimmune activity, including: (1) known neuroregulatory pathways, such as complement activation ( Schafer et al., 2013 [rid=R74] ), Toll-like receptor (TLR), and interleukin-1β (IL-1β) ( Heneka et al., 2013 [rid=R32] ; Figure 1E [rid=F1] ); (2) strong association with tau pathology, a neuronal driver of neurodegeneration ( Figures 1F [rid=F1] and S2A [this file] ); and (3) high overlap with the conserved neuroinflammatory module we identified in our prior study (NAI; Swarup et al., 2019 [rid=R85] ; Figures S2B [this file] and S2C [this file] ). \n",
      "In addition, all MNMs are preserved in three different transgenic mouse models expressing human mutations ( Swarup et al., 2019 [rid=R85] ; Figure S2D [this file] ) and in microglial-specific datasets from mouse models expressing ( Srinivasan et al., 2016 [rid=R80] ) and ( Wang et al., 2015 [rid=R90] ) mutations, with the exception of M_UP3, which is only weakly preserved in one of the two datasets from mice expressing early AD mutations ( Figure S2E [this file] ). A few modules display variability in their differential expression in different transgenic models, suggesting that they may be conditional on mutational interactions with disease-stage and disease-specific pathology, or possibly by other technical factors distinguishing these mouse models ( Figure S2F [this file] ). \n",
      "Finally, we observed an increased expression of genes involved in viral response and type II IFN response represented by M_UP3 ( Figures 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). \n",
      "We identified the earliest interconnected MNM genes present in pre-symptomatic disease tissue and called them early MNM sub-modules, reasoning that casual disease pathways should enrich among the earliest MNM components in disease ( Figure S3 [this file] ; STAR Methods [rid=S10] ). We confirmed significant associations using exome array data ( Chen et al., 2015 [rid=R9] ), replicating significant associations between AD and M_UP3 (β= 0.19, p < 0.001), FTD and early_UP1 (β= 0.25, p < 0.001), and FTD and early_DOWN1 (β= 0.15 p < 0.001) ( Figure S3G [this file] ). \n",
      "Consistent with these causal relationships defined by genetic enrichment, we were able to demonstrate that the expression of genes involved with viral defense, including the type I IFN response and dsRNA binding, were highly associated with the level of pathological tau phosphorylation in the brains of TPR50 mice ( Figures S3H [this file] – S3J [this file] ). \n",
      "We found that gene overexpression has inverse effects on modules A and B ( Figure 4D [rid=F4] ; 2,161 genes; STAR Methods [rid=S10] ), which we also confirmed at the levels of gene connectivity ( Figure S4A [this file] ) and PPI ( Figure S4B [this file] ). These patterns were confirmed in 12 independent transcriptome datasets ( Figures S4D [this file] – S4I [this file] ), indicating that these two modules represent robust, early microglia responses mediating opposing signaling pathways. \n",
      "Module A includes protein complexes and pathways of microglial innate neuroimmune activation, including the NLRP3 inflammasome ( , , , , and ) ( Ahmed et al., 2017 [rid=R2] ; Ising et al., 2019 [rid=R38] ; Lučiūnaitė et al., 2019 [rid=R54] ), TLRs ( , , , and ), and nuclear factor κB (NF-κB) targets ( Figures 4E [rid=F4] , 4G [rid=F4] , and S4A [this file] ). Supporting this interpretation, we found that purified pathological Aβ, an extracellular inducer of pro-inflammatory microglial activity ( Lučiūnaitė et al., 2019 [rid=R54] ), stimulates NAct in microglia ( Figures 4F [rid=F4] and S4C [this file] ). In contrast, module B includes viral and stress response genes, drivers of immune suppression ( [PD-L1], , , , and ), and the senescence marker, ( Figures 4H [rid=F4] and S4A [this file] ). \n",
      "Several features of NSupp suggest that in the context of tauopathy it may also represent an immunosuppressive type of IFN-β signaling in disease-associated microglia that are expressing inhibitory immune checkpoint genes (e.g., [PDL-1], , and ) ( Snell et al., 2017 [rid=R78] ; Wykes and Lewin, 2018 [rid=R94] ) and inhibitors of immune hyperactivity (e.g., ; Basters et al., 2018 [rid=R4] ; Taylor et al., 2018 [rid=R87] ; Figures 4G [rid=F4] , 4H [rid=F4] , and S4A [this file] ). \n",
      "NAct is activated by extracellular stimuli working through TLR and related receptors ( Figures 4E [rid=F4] , 4F [rid=F4] , S4A [this file] , and S4C [this file] ). \n",
      "Then, as tau pathology leads to chromatin and nucleotide dysregulation before neuronal loss, nucleotide damage signals activate dsRNA receptors to trigger a microglial type I IFN response that drives the expression of NSupp pathways involved in stress response, exosome, and viral entry ( Figure 4H [rid=F4] ), while suppressing the specific microglial immune attack pathways represented by NAct, including the NLRP3 inflammasome and TLR1 and −2 signaling ( Figures 4E [rid=F4] and S4A [this file] ). As disease progresses to include frank neurodegeneration, microglia undergo additional dynamic states, including increased type II IFN signaling, exogenous antigen presentation, T cell regulation, and cell death ( Figures 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). \n",
      "We pre-treated human-induced pluripotent stem cell (iPSC)-derived microglia ( Haenseler et al., 2017 [rid=R29] ; STAR Methods [rid=S10] ) with IFN-β and tested the ability of fibrillar Aβ, a NAct activator ( Figures 4F [rid=F4] and S4C [this file] ), to induce the secretion of IL-1β, a prominent member of the NAct pathway in these human cells ( Figures 4E [rid=F4] and 4G [rid=F4] ). \n",
      "Another prediction based on the mouse microglia module trajectory analysis ( Figure 2 [rid=F2] ) is a delayed, biphasic type II IFN response with early suppression and late increase (M_DOWN1, M_UP3; Figures 1C [rid=F1] , 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). Remarkably, we observed concordance between disease-associated signaling pathways represented in M_UP3 and microglia from patients with FTD-Pick’s; in particular in type II IFN, a specific activator of M_UP3 ( Figures 2A [rid=F2] , 2B [rid=F2] , 6C [rid=F6] , and S2G [this file] ). Significantly upregulated in bvFTD microglia are genes that can stimulate IFN-γ production ( , , , and ) ( Renkl et al., 2005 [rid=R66] ) and genes that mediate IFN-γ response ( , , , and ; Figures 6C [rid=F6] , 6D [rid=F6] , and S5E [this file] ). \n",
      "We find that both dsRNA detection and IFN pathways are highly correlated with pathological tau burden in brain ( Figures S3H [this file] – S3J [this file] ) and both the cytosolic dsRNA receptor (MDA5) and RIG1 pathway are hubs of the NSupp module ( Figure 4H [rid=F4] ). \n",
      "The significance of intramodular connectivity was assessed for each module using a permutation test (10,000 permutations), and all modules were confirmed to have permuted p value < 0.001. “Early submodules,” described in Figures 3 [rid=F3] and S3 [this file] , were derived by re-clustering M_UP1 and M_UP2 genes to generate “earlyUP” modules, or M_DOWN1, M_DOWN2 and M_DOWN3 genes to generate “earlyDOWN” modules, using the 3 month of age frontal cortex TPR50 gene expression data ( Swarup et al., 2019 [rid=R85] ). “NSupp and NAct,” described in Figures 4 [rid=F4] and 5 [rid=F5] , were derived from re-clustering the M_UP1 and M_UP2 genes, keeping only genes with annotated PPI in either Bio-grid ( Stark et al., 2006 [rid=R82] ) or Inweb databases ( Rossin et al., 2011 [rid=R69] ), using the same gene expression data used for the consensus WGCNA analysis (purified microglia from the Tg4510 model and frontal cortex TPR50 dataset (6 months of age)). \n",
      "Each cluster was then annotated as a major cell type using mean expression of groups of cell type marker genes, including the microglial markers CD74, CSF1R, C3 ( Mathys et al., 2019 [rid=R58] ), PTPRC, and DOCK8 ( Kelley et al., 2018 [rid=R42] ) ( Figure S5A [this file] ). \n",
      "In addition, to check that our findings were representative of group effects rather than driven by a few individuals, we compared all disease and control samples for average gene expression and total number of microglia contributed to the analysis ( Figure S5E [this file] ). \n",
      "For all boxplots, the center line is at the median and the upper and lower lines are at the 75 and 25 percentiles, respectively ( Figures 4I [rid=F4] , 5C [rid=F5] , 5F [rid=F5] , 5K [rid=F5] , S3B [this file] , S3C [this file] , S4G [this file] – S4I [this file] , and S5C [this file] ). Results of tissue culture experiments show each sample data point as a separate dot, the center line at the group mean, and error bars showing s.e.m. ( Figures 5D [rid=F5] , 5G [rid=F5] , and 5I [rid=F5] ) The barplot in Figure S2G [this file] has the center line at the group mean and error bars as s.e.m. In Figure S5D [this file] , the center line of the boxplot is the sample mean, the upper and lower lines are the 75 and 25 percentiles, respectively, and each dot is the average gene expression among the microglia from one subject, with the size of the dot proportional to the number of microglia contributing to the final analysis from that subject. \n",
      "See also Figure S1 [this file] . \n",
      "See also Figure S2 [this file] . \n",
      "See also Figure S3 [this file] . \n",
      "See also Figure S4 [this file] . \n",
      "See also Figure S5 [this file] and Table S3 [rid=SD4] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: We identified 13 distinct, robust co-expression modules, including 7 modules enriched for microglial genes, but varying in their relationship to disease, cell type, and temporal trajectories; we call these microglial neurodegeneration-associated modules, MNMs ( Figures 1 [rid=F1] , S1 [rid=SD1] , and S2 [rid=SD1] ; Table S1 [this file] ). \n",
      "To further explore the early microglial response in disease, we re-clustered only the genes from M_UP1 and M_UP2 to identify additional co-expression relationships among them ( Figure 4A [rid=F4] , Module A and Module B; Table S1 [this file] ). \n",
      "We next used the CMAP database to find candidate drivers of NSupp and NAct ( STAR Methods [rid=S10] ), identifying genes that regulate the viral activation of type I IFN among the top drivers of NSupp and suppressors of NAct ( ; Marazzi et al., 2012 [rid=R56] ; Table S1 [this file] ). \n",
      "For our analysis, we used the mean “connectivity scores” which is calculated from the combining data generated independently in 9 cell lines (see Table S1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: We compared them with six independent microglia-specific gene expression datasets from transgenic mouse models and humans with AD pathology to identify overlapping patterns of gene expression ( Figures 1B [rid=F1] , S1C [rid=SD1] – S1E [rid=SD1] , and S2F [rid=SD1] ; Table S2 [this file] ), supporting their generalizability. \n",
      "To explore their preservation across neurodegenerative dementias, we tested module preservation in multiple mouse and human disease datasets (total n = 525 samples from 17 datasets; STAR Methods [rid=S10] ; Table S2 [this file] ), finding significant preservation in transcriptome data from post-mortem human brain tissue from AD ( Allen et al., 2016 [rid=R3] ), FTD ( Chen-Plotkin et al., 2008 [rid=R10] ; Swarup et al., 2019 [rid=R85] ), and PSP ( Allen et al., 2016 [rid=R3] ) patients ( Figure 1G [rid=F1] ). \n",
      "Samples were balanced age (mean: 68 years old -Pick’s, 66 years old -control), PMI (mean: 15.98 hours – Pick’s, 12.78 hours - control) and tissue RNA integrity number (RIN; mean: 6.1 -Pick’s, 5.6 -controls) ( Table S2 [this file] ). \n",
      "Additional publicly available datasets were used throughout the study for validation or comparison ( Table S2 [this file] ; and Key Resources Table [rid=T1] ). \n",
      "One outlier sample was identified based on abnormal frequencies of major cell types and divergent gene expression patterns, and was removed from the analysis ( Table S2 [this file] ). \n",
      "While RIN, PMI and age were not significantly different between case and control subjects (Mann Whitney U test; Table S2 [this file] ); sex and brank bank were not evenly split across groups (UCSF brain bank –bvFTD = 70%, control = 12%; male sex –bvFTD = 62%, controls = 50% ( Table S2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: We used the top 250 genes associated with bvFTD (at significance thresholds of beta > 0.1 and p-adjusted < 5.0 E-04) for further analysis ( Table S3 [this file] ). We report the effect size output from the linear models as beta; which is approximate to the log2-fold change between groups ( Table S3 [this file] ). \n",
      "We identified all genes significantly associated with either UCSF brain bank or sex using the thresholds applied to diagnosis (beta 0.1, p value < 1.0 E-05, p-adjusted < 5.0 E-04) ( Table S3 [this file] ) and excluded them from further analysis. No genes were significantly associated with PMI or number of genes or UMI detected; and genes found to be significantly associated with RIN ( Table S3 [this file] ) were excluded from further analysis (beta >0.1, p value < 1.0 E-05, p-adjusted < 5.0 E-04). \n",
      "See also Figure S5 [rid=SD1] and Table S3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 278849\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 70464 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-dedd31d19774b32aa87ecdbb9dc0f3af1944967c491bb9772849bf5a23b4227e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 70464 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-57b3fa40-aa13-4ace-aae7-2b89628406b7\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE146866\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs002197.v1.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE90696\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75431\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE61499\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE61500\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65067\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103334\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57181\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE55627\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE98401\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE1432\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13162\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://repo-prod.prod.sagebase.org/repo/v1/doi/locate?id=syn3163039&type=ENTITY\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7818788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE138852\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE121654\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE98969\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"n/a\\\",\\n   \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE146866\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs002197.v1.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE90696\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE75431\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE61499\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE61500\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65067\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE103334\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57181\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE55627\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE98401\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE1432\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE13162\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://repo-prod.prod.sagebase.org/repo/v1/doi/locate?id=syn3163039&type=ENTITY\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7818788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE138852\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE121654\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE98969\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n   \\\"dataset_identifier\\\": \\\"n/a\\\",\\n   \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322174,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 87807,\n",
      "        \"completion_tokens\": 923,\n",
      "        \"total_tokens\": 88730,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE146866', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866', 'dataset_id': 'GSE146866'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE146866, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE146866', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'phs002197.v1.p1', 'repository_reference': 'dbgap', 'dataset_id': 'phs002197.v1.p1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: phs002197.v1.p1, dbgap, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'phs002197.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/dhglab/Tau-pathology-drives-dementia-risk-associated-gene-networks-towards-chronic-inflammatory-states-and\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE90696', 'repository_reference': 'geo', 'dataset_id': 'GSE90696'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE90696, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE75431', 'repository_reference': 'geo', 'dataset_id': 'GSE75431'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE75431, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75431', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE61499', 'repository_reference': 'geo', 'dataset_id': 'GSE61499'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE61499, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE61499', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE61500', 'repository_reference': 'geo', 'dataset_id': 'GSE61500'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE61500, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE61500\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE61500', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65067', 'repository_reference': 'geo', 'dataset_id': 'GSE65067'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE65067, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE103334', 'repository_reference': 'geo', 'dataset_id': 'GSE103334'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE103334, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE103334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE103334', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE57181', 'repository_reference': 'geo', 'dataset_id': 'GSE57181'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE57181, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE57181\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE57181', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE55627', 'repository_reference': 'geo', 'dataset_id': 'GSE55627'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE55627, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE55627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE55627', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE98401', 'repository_reference': 'geo', 'dataset_id': 'GSE98401'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE98401, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE98401', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE1432', 'repository_reference': 'geo', 'dataset_id': 'GSE1432'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE1432, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE1432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE1432', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE13162', 'repository_reference': 'geo', 'dataset_id': 'GSE13162'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE13162, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13162', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE138852', 'repository_reference': 'geo', 'dataset_id': 'GSE138852'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE138852, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE121654', 'repository_reference': 'geo', 'dataset_id': 'GSE121654'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE121654, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE121654', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE98969', 'repository_reference': 'geo', 'dataset_id': 'GSE98969'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE98969, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE98969', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE146866', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'phs002197.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE75431', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE61499', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE61500', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE103334', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE57181', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE55627', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE98401', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE1432', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13162', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE121654', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE98969', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 17 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE146866 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE146866 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE146866 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: dbgap, identifier: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for phs002197.v1.p1 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: dbgap and accession_id: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE90696 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75431 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75431 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE75431 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE61499 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE61499 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE61499 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE61500\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE61500\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE61500 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE61500 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE61500 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE61500\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61500\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE65067 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE103334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE103334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE103334 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE103334 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE103334 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE103334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE57181\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE57181\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE57181 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE57181 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE57181 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE57181\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57181\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE55627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE55627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE55627 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE55627 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE55627 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE55627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE55627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE98401 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE98401 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE98401 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE1432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE1432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE1432 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE1432 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE1432 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE1432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE1432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13162 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13162 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE13162 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE138852 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE121654 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE121654 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE121654 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE98969 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE98969 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE98969 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 17: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 17\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 22, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 45th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC3788619&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0cbbf50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#d33e24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for d33e24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#d33e30'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for d33e30: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#d33e36'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for d33e36: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 140130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35785 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ff930265d02f93067d757826556ccda5bdac0f113a3252ce030e736f6ee63a8f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35785 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2489d85f-b8fe-4c7c-9eda-86f5a8823d17\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2237931\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2237931\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322189,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 49064,\n",
      "        \"completion_tokens\": 88,\n",
      "        \"total_tokens\": 49152,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1729383', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn1729383'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1729383, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2237931', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn2237931'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2237931, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2237931\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2237931', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2237931', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1729383 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2237931\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2237931\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2237931 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2237931 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2237931 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2237931 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2237931 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2237931\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2237931\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 46th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6750256&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b147f710>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 17 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S20: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S22'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S22: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S31'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S31: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 45 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 7597) for SD1: We compiled ~40 million possible variants based on the union of SNPs called from RNA-seq data ( Supplementary Fig. 1a [href=this file] ) and SNPs already identified from microarray probe sets and the 1000 Genomes (Methods). To collect transcriptome wide genotyping data from our brain samples, we performed imputation from genotyping array and RNA-seq based genotyping (Methods; Supplementary Fig. 1b [href=this file] and 1c [href=this file] ), and ancestry was identified for all samples (Methods; Supplementary Fig. 1d [href=this file] ).\n",
      "The values of excluded covariates showed balanced distributions between control and ASD ( Supplementary Fig. 1e [href=this file] ; Methods).\n",
      "As expected, SNPs within the same haplotype blocks show similar allelic expression patterns ( Supplementary Fig. 1f [href=this file] and 1g [href=this file] ).\n",
      "Although this database contains data from mostly non-neural tissues, genes manifesting ASE in our study showed significant overlap with those in dbMAE ( Supplementary Fig. 2a [href=this file] ). For example, the overlap between genes showing ASE in the two cortical regions, BA9 and BA41 is extremely high (odds ratio (OR)=221.00 and p-value<2.2e-16; Supplementary Fig. 2b [href=this file] ), whereas the degree of overlap between two more developmentally and evolutionarily distinct brain regions, cortex and cerebellum, is substantial, but lower than in the two more similar cortical regions (between BA9 and vermis: OR=49.00 and p-value<2.2e-16; between BA41 and vermis: OR=72.54 and p-value<2.2e-16).\n",
      "For further external validation, we applied the specific methods and pipeline used in the primary analysis of GTEx ( Supplementary Fig. 2c [href=this file] ).\n",
      "The distribution of expression of imbalanced alleles was significantly shifted lower than balanced alleles (Kolmogorov-Smirnov (KS) test, p-value<2.0e-16, the means of log (RPSM): 0.37 vs 0.71; Supplementary Fig. 2d [href=this file] ). Genes showing evidence for ASE were also preferentially enriched for neuronal and astrocyte markers ( Fig. 2b [href=#F2] and Supplementary Fig. 2e [href=this file] ). Remarkably, across all three brain regions, ASE genes are enriched for genes previously shown to be down-regulated in postmortem brain from patients with ASD ( Supplementary Fig. 2f [href=this file] ).\n",
      "Similar to controls, idiopathic ASD cases showed tissue specific ASE across the three brain regions ( Fig. 3a [href=#F3] , Supplementary Fig. 3a [href=this file] , and Supplementary Fig. 3b [href=this file] ).\n",
      "Genes exhibiting control-specific ASE are significantly enriched for pathways related to growth cone and synapse ( Supplementary Fig. 3c [href=this file] and Supplementary Table 4 [href=#SD6] ). Though there are fewer idiopathic ASD-specific ASE genes than control-specific ASE genes, those showing idiopathic ASD-specific ASD are significantly enriched for pathways involved in transport ( Supplementary Fig. 3d [href=this file] and Supplementary Table 4 [href=#SD6] ). Genes exhibiting ASE in both idiopathic ASD and controls show significant enrichment in genes harboring ASD risk variants ( Supplementary Fig. 3e [href=this file] ; OR=1.68, p-value=5e-04 for ASD1; OR=3.83, p-value=0.001 for ASD2). However, control-specific and idiopathic ASD-specific ASE genes do not show any enrichment with either the rare ASD risk variants or common variants from GWAS data ( Supplementary Fig. 3e [href=this file] ).\n",
      "Controls had a higher density of balanced alleles than ASD cases while idiopathic ASD cases showed more imbalanced alleles ( Supplementary Fig. 4a [href=this file] and Fig. 4a [href=#F4] ). However, controls showed significantly more major allele MAE for low frequency alleles (minor allele frequency (MAF)<0.05) compared to idiopathic ASD ( Supplementary Fig. 4b [href=this file] ). These patterns were recapitulated in the pseudoautosomal region (PAR) of the chromosome X ( Supplementary Fig. 4c [href=this file] ).\n",
      "We next investigated whether the differential ASE pattern was related to the MAF genome-wide ( Fig. 4b [href=#F4] and Supplementary Fig. 4b [href=this file] ).\n",
      "Although MAE SNPs were observed across the genome ( Supplementary Fig. 5 [href=this file] ), there was an approximately 2-fold enrichment on chromosome 15 harboring a number of known imprinted loci ( Supplementary Table 5 [href=#SD2] ). This genome-wide enrichment pattern persisted when SNPs were analyzed at individual genes ( Supplementary Fig. 6a [href=this file] and 6b [href=this file] ).\n",
      "For example, genes showing minor allele MAE showed no overlap with astrocyte markers , PSD , or ASD-specific down-regulated genes ( Supplementary Fig. 6c [href=this file] ). Like genes that manifest ASE ( Fig. 2c [href=#F2] ), major allele MAE genes show significant enrichment with genes associated with the rare ASD risk variants ( Supplementary Fig. 6d [href=this file] ). However, MAE genes in control and ASD showed some enrichment with common variants from GWAS data of BPD and SCZ ( Supplementary Fig. 6d [href=this file] ).\n",
      "This analysis identities only two genomic regions, 14q32 (chr14:101302,638–101,544,745) and 15q11 (chr15:25,223,730–25,582,395), which are known imprinted loci that have many small splice junctions ( Fig. 6 [href=#F6] ) and show continuous high expression ( Supplementary Fig. 7 [href=this file] ) as single transcript units .\n",
      "The snoRNA targeting sites were located significantly proximal to alternatively spliced junctions of their target genes in ASD ( Supplementary Fig. 8a [href=this file] ), and their splicing changes show significant correlations with the expression changes of their specific snoRNAs ( Supplementary Fig. 8b [href=this file] and 8c [href=this file] ). When one snoRNA has two different targets, their splicing changes also show strong correlation ( Supplementary Fig. 8d [href=this file] ).\n",
      "Compared with the previously published RNA-seq based genotyping methods by Quinn et al. , these genotyping parameters show much higher accuracy with sensitivity exceeding 95% ( Supplementary Fig. 1b [href=this file] ) and specificity in the range of 90–95% ( Supplementary Fig. 1c [href=this file] ; see Quinn at el. for accuracy tests).\n",
      "We categorized our samples into European, Mexican, Asian, African, and ambiguous groups ( Supplementary Fig. 1d [href=this file] ).\n",
      "In BA9 samples, the distributions of the covariate values were balanced between control and ASD ( Supplementary Fig. 1e [href=this file] ).\n",
      "We calculated deviations of log (fold change) values (the beta values of allele covariate from the group based ASE identifications) per haplotype block in cortex and compared their distributions with normal distributions ( Supplementary Fig. 1f [href=this file] and 1g [href=this file] ). For further validation, we compared ASE genes identified in our study with those present in dbMAE ( https://mae.hms.harvard.edu/ [href=https://mae.hms.harvard.edu/] ) using Fisher’s exact test ( Supplementary Fig. 2a [href=this file] ). Like GTEx data, we calculated ASE rates per tissue ( Supplementary Fig. 2c [href=this file] ).\n",
      "We compared their log (RPSM) values using KS tests ( Supplementary Fig. 2d [href=this file] ).\n",
      "We visualized their distributions using ggplot2 R package ( http://ggplot2.org/ [href=http://ggplot2.org/] ) and compared them between control and idiopathic ASD at autosome ( Fig. 4a [href=#F4] and Supplementary Fig. 4b [href=this file] ) and chromosome X ( Supplementary Fig. 4c [href=#SD7] ).\n",
      "For major and minor allele MAE SNPs, we visualized them on the different lines ( Supplementary Fig. 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 532) for SD2: Although MAE SNPs were observed across the genome ( Supplementary Fig. 5 [href=#SD1] ), there was an approximately 2-fold enrichment on chromosome 15 harboring a number of known imprinted loci ( Supplementary Table 5 [href=this file] ).\n",
      "Compared to chromosome 14, chromosome 15 exhibited a substantially higher minor allele MAE fraction in both ASD and dup15q than controls (3.9- and 1.6-fold, respectively; chi-square test p-values<2.2e-16 and 0.0184, respectively; Fig. 5c [href=#F5] and Supplementary Table 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 7 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1249) for SD3: For investigation of group-level ASE patterns across cases and brain regions, 1,163,249 heterozygous SNPs in 21,929 genes were used following quality control (Methods), and their significance was quantified using a linear mixed model to account for sample variation and covariates ( Supplementary Table 1 [href=this file] ; Methods).\n",
      "Though most sample information was already published , we provided it again including further information ( Supplementary Table 1 [href=this file] ).\n",
      "To increase their SNP calling accuracy, we optimized bam files following the recommended GATK pipeline ( Supplementary Fig. 1a [href=this file] ).\n",
      "To ensure if groups were balanced with respect to these covariates, we removed samples based on age (≦7), brain banks (The UK Brain Bank for Autism and Related Developmental Research and the MRC London Neurodegenerative Diseases Brain Banks), and SNP annotation (mysterious) ( Supplementary Table 1 [href=this file] ) . GC content was tested using AT and GC dropout values from Picard ( https://broadinstitute.github.io/picard/ [href=https://broadinstitute.github.io/picard/] ; Supplementary Table 1 [href=this file] ).\n",
      "The detailed description of brain samples is provided in Supplementary Table 1 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 341) for SD4: Furthermore, although assessed separately in each brain region, there was strong concordance in common ASE genes between regions in our data ( Fig. 2a [href=#F2] and Supplementary Table 2 [href=this file] ).\n",
      "For each tissue, group-based ASE identification results are available in Supplementary Tables 2 [href=this file] and 3 [href=#SD5] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#SD5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 361) for SD5: The majority of genes exhibiting ASE (70.41%) are shared across the case and control groups providing further confidence in the reproducibility of these results ( Fig. 3c [href=#F3] and Supplementary Table 3 [href=this file] ).\n",
      "For each tissue, group-based ASE identification results are available in Supplementary Tables 2 [href=#SD4] and 3 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 4 <a> elements with href='#SD6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 769) for SD6: The p-values and other results of the GO analysis are at Supplementary Table 4 [href=this file] .\n",
      "Genes exhibiting ASE in both idiopathic ASD and control are enriched in phosphoprotein, RNA splicing, and neuron projection pathways ( Fig. 3e [href=#F3] and Supplementary Table 4 [href=this file] ). Genes exhibiting control-specific ASE are significantly enriched for pathways related to growth cone and synapse ( Supplementary Fig. 3c [href=#SD1] and Supplementary Table 4 [href=this file] ). Though there are fewer idiopathic ASD-specific ASE genes than control-specific ASE genes, those showing idiopathic ASD-specific ASD are significantly enriched for pathways involved in transport ( Supplementary Fig. 3d [href=#SD1] and Supplementary Table 4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 9 <a> elements with href='#SD7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 2164) for SD7: For known ASD risk gene list (SFARI gene), we selected genes from category 1, 2, and 3 corresponding to gene scoring criteria ( https://gene.sfari.org/autdb/GS_Home.do [href=https://gene.sfari.org/autdb/GS_Home.do] ); Supplementary Table 6 [href=this file] ) The gene list was previously used at the previous publication .\n",
      "For this gene set enrichment studies, we also used known imprinted genes ( www.geneimprint.com [href=http://www.geneimprint.com/] ), PSD , FMRP target , HuR target , and RBFOX1 target genes , cell marker genes (neuron, astrocyte, oligodendrocyte, microglia, and endothelial), expression up- and down-regulated genes in ASD cortex ( Supplementary Table 6 [href=this file] ). HuR target gene list was collected from photoactivatable ribonucleoside crosslinking and immunoprecipitation (PAR-CLIP) data instead of RNA binding proteins immunoprecipitation complementary DNA array (RIP-chip) data since PAR-CLIP data identified RNA binding proteins binding sites more precisely than RIP-chip ( Supplementary Table 6 [href=this file] ).\n",
      "For the variant data, we used the risk genes containing rare likely gene disrupting mutations in SCZ, ID, and ASD from the dataset of Iossifov et al. ( Supplementary Table 6 [href=this file] ). For the risk gene list, we analyzed gene set enrichment with different FDR cutoffs (FDR≦0.01, 0.01<FDR≦0.05, and 0.05<FDR≦0.1) and found the highest enrichments using the most stringent FDR cutoff. of FDR≦0.01, which we report in our study ( Supplementary Table 6 [href=this file] ).\n",
      "For both studies, brain expressed genes were used as background genes that have more than 10 allele counts ( Supplementary Table 6 [href=this file] ).\n",
      "We visualized their distributions using ggplot2 R package ( http://ggplot2.org/ [href=http://ggplot2.org/] ) and compared them between control and idiopathic ASD at autosome ( Fig. 4a [href=#F4] and Supplementary Fig. 4b [href=#SD1] ) and chromosome X ( Supplementary Fig. 4c [href=this file] ).\n",
      "We also include gene lists that we used for gene set enrichment and GO analysis that includes brain expressed gene that we used for our study ( Supplementary Table 6 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 199861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 50717 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3d82b79f177efb394edf8314ca650806d65b235f15e304d754b867aa8b8ad07e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 50717 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7adef698-89b1-4f79-b14f-5811b315b1da\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Raw RNA-seq\\\",\\n    \\\"repository_reference\\\": \\\"http://dx.doi.org/10.7303/syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"sncRNA-seq\\\",\\n    \\\"repository_reference\\\": \\\"http://dx.doi.org/10.7303/syn4587609\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Raw RNA-seq\\\",\\n    \\\"repository_reference\\\": \\\"http://dx.doi.org/10.7303/syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"sncRNA-seq\\\",\\n    \\\"repository_reference\\\": \\\"http://dx.doi.org/10.7303/syn4587609\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322208,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 68186,\n",
      "        \"completion_tokens\": 113,\n",
      "        \"total_tokens\": 68299,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Raw RNA-seq', 'repository_reference': 'http://dx.doi.org/10.7303/syn4587609', 'dataset_id': 'Raw RNA-seq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4587609, None, http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://dx.doi.org/10.7303/syn4587609, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://dx.doi.org/10.7303/syn4587609 in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'sncRNA-seq', 'repository_reference': 'http://dx.doi.org/10.7303/syn4587609', 'dataset_id': 'sncRNA-seq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4587609, None, http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://dx.doi.org/10.7303/syn4587609, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for http://dx.doi.org/10.7303/syn4587609 in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 9\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 47th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11370573&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b764d900>, <Element supplementary-material at 0x1b120b780>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b764d900>, <Element supplementary-material at 0x1b120b780>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: Clones with their genomic features of interest can be found in supplementary table 5 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: All 19 patients had cfDNA collected at or close to the time of first debulking surgery or laparoscopic biopsy (baseline), and a subset (n=10) had radiographically confirmed disease recurrence along with cfDNA collections post-recurrence and during therapy ( Supplementary Figure 1 [this file] ). See Supplementary Figure 1 [this file] for clinical details, treatment history and sample collections for all 19 patients. \n",
      "From the 19 patients included in this study we generated scWGS data from 19,454 cells (range 200-2015 cells per patient, Supplementary Figure 2 [this file] ) with mean coverage of 0.089X (range 0.002-0.392X per cell, Supplementary Figure 2). We inferred the clonal composition at the time of diagnosis based on copy number data ( Methods [rid=S11] ), with the aim of following these clones over time as patients received chemotherapy, maintenance therapies and experienced disease recurrences using cfDNA ( Supplementary Figure 3 [this file] ). \n",
      "We then merged cells from each clone and re-computed copy number at 10kb resolution using a new Hidden Markov Model (HMM) based copy number caller, ( Methods [rid=S11] ). improves the resolution of pseudobulk clone copy number profiles and enables more precise matching between copy number and SVs ( Supplementary Figure 4 [this file] ). \n",
      "For each patient, we constructed a panel of mutations comprising a mix of clonal and subclonal somatic SNVs/SVs (≥100 SVs per patient, Supplementary Figure 2 [this file] ) and a small number of germline single nucleotide polymorphisms (SNPs) for QC purposes. \n",
      "Clone-specific SVs inferred by scWGS analysis were present in all patients with at least 200 cells (18 patients, average n=144, range 29-361 Supplementary Figure 2 [this file] ). \n",
      "See Supplementary Figure 5 [this file] for the scWGS data for these 10 patients. For all patients, ctDNA VAF of truncal SVs decreased during initial chemotherapy, as patients responded to therapy with decreased burden and decreased serum CA-125 levels ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). All patients were positive for ctDNA at the time point closest to first recurrence (defined as average VAF across the panel exceeding LOD ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] )). In 6 patients with sufficient plasma samples, ctDNA was detected prior to clinically confirmed disease recurrence but subsequent to completion of initial chemotherapy: 002 (76 days), 004 (26 days), 009 (184 days), 045 (109 days), 075 (233 days), 081 (314 days) ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). \n",
      "Copy number profiles from these data were consistent with predictions derived from scWGS derived copy number profiles and ctDNA clone frequencies ( Supplementary Figure 7a [this file] – c [this file] ). In patient 045, the locus (chr19q) was enriched at baseline, and (chr1q) at recurrence ( Supplementary Figure 7d [this file] ), while in patient 107, and a region on chr20q were enriched at recurrence as expected ( Supplementary Figure 7e [this file] ). Clone frequencies across time for patient 045 showed highly similar patterns for SVs ( Supplementary Figure 7f [this file] ) and SNVs ( Supplementary Figure 7g [this file] ) and were consistent using these two distinct sets of genomic features for all patients ( Supplementary Figure 7h [this file] , R=0.93, p<10 , Pearson correlation). \n",
      "In 6/10 patients (002, 006, 014, 045, 081, 107; see Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ), multiple clones were detected in post-recurrence plasma samples, highlighting that chemo-resistance may be polyclonal in many patients. In 4 of these cases (009, 045, 081 and 107), clones that were not detected in cfDNA or were detected at minor frequencies at baseline became dominant at first recurrence, suggesting that although multiple clones may become chemo-resistant, some have relative fitness advantages in the context of treatment ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). Notably, while the presence of multiple resistant clones was a common observation, the overall diversity, as quantified by Shannon entropy, decreased in the final timepoint relative to baseline in 8/9 cases (p=0.027, t-test, Supplementary Figure 6h [this file] ). The number of clones detected also decreased at the final time point relative to baseline (p=0.086, t-test, Supplementary Figure 6i [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 138696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35426 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-1e7a57aa369847147fb60d5741a3effe4ecd4ffe9fd1ca66ce0a8ce14488a1a8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35426 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d623754b-c6db-4a23-a303-e6852d974850\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25569736\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25569736\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322210,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41659,\n",
      "        \"completion_tokens\": 64,\n",
      "        \"total_tokens\": 41723,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25569736', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053', 'dataset_id': 'syn25569736'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25569736, synapse.org, https://www.synapse.org/[#!]*Synapse:syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25569736 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 48th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4344205&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b7277e10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 13 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec006'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 25) for sec006: Results [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0117658.s001'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 80) for pone.0117658.s001: S1 [href=this file] – S4 [href=#pone.0117658.s004] Tables show the raw results.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0117658.s002'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 148) for pone.0117658.s002: Yet, as we can see from S2 Table [href=this file] , we can get fairly good prediction (concordance index equal to 0.654) using the top 50 features.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#pone.0117658.s003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for pone.0117658.s003: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0117658.s004'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 80) for pone.0117658.s004: S1 [href=#pone.0117658.s001] – S4 [href=this file] Tables show the raw results.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0117658.s005'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 175) for pone.0117658.s005: S5 Table [href=this file] shows the top 150 features extracted by ReliefF from the entire data set for Method 1 and Method 2, and for 5 year, 10 year, and 15 year prediction.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 122128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31284 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c0eddd2c88eeebad03b939ab4ff2f0cb598240ce0f5390c6e593798b00fb66ff\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31284 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-413d8cf8-f9f8-4ec7-b473-530b8b8635c3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi:10.7303/syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"doi:10.7303/syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322225,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41154,\n",
      "        \"completion_tokens\": 54,\n",
      "        \"total_tokens\": 41208,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'doi:10.7303/syn1688369', 'repository_reference': 'synapse.org', 'dataset_id': 'doi:10.7303/syn1688369'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: doi:10.7303/syn1688369, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: doi:10.7303/syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'doi:10.7303/syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'doi:10.7303/syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: doi:10.7303/syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: doi:10.7303/syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier doi:10.7303/syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier doi:10.7303/syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier doi:10.7303/syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier doi:10.7303/syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for doi:10.7303/syn1688369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier doi:10.7303/syn1688369 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: doi:10.7303/syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:doi:10.7303/syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 49th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9630780&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b66c88d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 20 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#fn1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for fn1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec5.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec5.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#sec6.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 147) for sec6.1: Any additional information required to reanalyze the data reported in this paper is available from the lead contact [href=this file] upon request.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#da00101'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 44) for da00101: Data and code availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 140781\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 35947 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e2415970a725c6a6dc76c8c2ad35d78c24f982d8dc57767a27751d9b317a1469\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35947 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d4355d7f-bb33-4baa-a2fb-15cda2b5e1bc\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20843407\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn20843407/wiki/597068\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn20843407\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542449\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542450\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34539904\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542574\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25792906\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25886762\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25876213\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/CAROM\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20843407\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn20843407/wiki/597068\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn20843407\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542449\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542450\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34539904\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn34542574\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25792906\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25886762\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25876213\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/CAROM\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322240,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 48571,\n",
      "        \"completion_tokens\": 442,\n",
      "        \"total_tokens\": 49013,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20843407', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20843407/wiki/597068', 'dataset_id': 'syn20843407'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20843407, synapse.org, https://www.synapse.org/#!Synapse:syn20843407/wiki/597068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20843407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn20843407', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn20843407'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn20843407, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn20843407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn34542449', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn34542449, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542449', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn34542450', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542450'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn34542450, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn34539904', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34539904'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn34539904, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn34539904', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn34542574', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542574'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn34542574, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542574', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25792906', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25792906'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25792906, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25792906', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25886762', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25886762'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25886762, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25886762', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25876213', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25876213'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn25876213, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn25876213', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/CAROM', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/CAROM'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://www.synapse.org/CAROM\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://www.synapse.org/CAROM\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org/CAROM, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/CAROM\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org/CAROM in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://www.synapse.org/CAROM could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.synapse.org/[#!]*Synapse:__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 914 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 918 - INFO - Extracted ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://www.synapse.org/CAROM is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://www.synapse.org/CAROM', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/CAROM'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn20843407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn20843407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34542449', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34542450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34539904', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34542574', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25792906', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25886762', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25876213', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20843407 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn20843407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn20843407 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.7303/syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn20843407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn34542449 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn34542450 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn34539904 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn34542574 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25792906 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25886762 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn25876213 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 29, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 10\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 50th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10813059&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <supplementary-material xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.or...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b75cd580>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b75cd580>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref biomolecules-14-00066-s001: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 206446\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 52364 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-28bf069e6fe4f9b6f6e01d0fb305e31e720d7cdb46b1553cc1a34a7a3ac8e894\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 52364 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-99724e69-9860-43d8-9f62-c171d87f819c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52138250\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31952297\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31947506\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52138250\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31952297\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31947506\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322243,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 61768,\n",
      "        \"completion_tokens\": 137,\n",
      "        \"total_tokens\": 61905,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn52138250', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn52138250'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn52138250, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn52138250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn31952297', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31952297'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn31952297, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn31952297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn31947506', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31947506'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn31947506, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn31947506', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn52138250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn31952297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn31947506', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn52138250 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn31952297 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn31947506 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 51/100 (51.00%) | Elapsed: 00:07:52 | ETA: 00:07:33\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 51th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6089972&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1390a10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 28 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec20: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec21'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec21: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec21'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec21: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 7 <a> elements with href='#MOESM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 2417) for MOESM1: Analysis of residuals (method estimate—ABSOLUTE estimate) suggested close concordance with ABSOLUTE estimates for MethylCIBERSORT, with larger deviations only seen when samples were of very high purity (>80%), while other methods tended to overestimate tumour cell content in samples of low purity (Supplementary Figure 1a [href=this file] ), resulting in statistically significant differences in distributions (FDR < 2.2e−16).\n",
      "We also compared the mRNA expression of a panel of cellular lineage markers with MethylCIBERSORT estimates and found significant associations for multiple cell types (Supplementary Figure 1b [href=this file] ) even though they are derived from different samplings of the same tumour.\n",
      "For nine of these we were able to compare our predictions of tumour purity with ABSOLUTE estimates and observed strong correlations and significantly lower error margins compared to LUMP and ESTIMATE (Supplementary Figure 2a, b [href=this file] ). Further, we observed a robust preservation of positive correlations between MethylCIBERSORT and marker expression pan-cancer (Supplementary Figure 2c [href=this file] ), again with the caveat that the samples were taken from different aliquots of the tumour. We therefore compared estimates pertaining to fresh frozen and matched FFPE samples ( = 21 from three tumours) and recorded very high correlations, indicating our method is applicable also to the archival material (Supplementary Figure 2d [href=this file] ).\n",
      "Among the genomic alterations we identify (for a full list of predicted driver events see Supplementary Data 8 [href=this file0] ), it is likely that some establish, while others are selected for, in different immune microenvironments. The positive correlation between EGFR levels (which are themselves correlated with EGFR phosphorylation (activation), Supplementary Figure 3 [href=this file] ) and the glycolytic signature is maintained across TCGA when matched RPPA profiles and RNA-seq data are compared (Fig. 5f [href=#Fig5] ).\n",
      "Signature features were derived from 450k profiles using the aforementioned heuristic (delta-Beta and FDR cutoffs) with a maximum of 100 features per cell type for a wide range of tumour types, using cell lines allocated to the corresponding tissue in GSE68379 [href=https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379] (Table S10 [href=this file] ) and the aforementioned infiltrating cell types.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for MOESM2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#MOESM3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 825) for MOESM3: Using limma-trend analysis, we identified 458 differentially expressed genes (DEGs) between the hot and cold clusters at a fold-change of greater than 2 (FDR < 0.01, Supplementary Data 1 [href=this file] , genes highlighted in bold). In extended analyses of all genes at FDR < 0.01 (Supplementary Data 1 [href=this file] ), multiple other genes, including the Class 1 MHC gene (FC = 1.39), PD-1 ligand (FC = 1.62) and , which encodes SMA (FC = 0.68), are also differentially expressed between the two clusters, the latter validating fibroblast estimates from MethylCIBERSORT (Fig. 2a [href=#Fig2] ).\n",
      "Fas-ligand ( ), an upstream activator of pro-apoptotic signalling through Caspase 8 is also upregulated in the immune hot tumours, further highlighting the importance of this pathway (Supplementary Data 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 332) for MOESM4: Ingenuity Pathway Analysis further confirmed observations made using MethylCIBERSORT estimates, identifying differential regulation of multiple canonical pathways associated with immune function and inflammatory conditions (Supplementary Data 2 [href=this file] ), consistent with differential lymphocyte infiltration and activity.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 191) for MOESM5: Diseases and functions ontology (Supplementary Data 3 [href=this file] ) indicated that the top few pathways activated in hot tumours were associated with leukocyte and lymphocyte migration.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 209) for MOESM6: Upstream regulatory analysis implicated increased activation of STAT1 and IRF1, and inhibition of Interferon-stimulated transcription mediated by IRF4, in hot tumours (Supplementary Data 4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 152) for MOESM7: Finally, analysis of RPPA data identified 10 differentially abundant (FDR < 0.1) proteins or phospho-proteins (Supplementary Data 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 250) for MOESM8: We identified 365 genes at FDR < 0.01, FC > 2 and in pathway analysis, the top pathways were significantly associated with T-helper 1 (Th1) versus T-helper 2 (Th2) lymphocyte responses (Fig. 3e [href=#Fig3] , Supplementary Data 6 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 409) for MOESM9: Next, we examined if the genomic features associated with immune cluster were also reproducible across cancer types, performing adjusted binomial regressions to estimate cluster association after controlling for tumour type for genes previously implicated as pan-cancer drivers based on signatures of positive selection and recorded 56 hits at FDR < 0.05 (Supplementary Data 7 and Fig. 5c [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#MOESM10'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 235) for MOESM10: Among the genomic alterations we identify (for a full list of predicted driver events see Supplementary Data 8 [href=this file] ), it is likely that some establish, while others are selected for, in different immune microenvironments.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#MOESM11'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for MOESM11: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for ack1: Acknowledgements [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 199452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 50615 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-5144befa22fed490674b5f3e236ffd8e36b6a21d42afe98f1ff1408ed8a020b2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 50615 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-48811110-ac04-4523-bb6b-63e4d5624fc6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE74877\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38270\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE68379\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38266\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38266\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB11844\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJEB11844\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJNA258216\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJNA258216\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB5468\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJEB5468\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7214402\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7214402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7343873\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7343873\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2812961\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2812961\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE35069\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE74877\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38270\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE68379\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE38266\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38266\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB11844\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJEB11844\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJNA258216\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJNA258216\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB5468\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/data/view/PRJEB5468\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7214402\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7214402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7343873\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7343873\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2812961\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2812961\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE35069\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322259,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 67736,\n",
      "        \"completion_tokens\": 635,\n",
      "        \"total_tokens\": 68371,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE74877', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877', 'dataset_id': 'GSE74877'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE74877, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE74877', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE38270', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270', 'dataset_id': 'GSE38270'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE38270, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38270', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE68379', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379', 'dataset_id': 'GSE68379'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE68379, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE68379', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE38266', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38266', 'dataset_id': 'GSE38266'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE38266, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38266', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB11844', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB11844', 'dataset_id': 'PRJEB11844'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PRJEB11844 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJEB11844, identifier: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB11844', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PRJNA258216', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJNA258216', 'dataset_id': 'PRJNA258216'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PRJNA258216 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJNA258216, identifier: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PRJNA258216', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB5468', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB5468', 'dataset_id': 'PRJEB5468'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PRJEB5468 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJEB5468, identifier: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB5468', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn7214402', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7214402', 'dataset_id': 'syn7214402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn7214402, synapse.org, https://www.synapse.org/#!Synapse:syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn7214402', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn7343873', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7343873', 'dataset_id': 'syn7343873'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn7343873, synapse.org, https://www.synapse.org/#!Synapse:syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn7343873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2812961', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2812961', 'dataset_id': 'syn2812961'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2812961, synapse.org, https://www.synapse.org/#!Synapse:syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2812961', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE35069', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069', 'dataset_id': 'GSE35069'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE35069, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE35069', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE74877', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38270', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE68379', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38266', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB11844', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJNA258216', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB5468', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn7214402', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn7343873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2812961', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE35069', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 11 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE74877 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE74877 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE74877 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38270 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38270 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE38270 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE68379 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE68379 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE68379 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38266 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38266 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE38266 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PRJEB11844 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: www.ebi.ac.uk and accession_id: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PRJNA258216 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: www.ebi.ac.uk and accession_id: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PRJEB5468 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: www.ebi.ac.uk and accession_id: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn7214402 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn7343873 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2812961 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE35069 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE35069 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE35069 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 11\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 39, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 23\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 52th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8130821&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b10af250>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 9 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 24 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 4412) for SD2: In the discovery phase, we performed a PWAS by integrating AD GWAS results (N=455,258) with 376 human brain proteomes profiled from the dorsolateral prefrontal cortex (dPFC; Supplementary Table 1a [href=this file] ) using the FUSION pipeline . The PWAS identified 13 genes whose -regulated brain protein levels were associated with AD at false discovery rate (FDR) p<0.05 ( Figure 1 [href=#F1] , Table 1 [href=#T1] , Extended Data Figure 1a [href=#F2] , Supplementary Table 2 [href=this file] ).\n",
      "A confirmatory PWAS was performed using the same AD GWAS and an independent set of 152 human brain proteomes profiled from the dPFC ( Supplementary Table 1b [href=this file] ) . Correlation between the protein weights in the discovery and confirmatory datasets was high (median 0.85, interquartile range 0.21; Supplementary table 3 [href=this file] ). Ten of these 13 proteins could be tested and all 10 proteins replicated in the confirmatory PWAS ( Table 1 [href=#T1] ; Extended Data Figure 1b [href=#F2] ; Supplementary table 4 [href=this file] ).\n",
      "We found 9 of 13 genes consistent with being causal ( Table 2 [href=#T2] ; Supplementary Table 5 [href=this file] ). SMR results suggest that the -regulated protein abundance mediates the association between genetic variants and AD for all these 13 genes, but HEIDI results argue against a causal role for 4 genes due to linkage disequilibrium ( Table 2 [href=#T2] ; Supplementary Table 6 [href=this file] ).\n",
      "Full results for TWAS is in Supplementary tables 17 [href=this file] - 18 [href=this file] . “ ” in “TWAS significant” column refers to genes with 0.05 <TWAS nominal p <0.1. refers to genes not within 1Mb window of SNPs with p < 5E-08 identified in Jansen et. al.\n",
      "That analysis found the 13 original PWAS-significant genes and 6 additional significant genes at FDR p<0.05 ( , , ; Supplementary table 7 [href=this file] ). Moreover, results from COLOC and SMR/HEIDI tests found the same evidence of causality as the original findings except that was now consistent with causality by both COLOC and SMR/HEIDI compared to mixed findings before ( Supplementary tables 8 [href=this file] - 9 [href=this file] ). The 6 additional genes were not consistent with being causal by COLOC ( Supplementary Table 8 [href=this file] ).\n",
      "The PWAS of clinical AD identified 4 genes, ALS 7 genes, PD 17 genes, neuroticism 72 genes, height 662 genes, BMI 395 genes, and WHRadjBMI 244 genes ( Supplementary Tables 10 [href=this file] - 16 [href=this file] ).\n",
      "The 888 transcriptomes were mainly from the frontal cortex donated by participants of European descent ( Supplementary table 1c [href=this file] ), and quality control was analogous to that of the proteomes to remove technical and clinical characteristics before estimating the effect of genetic variants on mRNA expression. The AD TWAS identified 40 genes whose genetically regulated mRNA expression levels were associated with AD at FDR p<0.05 ( Extended Data Figure 3 [href=#F4] ; Supplementary table 17 [href=this file] ). Among the 11 potentially causal genes identified at the protein-level, five genes, , , , , and showed at least nominal significance with similar directions of association with AD as seen at the protein-level ( Table 3 [href=#T3] ; Supplementary table 18a [href=this file] ).\n",
      "For the 5 genes with evidence at both the transcript and protein levels, results from SMR test for two molecular traits suggested their protein abundance is mediated by mRNA expression ( Supplementary table 18a [href=this file] , b [href=this file] ). For the three genes with suggestive evidence for -regulated mRNA’s association with AD ( , , and ), only had evidence to suggest protein expression is mediated by mRNA expression level ( Supplementary table 18a [href=this file] , b [href=this file] ).\n",
      "Using human single-cell RNA-sequencing data profiled from the dPFC we found cell-type specific enrichment for expression of 6 of the 11 causal genes at FDR p-value < 0.05 (adjusted for 17,775 genes). , , , and were enriched in excitatory neurons, whereas showed enrichment in oligodendrocytes and in astrocytes and microglia ( Extended Data Figure 4 [href=#F5] ; Supplementary table 19 [href=this file] ).\n",
      "Principal component analysis was performed to compare genetic ancestry of these individuals to CEU from 1000 Genomes Project ( Extended Data Figure 5 [href=#F6] ; Supplementary Table 20 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 148659\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 37917 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-6007952a18b574a7f67ab8ee6cf145200644b04536fe00b73a30f254b99a42fa\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 37917 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9892b3e6-89c6-4ab6-872d-25a29c073d5f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9884314\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9884314\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23191787\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23191787\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20803583\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn20803583\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9884314\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9884314\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23191787\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23191787\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20803583\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn20803583\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322278,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51162,\n",
      "        \"completion_tokens\": 226,\n",
      "        \"total_tokens\": 51388,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17015098', 'dataset_id': 'syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn17015098, synapse.org, https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn9884314', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9884314', 'dataset_id': 'syn9884314'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn9884314, synapse.org, https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn9884314', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn23191787', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23191787', 'dataset_id': 'syn23191787'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn23191787, synapse.org, https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20803583', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20803583', 'dataset_id': 'syn20803583'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20803583, synapse.org, https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20803583', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9884314', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20803583', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn9884314 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn23191787 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20803583 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 53th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11224609&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b10ae740>, <Element supplementary-material at 0x1b19d6900>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b10ae740>, <Element supplementary-material at 0x1b19d6900>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup1: To illustrate the overall workflow, we present a toy example showing a simple schematic view and the order of major computation steps in CWAS-Plus ( Supplementary Fig. 1 [this file] ). \n",
      "Additionally, we examined the memory usage of CWAS-Plus with an increasing number of samples and found an approximately linear increase ( Supplementary Fig. 2 [this file] ). \n",
      "Our observations revealed that variants from most clusters exhibited a higher percentage of variants in higher score bins than non-RNVs ( Supplementary Fig. 3 [this file] ), indicating that CWAS-Plus prioritized more pathogenic variants within the broader pool of noncoding variants. \n",
      "CWAS-Plus demonstrated significant case enrichment (OR = 1.3; = 5.7 10 , 95% CI: 1.1–1.6), while Sei, Enformer, and STAARpipeline showed no significant case enrichment ( Supplementary Fig. 4 [this file] , Supplementary Table 6 [rid=sup2] ). \n",
      "CWAS analyses for variants showed highly consistent estimates between the variant-level and sample-level tests ( Supplementary Fig. 5 [this file] , Supplementary Table 7 [rid=sup2] ). \n",
      "Our extended analysis comparing the ability to identify risk variants revealed that only CWAS-Plus demonstrated significant case enrichment in risk variant carriers, whereas Sei, Enformer, and STAARpipeline did not ( Supplementary Fig. 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup2: CWAS-Plus was 50 times faster than CWAS, highlighting its superior computational speed ( Fig. 2A [rid=f2] , Supplementary Table 1 [this file] ). \n",
      "The scalability of the CWAS-Plus package was tested for run with an increasing number of samples ( Fig. 2B [rid=f2] , Supplementary Table 2 [this file] ) and datasets ( Fig. 2C [rid=f2] , Supplementary Table 3 [this file] ). \n",
      "The results showed that noncoding, including intergenic ( = 0.20%, = 1.3 10 ; Supplementary Tables 4 [this file] and 5 [this file] ) and 5’ UTR ( = 0.15%, = 1.1 10 ), categories were significantly enriched ( Fig. 3B [rid=f3] ). \n",
      "CWAS-Plus demonstrated significant case enrichment (OR = 1.3; = 5.7 10 , 95% CI: 1.1–1.6), while Sei, Enformer, and STAARpipeline showed no significant case enrichment ( Supplementary Fig. 4 [rid=sup1] , Supplementary Table 6 [this file] ). \n",
      "CWAS analyses for variants showed highly consistent estimates between the variant-level and sample-level tests ( Supplementary Fig. 5 [rid=sup1] , Supplementary Table 7 [this file] ). \n",
      "Since no single category fully explained AD risk associated with rare noncoding variants, we conducted a risk score analysis to identify predictors among diverse categories ( Fig. 4A [rid=f4] , Supplementary Tables 8 [this file] and 9 [this file] ). \n",
      "Set 3 showed the highest performance, outperforming set 1, followed by set 2 ( = 0.08%, = 5.5 10 for set 1; = 0.17%, = 4.3 10 for set 2; = 0.33%, = 3.7 10 for set 3; Fig. 5B [rid=f5] , Supplementary Table 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 174591\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 44400 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4c532778e0c9e7226751b65fdf151a39490fafd3004a8cd517a9042cca160ad9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 44400 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ae59a575-fa53-4bf7-af30-9dab0c10125e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn10901595\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn10901595\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.10795678\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/joonan-lab/cwas\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.11240547\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://base.sfari.org\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn10901595\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn10901595\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.10795678\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/joonan-lab/cwas\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.11240547\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://base.sfari.org\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322280,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51662,\n",
      "        \"completion_tokens\": 230,\n",
      "        \"total_tokens\": 51892,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn10901595', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn10901595', 'dataset_id': 'syn10901595'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn10901595, synapse.org, https://www.synapse.org/#!Synapse:syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn10901595', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.10795678', 'repository_reference': 'doi.org', 'dataset_id': '10.5281/zenodo.10795678'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/zenodo.10795678, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.10795678\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.10795678', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/joonan-lab/cwas', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/joonan-lab/cwas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/joonan-lab/cwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/joonan-lab/cwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/joonan-lab/cwas, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/joonan-lab/cwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/joonan-lab/cwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/joonan-lab/cwas is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/joonan-lab/cwas', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/joonan-lab/cwas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.11240547', 'repository_reference': 'doi.org', 'dataset_id': '10.5281/zenodo.11240547'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/zenodo.11240547, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.11240547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.11240547', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://base.sfari.org', 'repository_reference': 'n/a', 'dataset_id': 'https://base.sfari.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://base.sfari.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://base.sfari.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://base.sfari.org, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://base.sfari.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://base.sfari.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: base.sfari.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://base.sfari.org is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://base.sfari.org', 'repository_reference': 'n/a', 'dataset_id': 'https://base.sfari.org'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn10901595', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.10795678', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.11240547', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn10901595 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn10901595 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn10901595 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn10901595 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn10901595 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn10901595\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/zenodo.10795678\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.10795678\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.10795678 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/zenodo.10795678 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.5281/zenodo.10795678\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/zenodo.10795678\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/zenodo.11240547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.11240547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.11240547 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/zenodo.11240547 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.5281/zenodo.11240547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/zenodo.11240547\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 54th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7577712&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6492d50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 10 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec34'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec34: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 33) for ack1: Acknowledgments [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#supplementary-material4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for supplementary-material4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#app2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for app2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 239594\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 60651 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-5893c7caad0e5a5a1be01460a3d831074b335269d701c083eee96edd3443c600\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 60651 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5b4e1559-513c-4089-a45e-57df7948ae6a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33000\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD009589\\\",\\n    \\\"repository_reference\\\": \\\"proteomecentral.proteomexchange.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20821165\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33000\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD009589\\\",\\n    \\\"repository_reference\\\": \\\"proteomecentral.proteomexchange.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20821165\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322295,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 79798,\n",
      "        \"completion_tokens\": 118,\n",
      "        \"total_tokens\": 79916,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE33000', 'repository_reference': 'geo', 'dataset_id': 'GSE33000'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE33000, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE33000\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33000', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PXD009589', 'repository_reference': 'proteomecentral.proteomexchange.org', 'dataset_id': 'PXD009589'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PXD009589 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: proteomecentral.proteomexchange.org, identifier: PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PXD009589', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20821165', 'repository_reference': 'synapse.org', 'dataset_id': 'syn20821165'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20821165, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20821165\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20821165', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE33000', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PXD009589', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20821165', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE33000\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE33000\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33000 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33000 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE33000 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE33000\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33000\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: proteomecentral.proteomexchange.org, identifier: PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PXD009589 in proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: proteomecentral.proteomexchange.org and accession_id: PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD009589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20821165\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20821165\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20821165 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20821165 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20821165 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20821165 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20821165 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn20821165\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20821165\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 55th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7257173&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1019800>, <Element supplementary-material at 0x1b6402940>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1019800>, <Element supplementary-material at 0x1b6402940>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: CSF samples from 88 individuals in the replication cohort included 20 healthy controls, 37 patients with mild, symptomatic AD (either prodromal AD with mild cognitive impairment or early stage AD dementia with positive AD biomarkers), and 31 with other (non-AD) neurological disease (Additional file 1 [this file] : Tables S1 and S2). \n",
      "Samples were thawed at room temperature and protein concentrations measured by the bicinchoninic acid assay (BCA) method (Additional file 1 [this file] : Table S2). \n",
      "The mass spectrometer was set to collect in PRM mode with an inclusion list consisting of each peptide (Additional file 1 [this file] : Table S3). \n",
      "The summed product ions peak areas were considered as raw peptide peak area (Additional file 1 [this file] : Table S3), which was normalized by external reference heavy peptide standards to correct for MS signal and sample loading variance (Additional file 1 [this file] : Table S4). The sum of all peptides belonging to the same protein was calculated and normalized by the average of global pooled (GIS) samples (Additional file 1 [this file] : Table S5). Protein normalized ratios were then regressed by age and gender, and centered to mean zero (Additional file 1 [this file] : Table S6) prior to statistical analyses. \n",
      "For proteins, standard Student’s t-test, or one-way ANOVA test were used to calculate significance (Additional file 1 [this file] : Table S7). \n",
      "In the discovery TMT dataset, the Bicor rho correlation, and Student’s significance p values, were calculated using the WGCNA R package using the bicor And P value function, comparing normalized protein sum of peptide measurements by MS to selected traits obtained on paired cases by ELISA measurements of total Tau, phospho-Tau (pT181), β-amyloid, and ratio of Tau/Aβ (Additional file 1 [this file] : Table S8). \n",
      "These three logistic regression classifiers for each protein were trained using a five-fold cross validation procedure and the performance of these classifiers was assessed on the testing set using the AUC of the ROC curve (Additional file 1 [this file] : Table S9). \n",
      "The latter group were cognitively impaired, but negative for Aβ and Tau biomarker status (Additional file 1 [this file] : Table S1 and S2). Nearly all of the 41 proteins were significantly correlated with Aβ, Tau or phosphorylated Tau (pTau) ELISA levels in the discovery cohort (Fig. 1 [rid=Fig1] b and Additional file 1 [this file] : Table S8). \n",
      "All protein correlations and p-values are provided in Additional file 1 [this file] : Table S8 \n",
      "All targeted peptides and CVs following batch normalization are provided in Additional file 1 [this file] : Table S4. Peptide measurements were summed for proteins with more than one peptide quantified to generate a single protein measurement (Additional file 1 [this file] : Table S5). \n",
      "S3 and Additional file 1 [this file] : Table S7). \n",
      "To assess the degree to which any of these CSF proteins could predict clinical diagnosis, a logistic regression classifier was employed (Fig. 4 [rid=Fig4] and Additional file 1 [this file] : Table S9). \n",
      "All protein AUCs are provided in Additional file 1 [this file] : Table S9. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Prior to mass spectrometry analyses, the 88 CSF sample digests were randomized and divided into 4 batches (Additional file 2 [this file] : Fig. We selected 94 peptides across each of the 41 targets proteins based on empirically generated proteomics data from control and AD CSF patients [ 6 [rid=CR6] ] (Additional file 2 [this file] : Fig. Finally, to assess the accuracy of the label-free PRM measurements heavy labeled isotopic standards (AQUA™ Pro, ThermoFisher) were included for a subset of peptides and the light/heavy ratios compared to the normalized peak area (Additional file 2 [this file] : Fig. \n",
      "The technical coefficient of variation (CV) of each peptide was calculated based on a pooled standard of all samples ( = 10 replicate injections) following batch normalization (Additional file 2 [this file] : Fig. After summing the values for a protein measurement, the average CV for each protein was ~ 25% (Additional file 2 [this file] : Fig. \n",
      "This included Tau (MAPT) and neurofilament light and medium chain proteins (NEFL and NEFM) (Fig. 3 [rid=Fig3] b and Additional file 2 [this file] : Figure S3). \n",
      "Notably, 26 of the 41 CSF markers quantified in this study, mapped to one of these five biomarker panels [ 6 [rid=CR6] ] (Fig. 3 [rid=Fig3] b, Additional file 2 [this file] : Fig. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 218967\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 55494 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-84ec7fdc615b5753ef671d1f9ac887000ab58d9139455fa285bd604a272f1c47\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 55494 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b72bea45-41d4-46a1-9152-229bb8254fb9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21541022\\\",\\n    \\\"repository_reference\\\": \\\"www.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21541022\\\",\\n    \\\"repository_reference\\\": \\\"www.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322297,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 60337,\n",
      "        \"completion_tokens\": 47,\n",
      "        \"total_tokens\": 60384,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21541022', 'repository_reference': 'www.synapse.org', 'dataset_id': 'syn21541022'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21541022, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21541022\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21541022', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn21541022', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21541022\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21541022\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21541022 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21541022 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21541022 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21541022 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21541022 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn21541022\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21541022\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 56th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9780309&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0bace90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#Sec1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 38) for Sec1: Background & Summary [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#Sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 25) for Sec2: Methods [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Sec6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Sec6: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 167367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 42594 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-89566aaa06f13002d40c6728a9aafb12290490e011438bf2458adcd60e652148\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 42594 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f16ad55d-f5d3-410a-b527-0f74317b2758\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26560886\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26560886\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322314,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 49236,\n",
      "        \"completion_tokens\": 64,\n",
      "        \"total_tokens\": 49300,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn26560886', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053', 'dataset_id': 'syn26560886'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn26560886, synapse.org, https://www.synapse.org/[#!]*Synapse:syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn26560886\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn26560886', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn26560886', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn26560886\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26560886\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26560886 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26560886 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26560886 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26560886 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn26560886 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn26560886\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26560886\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 57th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7658217&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1476680>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1476680>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: Figure 1 [this file] , Suppl. Figure 2 [this file] A,B, Suppl. Table 1 [this file] ). Figure 2 [this file] B). Figure 3 [this file] ). Figure 2 [this file] C) and on cluster marker gene expression (Suppl. Figure 4 [this file] ). Figure 2 [this file] D). Figures 4 [this file] , 5 [this file] and 6 [this file] ), albeit the exploration of this atlas exceeds the scope of this manuscript. \n",
      "Figure 7 [this file] A) as the starting cluster. Figure 7 [this file] B). Figure 7 [this file] A,C), similarly to ciliated and alveolar type 1 transit-amplifying progenitors (Fig. 7 [rid=Fig7] E,F). Figure 7 [this file] D). \n",
      "Figure 7 [this file] E) while expression of was mostly restricted to mucous cells (Suppl. Figure 7 [this file] F). \n",
      "Supplementary Table 1 [this file] summarizes the thresholds used by each sample in each individual study, and Supplementary Fig. 1 [this file] summarizes quality-control metrics for each dataset. \n",
      "Parameters used for dbMAP embedding for individual studies and the integrated atlas, as well as those used for UMAP embedding of the atlas, are listed in Supplementary Table 3 [this file] . \n",
      "Table 1 [this file] ), followed by clustering as described for the complete atlas with parameters. \n",
      "Table 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 155354\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 39591 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b3b18b618e4277f12bca8b3efca2ef5516da4766e5fb6a1da9115fb203de5e04\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 39591 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bc3e1a83-a9c8-4382-bd5f-0d42b1feb347\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE132355\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.1101/742320\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/humanlung\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/dbMAP\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://humanlung.iqm.unicamp.br\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/lung_covid\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE132355\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.1101/742320\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/humanlung\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/dbMAP\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://humanlung.iqm.unicamp.br\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/davisidarta/lung_covid\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322316,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 45009,\n",
      "        \"completion_tokens\": 450,\n",
      "        \"total_tokens\": 45459,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122960', 'repository_reference': 'geo', 'dataset_id': 'GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE122960, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE132355', 'repository_reference': 'geo', 'dataset_id': 'GSE132355'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE132355, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE132355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE132355', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'repository_reference': 'n/a', 'dataset_id': 'https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: data.humancellatlas.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'repository_reference': 'n/a', 'dataset_id': 'https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.1101/742320', 'repository_reference': 'doi.org', 'dataset_id': 'https://doi.org/10.1101/742320'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1101/, None, https://doi.org/10.1101/742320\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1101/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'repository_reference': 'synapse.org', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21041850, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/davisidarta/humanlung', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/humanlung'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/davisidarta/humanlung\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/davisidarta/humanlung\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/davisidarta/humanlung, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/davisidarta/humanlung\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/davisidarta/humanlung\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/davisidarta/humanlung is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/davisidarta/humanlung', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/humanlung'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/davisidarta/dbMAP', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/dbMAP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/davisidarta/dbMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/davisidarta/dbMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/davisidarta/dbMAP, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/davisidarta/dbMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/davisidarta/dbMAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/davisidarta/dbMAP is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/davisidarta/dbMAP', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/dbMAP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://humanlung.iqm.unicamp.br', 'repository_reference': 'n/a', 'dataset_id': 'https://humanlung.iqm.unicamp.br'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://humanlung.iqm.unicamp.br\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://humanlung.iqm.unicamp.br\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://humanlung.iqm.unicamp.br, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://humanlung.iqm.unicamp.br\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://humanlung.iqm.unicamp.br\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: humanlung.iqm.unicamp.br\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://humanlung.iqm.unicamp.br is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://humanlung.iqm.unicamp.br', 'repository_reference': 'n/a', 'dataset_id': 'https://humanlung.iqm.unicamp.br'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf', 'repository_reference': 'n/a', 'dataset_id': 'https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: version-11-0.string-db.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf', 'repository_reference': 'n/a', 'dataset_id': 'https://version-11-0.string-db.org/cgi/network.pl?taskId=KwL0Kho7pBZf'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/davisidarta/lung_covid', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/lung_covid'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/davisidarta/lung_covid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/davisidarta/lung_covid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/davisidarta/lung_covid, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/davisidarta/lung_covid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/davisidarta/lung_covid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/davisidarta/lung_covid is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/davisidarta/lung_covid', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/davisidarta/lung_covid'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE132355', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1101/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE122960 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE132355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE132355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE132355 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE132355 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE132355 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE132355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1101/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 58th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6289083&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b15d8f10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 30 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec4.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec4.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec4.4.7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec4.4.7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 14 <a> elements with href='#mmc1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 3704) for mmc1: To test whether a similar approach could be applied to hematopoietic stem cells, we used multiparameter flow cytometry to sort phenotypically defined long-term HSCs and MPPs obtained from human bone marrow biopsies ( Notta et al., 2016 [href=#bib28] ) and then clonally expanded them to obtain sufficient DNA for WGS analysis ( Figures 1 [href=#fig1] A and S1 [href=this file] ). The majority of the somatic mutations in these cultures displayed a variant allele frequency (VAF) clustered around 0.5, which indicates that these mutations were shared by all cells in the culture and therefore present in the originally expanded stem cell ( Figure S2 [href=this file] ). These accumulated mutations are discarded based on the low VAF ( Figure S2 [href=this file] ). We performed WGS on DNA from 18 HSCs/MPPs derived from adult marrow biopsies of 5 healthy donors, ranging from 26 to 63 years of age ( Table S1 [href=this file] ). Independent validations using single-molecule molecular inversion probes (smMIPs) of a subset of the identified somatic mutations revealed an overall confirmation rate of approximately 91% ( Tables S2 [href=this file] and S4 [href=#mmc3] B).\n",
      "Nonetheless, cells of the same donor shared only a limited number of mutations (60 out of 11,082 base substitutions; Table S1 [href=this file] ), indicating that mutagenesis did occur independently during the lifetime of each assessed cell.\n",
      "The strongest factor in our PCA (PC1) separates cells with either high or low contribution of signatures 1 or 5 ( Figure S3 [href=this file] ), which are signatures that were previously defined in a pan-cancer analysis ( Alexandrov et al., 2013b [href=#bib3] ) and reported to act in a “clock-like” manner.\n",
      "Directions of signature contributions between samples are indicated on the x and y axis; see Figure S3 [href=this file] .\n",
      "The presence of a third pattern, reflecting a recently defined signature 32 ( Inman et al., 2018 [href=#bib16] ), separates the mutation spectra in HSPCs from stem cells of the other human tissues (PC2; Figures 3 [href=#fig3] A and S3 [href=this file] ). Although the etiology of signature 32 is unknown, base substitutions specific for this signature were associated with a transcriptional strand bias in the HSPCs ( Figures 3 [href=#fig3] D and S4 [href=this file] ), consistent with activity of transcription-coupled repair ( Pleasance et al., 2010 [href=#bib32] ).\n",
      "Remarkably, the absolute contribution of all three signatures to the base substitution load in normal HSPCs was similar as those in a previously reported acute myeloid leukemia (AML) dataset ( Welch et al., 2012 [href=#bib36] ; Figure 3 [href=#fig3] E; p = 0.97; chi-square test; Figure S4 [href=this file] B and S4C), indicating that the genesis of this malignancy does not necessarily require enhanced mutagenesis.\n",
      "Sample B (26-year-old donor, male, white) was extracted at Boston Children’s Hospital, whereas samples A (33-year-old, male, white), C (55-year-old, female, hispanic), D (63-year-old, male, white), and E (41-year old, female, African American) were obtained from AllCells ( Table S1 [href=this file] ). 10-20 mL of whole bone marrow aspirate were drawn into a 60cc syringe containing heparin (80 U/mL of BM) from a unique puncture in the posterior iliac crest.\n",
      "Representative examples of sorted populations are shown in the Figure S1 [href=this file] B.\n",
      "When 70% or more of the reads with the same UMI had the same nucleotide, then the UMI was counted as valid as described( Yu et al., 2018 [href=#bib40] ) Mutation positions with less than 20 valid UMI’s were filtered out, and the rest mutations with higher than 0 VAF were counted as validated ( Table S2 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#mmc2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 675) for mmc2: We did not observe non-synonymous or truncating mutations in cancer driver genes for hematological neoplasms ( Ju et al., 2017 [href=#bib19] ), excluding selective clonal outgrowth of cells in culture ( Table S3 [href=this file] ).\n",
      "Nevertheless, comparing somatic mutation load in 72 known driver genes for hematological neoplasms ( Ju et al., 2017 [href=#bib19] ) revealed that the normal blood progenitors are depleted for potential cancer driver mutations (0 mutations in 72 genes out of 22 genomes; Table S3 [href=this file] ) compared to the AML samples (28 mutations in 72 genes out of 24 genomes; p < 0.05; one-sided binomial test; Welch et al., 2012 [href=#bib36] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#mmc3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 844) for mmc3: Independent validations using single-molecule molecular inversion probes (smMIPs) of a subset of the identified somatic mutations revealed an overall confirmation rate of approximately 91% ( Tables S2 [href=#mmc1] and S4 [href=this file] B).\n",
      "To evaluate the ability of our reconstructed map to explain lineage relationships in the bone marrow, we genotyped each base substitution defining a branch in 125 additional HSCs and MPP clones from the same donor ( Table S4 [href=this file] A).\n",
      "Clone-specific smMIPs were designed as described( Hiatt et al., 2013 [href=#bib15] , Yu et al., 2018 [href=#bib40] ), ( Table S4 [href=this file] B).\n",
      "For all of these early embryonic base substitutions primers were designed for amplicon-base re-sequencing in the mature populations and HSC and MPP clones as described above ( Table S4 [href=this file] A).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#mmc4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for mmc4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 156376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 39846 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f18050a5fbf13e8c02f07cda0703492221e1bb51a72d3b541b6077580d18cee4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 39846 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a6fe6011-1ed6-4095-a922-fc35b2c54ae7\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001003068\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/home\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001003068\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/home\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322334,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 53606,\n",
      "        \"completion_tokens\": 59,\n",
      "        \"total_tokens\": 53665,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001003068', 'repository_reference': 'https://www.ebi.ac.uk/ega/home', 'dataset_id': 'EGAS00001003068'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EGAS00001003068 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ega/home, identifier: EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001003068 for https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00001003068 for https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00001003068 for https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001003068 for https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001003068 for https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ega/home\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001003068', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'EGAS00001003068', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EGAS00001003068 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: www.ebi.ac.uk and accession_id: EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAS00001003068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 31, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 59th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8807579&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1c9bb50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 16 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 243896\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 61726 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-68f3994239f254a509f144a6863016a0d43c8b58f92a6d14a32cccf9cc244719\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 61726 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-81b8b4ae-629d-4a8e-bfac-52986837ac78\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE164466\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164466\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE149149\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149149\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB24858\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE158233\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158233\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE156167\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156167\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE151999\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151999\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE36980\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36980\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20808171\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE164466\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164466\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE149149\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149149\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJEB24858\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE158233\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158233\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE156167\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156167\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE151999\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151999\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE36980\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36980\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20808171\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322358,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 91620,\n",
      "        \"completion_tokens\": 481,\n",
      "        \"total_tokens\": 92101,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE164466', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164466', 'dataset_id': 'GSE164466'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE164466, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE164466', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE149149', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149149', 'dataset_id': 'GSE149149'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE149149, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE149149', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB24858', 'repository_reference': 'https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads', 'dataset_id': 'PRJEB24858'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PRJEB24858 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads, identifier: PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB24858 for https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB24858 for https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB24858 for https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB24858 for https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB24858 for https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/browser/view/PRJEB24858?show=reads\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB24858', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE158233', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158233', 'dataset_id': 'GSE158233'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE158233, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE158233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE156167', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156167', 'dataset_id': 'GSE156167'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE156167, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE156167', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE151999', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151999', 'dataset_id': 'GSE151999'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE151999, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE151999', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE36980', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36980', 'dataset_id': 'GSE36980'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE36980, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE36980', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20808171', 'repository_reference': 'https://adknowledgeportal.synapse.org/', 'dataset_id': 'syn20808171'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20808171, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20808171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20808171', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE164466', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE149149', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB24858', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE158233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE156167', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE151999', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE36980', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20808171', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE164466 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE164466 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE164466 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164466\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE149149 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE149149 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE149149 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PRJEB24858 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJEB24858\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE158233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE158233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE158233 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE156167 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE156167 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE156167 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE151999 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE151999 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE151999 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151999\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE36980 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE36980 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE36980 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36980\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20808171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20808171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20808171 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20808171 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20808171 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20808171 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20808171 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn20808171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20808171\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 24, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 10\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 60th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6700281&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0c75700>, <Element supplementary-material at 0x1b0c76640>, <Element supplementary-material at 0x1b0c76e40>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0c75700>, <Element supplementary-material at 0x1b0c76640>, <Element supplementary-material at 0x1b0c76e40>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: See also Figures S1 [rid=figs1] , S2 [rid=figs2] , and S3 [rid=figs3] and Tables S2 [rid=mmc2] and S3–S5 [this file] . \n",
      "See also Figure S2 [rid=figs2] and Table S1 [this file] . \n",
      "The vast majority (> 95%) of European ancestry HD chrs carry a canonical CAG repeat region with an uninterrupted CAG repeat followed by CAACAG ( Figure 2 [rid=fig2] B; Table S1 [this file] ). \n",
      "On CAA-loss chrs, the CAG repeat is followed by CCG CCT CAGCTTCCT and on CAACAG-duplication chrs, it is followed by CAA CAG CAA CAG CCG CCA CCG CCT CAGCTTCCT Sequencing of samples not tagged by rs764154313 or rs183415333 confirmed correct PCR-assay genotyping of the uninterrupted CAG sequence on canonical chrs and showed that the vast majority carry CAA CAG CCG CCA CCG CCT CAGCTTCCT after the CAG repeat ( Table S1 [this file] ). Indeed, the sequences of a selection of other HD and normal chrs revealed a wide variety of rarer non-canonical sequence variations ( Table S1 [this file] ). \n",
      "See also Figure S3 [rid=figs3] and Tables S2 [rid=mmc2] and S3–S5 [this file] . \n",
      "Pathway enrichment compared to GWA123 again pointed in GWA12345 (and GWA45) to mismatch repair but did not support mitochondrial regulation ( Table S3 [this file] ). In 77 DNA repair gene sets ( Pearl et al., 2015 [rid=bib59] ), the strongest enrichments were related to mismatch repair ( Tables S4 [this file] and S5 [this file] ). In a broader test of 14,210 pathways with 10 or more gene members, 77 pathways were significant after Bonferroni correction in GWA12345 ( Table S6 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc2: See also Figures S1 [rid=figs1] , S2 [rid=figs2] , and S3 [rid=figs3] and Tables S2 [this file] and S3–S5 [rid=mmc1] . \n",
      "See also Figure S3 [rid=figs3] and Tables S2 [this file] and S3–S5 [rid=mmc1] . \n",
      "Finally, we performed gene-wide association analysis ( Table S2 [this file] ), finding that all significant genes came from regions in Table 1 [rid=tbl1] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 308929\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 77984 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-627c04317ef4eb3cb0cfb363825fe846e97ea854127efc38ff682071460501f6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 77984 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9144d266-19ab-4a34-8bef-0d76129d7ecc\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs000424.v7.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn2759792/wiki/69613\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8\\\",\\n    \\\"repository_reference\\\": \\\"data.mendeley.com\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs000424.v7.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn2759792/wiki/69613\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8\\\",\\n    \\\"repository_reference\\\": \\\"data.mendeley.com\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322361,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 89540,\n",
      "        \"completion_tokens\": 165,\n",
      "        \"total_tokens\": 89705,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'phs000424.v7.p1', 'repository_reference': 'dbgap', 'dataset_id': 'phs000424.v7.p1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: phs000424.v7.p1, dbgap, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: phs000424.v7.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'phs000424.v7.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/#!Synapse:syn2759792/wiki/69613', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/#!Synapse:syn2759792/wiki/69613'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2759792, None, https://www.synapse.org/#!Synapse:syn2759792/wiki/69613\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8', 'repository_reference': 'data.mendeley.com', 'dataset_id': 'https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5061/, None, https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: data.mendeley.com, identifier: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5061/ for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.5061/ for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for data.mendeley.com in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5061/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'phs000424.v7.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5061/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: phs000424.v7.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: dbgap, identifier: phs000424.v7.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs000424.v7.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs000424.v7.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs000424.v7.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for phs000424.v7.p1 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: dbgap and accession_id: phs000424.v7.p1.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v7.p1.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2759792 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5061/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5061/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 61/100 (61.00%) | Elapsed: 00:09:49 | ETA: 00:06:17\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 61th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7029360&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6ce7c10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 121372\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31095 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cf2e39145e7f4ec8866cf6cff9793434c9a0c4e8d6169f93ff574af22b603c60\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31095 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-45515bbb-43e0-4f5c-9733-c27493b8f1f1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC7029360\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"31561387\\\",\\n    \\\"repository_reference\\\": \\\"https://pubmed.ncbi.nlm.nih.gov/31561387/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8717496\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8717496\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC7029360\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"31561387\\\",\\n    \\\"repository_reference\\\": \\\"https://pubmed.ncbi.nlm.nih.gov/31561387/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8717496\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8717496\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322382,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 41545,\n",
      "        \"completion_tokens\": 148,\n",
      "        \"total_tokens\": 41693,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PMC7029360', 'repository_reference': 'n/a', 'dataset_id': 'PMC7029360'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PMC7029360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PMC7029360 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: PMC7029360\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC7029360 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'PMC7029360', 'repository_reference': 'n/a', 'dataset_id': 'PMC7029360'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '31561387', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/31561387/', 'dataset_id': '31561387'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID 31561387 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://pubmed.ncbi.nlm.nih.gov/31561387/, identifier: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 31561387 for https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://pubmed.ncbi.nlm.nih.gov/31561387/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '31561387', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn8717496', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8717496', 'dataset_id': 'syn8717496'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn8717496, synapse.org, https://www.synapse.org/#!Synapse:syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn8717496', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '31561387', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn8717496', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: pubmed.ncbi.nlm.nih.gov, identifier: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 31561387 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL pubmed.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 31561387 in pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: pubmed.ncbi.nlm.nih.gov and accession_id: 31561387\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository pubmed.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8717496 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn8717496 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8717496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 62th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8219045&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6c59640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6c59640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup1: Details on the identified markers on cerebellum and pancreas are reported in Supplementary Tables S1–S2 [this file] , respectively. \n",
      "We also noticed that a recently published study ( 23 [rid=B23] ) that re-analyzed the same data has come to a similar conclusion with correlation coefficient = ∼-0.31 and -value = 0.00126 (see Supplementary Figure S1 [this file] ). \n",
      "We observed strong similarities between the average expressions of the inferred profiles between the same cell-types (Figure 4B [rid=F4] and Supplementary Figure S2 [this file] ). \n",
      "We ran CellR and looked for DE genes within each cell-type ( Supplementary Table S3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 191011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 48505 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-aa271f2b811e8822c8970d45ca30006a0fe02e6f747a8ab1d411577a4623268d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 48505 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d7890d17-e8fe-4742-a831-077e7fb4f589\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE64810\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE97942\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5061\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3163039\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE64810\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE97942\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-5061\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3163039\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322384,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 57515,\n",
      "        \"completion_tokens\": 143,\n",
      "        \"total_tokens\": 57658,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE64810', 'repository_reference': 'geo', 'dataset_id': 'GSE64810'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE64810, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE64810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE64810', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE97942', 'repository_reference': 'geo', 'dataset_id': 'GSE97942'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE97942, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE97942\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE97942', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5061', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-5061'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-5061 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3163039', 'repository_reference': 'synapse.org', 'dataset_id': 'syn3163039'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn3163039, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn3163039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn3163039', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE64810', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE97942', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3163039', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE64810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE64810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE64810 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE64810 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE64810 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE64810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE64810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE97942\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE97942\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE97942 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE97942 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE97942 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE97942\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97942\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-5061 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn3163039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3163039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3163039 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3163039 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3163039 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3163039 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn3163039 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn3163039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3163039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 63th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10543041&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b79e6f90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 16 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 7 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 2063) for SD1: Briefly, samples from the full cohort were genotyped across multiple Illumina microarray platforms (see Supplemental Methods [href=this file] ) and underwent a standard quality control (QC) protocol to remove low-quality and low-frequency variants.\n",
      "Samples were further excluded based on RNA-seq quality metrics (see Supplemental Methods [href=this file] ), low RIN score, discrepancies between self-reported sex and chromosome Y gene expression, and missing genotype data.\n",
      "Four different meQTL mapping regression models were fit: (1) a “baseline” model to test for association between genetic variants and DNAm β-values across both smoking cases and controls; this model included age at death, sex, estimated non-neuronal cell-type proportion, PC1 for DNAm array negative control probes, PC1 for imputed genotypes, and PC2 for imputed genotypes as covariates (see Supplemental Methods [href=this file] for model selection details); (2) a smoking cases–only model similar to the baseline model; (3) a smoking controls–only model similar to the baseline model; and (4) an interaction model to test for associations of a genetic variant-by-smoking status interaction with DNAm β-values. Additional details of each model are provided in the Supplemental Methods [href=this file] .\n",
      "The covariates included in each model were age at death, sex, exon mapping rate, ribosomal RNA mapping rate, PC1 for imputed genotypes, PC2 for imputed genotypes, and four latent variables estimated by PEER v1.3 to account for additional unmeasured sources of confounding (see Supplemental Methods [href=this file] for model selection details).\n",
      "For a given probe, the colocalization test region spanned all genetic variants that were 1) included in the meQTL mapping for the probe and 2) tested in the GSCAN GWAS ( Figure S1 [href=this file] ).\n",
      "Of the significant 41,695 unique CpGs, only five demonstrated strong evidence of an interaction with smoking ( Table 2 [href=#T2] , Figure S2 [href=this file] ) based on Bonferroni correction (significance threshold of 0.05/41,695=1.2e-6).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 244) for SD2: In general, we observed more colocalization of meQTLs than eQTLs with the GWAS loci ( Table 3 [href=#T3] , Tables S8 [href=this file] – S9 [href=#SD3] ).\n",
      "Full colocalization results are provided in Tables S8 [href=this file] – S9 [href=#SD3] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 244) for SD3: In general, we observed more colocalization of meQTLs than eQTLs with the GWAS loci ( Table 3 [href=#T3] , Tables S8 [href=#SD2] – S9 [href=this file] ).\n",
      "Full colocalization results are provided in Tables S8 [href=#SD2] – S9 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 4 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 363) for SD4: These results are shown in Supplemental Table S1 [href=this file] , which is restricted to the lead variant for each unique CpG because of the size of the meQTL analysis ( Table S1 [href=this file] , Column: P ).\n",
      "The top five most significant meQTLs include CpGs that map to , or ( Table S1 [href=this file] ).\n",
      "See Table S1 [href=this file] for full annotations.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 337) for SD5: Of these, 57,683 eQTL variants targeting 958 unique eGenes remained after post hoc filtering was applied to identify the most robust signals, keeping only eQTLs where the top variant had an MAF ≥0.05 and missingness ≤0.10 in both ancestries ( Table S2 [href=this file] ). All significant results are shown in Table S2 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#SD6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 308) for SD6: Table S3 [href=this file] is filtered to the lead variant for each significant eGene (N=1,050). The 10 most significant eGenes were , and ( Table S3 [href=this file] ).\n",
      "Of these 655, 509 (78%) met a Bonferroni correction significance threshold (0.05/655=7.6e-5) ( Table S3 [href=this file] , GTEx column: ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SD7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 193) for SD7: For eQTL-smoking interactions, cell cycle and wound response processes (cellular changes resulting from a stimulus indicated damage to an organism) were enriched ( Table S4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#SD8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 559) for SD8: Annotating the GWAS variants to our QTL maps revealed that 63% of the GWAS-identified variants were significant meQTLs, whereas only 7% were significant eQTLs ( Tables S5 [href=this file] – S7 [href=#SD10] ). Only a few (eQTL N=11, meQTL N=293) were significant at the genome-wide level ( Tables S5 [href=this file] – S6 [href=#SD9] , column ).\n",
      "The GWAS-identified variants mapped to 1,201 significant variant–CpG pairs from the meQTL 2DF model, of which 135 showed nominal evidence for variant-by-smoking interaction (P <0.05) ( Table S5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 345) for SD9: Only a few (eQTL N=11, meQTL N=293) were significant at the genome-wide level ( Tables S5 [href=#SD8] – S6 [href=this file] , column ).\n",
      "Additionally, the GWAS-identified variants mapped to 27 significant variant–eGene pairs from the eQTL 2DF model, of which three showed nominal evidence for interaction (P <0.05) ( Table S6 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SD10'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 208) for SD10: Annotating the GWAS variants to our QTL maps revealed that 63% of the GWAS-identified variants were significant meQTLs, whereas only 7% were significant eQTLs ( Tables S5 [href=#SD8] – S7 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 179060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 45517 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-9fe2f4ba3ab2fbc7b0d50c474063efb482b86feaf9f04b0266997f75659a0c59\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 45517 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7b9605d6-7cb3-4c9e-9ff7-780c6c65f031\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50996324\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50996324\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn50996324\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.13020/3b1n-ff32\\\",\\n    \\\"repository_reference\\\": \\\"data.mendeley.com\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50996324\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn50996324\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn50996324\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.13020/3b1n-ff32\\\",\\n    \\\"repository_reference\\\": \\\"data.mendeley.com\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322406,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 62705,\n",
      "        \"completion_tokens\": 158,\n",
      "        \"total_tokens\": 62863,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn50996324', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn50996324'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn50996324, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn50996324\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn50996324', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.13020/3b1n-ff32', 'repository_reference': 'data.mendeley.com', 'dataset_id': 'https://doi.org/10.13020/3b1n-ff32'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.13020/, None, https://doi.org/10.13020/3b1n-ff32\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: data.mendeley.com, identifier: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.13020/ for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.13020/ for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for data.mendeley.com in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.13020/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn50996324', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.13020/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn50996324\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn50996324\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50996324 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50996324 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50996324 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50996324 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn50996324 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn50996324\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn50996324\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.13020/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.13020/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.13020/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 18, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 13\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 64th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6546784&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 9 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6e36cc0>, <Element supplementary-material at 0x1b7376f80>, <Element supplementary-material at 0x1b1637b80>, <Element supplementary-material at 0x1b681d980>, <Element supplementary-material at 0x1b681dc00>, <Element supplementary-material at 0x1b681c0c0>, <Element supplementary-material at 0x1b681cfc0>, <Element supplementary-material at 0x1b681db40>, <Element supplementary-material at 0x1b681ee80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6e36cc0>, <Element supplementary-material at 0x1b7376f80>, <Element supplementary-material at 0x1b1637b80>, <Element supplementary-material at 0x1b681d980>, <Element supplementary-material at 0x1b681dc00>, <Element supplementary-material at 0x1b681c0c0>, <Element supplementary-material at 0x1b681cfc0>, <Element supplementary-material at 0x1b681db40>, <Element supplementary-material at 0x1b681ee80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: Consistent with previous results, HARs were significantly enriched in putative regulatory elements active prenatally (fetal adrenal gland, brain, kidney, lung, and muscle) , the strongest enrichment being observed in fetal brain (Fig. 1a [rid=Fig1] , Supplementary Fig. 1a [this file] ). We observed strong enrichment in predicted enhancer states, as well as H3K27ac, H3K4me1, and H3K4me2 marks, indicating that HARs are enriched in developing brain enhancers, more so than in adult brain (Supplementary Fig. 1b, c [this file] ). Further, HARs were enriched in regulatory elements that were significantly more likely to be accessible in the germinal (neurogenic) zones of the developing cortex , highlighting their potential role in cortical neurogenesis (Supplementary Fig. 1d [this file] ). \n",
      "We were able to assign 638 and 717 HARs to 972 and 1021 genes using contacts defined by three-dimensional chromatin conformation in cortical plate (CP, neocortical laminae containing post-mitotic neurons) and germinal zone (GZ, neocortical laminae consisting primarily of neural progenitors) in the fetal brain, respectively (Methods; Supplementary Fig. 2 [this file] ) . When combined with intragenic HARs, we were able to assign 1028 HARs to 1648 putative target genes (Supplementary Tables 1 [this file] – 2 [this file] , Supplementary Fig. 2 [this file] , subsequently referred to as putative target genes for HARs). \n",
      "The putative target genes of HARs are enriched for genes that regulate pathways involved in human brain development, regionalization, dorsal-ventral patterning, cortical lamination, and proliferation of neuronal progenitors (Fig. 1d [rid=Fig1] , Supplementary Fig. 3 [this file] ), suggesting that multiple aspects of human brain development are subject to human-specific regulation. \n",
      "Although this overlap is significant ( = 2.09 × 10 , OR = 2.5, Fisher’s exact test) and implicates a high confidence core set of target genes (Supplementary Table 1 [this file] ), most target genes are discordant (Fig. 1c [rid=Fig1] ), which may be due to differences between tissue specific gene regulation, or methodologic factors. \n",
      "We found that HARs interacted with 823 and 467 protein-coding genes in ES cells and IMR90 cells, respectively (Supplementary Fig. 4a, b [this file] ). The majority of genes (~60%) interacted with HARs in a cell type-specific manner, again highlighting the cell type specific nature of chromatin interactions, consistent with the notion of tissue specific gene regulation (Supplementary Fig. 4 [this file] ). Notably, many genes known to play major roles in cerebral cortex development and dorsal–ventral/anterior–posterior pattern specification, including , , , , , and , interacted with HARs in the developing cortex (Supplementary Fig. 4c [this file] ), consistent with the model that chromatin contact maps in developing brain will likely provide more biologically relevant targets for human brain evolution than other tissues. \n",
      "Also check Supplementary Fig. 5c [this file] . \n",
      "In contrast, HGEs interact with genes involved in collagen metabolism, TOR signaling, immune function, and lipid storage, and HLEs interact with genes involved in oxygen transport, autophagy, and thymus development (Supplementary Fig. 5a [this file] ). \n",
      "HGE- and HLE-associated genes show a pattern of more gradual upregulation throughout prenatal development, manifesting their highest expression after the peak of neuronal migration, during the period of synaptic formation and gliogenesis (Supplementary Fig. 5b [this file] ). \n",
      "Indeed, HAR-, HGE -, and HLE-associated genes are also under purifying selection when compared with the genome background (Methods; Supplementary Fig. 6 [this file] ). \n",
      "Genes associated with other human evolved elements do not show any deviation of expression -scores, indicating that they do not show stage-specific enrichment in human compared with rhesus macaque (Supplementary Fig. 7 [this file] ). \n",
      "HAR - and HLE-associated genes do not exhibit significant changes in breakpoints in the genes they regulate (Supplementary Fig. 7 [this file] ), indicating that the activity of different classes of human evolved regulatory elements manifest distinct developmental trajectories. \n",
      "Indeed, we observed that HAR- and HGE -associated genes are enriched with LoF-intolerant genes that harbor de novo mutations in ASD (ASD constrained genes) and DD risk genes (Fig. 2f [rid=Fig2] , Supplementary Fig. 5c [this file] ). Relative to HGE , genes putatively regulated by HARs show more enrichment in ASD constrained genes and de novo LOF variation, whereas HGE appear more enriched for constrained DD genes, or genes harboring LOF mutations in DD (Supplementary Fig. 5c [this file] . \n",
      "However, HAR genes appear more likely to be disrupted in ASD, while HGE genes are more substantially enriched in constrained or LOF DD genes although enriched in constrained ASD genes (Supplementary Fig. 5 [this file] ). \n",
      "This leads to a set of 10,000 ORs for each tissue type, which was plotted in Supplementary Fig. 1a [this file] . \n",
      "As some of these regions were overlapping, we ended up having 109,408 randomly selected genomic regions with matched GC content and length as HARs (Supplementary Table 4 [this file] ), which we used to construct a null distribution. \n",
      "Human-specific genes were obtained from Bakken et al. (Table S10 [this file] ), Sousa et al. (Table S5 [this file] ), and Brawand et al. (Table S1 [this file] ). \n",
      "To further confirm that this enrichment is not merely due to their genomic features, we performed a disease enrichment analysis for HAR-associated genes compared with other genes associated with evolutionary conserved elements (3,083,588 evolutionary conserved elements with phastCons score >0.40 (similar to most HARs) were mapped to 16,676 protein-coding genes), and HGE -associated genes over genes associated with fetal brain enhancers (186,304 enhancers reported in Reilly et al. were mapped to total 15,693 protein-coding genes), where we obtained similar results (Supplementary Fig. 5c [this file] ). \n",
      "To experimentally validate Hi-C predicted candidate target genes of HARs, we chose HARs that (1) overlap with H3K27ac marks in fetal brain and (2) interact with developmentally important genes expressed in human neural progenitors (Supplementary Fig. 8 [this file] ), and that are 3) predicted to regulate only one gene. RNA was extracted by miRNeasy Mini Kit (Qiagen) and the expression level of putative target genes ( , , and ) was measured by qPCR (LightCycler 480 SYBR Green I Master, Roche) and normalized to . gRNA and primer sequences for both genomic DNA and qPCR are described in Supplementary Table 3 [this file] . \n",
      "Promoter-based interaction maps for fetal brain and adult brain are available in Supplementary Tables 22 [this file] – 23 [this file] of Won et al. and the PsychENCODE knowledge portal, respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: Codes used to analyze and plot the results are available in the Supplementary Software 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 157437\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 40111 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e96923d38a15927dcfca07e82dde9d352c47e5db3f2a9de43d33fe5f004baf1d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 40111 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8f0192e3-ec14-4362-966d-8a719e8d0425\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE77565\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001190.v1.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4921369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE77565\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001190.v1.p1\\\",\\n    \\\"repository_reference\\\": \\\"dbgap\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4921369\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322409,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 45908,\n",
      "        \"completion_tokens\": 115,\n",
      "        \"total_tokens\": 46023,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE77565', 'repository_reference': 'geo', 'dataset_id': 'GSE77565'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE77565, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE77565\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE77565', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'phs001190.v1.p1', 'repository_reference': 'dbgap', 'dataset_id': 'phs001190.v1.p1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: phs001190.v1.p1, dbgap, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: phs001190.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'phs001190.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4921369', 'repository_reference': 'synapse.org', 'dataset_id': 'syn4921369'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4921369, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE77565', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'phs001190.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE77565\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE77565\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE77565 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE77565 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE77565 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE77565\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77565\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: phs001190.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: dbgap, identifier: phs001190.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs001190.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs001190.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs001190.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for phs001190.v1.p1 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: dbgap and accession_id: phs001190.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001190.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4921369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 65th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10696419&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0e5fb50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 4 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s21'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s21: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 38 <a> elements with href='#SMF1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 4631) for SMF1: A full list of Abs used in all experiments is shown in Supplemental Fig. 4G [href=this file] .\n",
      "In our aged mouse HMPV model, aged mock-infected CD4 T cells had increased expression compared with young CD4 T cells, but there was no difference in CD19 B cells ( Supplemental Fig. 1A [href=this file] ). We also found that upregulation of PD-1 during HMPV infection was specific to CD8 T cells with no difference in PD-1 on CD4 T or CD19 B cells between young and aged HMPV-infected mice ( Supplemental Fig 1B [href=this file] ). PD-1 expression robustly increased with infection in young mice but stayed elevated in aged mice in both mock and HMPV infection ( Fig. 1E [href=#fig01] ). scRNAseq data revealed no difference in (TIM-3), , or (2B4) inhibitory receptor expression on CD8 T cells ( Supplemental Fig. 1C [href=this file] – 1E [href=this file] ), which we confirmed at the protein level in mock-infected mice via flow cytometry ( Supplemental Fig. 1F [href=this file] ).\n",
      "There were no significant differences in PD-L1 or PD-L2 expression between ages in mock- or HMPV-infected groups across numerous innate immune cell types ( Fig. 1I [href=#fig01] and Supplemental Fig. 1G [href=this file] ), with the exception of a small increase in PD-L2 expression in aged mock interstitial macrophages ( Supplemental Fig. 1G [href=this file] ). We also found no difference between ages in (PD-L1) ( Fig. 1J [href=#fig01] ) or in (PD-L2) ( Supplemental Fig. 1H [href=this file] ) in the lung myeloid cell scRNAseq dataset ( 8 [href=#r8] ).\n",
      "PD-1 blockade had no significant impact on the production of HMPV-specific lung CD8 T cells, based on tetramer staining with the HMPV viral epitope H2-D/K N (N11) ( Fig. 2D [href=#fig02] , 2E [href=#fig02] ), and also did not impact the T CD8 T cells that expressed TOX and EOMES ( Fig. 2F [href=#fig02] , 2G [href=#fig02] ; Supplemental Fig. 1I [href=this file] – 1K [href=this file] ).\n",
      "We confirmed PD-1 expression was absent on CD8 T cells ( Supplemental Fig. 2A [href=this file] ). There was no significant difference in weight loss ( Supplemental Fig. 2B [href=this file] ) or viral titer ( Supplemental Fig. 2C [href=this file] ) in aged mice. No difference was seen in TIM-3 expression between aged and WT mice ( Supplemental Fig. 2D [href=this file] , 2E [href=this file] ). However, LAG-3 expression was increased ( Supplemental Fig. 2F [href=this file] , 2G [href=this file] ) on CD8 tet T cells. PD-1 absence did not significantly affect the expression levels or absolute cell numbers of CD44 and CD62L on CD8 T cells ( Supplemental Fig. 2H [href=this file] , 2K [href=this file] ). We found no significant differences in the frequency or absolute cell number of CD4 Foxp3 , T-bet , GATA3 ( Supplemental Fig. 2I [href=this file] , 2L [href=this file] ), or Th1/Th2 ratio between genotypes ( Supplemental Fig. 2J [href=this file] ).\n",
      "We next assessed whether this increase in CD8 T cell function was comparable to the increase in function we observed in young T cells compared with young WT T cells both in this study and in our previous studies ( Supplemental Fig. 2M [href=this file] ) ( 12–15 [href=#r12] ).\n",
      "We confirmed the identity of transplanted T cells by assessing PD-1 expression on CD45.2 CD8 T cells ( Supplemental Fig. 3A [href=this file] ). There was no difference in weight loss or viral burden between the four groups ( Supplemental Fig. 3B [href=this file] , 3C [href=this file] ). We also assessed engraftment of recipient (CD45.1) and donor (CD45.2) total CD3 lymphocytes, CD4 T cells, and CD8 T cells ( Supplemental Fig. 3D [href=this file] ). lymphocytes engrafted better than B6 lymphocytes ( Supplemental Fig. 3D [href=this file] – 3G [href=this file] ), especially in bulk CD3 ( Supplemental Fig. 3E [href=this file] ) and CD8 T lymphocytes ( Supplemental Fig. 3G [href=this file] ), as previously observed ( 15 [href=#r15] ).\n",
      "To remove irradiation as a confounding variable, we performed an adoptive transfer of young or aged CD8 T cells into young recipients along with young WT CD4 T cells and B cells ( Supplemental Fig. 4A [href=this file] ). When we compared the function of young and aged CD8 T cells in this model, we observed that young CD8 T cells were still more functional in granzyme B and IFN-γ production than aged CD8 T cells ( Supplemental Fig. 4B [href=this file] – 4D [href=this file] ).\n",
      "Fig. 4E [href=this file] ). Viral replication may have been limited in this rechallenge by anti-HMPV Ab responses in all groups ( Supplemental Fig. 4F [href=this file] ).\n",
      "The online version of this article contains supplemental material [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 173112\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 44030 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e20a9cc4c12c0f1b4c7523347798375d3343cb21fecb50af2997ed5d6764ffa1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 44030 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-35521ba7-b623-45b2-b4d5-e04731ea6c79\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22255433\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22255433/wiki/604556\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22255433\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22255433/wiki/604556\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322434,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 59466,\n",
      "        \"completion_tokens\": 72,\n",
      "        \"total_tokens\": 59538,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn22255433', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22255433/wiki/604556', 'dataset_id': 'syn22255433'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn22255433, synapse.org, https://www.synapse.org/#!Synapse:syn22255433/wiki/604556\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn22255433', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn22255433', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22255433 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22255433 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22255433 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22255433 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn22255433 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22255433\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 66th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5387919&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6858990>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 11 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#abstract1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 26) for abstract1: Abstract [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 29) for sec13: Usage Notes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 29) for sec13: Usage Notes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#d32e23'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for d32e23: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 143837\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 36711 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-2892b1d0be3bb7af01be081d038c08d1a65f70c43ad4f0955fd94122cb80dcfe\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 36711 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-56a4d20a-1896-47e3-b9b9-9f03ac02200d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5556216\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5522788\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5522790\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6022465\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6022474\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5004874\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6035832\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5492805\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5493036\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4984604\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.synapse.org/cutaneousNF\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5556216\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5522788\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5522790\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6022465\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6022474\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5004874\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn6035832\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5492805\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5493036\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4984604\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.synapse.org/cutaneousNF\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322452,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 50681,\n",
      "        \"completion_tokens\": 406,\n",
      "        \"total_tokens\": 51087,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5556216', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5556216'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5556216, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5556216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5556216', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5522788', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5522788'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5522788, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5522788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5522788', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5522790', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5522790'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5522790, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5522790\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5522790', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn6022465', 'repository_reference': 'synapse.org', 'dataset_id': 'syn6022465'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn6022465, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn6022465\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn6022465', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn6022474', 'repository_reference': 'synapse.org', 'dataset_id': 'syn6022474'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn6022474, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn6022474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn6022474', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5004874', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5004874'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5004874, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5004874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5004874', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn6035832', 'repository_reference': 'synapse.org', 'dataset_id': 'syn6035832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn6035832, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn6035832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn6035832', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5492805', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5492805'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5492805, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5492805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5492805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5493036', 'repository_reference': 'synapse.org', 'dataset_id': 'syn5493036'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn5493036, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn5493036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn5493036', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4984604', 'repository_reference': 'doi.org', 'dataset_id': 'syn4984604'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn4984604, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn4984604\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn4984604', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.synapse.org/cutaneousNF', 'repository_reference': 'synapse.org', 'dataset_id': 'http://www.synapse.org/cutaneousNF'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.synapse.org/cutaneousNF\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.synapse.org/cutaneousNF\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.synapse.org/cutaneousNF, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org/cutaneousNF\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://www.synapse.org/cutaneousNF in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link http://www.synapse.org/cutaneousNF could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.synapse.org/[#!]*Synapse:__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 914 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 918 - INFO - Extracted ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.synapse.org/cutaneousNF is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://www.synapse.org/cutaneousNF', 'repository_reference': 'synapse.org', 'dataset_id': 'http://www.synapse.org/cutaneousNF'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn5556216', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5522788', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5522790', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn6022465', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn6022474', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5004874', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn6035832', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5492805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5493036', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4984604', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 10 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5556216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5556216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5556216 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5556216 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5556216 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5556216 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5556216 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn5556216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5556216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5522788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5522788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5522788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5522788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5522788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5522788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5522788 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn5522788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5522788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5522790\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5522790\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5522790 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5522790 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5522790 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5522790 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5522790 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn5522790\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5522790\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn6022465\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn6022465\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn6022465 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn6022465 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn6022465 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn6022465 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn6022465 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn6022465\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn6022465\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn6022474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn6022474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn6022474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn6022474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn6022474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn6022474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn6022474 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn6022474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn6022474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5004874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5004874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5004874 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5004874 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5004874 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5004874 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5004874 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn5004874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5004874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn6035832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn6035832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn6035832 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn6035832 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn6035832 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn6035832 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn6035832 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn6035832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn6035832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5492805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5492805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5492805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5492805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5492805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5492805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5492805 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn5492805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5492805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn5493036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5493036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5493036 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5493036 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5493036 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5493036 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn5493036 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn5493036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5493036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn4984604\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4984604\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4984604 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4984604 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4984604 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4984604 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn4984604 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn4984604\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4984604\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 10\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 12\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 67th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8094983&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b77fa640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b77fa640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The full phylogenetic tree can be found in the supplementary (Additional file 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 110966\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 28494 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f814bed49bd6ae3768d3fa5908bed0d2d82b40ed92b9ce5d24afb4929826d29e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 28494 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6b301c67-57dc-48a1-83fe-e4d4d8bc0a76\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_415464\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461001\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461002\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461003\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461051\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461052\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461053\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461054\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461055\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461056\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461057\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461058\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461059\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461060\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23532550\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23532550/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_415464\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461001\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461002\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461003\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461051\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461052\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461053\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461054\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461055\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461056\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461057\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461058\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461059\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EPI_ISL_461060\\\",\\n    \\\"repository_reference\\\": \\\"GISAID\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23532550\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23532550/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322455,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 32630,\n",
      "        \"completion_tokens\": 581,\n",
      "        \"total_tokens\": 33211,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_415464', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_415464'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_415464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_415464 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_415464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_415464', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461001', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461001'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461001 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461001', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461002', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461002'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461002 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461002', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461003', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461003'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461003\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461003 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461003\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461003', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461051', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461051'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461051\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461051 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461051\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461051', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461052', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461052'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461052 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461052', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461053', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461053 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461053', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461054', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461054'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461054\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461054 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461054\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461054', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461055', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461055'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461055 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461055', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461056', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461056'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461056 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461056', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461057', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461057 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461057', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461058', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461058'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461058\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461058 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461058\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461058', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461059', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461059'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461059\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461059 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461059\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461059', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EPI_ISL_461060', 'repository_reference': 'GISAID', 'dataset_id': 'EPI_ISL_461060'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EPI_ISL_461060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EPI_ISL_461060 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EPI_ISL_461060', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn23532550', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23532550/', 'dataset_id': 'syn23532550'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn23532550, synapse.org, https://www.synapse.org/#!Synapse:syn23532550/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn23532550', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'EPI_ISL_415464', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461001', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461002', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461003', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461051', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461052', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461053', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461054', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461055', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461056', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461057', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461058', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461059', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EPI_ISL_461060', 'data_repository': 'GISAID', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23532550', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 15 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_415464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_415464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_415464 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_415464 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: gisaid and accession_id: EPI_ISL_415464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461001 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461001 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: gisaid and accession_id: EPI_ISL_461001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461002 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461002 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: gisaid and accession_id: EPI_ISL_461002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461003\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461003\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461003 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461003 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: gisaid and accession_id: EPI_ISL_461003\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461051\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461051\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461051 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461051 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: gisaid and accession_id: EPI_ISL_461051\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461052 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461052 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: gisaid and accession_id: EPI_ISL_461052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 6.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461053 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461053 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: gisaid and accession_id: EPI_ISL_461053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461054\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461054\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461054 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461054 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: gisaid and accession_id: EPI_ISL_461054\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 8.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461055 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461055 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: gisaid and accession_id: EPI_ISL_461055\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461056 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461056 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: gisaid and accession_id: EPI_ISL_461056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 10.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461057 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461057 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: gisaid and accession_id: EPI_ISL_461057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461058\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461058\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461058 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461058 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: gisaid and accession_id: EPI_ISL_461058\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 12.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461059\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461059\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461059 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461059 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: gisaid and accession_id: EPI_ISL_461059\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 13.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EPI_ISL_461060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GISAID, identifier: EPI_ISL_461060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EPI_ISL_461060 for GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GISAID\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EPI_ISL_461060 in gisaid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 14 with repo: gisaid and accession_id: EPI_ISL_461060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository gisaid unknown in Ontology. Skipping dataset page 14.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23532550 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23532550 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23532550 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23532550 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn23532550 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 15 with repo: synapse.org and accession_id: syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23532550\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 15: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 15\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 68th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7962955&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b75b0e80>, <Element supplementary-material at 0x1b6ba7bc0>, <Element supplementary-material at 0x1b0bdef00>, <Element supplementary-material at 0x1b0bddd00>, <Element supplementary-material at 0x1b0bdf500>, <Element supplementary-material at 0x1b0bdd9c0>, <Element supplementary-material at 0x1b0bdd640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b75b0e80>, <Element supplementary-material at 0x1b6ba7bc0>, <Element supplementary-material at 0x1b0bdef00>, <Element supplementary-material at 0x1b0bddd00>, <Element supplementary-material at 0x1b0bdf500>, <Element supplementary-material at 0x1b0bdd9c0>, <Element supplementary-material at 0x1b0bdd640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: See also Figure S1 [this file] . \n",
      "For further validation, by randomizing the set of tropisms, the LDA loses all its discriminating power ( Figure S1 [this file] A). Supporting the robustness of the clustering, we obtained the same performance by using another reduced list of human viruses from ViralZone ( Hulo et al., 2011 [rid=bib34] ; Figure S1 [this file] B; Table S1 [rid=mmc2] ). \n",
      "As shown in Figure S1 [this file] , we observed that RNA folding, as determined by the minimum free energy, was also non-randomly distributed among tropisms. \n",
      "This analysis was replicated with the other list of viruses from ViralZone, leading to comparable predictive outcomes ( Figure S2 [this file] ; p < 0.01, permutation test in Table S2 [rid=mmc3] ). In concordance, a LDA of the average SDA of each virus across tissues can similarly separate different clusters of viral tropism based on their translational efficiencies ( Figure S2 [this file] ). \n",
      "See also Figures S2–S5 [this file] . \n",
      "The results showed similar predictive performances compared to TCGA ( Figure S3 [this file] ). \n",
      "As a result, we observed that the SARS-CoV-2 proteome is especially adapted to the upper respiratory airways and alveoli but also to other tissues such as the gastrointestinal tract and brain ( Figures S4 [this file] and S5 [this file] ; Table S3 [rid=mmc4] ; see STAR Methods [rid=sec4] ). \n",
      "However, as suggested by Figure S1 [this file] , many other and overlapping forces, such as mRNA stability, frameshift motifs, transcriptional regulation, or codon-dependent immune responses, are also known to shape the composition of coding sequences ( Bergman and Tuller, 2020 [rid=bib10] ; Li et al., 2012 [rid=bib39] ). \n",
      "In particular, we found that SARS-CoV-2 is highly adapted to the upper respiratory tract and the alveoli ( Figure S4 [this file] C), which is in agreement with recent single-cell transcriptomic studies reporting the expression of ACE2 in the nasal goblet and ciliated cells as well as the type-2 alveolar epithelial cells ( Sungnak et al., 2020 [rid=bib66] ; Ziegler et al., 2020 [rid=bib83] ). In terms of the evolution of the new coronavirus, given the similarity of SARS-CoV-2 SDA with the phylogenetically closest bat coronavirus ( Figure S5 [this file] B), it seems that a translational selection to increase SDA would have acted before the putative zoonosis from bats or other intermediate hosts. \n",
      "We assigned each virus to its corresponding tropism ( Table S1 [rid=mmc2] ) in order to find the linear combination of tissue adaptation features that maximized differences between viral target tissues ( Figure S2 [this file] ; Table S2 [rid=mmc3] ). \n",
      "We then computed the SDA of viral proteins from the three pandemic coronaviruses of the last two decades SARS-CoV ( Drosten et al., 2003 [rid=bib19] ), MERS-CoV ( Zaki et al., 2012 [rid=bib75] ), and SARS-CoV-2 ( Zhou et al., 2020a [rid=bib81] ), as well as the common flu-causing influenza A virus (H1N1) along the respiratory tract ( Figure S4 [this file] A). For this reason, we also extended our translational analysis to all the 23 tissues of the TCGA dataset ( Figure S5 [this file] A). \n",
      "Moreover, given that the tropisms not only depend on the translational adaptation to the host, but also on the expression of the required entry receptors, we measured the respective receptors of each virus ( Figure S4 [this file] B; Table S3 [rid=mmc4] ). \n",
      "In an attempt to elucidate the translational selection that could have benefitted the evolution of the new coronavirus, we also compared the SDA adaptation of SARS-CoV-2 to those of close phylogenetic strains ( Figure S5 [this file] B): the human-infecting SARS-CoV and the bat coronavirus RatG13, with 79.6% and 96.2% of sequence identity, respectively ( Zhou et al., 2020a [rid=bib81] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc2: Publicly available genomic data comprised a total of 502 human-infecting viruses, distributed among 35 families and covering all 7 Baltimore categories ( Table S1 [this file] ). Their corresponding coding sequences constituted a total of 6,087 viral proteins ( Table S1 [this file] ), for which we determined the relative codon usage (RCU; i.e., the contribution of each synonymous codon to the amino acid it encodes; see STAR Methods [rid=sec4] ). \n",
      "Supporting the robustness of the clustering, we obtained the same performance by using another reduced list of human viruses from ViralZone ( Hulo et al., 2011 [rid=bib34] ; Figure S1 [rid=mmc1] B; Table S1 [this file] ). \n",
      "Also, for some viruses, multiple genotypes and variants are represented in the NCBI database, although they actually do showcase differences in codon usage ( Table S1 [this file] ; Figure 1 [rid=fig1] ). \n",
      "Table S1 [this file] contains all human-infecting viruses and their associated metadata. \n",
      "The coding sequences of human-infecting viruses from RefSeq were downloaded from the Codon/Codon Pair Usage Tables (CoCoPUTs) project release as of June 9, 2020 ( Alexaki et al., 2019 [rid=bib4] ; Athey et al., 2017 [rid=bib5] ; Table S1 [this file] ). \n",
      "We assigned each virus to its corresponding tropism ( Table S1 [this file] ) in order to find the linear combination of codon features that maximized differences between viral target tissues. \n",
      "We assigned each virus to its corresponding tropism ( Table S1 [this file] ) in order to find the linear combination of tissue adaptation features that maximized differences between viral target tissues ( Figure S2 [rid=mmc1] ; Table S2 [rid=mmc3] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc3: Therefore, taking the average of all healthy samples across each of the 23 TCGA cancer types, we determined the estimated translational efficiencies of viral proteins in different human tissues ( Table S2 [this file] ). \n",
      "The resulting performance of the models, based on the area under the curve (AUC) of their receiver operating characteristic (ROC) curves, ranges 0.79–0.92 ( Figure 2 [rid=fig2] A), which is clearly higher than the no-skill model of 0.5 (p < 0.01, permutation test in Table S2 [this file] ). This analysis was replicated with the other list of viruses from ViralZone, leading to comparable predictive outcomes ( Figure S2 [rid=mmc1] ; p < 0.01, permutation test in Table S2 [this file] ). \n",
      "Refer to Table S2 [this file] for full TCGA cancer type names. \n",
      "To evaluate the adaptation of the viral proteins to the SDAw of human tissues, we computed their average SDA to each of the 23 TCGA tissues ( Table S2 [this file] ). \n",
      "In addition, we also validate the predictive potential of the model by performing a permutation test over 100 randomizations of the tropism labels of the dataset ( Table S2 [this file] ). \n",
      "For the dataset of Hydro-tRNaseq of human cells, we computed the average viral RtAI to each of the seven cell lines ( Table S2 [this file] ). \n",
      "We assigned each virus to its corresponding tropism ( Table S1 [rid=mmc2] ) in order to find the linear combination of tissue adaptation features that maximized differences between viral target tissues ( Figure S2 [rid=mmc1] ; Table S2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc4: As a result, we observed that the SARS-CoV-2 proteome is especially adapted to the upper respiratory airways and alveoli but also to other tissues such as the gastrointestinal tract and brain ( Figures S4 [rid=mmc1] and S5 [rid=mmc1] ; Table S3 [this file] ; see STAR Methods [rid=sec4] ). \n",
      "The SARS-CoV-2 and RaTG13 sequences were directly downloaded from GenBank ( Table S3 [this file] ; NCBI RefSeq: NC_045512.2 (NC_045512.2) ; GenBank: MN996532.1 (MN996532.1) ). \n",
      "We initially reconstructed tRNA expression profiles along the respiratory tract making use of the spatial information associated with healthy TCGA samples from head and neck squamous cell carcinoma (HNSC), lung squamous cell carcinoma (LUSC) and lung adenocarcinoma (LUAD) ( Table S3 [this file] ). \n",
      "Moreover, given that the tropisms not only depend on the translational adaptation to the host, but also on the expression of the required entry receptors, we measured the respective receptors of each virus ( Figure S4 [rid=mmc1] B; Table S3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc5: To estimate the adaptation of each protein to the tRNA-based codon efficiencies of each tissue, we computed their SDA ( Hernandez-Alias et al., 2020 [rid=bib30] ; Table S4 [this file] ). For that purpose, we matched each virus to the tRNAs of their tissues of infection ( Table S4 [this file] ). \n",
      "(A) Average SDA of replication (Xr) and structural (Xs) proteins of a total of 104 annotated tropism-specific viruses matched to 461 samples of their tissues of infection ( Table S4 [this file] ). \n",
      "(B) Top 10 positive and negative virus orthologous groups upon gene set enrichment analysis of the SDA of all proteins of tropism-specific viruses ( Table S4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc6: Using 3 previously published small RNA-seq datasets of human cell lines upon viral infection ( Chang et al., 2013 [rid=bib15] ; Shi et al., 2018 [rid=bib62] ; Stark et al., 2012 [rid=bib63] ; GEO: GSE33584 (ncbi-geo:GSE33584) , GSE102470 (ncbi-geo:GSE102470) , GSE57763 (ncbi-geo:GSE57763) ), we quantified the tRNA abundances at different time points ( Table S5 [this file] ). \n",
      "(A) Relative tRNA adaptation index (see STAR Methods [rid=sec4] ; Table S5 [this file] ) of viral proteins upon effective viral infections in different cell lines. Proteins are allocated to different time expression classes based on current viral knowledge ( Knipe and Howley, 2013 [rid=bib36] ; Table S5 [this file] ). \n",
      "(B) Abundances of viral proteins (see STAR Methods [rid=sec4] ; Table S5 [this file] ) upon effective viral infections at different time points in different cell lines. \n",
      "All proteomic data are accessible in Table S5 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc7: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 233255\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 59066 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-def4a1020b675961f94a4d705471ea2388205c9b9d5966089243c1f65abb2b66\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 59066 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-98146dcb-a75c-41f2-a203-929ed3bdbb3b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"pdc.cancer.gov, gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20640275\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NC_045512.2\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"MN996532.1\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33584\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE102470\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57763\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE137834\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-9905\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"pdc.cancer.gov, gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20640275\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NC_045512.2\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"MN996532.1\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE33584\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE102470\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE57763\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE137834\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-9905\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322458,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 70098,\n",
      "        \"completion_tokens\": 320,\n",
      "        \"total_tokens\": 70418,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA', 'repository_reference': 'pdc.cancer.gov, gdc.cancer.gov', 'dataset_id': 'TCGA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TCGA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: pdc.cancer.gov, gdc.cancer.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - ERROR - Error extracting domain from URL: pdc.cancer.gov, gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'Unknown_Publisher', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20640275', 'repository_reference': 'synapse.org', 'dataset_id': 'syn20640275'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20640275, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20640275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20640275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NC_045512.2', 'repository_reference': 'nuccore', 'dataset_id': 'NC_045512.2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NC_045512.2 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'NC_045512.2', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'MN996532.1', 'repository_reference': 'nuccore', 'dataset_id': 'MN996532.1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID MN996532.1 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'MN996532.1', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE33584', 'repository_reference': 'geo', 'dataset_id': 'GSE33584'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE33584, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE33584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33584', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE102470', 'repository_reference': 'geo', 'dataset_id': 'GSE102470'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE102470, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE102470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE102470', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE57763', 'repository_reference': 'geo', 'dataset_id': 'GSE57763'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE57763, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE57763\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE57763', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE137834', 'repository_reference': 'geo', 'dataset_id': 'GSE137834'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE137834, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE137834\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE137834', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-9905', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-9905'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-9905 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-9905', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'TCGA', 'data_repository': 'Unknown_Publisher', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20640275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'NC_045512.2', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'MN996532.1', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE33584', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE102470', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE57763', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE137834', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-9905', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: Unknown_Publisher, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TCGA in unknown_publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: unknown_publisher and accession_id: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository unknown_publisher unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20640275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20640275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20640275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20640275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20640275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20640275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20640275 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn20640275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20640275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NC_045512.2 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for NC_045512.2 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: nuccore and accession_id: NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MN996532.1 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for MN996532.1 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: nuccore and accession_id: MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/MN996532.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE33584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE33584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33584 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33584 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE33584 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE33584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE102470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE102470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE102470 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE102470 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE102470 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE102470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE57763\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE57763\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE57763 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE57763 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE57763 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE57763\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57763\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE137834\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE137834\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE137834 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE137834 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE137834 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE137834\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137834\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9905 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-9905 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: arrayexpress and accession_id: E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-9905\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 69th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11601401&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b71ffbc0>, <Element supplementary-material at 0x1b1596dc0>, <Element supplementary-material at 0x1b630e640>, <Element supplementary-material at 0x1b1b9c6c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b71ffbc0>, <Element supplementary-material at 0x1b1596dc0>, <Element supplementary-material at 0x1b630e640>, <Element supplementary-material at 0x1b1b9c6c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: Following preprocessing and quality control, our dataset consisted of 59,588 tissue-covered spots, with each spot capturing RNA from an average of 3.7 nuclei and detecting an average of 2,361 genes ( Supplementary Figure 1a [this file] – d [this file] ). This analysis revealed significant correlations between the two data modalities, suggesting that immunofluorescent staining can be reliably combined with the Visium ST protocol ( Supplementary Figure 2a [this file] – d [this file] ). \n",
      "Comparing layer-specific transcriptional signatures derived from our dataset with those derived from a published DLPFC ST dataset with H&E-based manual layer annotation showed a strong correlation, confirming the concordance of our clustering-based annotation with classical histopathological annotation ( Supplementary Figure 3a [this file] ) . Furthermore, using our layer annotations and plaque locations, we found a higher rate of plaques in the deeper layers (L4 - L6) compared to the upper layers (L1 - L3), ( Supplementary Table 2 [rid=SD3] ; Supplementary Figure 3b [this file] – e [this file] ), similar to previously published results . \n",
      "Metallothioneins showed the highest transcription levels in the nuclei of astrocytes, although they were also transcribed by other cell types at lower levels ( Supplementary Figure 4a [this file] ). The upregulated gene was almost exclusively transcribed by astrocytes in our human gray matter DLPFC data ( Supplementary Figure 4a [this file] ). \n",
      "The predicted abundances of excitatory neuronal subpopulations reflected the cortical laminar structure, consistent with the layer-specific annotations of these subpopulations from the Allen Brain Map ( Supplementary Figure 5a [this file] – d [this file] ). Consistently, Ast.5 exhibited a reactive-like transcription signature in the snRNA-seq data with high levels of and expression ( Supplementary Figure 4b [this file] ), two of the four genes significantly upregulated at neuritic plaques . \n",
      "At a cellular level, astrocytes are known to exhibit the highest levels of metallothionein expression in the human brain ( Supplementary Figure 4 [this file] ), but other cell types may also contribute. \n",
      "GFAP MFI values per spot ( Supplementary Figure 2 [this file] ) were calculated in ImageJ (1.54f) by mapping spot coordinates to raw images. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Demographic and clinicopathologic characteristics are presented in Supplementary Table 1 [this file] . \n",
      "We next sought to confirm the overexpression of SERPINA3 in astrocytes proximal to neuritic plaques at the protein level using an independent set of 20 ROSMAP paraffin-embedded DLPFC tissue sections from donors with Braak stages 5 or 6 ( Supplementary Table 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: Furthermore, using our layer annotations and plaque locations, we found a higher rate of plaques in the deeper layers (L4 - L6) compared to the upper layers (L1 - L3), ( Supplementary Table 2 [this file] ; Supplementary Figure 3b [rid=SD1] – e [rid=SD1] ), similar to previously published results . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: This revealed a statistically significant upregulation of , , , and at the plaque niche ( Figure 1b [rid=F1] and Supplementary Table 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 92067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 23769 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c58fb99712507710df2bce721da83f0b637b5b7ed53864718586b0b653425283\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 23769 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a1a080b9-8081-46f2-a931-c19b5760d7ca\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51150434\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.58085/HR6S-2991\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000238\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000239\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn62110225\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51150434\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.58085/HR6S-2991\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000238\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000239\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn62110225\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322461,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 29256,\n",
      "        \"completion_tokens\": 207,\n",
      "        \"total_tokens\": 29463,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51150434', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn51150434'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn51150434, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn51150434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn51150434', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.58085/HR6S-2991', 'repository_reference': 'doi.org', 'dataset_id': 'https://doi.org/10.58085/HR6S-2991'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.58085/, None, https://doi.org/10.58085/HR6S-2991\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.58085/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.58085/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'CG000238', 'repository_reference': 'n/a', 'dataset_id': 'CG000238'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: CG000238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID CG000238 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: CG000238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CG000238 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier CG000238 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier CG000238 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CG000238 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CG000238 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'CG000238', 'repository_reference': 'n/a', 'dataset_id': 'CG000238'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'CG000239', 'repository_reference': 'n/a', 'dataset_id': 'CG000239'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: CG000239\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID CG000239 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: CG000239\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CG000239 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier CG000239 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier CG000239 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CG000239 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CG000239 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'CG000239', 'repository_reference': 'n/a', 'dataset_id': 'CG000239'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn62110225', 'repository_reference': 'https://adknowledgeportal.org/', 'dataset_id': 'syn62110225'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn62110225, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn62110225\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn62110225', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51150434', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.58085/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn62110225', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn51150434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51150434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51150434 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51150434 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51150434 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51150434 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn51150434 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51150434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51150434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.58085/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.58085/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.58085/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn62110225\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn62110225\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn62110225 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn62110225 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn62110225 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn62110225 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn62110225 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn62110225\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn62110225\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 70th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9712627&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b75b12d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 36 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s1a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s1a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s1c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s1c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s1c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s1c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3b9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3b9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 205308\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 52079 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-5e80f10d0decc66a97fcf11cdecc7c6456578d3f7dc9f446629aa6559a7bd57e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 52079 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-67c87d7d-491f-415d-8acc-dc3d0d3fea3a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12514624\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148822\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148822\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE137444\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137444\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE167494\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167494\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE183068\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183068\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE174367\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE174332\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174332\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000376\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC9712627\\\",\\n    \\\"repository_reference\\\": \\\"PMC\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12514624\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148822\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148822\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE137444\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137444\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE167494\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167494\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE183068\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183068\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE174367\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE174332\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174332\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE160936\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CG000376\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC9712627\\\",\\n    \\\"repository_reference\\\": \\\"PMC\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322482,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 77456,\n",
      "        \"completion_tokens\": 1103,\n",
      "        \"total_tokens\": 78559,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE160936', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936', 'dataset_id': 'GSE160936'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE160936, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE160936', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn12514624', 'repository_reference': 'synapse.org', 'dataset_id': 'syn12514624'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn12514624, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn12514624\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn12514624', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE178265', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265', 'dataset_id': 'GSE178265'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE178265, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178265', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE148822', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148822', 'dataset_id': 'GSE148822'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE148822, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE148822', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE137444', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137444', 'dataset_id': 'GSE137444'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE137444, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE137444', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE167494', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167494', 'dataset_id': 'GSE167494'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE167494, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE167494', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE183068', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183068', 'dataset_id': 'GSE183068'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE183068, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE183068', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE174367', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367', 'dataset_id': 'GSE174367'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE174367, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE174367', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE174332', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174332', 'dataset_id': 'GSE174332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE174332, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE174332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'CG000376', 'repository_reference': 'nuccore', 'dataset_id': 'CG000376'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID CG000376 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'CG000376', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PMC9712627', 'repository_reference': 'PMC', 'dataset_id': 'PMC9712627'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PMC9712627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PMC9712627 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PMC, identifier: PMC9712627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PMC9712627', 'data_repository': 'PMC', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE160936', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn12514624', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE178265', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE148822', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE137444', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE167494', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE183068', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE174367', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE174332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'CG000376', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PMC9712627', 'data_repository': 'PMC', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 11 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE160936 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE160936 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE160936 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160936\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn12514624\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn12514624\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn12514624 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn12514624 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn12514624 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn12514624 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn12514624 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn12514624\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn12514624\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178265 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178265 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE178265 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE148822 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE148822 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE148822 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE137444 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE137444 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE137444 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137444\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE167494 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE167494 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE167494 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167494\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE183068 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE183068 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE183068 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE174367 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE174367 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE174367 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE174332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE174332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE174332 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CG000376 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for CG000376 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: nuccore and accession_id: CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/CG000376\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PMC9712627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PMC, identifier: PMC9712627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC9712627 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PMC9712627 in pmc\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: pmc and accession_id: PMC9712627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository pmc unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 11\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 47, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 25\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 71/100 (71.00%) | Elapsed: 00:11:51 | ETA: 00:04:50\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 71th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4697768&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b67558d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 2 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 34173\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 9295 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-81996b5866870589201860ac2a760da0410c68d90bfe435f3caeb81d5d540418\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 9295 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1858e5e1-4793-434a-b291-2716ec153392\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2813581\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2813581\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"COSMIC\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2813581\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2813581\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"COSMIC\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322501,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 12096,\n",
      "        \"completion_tokens\": 90,\n",
      "        \"total_tokens\": 12186,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2813581', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2813581', 'dataset_id': 'syn2813581'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2813581, synapse.org, https://www.synapse.org/#!Synapse:syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2813581', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'COSMIC', 'repository_reference': 'n/a', 'dataset_id': 'COSMIC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID COSMIC is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier COSMIC for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier COSMIC for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier COSMIC for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier COSMIC for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier COSMIC for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'COSMIC', 'repository_reference': 'n/a', 'dataset_id': 'COSMIC'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn2813581', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2813581 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2813581 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2813581 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2813581 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2813581 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2813581\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 72th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9094742&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1b45780>, <Element supplementary-material at 0x1b7189800>, <Element supplementary-material at 0x1b77fa800>, <Element supplementary-material at 0x1b66d33c0>, <Element supplementary-material at 0x1b66f2200>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1b45780>, <Element supplementary-material at 0x1b7189800>, <Element supplementary-material at 0x1b77fa800>, <Element supplementary-material at 0x1b66d33c0>, <Element supplementary-material at 0x1b66f2200>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref fig1sdata1: When assessing myelin by MS , we quantified 332 proteins ( Figure 1—source data 1 [this file] ; labeled in green in Figure 1a [rid=fig1] ) with a false discovery rate (FDR) of <1% and an average sequence coverage of 39.6%. When using the ultra-definition (UD)MS mode, in which ion mobility spectrometry enables an additional separation of peptides after chromatography and before mass measurement, we identified and quantified 835 proteins with an average sequence coverage of 37.0% ( Figure 1—source data 1 [this file] ; labeled in blue in Figure 1a [rid=fig1] ). \n",
      "See Figure 1—source data 1 [this file] for datasets. ppm, parts per million. ( ) Scatter plot comparing the log -transformed relative abundance of proteins identified in myelin by MS against their abundance as identified by UDMS . \n",
      "See Figure 1—source data 1 [this file] for datasets. n = 5 biological replicates analyzed as two technical replicates each. \n",
      "Next, we compared our human ( Figure 1—source data 1 [this file] ) and mouse myelin ( Jahn et al., 2020 [rid=bib28] ) proteome datasets (by MS or UDMS ) with the averaged vdW-normalized mRNA abundance profile of MOL in the respective same species. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref fig2sdata1: Original immunoblots are provided in Figure 2—source data 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref fig3sdata1: To this aim, we retrieved previously published scRNA-seq datasets from the CNS of humans ( Jäkel et al., 2019 [rid=bib29] ; Zhou et al., 2020 [rid=bib71] ; Lake et al., 2018 [rid=bib34] ; Habib et al., 2017 [rid=bib23] ; Grubman et al., 2019 [rid=bib22] ; Wheeler et al., 2020 [rid=bib62] ) and mice ( Falcão et al., 2018 [rid=bib18] ; Wheeler et al., 2020 [rid=bib62] ; Marques et al., 2016 [rid=bib36] ; Saunders et al., 2018 [rid=bib48] ; Zeisel et al., 2018 [rid=bib69] ; Zeisel et al., 2015 [rid=bib68] ; Ximerakis et al., 2019 [rid=bib64] ) and evaluated all cells designated as oligodendrocyte progenitor cells (OPCs), newly formed oligodendrocytes (NFOs), and MOL from nondiseased subjects ( Figure 3—figure supplement 1a and b [rid=fig3s1] , Figure 3—source data 1 [this file] ). Indeed, only 132 cells designated as NFO were comprised in the available human scRNA-seq datasets, considerably fewer compared to 10,391 NFO recovered from the mouse datasets ( Figure 3—source data 1 [this file] ). \n",
      "For quality control, each of the retrieved datasets was analyzed using the Seurat R package (version 3.1.4; Butler et al., 2018 [rid=bib6] ) in an analysis pipeline, including validating sequencing quality, filtering for outlier cells (as specified in Figure 3—source data 1 [this file] ), log-normalizing the expression matrix with a scale factor 10,000, high variable gene selection and data scaling, linear dimensionality reduction using principal component analysis (PCA), and neighboring embedding using UMAP to ensure accurate cell type annotation and detect any potential batch effect. Specific parameters applied to individual datasets and the number of recovered cells are listed in Figure 3—source data 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref fig4sdata1: Cluster analysis was based on the KNN algorithm calculated with resolution 0.1, and clusters of differentially expressed genes were calculated using the model-based analysis of single-cell transcriptomics (MAST) algorithm ( Figure 4—source data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref transrepform: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 316290\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 79825 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-06d191d41143cf14ed8d8c036635cda845a242f66ea15d140509c14daf4d3aa1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 79825 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b8da960b-82fe-4258-9138-a6e65f802f66\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD029727\\\",\\n    \\\"repository_reference\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD020007\\\",\\n    \\\"repository_reference\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60361\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE775330\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75330\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE113973\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE116470\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116470\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRP135960\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/SRP135960\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE129788\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE130119\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130119\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE140511\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140511\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE140511\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE97930\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97930\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE138852\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE118257\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118257\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD029727\\\",\\n    \\\"repository_reference\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD020007\\\",\\n    \\\"repository_reference\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE60361\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE775330\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75330\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE113973\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE116470\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116470\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRP135960\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/SRP135960\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE129788\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE130119\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130119\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE140511\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140511\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"GSE140511\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE97930\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97930\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE138852\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE118257\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118257\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322506,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 90374,\n",
      "        \"completion_tokens\": 830,\n",
      "        \"total_tokens\": 91204,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PXD029727', 'repository_reference': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727', 'dataset_id': 'PXD029727'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PXD029727 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727, identifier: PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727 in ProteomeCentral\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PXD029727', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PXD020007', 'repository_reference': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007', 'dataset_id': 'PXD020007'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PXD020007 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007, identifier: PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007 in ProteomeCentral\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PXD020007', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE60361', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361', 'dataset_id': 'GSE60361'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE60361, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60361', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE775330', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75330', 'dataset_id': 'GSE775330'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE75330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE775330, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE775330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE775330', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE113973', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973', 'dataset_id': 'GSE113973'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE113973, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE113973', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE116470', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116470', 'dataset_id': 'GSE116470'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE116470, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE116470', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRP135960', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/SRP135960', 'dataset_id': 'SRP135960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRP135960 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/sra/SRP135960, identifier: SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRP135960 for https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRP135960 for https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRP135960 for https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRP135960 for https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRP135960 for https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/sra/SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRP135960', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE129788', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129788', 'dataset_id': 'GSE129788'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE129788, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE129788', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE130119', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130119', 'dataset_id': 'GSE130119'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE130119, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE130119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE140511', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140511', 'dataset_id': 'GSE140511'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE140511, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE140511', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE97930', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97930', 'dataset_id': 'GSE97930'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE97930, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE97930', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE138852', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852', 'dataset_id': 'GSE138852'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE138852, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE118257', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118257', 'dataset_id': 'GSE118257'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE118257, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE118257', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'PXD029727', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PXD020007', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE60361', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE775330', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE113973', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE116470', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRP135960', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE129788', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE130119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE140511', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE97930', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE118257', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 13 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: proteomecentral.proteomexchange.org, identifier: PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PXD029727 in proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: proteomecentral.proteomexchange.org and accession_id: PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD029727\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: proteomecentral.proteomexchange.org, identifier: PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PXD020007 in proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: proteomecentral.proteomexchange.org and accession_id: PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020007\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE60361 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE60361 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE60361 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE775330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE775330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE775330 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE775330 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE775330 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE775330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE775330\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE113973 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE113973 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE113973 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE116470 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE116470 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE116470 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRP135960 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRP135960 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRP135960 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRP135960 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRP135960 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRP135960 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: www.ncbi.nlm.nih.gov and accession_id: SRP135960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE129788 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE129788 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE129788 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE130119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE130119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE130119 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE140511 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE140511 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE140511 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE97930 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE97930 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE97930 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97930\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE138852 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE118257 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE118257 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE118257 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118257\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 13\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 18, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 73th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8565335&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7b81280>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7b81280>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup1: As we show in Supplementary Figure S6b [this file] , Epitome performs better when considering each epigenetic signal independently. \n",
      "All ENCODE accessions used are listed in Supplementary Table S2 [this file] . \n",
      "Experiments used from ChIP-Atlas are listed in Supplementary Table S3 [this file] . \n",
      "We then calculate the fraction of unique peaks shown in Figure 1 [rid=F1] A and Supplementary Figure S2 [this file] as: \n",
      "All ChIP-seq targets and cell types used in Epitome are listed in Supplementary Table S2 [this file] . \n",
      "However, Epitome can make predictions in a query cellular context using other assays that measure chromatin accessibility, such as ATAC-seq (see Supplementary Figure S14 [this file] ). \n",
      "This maximum genomic distance of 12 kb was determined through hyperparameter search using validation chromosome 7 (See Supplementary Figure S12a [this file] ). Without the CASV, Epitome is not able to provide cell type specific predictions for , as shown in Supplementary Figure S12a [this file] . \n",
      "However, ATAC-seq can also be used to compute the CASV (See Supplementary Figure S14 [this file] ). \n",
      "Training on multiple cell types allows Epitome to generalize well to new cell types, and allows Epitome to achieve similar precision to methods that see the cell type being evaluated during training ( Supplementary Figures S10 and S11 [this file] ). This procedure is visualized in Supplementary Figure S4 [this file] , where at each iteration, the choice of cell types used in and the cell type used for are rotated through until the model has converged. \n",
      "Supplementary Figure S16 [this file] visualizes changes in validation and training loss for a subset of 18 ChIP-seq targets in a joint Epitome model trained for 2000 iterations, where training loss is calculated as the mean sigmoid cross entropy across 10 000 randomly selected training regions, and the validation loss is calculated as the mean sigmoid cross entropy across all 200 bp regions on chromosome 7 that have epigenetic signal from at least one of the 18 ChIP-seq targets. \n",
      "For Epitome, we found 5 and 0 to be the most optimal patience and min-delta, respectively, after tuning and comparing the hyperparameters shown in Supplementary Figure S15b [this file] . \n",
      "Supplementary Figure S15a [this file] compares the model accuracy (auPRC) with and without the early stopping method on 85 ChIP-seq targets. \n",
      "Information regarding each algorithm and the features used for each method are listed in Supplementary Table S1 [this file] . We compared these methods by predicting binding sites for 77 TFs and chromatin modifiers in 40 held out cell lines, tissues, and primary cells, enumerated in Supplementary Table S6 [this file] . \n",
      "We additionally used motifs published by Kheradpour ( 50 [rid=B50] ), available at http://compbio.mit.edu/encode-motifs/ (http://compbio.mit.edu/encode-motifs/) (See Supplementary Table S5 [this file] for a list of motifs used). \n",
      "All combinations considered are listed in Supplementary Table S6 [this file] . In total, we trained 323 DeFCoM models. auPRC and pAUC scores for both methods can be found in Supplementary Table S7 [this file] . \n",
      "Therefore, we only compare Epitome and DeFCoM in regions overlapping motifs, shown in Figure 2 [rid=F2] B and Supplementary Figure S8 [this file] . \n",
      "This is demonstrated in Supplementary Figure S7 [this file] , which shows comparisons of pAUC and auPRC when evaluated on all 200 bp regions on chromosomes 8 and 9. While it is generally a simple classification problem to predict nonbinding in genomic loci with no measured regulatory activity, it is harder to predict nonbinding in regions that have some epigenetic signal. pAUC and auPRC values used to generate Figure 2 [rid=F2] using this sampling technique on chromosomes 8 and 9 can be found in Supplementary Table S8 [this file] . Example ROC and PR curves are shown in Supplementary Figure S9 [this file] . \n",
      "We used all motifs as those used to train DeFCoM, available in Cis-BP database and Kheradpour ( 49 [rid=B49] , 50 [rid=B50] ) ( Supplementary Table S5 [this file] ). \n",
      "Evaluations of Epitome and DeepSEA are shown in Supplementary Figures S10 and S11 [this file] . We selected all 200 bp regions for evaluation as defined in Defining the test set to compare Epitome, Catchitt and Avocado. pAUC and auPRC values used to generate Supplementary Figure S10 [this file] can be found in Supplementary Table S9 [this file] . \n",
      "Supplementary Figure S5 [this file] shows the ratio of ChIP-seq peaks that do not overlap any motif for a TF or chromatin modifier of interest. We considered all motifs specified in Supplementary Table S5 [this file] . TFs, chromatin modifiers, and cell types evaluated are listed in Supplementary Table S6 [this file] . Ratios used to generate Supplementary Figure S5 [this file] can be found in Supplementary Table S10 [this file] . \n",
      "Heatmaps shown in Supplementary Figure S12 [this file] were generated using Seaborn ( 53 [rid=B53] ). \n",
      "Supplementary Figure S14 [this file] shows that Epitome models that are trained on DNase-seq and validated using ATAC-seq perform comparably when validated using DNase-seq. \n",
      "listed in Supplementary Table S4 [this file] . \n",
      "We therefore compared Epitome and DeFCoM by evaluating the ability of each method to predict TFBS overlapping motifs for 77 TFs and chromatin modifiers across 40 cell lines, primary cells, and tissues ( Supplementary Table S6 [this file] ). \n",
      "This is demonstrated in Supplementary Figure S5 [this file] , which shows that the ratio of ChIP-seq peaks that do not overlap any motif for a TF in question can range from 0.45 to 1.0 for the 77 TFs and chromatin modifiers we evaluated (see Evaluation of motif overlap of ChIP-seq peaks for 77 transcription factors and chromatin modifiers in 40 cell types). To compare to these methods, we first evaluated chromosome wide predictions of Epitome, Avocado, and Catchitt, on all 77 TFs and chromatin modifiers across 40 cell lines, primary cells, and tissues ( Supplementary Figure S6 [this file] ) on held-out chromosomes 8 and 9 in Figure 2A [rid=F2] , for a total of 264 comparisons. While joint Epitome models perform similarly in terms of pAUC and better for many instances in terms of auPRC compared to single Catchitt models ( Supplementary Figure S6a [this file] ), Avocado performs poorest on these metrics (Figure 2A [rid=F2] ). Supplementary Figure S6b [this file] additionally shows that for many cases, single Epitome models perform better than jointly trained Epitome models. As shown in Supplementary Figure S16 [this file] , different ChIP-seq targets trained in a joint model converge at different iterations during training. \n",
      "Supplementary Figure S10 [this file] shows that even though DeepSEA trains using a given test cell line and Epitome does not, Epitome performs comparably to DeepSEA in both pAUC and auPRC. Supplementary Figure S11 [this file] show receiver operating characteristic curves and precision-recall curves for these methods in all four held out cell lines from ENCODE. \n",
      "This observation is most easily observed in change in AUC as wider genomic contexts are considered, shown in Supplementary Figure S12a [this file] . Supplementary Figure S12b and c [this file] shows that when histone modifications are trained on more than 10 cell types, performance decreases as wider genomic contexts are considered. \n",
      "This is also a common use case that a query cellular context has been partially characterized with multiple experiments, including chromatin accessibility and various histone modifications ( Supplementary Figure S3 [this file] ). \n",
      "Although the Epitome model primarily uses DNase-seq to compute the CASV and train its models, we show in Supplementary Figure S14 [this file] that Epitome can accurately predict histone modifications and TFBS when using ATAC-seq during evaluation in the CASV when Epitome has been trained using DNase-seq. \n",
      "Figure 5D [rid=F5] and Supplementary Figure S13a [this file] show Epitome predictions for H3K27ac at 72 h in peak and non-peak regions respectively. \n",
      "Figure 1 [rid=F1] A and Supplementary Figure S1 [this file] similarly show that across 152 ChIP-seq and DNase-seq experiments, the fraction of unique peaks discovered in each cell type used in Epitome decreases as more cell types are characterized for a given experiment. \n",
      "These results are shown in Supplementary Figure S3 [this file] , and show that many histone modifications, including H3K27ac, H3K27me3, and H3K4me1 and H3K4me2, are frequently collected, along with at least one other additional ChIP-seq target of interest. \n",
      "As shown in Figure 2 [rid=F2] and Supplementary Figure S6b [this file] , models that are trained individually on epigenetic events of interest often outperform models trained jointly. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 228916\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 57981 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ee3ce79985e558b52efed84be1497976a3a2c34c79a74da0810ae29583caaff0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 57981 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9a4caa28-e70e-4895-9c4a-19e728ddb16a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115046\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115046\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322508,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 65397,\n",
      "        \"completion_tokens\": 41,\n",
      "        \"total_tokens\": 65438,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE115046', 'repository_reference': 'geo', 'dataset_id': 'GSE115046'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE115046, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE115046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE115046', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE115046', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE115046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE115046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115046 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115046 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE115046 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE115046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 74th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10985232&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b64dfbc0>, <Element supplementary-material at 0x1b687f480>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b64dfbc0>, <Element supplementary-material at 0x1b687f480>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: Finally, instead of using a supporting read threshold to filter as most methods do, Anchored-fusion incorporates a model with a deep neural network called the HVLD model to filter out false positive fusions that may be caused by artificial factors during the experimental process ( Figures 1 [rid=fig1] B and S1 [this file] ). \n",
      "For example, the classical BCR-ABL1 fusion gene contains a 47 bp short homologous sequence with chromosome 5 near its fusion breakpoint, which causes some supporting evidences at that location to be incorrectly matched or filtered out due to multiple alignments ( Figures 1 [rid=fig1] C and S2 [this file] ). As a result, the supporting sequencing reads of the DUX4 fusion gene are matched to hundreds of loci, leading to few supporting evidences or even the erroneous removal of the fusion event ( Figures 1 [rid=fig1] D and S3 [this file] ). \n",
      "For all test cases, we took the real fusions downloaded from Pan-Cancer Analysis of Whole Genomes as positive samples ( Figure S4 [this file] ). \n",
      "The fusion breakpoints identified by each tool among the fusion partners are listed in Table S1 [this file] . \n",
      "The fusion gene with the breakpoint at 1,905,943 is derived from RRMM1 and RRMM2, and the one with the breakpoint at nucleotide 1,902,353 is derived from SMM0 ( Figures 6 [rid=fig6] B and S5 [this file] ). \n",
      "The specific process of obtaining training and testing data is shown in Figure S4 [this file] . \n",
      "Randomly selecting negative samples and using the entire set for training showed no significant differences in performance. ( Figure S6 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 176809\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 44954 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-04cae8078cace84ec0b3d48fe6d9106229f4d63dd87630f93b1e3369a853d254\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 44954 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-78c0deb3-85f9-4ecd-9353-60531794ad92\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP186687\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP291312\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP158590\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Synapse:syn10003873\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.5281/zenodo.10677267\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/ShenLab-Genomics/Anchored-Fusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiDsLab/scFusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/STAR-Fusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/FusionInspector/FusionInspector\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/ndaniel/fusioncatcher\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/suhrig/arriba\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"NM_001306068.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"NM_133330.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP186687\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP291312\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRA:SRP158590\\\",\\n    \\\"repository_reference\\\": \\\"sra\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Synapse:syn10003873\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.5281/zenodo.10677267\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/ShenLab-Genomics/Anchored-Fusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/XiDsLab/scFusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/STAR-Fusion\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/FusionInspector/FusionInspector\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/ndaniel/fusioncatcher\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/suhrig/arriba\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"NM_001306068.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"NM_133330.3\\\",\\n    \\\"repository_reference\\\": \\\"nuccore\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322511,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 50133,\n",
      "        \"completion_tokens\": 516,\n",
      "        \"total_tokens\": 50649,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRA:SRP186687', 'repository_reference': 'sra', 'dataset_id': 'SRA:SRP186687'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRA:SRP186687 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRA:SRP186687', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRA:SRP291312', 'repository_reference': 'sra', 'dataset_id': 'SRA:SRP291312'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRA:SRP291312 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRA:SRP291312', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRA:SRP158590', 'repository_reference': 'sra', 'dataset_id': 'SRA:SRP158590'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SRA:SRP158590 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'SRA:SRP158590', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Synapse:syn10003873', 'repository_reference': 'synapse.org', 'dataset_id': 'Synapse:syn10003873'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: Synapse:syn10003873, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: Synapse:syn10003873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'Synapse:syn10003873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.5281/zenodo.10677267', 'repository_reference': 'doi.org', 'dataset_id': 'https://doi.org/10.5281/zenodo.10677267'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/, None, https://doi.org/10.5281/zenodo.10677267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/ShenLab-Genomics/Anchored-Fusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/ShenLab-Genomics/Anchored-Fusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/ShenLab-Genomics/Anchored-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/ShenLab-Genomics/Anchored-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/ShenLab-Genomics/Anchored-Fusion, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/ShenLab-Genomics/Anchored-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/ShenLab-Genomics/Anchored-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/ShenLab-Genomics/Anchored-Fusion is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/ShenLab-Genomics/Anchored-Fusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/ShenLab-Genomics/Anchored-Fusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/XiDsLab/scFusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiDsLab/scFusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/XiDsLab/scFusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/XiDsLab/scFusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/XiDsLab/scFusion, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/XiDsLab/scFusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/XiDsLab/scFusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/XiDsLab/scFusion is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/XiDsLab/scFusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/XiDsLab/scFusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/STAR-Fusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/STAR-Fusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/STAR-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/STAR-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/STAR-Fusion, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/STAR-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/STAR-Fusion\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/STAR-Fusion is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/STAR-Fusion', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/STAR-Fusion'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/FusionInspector/FusionInspector', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/FusionInspector/FusionInspector'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/FusionInspector/FusionInspector\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/FusionInspector/FusionInspector\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/FusionInspector/FusionInspector, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/FusionInspector/FusionInspector\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/FusionInspector/FusionInspector\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/FusionInspector/FusionInspector is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/FusionInspector/FusionInspector', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/FusionInspector/FusionInspector'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/ndaniel/fusioncatcher', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/ndaniel/fusioncatcher'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/ndaniel/fusioncatcher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/ndaniel/fusioncatcher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/ndaniel/fusioncatcher, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/ndaniel/fusioncatcher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/ndaniel/fusioncatcher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/ndaniel/fusioncatcher is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/ndaniel/fusioncatcher', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/ndaniel/fusioncatcher'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/suhrig/arriba', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/suhrig/arriba'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/suhrig/arriba\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/suhrig/arriba\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/suhrig/arriba, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/suhrig/arriba\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/suhrig/arriba\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/suhrig/arriba is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://github.com/suhrig/arriba', 'repository_reference': 'n/a', 'dataset_id': 'https://github.com/suhrig/arriba'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NM_001306068.3', 'repository_reference': 'nuccore', 'dataset_id': 'NM_001306068.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NM_001306068.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'NM_001306068.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NM_133330.3', 'repository_reference': 'nuccore', 'dataset_id': 'NM_133330.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID NM_133330.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'NM_133330.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'SRA:SRP186687', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRA:SRP291312', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRA:SRP158590', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'Synapse:syn10003873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'NM_001306068.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'NM_133330.3', 'data_repository': 'nuccore', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 7 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP186687 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRA:SRP186687 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: sra and accession_id: SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRA:SRP186687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP291312 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRA:SRP291312 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: sra and accession_id: SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRA:SRP291312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: sra, identifier: SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA:SRP158590 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for SRA:SRP158590 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: sra and accession_id: SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRA:SRP158590\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: Synapse:syn10003873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: Synapse:syn10003873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Synapse:syn10003873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Synapse:syn10003873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Synapse:syn10003873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Synapse:syn10003873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for Synapse:syn10003873 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1076 - WARNING - Identifier Synapse:syn10003873 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: Synapse:syn10003873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:Synapse:syn10003873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: doi.org and accession_id: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_001306068.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for NM_001306068.3 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: nuccore and accession_id: NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/NM_001306068.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: nuccore, identifier: NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NM_133330.3 for nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for NM_133330.3 in nuccore\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: nuccore and accession_id: NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/nuccore/NM_133330.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 7\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 75th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8784774&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7352a00>, <Element supplementary-material at 0x1b7352b40>, <Element supplementary-material at 0x1b7351e00>, <Element supplementary-material at 0x1b7353580>, <Element supplementary-material at 0x1b7352500>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7352a00>, <Element supplementary-material at 0x1b7352b40>, <Element supplementary-material at 0x1b7351e00>, <Element supplementary-material at 0x1b7353580>, <Element supplementary-material at 0x1b7352500>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: By extension, compounds targeting the same proteins should induce similar transcriptional signatures, which in turn can shed insight into its MoA ( Figure S1 [this file] ). \n",
      "The most comprehensive such evaluation to date, the Kinome-Binding Resource (KBR), measured the affinity of 230 clinically relevant kinase inhibitors across 255 kinases. [xref] [rid=bib18] While restricted to this protein class, this dataset is well-suited to benchmarking methods aimed at predicting drug polypharmacology by providing criteria for the evaluation of systems pharmacology approaches ( Figure S2 [this file] ). \n",
      "Interestingly, SC1 and SC2 scores revealed distinct distribution profiles: on average, most predictions were statistically significantly enriched on the top 10 target metric (SC1) but not on the entire list enrichment (SC2) ( Figures S3 [this file] A and S3B) \n",
      "Figures S3 [this file] C and 3D summarize the results of this analysis, with each box showing a team’s bootstrapped scores, and the color of the box indicating the Bayes factor relative to the top performer. Using this criteria, Team Atom and Team Netphar were confirmed as the top performers in SC1 and SC2, respectively ( Figures S3 [this file] C and S3D), while team SBNB was a close second in SC2 (Bayes factor 3–5). \n",
      "This analysis ( Figures S3 [this file] E and S3F) changed the ranking of some teams in SC1, most notably Team Theragen, whose original second place fell to ninth place when non-KBR targets were not pre-filtered. \n",
      "To better understand the models and the difference in their performances, we examined sub-challenge scores on an individual drug basis ( Figures S3 [this file] G and S3H). \n",
      "To address this question, we evaluated the ratio between the scores of the top three winning teams when either DrugBank or KBR targets were used as bona fide high-affinity targets of the 32 drugs used in the challenge ( Figure S4 [this file] ). While the scores based on DrugBank targets were consistently higher (Netphar, 3:2; SBNB, 4:2; and Atom, 1.7:1.4), they all showed positive enrichment within the prediction vector ( Figure S4 [this file] ). \n",
      "We compared the ranked performance of each prediction using these various databases as gold standards (KBR/Kinome, ChEMBL, DrugTargetCommons, KinomeScan, and DrugBank) to evaluate the stability of the predictions with different ground truths ( Figure S5 [this file] ). \n",
      "To evaluate the general relationships between kinase targets and mRNA programs, we assessed the correlation of the KBR-reported with transcriptional hallmarks (as detailed in Supplemental information [this file] ) across 84 drugs present in both databases ( Figure 4 [rid=fig4] D). \n",
      "Drug sensitivity training data was obtained from DrugComb, a curated database that includes batch-corrected drug sensitivities for both single drugs and drug combinations. [xref] [rid=bib27] In addition to the commonly used IC (half maximum inhibitory concenration), DrugComb provides an AUC-based relative inhibition (RI) metric, [xref] [rid=bib35] which captures both the potency and efficacy of drug responses ( Figure S6 [this file] A). \n",
      "Examining correlations between predicted and gold-standard targets, we found that adding drug sensitivity data significantly improved prediction accuracy relative to transcriptional data alone ( Figure S6 [this file] B). Performance improvements were driven by several individual drugs whose targets were poorly predicted based on perturbational profile data only, including sunitinib, crizotinib, and crenolanib ( Figure S6 [this file] C). \n",
      "This addresses two major reasons for clinical trial failures: lack of safety and efficacy. [xref] [rid=bib37] [xref] [rid=bib38] Failure rates may be substantially reduced if compound MoAs could be assessed more accurately and comprehensively ( Figures S1 [this file] A and S1B). \n",
      "Most of the experimental approaches rely on direct binding assays, such as ATP competitive pull-down, [xref] [rid=bib18] affinity purification [xref] [rid=bib39] [xref] [rid=bib40] or affinity chromatography assays. [xref] [rid=bib41] These labor-intensive methods are generally limited to the identification of high-affinity binding targets rather than the full protein repertoire responsible for compound activity in a tissue and are often restricted to a specific protein family, such as protein kinases ( Figure S1 [this file] A). Thus, critically relevant targets outside of these relatively narrow confines may be missed, as shown by the recent reclassification of the MET tyrosine receptor kinase inhibitor tivantinib as a microtubule inhibitor. [xref] [rid=bib42] Indeed, drug polypharmacology is emerging as a critical concept that increasingly impacts the mechanistic understanding of a drug’s disease-specific impact, for instance via a field effect mediated by multiple targets rather than by their primary, high-affinity binding target ( Figure S1 [this file] B). \n",
      "More recently, systematic gene expression profiling (GEP) following compound perturbations in cell lines [xref] [rid=bib8] [xref] [rid=bib11] [xref] [rid=bib23] [xref] [rid=bib50] has furthered the development of computational methods for MoA analysis ( Figure S1 [this file] B). \n",
      "The objective benchmark used in this challenge is the Kinome-Binding Resource (KBR), a systematic set of ATP-competitive binding assays assessing the ability of 230 candidate kinase inhibitor molecules to bind to one of 255 protein kinases ( Figure S2 [this file] ). \n",
      "A critical issue emerging from the evaluation of individual prediction performance and individual databases is that the concept of is still poorly defined and inconsistent ( Figure S2 [this file] ). \n",
      "Indeed, the fact that the perturbational signature significantly contributed to correct target inference suggests that downstream transcriptional regulators represent a valuable reporter assay that can distinguish the MoA of different compounds ( Figures 4 [rid=fig4] and S6 [this file] ). \n",
      "While we did not observe major differences between model performances based on modeling strategy, generally, similarity-weighted average methodologies performed best in SC2 ( Figures S7 [this file] and S8 [this file] ), while neural-network-based methodologies performed best in SC1 ( Figure S9 [this file] ). \n",
      "In the post-challenge phase of this study, we re-evaluated the performance of each team ( Figure S3 [this file] ) by repeating this analysis but omitting the kinase filtering step described above. \n",
      "Detailed computational procedure Figure S4 [this file] \n",
      "Detailed algorithm for winning team “SBNB” ( Figure S8 [this file] ): \n",
      "Detailed algorithm for winning team “ATOM” ( Figure S9 [this file] ): \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc2: In the CTD2 Pancancer Drug Activity DREAM Challenge, participants were asked to use DREAM-provided and publicly available pharmacogenomic datasets—including cell-line-matched DRCs and gene expression profiles of drug-naive and -perturbed cells (perturbational profiles)—to predict compound binding proteins (high-affinity targets) of 32 anonymized drugs ( Figure 1 [rid=fig1] A, Table S1 [this file] ). \n",
      "In order to optimize the clinical translation potential of the perturbation databases, we used the CMax, defined as the maximum plasma concentration after the administration of the drug at the maximum tolerated dose in patients, (whenever available from published pharmacokinetic studies), as an upper bound for the perturbation studies ( Table S1 [this file] ). \n",
      "Sidebar annotations of canonical drug-targets were defined based on the DrugBank definitions of drug-targets as detailed in Table S1 [this file] . \n",
      "To better understand the type of inhibitors that models performed the best on we calculated the enrichment of each drugbank target kinases (as defined in Table S1 [this file] ) over the ranked 32-drug vector in Figures 4 [rid=fig4] A and 4B using the aREA algorithm in the viper package in R. [xref] [rid=bib12]  To visualize the kinases sampled by the Klaeger et. al [xref] [rid=bib18] definitions of drug-targets we color-coded individual kinase-nodes within the Human Kinome phylogenetic tree obtained from the CORAL tool. [xref] [rid=bib64] Kinases measured in the Kinome-dataset were color coded based the Kinase group that they were a member of as defined in Manning et al. [xref] [rid=bib31]  To better understand the relationships between individual kinases and down-stream transcriptional programs we calculated the correlation matrix between the Kinome Kd’s (250 kinases x 84 drugs) and the pan-cancer transcriptional signature PANACEA-data (50 signatures x 84 drugs) across 84 overlapping drugs that occurred in both datasets. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc3: The DREAM-provided dataset comprised 704 DRCs and matched perturbational profiles of these 32 drugs ( Table S2 [this file] ) in 11 cell lines representing molecularly distinct tumor subtypes in replicate ( Figure 1 [rid=fig1] E). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc4: Douglass Jr., Robert Allaway, Bence Szalai, Ron Realubit, Charles Karan, Wenyu Wang, Tingzhong Tian, Adrià Fernández-Torras, Jing Tang, Shuyu Zheng, Alberto Pessia, Ziaurrehman Tanoli, Mohieddin Jafari, Fangping Wan, Shuya Li, Yuanpeng Xiong, Jianyang Zeng, Miquel Duran-Frigola, Martino Bertoni, Pau Badia-i-Mompel, Lídia Mateo, Oriol Guitart-Pla, Patrick Aloy, Verena Chung, Julio Saez-Rodriguez, Justin Guinney, Daniela Gerhard, and Andrea Califano (see Data S1 [this file] for consortium author affiliations). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 257646\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 65164 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-610c498595ffacd3ca8c1c99189ae73c49240b09ab3e9a752138231069fe4659\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 65164 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b550f337-af13-4c71-8cbf-116cd821b298\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE186341\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20968331\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE186341\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20968331\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322513,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 74560,\n",
      "        \"completion_tokens\": 78,\n",
      "        \"total_tokens\": 74638,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE186341', 'repository_reference': 'geo', 'dataset_id': 'GSE186341'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE186341, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE186341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE186341', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn20968331', 'repository_reference': 'synapse.org', 'dataset_id': 'syn20968331'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn20968331, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn20968331\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn20968331', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE186341', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20968331', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE186341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE186341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE186341 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE186341 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE186341 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE186341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn20968331\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20968331\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20968331 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20968331 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20968331 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20968331 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn20968331 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn20968331\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20968331\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 76th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8082263&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b17e2800>, <Element supplementary-material at 0x1b1222340>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b17e2800>, <Element supplementary-material at 0x1b1222340>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: The PARADIGM algorithm integrated platform-corrected expression, gene-level copy number, and pathway interaction data and was used to infer activities of ~19K pathway features ( Data S1 [this file] ) for 9829 TCGA Pan-Cancer samples across 33 cancer types. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc2: Download and read in the pathway interaction dictionary containing ~45K interactions linking pathway features within a superimposed pathway structure ( key resources table [rid=sec9] , Data S2 [this file] ) \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 210622\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 53408 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f36c1d17933f85bef772ac05738f2553383e0e7aebaca07d1217c227b0a37b10\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 53408 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2eff5f69-25da-479b-b8fd-84a011ef3706\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"GDC publication page\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/node/1248\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Genomic Data Commons\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://firebrowse.org/api-docs/#!/Analyses\\\",\\n    \\\"repository_reference\\\": \\\"firebrowse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn11801497\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/legacy-archive/\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://firebrowse.org/api-docs/#!/Analyses\\\",\\n    \\\"repository_reference\\\": \\\"firebrowse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/legacy-archive/\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://zwdzwd.github.io/InfiniumAnnotation\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://zwdzwd.github.io/pmd\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://uswest.ensembl.org/info/data/ftp/index.html\\\",\\n    \\\"repository_reference\\\": \\\"ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://m.ensembl.org/info/website/archives/index.html\\\",\\n    \\\"repository_reference\\\": \\\"ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\\\",\\n    \\\"repository_reference\\\": \\\"ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.mirbase.org/ftp.shtml\\\",\\n    \\\"repository_reference\\\": \\\"mirbase.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://sbenz.github.io/Paradigm/\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.targetscan.org/faqs.Release_7.html\\\",\\n    \\\"repository_reference\\\": \\\"targetscan.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://bioconductor.org/packages/release/bioc/html/sesame.html\\\",\\n    \\\"repository_reference\\\": \\\"bioconductor.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"GDC publication page\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/node/1248\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Table S1\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Genomic Data Commons\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://firebrowse.org/api-docs/#!/Analyses\\\",\\n    \\\"repository_reference\\\": \\\"firebrowse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.synapse.org/#!Synapse:syn11801497\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/legacy-archive/\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://firebrowse.org/api-docs/#!/Analyses\\\",\\n    \\\"repository_reference\\\": \\\"firebrowse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://portal.gdc.cancer.gov/legacy-archive/\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://zwdzwd.github.io/InfiniumAnnotation\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://zwdzwd.github.io/pmd\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/pancanatlas\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://uswest.ensembl.org/info/data/ftp/index.html\\\",\\n    \\\"repository_reference\\\": \\\"ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://m.ensembl.org/info/website/archives/index.html\\\",\\n    \\\"repository_reference\\\": \\\"ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\\\",\\n    \\\"repository_reference\\\": \\\"ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.mirbase.org/ftp.shtml\\\",\\n    \\\"repository_reference\\\": \\\"mirbase.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://sbenz.github.io/Paradigm/\\\",\\n    \\\"repository_reference\\\": \\\"github.io\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://www.targetscan.org/faqs.Release_7.html\\\",\\n    \\\"repository_reference\\\": \\\"targetscan.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://bioconductor.org/packages/release/bioc/html/sesame.html\\\",\\n    \\\"repository_reference\\\": \\\"bioconductor.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322520,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 62039,\n",
      "        \"completion_tokens\": 1642,\n",
      "        \"total_tokens\": 63681,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Table S1', 'repository_reference': 'GDC publication page', 'dataset_id': 'Table S1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Table S1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Table S1 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: GDC publication page, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: GDC publication page\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'GDC publication page', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://gdc.cancer.gov/node/1248', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/node/1248'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://gdc.cancer.gov/node/1248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://gdc.cancer.gov/node/1248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://gdc.cancer.gov/node/1248, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://gdc.cancer.gov/node/1248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://gdc.cancer.gov/node/1248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://gdc.cancer.gov/node/1248 could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://gdc.cancer.gov/node/1248 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://gdc.cancer.gov/node/1248', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/node/1248'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ars.els-cdn.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ars.els-cdn.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S0092867417310565-mmc1.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/pancanatlas', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/pancanatlas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://gdc.cancer.gov/about-data/publications/pancanatlas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/pancanatlas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://gdc.cancer.gov/about-data/publications/pancanatlas, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/pancanatlas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://gdc.cancer.gov/about-data/publications/pancanatlas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://gdc.cancer.gov/about-data/publications/pancanatlas could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://gdc.cancer.gov/about-data/publications/pancanatlas is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/pancanatlas', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/pancanatlas'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Genomic Data Commons', 'repository_reference': 'n/a', 'dataset_id': 'Genomic Data Commons'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Genomic Data Commons\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 999 - INFO - Accession ID Genomic Data Commons is a known repository --> invalid, returning 'n/a'\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Genomic Data Commons', 'repository_reference': 'n/a', 'dataset_id': 'Genomic Data Commons'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://firebrowse.org/api-docs/#!/Analyses', 'repository_reference': 'firebrowse.org', 'dataset_id': 'http://firebrowse.org/api-docs/#!/Analyses'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://firebrowse.org/api-docs/#!/Analyses\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://firebrowse.org/api-docs/#!/Analyses\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://firebrowse.org/api-docs/#!/Analyses, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://firebrowse.org/api-docs/#!/Analyses\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://firebrowse.org/api-docs/#!/Analyses\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: firebrowse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://firebrowse.org/api-docs/#!/Analyses is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: firebrowse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: firebrowse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL firebrowse.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'firebrowse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.synapse.org/#!Synapse:syn11801497', 'repository_reference': 'synapse.org', 'dataset_id': 'https://www.synapse.org/#!Synapse:syn11801497'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn11801497, None, https://www.synapse.org/#!Synapse:syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn11801497', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://portal.gdc.cancer.gov/legacy-archive/', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://portal.gdc.cancer.gov/legacy-archive/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://portal.gdc.cancer.gov/legacy-archive/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://portal.gdc.cancer.gov/legacy-archive/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/legacy-archive/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/legacy-archive/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/legacy-archive/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://portal.gdc.cancer.gov/legacy-archive/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://portal.gdc.cancer.gov/legacy-archive/', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://portal.gdc.cancer.gov/legacy-archive/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ars.els-cdn.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610818301119-mmc2.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://zwdzwd.github.io/InfiniumAnnotation', 'repository_reference': 'github.io', 'dataset_id': 'http://zwdzwd.github.io/InfiniumAnnotation'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://zwdzwd.github.io/InfiniumAnnotation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://zwdzwd.github.io/InfiniumAnnotation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://zwdzwd.github.io/InfiniumAnnotation, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://zwdzwd.github.io/InfiniumAnnotation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://zwdzwd.github.io/InfiniumAnnotation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: zwdzwd.github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://zwdzwd.github.io/InfiniumAnnotation is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.io, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.io may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://zwdzwd.github.io/pmd', 'repository_reference': 'github.io', 'dataset_id': 'http://zwdzwd.github.io/pmd'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://zwdzwd.github.io/pmd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://zwdzwd.github.io/pmd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://zwdzwd.github.io/pmd, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://zwdzwd.github.io/pmd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://zwdzwd.github.io/pmd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: zwdzwd.github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://zwdzwd.github.io/pmd is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.io, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.io may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://uswest.ensembl.org/info/data/ftp/index.html', 'repository_reference': 'ensembl.org', 'dataset_id': 'https://uswest.ensembl.org/info/data/ftp/index.html'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://uswest.ensembl.org/info/data/ftp/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://uswest.ensembl.org/info/data/ftp/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://uswest.ensembl.org/info/data/ftp/index.html, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://uswest.ensembl.org/info/data/ftp/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://uswest.ensembl.org/info/data/ftp/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: uswest.ensembl.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://uswest.ensembl.org/info/data/ftp/index.html is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ensembl.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ensembl.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL ensembl.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'ensembl.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://m.ensembl.org/info/website/archives/index.html', 'repository_reference': 'ensembl.org', 'dataset_id': 'https://m.ensembl.org/info/website/archives/index.html'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://m.ensembl.org/info/website/archives/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://m.ensembl.org/info/website/archives/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://m.ensembl.org/info/website/archives/index.html, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://m.ensembl.org/info/website/archives/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://m.ensembl.org/info/website/archives/index.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: m.ensembl.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://m.ensembl.org/info/website/archives/index.html is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ensembl.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ensembl.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL ensembl.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'ensembl.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz', 'repository_reference': 'ucsc.edu', 'dataset_id': 'http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: hgdownload.soe.ucsc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ucsc.edu, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ucsc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL ucsc.edu may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'ucsc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020 could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.mirbase.org/ftp.shtml', 'repository_reference': 'mirbase.org', 'dataset_id': 'http://www.mirbase.org/ftp.shtml'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.mirbase.org/ftp.shtml\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.mirbase.org/ftp.shtml\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.mirbase.org/ftp.shtml, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.mirbase.org/ftp.shtml\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.mirbase.org/ftp.shtml\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.mirbase.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.mirbase.org/ftp.shtml is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: mirbase.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: mirbase.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL mirbase.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'mirbase.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://sbenz.github.io/Paradigm/', 'repository_reference': 'github.io', 'dataset_id': 'http://sbenz.github.io/Paradigm/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://sbenz.github.io/Paradigm/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://sbenz.github.io/Paradigm/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://sbenz.github.io/Paradigm/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://sbenz.github.io/Paradigm/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://sbenz.github.io/Paradigm/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: sbenz.github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://sbenz.github.io/Paradigm/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.io, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.io\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.io may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://www.targetscan.org/faqs.Release_7.html', 'repository_reference': 'targetscan.org', 'dataset_id': 'http://www.targetscan.org/faqs.Release_7.html'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://www.targetscan.org/faqs.Release_7.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://www.targetscan.org/faqs.Release_7.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://www.targetscan.org/faqs.Release_7.html, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://www.targetscan.org/faqs.Release_7.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://www.targetscan.org/faqs.Release_7.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: www.targetscan.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://www.targetscan.org/faqs.Release_7.html is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: targetscan.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: targetscan.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL targetscan.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'targetscan.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://bioconductor.org/packages/release/bioc/html/sesame.html', 'repository_reference': 'bioconductor.org', 'dataset_id': 'https://bioconductor.org/packages/release/bioc/html/sesame.html'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://bioconductor.org/packages/release/bioc/html/sesame.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://bioconductor.org/packages/release/bioc/html/sesame.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://bioconductor.org/packages/release/bioc/html/sesame.html, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://bioconductor.org/packages/release/bioc/html/sesame.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://bioconductor.org/packages/release/bioc/html/sesame.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: bioconductor.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://bioconductor.org/packages/release/bioc/html/sesame.html is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: bioconductor.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: bioconductor.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL bioconductor.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'bioconductor.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'GDC publication page', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'firebrowse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn11801497', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'ensembl.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'ensembl.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'ucsc.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'mirbase.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.io', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'targetscan.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'bioconductor.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 12 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11801497 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn11801497 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11801497\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 7: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 11: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 12: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 12\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 77th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11659981&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 12 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b735b140>, <Element supplementary-material at 0x1b1aa38c0>, <Element supplementary-material at 0x1b0e5e200>, <Element supplementary-material at 0x1b0e16f00>, <Element supplementary-material at 0x1b0e17c00>, <Element supplementary-material at 0x1b1b44380>, <Element supplementary-material at 0x1b7bf0980>, <Element supplementary-material at 0x1b7bf0c40>, <Element supplementary-material at 0x1b7128280>, <Element supplementary-material at 0x1b7583480>, <Element supplementary-material at 0x1b0c59cc0>, <Element supplementary-material at 0x1b114b280>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b735b140>, <Element supplementary-material at 0x1b1aa38c0>, <Element supplementary-material at 0x1b0e5e200>, <Element supplementary-material at 0x1b0e16f00>, <Element supplementary-material at 0x1b0e17c00>, <Element supplementary-material at 0x1b1b44380>, <Element supplementary-material at 0x1b7bf0980>, <Element supplementary-material at 0x1b7bf0c40>, <Element supplementary-material at 0x1b7128280>, <Element supplementary-material at 0x1b7583480>, <Element supplementary-material at 0x1b0c59cc0>, <Element supplementary-material at 0x1b114b280>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup6: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup7: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup8: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup9: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup10: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup11: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref sup12: We collected the scRNA-seq data from 6 healthy human tissues, which are brain [ 26 [rid=bib26] ], heart [ 27 [rid=bib27] ], kidney [ 28 [rid=bib28] ], liver [ 29 [rid=bib29] ], lung [ 30 [rid=bib30] ], and peripheral blood mononuclear cells (PBMCs) [ 31 [rid=bib31] ] ( Supplementary Table S1 [this file] ). \n",
      "On average, each tissue contained 66,395 cells (ranging from 2,638 cells in PBMCs to 287,269 in the heart) and 24,064 genes (ranging from 13,714 genes in PBMCs to 33,694 in the heart and kidney) ( Supplementary Table S1 [this file] ). \n",
      "S1A [this file] ). S1B [this file] ). \n",
      "S2A– [this file] C [this file] ). \n",
      "S2C [this file] , D [this file] ). \n",
      "S3A [this file] , B [this file] and S4A [this file] , B [this file] ). S3D [this file] , E [this file] and S4D [this file] , E [this file] ), indicating that the primary community of scGraph2Vec played important roles in generating gene clusters with similar behaviors in the network. \n",
      "S3C [this file] and S4C [this file] ). S5 [this file] ). S6 [this file] ). S7 [this file] ). \n",
      "S8 [this file] ). \n",
      "Notably, ARGA, ARGVA, SIG-VAE, and DGVAE were not designed specifically for biological data, and other methods were designed based on the gene expression matrix only ( Supplementary Table S2 [this file] ). \n",
      "S9 [this file] ). S10 [this file] ). \n",
      "S11 [this file] and Supplementary Table S3 [this file] ). S12 [this file] ). Each cluster contained tens to hundreds of genes, for example, 68 to 270 genes per cluster for the brain ( Supplementary Table S3 [this file] ). \n",
      "S13 [this file] ). \n",
      "S14 [this file] ). \n",
      "Taking the brain tissue as an example, we found a total of 37 clusters significantly enriched with at least 1 cell type ( < 0.05, Fig. 4A [rid=fig4] and Supplementary Table S4 [this file] ). S15 [this file] ). \n",
      "S16 [this file] ). S17 [this file] ). S18 [this file] ). S19 [this file] ). S20 [this file] ). \n",
      "S21 [this file] ). \n",
      "S22A [this file] , B [this file] and Supplementary Table S3 [this file] ). S22C [this file] , D [this file] ). \n",
      "S23 [this file] ). \n",
      "Based on the gene embeddings, we identified 149 clusters for the melanocytic state (62 significantly specific to melanocytic), 149 for the intermediate state (59 specific), and 133 for the mesenchymal state (47 specific) ( Supplementary Table S3 [this file] ). \n",
      "STRING analysis revealed close protein–protein interactions among these genes ( Supplementary Fig. 24 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 218630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 55410 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-5a662d0d30a417fd76ce500c6486dea99c14b1ffa0e0e52512005a042da1844a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 55410 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4a428763-433a-4da7-950e-db661d30a6b4\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE162170\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-6149\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-6653\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE134432\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SCP498\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE162170\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-6149\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-6653\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE134432\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SCP498\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322523,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 68844,\n",
      "        \"completion_tokens\": 212,\n",
      "        \"total_tokens\": 69056,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE162170', 'repository_reference': 'geo', 'dataset_id': 'GSE162170'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE162170, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE162170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE162170', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-6149', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-6149'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-6149 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-6653', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-6653'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-6653 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-6653', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE134432', 'repository_reference': 'geo', 'dataset_id': 'GSE134432'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE134432, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE134432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE134432', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'repository_reference': 'synapse.org', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn21041850, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SCP498', 'repository_reference': 'n/a', 'dataset_id': 'SCP498'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: SCP498\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID SCP498 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: SCP498\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SCP498 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SCP498 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SCP498 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SCP498 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SCP498 for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'SCP498', 'repository_reference': 'n/a', 'dataset_id': 'SCP498'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE162170', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-6653', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE134432', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE162170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE162170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE162170 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE162170 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE162170 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE162170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-6149 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6653 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-6653 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-6653\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE134432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE134432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134432 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134432 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE134432 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE134432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 78th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9356164&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b65b1fd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 51 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1.5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1.5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec1.6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 951) for sec1.6: To skip this step, download the controlled access “QC Unimputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the QC protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Stranded Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ), as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with the Stranding protocol steps provided below.\n",
      "To skip this step, download the controlled access “HRC Imputed Genotyping Data” generated from ( Sayaman et al., 2021 [href=#bib20] ) as described in step 11 of the “ prepare germline genetic variation dataset [href=this file] ” section of this protocol OR proceed with phasing and imputation protocol steps provided below.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.1: For reference, the “ quality control analysis of germline data [href=this file] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=this file] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.2: For reference, the “ quality control analysis of germline data [href=#sec3.1] ”, “ stranding [href=this file] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.2: For reference, the “ quality control analysis of germline data [href=#sec3.1] ”, “ stranding [href=this file] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.2: For reference, the “ quality control analysis of germline data [href=#sec3.1] ”, “ stranding [href=this file] ”, and “ genotype imputation [href=#sec3.3] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=this file] (steps 12 and 13); genotype imputation [href=#sec3.3] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.3: For reference, the “ quality control analysis of germline data [href=#sec3.1] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=this file] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=this file] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=this file] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#sec3.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1015) for sec3.3: For reference, the “ quality control analysis of germline data [href=#sec3.1] ”, “ stranding [href=#sec3.2] ”, and “ genotype imputation [href=this file] ” workflows were performed in the University of California, San Francisco (UCSF) high-performance compute environment which had 8 communal compute nodes and 1 dedicated node, each with 12–64 cores (each node had from 64 to 512 GB of RAM and at least 1.8 TB of fast local disk space).\n",
      "Issues or errors running commands on the high- performance compute server or implementing available code from GitHub. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=this file] (steps 14–17)).\n",
      "Inadequate computational power to run workflows. ( quality control analysis of germline data [href=#sec3.1] (steps 1–9); stranding [href=#sec3.2] (steps 12 and 13); genotype imputation [href=this file] (steps 14–17); determination of ancestry-associated SNPs [href=#sec3.4] (steps 18–21)).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec6.12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec6.12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec7.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec7.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 155752\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 39690 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-dfe3290b8d29c8dcc7d3830e600fd0a3f2cbaca5b6419e3fddbb0e8c115309e1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 39690 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-de66229f-7c36-4a1e-8b18-fde700c99ea1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"pdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169709\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169715\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169709\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://gong_lab.hzau.edu.cn/PancanQTL/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.cell.com/cms/10.1016/j.immuni.2021.01.011/attachment/edb228c3-a345-4292-9f60-8b18f2852bbe/mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"pdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169709\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169715\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn12169709\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"http://gong_lab.hzau.edu.cn/PancanQTL/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://www.cell.com/cms/10.1016/j.immuni.2021.01.011/attachment/edb228c3-a345-4292-9f60-8b18f2852bbe/mmc2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322545,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 52518,\n",
      "        \"completion_tokens\": 694,\n",
      "        \"total_tokens\": 53212,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'TCGA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TCGA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn12169709', 'repository_reference': 'synapse.org', 'dataset_id': 'syn12169709'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn12169709, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn12169709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn12169709', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn12169715', 'repository_reference': 'synapse.org', 'dataset_id': 'syn12169715'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn12169715, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn12169715\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn12169715', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: api.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'http://api.gdc.cancer.gov/data/3586c0da-64d0-4b74-a449-5ff4d9136611'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'http://gong_lab.hzau.edu.cn/PancanQTL/', 'repository_reference': 'n/a', 'dataset_id': 'http://gong_lab.hzau.edu.cn/PancanQTL/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: http://gong_lab.hzau.edu.cn/PancanQTL/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: http://gong_lab.hzau.edu.cn/PancanQTL/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: http://gong_lab.hzau.edu.cn/PancanQTL/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: http://gong_lab.hzau.edu.cn/PancanQTL/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: http://gong_lab.hzau.edu.cn/PancanQTL/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: gong_lab.hzau.edu.cn\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID http://gong_lab.hzau.edu.cn/PancanQTL/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'http://gong_lab.hzau.edu.cn/PancanQTL/', 'repository_reference': 'n/a', 'dataset_id': 'http://gong_lab.hzau.edu.cn/PancanQTL/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ars.els-cdn.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc2.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ars.els-cdn.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://ars.els-cdn.com/content/image/1-s2.0-S1535610820302117-mmc5.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://www.cell.com/cms/10.1016/j.immuni.2021.01.011/attachment/edb228c3-a345-4292-9f60-8b18f2852bbe/mmc2.xlsx', 'repository_reference': 'n/a', 'dataset_id': 'https://www.cell.com/cms/10.1016/j.immuni.2021.01.011/attachment/edb228c3-a345-4292-9f60-8b18f2852bbe/mmc2.xlsx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1016/, None, https://www.cell.com/cms/10.1016/j.immuni.2021.01.011/attachment/edb228c3-a345-4292-9f60-8b18f2852bbe/mmc2.xlsx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1016/ for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1016/ for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for n/a in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1016/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 896 - INFO - Link https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020 could point to dataset webpage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 909 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=__ID__ with __ID__ replaced\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'TCGA', 'data_repository': 'gdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn12169709', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn12169715', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1016/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TCGA in gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: gdc.cancer.gov and accession_id: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: ...\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn12169709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn12169709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn12169709 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn12169709 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn12169709 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn12169709 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn12169709 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn12169709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn12169709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn12169715\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn12169715\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn12169715 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn12169715 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn12169715 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn12169715 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn12169715 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn12169715\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn12169715\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1016/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1016/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: doi.org and accession_id: 10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1016/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 55, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 5\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 79th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11271596&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0e79550>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#Sec7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for Sec7: Data Records [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#notes2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for notes2: Code availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 154836\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 39461 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-459750fbddf80cb362b98ec503d6a372a9ed6d5ead4054be423da9afaa0e4b1f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 39461 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a183b094-4bd9-433e-94c7-78f0bc5b8313\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn53009474\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"VOC-ALS\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/giovannasannino/VOC-ALS.git\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn53009474\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/[#!]*Synapse:syn28580053\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"VOC-ALS\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/giovannasannino/VOC-ALS.git\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322564,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 51547,\n",
      "        \"completion_tokens\": 116,\n",
      "        \"total_tokens\": 51663,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn53009474', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053', 'dataset_id': '10.7303/syn53009474'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn53009474, synapse.org, https://www.synapse.org/[#!]*Synapse:syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn53009474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn53009474', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'VOC-ALS', 'repository_reference': 'https://github.com/giovannasannino/VOC-ALS.git', 'dataset_id': 'VOC-ALS'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: VOC-ALS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID VOC-ALS is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/giovannasannino/VOC-ALS.git, identifier: VOC-ALS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier VOC-ALS for https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier VOC-ALS for https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier VOC-ALS for https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier VOC-ALS for https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier VOC-ALS for https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/giovannasannino/VOC-ALS.git\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'VOC-ALS', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.7303/syn53009474', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'VOC-ALS', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn53009474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn53009474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn53009474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn53009474 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn53009474 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.7303/syn53009474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn53009474\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: VOC-ALS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: VOC-ALS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier VOC-ALS for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier VOC-ALS for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier VOC-ALS for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier VOC-ALS for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier VOC-ALS for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for VOC-ALS in github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: github.com and accession_id: VOC-ALS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository github.com unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 80th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7520714&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b1341790>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 22 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 25) for sec4: Methods [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec6: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec6: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec16'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec16: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec16'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec16: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for ack1: Acknowledgements [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 118881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 30472 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b28654c7315a7f6a759eb2d822bfd9145335b639c94ccce89412158912b05c1c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 30472 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f528769c-aaeb-47fb-85c9-25d3286231ad\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17009517\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17009517\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas-surveillance.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.8159525.v1\\\",\\n    \\\"repository_reference\\\": \\\"https://dx.doi.org/10.6084/m9.figshare.8159525.v1\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn18344812\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17009517\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17009517\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas-surveillance.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.8159525.v1\\\",\\n    \\\"repository_reference\\\": \\\"https://dx.doi.org/10.6084/m9.figshare.8159525.v1\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn18344812\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322586,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 37283,\n",
      "        \"completion_tokens\": 222,\n",
      "        \"total_tokens\": 37505,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17009517', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17009517', 'dataset_id': 'syn17009517'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn17009517, synapse.org, https://www.synapse.org/#!Synapse:syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn17009517', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://atlas-surveillance.com/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://atlas-surveillance.com/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://atlas-surveillance.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://atlas-surveillance.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'atlas-surveillance.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.6084/m9.figshare.8159525.v1', 'repository_reference': 'https://dx.doi.org/10.6084/m9.figshare.8159525.v1', 'dataset_id': '10.6084/m9.figshare.8159525.v1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.6084/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.6084/m9.figshare.8159525.v1, doi.org, https://dx.doi.org/10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.6084/m9.figshare.8159525.v1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn18344812', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn18344812'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn18344812, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn18344812\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn18344812', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn17009517', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'atlas-surveillance.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.6084/m9.figshare.8159525.v1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn18344812', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17009517 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17009517 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17009517 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17009517 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn17009517 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17009517\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.6084/m9.figshare.8159525.v1 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.6084/m9.figshare.8159525.v1 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.6084/m9.figshare.8159525.v1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn18344812\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn18344812\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn18344812 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn18344812 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn18344812 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: doi.org and accession_id: 10.7303/syn18344812\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn18344812\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 5\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 81/100 (81.00%) | Elapsed: 00:13:35 | ETA: 00:03:11\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 81th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10769298&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0a93250>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 14 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S20: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 18 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 3806) for SD1: From the snRNAseq data, we identified major cell types across all three regions from all donors, including oligodendrocytes, oligodendrocyte precursor cells (OPCs), astrocytes, ependymal cells, microglia, macrophages, vascular cells, glutamatergic neurons and GABAergic neurons ( Fig 1E [href=#F1] , 1F [href=#F1] and Table S1 [href=this file] ). We observed oligodendrocyte and CD44-high astrocyte enrichment in the white matter adjacent to cortical gray matter ( Fig. 1I [href=#F1] ), and also confirmed that our cortical glutamatergic neuron subsets ( Table S2 [href=this file] ) localized to grey matter layers predicted by Azimuth human motor cortex reference ( Fig 1J [href=#F1] and 1K [href=#F1] ).\n",
      "In donors with primary or secondary demyelination, however, we found only the GABA_6_CCK_LAMP5 subtype lower in the DLPFC in donors with primary demyelination and secondary demyelination ( Fig. 2A [href=#F2] , 2B [href=#F2] Table S3 [href=this file] ).\n",
      "From a global perspective, we identified genes such as Myelin basic protein ( ) and Alpha-crystallin B ( ) differentially expressed in oligodendrocytes between control and demyelination groups ( Fig 2E [href=#F2] , Table S4 [href=this file] ). We then clustered oligodendrocytes into subgroups ( Fig 2F [href=#F2] , 2G [href=#F2] , Table S5 [href=this file] ) and examined compositional differences across brain regions and demyelination states ( Table S6 [href=this file] ). Genes distinguishing this Oligo_0_MOBP compared to other oligodendrocyte subgroups showed enrichment of expression programs related to cellular morphogenesis and axonogenesis ( Fig S1A [href=#SD2] , Table S7 [href=this file] ), suggesting a myelinating phenotype for this subgroup.\n",
      "Analysis of these two subgroups showed enrichment of marker genes associated with distress or dysregulation by means of active protein folding, response to temperature stimulus and response to unfolded protein ( Fig S1B [href=#SD2] , S1C [href=#SD2] , Table S8 [href=this file] and S9 [href=this file] ).\n",
      "We next performed the same analysis on oligodendrocyte precursor cells (OPCs), where we identified subgroups and genes differentially abundant between control and demyelination groups ( Fig 2M [href=#F2] , 2N [href=#F2] , 2O [href=#F2] , Table S10 [href=this file] , S11 [href=this file] ). In our MS donors, we observed higher proportions of subgroup OPC_2_TPST1_ARHGEF3 in NAWM; this subgroup showed enrichment of expression programs related to cell projection organization ( Fig 2P [href=#F2] , S1J [href=#SD2] , Table S12 [href=this file] ).\n",
      "Taking all microglia together, we identified genes - including and - differentially expressed between control and both demyelination groups ( Fig 3A [href=#F3] , Table S13 [href=this file] ), which also distinguish subgroups of microglia identified by clustering analysis ( Fig 3B [href=#F3] , 3C [href=#F3] , Table S14 [href=this file] ).\n",
      "GO analysis of genes distinguishing these two clusters pointed to processes such as immune cell differentiation, proliferation, migration and reactive inflammatory response pathways respectively ( Fig S2A [href=#SD3] , S2B [href=#SD3] and Table S15 [href=this file] , S16 [href=this file] ). This suggests the existence of a microglial signature - enriched for genes associated with inflammatory activation ( Fig S2C [href=#SD3] , Table S17 [href=this file] ) - that is present in non-lesional tissue specifically in MS-associated demyelination. By contrast, we observed a fourth microglial cluster, Micro_2_SOCS6_MYO1E, which showed enrichment only in the secondary demyelination group, again in all 3 brain regions ( Fig 3G [href=#F3] , Fig S2D [href=#SD3] , Table S18 [href=this file] ), suggesting that this signature may be a unique hallmark of demyelination accompanying diseases such as AD.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 8 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1161) for SD2: Genes distinguishing this Oligo_0_MOBP compared to other oligodendrocyte subgroups showed enrichment of expression programs related to cellular morphogenesis and axonogenesis ( Fig S1A [href=this file] , Table S7 [href=#SD1] ), suggesting a myelinating phenotype for this subgroup.\n",
      "Analysis of these two subgroups showed enrichment of marker genes associated with distress or dysregulation by means of active protein folding, response to temperature stimulus and response to unfolded protein ( Fig S1B [href=this file] , S1C [href=this file] , Table S8 [href=#SD1] and S9 [href=#SD1] ). Further examination of snRNAseq-derived abundances of Oligo_1_CRYAB and Oligo_6_CRYAB showed higher proportions in our participants with MS in all three regions, and we validated this trend in NAWM in our ST data. ( Fig S1D [href=this file] - S1F [href=this file] and Fig S1G [href=this file] - S1I [href=this file] ).\n",
      "In our MS donors, we observed higher proportions of subgroup OPC_2_TPST1_ARHGEF3 in NAWM; this subgroup showed enrichment of expression programs related to cell projection organization ( Fig 2P [href=#F2] , S1J [href=this file] , Table S12 [href=#SD1] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 4 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 898) for SD3: GO analysis of genes distinguishing these two clusters pointed to processes such as immune cell differentiation, proliferation, migration and reactive inflammatory response pathways respectively ( Fig S2A [href=this file] , S2B [href=this file] and Table S15 [href=#SD1] , S16 [href=#SD1] ). This suggests the existence of a microglial signature - enriched for genes associated with inflammatory activation ( Fig S2C [href=this file] , Table S17 [href=#SD1] ) - that is present in non-lesional tissue specifically in MS-associated demyelination. By contrast, we observed a fourth microglial cluster, Micro_2_SOCS6_MYO1E, which showed enrichment only in the secondary demyelination group, again in all 3 brain regions ( Fig 3G [href=#F3] , Fig S2D [href=this file] , Table S18 [href=#SD1] ), suggesting that this signature may be a unique hallmark of demyelination accompanying diseases such as AD.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 124002\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31753 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-db0679e0972add9a962bfc2a71cfebc60318faedeeac1d7371831bc95f1de582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31753 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-90e0ead4-66b5-4005-be4b-b8a35511daee\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC10769298\\\",\\n    \\\"repository_reference\\\": \\\"PMC\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"38187779\\\",\\n    \\\"repository_reference\\\": \\\"PubMed\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.1101/2023.12.20.572491\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/statgen/popscle\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://ftp.ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Q32851\\\",\\n    \\\"repository_reference\\\": \\\"www.ncbi.nlm.nih.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://imputationserver.sph.umich.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/naumanjaved/fingerprint_maps\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/ivykos/R-Pseudobulk/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PMC10769298\\\",\\n    \\\"repository_reference\\\": \\\"PMC\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"38187779\\\",\\n    \\\"repository_reference\\\": \\\"PubMed\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.1101/2023.12.20.572491\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/statgen/popscle\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://ftp.ensembl.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Q32851\\\",\\n    \\\"repository_reference\\\": \\\"www.ncbi.nlm.nih.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://imputationserver.sph.umich.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/naumanjaved/fingerprint_maps\\\"\\n  },\\n   {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/ivykos/R-Pseudobulk/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322607,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 39868,\n",
      "        \"completion_tokens\": 452,\n",
      "        \"total_tokens\": 40320,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PMC10769298', 'repository_reference': 'PMC', 'dataset_id': 'PMC10769298'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: PMC10769298\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID PMC10769298 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PMC, identifier: PMC10769298\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'PMC10769298', 'data_repository': 'PMC', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '38187779', 'repository_reference': 'PubMed', 'dataset_id': '38187779'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: 38187779\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID 38187779 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PubMed, identifier: 38187779\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '38187779', 'data_repository': 'PubMed', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.1101/2023.12.20.572491', 'repository_reference': 'doi.org', 'dataset_id': '10.1101/2023.12.20.572491'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.1101/2023.12.20.572491, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.1101/2023.12.20.572491\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.1101/2023.12.20.572491', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'repository_reference': 'synapse.org', 'dataset_id': 'syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn11724057, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/statgen/popscle', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/statgen/popscle, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/statgen/popscle\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/statgen/popscle\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Q32851', 'repository_reference': 'www.ncbi.nlm.nih.gov', 'dataset_id': 'Q32851'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Q32851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Q32851 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: Q32851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'Q32851', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC10769298', 'data_repository': 'PMC', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '38187779', 'data_repository': 'PubMed', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1101/2023.12.20.572491', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'Q32851', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 6 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: PMC10769298\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PMC, identifier: PMC10769298\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC10769298 for PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for PMC10769298 in pmc\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: pmc and accession_id: PMC10769298\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository pmc unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 38187779\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: PubMed, identifier: 38187779\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 38187779 for PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: PubMed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 38187779 in pubmed\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: pubmed and accession_id: 38187779\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository pubmed unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.1101/2023.12.20.572491\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1101/2023.12.20.572491\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/2023.12.20.572491 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.1101/2023.12.20.572491 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.1101/2023.12.20.572491\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.1101/2023.12.20.572491\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: Q32851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: Q32851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Q32851 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for Q32851 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: www.ncbi.nlm.nih.gov and accession_id: Q32851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 6.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 6\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 11\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 82th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8226229&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b76d3200>, <Element supplementary-material at 0x1b1959f00>, <Element supplementary-material at 0x1b195b7c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b76d3200>, <Element supplementary-material at 0x1b1959f00>, <Element supplementary-material at 0x1b195b7c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SM1: The flowchart of deriving stemness index (mRNAsi) was shown in \n",
      " [this file] . The R code was shown in \n",
      " [this file] . The stemness signatures of the predictive model were shown in \n",
      " [this file] . \n",
      "After constructing the predictive model in PCBC stem cell samples and applying it in NJHNCC, we obtained the stemness index ( \n",
      " [this file] ). \n",
      "Some 2,925 differential expressed genes were obtained by differential gene expression analysis of RNA-seq data from NJHNCC (Wilcoxon test, FDR <0.05), including 1,294 upregulated genes and 1,631 downregulated genes ( \n",
      " [rid=f2] and \n",
      " [this file] ). \n",
      "Unsupervised hierarchical clustering showed that gene modules were obtained ( \n",
      " [rid=f2] ) with soft threshold β = 10 and scale-free R = 0.9 ( \n",
      " [this file] ). \n",
      "Details of the eight candidate stemness-related genes were shown in \n",
      " [this file] . \n",
      "Also, we explored the relationship of co-expressed genes in the STRING database and it showed that they had significant correlations with other genes ( \n",
      " [rid=f5] and \n",
      " [this file] ). \n",
      "In addition, genes in the module that negatively related to stemness were also analyzed and the result showed that they were related to extracellular matrix organization ( \n",
      " [this file] ). \n",
      "The expressions of stemness-related genes in HNSCC and adjacent normal oral epithelial tissues were evaluated by qRT-PCR ( \n",
      " [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 101833\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 26210 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-7fc014262c2b8e03721b7cf78a14bf8e16fe6e7b59b38e0539b635604666607c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 26210 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6abc58f2-5c71-4be3-80a7-e572ee320765\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2701943\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2701943\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE30784\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NJHNCC-RNAseq-stemsig [14540610]\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2701943\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2701943\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE30784\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NJHNCC-RNAseq-stemsig [14540610]\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322610,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 31519,\n",
      "        \"completion_tokens\": 214,\n",
      "        \"total_tokens\": 31733,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2701943', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2701943', 'dataset_id': 'syn2701943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn2701943, synapse.org, https://www.synapse.org/#!Synapse:syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn2701943', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA', 'repository_reference': 'https://portal.gdc.cancer.gov', 'dataset_id': 'TCGA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID TCGA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE30784', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GSE30784'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE30784, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE30784\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE30784', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NJHNCC-RNAseq-stemsig [14540610]', 'repository_reference': 'https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610', 'dataset_id': 'NJHNCC-RNAseq-stemsig [14540610]'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NJHNCC-RNAseq-stemsig [14540610]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID NJHNCC-RNAseq-stemsig [14540610] is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://figshare.com/articles/dataset/NJHNCC-RNAseq-stemsig_txt/14540610\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'figshare.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn2701943', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TCGA', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE30784', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'figshare.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2701943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2701943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2701943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2701943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn2701943 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2701943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: portal.gdc.cancer.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL portal.gdc.cancer.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for TCGA in portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: portal.gdc.cancer.gov and accession_id: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository portal.gdc.cancer.gov unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE30784\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE30784\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE30784 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE30784 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE30784 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE30784\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30784\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 83th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11716409&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b18abb40>, <Element supplementary-material at 0x1b167c380>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b18abb40>, <Element supplementary-material at 0x1b167c380>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 347627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 87659 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d24b0a7702a4c56d1e2aa660c63cc382429d3cb601b85e0bd6dece98304578e2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 87659 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-de1a7add-5113-463b-aa3b-07982cf60685\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE89413\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GPL18573\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE16476\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-8248\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE79209\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE89413\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GPL18573\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE16476\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E-MTAB-8248\\\",\\n    \\\"repository_reference\\\": \\\"arrayexpress\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE79209\\\",\\n    \\\"repository_reference\\\": \\\"geo\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322613,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 103917,\n",
      "        \"completion_tokens\": 170,\n",
      "        \"total_tokens\": 104087,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE89413', 'repository_reference': 'geo', 'dataset_id': 'GSE89413'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE89413, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE89413\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE89413', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GPL18573', 'repository_reference': 'geo', 'dataset_id': 'GPL18573'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GPL18573, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GPL18573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GPL18573', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE16476', 'repository_reference': 'geo', 'dataset_id': 'GSE16476'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE16476, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE16476\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE16476', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-8248', 'repository_reference': 'arrayexpress', 'dataset_id': 'E-MTAB-8248'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID E-MTAB-8248 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-8248', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE79209', 'repository_reference': 'geo', 'dataset_id': 'GSE79209'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE79209, geo, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE79209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE79209', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE89413', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GPL18573', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE16476', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-8248', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE79209', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE89413\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE89413\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE89413 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE89413 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE89413 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE89413\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89413\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GPL18573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GPL18573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GPL18573 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GPL18573 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GPL18573 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GPL18573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL18573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE16476\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE16476\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE16476 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE16476 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE16476 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE16476\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE16476\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8248 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for E-MTAB-8248 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: arrayexpress and accession_id: E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-8248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE79209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE79209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE79209 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE79209 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE79209 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE79209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 84th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8055881&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b76d3200>, <Element supplementary-material at 0x1b18ab6c0>, <Element supplementary-material at 0x1b18aa040>, <Element supplementary-material at 0x1b18a9180>, <Element supplementary-material at 0x1b18a97c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b76d3200>, <Element supplementary-material at 0x1b18ab6c0>, <Element supplementary-material at 0x1b18aa040>, <Element supplementary-material at 0x1b18a9180>, <Element supplementary-material at 0x1b18a97c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: In hybrid clusters, the 5% sensitivity corresponded to 97.4% specificity in a receiver operating characteristic (ROC) curve analysis (AUC = 0.58, Supplementary Fig. 1A [this file] ). \n",
      "First, while this threshold (cluster closeness score = 2.56) may permit false-positives if the simulated phosphosites only contain negatives, we observed many of the clusters containing activating or recurrent mutations with cluster closeness scores close to the threshold (Supplementary Data 1 [this file] ). While hybrid clusters involving driver genes showed a higher density at the higher-score mode, driver gene status did not guarantee high scores (Supplementary Fig. 1B [this file] ). The 5% score threshold showed a sensitivity = 0.17 and specificity = 96.0% in distinguishing hybrid clusters with driver genes (Supplementary Fig. 1C [this file] ). Finally, we examined the score distribution using 200 bins on both the simulated vs. observed clusters, finding multiple peaks and alternative thresholds, for example, thresholding using one of the higher local minima retained only the top 2.28%, or the top 216 clusters (Supplementary Fig. 1D [this file] ). Cluster closeness scores for all identified clusters are provided herein to prioritize a more stringent set of clusters (Supplementary Data 1 [this file] ). \n",
      "HotPho generated a final high-confidence set of 906 mutation-only, 127 phosphosite-only, and 474 hybrid clusters based on the top 5% cluster closeness score threshold (“Methods”, Supplementary Data 1 [this file] ). Phosphosite-only clusters are found in fewer instances: ANXA5, CLIP1, FLNB, GPI, HSPD1, PEBP1, and PTK2 each harbored two (Supplementary Fig. 2A [this file] ). \n",
      "The Ensembl transcripts used for mapping of the protein coordinates are described in Supplementary Data 1 [this file] . \n",
      "The top hybrid clusters of each protein—identified by the highest cluster closeness score—may span mutations and phosphosites that are far from one another in the linear distance (Fig. 2c [rid=Fig2] , Supplementary Fig. 2B [this file] ). The list of 48 mutations contained mutations of interest in PDE1B (5 mutations), SRSF7 (4 mutations), and PTPN12 p.S275F/C that co-localized with p.S275 and co-clustered with p.S39/p.T40 (Supplementary Data 2 [this file] ). \n",
      "Among the 1,255 co-clustered phosphosites, 291 sites directly overlap and 356 sites are proximal (within 2 amino acid residues linearly) to their co-clustered mutations (Supplementary Data 1 [this file] , Fig. 2b [rid=Fig2] ). \n",
      "We then examined whether proteins containing hybrid clusters are enriched in specific biological pathways curated by WikiPathways and the NCI-Nature Pathway Interaction Database using Enrichr (Supplementary Data 3 [this file] , Supplementary Fig. 3 [this file] , “Methods”). \n",
      "Mapping residues to PFAM domains, we identified 26 PFAM domains significantly enriched for mutations and phosphosites in hybrid clusters when comparing to the background of all mapped mutations and phosphosites (Supplementary Data 4 [this file] , Supplementary Fig. 3 [this file] , “Methods”). Another two top PFAM domains are protein tyrosine kinase and protein kinase domains (Supplementary Fig. 3 [this file] ). Notably, some kinase-domain clusters showed conserved mutation/phosphosite patterns across homologs, such as FGFR2 and FGFR3 (Supplementary Fig. 3 [this file] ). \n",
      "We found 29 hybrid clusters containing 90 of these activating mutations in 17 genes, suggesting the functional relevance of the 54 co-clustering phosphosites (Supplementary Data 5 [this file] ). \n",
      "We specifically examined these clusters on protein structures (Fig. 3b [rid=Fig3] , Supplementary Fig. 5 [this file] ). \n",
      "We calculated the frequency of each of the co-clustering mutations within each of the TCGA cancer cohorts and identified their spatially adjacent phosphosites (Supplementary Data 6 [this file] ). We found that co-clustering phosphosites of the most recurrent mutations aggregate in proteins, including CTNNB1, HRAS, IDH1, KRAS, NRAS, PIK3CA, and TP53 (Fig. 3a [rid=Fig3] , Supplementary Fig. 5 [this file] ). \n",
      "Using the TCGA Reverse Phase Protein Arrays (RPPA) dataset for each of the 33 cancer types, we conducted a differential expression analysis to search for protein/phosphoprotein markers expressed at different levels in carriers of clustered mutations (“Methods”), identifying 24 significant (FDR < 0.05, linear regression) gene-cancer associations (Fig. 4b [rid=Fig4] , Supplementary Data 7 [this file] ). mutations in hybrid clusters are significantly correlated with higher p53 protein expression in 14 cancer types, most strikingly in UCEC, BRCA, COAD, and OV, consistent with the previously reported -effect of functional TP53 missense mutations . \n",
      "To further validate these findings, we conducted similar analyses of the mutational impact on protein expression using global proteomics datasets from retrospective and prospective CPTAC cohorts of breast, ovarian, and colorectal cancers (2 cohorts/cancer) each comprised of 78–126 samples (“Methods”, Supplementary Data 8 [this file] , Supplementary Fig. 6 [this file] ). Other notable findings include that mutations in cluster 1357.1 are associated with high ESR1 in ovarian cancer, whereas mutations in cluster 756.0 (Supplementary Fig. 6 [this file] ) are associated with high AKT1 proteins. \n",
      "By considering missense mutations (1) directly overlapping phosphosites, (2) proximal to phosphosites, and (3) co-clustering with phosphosites, we noticed that considering co-clustering mutations contribute significantly to the fraction of potentially functional mutations in many cancer genes including , , and (Supplementary Fig. 7 [this file] ). \n",
      "We then examined whether the co-clustered mutations show significant enrichment of activating mutations compared to the other mutations on a gene-by-gene basis (Fig. 5a, b [rid=Fig5] , Supplementary Data 9 [this file] ). \n",
      "First, using a multivariate logistic regression model corrected for the mutated gene, we found that the number of co-clustering mutations was not significantly associated with the mutation functionality in either the BAF3 ( = 0.91) or MCF ( = 0.40) cell line (Supplementary Fig. 9A [this file] ). But, when adding the phosphosite co-clustering status to the regression model, the mutation functionality was no longer associated with recurrence ( > 0.21), but strongly associated with the co-clustering status in both BAF3 ( = 1.4e−10) and MCF10A ( = 1.5e−4) cell lines (Fig. 5c, d [rid=Fig5] , Supplementary Fig. 9A [this file] ). \n",
      "We also obtained similar results when comparing cell lines with co-clustered missense mutations versus other non-synonymous mutations (Supplementary Data 10 [this file] ). \n",
      "Of the 1255 co-clustered phosphosites, 259 were detected in at least one of the 5 cohorts (Fig. 6a [rid=Fig6] , Supplementary Data 11 [this file] ). \n",
      "Finally, we conducted a systematic literature review of co-clustering phosphosites regulated or implicated in cancer (“Methods”), finding 25 unique phosphosites across 18 proteins that were experimentally linked to cancer (Supplementary Data 12 [this file] ). \n",
      "Among the 1255 co-clustering phosphosites, only 25 were previously known to be associated with cancer (Supplementary Data 12 [this file] ). \n",
      "Currently, we only observed a handful of examples where a single tumor sample carries both of the co-clustering phosphosites and mutations in existing quantitative phosphoproteomics cohorts, precluding systematic investigations of their correlative relationships (Supplementary Fig. 9 [this file] ). \n",
      "To resolve the multi-modal distribution of cluster closeness scores, we further compared the score distributions for 299 mutation-enriched cancer driver genes versus other genes using a ROC curve analysis (Supplementary Fig. 1B, C [this file] ). \n",
      "We further evaluated the sensitivity, specificity, and ROC curves using different threshold of the Cc score (Supplementary Fig. 1D [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 116481\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 29872 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-db403ac052ac69db6a30facc44e34be0b1286f0da36c636a99d3335a3bdb3a99\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 29872 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c793f0d6-90fe-4271-866e-ab3453dd6caa\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7824274\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GDC\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S039\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S038\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S037\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S053\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S050\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ptmcosmos.wustl.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7824274\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GDC\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S039\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S038\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S037\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S053\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://cptac-data-portal.georgetown.edu/study-summary/S050\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://ptmcosmos.wustl.edu/\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322616,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 35651,\n",
      "        \"completion_tokens\": 399,\n",
      "        \"total_tokens\": 36050,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn7824274', 'repository_reference': 'synapse.org', 'dataset_id': 'syn7824274'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn7824274, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn7824274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn7824274', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GDC', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'GDC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID GDC is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GDC for gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GDC', 'data_repository': 'gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S039', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S039'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/study-summary/S039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S039, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S039\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/study-summary/S039 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S039', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S039'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S038', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S038'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/study-summary/S038\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S038\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S038, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S038\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S038\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/study-summary/S038 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S038', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S038'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S037', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S037'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/study-summary/S037\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S037\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S037, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S037\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S037\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/study-summary/S037 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S037', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S037'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S053', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/study-summary/S053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S053, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/study-summary/S053 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S053', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S050', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S050'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://cptac-data-portal.georgetown.edu/study-summary/S050\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S050\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://cptac-data-portal.georgetown.edu/study-summary/S050, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://cptac-data-portal.georgetown.edu/study-summary/S050\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://cptac-data-portal.georgetown.edu/study-summary/S050\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: cptac-data-portal.georgetown.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://cptac-data-portal.georgetown.edu/study-summary/S050 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://cptac-data-portal.georgetown.edu/study-summary/S050', 'repository_reference': 'n/a', 'dataset_id': 'https://cptac-data-portal.georgetown.edu/study-summary/S050'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://ptmcosmos.wustl.edu/', 'repository_reference': 'n/a', 'dataset_id': 'https://ptmcosmos.wustl.edu/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://ptmcosmos.wustl.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://ptmcosmos.wustl.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://ptmcosmos.wustl.edu/, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://ptmcosmos.wustl.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://ptmcosmos.wustl.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: ptmcosmos.wustl.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://ptmcosmos.wustl.edu/ is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://ptmcosmos.wustl.edu/', 'repository_reference': 'n/a', 'dataset_id': 'https://ptmcosmos.wustl.edu/'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn7824274', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GDC', 'data_repository': 'gdc.cancer.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn7824274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7824274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7824274 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7824274 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7824274 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7824274 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn7824274 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn7824274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7824274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GDC for gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GDC in gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: gdc.cancer.gov and accession_id: GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=GDC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: ...\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 85th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8102856&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0cbfd80>, <Element supplementary-material at 0x1b7d90dc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0cbfd80>, <Element supplementary-material at 0x1b7d90dc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 108250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 27815 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d237d9aa8f2b631acde35d2831e1841853d958ef1eecf7c3e6eaeae8ada8b0b8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 27815 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bc5c175d-5dc1-4dd7-a850-0dbbf49f7181\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"CNP0001438\\\",\\n    \\\"repository_reference\\\": \\\"bigd.big.ac.cn\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CNP0001438\\\",\\n    \\\"repository_reference\\\": \\\"cngb.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://gdc.cancer.gov/about-data/publications/mc3-2017\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7214402/wiki/406007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://cancergenome.nih.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://support.sentieon.com/appnotes/out_fields/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"CNP0001438\\\",\\n    \\\"repository_reference\\\": \\\"bigd.big.ac.cn\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CNP0001438\\\",\\n    \\\"repository_reference\\\": \\\"cngb.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://gdc.cancer.gov/about-data/publications/mc3-2017\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7214402/wiki/406007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://cancergenome.nih.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://support.sentieon.com/appnotes/out_fields/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322618,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 34128,\n",
      "        \"completion_tokens\": 268,\n",
      "        \"total_tokens\": 34396,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'CNP0001438', 'repository_reference': 'bigd.big.ac.cn', 'dataset_id': 'CNP0001438'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID CNP0001438 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: bigd.big.ac.cn, identifier: CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: bigd.big.ac.cn\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'CNP0001438', 'data_repository': 'bigd.big.ac.cn', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://gdc.cancer.gov/about-data/publications/mc3-2017', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://gdc.cancer.gov/about-data/publications/mc3-2017, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://gdc.cancer.gov/about-data/publications/mc3-2017\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://gdc.cancer.gov/about-data/publications/mc3-2017\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'https://gdc.cancer.gov/about-data/publications/mc3-2017', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'CNP0001438', 'data_repository': 'bigd.big.ac.cn', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: bigd.big.ac.cn, identifier: CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: bigd.big.ac.cn\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for CNP0001438 in bigd.big.ac.cn\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: bigd.big.ac.cn and accession_id: CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://bigd.big.ac.cn/browse/CNP0001438\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 86th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9085578&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 8 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1957700>, <Element supplementary-material at 0x1b760a0c0>, <Element supplementary-material at 0x1b6e17800>, <Element supplementary-material at 0x1b18a88c0>, <Element supplementary-material at 0x1b18a9600>, <Element supplementary-material at 0x1b18a8580>, <Element supplementary-material at 0x1b6bb9d00>, <Element supplementary-material at 0x1b0e5dec0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1957700>, <Element supplementary-material at 0x1b760a0c0>, <Element supplementary-material at 0x1b6e17800>, <Element supplementary-material at 0x1b18a88c0>, <Element supplementary-material at 0x1b18a9600>, <Element supplementary-material at 0x1b18a8580>, <Element supplementary-material at 0x1b6bb9d00>, <Element supplementary-material at 0x1b0e5dec0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS1: NCI (complete lists in Supplementary Table 1 [this file] ). \n",
      "From a similar analysis on subjects with the fastest and slowest global cognitive decline (average across the five cognitive domains assessed in the ROSMAP study), there were 21 genes common to both lists (with and again ranked highly) (Complete gene lists in Supplementary Table 1 [this file] ) including seven that were common to dementia vs. \n",
      "As a follow-up, we tested whether there were genes that predicted the slope of episodic memory decline across the entire cohort. 38 genes predicted the progression of episodic memory decline, with 23 of these genes also associated with the fast decliners group ( Supplementary Table 1 [this file] ). and were again the top-ranked genes along with several transporter genes ( ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS2: They are both linked to peripheral immune response and extracellular matrix maintenance (summarized in Supplementary Table 2 [this file] ) but there is little information on their role in the human brain. The genes predictive of ADAMTS2 levels included genes associated with glucocorticoid response (e.g., , , ) and neuropeptide signaling (e.g., ) ( Hofer et al., 2008 [rid=B15] ; Gutièrrez-Mecinas et al., 2011 [rid=B14] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS3: There were 205 genes predictive of and 98 genes predictive of levels ( Supplementary Table 3 [this file] ), although there were no overlaps between the two “co-expressed” gene lists or in the STRING protein-protein interaction database ( Supplementary Figures 1A,B [rid=DS1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS4: This resulted in 713 subjects with complete data for 62 variables being used as the input for the analysis (Variable descriptions and missing data summarized in Supplementary Table 4 [this file] ). \n",
      "PRTN3 was ranked 1st by the tree-based algorithms (“xgbTree” and “ranger”) in all analyses while ( Supplementary Table 4 [this file] ). \n",
      "AT8, Abnormally phosphorylated tau from immunohistochemistry; NFT, neurofibrillary tangles from silver staining; NP, neuritic plaques from silver staining; CERAD, Consortium to establish a registry for Alzheimer’s disease score; TDP, Tar-DNA Binding Protein stage, full variable description in Supplementary Table 4 [this file] . \n",
      "AT8, Abnormally phosphorylated tau from immunohistochemistry; NFT, neurofibrillary tangles from silver staining; NP, neuritic plaques from silver staining, full variable description in Supplementary Table 4 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS5: The second dataset included ROSMAP controls and AD subjects ( = 48) ( Mathys et al., 2019 [rid=B25] ), and it showed that was only expressed by excitatory and inhibitory neurons and was downregulated in AD. was expressed in and upregulated in AD in all cell types ( Supplementary Table 5 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS6: Following exclusions for missing data, 62 variables from 713 individuals were used as the input to Boruta ( Supplementary Table 6 [this file] ). 34 variables were identified for classifying dementia vs. \n",
      "There were 27 variables predictive of accelerated episodic memory decline and 26 for accelerated global cognitive decline (Top 10 shown in Table 4 [rid=T4] with a full list in Supplementary Table 6 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref TS7: Tangles in the ACC was again the highest-ranked feature in the “xgbTree” and “ranger” models for all analyses ( Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref DS1: There were 205 genes predictive of and 98 genes predictive of levels ( Supplementary Table 3 [rid=TS3] ), although there were no overlaps between the two “co-expressed” gene lists or in the STRING protein-protein interaction database ( Supplementary Figures 1A,B [this file] ). \n",
      "and were validated using the AD consensus transcriptomics online resource (see text footnote 1) ( Morabito et al., 2020 [rid=B26] ). was lower in symptomatic AD (AD pathology diagnosis and dementia) compared to controls and asymptomatic AD (AD pathology diagnosis without dementia) in the Mayo Clinic temporal cortex data (not present in the Mount Sinai Brain Bank analysis). levels were consistently higher in symptomatic AD relative to both controls and asymptomatic AD (no difference between controls and asymptomatic AD) in all datasets ( Supplementary Figures 1C,D [this file] ). \n",
      "The first dataset included controls only (aged 67–90) and it showed that and transcripts were mainly expressed by neurons with lower expression in other cell types (microglial expression of ADAMTS2 only) ( Supplementary Figures 1E,F [this file] ). \n",
      "The original contributions presented in the study are included in the article/ Supplementary Material [this file] , further inquiries can be directed to the corresponding author. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 124074\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31771 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a88e2e132d506b658433a2d428c1790f8692b25ca458e282b75ac8314b92656f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31771 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-46947f4f-260c-45f7-9838-6620c3ec88ac\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8612097\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8612097\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8612097\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn8612097\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322619,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 36586,\n",
      "        \"completion_tokens\": 61,\n",
      "        \"total_tokens\": 36647,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn8612097', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8612097', 'dataset_id': 'syn8612097'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn8612097, synapse.org, https://www.synapse.org/#!Synapse:syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn8612097', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn8612097', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8612097 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn8612097 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn8612097 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8612097 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn8612097 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8612097\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 87th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9840277&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 14 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0e5fbc0>, <Element supplementary-material at 0x1b7bcdf40>, <Element supplementary-material at 0x1b7379500>, <Element supplementary-material at 0x1b18f28c0>, <Element supplementary-material at 0x1b1901340>, <Element supplementary-material at 0x1b1900840>, <Element supplementary-material at 0x1b1902300>, <Element supplementary-material at 0x1b19007c0>, <Element supplementary-material at 0x1b1877e40>, <Element supplementary-material at 0x1b1876300>, <Element supplementary-material at 0x1b18b0100>, <Element supplementary-material at 0x1b1876b40>, <Element supplementary-material at 0x1b1877d40>, <Element supplementary-material at 0x1b1876d80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0e5fbc0>, <Element supplementary-material at 0x1b7bcdf40>, <Element supplementary-material at 0x1b7379500>, <Element supplementary-material at 0x1b18f28c0>, <Element supplementary-material at 0x1b1901340>, <Element supplementary-material at 0x1b1900840>, <Element supplementary-material at 0x1b1902300>, <Element supplementary-material at 0x1b19007c0>, <Element supplementary-material at 0x1b1877e40>, <Element supplementary-material at 0x1b1876300>, <Element supplementary-material at 0x1b18b0100>, <Element supplementary-material at 0x1b1876b40>, <Element supplementary-material at 0x1b1877d40>, <Element supplementary-material at 0x1b1876d80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: Primers were specific for the different genes, and sequences are shown in Supplementary Tab. 4 [this file] . \n",
      "The membrane was then placed in a 50-mL tube and incubated over night with the primary antibody (see Supplementary Tab. 3 [this file] ; dissolved in 5 % milk 0.05 % Tween TBS) at 4 °C on a roller bank. The following day, the membrane was washed three times 5 min in 0.05 % Tween TBS, after which it was incubated with the secondary antibody (see Supplementary Tab. 3 [this file] ), also diluted in 5 % milk 0.05 % Tween TBS (1 h, RT, on roller bank). \n",
      "The sequences were specific for 3R and 4R and are shown in Supplementary Tab. 4 [this file] . All primer sequences are shown in Supplementary Tab. 4 [this file] . \n",
      "The primary antibodies (see Supplementary Tab. 3 [this file] ) were diluted in 5% goat serum 0.1% Triton-X TBS, and the slices incubated overnight at 4°C in a humidified chamber. The following day, slices were washed in TBS three times, followed by incubation with the respective secondary antibodies (see Supplementary Tab. 3 [this file] ), which were diluted in 2% BSA TBS (1h, RT). \n",
      "Furthermore, mRNA analysis using NanoString technology and PCR corroborated the absence of murine tau expression in htau-KI mice (Fig. 1 [rid=Fig1] c, d) and 5xFADxhtau-KI crossbreds (Supplementary Fig. 1 [this file] b). \n",
      "Interestingly, for 3R/4R tau protein, a relative ratio of approx. 1 was observed in 7-month-old mice (Fig. 2 [rid=Fig2] d, Supplementary Fig. 2 [this file] a–d), resembling the human 3R/4R tau protein ratio. \n",
      "The comparison of WT, htau-KI, 5xFAD, and 5xFADxhtau-KI models revealed that a large fraction of variation in gene expression observed between WT and 5xFADxhtau-KI could be attributed to the 5xFAD model (Supplementary Fig. 1 [this file] , Supplementary Tab. 1 [this file] , 2 [this file] ). \n",
      "At 7 months, multiple pathways in the 5xFADxhtau-KI model showed a significant enrichment when compared to the 5xFAD model (Supplementary Fig. 3 [this file] a). Among these pathways, genes in the KEGG oxidative phosphorylation (oxphos) pathway were coordinatively upregulated in the presence of htau-KI (5xFADxhtau-KI) when compared to the 5xFAD model (mmu190, enrichment score= 0.69, adjusted -value= 0.0058) (Fig. 3 [rid=Fig3] , Supplementary Fig. 3 [this file] a). In contrast, genes in the oxphos pathway (hsa190, enrichment score= −0.59, adjusted -value= 0.0025) were downregulated in deceased AD patients from the MSBB cohort when compared to non-demented controls (Supplementary Fig. 3 [this file] b–d) (see also: https://agora.ampadportal.org/genes (https://agora.ampadportal.org/genes) for detailed human gene expression profiles). \n",
      "At 13 months of age, multiple pathways, which are linked to extracellular matrix and immune function, are downregulated in the 5xFADxhtau-KI model while the calcium and oxytocin signaling pathways are upregulated when compared to the 5xFAD model (Supplementary Fig. 4 [this file] a). The ECM-receptor interaction is upregulated (hsa04512, enrichment score= 0.67, FDR-adjusted -value= 0.0088) in the TCX brain region from the Mayo cohort associated with the TCXblue co-expression module (Fig. 3 [rid=Fig3] ) and downregulated in the 5xFADxhtau-KI model (mmu04512, enrichment score= −0.72, FDR-adjusted -value= 0.011) (Supplementary Fig. 4 [this file] a, b). Among the genes, which showed decreased expression in the 5xFADxhtau-KI model compared to the 5xFAD model, were multiple AD candidate genes (Supplementary Fig. 4 [this file] c), which are significantly upregulated in AD patients compared to controls. is an integrin that facilitates cell-extracellular matrix adhesion, which has been shown to act as a transcriptional hub gene for inflammatory AD modules [ 41 [rid=CR41] ]. has been highlighted as an important hub gene in an integrated proteomic and transcriptomic study in the ROS/MAP cohort [ 42 [rid=CR42] ]. \n",
      "Quantification of overall plaque deposition revealed no statistically significant differences in the total plaque area between 5xFAD and 5xFADxhtau-KI at 7 and 13 months of age (Fig. 4 [rid=Fig4] c, Supplementary Fig. 5 [this file] b-f). In addition, the inflammatory phenotype expected for 5xFAD mice could also be seen in 5xFADxhtau-KI crossbreds, but not in htau-KI, as studied by astroglial (GFAP, Supplementary Fig. 6 [this file] a, c) and microglial (Iba-1, Supplementary Fig. 6 [this file] b, d) staining. \n",
      "Additionally, we analyzed the distance covered by an animal to reach the platform (Supplementary Fig. 7 [this file] a, b) and swim speed (Supplementary Fig. 7 [this file] c, d) as score for cognitive learning curves as they also reflect the actual spatial learning abilities. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Notably, gene expression correlation with human co-expression modules was weaker in the htau-KI genotype when compared to the 5xFAD mice (see also Additional file 2 [this file] ), indicating that the effect of human tau expression in htau-KI mice on pathways that are typically altered in AD is not as pronounced as in 5xFAD mice. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Interestingly, we found a strong negative correlation between both genotypes associated with multiple human co-expression modules (Fig. 3 [rid=Fig3] , see also Additional file 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Original files can be found as Additional files 4 [this file] , 5 [rid=MOESM5] and 6 [rid=MOESM6] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Original files can be found as Additional files 4 [rid=MOESM4] , 5 [this file] and 6 [rid=MOESM6] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Original files can be found as Additional files 4 [rid=MOESM4] , 5 [rid=MOESM5] and 6 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Original image file can be found as Additional files 7 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: Uncropped image files can be found as Additional files 8 [this file] , 9 [rid=MOESM9] , 10 [rid=MOESM10] and 11 [rid=MOESM11] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: Uncropped image files can be found as Additional files 8 [rid=MOESM8] , 9 [this file] , 10 [rid=MOESM10] and 11 [rid=MOESM11] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM10: Uncropped image files can be found as Additional files 8 [rid=MOESM8] , 9 [rid=MOESM9] , 10 [this file] and 11 [rid=MOESM11] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM11: Uncropped image files can be found as Additional files 8 [rid=MOESM8] , 9 [rid=MOESM9] , 10 [rid=MOESM10] and 11 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM12: Uncropped image files can be found as Additional files 12 [this file] , 13 [rid=MOESM13] and 14 [rid=MOESM14] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM13: Uncropped image files can be found as Additional files 12 [rid=MOESM12] , 13 [this file] and 14 [rid=MOESM14] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM14: Uncropped image files can be found as Additional files 12 [rid=MOESM12] , 13 [rid=MOESM13] and 14 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 239419\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 60607 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-89d33599ce9af9ee7cf89b3f317b2ea85c3c33ecfb476ff2bc6592e2a5c26222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 60607 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-06faa35c-df50-4048-93cf-de23626d62fd\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11914606\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11914606\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23569889\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23569889\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn14237651\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn14237651\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11914606\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11914606\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23569889\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23569889\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn14237651\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn14237651\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322622,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 68507,\n",
      "        \"completion_tokens\": 173,\n",
      "        \"total_tokens\": 68680,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn11914606', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn11914606', 'dataset_id': 'syn11914606'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn11914606, synapse.org, https://www.synapse.org/#!Synapse:syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn11914606', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn23569889', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23569889', 'dataset_id': 'syn23569889'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn23569889, synapse.org, https://www.synapse.org/#!Synapse:syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn23569889', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn14237651', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn14237651', 'dataset_id': 'syn14237651'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn14237651, synapse.org, https://www.synapse.org/#!Synapse:syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn14237651', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn11914606', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23569889', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn14237651', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11914606 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11914606 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11914606 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11914606 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn11914606 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11914606\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23569889 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23569889 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23569889 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23569889 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn23569889 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23569889\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn14237651 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn14237651 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn14237651 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn14237651 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn14237651 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn14237651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 88th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10547713&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b11a1640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b11a1640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 127848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 32714 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d6f33a03c0a51c4248af69398d01fd52b5a735b5a6ff29e72011b03758e0a7d2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 32714 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a326ed66-f40a-4de2-ade5-efc5eebd07af\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn26707219\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CVC-ClinicDB\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GIANA 2021\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir-Seg\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Nerthus\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HyperKvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Colon10k\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Instrument-kvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn26707219\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CVC-ClinicDB\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GIANA 2021\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir-Seg\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Nerthus\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HyperKvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Colon10k\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Instrument-kvasir\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322624,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 39624,\n",
      "        \"completion_tokens\": 304,\n",
      "        \"total_tokens\": 39928,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn26707219', 'repository_reference': 'synapse.org', 'dataset_id': '10.7303/syn26707219'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn26707219, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn26707219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn26707219', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'CVC-ClinicDB', 'repository_reference': 'n/a', 'dataset_id': 'CVC-ClinicDB'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: CVC-ClinicDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID CVC-ClinicDB is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: CVC-ClinicDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CVC-ClinicDB for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier CVC-ClinicDB for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier CVC-ClinicDB for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CVC-ClinicDB for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CVC-ClinicDB for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'CVC-ClinicDB', 'repository_reference': 'n/a', 'dataset_id': 'CVC-ClinicDB'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GIANA 2021', 'repository_reference': 'n/a', 'dataset_id': 'GIANA 2021'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: GIANA 2021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID GIANA 2021 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'GIANA 2021', 'repository_reference': 'n/a', 'dataset_id': 'GIANA 2021'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Kvasir', 'repository_reference': 'n/a', 'dataset_id': 'Kvasir'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Kvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Kvasir is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: Kvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Kvasir', 'repository_reference': 'n/a', 'dataset_id': 'Kvasir'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Kvasir-Seg', 'repository_reference': 'n/a', 'dataset_id': 'Kvasir-Seg'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Kvasir-Seg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Kvasir-Seg is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: Kvasir-Seg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kvasir-Seg for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Kvasir-Seg for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Kvasir-Seg for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kvasir-Seg for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kvasir-Seg for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Kvasir-Seg', 'repository_reference': 'n/a', 'dataset_id': 'Kvasir-Seg'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Nerthus', 'repository_reference': 'n/a', 'dataset_id': 'Nerthus'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Nerthus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Nerthus is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: Nerthus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Nerthus for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Nerthus for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Nerthus for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Nerthus for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Nerthus for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Nerthus', 'repository_reference': 'n/a', 'dataset_id': 'Nerthus'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'HyperKvasir', 'repository_reference': 'n/a', 'dataset_id': 'HyperKvasir'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: HyperKvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID HyperKvasir is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: HyperKvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HyperKvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HyperKvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HyperKvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HyperKvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier HyperKvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'HyperKvasir', 'repository_reference': 'n/a', 'dataset_id': 'HyperKvasir'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Colon10k', 'repository_reference': 'n/a', 'dataset_id': 'Colon10k'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Colon10k\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Colon10k is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: Colon10k\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Colon10k for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Colon10k for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Colon10k for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Colon10k for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Colon10k for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Colon10k', 'repository_reference': 'n/a', 'dataset_id': 'Colon10k'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Instrument-kvasir', 'repository_reference': 'n/a', 'dataset_id': 'Instrument-kvasir'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Instrument-kvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID Instrument-kvasir is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: Instrument-kvasir\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Instrument-kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Instrument-kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Instrument-kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Instrument-kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Instrument-kvasir for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Instrument-kvasir', 'repository_reference': 'n/a', 'dataset_id': 'Instrument-kvasir'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': '10.7303/syn26707219', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn26707219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn26707219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn26707219 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn26707219 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn26707219 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.7303/syn26707219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn26707219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 89th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10571258&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1468d00>, <Element supplementary-material at 0x1b1468f80>, <Element supplementary-material at 0x1b1468d80>, <Element supplementary-material at 0x1b1469b80>, <Element supplementary-material at 0x1b1469c00>, <Element supplementary-material at 0x1b1469900>, <Element supplementary-material at 0x1b146a580>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1468d00>, <Element supplementary-material at 0x1b1468f80>, <Element supplementary-material at 0x1b1468d80>, <Element supplementary-material at 0x1b1469b80>, <Element supplementary-material at 0x1b1469c00>, <Element supplementary-material at 0x1b1469900>, <Element supplementary-material at 0x1b146a580>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The very highest relative modularity density measures are prone to instability as the raw values approach zero, which makes the raw values more reliable for that case (see Supplement [this file] ). \n",
      "S 4 [this file] ), or with edge weights between clusters held constant (Additional File 1 [this file] : Fig. S5), see Supplement [this file] for details on these tests. \n",
      "Differences in NMI and ARI are due to the large number of unclassified nodes from WGCNA (see Supplement [this file] ). \n",
      "Sinai brain bank, see Supplement [this file] ). \n",
      "Contrary to their name, the gold-standard cell types may not be independent definitions, as they are sometimes generated by clustering (see Supplement [this file] ). \n",
      "Correspondence to ground truth was highly dataset dependent: 3 datasets averaged ~ 10% difference between the methods, while on 4 datasets, SE2 correspondence by ARI was only ~ 50% of Seurat (Additional File 1 [this file] : Fig S7). If the study goal is to mirror the style of classes that have previously been found, then Seurat is a good choice; in contrast, if the application would benefit from discovery of more granular cellular classes, SE2 will more likely produce that type of result (see Supplement [this file] on multiscale results). \n",
      "Both SE2 and Louvain/Seurat performance (ARI with ground truth) were systematically affected by preprocessing parameters (R = 0.39, R = 0.28) with both methods being most sensitive to data reduction type (UMAP vs ICA, Additional File 1 [this file] : Fig. \n",
      "SE2 is 60% faster the original SE and several times faster than Louvain and Leiden, on fully connected networks (Additional File 1 [this file] : Fig. For details of performance testing and significant caveats to interpreting these, see Supplementary methods [this file] . \n",
      "Networks are also generated under three different regimes, where the cross-links between clusters (1) have covarying proportion (Fig. 2 [rid=Fig2] ) and weights, (2) constant proportion and varying weight (Additional File 1 [this file] : Fig S2), or (3) varying proportion and constant weight (Additional File 1 [this file] : Fig S3). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Overall, the operation of SE2 can be conceptualized as merging groups of nodes, balanced with a tendency for groups to burst open like bubbles when they have expanded too far (see “ Methods [rid=Sec14] ”; Additional File 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Recovery of gold-standard protein complexes demonstrates the importance of metric selection in perceived performance of clustering methods (Additional File 3 [this file] : Table 1). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: However, Louvain and Leiden are orders of magnitude faster than SE2 in clustering large sparse networks (Additional File 4 [this file] : Table 2). Infomap appears to falter in this setting, providing low modularity and returning most nodes in a single cluster (Additional File 4 [this file] : Table 2). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Champagne is to identify clusters by maximizing the specificity of the labels assigned to nodes, where the specificity of a node’s label is defined as the difference between the actual frequency of that label in neighboring nodes compared to the expected frequency, given the global popularity of each label (pseudocode in Additional File 5 [this file] , [ 89 [rid=CR89] ]). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: After looping through these four modes of activity, SE2 records the label state as a possible partition (Fig. 1 [rid=Fig1] H, Additional File 6 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: The review history is available as Additional file 7 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 207745\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 52688 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fc02e35619440f28ebf1ef2323a55761294e74c82f82bfb307c100a8b3d0f66e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 52688 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ca673b32-e21c-41c4-99b0-f66201234435\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ROSMAP gene expression\\\",\\n    \\\"repository_reference\\\": \\\"https://www.radc.rush.edu/requests.htm\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Mayo and Mt. Sinai gene expression\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn30821562\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"single-cell data\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"protein binding\\\",\\n    \\\"repository_reference\\\": \\\"https://paccanarolab.org/cluster-one\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"diverse network set\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/Aghasemian/CommunityFitNet\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"large networks\\\",\\n    \\\"repository_reference\\\": \\\"https://snap.stanford.edu/data\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.5281/zenodo.8322854\\\",\\n    \\\"repository_reference\\\": \\\"zenodo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/cogdishion/SE2\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/DavidRConnell/igraph-SE2\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.8338532\\\",\\n    \\\"repository_reference\\\": \\\"zenodo\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ROSMAP gene expression\\\",\\n    \\\"repository_reference\\\": \\\"https://www.radc.rush.edu/requests.htm\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Mayo and Mt. Sinai gene expression\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn30821562\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"single-cell data\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"protein binding\\\",\\n    \\\"repository_reference\\\": \\\"https://paccanarolab.org/cluster-one\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"diverse network set\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/Aghasemian/CommunityFitNet\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"large networks\\\",\\n    \\\"repository_reference\\\": \\\"https://snap.stanford.edu/data\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://doi.org/10.5281/zenodo.8322854\\\",\\n    \\\"repository_reference\\\": \\\"zenodo\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/cogdishion/SE2\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/DavidRConnell/igraph-SE2\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.8338532\\\",\\n    \\\"repository_reference\\\": \\\"zenodo\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322627,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 57151,\n",
      "        \"completion_tokens\": 440,\n",
      "        \"total_tokens\": 57591,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ROSMAP gene expression', 'repository_reference': 'https://www.radc.rush.edu/requests.htm', 'dataset_id': 'ROSMAP gene expression'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ROSMAP gene expression\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID ROSMAP gene expression is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.radc.rush.edu/requests.htm, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.radc.rush.edu/requests.htm\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.radc.rush.edu/requests.htm\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Mayo and Mt. Sinai gene expression', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn30821562', 'dataset_id': 'Mayo and Mt. Sinai gene expression'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn30821562, None, https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn30821562, identifier: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30821562 for https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30821562 for https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30821562 for https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30821562 for https://www.synapse.org/#!Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn30821562 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn30821562', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'single-cell data', 'repository_reference': 'https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq', 'dataset_id': 'single-cell data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: single-cell data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID single-cell data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/mkrzak/Benchmarking_Clustering_Methods_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'protein binding', 'repository_reference': 'https://paccanarolab.org/cluster-one', 'dataset_id': 'protein binding'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: protein binding\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID protein binding is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://paccanarolab.org/cluster-one, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://paccanarolab.org/cluster-one\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://paccanarolab.org/cluster-one\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'paccanarolab.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'diverse network set', 'repository_reference': 'https://github.com/Aghasemian/CommunityFitNet', 'dataset_id': 'diverse network set'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: diverse network set\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID diverse network set is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/Aghasemian/CommunityFitNet, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/Aghasemian/CommunityFitNet\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/Aghasemian/CommunityFitNet\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'large networks', 'repository_reference': 'https://snap.stanford.edu/data', 'dataset_id': 'large networks'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: large networks\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID large networks is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://snap.stanford.edu/data, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://snap.stanford.edu/data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://snap.stanford.edu/data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'snap.stanford.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.5281/zenodo.8322854', 'repository_reference': 'zenodo', 'dataset_id': 'https://doi.org/10.5281/zenodo.8322854'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/, None, https://doi.org/10.5281/zenodo.8322854\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: zenodo, identifier: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/ for zenodo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.5281/ for zenodo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for zenodo in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/cogdishion/SE2', 'repository_reference': 'github.com', 'dataset_id': 'https://github.com/cogdishion/SE2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/cogdishion/SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/cogdishion/SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/cogdishion/SE2, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/cogdishion/SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/cogdishion/SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/cogdishion/SE2 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/DavidRConnell/igraph-SE2', 'repository_reference': 'github.com', 'dataset_id': 'https://github.com/DavidRConnell/igraph-SE2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/DavidRConnell/igraph-SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/DavidRConnell/igraph-SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/DavidRConnell/igraph-SE2, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/DavidRConnell/igraph-SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/DavidRConnell/igraph-SE2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/DavidRConnell/igraph-SE2 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.8338532', 'repository_reference': 'zenodo', 'dataset_id': '10.5281/zenodo.8338532'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/zenodo.8338532, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.8338532\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.8338532', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn30821562', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'paccanarolab.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'snap.stanford.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.8338532', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 10 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30821562 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30821562 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30821562 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30821562 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn30821562 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn30821562\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 7 with repo: doi.org and accession_id: 10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/zenodo.8338532\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.8338532\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.8338532 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/zenodo.8338532 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 10 with repo: doi.org and accession_id: 10.5281/zenodo.8338532\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/zenodo.8338532\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 10\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 90th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6829597&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b7551210>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 8 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s18: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s29'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s29: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s32'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s32: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#PMC_2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for PMC_2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 176644\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 44913 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-262e84d2c102e3140d77a40722b2e6bc1b5e28083828e49bd01d3f681f755b3e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 44913 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4b2c7177-aea1-4a30-89fb-9c6a692224e0\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset 1\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18796452\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Dataset 1\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18796452\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322647,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 62113,\n",
      "        \"completion_tokens\": 84,\n",
      "        \"total_tokens\": 62197,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Dataset 1', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'Dataset 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: Dataset 1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID Dataset 1 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.synapse.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1131 - INFO - Found partial match for https://www.synapse.org/ in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 672 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Dataset 1', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'Dataset 1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18796452', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn18796452'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn18796452, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn18796452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn18796452', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn18796452', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn18796452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18796452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18796452 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18796452 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18796452 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18796452 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn18796452 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn18796452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18796452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 562 - INFO - \n",
      "Progress: 91/100 (91.00%) | Elapsed: 00:14:35 | ETA: 00:01:26\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 91th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11240079&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <supplementary-material xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.or...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 19 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0d8ce80>, <Element supplementary-material at 0x1b1a4ba40>, <Element supplementary-material at 0x1b14da800>, <Element supplementary-material at 0x1b14d9480>, <Element supplementary-material at 0x1b14d9580>, <Element supplementary-material at 0x1b14dbe40>, <Element supplementary-material at 0x1b14dae00>, <Element supplementary-material at 0x1b14dad80>, <Element supplementary-material at 0x1b14d9200>, <Element supplementary-material at 0x1b14dbc80>, <Element supplementary-material at 0x1b1143e00>, <Element supplementary-material at 0x1b14d9140>, <Element supplementary-material at 0x1b14dba80>, <Element supplementary-material at 0x1b14db600>, <Element supplementary-material at 0x1b14d8d40>, <Element supplementary-material at 0x1b14db980>, <Element supplementary-material at 0x1b14db040>, <Element supplementary-material at 0x1b14d8500>, <Element supplementary-material at 0x1b14da780>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0d8ce80>, <Element supplementary-material at 0x1b1a4ba40>, <Element supplementary-material at 0x1b14da800>, <Element supplementary-material at 0x1b14d9480>, <Element supplementary-material at 0x1b14d9580>, <Element supplementary-material at 0x1b14dbe40>, <Element supplementary-material at 0x1b14dae00>, <Element supplementary-material at 0x1b14dad80>, <Element supplementary-material at 0x1b14d9200>, <Element supplementary-material at 0x1b14dbc80>, <Element supplementary-material at 0x1b1143e00>, <Element supplementary-material at 0x1b14d9140>, <Element supplementary-material at 0x1b14dba80>, <Element supplementary-material at 0x1b14db600>, <Element supplementary-material at 0x1b14d8d40>, <Element supplementary-material at 0x1b14db980>, <Element supplementary-material at 0x1b14db040>, <Element supplementary-material at 0x1b14d8500>, <Element supplementary-material at 0x1b14da780>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app1: A secondary data set was created by merging patients with common features from both primary data sets of the aforementioned clinical trials ( Multimedia Appendix 1 [this file] ). \n",
      "Finally, the platform provides detailed reports and visualizations, called blueprints, that show how each model performed on the data set ( Multimedia Appendix 10 [this file0] ). In addition, a more detailed view of the performance metrics achieved during training is provided by the learning curves during holdout partition and further LogLoss values across all CV folds ( Multimedia Appendix 11 [this file1] ). Finally, to ensure the variability and stability of our models’ performance across different data splits, we reran all our models for validation, CV, and holdout splits 10 times with 10 different random seeds and plotted the medians of all reported metrics with 95% CIs ( Multimedia Appendices 12 [this file] - 15 [this file5] ). Our study complies with the TRIPOD (Transparent Reporting of a Multivariable Prediction Model for Individual Prognosis or Diagnosis) guidelines ( Multimedia Appendix 16 [this file6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app2: Basic cohort characteristics of the secondary data set are shown in Multimedia Appendix 2 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app3: In addition, selected features were additionally transformed from numeric to binary or multiclass classification to facilitate subsequent AutoML analysis and interpretation of the results ( Multimedia Appendix 3 [this file] ). \n",
      "The feature with the highest impact was , a newly classified multiclass feature ( Multimedia Appendix 3 [this file] ), followed by , which was the most influential feature in the previous model. \n",
      "The third model selected in this study analyzed the progression of skin involvement in PsV by incorporating all available features, including several binary features newly created from multiclass features such as obesity, depression, hypertension, and TNF-α–targeting therapy ( Multimedia Appendix 3 [this file] ). \n",
      "Patients were stratified into 2 groups based on their initial BASDAI scores: inconspicuous (BASDAI score of ≤3; 68/129, 52.7%) and conspicuous (BASDAI score of ≥4; 61/129, 47.3%; Multimedia Appendix 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app4: Details of the processing of missing values, called imputation, are provided in Multimedia Appendix 4 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app5: Therefore, modeling was performed with a reduced feature list for each target ( Multimedia Appendices 5 [this file] - 8 [rid=app8] ). \n",
      "The list of features used and reduced by the AutoML platform for this target is given in Multimedia Appendix 5 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app6: The range of features used, as reduced by the AutoML platform for this target, is shown in Multimedia Appendix 6 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app7: The list of features used and reduced by the AutoML platform for this target is given in Multimedia Appendix 7 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app8: Therefore, modeling was performed with a reduced feature list for each target ( Multimedia Appendices 5 [rid=app5] - 8 [this file] ). \n",
      "The list of features used by the AutoML platform to analyze this target can be found in Multimedia Appendix 8 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app9: On each model, the AutoML platform automatically explores different configurations (hyperparameters) to optimize the model ( Multimedia Appendix 9 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app10: Finally, the platform provides detailed reports and visualizations, called blueprints, that show how each model performed on the data set ( Multimedia Appendix 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app11: In addition, a more detailed view of the performance metrics achieved during training is provided by the learning curves during holdout partition and further LogLoss values across all CV folds ( Multimedia Appendix 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app12: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app13: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app14: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app15: Finally, to ensure the variability and stability of our models’ performance across different data splits, we reran all our models for validation, CV, and holdout splits 10 times with 10 different random seeds and plotted the medians of all reported metrics with 95% CIs ( Multimedia Appendices 12 [rid=app1] - 15 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app16: Our study complies with the TRIPOD (Transparent Reporting of a Multivariable Prediction Model for Individual Prognosis or Diagnosis) guidelines ( Multimedia Appendix 16 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app17: A threshold of 0.6998 was set to discriminate between positive and negative predictions, giving an -score of 0.7117, sensitivity of 0.5918, and precision of 0.8923 in CV ( Figure 1 [rid=figure1] B) and an -score of 0.8182, sensitivity of 0.7200, and precision of 0.9474 in the holdout (Figure S1 in Multimedia Appendix 17 [this file] ). \n",
      "This resulted in an -score of 0.8271, sensitivity of 0.8661, and precision of 0.7914 in CV and an -score of 0.8966, sensitivity of 0.9630, and precision of 0.8387 in holdout (Figure S2 in Multimedia Appendix 17 [this file] ). \n",
      "Setting the decision threshold at 0.4471 gave an -score of 0.8000, sensitivity of 0.8148, and precision of 0.7857 in CV ( Figure 4 [rid=figure4] B), while the holdout analysis gave an -score of 0.8000, sensitivity of 0.8571, and precision of 0.7500 (Figure S3 in Multimedia Appendix 17 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app18: To illustrate how our chosen model could be used in everyday clinical practice, we considered 2 hypothetical patients for whom the ML model calculated the likelihood of changing therapy after 24 weeks ( Multimedia Appendix 18 [this file] ). Her calculated therapy change prediction score was 0.5097, which was below the established threshold ( Multimedia Appendix 18 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref app19: The primary predictor of PASI score progression was found to be the initial PASI score ( Figure 3 [rid=figure3] C; Multimedia Appendix 19 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 349978\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 88247 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-490ba068abbe72929eeed1a7e105d0023d1e8a8faf28815667b6a0d0fc9461a5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 88247 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-25269f54-91ab-41a1-8580-a4e7c2bd0331\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52566791\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn52566791\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52566791\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn52566791\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322649,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 99961,\n",
      "        \"completion_tokens\": 62,\n",
      "        \"total_tokens\": 100023,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn52566791', 'repository_reference': 'https://www.synapse.org/Synapse:syn52566791', 'dataset_id': 'syn52566791'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn52566791, synapse.org, https://www.synapse.org/Synapse:syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn52566791', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn52566791', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52566791 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52566791 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52566791 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52566791 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn52566791 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52566791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 92th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7805663&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b76c57d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 3 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for fn-group1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 117409\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 30104 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-56be447c2872f490c5ec6ba2fa0c0d74851634ef218f73818b5938c66e93f8ac\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 30104 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4aaf6763-096a-4c78-ba50-127300734839\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11737848\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/BioroboticsLab/bb_tracking\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11737848\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/BioroboticsLab/bb_tracking\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322666,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 39076,\n",
      "        \"completion_tokens\": 87,\n",
      "        \"total_tokens\": 39163,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn11737848', 'repository_reference': 'synapse.org', 'dataset_id': 'syn11737848'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn11737848, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn11737848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn11737848', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/BioroboticsLab/bb_tracking', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/BioroboticsLab/bb_tracking, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/BioroboticsLab/bb_tracking\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/BioroboticsLab/bb_tracking\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn11737848', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn11737848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11737848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11737848 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11737848 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11737848 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11737848 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn11737848 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn11737848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11737848\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 93th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10600818&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b113b400>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b113b400>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref S0033291723000235sup001: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 122928\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 31484 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-669fa66255ad76bdf7e71cd03855f0b4b9eb9eb757471e61717374be55af3d05\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 31484 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bc52bd96-ca70-4ff6-9a09-4fe5103c6041\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322668,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 38302,\n",
      "        \"completion_tokens\": 63,\n",
      "        \"total_tokens\": 38365,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn23627957', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23627957', 'dataset_id': 'syn23627957'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn23627957, synapse.org, https://www.synapse.org/#!Synapse:syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23627957 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23627957 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23627957 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23627957 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn23627957 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23627957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 94th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5752539&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b167db90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 10 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#abstract1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 26) for abstract1: Abstract [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2_2_4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2_2_4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2_2_4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2_2_4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2_2_4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2_2_4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 15 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 2896) for SD1: As illustrated by two key miRNA-target interactions, miR-155: ( -value=0.001, log-rank test, Supplementary Figure 1A [href=this file] ) and miR-21: ( -value=0.006, Supplementary Figure 1B [href=this file] ), it seems that key regulatory pairs are responsible for clinical prognosis. In KIRC, the key miRNA-target interactions, such as miR-29a: and miR-17: , are highly predictive of clinical outcome ( -value<0.05, Supplementary Figure 1C [href=this file] and Supplementary Figure 1D [href=this file] ).\n",
      "While in KIRC, key targets of miRNAs in the low-risk group were mostly enriched in ‘cell death’, ‘cell proliferation’ and ‘vasculogenesis’, those in the high-risk group were mostly enriched in ‘cell adhesion’, ‘cell motion’ and ‘epithelial cell migration’ ( Supplementary Figure 2A [href=this file] ), which is consistent with that KIRC is a typical metabolic disease [ 32 [href=#R32] ].\n",
      "Key targets of miR-204 in KIRC were specifically enriched in ‘apoptosis’ and ‘programmed cell death’ ( Supplementary Figure 2B [href=this file] ), consistent with inhibition of renal clear cell carcinoma tumor growth [ 38 [href=#R38] ].\n",
      "Analogously, in KIRC, the top three hallmarks of cancer are ‘sustained angiogenesis’, ‘tissue invasion and metastasis’ and ‘insensitivity to antigrowth signal’ ( Supplementary Figure 2C [href=this file] ). It is of interest to note that a hallmark of metastasis ‘tissue invasion and metastasis’ is significantly influenced by prognosis-related key miRNA-target interactions in ten distinct types of cancer (Figure 3D [href=#F3] , Supplementary Figure 3 [href=this file] ).\n",
      "It may suggest the important role of key miRNAs in cancer metastasis, since similar results were also observed in KIRC ( Supplementary Figure 2D [href=this file] ).\n",
      "The results showed significant difference between two groups in terms of PFS ( -value=0.016, log-rank test, Supplementary Figure 4A [href=this file] ) and KPS ( -value=0.024, Fisher’s exact test, Supplementary Figure 4B [href=this file] ). The percentage of GBM patients with (M1) or without (M0) distant metastases showed extremely close to significance ( -value=0.077, Fisher’s exact test, Supplementary Figure 4C [href=this file] ).\n",
      "Most cancer types (11 out of 16; 68.8%) showed that key miRNA-target interactions remained an independent prognostic signature for survival ( Supplementary Figure 4D [href=this file] ).\n",
      "We found that in most TCGA cancer types (11 out of 16; 68.8%) key miRNA-target interactions did not provide prognostic value in more than one other cancer type ( Supplementary Figure 5 [href=this file] ).\n",
      "Interestingly, we found that the combination of key miRNA-target interactions identified in malignant adenocarcinomas including LUAD, STAD and COAD could be an independent predictor for overall survival of another adenocarcinoma PAAD patients, individually ( Supplementary Figure 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 309) for SD2: However, we noted that miRNAs tended to regulate distinct key target genes in different cancer types, revealing highly cancer-specific miRNA-target regulation (Figure 5A [href=#F5] , Supplementary Table 1 [href=this file] , Supplementary Table 2 [href=this file] and Supplementary Table 3 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 154004\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 39253 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-2127cd5516629f8f7a3fcf6d3303a051918928a6d7b275a615d9c8ae26255c6e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 39253 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1a97bacd-5c51-46a8-877e-670e0d987487\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE29623\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29623\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE63805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63805\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE63459\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63459\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1741407\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE29623\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29623\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE63805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63805\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE63459\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63459\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1741407\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322684,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 54184,\n",
      "        \"completion_tokens\": 224,\n",
      "        \"total_tokens\": 54408,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE29623', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29623', 'dataset_id': 'GSE29623'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE29623, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE29623', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE63805', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63805', 'dataset_id': 'GSE63805'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE63805, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE63805', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE63459', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63459', 'dataset_id': 'GSE63459'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE63459, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE63459', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1741407', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1741407'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1741407, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1741407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1741407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE29623', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE63805', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE63459', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1741407', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE29623 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE29623 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE29623 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29623\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE63805 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE63805 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE63805 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE63459 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE63459 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE63459 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63459\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1741407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1741407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1741407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1741407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1741407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1741407 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1741407 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn1741407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1741407\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 95th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6602064&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b1c058c0>, <Element supplementary-material at 0x1b70b3ec0>, <Element supplementary-material at 0x1b10ed500>, <Element supplementary-material at 0x1b10ef080>, <Element supplementary-material at 0x1b10ece40>, <Element supplementary-material at 0x1b10ed900>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b1c058c0>, <Element supplementary-material at 0x1b70b3ec0>, <Element supplementary-material at 0x1b10ed500>, <Element supplementary-material at 0x1b10ef080>, <Element supplementary-material at 0x1b10ece40>, <Element supplementary-material at 0x1b10ed900>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: All three F1 crosses share key features of the disease , including decreased survival ( Fig. 1b [rid=F1] ) and body weight ( Supplementary Fig. 1a [this file] – b [this file] ), progressive accumulation of hyperphosphorylated tau in the cortex and hippocampus (AT8 staining; Fig. 1c [rid=F1] ), and prominent astrogliosis and microgliosis ( Fig. 1d [rid=F1] – f [rid=F1] ). Consistent with previous reports that FVB mice can be more vulnerable to neurodegeneration, we observed that F1 Tau transgenics on the FVB/C57 (TPR50xFVB/NJ) background had decreased survival and increased pathological tau as compared to the C57/C57 (TPR50xC57BL/6J) and DBA/C57 (TPR50xDBA/2J) progeny ( Fig. 1b [rid=F1] ) ( Supplementary Fig. 1c [this file] – h [this file] ). Importantly, no significant decrease in the weight of any of the affected brain areas was detected in transgenic animals at six months ( Supplementary Fig. 1i [this file] ), consistent with previous data showing no cell loss at this stage . \n",
      "The differentially expressed genes from transgenic and wild type mice clearly separate tau mutant from wild type genotypes within affected brain regions ( Supplementary Fig. 2a [this file] – c [this file] ), but not in the cerebellum ( Supplementary Fig. 2a [this file] ), and are consistent are across strains ( Supplementary Fig. 2d [this file] , e [this file] ). There was significant overlap in differential expression ( Online Methods [rid=S19] ; Supplementary Fig. 3a [this file] ) between the cortex and hippocampus, and significant correlation between fold changes in differentially expressed genes (Rank-rank hypergeometric test; p<0.05) from these two regions (R =0.76, slope=0.92, Supplementary Fig. 3a [this file] ). We find lesser, but significant correlation in gene expression between the cortex and brain stem (R =0.33, slope=0.82, Supplementary Fig. 3a [this file] ). Consistent with the absence of significant cell loss at six months in this model , positive regulators of apoptosis (GO ID:0043065) do not show significant changes across all 3 strains ( Supplementary Fig. 3b [this file] – e [this file] ) and negative regulators of apoptosis (GO ID :0043066) show modest, more significant changes ( Supplementary Fig. 3f [this file] – i [this file] ).We note that although we do not observe frank apoptosis, we do see evidence for initial activation of cellular pathways associated with inflammation in cerebral cortex by six months, including increases in GFAP and Iba-1 immunoreactivity in cortex ( Fig. 1d [rid=F1] – f [rid=F1] ). \n",
      "We observed no significant overlap in gene expression between the cortex and cerebellum ( Supplementary Fig. 3j [this file] – k [this file] ) and no correlation between differentially expressed genes in cortex and cerebellum (R =0.01, slope=0.14, Supplementary Fig. 3a [this file] ). Even more striking is that the expression changes of the top upregulated genes in the cerebellum are reversed in the cortex ( Supplementary Fig. 3l [this file] ), implying that protective changes may potentially occur in the cerebellum, as has been previously suggested . \n",
      "To place gene expression changes in a systems level framework, we performed consensus Weighted Gene Co-expression Network Analysis (cWGCNA ) across all three heterozygous F1 mouse strains to identify relationships not observed in only a single background ( Supplementary Fig. 4a [this file] , Supplementary Table 1 [rid=SD3] ). Cell-type enrichment analysis demonstrated that the salmon module is enriched in neuronal markers, the magenta module for oligodendrocyte markers, and the turquoise module for microglial, astrocyte and endothelial markers ( Fig. 2b [rid=F2] ) across multiple different cell type specific gene expression datasets ( Supplementary Fig. 4b [this file] ). Gene ontology (GO) analyses is consistent with the cell type enrichments, showing that the salmon module is enriched for synaptic pathways, magenta for DNA repair and transcription, while the turquoise module is enriched for immune and inflammatory categories ( Supplementary Fig. 4c [this file] – d [this file] ). \n",
      "Genes correlated highly with phospho-tau levels (R >0.6; Online Methods [rid=S19] ) were found to be significantly enriched in the salmon (42-fold) and turquoise modules (9.5-fold), but not in the magenta and pink modules ( Supplementary Fig. 4e [this file] ). Given the consistent cell type enrichments and strong gene ontology enrichments ( Fig. 2b [rid=F2] , d-g; and Supplementary Fig 4b [this file] – d [this file] ) we relabel the salmon module, the Neurodegeneration-Associated Synaptic (NAS) module, and the turquoise module, the Neurodegeneration-Associated Inflammatory (NAI) module, so as to provide more intuitive names for these core disease-associated modules. \n",
      "Moreover, both the NAI and NAS module eigengenes show changes across affected brain regions (cortex, Fig. 2e [rid=F2] , h [rid=F2] ; hippocampus, Supplementary Fig. 4f [this file] – g [this file] ; brain stem, Supplementary Fig. 4h [this file] – i [this file] ) but not cerebellum ( Fig. 2i [rid=F2] , j [rid=F2] ), prior to decreases in brain weight that would be indicative of cell loss ( Supplementary Fig. 1i [this file] ). This is further supported by analysis of expression profiles from sorted neurons and glia from the Tg4510 and PS2APP AD model which show cell intrinsic changes in the NAI and NAS module trajectories ( Supplementary Fig. 5a [this file] – d [this file] ). The NAI module is highly preserved in both microglia and astrocytes and significantly upregulated in these populations in transgenics ( Supplementary Fig. 5a [this file] , c [this file] , d [this file] – l [this file] ), while the neuronal NAS module is highly preserved in sorted neurons and downregulated in transgenics ( Supplementary Fig. 5b [this file] , e [this file] – l [this file] ). \n",
      "Functional annotation of the NAS module with GO and KEGG terms, confirms its broader relationship to synaptic function ( Fig. 2d [rid=F2] and Supplementary Fig. 4c [this file] ). In contrast, functional annotation of the NAI module highlights pathways associated with astrogliosis and inflammatory changes , including T cell activation and Toll-like receptor signaling pathways ( Fig. 2g [rid=F2] and Supplementary Fig. 4d [this file] ). \n",
      "Further analysis of 4 independent datasets from mouse models harboring pathological tau mutations including a replication dataset that we generated using the Tg4510 tau model (Methods, Supplementary Fig 5f [this file] ) confirm the preservation of the NAI and NAS modules ( Supplementary Fig 5e [this file] ), and the expected changes in gene expression early in the disease course ( Supplementary Fig. 5f [this file] - h [this file] , l [this file] ). The NAS and NAI models are highly preserved and their disease associations are maintained and both the NAI and NAS module expression changes occur with similar temporal profiles across each of these models ( Supplementary Fig. 5e [this file] , i [this file] – l [this file] ). \n",
      "The NAS and NAI modules are dysregulated in patient cortex, but not in cerebellum, both in the previously published microarray-based data and in our newly generated RNA-seq dataset ( Fig. 3a [rid=F3] – d [rid=F3] and Supplementary Fig. 5e [this file] , m [this file] – n [this file] ) consisting of both tau-positive and tau-negative FTD ( Fig. 3c [rid=F3] ). Moreover, the NAS module and its disease-trait relationship are preserved in iPSC-derived neurons from human FTD patients carrying GRN mutations ( Supplementary Fig. 5e [this file] , o [this file] ). \n",
      "We also performed mass-spectrometry-based, label-free, quantitative proteomics in an independent set of GRN-positive and GRN-negative FTD frontal cortex ( Supplementary Fig. 5p [this file] , Supplementary Table 2c [rid=SD4] – d [rid=SD4] , Online Methods [rid=S19] ). \n",
      "Examination of RNA sequencing and proteomic post mortem brain datasets from AD patients showed preservation of both the NAI and the NAS modules at the mRNA ( Supplementary Fig. 5e [this file] , Fig. 3g [rid=F3] – h [rid=F3] , Supplementary Table 4a [rid=SD6] ), and at the protein level ( Fig. 3i [rid=F3] ). \n",
      "We identified two miRNA modules associated with the transgenic condition (Bonferroni corrected P < 0.05, Fig. 4a [rid=F4] , Supplementary Table 3 [rid=SD5] ): the miM12 module that is anti-correlated, and the miM16 module ( Fig. 4b [rid=F4] ), which is positively correlated with the transgenic condition in affected regions, but not cerebellum ( Fig. 4c [rid=F4] and Supplementary Fig. 6a [this file] – e [this file] ). The miM16 module eigengene is strongly anti-correlated with the NAS module eigengene ( Fig. 4d [rid=F4] and Supplementary Fig. 6f [this file] ), and predicted targets of miR-203, an miM16 hub, are highly enriched (see Online Methods [rid=S19] ) in the NAS module ( Fig. 4e [rid=F4] and Supplementary Fig. 6g [this file] , h [this file] ). Consistent with its potential regulatory role, miR-203 is robustly upregulated in transgenic mice at six months in disease-affected areas ( Fig. 4f [rid=F4] and Supplementary Fig. 6i [this file] – k [this file] ), and in the frontal cortex in both tau-positive and tau-negative FTD ( Fig. 4g [rid=F4] , h [rid=F4] ). \n",
      "To test the predicted causal relationship between miR-203 expression and NAS module down-regulation, we acutely over-expressed miR-203 ( Fig. 5a [rid=F5] ) in primary cortical mouse neuronal cultures ( Online Methods [rid=S19] , Supplementary Fig. 7a [this file] ), observing downregulation of its predicted targets ( Fig. 5b [rid=F5] ) and the NAS module more broadly ( Fig. 5c [rid=F5] , Supplementary Fig. 7b [this file] , Supplementary Table 4a [rid=SD6] ). We further experimentally validated several predicted direct targets of miR-203 ( , , , ) by luciferase reporter assays ( Fig. 5f [rid=F5] ) and western blot (BCL2L2 and VSNL1l; Fig. 5g [rid=F5] , see Supplementary Fig 9 [this file] for uncropped blots). \n",
      "To control for potential confounding by supra-physiological over-expression in vitro, we moderately overexpressed miR-203 (approximately 2-fold; Online Methods [rid=S19] ) or a scrambled miRNA control (sc-miRNA) ( Fig. 5h [rid=F5] ) targeting neurons in wild-type mice in vivo ( Supplementary Fig. 7c [this file] – f [this file] ). The NAS module is highly preserved ( Supplementary Fig. 7g [this file] , Supplementary Table 4a [rid=SD6] ) and down-regulated in neurons overexpressing miR-203 after six weeks ( Fig. 5i [rid=F5] ). Strikingly, the level of miR-203 expression correlates remarkably well with downregulation of both miR-203 predicted targets (R =0.81, slope=−2.3; Supplementary Fig. 7h [this file] ) and NAS module genes (R =0.81, slope=−2.2; Supplementary Fig. 7i [this file] ). Furthermore, apoptotic pathways are altered in miR-203-overexpressing neurons (, Figure 5j [rid=F5] , Supplementary Table 4b [rid=SD6] ), confirmed by Casp8 protein expression ( Figure 5k [rid=F5] , Supplementary Fig. 7j [this file] ). Additionally, miR-203 over-expression in one-month-old Tg4510 Tau transgenic cortex ( Supplementary Fig. 8a [this file] – b [this file] ) causes downregulation of predicted miR-203 targets ( Fig. 5l [rid=F5] ), the NAS module ( Fig. 5m [rid=F5] ), a significant increase in CASP8 protein ( Fig. 5n [rid=F5] ) and apoptotic pathways ( Supplementary Fig. 8c [this file] – d [this file] ) and genes involved in calcium signaling and neuroactive ligand receptors ( Supplementary Fig. 8e [this file] – f [this file] ). \n",
      "Both short term (6-weeks, Supplementary Fig. 8g [this file] – h [this file] ) and longer term (4-months, Supplementary Fig. 8i [this file] – j [this file] ) inhibition of miR-203 oppose the pathologic transcriptional changes in predicted miR-203 targets ( Supplementary Fig. 8k [this file] , Fig. 5o [rid=F5] ) and the NAS module broadly ( Supplementary Fig. 8l [this file] , Fig. 5p [rid=F5] ) including genes involved in calcium signaling and neuroactive ligand receptor pathways ( Supplementary Fig. 8m [this file] – n [this file] ), and apoptosis ( Fig. 5q [rid=F5] , Supplementary Fig. 8o [this file] – p [this file] ). Reducing miR-203 did not affect phospho-tau immunostaining ( Supplementary Fig. 8q [this file] – r [this file] ), consistent with its acting downstream of tau pathology. C-fos intensity was also significantly increased in mutant neurons expressing the miR-203 TuD construct, as compared to those expressing the control construct ( Fig. 5q [rid=F5] , Supplementary Fig. 8s [this file] , p-value <0.0001), suggesting functional rescue. \n",
      "It is noteworthy that modules identified in the initial individual F1 crosses that are not conserved and dysregulated across all three strains, are also not conserved in other mouse models, or human brain ( Supplementary Fig 10a [this file] – c [this file] ), supporting the approach that we have taken. \n",
      "As we show, long-term inhibition of miR-203 in the Tg4510 model, while upregulating anti-apoptotic pathways, downregulating pro-apoptotic pathways and increasing neuronal activity as measured by c-fos activation, does not reduce the phospho-tau burden ( Supplementary Fig. 8q [this file] – r [this file] ). \n",
      "Summary heatmap of module preservation of all datasets described in Supplementary Fig. 5e [this file] using module definitions from strain-aware network analysis. \n",
      "For a full list of enrichments refer to Supplementary Fig. 6g [this file] . \n",
      "Uncropped blots are shown in Supplementary Fig. 9 [this file] . \n",
      "Scriptaid treatment does not alter miR-203 overexpression levels ( Supplementary Fig. 9 [this file] , indicating that the decrease in cell death is not a result of changes in viral infection, miRNA processing or regulation of miR-203 by scriptaid. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: To place gene expression changes in a systems level framework, we performed consensus Weighted Gene Co-expression Network Analysis (cWGCNA ) across all three heterozygous F1 mouse strains to identify relationships not observed in only a single background ( Supplementary Fig. 4a [rid=SD2] , Supplementary Table 1 [this file] ). \n",
      "Several AD candidate genes mapped to the NAI module including ( Online Methods [rid=S19] ; Fig. 3p [rid=F3] , Supplementary Table 1i [this file] ); In contrast to AD, FTD and PSP GWAS data both showed significant enrichment in the NAS module, which contains risk genes such as , and ( Fig. 3p [rid=F3] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: Considering this cross-mouse model conservation, we next assessed module preservation in human disease models and post mortem tissue samples to validate their human relevance (See Online Methods [rid=S19] , Supplementary Table 2a [this file] for list and characteristics of all human datasets, Supplementary Table 2b [this file] – c [this file] ). \n",
      "We also performed mass-spectrometry-based, label-free, quantitative proteomics in an independent set of GRN-positive and GRN-negative FTD frontal cortex ( Supplementary Fig. 5p [rid=SD2] , Supplementary Table 2c [this file] – d [this file] , Online Methods [rid=S19] ). \n",
      "Brain samples and individual level metadata is available in Supplementary Table 2b [this file] . Proteomics samples were obtained from Penn ADRC as part of the Accelerating Medicine Partnership for Alzheimer’s Disease (AMP-AD) and sample level metadata is available in Supplementary Table 2c [this file] . \n",
      "Several datasets have been used throughout the study, details of which can be found in Supplemental Table 2a [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD5: We identified two miRNA modules associated with the transgenic condition (Bonferroni corrected P < 0.05, Fig. 4a [rid=F4] , Supplementary Table 3 [this file] ): the miM12 module that is anti-correlated, and the miM16 module ( Fig. 4b [rid=F4] ), which is positively correlated with the transgenic condition in affected regions, but not cerebellum ( Fig. 4c [rid=F4] and Supplementary Fig. 6a [rid=SD2] – e [rid=SD2] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD6: Examination of RNA sequencing and proteomic post mortem brain datasets from AD patients showed preservation of both the NAI and the NAS modules at the mRNA ( Supplementary Fig. 5e [rid=SD2] , Fig. 3g [rid=F3] – h [rid=F3] , Supplementary Table 4a [this file] ), and at the protein level ( Fig. 3i [rid=F3] ). \n",
      "To test the predicted causal relationship between miR-203 expression and NAS module down-regulation, we acutely over-expressed miR-203 ( Fig. 5a [rid=F5] ) in primary cortical mouse neuronal cultures ( Online Methods [rid=S19] , Supplementary Fig. 7a [rid=SD2] ), observing downregulation of its predicted targets ( Fig. 5b [rid=F5] ) and the NAS module more broadly ( Fig. 5c [rid=F5] , Supplementary Fig. 7b [rid=SD2] , Supplementary Table 4a [this file] ). \n",
      "The NAS module is highly preserved ( Supplementary Fig. 7g [rid=SD2] , Supplementary Table 4a [this file] ) and down-regulated in neurons overexpressing miR-203 after six weeks ( Fig. 5i [rid=F5] ). Furthermore, apoptotic pathways are altered in miR-203-overexpressing neurons (, Figure 5j [rid=F5] , Supplementary Table 4b [this file] ), confirmed by Casp8 protein expression ( Figure 5k [rid=F5] , Supplementary Fig. 7j [rid=SD2] ). \n",
      "Four of the top 10 hits were histone deacetylase inhibitors (HDACs, Supplementary Table 4c [this file] ), which was significant (permutated p< 10 ). \n",
      "NAI module enrichment for AD risk genes was still significant after omitting APOE from the analysis ( Supplementary Table 4d [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 221742\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 56188 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-62c0810b8f567e4c476931e66846f1c7a1e0f1d24b7e329c4d6b316d677be4e3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 56188 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-cf1fc31c-712e-45c8-9b00-1340cfcc1fb3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE90696\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7818788\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7818788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9884357\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9884357\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE90696\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7818788\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7818788\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn9884357\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn9884357\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322686,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 67711,\n",
      "        \"completion_tokens\": 174,\n",
      "        \"total_tokens\": 67885,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE90696', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696', 'dataset_id': 'GSE90696'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: GSE90696, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn7818788', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7818788', 'dataset_id': 'syn7818788'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn7818788, synapse.org, https://www.synapse.org/#!Synapse:syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn7818788', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn9884357', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9884357', 'dataset_id': 'syn9884357'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn9884357, synapse.org, https://www.synapse.org/#!Synapse:syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn9884357', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn7818788', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9884357', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: geo, identifier: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for GSE90696 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7818788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7818788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7818788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7818788 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn7818788 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7818788\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9884357 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9884357 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9884357 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9884357 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn9884357 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9884357\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 96th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7032692&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6db1090>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec003: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec003: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#notes1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 35) for notes1: Data Availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 92962\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 23993 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-be7240dc221387fd8744821f2f96e05d70b640a8529f5fadeb236d6e5e733cc8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 23993 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-207b1750-22aa-4fc4-85d3-0900092cb9b6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1710250\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Portal.html#!Synapse:syn1710250/wiki/27348\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"EGAC00001000484\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1710250\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Portal.html#!Synapse:syn1710250/wiki/27348\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n  },\\n    {\\n    \\\"dataset_identifier\\\": \\\"EGAC00001000484\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322704,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 33196,\n",
      "        \"completion_tokens\": 203,\n",
      "        \"total_tokens\": 33399,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1710250', 'repository_reference': 'https://www.synapse.org/Portal.html#!Synapse:syn1710250/wiki/27348', 'dataset_id': 'syn1710250'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1710250, synapse.org, https://www.synapse.org/Portal.html#!Synapse:syn1710250/wiki/27348\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1710250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00000000083', 'repository_reference': 'https://www.ebi.ac.uk/ega/studies/EGAS00000000083', 'dataset_id': 'EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EGAS00000000083 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ega/studies/EGAS00000000083, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00000000083 for https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00000000083 for https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00000000083 for https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00000000083 for https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00000000083 for https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAC00001000484', 'repository_reference': 'https://www.ebi.ac.uk/ega/dacs/EGAC00001000484', 'dataset_id': 'EGAC00001000484'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID EGAC00001000484 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ega/dacs/EGAC00001000484, identifier: EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAC00001000484 for https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAC00001000484 for https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAC00001000484 for https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAC00001000484 for https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAC00001000484 for https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ega/dacs/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'EGAC00001000484', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1710250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EGAC00001000484', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1710250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1710250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1710250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1710250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1710250 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1710250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EGAS00000000083 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: www.ebi.ac.uk and accession_id: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for EGAC00001000484 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAC00001000484\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 97th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10061294&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b71810c0>, <Element supplementary-material at 0x1b7180a00>, <Element supplementary-material at 0x1b71827c0>, <Element supplementary-material at 0x1b7182640>, <Element supplementary-material at 0x1b7180900>, <Element supplementary-material at 0x1b7181dc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b71810c0>, <Element supplementary-material at 0x1b7180a00>, <Element supplementary-material at 0x1b71827c0>, <Element supplementary-material at 0x1b7182640>, <Element supplementary-material at 0x1b7180900>, <Element supplementary-material at 0x1b7181dc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0001: S1 [this file] for a visual explanation of the Bayesian rank‐based clustering method. \n",
      "S2 [this file] A), and we observed almost perfect overlap between the clustering results obtained when using the SFEs‐associated gene set and the random gene set (see Fig. S2 [this file] B for the clustering of samples based on the random gene set, and Fig. S2 [this file] C for the concordance table). S3 [this file] A). S3 [this file] B), and the clustering was also robust when using the random gene set (Fig. S3 [this file] C). \n",
      "S4 [this file] for a plot of the cluster‐specific densities of the counts of the top‐ranked genes co‐occurrences). \n",
      "S5 [this file] ). of the top‐ranked genes of the RankClusters identified using the random gene set selection provided few or no significantly enriched overlaps with known gene sets. \n",
      "S6 [this file] A). S6 [this file] B), and this may partly explain the separation of these samples from the other pan‐squamous samples in RankClusters 5 and 6. S6 [this file] B); for the BRCA samples, there was no statistically significant difference between RankCluster 7 and 8, while there was a small difference between RankCluster 7/8 and RankCluster 11 ( ‐values 2.1e‐2 and 4.3e‐4, respectively; Fig. S6 [this file] B). \n",
      "S2 [this file] ), which illustrates that the clusters are robust and can be reproduced by many different gene sets without prior biological knowledge [ 36 [rid=mol213354-bib-0036] ]. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0002: To assess the robustness of the Bayes Mallows method and the importance of the choice of gene set, we also performed the Bayes Mallows clustering on a random selection of 1247 genes (see Table S1 [this file] ), and compared the distributions of samples across clusters. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0003: Specifically, each RankCluster can be characterized by the group of genes showing the largest probability of being top‐ranked: In Fig. 4 [rid=mol213354-fig-0004] , for example, the genes are ranked according to the probability of being among the top 100 in the cluster consensus ranking, for each of the 16 RankClusters (see Tables S2 [this file] and S3 [rid=mol213354-supitem-0004] for the rankings and probabilities of genes being top 10 for each RankCluster, and Tables S4 [rid=mol213354-supitem-0005] and S5 [rid=mol213354-supitem-0006] for the ranking and probabilities of genes being top 100 for each RankCluster). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0004: Specifically, each RankCluster can be characterized by the group of genes showing the largest probability of being top‐ranked: In Fig. 4 [rid=mol213354-fig-0004] , for example, the genes are ranked according to the probability of being among the top 100 in the cluster consensus ranking, for each of the 16 RankClusters (see Tables S2 [rid=mol213354-supitem-0003] and S3 [this file] for the rankings and probabilities of genes being top 10 for each RankCluster, and Tables S4 [rid=mol213354-supitem-0005] and S5 [rid=mol213354-supitem-0006] for the ranking and probabilities of genes being top 100 for each RankCluster). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0005: Specifically, each RankCluster can be characterized by the group of genes showing the largest probability of being top‐ranked: In Fig. 4 [rid=mol213354-fig-0004] , for example, the genes are ranked according to the probability of being among the top 100 in the cluster consensus ranking, for each of the 16 RankClusters (see Tables S2 [rid=mol213354-supitem-0003] and S3 [rid=mol213354-supitem-0004] for the rankings and probabilities of genes being top 10 for each RankCluster, and Tables S4 [this file] and S5 [rid=mol213354-supitem-0006] for the ranking and probabilities of genes being top 100 for each RankCluster). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mol213354-supitem-0006: Specifically, each RankCluster can be characterized by the group of genes showing the largest probability of being top‐ranked: In Fig. 4 [rid=mol213354-fig-0004] , for example, the genes are ranked according to the probability of being among the top 100 in the cluster consensus ranking, for each of the 16 RankClusters (see Tables S2 [rid=mol213354-supitem-0003] and S3 [rid=mol213354-supitem-0004] for the rankings and probabilities of genes being top 10 for each RankCluster, and Tables S4 [rid=mol213354-supitem-0005] and S5 [this file] for the ranking and probabilities of genes being top 100 for each RankCluster). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 158778\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 40447 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-9ce0490355f6f67468f51f0eaa5caf8ecaead467b3c81c7a9ee4d37ab1293048\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 40447 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9f1eb466-3dc2-4440-ae44-bb6fe9a2fe68\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1715755\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn2468297\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1715755\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn2468297\\\",\\n    \\\"repository_reference\\\": \\\"doi.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322707,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 50327,\n",
      "        \"completion_tokens\": 87,\n",
      "        \"total_tokens\": 50414,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1715755', 'repository_reference': 'synapse.org', 'dataset_id': 'syn1715755'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn1715755, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn1715755\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn1715755', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn2468297', 'repository_reference': 'doi.org', 'dataset_id': '10.7303/syn2468297'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.7303/syn2468297, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.7303/syn2468297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn2468297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1715755', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.7303/syn2468297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn1715755\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1715755\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1715755 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1715755 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1715755 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1715755 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn1715755 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1715755\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1715755\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.7303/syn2468297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn2468297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn2468297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn2468297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.7303/syn2468297 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.7303/syn2468297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.7303/syn2468297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 98th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8006362&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 396 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 420 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b730be40>, <Element supplementary-material at 0x1b73093c0>, <Element supplementary-material at 0x1b7309540>, <Element supplementary-material at 0x1b7309900>, <Element supplementary-material at 0x1b73098c0>, <Element supplementary-material at 0x1b730a440>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b730be40>, <Element supplementary-material at 0x1b73093c0>, <Element supplementary-material at 0x1b7309540>, <Element supplementary-material at 0x1b7309900>, <Element supplementary-material at 0x1b73098c0>, <Element supplementary-material at 0x1b730a440>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: We mixed genomic DNAs derived from transformed lymphoblastoid cell lines of four unrelated individuals at different proportions (Fig. 1 [rid=Fig1] a and Additional file 1 [this file] : Fig. Combining the Mix 1 and Mix 2 datasets to double sequencing coverage only marginally improved sensitivity (Fig. 1 [rid=Fig1] b and Additional file 1 [this file] : Fig. Moreover, MuTect2 and Strelka2, which are designed to detect somatic SNVs that are present in tumors but not matched normal samples, lacked the sensitivity to detect simulated mosaic SNVs shared between two matched samples (Fig. 1 [rid=Fig1] b and Additional file 1 [this file] : Fig. \n",
      "The resultant reads were uniformly processed using a common workflow (see “ Methods [rid=Sec13] ”) and mapped to the human genome (version GRCh37d5) (Additional file 1 [this file] : Fig. \n",
      "For example, none of the putative SNV calls were identified by each of the six working groups and only three calls were identified by five of six groups (Fig. 2 [rid=Fig2] b and Additional file 1 [this file] : Table S1). \n",
      "We subjected 400 putative mosaic SNVs to validation experiments (Tables S 1 [this file] & S 2 [rid=MOESM2] ). \n",
      "Finally, to verify the validation status, we performed droplet digital PCR (ddPCR) experiments for thirteen calls (Additional file 1 [this file] : Table S3). \n",
      "Moreover, consistent with previous data [ 17 [rid=CR17] ], the SNV mutation spectrum was dominated by C>T transitions (Additional file 1 [this file] : Fig. \n",
      "Finally, 14/29 putative mosaic SNVs were located within regions of the genome containing germline copy number gains, leading to < 0.5 VAFs (Additional file 1 [this file] : Fig. \n",
      "By comparison, bona fide validated mosaic SNVs were detected as alternative reads on a single haplotype, giving the appearance of three haplotype states (i.e., the two parental haplotypes plus a “third” haplotype containing the somatic SNV on a single parental haplotype; Additional file 1 [this file] : Fig. \n",
      "A systematic misalignment of these germline variants (Additional file 1 [this file] : Fig. S7) and/or a systematic reduction in read counts near the germline variant (Additional file 1 [this file] : Fig. \n",
      "We found the input “discovery” call set has a significant impact on final sensitivity and determined that a GATK ploidy option of 50 for the 250× coverage WGS replicate 6 dataset, which corresponds to approximately five sequencing reads per haplotype, generally allowed the best approach to identify validated mosaic SNVs (Fig. 4 [rid=Fig4] d and Additional file 1 [this file] : Fig. For our best practices framework, we incorporated MosaicForecast as an additional step to obtain the highest accuracy call sets (Additional file 1 [this file] : Fig. \n",
      "One neuron (Additional file 1 [this file] : Fig. Two lineages (L1 and L2) were clearly separable by diagnostic mosaic SNVs in the eleven single cells (Fig. 4 [rid=Fig4] c and Additional file 1 [this file] : Fig. \n",
      "The eight best scoring SNV pairs consisted of a SNV (SNVs 1–4) from the L1 lineage and a SNV from L2 lineage (SNV10 and SNV11; Additional file 1 [this file] : Fig. Using a similar strategy, we also were able to identify anti-correlated mosaic SNV VAFs that defined L1 and L2 sub-lineages that likely arose later in development (Additional file 1 [this file] : Fig. \n",
      "In addition, and consistent with the reconstructed lineage tree (Fig. 4 [rid=Fig4] c and Additional file 1 [this file] : Fig. \n",
      "S 13 [this file] ). S 13 [this file] ); the third highest allele frequency did not correlate with the variant allele frequency (Fig. \n",
      "We initially designed 29 ddPCR assays to assess candidate mosaic SNVs present at < 3% VAFs and/or near know SVs (Additional file 1 [this file] : Table S3). \n",
      "S 8 [this file] ). \n",
      "While this model suffers with respect to sensitivity, the chosen values maximize the rejection of false-positive calls at the expense of removing some false-negative calls (Additional file 1 [this file] : Table S4). \n",
      "S 12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: We subjected 400 putative mosaic SNVs to validation experiments (Tables S 1 [rid=MOESM1] & S 2 [this file] ). \n",
      "S8) resulted in 75 false-positive calls (Additional file 2 [this file] : Table S2). \n",
      "Reconstructed cell lineage trees using a cohort of mosaic SNVs (Table S 2 [this file] ) present in eleven single-cell datasets from the common brain sample. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: The specificity of primers was confirmed by UCSC in silico PCR ( http://genome.ucsc.edu/cgi-bin/hgPcr (http://genome.ucsc.edu/cgi-bin/hgPcr) ) (Additional file 3 [this file] :Table S5). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: We performed ddPCR using the primers and probe listed in Additional file 4 [this file] : Table S6. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: The review history is available as Additional file 6 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 242 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 205416\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 52106 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b7c2757549ce641a20bf021f2afc0b624a82496d6b478081879f1c98db706167\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 52106 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c96bca9f-3f25-4c7f-b19b-5b4fd0d9b8a1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NDA Study ID 792\\\",\\n    \\\"repository_reference\\\": \\\"NIMH Data Archive\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.15154/1504248\\\",\\n    \\\"repository_reference\\\": \\\"NIMH Data Archive\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22024464\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22024464\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\\\",\\n    \\\"repository_reference\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/bsmn/bsmn-pipeline\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/bsmn/bsmn-pipeline\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.4321679\\\",\\n    \\\"repository_reference\\\": \\\"zenodo.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NDA Study ID 792\\\",\\n    \\\"repository_reference\\\": \\\"NIMH Data Archive\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.15154/1504248\\\",\\n    \\\"repository_reference\\\": \\\"NIMH Data Archive\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22024464\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22024464\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\\\",\\n    \\\"repository_reference\\\": \\\"ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/bsmn/bsmn-pipeline\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/bsmn/bsmn-pipeline\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.4321679\\\",\\n    \\\"repository_reference\\\": \\\"zenodo.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322710,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58912,\n",
      "        \"completion_tokens\": 336,\n",
      "        \"total_tokens\": 59248,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'NDA Study ID 792', 'repository_reference': 'NIMH Data Archive', 'dataset_id': 'NDA Study ID 792'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: NDA Study ID 792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1002 - INFO - Accession ID NDA Study ID 792 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: NIMH Data Archive, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: NIMH Data Archive\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'NIMH Data Archive', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.15154/1504248', 'repository_reference': 'NIMH Data Archive', 'dataset_id': '10.15154/1504248'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.15154/1504248, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.15154/1504248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.15154/1504248', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn22024464', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22024464', 'dataset_id': 'syn22024464'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 640 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 644 - INFO - Extracted ID: syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn22024464, synapse.org, https://www.synapse.org/#!Synapse:syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn22024464', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage', 'repository_reference': 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage', 'dataset_id': 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Accession ID ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage, identifier: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - ERROR - Error extracting domain from URL: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage', 'data_repository': 'Unknown_Publisher', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/bsmn/bsmn-pipeline', 'repository_reference': 'https://github.com/bsmn/bsmn-pipeline', 'dataset_id': 'https://github.com/bsmn/bsmn-pipeline'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/bsmn/bsmn-pipeline, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/bsmn/bsmn-pipeline is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/bsmn/bsmn-pipeline, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/bsmn/bsmn-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.4321679', 'repository_reference': 'zenodo.org', 'dataset_id': '10.5281/zenodo.4321679'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: 10.5281/zenodo.4321679, doi.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.4321679\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.4321679', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 255 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'NIMH Data Archive', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.15154/1504248', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn22024464', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage', 'data_repository': 'Unknown_Publisher', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.4321679', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 6 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.15154/1504248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.15154/1504248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.15154/1504248 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.15154/1504248 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.15154/1504248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.15154/1504248\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22024464 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22024464 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22024464 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22024464 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn22024464 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22024464\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: Unknown_Publisher, identifier: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage for Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: Unknown_Publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage in unknown_publisher\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: unknown_publisher and accession_id: ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000G_2504_high_coverage\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - WARNING - Repository unknown_publisher unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: 10.5281/zenodo.4321679\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.4321679\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.4321679 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for 10.5281/zenodo.4321679 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 6 with repo: doi.org and accession_id: 10.5281/zenodo.4321679\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://doi.org/10.5281/zenodo.4321679\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 6\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 264 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 542 - INFO - 99th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 348 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 355 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7778917&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 384 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b67b2690>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 400 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 414 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 442 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 15 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#abstract1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 26) for abstract1: Abstract [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC2-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC2-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SEC3-5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SEC3-5: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 96605\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 24903 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4f61077323241a652a0e9f044d86773cef8d5487db3028d74afb0f01f3f04dad\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1285 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 24903 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4844c5b5-50dd-4ce9-a539-10d464faed59\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://darkkinome.org\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase/darkkinasekb\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase/kinase_expression\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18360482\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://darkkinome.org\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase/darkkinasekb\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase/kinase_expression\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"https://github.com/IDG-Kinase\\\",\\n    \\\"repository_reference\\\": \\\"github.com\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18360482\\\",\\n    \\\"repository_reference\\\": \\\"synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1756322727,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 32638,\n",
      "        \"completion_tokens\": 201,\n",
      "        \"total_tokens\": 32839,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://darkkinome.org', 'repository_reference': 'n/a', 'dataset_id': 'https://darkkinome.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://darkkinome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://darkkinome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://darkkinome.org, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://darkkinome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://darkkinome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: darkkinome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://darkkinome.org is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 676 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 604 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'https://darkkinome.org', 'repository_reference': 'n/a', 'dataset_id': 'https://darkkinome.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/IDG-Kinase/darkkinasekb', 'repository_reference': 'github.com', 'dataset_id': 'https://github.com/IDG-Kinase/darkkinasekb'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/IDG-Kinase/darkkinasekb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/IDG-Kinase/darkkinasekb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/IDG-Kinase/darkkinasekb, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/IDG-Kinase/darkkinasekb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/IDG-Kinase/darkkinasekb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/IDG-Kinase/darkkinasekb is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/IDG-Kinase/kinase_expression', 'repository_reference': 'github.com', 'dataset_id': 'https://github.com/IDG-Kinase/kinase_expression'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/IDG-Kinase/kinase_expression\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/IDG-Kinase/kinase_expression\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/IDG-Kinase/kinase_expression, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/IDG-Kinase/kinase_expression\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/IDG-Kinase/kinase_expression\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/IDG-Kinase/kinase_expression is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'https://github.com/IDG-Kinase', 'repository_reference': 'github.com', 'dataset_id': 'https://github.com/IDG-Kinase'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 997 - INFO - Validating accession ID: https://github.com/IDG-Kinase\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 891 - INFO - Checking if link points to dataset webpage: https://github.com/IDG-Kinase\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: https://github.com/IDG-Kinase, identifier: None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 872 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/IDG-Kinase\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: https://github.com/IDG-Kinase\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 893 - INFO - Extracted domain: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1010 - INFO - Accession ID https://github.com/IDG-Kinase is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: github.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 950 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 962 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1146 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18360482', 'repository_reference': 'synapse.org', 'dataset_id': 'syn18360482'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 648 - INFO - Schema validation vals: syn18360482, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 653 - INFO - Dataset ID found via pattern matching: syn18360482\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 661 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 619 - INFO - Extracted dataset: {'dataset_identifier': 'syn18360482', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn18360482', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1167 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1183 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1175 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1186 - INFO - Raw accession ID: syn18360482\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1089 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18360482\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18360482 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18360482 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18360482 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1137 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18360482 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1118 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1071 - INFO - Resolving accession ID for syn18360482 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn18360482\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1226 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18360482\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1233 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1244 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 459 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 469 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1037 - INFO - Output written to file: output/result.csv\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1039 - INFO - File Download Schedule: []\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1052 - INFO - Quitting the WebDriver.\u001B[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1b1166f10>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/72d6691f-6bb2-4f73-8b46-992eb00093c6\n",
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1b1777650>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/72d6691f-6bb2-4f73-8b46-992eb00093c6\n",
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1b71eef10>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/72d6691f-6bb2-4f73-8b46-992eb00093c6\n"
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:27.547725Z",
     "start_time": "2025-08-27T19:25:27.512366Z"
    }
   },
   "cell_type": "code",
   "source": "combined_df",
   "id": "9fd1be22241ced6b",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "    dataset_identifier data_repository  \\\n",
       "0           syn9702085     synapse.org   \n",
       "1           syn8456637     synapse.org   \n",
       "2               M27188         nuccore   \n",
       "3          syn52293417     synapse.org   \n",
       "4           syn2580853     synapse.org   \n",
       "..                 ...             ...   \n",
       "927                NaN             NaN   \n",
       "928                NaN             NaN   \n",
       "929                n/a      github.com   \n",
       "930        syn18360482     synapse.org   \n",
       "931                NaN             NaN   \n",
       "\n",
       "                                      dataset_webpage  \\\n",
       "0     https://www.synapse.org/[#!]*Synapse:syn9702085   \n",
       "1     https://www.synapse.org/[#!]*Synapse:syn8456637   \n",
       "2         https://www.ncbi.nlm.nih.gov/nuccore/M27188   \n",
       "3    https://www.synapse.org/[#!]*Synapse:syn52293417   \n",
       "4     https://www.synapse.org/[#!]*Synapse:syn2580853   \n",
       "..                                                ...   \n",
       "927                                               NaN   \n",
       "928                                               NaN   \n",
       "929                                               n/a   \n",
       "930  https://www.synapse.org/[#!]*Synapse:syn18360482   \n",
       "931                                               NaN   \n",
       "\n",
       "                                   access_mode  \\\n",
       "0    Complex download or application to access   \n",
       "1    Complex download or application to access   \n",
       "2                                          NaN   \n",
       "3    Complex download or application to access   \n",
       "4    Complex download or application to access   \n",
       "..                                         ...   \n",
       "927                                        NaN   \n",
       "928                                        NaN   \n",
       "929                                        NaN   \n",
       "930  Complex download or application to access   \n",
       "931                                        NaN   \n",
       "\n",
       "                                    link  \\\n",
       "0                                    NaN   \n",
       "1                                    NaN   \n",
       "2                                    NaN   \n",
       "3                                    NaN   \n",
       "4                                    NaN   \n",
       "..                                   ...   \n",
       "927       13059_2021_2285_MOESM5_ESM.pdf   \n",
       "928      13059_2021_2285_MOESM6_ESM.docx   \n",
       "929                                  NaN   \n",
       "930                                  NaN   \n",
       "931  https://doi.org/10.1093/nar/gkaa853   \n",
       "\n",
       "                                            source_url  \\\n",
       "0    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "1    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "2    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "3    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "4    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "..                                                 ...   \n",
       "927   https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362   \n",
       "928   https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362   \n",
       "929   https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917   \n",
       "930   https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917   \n",
       "931   https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917   \n",
       "\n",
       "                                         download_link                title  \\\n",
       "0                                                  NaN                  NaN   \n",
       "1                                                  NaN                  NaN   \n",
       "2                                                  NaN                  NaN   \n",
       "3                                                  NaN                  NaN   \n",
       "4                                                  NaN                  NaN   \n",
       "..                                                 ...                  ...   \n",
       "927  https://pmc.ncbi.nlm.nih.gov/articles/instance...             No Title   \n",
       "928  https://pmc.ncbi.nlm.nih.gov/articles/instance...             No Title   \n",
       "929                                                NaN                  NaN   \n",
       "930                                                NaN                  NaN   \n",
       "931                                               None  10.1093/nar/gkaa853   \n",
       "\n",
       "    content_type      id  ...  \\\n",
       "0            NaN     NaN  ...   \n",
       "1            NaN     NaN  ...   \n",
       "2            NaN     NaN  ...   \n",
       "3            NaN     NaN  ...   \n",
       "4            NaN     NaN  ...   \n",
       "..           ...     ...  ...   \n",
       "927   local-data  MOESM5  ...   \n",
       "928   local-data  MOESM6  ...   \n",
       "929          NaN     NaN  ...   \n",
       "930          NaN     NaN  ...   \n",
       "931          NaN     NaN  ...   \n",
       "\n",
       "                                   context_description file_extension  \\\n",
       "0                                                  NaN           None   \n",
       "1                                                  NaN           None   \n",
       "2                                                  NaN           None   \n",
       "3                                                  NaN           None   \n",
       "4                                                  NaN           None   \n",
       "..                                                 ...            ...   \n",
       "927                                                               pdf   \n",
       "928  The review history is available as Additional ...           docx   \n",
       "929                                                NaN           None   \n",
       "930                                                NaN           None   \n",
       "931                                                              None   \n",
       "\n",
       "                                             pub_title file_info  \\\n",
       "0    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "1    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "2    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "3    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "4    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "..                                                 ...       ...   \n",
       "927  Comprehensive identification of somatic nucleo...       NaN   \n",
       "928  Comprehensive identification of somatic nucleo...       NaN   \n",
       "929  The Dark Kinase Knowledgebase: an online compe...       NaN   \n",
       "930  The Dark Kinase Knowledgebase: an online compe...       NaN   \n",
       "931  The Dark Kinase Knowledgebase: an online compe...       n/a   \n",
       "\n",
       "                                   section_class  \\\n",
       "0                                            NaN   \n",
       "1                                            NaN   \n",
       "2                                            NaN   \n",
       "3                                            NaN   \n",
       "4                                            NaN   \n",
       "..                                           ...   \n",
       "927                                          NaN   \n",
       "928                                          NaN   \n",
       "929                                          NaN   \n",
       "930                                          NaN   \n",
       "931  pmc-layout__citation font-secondary font-xs   \n",
       "\n",
       "                             a_attr_href                 a_attr_class  \\\n",
       "0                                    NaN                          NaN   \n",
       "1                                    NaN                          NaN   \n",
       "2                                    NaN                          NaN   \n",
       "3                                    NaN                          NaN   \n",
       "4                                    NaN                          NaN   \n",
       "..                                   ...                          ...   \n",
       "927                                  NaN                          NaN   \n",
       "928                                  NaN                          NaN   \n",
       "929                                  NaN                          NaN   \n",
       "930                                  NaN                          NaN   \n",
       "931  https://doi.org/10.1093/nar/gkaa853  usa-link usa-link--external   \n",
       "\n",
       "    a_attr_data-ga-action a_attr_target           a_attr_rel  \n",
       "0                     NaN           NaN                  NaN  \n",
       "1                     NaN           NaN                  NaN  \n",
       "2                     NaN           NaN                  NaN  \n",
       "3                     NaN           NaN                  NaN  \n",
       "4                     NaN           NaN                  NaN  \n",
       "..                    ...           ...                  ...  \n",
       "927                   NaN           NaN                  NaN  \n",
       "928                   NaN           NaN                  NaN  \n",
       "929                   NaN           NaN                  NaN  \n",
       "930                   NaN           NaN                  NaN  \n",
       "931      click_feat_suppl        _blank  noopener noreferrer  \n",
       "\n",
       "[932 rows x 24 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>source_url</th>\n",
       "      <th>download_link</th>\n",
       "      <th>title</th>\n",
       "      <th>content_type</th>\n",
       "      <th>id</th>\n",
       "      <th>...</th>\n",
       "      <th>context_description</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>section_class</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>syn9702085</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn9702085</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>syn8456637</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn8456637</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>M27188</td>\n",
       "      <td>nuccore</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/nuccore/M27188</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>syn52293417</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn52293417</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>syn2580853</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn2580853</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>927</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13059_2021_2285_MOESM5_ESM.pdf</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM5</td>\n",
       "      <td>...</td>\n",
       "      <td></td>\n",
       "      <td>pdf</td>\n",
       "      <td>Comprehensive identification of somatic nucleo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>928</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13059_2021_2285_MOESM6_ESM.docx</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM6</td>\n",
       "      <td>...</td>\n",
       "      <td>The review history is available as Additional ...</td>\n",
       "      <td>docx</td>\n",
       "      <td>Comprehensive identification of somatic nucleo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>929</th>\n",
       "      <td>n/a</td>\n",
       "      <td>github.com</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>The Dark Kinase Knowledgebase: an online compe...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>930</th>\n",
       "      <td>syn18360482</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn18360482</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>The Dark Kinase Knowledgebase: an online compe...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>931</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.1093/nar/gkaa853</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917</td>\n",
       "      <td>None</td>\n",
       "      <td>10.1093/nar/gkaa853</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td></td>\n",
       "      <td>None</td>\n",
       "      <td>The Dark Kinase Knowledgebase: an online compe...</td>\n",
       "      <td>n/a</td>\n",
       "      <td>pmc-layout__citation font-secondary font-xs</td>\n",
       "      <td>https://doi.org/10.1093/nar/gkaa853</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>932 rows × 24 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:28.453907Z",
     "start_time": "2025-08-27T19:25:27.552222Z"
    }
   },
   "cell_type": "code",
   "source": "ground_truth = pd.read_parquet(\"exp_input/dataset_citation_records_Table.parquet\")",
   "id": "305d7d5f299551c1",
   "outputs": [],
   "execution_count": 9
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:28.478971Z",
     "start_time": "2025-08-27T19:25:28.457362Z"
    }
   },
   "cell_type": "code",
   "source": "ground_truth.head()",
   "id": "dadaf8654a9d103d",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "  identifier repository                            citing_publication_link  \\\n",
       "0  PXD059466      PRIDE      https://dx.doi.org/10.1038/S41467-025-56720-1   \n",
       "1  PXD051312      PRIDE               https://dx.doi.org/10.6019/PXD051312   \n",
       "2  PXD051312      PRIDE          https://dx.doi.org/10.1002/prca.202400095   \n",
       "3  PXD051312      PRIDE  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "4  PXD054431      PRIDE        https://dx.doi.org/10.17159/SAJS.2025/18571   \n",
       "\n",
       "       citation_record_source  citation_record_from_doi  \\\n",
       "0  proteomexchange_search.tsv                         1   \n",
       "1  proteomexchange_search.tsv                         1   \n",
       "2  proteomexchange_search.tsv                         1   \n",
       "3  proteomexchange_search.tsv                         0   \n",
       "4  proteomexchange_search.tsv                         1   \n",
       "\n",
       "                          doi        pmcid  \n",
       "0  10.1038/S41467-025-56720-1               \n",
       "1           10.6019/PXD051312               \n",
       "2      10.1002/prca.202400095               \n",
       "3                              PMC11895760  \n",
       "4    10.17159/SAJS.2025/18571               "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>repository</th>\n",
       "      <th>citing_publication_link</th>\n",
       "      <th>citation_record_source</th>\n",
       "      <th>citation_record_from_doi</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PXD059466</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1038/S41467-025-56720-1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.6019/PXD051312</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.6019/PXD051312</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1002/prca.202400095</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1002/prca.202400095</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>0</td>\n",
       "      <td></td>\n",
       "      <td>PMC11895760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PXD054431</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.17159/SAJS.2025/18571</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.17159/SAJS.2025/18571</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:28.517753Z",
     "start_time": "2025-08-27T19:25:28.484030Z"
    }
   },
   "cell_type": "code",
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output = [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        pub_id = source_page.split('/')[-1].lower()\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend(FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ],
   "id": "583f880224783dc7",
   "outputs": [],
   "execution_count": 11
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:42.613365Z",
     "start_time": "2025-08-27T19:25:28.521340Z"
    }
   },
   "cell_type": "code",
   "source": "results = evaluate_performance(combined_df, ground_truth, dg, \"output/false_positives.txt\")",
   "id": "3ae367545bb1b42b",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn8456637', 'syn9702085', 'syn2580853', 'syn52293417'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn9702085', 'syn2580853', 'syn52293417', 'm27187', 'm27188', 'syn8456637'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1720047', 'syn2455683', 'syn312572'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1688309', 'syn1682511', 'syn1732077', 'syn1701558'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1732077'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'syn3219045', 'gse87544', 'gse65133', 'gse67835', 'gse115449'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1446295'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse224398', 'syn21126450'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'srr21079265', 'srr21079266', 'srr21079267', 'srr21079269', 'gse224398', 'syn21126450', 'srr21079270', 'srr21079268'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse81861', 'syn30243143', 'syn213309', 'gse76312', 'gse75688', 'doi.org/10.1242/bio.059256', 'gse81383', 'gse103322', 'gse70630'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.1111111111111111\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gcst007511', 'syn51364943', 'gcst009325', 'gcst90027158'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn11724057', 'syn3388564', 'syn18485175', 'syn4896408', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', 'syn11724057', 'syn3388564', 'syn18485175', 'gse291402', 'syn4896408', '10.5281/zenodo.14804357', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.625\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse27131', 'gse32707', 'gse40012', 'gse65682', 'gse54514', 'gse60244', 'gse20346', 'gse95233', 'gse40586', 'e-mtab-5273/5274', 'e-mexp-3589', '10.7303/syn5612563', 'gse13015', 'gse25504', 'e-mtab-1548', 'gse57065', 'gse21802', 'gse66099'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5612563, gt_id=syn5612563\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn51514106', 'syn51514106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn3157238', 'syn3157329', 'syn3157322', 'syn3205821', 'syn9705278'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'w81xwh-12-2-0012', 'n01-ag-12100', 'r01ag36836', '3u01ag024904-09s4', '10.7303/syn3157238', 'rc2ag036547', 'u01ag32984', '10.7303/syn3157322', 'r01ag48015', 'r01ag17917', 'u01ag61356', 'p30ag10161', '10.7303/syn3205821.6', 'u01ag46161', 'rf1ag57473', 'rf1ag051550', 'r01ag046171', '10.7303/syn5592519', '10.7303/syn9705278', 'r01ag36042', 'r01ag15819', 'r01ag30146', 'u01ag46152', 'hhsn271201200022c', '10.7303/syn3157329'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157238, gt_id=syn3157238\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5592519, gt_id=syn5592519\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn9705278, gt_id=syn9705278\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157322, gt_id=syn3157322\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3205821.6, gt_id=syn3205821\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157329, gt_id=syn3157329\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.24\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn5609493', 'e-mtab-2323'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse178362', 'gse136832', 'gse122332', 'gse149563', 'lmex0000004396', 'gse161382', 'gse178360', 'gse171524', 'lmex0000004397', 'gse135893', 'egas00001004082', 'gse122960', 'gse136831', 'gse134174', 'gse135851', 'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.0625\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1710680', 'syn1729383', 'syn1734155'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1710680', 'syn1729383', 'syn1734155'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: set()\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn21788425', 'gse147807'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn21788425', 'gse147807'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: set()\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn25576878', 'syn39456106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'gse15222', 'gse95587', 'e-geod-44770', 'gse169185', 'syn25576878', 'syn39456106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.2857142857142857\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse56031', 'syn1757053', 'e-mtab-4145', 'egas00000000083'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'tba'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn26134616', 'doi.org/10.7303/syn26134616'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse14333', 'prjeb23709', 'gse103479', 'gse33113', 'gse104645', 'gse17536', 'syn2623706', 'gse39582', 'gse38832', 'gse135222', 'gse78220', 'gse13067', 'gse108989', 'gse13294', 'gse37892', 'gse35144'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0625\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3219045', 'syn3505720', 'syn3191087', 'syn23554292'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'syn3505720', 'e-mtab-5060', 'syn23554292', 'syn3191087', 'syn3219045', 'e-mtab-5061', 'gse72056'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse25710', 'syn1688369', 'gse56066', 'gse84515', 'gse73663', 'gse154873', 'gse96517', 'gse78167', 'gse62944', 'gse102441', 'gse99542', 'gse57436', 'gse51403', 'gse78913'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26164834', 'gse249687', 'syn4921369'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.1073/', 'gse126957', 'gse142881', 'gse84176', 'syn4921369', 'gse249687', 'gse96107', 'gse59119'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn20681023', 'syn20681023', '10.6084/m9.figshare.13342055'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 12. Element IDs: {'gse161797', 'gse150117', 'gse150129', 'gse150121', 'gse150118', 'syn18485175', 'gse167044', 'gse161796', 'gse150128', 'gse150130', 'gse150079', 'syn4587609'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse81912', 'gse113673', 'encsr677hxc', 'e-mtab-9045', 'gse84245', 'gse65776', 'gse59630', 'encsr198zyj', 'gse60192', 'phs001373.v2.p2', 'srx3381923', 'gse59318', 'syn18485175', 'encsr822cea', 'gse150130', 'gse96107', '10.1371/', 'syn4587609'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.16666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn13363290', 'gse214628'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'syn13363290-v49', 'nm_021272.3'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn51365301'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'adni.loni.usc.edu', 'syn4907804'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn9702085', 'syn3157275', 'syn20068543', 'syn2580853', 'syn26720676', 'syn22264775', 'syn4896408', 'syn17008935', 'syn3157325', 'syn10235592'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn3157325', 'syn9702085', 'syn3157275', 'syn20068543', 'syn2580853', 'syn26720676', 'syn22264775', 'syn4896408', 'syn338856', 'syn17008935', 'syn10235592'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'sra', 'encode', 'syn6131484', 'geo'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52264240', 'gse225157', 'syn52234396'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn52264240', 'gse225157', 'encsr443vtk', 'encsr998ydi', 'syn52234396'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.6\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse148394', 'syn2623706'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse14333', 'gse148394', 'gse33113', 'syn2623706', 'tcga-coad', 'gse39582', 'tcga-read', 'gse38832', 'gse13294', 'e-mtab-8152', 'gse2109', 'gse44076', 'gse18088'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.15384615384615385\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn26254632', 'syn3157322', 'syn3388564'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'gse146865', 'gse146866', 'syn2580853', 'gse146864'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse57181', 'gse1432', 'gse98401', 'gse61499', 'gse65067', 'gse55627', 'gse138852', 'gse75431', 'gse13162', 'gse121654', 'gse61500', 'phs002197.v1.p1', 'gse146866', 'gse98969', 'gse90696', 'gse103334'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.0625\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2237931', 'syn1729383'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2237931', 'syn1729383'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn4587609'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn25886762', 'syn25792906', 'syn34542574', 'syn34542449', 'syn25876213', 'syn34539904', 'syn34542450', 'syn20843407'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn25886762', 'syn25792906', 'syn34542574', 'syn34542449', '10.7303/syn20843407', 'syn25876213', 'syn34539904', 'syn34542450', 'syn20843407'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8888888888888888\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31947506', 'syn31952297'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn52138250', 'syn31947506', 'syn31952297'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7214402', 'syn7343873', 'syn2812961'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn7214402', 'syn7343873', 'gse68379', 'prjeb11844', 'syn2812961', 'gse38270', 'prjeb5468', 'prjna258216', 'gse35069', 'gse74877', 'gse38266'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.2727272727272727\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn20803583', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn23191787', 'syn9884314', 'syn20803583', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.10795678', 'syn10901595', '10.5281/zenodo.11240547'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse33000', 'pxd009589', 'syn20821165'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse122960', '10.1101/', 'gse132355', 'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'gse156167', 'gse158233', 'syn20808171', 'gse149149', 'gse151999', 'gse164466'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse158233', 'gse156167', 'syn20808171', 'gse149149', 'gse151999', 'prjeb24858', 'gse164466', 'gse36980'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.75\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.5061/', 'phs000424.v7.p1', 'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'31561387', 'syn8717496'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn3163039', 'gse97942', 'gse64810', 'e-mtab-5061'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.13020/', 'syn50996324'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse77565', 'syn4921369'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'syn4921369', 'phs001190.v1.p1'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5556216', 'syn4984604', 'syn6022474', 'syn5492805', 'syn5522788', 'syn5493036', 'syn5522790', 'syn5004874', 'syn6022465', 'syn6035832'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn5556216', 'syn4984604', 'syn6022474', 'syn5492805', 'syn5522788', 'syn5493036', 'syn5522790', 'syn5004874', 'syn6022465', 'syn6035832'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461003', 'epi_isl_461056', 'epi_isl_461060', 'syn23532550', 'epi_isl_461053', 'epi_isl_461059', 'epi_isl_461057', 'epi_isl_461001', 'epi_isl_461054', 'epi_isl_461051', 'epi_isl_415464', 'epi_isl_461052', 'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461055'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse137834', 'syn20640275'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'e-mtab-9905', 'nc_045512.2', 'gse33584', 'syn20640275', 'gse137834', 'gse102470', 'mn996532.1', 'tcga', 'gse57763'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.2222222222222222\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn51150434', 'syn62110225', '10.58085/'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'gse178265', 'gse160936', 'gse167494', 'pmc9712627', 'gse174332', 'cg000376', 'gse148822', 'gse174367', 'syn12514624', 'gse137444'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.09090909090909091\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2813581'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn21125841', 'pxd029727'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse140511', 'gse113973', 'gse118257', 'gse138852', 'gse129788', 'gse775330', 'pxd029727', 'gse97930', 'gse116470', 'srp135960', 'gse130119', 'pxd020007', 'gse60361'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.07692307692307693\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse115046'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'sra:srp291312', 'sra:srp186687', '10.5281/', 'nm_001306068.3', 'synapse:syn10003873', 'sra:srp158590', 'nm_133330.3'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=synapse:syn10003873, gt_id=syn10003873\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.14285714285714285\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse186341', 'syn20968331'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse186341', 'syn20968331'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn11801497'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse134432', 'gse162170', 'e-mtab-6653', 'e-mtab-6149', 'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.2\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169709', 'syn12169715'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.1016/', 'syn12169709', 'tcga', 'syn12169715'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'voc-als', '10.7303/syn53009474'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn18344812', '10.6084/m9.figshare.8159525.v1', 'syn17009517'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.1101/2023.12.20.572491', 'syn11724057', '38187779', 'pmc10769298', 'q32851'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.2\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'gse30784', 'tcga'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse16476', 'gse79209', 'e-mtab-8248', 'gse89413', 'gpl18573'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn7824274', 'gdc'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'cnp0001438'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn11914606', 'syn14237651'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn11914606', 'syn14237651'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52137895', 'syn26707219'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8338532', 'syn30821562', '10.5281/'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn18796452'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'gse63805', 'syn1741407', 'gse63459'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'gse115205', 'syn9884357', 'gse89983', 'gse115206', 'gse90696', 'syn7818788', 'gse89979', 'gse89580'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn7818788', 'syn9884357', 'gse90696'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.375\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'egac00001000484', 'egas00000000083'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1715755', '10.7303/syn2468297'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000g_2504_high_coverage', '10.5281/zenodo.4321679', 'syn22024464', '10.15154/1504248'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn18360482'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 88 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 12
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:42.620818Z",
     "start_time": "2025-08-27T19:25:42.615748Z"
    }
   },
   "cell_type": "code",
   "source": "results",
   "id": "fff3dc3075cb13b5",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.5702544094794096,\n",
       " 'average_recall': 0.8845833333333334,\n",
       " 'f1_score': 0.6934622762948975}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 13
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-27T19:25:42.628632Z",
     "start_time": "2025-08-27T19:25:42.626487Z"
    }
   },
   "cell_type": "code",
   "source": "",
   "id": "1869e71b9d253e7d",
   "outputs": [],
   "execution_count": 13
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
