{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "initial_id",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:30:53.289617Z",
     "start_time": "2025-09-04T16:30:29.500758Z"
    },
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "78d2b2bc4106d2ef",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:15.898169Z",
     "start_time": "2025-09-04T16:31:15.880961Z"
    }
   },
   "outputs": [],
   "source": [
    "input_file = \"scripts/exp_input/REV.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "cca05a29b7c100d1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:17.295747Z",
     "start_time": "2025-09-04T16:31:17.291744Z"
    }
   },
   "outputs": [],
   "source": [
    "model_name = \"gemini-2.0-flash\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"GPT_FewShot\"  # \"GPT_from_full_input_Examples\" or \"GPT_FewShot\"\n",
    "FDR = False\n",
    "semantic_retrieval = True\n",
    "section_filter= None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "a4ae25cbd54d4fe1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:19.865917Z",
     "start_time": "2025-09-04T16:31:19.860295Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 20\n"
     ]
    }
   ],
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()[:20]\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b79cbd7265b497e1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:22.492692Z",
     "start_time": "2025-09-04T16:31:22.479800Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 329 - INFO - Setting up data fetcher...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 355 - INFO - Data fetcher setup completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 108 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.0-flash\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "dg = DataGatherer(\n",
    "    llm_name=model_name, \n",
    "    log_level='INFO', \n",
    "    process_entire_document=FDR, \n",
    "    driver_path='../Firefox/geckodriver', \n",
    "    save_to_cache=False, \n",
    "    load_from_cache=False,\n",
    "    full_output_file=\"scripts/output/result.csv\"\n",
    ") #, save_dynamic_prompts=True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "5e0e0500474a6c34",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:25.329784Z",
     "start_time": "2025-09-04T16:31:25.325226Z"
    }
   },
   "outputs": [],
   "source": [
    "# These are the problematic urls for information extraction for GEMINI\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335/\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917/\"]\n",
    "# v1 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\"\n",
    "# v2 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362\"\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693/\"]\n",
    "\n",
    "# These are the problematic urls for information extraction for GPT - (2025 Aug 29) GPT gets stuck for some reason after 50 calls or so...\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\"]\n",
    "\n",
    "#out = dg.process_articles(urls, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
    "\n",
    "#out\n",
    "#out['https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "bfd0572331e32fe2",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T17:36:22.343172Z",
     "start_time": "2025-09-04T16:31:29.415907Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #0 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 296 - INFO - Creating EntrezFetcher with backup support\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 102 - INFO - Backup data store initialized: 1313 publications, valid: True\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 834 - INFO - Raw_data_format: XML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10418516 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10418516 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 8 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec2dot1-ijms-24-12380'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec2dot1-ijms-24-12380: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec3dot2-ijms-24-12380'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec3dot2-ijms-24-12380: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec4dot4dot4-ijms-24-12380'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec4dot4dot4-ijms-24-12380: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#app1-ijms-24-12380'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 241) for app1-ijms-24-12380: For a complete understanding of the experiment, its scheme is shown in Figure S1 [href=this file] .\n",
      "For a complete understanding of the experiment, its scheme is shown in Figure S2 [href=this file] .\n",
      "Supplementary Materials [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ijms-24-12380-s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ijms-24-12380-s001: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 45) for notes4: Data Availability Statement [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 328048\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 242467\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 242466\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 71 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 71 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 71 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 533 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 71 sections → 226 unique corpus documents (processed 307 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 226 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "bf5611ef8d1f499ea4bfac3ac86dd326",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/8 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 6631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 2007 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-b132cda2e05c2af6bb2e476468d9126b189a2025201870cc4360671186611012\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2007 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8f9e40e2-f9fc-413f-8729-7245ef14e508\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE207091\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD044299\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD044299\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE207091\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD044299\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD044299\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381922,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2624,\n",
      "        \"completion_tokens\": 149,\n",
      "        \"total_tokens\": 2773,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE207091', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091', 'dataset_id': 'GSE207091'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GEO, identifier: GSE207091, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1012 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE207091, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091, resolved_repo geo, dataset_id GSE207091\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.67s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (226, 384)\n",
      "Chunking results: 226 original documents → 226 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE207091, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE207091', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE207091', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD044299', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD044299', 'dataset_id': 'PXD044299'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD044299, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD044299, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD044299, resolved_repo www.ebi.ac.uk, dataset_id PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD044299, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD044299\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD044299', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD044299', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 2 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 4\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 719 - INFO - \n",
      "Progress: 1/20 (5.00%) | Elapsed: 00:00:05 | ETA: 00:01:53\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #1 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC6327059 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC6327059 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 11 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec26: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 42 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 8045) for MOESM1: To identify gene mutations that rescue cytotoxicity induced by loss of BRCA2, monoclonal KBM-7 cell lines were engineered to express doxycycline-inducible BRCA2 short hairpin RNAs (shRNAs; Fig. 1a [href=#Fig1] , Supplementary Fig. 1a [href=this file] ). After 48 h of doxycycline treatment, ionizing radiation (IR)-induced recruitment of RAD51 to foci was lost (Fig. 1b [href=#Fig1] , Supplementary Fig. 1b [href=this file] ).\n",
      "Interestingly, the TNFα pathway component on average showed lower median messenger RNA (mRNA) levels in -mutated vs. wildtype (wt) tumors (Supplementary Fig. 1c [href=this file] ). showed a larger difference in expression level when compared to , although differences for both genes were not statistically significant, likely due to the low number of mutant cancers.\n",
      "To validate whether TNFR1 or SAM68 inactivation confers a survival advantage upon BRCA2 depletion, KBM-7-shBRCA2 cells were infected with plasmids harboring shRNAs targeting TNFR1 or SAM68 while also encoding an internal ribosome entry site (IRES)-driven mCherry cassette (Supplementary Fig. 2a [href=this file] ). Notably, TNFR1- or SAM68-depleted KBM-7 cells did not confer a survival advantage through compromising the shRNA-induced knockdown of BRCA2 (Supplementary Fig. 2b [href=this file] ). In contrast, BRCA2 depletion was nullified by small interfering RNA (siRNA)-mediated knockdown of SUPTH3, in line with our expectations (Supplementary Fig. 2c [href=this file] ).\n",
      "Depletion of TNFR1 or SAM68 did not reduce the total level of DNA damage induced by BRCA2 loss, as γH2AX levels were similar (Supplementary Fig. 2d [href=this file] ). Moreover, loss of TNFR1 or SAM68 did not confer a generic survival advantage, as TNFR1 or SAM68 depletion did not rescue cytotoxicity induced by Eg5 depletion (Supplementary Fig. 2e [href=this file] ). When : mouse embryonic fibroblasts (MEFs) were infected with Cre recombinase to induce loss of BRCA2 and p53, this resulted in efficient gene inactivation and interfered with cellular viability (Supplementary Fig. 3a [href=this file] ). Of note, shRNA-mediated inactivation of TNFR1 or SAM68 did not significantly rescue cellular survival (Supplementary Fig. 3b,c [href=this file] ). In line with these cells not being responsive to TNFα receptor signaling, inactivation of did not confer sensitivity to recombinant TNFα (Supplementary Fig. 3d [href=this file] ).\n",
      "Next, two triple negative breast cancer (TNBC) cell lines, BT-549 and MDA-MB-231, were depleted for BRCA2 (Supplementary Fig. 4a [href=this file] ). In line with our results in KBM-7 cells, BRCA2 depletion interfered with long-term survival (Supplementary Fig. 4b [href=this file] ). Assessing the effects of TNFR1 inactivation on survival of BRCA2-depleted MDA-MB-231 was not feasible, because TNFR1 appeared essential for viability in this cell line, regardless of BRCA2 status (Supplementary Fig. 4c-e [href=this file] ). In contrast, a stable population of TNFR1-depleted BT-549 cells was established (Supplementary Fig. 4f,g [href=this file] ), and showed that TNFR1 inactivation results in a survival benefit in BRCA2-depleted BT-549 cells (Fig. 2d, e [href=#Fig2] ). However, SAM68 depletion interfered with survival of BT-549 cells independent of BRCA2 status, as hardly any cells survived constitutive SAM68 depletion (Supplementary Fig. 4g,h [href=this file] ).\n",
      "Notably, co-depletion of BRCA2 with SAM68 or TNFR1 rescued the observed sensitivity to TNFα (Fig. 4b [href=#Fig4] , Supplementary Fig. 5a [href=this file] ). These responses were not specific to KBM-7 cells, as increased TNFα sensitivity was also observed in a dose-dependent manner upon BRCA2 depletion in a panel of TNBC cell lines (Fig. 4c [href=#Fig4] , Supplementary Fig. 4a [href=this file] , 5b [href=this file] ) and in the colorectal cancer cell line DLD-1, in which the gene was inactivated using CRISPR/Cas9 (clustered regularly interspaced short palindromic repeats/CRISPR-associated 9) (Fig. 4d [href=#Fig4] , Supplementary Fig. 5c [href=this file] ). Importantly, the increased sensitivity to TNFα in BRCA2-depleted cells could not be attributed to changes in TNFR1 expression levels upon BRCA2 inactivation (Supplementary Fig. 5d,e [href=this file] ).\n",
      "To test if sustained activity of ASK1 or JNK kinases is required to mediate TNFα-induced cell death in BRCA2-depleted cells, we chemically inhibited JNK (Fig. 4e [href=#Fig4] , Supplementary Fig. 5f [href=this file] ) or ASK1 (Fig. 4f [href=#Fig4] , Supplementary Fig. 5f [href=this file] ), in combination with TNFα treatment. Control-depleted BT-549 and HCC38 cells were not sensitive to TNFα, and their viability was not affected by JNK or ASK1 inhibition (Fig. 4e, f [href=#Fig4] , Supplementary Fig. 5f [href=this file] ). Notably, these effects were dose-dependently reversed by JNK or ASK1 inhibition (Fig. 4e, f [href=#Fig4] , Supplementary Fig. 5f [href=this file] ). To test if the observed increase in TNFα sensitivity upon BRCA2 depletion was driven by apoptosis-mediated cell death, we depleted caspase-3, -8 or -9 using siRNAs (Supplementary Fig. 5g [href=this file] ). Especially depletion of caspase-8 and -9 resulted in reduced sensitivity to TNFα in BRCA2-depleted cells, while the viability of control-depleted cells was not significantly affected (Fig. 4g [href=#Fig4] , Supplementary Fig. 5h [href=this file] ). Blocking caspase activity using the broad-spectrum caspase inhibitor zVAD-FMK confirmed the requirement of caspase activity, as TNFα-induced cell death in BRCA2-depleted BT-549 and HCC38 cells was significantly rescued by zVAD-FMK treatment (Supplementary Fig. 5i [href=this file] ).\n",
      "To check whether increased TNFα sensitivity was selectively induced by BRCA2 inactivation, BT-549 and HCC38 cells were depleted for the DNA repair proteins BRCA1 or FANCD2 (Supplementary Fig. 6a,b [href=this file] ). Depletion of BRCA1 or FANCD2 decreased long-term survival, comparable to BRCA2 depletion (Supplementary Fig. 6c [href=this file] ). Importantly, depletion of BRCA1 or FANCD2 also resulted in sensitivity to recombinant TNFα, both in BT-549 and HCC38 cells (Fig. 4h [href=#Fig4] and Supplementary Fig. 6d [href=this file] ). Of note, induction of replication stress with a non-toxic dose of hydroxyurea (HU) (Supplementary Fig. 6e, f [href=this file] ) also sensitized TNBC cell lines to recombinant TNFα (Fig. 4i [href=#Fig4] , Supplementary Fig. 6g [href=this file] ).\n",
      "Clearly, the most significantly enriched gene sets in both cell lines included interferon-γ and interferon-α responses, as well as activation of TNFα-responsive pathways (Fig. 5d–g [href=#Fig5] , and Supplementary Fig. 7a,b [href=this file] ).\n",
      "The top 10 list of enriched pathways can be found in Supplementary Fig. 7a [href=this file]\n",
      "In line with this notion, we observed elevated levels of micronuclei and cGAS-positive micronuclei upon BRCA2 depletion in BT-549 and HCC38 cells (Fig. 6b, c [href=#Fig6] ), which was accompanied by elevated levels of phosphorylated signal transducer and activator of transcription 1 (STAT1), a key mediator of interferon-induced transcription (Fig. 6d [href=#Fig6] , Supplementary Fig. 7c [href=this file] ). Importantly, siRNA-mediated depletion of cGAS or STING resulted in reduced levels of STAT1 phosphorylation in BRCA2-depleted BT-549 and HCC38 cells (Fig. 6e, f [href=#Fig6] and Supplementary Fig. 7d,e [href=this file] ). Furthermore, depletion of cGAS or STING rescued the sensitivity of TNFα upon BRCA2 inactivation in BT-549 and HCC38 cells (Fig. 6g [href=#Fig6] and Supplementary Fig. 7f [href=this file] ). Specifically, mutation of cGAS rescued long-term viability in BRCA2-depleted cells (Supplementary Fig. 7g,h [href=this file] ). Also, TNFα sensitivity and STAT1 phosphorylation upon BRCA2 depletion were rescued in cGAS cells compared to cGAS wt cells (Fig. 6h,i [href=#Fig6] and Supplementary Fig. 7i,j [href=this file] ).\n",
      "Uncropped versions of all western blots can be found in Supplementary Fig. 8 [href=this file] – 13 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 188) for MOESM4: Massive parallel sequencing was performed on genomic DNA isolated from cells that were allowed to grow for 19 days in the presence of doxycycline (Supplementary Data 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 542) for MOESM5: To filter out mutations that affect doxycycline-mediated expression of shRNAs, we performed a cross-comparison with a screen for gene mutations that reversed cell death induced by shRNA-mediated loss of the essential mitotic spindle component Eg5 (Supplementary Data 2 [href=this file] ) . As expected, multiple dominant integration hotspots identified in the shBRCA2 screen marked doxycycline-related genes which will nullify shRNA-mediated BRCA2 depletion, including , and (Supplementary Data 2 [href=this file] and Fig. 1f [href=#Fig1] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 178) for MOESM6: Common differentially expressed proteins measured in at least three out of four independent MS runs were plotted (Fig. 5b [href=#Fig5] , Supplementary Data 3 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#MOESM7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 128) for MOESM7: For values, see Supplementary Data 4 [href=this file]\n",
      "Statistical analysis is provided in Supplementary Data 4 [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 138) for MOESM8: Further information on experimental design is available in the Nature Research Reporting Summary [href=this file] linked to this Article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 255604\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 187108\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 187107\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 72 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 72 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 72 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 377 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 72 sections → 174 unique corpus documents (processed 203 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 174 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "91b841d0d5d646a0a1e2154041dd3013",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 7854\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 2313 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-3f2c48c952afc36d9486ecff945c52e87a65a07e178d968bfc3b4f0c06c428ab\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2313 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3bd76a54-8ef3-44d5-8692-55a44e433145\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE116943\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD007253\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD007253\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PRJNA299537\\\",\\n        \\\"data_repository\\\": \\\"NCBI short read archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE116943\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD007253\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD007253\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PRJNA299537\\\",\\n        \\\"data_repository\\\": \\\"NCBI short read archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381928,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 3420,\n",
      "        \"completion_tokens\": 222,\n",
      "        \"total_tokens\": 3642,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE116943', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943', 'dataset_id': 'GSE116943'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GEO, identifier: GSE116943, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1012 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE116943, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943, resolved_repo geo, dataset_id GSE116943\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.22s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (174, 384)\n",
      "Chunking results: 174 original documents → 174 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE116943, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE116943', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116943', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD007253', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD007253', 'dataset_id': 'PXD007253'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD007253, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD007253, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD007253, resolved_repo www.ebi.ac.uk, dataset_id PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD007253, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD007253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD007253', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD007253', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PRJNA299537', 'data_repository': 'NCBI short read archive', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537', 'dataset_id': 'PRJNA299537'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI short read archive, identifier: PRJNA299537, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(NCBI short read archive,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: NCBI short read archive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PRJNA299537, ncbi short read archive, https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: ncbi short read archive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537, resolved_repo ncbi short read archive, dataset_id PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID PRJNA299537 found in resolved dataset page https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PRJNA299537, ncbi short read archive, https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PRJNA299537', 'data_repository': 'ncbi short read archive', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA299537', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 3 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 12\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #2 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10359415 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10359415 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 24 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec24: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec38'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec38: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec39'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec39: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec49'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec49: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec49'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec49: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec50'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec50: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec50'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec50: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 68 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 16576) for MOESM1: According to the ISCT criteria, we evaluated the expression of MSC surface markers (Supplementary Fig. 1a [href=this file] ) and tri-lineage differentiation potential (Supplementary Fig. 1b [href=this file] ), only samples that passed rigorous quality control were subjected to droplet-based single-cell RNA sequencing (scRNA-seq). To further confirm the accuracy of our MSC quality control, we verified in single-cell transcriptomes that cells were positive for expression of a panel of positive MSC markers ( , and ), and no expression of negative MSC markers ( , , and ) was detected in nearly all the cells (Supplementary Fig. 1c [href=this file] ), suggesting the accuracy of our sample processing procedure and the high quality of the datasets.\n",
      "C4 appeared to be a subset of cells in a transition state (Supplementary Fig. 1d [href=this file] ). Monocle analysis predicted that the evolution of mesenchymal clusters occurred on a onefold trajectory, starting from cells in the active proliferating phase (C1, C2 and C3) and ending with C7 (Supplementary Fig. 1e [href=this file] ). Consistent with the Monocle analysis, the PAGA path also suggested that active proliferating clusters gave rise to cells in C5, C6 and C7, while C4 appeared as a node that linking this transition (Fig. 1d [href=#Fig1] ; Supplementary Fig. 1e [href=this file] ).\n",
      "MSCs from C5-7 showed upregulation of genes related to cellular senescence (Fig. 1e [href=#Fig1] ) and decreased expression of pluripotency stem cell signature genes (Supplementary Fig. 1f [href=this file] ), accompanied by impaired DNA repair and proliferative function (Fig. 1f, g [href=#Fig1] ). Other senescence-related genes, such as important constituents of SASP factors ( , , ), significant regulators of epigenetic shift in senescence ( , , ) and genes encoding actin stress fibers ( , , , ) were found to be differentially expressed in senescent and non-senescent clusters (Fig. 1h [href=#Fig1] ; Supplementary Fig. 1g [href=this file] ).\n",
      "To further analyze aging-related perturbations in the transcriptome of C5 MSCs, we conducted GO enrichment analysis of marker genes of clusters C5 (Fig. 2d [href=#Fig2] ), C6 and C7 (Supplementary Fig. 2a, b [href=this file] ). To further confirm the ER stress in C5, we performed gene set scoring analysis, which suggested that the enhanced UPR related pathways (ATF6 signaling, PERK signaling, IRE1 signaling and ERAD pathway) are linked to the senescence of MSCs in C5 (Supplementary Fig. 2c, d [href=this file] ).\n",
      "Transcription factors (TFs) related to cell proliferation and anti-senescence factors, such as and , were highly expressed in the C1, C2 and C3 populations and gradually downregulated in aging clusters (Supplementary Fig. 2e [href=this file] ), indicating that transcription factors might contribute to the cellular senescence of MSCs. In line with the Monocle results (Supplementary Fig. 1e [href=this file] ), cells from C1 to C3 underwent continuous self-renewal cycling, with some of the cells from C3 progressing to the transition state (cluster C4). Interestingly, we also identified several regulons that are seldom reported in MSCs (Supplementary Fig. 2f [href=this file] ), including transcription factors contributing to nervous system development ( , ), embryonic stem cell self-renewal ( ) and hematopoiesis ( ) .\n",
      "Although RNA velocity analysis suggests that MSCs from adult and perinatal tissues share a similar cellular transition pattern (Fig. 3a [href=#Fig3] ), we found that MSCs from different tissue sources differed substantially in relative cell frequency per cluster (Supplementary Fig. 3a [href=this file] ). Non-aging clusters C1, C2, and C3 exhibited a significantly increased frequency in perinatal MSCs, whereas the frequency of C7 was significantly decreased (Fig. 3b [href=#Fig3] ; Supplementary Fig. 3a [href=this file] ). Furthermore, the heterogeneity in aging levels is reflected in an increase in the proportion of C7 in adult MSCs, but there was no significant expansion of C5 and C6 (Fig. 3b [href=#Fig3] ; Supplementary Fig. 3a [href=this file] ).\n",
      "Although MSCs from adult and perinatal tissue retain a similar set of core signature genes among subclusters (Supplementary Fig. 3b [href=this file] ), many aging related genes were differentially expressed in each population (Supplementary Fig. 3c [href=this file] ). Furthermore, gene set scoring also indicated a higher degree of senescence in MSCs from aging tissues (Fig. 3d [href=#Fig3] ), while genes related to pluripotency stem cell signature, DNA repair and proliferation pathways were transcriptionally downregulated in adult MSCs (Supplementary Fig. 3d [href=this file] , Fig. 3d [href=#Fig3] ). Finally, we verified the aging-related gene expression differences by quantitative real-time PCR (Supplementary Fig. 3e [href=this file] ), which corroborated the downregulation of anti-senescence genes ( , , and ) in adult MSCs. The results obtained from qPCR showed downregulated expression of , , and in adult MSCs (Supplementary Fig. 3e [href=this file] ), indicating that senescence may diminish the multipotent differentiation capacity of MSCs.\n",
      "Furthermore, MSCs from adult tissues were also observed to have increased phosphorylated γH2AX foci via immunofluorescence (Supplementary Fig. 3f [href=this file] ); these foci coordinate unresolved genotoxic stress and double-strand breaks.\n",
      "Next, we examined the underlying mechanism by which senescence inhibits the immunosuppressive function of cultured MSCs. ( ) was selected as an important candidate, as it was significantly upregulated in perinatal MSCs and gradually downregulated during the MSC senescence process (Fig. 4b [href=#Fig4] ; Supplementary Fig. 4a [href=this file] ). To directly determine the role of senescence in regulating MSC PD-L1 expression, we induced cellular senescence in UC-MSCs via the application of doxorubicin or H O (Fig. 4c [href=#Fig4] ) and verified the senescence status of the drug-induced senescence model. (Fig. 4d [href=#Fig4] ; Supplementary Fig. 4b [href=this file] ). The flow cytometry results indicated that compared to the normal control (NC) group, DOXO- or H O -treated MSCs had a lower percentage of PD-L1-positive cells (Supplementary Fig. 4c [href=this file] ). Similarly, protein bands on Western blots showed that drug-treated senescent MSCs exhibited reduced PD-L1 expression and upregulation of the aging markers P53 and P21 (Fig. 4e [href=#Fig4] ; Supplementary Fig. 4d, e [href=this file] ). Consistent with this, the downregulation of PD-L1 was also observed in replicative senescent UC MSCs from high culture passages (Supplementary Fig. 4f [href=this file] ).\n",
      "As expected, NC MSCs showed a much stronger suppressive effect, as evidenced by lower CD4 T cell counts (Fig. 4f [href=#Fig4] ) and a decreased relative proliferation rate (Supplementary Fig. 4g, h [href=this file] ). After PD-L1 blockade with specific antibodies, the inhibition of CD4 T cell proliferation by NC MSCs was largely reversed, while T cell proliferation in the presence of senescent MSCs was changed only slightly after PD-L1 blockade (Fig. 4f [href=#Fig4] ; Supplementary Fig. 4h [href=this file] ), indicating relatively low endogenous PD-L1 expression in senescent MSCs.\n",
      "As determined by flow cytometry, the proportion of PD-L1+ MSCs was significantly higher in perinatal tissue-derived MSCs than in adult MSCs (Fig. 4i [href=#Fig4] , Supplementary Fig. 4i, j [href=this file] ). The Western blot results also confirmed that there was a significantly higher level of PD-L1 expression in perinatal MSCs, accompanied by downregulation of the senescence markers P53 and P21 (Supplementary Fig. 4k, l [href=this file] ). To determine whether cellular senescence attenuates PD-L1 secretion by MSC, we collected extracellular vesicles (EVs) from the culture medium of UC- and BM-MSCs (Supplementary Fig. 4m [href=this file] ) and applied mass spectrometry sequencing to investigate their protein expression level. We found that compared to UC-MSCs, BM-MSCs from aging tissues lost the capacity to significantly inhibit T cell proliferation when coculturing (Fig. 4k [href=#Fig4] ; Supplementary Fig. 4n, o [href=this file] ). In contrast, the addition of anti-PD-L1 Fab to the coculture system with UC-MSCs substantially increased T cell proliferation (Fig. 4k [href=#Fig4] ; Supplementary Fig. 4n, o [href=this file] ), indicating that the aging-associated decline in MSC immunomodulatory capacity is related to the downregulation of PD-L1. Furthermore, the results of ELISA also suggested that perinatal MSCs showed stronger immunomodulatory ability, as T cells cocultured with UC MSCs secreted less TNF-α and IFN-γ, which was partially rescued by PD-L1 blockade (Supplementary Fig. 4p, q [href=this file] ).\n",
      "Consistent with previous reports , we found that adipogenesis of aged BM-MSCs was enhanced, while osteogenic differentiation was weakened under in vitro induction (Supplementary Fig. 5a [href=this file] ), indicating a skewed differentiation potential. The results from the CCK-8 assay also showed that the proliferation was decreased in aged BM-MSCs (Supplementary Fig. 5b [href=this file] ). We proved the purity of BM-MSC samples by detecting MSC markers defined by ISCT criteria (Supplementary Fig. 5c [href=this file] ). To characterize the functional characteristics of each BM-MSC cluster, we conducted GO analysis based on the DEGs of each cluster (Supplementary Fig. 5d [href=this file] , Supplementary Data 8 [href=#MOESM4] ). Compared to cluster BM1, BM2 and BM3, cells in BM5 and BM6 were projected into senescent cluster C6 and C7 with their upregulated genes enriched in senescence related pathways, including ‘protein folding’ and ‘signal by p53 class mediator’, while cells from BM1, BM2 and BM3 highly expressed genes related to proliferation and anti-senescence functions (Supplementary Fig. 5d [href=this file] ). Consistently, the gene set scoring analysis showed cells from BM5 and BM6 have upregulated expression of cellular senescence genes (Fig. 5d [href=#Fig5] ), accompanied by impaired DNA repair and proliferative function (Supplementary Fig. 5e, f [href=this file] ), the downregulation of pluripotency stem cell signature genes was also observed in BM5 and BM6 (Supplementary Fig. 5g [href=this file] ). Furthermore, BM-MSCs from senescent (BM5-6) or non-senescent (BM1-3) clusters possessed a similar set of marker genes (Fig. 5e [href=#Fig5] ) and activated regulons (Supplementary Fig. 5g [href=this file] , Supplementary Data 9 [href=#MOESM4] ) to those found in adult and perinatal MSC dataset.\n",
      "To further investigate the impact of cellular senescence on the multi-lineage differentiation capacity of MSCs, we calculated the gene set scores for adipogenic genes, osteogenic genes and chondrogenic genes for each BM-MSC cluster (Supplementary Fig. 5i [href=this file] , Supplementary Data 3 [href=#MOESM4] ). Besides, we found that BM-MSCs derived from aged donors exhibited upregulation of adipogenic genes but decreased expression of osteogenic genes (Supplementary Fig. 5j [href=this file] ), which was consistent with the results of the in vitro differentiation experiments (Supplementary Fig. 5a [href=this file] ). Although no significant difference in chondrogenic ability was observed during the differentiation induction experiment of young and aged BM-MSCs (Supplementary Fig. 5a [href=this file] ), our transcriptome-level analysis suggested that the expression of chondrogenic-related genes was downregulated in aged BM-MSCs. (Supplementary Fig. 5i, j [href=this file] ). Although osteogenic and chondrogenic regulators did not exhibit significant differences between aging and non-aging clusters, we found that the expression of transcription factors regulating adipogenic differentiation was significantly upregulated in aging BM-MSC clusters (Supplementary Fig. 5k [href=this file] ). While not statistically significant, we observed that CEBPB, which is indispensable for triggering adipogenesis , was upregulated in aged samples at the pseudobulk level, whereas the expression of RUNX2, an essential regulator that controls the initial stages of osteoblastogenesis , was decreased in aged BM-MSCs (Supplementary Fig. 5l [href=this file] ).\n",
      "Consistent with previous analysis, we found that MSCs from non-senescent clusters (BM1, BM2, BM3) showed higher expression of immunosuppressive genes (Fig. 5i [href=#Fig5] ), cells from young BM-MSC samples also exhibit an overall higher immunosuppressive score compared to aged BM-MSCs (Supplementary Fig. 5m [href=this file] ). To examine the effect of aging on expression, we detected the gene expression in BM-MSCs dataset, the result showed that was downregulated in senescent clusters (Fig. 5j [href=#Fig5] ) or aging MSC samples (Supplementary Fig. 5n [href=this file] ). Downregulation of expression at the transcript level of aged BM-MSCs resulted in a decrease in PD-L1 at the protein level, as we detected by flow cytometry (Supplementary Fig. 5o [href=this file] ) and western blotting (Fig. 5k [href=#Fig5] ). In addition, the levels of PD-L1 protein secreted by senescent BM-MSCs were also reduced, as we detected in culture supernatant using ELISA (Supplementary Fig. 5p [href=this file] ). In order to confirm whether the decreased expression of PD-L1 in aging donor MSCs affects the immunosuppressive function, we co-cultured CFSE labeled activated CD4 T cell with young and aged BM-MSC (Supplementary Fig. 5q [href=this file] ). From the results of CFSE proliferation assay and ELISA analysis, we found that compared to young BM-MSCs, BM-MSCs from aging donors lost the capacity to significantly inhibit activated T cells, while this gap was largely reversed after anti-PD-L1 Fab addition (Fig. 5l [href=#Fig5] ; Supplementary Fig. 5r–u [href=this file] ).\n",
      "We also conducted correlation analysis to confirm the PCA results (Supplementary Fig. 6a [href=this file] ). The observed expression changes in the transcriptome and proteome data indicated comparable results (Fig. 6d [href=#Fig6] , Supplementary Fig. 6b, c [href=this file] ).\n",
      "Overlaying the core TFs (Cor>0.9; value < 0.05) from the correlation analysis with the top upregulated regulon ( -value > 30; value < 0.05) in C1, C2 and C3 revealed 21 candidate TFs (Fig. 6e [href=#Fig6] , Supplementary Data 12 [href=#MOESM4] ), including the transcription factor , which was observed to downregulated during the MSC senescent progress (Supplementary Fig. 6d [href=this file] ).\n",
      "To prove the role of GATA2 in regulating MSC senescence, we used lentivirus vectors (Supplementary Fig. 7a [href=this file] ) to overexpress in adult BM-MSCs, and successful overexpression was verified by qPCR (Fig. 7a [href=#Fig7] ). Compared to MSCs infected with empty vector (EV), overexpressed ( -OE) BM-MSCs have weaker β-glucosidase expression at low (Fig. 7b [href=#Fig7] ) and high culture passage (Fig. 7c [href=#Fig7] ; Supplementary Fig. 7b [href=this file] ), which indicates the enhanced anti-senescence function of these cells.\n",
      "To confirm this finding, we next examined PD-L1 expression using western blot (Fig. 7h [href=#Fig7] ) and flow cytometry (Supplementary Fig. 7c [href=this file] ), which validated that the overexpression of upregulates the expression of PD-L1 on protein level. To determine whether the PD-L1 upregulation induced by overexpression enhances immunosuppressive function, we cocultured -OE MSCs with activated CD4 T cells (Supplementary Fig. 7d [href=this file] ) and tested the effect of overexpression on human MSC-mediated T cell suppression by evaluating the capacity to inhibit CFSE-labeled CD4 T cell proliferation (Fig. 7i [href=#Fig7] ; Supplementary Fig. 7e, f [href=this file] ) and proinflammatory factors secretion (Supplementary Fig. 7g, h [href=this file] ). To provide further evidence that GATA2 regulates MSC senescence and PD-L1 expression, we used lentivirus vectors (Supplementary Figure 7i [href=this file] ) to knockdown in perinatal UC-MSCs (Fig. 7j [href=#Fig7] ). Compared to MSCs infected with shRNA scramble control (CT), UC-MSCs after knockdown (KD) showed senescence phenotypes (Fig. 7k [href=#Fig7] ) and downregulations of PD-L1 at protein level (Fig. 7l [href=#Fig7] ; Supplementary Fig. 7j [href=this file] ). As expected, we also proved that the decreased PD-L1 expression mediates the impaired immunosuppressive capacity of -KD MSCs through the MSC and CD4 T cells co-culturing assay (Fig. 7m [href=#Fig7] , Supplementary Fig. 7k–o [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 26 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 5710) for MOESM4: Cells isolated from human adipose tissue (AD, = 3), bone marrow (BM, = 3), placental chorionic membrane (PM, = 3) and umbilical cord (UC, = 3) were cultured and purified in vitro (Supplementary Data 1 [href=this file] ). Clustering analysis divided 45,955 current MSC cells from 4 tissue origins into seven subsets (Fig. 1b [href=#Fig1] ) with distinct expression signatures (Supplementary Data 2 [href=this file] ).\n",
      "To further characterize the functional characteristics of each cell population, we performed GSVA analysis to define enrichment scores for each pseudobulk cluster (Fig. 1c [href=#Fig1] ; Supplementary Data 3 [href=this file] ). The top expressed gene sets in each cluster are displayed on the heatmap, which suggested that MSCs from C1, C2 and C3 highly expressed genes involved in cell cycle progression and proliferation, while the transcription of these genes was restricted in C5, C6 and C7, accompanied by apparently altered cellular functions (Fig. 1c [href=#Fig1] ; Supplementary Data 4 [href=this file] ).\n",
      "We also applied single-cell regulatory network inference and clustering (SCENIC) analysis to assess specific global gene regulatory networks associated with MSC senescence (Supplementary Data 5 [href=this file] ). Based on the results of RNA velocity analysis, we next constructed a generalized linear model (GLM; see Methods) to characterize regulatory events responsible for the transitions between consecutive MSC subpopulations (Supplementary Data 6 [href=this file] ).\n",
      "To systematically assess the effect of senescence on the immunomodulatory capacity of MSCs, we summarized multiple factors (Supplementary Data 3 [href=this file] ) previously reported to perform immunosuppressive functions in MSCs and used them for gene set analysis (see Methods).\n",
      "As compared to EVs secreted from the more senescent BM-MSCs, UC-MSC EVs specifically upregulated PD-L1 expression level (Fig. 4j [href=#Fig4] ; Supplementary Data 7 [href=this file] ).\n",
      "To characterize the functional characteristics of each BM-MSC cluster, we conducted GO analysis based on the DEGs of each cluster (Supplementary Fig. 5d [href=#MOESM1] , Supplementary Data 8 [href=this file] ). Furthermore, BM-MSCs from senescent (BM5-6) or non-senescent (BM1-3) clusters possessed a similar set of marker genes (Fig. 5e [href=#Fig5] ) and activated regulons (Supplementary Fig. 5g [href=#MOESM1] , Supplementary Data 9 [href=this file] ) to those found in adult and perinatal MSC dataset.\n",
      "To further investigate the impact of cellular senescence on the multi-lineage differentiation capacity of MSCs, we calculated the gene set scores for adipogenic genes, osteogenic genes and chondrogenic genes for each BM-MSC cluster (Supplementary Fig. 5i [href=#MOESM1] , Supplementary Data 3 [href=this file] ).\n",
      "To investigate common or distinct regulation of gene annotation categories at the transcriptomic or proteomic level, we performed a two-dimensional annotation enrichment analysis using Perseus software (Fig. 6c [href=#Fig6] , Supplementary Data 11 [href=this file] ). Furthermore, we compared the multiomics data between adult MSCs and perinatal MSCs to detect significantly differentially expressed genes and proteins ( .adj < 0.05) for each comparison (Fig. 6d [href=#Fig6] , Supplementary Data 10 [href=this file] ).\n",
      "The significant positive correlation of both datasets was calculated by Spearman’s test, values were computed using two-sided algorithm AS 89 ( = 114 pathways; Supplementary Data 11 [href=this file] ). The significant positive correlation of both datasets was calculated by Spearman’s test, values were computed using two-sided asymptotic t approximation ( = 3,664 genes; Supplementary Data 10 [href=this file] ).\n",
      "We applied two criteria to narrow down our candidates: candidate TFs should be positively correlated with upregulated non-TF genes at the transcriptomic and proteomic levels in UC-MSCs, and their regulon should be activated in single-cell clusters C1, C2 and C3 (Supplementary Data 12 [href=this file] ). Overlaying the core TFs (Cor>0.9; value < 0.05) from the correlation analysis with the top upregulated regulon ( -value > 30; value < 0.05) in C1, C2 and C3 revealed 21 candidate TFs (Fig. 6e [href=#Fig6] , Supplementary Data 12 [href=this file] ), including the transcription factor , which was observed to downregulated during the MSC senescent progress (Supplementary Fig. 6d [href=#MOESM1] ).\n",
      "Therefore, we next analyzed GATA2 enrichment at promoter by quantitative real-time PCR using primers designed to detect GATA2 binding fragments at −153 bp (primer 1) and 93 bp (primer 2) of (Supplementary Data 13 [href=this file] ).\n",
      "The age range, sex and other basic information of donors is provided in Supplementary Data 1 [href=this file] .\n",
      "The primer sequences are listed in Supplementary Data 13 [href=this file] .\n",
      "Primers for promoter binding site were listed in Supplementary Data 13 [href=this file] .\n",
      "We constructed 3 batches of single-cell libraries for sequencing due to the sample collection interval (details in Supplementary Data 1 [href=this file] ).\n",
      "Cells with low-quality and very high mitochondrial genome transcripts were removed according to a sample specific cutoff (details in Supplementary Data 1 [href=this file] ).\n",
      "The reference gene sets were collected from the Molecular Signatures Database (details in Supplementary Data 3 [href=this file] and Supplementary Data 4 [href=this file] ; https://www.gsea-msigdb.org/gsea/msigdb [href=https://www.gsea-msigdb.org/gsea/msigdb] ).\n",
      "Functional scores were calculated as the average normalized expression of the corresponding gene sets, with the full gene list provided in Supplementary Data 3 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 135) for MOESM5: Further information on research design is available in the Nature Portfolio Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#Sec54'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 59) for Sec54: Source data [href=this file] are provided with this paper.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes4: Code availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes4: Code availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 336497\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 259826\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 259825\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 97 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 97 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 97 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 643 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 97 sections → 322 unique corpus documents (processed 321 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 322 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "a9292ae858a04141b9e73f554f106243",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/11 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 19523\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 5230 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-2f4ac35485bd53aab2ef1f2e998d74b9fe7985aec6229c3566c406395a189802\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 5230 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f38f1044-9734-4b92-8223-ffae9fb441a3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE200161\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"HRA003258\\\",\\n        \\\"data_repository\\\": \\\"Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD033812\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD042977\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"8026174\\\",\\n        \\\"data_repository\\\": \\\"Zenodo\\\",\\n        \\\"dataset_webpage\\\": \\\"https://zenodo.org/record/8026174\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE200161\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"HRA003258\\\",\\n        \\\"data_repository\\\": \\\"Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD033812\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD042977\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"8026174\\\",\\n        \\\"data_repository\\\": \\\"Zenodo\\\",\\n        \\\"dataset_webpage\\\": \\\"https://zenodo.org/record/8026174\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381939,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 7079,\n",
      "        \"completion_tokens\": 389,\n",
      "        \"total_tokens\": 7468,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE200161', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161', 'dataset_id': 'GSE200161'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE200161, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE200161, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161, resolved_repo geo, dataset_id GSE200161\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 2.22s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (322, 384)\n",
      "Chunking results: 322 original documents → 322 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE200161, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE200161', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200161', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'HRA003258', 'data_repository': 'Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human', 'dataset_webpage': 'https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258', 'dataset_id': 'HRA003258'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human, identifier: HRA003258, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences  GSA-Human,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences  GSA-Human\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: HRA003258, genome sequence archive in national genomics data center, china national center for bioinformation / beijing institute of genomics, chinese academy of sciences  gsa-human, https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: genome sequence archive in national genomics data center, china national center for bioinformation / beijing institute of genomics, chinese academy of sciences  gsa-human\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258, resolved_repo genome sequence archive in national genomics data center, china national center for bioinformation / beijing institute of genomics, chinese academy of sciences  gsa-human, dataset_id HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID HRA003258 found in resolved dataset page https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: HRA003258, genome sequence archive in national genomics data center, china national center for bioinformation / beijing institute of genomics, chinese academy of sciences  gsa-human, https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'HRA003258', 'data_repository': 'genome sequence archive in national genomics data center, china national center for bioinformation / beijing institute of genomics, chinese academy of sciences  gsa-human', 'dataset_webpage': 'https://ngdc.cncb.ac.cn/gsa-human/browse/HRA003258', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD033812', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812', 'dataset_id': 'PXD033812'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange Consortium, identifier: PXD033812, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for ProteomeXchange Consortium in ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD033812, proteomecentral.proteomexchange.org, http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD033812, proteomecentral.proteomexchange.org, https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD033812', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD033812', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD042977', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977', 'dataset_id': 'PXD042977'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange Consortium, identifier: PXD042977, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for ProteomeXchange Consortium in ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD042977, proteomecentral.proteomexchange.org, http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD042977, proteomecentral.proteomexchange.org, https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD042977', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD042977', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': '8026174', 'data_repository': 'Zenodo', 'dataset_webpage': 'https://zenodo.org/record/8026174', 'dataset_id': '8026174'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Zenodo, identifier: 8026174, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1033 - INFO - Found id_pattern match for Zenodo in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://zenodo.org/record/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://zenodo.org/record/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: 8026174, pdb, https://zenodo.org/record/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://zenodo.org/record/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://zenodo.org/record/8026174, resolved_repo pdb, dataset_id 8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://zenodo.org/records/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.rcsb.org/structure/, resolved_dataset_page: https://zenodo.org/records/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 890 - WARNING - Link does not match expected pattern https://www.rcsb.org/structure/ but may still be valid after redirect.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 893 - INFO - Dataset ID 8026174 found in resolved URL, accepting as valid.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: 8026174, pdb, https://zenodo.org/records/8026174\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '8026174', 'data_repository': 'pdb', 'dataset_webpage': 'https://zenodo.org/records/8026174', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 5 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 3: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 5 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 29, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 12\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #3 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC8041347 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC8041347 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 19 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#s6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for s6: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#s10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for s10: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#s10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for s10: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#s10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for s10: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#r1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for r1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 197623\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 130183\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 130182\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 39 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 39 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 39 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 208 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 39 sections → 114 unique corpus documents (processed 94 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 114 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "05160a8fdd3d4ad39927fcc140256d09",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/4 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 3227\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 1156 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-a7414d538b726c09981fda6b2777bc6e5ae9bd0b44781239402b6f8c4b8acf06\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1156 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e2a31a70-d670-47fe-9c36-64c3724d684f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE157798\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE157798\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381952,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1505,\n",
      "        \"completion_tokens\": 82,\n",
      "        \"total_tokens\": 1587,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE157798', 'data_repository': 'NCBI GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798', 'dataset_id': 'GSE157798'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI GEO, identifier: GSE157798, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1033 - INFO - Found id_pattern match for NCBI GEO in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE157798, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798, resolved_repo geo, dataset_id GSE157798\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 0.82s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (114, 384)\n",
      "Chunking results: 114 original documents → 114 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE157798, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE157798', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157798', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 2\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #4 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC6601207 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC6601207 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 26 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec6: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec7: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec14'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec14: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#sec24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for sec24: Data Availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#sec24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for sec24: Data Availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_4: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_5: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_6: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_7: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_9: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_10: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 257040\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 189232\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 189231\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 69 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 69 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 69 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 406 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 69 sections → 147 unique corpus documents (processed 259 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 147 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "49c4d573e9a445e894cd4b8f43b3f535",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/5 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 11797\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 3299 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-6659c937a31ff2d7cf7e14955d77b8684b1a178fddc098b10781828e24a0bff3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3299 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a22eacef-7b03-4041-8be8-10e00dc6d30e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012548\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012548\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE130187\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381959,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 5525,\n",
      "        \"completion_tokens\": 580,\n",
      "        \"total_tokens\": 6105,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 8 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 596 - INFO - Duplicate found and skipped: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD012548', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012548', 'dataset_id': 'PXD012548'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD012548, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD012548, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD012548, resolved_repo www.ebi.ac.uk, dataset_id PXD012548\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.05s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (147, 384)\n",
      "Chunking results: 147 original documents → 147 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD012548, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012548\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD012548', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012548', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE130187', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187', 'dataset_id': 'GSE130187'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE130187, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE130187, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187, resolved_repo geo, dataset_id GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE130187, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE130187', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130187', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 2 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 28, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 13\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #5 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC7327418 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC7327418 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 3 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#utbl0001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for utbl0001: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ecom0001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ecom0001: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 152770\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 85699\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 1 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 85698\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 39 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 39 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 39 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 125 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 39 sections → 73 unique corpus documents (processed 52 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 73 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "84b9dfc67a154909923419daea33e444",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/3 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 1908\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 826 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-fe83c405a6cc8e8433ebe9065c2eedcf5eae2a4ffa0e766645039718f94922cd\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 826 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5b3dbcd9-e861-4aef-9cec-65108750919c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD018367\\\",\\n    \\\"data_repository\\\": \\\"PRIDE\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD018367\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148483\\\",\\n    \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD018367\\\",\\n    \\\"data_repository\\\": \\\"PRIDE\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD018367\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE148483\\\",\\n    \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381963,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1085,\n",
      "        \"completion_tokens\": 152,\n",
      "        \"total_tokens\": 1237,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD018367', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD018367', 'dataset_id': 'PXD018367'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD018367, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD018367, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD018367, resolved_repo www.ebi.ac.uk, dataset_id PXD018367\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 0.54s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (73, 384)\n",
      "Chunking results: 73 original documents → 73 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD018367, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD018367\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD018367', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD018367', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE148483', 'data_repository': 'NCBI Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483', 'dataset_id': 'GSE148483'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI Gene Expression Omnibus, identifier: GSE148483, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for NCBI Gene Expression Omnibus in Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE148483, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483, resolved_repo geo, dataset_id GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE148483, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE148483', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148483', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 2 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 4\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #6 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10664992 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10664992 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 8 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec15'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec15: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec21'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec21: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#ack1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 33) for ack1: Acknowledgments [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec25'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec25: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#supplementary-material1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for supplementary-material1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#supplementary-material2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for supplementary-material2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 278834\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 206190\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 1 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 206189\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 52 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 52 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 52 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 368 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 52 sections → 176 unique corpus documents (processed 192 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 176 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "50cddbfc01754d6d8640b978ca78b8a3",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 3179\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 1144 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-01813342e26253188be145e1500a55d6707691695c95e231fba6fec105bdf12a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1144 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e7ade22c-096a-4db9-97ef-2f2cc8b1cc6f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  [\\n    {\\n      \\\"dataset_identifier\\\": \\\"GSE235490\\\",\\n      \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n      \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\\\"\\n    }\\n  ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  [\\n    {\\n      \\\"dataset_identifier\\\": \\\"GSE235490\\\",\\n      \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n      \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\\\"\\n    }\\n  ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381969,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1515,\n",
      "        \"completion_tokens\": 88,\n",
      "        \"total_tokens\": 1603,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: [{'dataset_identifier': 'GSE235490', 'data_repository': 'NCBI GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 356 - WARNING - Dataset is a list - likely malformed LLM response. Attempting to extract valid dictionaries...\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 362 - INFO - Found valid dataset in list: {'dataset_identifier': 'GSE235490', 'data_repository': 'NCBI GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI GEO, identifier: GSE235490, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1033 - INFO - Found id_pattern match for NCBI GEO in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE235490, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490, resolved_repo geo, dataset_id GSE235490\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.24s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (176, 384)\n",
      "Chunking results: 176 original documents → 176 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE235490, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 379 - INFO - Successfully processed dataset from list: {'dataset_identifier': 'GSE235490', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235490', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 4\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #7 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC11130334 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC11130334 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 16 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec7: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec13'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec13: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec13'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec13: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 24 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 5445) for MOESM1: The cohort consists of 34 PDX profiles, eighteen more than we previously reported , and now includes ten longitudinal sets (Fig. 1 [href=#Fig1] , Supplementary Fig. 1 [href=this file] ). Four of the longitudinal models altered molecular subtypes with treatment, two of which (TNBC283 & TNBC047) switched to the M subtype following NACT (Supplementary Fig. 1 [href=this file] ). An additional two sets (TNBC117 and TNBC010) had an increasingly significant association with the M subtype (Supplementary Fig. 2 [href=this file] ), but not to the extent of switching the overall classification.\n",
      "Next, we applied a series of subjective filters to remove drugs that were either pan-active or pan-inactive by applying a threshold on the observed range of AUC values (RANGE ≥ 0.5), followed by removing potential growth-confounded drug responses (Growth index versus AUC r > 0.25) (Supplementary Fig. 3 [href=this file] ).\n",
      "To better define what drove these differences, we correlated the AUCs between matched pairs and saw a high overall correlation in drug susceptibility (AUC) with substantial variability in a small subset of drugs, as discussed below (Supplementary Fig. 4a, d [href=this file] ). A more granular analysis of the correlation between technical replicates across assay plates (Supplementary Fig. 4b, c, e–g [href=this file] ) showed a low correlation in a subset of wells from an isolated region of the assay plate for PIM328 (Supplementary Fig. 4c [href=this file] ), when compared to its pre-treatment pair PIM300 (Supplementary Fig. 4b [href=this file] ). Conversely, plate effects were not detected in the technical replicates of pre-treatment model PIM388 (Supplementary Fig. 4e [href=this file] ).\n",
      "When this analysis was applied to the CTD2 dataset , “REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION” was identified as the most predictive pathway (LOOCV Pearson R = 0.72, Supplementary Fig. 5a [href=this file] ). The “REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION” pathway was also identified as the most predictive gene set (LOOCV Pearson R = 0.69) when the lasso regression analysis was applied to the GDSC1 dataset (Supplementary Fig. 5b [href=this file] ).\n",
      "Here, we identified “REACTOME_SUMOYLATION_OF_RNA_BINDING_PROTEINS”, “REACTOME_SUMOYLATION_OF_DNA_DAMAGE_RESPONSE_AND_REPAIR_PROTEINS”, “REACTOME_SUMOYLATION_OF_CHROMATIN_ORGANIZATION_PROTEINS”, and “KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS” using the CTD2 dataset (Supplementary Fig. 5c [href=this file] ) and “REACTOME_SUMOYLATION_OF_UBIQUITINYLATION_PROTEINS”, “REACTOME_SUMOYLATION_OF_CHROMATIN_ORGANIZATION_PROTEINS”, and “REACTOME_SUMOYLATION_OF_DNA_REPLICATION_PROTEINS”, using the GDSC1 dataset (Supplementary Fig. 5d [href=this file] ).\n",
      "When the LOOCV lasso regression analysis was applied to our dataset, we identified the closely related sub-pathway “REACTOME_SUMO_IS_CONJUGATED_TO_E_UBA2_SAE1” (LOOCV Pearson R = 0.44) and “BIOCARTA_SUMO_PATHWAY” (LOOCV Pearson R = 0.41) among the top 50 pathways that predicted pevonedistat activity (Supplementary Fig. 5e [href=this file] ).\n",
      "Following Lehman et al. we evaluated the correlation between the continuous TNBCtype coefficients and drug response and found an association between pevonedistat activity and the BL1 and M TNBC subtypes (Supplementary Fig. 6 [href=this file] ).\n",
      "We chose three longitudinal pairs (TNBC231: PIM025 and PIM038, TNBC283: PIM190 and PIM231, TNBC139: PIM254 and PIM311) that showed strong enhancement of pevonedistat response in mid- or post-NACT tumors relative to pre-NACT tumors in vitro and one longitudinal pair that showed robust response in both the pre-NACT and post-NACT settings (TNBC249: PIM137 and PIM172) (Supplementary Fig. 7 [href=this file] ). This drug regimen was well-tolerated as there were no significant changes in activity or body weight between vehicle and pevonedistat-treated animals (Supplementary Fig. 8 [href=this file] ). Resistance to pevonedistat was not observed in PIM254 tumors, but a subset of PIM311 tumors developed resistance within two months of treatment (Supplementary Fig. 9 [href=this file] ).\n",
      "While we could appreciate heterogeneity in the nuclear/cytoplasmic staining patterns within individual tumors, the general high-to-low trends of NEDD8 and SUMO1 abundance were maintained whether we quantified nuclear, cytoplasmic, or cellular staining intensity (Supplementary Fig. 10 [href=this file] ).\n",
      "Importantly, this gene set captures multiple components of post-translational modifications, including sumoylation and neddylation machinery (Supplementary Fig. 11 [href=this file] ), which is consistent with what was identified when analyzing the transcriptomes of the longitudinal PDX models (Fig. 3b [href=#Fig3] ), providing further evidence of the connection between this pathway and pevonedistat response in chemoresistant TNBC.\n",
      "All PDX models tested here had a median robust Z’ greater than 0.5 (Supplementary Fig. 12A, C [href=this file] ). Similarly, we observed strong reproducibility with low MSR values, with only three MSR values greater than three (Supplementary Fig. 12B, C [href=this file] ).\n",
      "We applied ComBat and PCA to normalize and monitor the presence of technical artifacts (Supplementary Fig. 13 [href=this file] ).\n",
      "We verified that the best matches (by correlation) of each tumor sample was against other PDX, patient tumor, or patient germline samples from the same patient (Supplementary Fig. 14 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 134) for MOESM2: Further information on research design is available in the Nature Research Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 210) for MOESM3: We performed unbiased high throughput drug viability screens on this panel of 34 PDX models using a library of 618 mechanistically annotated probes and oncology drugs (Supplementary Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 246) for MOESM4: Drug classes that tended to become less active in mid-NACT or post-NACT versus pre-NACT PDX models included anthracyclines, taxanes, and multiple serine/threonine and receptor tyrosine kinase inhibitors (Supplementary Table 2 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 273) for MOESM5: Next, we performed a differential gene expression analysis and identified 341 genes that were differentially expressed across time points (DEGs; Supplementary Table 3 [href=this file] ) when tumors from the same patients were paired, compared to only 6 DEGs when unpaired.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 367) for MOESM6: These data are further supported by an analysis of previously published RNA-seq data from a PDX model treated with AC therapy , which showed significant enrichment of genes in the KEGG ubiquitin-mediated proteolysis and multiple sumoylation and neddylation pathways in residual tumor cells after AC when compared to vehicle (Supplementary Table 4 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 173) for MOESM7: The overall quality of the gene expression data was assessed using standard metrics (e.g., total read count, percent mapped reads; Supplementary Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 257587\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 172903\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 172902\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 68 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 68 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 68 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 309 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 68 sections → 160 unique corpus documents (processed 149 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 160 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "ed2fe0fcbf644e28ad8979581aab5be2",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/5 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 11954\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 3338 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-91fd8068cce3d91c81db160ba71954641e1d7aa3f87e0bc434a623f3d04309cc\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3338 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b64248d9-86c9-4edd-9c14-6ccf252f92e1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE264252\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE264252\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381974,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 4929,\n",
      "        \"completion_tokens\": 83,\n",
      "        \"total_tokens\": 5012,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE264252', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252', 'dataset_id': 'GSE264252'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE264252, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE264252, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252, resolved_repo geo, dataset_id GSE264252\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.11s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (160, 384)\n",
      "Chunking results: 160 original documents → 160 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE264252, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE264252', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264252', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 9\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #8 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC5659845 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC5659845 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 81 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec006: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec006: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec006: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec006: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec006: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec008: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec008: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec008: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec008: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec009'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec009: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec011'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec011: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 4 <a> elements with href='#pone.0186386.s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 726) for pone.0186386.s001: Indeed, in both Hsp90-dependent networks, the connectivity parameters, such as the percentage of hits included ( S1A Fig [href=this file] .) and the overall number of connections ( S1B Fig [href=this file] ) are much higher than for random genes (748 and 952 connections versus 12 and 16, p<0.001). The determined score is higher than 90 for both directions, while being close to 0 for the similar sized random gene groups ( S1C Fig [href=this file] ), showing that almost all predicted genes behaved as expected (p-value < 0.001). This is not only true for the averaged hit lists but also for each of the biological replicates ( S1C Fig [href=this file] ), which confirms the high data quality of each individual experiment.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pone.0186386.s002'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 543) for pone.0186386.s002: Thus the three biological replicates generate similar responses on the clustered level and this correlation is also evident from the same top hits being upregulated in all three experiments ( S2 [href=this file] – S4 [href=#pone.0186386.s004] Figs).\n",
      "The results for the respective replicate on the single-gene basis are show in S2 [href=this file] and S3 [href=#pone.0186386.s003] Figs and S4B Fig [href=#pone.0186386.s004] ) Calculation of the UpRegScore for each gene cluster in the microarray experiments for RNAi against or (left side), .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pone.0186386.s003'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 293) for pone.0186386.s003: The results for the respective replicate on the single-gene basis are show in S2 [href=#pone.0186386.s002] and S3 [href=this file] Figs and S4B Fig [href=#pone.0186386.s004] ) Calculation of the UpRegScore for each gene cluster in the microarray experiments for RNAi against or (left side), .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pone.0186386.s004'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 543) for pone.0186386.s004: Thus the three biological replicates generate similar responses on the clustered level and this correlation is also evident from the same top hits being upregulated in all three experiments ( S2 [href=#pone.0186386.s002] – S4 [href=this file] Figs).\n",
      "The results for the respective replicate on the single-gene basis are show in S2 [href=#pone.0186386.s002] and S3 [href=#pone.0186386.s003] Figs and S4B Fig [href=this file] ) Calculation of the UpRegScore for each gene cluster in the microarray experiments for RNAi against or (left side), .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#pone.0186386.s005'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 482) for pone.0186386.s005: The corresponding results on the single-gene basis are shown in S5 [href=this file] – S8 Fig [href=#pone.0186386.s008] .\n",
      "To quantify the overlap in the respective clusters, we calculated the UpRegScore for each of our clusters ( Fig 3B [href=#pone.0186386.g003] , S5 Fig [href=this file] ). Clearly the cluster up_1 is regulated similarly in these two experiments (UpRegScore of 80, p<0.001) while the other clusters do not show an overlapping response ( S5 Fig [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pone.0186386.s006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 141) for pone.0186386.s006: The same is true for the response to depletion of (UpRegScore of 84, p< 0.001, Fig 3B [href=#pone.0186386.g003] , S6 Fig [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pone.0186386.s007'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 323) for pone.0186386.s007: To directly test the overlap of up_1 with the pathogen-induced immune response, we evaluated the response of our clusters during immune responses against and [ 36 [href=#pone.0186386.ref036] , 37 [href=#pone.0186386.ref037] ] ( Fig 3B [href=#pone.0186386.g003] , S7 [href=this file] and S8 [href=#pone.0186386.s008] Figs).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pone.0186386.s008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 444) for pone.0186386.s008: The corresponding results on the single-gene basis are shown in S5 [href=#pone.0186386.s005] – S8 Fig [href=this file] .\n",
      "To directly test the overlap of up_1 with the pathogen-induced immune response, we evaluated the response of our clusters during immune responses against and [ 36 [href=#pone.0186386.ref036] , 37 [href=#pone.0186386.ref037] ] ( Fig 3B [href=#pone.0186386.g003] , S7 [href=#pone.0186386.s007] and S8 [href=this file] Figs).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#pone.0186386.s009'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 569) for pone.0186386.s009: Using this resource, we determined the ranking number for each clustered gene ( Fig 6A [href=#pone.0186386.g006] , S9 Fig [href=this file] ) and tested whether any of our clusters show significant DAF-16 correlation. It is evident that cluster Up_1 in particular contains genes which are regulated by DAF-16 ( Fig 6B [href=#pone.0186386.g006] left plot, S9 Fig [href=this file] ). Clusters Down_1 and Down_4 also show enrichment of DAF-16 targets, but they are formed from genes repressed by DAF-16/PQM-1 ( Fig 6C [href=#pone.0186386.g006] , S9 Fig [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pone.0186386.s010'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 225) for pone.0186386.s010: Proteins upregulated after Hsp90-depletion in both experiments included many known heat-shock proteins and other proteins associated with the chaperone system ( Fig 1B [href=#pone.0186386.g001] , S1 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pone.0186386.s011'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 138) for pone.0186386.s011: Several proteins also showed reduced levels in Hsp90-depleted nematodes ( Fig 1B [href=#pone.0186386.g001] , S2 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#ack1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 33) for ack1: Acknowledgments [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 335407\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 253950\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 253949\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 69 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 69 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 69 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 444 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 69 sections → 195 unique corpus documents (processed 249 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 195 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "1f2664644fe54a129c5df5d75400a914",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/7 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 18227\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 4906 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-ad4c596eead48376d7868fd5682cd73cc7626c0661a19dfce8c737375b491b24\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 4906 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4c517304-aeb7-4b9b-9bd2-e9fbe23a059b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE105030\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE105030\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381981,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 6291,\n",
      "        \"completion_tokens\": 83,\n",
      "        \"total_tokens\": 6374,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE105030', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030', 'dataset_id': 'GSE105030'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE105030, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE105030, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030, resolved_repo geo, dataset_id GSE105030\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.66s (0.009s per chunk)\n",
      "Corpus embedding completed. Shape: (195, 384)\n",
      "Chunking results: 195 original documents → 195 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE105030, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE105030', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE105030', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 82, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 13\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #9 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC5013304 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC5013304 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 11 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec9: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec9: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec9: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec11: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec19'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec19: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#db_ds_supplementary-material1_reqid_'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for db_ds_supplementary-material1_reqid_: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 253689\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 186623\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 1 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 186622\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 55 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 55 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 55 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 413 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 55 sections → 170 unique corpus documents (processed 243 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 170 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "05b11086d14d4f9f977c72d0e4e2b624",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 2600\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 999 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-ddebc3b95d0923228a6c5c94085c3a4438f3d1f6e939fe1b7163f3953ed8866c\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 999 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bb2f8477-8ced-443b-bba4-e785dced6bdb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD003324\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD003324\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD003324\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD003324\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381986,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1671,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 1747,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 443 - WARNING - Initial JSON parsing failed. Attempting json_repair on [\n",
      "    [\n",
      "        \"dataset_identifier\": \"PXD003324\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"data...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 463 - INFO - After malformed pattern fix: [\n",
      "    {\"dataset_identifier\": \"PXD003324\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD003324\"\n",
      "    }\n",
      "]...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 472 - INFO - Successfully parsed after repair: <class 'list'> with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD003324', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD003324', 'dataset_id': 'PXD003324'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD003324, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD003324, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD003324, resolved_repo www.ebi.ac.uk, dataset_id PXD003324\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.30s (0.008s per chunk)\n",
      "Corpus embedding completed. Shape: (170, 384)\n",
      "Chunking results: 170 original documents → 170 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD003324, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD003324\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD003324', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD003324', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 5\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #10 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC9213489 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC9213489 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 34 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#Sec1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 30) for Sec1: Introduction [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#Sec9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 28) for Sec9: Discussion [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec25'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec25: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 81 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 16748) for MOESM1: As shown in Supplementary Fig. 1a, b [href=this file] , we unexpectedly found that regardless of a high or low inocula, fungal loads in kidney, spleen and brain were maintained at a constantly high level over the first 7-day experimental period, which normally represents a time window for induction of innate immunity . Notably, infection with pathogenic SC5314 yeast cells (5 × 10 CFUs) continuously yielded high tissue fungal burdens during the course of experiments (Supplementary Fig. 1c [href=this file] ), evidence on disease causality. Moreover, histopathologic examination of Periodic acid-Schiff (PAS) and hematoxylin and eosin (H&E)-stained sections from -infected mice confirmed abundant tissue colonization of aggregate yeast cells with no pseudohyphae and filaments, however, showed less severe inflammation and tissue damage and had much smaller multifocal areas of abscess formation than did -infected mice (Supplementary Fig. 1d–f [href=this file] ).\n",
      "Longer incubation periods ( ., 24 h) were not considered in this study, since we observed robust cell death in BMDMs 24 h after co-incubation (Supplementary Fig. 2 [href=this file] ). When yeast-form cells of or were inactivated by 4% paraformaldehyde (PFA-killed) and treated with BMDMs, we also found that was able to stimulate higher levels of cytokine/chemokine gene expression (Supplementary Fig. 3 [href=this file] ), minimizing a possible effect of hyphal induction during co-culture.\n",
      "Principal component analysis (PCA) performed on accurately normalized data showed that mapped RNA-seq reads of both PBS and were clustered together, with minor variations between the biological replicates, whereas reads of exhibited major variance, as a clear separation from PBS and was observed (Fig. 1c [href=#Fig1] , Supplementary Fig. 4a [href=this file] ), suggesting that the host response of was very similar to that of PBS but significantly separated from that of . By contrast, treating BMDMs with C. significantly increased the number of DEGs, with 201 and 884 genes at the 3 and 6 h time points, respectively (Fig. 1d [href=#Fig1] , Supplementary Fig. 4b [href=this file] ). KEGG pathway enrichment analysis of DEGs (Supplementary Data 3 [href=#MOESM6] and 4 [href=#MOESM7] ) showed that the host response specific for stimulation includes genes related to chemokine, Th17, Toll-like receptor, JAK-STAT, NF-κB, and MAPK signaling pathways, whereas it is not the case for the response to (Fig. 1e [href=#Fig1] , Supplementary Fig. 4c [href=this file] ). This tendency became even more obvious when we compared changes in the expression of selected innate immune-related genes during the interaction of BMDMs with PBS, or s (Fig. 1f [href=#Fig1] , Supplementary Fig. 4d [href=this file] ).\n",
      "Interestingly, we observed peak levels of phosphorylated ERK, JNK, and p38 MAPKs in BMDMs after co-culture with live cells for 15 min (Fig. 1g [href=#Fig1] ), and at this time, the cells were still uniformly yeast-form (Supplementary Fig. 4e [href=this file] ), downplaying the influence of hyphal formation on host MAPK activation.\n",
      "Interestingly, no statistically significant difference was observed in adhesion between skin-derived HaCat and gut-derived Caco-2 (Supplementary Fig. 5a [href=this file] ), and an in vitro infection model using the commercial three-dimensional reconstructed epidermis-EpiSkin also verified the weak adherence ability of (Supplementary Fig. 5b, c [href=this file] ). However, in vivo model of murine skin, topical exposure revealed that established persistent colonization on the skin surface (Supplementary Fig. 5d, e [href=this file] ). Moreover, we found that unlike , failed to be endocytosed by Caco-2 cells (Supplementary Fig. 5f, g [href=this file] , Supplementary Movies 1 [href=#MOESM9] , 2 [href=this file0] ) and some other epithelial or endothelial cells (Supplementary Movies 3 [href=this file1] – 10 [href=this file8] ). Second, we investigated host-pathogen interactions by examining the abilities of different immune cells, including macrophages and neutrophils from humans and mice, to phagocytose and kill yeast cells in vitro. or cells were co-cultured over time with mouse BMDMs and primary human monocyte-derived macrophages (MDMs), respectively, and initial recognition by macrophages was investigated by live video microscopy (Supplementary Movies 11 [href=this file9] – 14 [href=#MOESM22] ). In both human and mouse macrophages, showed reduced rates of uptake compared to (Fig. 2f, g [href=#Fig2] , Supplementary Fig. 6a, b [href=this file] ) and the phagocytic index of was also less than that of after 30 and 60 min of co-culture (Fig. 2f, h [href=#Fig2] , Supplementary Fig. 6a, c [href=this file] ). Similar results were observed for PFA-killed yeast-form cells of both strains (Supplementary Fig. 6d–f [href=this file] ). Compared to , appeared to be taken up and internalized by neutrophils at lower rates (Fig. 2i–k [href=#Fig2] , Supplementary Fig. 6g–i [href=this file] ). Importantly, the weaker interaction between and innate immune cells correlated with stronger resistance to intracellular killing by phagocytic cells (Fig. 2l, m [href=#Fig2] , Supplementary Fig. 6j, k [href=this file] ). Indeed, further analysis using fluorescent microscopy proved that compared to , caused a small, but significant fall in the percentage of uptake (Supplementary Fig. 6l [href=this file] ) and phagocytic index (Supplementary Fig. 6m [href=this file] ), as well as less damage to the phagocytes like BMDMs (Fig. 2n [href=#Fig2] ).\n",
      "We further evaluated innate immune responses using the mouse model described above (Supplementary Fig. 1 [href=this file] ). It has to be noted that different inoculum doses of (5 × 10 CFU/mice) and (1 × 10 CFU/mice) were used, largely because of the fact that mice inoculated with at 1 × 10 CFU experience a clinical illness manifested by ruffled fur, reduced activity and weight loss, and rapidly reach the humane endpoints (within 5 days) (Supplementary Fig. 7a [href=this file] ). It has to be noted that all mice infected with cells survived and appeared to be clinically normal, although we did observe abnormal symptoms, including slight weight loss, head bobbing and body spinning, in mice receiving a high inoculum (2 × 10 CFU/mice) (Supplementary Fig. 7b, c [href=this file] ). Consistent with the results in Supplementary Fig. 1 [href=this file] , we observed in Fig. 3a [href=#Fig3] and Supplementary Fig. 8a [href=this file] that following inoculation, the fungal burden was fairly constant and high on day 2 and 5 post-inoculations, regardless of inoculum size.\n",
      "Consistently, we found that the distribution of innate immune cells (e.g., leukocytes, CD11b cells, macrophages, monocytes, and NK cells) mirrored the patterns described for neutrophils, is particularly present in significantly higher numbers in -infected tissues versus -infected tissues (Fig. 3d, e [href=#Fig3] and Supplementary Fig. 8b, c [href=this file] ). In terms of tissue fungal burden, there was no significant difference between and (Supplementary Fig. 8d [href=this file] ). However, we noticed that serum levels of cytokines/chemokines were significantly higher in -infected mice than in -inoculated mice (Supplementary Fig. 8e [href=this file] ) (the data on day 5 were unavailable because most mice reach the humane endpoint and have to be sacrificed before that time), minimizing the possible effect of inoculum size on innate immune activation.\n",
      "Our in vitro assays using different mammalian cells showed that all three cell types of displayed no differences in cell adherence (Supplementary Fig. 9a [href=this file] ) and macrophage cytotoxicity (Supplementary Fig. 9b [href=this file] ). Moreover, all three cell types of also failed to induce p38 MAPK activation (Supplementary Fig. 9c [href=this file] ) and cytokine production (Fig. 4a [href=#Fig4] , Supplementary Fig. 9d [href=this file] ). Consistently, when mice were challenged with either Y or FC form of in two different inoculum sizes, we observed that neither low (10 CFU/mice; L) nor high (2 × 10 CFU/mice; H) dose inoculum shows significant differences in fungal loads (Supplementary Fig. 9e, f [href=this file] ), innate immune cell populations (Fig. 4b, c [href=#Fig4] , Supplementary Fig. 9g, h [href=this file] ) and cytokine production (Fig. 4d [href=#Fig4] ), highly suggesting that innate immune evasion of appears to be morphology-independent.\n",
      "Among them, 14 strains were isolated in different geographical areas, including 4 from China (Cau-C #1–2, Clade I; Cau-C#3–4, Clade III), 2 from Japan (Cau-J #1–2, Clade II), 6 from India (Cau-I #1–4, Clade I; Cau-I #5-6, Clade III), 1 from Korea (Cau-K, Clade II), 1 from South America (Cau-S, Clade IV), and 2 strains were independently isolated following repeated serial passage of the parental strain BJCA001 in fluconazole-containing medium (BJCA001-R #1–2) (Supplementary Table 2 [href=this file] ). Fluconazole resistance in each strain was assessed by a checkerboard assay according to the respective minimal inhibitory concentration (MIC) of the drug (Supplementary Fig. 10a [href=this file] ). Similar to BJCA001, the rest of strains also failed to elicit meaningful proinflammatory response in BMDMs (MOI = 5; 3 or 6 h), as evidenced by unaltered p38 MAPK phosphorylation (Supplementary Fig. 10b [href=this file] ) and decreased expression and secretion of cytokines/chemokines (Fig. 4e, f [href=#Fig4] , Supplementary Fig. 10c, d [href=this file] ). Similar results were obtained in experiments with lower doses of inoculum (MOI = 0.1) and longer time in stimulation (24 h) (Fig. 4g [href=#Fig4] , Supplementary Fig. 10e [href=this file] ).\n",
      "The results are summarized in Supplementary Fig. 11a, b [href=this file] .\n",
      "Notably, the HPLC elution profiles of the two strains were identical (Supplementary Fig. 12 [href=this file] ) and no additional distinct peaks were present between the two HPLC chromatograms, suggesting that should have no alternative PAMPs which may differ considerably from . The changes in cell wall polysaccharide fractions in and were further confirmed by both fluorescent staining and flow cytometry (Fig. 5f [href=#Fig5] , Supplementary Fig. 13a [href=this file] ). A similar exposure pattern was observed during host-pathogen interaction (Supplementary Fig. 13b [href=this file] ). As shown in Supplementary Fig. 13c [href=this file] , same patterns of β-glucan exposure were shown in both staining methods. Confirmation of acid-labile mannan (phosphomannan) deficiency was observed by the reduced Alcian blue staining (Supplementary Fig. 13d [href=this file] ).\n",
      "Consistent with previous findings , the Δ/Δ mutant strain had a complete loss of phosphomannan (Supplementary Fig. 13e [href=this file] ) and displayed a significant reduction in macrophage uptake (Supplementary Fig. 13f [href=this file] ). Surprisingly, this mutant was still able to induce at least the same or even higher cytokine production when compared to the wild-type (Supplementary Fig. 13g [href=this file] ). Furthermore, we also found that deletion had no effect on pathogenicity because the Δ/Δ mutant was equally virulent in mice compared to its wild-type counterpart (Supplementary Fig. 13h [href=this file] ) .\n",
      "To further validate the essential role of the outer mannan layer of cell wall in protection against innate immune responses, we focused on various mannan-binding receptors in host, including the mannose receptor (MR), Mincle, DC-SIGN, TLR4, and TLR2. stimulation induced higher expression levels of these receptor-encoding genes in BMDMs than did (Supplementary Fig. 14a [href=this file] ). Moreover, blockade of Mincle or DC-SIGN in BMDMs, by using specific antibodies, resulted in unexpected cytokine responses seen either increasing or decreasing expression of cytokines in response to depending on the type of cytokine and blocking antibody tested (Supplementary Fig. 14b, c [href=this file] ).\n",
      "In , glycosylation of cell wall mannoproteins involves activities of a number of enzymes, for example, the families of fungal-specific mannosyltransferases (Supplementary Fig. 15a [href=this file] ). Searching the genome database for the number of genes encoding mannosyltransferases identified 51 in and 49 in , respectively (Supplementary Fig. 15b [href=this file] , Supplementary Data 5 [href=#MOESM8] ). Further RT-qPCR analyses indicated that the transcript levels of genes encoding the major protein - and - mannosyltransferases were significantly increased in when compared to their levels in (Supplementary Fig. 15c [href=this file] ), which may explain the observed phenotype that has an extensive outer mannan layer.\n",
      "Searching the released genome sequence of identified a gene orthologue encoding the protein product that shares 76.6% sequence similarity with Pmr1 of (Supplementary Fig. 16a [href=this file] ). In vitro analysis showed that Δ mutant of displayed significant vegetative growth defects, formed large aggregates and maintained yeast forms (Supplementary Fig. 16b–e [href=this file] ). Furthermore, similar to its counterpart in , the deletion mutant of also exhibited a weakened cell wall, exemplified by increased sensitivity to cell wall- and cell membrane-damaging agents such as Calcofluor White (CFW), Congo Red and SDS (Supplementary Fig. 16f [href=this file] ), and activation of the cell wall integrity pathway (Supplementary Fig. 16g [href=this file] ), suggesting that this gene is the functional ortholog of .\n",
      "belongs to one of the family members encoding a protein -mannosyltransferase catalyzing the initial step of protein -glycosylation (Supplementary Fig. 15a [href=this file] ) . Pmt1 shares a 55.2% sequence similarity with its orthologue Pmt1 (Supplementary Fig. 17a [href=this file] ). In vitro assays revealed that Δ mutant of had similar growth phenotypes as Δ mutant (Supplementary Fig. 17b–d [href=this file] ). Similar to Δ/Δ mutant of , Δ mutant of did exhibit growth and morphology defects in cell wall- and cell membrane-damaging agents (Supplementary Fig. 17e [href=this file] ), as well as activation of the cell wall integrity pathway (Supplementary Fig. 17f [href=this file] ).\n",
      "In yeast fungi like , -mannan outer chain synthesis is initiated by the α-(1,6)-mannosyltransferase encoded by whose protein product shares 76.6% sequence similarity with Och1 of (Supplementary Fig. 18a [href=this file] ). We, therefore, generated a Tetoff- mutant in which the endogenous promoter in BJCA001 was replaced with the tetracycline-regulatable promoter (Supplementary Fig. 18b, c [href=this file] ). When mutant cells were cultured with doxycycline, growth was unaffected on standard YPD medium (Supplementary Fig. 18d [href=this file] ) but the mannan content at the cell surface was remarkably decreased, based on fluorescent staining and flow cytometry (Supplementary Fig. 18e [href=this file] ), reconfirming the conserved function of Och1 in the elaboration of the outer mannan chains. More importantly, growth in 5 μg/ml doxycycline, which is sufficient to significantly reduce the expression of (Supplementary Fig. 18c [href=this file] ), significantly stimulated the production of cytokines (TNFα) by BMDMs (Supplementary Fig. 18f [href=this file] ), suggesting that protein glycosylation in the outer layer of cell wall also plays a key role in suppressing the innate immune response.\n",
      "All reagents and antibodies used in this study are listed in Supplementary Table 1 [href=this file] .\n",
      "All strains used in the experiments are listed in Supplementary Table 2 [href=this file] . strain BJCA001, the first clinical isolate of in China , and wild-type reference strain SC5314 , were utilized for all in vitro and in vivo studies.\n",
      "A detailed list of specific antibodies is provided in Supplementary Table 1 [href=this file] .\n",
      "A clinical score was determined to evaluate the disease severity of each animal, using parameters that were described previously and mainly include body weight, fur, lethargy, and motility (Supplementary Table 3 [href=this file] ).\n",
      "Construction of mutants was performed using protocols developed for use in , with minor modifications (Supplementary Figs. 16 [href=this file] b and 18b [href=this file] ) . The primers used for PCR amplification are listed in Supplementary Table 4 [href=this file] and plasmids used for gene deletion are listed in Supplementary Table 2 [href=this file] .\n",
      "All primers listed in Supplementary Table 4 [href=this file] .\n",
      "A total of 1 × 10 live cells were blocked with CD16/CD32 and stained with antibodies listed in Supplementary Table 1 [href=this file] . A detailed description of gating strategy is provided in Supplementary Fig. 19 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 134) for MOESM2: Further information on research design is available in the Nature Research Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 54) for MOESM3: Peer reviewer reports [href=this file] are available.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 97) for MOESM4: The entire list of DEGs is shown in Supplementary Data 1 [href=this file] and 2 [href=#MOESM5] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 97) for MOESM5: The entire list of DEGs is shown in Supplementary Data 1 [href=#MOESM4] and 2 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 378) for MOESM6: KEGG pathway enrichment analysis of DEGs (Supplementary Data 3 [href=this file] and 4 [href=#MOESM7] ) showed that the host response specific for stimulation includes genes related to chemokine, Th17, Toll-like receptor, JAK-STAT, NF-κB, and MAPK signaling pathways, whereas it is not the case for the response to (Fig. 1e [href=#Fig1] , Supplementary Fig. 4c [href=#MOESM1] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 378) for MOESM7: KEGG pathway enrichment analysis of DEGs (Supplementary Data 3 [href=#MOESM6] and 4 [href=this file] ) showed that the host response specific for stimulation includes genes related to chemokine, Th17, Toll-like receptor, JAK-STAT, NF-κB, and MAPK signaling pathways, whereas it is not the case for the response to (Fig. 1e [href=#Fig1] , Supplementary Fig. 4c [href=#MOESM1] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 208) for MOESM8: Searching the genome database for the number of genes encoding mannosyltransferases identified 51 in and 49 in , respectively (Supplementary Fig. 15b [href=#MOESM1] , Supplementary Data 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 290) for MOESM9: Moreover, we found that unlike , failed to be endocytosed by Caco-2 cells (Supplementary Fig. 5f, g [href=#MOESM1] , Supplementary Movies 1 [href=this file] , 2 [href=#MOESM10] ) and some other epithelial or endothelial cells (Supplementary Movies 3 [href=#MOESM11] – 10 [href=#MOESM18] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 289) for MOESM10: Moreover, we found that unlike , failed to be endocytosed by Caco-2 cells (Supplementary Fig. 5f, g [href=#MOESM1] , Supplementary Movies 1 [href=#MOESM9] , 2 [href=this file] ) and some other epithelial or endothelial cells (Supplementary Movies 3 [href=#MOESM11] – 10 [href=#MOESM18] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 289) for MOESM11: Moreover, we found that unlike , failed to be endocytosed by Caco-2 cells (Supplementary Fig. 5f, g [href=#MOESM1] , Supplementary Movies 1 [href=#MOESM9] , 2 [href=#MOESM10] ) and some other epithelial or endothelial cells (Supplementary Movies 3 [href=this file] – 10 [href=#MOESM18] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM12: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM13'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM13: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM14'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM14: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM15'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM15: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM16'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM16: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM17'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM17: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM18'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 289) for MOESM18: Moreover, we found that unlike , failed to be endocytosed by Caco-2 cells (Supplementary Fig. 5f, g [href=#MOESM1] , Supplementary Movies 1 [href=#MOESM9] , 2 [href=#MOESM10] ) and some other epithelial or endothelial cells (Supplementary Movies 3 [href=#MOESM11] – 10 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM19'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 480) for MOESM19: Second, we investigated host-pathogen interactions by examining the abilities of different immune cells, including macrophages and neutrophils from humans and mice, to phagocytose and kill yeast cells in vitro. or cells were co-cultured over time with mouse BMDMs and primary human monocyte-derived macrophages (MDMs), respectively, and initial recognition by macrophages was investigated by live video microscopy (Supplementary Movies 11 [href=this file] – 14 [href=#MOESM22] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM20'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM20: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM21'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM21: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM22'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 480) for MOESM22: Second, we investigated host-pathogen interactions by examining the abilities of different immune cells, including macrophages and neutrophils from humans and mice, to phagocytose and kill yeast cells in vitro. or cells were co-cultured over time with mouse BMDMs and primary human monocyte-derived macrophages (MDMs), respectively, and initial recognition by macrophages was investigated by live video microscopy (Supplementary Movies 11 [href=#MOESM19] – 14 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM23'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 211) for MOESM23: Furthermore, we investigated the interaction of murine and human neutrophils with or and also found differences in the host-pathogen interaction (Supplementary Movies 15 [href=this file] – 18 [href=#MOESM26] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM24: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM25'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM25: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 211) for MOESM26: Furthermore, we investigated the interaction of murine and human neutrophils with or and also found differences in the host-pathogen interaction (Supplementary Movies 15 [href=#MOESM23] – 18 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM27'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM27: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#Sec36'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 57) for Sec36: Source data [href=this file] are provided in this paper.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 332645\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 259504\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 259503\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 122 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 122 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 122 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 573 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 122 sections → 253 unique corpus documents (processed 320 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 253 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "cf7bb0b6694a4c4193d22e4ea4d932fb",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/8 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 16473\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 4468 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-092acd457c9828c9a7af726dacb4aa1a07ba8ed2540ae45d733406e518a7f67a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 4468 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8c691db5-bbe8-47af-b3dc-72913391aa6e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE203508\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE203508\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381992,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 7737,\n",
      "        \"completion_tokens\": 81,\n",
      "        \"total_tokens\": 7818,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE203508', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508', 'dataset_id': 'GSE203508'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GEO, identifier: GSE203508, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1012 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE203508, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508, resolved_repo geo, dataset_id GSE203508\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.75s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (253, 384)\n",
      "Chunking results: 253 original documents → 253 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE203508, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE203508', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE203508', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 35, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 30\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 719 - INFO - \n",
      "Progress: 11/20 (55.00%) | Elapsed: 00:00:56 | ETA: 00:00:46\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #11 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10442425 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10442425 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 14 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec17'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec17: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec18'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec18: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec19'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec19: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 36 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 6080) for MOESM1: Notably, the serum levels of E protein positively correlated with diagnostic coagulation parameters such as elevations in activated partial thromboplastin time (APTT), prothrombin time (PT), increased D-dimer and fibrinogen degradation products (FDP) (Fig. 1b–e [href=#Fig1] and Supplementary Fig. 1a–d [href=this file] ). Moreover, the serum levels of E protein negatively correlated with platelet counts (Fig. 1f [href=#Fig1] and Supplementary Fig. 1e [href=this file] ), though there was a significant positive correlation between the levels of CD62P in plasma, which serves as a marker for platelet activation (Supplementary Fig. 1f [href=this file] ).\n",
      "In such cases, immunofluorescence staining revealed elevated attachment of E protein to platelets, marked by CD41, at the thrombi site of pulmonary vessels (Fig. 2b [href=#Fig2] and Supplementary Fig. 2 [href=this file] ).\n",
      "We found that, like S protein, E protein could act in a dose-dependent manner to enhance ADP-dependent platelet aggregation (Fig. 3a [href=#Fig3] and Supplementary Fig. 3a [href=this file] – d [href=this file] ), P-selectin exposure (with or without thrombin; Fig. 3b [href=#Fig3] and Supplementary Fig. 3e [href=this file] ), fibrinogen (Fg) binding (Fig. 3c [href=#Fig3] and Supplementary Fig. 3e [href=this file] ), platelet spreading (Fig. 3d [href=#Fig3] and Supplementary Fig. 3f [href=this file] ) and clot retraction (Fig. 3e [href=#Fig3] and Supplementary Fig. 3g [href=this file] ). On the other hand, when incubated with endothelial cells, E protein did not seem to cause a change in the expression of ICAM-1 and VCAM-1 (Supplementary Fig. 4a [href=this file] – c [href=this file] ), an observation that might suggest that the E protein did not notably activate endothelial cells, though further studies are required to address the impact of E protein on the function of endothelial cells.\n",
      "Though these pathways were known to be mediated by a signaling cascade involving various protein kinases including the extracellular signal-regulated kinase (ERK), the c-Jun terminal kinase (JNK) and the p38 group of protein kinase (p38 MAPK) , we found that E protein treatment did not seem to affect the phosphorylation of ERK or JNK (Supplementary Fig. 5a–c [href=this file] ), while significantly promoting the phosphorylation of p38 and NF- B in platelets upon stimulation with low-dose of ADP (Fig. 4c–e [href=#Fig4] ). Consistent with previous studies, COVID-19 platelets exhibited higher reactivity, as characterized by increased aggregation capacity under ADP stimulation (Supplementary Fig. 6a [href=this file] , b [href=this file] ), elevated P-selectin exposure (Supplementary Fig. 6c [href=this file] ), increased spreading area on immobilized Fg (Supplementary Fig. 6d, e [href=this file] ), and enhanced clot contraction (Supplementary Fig. 6f [href=this file] , g [href=this file] ). Of note, both the protein expression of CD36 and the phosphorylation of p38 MAPK and NF- B were significantly upregulated in COVID-19 platelets with or without the presence of agonists (Supplementary Fig. 6h [href=this file] – k [href=this file] ).\n",
      "The result showed that SB203580 significantly blocked E protein-induced platelet aggregation upon stimulation with ADP, and this effect was more evident when platelets were treated with a lower concentration of the E protein (Fig. 4f [href=#Fig4] and Supplementary Fig. 7a [href=this file] ). Likewise, SB203580 suppressed a variety of E protein-induced platelet activation phenotypes, including P-selectin exposure (Fig. 4g [href=#Fig4] ), Fg binding (Fig. 4h [href=#Fig4] ), platelet spreading on Fg (Fig. 4i [href=#Fig4] and Supplementary Fig. 7b [href=this file] ), clot retraction (Fig. 4j [href=#Fig4] and Supplementary Fig. 7c [href=this file] ), and phosphorylation of p38 and NF- B (Fig. 4k–m [href=#Fig4] ).\n",
      "A notable membrane protein pulled down by the E protein was CD36 (Supplementary Table 2 [href=this file] ), a glycoprotein that has been implicated in mediating platelet activation and thrombosis under conditions of hyperlipidemia or inflammation . Consistently, the colocalization of the E protein and CD36 were identified on platelets isolated from COVID-19 patients (Supplementary Fig. 8 [href=this file] ).\n",
      "Notably, FA6-152 treatment attenuated the ability of E protein to promote platelet aggregation (Fig. 5e [href=#Fig5] and Supplementary Fig. 9a [href=this file] ). The anti-CD36 antibody also decreased the effect of E-protein-induced platelet -granule secretion (Fig. 5f [href=#Fig5] ), Fg binding (Fig. 5g [href=#Fig5] ), platelet spreading on immobilized Fg (Fig. 5h [href=#Fig5] and Supplementary Fig. 9b [href=this file] ), clot retraction (Fig. 5i [href=#Fig5] ; Supplementary Fig. 9c [href=this file] ), as well as E protein-induced elevation of the phosphorylation levels of p38 and NF- B (Fig. 5j–l [href=#Fig5] ).\n",
      "We first confirmed that the peripheral platelet counts in deficiency mice were not significantly changed as compared to WT mice (Supplementary Fig. 10a [href=this file] ) and reproduced the pro-thrombotic effect of E protein in WT mice (compare Fig. 6 [href=#Fig6] and Fig. 2 [href=#Fig2] ).\n",
      "The results showed that deficiency abolished various E protein-induced platelet activation phenotypes, including platelet aggregation in response to ADP (Fig. 6h [href=#Fig6] and Supplementary Fig. 10b [href=this file] , e, f [href=this file] ), platelet spreading on immobilized Fg (Fig. 6i [href=#Fig6] and Supplementary Fig. 10c [href=this file] ), clot retraction (Fig. 6j [href=#Fig6] and Supplementary Fig. 10d [href=this file] ), and increased phosphorylation of p38 and NF- B (Fig. 6k–m [href=#Fig6] ).\n",
      "The characteristics of all participants were summarized in Supplementary Table 1 [href=this file] .\n",
      "The data of the membrane proteins were searched against the UniprotKB Human Reference Proteome database ( http://www.uniprot.org/ [href=http://www.uniprot.org/] , up to date as of December 10, 2021), and the search results were filtered by peptides ≥ 2 (Supplementary Table 2 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 135) for MOESM3: Further information on research design is available in the Nature Portfolio Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM4: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes3: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 250777\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 179197\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 179196\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 70 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 70 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 70 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 363 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 70 sections → 167 unique corpus documents (processed 196 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 167 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "24cb521995e64fafaa4e06151d7ec03e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 10523\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 2980 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-3d6d286c0e9064807d23bdfcda67c0bd69d097bf14141e7a1275a410e3de88d6\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2980 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-418706b4-fbe1-4a53-bc76-98aefbe4fcac\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE214150\\\",\\n        \\\"data_repository\\\": \\\"GEO (Gene Expression Omnibus)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD037073\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE214150\\\",\\n        \\\"data_repository\\\": \\\"GEO (Gene Expression Omnibus)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD037073\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760381998,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 4125,\n",
      "        \"completion_tokens\": 159,\n",
      "        \"total_tokens\": 4284,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE214150', 'data_repository': 'GEO (Gene Expression Omnibus)', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150', 'dataset_id': 'GSE214150'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GEO (Gene Expression Omnibus), identifier: GSE214150, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for GEO  Gene Expression Omnibus  in Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE214150, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150, resolved_repo geo, dataset_id GSE214150\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.28s (0.008s per chunk)\n",
      "Corpus embedding completed. Shape: (167, 384)\n",
      "Chunking results: 167 original documents → 167 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE214150, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE214150', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214150', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD037073', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073', 'dataset_id': 'PXD037073'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange Consortium, identifier: PXD037073, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for ProteomeXchange Consortium in ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD037073, proteomecentral.proteomexchange.org, https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD037073, proteomecentral.proteomexchange.org, https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD037073', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD037073', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 2 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 7\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #12 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10967333 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10967333 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 5 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec2dot4-antibodies-13-00017'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec2dot4-antibodies-13-00017: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 5 <a> elements with href='#app1-antibodies-13-00017'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 716) for app1-antibodies-13-00017: Ratios of unlabeled and O-labeled peptides were based on peak heights in the MS1 spectra ( Figure S1 [href=this file] ).\n",
      "The high abundance of serotransferrin, which made up 66% of the sample, could be attributed to its endogenous expression by VK2 cells and subsequent transfer into the culture supernatant ( Table S1 [href=this file] ).\n",
      "We determined that the first heavy chain site is largely unoccupied ( Figure S2 [href=this file] ).\n",
      "The quantitative analysis of -glycosylation site occupancy revealed that only 49.1% of the EEQYNSTYR peptides was the N glycosylation site occupied by glycans ( Table 2 [href=#antibodies-13-00017-t002] and Figure S1 [href=this file] ).\n",
      "Supplementary Materials [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#antibodies-13-00017-s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for antibodies-13-00017-s001: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 216282\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 134423\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 134422\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 45 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 45 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 45 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 226 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 45 sections → 117 unique corpus documents (processed 109 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 117 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "dd6afa342583435b924a9fa11bb9ecd7",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/4 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 2421\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 955 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-d68cf8217af57d5e6f3425b625ceb1bb2282963b1b73ad5e609deb8100245722\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 955 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-02ad4ffe-71eb-49bc-8e3b-a8504da2adad\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD047910\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD047910\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD047910\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD047910\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382004,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1292,\n",
      "        \"completion_tokens\": 78,\n",
      "        \"total_tokens\": 1370,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD047910', 'data_repository': 'ProteomeXchange', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD047910', 'dataset_id': 'PXD047910'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange, identifier: PXD047910, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD047910, proteomecentral.proteomexchange.org, https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD047910, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD047910\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 0.87s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (117, 384)\n",
      "Chunking results: 117 original documents → 117 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 890 - WARNING - Link does not match expected pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= but may still be valid after redirect.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 893 - INFO - Dataset ID PXD047910 found in resolved URL, accepting as valid.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD047910, proteomecentral.proteomexchange.org, https://www.ebi.ac.uk/pride/archive/projects/PXD047910\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD047910', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD047910', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 3\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #13 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC10454042 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC10454042 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 5 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 17 <a> elements with href='#app1-ijms-24-12592'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 3806) for app1-ijms-24-12592: Using data-dependent nLC–MS/MS for discovery-driven assessment of the normotensive and OHT retina proteome in retired breeder male BN rats, Proteome Discoverer running the Mascot search engine and relying on the Uniprot rat protein database returned 4424 high-confidence protein identifications with at least two identified proteotypic peptides ( Table S1 of the Supporting Information [href=this file] available online). More than 1100 proteins were validated subsequently by Scaffold ( Table S2 [href=this file] ), which estimated a false discovery rate (FDR) of 0.1% at the protein level and 0.0% at the peptide level based on rigorous identification criteria and using a decoy-based method of FDR estimation. Our LFQ applied spectral counting and Fisher’s exact tests [ 32 [href=#B32-ijms-24-12592] ] with corrections for multiple testing [ 33 [href=#B33-ijms-24-12592] ] to obtain proteins with statistically significant differences in their expression in the OHT versus normotensive retinae ( Table S3 [href=this file] ).\n",
      "Proteins impacted by the induced OHT were organized into interaction networks by IPA as shown in Figure 1 [href=#ijms-24-12592-f001] (example) and Figures S1–S10 [href=this file] , as well as summarized in Table S4 [href=this file] .\n",
      "Abbreviations of proteins are listed in Table S6 [href=this file] .\n",
      "Bioinformatics also revealed over 100 canonical pathways that responded to IOP elevation ( Table S5 [href=this file] ). The protein interaction networks shown in Figure 1 [href=#ijms-24-12592-f001] b and Figures S1–S10 of the Supplementary Materials [href=this file] also label, along with RGCs, other cell types that may be involved in glaucomatous neurodegeneration and could be distinguished by bioinformatic analyses based on the results of our proteomics study.\n",
      "A subset of OHT-impacted proteins listed in Table S3 [href=this file] was chosen for targeted proteomics. The heuristic selection of target proteins aspired to achieve overlap in functions and canonical pathways among the interaction networks established through our discovery-driven study focusing on the effect of the IOP elevation in the chosen experimental glaucoma model ( Figure 1 [href=#ijms-24-12592-f001] b and Figures S1–S10 [href=this file] ).\n",
      "Abbreviations of proteins involved in the displayed IPA network ( ) are listed in Table S7 [href=this file] .\n",
      "Our discovery-driven proteomics study now presents over 200 proteins significantly affected by IOP elevation in the hypertonic saline injection model applied to male retired breeder Brown Norway rats ( Table S3 [href=this file] ). Since the pathophysiology of glaucoma is not well understood [ 4 [href=#B4-ijms-24-12592] ], many additional associations involving OHT-impacted proteins identified from our experiment and assembled to networks ( Figure 1 [href=#ijms-24-12592-f001] b and Figures S1–S10 [href=this file] ) may inspire expansion of hypothesis-driven research in the field.\n",
      "Although we could label many proteins in our networks ( Figure 1 [href=#ijms-24-12592-f001] b, Figure 2 [href=#ijms-24-12592-f002] a and Figures S1–S10 [href=this file] ) with their associated retinal cell types using IPA ’s cells and tissue overlay option, enrichment of target cells has been recommended for proteomics studies focusing on cell-specific subproteomes of the retina [ 64 [href=#B64-ijms-24-12592] ].\n",
      "Peptides for PRM were chosen with criteria including 100% identification probability, no missed cleavage and posttranslational modification, doubly charged precursor ions (z = 2), and listed with retention time windows in Table S8 [href=this file] .\n",
      "To obtain fold changes, all the technical replicates of all samples for each peptide ( Table S8 [href=this file] ) were averaged first, and then the treated samples were compared to controls.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ijms-24-12592-s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ijms-24-12592-s001: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 235799\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 168064\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 168063\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 42 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 42 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 42 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 246 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 42 sections → 107 unique corpus documents (processed 139 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 107 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "bbbb33a433764a1aa80b674e70008f24",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/4 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 3632\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 1257 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-e3e38827c73700f1f5b7033e0574e2f2dc7e6d956705d6c4637632528efea564\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1257 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-0a9c4e23-eb1e-43ea-8346-521d085da492\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD042729\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD042729\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD042729\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD042729\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382008,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1820,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 1896,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 443 - WARNING - Initial JSON parsing failed. Attempting json_repair on [\n",
      "    [\n",
      "        \"dataset_identifier\": \"PXD042729\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"data...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 463 - INFO - After malformed pattern fix: [\n",
      "    {\"dataset_identifier\": \"PXD042729\",\n",
      "        \"data_repository\": \"PRIDE\",\n",
      "        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD042729\"\n",
      "    }\n",
      "]...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 472 - INFO - Successfully parsed after repair: <class 'list'> with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD042729', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD042729', 'dataset_id': 'PXD042729'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD042729, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD042729, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD042729, resolved_repo www.ebi.ac.uk, dataset_id PXD042729\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 0.78s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (107, 384)\n",
      "Chunking results: 107 original documents → 107 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD042729, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD042729\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD042729', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD042729', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 3\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #14 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC6823857 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC6823857 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 6 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec23'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec23: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#sec31'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for sec31: DATA AVAILABILITY [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#PMC_3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for PMC_3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 246552\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 179501\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 179500\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 62 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 62 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 62 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 443 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 62 sections → 151 unique corpus documents (processed 292 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 151 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6c3c45d418d34c60991ea5d2aa92939f",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/5 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 5403\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 1700 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-70662a49afaf185f0f36ce74a4252af1515c8dd119f7b761178599ebbf4a27f0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1700 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b87df5fc-ee52-480d-b099-16e17f47e90b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012357\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012357\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012357\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012357\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382013,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2447,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 2523,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD012357', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012357', 'dataset_id': 'PXD012357'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD012357, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD012357, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD012357, resolved_repo www.ebi.ac.uk, dataset_id PXD012357\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.10s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (151, 384)\n",
      "Chunking results: 151 original documents → 151 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD012357, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012357\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD012357', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012357', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 5\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #15 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC7418398 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC7418398 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 17 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec18'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec18: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 620) for MOESM1: A total of 1508 proteins were absolutely quantified in 287 HLS9 samples using the DIA-TPA proteomic method (Additional file 1 [href=this file] : Fig. Human liver transcriptome data retrieved from the GTEx database revealed a different mRNA expression pattern (Additional file 1 [href=this file] : Fig.\n",
      "The median value of Spearman’s rho (correlation coefficient) of significantly correlated protein-protein pairs was 0.146 (Fig. 2 [href=#Fig2] c), which is lower than that (Spearman’s rho = 0.354) of the same genes at the transcript level in human livers, as reported in GTEx (Additional file 1 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 411) for MOESM2: S1 and Additional file 2 [href=this file] : Data file S1).\n",
      "Finally, the wealth of data generated by the study (Additional files 2 [href=this file] : Data file S1, 4 [href=#MOESM4] : Data file S2, 8 [href=#MOESM8] : Data file S3, 10 [href=#MOESM10] : Data file S4, 11 [href=#MOESM11] : Data file S5) provides a valuable resource for the scientific community of investigators in the field of hepatology research.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 150) for MOESM3: Most proteins had similar expression patterns across samples, although interindividual variability existed (Additional file 3 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 10 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 3092) for MOESM4: We identified 6155 pQTL variant-protein interactions having a genome-wide significance ( value < 2.99 × 10 ), involving 4886 pQTL variants and 648 proteins (Additional file 4 [href=this file] : Data file S2).\n",
      "A total of 1133 pQTL variants were found to be located in these transcriptional regulatory regions (Additional file 4 [href=this file] : Data file S2) and associated with the expression levels of 399 proteins. Among these, the metabolism of cobalamin associated B (MMAB) protein had the largest number of pQTL variants in the transcriptional regulatory regions, all of which were local pQTL (Additional file 4 [href=this file] : Data file S2). Levels of MIF and DDT proteins were highly correlated (Spearman’s rho = 0.800, value < 0.001, Fig. 5 [href=#Fig5] c), and they shared eight local pQTL variants including rs5760096 and rs5760124 (Fig. 5 [href=#Fig5] d). rs5760096 is located in a TF binding site in an enhancer region and positively associated with the expression of both DDT (beta = 0.997, value = 1.4 × 10 ) and MIF (beta = 0.490, value = 2.3 × 10 ) (Fig. 5 [href=#Fig5] a, b, Additional file 4 [href=this file] : Data file S2). rs5760124 resides in a CTCF binding site in an open chromatin region and was negatively associated with the expression of both DDT (beta = − 1.143, value = 7.7 × 10 ) and MIF (beta = − 0.607, value = 5.6 × 10 ) (Fig. 5 [href=#Fig5] a, b; Additional file 4 [href=this file] : Data file S2).\n",
      "DECR1 and PSMD4 were almost perfectly correlated (Spearman’s rho = 0.974, value < 0.001) (Fig. 6 [href=#Fig6] b, top-right) and shared 29 pQTL variants (Additional file 4 [href=this file] : Data file S2).\n",
      "In this study, rs698 was found to be a distant pQTL variant negatively associated with protein expression levels of S100 calcium-binding protein A11 (S100A11) and serpin family C member 1 (SERPINC1) (Fig. 7 [href=#Fig7] b, c; Additional file 4 [href=this file] : Data file S2), and S100A11and SERPINC1 are potential co-regulated proteins associated with pQTLs in hotspots 17 and 18 (Additional file 8 [href=#MOESM8] : Data file S3).\n",
      "This study discovered both transcriptional and post-transcriptional mechanisms involved in regulating protein expression in human livers (Additional file 4 [href=this file] : Data file S2 and Additional file 10 [href=#MOESM10] : Data file S4).\n",
      "Their protein levels were almost perfectly correlated (Spearman’s rho = 0.974, value < 0.001) (Fig. 6 [href=#Fig6] b, top-right) and shared 29 pQTL variants, of which 17 pQTL variants were in transcription regulatory regions, 11 pQTL variants were in coding genes, and 1 pQTL variant was in an intergenic region (Additional file 4 [href=this file] : Data file S2 and Additional file 10 [href=#MOESM10] : Data file S4).\n",
      "Finally, the wealth of data generated by the study (Additional files 2 [href=#MOESM2] : Data file S1, 4 [href=this file] : Data file S2, 8 [href=#MOESM8] : Data file S3, 10 [href=#MOESM10] : Data file S4, 11 [href=#MOESM11] : Data file S5) provides a valuable resource for the scientific community of investigators in the field of hepatology research.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 550) for MOESM5: Most pQTL variants were associated with the expression of a single protein, but 730 pQTL variants were associated with multiple proteins, including 67 local pQTL and 623 distant pQTL variants and 40 SNPs acting as both local and distant pQTLs (Fig. 3 [href=#Fig3] c, left; Additional file 5 [href=this file] : Fig. Most proteins had more than one pQTL variant; the median number of pQTL variants per protein was 4 (Fig. 3 [href=#Fig3] c, right) (4 and 4.5 for distant and local pQTL variants, respectively) (Additional file 5 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 4 <a> elements with href='#MOESM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 666) for MOESM6: A total of 256 genes were found to have both eQTLs and pQTLs, while 266 and 392 genes had only eQTLs and pQTLs, respectively (Additional file 6 [href=this file] : Fig. Across all genes, 1373 eQTL and 2750 pQTL associations were identified, but only 296 were shared associations found in both previous eQTL studies and the present pQTL investigation (Additional file 6 [href=this file] : Fig. The shared associations contained 256 variants (Additional file 6 [href=this file] : Fig. Of note, most of the eQTLs were local eQTLs whereas the identified pQTLs were more evenly distributed between local and distant pQTL regions (Additional file 6 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 6 <a> elements with href='#MOESM7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 701) for MOESM7: We further analyzed the colocalization of primary pQTL and eQTL signals and found that the lead pQTL variants of GMPPB, QPRT, DECR2, ETHE1, HAAO, DDAH2, GSTA2, and CNDP2 were colocalized with the previously reported eQTL variants (Additional file 7 [href=this file] : Table S1) [ 8 [href=#CR8] – 11 [href=#CR11] ].\n",
      "We further performed a colocalization analysis of primary pQTL and GWAS signals and found that the lead pQTL variant of AKR1A1 and HAAO were colocalized with the GWAS variants associated with blood protein levels and hypospadias, respectively (Additional file 7 [href=this file] : Table S2).\n",
      "Additional file 7 [href=this file] : Table S2. Additional file 7 [href=this file] : Table S3.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 7 <a> elements with href='#MOESM8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 1886) for MOESM8: We identified 53 pQTL hotspots for these correlated proteins (Fig. 4 [href=#Fig4] , Additional file 8 [href=this file] : Data file S3).\n",
      "We also identified 12 eQTL hotspots for the correlated proteins (Spearman’s rho > 0.5 or < − 0.5) using published human liver eQTL data [ 8 [href=#CR8] – 11 [href=#CR11] ] (Additional file 8 [href=this file] : Data file S3).\n",
      "Our results imply that transcriptional regulation of hepatic protein expression is a complex yet precisely regulated process, as exemplified by the pQTL variants of the macrophage migration inhibitory factor (MIF) (Fig. 5 [href=#Fig5] a) and -dopachrome tautomerase (DDT) proteins (Fig. 5 [href=#Fig5] b), which appear to be co-regulated proteins associated with pQTLs in hotspots 51 and 52 (Additional file 8 [href=this file] : Data file S3).\n",
      "In this study, rs698 was found to be a distant pQTL variant negatively associated with protein expression levels of S100 calcium-binding protein A11 (S100A11) and serpin family C member 1 (SERPINC1) (Fig. 7 [href=#Fig7] b, c; Additional file 4 [href=#MOESM4] : Data file S2), and S100A11and SERPINC1 are potential co-regulated proteins associated with pQTLs in hotspots 17 and 18 (Additional file 8 [href=this file] : Data file S3).\n",
      "Proteins associated with pQTLs in a hotspot often share the same function or belong to the same biological pathway (Additional file 8 [href=this file] : Data file S3).\n",
      "DECR1 and PSMD4 protein levels associated with pQTLs in hotspots 32 and 33 (Additional file 8 [href=this file] : Data file S3).\n",
      "Finally, the wealth of data generated by the study (Additional files 2 [href=#MOESM2] : Data file S1, 4 [href=#MOESM4] : Data file S2, 8 [href=this file] : Data file S3, 10 [href=#MOESM10] : Data file S4, 11 [href=#MOESM11] : Data file S5) provides a valuable resource for the scientific community of investigators in the field of hepatology research.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 137) for MOESM9: The median value of Spearman’s rho among proteins associated with pQTLs in hotspots was 0.826 (Additional file 9 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 5 <a> elements with href='#MOESM10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 1412) for MOESM10: The analysis predicted 595 regulators within protein-coding genes and non-coding RNA genes for 394 proteins (Additional file 10 [href=this file] : Data file S4).\n",
      "This study discovered both transcriptional and post-transcriptional mechanisms involved in regulating protein expression in human livers (Additional file 4 [href=#MOESM4] : Data file S2 and Additional file 10 [href=this file] : Data file S4). The pQTL variants associated with DDT and MIF were significantly enriched (Bonferroni-adjusted value < 0.05) in exon regions of these three non-coding RNA genes (Fig. 5 [href=#Fig5] d, Additional file 10 [href=this file] : Data file S4).\n",
      "Their protein levels were almost perfectly correlated (Spearman’s rho = 0.974, value < 0.001) (Fig. 6 [href=#Fig6] b, top-right) and shared 29 pQTL variants, of which 17 pQTL variants were in transcription regulatory regions, 11 pQTL variants were in coding genes, and 1 pQTL variant was in an intergenic region (Additional file 4 [href=#MOESM4] : Data file S2 and Additional file 10 [href=this file] : Data file S4).\n",
      "Finally, the wealth of data generated by the study (Additional files 2 [href=#MOESM2] : Data file S1, 4 [href=#MOESM4] : Data file S2, 8 [href=#MOESM8] : Data file S3, 10 [href=this file] : Data file S4, 11 [href=#MOESM11] : Data file S5) provides a valuable resource for the scientific community of investigators in the field of hepatology research.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 4 <a> elements with href='#MOESM11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 1027) for MOESM11: We found that 131 pQTL variants mapped to 201 traits (Additional file 11 [href=this file] : Data file S5), suggesting 252 protein-trait interactions (Fig. 7 [href=#Fig7] a).\n",
      "This study not only uncovered protein regulatory networks but also linked proteins to specific clinical traits, such as diseases and drug responses, through integrated pQTL and GWAS analysis (Additional file 11 [href=this file] : Data file S5). For instance, we were able to link S100A11 and SERPINC1, proteins associated with pQTLs in hotspots 17 and 18, to alcohol dependence through their distant pQTL variants located in (Additional file 11 [href=this file] : Data file S5, Fig. 7 [href=#Fig7] a).\n",
      "Finally, the wealth of data generated by the study (Additional files 2 [href=#MOESM2] : Data file S1, 4 [href=#MOESM4] : Data file S2, 8 [href=#MOESM8] : Data file S3, 10 [href=#MOESM10] : Data file S4, 11 [href=this file] : Data file S5) provides a valuable resource for the scientific community of investigators in the field of hepatology research.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#MOESM12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 661) for MOESM12: The demographic information of the donors is limited, and we summarized the gender and ethnicity information in Supplementary Table S3 (Additional file 12 [href=this file] ). To avoid possible value inflation caused by population structure, we performed a genotype principal component analysis to identify outliers (Additional file 12 [href=this file] : Fig.\n",
      "We determined the additive effects of each SNP on protein expression utilizing PLINK (version 1.9) with a linear regression model, which used ethnicity, gender, and the top three principal components identified from the genotype principal component analysis (Additional file 12 [href=this file] : Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 241063\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 159576\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 159575\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 82 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 82 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 82 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 347 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 82 sections → 190 unique corpus documents (processed 157 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 190 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "06937918fa74481fad3d77038180157a",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 20822\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 5555 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-3b54a1bccd6fafccd1821bf818c7b90686f4f0600e052b6168a21c31e1e30034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 5555 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4d008f9f-415c-41dd-9678-e74b77db8c81\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD019169\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD019169\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"v1.0.2\\\",\\n        \\\"data_repository\\\": \\\"GWAS Catalog\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ClinVar\\\",\\n        \\\"data_repository\\\": \\\"ClinVar database\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/clinvar/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PharmGKB\\\",\\n        \\\"data_repository\\\": \\\"PharmGKB\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.pharmgkb.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GTEx\\\",\\n        \\\"data_repository\\\": \\\"GTEx\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gtexportal.org/home/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD019169\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD019169\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"v1.0.2\\\",\\n        \\\"data_repository\\\": \\\"GWAS Catalog\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ClinVar\\\",\\n        \\\"data_repository\\\": \\\"ClinVar database\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/clinvar/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PharmGKB\\\",\\n        \\\"data_repository\\\": \\\"PharmGKB\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.pharmgkb.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GTEx\\\",\\n        \\\"data_repository\\\": \\\"GTEx\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gtexportal.org/home/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382021,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 7721,\n",
      "        \"completion_tokens\": 285,\n",
      "        \"total_tokens\": 8006,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD019169', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD019169', 'dataset_id': 'PXD019169'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange Consortium, identifier: PXD019169, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for ProteomeXchange Consortium in ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD019169, proteomecentral.proteomexchange.org, https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD019169, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD019169\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.32s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (190, 384)\n",
      "Chunking results: 190 original documents → 190 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 890 - WARNING - Link does not match expected pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= but may still be valid after redirect.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 893 - INFO - Dataset ID PXD019169 found in resolved URL, accepting as valid.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD019169, proteomecentral.proteomexchange.org, https://www.ebi.ac.uk/pride/archive/projects/PXD019169\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD019169', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD019169', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'v1.0.2', 'data_repository': 'GWAS Catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/', 'dataset_id': 'v1.0.2'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID v1.0.2 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GWAS Catalog, identifier: v1.0.2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(GWAS Catalog,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: GWAS Catalog\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas/, resolved_repo gwas catalog, dataset_id v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: GWAS Catalog\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: GWAS Catalog\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: v1.0.2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository gwas catalog not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: v1.0.2, gwas catalog, https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'v1.0.2', 'data_repository': 'gwas catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'ClinVar', 'data_repository': 'ClinVar database', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/clinvar/', 'dataset_id': 'ClinVar'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID ClinVar is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ClinVar database, identifier: ClinVar, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(ClinVar database,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: ClinVar database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/clinvar/, resolved_repo clinvar database, dataset_id ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/clinvar/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: ClinVar database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: ClinVar database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: https://www.ncbi.nlm.nih.gov/clinvar/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/clinvar/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: ClinVar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository clinvar database not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: ClinVar, clinvar database, https://www.ncbi.nlm.nih.gov/clinvar/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'ClinVar', 'data_repository': 'clinvar database', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/clinvar/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PharmGKB', 'data_repository': 'PharmGKB', 'dataset_webpage': 'https://www.pharmgkb.org/', 'dataset_id': 'PharmGKB'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID PharmGKB is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PharmGKB, identifier: PharmGKB, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(PharmGKB,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.pharmgkb.org/, resolved_repo pharmgkb, dataset_id PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.clinpgx.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: https://www.pharmgkb.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.pharmgkb.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: PharmGKB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository pharmgkb not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PharmGKB, pharmgkb, https://www.clinpgx.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PharmGKB', 'data_repository': 'pharmgkb', 'dataset_webpage': 'https://www.clinpgx.org/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GTEx', 'data_repository': 'GTEx', 'dataset_webpage': 'https://gtexportal.org/home/', 'dataset_id': 'GTEx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 866 - INFO - Accession ID GTEx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GTEx, identifier: GTEx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(GTEx,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://gtexportal.org/home/, resolved_repo gtex, dataset_id GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: https://gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository gtex not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GTEx, gtex, https://gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GTEx', 'data_repository': 'gtex', 'dataset_webpage': 'https://gtexportal.org/home/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 5 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 5 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 22, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 18\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #16 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC8213729 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC8213729 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 20 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec6: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#Sec9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 28) for Sec9: Discussion [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec15'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec15: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec16'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec16: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec24'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec24: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 91 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 17717) for MOESM1: Through interrogating the lncRNA expression data in the Cancer Genome Atlas (TCGA) , we identified a panel of 18 pan-cancer upregulated lncRNAs that were increased in expression in at least 19 of 20 cancer types in relation to corresponding normal tissues (Supplementary Fig. 1a [href=this file] ). Among them was melanotransferrin (MELTF, also known as MFI2) antisense RNA1 (MELTF-AS1 or MFI2-AS1) that is encoded by a gene located to the distal portion of chromosome 3q (3q29) (Supplementary Fig. 1a–c [href=this file] ), whose amplification is one of the most prevalent chromosomal abnormalities observed in various cancer types . Indeed, was the most frequently amplified gene among those that encode the pan-cancer upregulated lncRNAs (Supplementary Fig. 1d [href=this file] ). Of the five annotated MELTF-AS1 isoforms (Vega Genome Browser), the longest isoform was markedly more abundant than others in multiple cancer cell lines, including A549 and H1299 LUAD, NCI-H226 lung squamous cell carcinoma (LUSC), HCT116 colon adenocarcinoma (COAD), MCF-7 BRCA and Eca109 esophageal squamous cell carcinoma (ESCC), as shown in RT-PCR analysis with isoform targeting primers including those recognise a region spanning across exon 2 and 3 (Supplementary Fig. 1e, f [href=this file] ). PLANE consists of four exons (E1–E4), with minimum free energy modelling predicting a broadly symmetrical structure with E1 and E3 constituting each pole, whereas E2 and E4 contributing to both poles of the molecule (Supplementary Fig. 1g [href=this file] ).\n",
      "We confirmed the cancer-associated upregulation of PLANE in cohorts of formalin-fixed paraffin-embedded (FFPE) LUSC, LUAD and COAD and freshly isolated ESCC and BRCA samples compared to adjacent normal tissues (Fig. 1a [href=#Fig1] and Supplementary Fig. 2a [href=this file] ). Noticeably, despite the common increase in cancer tissues, PLANE levels did not differ among tumours of different stages (Supplementary Fig. 2b [href=this file] and Supplementary Tables 1 [href=this file] – 3 [href=this file] ). Likewise, there were no significant differences in PLANE expression between LUSC, LUAD and COAD of different groups stratified by tumour grade and patient gender as well as their median age at diagnosis (Supplementary Tables 1 [href=this file] – 3 [href=this file] ). Moreover, no significant changes were found in PLANE expression levels between COAD and colon adenomas (pre-neoplastic colon lesions), whereas PLANE expression was increased in colon adenomas compared with normal colon epithelia (Supplementary Fig. 2c [href=this file] ). Furthermore, high PLANE expression was associated with poorer overall patient survival (OS) in diverse cancer types (Fig. 1b [href=#Fig1] and Supplementary Fig. 2d [href=this file] ), implicating its broad involvement in cancer development and progression.\n",
      "Consistent with the contribution of genomic amplification to the upregulation of PLANE in some cancer tissues (Supplementary Fig. 1d [href=this file] ), qPCR analysis of genomic DNA demonstrated copy number gains in ~36% of LUSC (8 of 22) and ~4% of LUAD (1 of 24) (Fig. 1c [href=#Fig1] ). Nonetheless, similar to the pan-cancer upregulation of PLANE in tissue samples, all the cancer cell lines examined expressed higher levels of PLANE than the CCC-HSF-1 normal skin fibroblast cell line irrespective of their amplification status (Supplementary Fig. 2e [href=this file] ), suggesting that additional causal mechanisms such as transcriptional regulation are involved in the upregulation of PLANE in cancer cells.\n",
      "This predicted multiple E2F1-binding motifs located to the −314/−14 region of the proximal promoter of the gene (Supplementary Fig. 3a [href=this file] ) . Moreover, co-transfection of E2F1 selectively enhanced the transcriptional activity of the reporter whereas knockdown of E2F1 diminished reporter activity (Supplementary Fig. 3b, c [href=this file] ), supporting the notion that is transcriptionally activated by E2F1 through the identified E2F1-BR. In accordance, knockdown of E2F1 reduced the endogenous PLANE levels, whereas overexpression of E2F1 caused PLANE upregulation (Fig. 1h [href=#Fig1] and Supplementary Fig. 3d [href=this file] ). There was a trend that PLANE levels were positively associated with the levels of E2F1 in diverse cancer types in TCGA datasets (Supplementary Fig. 3e [href=this file] ). Furthermore, qPCR analysis of cohorts of freshly isolated ESCC and LUAD samples showed that the levels of PLANE were indeed correlated with E2F1 expression levels (Supplementary Fig. 3f [href=this file] ).\n",
      "The gene encoding PLANE is divergently located opposite to the protein-coding gene (Supplementary Fig. 4a [href=this file] ). Notably, high MELTF mRNA expression is also negatively associated with survival of patients with diverse types of cancers in the TCGA datasets (Supplementary Fig. 4b [href=this file] ). Nevertheless, knockdown of PLANE did not impinge upon MELTF mRNA expression (Supplementary Fig. 4c [href=this file] ), and similarly, knockdown of MELTF did not affect PLANE expression levels (Supplementary Fig. 4d [href=this file] ).\n",
      "SiRNA knockdown of PLANE inhibited cell proliferation and reduced clonogenicity in diverse cancer cell lines (Fig. 2a–c [href=#Fig2] ), which was associated with G0/G1 cell cycle arrest (Supplementary Fig. 5a [href=this file] ). Conversely, overexpression of PLANE increased, albeit moderately, proliferation in A549 and H1299 cells (Supplementary Fig. 5b [href=this file] ). Gene set enrichment analysis (GSEA) of the RNA-sequencing (RNA-seq) data from A549 cells revealed that knockdown of PLANE caused downregulation of numerous genes of signalling pathways involved in cell cycle progression, including the E2F1, G2/M checkpoint and mitotic spindle assembly pathways (Supplementary Fig. 5c [href=this file] ).\n",
      "Induced PLANE knockdown similarly triggered reductions in cell proliferation and clonogenicity and induced anchorage-independent growth in A549.shPLANE and H1299.shPLANE sublines (Fig. 2e–g [href=#Fig2] and Supplementary Fig. 5d [href=this file] ). Moreover, Dox treatment of nu/nu mice retarded the growth of A549.shPLANE.1 xenografts (Fig. 2h, i [href=#Fig2] and Supplementary Fig. 5e, f [href=this file] ). Cessation of Dox treatment restored the expression of PLANE and recovered, at least in part, the clonogenic potential in vitro and tumour xenograft growth in mice (Fig. 2f–i [href=#Fig2] and Supplementary Fig. 5e, f [href=this file] ), further consolidating the role of PLANE in tumorigenicity.\n",
      "In contrast, neither short- nor long-read RNA-seq identified significant changes in the levels of NCOR2-001 and NCOR2-005, which along with NCOR2-202, give rise to the three major NCOR2 protein isoforms, NCOR2-3, -1 and -2, respectively (Fig. 3a, b [href=#Fig3] , Supplementary Fig. 6 [href=this file] and Supplementary Data 1 [href=#MOESM4] ) . Intriguingly, knockdown or overexpression of PLANE did not alter NCOR2 pre-mRNA expression (Supplementary Fig. 7a, b [href=this file] ). Moreover, knockdown of PLANE did not affect the enrichment of the transcriptional activation mark H3K4me3 and the transcriptional repression mark H3K27me3 at the promoter (Supplementary Fig. 7c [href=this file] ) .\n",
      "The NCOR2-202 mRNA variant differs from NCOR2-001 and NCOR2-005 in its generation through an alternative 5′ splice site within intron 45 (Supplementary Fig. 8a [href=this file] ). Noticeably, the NCOR2-017 mRNA variant is also spliced out from this site (Supplementary Fig. 8a [href=this file] ). For simplicity, we hereafter collectively refer to these mRNAs as NCOR2-001/005-like mRNA variants (Supplementary Fig. 8a [href=this file] ). Due to sequence overlaps it was not feasible to specifically detect NCOR2-202 using qPCR to verify directly its increase caused by PLANE knockdown (Supplementary Fig. 6 [href=this file] ). However, by use of primers recognising the 1−138 fragment of intron 45 that is not contained by NCOR2-202, we could collectively detect the NCOR2-001/005-like mRNA variants using qPCR (Supplementary Fig. 8a [href=this file] ). In contrast, qPCR analysis using primers recognising a common region present in NCOR2-202 and the NCOR2-001/005-like mRNA variants except for NCOR2-018 revealed an increase in expression caused by knockdown of PLANE (Fig. 3d [href=#Fig3] and Supplementary Fig. 6 [href=this file] ). Importantly, qPCR analysis using primers recognising only NCOR2-018 and NCOR2-203 displayed reduced expression in PLANE knockdown cells (Supplementary Fig. 8b [href=this file] ). Moreover, analysis using primers specifically directed to NCOR2-017 showed similarly low levels of expression in cells with and without knockdown of PLANE (Supplementary Fig. 8c [href=this file] ). Taken together, these results indicate that PLANE indeed selectively inhibits the expression of the NCOR2-202 mRNA variant (Supplementary Fig. 8d [href=this file] ). Nonetheless, there was no significant relationship between PLANE and NCOR2-202 expression levels in freshly isolated ESCC and LUAD samples as shown with semi-quantitative RT-PCR (Supplementary Fig. 8e [href=this file] ), implying that additional mechanisms are likely involved in regulating NCOR2-202 mRNA expression.\n",
      "To substantiate these data, we carried out semi-quantitative RT-PCR analysis using primers flanking the splice site that generates NCOR2-202 and NCOR2-017 (Supplementary Fig. 8a [href=this file] ). Thus, PLANE inhibits NCOR2-202 expression through repressing its generation by AS (Supplementary Fig. 8d [href=this file] ).\n",
      "Immunoblotting with NCOR2 antibodies against residues near its N-terminus that are conserved in NCOR2 protein isoforms 1–3 demonstrated that knockdown of PLANE indeed caused an increase in NCOR2 protein levels (Figs. 2 [href=#Fig2] d, 3d [href=#Fig3] and Supplementary Fig. 6 [href=this file] ), whereas overexpression of PLANE led to downregulation of NCOR2 protein levels (Supplementary Fig. 9a [href=this file] ), implicating that regulation of NCOR2-202 AS variant expression is indeed associated with changes in NCOR2 protein expression. We also endeavoured to use siRNAs to selectively target the 1–138 fragment within intron 45 present in NCOR2-001 and NCOR2-005 but not NCOR2-202, an approach that would further interrogate the contribution of the increase in NCOR2-202 to the upregulation of NCOR2 protein expression caused by PLANE knockdown (Supplementary Table 4 [href=this file] ). However, we did not achieve meaningful knockdown of NCOR2-001 and/or NCOR2-005 expression after numerous attempts with two different siRNAs (Supplementary Fig. 9b [href=this file] ).\n",
      "The results showed that PLANE knockdown did not cause any significant changes in the proportions of these NCOR2 AS variants distributing to the nuclear and cytoplasmic fractions (Supplementary Fig. 9c [href=this file] ).\n",
      "To investigate the biological impact of PLANE regulation of AS production of NCOR2-202, we tested the effect of siRNAs targeting common regions of NCOR2-001, NCOR2-005 and NCOR2-202 on inhibition of cell proliferation caused by PLANE knockdown (Supplementary Fig. 6 [href=this file] ), which conceivably reflected the consequence of inhibition NCOR2-202 expression, as NCOR2 protein upregulation in PLANE knockdown cells was closely associated with the increase in NCOR2-202 (Fig. 3a–g [href=#Fig3] and Supplementary Fig. 8d [href=this file] ). As anticipated, introduction of the NCOR2 siRNAs diminished the upregulation of NCOR2 and reversed, at least partially, the reduction in cell proliferation and clonogenicity caused by knockdown of PLANE (Fig. 3j, k [href=#Fig3] and Supplementary Fig. 9d [href=this file] ). Of note, NCOR2 knockdown alone caused moderate increases in cell proliferation (Fig. 3j, k [href=#Fig3] and Supplementary Fig. 9d [href=this file] ).\n",
      "MAJIQ analysis identified 46916 and 48635 AS events across 10187 genes in cells with and without PLANE knockdown, respectively, with 55 significant LSVs apart from the NCOR2-202-generating splicing event triggered by knockdown of PLANE (delta PSI ≥ 20%; confidence threshold >95%) (Supplementary Fig. 10a [href=this file] ). By use of semi-quantitative RT-PCR we validated 5 randomly selected LSVs caused by knockdown of PLANE (Supplementary Fig. 10b [href=this file] ).\n",
      "Bioinformatics analysis using the IntaRNA program ( http://rna.informatik.uni-freiburg.de [href=http://rna.informatik.uni-freiburg.de] ) identified a potential PLANE-binding region (PLANE-BR) at intron 45 of the NCOR2 pre-mRNA that complements to a fragment enriched of duplex-forming oligonucleotides (DFO) contained in PLANE (Supplementary Fig. 11a [href=this file] ) .\n",
      "In contrast to overexpression of wild-type PLANE (Fig. 3h, i [href=#Fig3] and Supplementary Fig. 5b [href=this file] ), introduction of a PLANE mutant lacking the DFO into A549 and H1299 cells had no effect on the NCOR2-202-generating splicing event and cell proliferation (Fig. 4h, i [href=#Fig4] and Supplementary Fig. 11b [href=this file] ). Taken together with preceding data (Fig. 4c–k [href=#Fig4] and Supplementary Fig. 11b [href=this file] ), these findings indicate that the formation of the RNA–RNA duplex is required for the PLANE effects on NCOR2 pre-mRNA AS and cell proliferation.\n",
      "The most abundant protein that co-precipitated with PLANE was hnRNPM (Fig. 5a [href=#Fig5] and Supplementary Table 5 [href=this file] ), one of the hnRNP proteins that complex with heterogeneous nuclear RNA and are essential in regulating mRNA maturation processes including pre-mRNA splicing .\n",
      "To define the region of PLANE responsible for its interaction with hnRNPM, we carried out mapping experiments with PLANE mutants transcribed in vitro (Supplementary Fig. 12a [href=this file] ). We also conducted mapping experiments to identify the structural determinant for the binding of hnRNPM with PLANE. hnRNPM contains three RNA recognition motifs (RRMs) that are located at aa 72–147, aa 206–279 and aa 654–730, respectively (Supplementary Fig. 12b [href=this file] ).\n",
      "As anticipated, hnRNPM predominantly localised to the nucleus in A549 and H1299 cells (Supplementary Fig. 12c [href=this file] ). Noticeably, while a proportion of both hnRNPM and PLANE colocalised with the splicing factor SC35, a marker of nuclear speckles where the pre-mRNA splicing machinery is assembled, modified and stored , little NCOR2 pre-mRNA was found to colocalise with SC35 (Fig. 5g [href=#Fig5] and Supplementary Fig. 12d, e [href=this file] ).\n",
      "Through in silico analysis we identified multiple fragments enriched of consensus hnRNPM-binding sites (hnRNPM-BSs) in the NCOR2 pre-mRNA sequence including one site within intron 45 (Supplementary Fig. 12f [href=this file] and Supplementary Data 3 [href=#MOESM6] ). Notably, all fragments except 105067–105217 were co-precipitated with hnRNPM (Fig. 6a, b [href=#Fig6] and Supplementary Fig. 12g [href=this file] ). Instructively, knockdown of PLANE reduced the relative amounts of hnRNPM binding to the hnRNPM-BSs within intron 45 but did not affect hnRNPM associations with the 158355–158512 and 176814–177141 fragments (Fig. 6c [href=#Fig6] and Supplementary Fig. 12h [href=this file] ). In contrast, hnRNPM did not appear to be necessary for the interaction between PLANE and the hnRNPM-BSs within intron 45, as knockdown of hnRNPM did not affect the association between PLANE and the fragment (Supplementary Fig. 12i [href=this file] ).\n",
      "By use of bioinformatics analysis, we identified a Gorilla transcript that is highly homologous to human PLANE with 92% sequence similarity (Supplementary Table 6 [href=this file] ), suggesting evolutionary conservation of PLANE between Hominidae. Nonetheless, no similarity was found between PLANE and Mus musculus, a finding that precluded further testing the role of PLANE in transgenic mouse models (Supplementary Table 6 [href=this file] ). Irrespectively, our results from functional and correlative studies using human cell line models and human tissue samples suggest that PLANE contributes to cancer development and progression driven by genomic amplification of the distal portion of chromosome 3q and the cancer-promoting axis of E2F1 signalling (Supplementary Fig. 13 [href=this file] ).\n",
      "Information on cell lines is provided in Supplementary Table 7 [href=this file] .\n",
      "Information on antibodies and reagents used in this study is provided in Supplementary Tables 8 [href=this file] and 9 [href=this file] , respectively.\n",
      "Primers used in the fusion PCR are shown in Supplementary Table 10 [href=this file] .\n",
      "Primer sequences are listed in Supplementary Table 11 [href=this file] .\n",
      "Information on antibodies and primers used in this study are shown in Supplementary Tables 8 [href=this file] and 12 [href=this file] , respectively.\n",
      "The gating strategy is shown in Supplementary Fig. 14 [href=this file] .\n",
      "Information of antibodies used in this study was shown in Supplementary Table 8 [href=this file] .\n",
      "Primer sequences are shown in Supplementary Table 12 [href=this file] .\n",
      "Probe sequences are shown in Supplementary Table 12 [href=this file] .\n",
      "Information of antibodies and probes is shown in Supplementary Tables 8 [href=this file] and 12 [href=this file] , respectively.\n",
      "Proteins identified from the mass spectrometry analysis are listed in Supplementary Table 5 [href=this file] .\n",
      "Information on antibody and primers used in this study is shown in Supplementary Tables 8 [href=this file] and 12 [href=this file] .\n",
      "Antibodies used in this study are shown in Supplementary Table 8 [href=this file] .\n",
      "Primer sequences used are listed in Supplementary Table 11 [href=this file] .\n",
      "ShRNA sequences are shown in Supplementary Table 4 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#MOESM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 990) for MOESM4: Our initial short-read (150 bp) RNA-seq data showed that the NCOR2 mRNA variant, NCOR2-202 (ENST00000397355.1; grch37.ensembl.org), was the most highly upregulated transcript among those (44 transcripts) that were commonly increased after PLANE knockdown with two different siRNAs (Fig. 3a [href=#Fig3] and Supplementary Data 1 [href=this file] ). In contrast, neither short- nor long-read RNA-seq identified significant changes in the levels of NCOR2-001 and NCOR2-005, which along with NCOR2-202, give rise to the three major NCOR2 protein isoforms, NCOR2-3, -1 and -2, respectively (Fig. 3a, b [href=#Fig3] , Supplementary Fig. 6 [href=#MOESM1] and Supplementary Data 1 [href=this file] ) .\n",
      "On the other hand, while the NCOR2-001 and NCOR2-005 levels remained unchanged, the levels of NCOR2-202 was apparently increased, concurring with the upregulation of the NCOR2 protein in cells with PLANE knocked down (Fig. 3a, b, d, f, g [href=#Fig3] and Supplementary Data 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 402) for MOESM5: Quantitative visualisation of AS frequency within the gene using integrated genome viewer (IGV)-Sashimi plot revealed that the frequency of the AS event generating NCOR2-202 and NCOR2-017 was increased, whereas the frequency of the AS event giving rise to the NCOR2-001/005-like mRNA variants was reduced in cells with PLANE knockdown (Fig. 3e [href=#Fig3] and Supplementary Data 2 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 267) for MOESM6: Through in silico analysis we identified multiple fragments enriched of consensus hnRNPM-binding sites (hnRNPM-BSs) in the NCOR2 pre-mRNA sequence including one site within intron 45 (Supplementary Fig. 12f [href=#MOESM1] and Supplementary Data 3 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 134) for MOESM7: Further information on research design is available in the Nature Research Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 295439\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 222654\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 222653\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 82 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 82 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 82 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 486 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 82 sections → 206 unique corpus documents (processed 280 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 206 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "f8b6289fddfe42f08542ac93e7ca12ee",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/7 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 15693\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 4273 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-d4c5abc219be34efb5e89220f4a9af3dcc4158b8d5c86cc9bd37aeb50d3d88f5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 4273 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-eb8d4ec1-ffa7-41de-b86b-559c7162963f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE162215\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE172124\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD022747\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\\\",\\n        \\\"data_repository\\\": \\\"GEPIA\\\",\\n        \\\"dataset_webpage\\\": \\\"http://gepia.cancer-pku.cn/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA PanCancer Atlas Studies\\\",\\n        \\\"data_repository\\\": \\\"cBioPortal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cbioportal.org/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE162215\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE172124\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD022747\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange Consortium\\\",\\n        \\\"dataset_webpage\\\": \\\"http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\\\",\\n        \\\"data_repository\\\": \\\"GEPIA\\\",\\n        \\\"dataset_webpage\\\": \\\"http://gepia.cancer-pku.cn/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA PanCancer Atlas Studies\\\",\\n        \\\"data_repository\\\": \\\"cBioPortal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cbioportal.org/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382030,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 5891,\n",
      "        \"completion_tokens\": 356,\n",
      "        \"total_tokens\": 6247,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE162215', 'data_repository': 'NCBI Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215', 'dataset_id': 'GSE162215'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI Gene Expression Omnibus, identifier: GSE162215, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for NCBI Gene Expression Omnibus in Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE162215, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215, resolved_repo geo, dataset_id GSE162215\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.38s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (206, 384)\n",
      "Chunking results: 206 original documents → 206 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE162215, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE162215', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162215', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE172124', 'data_repository': 'NCBI Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124', 'dataset_id': 'GSE172124'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: NCBI Gene Expression Omnibus, identifier: GSE172124, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for NCBI Gene Expression Omnibus in Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE172124, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124, resolved_repo geo, dataset_id GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE172124, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE172124', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172124', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD022747', 'data_repository': 'ProteomeXchange Consortium', 'dataset_webpage': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747', 'dataset_id': 'PXD022747'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: ProteomeXchange Consortium, identifier: PXD022747, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for ProteomeXchange Consortium in ProteomeXchange\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1045 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD022747, proteomecentral.proteomexchange.org, http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: proteomecentral.proteomexchange.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747, resolved_repo proteomecentral.proteomexchange.org, dataset_id PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=, resolved_dataset_page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD022747, proteomecentral.proteomexchange.org, https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD022747', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD022747', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC', 'data_repository': 'GEPIA', 'dataset_webpage': 'http://gepia.cancer-pku.cn/', 'dataset_id': 'TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 853 - INFO - Accession ID TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: GEPIA, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(GEPIA,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: GEPIA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: http://gepia.cancer-pku.cn/, resolved_repo gepia, dataset_id n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: http://gepia.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: GEPIA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: GEPIA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: http://gepia.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: http://gepia.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: TCGA-LUSC, TCGA-COAD, TCGA-KIRC and TCGA-UCEC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository gepia not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: n/a, gepia, http://gepia.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gepia', 'dataset_webpage': 'http://gepia.cancer-pku.cn/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'TCGA PanCancer Atlas Studies', 'data_repository': 'cBioPortal', 'dataset_webpage': 'https://www.cbioportal.org/', 'dataset_id': 'TCGA PanCancer Atlas Studies'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 848 - INFO - Validating accession ID: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 853 - INFO - Accession ID TCGA PanCancer Atlas Studies is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: cBioPortal, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(cBioPortal,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: cBioPortal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.cbioportal.org/, resolved_repo cbioportal, dataset_id n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_identifier: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata data_repository: cBioPortal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: cBioPortal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_webpage: https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 911 - INFO - Brute-force checking old metadata dataset_id: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 723 - INFO - Brute-force checking if link points to dataset webpage: TCGA PanCancer Atlas Studies\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 919 - WARNING - Repository cbioportal not found in ontology, but preserving resolved URL.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 925 - WARNING - All validation methods failed, returning original URL to preserve information.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: n/a, cbioportal, https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cbioportal', 'dataset_webpage': 'https://www.cbioportal.org/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 5 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 3: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 5 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 25, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 13\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #17 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC5339665 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC5339665 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 26 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec11: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec11: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec11: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec18'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec18: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec18'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec18: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#T5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 414) for T5: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=this file] ; Table S4 [href=#SM4] ).\n",
      "According to the phosphoproteomic analysis in this study, phosphoproteins associated with cell growth and division, such as DivIVA and MapZ, were down-regulated in Δ (Table 5 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#s7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 40) for s7: Supplementary material [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#SM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 527) for SM1: Primers used in this study were designed according to the genome sequence of 05ZYH33 and are listed in Table S1 [href=this file] .\n",
      "To confirm whether the upstream and downstream genes of in the mutant were affected or not, the genes were amplified using the primers SSU05_0427-F/SSU05_0427-R (for upstream gene) and SSU05_0429-F/SSU05_0429-R (for downstream gene; Table S1 [href=this file] ) using the SC-19 cDNA as template.\n",
      "The primers (Table S1 [href=this file] ) were designed according to the genomic sequence of 05ZYH33.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#SM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 220) for SM2: Among these, 539 were differentially expressed (25% of expressed genes) in Δ compared with SC-19, with 320 up-regulated and 219 down-regulated ( < 0.05 and log transformed fold change > 2.0; Table S2 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#SM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 141) for SM3: More than two thirds of the genes (34 out of 54) involved in the purine metabolism pathway were repressed in Δ (Table S3 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#SM4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 218) for SM4: Of these 12 were differentially expressed phosphoproteins (DEPPs), including 9 down-regulated phosphoproteins (FC < 0.83) and 3 up-regulated phosphoproteins (FC > 1.2; Table 5 [href=#T5] ; Table S4 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#SM5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 141) for SM5: We constructed a deletion mutant of , as confirmed by RT-PCR (Figure S1A [href=this file] ) and Western blot (Figure S1B [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#SM6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 217) for SM6: The correlation between the two methods was high ( = 0.903; Figure S2A [href=this file] ).\n",
      "The transcription levels of 10 selected genes were compared between SC-19 and CΔ related to Δ (Figure S2B [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 353520\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 271641\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 1 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 271640\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 69 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 69 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 69 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 378 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 69 sections → 190 unique corpus documents (processed 188 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 190 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "36d3be5399a94c06a8e08a8ee3776b5a",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 3739\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 1284 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-64290da4086b4a1def9ee9a4cbccb154eede713bdd18a4f8112e998785216f53\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1284 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b3d9ab1b-fcf4-4123-9f08-832239026d18\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table1.DOCX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table1.DOCX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table2.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table2.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table3.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table3.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table4.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table4.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Image1.pdf\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Image1.pdf\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Image2.pdf\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Image2.pdf\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table1.DOCX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table1.DOCX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table2.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table2.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table3.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table3.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Table4.XLSX\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Table4.XLSX\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Image1.pdf\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Image1.pdf\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"Image2.pdf\\\",\\n        \\\"data_repository\\\": \\\"Figshare\\\",\\n        \\\"dataset_webpage\\\": \\\"/articles/instance/5339665/bin/Image2.pdf\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382041,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1734,\n",
      "        \"completion_tokens\": 356,\n",
      "        \"total_tokens\": 2090,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mllm_client.py - line 443 - WARNING - Initial JSON parsing failed. Attempting json_repair on [\n",
      "    [\n",
      "        \"dataset_identifier\": \"Table1.DOCX\",\n",
      "        \"data_repository\": \"Figshare\",\n",
      "        ...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 463 - INFO - After malformed pattern fix: [\n",
      "    {\"dataset_identifier\": \"Table1.DOCX\",\n",
      "        \"data_repository\": \"Figshare\",\n",
      "        \"dataset_webpage\": \"/articles/instance/5339665/bin/Table1.DOCX\"\n",
      "    },\n",
      "    {\"dataset_identifier\": \"Table2.XLS...\u001b[0m\n",
      "\u001b[97mllm_client.py - line 472 - INFO - Successfully parsed after repair: <class 'list'> with 6 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 6 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table1.DOCX', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table1.DOCX', 'dataset_id': 'Table1.DOCX'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Table1.DOCX, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Table1.DOCX, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Table1.DOCX, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Table1.DOCX: Invalid URL '/articles/instance/5339665/bin/Table1.DOCX': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Table1.DOCX?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Table1.DOCX found in resolved dataset page /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Table1.DOCX, figshare, /articles/instance/5339665/bin/Table1.DOCX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Table1.DOCX', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table1.DOCX', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table2.XLSX', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table2.XLSX', 'dataset_id': 'Table2.XLSX'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Table2.XLSX, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Table2.XLSX, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Table2.XLSX, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Table2.XLSX: Invalid URL '/articles/instance/5339665/bin/Table2.XLSX': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Table2.XLSX?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Table2.XLSX found in resolved dataset page /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Table2.XLSX, figshare, /articles/instance/5339665/bin/Table2.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Table2.XLSX', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table2.XLSX', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table3.XLSX', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table3.XLSX', 'dataset_id': 'Table3.XLSX'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Table3.XLSX, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Table3.XLSX, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Table3.XLSX, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Table3.XLSX: Invalid URL '/articles/instance/5339665/bin/Table3.XLSX': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Table3.XLSX?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Table3.XLSX found in resolved dataset page /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Table3.XLSX, figshare, /articles/instance/5339665/bin/Table3.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Table3.XLSX', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table3.XLSX', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Table4.XLSX', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table4.XLSX', 'dataset_id': 'Table4.XLSX'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Table4.XLSX, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Table4.XLSX, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Table4.XLSX, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Table4.XLSX: Invalid URL '/articles/instance/5339665/bin/Table4.XLSX': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Table4.XLSX?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Table4.XLSX found in resolved dataset page /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Table4.XLSX, figshare, /articles/instance/5339665/bin/Table4.XLSX\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Table4.XLSX', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Table4.XLSX', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Image1.pdf', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Image1.pdf', 'dataset_id': 'Image1.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Image1.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Image1.pdf, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Image1.pdf, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Image1.pdf: Invalid URL '/articles/instance/5339665/bin/Image1.pdf': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Image1.pdf?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Image1.pdf found in resolved dataset page /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Image1.pdf, figshare, /articles/instance/5339665/bin/Image1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Image1.pdf', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Image1.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'Image2.pdf', 'data_repository': 'Figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Image2.pdf', 'dataset_id': 'Image2.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Figshare, identifier: Image2.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 793 - INFO - Function call: url_to_repo_domain(Figshare,None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 801 - INFO - Extracting repo domain from URL: Figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1040 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: /articles/instance/5339665/bin/Image2.pdf, figshare, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: figshare\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: /articles/instance/5339665/bin/Image2.pdf, resolved_repo figshare, dataset_id /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 934 - WARNING - Error resolving URL /articles/instance/5339665/bin/Image2.pdf: Invalid URL '/articles/instance/5339665/bin/Image2.pdf': No scheme supplied. Perhaps you meant https:///articles/instance/5339665/bin/Image2.pdf?\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 907 - INFO - Dataset ID /articles/instance/5339665/bin/Image2.pdf found in resolved dataset page /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: /articles/instance/5339665/bin/Image2.pdf, figshare, /articles/instance/5339665/bin/Image2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': '/articles/instance/5339665/bin/Image2.pdf', 'data_repository': 'figshare', 'dataset_webpage': '/articles/instance/5339665/bin/Image2.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 6 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 6 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 32, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.45s (0.008s per chunk)\n",
      "Corpus embedding completed. Shape: (190, 384)\n",
      "Chunking results: 190 original documents → 190 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 13\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #18 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC5940180 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC5940180 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 117 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#sec016'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for sec016: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 13 <a> elements with href='#pcbi.1006088.s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 2871) for pcbi.1006088.s001: A full description of the phospho-proteomic data collection is found in the S1 Supporting Information [href=this file] Section 1.\n",
      "The method works through three main steps ( Fig 1 [href=#pcbi.1006088.g001] , complete details in S1 Supporting Information [href=this file] Section 2):\n",
      "Rectangular submodules are colored yellow or blue if their phospho-peptides showed increasing or decreasing phosphorylation upon NaCl treatment. ( Precision-recall curves were calculated using a true positive list, excluding submodules and sources (see Methods [href=#sec011] and S1 Supporting Information [href=this file] Section 3 for evaluation details). is the percentage of network proteins that are true positives, while is the percentage of true positives retrieved.\n",
      "We next compared our method to three other established procedures (see S1 Supporting Information [href=this file] Section 3 for full details). Nonetheless, our approach identified significantly more HOG network components and many more kinases ( S1 Supporting Information [href=this file] Section 3 and Fig 2B [href=#pcbi.1006088.g002] ).\n",
      "Our method outperformed the Steiner forest method by all metrics, with greater area under the precision-recall curve (AUC) and more target submodules incorporated ( Fig 2B [href=#pcbi.1006088.g002] , Table 1 [href=#pcbi.1006088.t001] , S1 Supporting Information [href=this file] Section 3).\n",
      "PathLinker networks also captured substantially fewer no-phenotype submodules that are added back to the network in the last step, preventing several insights afforded by the ILP network ( S1 Supporting Information [href=this file] Section 3). Finally, the PathLinker network structure did not conform to the typical signaling pathway topology but instead produced extensive looping for some nodes ( S2 Fig [href=#pcbi.1006088.s003] , S1 Supporting Information [href=this file] Section 3).\n",
      "Peptides were analyzed on a quadrupole-ion trap-Orbitrap hybrid Fusion or Fusion Lumos mass spectrometer (Thermo Scientific, San Jose, CA), as described in detail in S1 Supporting Information [href=this file] .\n",
      "Searches were performed against a target-decoy database of reviewed yeast proteins plus isoforms (Uniprot, downloaded January 20, 2013) using the Andromeda search algorithm with precursor search tolerance of 4.5 ppm and a product mass tolerance of 20 ppm, as further described in S1 Supporting Information [href=this file] (peptide and protein FDR < 1%).\n",
      "An overview of the approach is described below with a detailed description provided in S1 Supporting Information [href=this file] Section 2.\n",
      "We developed a multi-part objective procedure similar to our previous work [ 25 [href=#pcbi.1006088.ref025] ] and described in detail in S1 Supporting Information [href=this file] Section 2. Precision-recall analysis is described in S1 Supporting Information [href=this file] Section 3.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s002'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 288) for pcbi.1006088.s002: The networks generated from scrambled data fared significantly worse than the network inferred from real data, based on precision and recall of true positive regulators and connection of SIs to targets submodules (see S1 Fig [href=this file] and Table 1 [href=#pcbi.1006088.t001] below).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s003'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 246) for pcbi.1006088.s003: Finally, the PathLinker network structure did not conform to the typical signaling pathway topology but instead produced extensive looping for some nodes ( S2 Fig [href=this file] , S1 Supporting Information [href=#pcbi.1006088.s001] Section 3).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pcbi.1006088.s004'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 464) for pcbi.1006088.s004: We leveraged replicates at specific time points to identify 1,249 peptides from 618 proteins that showed reproducible phosphorylation changes ( S1 Table [href=this file] ) in wild-type cells responding to NaCl treatment (FDR < 0.05 or meeting our selection criteria, see Methods [href=#sec011] ).\n",
      "This identified 1,249 phospho-peptides that respond to NaCl, which were split into peptides that increased or decreased phosphorylation ( S1 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 4 <a> elements with href='#pcbi.1006088.s005'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 1438) for pcbi.1006088.s005: We identified phospho-peptides with reproducible phosphorylation defects upon NaCl treatment (see Methods [href=#sec011] ), implicating 211 defective phospho-peptides in the Δ strain and 140 in the Δ strain ( S2 Table [href=this file] ). To investigate Cdc14, we used the temperature-sensitive mutant paired with an identically treated, isogenic wild type to identify 161 phospho-peptides with in some cases subtle but reproducible phosphorylation defects (see Methods [href=#sec011] ) ( S2 Table [href=this file] ); these were linked to budding, cell polarity and filament formation, GTPase signal transduction, and kinase activity (P < 1x10 , hypergeometric test [ 45 [href=#pcbi.1006088.ref045] ] ( S3 Table [href=#pcbi.1006088.s006] ), as expected if Cdc14 is inhibited.\n",
      "Functional enrichments of the selected proteins support the approach: peptides identified as Hog1 dependent were enriched for ER to Golgi transport and cell growth, whereas Pde2 was enriched for cell growth, actin cytoskeleton, budding, cell polarity and filament formation, and guanine nucleotide exchange factors (GEFs) ( S2 Table [href=this file] ) (P < 1x10 ) [ 45 [href=#pcbi.1006088.ref045] ]. Phospho-peptides identified as Cdc14-affected were enriched for functions related to Cdc14 activity, including cell cycle, budding, cell polarity and filament formation, in addition to GTPase signal transduction and kinase activity ( S2 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 537) for pcbi.1006088.s006: To investigate Cdc14, we used the temperature-sensitive mutant paired with an identically treated, isogenic wild type to identify 161 phospho-peptides with in some cases subtle but reproducible phosphorylation defects (see Methods [href=#sec011] ) ( S2 Table [href=#pcbi.1006088.s005] ); these were linked to budding, cell polarity and filament formation, GTPase signal transduction, and kinase activity (P < 1x10 , hypergeometric test [ 45 [href=#pcbi.1006088.ref045] ] ( S3 Table [href=this file] ), as expected if Cdc14 is inhibited.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s007'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 367) for pcbi.1006088.s007: This generated 76 of peptides ( Fig 1C [href=#pcbi.1006088.g001] ), including 16 submodules (capturing 159 peptides) that were dependent on Hog1, 19 submodules (100 peptides) dependent on Pde2, 24 submodules (110 peptides) affected by Cdc14, and 17 submodules (563 peptides) with no detectible dependency on any of the three regulators ( S4 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pcbi.1006088.s008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 308) for pcbi.1006088.s008: Using this approach, we identified a total of 472 SIs for 54 of the 76 submodules ( S5 Table [href=this file] ).\n",
      "We first augmented the background network with 1,835 edges capturing SI-to-submodule edges and submodule-to-constituent or submodule-to-SI edges as described above ( S5 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 2 <a> elements with href='#pcbi.1006088.s009'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 404) for pcbi.1006088.s009: The previously inferred transcriptome-regulating network was heavily enriched for proteins involved in transcription, mRNA transport, chromatin modification, and those localized to the nucleus, among other functions including proteasome degradation ( < 1x10 , hypergeometric test [ 45 [href=#pcbi.1006088.ref045] ], S6 Table [href=this file] ).\n",
      "Complete GO enrichments are in S6 Table [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 3 <a> elements with href='#pcbi.1006088.s010'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 587) for pcbi.1006088.s010: To test these predictions, we immunoprecipitated GFP-tagged Pde2, Cdc14, Rck2, and Hog1 before and 10 min after salt stress and used mass spectrometry to identify other recovered proteins ( S7 Table [href=this file] ).\n",
      "But an interesting result emerged from IP of Pde2: nearly a quarter of co-IP’d proteins (17 of 73) were transcription factors, which is much higher than predicted by chance ( = 2.6x10 ) ( Fig 7 [href=#pcbi.1006088.g007] and S7 Table [href=this file] ).\n",
      "This included interactions between Rck2, Cdc14, Hog1, and several predicted targets ( S7 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s011'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 337) for pcbi.1006088.s011: Collectively, the constituents of PKA-connected submodules with reduced phosphorylation upon NaCl were enriched for proteins involved in budding, cell polarity and filament formation, GTPase- and cAMP-mediated signal transduction, stress response, cell wall structure, and included many kinases ( < 1x10 ) ( S8 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#pcbi.1006088.s012'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 184) for pcbi.1006088.s012: We first generated a position weight matrix (PWM) for module peptides distinguished by , adding a pseudocount for each amino acid to prevent overfitting ( S9 Table [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#pcbi.1006088.s013'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for pcbi.1006088.s013: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes1: Data Availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 369454\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 296323\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 4 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 296322\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 71 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 71 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 71 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 523 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 71 sections → 207 unique corpus documents (processed 316 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 207 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "2652e41cc0c64fb4a4ad489e6b4e5f99",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/7 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 16179\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 4394 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-80d5bb9b3546711b01f3bb7ba2f84b59cf248e61562d65476aa15cafa8d1c4dd\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 4394 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-defbe5eb-aafb-4b59-8def-5d4b6f1078bf\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD006192\\\",\\n    \\\"data_repository\\\": \\\"PRIDE\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD006192\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PXD006192\\\",\\n    \\\"data_repository\\\": \\\"PRIDE\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD006192\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382047,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 6084,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 6160,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD006192', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD006192', 'dataset_id': 'PXD006192'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD006192, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD006192, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD006192, resolved_repo www.ebi.ac.uk, dataset_id PXD006192\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.50s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (207, 384)\n",
      "Chunking results: 207 original documents → 207 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD006192, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD006192\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD006192', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD006192', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 1 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 118, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 15\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 696 - INFO - #19 function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 421 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 288 - INFO - API detected: PMC\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - INFO - Reusing existing EntrezFetcher instance with backup support.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 429 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 118 - INFO - Found PMC6976629 in backup data store (format: HTML)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 860 - INFO - Found PMC6976629 in local backup data (fast path, format: HTML)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 254 - INFO - Checking if HTML data is complete\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for data_availability_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 213 - INFO - Checking for supplementary_data_sections section in raw HTML data.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 190 - INFO - Loading target selectors for section name: supplementary_data_sections\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 223 - INFO - Found section with CSS selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 261 - INFO - HTML data contains all required sections.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 473 - INFO - HTML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 146 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 477 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Skipping raw HTML/XML/PDF saving. Param write_htmls_xmls set to False.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 499 - INFO - Parsing Raw content from format: HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 525 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 45 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mllm_client.py - line 19 - INFO - Initializing LLMClient with model: gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 83 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 36 - INFO - Metal Performance Shaders available - using Apple Silicon acceleration\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 48 - INFO - Initialized model: SentenceTransformer(\n",
      "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
      "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
      "  (2): Normalize()\n",
      ")\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 54 - INFO - Using model's actual max sequence length: 256\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 429 - INFO - Function call: parse_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 629 - INFO - Extracted 7 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 639 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 43 <a> elements with href='#MOESM1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 10759) for MOESM1: To investigate if systemic arginine manipulation modulated bone diseases, we first used recArg1 in serum transfer arthritis (K/BxN), which is characterized by RANKL-induced bone erosion (Supplementary Fig. 1a [href=this file] ) . Using a different arthritis model, we found arginine was equally important in tumour necrosis factor (TNF)-dependent arthritis (hTNF , Supplementary Fig. 1b [href=this file] ) . Finally, we systemically depleted arginine in collagen-induced arthritis (CIA), a model also dependent on adaptive immunity (Supplementary Fig. 1c [href=this file] ) . Although arginine depletion decreased and mRNA expression, no effects were observed on mRNA, indicating that arginine depletion had a preferential effect on the transcriptional program of osteoclastogenesis rather than a global effect on inflammation (Supplementary Fig. 1d [href=this file] ). Concordantly, while a secondary collagen challenge ex vivo induced robust splenocyte proliferation in diseased versus healthy animals, we detected no difference in recArg1-treated versus sham-treated arthritic animals (Supplementary Fig. 1e [href=this file] ). This was associated with increases in ornithine and other AAs (Supplementary Fig. 1f [href=this file] ). Of note, in CIA, myeloid populations including osteoclast precursors were unaffected by arginine restriction, as was the systemic RANKL/OPG ratio in all models studied (Supplementary Fig. 1g–i [href=this file] ), suggesting that decreased osteoclast numbers found in arthritis were due to differentiation of osteoclasts.\n",
      "As expected, the latter included hallmark osteoclastic genes; , , and and KEGG analysis showed enrichment for osteoclast differentiation (Supplementary Fig. 2a, b [href=this file] ). Treatment with recArg1 affected RANKL-induced gene expression already after 24 h, preceding RANKL effects on osteoclast fusion that mainly occurred at 72–96 h (Fig. 2e [href=#Fig2] , Supplementary Figs. 2c [href=this file] and 3a [href=this file] ), indicating recArg1 affects osteoclastogenesis by selectively reducing RANKL-dependent gene expression.\n",
      "Underscoring the negligible effects observed of conditional deletion on osteoclastogenesis, we observed persistent downregulation of transcript and protein levels post RANKL treatment during osteoclastogenesis of wildtype preosteoclasts (Fig. 2h–i [href=#Fig2] , Supplementary Fig. 2a [href=this file] ). RecArg1 exerted minor influences on macrophage and dendritic cell differentiation (Supplementary Fig. 4 [href=this file] ), suggesting differential environmental arginine requirements for M-CSF/GM-CSF versus RANKL signalling, the latter of which is well described to be important for multinucleated osteoclast formation.\n",
      "RANKL/Arg-Depletion demonstrated more pronounced segregation from RANKL than RANKL/Arg-Starvation samples, whilst both profiles were distinct from M-CSF conditions (Fig. 3b [href=#Fig3] and Supplementary Fig. 5a [href=this file] ). While recArg1 had the strongest effect on the RANKL transcriptome, most genes (119) significantly modified by Arg-Starvation (150) were shared with Arg-Depletion (420), although genes specific for both were identified (Supplementary Fig. 5b [href=this file] ).\n",
      "Heatmap of proteomics data in , showing -score of abundance level per protein across all conditions (RANKL-conditions shown only, full heatmap in Supplementary Fig. 5d [href=this file] ).\n",
      "To dissect the cellular and metabolic changes caused by arginine restriction during osteoclast development, we next used a proteomic approach to probe whether Arg-Depletion in preosteoclasts caused an altered proteome (Fig. 3a [href=#Fig3] , Supplementary Fig. 5c [href=this file] ). We did not observe a global translational decrease upon Arg-Starvation or Arg-Depletion, and indeed, many proteins were induced under low arginine conditions (Fig. 3d [href=#Fig3] , Supplementary Fig. 5d [href=this file] ). Examining cell cycle status revealed that while arginine was required for RANKL-induced G /M cell cycle progression that preceded cell fusion, its withdrawal was equivalent to M-CSF- treated cells, suggesting diminished cellular proliferation (Supplementary Fig. 3a, b [href=this file] ).\n",
      "Data normalized against M-CSF (left) and represent mean ( = 4, RANKL intracellular = 3; see Supplementary Fig. 7c, d [href=this file] ) Transcriptional profiles of selected KEGG pathways.\n",
      "Integrative network analysis of arginine effects on the RANKL signature, unifying transcriptomic and proteomic data, suggested that oxidative metabolism represented an important metabolic hub linked to arginine presence (Supplementary Fig. 6a [href=this file] and Supplementary Table 3 [href=this file] ). These RANKL-mediated increases were counteracted by recArg1, in a manner that was independent of changes in mitochondrial mass (Fig. 4c [href=#Fig4] and Supplementary Fig. 7a, b [href=this file] ) . Using a metabolomic approach, we observed RANKL-dependent induction of key intracellular tricarboxylic acid (TCA) cycle intermediates (e.g. malate, fumarate, succinate, α-ketoglutarate) 24 h post stimulation and only subtle changes in intracellular AAs (e.g. arginine, glycine, alanine) (Fig. 4f [href=#Fig4] , Supplementary Fig. 7c [href=this file] ). Oxygen consumption rates (OCRs) 24 h post RANKL treatment were unchanged compared to M-CSF, indicating that RANKL-induced changes in TCA metabolites preceded the observed boost in OXPHOS (Supplementary Fig. 7e [href=this file] ).\n",
      "RecArg1 further led to extracellular arginine depletion via conversion to ornithine and urea and reversed the intracellular build-up of malate, fumarate, succinate and α-ketoglutarate induced by RANKL (Fig. 4f [href=#Fig4] and Supplementary Fig. 7c, d [href=this file] ), suggesting arginine depletion was linked to a rewired TCA cycle. Indeed, recArg1 counteracted specific RANKL-induced TCA enzyme expression, while upregulating serine and purine biosynthetic enzymes (Pck2, Eno2, Phgdh) (Fig. 4g [href=#Fig4] , Supplementary Fig. 2f [href=this file] ) . Consistent with the notion of diverted biosynthetic fluxes upon arginine depletion, increased expression of the pyruvate anabolic genes and occurred (Supplementary Fig. 2d [href=this file] ).\n",
      "In line with negligible increases in arginine transporter transcription ( ) (Supplementary Fig. 2e [href=this file] ) , we observed no difference in C -labelled arginine uptake upon RANKL treatment post 24 h. Notably, as arginine was readily converted to ornithine and urea by recArg1 in the supernatant, we could barely detect labelled intracellular C arginine upon recArg1 treatment (Figs. 4f [href=#Fig4] , 5b [href=#Fig5] , Supplementary Fig. 7d [href=this file] ). Suggestive of uptake and/or synthesis from + 5 ornithine occurring under C labelling, total intracellular amounts of the polyamines putrescine and spermidine were decreased upon recArg1 treatment in comparison to RANKL (Figs. 4f [href=#Fig4] , 5c [href=#Fig5] , Supplementary Fig. 7c [href=this file] ).\n",
      "As we wanted to test if inhibitory effects of recArg1 treatment on osteoclastogenesis were mediated by decreased intracellular polyamine synthesis (Fig. 4f [href=#Fig4] , Supplementary Fig. 7c [href=this file] ), we incubated cells with alpha-difluoromethylornithine (DFMO), thereby blocking conversion of ornithine to putrescine .\n",
      "So far we showed that the arginine-dependent dysregulated TCA cycle-initiated cell cycle arrest was not associated with known properties of quiescence including decreased glycolysis, reduced translation rates or mTORC1 deactivation (Figs. 3 [href=#Fig3] , 4 [href=#Fig4] and Supplementary Fig. 3b [href=this file] ). Indeed, osteoclast formation could be restarted even after prolonged arginine starvation by arginine resupplementation and even exhibited a tendency towards being increased relative to controls (Supplementary Fig. 3c, d [href=this file] ). Notably, the Arg-Starvation-mediated inhibition of polykaryons was restored by argininosuccinate and citrulline supplementation, but not by ornithine and proline, indicating that select arginine precursor metabolites bypass the dependence of arginine for RANKL signalling (Fig. 6a [href=#Fig6] ). α-Ketoglutarate was not able to rescue arginine absence (Supplementary Fig. 3e [href=this file] ), suggesting that arginine scarcity could not be compensated by direct TCA cycle refuelling but only by AA derivatives.\n",
      "To evaluate whether other AAs were essential for RANKL cellular programming, we next cultured preosteoclasts in media selectively devoid of AAs and demonstrated commonalities in metabolic blockage during osteoclastogenesis to a nutrient-limited environment, especially regarding essential AAs (Fig. 6b [href=#Fig6] , Supplementary Fig. 8 [href=this file] , Supplementary Table 1 [href=this file] ).\n",
      "Noting that mononuclear innate immune cell in vitro differentiation efficiency is independent of arginine absence (Supplementary Fig. 4 [href=this file] ) and that RANKL induces giant cell formation, we next examined whether the concept of arginine reliance holds true for other MGCs.\n",
      "Preosteoclasts in tryptophan and cysteine-free medium were cultured in 5% dialyzed FCS to control for serum-bound AAs (Supplementary Table 1 [href=this file] ). 2-(difluoromethyl)ornithine (DFMO) was a kind gift from the Woster Lab (MUSC) and used at a concentration of 2.5 mM. and LysMcre have previously been described . (B6.129S6- /J) and Tie2 bone marrow was provided by the Murray Lab . (B6.129S(Cg)- /J) animals were acquired from the Jackson Laboratories. (unpublished) bone marrow was a kind gift from the Surperti-Furga Lab .\n",
      "The differential expression analysis was thus first based solely on counts obtained allowing multimapping reads (Figs. 2 [href=#Fig3] b, 3 [href=#Fig4] b, c, 4g [href=#Fig5] and Supplementary Fig. 2 [href=this file] ).\n",
      "This tool circumvents the low overlap between proteomic and transcriptomic hits (Supplementary Fig. 6b [href=this file] ), by inferring transcription factors likely to explain the transcriptional changes and searching for minimal sets of connected genes including these transcription factors and abundantly changed proteins. The protein interactions used were mouse–mouse interactions from BioGRID Release 3.4.160 (ref. ), weighted by directness and experimental strength of the assay (Supplementary Table 2 [href=this file] ). Major network structures were identified according to coding theory, by looking at which grouping of the nodes allows the minimal coding of trajectories of random walks on the network (Supplementary Table 3 [href=this file] ).\n",
      "Mass spectral data on the 7000 QqQ were acquired in scan mode monitoring selected ion clusters of the different metabolites (Supplementary Table 4 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for MOESM2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#MOESM3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 134) for MOESM3: Further information on research design is available in the Nature Research Reporting Summary [href=this file] linked to this article.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 0 <a> elements with href='#Sec36'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 0) for Sec36: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 650 - INFO - Found 1 <a> elements with href='#notes2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 658 - INFO - Extracted context description (len: 35) for notes2: Data availability [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 102 - INFO - Length of original HTML: 293429\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 148 - INFO - Length of normalized HTML: 220075\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 465 - INFO - Chunking the HTML content for the parsing step.\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 115 - INFO - Extracting data availability elements from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.data-availability-statement\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: section.associated-data\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 126 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 164 - INFO - Found 3 data availability elements from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 245 - INFO - Function_call: extract_sections_from_html(html_content) with content length 220074\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 325 - INFO - Extracted 73 sections from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 916 - INFO - HTML semantic retrieval from corpus with 73 sections\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 725 - INFO - Converting 73 HTML sections to embeddings corpus\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 821 - INFO - Pre-deduplication: 468 corpus documents\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 852 - INFO - HTML sections converted: 73 sections → 220 unique corpus documents (processed 248 duplicates with title merging)\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding corpus of 220 documents using sentence-transformers/all-MiniLM-L6-v2\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "3eda4d8c03e643e393f52db367562e9a",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Batches:   0%|          | 0/7 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97membeddings_retriever.py - line 169 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001b[0m\n",
      "\u001b[97membeddings_retriever.py - line 151 - INFO - Computing L2 distances using numpy.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 935 - INFO - HTML semantic retrieval completed: found 2 relevant sections\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 258 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FewShot from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 266 - INFO - Content length: 7733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 283 - INFO - Prompt messages total length: 2283 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 289 - INFO - Prompt ID: gemini-2.0-flash-0-5f106719485ef2951540c7150644dc9236f7b4b8033dfaf918a163f772d23154\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1205 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 313 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2283 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 168 - INFO - Making LLM call with model: gemini-2.0-flash, temperature: 0\u001b[0m\n",
      "\u001b[97mllm_client.py - line 223 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ee65e2f9-9d09-4826-81d3-ee1779e9315f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE125101\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012405\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012405\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE125101\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD012405\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD012405\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1760382053,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 3235,\n",
      "        \"completion_tokens\": 153,\n",
      "        \"total_tokens\": 3388,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mllm_client.py - line 400 - INFO - Function_call: _safe_parse_json_internal(response_text <class 'portkey_ai.api_resources.types.chat_complete_type.ChatCompletions'>)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 544 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 586 - INFO - Adding unique item: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'GSE125101', 'data_repository': 'Gene Expression Omnibus (GEO)', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101', 'dataset_id': 'GSE125101'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 487 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: Gene Expression Omnibus (GEO), identifier: GSE125101, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1025 - INFO - Found partial match for Gene Expression Omnibus  GEO  in Gene Expression Omnibus\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 469 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: GSE125101, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101, resolved_repo geo, dataset_id GSE125101\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Embedding time: 1.56s (0.007s per chunk)\n",
      "Corpus embedding completed. Shape: (220, 384)\n",
      "Chunking results: 220 original documents → 220 embedded chunks\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=, resolved_dataset_page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc= of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: GSE125101, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'GSE125101', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE125101', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 347 - INFO - Processing dataset: {'dataset_identifier': 'PXD012405', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012405', 'dataset_id': 'PXD012405'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 398 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Matched dataset_webpage_url_ptr: https://www.ebi.ac.uk/pride/archive/projects/__ID__ to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 454 - INFO - Candidate dataset_webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 457 - INFO - Candidate data_repository: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 974 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD012405, dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1019 - INFO - Found repo_name match for PRIDE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 484 - INFO - Setting dataset_id to value: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 472 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 477 - INFO - Extracted ID: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 466 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 494 - INFO - Schema validation vals: PXD012405, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 499 - INFO - Dataset ID found via pattern matching: PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 507 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 515 - INFO - Dataset webpage found via pattern matching: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 878 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD012405, resolved_repo www.ebi.ac.uk, dataset_id PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 931 - INFO - Resolved URL: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 885 - INFO - Pattern: https://www.ebi.ac.uk/pride/archive/projects/, resolved_dataset_page: https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 887 - INFO - Link matches the pattern https://www.ebi.ac.uk/pride/archive/projects/ of resolved_dataset_page.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 518 - INFO - Final schema validation vals: PXD012405, www.ebi.ac.uk, https://www.ebi.ac.uk/pride/archive/projects/PXD012405\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 431 - INFO - Extracted dataset: {'dataset_identifier': 'PXD012405', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD012405', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1067 - INFO - Enhancing dataset pages for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1161 - INFO - Added access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1117 - INFO - Dataset enhancement completed: 2 datasets processed\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 493 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 944 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 537 - INFO - Parsed data extraction completed. Elements collected: 7\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 570 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 580 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1274 - INFO - Output written to file: scripts/output/result.csv\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1276 - INFO - File Download Schedule: []\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1293 - INFO - Closing the EntrezFetcher.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "combined_df = dg.run(input_file=pmcids, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "9fd1be22241ced6b",
   "metadata": {
    "jupyter": {
     "is_executing": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>citation_type</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>context_description</th>\n",
       "      <th>source_url</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>raw_data_format</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE207091</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>Regulatory Effects of ABA and GA on the Expres...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PXD044299</td>\n",
       "      <td>www.ebi.ac.uk</td>\n",
       "      <td>https://www.ebi.ac.uk/pride/archive/projects/P...</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>Regulatory Effects of ABA and GA on the Expres...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.3390/ijms241512380</td>\n",
       "      <td>10.3390/ijms241512380</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>https://doi.org/10.3390/ijms241512380</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "      <td></td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>Regulatory Effects of ABA and GA on the Expres...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/10418516/bin/ijms-24-12380-...</td>\n",
       "      <td>Click here for additional data file.</td>\n",
       "      <td>(420.9KB, zip)</td>\n",
       "      <td>n/a</td>\n",
       "      <td>ijms-24-12380-s001</td>\n",
       "      <td>...</td>\n",
       "      <td>zip</td>\n",
       "      <td>/articles/instance/10418516/bin/ijms-24-12380-...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td></td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>Regulatory Effects of ABA and GA on the Expres...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE116943</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>BRCA2 deficiency instigates cGAS-mediated infl...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 22 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  dataset_identifier data_repository  \\\n",
       "0          GSE207091             geo   \n",
       "1          PXD044299   www.ebi.ac.uk   \n",
       "2                NaN             NaN   \n",
       "3                NaN             NaN   \n",
       "4          GSE116943             geo   \n",
       "\n",
       "                                     dataset_webpage citation_type  \\\n",
       "0  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...           n/a   \n",
       "1  https://www.ebi.ac.uk/pride/archive/projects/P...           n/a   \n",
       "2                                                NaN           NaN   \n",
       "3                                                NaN           NaN   \n",
       "4  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...           n/a   \n",
       "\n",
       "        access_mode                                               link  \\\n",
       "0  Complex download                                                NaN   \n",
       "1  Complex download                                                NaN   \n",
       "2               NaN              https://doi.org/10.3390/ijms241512380   \n",
       "3               NaN  /articles/instance/10418516/bin/ijms-24-12380-...   \n",
       "4  Complex download                                                NaN   \n",
       "\n",
       "                                  title       file_info description  \\\n",
       "0                                   NaN             NaN         NaN   \n",
       "1                                   NaN             NaN         NaN   \n",
       "2                 10.3390/ijms241512380             n/a         n/a   \n",
       "3  Click here for additional data file.  (420.9KB, zip)         n/a   \n",
       "4                                   NaN             NaN         NaN   \n",
       "\n",
       "       source_section  ... file_extension  \\\n",
       "0                 NaN  ...           None   \n",
       "1                 NaN  ...           None   \n",
       "2                 n/a  ...           None   \n",
       "3  ijms-24-12380-s001  ...            zip   \n",
       "4                 NaN  ...           None   \n",
       "\n",
       "                                         a_attr_href  \\\n",
       "0                                                NaN   \n",
       "1                                                NaN   \n",
       "2              https://doi.org/10.3390/ijms241512380   \n",
       "3  /articles/instance/10418516/bin/ijms-24-12380-...   \n",
       "4                                                NaN   \n",
       "\n",
       "                  a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0                          NaN                   NaN           NaN   \n",
       "1                          NaN                   NaN           NaN   \n",
       "2  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       "3                     usa-link      click_feat_suppl           NaN   \n",
       "4                          NaN                   NaN           NaN   \n",
       "\n",
       "            a_attr_rel context_description  \\\n",
       "0                  NaN                 NaN   \n",
       "1                  NaN                 NaN   \n",
       "2  noopener noreferrer                       \n",
       "3                  NaN                       \n",
       "4                  NaN                 NaN   \n",
       "\n",
       "                                          source_url  \\\n",
       "0  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "1  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "2  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "3  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "4  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "\n",
       "                                           pub_title raw_data_format  \n",
       "0  Regulatory Effects of ABA and GA on the Expres...            HTML  \n",
       "1  Regulatory Effects of ABA and GA on the Expres...            HTML  \n",
       "2  Regulatory Effects of ABA and GA on the Expres...            HTML  \n",
       "3  Regulatory Effects of ABA and GA on the Expres...            HTML  \n",
       "4  BRCA2 deficiency instigates cGAS-mediated infl...            HTML  \n",
       "\n",
       "[5 rows x 22 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#combined_df = pd.read_csv(\"scripts/output/result.csv\")\n",
    "combined_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "305d7d5f299551c1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.136808Z",
     "start_time": "2025-09-04T21:20:38.134052Z"
    }
   },
   "outputs": [],
   "source": [
    "#ground_truth = pd.read_parquet(\"scripts/exp_input/dataset_citation_records_Table.parquet\")\n",
    "ground_truth = pd.read_parquet(\"scripts/output/gold/dataset_citation_records_Table.parquet\")\n",
    "#ground_truth = df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "dadaf8654a9d103d",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.857028Z",
     "start_time": "2025-09-04T21:20:38.851074Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>repository</th>\n",
       "      <th>citing_publication_link</th>\n",
       "      <th>citation_record_source</th>\n",
       "      <th>citation_record_from_doi</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PXD059466</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1038/S41467-025-56720-1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.6019/PXD051312</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.6019/PXD051312</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1002/prca.202400095</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1002/prca.202400095</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>0</td>\n",
       "      <td></td>\n",
       "      <td>PMC11895760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PXD054431</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.17159/SAJS.2025/18571</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.17159/SAJS.2025/18571</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  identifier repository                            citing_publication_link  \\\n",
       "0  PXD059466      PRIDE      https://dx.doi.org/10.1038/S41467-025-56720-1   \n",
       "1  PXD051312      PRIDE               https://dx.doi.org/10.6019/PXD051312   \n",
       "2  PXD051312      PRIDE          https://dx.doi.org/10.1002/prca.202400095   \n",
       "3  PXD051312      PRIDE  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "4  PXD054431      PRIDE        https://dx.doi.org/10.17159/SAJS.2025/18571   \n",
       "\n",
       "       citation_record_source  citation_record_from_doi  \\\n",
       "0  proteomexchange_search.tsv                         1   \n",
       "1  proteomexchange_search.tsv                         1   \n",
       "2  proteomexchange_search.tsv                         1   \n",
       "3  proteomexchange_search.tsv                         0   \n",
       "4  proteomexchange_search.tsv                         1   \n",
       "\n",
       "                          doi        pmcid  \n",
       "0  10.1038/S41467-025-56720-1               \n",
       "1           10.6019/PXD051312               \n",
       "2      10.1002/prca.202400095               \n",
       "3                              PMC11895760  \n",
       "4    10.17159/SAJS.2025/18571               "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ground_truth.head()\n",
    "# rename colum synid to 'identifier'\n",
    "#ground_truth = ground_truth.rename(columns={'synid': 'identifier'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "583f880224783dc7",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:52.754959Z",
     "start_time": "2025-09-04T21:26:52.722141Z"
    }
   },
   "outputs": [],
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file, false_negatives_file=None):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output, false_negatives_output = [], [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        #pub_id = source_page.split('/')[-1].lower()\n",
    "        pub_id = source_page\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        #gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "        gt_data = ground_truth[ground_truth['citing_publication_link'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend([false_p, pub_id] for false_p in FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "        false_negatives_output.extend((FN, pub_id)) if len(FN) > 0 else None\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    if false_negatives_file:\n",
    "        with open(false_negatives_file, 'w') as f:\n",
    "            for item in false_negatives_output:\n",
    "                f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "d5bd9405ee5dcd60",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:53.669418Z",
     "start_time": "2025-09-04T21:26:53.666829Z"
    }
   },
   "outputs": [],
   "source": [
    "#results = evaluate_performance(out, ground_truth, dg, \"output/false_positives.txt\")\n",
    "# PMC4318527, "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "3ae367545bb1b42b",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:55.530950Z",
     "start_time": "2025-09-04T21:26:54.622805Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'pxd044299', 'gse207091'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse207091', 'pxd044299'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10418516: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'pxd007253', 'gse116943'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'prjna299537', 'pxd007253', 'gse116943'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6327059: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 3. Element IDs: {'gse200161', 'pxd042977', 'pxd033812'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd042977', 'hra003258', 'gse200161', '8026174', 'pxd033812'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415: 0.6\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359415: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'gse157798'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse157798'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041347: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'gse130187', 'pxd012548'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse130187', 'pxd012548'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601207: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'gse148483', 'pxd018367'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse148483', 'pxd018367'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7327418: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'gse235490'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse235490'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10664992: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'gse264252'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse264252'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11130334: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'gse105030', 'pxd006185'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse105030'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659845: 0.5\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd003324'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd003324'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013304: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'gse203508'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse203508'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9213489: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'pxd037073', 'gse214150'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd037073', 'gse214150'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10442425: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd047910'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd047910'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10967333: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd042729'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd042729'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10454042: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd012357'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd012357'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823857: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd019169'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd019169', 'pharmgkb', 'v1.0.2', 'clinvar', 'gtex'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398: 0.2\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7418398: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 3. Element IDs: {'gse172124', 'gse162215', 'pxd022747'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse162215', 'gse172124', 'pxd022747'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213729: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'gse87759', 'pxd005663'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'/articles/instance/5339665/bin/image2.pdf', '/articles/instance/5339665/bin/table4.xlsx', '/articles/instance/5339665/bin/table1.docx', '/articles/instance/5339665/bin/table2.xlsx', '/articles/instance/5339665/bin/table3.xlsx', '/articles/instance/5339665/bin/image1.pdf'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665: 0.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5339665: 0.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 1. Element IDs: {'pxd006192'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'pxd006192'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940180: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 11 - INFO - Evaluating pub_id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629\u001b[0m\n",
      "\u001b[97m3090732644.py - line 19 - INFO - # of elements in gt_data: 2. Element IDs: {'gse125101', 'pxd012405'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 30 - INFO - Evaluation datasets: {'gse125101', 'pxd012405'}\u001b[0m\n",
      "\u001b[97m3090732644.py - line 72 - INFO - Precision for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 73 - INFO - Recall for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976629: 1.0\u001b[0m\n",
      "\u001b[97m3090732644.py - line 91 - INFO - \n",
      "Performance evaluation completed for 20 source pages.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "results = evaluate_performance(combined_df, ground_truth, dg, \"scripts/output/false_positives.txt\", \"scripts/output/false_negatives.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "fff3dc3075cb13b5",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:25:01.310798Z",
     "start_time": "2025-09-04T21:25:01.304645Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.8733333333333333,\n",
       " 'average_recall': 0.925,\n",
       " 'f1_score': 0.8984244670991658}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "37668da6",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "torcharm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
